<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<!-- saved from url=(0070)file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html -->
<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1"> <title></title> 
 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<!-- html,htex4ht,xhtml --> 
<meta name="src" content="RoyC_Umass_Thesis.tex"> 
<meta name="date" content="2014-04-29 21:15:00"> 
<link rel="stylesheet" type="text/css" href="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis.css"> 
</head><body>
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 4--><p class="noindent">
                                                                                

                                                                                
</p>
<div class="center">
<!--l. 5--><p class="noindent">
</p><!--l. 6--><p class="noindent"><span class="phvr8t-x-x-207">Putting the Pieces Together:</span><br>
<span class="phvr8t-x-x-207">Exons and piRNAs</span><br><br>
<span class="phvr8t-x-x-144">A Dissertation Presented</span><br><br>
<span class="phvr8t-x-x-144">By</span><br><br>
<span class="phvr8t-x-x-144">Christian K. Roy</span><br><br>
<span class="phvr8t-x-x-144">Submitted to the Faculty of the University of the</span><br>
<span class="phvr8t-x-x-144">Massachusetts Graduate School of Biomedical Sciences,</span>
<span class="phvr8t-x-x-144">Worcester</span><br>
<span class="phvr8t-x-x-144">in partial filfillment of the requirements</span><br>
<span class="phvr8t-x-x-144">for the degree of</span><br><br>
<span class="phvr8t-x-x-144">DOCTOR OF PHILOSOPHY</span><br><br>
<span class="phvr8t-x-x-144">MAY 21st 2014</span><br><br>
<span class="phvr8t-x-x-144">BIOCHEMISTRY</span><br><br>
</p>
                                                                                

                                                                                
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 4--><p class="noindent">
                                                                                

                                                                                
<a id="x1-1doc"></a>
</p>
<div class="center">
<!--l. 5--><p class="noindent">
</p><!--l. 6--><p class="noindent">Putting the Pieces Together:<br>
Exons and piRNAs<br>
A Dissertation Presented<br>
By<br>
Christian K. Roy<br>
The signatures of the Dissertation Defense Committee signify<br>
completion and approval as to style and content of the Dissertation<br><br>
_________________________________________________________
                                                         <br>Melissa J. Moore, Co-Thesis Advisor<br><br>
_________________________________________________________
                                                         <br>Phillip D. Zamore, Co-Thesis Advisor<br><br>
_________________________________________________________
                                                         <br>Scot Wolfe, Member of Committee<br><br>
_________________________________________________________
                                                         <br>Job Dekker, Member of Committee<br>
The signature of the Chair of the Committee signifies that the written
dissertation meets the requirements of the Dissertation Committee<br><br>
_________________________________________________________
                                                         <br>Zhiping Weng, Chair of Committee<br>
The signature of the Dean of the Graduate School of Biomedical Sciences
signifies that the student has met all graduation requirements of the school.<br><br>
_________________________________________________________
                                                         <br>Anthony Carruthers, Ph.D.,<br>
Dean of the Graduate School of Biomedical Sciences<br>
Biochemistry and Molecular Pharmacology<br>
                                                                                

                                                                                
MAY 21st 2014
</p>
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<a id="x1-2r1"></a>
<a id="Q1-1-1"></a>
<div class="center">
<!--l. 3--><p class="noindent">
</p><!--l. 3--><p class="noindent"><span class="phvro8t-x-x-207">Acknowledgements</span>  </p></div>
<!--l. 5--><p class="noindent">First I would like to thank Laura Geagan and Rebecca Sendak. They were my
supervisors while a research associate at Genzyme. Without the confidence they
instilled in my abilities as a young researcher I doubt I would have ever signed up for
more school.
</p><!--l. 7--><p class="noindent">During my 1st year retreat at Wood’s Hole I first observed Melissa’s fantastic ability as
a communicator of interesting and important Science. She brought out her rope
representing unspliced dystrophin—a rope that reached to the back of a rather large
auditorium—and then dramatically held up a “No.2” pencil representing the final
mRNA product, both to scale, I knew the that I wanted join her lab. Thanks Melissa for
teaching me so much.
</p><!--l. 9--><p class="noindent">Soon after joining Melissa’s lab, with a project going well, it was proposed that
I be a joint student between Melissa and Phil. It was not difficult to jump
at the opportunity to be advised by <span class="phvro8t-x-x-120">two </span>Hughes Investigators. I have been
continually amazed at the depth of Phil’s knowledge in scientific and general
topics. He is a careful, meticulous, quantitative, and calculating mentor. Phil
forces me to think and act outside my comfort zone, something I always tell
myself is critical for change and growth. Thank you Phil for everything I’ve
learned.
</p><!--l. 11--><p class="noindent">My committee has also been very supportive throughput my PhD. I hardly believed
the ease with which I passed my qualifying exam and figured there must have been
some mistake. Yet TRAC meetings confirmed that I was not thinking off-base.
One-on-one meetings just prior to my QE were especially helpful. Thanks to Scot,
Job, and Zhiping for all your guidance.
</p><!--l. 13--><p class="noindent">Thanks to all members of the Za(Moore) labs, past and present. Specifically I would
like to thank other members of my year in the Moore Lab, Eric and Erin. Thanks to
fantastic past post-docs Amrit, Alper, and Aaron. Also thanks again to Alper and to Bo
and Can for my novice bioinformatic questions.
</p><!--l. 15--><p class="noindent">Thanks to the faculty I have interacted with throughout my time at Umass, including
Dave Weaver, Brent Graveley, Andres Muro, Manuel Garber, and Heinrich Gottlinger.
And thanks to Anna from the Gottlinger lab for involving me in her HIV and hUPF1
story.
                                                                                

                                                                                
</p><!--l. 17--><p class="noindent">My project was pushed forward by a pair of great rotation students: Ben Landry and
Ogooluwa (“Ogo”) Adegoke Ojelabi . Thanks you two for letting me know SeqZip was
robust beyond my own hands.
</p><!--l. 19--><p class="noindent">My family always instilled the importance and value of education. Thank you Mom,
Dad, Grandpa and Grandma for sacrificing to provide me so much opportunity. I hope
I have made you proud.
</p><!--l. 21--><p class="noindent">Finally, and most importantly, I would like to acknowledge the love and support of my
wife, Julie. Much of my success can be attributed to her hard work and support over
the last 15 years. Our son, Owen, is lucky to have such an amazing mom. Our family
is truly blessed. Thanks P :)
                                                                                

                                                                                
</p><div class="quote">
<!--l. 8--><p class="noindent"><span class="phvro8t-x-x-144">For the Grump</span><span class="phvro8t-x-x-144">…I did it.</span> </p></div>
                                                                                

                                                                                
<a id="x1-3r2"></a>
<a id="Q1-1-2"></a>
<div class="center">
<!--l. 4--><p class="noindent">
</p><!--l. 4--><p class="noindent">UNIVERSITY OF MASSACHUSETTS MEDICAL SCHOOL
</p><!--l. 4--><p class="noindent"><span class="phvro8t-x-x-248">Abstract</span>
</p><!--l. 4--><p class="noindent">BIOCHEMISTRY AND MOLECULAR PHARMACOLOGY
</p><!--l. 4--><p class="noindent">Doctor of Philosophy
</p><!--l. 4--><p class="noindent"><span class="phvb8t-x-x-120">Putting the Pieces Together:</span><br>
<span class="phvb8t-x-x-120">Exons and piRNAs</span>
</p><!--l. 4--><p class="noindent">by Christian K. Roy
</p>
</div>
                                                                                

                                                                                
<!--l. 6--><p class="noindent">Analysis of gene expression has undergone a technological revolution. What was
impossible 6 years ago is now routine and high-throughput DNA sequencing
machines capable of generating hundreds of millions of reads allow, indeed force, a
major revision toward the study of the genome’s functional output—the transcriptome.
This thesis examines the history of DNA sequencing, measurement of gene
expression by sequencing, isoform complexity driven by alternative splicing and
mammalian piRNA precursor biogenesis. Examination of these topics is framed
around development of a novel RNA-templated DNA-DNA ligation assay (SeqZip)
that allows for efficient analysis of abundant, complex, and functional long RNAs. The
discussion focuses on the future of transcriptome analysis, development and
applications of SeqZip, and challenges presented to biomedical researchers by
extremely large and rich datasets.
                                                                                

                                                                                
                                                                                

                                                                                
</p>
<h2 class="likechapterHead"><a id="x1-10002"></a>Contents</h2> <div class="tableofcontents">
<span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-1">Acknowledgements</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-2">Abstract</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-4">List of Figures</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-6">List of Tables</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-8">Abbreviations and Symbols</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-9">Definitions</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-10">Preface</a></span>
<br><span class="chapterToc">1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-40001" id="QQ2-1-11">Introduction</a></span>
<br>&nbsp;<span class="sectionToc">1.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001" id="QQ2-1-12">Fixed Genomes and Flexible Genes</a></span>
<br>&nbsp;<span class="sectionToc">1.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-60002" id="QQ2-1-14">Nucleic Acid Sequencing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.2.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-70001" id="QQ2-1-15">DNA Sequencing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.2.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-80002" id="QQ2-1-16">High-throughput Sequencing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.2.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-90003" id="QQ2-1-18">RNA Sequencing</a></span>
<br>&nbsp;<span class="sectionToc">1.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-100003" id="QQ2-1-20">Nucleic Acid Splicing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-110001" id="QQ2-1-21">Alternative Splicing</a></span>
                                                                                

                                                                                
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-120002" id="QQ2-1-23">A Splicing Code</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-130003" id="QQ2-1-24">The Isoform Problem</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-140004" id="QQ2-1-26">Coordinated Splicing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-150005" id="QQ2-1-28">One Gene. Many Isoforms</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.3.6 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-160006" id="QQ2-1-32"><span class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></a></span>
<br>&nbsp;<span class="sectionToc">1.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-170004" id="QQ2-1-35">Nucleic Acid Ligation</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-180001" id="QQ2-1-36">RNA-templated DNA-DNA ligation</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-190002" id="QQ2-1-38">T4 RNA Ligase 2</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.4.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-200003" id="QQ2-1-41">Ligases as molecular tools</a></span>
<br>&nbsp;<span class="sectionToc">1.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-210005" id="QQ2-1-42">Nucleic Acid Polymers</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.5.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-220001" id="QQ2-1-43">It Started Small: Mammalian piRNAs</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.5.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-230002" id="QQ2-1-46">From Long to Small: Precursor processing to mature piRNAs</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">1.5.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-240003" id="QQ2-1-48">From Short to Long: Transcript Assembly</a></span>
<br>&nbsp;<span class="sectionToc">1.6 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-250006" id="QQ2-1-49">Nucleic Acid ’Omics</a></span>
<br><span class="chapterToc">2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002" id="QQ2-1-50">Simultaneous analysis of multiple site alternative splicing via
RNA-templated DNA-to-DNA ligation</a></span>
<br>&nbsp;<span class="sectionToc">2.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-270001" id="QQ2-1-51">Introduction</a></span>
<br>&nbsp;<span class="sectionToc">2.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-280002" id="QQ2-1-53">Results</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-290001" id="QQ2-1-54">Method development and validation</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">2.2.1.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-300001" id="QQ2-1-56">Caption for Figure 2.2</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-310002" id="QQ2-1-57">Trans-transcript hybridization and ligation is minimal</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-320003" id="QQ2-1-58">Reverse transcriptase-based detection versus SeqZip</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">2.2.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-330001" id="QQ2-1-60">Figure 2.3 Caption</a></span>
                                                                                

                                                                                
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-340004" id="QQ2-1-61">SeqZip maintains connectivity and abundance between
many sites of alternative splicing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-350005" id="QQ2-1-62"><span class="phvro8t-x-x-120">Dscam1 </span>analysis by high-throughput sequencing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.2.6 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-360006" id="QQ2-1-67">Analysis of <span class="phvro8t-x-x-120">Dscam1 </span>transcripts</a></span>
<br>&nbsp;<span class="sectionToc">2.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-370003" id="QQ2-1-71">Discussion</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-380001" id="QQ2-1-72">Deconvoluting <span class="phvro8t-x-x-120">Dscam1</span></a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">2.3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-390002" id="QQ2-1-73">Other applications of SeqZip</a></span>
<br>&nbsp;<span class="sectionToc">2.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004" id="QQ2-1-74">Materials and Methods</a></span>
<br>&nbsp;<span class="sectionToc">2.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-410005" id="QQ2-1-75">End Matter</a></span>
<br><span class="chapterToc">3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003" id="QQ2-1-76">An Ancient Transcription Factor Initiates the Burst of piRNA
Production during Early Meiosis in Mouse Testes</a></span>
<br>&nbsp;<span class="sectionToc">3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-430001" id="QQ2-1-77">Preface</a></span>
<br>&nbsp;<span class="sectionToc">3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-440002" id="QQ2-1-78">Introduction</a></span>
<br>&nbsp;<span class="sectionToc">3.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-450003" id="QQ2-1-79">Results</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-460001" id="QQ2-1-80">Defining piRNA-Producing Transcripts in the Mouse Testis</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.1.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-470001" id="QQ2-1-83">Caption for Figure 3.2</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-480002" id="QQ2-1-84">piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-490003" id="QQ2-1-85">A Transcript-based Set of piRNA Loci</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-500004" id="QQ2-1-86">Three Classes of piRNAs During Post-Natal Spermatogenesis</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-510001" id="QQ2-1-91">Caption for Figure 3.3</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-520002" id="QQ2-1-92">Caption for Figure 3.4</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-530005" id="QQ2-1-93"><span class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.6 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-540006" id="QQ2-1-96"><span class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.7 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-550007" id="QQ2-1-99"><span class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</a></span>
                                                                                

                                                                                
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.8 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-560008" id="QQ2-1-102">A-MYB and the Pachytene piRNA Regulatory Circuitry</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.8.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-570001" id="QQ2-1-105">Caption for Figure 3.13</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.8.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-580002" id="QQ2-1-106">Caption for Figure 3.14</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">3.3.9 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-590009" id="QQ2-1-107">Feed-Forward Regulation of piRNA Production is Evolutionarily
Conserved</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">3.3.9.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-600001" id="QQ2-1-110">Caption for figure 3.15</a></span>
<br>&nbsp;<span class="sectionToc">3.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-610004" id="QQ2-1-111">Discussion</a></span>
<br>&nbsp;<span class="sectionToc">3.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-620005" id="QQ2-1-112">Experimental Procedures</a></span>
<br><span class="chapterToc">4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004" id="QQ2-1-113">SeqZip - Development and Applications</a></span>
<br>&nbsp;<span class="sectionToc">4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-640001" id="QQ2-1-114">Overview</a></span>
<br>&nbsp;<span class="sectionToc">4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002" id="QQ2-1-116">Multiplex SeqZip Application</a></span>
<br>&nbsp;<span class="sectionToc">4.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-660003" id="QQ2-1-120">RNA Integrity</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-670001" id="QQ2-1-121">Demonstration of Concept</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-680002" id="QQ2-1-124">HIV Genome Integrity</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.3.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-690003" id="QQ2-1-125">HIV Ligamer Design</a></span>
<br>&nbsp;<span class="sectionToc">4.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-700004" id="QQ2-1-127">piRNA Precursors</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-710001" id="QQ2-1-128">Mammalian piRNA Precursor Loci</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-720002" id="QQ2-1-131">Pachytene Precursors are Unique Pol II Transcripts</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.4.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-730003" id="QQ2-1-134">Connectivity of Distance Intramolecular Sequences</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">4.4.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-740004" id="QQ2-1-137">Precursor Transcript Continuity</a></span>
<br>&nbsp;<span class="sectionToc">4.5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-750005" id="QQ2-1-141">Precursor Splicing</a></span>
<br><span class="chapterToc">5 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-760005" id="QQ2-1-146">Discussion</a></span>
<br>&nbsp;<span class="sectionToc">5.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-770001" id="QQ2-1-147">Concerning the Transcriptome</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.1.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-780001" id="QQ2-1-148">Pervasive transcription</a></span>
                                                                                

                                                                                
<br>&nbsp;&nbsp;<span class="subsectionToc">5.1.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-790002" id="QQ2-1-149">A Need for Transcript Assembly</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.1.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-800003" id="QQ2-1-150">Tissue and Cell Specificity</a></span>
<br>&nbsp;<span class="sectionToc">5.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-810002" id="QQ2-1-151">In the haystack: piRNA Precursors</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.2.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-820001" id="QQ2-1-152">Precursor Identity</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.2.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-830002" id="QQ2-1-154">Precursor Interactions</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.2.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-840003" id="QQ2-1-155">Precursor Location</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.2.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-850004" id="QQ2-1-156">Precursor Sequencing</a></span>
<br>&nbsp;<span class="sectionToc">5.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-860003" id="QQ2-1-157">Future of RNA-templated DNA-DNA ligation</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.3.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-870001" id="QQ2-1-158">An Optimized SeqZip Examination of Coordinated Splicing</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.3.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-880002" id="QQ2-1-159">LNA-containing ligamers and T39A Rnl2</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.3.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-890003" id="QQ2-1-161">Thermostable Ligases</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.3.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-900004" id="QQ2-1-162">Other SeqZip Applications</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">5.3.4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-910001" id="QQ2-1-164">Multi-site SNP detection</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">5.3.4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-920002" id="QQ2-1-165">SeqZip and single-molecule multi-site FISH</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">5.3.4.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-930003" id="QQ2-1-167">Introduction of destruction sequences</a></span>
<br>&nbsp;&nbsp;&nbsp;<span class="subsubsectionToc">5.3.4.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-940004" id="QQ2-1-168">Re-purposing the SOLiD Platform</a></span>
<br>&nbsp;<span class="sectionToc">5.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-950004" id="QQ2-1-169">Final Thoughts</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.4.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-960001" id="QQ2-1-170">Science vs. Engineering</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.4.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-970002" id="QQ2-1-171">The Data Deluge</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">5.4.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-980003" id="QQ2-1-172">Biologists need Computation Biological Skills</a></span>
<br><span class="appendixToc">A <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-99000A" id="QQ2-1-174">Appendix A: Misc Information</a></span>
<br>&nbsp;<span class="sectionToc">A.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1000001" id="QQ2-1-175">Buffers</a></span>
<br>&nbsp;<span class="sectionToc">A.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1010002" id="QQ2-1-177">Equations</a></span>
                                                                                

                                                                                
<br>&nbsp;&nbsp;<span class="subsectionToc">A.2.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1020001" id="QQ2-1-178">RNA concentration from radioactive in vitro transcription</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">A.2.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1030002" id="QQ2-1-179">RNA concentration based on absorbance</a></span>
<br>&nbsp;&nbsp;<span class="subsectionToc">A.2.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1040003" id="QQ2-1-180">Normalize oxidized small RNA libraries size to time-matched
unoxidized library</a></span>
<br>&nbsp;<span class="sectionToc">A.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1050003" id="QQ2-1-181">PCR Programs</a></span>
<br>&nbsp;<span class="sectionToc">A.4 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1060004" id="QQ2-1-182">Intergenic and genic piRNA-producing Loci</a></span>
<br><span class="appendixToc">B <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-107000B" id="QQ2-1-184">Appendix B: Automated Ligamer Assembly</a></span>
<br>&nbsp;<span class="sectionToc">B.1 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1080001" id="QQ2-1-185">Installation</a></span>
<br>&nbsp;<span class="sectionToc">B.2 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1090002" id="QQ2-1-186">Example Input Format</a></span>
<br>&nbsp;<span class="sectionToc">B.3 <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-1100003" id="QQ2-1-187">Ligamer Assembler Source Code</a></span>
<br><span class="chapterToc"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#Q1-1-189">Bibliography</a></span>
</div><a id="x1-1001r3"></a><a id="Q1-1-4"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a id="x1-20003"></a>List of Figures</h2> <div class="tableofcontents"><span class="lofToc">1.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-5001r1">The Solitary and Gregarious forms of <span class="phvro8t-x-x-120">Schistocerca</span>
<span class="phvro8t-x-x-120">gregaria</span></a></span><br><span class="lofToc">1.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-8001r2">Cost of sequencing the human genome over time</a></span><br><span class="lofToc">1.3&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-9001r3">Methods
for High-throughput sequencing of RNA</a></span><br><span class="lofToc">1.4&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-11001r4">Estimates of number of
human genes, and percentage alternatively spliced over time</a></span><br><span class="lofToc">1.5&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-13001r5">HTS
read lengths are not sufficient to maintain alternative splicing
connectivity</a></span><br><span class="lofToc">1.6&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-14001r6">Mouse <span class="phvro8t-x-x-120">Fn1 </span>contains multiple sites of Alternative
Splicing</a></span><br><span class="lofToc">1.7&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15001r7">Number of hg19 Alternative Event types</a></span><br><span class="lofToc">1.8&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15002r8">Number of
transcripts per <span class="phvro8t-x-x-120">Drosophila melanogaster </span>gene</a></span><br><span class="lofToc">1.9&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16001r9">The architecture
of the <span class="phvro8t-x-x-120">Drosophila melanogaster </span>gene <span class="phvro8t-x-x-120">Dscam1</span></a></span><br><span class="lofToc">1.10&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16002r10">Important sites
of <span class="phvro8t-x-x-120">Dscam1 </span>expression in <span class="phvro8t-x-x-120">Drosophila melanogaster</span></a></span><br><span class="lofToc">1.11&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-18001r11">Mechanism
of Rnl2 ATP-dependent ligation</a></span><br><span class="lofToc">1.12&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-19001r12">Structure and active site of
pre-adenylated Rnl2</a></span><br><span class="lofToc">1.13&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-19002r13">Active site of T4 RNA Ligase 2 with highlighted
residues</a></span><br><span class="lofToc">1.14&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-22001r14">Different Classes of mammalian piRNAs</a></span><br><span class="lofToc">1.15&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-22002r15">Genetic
evidence for long, continuous fly piRNA precursor transcripts</a></span><br><span class="lofToc">1.16&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-23001r16">A
model for Mammalian piRNA biogenesis</a></span><br><span class="lofToc">2.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-27001r1">Principles of the SeqZip
Assay</a></span><br><span class="lofToc">2.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">T4 RNA Ligase 2 will catalyze RNA-templated DNA-to-DNA
ligation</a></span><br><span class="lofToc">2.3&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">SeqZip on endogenously expressed RNAs</a></span><br><span class="lofToc">2.4&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">Analysis of
<span class="phvro8t-x-x-120">Dscam1 </span>isoforms via High-Throughput Sequencing</a></span><br><span class="lofToc">2.5&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">HTS Analysis
of <span class="phvro8t-x-x-120">Dscam1 in vitro </span>transcripts</a></span><br><span class="lofToc">2.6&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35003r6">SeqZip <span class="phvro8t-x-x-120">in vitro Dscam1 </span>controls and
near-cognate ligation</a></span><br><span class="lofToc">2.7&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35004r7">Cognate and Nearest-Near cognate folding
energies for <span class="phvro8t-x-x-120">Dscam1 </span>exon 6</a></span><br><span class="lofToc">2.8&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36001r8">Roy et al 2014 Figure S4</a></span><br><span class="lofToc">2.9&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36002r9"><span class="phvro8t-x-x-120">Dscam1</span>
individual exon usage measured by SeqZip</a></span><br><span class="lofToc">2.10&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36003r10">Comparison
between SeqZip and CAMSeq analysis of <span class="phvro8t-x-x-120">Dscam1 </span>isoforms in S2
cells</a></span><br><span class="lofToc">3.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46001r1">piRNA Precursors are RNA Pol II Transcripts</a></span><br><span class="lofToc">3.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">The Major
piRNA-Producing Genes of the Post-Partum Mouse Testis</a></span><br><span class="lofToc">3.3&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">Three
Classes of piRNA-Generating Loci</a></span><br><span class="lofToc">3.4&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">Pre-pachytene piRNAs Persist
in Pachytene Spermatocytes</a></span><br><span class="lofToc">3.5&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50003r5">Examples of Pachytene piRNA
Genes</a></span><br><span class="lofToc">3.6&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50004r6">Examples of Pre-Pachytene piRNA Genes</a></span><br><span class="lofToc">3.7&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">A-MYB Binds
the Promoters of Pachytene piRNA Genes</a></span><br><span class="lofToc">3.8&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53002r8">ChIP-qPCR Confirms
ChIP-seq Data</a></span><br><span class="lofToc">3.9&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">Pachytene piRNAs and Precursors Decrease in <span class="phvro8t-x-x-120">A-Myb</span>
Mutant Testes</a></span><br><span class="lofToc">3.10&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54002r10">Change in piRNA Expression in <span class="phvro8t-x-x-120">Spo11</span>, <span class="phvro8t-x-x-120">Miwi</span>, <span class="phvro8t-x-x-120">Tdrd6</span>,
                                                                                

                                                                                
and <span class="phvro8t-x-x-120">Tdrd9 </span>Mutants</a></span><br><span class="lofToc">3.11&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-55001r11">Examples of the Effect of the <span class="phvro8t-x-x-120">A-Myb </span>Mutation
on piRNA Expression</a></span><br><span class="lofToc">3.12&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-55002r12">Pachytene piRNA Precursor Abundance in
<span class="phvro8t-x-x-120">A-Myb</span>, <span class="phvro8t-x-x-120">Miwi</span>, and <span class="phvro8t-x-x-120">Trip13 </span>Mutants</a></span><br><span class="lofToc">3.13&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">A-MYB Regulates Expression
of mRNAs Encoding piRNA Pathway Proteins</a></span><br><span class="lofToc">3.14&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56002r14"><span class="phvro8t-x-x-120">A-Myb </span>mutants, but
Not <span class="phvro8t-x-x-120">Miwi </span>Mutants, Change the Expression of RNA Silencing Pathway
Genes</a></span><br><span class="lofToc">3.15&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">Feed-Forward Regulation of piRNA Biogenesis by A-MYB
is Conserved in Rooster</a></span><br><span class="lofToc">3.16&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59002r16">Genomic Locations of piRNA Clusters in
the Rooster (Gallus gallus) Testis.</a></span><br><span class="lofToc">4.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-64001r1">Original SeqZip Diagram</a></span><br><span class="lofToc">4.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-65002r2">10
Gene Set study schematic</a></span><br><span class="lofToc">4.3&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-65003r3">Measuring <span class="phvro8t-x-x-120">Apbb1 </span>via SeqZip in
multiplex study</a></span><br><span class="lofToc">4.4&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67001r4">Ligation product tied to RNA integrity</a></span><br><span class="lofToc">4.5&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67002r5">Trans
Transcript investigation</a></span><br><span class="lofToc">4.6&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-69001r6">SeqZip can examine HIV transcript
integrity</a></span><br><span class="lofToc">4.7&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-71001r7">Pachytene piRNA precursor locations in mice</a></span><br><span class="lofToc">4.8&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72001r8">Some
general features of piRNA transcripts</a></span><br><span class="lofToc">4.9&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72002r9">pRT Doesn’t Work for piRNA
precursors</a></span><br><span class="lofToc">4.10&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-73001r10">SeqZip on a very long mRNA (<span class="phvro8t-x-x-120">Dst1</span>)</a></span><br><span class="lofToc">4.11&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-73002r11">Three
sites of alternative splicing in <span class="phvro8t-x-x-120">Fn1 </span>by SeqZip</a></span><br><span class="lofToc">4.12&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74001r12">Testes-specific
ligation product signal from piRNA precursor</a></span><br><span class="lofToc">4.13&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74002r13">SeqZip signal from
piRNA-producing loci <span class="phvro8t-x-x-120">17-qA3.3-27363.1</span></a></span><br><span class="lofToc">4.14&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74003r14">SeqZip signal from
piRNA precursor transcripts decreases with loop length</a></span><br><span class="lofToc">4.15&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75001r15">Example
small RNA and RNA-Seq data aligned to a piRNA-generating
loci</a></span><br><span class="lofToc">4.16&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75002r16">Introns in mammalian piRNA precursors</a></span><br><span class="lofToc">4.17&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75003r17">piRNAs map
to precursor transcript splice junctions</a></span><br><span class="lofToc">4.18&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75004r18"><span class="phvro8t-x-x-120">A-Myb </span>Mutants produce
no splice-junction mapping piRNAs for genic piRNA-producing
loci</a></span><br><span class="lofToc">5.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-82001r1"><span class="phvro8t-x-x-120">wdfy3 </span>locus expresses both mRNA and piRNA precursor form in
testes</a></span><br><span class="lofToc">5.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-88001r2">Sugar pucker in Rnl2 structures</a></span><br><span class="lofToc">5.3&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-90001r3">Proposed uses of the SeqZip
methodology</a></span><br><span class="lofToc">5.4&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-92001r4">Multi-Site smFISH using flourophore-containing ligamers</a></span><br>
</div>
                                                                                

                                                                                
<a id="x1-2001r4"></a>
<a id="Q1-1-6"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a id="x1-30004"></a>List of Tables</h2> <div class="tableofcontents"><span class="lotToc">1.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15003r1">Fly genes with &gt;2,000 assembled transcripts
according to [Brown et&nbsp;al.,&nbsp;2014].</a></span><br><span class="lotToc">4.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-65001r1">Mouse genes with large
sequence between suggested coordinated cassette exons</a></span><br><span class="lotToc">4.2&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-71002r2">Just
9 piRNA genes create &gt;50% of mammalian piRNAs</a></span><br><span class="lotToc">5.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-98001r1">Changing
computational tools for Molecular Biologists</a></span><br><span class="lotToc">A.1&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-100001r1">SeqZip Hybridization
and Ligation Buffer</a></span><br><span class="lotToc">&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-106002">A.2
Intergenic and genic subclassifications of piRNA-producing loci</a></span><br>
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<a id="x1-3001r5"></a>
<a id="Q1-1-8"></a>
<div class="table">
                                                                                

                                                                                
<!--l. 5--><p class="noindent"></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="tabular"><table id="TBL-1" class="tabular" cellspacing="0" cellpadding="0" rules="groups"><colgroup id="TBL-1-1g"><col id="TBL-1-1"></colgroup><colgroup id="TBL-1-2g"><col id="TBL-1-2"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-1-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-1-1" class="td11">aka </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-1-2" class="td11">also known as                                                                                           </td></tr><tr style="vertical-align:baseline;" id="TBL-1-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-2-1" class="td11">ATP </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-2-2" class="td11">Adenosine triphosphate</td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-3-1" class="td11">AS             </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-3-2" class="td11">Alternative Splicing                                                                                   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-4-1" class="td11">bp             </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-4-2" class="td11">A base pair of DNA                                                                                   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-5-1" class="td11">CAP          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-5-2" class="td11">5<span class="cmsy-8">&#8242;</span>&nbsp; 7meG structure attached to mRNAs                                                   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-6-1" class="td11">ChIP-Seq  </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-6-2" class="td11">Chromatin Immunoprecipitation followed by sequencing                          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-7-1" class="td11">CNS          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-7-2" class="td11">Central Nervous System                                                                           </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-8-1" class="td11">DNA          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-8-2" class="td11">Deoxyribonucleic acid                                                                               </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-9-1" class="td11">dpp           </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-9-2" class="td11">days post partum                                                                                      </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-10-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-10-1" class="td11">EST          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-10-2" class="td11">Expressed Sequence Tag                                                                         </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-11-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-11-1" class="td11">FLLP         </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-11-2" class="td11">Full length ligation product                                                                        </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-12-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-12-1" class="td11">FISH         </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-12-2" class="td11">Fluorescence in situ hybridization                                                             </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-13-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-13-1" class="td11">FISSEQ    </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-13-2" class="td11">fluorescent in situ RNA sequencing                                                          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-14-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-14-1" class="td11">GFP          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-14-2" class="td11">Green Fluorescencent Protein                                                                  </td></tr><tr style="vertical-align:baseline;" id="TBL-1-15-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-15-1" class="td
11">HITS-CLIP</td><td style="white-space:nowrap; text-align:left;" id="TBL-1-15-2" class="td11">high-throughput sequencing after cross-linking and immunoprecipitation</td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-16-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-16-1" class="td11">hnRNP      </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-16-2" class="td11">heterogeneous nuclear ribonucleoprotein                                                 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-17-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-17-1" class="td11">HTS          </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-17-2" class="td11">High-throughput sequencing (see also NGS)                                           </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-18-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-18-1" class="td11">IAP            </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-18-2" class="td11">intra-cisternal A particle                                                                            </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-19-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-19-1" class="td11">IRE           </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-19-2" class="td11">Intron Recognition Element                                                                       </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-20-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-20-1" class="td11">kb              </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-20-2" class="td11">kilo-base of RNA (in nt)                                                                             </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-21-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-21-1" class="td11">kbp            </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-21-2" class="td11">kilo-base-pair of DNA (in nt)                                                                      </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-22-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-22-1" class="td11">LINE1 </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-22-2" class="td11">Long interspersed element 1                                                                    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-23-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-23-1" class="td11">LNA </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-23-2" class="td11">Locked Nucleic Acid                                                                                  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-24-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-24-1" class="td11">MPSS </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-24-2" class="td11">Massively Parallel Signature sequencing                                                  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-25-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-25-1" class="td11">NAD </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-25-2" class="td11">Nicotinamide adenine dinucleotide                                                           </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-26-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-26-1" class="td11">NGS </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-26-2" class="td11">Next-generation sequencing                                                                     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-27-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-27-1" class="td11">nt </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-27-2" class="td11">A nucleotide of either DNA or RNA                                                           </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-28-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-28-1" class="td11">Pol II </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-28-2" class="td11">Polymerase II, one of the main RNA polymerases                                   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-29-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-29-1" class="td11">RNA </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-29-2" class="td11">Ribonucleic acid                                                                                        </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-30-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-30-1" class="td11">SAGE </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-30-2" class="td11">Serial Analysis of Gene Expression                                                          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-31-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-31-1" class="td11">SNP </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-31-2" class="td11">Single Nucleotide Polymorphism                                                              </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-32-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-32-1" class="td11">SRE </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-32-2" class="td11">Splicing Regulatory Element                                                                     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-33-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-33-1" class="td11">ssDNA </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-33-2" class="td11">Single-stranded DNA                                                                                </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-34-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-34-1" class="td11">ssRNA </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-34-2" class="td11">Single-stranded RNA                                                                                </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-35-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-35-1" class="td11">TSS </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-35-2" class="td11">{Transcription or Translation} Start Site                                                     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-36-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-36-1" class="td11">TTS </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-36-2" class="td11">{Transcription or Translation} Termination Site                                          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-37-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-37-1" class="td11">5<span class="cmsy-8">&#8242;</span> </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-37-2" class="td11">The 5 prime end of a DNA or RNA molecule                                            </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-38-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-38-1" class="td11">3<span class="cmsy-8">&#8242;</span> </td><td style="white-space:nowrap; text-align:left;" id="TBL-1-38-2" class="td11">The 5 prime end of a DNA or RNA molecule                                            </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-39-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-39-1" class="td11"><span class="cmmi-12">&#956; </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-1-39-2" class="td11">Micro. A value of 1x10<sup class="textsuperscript"><span class="phvr8t-x-x-109">-6</span></sup> standard units                                                    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-40-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-40-1" class="td11"><span class="cmmi-12">r </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-1-40-2" class="td11">Pearson product-moment correlation coefficient                                       </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-41-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-41-1" class="td11"><span class="cmmi-12">&#961; </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-1-41-2" class="td11">Spearman’s rank correlation coefficient                                                    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-1-42-"><td style="white-space:nowrap; text-align:left;" id="TBL-1-42-1" class="td11"> </td></tr></tbody></table></div>
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
                                                                                

                                                                                
<a id="x1-3002r6"></a>
<a id="Q1-1-9"></a>
<dl class="description"><dt class="description">
<span class="phvb8t-x-x-120">RNA-Seq</span></dt><dd class="description">A technology wherein RNA is fragmented, converted to DNA, and
analyzed on a high-throughput sequencing instrument
</dd><dt class="description">
<span class="phvb8t-x-x-120">A “Read”</span></dt><dd class="description">The  sequence  of  nucleotides  produced  from  each  spot  on  a
high-throughput sequencing machine
</dd><dt class="description">
<span class="phvb8t-x-x-120">Insert</span></dt><dd class="description">The RNA  molecule  captured  between  two  cloning  sequences  in  a
high-throughput sequencing library preparation workflow
</dd><dt class="description">
<span class="phvb8t-x-x-120">Read length</span> </dt><dd class="description">The number of nucleotides for each given “read”
</dd><dt class="description">
<span class="phvb8t-x-x-120">Read depth</span></dt><dd class="description">The   number   of   reads   obtained   from   each   high-throughput
sequencing analysis
</dd><dt class="description">
<span class="phvb8t-x-x-120">Coverage</span></dt><dd class="description">A  measure  of  the  number  of  times  each  nt  of  a  genome  is
sequenced.  E.g.  100  million  reads  of  a  10  million  nt  genome  =  10X
coverage, assuming uniform distribution of the “reads”
</dd><dt class="description">
<span class="phvb8t-x-x-120">Paired-end</span></dt><dd class="description">When  both  sides  of  a  DNA  insert  or  template  are  sequenced,
utilizing the original length of DNA between the reads to facilitate mapping
(<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRoach1995">Roach et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRoach1995">1995</a>]).
</dd><dt class="description">
                                                                                

                                                                                
<span class="phvb8t-x-x-120">Scaffold or Contig</span> </dt><dd class="description">A  draft  sequence  of  nucleotides,  meant  to  represent
the actual   biological   sequence   as   closely   as   possible,   examples
include unassembled fragments of chromosomes or fragments of mRNA
transcripts.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Argonaute</span></dt><dd class="description">Protein(s)  belonging  to  a  group  containing  a  Piwi  (P-element
induced  wimpy  testes)  domain,  that  bind  nucleic  acids  and  participate
in many   target-guided   processes,   including   RNA   Interference,   and
RNA-induced transcript/gene silencing.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Ligamer</span></dt><dd class="description">A                    DNA                    oligonucleotide                    containing
two distinct regions of complementarity to a 5<span class="cmsy-8">&#8242;</span>&nbsp; and 3<span class="cmsy-8">&#8242;</span>&nbsp; section of RNA.
Each region is normalized for <sup><span class="cmmi-8">Tm</span></sup> such that the length of each section is
~15–30 nt. These two regions are connected by a short sequence of the
designer’s choice, usually &gt;5 nt in length. Each ligamers overall length is
~45–60 nt. See figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-64001r1">4.1<!--tex4ht:ref: SeqZipMethod:fig:Original SeqZip Diagram --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-27001r1">2.1<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F1 --></a>.
</dd></dl>
                                                                                

                                                                                
<a id="x1-3003r7"></a>
<a id="Q1-1-10"></a>
<div class="center">
<!--l. 4--><p class="noindent">
</p><!--l. 4--><p class="noindent"><span class="phvro8t-x-x-144">Preface</span>
</p>
</div>
<!--l. 6--><p class="noindent">The work reported in this dissertation has been published in the following
articles.
</p><!--l. 8--><p class="noindent">Some contents of Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> are included in a currently submitted manuscript.
</p><!--l. 10--><p class="noindent">Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> has been published previously as:
</p><div class="quote">
<!--l. 15--><p class="noindent"><a id="XLi2013h"></a><span class="phvro8t-x-x-120">Li, X.</span><span class="phvro8t-x-x-120">&nbsp;Z.,  Roy,  C.</span><span class="phvro8t-x-x-120">&nbsp;K.,  Dong,  X.,  Bolcun-Filas,  E.,  Wang,  J.,  Han,</span>
<span class="phvro8t-x-x-120">B.</span><span class="phvro8t-x-x-120">&nbsp;W., Xu, J., Moore, M.</span><span class="phvro8t-x-x-120">&nbsp;J., Schimenti, J.</span><span class="phvro8t-x-x-120">&nbsp;C., Weng, Z., and Zamore,</span>
<span class="phvro8t-x-x-120">P.</span><span class="phvro8t-x-x-120">&nbsp;D. (2013a).   An ancient transcription factor initiates the burst of</span>
<span class="phvro8t-x-x-120">piRNA production during early meiosis in mouse testes.  </span>Molecular
cell<span class="phvro8t-x-x-120">, 50(1):67–81</span></p></div>
                                                                                

                                                                                
                                                                                

                                                                                
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&nbsp;1</span><br><a id="x1-40001"></a>Introduction</h2>
<a id="x1-4001r1"></a>
<h3 class="sectionHead"><span class="titlemark">1.1 </span> <a id="x1-50001"></a>Fixed Genomes and Flexible Genes</h3>
<!--l. 9--><p class="noindent">Exodus tells of the liberation of the Israelites from Egyptian slavery. Humble and
reluctant Moses, their divine-appointed messiah, attempts to force the Pharaoh
Ramses to release the Israelites through inflicting 10 plagues. Pharaoh is stalwart
and stubborn as water turns to blood and the streets are flooded with frogs,
lice, and flies. As livestock falls dead from disease, people and animals both
are covered in boils, and the land burns in storms of fire, Pharaoh does not
bend.
</p><!--l. 11--><p class="noindent">The 8th plague was a swarm of Locusts, described in Exodus 10: 14-15:
</p><div class="quote">
<!--l. 16--><p class="noindent"><sup class="textsuperscript"><span class="phvro8t-x-x-109">14</span></sup><span class="phvro8t-x-x-120">And the locusts went up over all the land of Egypt, and rested in</span>
<span class="phvro8t-x-x-120">all the coasts of Egypt: very grievous were they; before them there</span>
<span class="phvro8t-x-x-120">were no such locusts as they, neither after them shall be such.</span><br class="newline"><br class="newline"><sup class="textsuperscript"><span class="phvro8t-x-x-109">15</span></sup><span class="phvro8t-x-x-120">For they covered the face of the whole earth, so that the land was</span>
<span class="phvro8t-x-x-120">darkened; and they did eat every herb of the land, and all the fruit of</span>
<span class="phvro8t-x-x-120">the trees which the hail had left: and there remained not any green</span>
<span class="phvro8t-x-x-120">thing in the trees, or in the herbs of the field, through all the land of</span>
<span class="phvro8t-x-x-120">Egypt.</span></p></div>
<!--l. 21--><p class="noindent">The desolation of a locust plague was still not enough to persuade Ramses. Nor was
                                                                                

                                                                                
three days of darkness. Only after the death of all first-born Egyptians, including
Ramses own son, was Pharaoh persuaded to liberate the Israelites.
</p><!--l. 23--><p class="noindent">Locust swarms are not biblical fantasy. Today the United Nations’ Food and
Agriculture division maintains a <a href="http://www.fao.org/ag/locusts/en/info/info/news/index.html">Locust watch website</a> that provides weekly updates
on potential swarms in northern Africa and the Middle East. Locusts have long been,
and continue to be, a powerful and feared force of Nature.
</p><!--l. 25--><p class="noindent">Unlike fire and brimstone, locusts are something that can be observed and studied.
What triggers a swam? We know that the desert locust, <span class="phvro8t-x-x-120">Schistocerca gregaria</span>, is the
one of 10 species that swarm and cause massive crop damage. <span class="phvro8t-x-x-120">Schistocerca</span>
<span class="phvro8t-x-x-120">gregaria </span>are in the insect Order Orthoptera, along with crickets and katydids.
Orthoptern members make sound known as <span class="phvro8t-x-x-120">stridulation </span>by vigorously rubbing their
wings, making for a noisy cloud of devastation. They only weigh 0.05–0.07 ounces
and are less than 2.5 inches long but can consume their own body weight in
vegetation per day. One <a href="http://animaldiversity.ummz.umich.edu/site/accounts/information/Melanoplus_spretus.html">swarm</a> of the infamous, and now curiously extinct, Rocky
Mountain locust contained 12.2 <span class="phvro8t-x-x-120">trillion </span>insects. Its estimated total weight was 27.5
<span class="phvro8t-x-x-120">million </span>tons. The swarm covered almost 200 square miles (2/3 the size of Manhattan),
and could travel 60 miles in a day. A locust swarm is truly a modern biblical
plague.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-5001r1"></a>
                                                                                

                                                                                

<!--l. 29--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/DesertLocust.jpeg" alt="PIC">
<br> </p><div class="caption"><span class="id">Figure&nbsp;1.1: </span><span class="content">The Solitary and Gregarious forms of <span class="phvro8t-x-x-120">Schistocerca gregaria</span><br class="newline"> The  two  phenotypic  forms  of  <span class="phvro8t-x-x-120">Schistocerca  gregaria  </span>appear  very  different.
The  solitary  form  is  green  and  generally  larger,  while  its  gregarious  form
is  more  brightly  colored,  smaller,  and  swarms  in  vast  numbers.  Photo  from
<a href="http://www.wikicommons.com/">Wikicommons</a>. </span></div><!--tex4ht:label?: x1-5001r1 -->
                                                                                

                                                                                
<!--l. 38--><p class="noindent">By definition swarms are temporary; the movement, en masse, from one location to
another. But where do 12.2 trillion locusts go when not swarming? Does anyone care
if their crops aren’t under assault? It seemed no one cared until 1921 when an
important realization was made.
</p><!--l. 40--><p class="noindent">The power and destruction <span class="phvro8t-x-x-120">Schistocerca gregaria </span>can inflict makes it difficult to
believe that they are nothing more than common grasshoppers. Nothing more than
grasshoppers not just by analogy, but by actual <span class="phvro8t-x-x-120">Taxonomy</span>. “Desert” locusts are
actually the <span class="phvro8t-x-x-120">gregarious </span>form of <span class="phvro8t-x-x-120">Schistocerca gregaria </span>(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-5001r1">1.1<!--tex4ht:ref: Intro:fig:Locust --></a>), while
the more familiar and docile looking “grasshopper” is the <span class="phvro8t-x-x-120">solitary form</span>. How
does such a dichotomy exist within the same organism—indeed the same
<span class="phvro8t-x-x-120">genome</span>?
</p><!--l. 42--><p class="noindent"><span class="phvro8t-x-x-120">Schistocerca gregaria </span>are <span class="phvro8t-x-x-120">polyphenic</span>, meaning that they have multiple (poly) physical
forms (phenotypes). Polyphenism is a general feature among insects. These
phenotypes are often extremely different. For example, pea aphids (<span class="phvro8t-x-x-120">Acyrthosiphon</span>
<span class="phvro8t-x-x-120">pisum</span>), which usually exist in an asexually reproducing, wingless female
form, respond to reduced food supply and overcrowding by producing winged
sexually-reproducing offspring. Winged organisms travel to new sources of food and
revert back to the asexually reproducing form [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPurandare2014b">Purandare et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPurandare2014b">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShingleton2003">Shingleton
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShingleton2003">2003</a>]. In the case of <span class="phvro8t-x-x-120">Schistocerca gregaria</span>, the gregarious form is smaller and
more brightly colored compared to its solitary cousins. This transformation
                                                                                

                                                                                
can happen in as little as two hours. What is the underlying cause of this
transformation?
</p><!--l. 44--><p class="noindent">In 2009, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnstey2009">Anstey et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnstey2009">2009</a>] reported that after two hours of forced crowding
<span class="phvro8t-x-x-120">Schistocerca gregaria </span>displayed elevated levels of the neurotransmitter serotonin in
the ganglia (brain). Serotonin levels were strongly correlated with other gregarious
form indicators. Serotonin regulates neuronal junctions and wiring in the brain
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHoeffer2003">Hoeffer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHoeffer2003">2003</a>]. Through the integration of environmental and social cues, the
grasshopper brain can be re-wired, resulting in tremendous changes in behavior and
phenotype. These changes prepare the organism to deal with a different world. It
allows the organism to survive. Survival that is to the detriment of surrounding
agriculture.
</p><!--l. 46--><p class="noindent">In an extremely interesting <a href="http://aeon.co/magazine/nature-and-cosmos/why-its-time-to-lay-the-selfish-gene-to-rest/">article</a>, David Dobbs compares the two forms
of <span class="phvro8t-x-x-120">Schistocerca gregaria </span>to that of Dr. Jekyll and Mr. Hyde, the principle
characters in the Robert Louis Stevenson novella. For Dr. Jekyll in fiction,
and for <span class="phvro8t-x-x-120">Schistocerca gregaria </span>in reality, the power to morph into multiple
forms demonstrates the incredible power of a fixed genome yet plastic gene
expression.
</p><!--l. 48--><p class="noindent">It is often said that something is “in the genes.” Another oft-heard idiom that is
perhaps more appropriate is: “it’s how you use them.” This thesis will illustrate that,
with ever increasing resolution in the measurement of functional gene products (i.e.
                                                                                

                                                                                
the “transcriptome”), we are beginning to realize the tremendous diversity and
complexity of gene expression.
<a id="x1-5002r12"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.2 </span> <a id="x1-60002"></a>Nucleic Acid Sequencing</h3>
<a id="x1-6001r1"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.1 </span> <a id="x1-70001"></a>DNA Sequencing</h4>
<!--l. 56--><p class="noindent">Soon after it was realized that DNA is the source of genetic information in all living
organisms [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson1953a">Watson and Crick</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson1953a">1953</a>], and the <span class="phvro8t-x-x-120">pretty </span>and <span class="phvro8t-x-x-120">elegant </span>arrangement of
complementary, antiparrallel, DNA strands was known [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2012a">Watson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2012a">2012</a>], the
ability to determine specific arrangements of nucleotide bases in a given length of
DNA (i.e. to “sequence”) was seen as a critical missing piece of technology. It took
25 years after the structure of DNA was known to be able sequence it. By
1977, two completely different methods developed by Sanger [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1975a">Sanger and
Coulson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1975a">1975</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1977b">Sanger et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1977b">1977</a>] and Maxam-Gilbert [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMaxam1977a">Maxam and Gilbert</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMaxam1977a">1992</a>]
were reported. These sequencing technologies, from then on referred to
eponymously as “Sanger” or “Maxam-Gilbert” sequencing, were used to
determine the specific order of a small piece of DNA (200–300 nt). Sanger
sequencing soon dominated most sequencing reactions, likely due to the
                                                                                

                                                                                
conceptually more intuitive nature of the technology. Over the next 35 years, DNA
sequences were slowly cloned, sequenced, analyzed, and dutifully cataloged into
knowledge.
</p><!--l. 58--><p class="noindent">During the late 1970’s and throughout the 1980’s, DNA sequences were typically
communicated in important publications [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCordell1980a">Bell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCordell1980a">1980</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1978a">Sanger et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSanger1978a">1978</a>]. The
birth of the Internet in the 1990’s allowed publically-funded repositories to
store sequence information [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBenson2011a">Benson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBenson2011a">2011</a>]. Yet it was the human
genome project [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLander2011a">Lander</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLander2011a">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVenter2001">Venter et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVenter2001">2001</a>], provided the important
activation energy that brought DNA sequencing from a hard-to-perform, but
necessary, analysis, to an organized large-scale effort of assembling the complete
genetic material of complex genomes. An often criticized, but undeniably
disrupting force in the human genome project was the competing efforts by the
privately-owned company Celera [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVenter2008a">Venter</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVenter2008a">2007</a>]. Taking a higher-throughput and
centralized approach to determining the sequence of the human genome,
Celera fundamentally changed the landscape of genome assembly. Instead of
assigning specific sections of the genome to be worked out by individual labs,
Celera parallelized the effort. Celera collected the best “high-throughput”
Sanger-sequencing devices from Agilent (ABI 3700 DNA Analyzer) and used a
“shotgun” approach [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XStaden1979">Staden</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XStaden1979">1979</a>], sequenced pairwise [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRoach1995">Roach et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRoach1995">1995</a>].
Combined with sequence scaffolds made available by the publicly-funded
project, Celera was able to assemble a high-quality genome quickly. Arguably,
                                                                                

                                                                                
this was the first deep sequencing effort. Coincident with the beginning of a
new millennium, it changed the landscape of molecular and biochemical
research.
<a id="x1-7001r15"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.2.2 </span> <a id="x1-80002"></a>High-throughput Sequencing</h4>
<!--l. 63--><p class="noindent">Sequencing DNA by Sanger’s technology remains a valuable tool in every biological
scientist’s toolbox. However, Sanger sequencing has a practical throughput
limit. Each DNA molecule to be sequenced must be isolated, cloned, and
amplified—using bacteria. Given that the human genome [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2005a">Consortium</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2005a">2004</a>]
comprises &gt;3 billion bp, and each Sanger reaction provides ~800 nt of quality
sequence, at least ~4 million individual reactions are needed to determine the
sequence of the human genome. This number assumes all “reads” are of
sufficient quality, length, and do not overlap by even 1 nt. The use of reads with
no overlap is out of the question, as overlaps are essential for assembling
individual sequences via overlaps, critical for assembly through repeated
sequences.
</p><!--l. 65--><p class="noindent">Even the best practical improvements to Sanger work-flows could not bring
the technology in-line with aspirations of analyzing many species and/or
organisms. Yet, the early 2000’s saw multiple efforts to improve the scale
                                                                                

                                                                                
of DNA sequencing, first using MPSS [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrenner2000a">Brenner et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrenner2000a">2000</a>], but perhaps
more importantly, by Pyro- [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRonaghi1998a">Ronaghi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRonaghi1998a">1998</a>] and Polony sequencing
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2005">Shendure et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2005">2005</a>]. Both pyro- and polony sequencing utilize emulsion PCR
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNakano2003a">Nakano et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNakano2003a">2003</a>] for clonal amplification prior to sequencing, removing the
bottleneck of bacterial cloning. In contrast to Sanger sequencing, where
fluorescence signal from the last incorporated chain-terminating nucleotide is
observed, pyrosequencing visualizes light given off by luciferase reacting with
pyrophosphate (PPi), a by-product of nucleotide incorporation. This approach was
later commercialized by 454 technologies. Polony sequencing involves a more
complicated sequencing-by-ligation method, eventually commercialized by
Applied BioSystems and branded as SOLiD sequencing. While both of these
technologies provided valuable high-throughput sequences, neither has been
as successful as the approach commercialized by Solexa, now known as
Illumina.
</p><!--l. 67--><p class="noindent">Illumina sequencers use a sequencing-by-synthesis approach. After clonal
amplification of DNA on a slide surface [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBentley2008">Bentley et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBentley2008">2008</a>], fluorescent nucleotides
are visualized as they are incorporated into the growing DNA strand. Since 2006,
iterations of the Illumina platform (e.g. GE, GE-II(x), HiSeq, HiSeq 2500, Hi X) have
demonstrated a steady and impressive increases in both read depth and length. On
February 15th 2012, Illumina announced on its <a href="http://blog.basespace.illumina.com/">Basespace blog</a>, that they had
sequenced a HapMap sample at 40X coverage, using the HiSeq 2500 platform and
                                                                                

                                                                                
paired-end 100 nt reads in a single run. On January 14th, 2014, Illumina
<a href="http://bit.ly/PZpegZ">announced</a> its HiSeq X system, the first platform to truly attain the mythical
$1,000 genome. These machines demonstrate that sequencing genomes is
no longer the monumental endeavor it once was and that completely new
experimental possibilities are a reality for life science researchers (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-8001r2">1.2<!--tex4ht:ref: Intro:fig:SeqCosts --></a>).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-8001r2"></a>
                                                                                

                                                                                
<!--l. 71--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis0x.png" alt="PIC" class="graphics" width="346.29375pt" height="155.58124pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis0x.png" src="Figures/Intro/Sequencing_costs_over_time.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.2: </span><span class="content">Cost of sequencing the human genome over time<br class="newline">The costs of  sequencing  the  human  genome  has  decreased  on  a  log
scale over a  10  year  period  due  to  major  improvements  in  high-throughput
sequencing.   Data   from   Wetterstrand   KA.   DNA   Sequencing   Costs:   Data
from the NHGRI   Genome   Sequencing   Program   (GSP)   Available   at:
<a href="file://localhost/Users/royc/Google_Drive/Thesis/www.genome.gov/sequencingcosts" class="url"><span class="ectt-1200">www.genome.gov/sequencingcosts</span></a>. Accessed 2013-09-03). </span></div><!--tex4ht:label?: x1-8001r2 -->
                                                                                

                                                                                
<a id="x1-8002r16"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.3 </span> <a id="x1-90003"></a>RNA Sequencing</h4>
<!--l. 83--><p class="noindent">Before reviewing sequencing technologies’ application to measuring gene
expression, microarray technology must first be discussed. The importance of
microarrays in the measurement of gene expression cannot be overstated [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMarioni2008">Marioni
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMarioni2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2008">Shendure and Ji</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2008">2008</a>]. Microarray use in recent publications, such as
defining the transcriptome of the developing human brain <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiller2014">Miller et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiller2014">2014</a>], was
correct and essential. However, limitations of novel sequences discovery combined
with analogue signal, make the relevance of microarray technology off-topic to this
section.
</p><!--l. 85--><p class="noindent">The first widely-accepted large scale method used to measured gene expression was
Serial Analysis of Gene Expression (SAGE) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVelculescu1995a">Velculescu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVelculescu1995a">1995</a>]. SAGE, like
the before mentioned MPSS technique, produces a digital output of gene
expression using a clever procedure of restriction endonucleolytic cDNA
cleavage. Cleaved-product sticky ends are concatenated together to form long
DNA fragments. Fragments are cloned into a vector, amplified, and Sanger
sequenced. Using known sequences incorporated during concatenation,
the number of sequenced fragments that align to a given gene is related
to the abundance of the original RNA molecule. A clever molecular trick,
                                                                                

                                                                                
SAGE allowed researches to dip into the 5-log range of mRNA expression.
However, the technique it is still limited by Sanger sequencing read lengths and
depth.
</p><!--l. 87--><p class="noindent">The Solexa/Illumina platform relies on clonal amplification of a single template directly
on a slide surface and is therefore not restricted by bacterial cloning. Imaging
spots with sensitive digital cameras after sequential addition of fluorescent
nucleotides, <span class="phvro8t-x-x-120">sequencing by synthesis</span>, turned out to be the right mix for a “second
generation” HTS platform. Not long after the Solexa/Illumina platform achieved read
lengths of sufficient length and depth necessary to measure gene expression
were the first RNA-Seq papers published [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLister2008">Lister et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLister2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMortazavi2008">Mortazavi
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMortazavi2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNagalakshmi2008">Nagalakshmi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNagalakshmi2008">2008</a>]. These papers provided a glimpse
into the future of molecular biology. Indeed, in the years since, analysis by
RNA-Seq has quickly overtaken other forms of gene expression analysis, as
demonstrated by the number of accessions created in the publically-funded
repository of sequencing data, the Gene Expression OmniBus (GEO) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarrett2013">Barrett
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarrett2013">2013</a>]. RNA-Seq allows for digital quantification of RNA expression across
physiologically-relevant ranges [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlencowe2009">Blencowe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlencowe2009">2009</a>], novel sequence
discovery, measuring RNA-editing [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2011">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2011">2011</a>], and fuels the novel area
of transcript assembly [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">2010</a>]. Through modification of the
basic protocol or performing additional biochemical steps, RNA-Seq can
be used to investigate many aspects of RNA biology (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-9001r3">1.3<!--tex4ht:ref: Intro:fig:htsMethods --></a> and [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMutz2013">Mutz
                                                                                

                                                                                
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMutz2013">2013</a>]).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-9001r3"></a>
                                                                                

                                                                                
<!--l. 91--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis1x.png" alt="PIC" class="graphics" width="420.57124pt" height="311.16249pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis1x.png" src="Figures/Intro/RNA_Sequencing_methodologies.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.3: </span><span class="content">Methods for High-throughput sequencing of RNA<br class="newline">In the short years since the first report of RNA-Seq, many variations have been
reported. The figure above provides an incomplete graphical illustration of these
variations. A more complete list of “*Seq” applications is maintained on this <a href="http://liorpachter.wordpress.com/seq/">blog</a>
<a href="http://liorpachter.wordpress.com/seq" class="url"><span class="ectt-1200">http://liorpachter.wordpress.com/seq</span></a>. </span></div><!--tex4ht:label?: x1-9001r3 -->
                                                                                

                                                                                
<!--l. 100--><p class="noindent">RNA processing begins the moment the nascent RNA is exposed from the
polymerase exit channel. Numerous methodologies enrich RNA-Seq libraries for
particular types of RNA. For example, measurement of nascent transcripts can be
performed via GRO-Seq [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCore2008a">Core et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCore2008a">2008</a>], and the extremely complicated process
of RNA turnover (referring to the rates at which RNAs both are produced and
degraded) has been examined [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhosh2010a">Ghosh and Jacobson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhosh2010a">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTani2012">Tani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTani2012">2012</a>].
RNA::Protein interactions can be measured with or without cross-linking
the protein to the RNA, via CLIP or RIP, respectively (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-830002">5.2.2<!--tex4ht:ref: Disc:subsec:Labeling of precursors --></a>)
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2006">Licatalosi and Darnell</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSingh2013">Singh et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSingh2013">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUle2005">Ule et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUle2005">2005</a>]. Once an
RNA has been fully transcribed, known processing steps such as 5<span class="cmsy-8">&#8242;</span>&nbsp;7meG
CAP formation and poly(A)+ tail formation can be measured using any of the
Cap-Seq/CAGE methodologies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShiraki2003a">Shiraki et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShiraki2003a">2003</a>], or PAS/TAIL/PAL [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014b">Chang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014b">2014a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShepard2011">Shepard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShepard2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSubtelny2014">Subtelny et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSubtelny2014">2014</a>]. With appropriate
size-selection steps, small RNAs [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">Ghildiyal et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">2008</a>] can also be captured.
Finally, traditional RNA-Seq can capture many of the same RNA fragments as the
above mentioned methods, even though it is mainly associated with measurement of
traditional mRNA.
</p><!--l. 102--><p class="noindent">RNA-Seq (and all its flavors) are traditionally associated with quantification of RNA
obtained from <span class="phvro8t-x-x-120">many </span>tissue culture cells or bulk pieces of tissue. Recently, efforts to
measure RNA expression occurring in individual cells has gained attention
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShapiro2013b">Shapiro et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShapiro2013b">2013</a>]. Perhaps the most interesting concept when thinking
                                                                                

                                                                                
about measurement of gene expression in a single cell is the “biological
uncertainty principle”, wherein it is possible to either know, or change — but
not both—&nbsp;the RNA composition of a single cell. The name borrows from
Heisenberg’s uncertainty principle [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKennard1927">Kennard</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKennard1927">1927</a>] and is often confused with the
more appropriate “Observer effect” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRiley2013">Riley and Steitz</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRiley2013">2013</a>]. Leaving that
issue aside, measuring the unique transcriptome of a cell is surly an exciting
and informative endeavor [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMarinov2013">Marinov et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMarinov2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShalek2013b">Shalek et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShalek2013b">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWills2013">Wills
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWills2013">2013</a>]. Compared to DNA, the diversity of RNA synthesis within living
cells is potentially much more complicated [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2012">Shendure and Aiden</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2012">2012</a>], and
the ability to accurately measure RNA dynamics should allow us to make
much more informative observations concerning biology then is currently
possible.
<a id="x1-9002r14"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.3 </span> <a id="x1-100003"></a>Nucleic Acid Splicing</h3>
<!--l. 107--><p class="noindent">1977 brought the discovery of “split genes” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBerget1977">Berget et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBerget1977">1977</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChow1977">Chow et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChow1977">1977</a>].
Almost immediately it was reasoned that split genes could be arranged in different
combinations, greatly increasing the coding potential of a genome [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGilbert1978a">Gilbert</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGilbert1978a">1978</a>].
The process of rearranging genes, now known as alternative splicing, has proven to
                                                                                

                                                                                
be an integral part of eukaryotic gene expression.
<a id="x1-10001r18"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.1 </span> <a id="x1-110001"></a>Alternative Splicing</h4>
<!--l. 112--><p class="noindent">The number of genes estimated to be alternatively spliced has grown considerably. In
1993, Phillip Sharp, Co-Nobel-prize winner for the discovery of splicing, stated that:
“Approximately, one of every twenty genes is expressed by alternative pathways of
RNA splicing in different cell types or growth states” <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharp2014">Sharp</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharp2014">2014</a>]. Not long after the
assembly of the first human genome, a number of groups combed through Expressed
Sequence Tag (EST) databases to increase that estimate to 35%-59% [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModrek2002">Modrek and
Lee</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModrek2002">2002</a>]. Soon after, analysis using specially designed microarrays resulted in
an increased estimate of 74% [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2003">Johnson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2003">2003</a>]. In late 2008, three
groups used RNA-Seq to demonstrate that between 86% and 95% of human
multi-exon genes are subject to alternative splicing [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">Pan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSultan2008">Sultan
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSultan2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">Wang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">2008</a>]. Not only did they demonstrate that almost all
genes are alternatively spliced, they also showed that alternative splicing
often occurs in a tissue- and cell type-specific manner. In combination with
regulation of transcription itself, the study of alternative splicing is critical to
advance our understanding of the connections between the comparably
static genomic DNA sequence and the highly flexible and adaptive abilities of
                                                                                

                                                                                
organisms.
</p><!--l. 114--><p class="noindent">A pair of papers recently published in Science best illustrate the amazing complexity
alternative splicing can generate between, and perhaps more importantly <span class="phvro8t-x-x-120">within</span>,
organisms [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarbosa-Morais2012">Barbosa-Morais et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarbosa-Morais2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkin2012">Merkin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkin2012">2012</a>]. Analysis of RNA
obtained from a diverse array of organisms and tissues by RNA-Seq has revealed
that splicing patterns are shared more closely between organs of different species
than between different organs of the same species. It seems that alternative
splicing that is essential to the use of a common genome in roles required by
physiologically-specialized organs.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-11001r4"></a>
                                                                                

                                                                                
<!--l. 118--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis2x.png" alt="PIC" class="graphics" width="332.24124pt" height="160.59999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis2x.png" src="Figures/Intro/numberHumanGenesAndNumberSpliced.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.4: </span><span class="content">Dark Grey - Estimates of number of human genes; Light Grey -
Estimates of what percent of genes undergo some form of alternative splicing. </span></div><!--tex4ht:label?: x1-11001r4 -->
                                                                                

                                                                                
<a id="x1-11002r21"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.2 </span> <a id="x1-120002"></a>A Splicing Code</h4>
<!--l. 129--><p class="noindent">A gene is alternatively spliced when, as a result of transcription and processing, there
are at least two unique transcripts produced. Beyond isoform counting by RNA-Seq, a
major area of effort is decoding sequence regulatory elements (SREs) contained in
pre-mRNA that define alternative splicing site selection [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">Wang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">2008</a>]. In
contrast to core splicing signals, there exists limited knowledge of the SREs that
serve to increase and decrease the strength of a particular splice site. Regulatory
elements are always surrounded by a sea of other potential sites. Through a variety of
mechanisms, SREs serve as <span class="phvro8t-x-x-120">cis</span>-acting sequences and binding sites for
<span class="phvro8t-x-x-120">trans</span>-acting factors. Some of the best-studied SREs include Exon Splicing
Enhancers and Silencers (ESEs and ESSs). Members of the Serine-Arginine (SR)
protein family typically bind to ESEs located in an exon, promoting exon
definition and thereby increasing the probability that the exon will be included in
the final transcript [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2000">Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2000">2000</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLong2009">Long and Caceres</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLong2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsen2010">Nilsen and
Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsen2010">2010</a>]. Meanwhile, ESS recognition squelches inclusion through
binding trans-acting heterogeneous ribonucleoprotein particles (hnRNPs)
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMartinez-Contreras2007">Martinez-Contreras et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMartinez-Contreras2007">2007</a>]. Therefore, trans-acting factor SRE binding
can either promote or inhibit splicing machinery::pre-mRNA interactions.
The current working hypothesis is that a finely tuned combination of these
                                                                                

                                                                                
binding events determines the final exonic content of each isoform [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouse2008">House and
Lynch</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouse2008">2008</a>].
</p><!--l. 131--><p class="noindent">Sequence motifs that compose the alternative splicing code have been teased out
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarash2010">Barash et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarash2010">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLadd2002">Ladd and Cooper</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLadd2002">2002</a>]. Additionally, assignment of
binding motifs to tissue-specific trans-acting factors has also progressed [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJin2003">Jin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJin2003">2003</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">Licatalosi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUle2005">Ule et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUle2005">2005</a>]. Many of these binding motifs
were identified using combined computational and biochemical approaches.
Computational approaches involve searching for a comparative enrichment of
sequences near splice sites. Biochemical approaches include gel shift, SELEX,
and cross-linking. Many of these approaches are performed <span class="phvro8t-x-x-120">in vitro </span>and
disregard the importance of cellular context on binding affinities. However, with
increasing accessibility of HTS, many groups are extracting physiologically
relevant, high-resolution data from traditional biochemical techniques [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIngolia2009">Ingolia
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIngolia2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIngolia2011">2011</a>]. Deep-sequencing approaches are also being applied to
questions involving mechanisms of alternative splicing. In addition to the
RNA-Seq experiments, High-Throughput Sequencing [following] Cross-Linking
Immunoprecipitation (HTS-CLIP) has confirmed SRE motif data predicted from
computational and microarray experiments [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2010">Hafner et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2010">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">Licatalosi
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">2008</a>]. Using HTS-CLIP, researchers can now enrich their samples for
sequences that bind trans-acting factors of interest.
<a id="x1-12001r23"></a>
                                                                                

                                                                                
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.3 </span> <a id="x1-130003"></a>The Isoform Problem</h4>
<!--l. 136--><p class="noindent">Alternative splicing research now relies on large-scale (aka: <span class="phvro8t-x-x-120">global</span>, <span class="phvro8t-x-x-120">genome-wide</span>,
<span class="phvro8t-x-x-120">high-throughput</span>) techniques. Two of the most widely-applied technologies employed
for large-scale analysis of gene expression are microarrays and “2nd generation”
HTS. Unfortunately, both of these techniques have fundamental limitations, with the
major issues being probe specificity for the former and read length for the
latter.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-13001r5"></a>
                                                                                

                                                                                
<!--l. 140--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis3x.png" alt="PIC" class="graphics" width="415.55249pt" height="327.22249pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis3x.png" src="Figures/Intro/SeqLengths_and_Connectivity.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.5: </span><span class="content">HTS read lengths are not sufficient to maintain alternative splicing
connectivity<br class="newline">A) Long RNAs may have multiple sites of alternative splicing, separated by
1000’s of nt; B) Most mRNAs have ~10 exons of ~150 nt each. Some have many
more (and longer) exons. Read lengths of current sequencing technologies do
not maintain connectivity between distant sites. </span></div><!--tex4ht:label?: x1-13001r5 -->
                                                                                

                                                                                
<!--l. 149--><p class="noindent">Microarrays rely on hybridization of a target sequence to a known probe averaging
25–100 nt in length [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSouthern2001">Southern</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSouthern2001">2001</a>]. Therefore, microarrays only report the
presence of short sequences in the target sample and do not provide linkage
information between the sequences. A hypothetical scenario can be used to describe
this issue. The goal of this example is to investigate a transcript known to display two
different regions of alternative splicing (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-13001r5">1.5<!--tex4ht:ref: Intro:fig:NoConnectivityInHTSMethods --></a>). Probes targeting these
two regions demonstrate an increase in signal for both alternative splicing
events. Unfortunately, it is not known if we observe an increase in unique
transcripts, each containing only one region of alternative splicing , or an
increase in production of a single transcript containing both regions [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007">Calarco
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007">2007a</a>]. This binary analysis is the heart of the “connectivity problem.”
Microarrays have proven extremely informative and will likely continue to
do so in more targeted applications. However, this issue, combined with
concerns of cross-hybridization, reproducibility, and a comparably small dynamic
range, has hastened the displacement of microarray by RNA-Seq as the
preferred method for comprehensive analysis of gene expression [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2008">Shendure and
Ji</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2008">2008</a>].
</p><!--l. 151--><p class="noindent">2nd generation HTS methodology is the <span class="phvro8t-x-x-120">de facto </span>method for comprehensive
transcriptome analysis. HTS allows for <span class="phvro8t-x-x-120">de novo </span>identification of isoforms. RNA-Seq
operates over a larger dynamic range and in a quantitative manner [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMortazavi2008">Mortazavi
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMortazavi2008">2008</a>]. Additionally, techniques exist to enrich samples for low-abundance
                                                                                

                                                                                
isoforms, making the complete cataloging of alternative splicing events a possibility
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2008">Djebali et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSalehi-Ashtiani2008">Salehi-Ashtiani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSalehi-Ashtiani2008">2008</a>]. Unfortunately, the current
read-length abilities (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-13001r5">1.5<!--tex4ht:ref: Intro:fig:NoConnectivityInHTSMethods --></a>) of all sequencing platforms do not solve the
connectivity problem. Excluding single-molecule read lengths of sufficient
length [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2004">Shendure et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShendure2004">2004</a>], other approaches proposed to solve the
connectivity problem include traditional cloning and sequencing or hybridization of
query oligos to single-molecule transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007">Calarco et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007">2007a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEmerick2007">Emerick
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEmerick2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhu2003">Zhu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhu2003">2003</a>]. While these approaches can determine exon
sequence connectivity, they scale poorly and are not feasible for large-scale
applications.
</p><!--l. 153--><p class="noindent">Alternative splicing is an essential regulatory mechanism involved in the control of
human gene expression. Its combinatorial nature could potentially answer many basic
questions concerning gene expression, such as a physical explanation of what
separates us from our closest evolutionary ancestor, the chimpanzee [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007a">Calarco
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCalarco2007a">2007b</a>]. Additionally, the influence of alternative splicing on disease and
cancer is slowly coming to light [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTazi2009">Tazi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTazi2009">2009</a>]. Unfortunately, because of the
limitations of methods currently used for the large-scale analysis of isoform
expression, we fail to obtain the complete picture of alternative splicing. One specific
missing element of that picture is the prevalence of coordination between different
regions of alternative splicing separated by large spans of sequence. An efficient,
large-scale, single-molecule technique that maintains isoform sequence
                                                                                

                                                                                
connectivity is required to complete the complicated picture of alternative
splicing.
</p><!--l. 155--><p class="noindent">Identification of proximally-acting SREs is progressing at a rapid pace. New and
traditional biochemical methods, coupled with HTS, will undoubtedly fuel this
progress. Unfortunately, a critical component of alternative splicing regulation
currently neglected by the field is that of SREs acting across a considerable distance
(&gt;800 nt). One observation that may lead to the identification of long-range SREs is
intramolecular coordination between distal splicing decisions. Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-13001r5">1.5<!--tex4ht:ref: Intro:fig:NoConnectivityInHTSMethods --></a> shows a
model transcript that may exhibit coordinated distal regions of alternative splicing. In
this model, the 5<span class="cmsy-8">&#8242;</span>&nbsp;region of alternative splicing contains a cassette exon, which may
or may not be included. This region is separated from the 3<span class="cmsy-8">&#8242;</span>&nbsp;region of alternative
splicing by many thousands of nucleotides. Does the decision to include the cassette
exon have an effect on which of the mutually exclusive exons is included? This type
of alternative splicing regulation may represent a general and pervasive
phenomenon.
<a id="x1-13002r24"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.4 </span> <a id="x1-140004"></a>Coordinated Splicing</h4>
<!--l. 160--><p class="noindent">The provocative “Miller Spread”, showing spliceosomes associated with RNA
                                                                                

                                                                                
transcripts, ignited the first thoughts that transcription and splicing are intricately
linked [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOsheim1985">Osheim et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOsheim1985">1985</a>]. Twelve years later, the observation that polymerase
speed can affect downstream splicing decisions was reported [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCramer1997">Cramer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCramer1997">1997</a>],
spawning a new field of research into linked transcription and splicing.
</p><!--l. 162--><p class="noindent">How would linkage between transcription and splicing manifest? One of the
clearest examples is mouse <span class="phvro8t-x-x-120">Fibronectin </span>(<span class="phvro8t-x-x-120">Fn1</span>) (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-14001r6">1.6<!--tex4ht:ref: Intro:fig:mouseFn1 --></a>) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchwarzbauer1983">Schwarzbauer
and Tamkun</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchwarzbauer1983">1983</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWhite2011a">White and Muro</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWhite2011a">2011</a>]. In this gene, inclusion of the
alternatively spliced Extra Domain A (aka “EDI” or “EDA”) region promotes
splicing from one of three alternative 3<span class="cmsy-8">&#8242;</span>&nbsp;Splice Sites (3<span class="cmsy-8">&#8242;</span>&nbsp;SS) in the type III
homology connecting segment (IIICS) region, resulting in more frequent
production of shorter transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">Fededa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">2005</a>]. This effect occurs
over six constitutively expressed exons and 800 nt of sequence (5,400 nt if
introns are considered). <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">Fededa et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">2005</a>] also analyzed EST databases,
concluding that approximately 25% of human genes contain multiple regions of
alternative splicing. How many of these regions could show a coordinated effect,
similar to that observed in <span class="phvro8t-x-x-120">Fn1</span>? Providing some insight into this question,
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>] used microarrays designed to report on inclusion levels
of cassette exons in mammalian central nervous system tissues [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>]. The results produced a set of 38 pairs of exons mapping to the
same gene that showed a coordinated increase or decrease of inclusion
levels.
                                                                                

                                                                                
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-14001r6"></a>
                                                                                

                                                                                
<!--l. 166--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis4x.png" alt="PIC" class="graphics" width="390.45874pt" height="289.07999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis4x.png" src="Figures/Intro/Fibronectin.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.6: </span><span class="content">Mouse <span class="phvro8t-x-x-120">Fn1 </span>contains multiple sites of Alternative Splicing<br class="newline">A) There are three highly-studied regions of alternative splicing in mouse <span class="phvro8t-x-x-120">Fn1</span>:
Cassette exons EDB and EDA and the Variable(V)-region exon, which displays
multiple 3<span class="cmsy-8">&#8242;</span> &nbsp;splice sites. Each of these sites is separated by multiple constitutive
exons. B) Considering  simplistic  splicing  of  these  three  exons,  there  are  12
different isoforms of mouse <span class="phvro8t-x-x-120">Fn1</span>. </span></div><!--tex4ht:label?: x1-14001r6 -->
                                                                                

                                                                                
<!--l. 175--><p class="noindent">Few studies have investigated coordinated splicing between adjacent exons present
in mRNA. The vertebrate genes <span class="phvro8t-x-x-120">4.1B </span>and <span class="phvro8t-x-x-120">4.1R</span>, members of the protein 4.1 family
and encode cytoskeletal adaptor proteins. Both genes undergo splicing of upstream
5<span class="cmsy-8">&#8242;</span> &nbsp;first exons to distal 3<span class="cmsy-8">&#8242;</span>&nbsp;second exons, skipping a stronger proximal 3<span class="cmsy-8">&#8242;</span>&nbsp;second exon
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XParra2012">Parra et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XParra2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XParra2008">2008</a>]. This is accomplished through “intrasplicing” involving an
intronic sequence element (“intraexon”) only present when transcription begins at the
upstream 5<span class="cmsy-8">&#8242;</span> &nbsp;exon. This allows the exon to ligate to the weaker distal 3<span class="cmsy-8">&#8242;</span>&nbsp;second exon
via an intermediate splicing event. Importantly, this type of splicing would be similar,
but different from, recursive splicing seen in <span class="phvro8t-x-x-120">Drosophila melanogaster </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBurnette2005a">Burnette
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBurnette2005a">2005</a>].
</p><!--l. 177--><p class="noindent">Another example of the importance of Intronic Recognition Elements (ISE) on
alternative splicing is that observed in equine <span class="cmmi-12">&#946;</span><span class="phvro8t-x-x-120">-casin</span>, where the authors propose a
model involving an IRE bound to the exit channel of the elongating polymerase. IRE
binding promotes inclusion of downstream cassette exons [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLenasi2006">Lenasi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLenasi2006">2006</a>].
Taking a more genome-wide approach <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPeng2008">Peng et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPeng2008">2008</a>] examined human
and mouse EST data looking for correlations between adjacent alternative
splicing cassette exons. The authors note that positively correlated pairs of
adjacent cassette exons typically resemble constitutive exons in splice strength,
whereas negatively, or weakly correlated pairs are likely to be newly emerging
exons, whose strength of splicing has not evolved enough to be constitutively
included.
                                                                                

                                                                                
</p><!--l. 179--><p class="noindent">The last, most current, and thorough study of intra-gene splicing coordination
involves the <span class="phvro8t-x-x-120">Caenorhabditis elegans </span>gene <span class="phvro8t-x-x-120">slo1 </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">Glauser et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2011">Johnson
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2011">2011</a>]. <span class="phvro8t-x-x-120">slo1 </span>is the <span class="phvro8t-x-x-120">Caenorhabditis elegans </span>orthologue of the human BK
channel gene <span class="phvro8t-x-x-120">Kcnma</span>, also known to undergo extensive alternative splicing
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsen2010">Nilsen and Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsen2010">2010</a>] via 13 cassette exons, potentially coding for over
1,000 different isoforms. <span class="phvro8t-x-x-120">Kcnma </span>is developmentally, spatially, and tissue
regulated. It is involved in a diverse range of cellular processes, including
hearing, circadian rhythms, urinary function, and vasoregulation [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFodor2009a">Fodor and
Aldrich</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFodor2009a">2009</a>].
</p><!--l. 181--><p class="noindent">In worms, <span class="phvro8t-x-x-120">slo1 </span>can produce up to 12 different isoforms. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">Glauser et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">2011</a>] used
QPCR to demonstrate individual, alternative region inclusion frequencies do not
correspond to complete isoform frequencies, when measured via TaqMan. They go
on to describe a interdependent-splicing model that best fits the data and support
interdependence via mutations at one sight altering both upstream and downstream
sites of alternative splicing, separated by at least one other splicing event. After
measuring the biophysical properties of the isoforms [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2011">Johnson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJohnson2011">2011</a>]. [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">Glauser
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGlauser2011">2011</a>] conclude that coordinated alternative splicing is critical for proper BK
channel function <span class="phvro8t-x-x-120">in vivo</span>. It is interesting to note that this study also identified an IRE
that displayed some type of coordinated, or co-regulated effect on alternative splicing
.
                                                                                

                                                                                
</p><!--l. 183--><p class="noindent">Indeed the “Miller Spread” was an early glimpse into another aspect of Nature’s
complexity. Over just the past ~5 years, layers of functional coupling between
transcription and splicing have been observed [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkhofer2014">Merkhofer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkhofer2014">2014</a>]. For example,
specific chromatin marks seem to not only demarcate transcriptionally active regions
of chromatin for silent ones, but also alternative splicing exons from constitutively
chosen ones [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKolasinska-Zwierz2009">Kolasinska-Zwierz et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKolasinska-Zwierz2009">2009</a>] and first exons appear to be
epigenetically marked to aid in transcriptional identification [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBieberstein2012">Bieberstein et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBieberstein2012">2012</a>].
Tied to the recurring theme of this thesis, increasing resolution of genome-wide data
should reveal more ties between these cellular processes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2012">Brown et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuco2011a">Luco
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuco2011a">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchwartz2010">Schwartz and Ast</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchwartz2010">2010</a>].
<a id="x1-14002r26"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.3.5 </span> <a id="x1-150005"></a>One Gene. Many Isoforms</h4>
<!--l. 188--><p class="noindent">Just as it is tempting to uncouple, for the purpose of research, the evolutionarily
intertwined processes such as transcription and splicing, it is easy to think of
alternative splicing as a binary process. Isoform A or B is produced based upon
picking either exon A or B. What quickly becomes evident (and is far too real for
researchers building transcriptome assembly algorithms) is that the combinatorial
nature of alternative splicing makes it both a power means of generating isoform
diversity <span class="phvro8t-x-x-120">and </span>a difficult problem to study [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2012a">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2012a">2012</a>].
                                                                                

                                                                                
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-15001r7"></a>
                                                                                

                                                                                
<!--l. 192--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis5x.png" alt="PIC" class="graphics" width="393.46999pt" height="170.6375pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis5x.png" src="Figures/Intro/ASEventTypesPlot.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.7: </span><span class="content">Number of hg19 Alternative event types per gene<br class="newline">Alternative Event types per gene. Accessed from RefSeq on 2014-03-24. </span></div><!--tex4ht:label?: x1-15001r7 -->
                                                                                

                                                                                
<!--l. 201--><p class="noindent">One of the most recent attempts to investigate the breath of combinations produced
by alternative splicing is the ENCODE project [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBirney2007">Birney et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBirney2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDunham2012">Dunham
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDunham2012">2012</a>]. The transcriptional annotation arm of the ENCODE project
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDerrien2012">Derrien et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDerrien2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">Djebali et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">2012</a>] used the data from analysis of
15 human cancerous cell lines, and found that each gene produces ~10
isoforms per gene, with a broad distribution in terms of isoforms expressed per
sample.
</p><!--l. 203--><p class="noindent">The ENCODE project added to the building evidence that most human genes
undergo alternative splicing in many more ways than previously appreciated [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">Pan
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">Wang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">2008</a>]. Most genes contain multiple forms of alternative
splicing (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15001r7">1.7<!--tex4ht:ref: Intro:fig:asEventsBarChart --></a>). Despite the prevalence of complex alternative spliced genes,
just a few genes are routinely used as examples to illustrate numerical possibilities
and biological significance. For example, the human immune system relies heavily on
alternative splicing for plastic antigen recognition and response [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLynch2004">Lynch</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLynch2004">2004</a>].
Modulation of extracellular signaling proteins such as <span class="phvro8t-x-x-120">CD44 </span>and cellular adhesion
protein <span class="phvro8t-x-x-120">CD45 </span>have been well-studied [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPonta2003">Ponta et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPonta2003">2003</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZikherman2008">Zikherman and
Weiss</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZikherman2008">2008</a>].
</p><!--l. 205--><p class="noindent">Alternative splicing in humans, however, does not seem to produce the number of
unique possible combinations as alternative splicing of genes in simpler organisms,
such as <span class="phvro8t-x-x-120">Drosophila melanogaster </span>(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15002r8">1.8<!--tex4ht:ref: Intro:fig:txPerFlyGene --></a> and Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15002r8">1.8<!--tex4ht:ref: Intro:fig:txPerFlyGene --></a>). Perhaps this reduced
                                                                                

                                                                                
alternative splicing <span class="phvro8t-x-x-120">per gene </span>is due to gene specialization, with different genes
working in combination, as oppose to utilizing unique isoforms from a single gene
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2007">Park and Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2007">2007</a>]. For example, the fruit fly gene muscle myosin heavy chain
(<span class="phvro8t-x-x-120">Mhc</span>) can produce up to 480 different isoforms through alternative splicing of 17
different cassette exons [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBernstein1983a">Bernstein et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBernstein1983a">1983</a>]. Section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-160006">1.3.6<!--tex4ht:ref: Intro:subsec:Dscam --></a> will discuss an even
more appropriate comparison between the human and fly extracellular binding protein
DSCAM.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-15002r8"></a>
                                                                                

                                                                                
<!--l. 209--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis6x.png" alt="PIC" class="graphics" width="413.545pt" height="291.0875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis6x.png" src="Figures/Intro/NumberOFTranscriptsPerFlyGene.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.8: </span><span class="content">Number of transcripts per <span class="phvro8t-x-x-120">Drosophila melanogaster </span>gene<br class="newline">Data from [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">Brown et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">2014</a>], Supplemental Table 3. Number of transcript
per bin, with bin sizes “closed” on the upper part of range. </span></div><!--tex4ht:label?: x1-15002r8 -->
                                                                                

                                                                                
<div class="table">
                                                                                

                                                                                
<!--l. 219--><p class="noindent"><a id="x1-15003r1"></a></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="caption"><span class="id">Table&nbsp;1.1: </span><span class="content">Fly genes with &gt;2,000 assembled transcripts according to [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">Brown
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">2014</a>].</span></div><!--tex4ht:label?: x1-15003r1 -->
<div class="tabular"><table id="TBL-2" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-2-1g"><col id="TBL-2-1"><col id="TBL-2-2"><col id="TBL-2-3"><col id="TBL-2-4"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-2-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-1-1" class="td11"><span class="phvb8t-x-x-109">Gene Name</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-1-2" class="td11"><span class="phvb8t-x-x-109"># Introns</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-1-3" class="td11"><span class="phvb8t-x-x-109"># Transcripts</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-1-4" class="td11"><span class="phvb8t-x-x-109"># Proteins</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-2-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-2-1" class="td11"><span class="phvr8t-x-x-109">Mhc              </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-2-2" class="td11">   <span class="phvr8t-x-x-109">60      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-2-3" class="td11">    <span class="phvr8t-x-x-109">2040       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-2-4" class="td11">   <span class="phvr8t-x-x-109">511      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-3-1" class="td
11"><span class="phvr8t-x-x-109">slo                </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-3-2" class="td11">   <span class="phvr8t-x-x-109">49      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-3-3" class="td11">    <span class="phvr8t-x-x-109">2070       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-3-4" class="td11">   <span class="phvr8t-x-x-109">279      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-4-1" class="td
11"><span class="phvr8t-x-x-109">ps                 </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-4-2" class="td11">   <span class="phvr8t-x-x-109">30      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-4-3" class="td11">    <span class="phvr8t-x-x-109">2099       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-4-4" class="td11">    <span class="phvr8t-x-x-109">27       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-5-1" class="td11"><span class="phvr8t-x-x-109">rg                  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-5-2" class="td11">   <span class="phvr8t-x-x-109">45      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-5-3" class="td11">    <span class="phvr8t-x-x-109">2178       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-5-4" class="td11">    <span class="phvr8t-x-x-109">23       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-6-1" class="td11"><span class="phvr8t-x-x-109">shot              </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-6-2" class="td11">   <span class="phvr8t-x-x-109">60      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-6-3" class="td11">    <span class="phvr8t-x-x-109">2478       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-6-4" class="td11">   <span class="phvr8t-x-x-109">886      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-7-1" class="td
11"><span class="phvr8t-x-x-109">scrib             </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-7-2" class="td11">   <span class="phvr8t-x-x-109">53      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-7-3" class="td11">    <span class="phvr8t-x-x-109">2555       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-7-4" class="td11">   <span class="phvr8t-x-x-109">259      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-8-1" class="td
11"><span class="phvr8t-x-x-109">heph             </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-8-2" class="td11">   <span class="phvr8t-x-x-109">75      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-8-3" class="td11">    <span class="phvr8t-x-x-109">2876       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-8-4" class="td11">    <span class="phvr8t-x-x-109">52       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-9-1" class="td11"><span class="phvr8t-x-x-109">CG42748     </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-9-2" class="td11">   <span class="phvr8t-x-x-109">26      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-9-3" class="td11">    <span class="phvr8t-x-x-109">2876       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-9-4" class="td11">    <span class="phvr8t-x-x-109">51       </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-10-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-10-1" class="td
11"><span class="phvr8t-x-x-109">rdgA             </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-10-2" class="td11">   <span class="phvr8t-x-x-109">35      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-10-3" class="td11">    <span class="phvr8t-x-x-109">3003       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-10-4" class="td11">    <span class="phvr8t-x-x-109">89       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-11-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-11-1" class="td11"><span class="phvr8t-x-x-109">Mbs              </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-11-2" class="td11">   <span class="phvr8t-x-x-109">39      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-11-3" class="td11">    <span class="phvr8t-x-x-109">3080       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-11-4" class="td11">   <span class="phvr8t-x-x-109">119      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-2-12-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-12-1" class="td
11"><span class="phvr8t-x-x-109">CaMKI          </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-12-2" class="td11">   <span class="phvr8t-x-x-109">41      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-12-3" class="td11">    <span class="phvr8t-x-x-109">3992       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-12-4" class="td11">    <span class="phvr8t-x-x-109">7        </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-13-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-13-1" class="td11"><span class="phvr8t-x-x-109">par-1            </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-13-2" class="td11">   <span class="phvr8t-x-x-109">48      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-13-3" class="td11">    <span class="phvr8t-x-x-109">4410       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-13-4" class="td11">   <span class="phvr8t-x-x-109">142      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-14-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-14-1" class="td11"><span class="phvr8t-x-x-109">GluClalpha   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-14-2" class="td11">   <span class="phvr8t-x-x-109">27      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-14-3" class="td11">    <span class="phvr8t-x-x-109">4945       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-14-4" class="td11">   <span class="phvr8t-x-x-109">188      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-15-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-15-1" class="td11"><span class="phvr8t-x-x-109">Sap47          </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-15-2" class="td11">   <span class="phvr8t-x-x-109">24      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-15-3" class="td11">    <span class="phvr8t-x-x-109">5011       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-15-4" class="td11">    <span class="phvr8t-x-x-109">49       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-16-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-16-1" class="td11"><span class="phvr8t-x-x-109">Patronin        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-16-2" class="td11">   <span class="phvr8t-x-x-109">50      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-16-3" class="td11">    <span class="phvr8t-x-x-109">5615       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-16-4" class="td11">   <span class="phvr8t-x-x-109">590      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-17-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-17-1" class="td11"><span class="phvr8t-x-x-109">CG17838     </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-17-2" class="td11">   <span class="phvr8t-x-x-109">37      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-17-3" class="td11">    <span class="phvr8t-x-x-109">8333       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-17-4" class="td11">   <span class="phvr8t-x-x-109">147      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-18-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-18-1" class="td11"><span class="phvr8t-x-x-109">unc-13          </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-18-2" class="td11">   <span class="phvr8t-x-x-109">52      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-18-3" class="td11">    <span class="phvr8t-x-x-109">8391       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-18-4" class="td11">   <span class="phvr8t-x-x-109">279      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-19-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-19-1" class="td11"><span class="phvr8t-x-x-109">A2bp1          </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-19-2" class="td11">   <span class="phvr8t-x-x-109">29      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-19-3" class="td11">    <span class="phvr8t-x-x-109">9055       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-19-4" class="td11">    <span class="phvr8t-x-x-109">58       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-20-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-20-1" class="td11"><span class="phvr8t-x-x-109">Imp               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-20-2" class="td11">   <span class="phvr8t-x-x-109">33      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-20-3" class="td11">    <span class="phvr8t-x-x-109">9131       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-20-4" class="td11">    <span class="phvr8t-x-x-109">12       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-21-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-21-1" class="td11"><span class="phvr8t-x-x-109">pan               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-21-2" class="td11">   <span class="phvr8t-x-x-109">38      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-21-3" class="td11">    <span class="phvr8t-x-x-109">9432       </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-21-4" class="td11">    <span class="phvr8t-x-x-109">72       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-22-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-22-1" class="td11"><span class="phvr8t-x-x-109">Sh                </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-22-2" class="td11">   <span class="phvr8t-x-x-109">40      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-22-3" class="td11">   <span class="phvr8t-x-x-109">15995      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-22-4" class="td11">    <span class="phvr8t-x-x-109">66       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-23-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-23-1" class="td11"><span class="phvr8t-x-x-109">gish              </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-23-2" class="td11">   <span class="phvr8t-x-x-109">48      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-23-3" class="td11">   <span class="phvr8t-x-x-109">18972      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-2-23-4" class="td11">   <span class="phvr8t-x-x-109">142      </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-2-24-"><td style="white-space:nowrap; text-align:left;" id="TBL-2-24-1" class="td11">           </td></tr></tbody></table></div>
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
<a id="x1-15004r28"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.6 </span> <a id="x1-160006"></a><span class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></h4>
<!--l. 227--><p class="noindent">Unquestionably, the gene most frequently used to demonstrate the combinatorial
power of alternative splicing is <span class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span>. The “architecture”
of <span class="phvro8t-x-x-120">Dscam1 </span>is rather unique, but as we see in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15002r8">1.8<!--tex4ht:ref: Intro:fig:txPerFlyGene --></a> and Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15002r8">1.8<!--tex4ht:ref: Intro:fig:txPerFlyGene --></a>, <span class="phvro8t-x-x-120">Drosophila</span>
<span class="phvro8t-x-x-120">melanogaster </span>contain numerous genes that generate tremendous isoform diversity
from a single locus [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">Brown et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">2014</a>]. The basic structure of <span class="phvro8t-x-x-120">Dscam1 </span>is shown in
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16001r9">1.9<!--tex4ht:ref: Intro:fig:DscamArch --></a>.
</p><!--l. 229--><p class="noindent">Human <span class="phvro8t-x-x-120">Dscam </span>(Down Syndrome Cellular Adhesion Molecule), for which <span class="phvro8t-x-x-120">Dscam1 </span>was
named, was identified while looking for genes on chromosome 21, specifically band
21q22, where extra copies expressed in Down syndrome patients, a trisomy 21
disorder, maybe causative for disease [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYamakawa1998a">Yamakawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYamakawa1998a">1998</a>]. <span class="phvro8t-x-x-120">Dscam </span>was named
according to this association, and its membership in the immunoglobulin super family
of proteins with extracellular adhesion functions. Human <span class="phvro8t-x-x-120">Dscam </span>does undergo some
alternatively splicing and is broadly expressed in the developing nervous system.
Yet, it does not contain the same impressive number of cassette exons as
<span class="phvro8t-x-x-120">Dscam1</span>.
                                                                                

                                                                                
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-16001r9"></a>
                                                                                

                                                                                
<!--l. 233--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis7x.png" alt="PIC" class="graphics" width="391.4625pt" height="73.27374pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis7x.png" src="Figures/Intro/DscamArch.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.9: </span><span class="content">The architecture of the <span class="phvro8t-x-x-120">Drosophila melanogaster </span>gene <span class="phvro8t-x-x-120">Dscam1</span><br class="newline"><span class="phvro8t-x-x-120">Dscam1 </span>has three <span class="phvro8t-x-x-120">clusters </span>or “banks” of alternative cassette exons that are
splicing out in a mutually-exclusive manner. The first bank, “Exon 4”, contains
12 different variants, of which only one is ever included into the mRNA. Similarly,
banks 6 &amp; 9 each contain 48 and 33 different variants, respectively. These three
banks code for extracellular IgG domains, while the final region of alternative
splicing, exon 17, encodes two different trans-membrane domains, again only
one of which only one is included in the final mRNA. </span></div><!--tex4ht:label?: x1-16001r9 -->
                                                                                

                                                                                
<!--l. 242--><p class="noindent">Complex alternative splicing of<span class="phvro8t-x-x-120">Dscam1 </span>was first noticed by the Zipursky lab in 2000
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchmucker2000">Schmucker et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchmucker2000">2000</a>]. While looking for proteins associated with <span class="phvro8t-x-x-120">dock </span>and <span class="phvro8t-x-x-120">pak</span>,
two proteins important for neuronal growth cone guidance, they biochemically
co-purified DSCAM1. Sequencing of <span class="phvro8t-x-x-120">Dscam1 </span>clones revealed that all clones
contained different combinations of exons 4,6, and 9. In fact, these three exons are
chosen from three clusters of mutually-exclusive cassette exons, containing 12, 48,
and 33 different options each (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16001r9">1.9<!--tex4ht:ref: Intro:fig:DscamArch --></a>). The initial report kicked off an exciting
period of research into <span class="phvro8t-x-x-120">Dscam1 </span>structure and function. Determining the mechanism
and functional significance of <span class="phvro8t-x-x-120">Dscam1 </span>isoform diversity was a major goal of multiple
labs.
</p><!--l. 244--><p class="noindent">Before the highlights of <span class="phvro8t-x-x-120">Dscam1 </span>research are reviewed, it is illustrative to discuss
some basic <span class="phvro8t-x-x-120">Drosophila melanogaster </span>anatomy. There are 4 main regions where
<span class="phvro8t-x-x-120">Dscam1 </span>expression has been highly-studied. These four biologically important roles
are shown in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16002r10">1.10<!--tex4ht:ref: Intro:fig:DscamAnatomy --></a>.
</p>
<ul class="itemize1">
<li class="itemize">Hemocyte cells of the immune system
</li>
<li class="itemize">Larva Class IV da Neurons
</li>
<li class="itemize">Pupal Mushroom-body neurons in the developing brain
</li>
<li class="itemize">Tetrad synapses of the eye</li></ul>
<!--l. 254--><p class="noindent">During larval development, <span class="phvro8t-x-x-120">Dscam1 </span>is expressed in the da neurons of the larval body
                                                                                

                                                                                
wall. The da neurons create a uniform sensory field that allow larva to respond to
mechanical stimulus. Morphologically, da neurons resemble oak trees growing in a
sunny field. In order to maximize coverage of the field, every {cell::cell} interaction (i.e.
every synapse) must be a productive one. Molecularly, this is accomplished via an
extracellular handshake between two copies of DSCAM1. If this handshake feels too
familiar, a stable, lasting, and <span class="phvro8t-x-x-120">productive </span>synapse is actively discouraged
until a new and different handshake is felt [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWojtowicz2004">Wojtowicz et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWojtowicz2004">2004</a>]. The
use of DSCAM1 to discern self from non-self determination is not unique to
da neurons. It is also essential for an equally critical, and arguably more
complex, nervous systems including the eye and brain. In the developing brain,
<span class="phvro8t-x-x-120">Dscam1 </span>is expressed in both axonal projections of neurons as they extend
from Kenyon cell bodies and bifurcate into the two different mushroom body
lobs [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">Zhan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">2004</a>]. Finally, the involvement of <span class="phvro8t-x-x-120">Dscam1 </span>in the innate
immune system of insects has been demonstrated [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDong2006">Dong et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDong2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">Watson
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">2005</a>] where DSCAM1 recognizes and antigen via similar self vs non-self
interactions.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-16002r10"></a>
                                                                                

                                                                                
<!--l. 258--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis8x.png" alt="PIC" class="graphics" width="384.43625pt" height="499.8675pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis8x.png" src="Figures/Intro/DscamAnatomy.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.10:    </span><span class="content">Important    sites    of    <span class="phvro8t-x-x-120">Dscam1    </span>expression    in    <span class="phvro8t-x-x-120">Drosophila</span>
<span class="phvro8t-x-x-120">melanogaster</span><br class="newline"><span class="phvro8t-x-x-120">Dscam1 </span>has  been  high-studied  in  four  different  regions/cell  types.  (1)
                                                                                

                                                                                
Hemocytes of  the  immune  system,  where  DSCAM1  is  involved  in  antigen
recognition; (2) In Class IV da neurons, which sense mechanical stimulation of
the larval body wall; (3) In mushroom body neurons of the pupal developing
brain; and (4) (not shown) in Tetrad neurons of the eyes. </span></div><!--tex4ht:label?: x1-16002r10 -->
                                                                                

                                                                                
<!--l. 267--><p class="noindent">With the functionally significance of <span class="phvro8t-x-x-120">Dscam1 </span>expression clear, it is important to review
research into the mechanisms of generating tremendous isoform diversity. Soon after
the initial <span class="phvro8t-x-x-120">Dscam1 </span>report <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCelotto2001">Celotto and Graveley</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCelotto2001">2001</a>] investigated the genes
developmental regulation. They focused on the 12 variants of cluster 4 and observed
regulation of exon 4.2. Embryonic transcripts show little inclusion of this exon, while
adult transcripts show frequent inclusion. Exon 4.8 demonstrates the opposite
behavior. Similar regulation of cluster 4 exons was also observed in a closely related
species, <span class="phvro8t-x-x-120">Drosophila yakuba</span>.
</p><!--l. 269--><p class="noindent">In 2004, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>] used a specially designed microarray to robustly
characterize <span class="phvro8t-x-x-120">Dscam1 </span>molecular diversity. They observed, at some rate, inclusion of
virtually all alternative exons from clusters 4, 6, and 9. Additionally, they examined
<span class="phvro8t-x-x-120">Dscam1 </span>transcripts obtained from colonies grown from single cells and reported that
multiple <span class="phvro8t-x-x-120">Dscam1 </span>transcripts were expressed per cell, estimating between 7–50
different combinations per cell, depending on the cell type. As discussed above, the
use of microarrays to perform this analysis precluded observing any potential
coordination between variant exons.
</p><!--l. 271--><p class="noindent">Quickly after <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>] published their results, the Zipursky lab also
published a microarray study of <span class="phvro8t-x-x-120">Dscam1 </span>isoforms [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">Zhan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">2004</a>]. They focused
their analysis on neurons of the developing Mushroom body (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-16002r10">1.10<!--tex4ht:ref: Intro:fig:DscamAnatomy --></a>). Not only
did they also show that most <span class="phvro8t-x-x-120">Dscam1 </span>combinations are likely produced at some level,
                                                                                

                                                                                
but that diversity of isoforms is required for bifurcation of neurons into different
lobes of the developing mushroom body. These results highlighted a critical
function for DSCAM1-mediating extracellular interactions via homophillic
binding.
</p><!--l. 273--><p class="noindent">How is mutually-exclusive exon usage among 48 different options possible?
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2005b">Graveley</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2005b">2005</a>] observed a single “Docking site” within the intronic sequence just
5<span class="cmsy-8">&#8242;</span> &nbsp;to exon 6.1. This docking site was conserved among 15 insect species examined,
from closely-related <span class="phvro8t-x-x-120">Drosophila simulans </span>to a distantly-related <span class="phvro8t-x-x-120">Tribolium castaneum</span>
(Red flour beetle). Astonishingly, the docking site was complementary to “selector
sites” within intronic regions just 5<span class="cmsy-8">&#8242;</span>&nbsp;of each of the 48 variant exons. A model was
proposed where {docking::selector} interaction is required to choose which variant
exons is included, while a splicing regulator protein, likely an hnRNP due to the
repressive nature of the interaction, binds to unused selector sites contained in the
pre-mRNA [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2000">Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraveley2000">2000</a>]. Additional mechanisms have been reported for other
clusters, including the <span class="phvro8t-x-x-120">iStem </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKreahling2005">Kreahling and Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKreahling2005">2005</a>] in cluster 4, and the
hnRNP protein hrp36 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOlson2007">Olson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOlson2007">2007</a>].
</p><!--l. 275--><p class="noindent">[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>] examined <span class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells, and their
results clearly show reduced variability in cluster 9 inclusion. Virtually all of the signal
obtained from hemocyte cells for cluster 9 was seen in variants 9.[6,9,13,30,and 31].
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">Watson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">2005</a>] also examined <span class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells,
                                                                                

                                                                                
comparing it to that of neuronal cells. They propose that secreted forms of
<span class="phvro8t-x-x-120">Dscam1 </span>are essential for a robust innate immune system in insects. These
studies highlight how nature has applied one gene that produces extreme
molecular diversity to multiple problems involving determining self from non-self
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2008">Hattori et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShi2012a">Shi and Lee</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShi2012a">2012</a>]. <span class="phvro8t-x-x-120">Dscam1 </span>use in these two very
different biologically roles has been summarized previously [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHemani2012">Hemani and
Soller</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHemani2012">2012</a>].
</p><!--l. 277--><p class="noindent">In 2007 the Zipersky lab published [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2007">Hattori et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2007">2007</a>] the first in a series of quality
genetic reports describing the function and diversity of <span class="phvro8t-x-x-120">Dscam1</span>. Using homologous
recombination, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2007">Hattori et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2007">2007</a>] showed that <span class="phvro8t-x-x-120">Dscam1 </span>diversity is required for
proper neural wiring but that individual neuronal-isoform identity is not important. Two
years later, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2009">Hattori et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2009">2009</a>] observed that flies capable of expressing at
least 4,752 different <span class="phvro8t-x-x-120">Dscam1 </span>isoforms were indistinguishable from wild-type
controls. This series was recently advanced with another tour-de-force of
genetic manipulation. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">Miura et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">2013</a>] used a collection of <span class="phvro8t-x-x-120">Dscam1 </span>mutants
allowing for visualization via GFP of specific cluster 4.X variant expression in
real time. They concluded that a single neuron expresses multiple <span class="phvro8t-x-x-120">Dscam1</span>
isoforms over time, and <span class="phvro8t-x-x-120">Dscam1 </span>is expressed via “stochastic and probabilistic”
mechanisms.
</p><!--l. 279--><p class="noindent">Research into <span class="phvro8t-x-x-120">Drosophila melanogaster Dscam1 </span>has provided major advancements
                                                                                

                                                                                
to our understanding of multiple aspects of transcription, including: 1) Exon definition;
2) Alternative Splicing of cassette exons; 3) Neuronal and cellular recognition; and
finally 4) Allowing comparisons between how points 1–3 are accomplished in “simple”
model organisms and humans. See sections <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-280002">2.2<!--tex4ht:ref: SeqZipPaper:sec:Results --></a> for more information concerning
<span class="phvro8t-x-x-120">Dscam1</span>.
<a id="x1-16003r20"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.4 </span> <a id="x1-170004"></a>Nucleic Acid Ligation</h3>
<!--l. 284--><p class="noindent">Biomedical research has always been advanced by countless technologies. Section
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-60002">1.2<!--tex4ht:ref: Intro:sec:Nucleic Acid Sequencing --></a> discusses implications of cheap DNA and RNA sequencing. This section
discusses how the ability to <span class="phvro8t-x-x-120">join </span>pieces nucleic acid has also advanced our
understanding of biology. A particular focus is placed on an enzyme with relevance to
Chapters <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a>—T4 RNA Ligase 2.
<a id="x1-17001r32"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.4.1 </span> <a id="x1-180001"></a>RNA-templated DNA-DNA ligation</h4>
<!--l. 289--><p class="noindent">In the late 1960’s and early 1970’s, the Lehman and Richardson labs characterized
two workhorse-enzymes of modern molecular biology. Robert Lehman and
colleagues, working at Stanford Medical School, first described the activity of
                                                                                

                                                                                
<span class="phvro8t-x-x-120">polynucleotide-joining enzyme </span>from <span class="phvro8t-x-x-120">Escherichia coli </span>(now known as <span class="phvro8t-x-x-120">E. Coli </span>DNA
Ligase) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOlivera1967b">Olivera and Lehman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOlivera1967b">1967</a>]. Work on this enzyme paralleled that
from the Richardson lab at Harvard Medical School, where they focused on
<span class="phvro8t-x-x-120">polynucleotide ligase </span>from <span class="phvro8t-x-x-120">Escherichia coli </span>infected with T4 bacteriophage
(now known as T4 DNA ligase) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeiss1967a">Weiss and Richardson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeiss1967a">1967</a>]. It became
clear that while these two enzyme’s shared a common mechanism—later
elucidated by [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModrich1973a">Modrich et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModrich1973a">1973</a>]—they had important differences. First,
T4 DNA ligase required ATP as a cofactor, which <span class="phvro8t-x-x-120">E. Coli </span>DNA Ligase did
not (it was later discovered that DNA ligase required NAD as a cofactor).
Second, only T4 DNA ligase could catalyze ligation of blunt-ended DNA
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTabor1987a">Tabor</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTabor1987a">1987</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-18001r11"></a>
                                                                                

                                                                                
<!--l. 293--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis9x.png" alt="PIC" class="graphics" width="374.39874pt" height="518.93874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis9x.png" src="Figures/Intro/LigationMechanism.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.11: </span><span class="content">Mechanism of ATP-dependent ligation<br class="newline">Adapted from [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>] and specifically for that of T4 RNA
                                                                                

                                                                                
ligase 2. </span></div><!--tex4ht:label?: x1-18001r11 -->
                                                                                

                                                                                
<!--l. 302--><p class="noindent">The general mechanism of ligation (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-18001r11">1.11<!--tex4ht:ref: Intro:fig:Ligation Mechanism --></a>) involves three steps: Step 1 (A) the
<span class="cmmi-12">&#1013;</span>-amino group from the active site lysine performs a nucleophilic attack on the
<span class="cmmi-12">&#945;</span>-phosphate of ATP in solution. B) The ligase is now charged with AMP and inorganic
phosphate (PPi) is freed into solution. C) Step 2: Nucleophilic attack by the 5<span class="cmsy-8">&#8242;</span>&nbsp;DNA
phosphate on the 3<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick to the AMP:ligase phosphate. D) Adenylated
DNA is now competent for DNA ligation. E) Step 3: the 3<span class="cmsy-8">&#8242;</span>&nbsp;OH on the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of
the nick performs a nucleophilic attack on the 5<span class="cmsy-8">&#8242;</span>&nbsp;PO<sub>
<span class="cmr-8">4</span></sub> across the DNA nick,
liberating AMP into solution. F) Sealed nick resulting in: Ligase; AMP; and intact
dsDNA.
</p><!--l. 304--><p class="noindent">In addition to elucidating the general mechanism of ligation, it was also discovered
that T4 DNA ligase lacks a preference for terminal polynucleotide structures. The
Khorana and Richardson labs both reported the activity of this enzyme on
combinations of RNA and DNA duplexes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFareed1971">Fareed et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFareed1971">1971</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKleppe1970b">Kleppe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKleppe1970b">1970</a>].
Both of these papers describe an activity of T4 DNA ligase, RNA-templated
DNA-DNA ligation, that is of particular relevance to this thesis work. Unlike T4 DNA
ligase, <span class="phvro8t-x-x-120">E. Coli </span>DNA Ligase, will not join DNA strands on an RNA template [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">Bullard
and Bowater</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">2006</a>]. Soon after demonstrating these activities <span class="phvro8t-x-x-120">in vitro</span>, the Khorana
lab reported detection of organism-generated DNA [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBesmer1972b">Besmer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBesmer1972b">1972</a>], setting up
an orthogonal field (respective to PCR) of nucleic acid sequence characterization
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XConze2009c">Conze et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XConze2009c">2009</a>].
                                                                                

                                                                                
</p><!--l. 306--><p class="noindent">An enzyme that can catalyze an RNA-templated DNA-DNA ligation is a very useful
molecular biology tool for two main reasons. First, using RNA as a ligation guide
means no modification is made to the template molecule. This contrasts
cDNA analysis, where the RNA has been enzymatically converted by reverse
transcription, potentially losing valuable RNA-coded information, such as
modified bases. Second, synthesis of the DNA probes used in ligation is
inherently easier and cheaper compared to synthesis of RNA probes (see
section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-880002">5.3.2<!--tex4ht:ref: Disc:subsec:LNA-Containing ligamers and T39A Rnk2 --></a>). In addition to being cheaper, synthesis of DNA probes has
become high-throughput since the adoption of microarrays as a standard
gene expression measurement tool [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchena1995a">Schena et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchena1995a">1995</a>]. A pair of papers
from the Landegren lab first reported the utility of RNA-templated DNA-DNA
ligation for analysis of RNA transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">Nilsson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">2001</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2000">2000</a>]. The Fu lab
applied this approach in a multiplex experimental design in collaboration
with Illumina [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2012c">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2012c">2012a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">Yeakley et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">2002</a>], while the Nilsson and
Landegren labs developed a single molecule application [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XConze2010">Conze et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XConze2010">2010</a>]. It
is important to note that <span class="phvro8t-x-x-120">all </span>of these studies used T4 DNA ligase. Clearly,
there is interest and utility in analyzing RNA in both high-throughput and
multiplex experimental designs, using cheap DNA probes, and without cDNA
conversion.
</p><!--l. 308--><p class="noindent">For more than 40 years after its first description, T4 DNA ligase was the only choice
for RNA-templated DNA-DNA ligation. However, a recent publication from New
                                                                                

                                                                                
England Biolabs (NEB) describes this activity by another well-studied ligase, Chlorella
Virus PBCV-1 DNA ligase (herein Chlorella DNA ligase) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">Lohman et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">2013</a>].
Chlorella DNA ligase is a long-studied enzyme and had been reported to <span class="phvro8t-x-x-120">not </span>display
RNA-templated DNA:DNA ligation activity [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo1997b">Ho et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo1997b">1997</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSriskanda1998c">Sriskanda and
Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSriskanda1998c">1998</a>]. However, at high enough concentrations and under special buffer
conditions (specifically a critical concentration of ATP), <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">Lohman et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">2013</a>] have
shown that Chlorella DNA ligase will join two DNA strands hybridized to
an RNA template. They further demonstrated that it performs no worse in
this activity than traditional T4 DNA ligase [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">Nilsson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">2001</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">Yeakley
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">2002</a>].
</p><!--l. 310--><p class="noindent">Building on the list of available enzymes that join hybrid polymer substrates Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a>
presents data supporting RNA-templated DNA-DNA ligation activity for another
enzyme, T4 RNA Ligase 2.
<a id="x1-18002r36"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.4.2 </span> <a id="x1-190002"></a>T4 RNA Ligase 2</h4>
<!--l. 315--><p class="noindent">Proteins of the T4 and T7 bacteriophages have been a boon for molecular biology.
Without enzymes like polynucleotide kinase [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRichardson1965a">Richardson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRichardson1965a">1965</a>], T7 RNA
polymerase [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSummers1970b">Summers and Siegel</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSummers1970b">1970</a>], and T4 DNA ligase [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeiss1967a">Weiss and
                                                                                

                                                                                
Richardson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeiss1967a">1967</a>], many essential manipulations of nucleic acids would have been
impossible for decades. Obviously, these enzymes also have essential phage
functions. T7 RNA polymerase is responsible for late stage replication of T7
phage transcripts, while T4 PNK works in concert with T4 DNA and RNA
ligases to repair cleaved nucleic acids resulting from bacterial pathogens
defense systems [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2002b">Wang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2002b">2002</a>]. Specifically, T4 RNA ligase 1 (herein
“Rnl1”, also known as <span class="phvro8t-x-x-120">gene 63</span>) maintains phage replication by repairing
tRNAs cleaved by an anticodon nuclease produces from the <span class="phvro8t-x-x-120">prr </span>locus [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAmitsur1987d">Amitsur
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAmitsur1987d">1987</a>].
</p><!--l. 317--><p class="noindent">Given the utility and importance of these enzymes, novel enzyme discovery is a
fruitful area of research. The Shuman lab has a distinguished record of discovering
and characterizing numerous such enzymes, including many involved in nucleic acid
synthesis, modification, and repair. Through a BLAST search looking for novel
ligases with sequences related to <span class="phvro8t-x-x-120">Trypanosoma brucei </span>RNA-editing ligases
TbMP52 and TbMP48 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">Ho and Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">2002</a>], they identified motifs in correct
arrangement, spacing, and number indicative of an RNA ligase. The gene,
identified as <span class="phvro8t-x-x-120">gp24.1</span>, has quickly become an essential tool in the era of modern
genomics.
</p><!--l. 319--><p class="noindent">Initial biochemical purification and characterization of <span class="phvro8t-x-x-120">gp24.1 </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">Ho and Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">2002</a>]
revealed that it indeed codes for an RNA ligase, which was renamed T4 RNA ligase 2
                                                                                

                                                                                
(herein “Rnl2”). Rnl2 is a 374 amino acid monomeric protein composed of 2 distinct
domains initially purified as a 42-kDA His-tagged recombinant protein. The N-terminal
domain (1–243) is responsible for steps (1) and (3) of the general ligation
mechanisms (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-18001r11">1.11<!--tex4ht:ref: Intro:fig:Ligation Mechanism --></a>), while the C-terminal domain (244–329) is responsible for
adenylation of the 5<span class="cmsy-8">&#8242;</span>&nbsp;PO<sub>
<span class="cmr-8">4</span></sub> on the 5<span class="cmsy-8">&#8242;</span>&nbsp;residue at the 3<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick, as shown
in step (2). Additionally, Rnl2 is routinely purified as a pre-adenylated and
immediately poised for its first ligation. In contrast to the N-terminal domain,
which is composed of motifs typical to main ligases, the C-terminal domain is
not contained in other DNA ligases. While the biological function of Rnl1
is known, the biological function of Rnl2 remains a mystery more than 12
years after its discovery [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">Chauleau and Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">2013</a>]. However, there is
some speculation that the flurry of research into bacterial CRISPR phage
defense may reveal a role for Rnl2 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarrangou2007c">Barrangou et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarrangou2007c">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">Chauleau and
Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">2013</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-19001r12"></a>
                                                                                

                                                                                
<!--l. 323--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis10x.png" alt="PIC" class="graphics" width="355.3275pt" height="292.09125pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis10x.png" src="Figures/Intro/Rnl2_Structure.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.12: </span><span class="content">Structure and active site of pre-adenylated of Rnl2<br class="newline">Rnl2 as crystalized and described by [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>]. Structures
from {PDB:2HVQ}  were  generated  with  PyMol.  Top  left)  Rnl2  is  composed
of a C-terminal and N-terminal domain. Top Right) The active site of Rnl2 is
highlighted. Bottom left) Active site of Rnl2 as shown from bottom. This face
interacts with substrate. </span></div><!--tex4ht:label?: x1-19001r12 -->
                                                                                

                                                                                
<!--l. 332--><p class="noindent">Mutational analysis of Rnl2, and later a crystal structure of the enzyme, have identified
key functional residues [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2004">Ho et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYin2003d">Yin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYin2003d">2003</a>]. The lysine residue at position 35 (K35) receives the AMP in Step 1. The
K227 residue in the C-terminal domain is essential for both forward and reverse
adenylation of the 5<span class="cmsy-8">&#8242;</span>&nbsp;PO<sub>
<span class="cmr-8">4</span></sub> at the nick [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">Viollet et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">2011</a>]. Mutation of H37
results in an ~102 reduced ligation rate, indicating the important nature of
this residue. Finally, T39 has been shown to interact with the 2<span class="cmsy-8">&#8242;</span>&nbsp;OH on the
3<span class="cmsy-8">&#8242;</span> &nbsp;side of the nick, preferring a C3<span class="cmsy-8">&#8242;</span>&nbsp;endo sugar pucker conformation (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-19002r13">1.13<!--tex4ht:ref: Intro:fig:Rnl2 Active Site Residues --></a>).
</p><!--l. 334--><p class="noindent">Rnl2 has a minimal footprint of 13 nt, centered on the nick, and only requires
magnesium for transfer of AMP to the 5<span class="cmsy-8">&#8242;</span>&nbsp;phosphate. Work done in the Shuman lab
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>] observed that 2<span class="cmsy-8">&#8242;</span>&nbsp;deoxyribose residues on the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of
the nick (i.e. DNA) adopt an RNA-like sugar pucker, leading to the correct orientation
of the 3<span class="cmsy-8">&#8242;</span> &nbsp;OH relative to the AMP leaving group and resulting in ligation. This
conformation is of particular importance to results presented in Chapters <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-19002r13"></a>
                                                                                

                                                                                
<!--l. 338--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis11x.png" alt="PIC" class="graphics" width="216.81pt" height="216.81pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis11x.png" src="Figures/Intro/Rnl2_Active_Site_Residues.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.13: </span><span class="content">Active site of T4 RNA Ligase 2 with highlighted residues<br class="newline">Rnl2 complexed   with   nicked   dsDNA   as   crystallized   and   described
by [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>].  Structures  from  {PDB:2HVR}  and  images
generated with PyMol. </span></div><!--tex4ht:label?: x1-19002r13 -->
                                                                                

                                                                                
<!--l. 347--><p class="noindent">While Rnl2 is extremely efficient at high concentration, displaying little or no reversible
chemistry, a modified version of the enzyme containing only the N-terminal domain
and a K227A point mutation (“Truncated mutant”) has no adenyltransferase activity
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">Viollet et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">2011</a>]. In this case, adenyltransferase refers to the ligase transferring
AMP from an adenylated substrate to itself; reverse chemistry of step 2 in
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-18001r11">1.11<!--tex4ht:ref: Intro:fig:Ligation Mechanism --></a>). This mutant has been used in specialized cloning applications
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">Ghildiyal et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2008a">Hafner et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2008a">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">Viollet et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XViollet2011">2011</a>] that take
advantage of this activity. In these reactions, the use of pre-adenylated 3<span class="cmsy-8">&#8242;</span>&nbsp;DNA
adapters allows for selective ligation among already phosphorylated species by
limiting the enzyme-catalyzed transfer of AMP from the adapter to other
phosphorylated species. Use of this truncated mutant to create a hybrid
RNA/DNA molecule has greatly improved high-throughput sequencing work
flows.
</p><!--l. 349--><p class="noindent">Ligation of hybrid substrates (e.g.. DNA-templated RNA:DNA vs DNA-templated
DNA:DNA) have revealed general substrate preferences. DNA ligases appear to
prefer the residue bearing the 5<span class="cmsy-8">&#8242;</span>&nbsp;phosphate on the 3<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick to be
2<span class="cmsy-8">&#8242;</span> &nbsp;deoxyribose, and have a relaxed requirement for the sugar on the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of the
nick. RNA ligases have the reverse preference, demonstrating higher activities when
the 5<span class="cmsy-8">&#8242;</span> &nbsp;strand, 3<span class="cmsy-8">&#8242;</span>&nbsp;OH residue also bears a 2<span class="cmsy-8">&#8242;</span>&nbsp;OH. Rnl2 has an additional
preference for an RNA residue at the penultimate 3<span class="cmsy-8">&#8242;</span>&nbsp;side of a residue [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">Ho and
Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">2002</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2004">Ho et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>]. The two base
                                                                                

                                                                                
requirement for RNA at the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of the double stranded nick biases Rnl2 to join
RNA:[RNA/DNA] strands. Independent labs have measured this preference and have
reported that the RNA-templated DNA-DNA joining activity of Rnl2 is below assay
limits of detection [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">Bullard and Bowater</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">2006</a>]. However, results discussed in
this work clearly show that with enough enzyme and sensitive downstream
measurements, Rnl2 will catalyze RNA-templated DNA:DNA ligation (see Chapters
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a>). Previous reports of Rnl2 lacking this activity are likely due to a
single turnover mechanism in this reaction imposed by a non-natural sugar
pucker of the ligated DNA kinetically trapping the enzyme on the ligated
duplex.
<a id="x1-19003r38"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.4.3 </span> <a id="x1-200003"></a>Ligases as molecular tools</h4>
<!--l. 353--><p class="noindent">Section describes the identification and development of ligases as tools in
molecular biology. Ligation of templated duplexes has multiple uses in cloning
and sequence characterization. The following section (<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-210005">1.5<!--tex4ht:ref: Intro:sec:Nucleic Acid Polymers --></a>) discusses long
nucleic acid polymers, specifically mammalian piRNA precursor transcripts.
Little biology is known concerning these long transcripts. Chapters <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a> discuss the application of Rnl2 to characterization of long nucleic acid
polymers. This Chapter’s final section (<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-250006">1.6<!--tex4ht:ref: Intro:sec:Unique Time in Omics --></a>) discusses what is in effect <span class="phvro8t-x-x-120">in silico</span>
                                                                                

                                                                                
ligation of sequence reads–Assembly of full-length transcripts from short
reads.
<a id="x1-20001r35"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.5 </span> <a id="x1-210005"></a>Nucleic Acid Polymers</h3>
<!--l. 358--><p class="noindent"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFire1998">Fire et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFire1998">1998</a>] brought small RNAs to the forefront of research. Recently lncRNA
research has been in similarity exciting period [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2009">Guttman et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKhalil2009">Khalil
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKhalil2009">2009</a>]. Whether “small” or “long” all classes of RNA are polymers of
ribonucleotides. This section will focus on an interesting class of nucleic acid
polymer—Mammalian piRNA precursor transcripts. These transcripts, which share
similarities to traditional mRNAs, are processed down piRNAs. Finally, the section
ends with a history of transcript assembly of HTS data, without which the study of
both types of polymer would not be possible.
<a id="x1-21001r41"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.5.1 </span> <a id="x1-220001"></a>It Started Small: Mammalian piRNAs</h4>
<!--l. 362--><p class="noindent">piRNAs are small RNAs 23–35 nt long. They are longer than other small RNAs (e.g..
miRNAs or siRNAs) and are believed to derive from single-stranded RNA
precursors because unlike other small RNAs, their biogenesis does not the require
                                                                                

                                                                                
double-stranded RNA-specific ribonuclease Dicer [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">Houwing et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2006">Vagin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2006">2006</a>]. Yet, similar to other small RNAs, they bind a sub group of the
Argonaute family of proteins, PIWI proteins, from which their name is derived (<span class="phvro8t-x-x-120">PIWI</span>
<span class="phvro8t-x-x-120">Interacting RNAs</span>).
</p><!--l. 364--><p class="noindent"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2001">Aravin et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2001">2001</a>] first identified piRNAs in <span class="phvro8t-x-x-120">Drosophila melanogaster </span>via
northern blot probes from the <span class="phvro8t-x-x-120">Su(Ste) </span>locus as heterogeneous 25–27 RNAs
essential for silencing of <span class="phvro8t-x-x-120">Stellate </span>and, more importantly, male fertility. In the
years since the initial report, piRNAs have been cataloged, characterized,
manipulated and mutated. Yet, very little about what they do is known. What is
known is that without a functioning piRNA pathway male mice are also sterile
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">2004</a>]. Studies in humans
have correlated SNPs in human PIWI proteins and decreased fertility [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGu2010a">Gu
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGu2010a">2010</a>]. What is unknown includes biogenesis of initial transcript to small RNA,
physiological targets, and terminal function of sterility maintenance. Especially little is
known concerning mammalian piRNAs, on which the following section will
focus.
</p><!--l. 366--><p class="noindent">Virtually all known concerning piRNA biogensis is infered from flies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHirose2014">Hirose
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHirose2014">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">Luteijn and Ketting</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSiomi2011">Siomi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSiomi2011">2011</a>]. Where parallels break
down is what is known concerning <span class="phvro8t-x-x-120">function</span>. The most famous function for piRNAs in
<span class="phvro8t-x-x-120">Drosophila melanogaster </span>is suppression of transposon transcripts during
                                                                                

                                                                                
gameotogensis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMalone2009">Malone and Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMalone2009">2009</a>]. The Ping-Pong model elegantly
explains how this might be accomplished: cyclic cleavage of transposon transcripts
and piRNA precursor transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">Gunawardane
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">2007</a>]. Yet for mammals, it appears that piRNAs have diversified beyond just
transposon silencing.
</p><!--l. 368--><p class="noindent">Four reports in 2006 defined the beginning of mammalian piRNA research [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">Aravin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">Grivna et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>]. Each
observed small RNA species, most are 23–35 nt long, that bound PIWI proteins. They
also noticed that when aligned to the mouse genome, most piRNAs map to the
genome as “clusters” of discrete genomic loci. Overtime, it became clear that
mammalian piRNAs can divided into three major classes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-22001r14">1.14<!--tex4ht:ref: Intro:fig:Mammalian piRNA classes --></a>). There
are also three PIWI proteins in mice, each displaying a distinct expression
profile during development and an association with piRNAs of a specific
length.
                                                                                

                                                                                
                                                                                </p>
<hr class="figure"><div class="figure">
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <a id="x1-22001r14"></a>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 373--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis12x.png" alt="PIC" class="graphics" width="477.78499pt" height="373.39499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis12x.png" src="Figures/Intro/MammalianPiRNAClassesOverTime.eps"  
-->
<br> </p><div class="caption"><span class="id">Figure&nbsp;1.14: </span><span class="content">Overall view of the three classes of mammalian piRNAs. Figure design assisted by <a href="mailto:xin.li@umassmed.edu">Xin Zhiguo Li</a>. </span></div><!--tex4ht:label?: x1-22001r14 -->
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 381--><p class="noindent"><span class="phvro8t-x-x-120">Fetal piRNAs </span>are present before birth. These piRNAs tend to be short, bind the PIWI
protein MIWI2 (PIWIL4) in mice, and have sequences found in transposable elements
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">Carmell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">2007</a>]. Quickly following MIWI2 in expression is the second PIWI
protein expressed MILI (PIWIL2). It is during the “fetal” stage of piRNA biogenesis in
mice that MIWI2 and MILI undergo ping-pong amplification, similar to that
observed in flies, in order to silence expression of transposons during germ line
formation [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">Aravin and Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">2004</a>].
</p><!--l. 383--><p class="noindent">The next class of piRNAs, historically but confusing grouped with the previous class,
are called <span class="phvro8t-x-x-120">Pre-pachytene piRNAs</span>. Pre-pachytene piRNAs are expressed just before
birth and continue to be expressed in functioning testes. These piRNAs tend to map
to traditional, annotated, protein-coding genes. After birth, and during the “neonatal
stage” only MILI is expressed, and piRNAs shift from mostly transposon-mapping to
3<span class="cmsy-8">&#8242;</span> &nbsp;UTR mapping [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">Robine et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">2009</a>].
</p><!--l. 385--><p class="noindent">Finally, due to their unique sequence in the genome, the genetic origin of millions of
piRNAs belonging to the third class, the <span class="phvro8t-x-x-120">pachytene piRNAs</span>, was immediately known.
Once the “first wave” of spermatogenesis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLaiho2013a">Laiho et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLaiho2013a">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOakberg1956b">Oakberg and
Oakberq</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOakberg1956b">1956</a>] reaches the pachytene stage of meiosis I when chromosomes pair
up, cross over, and exchange genetic material, the “pachytene piRNAs” are
expressed [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">2013a</a>]. Pachytene
                                                                                

                                                                                
piRNAs are extremely abundant compared to fetal and pre-pachytene piRNAs. They
bind another Piwi protein MIWI (PIWIL1). The genomic origin of pachytene piRNAs,
often unique in terms of genomic sequence, map within “gene deserts”—unannotated
and devoid of introns. This is comparable to piRNA clusters in flies, such as
<span class="phvro8t-x-x-120">flamenco</span>, whose transcription can be abolished with a P-element insertion
into a putative promoter, as measured by northern blot looking for piRNAs
generated 168 kb downstream (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-22002r15">1.15<!--tex4ht:ref: Intro:fig:flamenco --></a> [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGoriaux2014">Goriaux
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGoriaux2014">2014</a>]. This gene architecture makes the pachytene piRNA loci some of the
most interesting RNA-producing regions of the mammalian genome. These
clusters, (aka: “piRNA-producing loci” or “genes”) appear to produce single,
continuous, relatively long, and un-spliced Pol II transcripts (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a>) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">Li
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">2013a</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-22002r15"></a>
                                                                                

                                                                                
<!--l. 389--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis13x.png" alt="PIC" class="graphics" width="375.4025pt" height="234.87749pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis13x.png" src="Figures/Intro/FlamencoLocus.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;1.15:  </span><span class="content">A  the  <span class="phvro8t-x-x-120">Drosophila  melanogaster  </span>gene  <span class="phvro8t-x-x-120">flamenco  </span>is  a  graveyard
for transposon sequences [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPelisson1994">Pélisson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPelisson1994">1994</a>]. Evidence for expression of a
single-contiguous RNA transcript from <span class="phvro8t-x-x-120">flamenco </span>(A) is provided by a P-element
insertion into the suspected promoter region (B). [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>] could
not detect specific piRNAs (red X’s) by northern blot in the P-element mutant. </span></div><!--tex4ht:label?: x1-22002r15 -->
                                                                                

                                                                                
<a id="x1-22003r43"></a>
<h4 class="subsectionHead"><span class="titlemark">1.5.2 </span> <a id="x1-230002"></a>From Long to Small: Precursor processing to mature piRNAs</h4>
<!--l. 400--><p class="noindent">The process by which a long, single-stranded piRNA precursor transcript become
small mature piRNA is full of black boxes and question marks. While we do know
they are long, single-stranded piRNA transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">Li
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">2013b</a>], we are very unsure of many steps between transcription and terminal
function of {piRNA::PIWI} complexes (PIWI-piRISC).
</p><!--l. 402--><p class="noindent">For example, how do piRNA precursor transcripts exit the nucleus? This is not known
in mice, but there are clues from <span class="phvro8t-x-x-120">Drosophila melanogaster</span>, where some piRNA
clusters are bidirectionally transcribed and bound by the HP1 homologue Rhino
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKlattenhoff2009">Klattenhoff et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKlattenhoff2009">2009</a>]. Rhino co-localizes with the DEAD box protein UAP56 near
the perinuclear compartment known as nuage [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">Zhang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">2012a</a>]. It is believed
that Rhino and UAP56 assist in a hand off of large precursor transcripts across the
nuclear envelope where they are bound by the nuage protein VASA [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">Zhang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">2012a</a>].
</p><!--l. 404--><p class="noindent">Once precursor transcripts exit the nuclease they likely enter chromatoid bodies
(comparable to nuage in flies) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLim2007">Lim and Kai</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLim2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeikar2011">Meikar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeikar2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">Zhang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">2012a</a>]. There they are then believed to be “fragmented” into shorter <span class="phvro8t-x-x-120">piRNA</span>
<span class="phvro8t-x-x-120">intermediates </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">2013c</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSaito2010">Saito et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSaito2010">2010</a>] by the protein MitoPLD (aka:
                                                                                

                                                                                
PLD6) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011a">Watanabe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011a">2011a</a>]. Structures of the fly homologue of MitoPLD,
<span class="phvro8t-x-x-120">Zucchini</span>, support slicing activity <span class="phvro8t-x-x-120">in vitro </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">Ipsaro et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">Nishimasu
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">2012</a>], its activity has yet to be shown <span class="phvro8t-x-x-120">in vivo </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">Luteijn and Ketting</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">2013</a>].
Fragmentation may, or may not, impart the 5<span class="cmsy-8">&#8242;</span>&nbsp;U preference seen in mature piRNAs
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">Gunawardane et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">2007</a>] and indeed Zucchini does not
show a 5<span class="cmsy-8">&#8242;</span> &nbsp;U bias <span class="phvro8t-x-x-120">in vitro </span>[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">Ipsaro et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">Nishimasu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">2012</a>]. However, this
preference may result from downstream sequence preference of PIWI-protein binding
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCora2014">Cora et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCora2014">2014</a>].
</p><!--l. 406--><p class="noindent">Once fragmented into shorter RNAs, piRNA intermediates seem to be “loaded,” into a
specific time- and expression-appropriate PIWI proteins (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-22001r14">1.14<!--tex4ht:ref: Intro:fig:Mammalian piRNA classes --></a>). Following
“loading,” piRNA intermediates are trimmed down to the length characterstic of bound
Piwi by the appropriately named, but <span class="phvro8t-x-x-120">hypothetical</span>, enzyme “Trimmer” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">Li
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">2013c</a>]. Both “Loading” and “Trimmer” activity have not been shown in
mammalian systems but are inferred from Silk worm (<span class="phvro8t-x-x-120">Bombyx mori</span>) cellular
extracts of ovary-derived BmN4 cells [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2009">Kawaoka et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2009">2009</a>]. Once trimmed,
piRNAs are methylated on the 2<span class="cmsy-8">&#8242;</span>&nbsp;OH position by the enzyme HEN1 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorwich2007">Horwich
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorwich2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2011">Kawaoka et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKirino2007">Kirino and Mourelatos</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKirino2007">2007</a>]. At this point, a
mature piRNA, complexed with a PIWI protein (PIWI-piRISC), is poised to perform
cellular function(s).
</p>
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-23001r16"></a>
                                                                                

                                                                                
<!--l. 410--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/mammalian_piRNA_pathway.png" alt="pict" width="72.26999pt">
<br></p><div class="caption"><span class="id">Figure&nbsp;1.16:   </span><span class="content">Figure   taken   from   [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">Li   et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">2013b</a>]:   A   model   for   piRNA
biogenesis. Primary  piRNA  transcripts  are  transcribed  by  RNA  polymerase
II and contain  5<span class="cmsy-8">&#8242;</span>&nbsp;caps,  exons,  introns,  and  poly(A)  tails.  The  transcription
of pachytene  piRNA  genes  is  controlled  by  A-MYB;  transcription  factor(s)
(TF) controlling pre-pachytene piRNA genes remain to be discovered. Current
models of piRNA  biogenesis  propose  that  PLD6  determines  the  5<span class="cmsy-8">&#8242;</span>&nbsp;end  of
piRNA intermediates with lengths &gt;30 nt. These intermediates are proposed
to then be loaded  into  PIWI  proteins.  After  PIWI  binding,  a  nuclease  is
thought to trim  the  3<span class="cmsy-8">&#8242;</span>&nbsp;end  of  the  piRNA  to  the  length  characteristic  of
the particular  bound  PIWI  protein.  Finally,  further  trimming  is  prevented  by
addition of a  2<span class="cmsy-8">&#8242;</span>&nbsp;O-methyl  group  to  the  3<span class="cmsy-8">&#8242;</span>&nbsp;end  of  the  mature  piRNA  by  the
S–adenosylmethionine-dependent  methyltransferase  HEN1.  Figure  adapted
from [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013">2013c</a>]. </span></div><!--tex4ht:label?: x1-23001r16 -->
                                                                                

                                                                                
<!--l. 418--><p class="noindent">What are the cellular activities of PIWI-piRISC? MILI and MIWI2 have been shown to
direct epigenetic LINE1 and IAP transposon silencing in the embryonic male germline
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007">2007b</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">Carmell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi2008">Kuramochi-Miyagawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi2008">2008</a>]. Two
studies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeFazio2011">De Fazio et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeFazio2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">Reuter et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">2011</a>] used point mutations in the
catalytic triad of MIWI, MIWI2, and MILI to remove slicer activity. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeFazio2011">De Fazio
et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeFazio2011">2011</a>] found that MIWI2-deficient mice are fertile, silence transposons, and
display all signs of secondary piRNA biogenesis and concluded that MILI
(which is sterile) was required for transposon silencing. This finding was
later elaborated upon by <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDiGiacomo2013">Di Giacomo et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDiGiacomo2013">2013</a>] to work in concert with
other forms of epigenetic silencing to repress LINE1 expression. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">Reuter
et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">2011</a>] focused on MIWI and found that it required for silencing of LINE1
transcripts long after they were epigentically silenced (i.e. in the adolescent
mouse).
</p><!--l. 420--><p class="noindent">The above studies point to a familiar scenario of piRNA-mediated target cleavage
and/or transcriptional silencing by PIWI-piRISC [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeister2013">Meister</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeister2013">2013</a>]. Yet confusingly,
HITS-CLIP of MIWI revealed that MIWI binds spermiogenic mRNAs without a piRNA
guide [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>] and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">Reuter et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">2011</a>] demonstrated that slicing of
target by MIWI RISC requires near perfect binding.
</p><!--l. 422--><p class="noindent">How does does one reconcile these findings with the extremely uniquely-mapping
quality of virtually all pachytene piRNAs? Put another way, if MIWI requires near
                                                                                

                                                                                
perfect pairing between guide and target, and pachytene piRNAs perfectly pair with
nothing else in the genome but antisense transcripts from their own loci, what is the
mechanism target recognition? In fact, when taking all studies on the function of
mouse piRNAs, frustratingly little is actually known or internally consistent.
Indeed, the catalytic nature of PIWI proteins is a debated topic [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">Luteijn and
Ketting</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLuteijn2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeister2013">Meister</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeister2013">2013</a>]. A recent report that the DNA modification 5meC is
high in piRNA intergenic gene bodies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2013">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2013">2013</a>], combined with known
functions of fetal PIWI-piRISC alludes to a function for self-mapping pachytene
piRNAs.
</p><!--l. 424--><p class="noindent">Perhaps the site of PIWI-piRISC function is not cytoplasm. Indeed, fly PIWI is
localized in the nucleus, and MILI and MIWI2 have been shown in induce DNA
methylation [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCox2000">Cox et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCox2000">2000</a>]. This is a potentially misleading
course of logic. Localization does not confirm interaction [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNorth2006">North</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNorth2006">2006</a>] or function
and inferring such from localization can be as dangerous as assuming cars function
in parking lots. Finally, a extremely tantalizing additional potential function for
mammalian piRNAs is that of genomic imprinting [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011">Watanabe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011">2011b</a>]. This
function is in good agreement with germ line-specific and developmentally timed
nature of Piwi protein expression.
</p><!--l. 426--><p class="noindent">In summary, there are still many holes and black boxes in the story of mammalian
piRNAs. Continued study is easily justified by the sterile phenotypes of all pathway
                                                                                

                                                                                
mutants. Time will tell if mammalian piRNAs are involved in a satisfying process of
biology or are crude side quest of Nature.
<a id="x1-23002r46"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">1.5.3 </span> <a id="x1-240003"></a>From Short to Long: Transcript Assembly</h4>
<!--l. 432--><p class="noindent">Initial genome-wide HTS of piRNAs revealed a tremendous amount of biology
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">Gunawardane et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">2007</a>], but could provide little
information as to the original transcriptional unit. The ability to reconstruct piRNA
precursors had to wait for technological improvements in HTS read length and
alignment algorithms.
</p><!--l. 434--><p class="noindent">Working backwards from small RNA-Seq data to original transcription units was
impossible. Mammalian piRNAs are too short (~30 nt) to allow for quality assembly
using even the most current algorithms. They simply do not provide the necessary
overlap of sequence to build scaffolds. Also, repeat elements are extremely abundant
in mice [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNellaker2012">Nellå&nbsp;ker et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNellaker2012">2012</a>], and combined with short reads further reduce the
ability to assemble full-length sequences. Therefore, it was necessary to sequence
RNAs prior to mature piRNA formation.
</p><!--l. 436--><p class="noindent">Even with longer read lengths and the best assembly algorithms, the 5<span class="cmsy-8">&#8242;</span>&nbsp; and 3<span class="cmsy-8">&#8242;</span>&nbsp; ends
of long and diverse transcripts like piRNA precursors often requires a combination of
                                                                                

                                                                                
multiple HTS datatypes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlower2013">Blower et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlower2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">2013b</a>]. Tailored versions
of RNA-Seq, such as CAP-Seq (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-80002">1.2.2<!--tex4ht:ref: Intro:subsec: History of HTS --></a>), are not sufficient for
accurate 5<span class="cmsy-8">&#8242;</span> &nbsp;end determination, and require orthogonal datasets to verify TSSs.
Taking a page from lncRNA transcript discovery, orthogonal data sets such as
ChIP-Seq of H3K4 methylated histones, a marker for transcriptional initiation can
supplement RNA expression data [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKhalil2009">Khalil et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKhalil2009">2009</a>]. More information
about how multiple HTS datasets can be—and were used—to define the
transcriptional unit of piRNA precursors transcripts is provided in Chapter
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a>.
</p><!--l. 438--><p class="noindent">General assembly of full length transcripts (not just piRNA precursor transcripts) is
difficult for at least 3 reasons: (1) The transcriptome is expressed across 5 orders of
magnitude and a typical RNA-Seq library contains many reads from a few
highly-expressed genes and many fewer reads from lowly-expressed genes
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlencowe2009">Blencowe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlencowe2009">2009</a>]; (2) RNA-Seq libraries are often not created from a
completely pure source of mRNA and can contain reads from other RNA classes (e.g.
tRNAs) or intronic reads from pre-mRNAs; and (3) Reads are often much shorter
than a typical mRNA, making it difficult to assign which read goes to which isoform of
a given gene (see the “connectivity problem” discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-130003">1.3.3<!--tex4ht:ref: Intro:sec:Isoform Problem --></a>. With these
challenges in mind, what is the current state of transcript reconstruction (herein
<span class="phvro8t-x-x-120">transcript assembly</span>)?
                                                                                

                                                                                
</p><!--l. 440--><p class="noindent">Computational transcriptome assembly of short reads is currently performed in one of
two modes: genome-guided and genome-independent [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGarber2011a">Garber et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGarber2011a">2011</a>]. The
difference between these two approaches is use of a high-quality genome during the
assembly process. Popular assembly programs such as Cufflinks [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">Trapnell
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">2010</a>] and Scripture [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2010">Guttman et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2010">2010</a>], use genome-aligned short reads
as the bases for calling transcripts. Genome-independent methods include Trinity,
Oasis, and Velvet [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaas2013c">Haas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaas2013c">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchulz2012">Schulz et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchulz2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZerbino2008">Zerbino and
Birney</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZerbino2008">2008</a>].
</p><!--l. 442--><p class="noindent">As mentioned previously, constraints imposed by the huge dynamic range of RNA
expression is the major complicating factor with current transcript assembly
programs. These programs frequently generate short transcript fragments (“contigs”)
due to poor coverage of long and lowly-expressed transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRehrauer2013">Rehrauer
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRehrauer2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">Steijger et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">2013</a>]. Merging these contigs into a continuous
annotation is a major goal. Improvements will surely come from greater sequencing
depth, longer reads, and mRNA enrichment schemes, albeit with diminishing returns
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014c">Chang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014c">2014b</a>]. See section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-790002">5.1.2<!--tex4ht:ref: Disc:subsec:need for Tx assembly --></a> for more thoughts concerning transcript
assembly. Longer-term barriers include repetitive sequences, transcript secondary
structure [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWan2014">Wan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWan2014">2014</a>], and mRNA processing including hydrolysis and RT
processivity [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">Sharon et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">2013</a>]. Finally, multiple forms of mRNA enrichment
and purification, specifically combining poly(A)+ tail and 5<span class="cmsy-8">&#8242;</span>&nbsp;CAP selection,
can be used to increase the accuracy of mRNA transcript assembly [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlower2013">Blower
                                                                                

                                                                                
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlower2013">2013</a>].
<a id="x1-24001r42"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">1.6 </span> <a id="x1-250006"></a>Nucleic Acid ’Omics</h3>
<!--l. 447--><p class="noindent">Biomedical science has just taken a very sharp step (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-8001r2">1.2<!--tex4ht:ref: Intro:fig:SeqCosts --></a> into an era
of cheap genomics. Most questions, including those of gene expression,
molecular interactions, and evolution should no longer be asked on a small
scale. Indeed questions and hypothesis concerning these topics should be
approached on a “big scale” from the very onset in order to maximize return
on investment. Much of the work required to reap maximum benefit falls
squarely on our own education and experience. See section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-950004">5.4<!--tex4ht:ref: Disc:sec:Final Thoughts --></a> for concluding
thoughts.
                                                                                

                                                                                
                                                                                

                                                                                
</p>
<h2 class="chapterHead"><span class="titlemark">Chapter&nbsp;2</span><br><a id="x1-260002"></a>Simultaneous analysis of multiple site alternative splicing via RNA-templated
DNA-to-DNA ligation</h2>
<a id="x1-26001r49"></a>
<h3 class="sectionHead"><span class="titlemark">2.1 </span> <a id="x1-270001"></a>Introduction</h3>
<!--l. 9--><p class="noindent">In eukaryotes, genome size does not scale with complexity, in large part due to
expression of alternative mRNA isoforms. High-throughput sequencing has revealed
that ~58% of <span class="phvro8t-x-x-120">Drosophila melanogaster </span>genes and &gt;95% of human genes produce
multiple transcripts per gene [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">Brown et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">Pan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">Wang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">2008</a>], with many human genes expressing 10 or more isoforms [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">Djebali
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">2012</a>]. Isoform diversity is driven by alternative promoter use (i.e., alternative
first exons), alternative splicing at internal sites, and alternative polyadenylation. With
regard to alternative splicing, more than a quarter of human genes contain multiple
alternative splicing regions separated by stretches of constitutively-included exons
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">Fededa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">2005</a>]. The combinatorial potential of such multi-site alternative
splicing exponentially increases the number of possible isoforms, with some
human genes predicted to have &gt;100 isoforms and Drosophila <span class="phvro8t-x-x-120">Dscam1</span>, which
utilizes four regions of mutually exclusive cassette exons, predicted to have
37,224!
</p><!--l. 11--><p class="noindent">Although bioinformatic analysis of high throughput sequencing data has proven
                                                                                

                                                                                
incredibly powerful for identifying individual alternative splicing regions and
characterizing the diversity exon utilization within them, current technology is
limited to ~500 nt of contiguous sequence. Thus complete transcripts must be
intuited by piecing together multiple short reads [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBoley2014">Boley et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBoley2014">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGarber2011a">Garber
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGarber2011a">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">Grabherr et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaas2013c">Haas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaas2013c">2013</a>]. As a result, connectivity
information present in individual mRNA molecules is lost. With regard to distal
alternative splicing regions, this limits our ability to know (1) whether all possible
combinations are actually produced and (2) whether there is any long-range
coordination between different alternative splicing regions.
</p><!--l. 13--><p class="noindent">Linked processing events have been largely observed in reporter constructs as a
dependence of downstream exon inclusion due to promoter-dependent polymerase
speed [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKornblihtt2013">Kornblihtt et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKornblihtt2013">2013</a>]. There are also reports of coordinated endogenous
exon usage [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>], most notably for mouse <span class="phvro8t-x-x-120">Fibronectin </span>(<span class="phvro8t-x-x-120">Fn1</span>)
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">Fededa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">2005</a>]. However, for many genes, including the notorious <span class="phvro8t-x-x-120">Drosophila</span>
<span class="phvro8t-x-x-120">melanogaster Dscam1</span>, no coordination has been observed between distinct splicing
decisions [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">Miura et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>]. Rigorous examination of
coordinated splicing remains technically challenging.
</p><!--l. 15--><p class="noindent">With an end goal of thorough analysis of linked splicing decisions, we designed a
novel experimental approach, SeqZip (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-27001r1">2.1<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F1 --></a>, with which to accurately and rapidly
profile multiple, distant (&gt;1,000 nt) sites of alternative splicing contained in the
                                                                                

                                                                                
same transcript. SeqZip employs RNA-templated DNA ligation of specific
DNA oligonucleotides (oligos), termed “ligamers,” whose targeted sequences
can be separated by hundreds or thousands of nucleotides. Each ~40 nt
ligamer spans the ends of a single alternatively spliced exon, or the beginning
and end of a large block of constitutively included exons, looping out the
sequence in between the ends. Unique ligamer sets hybridized to individual RNA
molecules are then joined by enzymatic ligation with T4 RNA ligase 2 (Rnl2)
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">Ho and Shuman</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHo2002b">2002</a>]. The resultant multi-ligamer product reduces the
sequence space occupied by the looped out regions of the target RNA while
retaining targeted exon connectivity. This connectivity can then be assessed
by either size separation or sequencing of ligation products. SeqZip can
quantitatively report on RNA isoform abundance, and has a usable dynamic
range spanning 6 orders magnitude. Further, SeqZip does not use reverse
transcriptase (RT), so is not subject to the problems associated with RT of long
RNAs.
</p><!--l. 17--><p class="noindent">Here we describe development and validation of SeqZip, its initial application to
investigate potential connectivity among alternatively spliced exons in <span class="phvro8t-x-x-120">Fn1 </span>and its use
to characterize of the immense molecular diversity of <span class="phvro8t-x-x-120">Dscam1</span>. We also suggest other
potential applications for SeqZip, including multi-site SNP detection, multi-site
smFISH probes, and Q-PCR improvements.
</p>
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-27001r1"></a>
                                                                                

                                                                                
<!--l. 21--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis14x.png" alt="PIC" class="graphics" width="409.53pt" height="143.53624pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis14x.png" src="Figures/SeqZipPaper/Roy2014Fig1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.1: </span><span class="content">Principles of the SeqZip Assay<br class="newline">Using an RNA template, custom synthesized DNA oligonucleotide (“ligamers”)
that have either  one,  or  two  regions  of  complementarity  to  the  RNA  are
allowed to hybridize.  Ligamers  containing  one  region  of  complementarity
target the terminal,  flanking,  constant  sequences,  and  also  contain  primer
sequences for subsequent amplification. Internal ligamers contain two regions
of complementarity,  separated  by  a  spacer  sequence.  Hybridization  of  the
internal ligamers encourages the RNA between the hybridization sites to loop
out. Once all ligamers are hybridized, Rnl2 is added in excess, and the ligated
DNA is amplified and analyzed. </span></div><!--tex4ht:label?: x1-27001r1 -->
                                                                                

                                                                                
<a id="x1-27002r51"></a>
<h3 class="sectionHead"><span class="titlemark">2.2 </span> <a id="x1-280002"></a>Results</h3>
<a id="x1-28001r48"></a>
<h4 class="subsectionHead"><span class="titlemark">2.2.1 </span> <a id="x1-290001"></a>Method development and validation</h4>
<!--l. 36--><p class="noindent">The SeqZip assay requires efficient enzymatic ligation of DNA oligos hybridized to a
RNA template (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-27001r1">2.1<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F1 --></a>). Although numerous ligases can join DNA or RNA
fragments hybridized a DNA template [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">Bullard and Bowater</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">2006</a>], only T4 DNA
ligase, and recently Chlorella virus DNA ligase, are known to previously shown to join
DNA fragments hybridized to a RNA template [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">Lohman et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLohman2013c">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">Nilsson
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNilsson2001">2001</a>]. The commonly used T4 DNA ligase has a high proclivity for
promiscuous ligation (NTL) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuhn2005">Kuhn and Frank-Kamenetskii</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuhn2005">2005</a>]. Therefore, we
tested the ability of several other commercially available enzymes to perform ligation
reactions with four or five 5<span class="cmsy-8">&#8242;</span>&nbsp;<sup><span class="cmr-8">32</span></sup>P-labeled 20 nt DNA oligos hybridized to adjacent
positions on either a DNA or RNA template (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>). As expected, all DNA ligases
tested [Tth DNA ligase (Thermo), Tsc DNA ligase (Prokaria), Thermostable DNA
ligase (Bioline), T4 DNA ligase (NEB), and E. coli DNA ligase (NEB)] efficiently
joined all four oligos hybridized to the DNA template, and Rnl2 (NEB) lacked
this activity (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>A, left, lanes 1-6) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">Bullard and Bowater</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBullard2006">2006</a>]. Also
as expected, T4 DNA ligase generated multiple slower migrating products,
                                                                                

                                                                                
likely resulting from non-templated ligation (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>A, left, lane 4). When
the oligos were hybridized to an RNA template, only T4 DNA ligase and
Rnl2 produced ligation products (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>A, left, lanes 11-12). Titration
of both enzymes revealed that Rnl2 had significantly higher activity on the
RNA template than did T4 DNA ligase (data not shown). Further, at enzyme
concentrations that yielded maximal ligation efficiencies after 8 hours, Rnl2 produced
significantly less (7.5-fold) non-templated product than did T4 DNA ligase
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>A, right, compare lanes 9,10 and 18,19), indicating that Rnl2 has a
lower propensity for promiscuous ligation. Further, the inability of Rnl2 to
mediate DNA-templated DNA-DNA ligation minimizes the possibility that
contaminating genomic DNA in biological samples would confound SeqZip
analysis (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>B, left, lane 5). Therefore, we decided to move forward with
Rnl2.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-29001r2"></a>
                                                                                

                                                                                
<!--l. 40--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis15x.png" alt="PIC" class="graphics" width="387.4475pt" height="358.33875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis15x.png" src="Figures/SeqZipPaper/Roy2014Fig2.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.2: </span><span class="content">T4 RNA Ligase 2 will catalyze RNA-templated DNA-to-DNA ligation<br class="newline">See section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-300001">2.2.1.1<!--tex4ht:ref: SeqZipPaper:figCap: Roy2014 F2 --></a> for caption </span></div><!--tex4ht:label?: x1-29001r2 -->
                                                                                

                                                                                
<!--l. 49--><p class="noindent">We next assessed the feasibility of ligating multiple DNA oligos (ligamers), each
spanning a loop in an RNA template (schematized in Figure-2B). Four different
ligamers were constructed to loop out various lengths of a 307 nt transcript. Each 26
nt ligamer consisted of 10 nt hybridizing to either side of the loop separated by a 6 nt
spacer. Ligamers were 5<span class="cmsy-8">&#8242;</span>&nbsp;end labeled with <sup><span class="cmr-8">32</span></sup>P and hybridized to the template RNA
individually, pairwise, in threes, or as a complete set. Ligation products were only
observed when ligamers targeting adjacent RNA sequences were present in the
reaction, and 4-way ligation products were obtained only when all ligamers
were present. Thus DNA oligos designed to loop out various lengths of a
template RNA can be used to condense the connectivity information in an RNA
by more than 3 fold (i.e., a 94 nt RNA was condensed to a 26 nt DNA). In
subsequent studies we were able to push this condensation ratio to &gt;49 fold (see
below).
<a id="x1-29002r1"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">2.2.1.1 </span> <a id="x1-300001"></a>Caption for Figure 2.2</h5>
<!--l. 54--><p class="noindent">T4 RNA Ligase 2 will catalyze RNA-templated DNA-to-DNA ligation
</p><!--l. 56--><p class="noindent">(left) A screen of ligases was performed (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). Ligases were incubated
with an RNA or DNA template and a common pool of end-labeled DNA oligos.
                                                                                

                                                                                
Importantly, the DNA template was only 80 nt long and could therefore only
accommodate 4 oligos. Successful ligation is visualized as products of 40, 60,
80, or 100 nt. The doublet visible at 20 and 40 nt represents intermediate
adenylated oligos. The viability of each enzyme was confirmed using the DNA
template. Of note is the inability of Rnl2 to create ligation products longer than 40
nt using a DNA template. Ligases examined and venders are as follows:
Lanes 1&amp;8) Tth DNA ligase (Thermo), 2&amp;9) Tsc DNA ligase (Prokaria), 3&amp;10)
Thermostable DNA ligase (Bioline), 4&amp;11) T4 DNA ligase (NEB), 5&amp;12) T4 RNA
ligase 2 (Rnl2)(NEB), 6&amp;13, E. Coli DNA ligase (NEB). Lanes 7&amp;15 were
not loaded. Lane 14 contains radio-labeled oligos but no RNA template,
Lane 15 contains the radiolabeled template, Lane 16 shows a 5 nt RNA
ladder. (right) A ligation time course was performed for Rnl2 and T4 DNA
ligase (section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). Non-templated ligation (NTL) products are annotated as
“x-6*” as there are only 5 hybridization sites on the RNA template. (b) The
ability of Rnl2 to ligate adjacently hybridizing ligamers was tested by adding
combinations of ligamers to individual reactions and visualizing the mobility
of the ligation products on a denaturing PAGE gel. Only when adjacently
hybridizing ligamers are included in a reaction are bands of the expected mobility
visible. (c) Four different <span class="phvro8t-x-x-120">in vitro </span>transcribed RNAs were created by through
amplification of a plasmid with specific pairs of primers, creating a 1,163
nt RNA with unique sequences on the 5<span class="cmsy-8">&#8242;</span>&nbsp;and 3<span class="cmsy-8">&#8242;</span>&nbsp;ends. These RNAs were
                                                                                

                                                                                
incubated in pairs at different concentrations, along with a pool of common
ligamers hybridizing to the unique sequences and a ligamer designed to
loop out the common, 1,046 nt, internal sequence. After radioactive (left) or
endpoint (right), ligation products were visualized on a native PAGE gel.
Trans-transcript products are not visible in radioactive PCR, are only at RNA
concentrations &gt;10 nM in the endpoint PCR analysis. (d) “ABC” and “DBE”
RNAs were combined in different ratios (blue) in the same background of
poly(A)+ RNA. Ligation product band intensity (red) was obtained by radioactive
PCR.
<a id="x1-30001r54"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.2.2 </span> <a id="x1-310002"></a>Trans-transcript hybridization and ligation is minimal</h4>
<!--l. 65--><p class="noindent">One concern when looping out sections of RNA is the potential for an individual
ligamer to hybridize simultaneously to two different RNA molecules. This could result
in the undesired formation of ligation products from intermolecular (<span class="phvro8t-x-x-120">trans</span>) rather
than intramolecular (<span class="phvro8t-x-x-120">cis</span>) hybridization (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>C). To investigate this, we
combined equimolar concentrations of two 1,127 nt RNA transcripts, each
containing a common 1,106 nt internal sequence connected to unique 5<span class="cmsy-8">&#8242;</span>&nbsp;and
3<span class="cmsy-8">&#8242;</span> &nbsp;termini, with a ligamer set designed to loop out 1,046 nt of the common internal
sequence (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>C). Following incubation with Rnl2, ligation products
                                                                                

                                                                                
were PCR amplified. Ligations arising from cis-transcript hybridization result
in 177 and 143 nt PCR products, whereas ligations from trans-transcript
hybridization result in 165 and 155 nt PCR products. Trans hybridization
efficiency should be much more sensitive to RNA concentration than cis
hybridization efficiency. Consistent with this, <span class="phvro8t-x-x-120">trans </span>hybridization products were only
abundant in endpoint PCR reactions when the two RNA targets were <span class="cmsy-10x-x-120">&#8805;</span>10 nM,
whereas <span class="phvro8t-x-x-120">cis </span>hybridization products were readily detectable for reactions
containing as little as 0.01 nM target RNA (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>C, lower left). Even
in samples containing 10 or 50 nM target RNA, radioactive PCR revealed
that cis hybridization products predominated at low cycle numbers (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>C, lower left). As detailed below, both ligamer hybridization and ligation for
SeqZip analysis of complex samples are carried out on magnetic beads on
which poly(A) RNA derived from 5 <span class="cmmi-12">&#956;</span>g of total RNA is adhered. Based on
calculations of known mRNA abundances, the vast majority of cellular mRNAs
are present at concentrations well below 1 nM on these beads. Thus trans
hybridization is highly disfavored compared to cis hybridization under these
conditions.
</p><!--l. 67--><p class="noindent">To be useful for measuring the abundance of isoforms, SeqZip should faithfully
recapitulate input isoform ratios. To test the ability of SeqZip to measure input isoform
ratios, we combined two RNA transcripts at ratios varying from 1:1 to 1:100 (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-29001r2">2.2<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F2 --></a>D). Output ratios determined by radioactive PCR were indistinguishable from input.
                                                                                

                                                                                
With regard to dynamic range, we have obtained ligation products from as
little as 903 molecules (<span class="cmr-12">50 </span><span class="cmsy-10x-x-120">* </span><span class="cmr-12">10</span><sup><span class="cmsy-8">-</span><span class="cmr-8">18</span></sup> mole in 30 <span class="cmmi-12">&#956;</span>L) of target mRNA (data not
shown).
<a id="x1-31001r57"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.2.3 </span> <a id="x1-320003"></a>Reverse transcriptase-based detection versus SeqZip</h4>
<!--l. 72--><p class="noindent">As a first test of the SeqZip method for measuring relative spliced isoform
abundances of an endogenous target, we chose human <span class="phvro8t-x-x-120">CD45 </span>(aka <span class="phvro8t-x-x-120">PTPRC</span>) mRNA
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZikherman2008">Zikherman and Weiss</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZikherman2008">2008</a>]. <span class="phvro8t-x-x-120">CD45 </span>is expressed in T-cells, where mRNA isoforms
(R) contain various combinations of exons 4, 5, and 6 (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>A). Jurkat
cells (resembling naïve primary T-cells) predominantly express isoforms
R56, R5 and R0 (R0 denotes exclusion of all three exons) while U-937 cells
(resembling activated T-cells) predominantly express R456 and R56 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">Yeakley
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYeakley2002">2002</a>]. These three cassette exons are adjacent to one another and
together comprise only 585 nt, making this region of <span class="phvro8t-x-x-120">CD45 </span>mRNA amenable to
analysis by both RT and SeqZip, and thus provided an excellent benchmark for
comparison between methods. Whereas the RT-PCR products varied in length
between 365 and 848 nt, shorter SeqZip-PCR products varied between 132 and
260 nt (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>B), representing a ~3-fold compression of the connectivity
information.
                                                                                

                                                                                
</p><!--l. 74--><p class="noindent">Poly(A)+-selected RNA samples from Jurkat and U-937 cells, or an equimolar
combination of the two (Mix), were assessed by RT or SeqZip, both followed by
radioactive PCR. Both methods reported the expected isoform profiles for every
sample and relative isoform abundances reported by the two techniques were
remarkably similar (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>C). Importantly, SeqZip did not underreport the
abundance of R456 in the Mix sample (which contained all four isoforms), even
though detection of R456 required three more ligation events than R0. Thus SeqZip is
a robust method for detecting and quantifying spliced isoforms of endogenous
transcripts.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-32001r3"></a>
                                                                                

                                                                                
<!--l. 78--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis16x.png" alt="PIC" class="graphics" width="390.45874pt" height="478.78874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis16x.png" src="Figures/SeqZipPaper/Roy2014Fig3.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.3: </span><span class="content">SeqZip on endogenously expressed RNAs<br class="newline">See subsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-330001">2.2.3.1<!--tex4ht:ref: SeqZipPaper:figCap: Roy2014 F3 --></a> for caption. </span></div><!--tex4ht:label?: x1-32001r3 -->
                                                                                

                                                                                
                                                                                

                                                                                
<a id="x1-32002r56"></a>
<h5 class="subsubsectionHead"><span class="titlemark">2.2.3.1 </span> <a id="x1-330001"></a>Figure 2.3 Caption</h5>
<!--l. 90--><p class="noindent">SeqZip on endogenously expressed RNAs
</p><!--l. 92--><p class="noindent">(a) Schematic demonstrated analysis of isoforms of the human <span class="phvro8t-x-x-120">CD45 </span>gene by
SeqZip. (b) Denaturing PAGE gels showing products of RT (top) or SeqZip (bottom)
<span class="phvro8t-x-x-120">CD45 </span>obtained from two different human T-cell lines, or a 1:1 mixture of the two. (c)
Quantified band intensity from the gels shown in (b) (top) and a mirror image of the
lane profile from the mix lanes (bottom). (d) When considering the cassette exons
EDA (blue), and the Variable region (light blue), mouse <span class="phvro8t-x-x-120">Fn1 </span>can produce 6
different isoforms. Isoform nomenclature used in the rest of the figure is
shown next to the block schematic. Filled boxes depict exons, diagonal lines
indicate isoform sequences not shown, straight lines show absence of exon in
the final mRNA. (e) schematic showing more detail for the three regions of
<span class="phvro8t-x-x-120">Fn1 </span>alternative splicing investigated. Also shown are different ligamer pools
(red). Looped regions indicated by dashed red lines, priming sequences as
black arrows. (f) SeqZip was performed on poly(A)+ RNA from various <span class="phvro8t-x-x-120">Fn1</span>
modified cell lines (see Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). Ligation products were amplified via
radioactive PCR. (g) isoform band intensities are from (f) were quantified in
triplicate and are plotted. Black bars indicate individual exon intensities (EDA;
                                                                                

                                                                                
V-Region), or calculated expected frequencies (black bars under “Combination
pool”). Shown in light grey are observed combination isoform intensities
and when available, frequencies of PacBio reads from RNA of each sample
(grey).
<a id="x1-33001r58"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.2.4 </span> <a id="x1-340004"></a>SeqZip maintains connectivity and abundance between many sites of
alternative splicing</h4>
<!--l. 103--><p class="noindent">Mouse Fn1 contains three well-characterized sites of alternative splicing: Extra
Domain B (EDB,B), which is included in embryos but excluded in almost all adult
tissues except brain; Extra Domain A (EDA (A), which displays a more variable
alternative splicing pattern both across developmental stages and adult tissue types;
and Variable Region (V) wherein three alternative 3<span class="cmsy-8">&#8242;</span>&nbsp;splice sites lead to
120, 95, or 0 additional amino acids in FN1 (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>D). One study has
suggested that EDA exclusion is associated with preferential use of the most
promoter-proximal 3<span class="cmsy-8">&#8242;</span>&nbsp;splice site (120) in the V region [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">Fededa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFededa2005">2005</a>].
However, another concluded that splicing of the EDA and V regions occurs
autonomously [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">Chauhan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">2004</a>]. These contrary results are possibly
explainable by differences in both mRNA source (MEFs vs. adult tissues,
respectively) and quantitation methods (region-specific or exon-specific RT-PCR,
                                                                                

                                                                                
respectively).
</p><!--l. 105--><p class="noindent">Together, the EDA and V regions produce up to six different spliced isoforms (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>D). These two alternative splicing regions are separated by constitutively included
exons 34-39 comprising 815 nt. Analysis of the EDA and V exons using traditional
RT-PCR generates cDNAs ranging in size from 1 to 1.6 kilobases (kb). Using SeqZip,
we could compress the exon connectivity information &gt;5-fold into uniquely sized
ligation products (139 to 318 nt), greatly simplifying connectivity and abundance
analysis.
</p><!--l. 107--><p class="noindent">To examine possible coordination between the EDA and V regions, we first made
ligamer sets that reported on each region individually (individual pools A and V) and
in combination (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>E). Using these ligamer pools, we obtained ligation
products from poly(A)+ RNA of MEF cells and analyzed them using radioactive PCR
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>F and G). EDA and V isoform distributions were analyzed separately
or in combination (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>F, top row, black bars). The expected isoform
distributions for all EDA and V combinations, assuming the null hypothesis of no
coordination, were calculated from the individual frequencies (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>G,
light grey bars). Connected splicing decisions should appear as deviations
from null-hypothesis frequencies. In no case did the SeqZip data exhibit any
significant deviance from the null hypothesis expectation. Thus, at least in primary
MEFs, EDA and V region alternative splicing appears to be independently
                                                                                

                                                                                
regulated.
</p><!--l. 109--><p class="noindent">To directly test the effects of EDA inclusion or exclusion on splicing of the V
region, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">Chauhan et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">2004</a>] used homologous recombination to create
mice where the intronic splicing enhancers were modified to favor either
constitutive inclusion (+/+) or exclusion (-/-) of exon 33. They also analyzed
the parental strain (wt) and mice heterozygous for the modified locus (+/-).
We analyzed RNA obtained from immortalized MEFs of all four genotypes
using SeqZip as above(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>G, lower 4 rows). While we did observe
constitutive inclusion or exclusion of the EDA exon as expected, in none
of the lines did we observe an effect of EDA inclusion or exclusion on the
V-region. We further confirmed these mRNA isoform distributions by directly
sequencing full-length RT-PCR products on the PacBio platform (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-32001r3">2.3<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F3 --></a>G,
grey bars). Thus, our results are consistent with the findings of <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">Chauhan
et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauhan2004">2004</a>] that the EDA and V regions are indeed autonomous with regard to
splicing.
<a id="x1-34001r61"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.2.5 </span> <a id="x1-350005"></a><span class="phvro8t-x-x-120">Dscam1 </span>analysis by high-throughput sequencing</h4>
<!--l. 114--><p class="noindent">Unquestionably, the most challenging system for measuring multi-site alternative
splicing exon connectivity and isoform abundance is <span class="phvro8t-x-x-120">Dscam1 </span>(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>A).
                                                                                

                                                                                
<span class="phvro8t-x-x-120">Dscam1 </span>has four regions of alternative splicing (exons 4, 6, 9 and 17), all
utilizing mutually exclusive cassette exons (with 12, 47, 33 and 2 variants,
respectively). Thus there are 37,224 possible <span class="phvro8t-x-x-120">Dscam1 </span>mRNA isoforms, and previous
studies have suggested that all isoforms have the potential to be generated
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">Zhan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">2004</a>]. Consistent with
this, a recent study examining <span class="phvro8t-x-x-120">Dscam1 </span>expression in individual neurons
showed that all exon 4 variants are used and are incorporated in a stochastic
and probabilistic manner [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">Miura et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">2013</a>]. Only one study so far has
attempted to assess the extent to which specific exon choices in the different
cassette regions influence one another [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>]. That study examined
connectivity between exons 4, 6 and 9 using an RT-PCR-based method dubbed
“CAMSeq”.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-35001r4"></a>
                                                                                

                                                                                
<!--l. 118--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis17x.png" alt="PIC" class="graphics" width="382.42874pt" height="461.72499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis17x.png" src="Figures/SeqZipPaper/Roy2014Fig4.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.4: </span><span class="content">Analysis of <span class="phvro8t-x-x-120">Dscam1 </span>isoforms via High-Throughput Sequencing<br class="newline">A) The <span class="phvro8t-x-x-120">Dscam1  </span>gene.  B)  Triple-Read  library  preparation  schematic.  C)
Rank-order expression  for  different  <span class="phvro8t-x-x-120">in  vitro  Dscam1  </span>isoforms.  Blue  -  <span class="phvro8t-x-x-120">in  vitro</span>
products added to libraries. with isoform identities at 4.X, 6.X, and 9.X. Red
                                                                                

                                                                                
- template-switched  isoforms,  Yellow  -  ligation  products  created  via  inclusion
of near-cognate  ligamers.  Green  -  <span class="phvro8t-x-x-120">in  vivo  </span>isoforms  measured  from  S2  cells.
CAM-Seq data  from  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>].  See  also  Figure  <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36001r8">2.8<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S4 --></a>.  D)  Schematic
showing how ligamers include exon-specific barcodes. E) SeqZip application to
<span class="phvro8t-x-x-120">Dscam1 </span>isoform profiling. F) Technical and biological replicates for S2 cells. and
14-16 hr embryos using SeqZip G) Number of reads and isoforms observed
per sample type. Also shared isoforms between libraries. H) Rank order isoform
expression by sample type. </span></div><!--tex4ht:label?: x1-35001r4 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-35002r5"></a>
                                                                                

                                                                                
<!--l. 129--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis18x.png" alt="PIC" class="graphics" width="362.35374pt" height="382.42874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis18x.png" src="Figures/SeqZipPaper/Roy2014FigS1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.5: </span><span class="content">HTS Analysis of <span class="phvro8t-x-x-120">Dscam1 in vitro </span>transcripts<br class="newline">A) The similarity of sequence between cDNA, circularized cDNA, and ligation
products was measured [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNotredame2000">Notredame et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNotredame2000">2000</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWaterhouse2009">Waterhouse et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWaterhouse2009">2009</a>].
B) The rank-order in terms of expression for different <span class="phvro8t-x-x-120">in vitro Dscam1 </span>isoforms
is plotted. Blue  dots  indicate  true  <span class="phvro8t-x-x-120">in  vitro  </span>products  added  to  the  libraries,
Red dots indicate  template-switch  isoforms  sequenced  from,  but  not  added
to, each library.  Yellow  dots  indicate  ligation  products  created  via  inclusion
                                                                                

                                                                                
of near-cognate  ligamers  during  SeqZip  ligation.  Green  dots  indicate  <span class="phvro8t-x-x-120">in  vivo</span>
isoforms measured  from  S2  cells  using  SeqZip.  CAMSeq  data  is  from  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>]. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>. </span></div><!--tex4ht:label?: x1-35002r5 -->
                                                                                

                                                                                
<!--l. 138--><p class="noindent">Potentially problematic for RT-PCR-based approaches are long stretches of sequence
identity in the constitutive exons separating each cluster, as well as the high
sequence similarity between individual exon 4, 6 and 9 variants (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>). Long
regions of sequence homology favor template switching during both RT and PCR
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouseley2010a">Houseley and Tollervey</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouseley2010a">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJudo1998">Judo et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJudo1998">1998</a>]. That is, an incompletely extended
DNA strand can hybridize to a different template during the RT step or any
subsequent PCR cycle, leading to novel “switched ” isoforms not present in the
original biological sample.
</p><!--l. 140--><p class="noindent">To test how template switching might affect <span class="phvro8t-x-x-120">Dscam1 </span>isoform interrogation, we
developed an RT-PCR-based Triple-read sequencing method for examining exon
connectivity between alternative splicing regions 4, 6 and 9, similar to CAMSeq
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>B, &amp; section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). To specifically assess template switching, we
generated four full-length cDNA clones (<span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">1</span><span class="cmmi-8">,</span><span class="cmr-8">33</span><span class="cmmi-8">,</span><span class="cmr-8">9</span></sup>, <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">12</span><span class="cmmi-8">,</span><span class="cmr-8">32</span><span class="cmmi-8">,</span><span class="cmr-8">9</span></sup>, <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">1</span><span class="cmmi-8">,</span><span class="cmr-8">24</span><span class="cmmi-8">,</span><span class="cmr-8">6</span></sup>,
and <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">7</span><span class="cmmi-8">,</span><span class="cmr-8">9</span><span class="cmmi-8">,</span><span class="cmr-8">6</span></sup>; where the superscripted numbers indicate the included
4.X, 6.Y, and 9.Z exon variants respectively). Runoff RNA transcripts were
mixed together in three different ratios (1:1:1:1, 3:3:1:1, and 5:1:1:1) and the
mixtures were then processed in parallel to generate deep sequencing data.
Indicative of template switching, many novel transcript isoforms containing exon
combinations not present in original control transcripts were observed (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>C).
These template-switched isoforms made up 34% to 55% of the total isoforms
detected, with many being significantly more abundant than one or more of the
                                                                                

                                                                                
input isoform(s). We next analyzed similar control data from the CAMSeq
study [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>]. CAMSeq proved to be much more robust than our
Triple-read protocol for accurately reporting both individual exon and individual
transcript isoform abundances over five orders of magnitude. Nonetheless,
template-switched isoforms were present in the CAMSeq data, with many switched
species being more abundant than the lowest input species. Further, many
additional invasive isoforms of unknown source were detected. Although the
template-switched and unknown source isoforms represented just 0.6–0.94% of
total reads, they represented 99.9% of all isoforms detected (5,378–5,906
switched and source-unknown isoforms versus 8 input isoforms) (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>B).
Thus, extensive template in both RT-PCR-based approaches (Triple-read and
CAMSeq) compromised their ability to accurately identify the exact isoform
set present in the original sample and faithfully report their relative input
levels.
</p><!--l. 142--><p class="noindent">The technical issue of template switching can be greatly minimized using SeqZip and
ligamers containing an exon-specific barcode in the linker region (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>D). The
added sequence diversity and decreased length make SeqZip ligation products, in
principle, much less prone to template switching (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>A). Exon-specific
barcodes also ensure unambiguous isoform alignment. To apply SeqZip to <span class="phvro8t-x-x-120">Dscam1</span>,
we designed ligamers targeting each variant exon in clusters 4, 6, and 9 plus
constitutive exons 3, 5, 7–8, and 10 (97 ligamers in all) (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>E). Whereas the
                                                                                

                                                                                
exon 3–10 region ranges in size from 1,722–1,751 nt (median = 1,734 nt) in
<span class="phvro8t-x-x-120">Dscam1 </span>mRNAs, our ligamer design compresses this ~5-fold to a 356 nt
median-length, 7 ligamer-containing (6 ligation events) products. These products
could be completely sequenced in a single Illumina MiSeq paired-end 250 nt
run.
</p><!--l. 144--><p class="noindent">To assess (1) maintenance of input isoform ratios, (2) template switching during PCR
amplification, and (3) inappropriate ligamer hybridization during the ligation reaction,
we performed control experiments containing 3 different <span class="phvro8t-x-x-120">in vitro </span>transcribed <span class="phvro8t-x-x-120">Dscam1</span>
isoforms mixed at a 1:10:100 ratio in a background of total RNA from mouse Hepa
1-6 cells (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35003r6">2.6<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S2 --></a>A, section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). Separate ligation reactions each containing a
97-ligamer pool wherein two highly expressed exons within each cluster were
differently coded between the pools allowed for detection of template-switched
isoforms, both in our control experiments and in biological samples. Following
ligation, differentially coded samples are mixed together, subjected to PCR and
sequenced (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>C, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>B &amp; section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). As expected, SeqZip faithfully
maintained Dscam input isoform abundance in the control samples (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>BB).
Further, we detected no template-switched isoforms (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>C). Thus, as
expected, our ligamer design strongly disfavors template switching during PCR
amplification. This also held true for a complex biological sample (poly(A)+-selected
RNA from S2 cells) where the same differentially coding approach revealed just
17 of 111,242 reads (0.015%) corresponding to switched isoforms (Figure
                                                                                

                                                                                
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>C).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-35003r6"></a>
                                                                                

                                                                                
<!--l. 148--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis19x.png" alt="PIC" class="graphics" width="387.4475pt" height="490.83374pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis19x.png" src="Figures/SeqZipPaper/Roy2014FigS2.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.6: </span><span class="content">SeqZip <span class="phvro8t-x-x-120">in vitro Dscam1 </span>controls and near-cognate ligation<br class="newline">A) The three   in   vitro   cDNAs   used   as   <span class="phvro8t-x-x-120">in   vitro   </span>transcript   controls.   B)
Expression of  <span class="phvro8t-x-x-120">in  vitro  </span>controls  in  SeqZip  analysis.  Blue  are  true  transcripts,
                                                                                

                                                                                
yellow near-cognate  transcripts.  C)  Schematic  showing  template-switching
observation scheme. Also shown is quantified template-switched products from
Control and S2 libraries. D) Comparison of E6.[8 &amp; 9] ligamer sequences. E)
Folding energies of between ligamers 6.9 and 6.24 to all exons 6.X sequences.
</span></div><!--tex4ht:label?: x1-35003r6 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-35004r7"></a>
                                                                                

                                                                                
<!--l. 159--><p class="noindent"><img src="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis20x.png" alt="PIC" class="graphics" width="392.46626pt" height="186.6975pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis20x.png" src="Figures/SeqZipPaper/Roy2014FigS3.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.7: </span><span class="content">Cognate and Nearest-Near cognate folding energies for <span class="phvro8t-x-x-120">Dscam1</span>
exon 6<br class="newline">Left) Folding  energies  between  all  cognate  exon:ligamer  6  pairs.  Right)
Nearest-NearCognate folding energies for all ligamer:exon pairs. On both Left
and Right panels, Yellow line = folding energy of -36 J/mol and Blue line = -67
J/mol. </span></div><!--tex4ht:label?: x1-35004r7 -->
                                                                                

                                                                                
<!--l. 168--><p class="noindent">Given the high sequence similarity between variant exons within each cluster, ligamer
hybridization to near cognate sequences is potentially problematic. Ligamer
hybridization is specified by the sequences at the ends of target exons (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-27001r1">2.1<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F1 --></a>). To assess the potential for mis-pairing, we calculated the free energy of
hybridization [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2010">Reuter and Mathews</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2010">2010</a>] between each ligamer and all exon
variants within its target cluster. As expected, cognate ligamer-exon pairs had
predicted hybridization energies lower than <span class="cmr-12">&#916;</span><span class="cmmi-12">G</span><sup><span class="cmsy-8">&#8728;</span></sup> = -67 kcal/mol, with the closest
near-cognate pair being at least 12 units higher (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35004r7">2.7<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 FS3 --></a>, section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a>). In
the control samples, only 642 of 50,475 high-confidence alignments (1.3%)
contained ligamers for exons not present in any input transcript, with the vast
majority of these species (221/236) being represented by 3 or fewer reads.
The two highest near-cognate hybridization products had abundances well
below those of the true targets (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>C, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>B, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35003r6">2.6<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S2 --></a>D, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35003r6">2.6<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S2 --></a>E), indicating
that, while detectable, near-cognate hybridization is not a major problem for
SeqZip.
<a id="x1-35005r62"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.2.6 </span> <a id="x1-360006"></a>Analysis of <span class="phvro8t-x-x-120">Dscam1 </span>transcripts</h4>
<!--l. 173--><p class="noindent">Having validated SeqZip as a reliable approach for analyzing <span class="phvro8t-x-x-120">Dscam1 </span>isoforms, we
next analyzed RNA samples from S2 cells, 4–6 and 14–16 h embryos. Between
                                                                                

                                                                                
~450,000–1,000,000 reads were obtained for each sample. In total, 8,397 of the
possible 18,612 unique isoforms were detected (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>G). Individual isoform
abundances were highly correlated in both technical and biological replicates (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35001r4">2.4<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F4 --></a>F; <span class="cmmi-12">r</span>=0.95-0.8). Of the 97 possible exons represented in our ligamer set, all were
detected except 6.11, for which substantial evidence indicates it to be an unused
pseudo-exon [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">Watson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">2005</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">Zhan
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">2004</a>]. Thus absence of exon 6.11 reads from our libraries additionally
confirms the specificity of our technique. Yet another confirmation was the
individual exon utilization patterns observed in S2 cells, the only sample directly
comparable between the SeqZip and CAMSeq datasets (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35004r7">2.7<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 FS3 --></a>A &amp; B).
Overall, the S2 exon utilization patterns were remarkably similar between the
two analyses. The exception was exon 6.47, which was well represented in
the CAMSeq data, but undetectable in the SeqZip data. We currently do
not understand this failure of ligamer 6.47 to capture exon 6.47 containing
transcripts. Nonetheless, the similarity between the SeqZip and CAMSeq
data with regard to all other exon abundances demonstrates the general
robustness of SeqZip for accurately reporting exon abundances in highly complex
samples.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-36001r8"></a>
                                                                                

                                                                                
<!--l. 177--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis21x.png" alt="PIC" class="graphics" width="401.5pt" height="159.59624pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis21x.png" src="Figures/SeqZipPaper/Roy2014FigS4.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.8:  </span><span class="content">Correlograms  of  expression  for  all  exon  pairs  in  <span class="phvro8t-x-x-120">Dscam1  </span>as
measured by SeqZip<br class="newline">Pearson correlations  of  expression  for  each  exon  from  all  libraries  were
calculated and are plotted as shaded boxes with Yellow = <span class="cmmi-12">r </span>of 1 and Blue = <span class="cmmi-12">r </span>of
-1. </span></div><!--tex4ht:label?: x1-36001r8 -->
                                                                                

                                                                                
<!--l. 185--><p class="noindent">Comparison of exon utilization patterns across biological samples in our SeqZip data
revealed a substantial increase in diversity going from S2 cells (least diversity) to 4-6
hour embryos (intermediate) to 14-16 hour embryos (highest) (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36002r9">2.9<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F5 --></a>A). As
previously reported, the utilization patterns across clusters 4 and 9 exhibit
dramatically change during development, whereas the utilization pattern
across cluster 6 remains relatively static [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCelotto2001">Celotto and Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCelotto2001">2001</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">Miura
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">Zhan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhan2004">2004</a>]. S2 cells
are characterized by poor utilization of exons 4.[2,9], and almost exclusive
utilization of exons 9.[6,9,13,30,31]. This pattern is characteristic of hemocytes
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">Watson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">2005</a>], consistent with the macrophage-like nature of S2 cells
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchneider1972">Schneider</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchneider1972">1972</a>]. Whereas 4-6 hour embryos are very similar to S2 cells in their
cluster 4 and 9 exon utilization patterns, 14-16 hour embryos show a significant
increase in exon diversity, especially in cluster 9. New isoforms are likely neuronal in
origin. To compare the expression levels of hemocyte- and non-hemocytes indicative
isoforms within each sample, we color coded them in the scatter plots in Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36002r9">2.9<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F5 --></a>B. This makes it easy to see that as a class, the “hemocyte-indicative”
isoforms (i.e., those lacking exon 4.[2,9] or containing exon 9.[6,9,13,30,31])
dominate all samples in terms of abundance. As development proceeds,
however, “non-hemocyte indicative” isoforms increase in both number and
abundance.
</p><!--l. 187--><p class="noindent">For all three samples (S2 cells, 4-6 h and 14-16 h embryos), we calculated
                                                                                

                                                                                
expected pairwise and triple combination frequencies in individual transcript
isoforms by simple multiplication of individual variant exon frequencies in each
cluster. Plotting these expected frequencies against observed frequencies
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36002r9">2.9<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F5 --></a>B) revealed no obvious outliers. Therefore, consistent with previous
analyses [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>], we conclude that there is no
coordination between <span class="phvro8t-x-x-120">Dscam1 </span>clusters 4, 6 and 9 with regard to alternative exon
choice.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-36002r9"></a>
                                                                                

                                                                                
<!--l. 191--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis22x.png" alt="PIC" class="graphics" width="348.30124pt" height="427.59749pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis22x.png" src="Figures/SeqZipPaper/Roy2014Fig5.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.9: </span><span class="content"><span class="phvro8t-x-x-120">Dscam1 </span>individual exon usage measured by SeqZip<br class="newline">A) Expression  of  each  variant  exon,  for  each  library.  Individual  libraries  are
plotted as gray-shaded bars. B) Scatterplots showing mean Observed versus
Expected individual exon abundances for the pairs, or all exon clusters (4, 6,
and 9). Isoforms are colored red when hemocyte indicative or blue when not. </span></div><!--tex4ht:label?: x1-36002r9 -->
                                                                                

                                                                                
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-36003r10"></a>
                                                                                

                                                                                
<!--l. 201--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis23x.png" alt="PIC" class="graphics" width="366.36874pt" height="396.48125pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis23x.png" src="Figures/SeqZipPaper/Roy2014FigS5.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;2.10: </span><span class="content">Comparison between SeqZip and CAMSeq analysis of <span class="phvro8t-x-x-120">Dscam1</span>
isoforms in S2 cells<br class="newline">A) Individual exon usages observed for S2 cells observed using either SeqZip
(Top) or CAMSeq (Bottom). </span></div><!--tex4ht:label?: x1-36003r10 -->
                                                                                

                                                                                
<a id="x1-36004r53"></a>
<h3 class="sectionHead"><span class="titlemark">2.3 </span> <a id="x1-370003"></a>Discussion</h3>
<!--l. 212--><p class="noindent">Here we describe development and implementation of a new method, SeqZip, for
compressing sequence information of long RNAs while maintaining connectivity
between distant regions of individual molecules. Completely orthogonal to traditional
methods of RNA sequence investigation such as RT-PCR, SeqZip can be used to
quickly and efficiently examine complex alternative splicing events, and is particularly
useful for investigating genes harboring multiple distal sites of alternative splicing.
Using SeqZip to investigate the splicing coordination in mouse <span class="phvro8t-x-x-120">Fn1 </span>transcripts and in
<span class="phvro8t-x-x-120">Drosophila melanogaster Dscam1 </span>we found no evidence of splicing coordination in
either gene.
<a id="x1-37001r67"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.3.1 </span> <a id="x1-380001"></a>Deconvoluting <span class="phvro8t-x-x-120">Dscam1</span></h4>
<!--l. 216--><p class="noindent">Many of the <span class="phvro8t-x-x-120">Dscam1 </span>variant exons arose from exon-duplication and, therefore, have
very high sequence similarity [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2010b">Lee et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2010b">2010</a>]. The extreme diversity of
<span class="phvro8t-x-x-120">Dscam1 </span>has been implicated in important biological functions, including
neuronal self recognition and immune function [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">Watson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatson2005">2005</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWojtowicz2004">Wojtowicz
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWojtowicz2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">Zipursky and Grueber</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">2013</a>]. While flies coding for 4,752 unique
                                                                                

                                                                                
isoforms have been shown to display equivalent neurite formation as wild-type
controls, animals expressing 1,152 isoforms display neuronal branding defects,
supporting the view that diversity of molecules, and not sequence, is essential for
biological function [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2009">Hattori et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2009">2009</a>].
</p><!--l. 218--><p class="noindent">Multiple technical hurdles currently hamper characterization of <span class="phvro8t-x-x-120">Dscam1 </span>isoforms: (1)
determining the relative utilization of individual exons within each cassette region,
while maintaining connectivity information between cassettes using current
Illumina-platform RNA-Seq read lengths is currently not possible [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLeGault2013">LeGault and
Dewey</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLeGault2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">Neves et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNeves2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">Zipursky and Grueber</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">2013</a>]; (2) sequencing
full-length mRNAs expressed across 5 orders of magnitude is technically challenging
and costly [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2008">Hattori et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHattori2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">Sharon et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">2013</a>]; and (3) due to sequence
similarity between <span class="phvro8t-x-x-120">Dscam1 </span>isoforms, template switching artifacts complicate
high-throughput sequencing library preparation.
</p><!--l. 220--><p class="noindent">Recently, two studies have lent additional support to a longstanding hypothesis that
<span class="phvro8t-x-x-120">Dscam1 </span>isoforms are produced via stochastic processing. The first is an elegant
genetic investigation into exon 4 variants, demonstrating that changes in variant
expression are not due to any requirement at a specific time, cell, or tissue,
and instead is determined by modulating the probability of choosing certain
variants [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">Miura et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMiura2013b">2013</a>]. In the second study, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">Sun et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSun2013">2013</a>] employed a
novel high throughput sequencing approach (CAMSeq). CAMSeq begins
                                                                                

                                                                                
by amplifying barcoded 2 kb <span class="phvro8t-x-x-120">Dscam1 </span>RT products circularized into ~1 kb
inserts containing exon clusters 4, 6 and 9. Circularized products are amplified
again, and deep sequenced via sequential hybridization of three constitutive
exon-specific primers, sequencing exons 4, 6, and 9. CAMSeq is extremely
similar to the Triple-read approach described above. CAMSeq analysis of
<span class="phvro8t-x-x-120">Dscam1 </span>diversity in multiple <span class="phvro8t-x-x-120">Drosophila melanogaster </span>(S2 cells, embryos,
larva, pupae, and adult brains) led the authors to conclude that all possible
isoforms are expressed at all developmental stages, again in a stochastic
fashion.
</p><!--l. 222--><p class="noindent">Two potential complications of the CAMSeq approach are: (1) chimera formation
via intermolecular ligation during the circularization step, and (2) template
switching during the RT step or either round of PCR. Using a clever barcoding
scheme, the authors were able to determine that chimeras represented ~1% of
their CAMSeq libraries, indicating that this is a non-trivial problem. Indeed,
in control libraries made from a mixture of 8 <span class="phvro8t-x-x-120">in vitro </span>transcribed <span class="phvro8t-x-x-120">Dscam1</span>
isoforms, chimera formation resulted in detection of &gt;5,000 other isoforms not
present in the original mix, with ~500 being represented by <span class="cmsy-10x-x-120">&#8805; </span>10 reads (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-35002r5">2.5<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 S1 --></a>B).
</p><!--l. 224--><p class="noindent">Whereas barcoding can distinguish bona fide isoforms from chimeras, there is no way
to distinguish isoforms present in the original RNA sample from artificial combinations
                                                                                

                                                                                
created by template switching. Template switching can occur during the initial RT step
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouseley2010a">Houseley and Tollervey</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouseley2010a">2010</a>], or during either PCR amplification step [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJudo1998">Judo
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJudo1998">1998</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeyerhans1990a">Meyerhans et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeyerhans1990a">1990</a>]. With these potential experimental
complications in mind, we decided to investigate and characterize isoforms of
<span class="phvro8t-x-x-120">Dscam1 </span>using the SeqZip method.
</p><!--l. 226--><p class="noindent">Our analysis of <span class="phvro8t-x-x-120">Dscam1 </span>yielded a very similar exon usage frequency to that of
CAMSeq at each stage examined (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36003r10">2.10<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 FS5 --></a>). Additionally, we also observed no
connectivity between exon choices from any of the three clusters (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-36002r9">2.9<!--tex4ht:ref: SeqZipPaper:fig:Roy2014 F5 --></a>B). We do observe increased exon 9 diversity in 14-16 h embryos. Taken
together, our data also support the view that flies use a complex mixture
of <span class="phvro8t-x-x-120">Dscam1 </span>isoforms, produced via stochastic and probabilistic splicing, in
order to discriminate self from non-self neuronal processes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">Zipursky and
Grueber</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLawrenceZipursky2013">2013</a>].
</p><!--l. 228--><p class="noindent">Even a relatively shallow analysis of the human transcriptome using single-molecule
sequencing on the PacBio platform has revealed a rich population of previously
unreported isoforms, and thorough analysis of complex spliced genes is becoming a
reality [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">Sharon et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">2013</a>]. However, the most interesting spliced genes often
produce long (<span class="cmmi-12">&gt;</span>1,500 nt) transcripts that are often expressed in the central nervous
system [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2007">Park and Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2007">2007</a>]. While human <span class="phvro8t-x-x-120">Dscam </span>does play a role in the
neurologically disorder down syndrome, it does not undergo extensive splicing
                                                                                

                                                                                
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYamakawa1998a">Yamakawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYamakawa1998a">1998</a>]. Unlike <span class="phvro8t-x-x-120">Dscam</span>, human <span class="phvro8t-x-x-120">protocadherin </span>and <span class="phvro8t-x-x-120">neurexins </span>are
heavily processed and, similar to <span class="phvro8t-x-x-120">Dscam1</span>, are involved in neuronal wiring [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUshkaryov1992">Ushkaryov
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XUshkaryov1992">1992</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWu1999">Wu and Maniatis</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWu1999">1999</a>]. Recently, PacBio was used to rigorously
determine <span class="phvro8t-x-x-120">neurexin </span>isoforms, and found that these genes do produce many different
isoforms, but lack any coordination in their alternative processing events [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTreutlein2014">Treutlein
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTreutlein2014">2014</a>]. Perhaps efficient cell-cell recognition is accomplished not by an
ordained and complicated system, but by random and frequent shuffling of
exons.
<a id="x1-38001r72"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">2.3.2 </span> <a id="x1-390002"></a>Other applications of SeqZip</h4>
<!--l. 232--><p class="noindent">A potentially more routine use of the SeqZip methodology is highlighted by our
analysis of mouse <span class="phvro8t-x-x-120">Fn1</span>, where we simultaneously measured 12 different alternative
splicing isoforms and determined their relative expression using traditional
PAGE. While it is intriguing to think that inclusion of the EDA exon in this gene
influences alternative splicing decisions over a kilobase and multiple exons
away, we saw no evidence for this type of regulation in any of the cell lines
investigated.
</p><!--l. 234--><p class="noindent">SeqZip could also be used to assess the integrity of long RNAs, extended 3<span class="cmsy-8">&#8242;</span>&nbsp;UTRs
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2013b">Wang and Yi</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2013b">2013</a>], or piRNA-precursor transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013h">2013a</a>]. A more
                                                                                

                                                                                
routine laboratory task where SeqZip could prove useful is Q-PCR. SeqZip does
not include an RT step, providing an orthogonal means of measuring RNA
abundance. Also, any sequence can be placed between each ligamers two regions
of complementarity. Therefore sequences for custom priming, restriction
digestion, recombination, etc., can be introduced allowing for quantification or
manipulation of ligation products. Analysis of ligation products can even be
multiplexed, allowing for simultaneous generation and analysis of internal controls
and primer sets. As demonstrated by our <span class="phvro8t-x-x-120">Dscam1 </span>study, SeqZip ligation
products can be analyzed with high-throughput sequencing via incorporation of
platform-appropriate priming sequences either in the terminal PCR primers, or in the
spacer sequences of the internal ligamers. Therefore, this robust methodology, which
only takes 1.5 days to complete, complements more traditional analysis via
RT-PCR.
<a id="x1-39001r71"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">2.4 </span> <a id="x1-400004"></a>Materials and Methods</h3>
<dl class="description"><dt class="description">
<span class="phvb8t-x-x-120">Cell lines and Drosophila melanogaster stocks</span> </dt><dd class="description">
<!--l. 242--><p class="noindent">U-937 (CRL-1593.2),  Jurkat  (TIB-152),  and  S2  (CRL-1963)  cell  lines
were obtained  from  ATCC.  Primary  MEF  cells  were  from  C57BL/6J
                                                                                

                                                                                
strain background  and  were  obtained  from  The  Jackson  Labs.  MEF
lines were immortalized using SV40 retroviral infection. Mixed Drosophila
melanogaster  Oregon-R  males  and  females  were  maintained  at  25 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C</span>.
Embryos 4-6 hour, and 14-16 were collected.
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">Ligamer design</span> </dt><dd class="description">The  5<span class="cmsy-8">&#8242;</span>&nbsp;and  3<span class="cmsy-8">&#8242;</span>&nbsp;most  sequences  of  a  target  sequence  (ex.
exon or multiple exons) were obtained from online databases (Ensembl,
UCSC, etc.).  The  <span class="phvro8t-x-x-120">T</span><sub><span class="cmmi-8">m</span></sub>  of  these  sequences  was  normalized  to  60 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span><span class="cmsy-10x-x-120">±</span>
5 <sup><span class="cmsy-8">&#8728;</span> </sup> <span class="cmr-12">C </span>according to nearest-neighbor rules [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XXia1998">Xia et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XXia1998">1998</a>] by adding or
removing flanking nucleotides. Most regions of complementarity fell 12–25
nt in length. After assembling complementary region sequences, matching
sequences (i.e. the 5<span class="cmsy-8">&#8242;</span>&nbsp;and 3<span class="cmsy-8">&#8242;</span>&nbsp;edge sequences of a specific exon) were
combined  with  a  short  spacer  sequences  included  between  them.  For
this study the spacer was restricted to (TGA)*N, where N was typically
2. With the full sequence now assembled, the reverse complement was
taken, ligamers requiring 5<span class="cmsy-8">&#8242;</span>&nbsp;phosphorylation for subsequent ligation were
marked, and ligamers were ordered in 96 well format from Integrated DNA
Technologies (IDT). Ligamers were reconstituted to 1 <span class="cmmi-12">&#956;</span>M in sets targeting
specific regions on a specific gene and subsequently diluted further for
use in the SeqZip protocol.
                                                                                

                                                                                
</dd><dt class="description">
<span class="phvb8t-x-x-120">SeqZip</span></dt><dd class="description">Total RNA was isolated from a cell line or tissue type using TriReagent
(MRC) according to the manufacturer’s instructions. Poly(A)+ RNA was
isolated using the Poly(A)Purist MAG kit from Ambion (AM1922). Poly(A)+
RNA was  not  eluted  from  magnetic  beads,  and  after  the  last  wash
step, beads were aliquoted into appropriate amounts and reconstituted in
hybridization buffer (60 mM Tris-HCl pH 7.5 @ 25 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C</span>, 1.2 mM DTT 2.4 mM
MgCl<sub><span class="cmr-8">2</span> </sub> , 480 <span class="cmmi-12">&#956;</span>M ATP) including 10 nM appropriate ligamers. Hybridization
was performed  in  a  thermocycler  by  heating  samples  to  62 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C  </span>for  5
minutes and cooling to 45 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>in 3 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>by 10 minute increments. Samples
were left  at  45 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>for  1  hour,  then  cooled  again  in  3 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>by  10  minute
increments until 37 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>was reached, followed by enzyme addition. T4 RNA
ligase 2 (NEB, M0239) was added to compose 10% of final volume (ex.
2.5 <span class="cmmi-12">&#956;</span>l in 22.5 <span class="cmmi-12">&#956;</span>L previous samples). At this point the samples were in 1X
ligation buffer (51 mM Tris-HCl pH 7.5 @ 25 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C</span>, 2.01 mM DTT, 5 mM KCl,
2 mM MgCl<sub><span class="cmr-8">2</span></sub>,  400  <span class="cmmi-12">&#956;</span>M  ATP,  3.5  mM  (NH<sub><span class="cmr-8">4</span></sub>)<sub><span class="cmr-8">2</span></sub>SO<sub><span class="cmr-8">4</span></sub>,  5%  glycerol).  Samples
were incubated at 37 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>for 8–16 hours. Ligation products were amplified
by PCR and analyzed by either native PAGE or sequencing.
</dd><dt class="description">
<span class="phvb8t-x-x-120">PacBio FN1 Analysis</span> </dt><dd class="description">cDNA  prepared  using  primers  designed  to  amplify
EDB-&gt;IIICS  region  was  submitted  for  library  construction  using  “The
                                                                                

                                                                                
DNA Template Prep Kit 2.0 ”and sequenced on a PacBio RS II. Circular
Consensus reads were aligned to an index of FN1 isoforms using BLAT.
</dd><dt class="description">
<span class="phvb8t-x-x-120">MiSeq Library Preparation</span> </dt><dd class="description">Individual SeqZip ligation reactions were amplified
for 12 cycles using common primers, in individual PCR reactions. After
amplification,  PCR  reactions  were  run  on  a  5%  acrylamide  gel,  and
DNA in the appropriate size range of full-length ligation products was cut
from the gel, and eluted. Eluted DNA was precipitated, and amplified for
another 22 cycles using primers containing Illumina priming sequences
with integrated  barcodes.  These  PCR  reactions  were  cleaned  using  a
PCR clean  up  kit  (Qiagen)  and  quantified.  Samples  were  mixed  and
submitted for sequencing on the MiSeq instrument, paired-end 250 nt read
option.
</dd><dt class="description">
<span class="phvb8t-x-x-120">MiSeq Library Analysis</span> </dt><dd class="description">Paired reads were split according to the index read.
Libraries were aligned against an index of all possible <span class="phvro8t-x-x-120">Dscam1 </span>ligamer
arrangements  using  bowtie2  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2012">Langmead  and  Salzberg</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2012">2012</a>]  in  the
“-very-sensitive-local”  mode  and  constrained  using  “-no-discordant”  to
only look for reads where both pairs aligned to the same isoform. Read
counts per isoform were extracted using the SAMtools software package
                                                                                

                                                                                
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009d">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009d">2009a</a>].  Analysis  of  count  values  and  graph  generation  was
performed using R [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRDevelopmentCoreTeam2011">R Development Core Team</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRDevelopmentCoreTeam2011">2011</a>].
</dd><dt class="description">
<span class="phvb8t-x-x-120">Triple Read Sequencing</span> </dt><dd class="description">To interrogate, RNA samples, reverse transcription
was performed  using  5  <span class="cmmi-12">&#956;</span>g  total  RNA,  Superscript  II  (Invitrogen)  and
random hexamer at 42o C for one hour. Three strand-switching control
experiments  were  performed  by  mixing  plasmids  encoding  isoforms
<span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">1</span><span class="cmmi-8">.</span><span class="cmr-8">33</span><span class="cmmi-8">.</span><span class="cmr-8">9</span></sup>, <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">12</span><span class="cmmi-8">.</span><span class="cmr-8">32</span><span class="cmmi-8">.</span><span class="cmr-8">9</span></sup>, <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">1</span><span class="cmmi-8">.</span><span class="cmr-8">24</span><span class="cmmi-8">.</span><span class="cmr-8">6</span></sup>, and <span class="phvro8t-x-x-120">Dscam1</span><sup><span class="cmr-8">7</span><span class="cmmi-8">.</span><span class="cmr-8">9</span><span class="cmmi-8">.</span><span class="cmr-8">6</span></sup> in the ratios of
3:3:1:1, 1:1:1:5, and 1:1:1:1. PCR primers specific to exon 3 (Not1Ex3For:
TAT CGG CGG CCG CGG ACG TCC ATG TGC GAG CCG) and exon
10 (Ex10RevNot1: ATA TCG CGG CCG CGA GGA TCC ATC TGG GAG
GTA) with  a  Not  I  restriction  enzyme  site  on  the  5<span class="cmsy-8">&#8242;</span>&nbsp;ends  were  used
to amplify  the  cDNA  or  plasmids  containing  the  region  encompassing
exons 4,  6,  and  9  using  Phusion  polymerase  (NEB)  with  an  annealing
temperature  of  55 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>and  1  minute  extension.  PCR  products  were  gel
purified and digested with Not1 for 2 hours at 37 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C</span>, followed by a heat
inactivation at 65 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>for 20 minutes. 0.5 <span class="cmmi-12">&#956;</span>g of the digested PCR products
were circularized in 500 <span class="cmmi-12">&#956;</span>L 1X T4 ligase buffer (NEB) with 1 <span class="cmmi-12">&#956;</span>l T4 ligase at
18 <sup><span class="cmsy-8">&#8728;</span> </sup> <span class="cmr-12">C </span>overnight. Inverse PCR was then performed with primers specific to
exons 7 (PEex7Rev:CAA GCA GAA GAC GGC ATA CGA GAT CGG TCT
CGG CAT TCC TGC TGA ACC GCT CTT CCG ATC TAT GAA CTT GTA
                                                                                

                                                                                
CCA T) and 8 (PEex8For: AAT GAT ACG GCG ACC ACC GAG ATC TAC
ACT GTT TCC CTA CAC GAC GCT CTT CCG ATC TAA GTG CAA GTC
ATG G) that containing Illumina paired-end clustering sequences using
Phusion polymerase (NEB) with a 55 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>annealing temperature and 30
second extension. Libraries were gel purified, quantified and clustered on
a Genome Analyzer IIx paired-end flow cell on an Illumina cluster station
using the standard clustering protocol.
<!--l. 262--><p class="noindent">Sequencing was performed on an Illumina GAIIx by modifying the protocol
for paired-end  sequencing  with  an  index  read.  Briefly,  read  1  was
performed for 24 cycles with a primer complementary to the 5<span class="cmsy-8">&#8242;</span>&nbsp;end of exon
8 (Ex8For:ACG ACG CTC TTC CGA TCT AAG TGC AAG TCA TGG).
The flow cell was denatured to remove the exon 9 sequencing products,
a primer complementary to exon 3 (Ex3For:CCC GGG ACG TCC ATG
TGC GAG  CCG)  was  annealed,  and  read  2  sequenced  for  12  cycles.
Next, the  flow  cell  was  re-clustered  using  the  paired-end  protocol  and
read 3 performed for 20 cycles using a primer complementary to exon 7
(Ex7Rev:GAA CCG CTC TTC CGA TCT ATG AAC TTG TAC CAT).
</p><!--l. 264--><p class="noindent">Base calling  was  performed  from  the  raw  images  using  the  Firecrest,
Bustard,   and   Gerald   software   modules   of   GAPipeline-1.4.0   and   a
matrix.txt  file  for  a  PhiX  lane  from  a  previous  flow  cell  for  calibration.
                                                                                

                                                                                
This generated  a  single  FastQ  file  per  lane  containing  the  three  reads
from each cluster concatenated together. The reads within the FastQ files
were parsed to separate the three reads and the identity of each exon
within each cluster, and thus the full isoform, determined by matching to a
database of known exon sequences using Perl scripts.
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">Determining Sequencing Similarity of </span><span class="phvbo8t-x-x-120">Dscam1 </span><span class="phvb8t-x-x-120">Sequences</span> </dt><dd class="description">
Endogenous  <span class="phvro8t-x-x-120">Dscam1  </span>sequences  were  obtained  from  genomic  build
DM3 using  BEDTools  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XQuinlan2010">Quinlan  and  Hall</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XQuinlan2010">2010</a>].  All  possible  <span class="phvro8t-x-x-120">Dscam1</span>
were assembled  using  a  PERL  script.  Five  hundred  random  isoforms
were obtained,  and  aligned  using  TCOFFEE  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNotredame2000">Notredame  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNotredame2000">2000</a>]
in the Jalview package [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWaterhouse2009">Waterhouse et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWaterhouse2009">2009</a>]. Consensus scores of
alignments  were  exported  and  graphed  in  R.  The  same  analysis  was
performed on <span class="phvro8t-x-x-120">Dscam1 </span>ligation products, except ligamer sequences were
used in place of endogenous exonic sequences.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Trans-transcript RNA design</span> </dt><dd class="description">PCR  was  performed  using  oligos  specified
in Table   S2   in   desired   combinations.   These   oligos   have   partial
complementarity  to  the  open  reading  frame  (ORF)  of  human  eiF4A3.
A plasmid  containing  this  ORF  was  used  (RefSeq:  NM_014740  )  as  a
                                                                                

                                                                                
template for PCR. The sequences of the individual RNAs were confirmed
by sequencing.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Reverse Transcription</span> </dt><dd class="description">Reverse                                                     transcription
was performed using SuperScript III (Invitrogen), 200 ng poly(A) selected
RNA, and either a gene-specific antisense primer or anchored oligo(dT).
</dd><dt class="description">
<span class="phvb8t-x-x-120">Radioactive PCR</span> </dt><dd class="description">A  5<span class="cmsy-8">&#8242;</span>&nbsp;<sup><span class="cmr-8">32</span></sup>P-radiolabeled  antisense  primer  was  used  in  PCR
reactions run for a limiting number of PCR cycles. Multiple cycle numbers
were performed to test for expected increases in signal (typically 15,18,
and 21 cycles). Reactions analyzed on denaturing acrylamide gels to size
resolve ligation products. Bands were quantified using a Typhoon imager
(GE Healthcare) and the ImageQuant software package (GE Healthcare).
</dd><dt class="description">
<span class="phvb8t-x-x-120">Endpoint PCR</span> </dt><dd class="description">Using a 25 <span class="cmmi-12">&#956;</span>l reaction volume, after 8 hours of ligation, 2 <span class="cmmi-12">&#956;</span>l
reaction volume was added into a 20 <span class="cmmi-12">&#956;</span>l PCR reaction with 500 nM primers
and 50% Green master mix from Promega. Samples were amplified for 35
cycles using a hybridization temperature 5 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>below the <span class="phvro8t-x-x-120">T</span><sub>
<span class="cmmi-8">m</span></sub> of the primers.
10 <span class="cmmi-12">&#956;</span>l of each PCR reaction was run out on an appropriate percentage
native 29:1 (acrylamide: bis-acrylamide) native acrylamide gel.
                                                                                

                                                                                
</dd><dt class="description">
<span class="phvb8t-x-x-120">Statistical Analysis</span> </dt><dd class="description">Error bars represent the standard error of the mean of the
experimental replicates. Errors were propagated from individual standard
deviations
according to the formula <span class="cmr-12">&#916;</span>Z = Z(SQRT(((<span class="cmr-12">&#916;</span>A/A)^2)+((<span class="cmr-12">&#916;</span>B/B)^2))) where Z
= A/B.
</dd></dl>
<a id="x1-40001r74"></a>
<h3 class="sectionHead"><span class="titlemark">2.5 </span> <a id="x1-410005"></a>End Matter</h3>
<dl class="description"><dt class="description">
<span class="phvb8t-x-x-120">ACKNOWLEDGEMENTS</span> </dt><dd class="description">Alper Kucukural for bioinformatics support. Kristen
Lynch and Eric Shuman for plasmids. Andrés Muro for Fn1 cell lines.
</dd><dt class="description">
<span class="phvb8t-x-x-120">AUTHOR CONTRIBUTIONS</span> </dt><dd class="description">CKR,   BRG,   PDZ,   and   MJM   conceived   the
experiments. CKR and SO performed the experiments and conducted the
analysis. CKR, PDZ, and MJM wrote the paper.
</dd><dt class="description">
<span class="phvb8t-x-x-120">COMPETING FINANCIAL INTERESTS</span> </dt><dd class="description">The authors have applied for a patent
(12/906,678) concerning the SeqZip protocol.
                                                                                

                                                                                
</dd></dl>
                                                                                

                                                                                
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&nbsp;3</span><br><a id="x1-420003"></a>An Ancient Transcription Factor Initiates the Burst of piRNA Production
during Early Meiosis in Mouse Testes</h2>
<a id="x1-42001r75"></a>
<h3 class="sectionHead"><span class="titlemark">3.1 </span> <a id="x1-430001"></a>Preface</h3>
<!--l. 9--><p class="noindent">The contents of this Chapter have been published previously as:
</p><div class="quote">
<!--l. 14--><p class="noindent"><a id="XLi2013h"></a><span class="phvro8t-x-x-120">Li, X.</span><span class="phvro8t-x-x-120">&nbsp;Z.,  Roy,  C.</span><span class="phvro8t-x-x-120">&nbsp;K.,  Dong,  X.,  Bolcun-Filas,  E.,  Wang,  J.,  Han,</span>
<span class="phvro8t-x-x-120">B.</span><span class="phvro8t-x-x-120">&nbsp;W., Xu, J., Moore, M.</span><span class="phvro8t-x-x-120">&nbsp;J., Schimenti, J.</span><span class="phvro8t-x-x-120">&nbsp;C., Weng, Z., and Zamore,</span>
<span class="phvro8t-x-x-120">P.</span><span class="phvro8t-x-x-120">&nbsp;D. (2013a).   An ancient transcription factor initiates the burst of</span>
<span class="phvro8t-x-x-120">piRNA production during early meiosis in mouse testes.  </span>Molecular
cell<span class="phvro8t-x-x-120">, 50(1):67–81</span></p></div>
<!--l. 17--><p class="noindent">For information not contained in this chapter (i.e. supplemental tables), please refer to
the following locations:
</p><!--l. 19--><p class="noindent">NCBI: <a href="http://www.ncbi.nlm.nih.gov/pubmed/23523368" class="url"><span class="ectt-1200">http://www.ncbi.nlm.nih.gov/pubmed/23523368</span></a> <br class="newline">Molecular Cell: <a href="http://www.cell.com/molecular-cell/abstract/S1097-2765(13)00172-X" class="url"><span class="ectt-1200">http://www.cell.com/molecular-_cell/abstract/S1097-_2765(13)00172-_X</span></a>
</p><!--l. 22--><p class="noindent">Supplemental tables can also be found on the Zamore Lab website:
<a href="http://www.sciencedirect.com/science/article/pii/S109727651300172X#appd003" class="url"><span class="ectt-1200">http://www.sciencedirect.com/science/article/pii/S109727651300172X#appd003</span></a>
<a id="x1-43001r77"></a>
</p>
                                                                                

                                                                                
<h3 class="sectionHead"><span class="titlemark">3.2 </span> <a id="x1-440002"></a>Introduction</h3>
<!--l. 27--><p class="noindent">P-element induced wimpy testis (PIWI)-interacting RNAs (piRNAs) can be
distinguished from other animal small silencing RNAs by their longer length (typically
23–35 nt), 2<span class="cmsy-8">&#8242;</span> -O-methyl-modified 3<span class="cmsy-8">&#8242;</span>&nbsp;termini, and association with PIWI proteins,
a distinct subgroup of Argonaute proteins, the small RNA-guided proteins
responsible for RNA interference and related pathways [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCenik2011">Cenik
and Zamore</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCenik2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFarazi2008">Farazi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFarazi2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKim2009">Kim et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKim2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKumar1998">Kumar and
Carmichael</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKumar1998">1998</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XThomson2009">Thomson and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XThomson2009">2009</a>]. piRNA production does not require
Dicer, the double-stranded RNA endonuclease that makes microRNAs (miRNAs) and
small interfering RNAs (siRNAs), and piRNAs are thought to derive from
single-stranded rather than double-stranded RNA [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">Houwing et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2006">Vagin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2006">2006</a>].
</p><!--l. 29--><p class="noindent">In most bilateral animals, germline piRNAs protect the genome from transposon
activation, but also have other functions [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">Aravin and Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">Aravin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">2007a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2001">2001</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAshe2012">Ashe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAshe2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">Carmell
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHartig2007">Hartig et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHartig2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi2008">Kuramochi-Miyagawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2012">Lee
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShirayama2012">Shirayama et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShirayama2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2004">Vagin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2004">2004</a>]. A few days after
birth, the majority of piRNAs in the mouse testis are pre-pachytene piRNAs;
25% of these piRNA species map to more than one location in the genome.
A second class of piRNAs, typically derived from intergenic regions, has
                                                                                

                                                                                
been reported to emerge in the mouse testis 14.5 days postpartum (dpp),
when the developing spermatocytes synchronously enter the pachytene
phase of meiotic prophase I. These pachytene piRNAs compose &gt;95% of
piRNAs in the adult mouse testis. Loss of genes required to make pachytene
piRNAs blocks production of mature sperm [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2001">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2001">2001</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and
Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">Reuter et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>]. What triggers the
accumulation of pachytene piRNAs when spermatocytes enter the pachynema is
unknown.
</p><!--l. 31--><p class="noindent">In Caenorhabditis elegans, each piRNA is processed from its own short RNA
polymerase II (Pol II) transcript [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGu2012">Gu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGu2012">2012</a>]. In contrast, insect and mouse
piRNAs are thought to be processed from long RNAs transcribed from large
piRNA loci. Supporting this view, a transposon inserted into the 5<span class="cmsy-8">&#8242;</span>&nbsp;end of the
flamenco piRNA cluster in flies reduces the production of flamenco piRNAs 168
kbp 3<span class="cmsy-8">&#8242;</span> &nbsp;to the insertion, suggesting that it disrupts transcription of the entire
locus [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>]. High-throughput sequencing and chromatin
immunoprecipitation (ChIP) has been used to define the genomic structure of the
piRNA-producing genes of immortalized, cultured silk moth BmN4 cells [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2012">Kawaoka
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2012">2012</a>]. However, for flies and mice, we do not know the structure of
piRNA-producing genes, their transcripts, or the nature of the promoters that control
their expression.
                                                                                

                                                                                
</p><!--l. 33--><p class="noindent">Instead, piRNA loci have been defined as clusters: regions of the genome with a
high density of mapping piRNA sequences [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">Grivna et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRo2007">Ro
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRo2007">2007</a>]. In reality, piRNA-producing loci correspond to discrete transcription
units that include both intergenic loci believed to encode no protein [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2008">Brennecke and Malone</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>] and
protein-coding genes that also produce piRNAs [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007">2007b</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">Robine
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSaito2009">Saito et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSaito2009">2009</a>].
</p><!--l. 35--><p class="noindent">We used high-throughput sequencing data to define the genes and transcripts that
produce piRNAs in the juvenile and adult mouse testis. Using these data, we
identified the factor that initiates transcription of pachytene piRNA genes: A-MYB
(MYBL1), a spermatocyte protein that serves as a master regulator of genes
encoding proteins required for cell-cycle progression through the pachytene stage of
meiosis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrauth1994">Trauth et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrauth1994">1994</a>]. A-MYB also initiates
transcription of the genes encoding many piRNA biogenesis factors. The
combined action of A-MYB at the promoters of genes producing pachytene
piRNA precursor transcripts and genes encoding piRNA biogenesis proteins
creates a coherent feedforward loop that triggers a &gt;6,000-fold increase in
pachytene piRNA abundance during the ~5 days between the early and late
phases of the pachytene stage of male meiosis. A-MYB also promotes its own
transcription through a positive feedback loop. The A-MYB-regulated feedforward
                                                                                

                                                                                
loop is evolutionarily conserved: A-MYB is bound to the promoters of both
piRNA clusters and PIWIL1, TDRD1, and TDRD3 in the rooster (Gallus gallus)
testis.
<a id="x1-44001r78"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">3.3 </span> <a id="x1-450003"></a>Results</h3>
<a id="x1-45001r73"></a>
<h4 class="subsectionHead"><span class="titlemark">3.3.1 </span> <a id="x1-460001"></a>Defining piRNA-Producing Transcripts in the Mouse Testis</h4>
<!--l. 42--><p class="noindent">To define the structure of piRNA-producing loci in the testis of wild-type adult
mice, we assembled the transcripts detected by three biological replicates of
strand-specific, paired-end, rRNA-depleted, total RNA sequencing (RNA-seq;
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46001r1">3.1<!--tex4ht:ref: MolCel:fig:MolCelF1 --></a>A). We mapped reads to the mouse genome using TopHat [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">Trapnell
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">2009</a>] and performed de novo transcriptome assembly using Trinity
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">Grabherr et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">2011</a>] to identify unannotated exon-exon junctions. We used all
mapped reads, including reads corresponding to unannotated exon-exon
junctions, to perform reference-based transcript assembly (Cufflinks; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">Trapnell
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">2010</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-46001r1"></a>
                                                                                

                                                                                
<!--l. 46--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis24x.png" alt="PIC" class="graphics" width="320.19624pt" height="444.66124pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis24x.png" src="Figures/MolCel/MolCel2013_Fig1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.1: </span><span class="content">piRNA Precursors are RNA Pol II Transcripts<br class="newline">(A) Strategy  to  assemble  the  mouse  testis  transcriptome.  Rectangles  with
rounded corners,  input  or  output  data;  rectangles,  processes.  Decisions  are
shown without boxing.(B) Aggregated data for piRNA-producing transcripts (5%
trimmed mean). Oxidized small RNA (&gt;23 nt) sequencing data were used to
                                                                                

                                                                                
detect piRNAs; transcript abundance was measured using total RNA depleted
of rRNA (RNA-seq). RNA Pol III data were from SRA001030. Dotted lines show
the transcriptional start site (Start) and site of polyadenylation (End). See also
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">3.2<!--tex4ht:ref: MolCel:fig:MolCelS1 --></a>. </span></div><!--tex4ht:label?: x1-46001r1 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-46002r2"></a>
                                                                                

                                                                                
<!--l. 56--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis25x.png" alt="PIC" class="graphics" width="369.37999pt" height="504.88625pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis25x.png" src="Figures/MolCel/MolCel2013_FigS1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.2: </span><span class="content">See subsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-470001">3.3.1.1<!--tex4ht:ref: MolCel:subsubsec:cap:Figure S1 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-46002r2 -->
                                                                                

                                                                                
<!--l. 64--><p class="noindent">To identify the transcripts that produce piRNAs, we sequenced piRNAs from six
developmental stages of mouse testes (10.5 dpp, 12.5 dpp, 14.5 dpp, 17.5 dpp, 20.5
dpp, and adult) and mapped them to the assembled transcripts. The first round of
spermatogenesis proceeds synchronously among the tubules of the testis:
mouse testes at 10.5 dpp advance no further than the zygotene stage (staging
according to [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNEBEL1961">Nebel et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNEBEL1961">1961</a>]; 12.5 dpp to the early pachytene; 14.5
dpp to the middle pachytene; 17.5 to the late pachytene; and 20.5 dpp to
the round spermatid stage. For each stage, we prepared two sequencing
libraries: one comprising all small RNAs and one in which oxidation was used to
enrich for piRNAs by virtue of their 2<span class="cmsy-8">&#8242;</span>-O-methyl-modified 3<span class="cmsy-8">&#8242;</span>&nbsp;termini [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">Ghildiyal
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">2008</a>].
</p><!--l. 66--><p class="noindent">To qualify as a piRNA-producing transcript, an assembled RNA was required to
produce either a sufficiently high piRNA abundance (&gt;100 ppm; parts per million
uniquely mapped reads) or density (&gt;100 rpkm; reads per kilobase of transcript per
million uniquely mapped reads). These criteria retained both long transcripts
producing an abundance of piRNAs and short transcripts generating many piRNAs
per unit of length. To refine the termini of each piRNA-producing transcript, we
supplemented the RNA-seq data with high-throughput sequencing of the 5<span class="cmsy-8">&#8242;</span>&nbsp;ends of
RNAs bearing an N(5<span class="cmsy-8">&#8242;</span>&nbsp;)ppp(5<span class="cmsy-8">&#8242;</span>&nbsp;)N cap structure (cap analysis of gene expression;
CAGE) and the 3<span class="cmsy-8">&#8242;</span>&nbsp;ends of transcripts preceding the poly(A) tail (polyadenylation site
sequencing; PAS-seq). The assembled piRNA-producing transcripts likely correspond
                                                                                

                                                                                
to continuous RNAs <span class="phvro8t-x-x-120">in vivo </span>because the CAGE library used to annotate transcript
5<span class="cmsy-8">&#8242;</span> &nbsp;ends was constructed after two rounds of poly(A) selection. Thus, the RNA
molecules in the library derive from complete transcripts extending from
the 5<span class="cmsy-8">&#8242;</span> &nbsp;cap to the poly(A) tail (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46001r1">3.1<!--tex4ht:ref: MolCel:fig:MolCelF1 --></a>B). Conventional 5<span class="cmsy-8">&#8242;</span>&nbsp;and 3<span class="cmsy-8">&#8242;</span>&nbsp;RACE
(rapid amplification of cDNA ends) analysis of piRNA-producing transcripts
confirmed the ends of 16 loci (data not shown). To provide additional confirmation
of the 5<span class="cmsy-8">&#8242;</span> &nbsp;end of each piRNA-producing transcript, we also determined the
locations of histone H3 bearing trimethylated lysine 4 (H3K4me3), a histone
modification associated with RNA Pol II transcription start sites <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuenther2007">Guenther
et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuenther2007">2007</a>].
<a id="x1-46003r60"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.1.1 </span> <a id="x1-470001"></a>Caption for Figure 3.2</h5>
<!--l. 70--><p class="noindent">(A) Positions of the 214 major piRNA-producing genes on the 19 autosomes
of mice. We detected no loci on the X or Y chromosomes. (B) Cumulative
distributions for all piRNAs and for uniquely mapping piRNAs comparing
the piRNA loci defined by our methods and by previous approaches [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>]. (C) Histogram of distances (in 200 bp bins)
from the annotated 5<span class="cmsy-8">&#8242;</span>&nbsp;or 3<span class="cmsy-8">&#8242;</span>&nbsp;end of a piRNA gene (this study) or cluster to
the nearest peak of reads from high-throughput sequencing for transcript
                                                                                

                                                                                
5<span class="cmsy-8">&#8242;</span> &nbsp;(CAGE-seq) or 3<span class="cmsy-8">&#8242;</span>&nbsp;(PAS-seq) ends, transcription start sites (H3K4me3) or A-MYB
binding.
<a id="x1-47001r80"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.2 </span> <a id="x1-480002"></a>piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</h4>
<!--l. 75--><p class="noindent">The presence of 5<span class="cmsy-8">&#8242;</span>&nbsp;caps and poly(A) tails and the binding of histone H3K4me3 to the
genomic DNA immediately upstream of the transcription start site of each piRNA
locus suggest that piRNA transcripts are produced by RNA pol II <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46001r1">3.1<!--tex4ht:ref: MolCel:fig:MolCelF1 --></a>. Moreover, using
antibodies to RNA pol II but not RNA pol III, ChIP-seq showed a peak at the
transcription start site as well as polymerase occupancy across the entire
piRNA gene (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46001r1">3.1<!--tex4ht:ref: MolCel:fig:MolCelF1 --></a>B; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKutter2011">Kutter et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKutter2011">2011</a>]. We conclude that piRNA
transcripts are conventional RNA pol II transcripts bearing 5<span class="cmsy-8">&#8242;</span>&nbsp;caps and 3<span class="cmsy-8">&#8242;</span>&nbsp;poly(A)
tails.
<a id="x1-48001r84"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.3 </span> <a id="x1-490003"></a>A Transcript-based Set of piRNA Loci</h4>
<!--l. 80--><p class="noindent">Our transcriptome assembly yielded 467 piRNA-producing transcripts that define 214
genomic loci (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">3.2<!--tex4ht:ref: MolCel:fig:MolCelS1 --></a>A and Table S1). Among the ~2.2 million distinct piRNA
species and ~8.8 million piRNA reads from the adult mouse testis, the 214 genomic
                                                                                

                                                                                
loci account for 95% of all piRNAs.
</p><!--l. 82--><p class="noindent">Previous studies defined piRNA clusters based solely on small RNA sequencing data
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">2007a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>]. Our approach differs in
that it (1) uses RNA-seq data, whose greater read length facilitates the identification
of introns, allowing us to define the architecture of piRNA precursor transcripts and
(2) uses CAGE, PAS-seq, and H3K4me3 ChIP-seq data to refine the 5<span class="cmsy-8">&#8242;</span>&nbsp;and
3<span class="cmsy-8">&#8242;</span> &nbsp;ends of the piRNA transcripts. Consequently, the piRNA loci presented here
account for more piRNAs using fewer genomic base pairs than those previously
defined (Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">3.2<!--tex4ht:ref: MolCel:fig:MolCelS1 --></a>B and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">3.2<!--tex4ht:ref: MolCel:fig:MolCelS1 --></a>C; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>]. Our
piRNA-producing loci include 41 piRNA loci that escaped previous detection [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">Aravin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">2007a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>], 37 of which contain introns.
The 41 loci account for 2% of piRNAs at 10.5 dpp and 0.36% in the adult
testis.
<a id="x1-49001r85"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.4 </span> <a id="x1-500004"></a>Three Classes of piRNAs During Post-Natal Spermatogenesis</h4>
<!--l. 87--><p class="noindent">Mice produce three PIWI proteins: MIWI2 (PIWIL4), which binds piRNAs
in perinatal testis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">Aravin and Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">Carmell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XCarmell2007">2007</a>]; MILI
(PIWIL2), which binds piRNAs at least until the round spermatid stage of
spermatogenesis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2007a">2007a</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKuramochi-Miyagawa2004">2004</a>];
                                                                                

                                                                                
and MIWI (PIWILl), which is first produced during the pachytene stage of
meiosis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>]. From 10.5 to 20.5 dpp, piRNA abundance
increases and longer piRNAs appear, reflecting a switch from MILI-bound
piRNAs, which have a 26-27 nt modal length [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">Aravin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2006">2006</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">Aravin and
Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMontgomery1998">Montgomery et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMontgomery1998">1998</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">Robine et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">2009</a>], to MIWI-bound
piRNAs, which have a 30 nt modal length (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>A; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2009">Reuter et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XReuter2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">Robine
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">2009</a>]. This switch occurs at the pachytene phase of meiosis. MILI-bound
pre-pachytene piRNAs predominate before the onset of pachynema; at the
pachytene and round spermatid stages, most piRNAs are MIWI-bound pachytene
piRNAs.
</p><!--l. 89--><p class="noindent">We used hierarchical clustering to analyze the change in piRNA abundance from 10.5
to 20.5 dpp for the 214 genes defined by our data (Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>A and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>A and Table
S2). Three types of piRNA-producing genes were identified according to when their
piRNAs first accumulate and how their expression changes during spermatogenesis:
84 pre-pachytene, 100 pachytene, and 30 hybrid loci. At 10.5 dpp, the earliest time
we evaluated, 84 genes dominate piRNA production (median piRNA abundance per
gene = 16 rpkm; Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>B). Nearly all (81 out of 84) were congruent with
protein-coding genes. The 84 pre-pachytene piRNA genes account for 13% of
piRNAs at 10.5 dpp, but only 0.31% of piRNAs in the adult testis. Of the
pre-pachytene piRNAs accounted for by the 84 loci, 15% derive from 31
piRNA-producing genes that, to our knowledge, have not previously been
                                                                                

                                                                                
described.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-50001r3"></a>
                                                                                

                                                                                
<!--l. 93--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis26x.png" alt="PIC" class="graphics" width="370.38374pt" height="485.81499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis26x.png" src="Figures/MolCel/MolCel2013_Fig2.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.3: </span><span class="content">See subsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-510001">3.3.4.1<!--tex4ht:ref: MolCel:subsubsection:cap:Figure F2 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-50001r3 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-50002r4"></a>
                                                                                

                                                                                
<!--l. 102--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis27x.png" alt="PIC" class="graphics" width="408.52625pt" height="536.00249pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis27x.png" src="Figures/MolCel/MolCel2013_FigS2.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.4: </span><span class="content">See subsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-520002">3.3.4.2<!--tex4ht:ref: MolCel:subsubsection:cap:Figure S2 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-50002r4 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 110--><p class="noindent">A parallel analysis of piRNA precursor transcription using RNA-seq (&gt;100 nt)
corroborated the classification based on piRNA abundance; of the 100 piRNA genes
classified as pachytene based on the developmental expression profile of their
piRNAs, 93 were grouped as pachytene according to the developmental expression
profile of their transcripts. Of these 93, 89 are intergenic. All 84 piRNA genes
designated pre-pachytene using piRNA data were classified as pre-pachytene
according to their transcript abundance.
</p><!--l. 112--><p class="noindent">Despite their name, pre-pachytene piRNAs were readily detected in &gt;90% and
~95% pure pachytene spermatocytes, as well as round spermatids (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>B; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>]. Transcript abundance
from the 84 pre-pachytene loci was high at 3 dpp (median abundance = 11
rpkm), higher by 8 dpp (18 rpkm), and lower in purified leptotene/zygotene
spermatocytes (3.3 rpkm; <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>B). Yet piRNA precursor transcripts were readily
detectable in purified pachytene spermatocytes at a level (4.6 rpkm) comparable
to that in purified leptotene/zygotene spermatocytes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>B); [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>]. From 10.5 to 20.5 dpp, the steady-state
level of pre-pachytene piRNA precursor transcripts remained constant (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>B).
</p><!--l. 114--><p class="noindent">Finally, the abundance of pre-pachytene piRNA precursor transcripts was better
correlated with pre-pachytene piRNA abundance at 17.5 dpp (<span class="cmmi-12">&#961; </span>= 0.47), when
                                                                                

                                                                                
pachytene spermatocytes compose a larger fraction of the testis, than at 10.5,
12.5, or 14.5 dpp (0.32 <span class="cmsy-10x-x-120">&#8805; </span><span class="cmmi-12">&#961; </span><span class="cmsy-10x-x-120">&#8804; </span>0.40; Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>C). Our data suggest that the
pre-pachytene loci continue to be transcribed and processed into piRNAs long
after spermatocytes enter the pachytene stage of meiosis. Thus, the name
pre-pachytene piRNA is a misnomer that should be retained only for historical
reasons.
</p><!--l. 116--><p class="noindent">Hierarchical clustering identified 100 pachytene genes whose piRNAs emerge at 12.5
dpp, 2 days earlier than previously reported [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>]. Nearly all the
pachytene genes are intergenic (93 out of 100). piRNA expression from pachytene
piRNA genes peaks at 17.5 dpp (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>B). Overall, the median abundance of
piRNAs from these 100 loci increased &gt;6,000-fold from 10.5 to 17.5 dpp. Transcripts
from pachytene genes were low at 10.5 dpp (median abundance = 0.15 rpkm) and
increased 116-fold from 10.5 to 17.5 dpp. From 10.5 to 20.5 dpp, the dynamics of
pachytene piRNA abundance from each piRNA gene correlated with the increase in
abundance of its precursor transcripts (0.39 <span class="cmsy-10x-x-120">&#8805; </span><span class="cmmi-12">&#961; </span><span class="cmsy-10x-x-120">&#8804; </span>0.63; <span class="cmmi-12">&#961;value </span><span class="cmsy-10x-x-120">&#8804; </span><span class="cmr-12">7</span><span class="cmmi-12">.</span><span class="cmr-12">3</span><span class="cmmi-12">x</span><span class="cmr-12">10 </span><span class="cmsy-10x-x-120">- </span><span class="cmr-12">5</span>; Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a>C). The 100 pachytene genes account for 92% of piRNAs in the adult testis,
making it unlikely that biologically functional pachytene piRNAs originate from
thousands of genomic loci [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>]. Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50003r5">3.5<!--tex4ht:ref: MolCel:fig:MolCelF3 --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50004r6">3.6<!--tex4ht:ref: MolCel:fig:MolCelS3 --></a> provide
examples of pachytene and pre-pachytene piRNA genes defined by our
data.
                                                                                

                                                                                
                                                                                

                                                                                
                                                                                </p>
<hr class="figure"><div class="figure">
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <a id="x1-50003r5"></a>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 121--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis28x.png" alt="PIC" class="graphics" width="482.80374pt" height="289.07999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis28x.png" src="Figures/MolCel/MolCel2013_Fig3.eps"  
-->
<br> </p><div class="caption"><span class="id">Figure&nbsp;3.5: </span><span class="content">Previous cluster boundaries are from <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>] in gray and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>] in dark gray). </span></div><!--tex4ht:label?: x1-50003r5 -->
                                                                                                                                                                                      
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-50004r6"></a>
                                                                                

                                                                                
<!--l. 131--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis29x.png" alt="PIC" class="graphics" width="387.4475pt" height="289.07999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis29x.png" src="Figures/MolCel/MolCel2013_FigS3.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.6: </span><span class="content">Previous cluster boundaries are from <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">Lau et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLau2006">2006</a>] in gray and
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">Girard et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGirard2006">2006</a>] in dark gray). </span></div><!--tex4ht:label?: x1-50004r6 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 140--><p class="noindent">Hierarchical clustering detected a third class, hybrid piRNAs, which derives from 30
genes with characteristics of both pre-pachytene and pachytene piRNA loci. Like
pre-pachytene, hybrid piRNAs were detected at 10.5 dpp (median abundance = 3.7
rpkm) and in purified spermatogonia [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>]. Like pachytene piRNAs,
hybrid piRNA abundance increased during the pachytene stage of meiosis, but the
increase was delayed until late (17.5 dpp) rather than early pachynema (14.5 dpp).
Overall, piRNAs from hybrid genes increased &gt;10-fold from 14.5 to 17.5 dpp. The
median abundance of piRNAs from hybrid piRNA genes ranged from 90-120 rpkm
in purified pachytene spermatocytes, &gt;20-fold greater than their median
abundance in spermatogonia [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>].
Moreover, hybrid piRNA precursor transcripts were readily detected in purified
pachytene spermatocytes (median abundance = 9.0 rpkm; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>]).
<a id="x1-50005r83"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.4.1 </span> <a id="x1-510001"></a>Caption for Figure 3.3</h5>
<!--l. 144--><p class="noindent">(A) Normalized piRNA density (rpkm) for each piRNA-producing gene is shown as a
heatmap across the developmental stages. Hierarchical clustering divided the genes
into three classes. Arrowheads mark seven pachytene piRNA genes that were not
classified as pachytene according to the change in the abundance of their precursor
                                                                                

                                                                                
RNAs from 10.5 to 17.5 dpp.(B) Top: box plots present piRNA density per gene as
spermatogenesis progresses (here and elsewhere, pre-pachytene in yellow and
pachytene in purple). Middle: expression of <span class="phvro8t-x-x-120">A-Myb</span>, <span class="phvro8t-x-x-120">B-Myb</span>, <span class="phvro8t-x-x-120">Mili</span>, and <span class="phvro8t-x-x-120">Miwi </span>was
measured by RNA-seq. Bottom: box plots present piRNA precursor expression per
gene, measured by RNA-seq, from 10.5 to 20.5 dpp. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50002r4">3.4<!--tex4ht:ref: MolCel:fig:MolCelS2 --></a> and Table
S2.
<a id="x1-51001r91"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.4.2 </span> <a id="x1-520002"></a>Caption for Figure 3.4</h5>
<!--l. 148--><p class="noindent">(A) As shown previously by others using lower temporal resolution, the modal length
of piRNAs increases as spermatogenesis proceeds to more advanced stages. (B)
Total piRNA rpkm abundance and piRNA transcript abundance per locus by class,
from purified spermatogonia, spermatocytes, round spermatids, and 3 dpp and 8 dpp
testis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>]. (C) Correlation between piRNA
abundance per locus and piRNA precursor transcription from 10.5 to 20.5 dpp.
Throughout the Figures, gold indicates pre-pachytene and purple indicates pachytene
piRNA loci.
<a id="x1-52001r86"></a>
</p>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">3.3.5 </span> <a id="x1-530005"></a><span class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</h4>
<!--l. 153--><p class="noindent">The coordinated increase in pachytene piRNA precursor transcripts suggests their
regulation by a common transcription factor or factors. Among the 100 pachytene
piRNA genes, 15 pairs (30 genes) are divergently transcribed. The 5<span class="cmsy-8">&#8242;</span>&nbsp;ends of the
piRNA precursor RNAs from each pair are close in genomic distance (median = 127
bp), suggesting that a shared promoter lies between the two transcription start
sites.
</p><!--l. 155--><p class="noindent">We took advantage of the unique genomic organization of these 15 pairs of
divergently transcribed piRNA genes to search for sequence motifs common to their
promoters. The MEME algorithm [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey1994">Bailey and Elkan</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey1994">1994</a>] revealed a motif highly
enriched in these bidirectional promoters (E = 8.3 x 10<sup><span class="cmr-8">12</span></sup>; Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>A). This motif
matches the binding site of the Myb family of transcription factors (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>A; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGupta2007">Gupta et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGupta2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNewburger2009">Newburger and Bulyk</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNewburger2009">2009</a>]. The Myb motif is
not restricted to bidirectional promoters; MEME identified the same motif
using the promoters of all pachytene piRNA genes (E = 9.1 x 10<sup><span class="cmsy-8">-</span><span class="cmr-8">28</span></sup>; Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>B).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-53001r7"></a>
                                                                                

                                                                                
<!--l. 159--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis30x.png" alt="PIC" class="graphics" width="393.46999pt" height="391.4625pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis30x.png" src="Figures/MolCel/MolCel2013_Fig4.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.7: </span><span class="content">(A)  Top:  MEME  identified  a  sequence  motif  in  the  bidirectional
promoters of the 15 pairs of divergently transcribed pachytene piRNA genes.
E value computed  by  MEME  measures  the  statistical  significance  of  the
motif. Middle:  Myb  motif  from  the  mouse  UniPROBE  database.  Bottom:
MEME-reported motif for the top 500 (by peak score) A-MYB ChIP-seq peaks
from adult mouse testes.(B) A-MYB ChIP-seq data for the common promoter
                                                                                

                                                                                
of the divergently transcribed pachytene piRNA genes <span class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and
<span class="phvro8t-x-x-120">17-qA3.3-26735.1</span>.(C) The distance from the annotated transcription start site
(TSS) of each piRNA gene to the nearest A-MYB peak. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53002r8">3.8<!--tex4ht:ref: MolCel:fig:MolCelS4 --></a>. </span></div><!--tex4ht:label?: x1-53001r7 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-53002r8"></a>
                                                                                

                                                                                
<!--l. 168--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis31x.png" alt="PIC" class="graphics" width="385.43999pt" height="454.69875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis31x.png" src="Figures/MolCel/MolCel2013_FigS4.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.8:   </span><span class="content">(A)   A-MYB   binds   to   the   common   promoter   of   divergently
transcribed pachytene  piRNA  loci  <span class="phvro8t-x-x-120">17-qA3.3-27363.1  </span>and  <span class="phvro8t-x-x-120">17-qA3.3-26735.1</span>.
The abundance of DNA fragments at the amplified region relative to a control
region (mean <span class="cmsy-10x-x-120">± </span>standard deviation; n = 3) was measured by qPCR (top). The
A-MYB ChIP-seq (red) and input (black) data for this pair of genes is presented
                                                                                

                                                                                
as in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>B. (B) ChIP-seq and qPCR were as in (A), but for the promoter
region of <span class="phvro8t-x-x-120">Miwi  </span>(Piwil1). Also shown is the RefSeq gene model. Exons, black;
introns, gray. </span></div><!--tex4ht:label?: x1-53002r8 -->
                                                                                

                                                                                
<!--l. 176--><p class="noindent">The Myb transcription factor family is conserved among eukaryotes. Like other
vertebrates, mice produce three Myb proteins, A-MYB (MYBL1), B-MYB
(MYBL2), and C-MYB (MYB), each with a distinct tissue distribution [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLatham1996">Latham
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLatham1996">1996</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMettus1994">Mettus et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMettus1994">1994</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOh1999">Oh and Reddy</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOh1999">1999</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrauth1994">Trauth et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrauth1994">1994</a>].
Testes produce both A- and B-MYB proteins. Multiple lines of evidence implicate
A-MYB, rather than B-MYB, as a candidate for regulating pachytene piRNA
transcription. First, the expression of <span class="phvro8t-x-x-120">A-Myb </span>during spermatogenesis resembles that
of pachytene piRNAs: <span class="phvro8t-x-x-120">A-Myb </span>transcripts appear at ~12.5 dpp and peak at 17.5 dpp
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>B; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>]. The expression of <span class="phvro8t-x-x-120">A-Myb </span>messenger RNA
(mRNA) increases ~15-fold from 8 dpp to 19 dpp, whereas <span class="phvro8t-x-x-120">B-Myb </span>mRNA expression
remains constant and low during the same time frame and into adulthood [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">Horvath
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">2009</a>]. Our RNA-seq data (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>B) corroborate these findings. Indeed, in
our RNA-seq analysis of adult testes, <span class="phvro8t-x-x-120">A-Myb </span>mRNA was 24-fold more abundant
than <span class="phvro8t-x-x-120">B-Myb</span>. Second, a testis-specific <span class="phvro8t-x-x-120">A-Myb </span>point-mutant allele, <span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span>,
which is caused by a cytosine-to-adenine transversion that changes alanine
213 to glutamic acid, leads to meiotic arrest at the pachytene stage with
subtle defects in autosome synapsis; <span class="phvro8t-x-x-120">A-Myb </span>null mutant mice have defects in
multiple tissues, including the testis and the mammary gland [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XToscani1997">Toscani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XToscani1997">1997</a>]. Third, our RNA-seq analysis of <span class="phvro8t-x-x-120">A-Myb</span>
mutant testes shows that there is no significant change in <span class="phvro8t-x-x-120">B-Myb </span>expression in
the mutant, compared to the heterozygous controls, at 14.5 or 17.5 dpp.
                                                                                

                                                                                
Finally, B-MYB protein is not detectable in pachytene spermatocytes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">Horvath
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">2009</a>].
</p><!--l. 178--><p class="noindent">To assess more directly the role of A-MYB in pachytene piRNA precursor
transcription, we used anti-A-MYB antibody to perform ChIP followed by
high-throughput sequencing of the A-MYB-bound DNA. The anti-A-MYB
antibody is specific for A-MYB, and the peptide used to raise the antibody is not
present in B-MYB. The model-based analysis of ChIP-seq (MACS) algorithm
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">Zhang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">2008</a>] reported 3,815 genomic regions with significant A-MYB
binding (false discovery rate, FDR &lt; 10<sup><span class="cmr-8">25</span></sup>); we call these regions A-MYB peaks
or peaks. Among the 500 peaks with the lowest FDR values, 394 (80%)
contained at least one significant site (<span class="cmmi-12">&#961; &lt; </span><span class="cmr-12">10</span><sup><span class="cmr-8">4</span></sup>) for the MYB binding motif
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>A). Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>B shows an example of such an A-MYB peak at the
bidirectional promoter of the divergently transcribed pair of pachytene piRNA
genes <span class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and <span class="phvro8t-x-x-120">17-qA3.3-26735.1</span>. A-MYB occupancy of this
genomic site was confirmed by ChIP and quantitative PCR (ChIP-qPCR) (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53002r8">3.8<!--tex4ht:ref: MolCel:fig:MolCelS4 --></a>A).
</p><!--l. 180--><p class="noindent">The median distance from the transcription start site to the nearest A-MYB peak was
~43 bp for the 100 pachytene piRNA genes but &gt;66,000 bp for the 84 pre-pachytene
genes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>C). Our data suggest that during mouse spermatogenesis A-MYB
binds to the promoters of both divergently and unidirectionally transcribed pachytene
                                                                                

                                                                                
piRNA genes.
</p><!--l. 182--><p class="noindent">To test the idea that A-MYB promotes transcription of pachytene, but not
pre-pachytene, piRNA genes, we used RNA-seq to measure the abundance of RNA &gt;
100 nt long from the testes of <span class="phvro8t-x-x-120">A-Myb </span>point-mutant (<span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span>) mice and their
heterozygous littermates (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">3.9<!--tex4ht:ref: MolCel:fig:MolCelF5 --></a>). Pachytene piRNA precursor transcripts—both
divergently and unidirectionally transcribed—were significantly depleted in <span class="phvro8t-x-x-120">A-Myb</span>
mutant testes compared to the heterozygotes: the median decrease was 45-fold at
14.5 dpp (q = 1.1 x 10<sup><span class="cmsy-8">-</span><span class="cmr-8">13</span></sup>) and 248-fold at 17.5 dpp (q = 3.9 x 10<sup><span class="cmsy-8">-</span><span class="cmr-8">23</span></sup>). The abundance
of pre-pachytene piRNA transcripts was not significantly changed (q <span class="cmsy-10x-x-120">&#8805; </span>0.34). The
binding of A-MYB to the promoters of pachytene piRNA genes, together with the
depletion of pachytene piRNA transcripts in the <span class="phvro8t-x-x-120">A-Myb </span>mutant, further supports
the view that A-MYB directly regulates transcription of pachytene piRNA
genes.
<a id="x1-53003r93"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.6 </span> <a id="x1-540006"></a><span class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</h4>
<!--l. 187--><p class="noindent">To test the consequences of the loss of piRNA precursor transcripts, we measured
piRNA abundance in the <span class="phvro8t-x-x-120">A-Myb </span>mutant. Like pachytene piRNA precursor
transcription, pachytene piRNA abundance significantly decreased in mutant testes.
                                                                                

                                                                                
At 14.5 dpp, median piRNA abundance per pachytene gene decreased 87-fold in
<span class="phvro8t-x-x-120">A-Myb </span>homozygous mutant testes compared to heterozygotes (<span class="cmmi-12">&#961; &lt; </span><span class="cmr-12">2</span><span class="cmmi-12">.</span><span class="cmr-12">2</span><span class="cmmi-12">X</span><span class="cmr-12">10</span><sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>; Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">3.9<!--tex4ht:ref: MolCel:fig:MolCelF5 --></a>. By 17.5 dpp, median pachytene piRNA abundance was &gt;9,000 times lower in the
<span class="phvro8t-x-x-120">A-Myb </span>mutant than the heterozygotes (P &lt; 2.2 x 10<sup><span class="cmsy-8">-</span><span class="cmr-8">16</span></sup>). In contrast, pre-pachytene
piRNA levels were essentially unaltered. Figure 6 presents examples of the effect at
14.5 and 17.5 dpp of the <span class="phvro8t-x-x-120">A-Myb </span>mutant on piRNA precursor transcript and
mature piRNA abundance for one pre-pachytene and three pachytene piRNA
genes.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-54001r9"></a>
                                                                                

                                                                                
<!--l. 191--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis32x.png" alt="PIC" class="graphics" width="399.4925pt" height="415.55249pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis32x.png" src="Figures/MolCel/MolCel2013_Fig5.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.9: </span><span class="content">The change in transcript or piRNA abundance per gene in <span class="phvro8t-x-x-120">A-Myb</span>
(n = 3) and <span class="phvro8t-x-x-120">Miwi </span>(n = 1) mutants compared to heterozygotes in testes isolated
at 14.5 and 17.5 dpp. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54002r10">3.10<!--tex4ht:ref: MolCel:fig:MolCelS5 --></a>. </span></div><!--tex4ht:label?: x1-54001r9 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-54002r10"></a>
                                                                                

                                                                                
<!--l. 200--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis33x.png" alt="PIC" class="graphics" width="308.15125pt" height="254.9525pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis33x.png" src="Figures/MolCel/MolCel2013_FigS5.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.10: </span><span class="content">Change in piRNA abundance per locus (rpkm) for <span class="phvro8t-x-x-120">Spo11 </span>(14.5
dpp), <span class="phvro8t-x-x-120">Miwi </span>(<span class="phvro8t-x-x-120">Piwil2</span>;  10.5  dpp),  <span class="phvro8t-x-x-120">Tdrd6  </span>(18  dpp),  and  <span class="phvro8t-x-x-120">Tdrd9  </span>(14  dpp)  mutants
compared to heterozygous controls. </span></div><!--tex4ht:label?: x1-54002r10 -->
                                                                                

                                                                                
<!--l. 208--><p class="noindent">Our data show that A-MYB binds to the promoters of pachytene piRNA genes;
<span class="phvro8t-x-x-120">A-Myb</span>, <span class="phvro8t-x-x-120">Miwi</span>, and pachytene piRNA precursor transcription begins at 12.5 dpp; and
<span class="phvro8t-x-x-120">A-Myb </span>mutant spermatocytes reach pachynema with subtle defects in autosome
synapsis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>]. Could pachytene piRNA depletion nonetheless
be an indirect consequence of the meiotic arrest caused by the <span class="phvro8t-x-x-120">A-Myb </span>mutant? To
test this possibility, we sequenced small RNAs from <span class="phvro8t-x-x-120">Spo11 </span>mutant testes, which failed
to generate double-stranded DNA breaks at the leptotene stage and display a meiotic
arrest [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">Baudat et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">2000</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRomanienko2000">Romanienko and Camerini-Otero</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRomanienko2000">2000</a>]. The median
abundance of piRNAs from pre-pachytene genes did not decrease at 14.5 dpp. By
17.5 dpp, piRNA from pachytene genes decreased just 5.9-fold in the <span class="phvro8t-x-x-120">Spo11</span>
mutant testes compared to the heterozygotes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54002r10">3.10<!--tex4ht:ref: MolCel:fig:MolCelS5 --></a>). We note that
A-MYB protein abundance is reduced in the <span class="phvro8t-x-x-120">Spo11 </span>mutant [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>].
</p><!--l. 210--><p class="noindent"><span class="phvro8t-x-x-120">Trip13 </span>is required to complete the repair of double-strand DNA breaks on fully
synapsed chromosomes. <span class="phvro8t-x-x-120">Trip13 </span>mutants display a meiotic arrest similar to
that in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2007">Li and Schimenti</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2007">2007</a>]: pachytene arrest with
synapsed chromosomes. To further test whether the loss of pachytene piRNA
precursor transcripts in <span class="phvro8t-x-x-120">A-Myb </span>mutants reflects a general effect of meiotic
arrest, we measured piRNA precursor transcript abundance in <span class="phvro8t-x-x-120">Trip13 </span>mutant
testes at 17.5 dpp. Unlike <span class="phvro8t-x-x-120">A-Myb</span>, piRNA precursor transcripts were readily
detectable in the <span class="phvro8t-x-x-120">Trip13 </span>mutant (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-55002r12">3.12<!--tex4ht:ref: MolCel:fig:MolCelS6 --></a>). We conclude that the loss of
                                                                                

                                                                                
pachytene piRNA precursor transcripts and piRNAs in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes is a
direct consequence of the requirement for A-MYB to transcribe pachytene
piRNA genes and not a general feature of meiotic arrest at the pachytene
stage.
<a id="x1-54003r96"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.7 </span> <a id="x1-550007"></a><span class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</h4>
<!--l. 215--><p class="noindent">The <span class="phvro8t-x-x-120">A-Myb </span>mutant more strongly affected pachytene piRNA accumulation than it did
the steady-state abundance of the corresponding piRNA precursor transcripts (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">3.9<!--tex4ht:ref: MolCel:fig:MolCelF5 --></a>; the median decrease in pachytene piRNA abundance was 2-fold greater at 14.5
dpp and 38-fold greater at 17.5 dpp than the decrease in the steady-state abundance
of pachytene precursor transcripts (Table S1). These data suggest that A-MYB exerts
a layer of control on piRNA accumulation beyond its role in promoting pachytene
piRNA precursor transcription.
</p><!--l. 217--><p class="noindent"><span class="phvro8t-x-x-120">Miwi </span>has previously been proposed to be a direct target of A-MYB; <span class="phvro8t-x-x-120">Miwi </span>mRNA
abundance is reduced in A-MYB mutant testes, and ChIP microarray data
place A-MYB on the <span class="phvro8t-x-x-120">Miwi </span>promoter [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>]. Our RNA-seq
data confirm that accumulation of <span class="phvro8t-x-x-120">Miwi </span>mRNA requires A-MYB: <span class="phvro8t-x-x-120">Miwi </span>mRNA
decreased more than 50-fold in testes isolated from <span class="phvro8t-x-x-120">A-Myb </span>mutant mice at
                                                                                

                                                                                
14.5 dpp compared to their heterozygous littermates (Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>A and
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56002r14">3.14<!--tex4ht:ref: MolCel:fig:MolCelS7 --></a> and Table S3). Furthermore, our ChIP data confirm that A-MYB binds
the <span class="phvro8t-x-x-120">Miwi </span>promoter <span class="phvro8t-x-x-120">in vivo </span>(Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>B, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53002r8">3.8<!--tex4ht:ref: MolCel:fig:MolCelS4 --></a>B, and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56002r14">3.14<!--tex4ht:ref: MolCel:fig:MolCelS7 --></a>). Like pachytene
piRNAs, <span class="phvro8t-x-x-120">Miwi </span>transcripts first appear at 12.5 dpp (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>B), and MIWI
protein is first detected in testes at 14.5 dpp [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>]. Loss
of MIWI arrests spermatogenesis at the round spermatid stage [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and
Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>].
</p><!--l. 219--><p class="noindent">A previous study reported that piRNAs fail to accumulate to wild-type levels in <span class="phvro8t-x-x-120">Miwi</span>
mutant testes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">Grivna et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrivna2006">2006</a>]. However, our data suggest that the overall
change in piRNA abundance caused by loss of MIWI is quite small: RNA-seq
detected no change at 14.5 dpp (change in total piRNA abundance = 1.1; n = 2) and
only a modest decrease at 17.5 dpp (change in total piRNA abundance = 0.58; n = 1).
piRNAs from pachytene loci decreased just 2.7-fold at 14.5 dpp (p = 0.0046) and
3.5-fold at 17.5 dpp (p = 1.8 x 10-6) in <span class="phvro8t-x-x-120">Miwi </span>mutant testes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">3.9<!--tex4ht:ref: MolCel:fig:MolCelF5 --></a>). By
comparison, pachytene piRNAs declined 87-fold at 14.5 dpp and 9,400-fold at 17.5
dpp in the <span class="phvro8t-x-x-120">A-Myb </span>mutant.
</p><!--l. 221--><p class="noindent">Does the loss of MIWI affect piRNA precursor transcription? We measured transcript
abundance and piRNA expression in <span class="phvro8t-x-x-120">Miwi </span>null mutant testes at 14.5 and 17.5 dpp. In
<span class="phvro8t-x-x-120">Miwi</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">&#8725;</span><span class="cmsy-8">-</span> </sup> testes, pachytene piRNA precursor transcripts were present at levels
indistinguishable from <span class="phvro8t-x-x-120">Miwi </span>heterozygotes (median change = 1.0- to 1.4-fold; q = 1;
                                                                                

                                                                                
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-54001r9">3.9<!--tex4ht:ref: MolCel:fig:MolCelF5 --></a>). Thus, loss of MIWI does not explain loss of pachytene piRNA precursor
transcripts in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-55001r11"></a>
                                                                                

                                                                                
<!--l. 225--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis34x.png" alt="PIC" class="graphics" width="396.48125pt" height="345.29pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis34x.png" src="Figures/MolCel/MolCel2013_Fig6.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.11:  </span><span class="content">Transcript  and  piRNA  abundance  in  heterozygous  (Het)  and
homozygous  <span class="phvro8t-x-x-120">A-Myb  </span>(Mut)  point-mutant  testes  is  shown  for  four  illustrative
examples at  14.5  and  17.5  dpp.  Also  shown  is  the  abundance  of  piRNA
sequencing reads  that  map  to  the  exon-exon  junctions.  Gene  <span class="phvro8t-x-x-120">11-qE1-9443</span>
does not have an intron. Exons, blue boxes; splice junctions, gaps; the last exon
is compressed and not to scale. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-55002r12">3.12<!--tex4ht:ref: MolCel:fig:MolCelS6 --></a>. </span></div><!--tex4ht:label?: x1-55001r11 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-55002r12"></a>
                                                                                

                                                                                
<!--l. 234--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis35x.png" alt="PIC" class="graphics" width="342.27875pt" height="357.33499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis35x.png" src="Figures/MolCel/MolCel2013_FigS6.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.12:  </span><span class="content">Transcripts  were  detected  in  total  RNA  from  adult  testes  by
RT-PCR (using random primers) for five pachytene piRNA loci as well as <span class="phvro8t-x-x-120">Miwi</span>
and <span class="phvro8t-x-x-120">Actin</span>. Mut, mutant; Het or H, heterozygote; wt, wild type. </span></div><!--tex4ht:label?: x1-55002r12 -->
                                                                                

                                                                                
<!--l. 242--><p class="noindent">In addition to <span class="phvro8t-x-x-120">Miwi</span>, ChIP-seq detected A-MYB bound to the promoters of 12 other
RNA-silencing-pathway genes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>B and Table S3). Of these, the mRNA
abundance—measured by three biologically independent RNA-seq experiments—of
<span class="phvro8t-x-x-120">Ago2</span>, <span class="phvro8t-x-x-120">Ddx39 </span>(uap56 in flies), <span class="phvro8t-x-x-120">Mael</span>, <span class="phvro8t-x-x-120">Mili</span>, <span class="phvro8t-x-x-120">Mov10l1</span>, <span class="phvro8t-x-x-120">Tdrd9</span>, and <span class="phvro8t-x-x-120">Vasa </span>did not change
significantly at 14.5 dpp in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes compared to heterozygotes (q &gt;
0.05); except for <span class="phvro8t-x-x-120">Ago2</span>, all decreased significantly in the mutant at 17.5 dpp. In
contrast, the abundance of the mRNAs encoding Tudor domain proteins decreased
significantly in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes: <span class="phvro8t-x-x-120">Tdrd6 </span>(64-fold decrease; q = 3.1 x 10-5) and
<span class="phvro8t-x-x-120">Tdrd5 </span>(7.5-fold decrease; q = 1.0 x 10-5). <span class="phvro8t-x-x-120">Tdrd5 </span>is expressed in embryonic testes
then decreases around birth [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">Yabuta et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">2011</a>]. <span class="phvro8t-x-x-120">TDRD5 </span>protein reappears at
12 dpp, increasing throughout the pachynema [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSmith2004">Smith et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSmith2004">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">Yabuta
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">2011</a>]. Our data indicate that A-MYB activates <span class="phvro8t-x-x-120">Tdrd5 </span>transcription at the
onset of the pachytene stage of meiosis. Similarly, <span class="phvro8t-x-x-120">Tdrd6 </span>mRNA can be
detected at the middle pachytene, but not the zygotene stage, and peaks after
late pachytene; TDRD6 protein can be detected at 17 dpp and continues
to increase until 21 dpp [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">Vasileva et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">2009</a>]. The findings that TDRD5
and TDRD6 colocalize with MIWI in pachytene spermatocytes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHosokawa2007">Hosokawa
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHosokawa2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">Vasileva et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">Yabuta et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">2011</a>] and that TDRD6 binds
MIWI [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2009a">Chen et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2009a">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">Vagin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">Vasileva et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">2009</a>] suggest
a role for these Tudor domain proteins in pachytene piRNA production or
function. As in <span class="phvro8t-x-x-120">Miwi</span>-/- testes, spermatogenesis arrests at the round spermatid
                                                                                

                                                                                
stage in <span class="phvro8t-x-x-120">Tdrd5</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">&#8725;</span><span class="cmsy-8">-</span></sup> and <span class="phvro8t-x-x-120">Tdrd6</span><sup><span class="cmsy-8">-</span><span class="cmmi-8">&#8725;</span><span class="cmsy-8">-</span></sup> mutant testes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">Vasileva et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVasileva2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">Yabuta
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYabuta2011">2011</a>]. Loss of <span class="phvro8t-x-x-120">Tdrd6 </span>expression has little effect on piRNA levels (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50004r6">3.6<!--tex4ht:ref: MolCel:fig:MolCelS3 --></a>;
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">Vagin et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">2009</a>], perhaps because the functions of Tudor domain proteins
overlap.
</p><!--l. 244--><p class="noindent">Other genes encoding piRNA pathway proteins whose promoters are bound by
A-MYB and whose expression decreased significantly in <span class="phvro8t-x-x-120">A-Myb </span>mutant testes include
<span class="phvro8t-x-x-120">MitoPld </span>(<span class="phvro8t-x-x-120">Pld6</span>; 3.9-fold decrease; q = 0.0095) and <span class="phvro8t-x-x-120">Tdrd12 </span>(5.3-fold decrease; q
= 0.0046). <span class="phvro8t-x-x-120">MitoPld </span>encodes an endoribonuclease implicated in an early
step in piRNA biogenesis in mice and flies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaase2010">Haase et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHaase2010">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">Houwing
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHuang2011">Huang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHuang2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">Ipsaro et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIpsaro2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">Nishimasu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNishimasu2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPane2007">Pane
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPane2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011a">Watanabe et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWatanabe2011a">2011a</a>]. The function of Tdrd12 is not known, but its
fly homologs (Yb, Brother of Yb, and Sister of Yb) are all required for piRNA
production [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHandler2011">Handler et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHandler2011">2011</a>]. <span class="phvro8t-x-x-120">Tdrd1 </span>decreased 3.4-fold, but with q value = 0.015.
<span class="phvro8t-x-x-120">Tdrd1 </span>is first expressed in fetal prospermatogonia, then re-expressed in
pachytene spermatocytes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChuma2006a">Chuma and Hosokawa</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChuma2006a">2006</a>]. In Tdrd1 mutant testes,
spermatogenesis fails, with no spermatocytes progressing past the round spermatid
stage [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChuma2006a">Chuma and Hosokawa</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChuma2006a">2006</a>]. TDRD1 binds MILI and MIWI [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2009a">Chen
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2009a">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKojima2009">Kojima et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKojima2009">2009</a>] and colocalizes with TDRD5 and TDRD6 in the
chromatoid body [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHosokawa2007">Hosokawa et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHosokawa2007">2007</a>].
</p><!--l. 246--><p class="noindent">Together, these data support the idea that at the onset of the pachytene phase of
                                                                                

                                                                                
meiosis, A-MYB coordinately activates transcription of many genes encoding piRNA
pathway proteins.
<a id="x1-55003r99"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.8 </span> <a id="x1-560008"></a>A-MYB and the Pachytene piRNA Regulatory Circuitry</h4>
<!--l. 251--><p class="noindent">A number of genes encoding known and suspected piRNA pathway proteins are
bound and regulated by A-MYB (Figures <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>B and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56002r14">3.14<!--tex4ht:ref: MolCel:fig:MolCelS7 --></a>C). Our data support a
model in which A-MYB drives both the transcription of pachytene piRNA genes and
the mRNAs encoding genes required for piRNA production including <span class="phvro8t-x-x-120">Miwi</span>, <span class="phvro8t-x-x-120">MitoPld</span>,
and <span class="phvro8t-x-x-120">Tdrd9</span>. Regulation by A-MYB of both the sources of pachytene piRNAs
and the piRNA biogenesis machinery creates a coherent feedforward loop
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>C). Feedforward loops amplify initiating signals to increase target
gene expression. Furthermore, they function as switches that are sensitive to
sustained signals; they reject transient signals [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOsella2011">Osella et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOsella2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShen-Orr2002">Shen-Orr
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShen-Orr2002">2002</a>].
</p><!--l. 253--><p class="noindent">A-MYB also bound to the <span class="phvro8t-x-x-120">A-Myb </span>promoter (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>B), and <span class="phvro8t-x-x-120">A-Myb </span>transcripts
decreased 4.2-fold in testes from an <span class="phvro8t-x-x-120">A-Myb </span>point mutant (<span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span>; Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56001r13">3.13<!--tex4ht:ref: MolCel:fig:MolCelF7 --></a>B). The <span class="phvro8t-x-x-120">A-Myb </span>mutant fails to produce the high level of A-MYB protein
observed in wild-type testes at the late pachytene stage of meiosis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas
                                                                                

                                                                                
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>]. Instead, A-MYB protein never becomes more abundant than the level
achieved in wild-type testes by the beginning of the pachytene stage. While the
lower level of A-MYB in the <span class="phvro8t-x-x-120">A-Myb </span>mutant may reflect instability of the mutant
protein, a simpler explanation is that mutant A-MYB cannot activate <span class="phvro8t-x-x-120">A-Myb</span>
transcription.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-56001r13"></a>
                                                                                

                                                                                
<!--l. 257--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis36x.png" alt="PIC" class="graphics" width="415.55249pt" height="502.87874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis36x.png" src="Figures/MolCel/MolCel2013_Fig7.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.13: </span><span class="content">See subsubsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-570001">3.3.8.1<!--tex4ht:ref: MolCel:subsubsection:cap:Figure F7 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-56001r13 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-56002r14"></a>
                                                                                

                                                                                
<!--l. 267--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis37x.png" alt="PIC" class="graphics" width="336.25624pt" height="414.54874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis37x.png" src="Figures/MolCel/MolCel2013_FigS7.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.14: </span><span class="content">See subsubsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-580002">3.3.8.2<!--tex4ht:ref: MolCel:subsubsection:cap:Figure S7 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-56002r14 -->
                                                                                

                                                                                
<a id="x1-56003r92"></a>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.8.1 </span> <a id="x1-570001"></a>Caption for Figure 3.13</h5>
<!--l. 277--><p class="noindent">(A) mRNA abundance in <span class="phvro8t-x-x-120">A-Myb </span>mutant versus heterozygous testes. The 407 genes
with a significant (q &lt; 0.05) change in steady-state mRNA levels are shown as
red circles. The 203 with A-MYB peaks within 500 bp of their transcription
start site are filled. (B) A-MYB ChIP-seq signal at the transcription start sites
of <span class="phvro8t-x-x-120">A-Myb </span>and genes implicated in RNA silencing pathways. For each, the
figure reports the change in mRNA abundance between 17.5 and 10.5 dpp in
wild-type testes and the mean change between <span class="phvro8t-x-x-120">A-Myb </span>mutant and heterozygous
testes at 14.5 dpp (mean <span class="cmsy-10x-x-120">± </span>SD; n = 3). (C) A model for the regulation of
pachytene piRNA biogenesis by A-MYB. See also Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-56002r14">3.14<!--tex4ht:ref: MolCel:fig:MolCelS7 --></a> and Table
S3.
<a id="x1-57001r105"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.8.2 </span> <a id="x1-580002"></a>Caption for Figure 3.14</h5>
<!--l. 283--><p class="noindent">A) mRNA abundance in 17.5 dpp <span class="phvro8t-x-x-120">A-Myb </span>versus heterozygous testes. The 2,853
genes with a significant (q &lt; 0.051) change in steady-state mRNA abundance are
shown as open red circles. Among them, 8721,009 genes also had A-MYB peaks
within 500 bp of their transcription start sites. These “A-MYB targets” are marked
                                                                                

                                                                                
with filled red circles. (B) Same as (A) but in 14.5 dpp <span class="phvro8t-x-x-120">Miwi </span>mutant versus
heterozygous testes. The genes encoding proteins implicated in RNA silencing
pathways that were labeled in (A) and that showed no change in expression in
<span class="phvro8t-x-x-120">Miwi </span>mutant testes are highlighted as green filled circles. As expected, <span class="phvro8t-x-x-120">Miwi</span>,
showed a significant decrease in mRNA abundance in <span class="phvro8t-x-x-120">Miwi</span>-/- testes. (C) The
change in mRNA abundance (rpkm) in <span class="phvro8t-x-x-120">A-Myb </span>and <span class="phvro8t-x-x-120">Miwi </span>mutant testes versus
heterozygous controls for the RNA silencing genes highlighted in (A) and
(B).
<a id="x1-58001r102"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">3.3.9 </span> <a id="x1-590009"></a>Feed-Forward Regulation of piRNA Production is Evolutionarily Conserved</h4>
<!--l. 288--><p class="noindent">Is A-MYB-mediated, feedforward control a general feature of regulation of piRNA
production among vertebrates? To test whether A-MYB control of piRNA precursor
transcription is evolutionarily conserved, we used high-throughput sequencing to
identify piRNAs in adult rooster testes. Birds and mammals diverged 330 million years
ago [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBenton2007">Benton and Donoghue</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBenton2007">2007</a>]. After removing the sequences of identifiable
miRNAs [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBurnside2008">Burnside et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBurnside2008">2008</a>] and annotated noncoding RNAs, total small
RNA from the adult rooster testis showed peaks at both 23 and 25 nt (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>A). When the RNA was oxidized before being prepared for sequencing,
only a single 25 nt peak remained, consistent with the 25 nt small RNAs
                                                                                

                                                                                
corresponding to piRNAs containing 2<span class="cmsy-8">&#8242;</span>-O-methyl-modified 3<span class="cmsy-8">&#8242;</span>&nbsp;termini. These
longer, oxidation-resistant species typically began with uracil (62% of species
and 65% of reads; Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>B), and we detected a significant Ping-Pong
amplification signature (Z score = 31; Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>C). We conclude that the
oxidation-resistant, 24-30 nt long small RNAs correspond to rooster piRNAs. Like
piRNAs generally, rooster piRNAs are diverse, with 5,742,529 species present among
81,121,893 genome-mapping reads. Like mouse pachytene piRNAs, 70% of
piRNAs from adult rooster testes mapped to unannotated intergenic regions,
19% mapped to transposons, and 14% mapped to protein-coding genes. Of
the piRNAs that map to protein-coding genes, &gt;95% derive from introns.
Forty-two percent of piRNA species mapped uniquely to the Gallus gallus
genome.
</p><!--l. 290--><p class="noindent">Using 24-30 nt piRNAs from oxidized libraries, we identified 327 rooster piRNA
clusters (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59002r16">3.16<!--tex4ht:ref: MolCel:fig:MolCelS8 --></a>). These account for 76% of all uniquely mapping piRNAs. Of the
327 clusters, 25 overlapped with protein-coding genes. To begin to identify the
transcription start sites for the rooster piRNA clusters, we analyzed adult
rooster testes by H3K4me3 ChIP-seq. More than 81% (268 out of 327) of
the clusters contained a readily detectable H3K4me3 peak within 1 kbp of
the piRNA cluster. In contrast, the median distance from a cluster to the
nearest transcription start site of an annotated gene was 73 kbp, suggesting
that the H3K4me3 peaks reflect the start sites for rooster piRNA precursor
                                                                                

                                                                                
transcripts.
</p><!--l. 292--><p class="noindent">Next, we asked where in the genome A-MYB bound in adult rooster testes. A-MYB
ChIP-seq identified 5,509 significant peaks (FDR &lt; 10-25). MEME analysis of the top
500 peaks with the lowest FDR values identified a motif (E = 2.6 x 10-201; Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>D) similar to that found in the mouse (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-53001r7">3.7<!--tex4ht:ref: MolCel:fig:MolCelF4 --></a>A). A-MYB is the only one of the
three chicken MYB genes expressed in adult testis (X.Z.L. and P.D.Z., unpublished
data), supporting the view that these peaks correspond to A-MYB binding. The
core sequence motif associated with A-MYB binding in mouse differs at one
position (CAGTT) from that in rooster (C C/G GTT). This difference between
mammalian and chicken MYB proteins has been noted previously [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng1996">Deng
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng1996">1996</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeston1992">Weston</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWeston1992">1992</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-59001r15"></a>
                                                                                

                                                                                
<!--l. 297--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis38x.png" alt="PIC" class="graphics" width="401.5pt" height="526.96875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis38x.png" src="Figures/MolCel/MolCel2013_Fig8.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.15: </span><span class="content">See subsubsection <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-600001">3.3.9.1<!--tex4ht:ref: MolCel:subsubsection:cap:Figure F8 --></a> for full figure caption. </span></div><!--tex4ht:label?: x1-59001r15 -->
                                                                                

                                                                                
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-59002r16"></a>
                                                                                

                                                                                
<!--l. 306--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis39x.png" alt="PIC" class="graphics" width="407.52249pt" height="459.7175pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis39x.png" src="Figures/MolCel/MolCel2013_FigS8.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;3.16:  </span><span class="content">Black  horizontal  lines  denote  the  locations  on  the  Gallus
gallus (galGal3) chromosomes of the piRNA clusters identified by small RNA
sequencing. The figure shows 324 clusters; clusters on E64 (cluster 370) and
E22C19W28_E50C23 (clusters 109 and 563) are not shown. </span></div><!--tex4ht:label?: x1-59002r16 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 314--><p class="noindent">To determine whether chicken A-MYB might regulate transcription of some piRNA
clusters in the testis, we compared the A-MYB peak nearest to each piRNA cluster
with the nearest H3K4me3 peak. Of the 327 rooster piRNA clusters, at least 104
were occupied by A-MYB at their promoters, as defined by an overlapping
H3K4me3 peak. These 104 clusters account for 31% of uniquely mapping rooster
piRNAs.
</p><!--l. 316--><p class="noindent">The chicken genome encodes at least two PIWI proteins: PIWIL1 and PIWIL2.
Remarkably, the promoter of Gallus gallus PIWIL1, the homolog of mouse <span class="phvro8t-x-x-120">Miwi</span>,
contained a prominent A-MYB peak (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>E). TDRD1 and TDRD3 also
showed A-MYB peaks (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>E). Thus, as in mice, Gallus gallus A-MYB
controls the transcription of both piRNA clusters and genes encoding piRNA
pathway proteins. We conclude that A-MYB-mediated feedforward regulation of
piRNA production was likely present in the last common ancestor of birds and
mammals.
</p><!--l. 318--><p class="noindent">In mice, we found no piRNA-producing genes on the sex chromosomes (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-46002r2">3.2<!--tex4ht:ref: MolCel:fig:MolCelS1 --></a>A),
perhaps because mouse sex chromosomes are silenced during the pachytene stage
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009d">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009d">2009a</a>]. Birds use a ZW rather than an XY mechanism for sex
determination, so roosters are homogametic (ZZ), allowing the sex chromosomes to
remain transcriptionally active in males [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNamekawa2009">Namekawa and Lee</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNamekawa2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchoenmakers2009">Schoenmakers
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchoenmakers2009">2009</a>]. Indeed, we find that 39 of the 327 rooster piRNA clusters are on the
                                                                                

                                                                                
Z chromosome, accounting for 12% of uniquely mapping piRNAs (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59002r16">3.16<!--tex4ht:ref: MolCel:fig:MolCelS8 --></a>). Of the 39 Z chromosome clusters, 18 had an A-MYB peak at their
promoter.
<a id="x1-59003r106"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">3.3.9.1 </span> <a id="x1-600001"></a>Caption for figure 3.15</h5>
<!--l. 322--><p class="noindent">(A) Length distributions of total rooster testis small RNAs (black) and miRNAs
(gray).(B) Sequence logo showing the nucleotide composition of piRNA reads and
species.(C) The 5<span class="cmsy-8">&#8242;</span>&nbsp;-5<span class="cmsy-8">&#8242;</span>&nbsp;overlap between piRNAs from opposite strands was analyzed
to determine if rooster piRNAs display Ping-Pong amplification. The number of pairs
of piRNA reads at each position is reported. Z score indicates that a significant 10 nt
overlap (Ping-Pong) was detected. Z score &gt; 1.96 corresponds to p value &lt; 0.05.(D)
MEME-reported motif of the top 500 (by peak score) A-MYB ChIP-seq peaks from
adult rooster testes.(E) A-MYB, H3K4me3, and input ChIP-seq signals at the
transcription start sites of rooster PIWIL1, TDRD1, and TDRD3. See also Figure
S8.
<a id="x1-60001r79"></a>
</p>
                                                                                

                                                                                
<h3 class="sectionHead"><span class="titlemark">3.4 </span> <a id="x1-610004"></a>Discussion</h3>
<!--l. 327--><p class="noindent">The data presented here provide strong support for the view that piRNAs in mammals
begin as long, single-stranded precursors generated by testis-specific, RNA Pol II
transcription of individual piRNA genes (see also <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>].
Transcription by RNA Pol II affords piRNA genes the same rich set of transcriptional
controls available to regulate mRNA expression. Our data establish that
developmentally regulated transcription of piRNA genes determines when specific
classes of piRNAs emerge during spermatogenesis.
</p><!--l. 329--><p class="noindent">During mouse spermatogenesis, transcription of pachytene piRNA genes begins at
the onset of the pachytene stage of meiosis; pachytene piRNAs accumulate
subsequently. The presence of the MYB binding motif near the transcription start sites
of pachytene piRNA genes, the physical binding of A-MYB to those genes, and
the loss of pachytene piRNA precursor transcripts and piRNAs in testes
from <span class="phvro8t-x-x-120">A-Myb </span>mutant mice all argue that A-MYB regulates pachytene piRNA
production.
</p><!--l. 331--><p class="noindent">A-MYB also drives increased expression of piRNA pathway genes. Among these,
<span class="phvro8t-x-x-120">Miwi </span>expression shows the greatest dependence on A-MYB, but A-MYB also drives
transcription of genes encoding other proteins in the piRNA pathway, including
MitoPld, Mael, and five genes encoding Tudor domain proteins. For example, A-MYB
increases expression of Tdrd6 more than 500-fold. Loss of A-MYB function more
                                                                                

                                                                                
strongly depletes pachytene piRNAs than loss of MIWI, in part because
pachytene piRNAs can still be loaded into MILI in <span class="phvro8t-x-x-120">Miwi </span>mutant testes, although
MILI-loaded pachytene piRNAs do not suffice to produce functional sperm. In the
<span class="phvro8t-x-x-120">A-Myb </span>mutant, expression of mRNAs encoding multiple piRNA pathway
proteins decreases. We speculate that in wild-type male mice, the increased
expression of these mRNAs at the onset of the pachytene stage of meiosis
ensures that sufficient piRNA-precursor-processing and MIWI-loading factors
are available to cope with the large increase in pachytene piRNA precursor
transcription.
</p><!--l. 333--><p class="noindent">We propose that induction of A-MYB during the early pachytene stage of
spermatogenesis initiates a feedforward loop that ensures the precisely timed
production of these piRNAs. Coherent feedforward loops show delayed kinetics in
order to reject background stimuli [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMangan2003">Mangan and Alon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMangan2003">2003</a>]. Indeed, we observed a
delay from the early to middle pachytene in the accumulation of pachytene piRNAs,
despite the continued increase in <span class="phvro8t-x-x-120">A-Myb </span>expression (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-50001r3">3.3<!--tex4ht:ref: MolCel:fig:MolCelF2 --></a>A). Pachytene
piRNA levels increase 75-fold (median for the 100 genes) from 10.5 to 12.5
dpp, coincident with increased expression of <span class="phvro8t-x-x-120">A-Myb</span>. However, from 12.5
to 14.5 dpp, pachytene piRNAs increase only 1.2-fold. Pachytene piRNAs
subsequently resume their accumulation, increasing 65-fold from 14.5 to
17.5 dpp. We believe this delay is a consequence of a feedforward loop that
ensures the production of pachytene piRNAs only at the pachytene stage of
                                                                                

                                                                                
spermatogenesis. Regulation by a feedforward loop also predicts a rapid shutdown of
pachytene piRNA pathways at round spermatid stage VIII, when A-MYB
protein levels decrease [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">Horvath et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHorvath2009">2009</a>]. Supporting this idea, the
abundance of MIWI decreases sharply by the elongated spermatid stage of
spermatogenesis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>]. Testing this proposal is a clear challenge for
the future.
</p><!--l. 335--><p class="noindent">In fruit flies and zebrafish [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">Houwing et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHouwing2007">2007</a>], most
piRNAs map to repetitive regions, whereas in mammals, uniquely mapping intergenic
piRNAs predominate in the adult testis. The discovery that 70% of rooster piRNA
reads map to intergenic regions suggests that the expansion of intergenic piRNAs
controlled by A-MYB feedforward regulation arose before the divergence of birds and
mammals. In the future, detailed analysis of piRNA production across avian
spermatogenesis should provide insight into the evolutionary origins and
functions of pachytene piRNAs, a class of piRNAs thus far only detected in
mammals.
</p><!--l. 337--><p class="noindent">In summary, we have shown that mouse piRNA genes are coregulated transcriptionally,
establishing that A-MYB coordinately regulates the biogenesis of an entire piRNA
class, the pachytene piRNAs. The discovery that a loss-of-function <span class="phvro8t-x-x-120">A-Myb </span>mutant,
<span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span> , disrupts piRNA precursor transcription in vertebrates provides a tool to
understand the transformation of long, single-stranded piRNA precursors into
                                                                                

                                                                                
mature piRNAs and to explore the functions and targets of the pachytene
piRNAs.
<a id="x1-61001r111"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">3.5 </span> <a id="x1-620005"></a>Experimental Procedures</h3>
<dl class="description"><dt class="description">
<span class="phvb8t-x-x-120">Mice</span></dt><dd class="description">                                                                         <br class="newline"><span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span>,  <span class="phvro8t-x-x-120">Spo11</span><sup><span class="cmmi-8">tm</span><span class="cmr-8">1</span><span class="cmmi-8">Sky</span></sup>,  and  <span class="phvro8t-x-x-120">Piwil1</span><sup><span class="cmmi-8">tm</span><span class="cmr-8">1</span><span class="cmmi-8">Hf</span></sup>  mice  were  maintained  and
used according   to   the   guidelines   of   the   Institutional   Animal   Care
and Use   Committee   of   the   University   of   Massachusetts   Medical
School and genotyped as described [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">Baudat et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">2000</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>].
</dd><dt class="description">
<span class="phvb8t-x-x-120">Sequencing</span></dt><dd class="description">                                                                 <br class="newline">Small [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">Ghildiyal  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSeitz2008">Seitz  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSeitz2008">2008</a>]  and  long  RNA-seq
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012a">Zhang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012a">2012b</a>] and analysis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">2009b</a>] were as described.
Reads that did not map to mouse genome mm9 were mapped to piRNA
precursor  transcripts  to  obtain  splice  junction  mapping  small  RNAs.
Total small RNA libraries from different developmental stages and from
mutants were normalized to the sum of all miRNA hairpin mapping reads.
                                                                                

                                                                                
Oxidized samples were calibrated to the corresponding total small RNA
library via  the  abundance  of  shared,  uniquely  mapped  piRNA  species.
piRNA expression data were grouped with Cluster 3.0. Differential gene
expression  was  analyzed  with  DESeq  R  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnders2010a">Anders  and  Huber</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnders2010a">2010</a>];
ChIP-seq reads were aligned to the genome using Bowtie version 0.12.7
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">Langmead et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">2009</a>], and peaks were identified using MACS [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">Zhang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">2008</a>].
</dd><dt class="description">
<span class="phvb8t-x-x-120">Acknowledgments</span> </dt><dd class="description">                                                                                            <br class="newline">We thank K. Chase and K. Schimenti for help collecting tissues; C. Tipping
for help with mouse husbandry; P. Johnson and B. Keagle for providing
rooster testes; G. Farley for technical assistance; H. Lin for reagents; Xi
Chen, Xiaotu Ma, Oliver Rando, and Benjamin Carone for advice on ChIP;
and members of our laboratories for critical comments on the manuscript.
X.Z.L. was supported by the Lalor Foundation and the Jane Coffin Childs
Memorial Fund for Medical Research.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Accession Numbers</span> </dt><dd class="description">                                                                                         <br class="newline">The Gene   Expression   Omnibus   (GEO)   accession   number   for   the
RNA-seq,  ChIP-seq,  and  small  RNA  data  reported  in  this  paper  is
                                                                                

                                                                                
GSE44690.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Animals</span></dt><dd class="description">                                                                      <br class="newline">Mice were  maintained  and  used  according  to  the  guidelines  of  the
Institutional   Animal   Care   and   Use   Committee   of   the   University   of
Massachusetts  Medical  School.  C57BL/6J  (Jackson  Labs,  Bar  Harbor,
ME, USA;  stock  number  664);  <span class="phvro8t-x-x-120">Mybl1</span><span class="cmmi-8">repro</span><span class="cmr-8">9</span>   in  a  mixed  129X1/SvJ  x
C57BL/6J  background;  Spo11tm1Sky  in  a  C57BL/6J  background;  and
Piwil<sup><span class="cmr-8">1</span><span class="cmmi-8">tm</span><span class="cmr-8">1</span><span class="cmmi-8">Hf</span></sup>  in  a  mixed  129X1/SvJ  x  C57BL/6J  background  (“<span class="phvro8t-x-x-120">Miwi</span>”)
mice were  genotyped  as  described  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">Baudat  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBaudat2000c">2000</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">Bolcun-Filas
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBolcun-Filas2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">Deng and Lin</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDeng2002c">2002</a>]. Rooster testes from White Leghorn of
the Cornell Special C strain, about 15 months old, were used for small
RNA analysis; and testes from the Brown Leghorn strain, about one year
old, were used for ChIP analysis.
</dd><dt class="description">
<span class="phvb8t-x-x-120">RNA Sequencing</span> </dt><dd class="description">                                                                                              <br class="newline">Small RNA  libraries  were  constructed  and  sequenced  as  described
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">Ghildiyal et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGhildiyal2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSeitz2008">Seitz et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSeitz2008">2008</a>] except that 18-35 nt RNA was
isolated  and  2S  rRNA  depletion  omitted.  Sequencing  was  performed
using either  a  Genome  Analyzer  GAII  (36  or  76  nt  reads)  or  HiSeq
                                                                                

                                                                                
2000 (50  nt)  instrument  (Illumina,  San  Diego,  CA,  USA).  We  analyzed
published   small   RNA   libraries   from   purified   mouse   spermatogonia
(SRR069809), spermatocytes (SRR069810, GSE39652), or spermatids
(SRR069811;  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">Gan  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGan2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">Modzelewski  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XModzelewski2012">2012</a>];  from  <span class="phvro8t-x-x-120">Mili</span>
mutant or heterozygous testes at 10 dpp (SRX003089 and SRX003088;
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">Aravin and Hannon</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAravin2008a">2008</a>]; from Tdrd6 mutant or heterozygous testes
at 18 dpp  (SRX012165  and  SRX012166;  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">Vagin  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVagin2009">2009</a>];  and
MILI IP  samples  from  Tdrd9  mutant  or  heterozygous  testes  at  14
dpp (SRX015795,  SRX015796,  SRX015797,  and  SRX015798;  [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShoji2009">Shoji
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShoji2009">2009</a>].
<!--l. 361--><p class="noindent">Strand-specific RNA-seq libraries [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">Zhang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2012">2012a</a>] using Ribo-Zero
Gold (Epicentre  Biotechnologies,  Madison,  WI,  USA)  were  sequenced
using the paired-end protocol on a HiSeq 2000.
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd class="description">                                                                                         <br class="newline">Small RNA sequence analysis was as described [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">2009b</a>] using
mouse  genome  release  mm9  and  chicken  genome  release  galGal3.
Non-coding  RNA  annotations  comprised  data  from  ncRNAscan,  the
known tRNAs from UCSC, and 18S, 28S and 5.8S rRNAs. miRNA hairpin
and mature miRNA annotation was from miRBase Release 19. Mouse and
                                                                                

                                                                                
chicken transposons were annotated using Repeat Masker from UCSC.
Reads that  did  not  map  to  the  mouse  genome  (mm9)  were  mapped
to piRNA  precursor  transcripts  to  obtain  splice  junction-mapping  small
RNAs. Total small RNA libraries from different developmental stages and
from mutants were normalized to the sum of all miRNA hairpin-mapping
reads. Oxidized samples were calibrated to the corresponding total small
RNA library  via  the  abundance  of  shared,  uniquely  mapped  piRNA
species. Table S1 reports the statistics for high-throughput sequencing.
For oxidized  (i.e.,  piRNA-enriched)  samples,  uniquely  mapping  small
RNAs &gt;23 nt were mapped to each assembled piRNA precursor transcript
and reported as reads per kilobase pair per million reads mapped to the
genome (rpkm) using a pseudo count of 0.001.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd class="description">                                                                                         <br class="newline">RNA-seq reads were aligned to the genome (NCBI 37/mm9) using TopHat
2.0.4 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">2009</a>]. Reads were mapped uniquely using the “-g
1” switch.  We  assembled  the  mouse  testes  transcriptome  (see  below).
For genes with multiple isoforms, the transcript with the highest average
rpkm value among the three replicates of adult testes was selected for
further analysis. Fragments with both reads mapped within a transcript,
or to piRNA precursor transcripts, were counted using BEDTools [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XQuinlan2010">Quinlan
                                                                                

                                                                                
and Hall</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XQuinlan2010">2010</a>].  The  sum  of  the  reads  aligning  to  the  top  quartile  of
expressed transcripts per library was used to calibrate the samples. The
number of reads per transcript was normalized by length, divided by the
library-specific  calibration  factor,  and  reported  as  rpkm  with  a  pseudo
count of  0.001.  Table  S1  presents  the  statistics  for  the  RNA-seq  data.
Sequences mapping to five genes (Table S1) that overlapped with or were
embedded within a piRNA gene were excluded when calculating piRNA
precursor transcript abundance.
</dd><dt class="description">
<span class="phvb8t-x-x-120">PAS-seq Library Construction and Analysis</span> </dt><dd class="description">                                                 <br class="newline">PAS-seq  libraries  (Table  S1)  were  prepared  essentially  as  described
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShepard2011">Shepard  et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShepard2011">2011</a>]  and  sequenced  using  a  HiSeq  2000  (100  nt
read length).  We  first  removed  adaptors  and  performed  quality  control
using Flexbar  2.2  (http://sourceforge.net/projects/theflexibleadap)  with
the parameters  “-at  3  -ao  10  –min-readlength  30  –max-uncalled  70
–phred-pre-trim 10.” For reads beginning with GGG including (NGG, NNG
or GNG) and ending with three or more adenosines, we removed the first
three nucleotides and mapped the remaining sequence with and without
the tailing  adenosines  to  the  mouse  genome  using  TopHat  2.0.4.  We
retained only those reads that could be mapped to the genome without the
trailing adenosine  residues.  Genome-mapping  reads  containing  trailing
                                                                                

                                                                                
adenosines were regarded as potentially originating from internal priming
and thus  discarded.  The  3<span class="cmsy-8">&#8242;</span>&nbsp;end  of  the  mapped,  retained  read  was
reported as the site of cleavage and polyadenylation.
</dd><dt class="description">
<span class="phvb8t-x-x-120">CAGE Library Construction and Analysis</span> </dt><dd class="description">                                                     <br class="newline">CAGE (cap analysis of gene expression; Table S1) was as described [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYang2011">Yang
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYang2011">2011</a>] and sequenced using a HiSeq 2000 (100 nt reads). After removing
adaptor sequences and checking read quality using Flexbar 2.2 with the
parameters of  <!--l. 375-->
<div class="lstlisting" id="listing-1"><span class="label"><a id="x1-62001r1"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">at</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">ao</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">10</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">min</span><span class="ectt-1000">-</span><span class="ectt-1000">readlength</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">20</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">max</span><span class="ectt-1000">-</span><span class="ectt-1000">uncalled</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">70</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">phred</span><span class="ectt-1000">-</span><span class="ectt-1000">pre</span><span class="ectt-1000">-</span><span class="ectt-1000">trim</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">10</span>

</div>
<!--l. 378--><p class="noindent">, we retained only reads beginning with NG or GG (the last two nucleotides on
the 5<span class="cmsy-8">&#8242;</span> &nbsp;adaptor). We then removed the first two nucleotides and mapped the
sequences to the mouse genome using TopHat 2.0.4. All unique 5<span class="cmsy-8">&#8242;</span>&nbsp;ends of the
mapped positions were considered as CAGE-tag starting sites and grouped into
tag clusters using a distance-based method in which the maximal distance
between two neighboring tags was required to be &lt;20 bp. The peak
position of a tag cluster was then reported as the transcription start
site.
                                                                                

                                                                                
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">Transcriptome Assembly and Annotation</span> </dt><dd class="description">                                                     <br class="newline">De novo transcriptome assembly from three biological replicates of
strand-specific RNA-seq data from adult testes was performed using Trinity
(r2012-06-08) with default parameters [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">Grabherr et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrabherr2011">2011</a>]. The assembled
RNA sequences were aligned to the mouse genome (mm9) with BLAT
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKent2002">Kent</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKent2002">2002</a>], and the alignments with more than 95% of sequence length
mapped and fewer than 1% mismatches retained.
<!--l. 383--><p class="noindent">We extracted novel junctions from Trinity (i.e., reads with [0-9]+M[0-9]+N[0-9]+M
pattern in the CIGAR string of SAM output), and re-mapped all RNA-seq reads
to these junctions, rescuing 1,402,444 reads in three replicates. Rescued reads
were combined with TopHat alignments (supplied with “-max-multihits 100” to
assembly through repetitive regions) and used as input for reference-based
assembly.
</p><!--l. 385--><p class="noindent">We used Cufflinks v2.0.2 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">2010</a>] with parameters of:
</p><!--l. 387-->
<div class="lstlisting" id="listing-2"><span class="label"><a id="x1-62002r1"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">u</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">j</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">0.2</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">min</span><span class="ectt-1000">-</span><span class="ectt-1000">frags</span><span class="ectt-1000">-</span><span class="ectt-1000">per</span><span class="ectt-1000">-</span><span class="ectt-1000">transfrag</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">40</span>

</div>
<!--l. 390--><p class="noindent">to assemble transcripts. To join small transcript fragments caused by insufficient
                                                                                

                                                                                
read coverage or embedded repetitive elements, two different gap-joining
distance cutoffs were used for the assembly of genes (“overlap-radius 100”)
and piRNA loci (“–overlap-radius 250”). We used Cuffcompare v2.0.2
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2010">2010</a>] to annotate the 49,840 Cufflinks-assembled
transcripts using parameters optimized for genic conditions (“overlap-radius
100”).
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">piRNA Precursor Transcript Annotation</span> </dt><dd class="description">                                                        <br class="newline">We combined transcripts from the two Cufflink assemblies with those from the
Trinity assembly, producing 136,069 unique transcripts. Those transcripts with
100 ppm or 100 rpkm unique mapping piRNAs at any time point (10.5, 12.5,
14.5, 17.5, 20.5 dpp and adult oxidized small RNA from testis) were selected for
manual annotation.
<!--l. 395--><p class="noindent">To refine the termini of the piRNA-producing transcripts, we supplemented the
RNA-seq data with high-throughput sequencing of 5<span class="cmsy-8">&#8242;</span>&nbsp;ends of RNAs bearing
(5<span class="cmsy-8">&#8242;</span> &nbsp;)ppp(5<span class="cmsy-8">&#8242;</span>&nbsp;) cap structures (CAGE) and of the 3<span class="cmsy-8">&#8242;</span>&nbsp;ends of transcripts flanking
the poly(A) tail (PAS-seq). To provide independent confirmation of the 5<span class="cmsy-8">&#8242;</span>&nbsp;ends of
each piRNA-producing transcript, we used chromatin immunoprecipitation
(ChIP-seq) of RNA polymerase II (pol II) and histone H3 bearing trimethylated
lysine-4 (H3K4me3). Refinement of transcriptional starts required both a CAGE
                                                                                

                                                                                
and a H3K4me3 peak to support the 5<span class="cmsy-8">&#8242;</span>&nbsp;end of the transcript. When no
H3K4me3 peak corroborated alternative transcription start sites proposed by
the CAGE data, the alternative transcripts were merged with the fully
substantiated transcript.
</p></dd><dt class="description">
<span class="phvb8t-x-x-120">piRNA Gene Nomenclature</span> </dt><dd class="description">                                                                             <br class="newline">When piRNA-producing genes overlap an annotated protein coding gene, we
refer to them using the name of the overlapping gene preceded by “pi-”; when
they do not, their names refer to their genomic location followed by a number
indicating the piRNA abundance in ppm at 6 weeks post-partum. The last
digit of a piRNA gene name specifies the rank order of expression
among isoforms, determined by the highest abundance of transcripts
(rpkm) observed for that gene among the six developmental stages of
testis.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Grouping piRNA Precursor Transcripts</span> </dt><dd class="description">                                                         <br class="newline">For the most abundant transcript in each locus, the abundance (rpkm) of
piRNAs at each stage was expressed as a fraction of the maximum abundance
reached during the developmental time course. These data were then analyzed
by hierarchical clustering according to Euclidean distance and complete linkage
                                                                                

                                                                                
using Cluster 3.0. Clustering results were visualized using Java Tree View
1.1.3.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Analysis of Differential Gene Expression</span>  </dt><dd class="description">                                                     <br class="newline">We determined differential gene expression using DESeq R [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnders2010a">Anders and
Huber</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAnders2010a">2010</a>]. For each annotated mRNA, reads from each library were aligned
to the most abundant assembled transcript. Transcripts with q &lt; 0.05 were
considered to be differentially expressed. Table S3 lists the genes that were
differentially expressed in <span class="phvro8t-x-x-120">A-Myb </span>at 14.5 dpp. Three biologically independent
replicates were used for <span class="phvro8t-x-x-120">A-Myb</span>homozygotes and heterozygotes at 14.5 and at
17.5 dpp.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Motif Discovery</span> </dt><dd class="description">                                                                                                 <br class="newline">For divergently transcribed piRNA gene pairs, the promoter region was defined
as the region between the transcription start sites defined by CAGE peaks.
Sequence motifs in these putative promoter regions were detected ab initio
using MEME [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey2009">Bailey et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey1994">Bailey and Elkan</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBailey1994">1994</a>] in TCM mod (any
number of repetitions per sequence) and compared to existing JASPAR and
TRANSFAC libraries via TOMTOM [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGupta2007">Gupta et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGupta2007">2007</a>]. FIMO was used to
detect motif sites within the putative promoters (default p &lt; 10<sup><span class="cmsy-8">-</span><span class="cmr-8">4</span></sup>; [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrant2011">Grant
                                                                                

                                                                                
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGrant2011">2011</a>].
</dd><dt class="description">
<span class="phvb8t-x-x-120">Chromatin Immunoprecipitation (ChIP)</span> </dt><dd class="description">                                                         <br class="newline">ChIP was performed as described [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2008">Chen et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2008">2008</a>] except that testes were
macerated on ice and then fixed with 1.5% (w/v) formaldehyde for 20 min.
Samples were then further crushed using 20 strokes with a “B” pestle in a
Dounce homogenizer (Kimble-Chase, Vineland, NJ, USA). Chromatin was
sheared by sonication and immunoprecipitated using anti-A-MYB (HPA008791;
Sigma, St. Louis, MO, USA) or anti-H3K4me3 (ab8580; Abcam, Cambridge,
MA, USA) antibody; immunoglobulin G (IgG; Sigma, item 2729) served as a
control. ChIP-quantitative PCR (qPCR) was performed using the CFX96
Real-Time PCR Detection System with SsoFast EvaGreen Supermix (Bio-Rad,
Hercules, CA, USA). Data were analyzed using DART-PCR [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPeirson2003">Peirson</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPeirson2003">2003</a>].
Relative ChIP enrichment values were normalized to <span class="phvro8t-x-x-120">MyoD1</span>, a gene
not expressed in testes. Table S1 lists ChIP-qPCR primers. ChIP-seq
libraries for anti-A-MYB and control input DNA were prepared following the
Illumina ChIP-seq protocol and sequenced on a HiSeq 2000 (50 nt
reads).
</dd><dt class="description">
<span class="phvb8t-x-x-120">ChIP-seq Analysis</span> </dt><dd class="description">                                                                                            <br class="newline">ChIP-seq reads were aligned to the genome using Bowtie version 0.12.7
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">Langmead et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">2009</a>]. Reads were mapped uniquely using the “-M 1 –best
–strata” switches and one mismatch was allowed (-v 1). ChIP peaks were
identified using MACS version 1.4.1 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">Zhang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZhang2008">2008</a>] using default
arguments, input as control, and a cutoff p-value = 10<sup><span class="cmsy-8">-</span><span class="cmr-8">25</span></sup> was used. BEDTools
was used to assign peaks to the nearest 5<span class="cmsy-8">&#8242;</span>&nbsp;end of genes. Table S1 reports
sequencing statistics for ChIP-seq.
</dd><dt class="description">
<span class="phvb8t-x-x-120">RT-PCR</span></dt><dd class="description">                                                                      <br class="newline">Total RNA was treated with Turbo DNase (Ambion, Austin, TX, USA), and then
reverse transcribed using SuperScript III (Invitrogen, Eugene, OR, USA) with
random primers (Promega, Madison, WI, USA). The resulting cDNA was
analyzed by conventional PCR. Table S1 lists the primers used in Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-55002r12">3.12<!--tex4ht:ref: MolCel:fig:MolCelS6 --></a>.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Ping-Pong Analysis</span> </dt><dd class="description">                                                                                          <br class="newline">Ping-Pong amplification was analyzed by the 5<span class="cmsy-8">&#8242;</span>&nbsp;-5<span class="cmsy-8">&#8242;</span>&nbsp;overlap between piRNA
pairs from opposite genomic strands [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2009a">2009b</a>]. Overlap scores for each
overlapping pair were the product of the number of reads of each of the piRNAs
from opposite strands. The overall score for each overlap extend (1-30) was
                                                                                

                                                                                
the sum of all such products for all chromosomes. Heterogeneity at
the 3<span class="cmsy-8">&#8242;</span> &nbsp;ends of small RNAs was neglected. Z-score for 10 bp overlap
was calculated using the scores of overlaps from 1-9 and 11-30 as
background.
</dd><dt class="description">
<span class="phvb8t-x-x-120">Rooster piRNA Cluster Detection</span> </dt><dd class="description">                                                                   <br class="newline">We developed a dynamic programming algorithm to identify the genomic
regions with the highest piRNA density. We used oxidized small RNA reads
(&gt;23 nt) to detect clusters. We used the conservative assumption that piRNA
clusters compose at most 2% of the chicken genome. We first split the genome
into 1 kbp non-overlapping windows and computed piRNA abundance for each
window. The mean of the top 2% of windows was used as the penalty score for
the dynamic programming algorithm. The algorithm computes the cumulative
piRNA abundance score as a function of the window index along each
chromosome. The score at a window is the sum of the score in the previous
window and the piRNA abundance in the current window, minus the
penalty score; if the resulting score was negative it was reset to 0. The
maximal score points to the largest piRNA cluster. We extracted the
largest piRNA cluster, recomputed the scores at the corresponding
windows, and searched for the next cluster. The process continued
until the scores for all windows were zero. The boundaries of each
                                                                                

                                                                                
cluster were further refined by including those base pairs for which
piRNA abundance exceeded the mean piRNA abundance of the top
2% windows. We considered only those clusters with abundance &gt;10
ppm for uniquely mapping piRNAs. In Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-59001r15">3.15<!--tex4ht:ref: MolCel:fig:MolCelF8 --></a>E, gene models were
corrected using data from our unpublished adult rooster testis RNA-seq
data.
</dd></dl>
                                                                                

                                                                                
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&nbsp;4</span><br><a id="x1-630004"></a>SeqZip - Development and Applications</h2>
<a id="x1-63001r112"></a>
<h3 class="sectionHead"><span class="titlemark">4.1 </span> <a id="x1-640001"></a>Overview</h3>
<!--l. 9--><p class="noindent">Development of the SeqZip methodology began with an attempt to circumvent an
obvious shortcoming in second generation HTS—short read lengths. Until
second generation HTS (i.e. reads &lt;100nt on either the Illumina or SOLiD
platforms), most sequencing was done using cloned fragments, stored in a
bacteria, and analyzed using dideoxy “Sanger Sequencing” (see <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-70001">1.2.1<!--tex4ht:ref: Intro:subsec:DNA Sequencing History --></a>). Indeed,
this is how most <span class="phvb8t-x-x-120">??</span><a href="file://localhost/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20/endcsname%20/relax%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20::)Q/endcsname%20/relax%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20::)F/cw:x1-3001r%20/protect%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20::)Q/endcsname%20F-/endcsname%20/relax%20/cw:x1-3001r%20/protect%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20::)F/cw:x1-3001r%20/protect%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B(caption)%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BPackage%20caption%20Warning:%20/label%20without%20proper%20/caption%20on%20input%20line%206.See%20the%20caption%20package%20documentation%20for%20explanation.%7D/endgroup%20/edef%20hd:abrevs%7Bhd:abrevs%7D/protect%20/gdef%20/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7B/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20There%20were%20undefined%20references.%7D/endgroup%20%7D%7B/unhbox%20/voidb@x%20/hbox%20%7B/a:mbox%20%7B/protect%20/protect%20/edef%20T1%7BT1%7D/let%20/enc@update%20/relax%20/protect%20/edef%20phv%7Bphv%7D/protect%20/edef%20m%7Bm%7D/protect%20/edef%20n%7Bn%7D/protect%20/xdef%20/T1/phv/bx/n/12%20%7B/T1/phv/m/n/12%20%7D/begingroup%20/escapechar%20/m@ne%20/let%20/MT@subst@/T1/phv/bx/n/12%20/def%20/par%20%7D/b:mbox%20%7D%7D/protect%20/begingroup%20/def%20/MessageBreak%20%7B%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D/let%20/protect%20/immediate/write%20/@unused%20%7BLaTeX%20Warning:%20Reference%20%60hd:abrevs'%20on%20page%20275%20undefined%20on%20input%20line%209.%7D/endgroup%20/relax%20/hbox%20%7B%7D/let%20::)Q/endcsname%20F-/endcsname%20/par%20EST%3C/a%3Es%20where%20analyzed.%20An%20extremely%20powerful%20feature%20ofthese%20ESTs%20is%20that%20as%20they%20represented%20the%20sequence%20of%20a%20single%20clone,%20froma%20single%20original%20molecule%20of%20RNA.%20Connectivity%20between%20sequences%20thatwere%20far%20apart%20(%3E1,000%20nt)%20in%20the%20original%20sequence%20was%20maintained.%20It%20isthis%20very%20feature,%20the%20continuity%20of%20sequence,%20that%20allowed%20whole%20genomeshotgun%20sequencing%20to%20be%20used,%20and%20ESTs%20to%20be%20assembled%20into%20completegenomes,%20despite%20sometimes%20lengthly,%20repetitive,%20stretches%20of%20DNA%20(see%20section%3Ca%20href=" #x1-70001"="">1.2.1<!--tex4ht:ref: Intro:subsec:DNA Sequencing History --></a>). Once research transitioned over to heavy use of the second generation
HTS, all of that connectivity was lost, and all the inherent information with
it.
</p><!--l. 11--><p class="noindent">Second generation HTS can be supplemented with other technologies. This has been
demonstrated perhaps most successfully with long-read assisted genome assembly
                                                                                

                                                                                
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKoren2012a">Koren et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKoren2012a">2012</a>]. Why not supplement the disconnected nature of short reads
with another technology? To that end, Phillip D. Zamore proposed an RNA-templated
DNA-DNA ligation approach as drawn in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-64001r1">4.1<!--tex4ht:ref: SeqZipMethod:fig:Original SeqZip Diagram --></a> (see US Patent application
<a href="http://1.usa.gov/PTG9BB">12/906,678</a>). Using this approach, two or more distant sequences of RNA
are investigated using short DNA oligonucleotides to force the intervening
sequences to loop out. Incorporation of the hybridized DNA via ligation with those
of DNAs adjacently hybridized generates a positive read out of sequence
presence.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-64001r1"></a>
                                                                                

                                                                                
<!--l. 15--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis40x.png" alt="PIC" class="graphics" width="396.48125pt" height="197.73874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis40x.png" src="Figures/SeqZipMethod/OriginalSeqZipDiagram.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.1: </span><span class="content">Original SeqZip Diagram<br class="newline">This is the original concept diagram of the SeqZip methodology. (A) Specific
DNA oligos target an mRNA and loop out the RNA sequence. Ligases is added
to join the DNA oligos together; (B) &amp; (C) Two different possibilities of ligation
products templated from the RNA in (A), where Exon 2 is an cassette exon. </span></div><!--tex4ht:label?: x1-64001r1 -->
                                                                                

                                                                                
<!--l. 24--><p class="noindent">Along with Patent <a href="http://1.usa.gov/PTG9BB">12/906,678</a>, Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> presents much of the early and important
developmental work demonstrating reduction to practice of this method (termed
“SeqZip"), and its application to investigating connectivity of sequence content in the
biologically-interesting genes <span class="phvro8t-x-x-120">Fn1 </span>and <span class="phvro8t-x-x-120">Dscam1</span>.
</p><!--l. 26--><p class="noindent">Presented in this Chapter are experiments demonstrating SeqZip application to the
following questions and issues:
</p>
<ul class="itemize1">
<li class="itemize">Section  <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002">4.2<!--tex4ht:ref: SeqZipMethod:sec:Multiplex Gene Study --></a>:  Simultaneous  investigation  of  10  genes  ("Multiplex")  for
coordinated alternative splicing.
</li>
<li class="itemize">Section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-660003">4.3<!--tex4ht:ref: SeqZipMethod:sec:RNA Integrity via SeqZip --></a>: Investigate of RNA integrity using SeqZip.
</li>
<li class="itemize">Section  <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-740004">4.4.4<!--tex4ht:ref: SeqZipMethod:sec:piRNA precursor by SeqZip --></a>:  Demonstrating  the  presence  of  long,  continuous  piRNA
precusors by SeqZip</li></ul>
<!--l. 38--><p class="noindent">The three sections add to the data discussed in Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> in some important ways.
Section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002">4.2<!--tex4ht:ref: SeqZipMethod:sec:Multiplex Gene Study --></a> demonstrates that the SeqZip method can not only be used to investigate
an extremely complex alternatively spliced gene (<span class="phvro8t-x-x-120">Dscam1</span>) in a comprehensive
manner, but can also be applied to looking at multiple genes at once. Section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-660003">4.3<!--tex4ht:ref: SeqZipMethod:sec:RNA Integrity via SeqZip --></a>
exploits a subtle feature of the method—that the RNA must be intact in order to
produce a ligation product. This can be used to report on a fraction of intact RNA and
deduce meaningful information such as the amount of intact RNA virus (see, or the
existence of as-yet unobserved mega transcripts, like mammalian piRNA precursors
                                                                                

                                                                                
(section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-700004">4.4<!--tex4ht:ref: SeqZipMethod:sec:Demonstrating continuous precursor TX by SeqZip --></a>).
<a id="x1-64002r114"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">4.2 </span> <a id="x1-650002"></a>Multiplex SeqZip Application</h3>
<!--l. 43--><p class="noindent">Is the coordination discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-140004">1.3.4<!--tex4ht:ref: Intro:subsec:Coordination in splicing --></a> a general phenomenon? One of the
major goals of developing SeqZip was to investigate potential coordination
genome-wide. By genome-wide, what we really mean is to analyze many (or all) of
the RNA transcripts in a tissue for evidence of coordinated splicing decisions. When
development of the method reached the point that it could be applied to a multiplex
study, I did not posses the bioinformatic skills necessary to (1) identify target
transcripts, exons, and sequences to investigate for potential connectivity and (2)
design ligamers in an automated and high-throughput fashion. Both of these points
are discussed later (see <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-700004">4.4<!--tex4ht:ref: SeqZipMethod:sec:Demonstrating continuous precursor TX by SeqZip --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-760005">5<!--tex4ht:ref: Disc --></a>).
</p><!--l. 45--><p class="noindent">In order to make some progress on applying the technique to multiple genes at once,
I used data presented by <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>]. This paper identified genes
displaying tissue-specific splicing patterns, focusing on those with CNS-specific
patterns. One section focused on “Coordination between alternative splicing events
belonging to the same genes,” and seemed to be the exact type of genes we were
interested in applying the SeqZip method too. Five hundred of the 3,044
                                                                                

                                                                                
genes investigated by their microarrays contained 2–5 alternative exons.
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>] contained an additional data file listing all pair-wise
combinations of alternative exons in the same gene (with that gene having significant
expression in &gt;20 different tissues), along with the standard and partial spearman
correlations.
</p><!--l. 47--><p class="noindent">It is important to note that the genes above also contain alternative first exons, a
prominent type of alternative splicing (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-15001r7">1.7<!--tex4ht:ref: Intro:fig:asEventsBarChart --></a>). Indeed, from microarrays studies, it
has been estimated that approximately 16%–23% of all alternative splicing events
involve alternative first and last exons [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBingham2008">Bingham et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBingham2008">2008</a>]. It is known
that, through alternative use of first and last exons, cells can fine-tune a
transcript’s untranslated region (UTR) and control many aspects of mRNA
regulation including nuclear export, localization, expression, and stability
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHughes2006">Hughes</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHughes2006">2006</a>].
</p><!--l. 49--><p class="noindent">Using the <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>] data, I filtered exon pairs to those with a distance
&gt;350 nt between exons in the final pre-mRNA. I also visualized their transcript
architecture, and EST evidence using NCBIs AceView tool [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XThierry-Mieg2006">Thierry-Mieg and
Thierry-Mieg</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XThierry-Mieg2006">2006</a>]. For example, the exons with strong correlation of expression in
<span class="phvro8t-x-x-120">Chl1 </span>are in the beginning (second exon) and end (fourth from last exon, accession
BC060216) with plenty of supporting evidence for these exons being expressed and
skipped. After combing through [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>] data, I assembled a list of 11
                                                                                

                                                                                
genes (Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-65001r1">4.1<!--tex4ht:ref: table: BigSpanGenes --></a>).
</p>
<div class="table">
                                                                                

                                                                                
<!--l. 52--><p class="noindent"><a id="x1-65001r1"></a></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="caption"><span class="id">Table&nbsp;4.1: </span><span class="content">A list of 11 genes investigated in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002">4.2<!--tex4ht:ref: SeqZipMethod:sec:Multiplex Gene Study --></a>. Coordination between
exons first suggested by [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">Fagnani et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFagnani2007">2007</a>]. </span></div><!--tex4ht:label?: x1-65001r1 -->
<div class="tabular"><table id="TBL-3" class="tabular" cellspacing="0" cellpadding="0" rules="groups"><colgroup id="TBL-3-1g"><col id="TBL-3-1"></colgroup><colgroup id="TBL-3-2g"><col id="TBL-3-2"></colgroup><colgroup id="TBL-3-3g"><col id="TBL-3-3"></colgroup><colgroup id="TBL-3-4g"><col id="TBL-3-4"></colgroup><colgroup id="TBL-3-5g"><col id="TBL-3-5"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-3-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-1-1" class="td11"><span class="phvb8t-x-x-109">Gene name</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-1-2" class="td11"><span class="phvb8t-x-x-109">nt mRNA between</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-1-3" class="td11"><span class="phvb8t-x-x-109">possible isoforms</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-1-4" class="td11"><span class="phvb8t-x-x-109">Exon 1</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-1-5" class="td11"><span class="phvb8t-x-x-109">Exon 2</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-2-1" class="td11"><span class="phvr8t-x-x-109">Chl1             </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-2-2" class="td11">            <span class="phvr8t-x-x-109">4665</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-2-3" class="td11">              <span class="phvr8t-x-x-109">18</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-2-4" class="td11">     <span class="phvr8t-x-x-109">2</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-2-5" class="td11">    <span class="phvr8t-x-x-109">24</span></td></tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-3-1" class="td
11"><span class="phvr8t-x-x-109">Mdm1          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-3-2" class="td11">            <span class="phvr8t-x-x-109">1846</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-3-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-3-4" class="td11">  <span class="phvr8t-x-x-109">EDA</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-3-5" class="td11">  <span class="phvr8t-x-x-109">IIICS</span></td></tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-4-1" class="td
11"><span class="phvr8t-x-x-109">PTPRF-Y     </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-4-2" class="td11">            <span class="phvr8t-x-x-109">1633</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-4-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-4-4" class="td11">     <span class="phvr8t-x-x-109">2</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-4-5" class="td11">    <span class="phvr8t-x-x-109">13</span></td></tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-5-1" class="td
11"><span class="phvr8t-x-x-109">Cacna1c      </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-5-2" class="td11">            <span class="phvr8t-x-x-109">1403</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-5-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-5-4" class="td11">    <span class="phvr8t-x-x-109">15</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-5-5" class="td11">  <span class="phvr8t-x-x-109">21/22</span></td></tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-6-1" class="td
11"><span class="phvr8t-x-x-109">PTPRF-X     </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-6-2" class="td11">             <span class="phvr8t-x-x-109">936</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-6-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-6-4" class="td11">   <span class="phvr8t-x-x-109">9/10</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-6-5" class="td11">    <span class="phvr8t-x-x-109">21</span></td></tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-7-1" class="td
11"><span class="phvr8t-x-x-109">FN1              </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-7-2" class="td11">             <span class="phvr8t-x-x-109">813</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-7-3" class="td11">               <span class="phvr8t-x-x-109">8</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-7-4" class="td11">  <span class="phvr8t-x-x-109">13/14</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-7-5" class="td11">  <span class="phvr8t-x-x-109">21/22</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-8-1" class="td11"><span class="phvr8t-x-x-109">Apbb1          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-8-2" class="td11">             <span class="phvr8t-x-x-109">802</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-8-3" class="td11">             <span class="phvr8t-x-x-109">260</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-8-4" class="td11">   <span class="phvr8t-x-x-109">1/2b</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-8-5" class="td11">   <span class="phvr8t-x-x-109">2/3e</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-9-1" class="td11"><span class="phvr8t-x-x-109">Agrn             </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-9-2" class="td11">             <span class="phvr8t-x-x-109">736</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-9-3" class="td11">               <span class="phvr8t-x-x-109">8</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-9-4" class="td11"> <span class="phvr8t-x-x-109">33/34c</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-9-5" class="td11"> <span class="phvr8t-x-x-109">33/34a</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-10-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-10-1" class="td11"><span class="phvr8t-x-x-109">Exoc7          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-10-2" class="td11">             <span class="phvr8t-x-x-109">513</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-10-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-10-4" class="td11">     <span class="phvr8t-x-x-109">7</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-10-5" class="td11">    <span class="phvr8t-x-x-109">13</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-11-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-11-1" class="td11"><span class="phvr8t-x-x-109">Prom1          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-11-2" class="td11">             <span class="phvr8t-x-x-109">512</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-11-3" class="td11">               <span class="phvr8t-x-x-109">4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-11-4" class="td11">     <span class="phvr8t-x-x-109">7</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-11-5" class="td11">     <span class="phvr8t-x-x-109">9</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-12-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-12-1" class="td11"><span class="phvr8t-x-x-109">Lphn2          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-12-2" class="td11">             <span class="phvr8t-x-x-109">396</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-12-3" class="td11">              <span class="phvr8t-x-x-109">32</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-12-4" class="td11">    <span class="phvr8t-x-x-109">19</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-12-5" class="td11"> <span class="phvr8t-x-x-109">24/25a</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-13-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-13-1" class="td11"><span class="phvr8t-x-x-109">Lphn2          </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-13-2" class="td11">             <span class="phvr8t-x-x-109">396</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-13-3" class="td11">              <span class="phvr8t-x-x-109">32</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-13-4" class="td11">    <span class="phvr8t-x-x-109">19</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-3-13-5" class="td11"> <span class="phvr8t-x-x-109">24/25a</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-3-14-"><td style="white-space:nowrap; text-align:left;" id="TBL-3-14-1" class="td11">           </td></tr></tbody></table></div>
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
<!--l. 60--><p class="noindent">I hand-designed ligamers to observe potentially coordinated splicing decisions. These
oligos were ordered from IDT in a 96-well plate format, pooled according to gene, and
used to develop a multiplex approach to applying SeqZip, as well as investigate
coordination between these exons, in these genes, using mouse total RNA from
brains (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-400004">2.4<!--tex4ht:ref: SeqZipPaper:sec: Methods and Materials --></a> for methods).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-65002r2"></a>
                                                                                

                                                                                
<!--l. 64--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis41x.png" alt="PIC" class="graphics" width="344.28624pt" height="370.38374pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis41x.png" src="Figures/SeqZipMethod/10GeneSetSchematic.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.2: </span><span class="content">10 Gene Set study schematic<br class="newline">A) Two pools   of   ligamers   “Combination”   and   “Individual”   were   used
to investigate  splicing  of  cassette  exons  individually  or  while  maintaining
connectivity. Ligation products were sequenced on the Illumina platform using
a paired-end approach. B) A hypothetical example of showing how individual
splicing rates  can  confound  true  isoform  identity  when  observing  regions
independently. </span></div><!--tex4ht:label?: x1-65002r2 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 73--><p class="noindent">After attempts to perform SeqZip on all 10 genes in one ligation failed, I reverted back
to <span class="phvro8t-x-x-120">per-gene </span>ligation reactions in order to trouble shoot and optimize the assay. Once I
had obtained ligation products from per-gene ligation reactions for both the individual
and combination ligamers pools, I pooled all the ligation products and amplified them.
Amplified products were sent for paired-end 100 sequencing on the Illumina GEIIx
platform.
</p><!--l. 75--><p class="noindent">After considerable delay and optimization from the Umass Sequencing Core (due to
low library sequence diversity) the analyzed data demonstrated little alternative
splicing in the genes examined. Put another way—most of the transcripts observed
via SeqZip were uniform in exon inclusion, and showed little variation for cassette
exon inclusion (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-65003r3">4.3<!--tex4ht:ref: SeqZipMethod:fig:Apbb1 Results --></a>). These results forced us to rethink applying SeqZip to
multiple genes or complex alternative splicing (i.e. <span class="phvro8t-x-x-120">Dscam1</span>). For most genes, there
was too few reads aligning to combination products, arguing for more careful mixing
of the more efficient individual products with lower-efficiency combination products
prior to sequencing. For a discussion of an “ideal” multiplex study, see section
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-870001">5.3.1<!--tex4ht:ref: Disc:subsec: Ideal SeqZip exp. to look for Coordination --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-65003r3"></a>
                                                                                

                                                                                
<!--l. 79--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis42x.png" alt="PIC" class="graphics" width="393.46999pt" height="271.0125pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis42x.png" src="Figures/SeqZipMethod/Apbb1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.3: </span><span class="content">Measuring <span class="phvro8t-x-x-120">Apbb1 </span>via SeqZip in multiplex study<br class="newline">A) Region of  <span class="phvro8t-x-x-120">Apbb1  </span>investigated.  Two  alternative  promoters,  a  5<span class="cmsy-8">&#8242;</span>&nbsp;cassette
exon (“E:1/2”) and a 3<span class="cmsy-8">&#8242;</span>&nbsp;group of cassette exons (“Exon 2/3 {A–G}”). B) Number
of sequencing reads mapping to individual <span class="phvro8t-x-x-120">Apbb1 </span>ligation products (left) and
combination (right). C) RT-PCR of total RNA taken from a mouse brain looking
for cassette exon usage at each position shown in (A). Also shown in the size in
nt of the expected bands. D) Schematic of cDNAs cloned and sequencing from
mouse brain total RNA. </span></div><!--tex4ht:label?: x1-65003r3 -->
                                                                                

                                                                                
<a id="x1-65004r116"></a>
<h3 class="sectionHead"><span class="titlemark">4.3 </span> <a id="x1-660003"></a>RNA Integrity</h3>
<!--l. 91--><p class="noindent">An exciting use of SeqZip is rapid quantification of RNA integrity. Integrity defined as
the faction of molecules that are continuous and unbroken nucleic acid polymers,
from the original site of transcript to 3<span class="cmsy-8">&#8242;</span>&nbsp;processed end. Quantification of integrity has
many uses including: (1) quality control of RNA before downstream analysis such as
RT or sequencing, and (2) implications of infectivity for viruses that package and RNA
genome in the virion.
<a id="x1-66001r107"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.1 </span> <a id="x1-670001"></a>Demonstration of Concept</h4>
<!--l. 96--><p class="noindent">In order to demonstrate the feasibility of the SeqZip assay toward performing these
type of analysis, I <span class="phvro8t-x-x-120">in vitro </span>transcribed a 9,800 nt long RNA that I digested using ZnCl<sub><span class="cmr-8">2</span></sub>
at two different concentrations and times (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67001r4">4.4<!--tex4ht:ref: SeqZipMethod:fig:Ligation product and RNA integrity --></a>). The RNA was probed using
three ligamers, two to the very edges of the RNA and one that looped out the
intervening 8,000 nt. The amount of product observed should be directly tied to the
abundance of the full length template.
</p>
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-67001r4"></a>
                                                                                

                                                                                
<!--l. 100--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis43x.png" alt="PIC" class="graphics" width="288.07625pt" height="251.94124pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis43x.png" src="Figures/SeqZipMethod/DegreadedRNABySeqZip.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.4: </span><span class="content">Ligation product tied to RNA integrity<br class="newline">Top) Schematic demonstrating the experimental design to test ability of SeqZip
to report on transcript integrity. A middle ligamer (L), that hybridizes to the edges
of a 8,000 nt section of RNA should only ligate to flanking ligamers when the
template RNA is intact. <br class="newline">Middle) Intensity of PCR products amplified using end-labeled primers such that
the intensities of all bands can be quantitatively compared (i.e. semi-quantitative
PCR). Bottom)  A  denaturing  agarose  gel  stained  with  EtBr  showing  the
intactness of the template RNA used in position-matched ligation reactions in
the middle panel. </span></div><!--tex4ht:label?: x1-67001r4 -->
                                                                                

                                                                                
<!--l. 110--><p class="noindent">Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67001r4">4.4<!--tex4ht:ref: SeqZipMethod:fig:Ligation product and RNA integrity --></a> shows promising results toward the ability of SeqZip to report on RNA
integrity. The apparent intensity of the bands shown in (middle) was tied to the
amount of intact RNA seen in (bottom). However, the lane where the RNA was
degraded for two minutes with 10 mM ZnCl<sub><span class="cmr-8">2</span></sub> compared to 30 seconds with 100 mM
ZnCl<sub><span class="cmr-8">2</span> </sub> were not in good agreement, with clearly less intact RNA in the two minute
lane, but just as much ligation product. We hypothesized that this was due to inherent
secondary structure in the template we used (a section of the HIV genome, discussed
in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-660003">4.3<!--tex4ht:ref: SeqZipMethod:sec:RNA Integrity via SeqZip --></a>).
</p><!--l. 112--><p class="noindent">At what concentration of template do “long” ligamers generate ligation products from
template fragments? Using pools of ligamers targeting fragments and the complete
template (the same template used in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67001r4">4.4<!--tex4ht:ref: SeqZipMethod:fig:Ligation product and RNA integrity --></a>). SeqZip was performed using a 1:1
ratio of RNA fragments. Results (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-67002r5">4.5<!--tex4ht:ref: SeqZipMethod:fig: trans Tx for degradation --></a>) show that SeqZip accurately
reports on the presence of fragments, and not full length transcripts at <span class="cmsy-10x-x-120">&#8804;</span>1
nM template. This is in good agreement with results presented in Chapter
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-67002r5"></a>
                                                                                

                                                                                
<!--l. 116--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis44x.png" alt="PIC" class="graphics" width="363.3575pt" height="258.9675pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis44x.png" src="Figures/SeqZipMethod/TransRNAWithSeqZip.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.5: </span><span class="content"><span class="phvro8t-x-x-120">Trans</span>-transcript investigation<br class="newline">Left) Schematic of experimental design: Three pools of ligamers were used.
Two (labeled  “3”  and  “5”)  hybridize  to  the  5<span class="cmsy-8">&#8242;</span>&nbsp;and  3<span class="cmsy-8">&#8242;</span>&nbsp;sections  of  a  9,800  nt
template RNA. The last, labeled “L” connects these two regions via a long longer
with target 5<span class="cmsy-8">&#8242;</span> &nbsp;and 3<span class="cmsy-8">&#8242;</span>&nbsp;regions of complementarity. (Right;Top) Combinations of
the ligamer pools were used with different concentrations of template RNA in the
SeqZip assay. Ligation products were amplified with end-labeled PCR primers
and amplified  using  radioactive  PCR.  Shown  are  low  (left  column)  and  high
(right column) versions of two different exposure times (1x on bottom and 6x on
top). Right Bottom) Quantification of the bands shown in the gel above, grouped
by input template  RNA  concentration.  The  fold  difference  in  band  intensity
                                                                                

                                                                                
between the lowest signal “5” or “3” ligamer pool and the “L” pool is indicated.
Y-Axis is the raw band intensity. </span></div><!--tex4ht:label?: x1-67002r5 -->
                                                                                

                                                                                
<!--l. 125--><p class="noindent">These results are encouraging, but bear repeating in order to address the issues of
potential secondary structure and repetitive regions inherent to the template RNA
used. Put differently—they should be repeated with a template RNA of mRNA
original, instead of a highly-structured and repetitive template such as the HIV
genome.
<a id="x1-67003r121"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.2 </span> <a id="x1-680002"></a>HIV Genome Integrity</h4>
<!--l. 130--><p class="noindent">In late 2010–early 2011, a graduate student in the <a href="http://profiles.umassmed.edu/profiles/display/133484">Gottlinger</a> lab, Anna Kristina
Serquiña observed that a cell line expressing ATPase-defective forms of the SF1
helicase UPF1 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBhattacharya2000">Bhattacharya et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBhattacharya2000">2000</a>] did not infect reporter cell lines to the
same as control. Previous Mass-spec results had reported MOV10 (a SF1 family
helicase [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGregersen2014">Gregersen et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGregersen2014">2014</a>]) was packaged into extracellular viral particles.
Anna hypothesized that the decrease in infectivity was due to a problem with RT
when the genetic material is injected into target cells. The results of this study were
recently published [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSerquina2013">Serquiña et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSerquina2013">2013</a>].
</p><!--l. 132--><p class="noindent">Anna was interested in using the SeqZip methodology to quantify intact HIV virus in
virus-producing cells and extracellular virions. The first step in applying SeqZip to HIV
was to design ligamers.
                                                                                

                                                                                
<a id="x1-68001r124"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.3.3 </span> <a id="x1-690003"></a>HIV Ligamer Design</h4>
<!--l. 137--><p class="noindent">Research into the integrity of the HIV RNA genome using SeqZip began with
designing a set of ligamers against two different clones. The first, targeting transcripts
from the M19921 plasmid (so called “M” clone), and the section from the K03455
clone containing nearly identical sequence. We targeted a difference in sequence for
one site of ligation (<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-69001r6">4.6<!--tex4ht:ref: SeqZipMethod:fig:Hiv tx via SeqZip --></a>)A). Three different pools of ligamers were created: a Five(5)
ligamer pool, with three ligamers designed to test for the presence of sequence in the
first 1,140 nt of the HIV genome, importantly the first site of ligation in the 5
region pool should contain a mismatch in the K clone sequence; a three(3)
pool, testing the last 1,210 nt of the genome, and a Long (L) ligamer pool,
also containing three ligamers, but the middle ligamer of which spans the 5
and 3 regions, looping out 8,633 nt of sequence in the middle of the HIV
genome.
</p><!--l. 139--><p class="noindent"><span class="phvro8t-x-x-120">In vitro </span>transcripts were created using both the K and M clones. These transcripts
were added to a background of total MEF RNA, and SeqZip was performed. Ligation
products were successfully amplified from all ligamer pools when using the M clone
transcript and all three ligamer pools. Also the abundance of these ligation
products, as measured by endpoint PCR, seemed to be spike-concentration
                                                                                

                                                                                
dependent. Notably, Ligation products were not obtained from the K clone
using either the 5 or L ligamer pools, likely due to the mismatch between
the transcript and the ligamers at the site of ligation. Also of note was the
appearance of ligation products from purified endogenous virions of the M
clone from all three ligamer pools, and the absence of products from virions
purified from plasmids containing a defective protein, Gag, essential for viral
packaging.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-69001r6"></a>
                                                                                

                                                                                
<!--l. 143--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis45x.png" alt="PIC" class="graphics" width="392.46625pt" height="346.29375pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis45x.png" src="Figures/SeqZipMethod/HIVviaSeqZip.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.6: </span><span class="content">SeqZip can examine HIV transcript integrity<br class="newline">A) Schematic  demonstrating  the  experimental  design.  Three  different  pools
are used to probe for connectivity on the 5<span class="cmsy-8">&#8242;</span>&nbsp;(Five(5)) and 3<span class="cmsy-8">&#8242;</span>&nbsp;(Three(3)) ends.
Additionally, a Long (L) ligamer is used to check for connectivity between the
two ends. We used two different clones of the HIV genome, described in the text
and denoted as “M” and “K”. Important here is that the “K” contains difference
base at a ligation site of the 5 ligamer pool. B) A series of end-point PCR gels
                                                                                

                                                                                
showing amplified ligation products templated with <span class="phvro8t-x-x-120">in vitro </span>transcribed RNA at
10 nM or 10 pM of either the K or M clones, or from purified virions of (M clone
origin). Show are two different end points of PCR, 12 cycles (top) or 22 cycles
(bottom). Also shown is a legend of expected ligation products lengths </span></div><!--tex4ht:label?: x1-69001r6 -->
                                                                                

                                                                                
<!--l. 153--><p class="noindent">The results show (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-69001r6">4.6<!--tex4ht:ref: SeqZipMethod:fig:Hiv tx via SeqZip --></a>) that SeqZip and these three pools of ligamers can be
used to profile <span class="phvro8t-x-x-120">in vitro </span>HIV transcripts and RNA from purified virions. Important
features of the figure are: (1) Ligation products are <span class="phvro8t-x-x-120">not </span>observed for ligation reactions
using the K clone template RNA and the Five(5) pool of ligamers, verifying the
specificity of the ligamers to the different base of the M clone; and (2) That
the amount of product from reactions using the L pool of ligamers required
more cycles (22<span class="cmmi-12">vs.</span>12) in order to be visualized, as would be expected given
the physical constrain of hybridizing to two sequences separated by &gt;8,000
nt.
</p><!--l. 155--><p class="noindent">Again, while these results were encouraging, access to purified material and a
general push to publish Anna’s UPF1 story lead the Gottlinger lab to substantiate the
viral genome integrity claims effecting infectivity using a traditional northern blot
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSerquina2013">Serquiña et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSerquina2013">2013</a>]. However, these results warrant additional optimization and
application.
<a id="x1-69002r120"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">4.4 </span> <a id="x1-700004"></a>piRNA Precursors</h3>
<!--l. 160--><p class="noindent">The first genome-wide studies of piRNAs in <span class="phvro8t-x-x-120">Drosophila melanogaster </span>suggested their
production from a long, single-stranded RNA, as discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-210005">1.5<!--tex4ht:ref: Intro:sec:Nucleic Acid Polymers --></a> [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">Brennecke
                                                                                

                                                                                
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrennecke2007">2007</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">Gunawardane et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGunawardane2007">2007</a>]. Yet, demonstration of precursor transcripts
existing as continuous, long, RNA molecules had, as of 2010, yet to be demonstrated.
If it could be shown through experimentation that precursors existed as long RNAs, it
would provide valuable clues as to their biogenesis, included how such a long RNA is
packaged and transported around the cell. With these goals in mind, the following
section describes efforts to demonstrating the continuity of precursor transcripts using
SeqZip.
<a id="x1-70001r125"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.4.1 </span> <a id="x1-710001"></a>Mammalian piRNA Precursor Loci</h4>
<!--l. 165--><p class="noindent">Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> discuses 214 genomic loci that account for &gt;95% of all pachytene piRNAs.
Many of these loci are intergenic. That is they reside many thousands of base
pairs away from another protein-coding gene. Yet, many of these loci <span class="phvro8t-x-x-120">are</span>
traditional protein coding genes themselves, making investigation into their
eventual biogenesis to mature piRNAs more complicated. Finally, some loci
are generated from what appear to be bidirectional promoters. Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-71001r7">4.7<!--tex4ht:ref: SeqZipMethod:fig:precursor Loci Locations --></a>
shows the location of each of these types of precursor loci on each of the
19 autosomal chromosomes of the mouse. There were no loci identified
on the X and Y chromosomes, likely due to transcription silencing during
gametogenesis.
                                                                                

                                                                                
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-71001r7"></a>
                                                                                

                                                                                
<!--l. 169--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis46x.png" alt="PIC" class="graphics" width="378.41374pt" height="294.09874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis46x.png" src="Figures/SeqZipMethod/PrecursorLocations.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.7: </span><span class="content">piRNA precursor locations in mice<br class="newline">Shown are the  19  autosomal  and  2  allosomal  mouse  chromosomes.  They
are banded according to ideogram staining and oriented with the centromere
(dark black circle)  on  the  top.  Yellow  bars  indicate  the  location  of  classified
“pre-pachytene”  loci,  which  are  mostly  coincident  with  previously  annotated
mRNAs. Purple  bars  indicate  pachytene  loci,  and  are  usually  far  from  any
other annotated transcript. Finally green arrows, pointing in opposite directions,
represent those pachytene loci that are divergently transcribed from a single
promoter. </span></div><!--tex4ht:label?: x1-71001r7 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 178--><p class="noindent">The bidirectionally-transcribed sub-type of the pachytene loci are extremely
interesting, and useful. A motif search of the small sequence between the
annotated 5<span class="cmsy-8">&#8242;</span> &nbsp;TSSs of these transcripts allowed for identification of A-MYB as the
transcription factor that drove loci transcription (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-530005">3.3.5<!--tex4ht:ref: MolCel:subsec:A-Myb controls Pachytene precursor Tx --></a>). Also,
even as the 214 loci account for &gt;95% of the adult pachytene piRNAs, one
could consider just 5 of these promoters, including 4 that drive bidirectional
transcription, and account for &gt;50% of the pachytene piRNAs. Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-71002r2">4.2<!--tex4ht:ref: SeqZipMethod:tab:matchedClusterValues --></a>
describes these loci and transcripts, along with the cumulative number of piRNAs
accounted.
</p>
<div class="table">
                                                                                

                                                                                
<!--l. 181--><p class="noindent"><a id="x1-71002r2"></a></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="caption"><span class="id">Table&nbsp;4.2: </span><span class="content">Just 9 piRNA genes create &gt;50% of mammalian piRNAs</span></div><!--tex4ht:label?: x1-71002r2 -->
<div class="tabular"><table id="TBL-4" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-4-1g"><col id="TBL-4-1"><col id="TBL-4-2"><col id="TBL-4-3"><col id="TBL-4-4"><col id="TBL-4-5"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-4-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-1-1" class="td11"><span class="phvb8t-x-x-109">Cluster Name     </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-1-2" class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-5" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-5-1g"><col id="TBL-5-1"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-5-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-5-1-1" class="td00"><span class="phvb8t-x-x-109">Matched</span></td></tr><tr style="vertical-align:baseline;" id="TBL-5-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-5-2-1" class="td00"> <span class="phvb8t-x-x-109">Cluster</span></td></tr></tbody></table>                                                                                                                      </div></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-1-3" class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-6" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-6-1g"><col id="TBL-6-1"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-6-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-6-1-1" class="td00"><span class="phvb8t-x-x-109">Unique-mapping</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-6-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-6-2-1" class="td00">    <span class="phvb8t-x-x-109">wt.14dpp       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-6-3-"><td style="white-space:nowrap; text-align:center;" id="TBL-6-3-1" class="td00">    <span class="phvb8t-x-x-109">piRNAs        </span></td></tr></tbody></table>                                                                                                        </div></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-1-4" class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-7" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-7-1g"><col id="TBL-7-1"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-7-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-7-1-1" class="td00"><span class="phvb8t-x-x-109">Fraction of</span></td></tr><tr style="vertical-align:baseline;" id="TBL-7-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-7-2-1" class="td00"> <span class="phvb8t-x-x-109">pachytene</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-7-3-"><td style="white-space:nowrap; text-align:center;" id="TBL-7-3-1" class="td00">  <span class="phvb8t-x-x-109">piRNAs   </span></td></tr></tbody></table>                                                                                                                  </div></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-1-5" class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-8" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-8-1g"><col id="TBL-8-1"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-8-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-8-1-1" class="td00"><span class="phvb8t-x-x-109">Cumulative</span></td></tr><tr style="vertical-align:baseline;" id="TBL-8-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-8-2-1" class="td00"> <span class="phvb8t-x-x-109">pachytene</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-8-3-"><td style="white-space:nowrap; text-align:center;" id="TBL-8-3-1" class="td00">  <span class="phvb8t-x-x-109">piRNAs    </span></td></tr></tbody></table>                                                                                                                 </div></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-4-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-2-1" class="td11"><span class="phvr8t-x-x-109">17-qA3.3-26735.1</span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-2-2" class="td11"><span class="phvr8t-x-x-109">17-qA3.3-27363                                                                                                               </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-2-3" class="td11">                                                                 <span class="phvr8t-x-x-109">3,021,022</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-2-4" class="td11">                                                                      <span class="phvr8t-x-x-109">17.2</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-2-5" class="td11">                                                                      <span class="phvr8t-x-x-109">17.2</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-3-1" class="td11"><span class="phvr8t-x-x-109">17-qA3.3-27363.1</span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-3-2" class="td11"><span class="phvr8t-x-x-109">17-qA3.3-26735                                                                                                               </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-3-3" class="td11">                                                                 <span class="phvr8t-x-x-109">1,742,695</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-3-4" class="td11">                                                                       <span class="phvr8t-x-x-109">9.9</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-3-5" class="td11">                                                                      <span class="phvr8t-x-x-109">27.2</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-4-1" class="td11"><span class="phvr8t-x-x-109">9-qC-31469.1       </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-4-2" class="td11"><span class="phvr8t-x-x-109">9-qC-10667                                                                                                                      </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-4-3" class="td11">                                                                 <span class="phvr8t-x-x-109">1,006,333</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-4-4" class="td11">                                                                       <span class="phvr8t-x-x-109">5.7</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-4-5" class="td11">                                                                      <span class="phvr8t-x-x-109">32.9</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-5-1" class="td11"><span class="phvr8t-x-x-109">9-qC-10667.1       </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-5-2" class="td11"><span class="phvr8t-x-x-109">9-qC-31469                                                                                                                      </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-5-3" class="td11">                                                                   <span class="phvr8t-x-x-109">272,385</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-5-4" class="td11">                                                                       <span class="phvr8t-x-x-109">1.6</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-5-5" class="td11">                                                                      <span class="phvr8t-x-x-109">34.5</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-6-1" class="td11"><span class="phvr8t-x-x-109">7-qD2-24830.1     </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-6-2" class="td11"><span class="phvr8t-x-x-109">7-qD2-11976                                                                                                                    </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-6-3" class="td11">                                                                   <span class="phvr8t-x-x-109">652,564</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-6-4" class="td11">                                                                       <span class="phvr8t-x-x-109">3.7</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-6-5" class="td11">                                                                      <span class="phvr8t-x-x-109">38.2</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-7-1" class="td11"><span class="phvr8t-x-x-109">7-qD2-11976.1     </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-7-2" class="td11"><span class="phvr8t-x-x-109">7-qD2-24830                                                                                                                    </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-7-3" class="td11">                                                                   <span class="phvr8t-x-x-109">280,312</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-7-4" class="td11">                                                                       <span class="phvr8t-x-x-109">1.6</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-7-5" class="td11">                                                                      <span class="phvr8t-x-x-109">39.8</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-8-1" class="td11"><span class="phvr8t-x-x-109">6-qF3-28913.1     </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-8-2" class="td11"><span class="phvr8t-x-x-109">6-qF3-8009                                                                                                                       </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-8-3" class="td11">                                                                   <span class="phvr8t-x-x-109">564,930</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-8-4" class="td11">                                                                       <span class="phvr8t-x-x-109">3.2</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-8-5" class="td11">                                                                      <span class="phvr8t-x-x-109">43.0</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-9-1" class="td11"><span class="phvr8t-x-x-109">6-qF3-8009.1       </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-9-2" class="td11"><span class="phvr8t-x-x-109">6-qF3-28913                                                                                                                     </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-9-3" class="td11">                                                                   <span class="phvr8t-x-x-109">180,210</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-9-4" class="td11">                                                                       <span class="phvr8t-x-x-109">1.0</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-9-5" class="td11">                                                                      <span class="phvr8t-x-x-109">44.0</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-4-10-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-10-1" class="td11"><span class="phvr8t-x-x-109">2-qE1-35981.1     </span></td><td style="white-space:nowrap; text-align:left;" id="TBL-4-10-2" class="td11"><span class="phvr8t-x-x-109">NA                                                                                                                                    </span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-10-3" class="td11">                                                                  <span class="phvr8t-x-x-109">1121042</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-10-4" class="td11">                                                                       <span class="phvr8t-x-x-109">6.4</span></td><td style="white-space:nowrap; text-align:right;" id="TBL-4-10-5" class="td11">                                                                      <span class="phvr8t-x-x-109">50.4</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-4-11-"><td style="white-space:nowrap; text-align:left;" id="TBL-4-11-1" class="td11">               </td></tr></tbody></table>
</div>
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
<!--l. 187--><p class="noindent">Currently, the Zamore lab is designing sequence-specific DNA modifications (via
TALENs and CRISPRs) to remove these promoters from the mouse genome.
Once strains are created with these promoters removed, it is hoped that the
phenotypes displayed will provide clues to the function of pachytene piRNAs in
mice.
<a id="x1-71003r128"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.4.2 </span> <a id="x1-720002"></a>Pachytene Precursors are Unique Pol II Transcripts</h4>
<!--l. 192--><p class="noindent">Though mammalian piRNA precursor transcription is driven by Pol II, transcripts
themselves have a unique architecture. They tend be very long (some are &gt;100 kb).
While not especially long compared to some annotated mRNAs, what is unique is that
many are not interrupted by introns for tens of thousands of nucleotides. Given the
coupling between splicing and transcription (discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-140004">1.3.4<!--tex4ht:ref: Intro:subsec:Coordination in splicing --></a>) it is
strange to see so much transcribed RNA, surely containing cryptic splice
sites, be largely skipped by the spliceosome. Perhaps more confusing is that
pre-pachytene precursors <span class="phvro8t-x-x-120">do have </span>traditional mRNA-like design and introns
typical of Pol II transcripts. Yet, both types of transcripts are processed into
                                                                                

                                                                                
piRNAs. How does the cell partition these transcripts (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-820001">5.2.1<!--tex4ht:ref: Disc:subsec:How are precursors generated --></a>)?
Also refer to Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72001r8">4.8<!--tex4ht:ref: SeqZipMethod:fig:piRNA precusor Tx features --></a> for comparisons between “genic” (i.e. prepachytene)
and “intergenic” (i.e. pachytene) precursor transcripts (see Appendix <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-106001r2">A.2<!--tex4ht:ref: Appendix:tab:GenicAndInterGenicLoci --></a>)
and two other classes of Pol II transcripts, mRNAs and non-coding RNAs
(ncRNA).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-72001r8"></a>
                                                                                

                                                                                
<!--l. 196--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis47x.png" alt="PIC" class="graphics" width="415.55249pt" height="221.82874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis47x.png" src="Figures/SeqZipMethod/piRNAPrecusorTXFeatures.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.8: </span><span class="content">Some general features of piRNA transcripts<br class="newline">Top, left) Comparison   between   genic   piRNA   precursor   transcripts   (i.e.
pre-pachytene), mRNAs, Intergenic precursor transcripts (i.e. pachytene), and
non-coding RNAs (ncRNA) for overall length in nucleotides. Top, middle) Intron
length. Top, right) Number of introns. Bottom) Same as above, but considering
fully processed (i.e. “spliced”) versions of the transcripts. </span></div><!--tex4ht:label?: x1-72001r8 -->
                                                                                

                                                                                
<!--l. 205--><p class="noindent">An initial goal of characterizing piRNA precursor transcripts was to demonstrate their
existence as continuous RNA polymers in total RNA obtained from mouse testes.
Given the tremendous length of these transcripts (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72001r8">4.8<!--tex4ht:ref: SeqZipMethod:fig:piRNA precusor Tx features --></a>), the go-to experimental
approach one would use to demonstrate continuity would be gene-specific RT-PCR. A
DNA oligo was designed to hybridize near the 3<span class="cmsy-8">&#8242;</span>&nbsp;end of the loci <span class="phvro8t-x-x-120">17-qA3.3-27363.1</span>
(<span class="phvro8t-x-x-120">aka </span>“M1”), the longest and most studied of the mice piRNA-generating loci. RT would
be primed using this oligo, generating cDNAs that would extend (1) until the
5<span class="cmsy-8">&#8242;</span> &nbsp;end of the transcript was reached or (2) RT fell off the template. Following
cDNA generation, pairs of DNA primers hybridizing to 5<span class="cmsy-8">&#8242;</span>&nbsp;sections of the
proposed transcript were used in PCR reactions. Boundaries of the proposed
transcript were determined using a combination of small RNA sequencing and
poly(A)+-unstranded RNA-Seq. A schematic of the approach is shown in Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72002r9">4.9<!--tex4ht:ref: SeqZipMethod:fig:RT doesn't work for precursors --></a>A.
</p><!--l. 207--><p class="noindent">One expected issue when performing RT on such a long transcript expressed at low
levels is the <span class="phvro8t-x-x-120">lack </span>of dependence on the RT primer. This is illustrated in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72002r9">4.9<!--tex4ht:ref: SeqZipMethod:fig:RT doesn't work for precursors --></a>B,
where in the “+RT; -Primer” lanes there is still a clear signal for all 7 primer
pairs. The signal is virtually gone when leaving out RT, suggesting that an
RNA template is the source of the signal. It is believed that extremely short
DNA species (as short as 4 nt) are priming cDNA at some very low rate in
the “-Primer” reactions. This complication removes RT-PCR as a suitable
experimental approach to demonstrate the continuity of piRNA precursor
                                                                                

                                                                                
transcripts.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-72002r9"></a>
                                                                                

                                                                                
<!--l. 211--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis48x.png" alt="PIC" class="graphics" width="341.275pt" height="346.29375pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis48x.png" src="Figures/SeqZipMethod/RTDoesntWork.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.9: </span><span class="content">RT Doesn’t Work for piRNA precursors<br class="newline">A) Experimental   design   of   RT-PCR   demonstration   of   piRNA   precursor
<span class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>continuity. Shown in black, numbered 1-7 are primer pairs
amplified by PCR, after cDNA generation using the red “RT primer”. Also shown
is the length of the locus, the small RNA signal in green, and the RNA-Seq
signal in black. The locus is shown 5<span class="cmsy-8">&#8242;</span>&nbsp;(left) to 3<span class="cmsy-8">&#8242;</span>&nbsp;(right). B) Results from RT-PCR
using the 7 primer pairs shown in A, and various combinations of <span class="cmsy-10x-x-120">± </span>RT-primer
and RT-PCR enzyme. </span></div><!--tex4ht:label?: x1-72002r9 -->
                                                                                

                                                                                
                                                                                

                                                                                
<a id="x1-72003r131"></a>
<h4 class="subsectionHead"><span class="titlemark">4.4.3 </span> <a id="x1-730003"></a>Connectivity of Distance Intramolecular Sequences</h4>
<!--l. 223--><p class="noindent">Before applying SeqZip to these extremely difficult transcripts, we designed a set of
oligos to demonstrate the continuity of a traditional mRNA. The mRNA picked, <span class="phvro8t-x-x-120">Dst1</span>,
was (1) of sufficient length (&gt;23 kb as a fully processed mRNA) and (2) expressed in
mouse testes. Ligamers were designed to loop out ~5kb sections spaced evenly
along the length of the transcript. A ligamer was designed to loop out 22 kb of the
message, from 5<span class="cmsy-8">&#8242;</span>&nbsp;to 3<span class="cmsy-8">&#8242;</span>&nbsp;end. An illustration of the experimental design is shown in
Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-73001r10">4.10<!--tex4ht:ref: SeqZipMethod:fig:SeqZip on dst1 --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-73001r10"></a>
                                                                                

                                                                                
<!--l. 227--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis49x.png" alt="PIC" class="graphics" width="291.0875pt" height="160.59999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis49x.png" src="Figures/SeqZipMethod/dst1.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.10: </span><span class="content">SeqZip on a very long mRNA (<span class="phvro8t-x-x-120">Dst1</span>)<br class="newline">A) A model of the <span class="phvro8t-x-x-120">Dst1 </span>gene. Arrows show direction of transcription (5<span class="cmsy-8">&#8242;</span>&nbsp;to 3<span class="cmsy-8">&#8242;</span>&nbsp;).
Exons are tall lines, intronic regions join the exons. A scale bar is shown for size
in kb. B) A schematic showing how SeqZip was used to investigate 5 different
regions of <span class="phvro8t-x-x-120">Dst1 </span>transcripts (called A-E). Indicated are the nt of each loop in kb.
C) End-point PCR of SeqZip ligation products from each of the ligamer sets
shown in (B). </span></div><!--tex4ht:label?: x1-73001r10 -->
                                                                                

                                                                                
<!--l. 236--><p class="noindent">As seen in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-73001r10">4.10<!--tex4ht:ref: SeqZipMethod:fig:SeqZip on dst1 --></a>C, ligation products were obtained from every ligamer
combination, including the critical set (“AE”) where &gt;22 kb of the message was
looped out. In the control experiment, no ligation products were observed. This
experiment represents the longest successful “looping” in a SeqZip experiment
targeting an endogenously expressed RNA.
</p><!--l. 238--><p class="noindent">An additional demonstration of SeqZip’s application to profile long RNAs at multiple
sites are experiments involving <span class="phvro8t-x-x-120">Fn1</span>. As described previously (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-290001">2.2.1<!--tex4ht:ref: SeqZipPaper:subsec: Method Development and Validation --></a> <span class="phvro8t-x-x-120">Fn1</span>
contains three main sites of alternative splicing: EDB, EDA, and V-region. Using the
proper mix of ligamers, SeqZip examines and maintains connectivity at all three of
these sites, correctly reporting on their usage in the RNA template (Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-73002r11">4.11<!--tex4ht:ref: SeqZipMethod:fig:Three Site FN1 by SeqZip --></a>).
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-73002r11"></a>
                                                                                

                                                                                
<!--l. 242--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis50x.png" alt="PIC" class="graphics" width="372.39124pt" height="371.3875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis50x.png" src="Figures/SeqZipMethod/fn1ThreeSite.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.11: </span><span class="content">Three sites of alternative splicing in <span class="phvro8t-x-x-120">Fn1 </span>by SeqZip<br class="newline">A) Graphical representation of the 12 possible isoforms from mouse <span class="phvro8t-x-x-120">Fn1</span>. B)
Radioactive PCR gel showing amplified ligation products templated with specific
loops of ligamers. Pools are specified by top row: B = EDB exon only; A = EDA
exon only; V = V-Region only; B:A = EDB and EDA exon combinations; B:V
= EDB and V-Region combinations; A:V = EDA and V-Region combinations;
B:A:V = All two combinations, as shown in panel (A). Marked in nt is shown on
                                                                                

                                                                                
left, expected size of specific ligation ligation products indicated in white letters
on the gel, or black on right side. Where identity is not obvious from size, identity
of isoform provided. C) Quantification of bands from panel (B). Black bars =
observed signal of indicated band, Grey = product of individual frequencies. Top
only describes A:V combinations, lower shows all combinations. </span></div><!--tex4ht:label?: x1-73002r11 -->
                                                                                

                                                                                
<!--l. 251--><p class="noindent">With these results in hand, we felt confident that SeqZip could be used to analyze
piRNA precursor transcripts.
<a id="x1-73003r134"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">4.4.4 </span> <a id="x1-740004"></a>Precursor Transcript Continuity</h4>
<!--l. 256--><p class="noindent">Applying the same logic as that used to examine multiple distant sequences in <span class="phvro8t-x-x-120">Dst1</span>
and <span class="phvro8t-x-x-120">Fn1</span>, ligamers were designed against a highly-expressed piRNA-producing loci,
<span class="phvro8t-x-x-120">7-qD2-11976 </span>(aka - “M11”). Five unique sites were picked, again named A-E. Sites
were picked to (1) avoid repetitive regions; (2) overlap with expression evidence from
small RNA and RNA-Seq data; (3) contain loops of ~5 kb in length; and 4)
be unique in the genome. A schematic of the approach is shown in Figure
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74001r12">4.12<!--tex4ht:ref: SeqZipMethod:fig:Precursors are testes-specific --></a>A.
</p><!--l. 258--><p class="noindent">Using total RNA obtained from adult mouse testes, analyzed by SeqZip and the
ligamers shown in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74001r12">4.12<!--tex4ht:ref: SeqZipMethod:fig:Precursors are testes-specific --></a>A, signal from ligation products could routinely be
observed from loops of ~5 kb (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74001r12">4.12<!--tex4ht:ref: SeqZipMethod:fig:Precursors are testes-specific --></a>B-right and Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74002r13">4.13<!--tex4ht:ref: SeqZipMethod:fig:M1 analysis by SeqZip --></a>B). Also the signal is
dependent on source RNA (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74001r12">4.12<!--tex4ht:ref: SeqZipMethod:fig:Precursors are testes-specific --></a>B-left) and RNA from mouse spleen
did not produce ligation products. The M1 and M11 clusters are both long
and have reasonably high expression compared to the other precursors.
Yet, no ligation products were ever obtained for either cluster when loops
                                                                                

                                                                                
&gt;~5 kb were used (data not shown). What was the cause of this negative
signal?
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-74001r12"></a>
                                                                                

                                                                                
<!--l. 262--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis51x.png" alt="PIC" class="graphics" width="336.25624pt" height="208.78pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis51x.png" src="Figures/SeqZipMethod/testesSpecificRnaseqPrecursors.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.12: </span><span class="content">Testes-specific ligation product signal from piRNA precursor<br class="newline">A) Schematic of the piRNA-producing loci (“gene”) <span class="phvro8t-x-x-120">7-qD2-11976 </span>(aka “M11”)
shown with scale bar, and relative looping ligamer locations. Loops are labeled
A-E, and the length of the loop in kb is shown. Also shown in green is small
RNA expression along this locus. B Left) Ligation products obtained from each
set shown in (A) using mouse testes RNA, or B Right) mouse spleen RNA. </span></div><!--tex4ht:label?: x1-74001r12 -->
                                                                                

                                                                                
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-74002r13"></a>
                                                                                

                                                                                
<!--l. 273--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis52x.png" alt="PIC" class="graphics" width="352.31624pt" height="264.98999pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis52x.png" src="Figures/SeqZipMethod/piRNAPrecurserAnalyisBySeqZip.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.13: </span><span class="content">SeqZip signal from piRNA-producing loci <span class="phvro8t-x-x-120">17-qA3.3-27363.1</span><br class="newline">A) Schematic of the piRNA-producing loci <span class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>(aka “M1”) shown
with scale bar, and relative looping ligamer locations. Loops are labeled A–G
and the length of the loop in kb. Green is small RNA expression along this locus
and RNA-Seq in black. B) Ligation products obtained from each set shown in
(A) using mouse testes RNA. </span></div><!--tex4ht:label?: x1-74002r13 -->
                                                                                

                                                                                
<!--l. 282--><p class="noindent">As first alluded to in Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002">4.2<!--tex4ht:ref: SeqZipMethod:sec:Multiplex Gene Study --></a>, ligation efficiency should
decrease with loop length and additional required ligations. All of the ligation
products used to profile precursors only required two ligation events. Numerous
other genes had been investigated with SeqZip that contained &gt;2 sites of
ligation (sections <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-370003">2.3<!--tex4ht:ref: SeqZipPaper:sec:Discussion --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-730003">4.4.3<!--tex4ht:ref: SeqZipMethod:subsec:SeqZip on long RNAs at multiple locations --></a>). This suggested that the length of the loops
was the limiting factor in obtaining ligation products templated off piRNA
precursors.
</p><!--l. 284--><p class="noindent">We investigated this potential explanation by designing a series of ligamer sets with
increasing 1 kb increment loop lengths from 5–10 kb. Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-74003r14">4.14<!--tex4ht:ref: SeqZipMethod:fig:piRNA precusors and loop length --></a> shows results
typical of this series of experiments. The amount of ligation product when using
ligamers of increasing loop size decreases with the length of the loop. The signal,
after 35 cycles of end-point PCR, is barely visible when the loop is 9 kb, and
extremely faint when 10 kb. Ten kilo-bases represents just a fraction of the length of
most pachytene piRNA precursor transcripts.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-74003r14"></a>
                                                                                

                                                                                
<!--l. 288--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis53x.png" alt="PIC" class="graphics" width="369.37999pt" height="85.31874pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis53x.png" src="Figures/SeqZipMethod/piRNAprecusorM1SignalDecreasesWithLoopLength.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.14: </span><span class="content">SeqZip signal from piRNA precursor transcripts decreases with
loop length<br class="newline">A series of   ligamers   were   design   against   the   5<span class="cmsy-8">&#8242;</span>&nbsp;portion   of   cluster
<span class="phvro8t-x-x-120">17-qA3.3-27363.1  </span>(“M1”).  Sets  forcing  increasing  lengths  were  used,  and
ligation products were analyzed by end-point PCR. </span></div><!--tex4ht:label?: x1-74003r14 -->
                                                                                

                                                                                
<!--l. 297--><p class="noindent">Even after numerous attempts, ligation products could not be obtained for loop sizes
&gt;10 kb, no matter what the target transcript. At this point in the study, we decided to
abandon the demonstration of piRNA precursor transcripts as continuous transcripts
via SeqZip, and instead turned our attention to splicing within the transcripts
(discussed in the next section, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-750005">4.5<!--tex4ht:ref: SeqZipMethod:sec:piRNA precursors are spliced --></a>) which eventually lead to the study presented in
Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a>.
</p><!--l. 299--><p class="noindent">What could be the cause of our inability to create ligation products? The method
worked so well, without any optimization, for mRNAs of similar length and expression
(e.g. <span class="phvro8t-x-x-120">Dst1</span>). Our current hypothesis is that at steady-state levels, the amount of
full-length piRNA precursors that exist—in continuous polymers of length
&gt;10kb—is extremely low. Low to the point of being below the SeqZip limit of
detection. Indeed, many nucleases appear to act on piRNA precursors along
their journey from Pol II transcript to mature piRNA (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-230002">1.5.2<!--tex4ht:ref: Intro:subsec:Processing of piRNAs in mice --></a>). The
piRNA machinery is perhaps too fast and efficient for us to capture these
extremely long RNAs. Future experiments that somehow perturb the pathway,
such as <span class="phvro8t-x-x-120">Pld6 </span>(aka <span class="phvro8t-x-x-120">MmZuc, MitoPLD </span>and <span class="phvro8t-x-x-120">Zucchini </span>in flies) could accumulate
precursors before cleavage occurs. This is discussed in more detail in section
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-810002">5.2<!--tex4ht:ref: Disc:sec:piRNA precursors --></a>.
<a id="x1-74004r127"></a>
</p>
                                                                                

                                                                                
<h3 class="sectionHead"><span class="titlemark">4.5 </span> <a id="x1-750005"></a>Precursor Splicing</h3>
<!--l. 304--><p class="noindent">Once it was determined that the existence of piRNA precursor transcripts as
continuous piRNA precursors could not be demonstrated using SeqZip, careful
attention was paid to RNA-Seq data used to determine the edges of precursor loci
transcription. The RNA-Seq data, once aligned with a splicing-sensitive algorithm (i.e.
“Tophat” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">2009</a>]), showed that piRNA precursors were spliced.
Multiple reads and species supported intronic segments, each containing little
to no RNA-Seq and small RNA reads. A good example of the high-level
type of data observation that was being performed until this point is shown
in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75001r15">4.15<!--tex4ht:ref: SeqZipMethod:fig:evidence for precusor splicing --></a>. In this figure, small RNA data is shown in green along with
RNA-Seq data in black. For this particular cluster, the RNA-Seq data and small
RNA data appear continuous with the length of gene, as typical for many loci
in <span class="phvro8t-x-x-120">Drosophila melanogaster</span>. It was necessary to increase the resolution
used to study the piRNA-generating loci in mice in order to accurately define
transcripts.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-75001r15"></a>
                                                                                

                                                                                
<!--l. 308--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis54x.png" alt="PIC" class="graphics" width="393.46999pt" height="182.6825pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis54x.png" src="Figures/SeqZipMethod/evidenceForPrecursorSplicing.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.15:   </span><span class="content">Example   small   RNA   and   RNA-Seq   data   aligned   to   a
piRNA-generating loci (<span class="phvro8t-x-x-120">17-qA3.3-26735</span>)<br class="newline">Show in the  context  of  the  genome  and  surrounding  genes  (blue)  is  a
piRNA-generating loci, with signal in green. Bottom) zoomed view of the small
RNA signal (green) along with poly(A)+-unstranded RNA-Seq (black). </span></div><!--tex4ht:label?: x1-75001r15 -->
                                                                                

                                                                                
<!--l. 317--><p class="noindent">One of the most illustrative piRNA-generating loci is that containing the genes
<span class="phvro8t-x-x-120">17-qA3.3-27363.1 and 17-qA3.3-26735 </span>(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75002r16">4.16<!--tex4ht:ref: SeqZipMethod:fig:no piRNAs within Precursor Introns --></a>). These two genes are
expressed in pre-pachytene testes and increase expression once mice hit 14.5 dpp.
These two genes along account for 27% of all the piRNAs sequenced at 14.5 dpp
(see Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> and Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-71002r2">4.2<!--tex4ht:ref: SeqZipMethod:tab:matchedClusterValues --></a>). A extremely informative feature, detected early from
initial poly(A)+-unstranded RNA-Seq libraries, was the absence of signal
near the apparent 3<span class="cmsy-8">&#8242;</span>&nbsp;end of the loci. There were many reads that could be
aligned across this gap, as if it was a traditional mRNA intron. There were no
repeat element that would have depleted this region of the message for reads,
as with other sections of the locus. The most obvious explanation was that
the precursor contained an intron, which was spliced out prior to poly(A)
tailing..
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-75002r16"></a>
                                                                                

                                                                                
<!--l. 321--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis55x.png" alt="PIC" class="graphics" width="403.50749pt" height="367.3725pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis55x.png" src="Figures/SeqZipMethod/noPiRNAswithinPrecusorIntrons.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.16: </span><span class="content">Introns in mammalian piRNA precursors<br class="newline">Top) Divergently transcribed piRNA-producing genes <span class="phvro8t-x-x-120">17-qA3.3-27363.1 and</span>
<span class="phvro8t-x-x-120">17-qA3.3-26735</span>. These genes are transcribed from a common promoter. Plus
strand small  RNAs  are  shown  in  blue,  minus  stranded  small  RNAs  in  red.
poly(A)+-unstranded) RNA-Seq is shown in black. Bottom) Zoomed portion of
the message  near  the  3<span class="cmsy-8">&#8242;</span>&nbsp;end  of  <span class="phvro8t-x-x-120">17-qA3.3-26735</span>.  Plus-stranded  small  RNA
                                                                                

                                                                                
(blue) and RNA-Seq reads in black. Multiple RNA reads and species aligned
across a intron. This region was also largely free of small RNA signal. </span></div><!--tex4ht:label?: x1-75002r16 -->
                                                                                

                                                                                
<!--l. 330--><p class="noindent">The results shown in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75002r16">4.16<!--tex4ht:ref: SeqZipMethod:fig:no piRNAs within Precursor Introns --></a> were very exciting initially, and provided important
clues to the biogenesis of piRNAs. The presence of an intron indicates Pol II origin.
The lack of small RNA within the intron supported mature piRNA creation after
precursor splicing. A major reason why this feature had not already been noticed is
that small RNA data is not long enough to accurately and confidently align
across splice junctions. Therefore, intron detection had to wait for application
of longer RNA-Seq reads and splicing-sensitive alignment software. Once
these introns were known, supporting their use with small RNA data become
possible.
</p><!--l. 332--><p class="noindent">Using genomic coordinates supplied by the splicing-sensitive alignment algorithm
([<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">Trapnell et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XTrapnell2009">2009</a>]), an alignment index of transcript sequences <span class="phvro8t-x-x-120">flanking </span>was
created. Then, using a more traditional (in terms of small RNA alignment) aligner,
Bowtie [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">Langmead et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">2009</a>], those piRNAs that did <span class="phvro8t-x-x-120">not </span>map to the genome could
be aligned to index containg piRNA precursor splice junctions. This experiment is
shown graphically in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75003r17">4.17<!--tex4ht:ref: SeqZipMethod:fig: piRNAs map to SJ --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-75003r17"></a>
                                                                                

                                                                                
<!--l. 336--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis56x.png" alt="PIC" class="graphics" width="415.55249pt" height="258.9675pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis56x.png" src="Figures/SeqZipMethod/smallRNAsMapToPrecursorSJ.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.17: </span><span class="content">piRNAs map to precursor transcript splice junctions<br class="newline">Top) piRNA density (green) and RNA-Seq density at the 3<span class="cmsy-8">&#8242;</span>&nbsp;most intron within
<span class="phvro8t-x-x-120">17-qA3.3-26735</span>. Bottom) A splice junction sequence (blue) created by joining
the sequences  just  outside  the  intron  shown  in  (Top)  is  sufficient  to  align
non-genome mapping piRNAs. </span></div><!--tex4ht:label?: x1-75003r17 -->
                                                                                

                                                                                
<!--l. 345--><p class="noindent">Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> discusses the ultimate refinement of the observations described
above, including the generality of splicing within precursor transcripts. In
fact, there are a total of 383 introns within the “intergenic” sub-classified
214 piRNA-generating loci from [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e">2013b</a>]. These introns display a
A-MYB–dependent small RNA signal across their exon-exon junctions (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-75004r18">4.18<!--tex4ht:ref: SeqZipMethod:fig: amyb makes SJ mapping --></a>).
The more traditionally looking piRNA-producing loci of the “genic” subclass, contain
far more introns (2,113). The signal for these transcripts does not display the same
A-MYB–dependent small RNA signal. These subclasses are provided in Appendix
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-106001r2">A.2<!--tex4ht:ref: Appendix:tab:GenicAndInterGenicLoci --></a>.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-75004r18"></a>
                                                                                

                                                                                
<!--l. 349--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis57x.png" alt="PIC" class="graphics" width="415.55249pt" height="188.70499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis57x.png" src="Figures/SeqZipMethod/aggregatePiRNAsatSpliceJunctions.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;4.18: </span><span class="content"><span class="phvro8t-x-x-120">A-Myb </span>Mutants produce no splice-junction mapping piRNAs for
genic piRNA-producing loci<br class="newline">Trimmed mean ppm of junction-mapping piRNAs within two classes (“genic &amp;
Intergenic”) loci. Shown in red is signal from <span class="phvro8t-x-x-120">A-Myb </span>mutant mice, black <span class="phvro8t-x-x-120">A-Myb</span>
heterozygous  mice.  All  data  from  stranded  RNA-Seq  (strand  accounted  for
during alignment and signal aggregation). </span></div><!--tex4ht:label?: x1-75004r18 -->
                                                                                

                                                                                
<!--l. 358--><p class="noindent">While it was not possible to demonstrate continuity of piRNA-producing precursors
using the SeqZip method, development of advanced HTS methods and computational
approaches provides clear evidence that they are (see Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a>). Proposed future
experiments into mammalian piRNA precursors are discussed in section
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-810002">5.2<!--tex4ht:ref: Disc:sec:piRNA precursors --></a>.
                                                                                

                                                                                
                                                                                

                                                                                
</p>
<h2 class="chapterHead"><span class="titlemark">Chapter&nbsp;5</span><br><a id="x1-760005"></a>Discussion</h2>
<a id="x1-76001r141"></a>
<h3 class="sectionHead"><span class="titlemark">5.1 </span> <a id="x1-770001"></a>Concerning the Transcriptome</h3>
<!--l. 9--><p class="noindent">Deep sequencing of transcriptomes has revolutionized biology. Previously, transcript
identification and characterization involved significant labor, cost, and materials. In
the mid-90’s, microarray technology [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchena1995a">Schena et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSchena1995a">1995</a>] gave a tantalizing glimpse
into how genes were expressed, but were limited to probed, and therefore known,
sequences. Yet, the green and red landscapes hinted at incredible complexity.
Full realization of this complexity would have to wait for technology to catch
up.
</p><!--l. 11--><p class="noindent">RNA-seq is possible due to incremental improvements in numerous supportive
technologies included: (1) digital optics; (2) microscopy; (3) slide chemistry; (4) colony
PCR; and (4) nucleic-acid alignment. A HiSeq 2500 relies on all of these technologies
(and others) to produce the &gt;100,000,000 sequences that allow scientists to peer into
the transcriptional output of a genome.
</p><!--l. 13--><p class="noindent">Biologists can now think beyond mRNAs and small RNAs. The former captured our
interest for 30+ years [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFuruichi1975">Furuichi</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFuruichi1975">1975</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWei1975">Wei et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWei1975">1975</a>], while the later has been on
a run-away train since 1998 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFire1998">Fire et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XFire1998">1998</a>]. Now included on the list of
                                                                                

                                                                                
captivating RNAs are Long non-coding RNAs (lncRNAs). All classes of RNA are
now routinely measured by HTS. However, many biologically-trained and
minded scientists find themselves overwhelmed by methods and approaches
used to tackle these biological “big data.” Current training programs do not
provide most with required skills in statistics, programing, and experimental
design necessary to work with genome-wide data (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-980003">5.4.3<!--tex4ht:ref: Disc:subsec:Biologists need Comp Skills --></a>). The
richness of these data often results in unasked—and unanswered—testable
hypothesis, answers to which are just sitting in public data repositories [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013">Plocik and
Graveley</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013">2013</a>].
</p><!--l. 15--><p class="noindent">This discussion will focus on <span class="phvro8t-x-x-120">long </span>RNA class that contain traditional mRNA
features—a 5<span class="cmsy-8">&#8242;</span>&nbsp;m7G Cap, ligated exons, and a poly(A)+ tail. Many of these long
mRNAs are extremely dynamic. So much so that until HTS and RNA-Seq,
comprehensive investigation of their complexity was impossible.
<a id="x1-77001r137"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.1.1 </span> <a id="x1-780001"></a>Pervasive transcription</h4>
<!--l. 20--><p class="noindent">There are 2,598,960 different Poker hands possible from a 52-card deck. There are
1,098,240 different single-pair combinations, with a probability of obtaining one being
almost 50%. Compare this to a “Royal Flush”, for which there are only 4 options, and
a probability of 649,739:1 or 1.54 * 10<sup><span class="cmsy-8">-</span><span class="cmr-8">6</span></sup>! It is these numbers that makes it possible to
                                                                                

                                                                                
play Poker for hours on end.
</p><!--l. 22--><p class="noindent">Biology uses similar combinatorics to arrange exons into unique and rare
combinations. This is especially true complex eukaryotic organisms, where virtually
all genes are alternatively spliced (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-11001r4">1.4<!--tex4ht:ref: Intro:fig:numGenesAndNumSpliced --></a>). Accurate determination and assembly
of each card (exon) that comprises a hand (transcript) is a major “known unknown”
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRumsfeld2011">Rumsfeld</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRumsfeld2011">2011</a>] of research into long RNAs.
</p><!--l. 24--><p class="noindent">The ENCODE papers of late 2012 suggest that 95% of the genome is functional
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDunham2012">Dunham et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDunham2012">2012</a>], a heavily debated finding [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBhattacharjee2014">Bhattacharjee</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBhattacharjee2014">2014</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraur2013">Graur
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGraur2013">2013</a>]. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">Djebali et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDjebali2012">2012</a>] focused on transcription in the ENCODE cell lines
(discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-150005">1.3.5<!--tex4ht:ref: Intro:subsec:IsoformsPerGene --></a>) and concluded that 75% of the genome is transcribed into
RNA. Additionally, “GENCODEv7” includes 9,640 manually curated lncRNA loci.
These lncRNA are some of the most novel and functionally interesting class of long
RNAs [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDerrien2012">Derrien et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XDerrien2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPauli2011">Pauli et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPauli2011">2011</a>]. While ENCODE was performed
using multiple human cancerous cell lines, these results do support the ability of HTS
to reveal transcriptional diversity.
<a id="x1-78001r148"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.1.2 </span> <a id="x1-790002"></a>A Need for Transcript Assembly</h4>
<!--l. 29--><p class="noindent">The field of transcriptome assembly is in its infancy (see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-240003">1.5.3<!--tex4ht:ref: Intro:subsec:Tx Assembly --></a>). Current
transcript assembly algorithms only provide predictions and probabilities for the
                                                                                

                                                                                
existence of real molecules. Until RNA is directly and completely sequenced
from single cells or molecular compartments, researchers will always be
forced to make compromises in annotation and quantification [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOzsolak2010">Ozsolak and
Milos</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XOzsolak2010">2010</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">Steijger et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">2013</a>]. Once technology advances to the point
where a transcriptome is as accurately and quickly determined as a genome,
exciting research into the subtle and nuanced complexity of transcriptome
regulation will be revealed (e.g. what makes one twin <span class="phvro8t-x-x-120">molecularly </span>different from
another?).
</p><!--l. 31--><p class="noindent">What is required to improve our ability to quickly and accurately assemble
transcriptomes? Simulations indicate that improvements will not come from longer
read lengths [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014c">Chang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChang2014c">2014b</a>]. These simulations also demonstrate that
the accuracy of current <span class="phvro8t-x-x-120">de novo </span>(see section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-240003">1.5.3<!--tex4ht:ref: Intro:subsec:Tx Assembly --></a>) assemblers decreases
sharply with increased increase alternative splicing within the transcriptome.
Systemic assessment of RNA-Seq transcript reconstruction methods have
concluded what is likely to be the most revolutionary step toward accurate
transcriptome assembly—single pass sequencing of single transcripts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEngstrom2013">Engström
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEngstrom2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">Steijger et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSteijger2013">2013</a>]. Results presented by [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">Sharon et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSharon2013">2013</a>],
demonstrating inherent constraints imposed by requiring RT to convert long RNAs to
long cDNAs, mean that this future single-molecule sequencing will have to be of the
RNA directly. This will be especially important and informative to measure RNA
modifications such as N6-methyl-adenosine [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2013">Pan</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2013">2013</a>]
                                                                                

                                                                                
<a id="x1-79001r149"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.1.3 </span> <a id="x1-800003"></a>Tissue and Cell Specificity</h4>
<!--l. 36--><p class="noindent">As discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-110001">1.3.1<!--tex4ht:ref: Intro:subsec:Alternative Splicing --></a>, mechanisms of alternative splicing are frequently
tissue-, time-, and cell- specific. Landmark studies examining alternative splicing in
different organ systems, from evolutionarily-distant organisms, found that alternative
splicing is more comparable between organs of different animals than between
different organs from the <span class="phvro8t-x-x-120">same </span>animal [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarbosa-Morais2012">Barbosa-Morais et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarbosa-Morais2012">2012</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkin2012">Merkin
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMerkin2012">2012</a>]. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">Brown et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBrown2014">2014</a>] analyzed the <span class="phvro8t-x-x-120">Drosophila melanogaster</span>
transcriptome, and observed that alternative splicing could be better described as
“tissue-specific splicing”. Further, tissue-specific lncRNA expression has been
recently reported [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWashietl2014">Washietl et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWashietl2014">2014</a>], adding to the importance of sample
resolution when performing transcriptome analysis.
</p><!--l. 38--><p class="noindent">The concept of “tissue-specific splicing” brings up a subtle but important
consideration concerning alternative splicing research. The concept of alternative
splicing conjures an image of dynamic post-transcription RNA processing, allowing
cells to quickly respond changes in environment, programming, or stimuli. Yet, as
demonstrated by the studies just mentioned, most alternative splicing is not that
dynamic. In fact, the main reason many of these events are even considered
<span class="phvro8t-x-x-120">alternative </span>is because we are comparing transcriptomes of different tissues. Is that a
                                                                                

                                                                                
fair comparison?
</p><!--l. 40--><p class="noindent">By the current definition these events are indeed <span class="phvro8t-x-x-120">alternative</span>, but significance does
that label carry? The label of alternative communicates that RNA products of a gene
can be processed in multiple ways, but if they are typically <span class="phvro8t-x-x-120">not</span>, does this capability
matter when HTS allows analysis of very specific biological samples and time points?
These questions underscore the importance of advances in transcriptome assembly
keeping step with advancements in HTS technology and ever-increasing sample
resolution.
<a id="x1-80001r147"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">5.2 </span> <a id="x1-810002"></a>In the haystack: piRNA Precursors</h3>
<!--l. 45--><p class="noindent">Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> describes the annotation of 467 transcripts from 214 loci. These loci
account for 95% of the total piRNAs in 14.5 dpp mice. These transcripts possess the
archetypical molecular signatures of Pol II origin, including 5<span class="cmsy-8">&#8242;</span>&nbsp;7meG CAP, introns, and
poly(A)+ tails. Yet RNA from these molecules appears to be rapidly consumed and
processed into millions of unique small RNA (&nbsp;23–35 nt) species. How does
the cell partition mRNAs for translation by the ribosome or maturation into
piRNAs?
<a id="x1-81001r150"></a>
</p>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">5.2.1 </span> <a id="x1-820001"></a>Precursor Identity</h4>
<!--l. 50--><p class="noindent">A good example that highlights why this issue is shown in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-82001r1">5.1<!--tex4ht:ref: Disc:fig:wdfy3 --></a>. In testes
of mice, the <span class="phvro8t-x-x-120">Wdfy3 </span>gene produces at least two different transcripts from
different promoters. Virtually all piRNAs map within the bounds of the shorter
isoform (<span class="phvro8t-x-x-120">PI-Wdfy3.1</span>). The promoter that falls more 3<span class="cmsy-8">&#8242;</span>&nbsp; within the gene is
also bound by A-MYB. The more 5<span class="cmsy-8">&#8242;</span>&nbsp; promoter, which presumably drives
transcription of the longer isoform (<span class="phvro8t-x-x-120">PI-Wdfy3.2</span>), is not bound by A-MYB. Also, in
<span class="phvro8t-x-x-120">A-Myb </span>mutants, piRNAs from the shorter locus are drastically reduced as are
RNA-Seq reads. RNA-Seq reads aligning to the longer transcript did not
decrease.
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-82001r1"></a>
                                                                                

                                                                                
<!--l. 54--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis58x.png" alt="PIC" class="graphics" width="397.48499pt" height="231.86624pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis58x.png" src="Figures/Discussion/pi-wdfy3.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;5.1: </span><span class="content"><span class="phvro8t-x-x-120">wdfy3 </span>locus expresses both mRNA and piRNA precursor form in
testes<br class="newline">The mouse  genomic  locus  <span class="phvro8t-x-x-120">wdfy3  </span>expresses  both  a  traditional  mRNA  form,
originating from  an  upstream  TSS,  and  a  piRNA  precursor  transcript  from  a
downstream TSS.  The  piRNA  precursor  form  appears  to  originate  from  an
A-MYB-bound promoter, and is expressed in <span class="phvro8t-x-x-120">A-Myb </span>heterozygous mice. Also
small RNAs  (piRNAs)  mapping  to  this  locus  are  only  observed  in  <span class="phvro8t-x-x-120">A-Myb</span>
heterozygous mice, and not in <span class="phvro8t-x-x-120">A-Myb </span>mutant mice. </span></div><!--tex4ht:label?: x1-82001r1 -->
                                                                                

                                                                                
<!--l. 62--><p class="noindent">These results indicate that A-MYB drives transcription of the shorter <span class="phvro8t-x-x-120">pi-Wdfy3.1</span>
isoform, but not longer isoform, annotated elsewhere simply as <span class="phvro8t-x-x-120">Wdfy3</span>. A
general phenomenon of piRNAs mapping to the 3<span class="cmsy-8">&#8242;</span>&nbsp;-UTR of mRNAs has been
reported [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">Robine et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobine2009">2009</a>]. How does a cell discriminate between these two
transcripts
</p><!--l. 64--><p class="noindent">Recently it was demonstrated that virtually all RNAs interact with the ribosome
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIngolia2011">Ingolia et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XIngolia2011">2011</a>]. This observation was later refined to state that only mRNAs
display a strong “Ribosome Release Score (RRS)” indicative of read-frame
engagement [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2013">Guttman et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XGuttman2013">2013</a>]. Therefore, it is not surprising that preliminary
results support precursors being traversed by ribosomes (Xin Zhiguo Li,
unpublished). Additional experiments and bioinformatic analysis may tease out
sequence elements the assist in precursor discrimination from mRNAs by the
ribosome, similar to the RRS for traditional ncRNAs.
</p><!--l. 66--><p class="noindent">Beyond ribosome profiling, what are other potential experimental approaches that
could be used to gain insight into the biology of mammalian piRNA precursors?
<a id="x1-82002r152"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.2.2 </span> <a id="x1-830002"></a>Precursor Interactions</h4>
<!--l. 71--><p class="noindent">While intergenic piRNA loci share many features with other Pol II transcript classes
                                                                                

                                                                                
(Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-72001r8">4.8<!--tex4ht:ref: SeqZipMethod:fig:piRNA precusor Tx features --></a>) one almost half contain no introns. Visual inspection using genome
browsers and the HTS datasets described in Chapter <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003">3<!--tex4ht:ref: MolCel --></a> revealed how unique these
loci are from the rest of the transcriptome.
</p><!--l. 73--><p class="noindent">As discussed in sections <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-110001">1.3.1<!--tex4ht:ref: Intro:subsec:Alternative Splicing --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-120002">1.3.2<!--tex4ht:ref: Intro:subsec:Splicing Code --></a>, splice sites and SRE are recognized
amidst a sea of extremely similar “crytic” sequences. The spliceosome is
remarkably efficient at determining the correct elements to use. Spliceosomal
components assist in choosing from this the overwhelming set of sequences. <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBerg2012">Berg
et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBerg2012">2012</a>] identified the snRNP U1 as a key suppressor of cryptic polyadenylation
site (PAS) use. This suppressor activity is in contrast to its primary role in
the definition of 5<span class="cmsy-8">&#8242;</span>&nbsp;splice sites. Perhaps a similar mechanism is acting on
cryptic splice sites contained within precursor transcripts. What experiments
could be used to identify precursor interacting molecules, protein and RNA
both?
</p><!--l. 75--><p class="noindent">Coincident with HTS development, methodology to measure genome-wide
interactions have also made considerable advances [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKonig2011">König et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKonig2011">2011</a>].
Methodologies to capture {Protein::RNA} interactions include “HTS-CLIP” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">Licatalosi
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLicatalosi2008">2008</a>], “PAR-CLIP” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2010">Hafner et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHafner2010">2010</a>], and “iCLIP” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKonig2010">König et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKonig2010">2010</a>].
{DNA::RNA} interactions are measurable using “ChIRP” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChu2012">Chu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChu2012">2012</a>],
{RNA::RNA} interactions by “RAP” and “CLASH” [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEngreitz2013">Engreitz et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XEngreitz2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHelwak2014">Helwak and
Tollervey</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHelwak2014">2014</a>]. These approaches could be applied to determining piRNA
                                                                                

                                                                                
precursor transcript interacting molecules. However, there are some important
caveats that warrant discussion.
</p><!--l. 77--><p class="noindent">Techniques mentioned above that investigate {Nucleic acid::Protein} interactions
require a target protein. This has already been done for MILI and MIWI in postnatal
testes [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>], and few additional interacting proteins are known,
limiting the number of proteins to investigate. Obvious initial candidates include
MitoPLD, Mvh [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLasko2013">Lasko</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLasko2013">2013</a>] (the mouse homologue of Vasa) and numerous
Tudor-related proteins [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2011">Chen et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2011">2011</a>].
</p><!--l. 79--><p class="noindent">Genome-wide study of {Nucleic Acid::Protein} interactions typically require
cross-linking [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChodosh2001">Chodosh</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChodosh2001">2001</a>] using either ultra violet light or reagents such as
glutathione. This requirement is why most original reports of these techniques are
performed in cell culture (due to the relative easy of exposing the sample to
the cross-linking reagent). Currently, the process of piRNA biogenesis has
only been reproduced <span class="phvro8t-x-x-120">in vitro </span>using silk worm cell culture extracts [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2009">Kawaoka
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2009">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKawaoka2011">2011</a>], a system which is likely far from that of pachytene biogenesis
and function in mice. Therefore, application of these techniques to mammalian piRNA
pathway study would require testes sectioning prior to cross-linking. [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">Vourekas
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XVourekas2012">2012</a>] worked around this requirement by detunicated testes and creating a
cell suspension in a petri dish which was then irradiated.
</p><!--l. 81--><p class="noindent">Another potential work-around for would be to perform these studies in mature (or
                                                                                

                                                                                
<span class="phvro8t-x-x-120">maturing</span>) sperm. Sperm develop and mature as they move through the seminiferous
tubules and into the epididymis, where piRNAs are known to be “sequence-able” in
humans [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJones1999">Jones</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XJones1999">1999</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2012a">Li et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2012a">2012b</a>]. However, how much of exciting biology
driven by piRNAs has already occurring once sperm have transitioned into the
epididymis?
</p><!--l. 83--><p class="noindent">{Nucleic Acid::Nucleic acid} interactions typically require a ligation step, the efficiency
of which is typically very low [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHelwak2014">Helwak and Tollervey</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XHelwak2014">2014</a>]. Also, the “ChIRP” protocol
is done in crude cell extract, where RNAse H is a concern when using DNA probes to
pull down and query RNA. Given that precursor transcripts seem to be rapidly
processed (section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-740004">4.4.4<!--tex4ht:ref: SeqZipMethod:sec:piRNA precursor by SeqZip --></a>), these methods may require prior enrichment, perhaps
using RNACapture [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMercer2014">Mercer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMercer2014">2014</a>]. Given recent developments into the
CRISPR/CAS9 system for genome-editing [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSander2014">Sander and Joung</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XSander2014">2014</a>], the
“CLASH” approach to look at {RNA::RNA} interactions for precursor transcripts is
attractive as the requirement for a tagged protein is no longer as large a
barrier.
</p><!--l. 85--><p class="noindent">There are many applications to piRNA biogenesis biology for these experimental
techniques as they evolve and become more robust. Increased resolution of time
points, proteins, and species examined will help to create a comprehensive purpose
for piRNA in the maintenance of mammalian male fertility.
<a id="x1-83001r154"></a>
                                                                                

                                                                                
</p>
<h4 class="subsectionHead"><span class="titlemark">5.2.3 </span> <a id="x1-840003"></a>Precursor Location</h4>
<!--l. 90--><p class="noindent">A drawback of all the methods and approaches discussed above is that they do not
maintain the anatomical and cellular location of transcripts. Localization of RNA has
been important for decades [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRebagliati1985">Rebagliati et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRebagliati1985">1985</a>], and was recently shown in a
large screen in <span class="phvro8t-x-x-120">Drosophila melanogaster </span>embryos to be the rule rather than the
exception [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLecuyer2007">Lécuyer et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLecuyer2007">2007</a>].
</p><!--l. 92--><p class="noindent">The most important question for mammalian <span class="phvro8t-x-x-120">pachytene </span>piRNAs is <span class="phvro8t-x-x-120">What are</span>
<span class="phvro8t-x-x-120">they doing</span>? We know that they are essential for the health of the species,
as discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-440002">3.2<!--tex4ht:ref: MolCel:sec:Introduction --></a>, and piRNA-pathway mutants are sterile. What
could these small RNAs, with complementarity to nothing but themselves, be
doing?
</p><!--l. 94--><p class="noindent">The cellular location of precursor piRNA transcript processing is not known. The most
accepted hypothesis is that precursor transcripts are processed into mature piRNAs
with machinery tethered to chromatoid bodies [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeikar2011">Meikar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeikar2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMeikar2014">2014</a>] or another
structure similar to Drosophila Nuage. Knowledge of <span class="phvro8t-x-x-120">where </span>where mature
piRNAs are generated would provide clues into larger biogenesis mechanistic
details.
</p><!--l. 96--><p class="noindent">Identifying the location of mature piRNA processing from precursor transcripts could
                                                                                

                                                                                
be achieved through development and application of techniques to visualize
precursors in a dense sea of other RNA, including mature piRNAs. Improvements in
<span class="phvro8t-x-x-120">in vitro </span>FISH experiments allow for discrimination of isoforms resulting from
alternative splicing [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014">Lee et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014">2014b</a>]. Robust FISH-type experiments could be used
to investigate cellular and anatomical locations of precursor transcript processing.
The SeqZip methodology could even be used in this regard (see section
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-920002">5.3.4.2<!--tex4ht:ref: Disc:subsubsec:SeqZip and Single-Molecule FISH --></a>).
</p><!--l. 98--><p class="noindent">Beyond FISH, direct imaging of precursor transcripts could be accomplish by
engineering MS2 loop sequences into piRNA-generating genes similar to
experiments performed in the the Singer lab [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2014">Park et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPark2014">2014</a>]. This would be
assisted by the previously mentioned CRISPR/CAS9 systems. Mice expressing
precursors containing MS2 loops could be crossed with those containing
MS2 Bacteriophage capsid protein fused to GFP (MCP-GFP). Using this
system precursors could potentially be visualized in real time or at least in real
locations.
<a id="x1-84001r155"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.2.4 </span> <a id="x1-850004"></a>Precursor Sequencing</h4>
<!--l. 103--><p class="noindent">Very recently methods demonstrating sequencing <span class="phvro8t-x-x-120">in situ </span>have been published [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKe2013">Ke
et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XKe2013">2013</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014a">Lee et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014a">2014a</a>]. These methods represent a major improvement
                                                                                

                                                                                
over the single-cell sequencing approaches discussed in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-90003">1.2.3<!--tex4ht:ref: Intro:subsec:Types of HTS --></a>. Building upon
the principles of FISH, <span class="phvro8t-x-x-120">in situ </span>sequencing allows for novel sequence discovery,
multiplex investigation, and cellular location RNA. Could <span class="phvro8t-x-x-120">in situ </span>sequencing be used
to learn more about piRNA precursor biology?
</p><!--l. 105--><p class="noindent">FISSEQ, reported by <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014a">Lee et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLee2014a">2014a</a>], uses rolling circle amplification to create a
3D grid of highly-concentrated DNA (“nanoballs”), originating from a single
RNA/cDNA. SOLiD sequencing is used to determine 27–30 bases from each
nanoball. Confocal microscopy is used to assign the sequence to 3D location within
the sample. Read lengths for FISSEQ would make it difficult to distinguish mature
piRNAs from precursors. An experimental scheme, perhaps exploiting the methylation
of mature piRNAs, would be necessary to ensure sequencing of piRNA precursors or
intermediates.
</p><!--l. 107--><p class="noindent">Whether by determining the interacting molecules, physical location, or <span class="phvro8t-x-x-120">in situ</span>
sequence, more advanced techniques are required to advance our understanding of
this novel and exciting cellular process in a complex organ that perpetuates
mammalian species.
<a id="x1-85001r151"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">5.3 </span> <a id="x1-860003"></a>Future of RNA-templated DNA-DNA ligation</h3>
<!--l. 112--><p class="noindent">The SeqZip methodology as developed and described in Chapters <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a> and <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a> works
                                                                                

                                                                                
adequately and robustly for characterization of relatively simple (<span class="phvro8t-x-x-120">CD45</span>) to extremely
complex (<span class="phvro8t-x-x-120">Dscam1</span>) genes. However, there is substantial room for optimization. The
improvements, modifications, and applications discussed below support continued
use of SeqZip in RNA research.
<a id="x1-86001r156"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.3.1 </span> <a id="x1-870001"></a>An Optimized SeqZip Examination of Coordinated Splicing</h4>
<!--l. 117--><p class="noindent">If I could go back in time 4 years and still possess the knowledge and abilities that I
do now, I would have approached a genome-wide study of coordination in splicing
using SeqZip differently. I would have focused on alternative first exon use and
potential coordination with downstream cassette exons. I would have mined
newly-generated RNA-Seq data [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">Pan et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPan2008">2008</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">Wang et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XWang2008">2008</a>] to limit
identified targets for sufficient variation and expression.
</p><!--l. 119--><p class="noindent">Following target identification, I would have automated the ligamer design process
(see Appendix <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-107000B">B<!--tex4ht:ref: Appendix:AutoMatedLigamerAssembly --></a>), to create a database of the required ligamers. At least three
ligamers would be required per event, with very little duplicated use of ligamers. The
number of ligamers required would preclude the use of standard synthesis (even in a
384-well plate format). Therefore, I would have pursued printing the ligamers
on a custom microarray, similar to products offered by <a href="http://www.nimblegen.com/">Nimblogen</a>. These
ligamers would be barcoded and priming sequences included such that short
                                                                                

                                                                                
(50nt) paired-end reads could reliably identify the templated first and cassette
exons.
</p><!--l. 121--><p class="noindent">Using this complex ligamer pool, I would have further optimized the SeqZip assay,
including a barcoding scheme to quantify the number of ligation events per {alt first
exon::cassette exon} pair. Also, libraries would have been amplified using a digital
PCR scheme. This would allow determination of PCR jackpots and enhance read
quantification [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShiroguchi2012a">Shiroguchi et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XShiroguchi2012a">2012</a>]. Finally, the data would be aligned against a
reference of all {alt first exon::cassette exon} pairs and any potential coordination
determined.
</p><!--l. 123--><p class="noindent">If I could have done the experiment described above, I feel the full potential and utility
of the SeqZip method could have been realized and generated new and valuable
knowledge for the field of gene expression.
<a id="x1-87001r158"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.3.2 </span> <a id="x1-880002"></a>LNA-containing ligamers and T39A Rnl2</h4>
<!--l. 128--><p class="noindent">The use of an RNA-base on the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick encourages a C3<span class="cmsy-8">&#8242;</span>&nbsp;<span class="phvro8t-x-x-120">endo </span>sugar
pucker for the base, placing the 3<span class="cmsy-8">&#8242;</span>&nbsp;OH in an apical orientation relative to the the AMP
leaving group (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-88001r2">5.2<!--tex4ht:ref: Disc:fig:Rnl2 and suger pucker --></a>) [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>]. With lowered costs of oligo
synthesis, incorporation of 2<span class="cmsy-8">&#8242;</span>&nbsp;OMe at the penultimate and ultimate bases of the
5<span class="cmsy-8">&#8242;</span> &nbsp;nick ligamers should greatly increase ligation efficiency, as these are the primary
                                                                                

                                                                                
substrate-specificity determinants of Rnl2 due to this conformational constraint
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">Nandakumar et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">2004</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>].
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-88001r2"></a>
                                                                                

                                                                                
<!--l. 132--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis59x.png" alt="PIC" class="graphics" width="361.34999pt" height="188.70499pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis59x.png" src="Figures/Discussion/Rnl2_and_suger_pucker.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;5.2: </span><span class="content">Sugar pucker in Rnl2 structures<br class="newline">Using two different nucleic acid substrate combinations crystallized with Rnl2,
<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">Nandakumar  et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2006">2006</a>]  demonstrates  the  effect  of  3<span class="cmsy-8">&#8242;</span>&nbsp;and  2<span class="cmsy-8">&#8242;</span>&nbsp;  identify  of
the base at the 5<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick: Left) The crystal structure (PDB: 2HVS),
containing a 2<span class="cmsy-8">&#8242;</span>&nbsp; position deoxy residue, displays a DNA-like C2<span class="cmsy-8">&#8242;</span>&nbsp;<span class="phvro8t-x-x-120">endo  </span>sugar
pucker. In contrast to Right) where the crystal structure (2HVR) contains a 2<span class="cmsy-8">&#8242;</span>&nbsp;
hydroxyl and displays an RNA-like C3<span class="cmsy-8">&#8242;</span>&nbsp;<span class="phvro8t-x-x-120">endo </span>sugar pucker. </span></div><!--tex4ht:label?: x1-88001r2 -->
                                                                                

                                                                                
<!--l. 140--><p class="noindent"><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">Nandakumar et&nbsp;al.</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XNandakumar2004a">2004</a>] demonstrated the importance of the ribose at this
penultimate position, evidenced by a 50-fold reduction in turnover for substrates
containing a 2<span class="cmsy-8">&#8242;</span>&nbsp;H substitutions. Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-88001r2">5.2<!--tex4ht:ref: Disc:fig:Rnl2 and suger pucker --></a> shows that Threonine 39 (T39) hydrogen
bonds with both the 2<span class="cmsy-8">&#8242;</span>&nbsp;OMe and 3<span class="cmsy-8">&#8242;</span>&nbsp;O of the penultimate sugar. A T39A mutation did
not phenocopy the 2<span class="cmsy-8">&#8242;</span>&nbsp;H substitution on the penultimate sugar, indicating that the
structural constraint of sugar pucker is important for efficient ligation. These results
support the use of a T39A Rnl2 mutant for increased RNA-templated DNA-DNA
ligation efficiency, as the mutant would have one less molecular requirement for an
RNA substrate.
</p><!--l. 142--><p class="noindent">Future versions of the SeqZip assay could use a combination of LNA modified bases
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYou2006">You et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XYou2006">2006</a>] at either the penultimate or terminal residues (or both) on the
5<span class="cmsy-8">&#8242;</span>&nbsp;side of the nick in order to increase specificity and efficiency. The combination of
modified ligamers and the T39A Rnl2 mutant could enhance the efficiency of
RNA-templated DNA-DNA ligations required by SeqZip.
<a id="x1-88002r159"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.3.3    </span> <a id="x1-890003"></a>Thermostable Ligases</h4>
<!--l. 147--><p class="noindent">The use of LNA-containing brings up issues involving off-target hybridization.
Directed protein evolution of Rnl2 [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRomero2009a">Romero and Arnold</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRomero2009a">2009</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XStemmer1994">Stemmer</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XStemmer1994">1994</a>] could
                                                                                

                                                                                
be used to develop a thermostable variant of the enzyme, similar to variations of DNA
ligase that have been used for years [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarany1991">Barany</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBarany1991">1991</a>]. Use of LNAs and elevated
ligation temperatures could alleviate off-target hybridization events reducing
both nonproductive hybridization and non-templated ligation. This would
also allow for use of reduced overall ligamer concentrations, in line with the
optimal SeqZip experiment described in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-870001">5.3.1<!--tex4ht:ref: Disc:subsec: Ideal SeqZip exp. to look for Coordination --></a> and ligamer synthesis on
microarrays.
<a id="x1-89001r161"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.3.4 </span> <a id="x1-900004"></a>Other SeqZip Applications</h4>
<!--l. 152--><p class="noindent">SeqZip can be used in many different forms of RNA sequence characterization. An
incomplete illustration of these applications is shown in Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-90001r3">5.3<!--tex4ht:ref: Disc:fig: Panel of SeqZip Applications --></a>. Three novel
applications are discussed below:
</p>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-90001r3"></a>
                                                                                

                                                                                
<!--l. 156--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis60x.png" alt="PIC" class="graphics" width="397.48499pt" height="324.21124pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis60x.png" src="Figures/Discussion/SeqZipUses.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;5.3: </span><span class="content">Proposed uses of the SeqZip methodology<br class="newline">Shown are  general  applications  of  the  SeqZip  method  to  profiling  RNA
sequences. Top  row  examples  are  substantiated  by  experiments  described
on Chapters  <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-260002">2<!--tex4ht:ref: SeqZipPaper --></a>  and  <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-630004">4<!--tex4ht:ref: SeqZipMethod --></a>.  Middle  and  bottom  rows  are  hypothetical,  but  logical,
extensions of the method. </span></div><!--tex4ht:label?: x1-90001r3 -->
                                                                                

                                                                                
<a id="x1-90002r110"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.3.4.1 </span> <a id="x1-910001"></a>Multi-site SNP detection</h5>
<!--l. 167--><p class="noindent">The concept of connectivity in sequence can be applied not only to exons, or long
stretches of RNA, but even to single-nucleotide polymorphisms (SNPs). SeqZip could
be used to profile potential SNPs contained within the same transcript and therefore
within the same allele (Figure <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-90001r3">5.3<!--tex4ht:ref: Disc:fig: Panel of SeqZip Applications --></a>). For maximal benefit and specificity, care should be
taken as to where the variant ligamers bases are placed respective to the 5<span class="cmsy-8">&#8242;</span>&nbsp; or 3<span class="cmsy-8">&#8242;</span>&nbsp;
side of the ligation site. For example, <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">Chauleau and Shuman</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChauleau2013b">2013</a>] have
demonstrated the importance of proper base pairing at the 3<span class="cmsy-8">&#8242;</span>&nbsp;OH side of the
nick.
<a id="x1-91001r164"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">5.3.4.2 </span> <a id="x1-920002"></a>SeqZip and single-molecule multi-site FISH</h5>
<hr class="figure"><div class="figure">
                                                                                

                                                                                
<a id="x1-92001r4"></a>
                                                                                

                                                                                
<!--l. 174--><p class="noindent"><img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis61x.png" alt="PIC" class="graphics" width="369.37999pt" height="125.46875pt"><!--tex4ht:graphics  
name="RoyC_Umass_Thesis61x.png" src="Figures/Discussion/MultiSiteFish.eps"  
-->
<br></p><div class="caption"><span class="id">Figure&nbsp;5.4: </span><span class="content">Multi-Site smFISH using flourophore-containing ligamers<br class="newline">The simplest form of a multi-site FISH SeqZip experiment. Five ligamers are
used, with two hybridizing to the beginning and end of a target RNA sequence
(e.g. and exon),  and  having  unique  fluorescent  labels  (Cy3  and  Cy5  in  this
case). The use of the third ligamer, containing an addressable barcode is used
to report on these two ligamers hybridizing to the same RNA. Use of flanking
ligamers allows for amplification, downstream analysis, and trouble shooting. </span></div><!--tex4ht:label?: x1-92001r4 -->
                                                                                

                                                                                
<!--l. 184--><p class="noindent">A logical extension of the multi-site SNP detection application described above is the
use of SeqZip in multi-site FISH probes Figure (<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-92001r4">5.4<!--tex4ht:ref: Disc:fig:MultiSite FISH using SeqZip --></a>). Advances in FISH, microscopy,
fluorescent moieties, and image processing are making this type of experiment more
approachable. A multi-site FISH SeqZip experiment could be used to ask some of the
questions described in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-840003">5.2.3<!--tex4ht:ref: Disc:subsec:Imaging of precursors --></a>, including precursor integrity and location,
without the need for downstream processing.
<a id="x1-92002r165"></a>
</p>
<h5 class="subsubsectionHead"><span class="titlemark">5.3.4.3 </span> <a id="x1-930003"></a>Introduction of destruction sequences</h5>
<!--l. 189--><p class="noindent">Introduction of unique sequences into a ligation product is a powerful feature of
SeqZip. In addition to the other proposed uses (barcoding, sites of priming,
sequence diversity, etc) the sequences introduced could also be used to
<span class="phvro8t-x-x-120">eliminate </span>ligation products. Restriction enzymes or elimination via selective
hybridization, similar removal of ribosomal RNA sequences during HTS library
preparation [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2011a">Chen and Duan</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XChen2011a">2011</a>], are two potential ways to remove products of
ligation.
<a id="x1-93001r167"></a>
</p>
                                                                                

                                                                                
<h5 class="subsubsectionHead"><span class="titlemark">5.3.4.4 </span> <a id="x1-940004"></a>Re-purposing the SOLiD Platform</h5>
<!--l. 194--><p class="noindent">Custom sequences within ligamers could also be used to generalize exon identity. Put
another way, ligamers representing the first exon of a message could be given a
specific barcode, second exons another barcode, and so on. Then, using a
sequencing platform such as SOLiD and custom hybridization/sequencing oligos,
florescent signal would report not the sequence, but the numeric ID of the exon within
the target message. A few rounds of traditional sequencing could identify the mRNA
from each spot, and a simplistic schema of exon arrangement could be interpreted
from the ligation product. This would require major SeqZip optimization, bioinformatic
transformation of a given transcriptome annotation, transcriptome-wide design and
synthesis of ligamers, and customization of the SOLiD ligation chemistry. But it would
be extremely useful and informative for complete and routine transcriptome
quantification.
<a id="x1-94001r157"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">5.4 </span> <a id="x1-950004"></a>Final Thoughts</h3>
<!--l. 199--><p class="noindent">This thesis has introduced the complexity, purpose, potential, and challenges of
transcriptome study. There is no comparison between these issues with that of the
DNA genome. The next period of biomedical knowledge will be heralded by advances
                                                                                

                                                                                
in transcriptome analysis. This section discusses how scientists need to grow with
technology.
<a id="x1-95001r162"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.4.1 </span> <a id="x1-960001"></a>Science vs. Engineering</h4>
<div class="quote">
<!--l. 207--><p class="noindent"><span class="phvro8t-x-x-120">“There is  a  general  attitude  among  the  scientific  community  that</span>
<span class="phvro8t-x-x-120">science is superior to engineering.” — [</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMacilwain2010"><span class="phvro8t-x-x-120">Macilwain</span></a><span class="phvro8t-x-x-120">,</span><span class="phvro8t-x-x-120">&nbsp;</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XMacilwain2010"><span class="phvro8t-x-x-120">2010</span></a><span class="phvro8t-x-x-120">]</span>
</p><!--l. 209--><p class="noindent"><span class="phvro8t-x-x-120">“Science  is  about  what  is;  engineering  is  about  what  can  be.</span>
<span class="phvro8t-x-x-120">Engineers  are  dedicated  to  solving  problems  and  creating  new,</span>
<span class="phvro8t-x-x-120">useful, and efficient things.” — Neil Armstrong</span></p></div>
<!--l. 212--><p class="noindent">A common schism between technically-oriented individuals is whether or not they
identify themselves as an engineer or a scientist. The first quote, from an article
published in <span class="phvro8t-x-x-120">Nature</span>, communicates a clear bias in academic circles of the
importance of the <span class="phvro8t-x-x-120">why </span>over the <span class="phvro8t-x-x-120">how</span>. In essence, how one prioritizes these
questions categorizes individuals as scientists (why is important) or engineers
(how is more important). The second quote, from the first man to walk on the
Moon, Neil Armstrong, highlights what motivates a self described “engineer”
and “geek.” How does a graduate system—training PhDs for careers in life
science—educate individuals who fall into these two fundamentally different belief
                                                                                

                                                                                
systems?
</p><!--l. 214--><p class="noindent">When searching for a lab I told professors that I wanted to work on a technology
development project. A typical response was, “That’s not what we do here.” As
someone who is more interested in the “how” over the “why,” this began a brief period
when I thought I had made the wrong choice leaving industry and going back to
graduate school. What was the basis for this aversion to technology development?
The same article in <span class="phvro8t-x-x-120">Nature </span>states that this feeling toward engineering may be
attributed…
</p><div class="quote">


<!--l. 219--><p class="noindent"><span class="phvro8t-x-x-120">…partly to a “linear” model of innovation, which holds that scientific</span>
<span class="phvro8t-x-x-120">discovery  leads  to  technology,  which  in  turn  leads  to  human</span>
<span class="phvro8t-x-x-120">betterment.  This  model  is  as  firmly  entrenched  in  policy-makers’</span>
<span class="phvro8t-x-x-120">minds as  it  is  intellectually  discredited.  As  any  engineer  will  tell</span>
<span class="phvro8t-x-x-120">you, innovations, such as aviation and the steam engine, commonly</span>
<span class="phvro8t-x-x-120">precede scientific understanding of how things work.</span></p></div>
<!--l. 222--><p class="noindent">In fact, some of the most notable breakthrough scientific discoveries, including many
made by Nobel Laureates, demonstrate a clear integration of both the scientific
method and practical application. For example, the 2007 award in Physiology and
Medicine was given for “discoveries of principles for introducing specific gene
modifications in mice by the use of embryonic stem cells.” By combining
principle discoveries an indispensable technique in modern genetics was
                                                                                

                                                                                
created—gene targeting. The feeling of which is more important, the principle
discoveries or the application thereof, is likely what separates a scientist from an
engineer.
</p><!--l. 224--><p class="noindent">The importance of technology to the advancement of science in general is not limited
to anecdotes resulting in a Nobel prize. A quick scan of the most <a href="http://www.pnas.org/reports/most-cited">highly-cited</a> papers
in the <span class="phvro8t-x-x-120">PNAS </span>reveals that the top 13, indeed <span class="phvro8t-x-x-120">all </span>13, describe a novel methodology or
technique. Sequencing of DNA, microarray analysis, tetracycline-inducible promoters,
recombinant adenovirus, and site-specific mutagenesis are just a handful of
the tools on the list. This effect can also be seen in <a href="http://simplystatistics.org/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/">computation biology</a>,
where transformative algorithms, such as BLAT [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAltschul1990">Altschul et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XAltschul1990">1990</a>] and
Bowtie [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">Langmead et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLangmead2009">2009</a>] attain citations well beyond a typical paper
in their journal of publication, indeed far more than most primary research
articles.
</p><!--l. 226--><p class="noindent">The growth of big datasets is forcing all in biomedical research to think like an
Engineer. At least two major concerns require immediate attention: How to store the
data and how to analyze it?
<a id="x1-96001r170"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.4.2 </span> <a id="x1-970002"></a>The Data Deluge</h4>
<div class="quote">
                                                                                

                                                                                
<!--l. 234--><p class="noindent"><span class="phvro8t-x-x-120">“The HiSeq X Ten is sold as a set of 10 or more ultra-high throughput</span>
<span class="phvro8t-x-x-120">sequencing systems, each generating up to 1.8 terabases (Tb) of</span>
<span class="phvro8t-x-x-120">sequencing data in less than three days or up to 600 gigabases (Gb)</span>
<span class="phvro8t-x-x-120">per day, per system, providing the throughput to sequence tens of</span>
<span class="phvro8t-x-x-120">thousands of high-quality, high-coverage genomes per year.” </span><br class="newline"><span class="phvro8t-x-x-120">—</span><a href="http://bit.ly/PZpegZ"><span class="phvro8t-x-x-120">Illumina Press Release</span></a></p></div>
<!--l. 238--><p class="noindent">In a world where the HiSeq X, described above is a reality, biomedical researchers
need to change how they approach every aspect of data analysis including storage,
processing, and visualization. Evidence is mounting that replicates, not depth,
are essential in differential gene expression [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLiu2014">Liu et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLiu2014">2014</a>]. Replicates
compound problems of keeping similar data and file types separate and
tracked.
</p><!--l. 240--><p class="noindent">How do we work with all this data? Systems need to be in place to track the
necessary sample meta-data, analysis and modifications performed. Systems should
aid in eventual public posting and sharing of HTS data. Laboratory information
management systems (LIMs) and Electronic lab notebooks (ELNs) must be
implemented in academic labs participating in copious amounts of HTS data
generation and analysis.
</p><!--l. 242--><p class="noindent">Once these systems are in place, the ability to navigate, via genome browsers
[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobinson2011">Robinson et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRobinson2011">2011</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZweig2008">Zweig et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XZweig2008">2008</a>], will be of critical importance to allow
other members of the lab to reuse valuable datasets. Recent changes to the way
                                                                                

                                                                                
aligned genomic data is stored and added to UCSC genome browsers is a good
example of needed process improvements [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRaney2013">Raney et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XRaney2013">2013</a>]. Finally, efforts such
as the Galaxy project will define how most academic labs perform future “routine”
HTS analysis [<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlankenberg2010">Blankenberg et&nbsp;al.</a>,&nbsp;<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XBlankenberg2010">2010</a>].
<a id="x1-97001r171"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">5.4.3 </span> <a id="x1-980003"></a>Biologists need Computation Biological Skills</h4>
<!--l. 247--><p class="noindent">Just 10 years ago, graduate students and PhDs in the fields of Molecular Biology or
Biochemistry need not venture far from Excel or perhaps a statistical program with an
advanced graphical interface (e.g. Prism or Graphpad). Software knowledge that
stops at these tools and the rest of the Microsoft Office suite is no longer enough to
generate big strides in Biomedical research.
</p><!--l. 249--><p class="noindent">Working with tens of even hundreds of lines of data within a spreadsheet is
manageable. Computers from 20 years ago had more then enough computing power
to process these type of data. Data generated from most cutting edge projects can no
longer be analyzed in a spreadsheet program. Many students and post-docs find that
they are unable to analyze the data generated from months or years of tireless bench
work. Faced with learning what is effectively a collection of new languages and awash
in a sea of acronyms (e.g. LINUX, BASH, GNU, PERL, R) they reach out for
help from a “bioinformatics person.” Perhaps the relationship and interaction
                                                                                

                                                                                
with this personal is productive, leading to a collaboration and exciting new
knowledge. Sometimes it isn’t and the bench scientist shifts into one of three
modes: wait, find another bioinformatic-minded collaborator; or collect more
data.
</p>
<div class="table">
                                                                                

                                                                                
<!--l. 252--><p class="noindent"><a id="x1-98001r1"></a></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="caption"><span class="id">Table&nbsp;5.1: </span><span class="content">Changing computational tools for Molecular Biologists </span></div><!--tex4ht:label?: x1-98001r1 -->
<div class="tabular"><table id="TBL-9" class="tabular" cellspacing="0" cellpadding="0" rules="groups"><colgroup id="TBL-9-1g"><col id="TBL-9-1"></colgroup><colgroup id="TBL-9-2g"><col id="TBL-9-2"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-9-1-"><td style="white-space:wrap; text-align:left;" id="TBL-9-1-1" class="td11"><!--l. 4--><p class="noindent"><span class="phvb8t-">What is Used</span>                                       </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-1-2" class="td11"><!--l. 4--><p class="noindent"><span class="phvb8t-">What Should be Used</span>                                             </p></td></tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-2-"><td style="white-space:wrap; text-align:left;" id="TBL-9-2-1" class="td11"><!--l. 5--><p class="noindent"><span class="phvr8t-">Word</span> </p> </td><td style="white-space:wrap; text-align:left;" id="TBL-9-2-2" class="td11"><!--l. 5--><p class="noindent"><span class="phvr8t-">Plain Text</span></p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-3-"><td style="white-space:wrap; text-align:left;" id="TBL-9-3-1" class="td11"><!--l. 6--><p class="noindent"><span class="phvr8t-">Hand-inserted Citations</span>                         </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-3-2" class="td11"><!--l. 6--><p class="noindent"><span class="phvr8t-">Citation-management Software (e.g. Papers )</span>          </p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-4-"><td style="white-space:wrap; text-align:left;" id="TBL-9-4-1" class="td11"><!--l. 7--><p class="noindent"><span class="phvr8t-">Paper Notebooks</span>                                   </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-4-2" class="td11"><!--l. 7--><p class="noindent"><span class="phvr8t-">Evernote or Commercial ELNs</span>                                 </p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-5-"><td style="white-space:wrap; text-align:left;" id="TBL-9-5-1" class="td11"><!--l. 8--><p class="noindent"><span class="phvr8t-">Excel for Storage</span>                                   </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-5-2" class="td11"><!--l. 8--><p class="noindent"><span class="phvr8t-">Relational Databases (i.e. MySQL)</span>                           </p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-6-"><td style="white-space:wrap; text-align:left;" id="TBL-9-6-1" class="td11"><!--l. 9--><p class="noindent"><span class="phvr8t-">Excel for Analysis</span>                                  </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-6-2" class="td11"><!--l. 9--><p class="noindent"><span class="phvr8t-">R</span>                                                                               </p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-7-"><td style="white-space:wrap; text-align:left;" id="TBL-9-7-1" class="td11"><!--l. 10--><p class="noindent"><span class="phvr8t-">Local Code development &amp; backup</span>       </p></td><td style="white-space:wrap; text-align:left;" id="TBL-9-7-2" class="td11"><!--l. 10--><p class="noindent"><span class="phvr8t-">Online Code development (GitHub)</span>                          </p></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-9-8-"><td style="white-space:wrap; text-align:left;" id="TBL-9-8-1" class="td11">                                 </td></tr></tbody></table></div>
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
<!--l. 260--><p class="noindent">The “wait” mode is the most damaging, as it delays the progress of one’s work and
the advancement of science in general. Personally, I did not want to fall into this
mode. Once the multiplex study described in section <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-650002">4.2<!--tex4ht:ref: SeqZipMethod:sec:Multiplex Gene Study --></a> reached a point where I had
millions of sequencing reads but I could not find anyone to help me analyze the data, I
decided to educate myself on the basic principles of Linux, the command line, and
analysis of HTS data.
</p><!--l. 262--><p class="noindent">A biologically-train individual who posses the knowledge of analysis of HTS datasets
is an extremely powerful and empowering situation. This was recently communicated
in <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013">Plocik and Graveley</a>&nbsp;[<a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013">2013</a>]:
</p><div class="quote">
<!--l. 266--><p class="noindent"><span class="phvro8t-x-x-120">“Such exercises will empower students to explore and assess the</span>
<span class="phvro8t-x-x-120">quantitative data published in the manuscripts that they read, which</span>
<span class="phvro8t-x-x-120">can no longer be assessed at a glance like the qualitative gel-based</span>
<span class="phvro8t-x-x-120">results on which molecular biology was founded. Ultimately, it will be</span>
<span class="phvro8t-x-x-120">equally important to know how to write code as it is to pipette.” —</span>
<span class="phvro8t-x-x-120">[</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013"><span class="phvro8t-x-x-120">Plocik and Graveley</span></a><span class="phvro8t-x-x-120">,</span><span class="phvro8t-x-x-120">&nbsp;</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XPlocik2013"><span class="phvro8t-x-x-120">2013</span></a><span class="phvro8t-x-x-120">]</span>
</p>
</div>
<!--l. 270--><p class="noindent">The fact is that no one will care about a project as much as the student or post-doc
who generated the data. Learning and training of computational skills bent on
                                                                                

                                                                                
analyzing large datasets (see Table <a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-98001r1">5.1<!--tex4ht:ref: Disc:tab:Comp tools for Molecular Biology --></a>)should be central to the education in
Biomedical sciences in the future.
                                                                                

                                                                                
                                                                                

                                                                                
<a id="x1-98002r146"></a>
</p>
<h2 class="appendixHead"><span class="titlemark">Appendix&nbsp;A</span><br><a id="x1-99000A"></a>Appendix A: Misc Information</h2>
<a id="x1-99001r169"></a>
<h3 class="sectionHead"><span class="titlemark">A.1 </span> <a id="x1-1000001"></a>Buffers</h3>
<div class="table">
                                                                                

                                                                                
<!--l. 9--><p class="noindent"><a id="x1-100001r1"></a></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="tabular"><table id="TBL-10" class="tabular" cellspacing="0" cellpadding="0" rules="groups"><colgroup id="TBL-10-1g"><col id="TBL-10-1"></colgroup><colgroup id="TBL-10-2g"><col id="TBL-10-2"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-10-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-1-1" class="td11">Component</td><td style="white-space:nowrap; text-align:center;" id="TBL-10-1-2" class="td11">Concentration</td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-10-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-2-1" class="td11">  Tris-HCl   </td><td style="white-space:nowrap; text-align:center;" id="TBL-10-2-2" class="td11">   50 mM      </td></tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-10-3-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-3-1" class="td
11">  MgCl2     </td><td style="white-space:nowrap; text-align:center;" id="TBL-10-3-2" class="td11">    2 mM       </td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-10-4-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-4-1" class="td11"> DTT      </td><td style="white-space:nowrap; text-align:center;" id="TBL-10-4-2" class="td11">    1 mM       </td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-10-5-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-5-1" class="td11"> ATP       </td><td style="white-space:nowrap; text-align:center;" id="TBL-10-5-2" class="td11">   400 <span class="cmmi-12">&#956;</span>M     </td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-10-6-"><td style="white-space:nowrap; text-align:center;" id="TBL-10-6-1" class="td11"> pH       </td><td style="white-space:nowrap; text-align:center;" id="TBL-10-6-2" class="td11"> 7.5 @ 25 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span></td>
</tr></tbody></table></div>
<br><div class="caption"><span class="id">Table&nbsp;A.1: </span><span class="content">SeqZip Hybridization and Ligation Buffer </span></div><!--tex4ht:label?: x1-100001r1 -->
                                                                                

                                                                                
</div><hr class="endfloat">
</div>
<a id="x1-100002r175"></a>
<h3 class="sectionHead"><span class="titlemark">A.2 </span> <a id="x1-1010002"></a>Equations</h3>
<a id="x1-101001r172"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.1 </span> <a id="x1-1020001"></a>RNA concentration from radioactive in vitro transcription</h4>
<div class="eqnarray">
<center class="math-display">
<img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis62x.png" alt="()(                  )  (               )  (              )
&#956;M=pmol=cpmafterpurification×-dilutionfactor- ×  molUTPinoriginalreaction-×  ------1------- × 10-12
&#956;Lcpmbeforepurification×dilutionfactor      ReactionVolume       NumberUTPsintranscript
" class="math-display"></center>
</div>
<a id="x1-102002r178"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.2 </span> <a id="x1-1030002"></a>RNA concentration based on absorbance</h4>
<!--l. 56--><p class="noindent">
</p>
<center class="math-display">
<img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis63x.png" alt="          (                                              )
           ------------A260-×-Dilution-Factor-------------
[RNAin M ] =  10, 313 &lt; note 1 &gt; ×  nucleotides in message
" class="math-display"></center>
                                                                                

                                                                                
<!--l. 61--><p class="noindent">note 1: This value represents an average RNA extinction (<span class="cmmi-12">&#1013;</span>) coefficient value
<br class="newline">
<a id="x1-103001r179"></a>
</p>
<h4 class="subsectionHead"><span class="titlemark">A.2.3 </span> <a id="x1-1040003"></a>Normalize oxidized small RNA libraries size to time-matched unoxidized
library</h4>
<!--l. 65--><p class="noindent">NB: this equation assumes calibration against a specific time-point , in this case data
obtained from 6 week-old testes.
</p>
<div class="eqnarray">
<center class="math-display">
<img src="./RoyC_Umass_Thesis-2014-04-29_files/RoyC_Umass_Thesis64x.png" alt="        ( (  &#8721;                   )               )
                miRNA   reads &#964;
        ||   &#8721;--------------------- ×  depth  6wk ||
unox&#964;norm   =   || -----miRNA---reads--6wk-----------------||
  1     |(               1,000, 000               |)

                        (   &#8721;                                  )
                               oxidized  shared  &#8805; 23 nt reads
ox&#964;norm1  =   unox  &#964; norm1 × ( &#8721;------------------------------------)
                             unoxidized  shared  &#8805;  23 nt reads
" class="math-display"></center>
</div>
<a id="x1-104002r177"></a>
<h3 class="sectionHead"><span class="titlemark">A.3 </span> <a id="x1-1050003"></a>PCR Programs</h3>
<!--l. 80--><p class="noindent"><span class="phvb8t-x-x-120">Ligamer Hybridization </span>ROY-H2 | Ligamer Hybridization<br class="newline">Steps 1–9 are 10 minute incubations at the following temperatures:<br class="newline">69;66;63;58;54;52;50;48;46 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C</span><br class="newline">Step 10 is a 45 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>incubation for 1 hour<br class="newline">Steps 11–14 are 10 minute incubations at the following templates:<br class="newline">43;41;39;37 <sup><span class="cmsy-8">&#8728;</span> </sup> <span class="cmr-12">C</span><br class="newline">Final incubation is at 37 <sup><span class="cmsy-8">&#8728;</span></sup><span class="cmr-12">C </span>for <span class="cmsy-10x-x-120">&#8734;</span><br class="newline">
</p><!--l. 89--><p class="noindent"><span class="phvb8t-x-x-120">SeqZip ligation program </span>ROY-37-4 | T4 Rnl2 RNA-template DNA:DNA
ligation<br class="newline">1. 37 <sup><span class="cmsy-8">&#8728;</span> </sup> <span class="cmr-12">C </span>for 18 hours<br class="newline">2. 10 <sup><span class="cmsy-8">&#8728;</span> </sup> <span class="cmr-12">C </span>for <span class="cmsy-10x-x-120">&#8734;</span><br class="newline">
<a id="x1-105001r181"></a>
</p>
<h3 class="sectionHead"><span class="titlemark">A.4 </span> <a id="x1-1060004"></a>Intergenic and genic piRNA-producing Loci</h3>
<a id="x1-106001r2"></a><!--l. 100--><div class="longtable"><table id="TBL-11" class="longtable" cellspacing="0" cellpadding="0" rules="groups"><colgroup id="TBL-11-1g"><col id="TBL-11-1"></colgroup><colgroup id="TBL-11-2g"><col id="TBL-11-2"></colgroup>
<tbody><tr style="vertical-align:baseline;" id="TBL-11-1-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-1-1" class="td11">                </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-2-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-2-1" class="td11">                </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-3-"><td colspan="2" style="white-space:nowrap; text-align:center;" id="TBL-11-3-1" class="td11">                 <div class="multicolumn" style="white-space:nowrap; text-align:center;">
&nbsp;      <span class="phvrc8t-">T<span class="small-caps">A</span><span class="small-caps">B</span><span class="small-caps">L</span><span class="small-caps">E</span></span><span class="phvrc8t-">&nbsp;A.2: </span><span class="phvr8t-">Intergenic and genic subclassifications of piRNA-producing loci from </span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#x1-420003"><span class="phvr8t-">3</span><!--tex4ht:ref: MolCel --></a>
                                                         <span class="phvr8t-">and [</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e"><span class="phvr8t-">Li et</span><span class="phvr8t-">&nbsp;al.</span></a><span class="phvr8t-">,</span><span class="phvr8t-">&nbsp;</span><a href="file://localhost/Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.html#XLi2013e"><span class="phvr8t-">2013b</span></a><span class="phvr8t-">].</span>                                                  &nbsp;</div> <a id="x1-106002"></a>
</td></tr><tr style="vertical-align:baseline;" id="TBL-11-4-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-4-1" class="td11"> <span class="phvb8t-x-x-120">Genic Locus    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-11-4-2" class="td11"><span class="phvb8t-x-x-120">Intergenic Locus</span></td>
</tr><tr class="hline"><td><hr></td><td><hr></td></tr><tr style="vertical-align:baseline;" id="TBL-11-5-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-5-1" class="td11"> pi-Zbtb37       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-5-2" class="td11">  1-qC1.3-637    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-6-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-6-1" class="td11"> pi-Zbtb37       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-6-2" class="td11">  1-qC1.3-637    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-7-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-7-1" class="td11"> pi-Gm5878      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-7-2" class="td11">   1-qD-2017      </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-8-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-8-1" class="td11">pi-1700006A11Rik</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-8-2" class="td11">  10-qA3-2592    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-9-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-9-1" class="td11">pi-1700016M24Rik</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-9-2" class="td11">  10-qB4-6488    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-10-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-10-1" class="td11"> pi-Abl2          </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-10-2" class="td11">   10-qC1-875     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-11-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-11-1" class="td11"> pi-Acvr2b       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-11-2" class="td11">   10-qC2-545     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-12-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-12-1" class="td11"> pi-Ankrd11      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-12-2" class="td11">  11-qB1.3-590   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-13-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-13-1" class="td11"> pi-Arhgap20     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-13-2" class="td11">  11-qE1-3997    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-14-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-14-1" class="td11"> pi-Asb1         </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-14-2" class="td11">  12-qE-23911    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-15-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-15-1" class="td11"> pi-Atxn1l        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-15-2" class="td11">  13-qA3.1-213   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-16-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-16-1" class="td11"> pi-BC026590     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-16-2" class="td11">  13-qA3.1-355   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-17-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-17-1" class="td11"> pi-Bcl2l13       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-17-2" class="td11">   13-qA5-156     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-18-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-18-1" class="td11"> pi-Bend4        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-18-2" class="td11">   13-qA5-208     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-19-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-19-1" class="td11"> pi-Cbfa2t2       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-19-2" class="td11">   13-qA5-464     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-20-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-20-1" class="td11"> pi-Cbl          </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-20-2" class="td11">   13-qA5-703     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-21-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-21-1" class="td11"> pi-Cbx5         </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-21-2" class="td11">   13-qA5-967     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-22-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-22-1" class="td11"> pi-Ccdc117      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-22-2" class="td11">  13-qB1-1517    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-23-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-23-1" class="td11">        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-23-2" class="td11">
</td></tr>
<tr style="vertical-align:baseline;" id="TBL-11-24-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-24-1" class="td11"> pi-Ccrn4l        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-24-2" class="td11"> 14-qC1-1010  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-25-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-25-1" class="td11"> pi-Cdc42ep3     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-25-2" class="td11"> 15-qD1-4001  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-26-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-26-1" class="td11"> pi-Cmtm4       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-26-2" class="td11"> 15-qE1-3710   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-27-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-27-1" class="td11"> pi-Cramp1l      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-27-2" class="td11">17-qA3.3-26735</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-28-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-28-1" class="td11"> pi-Crkl          </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-28-2" class="td11"> 17-qA3.3-352  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-29-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-29-1" class="td11"> pi-Ctdsp2       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-29-2" class="td11">  17-qC-935    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-30-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-30-1" class="td11"> pi-D10Wsu102e  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-30-2" class="td11"> 19-qC2-1361  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-31-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-31-1" class="td11">pi-D730040F13Rik</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-31-2" class="td11"> 2-qE1-35981   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-32-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-32-1" class="td11"> pi-Dcaf7        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-32-2" class="td11">    2-qE5-4       </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-33-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-33-1" class="td11"> pi-Ddx19b       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-33-2" class="td11">  2-qF1-2536    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-34-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-34-1" class="td11"> pi-Dnmt3a       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-34-2" class="td11">  2-qG3-1029   </td>
                                                                                

                                                                                
</tr><tr style="vertical-align:baseline;" id="TBL-11-35-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-35-1" class="td11"> pi-Eif2c2        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-35-2" class="td11">  3-qA2-617     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-36-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-36-1" class="td11"> pi-Eif4ebp2      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-36-2" class="td11">  3-qA3-2052    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-37-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-37-1" class="td11"> pi-Elk4         </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-37-2" class="td11">  4-qB3-277     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-38-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-38-1" class="td11"> pi-Exog         </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-38-2" class="td11">  4-qB3-3994    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-39-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-39-1" class="td11"> pi-Eya3         </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-39-2" class="td11">  4-qB3-639     </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-40-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-40-1" class="td11"> pi-Fam168b     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-40-2" class="td11">  4-qD3-2082   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-41-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-41-1" class="td11"> pi-Fam53b      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-41-2" class="td11">  5-qF-14224    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-42-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-42-1" class="td11"> pi-Fbxl18        </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-42-2" class="td11">  5-qG2-2301   </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-43-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-43-1" class="td11"> pi-Fbxo41       </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-43-2" class="td11">  5-qG2-950    </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-44-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-44-1" class="td11">                </td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-11-45-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-45-1" class="td11"> pi-Foxo3 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-45-2" class="td11"> 6-qC3-100  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-46-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-46-1" class="td11"> pi-Fth1 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-46-2" class="td11"> 6-qC3-2394 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-47-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-47-1" class="td11">pi-Gabpb2</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-47-2" class="td11"> 6-qC3-6258 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-48-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-48-1" class="td11"> pi-Gan </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-48-2" class="td11"> 6-qD1-2831 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-49-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-49-1" class="td11"> pi-Gtf3c4 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-49-2" class="td11"> 6-qF3-1063 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-50-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-50-1" class="td11"> pi-Hic2 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-50-2" class="td11"> 6-qF3-3040 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-51-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-51-1" class="td11"> pi-Hif1an </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-51-2" class="td11"> 6-qF3-8009 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-52-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-52-1" class="td11"> pi-Hinfp </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-52-2" class="td11"> 7-qB5-6255 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-53-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-53-1" class="td11"> pi-Hjurp </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-53-2" class="td11">7-qD1-16444</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-54-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-54-1" class="td11">pi-Hmbox1</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-54-2" class="td11"> 7-qD1-654  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-55-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-55-1" class="td11"> pi-Igf2bp1 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-55-2" class="td11">7-qD2-11976</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-56-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-56-1" class="td11"> pi-Igsf9b </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-56-2" class="td11">7-qD2-24830</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-57-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-57-1" class="td11"> pi-Il17rd </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-57-2" class="td11"> 7-qF3-3125 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-58-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-58-1" class="td11"> pi-Ing5 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-58-2" class="td11"> 8-qC5-2209 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-59-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-59-1" class="td11"> pi-Ip6k1 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-59-2" class="td11"> 8-qE1-3748 </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-60-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-60-1" class="td11"> pi-Ipmk </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-60-2" class="td11"> 9-qA1-178  </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-61-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-61-1" class="td11"> pi-Kcng3 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-61-2" class="td11">     –         </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-62-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-62-1" class="td11"> pi-Kctd7 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-62-2" class="td11">     –         </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-63-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-63-1" class="td11"> pi-Klf13 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-63-2" class="td11">     –         </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-64-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-64-1" class="td11"> pi-Klhl11 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-64-2" class="td11">     –         </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-65-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-65-1" class="td11"> </td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-11-66-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-66-1" class="td11"> pi-Lpp     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-66-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-67-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-67-1" class="td11"> pi-Lsmd1  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-67-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-68-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-68-1" class="td11"> pi-Luzp1   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-68-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-69-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-69-1" class="td11"> pi-Mafg    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-69-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-70-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-70-1" class="td11"> pi-Map3k9 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-70-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-71-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-71-1" class="td11"> pi-March08 </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-71-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-72-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-72-1" class="td11"> pi-Mcart1  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-72-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-73-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-73-1" class="td11"> pi-Mgll     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-73-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-74-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-74-1" class="td11"> pi-Mlc1    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-74-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-75-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-75-1" class="td11"> pi-Mlec    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-75-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-76-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-76-1" class="td11"> pi-Mllt6    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-76-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-77-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-77-1" class="td11"> pi-Mrs2    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-77-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-78-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-78-1" class="td11"> pi-Ndst1   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-78-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-79-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-79-1" class="td11"> pi-Nr2c2   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-79-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-80-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-80-1" class="td11"> pi-Nsd1    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-80-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-81-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-81-1" class="td11"> pi-Phf20   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-81-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-82-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-82-1" class="td11"> pi-Pou6f1  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-82-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-83-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-83-1" class="td11"> pi-Ppm1f   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-83-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-84-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-84-1" class="td11">pi-Ppp1r12b</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-84-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-85-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-85-1" class="td11">pi-Ppp1r15b</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-85-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-86-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-86-1" class="td11">           </td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-11-87-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-87-1" class="td11">pi-Rab11fip4</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-87-2" class="td11">–</td>
                                                                                

                                                                                
</tr><tr style="vertical-align:baseline;" id="TBL-11-88-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-88-1" class="td11"> pi-Rad54l2  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-88-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-89-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-89-1" class="td11"> pi-Rc3h1   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-89-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-90-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-90-1" class="td11"> pi-Rcan3   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-90-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-91-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-91-1" class="td11"> pi-Rnf169  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-91-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-92-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-92-1" class="td11"> pi-Rplp1    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-92-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-93-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-93-1" class="td11"> pi-Shank3  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-93-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-94-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-94-1" class="td11"> pi-Slc43a2  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-94-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-95-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-95-1" class="td11"> pi-Smcr8   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-95-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-96-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-96-1" class="td11"> pi-Snx30   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-96-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-97-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-97-1" class="td11"> pi-Socs7   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-97-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-98-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-98-1" class="td11"> pi-Ssh1    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-98-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-99-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-99-1" class="td11"> pi-Strbp    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-99-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-100-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-100-1" class="td11"> pi-Tacc1    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-100-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-101-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-101-1" class="td11"> pi-Tbl2     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-101-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-102-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-102-1" class="td11"> pi-Tef      </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-102-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-103-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-103-1" class="td11"> pi-Tet3     </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-103-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-104-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-104-1" class="td11"> pi-Tfcp2l1  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-104-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-105-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-105-1" class="td11"> pi-Tktl2    </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-105-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-106-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-106-1" class="td11">pi-Tmem194</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-106-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-107-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-107-1" class="td11">           </td>
</tr>
<tr style="vertical-align:baseline;" id="TBL-11-108-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-108-1" class="td11"> pi-Trim71  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-108-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-109-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-109-1" class="td11"> pi-Uhmk1  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-109-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-110-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-110-1" class="td11">pi-Uhrf1bp1</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-110-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-111-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-111-1" class="td11"> pi-Wdfy3  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-111-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-112-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-112-1" class="td11"> pi-Wipf2   </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-112-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-113-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-113-1" class="td11"> pi-Zbtb49  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-113-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-114-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-114-1" class="td11">pi-Zdhhc23</td><td style="white-space:nowrap; text-align:center;" id="TBL-11-114-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-115-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-115-1" class="td11"> pi-Zfp280b </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-115-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-116-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-116-1" class="td11"> pi-Zfp346  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-116-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-117-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-117-1" class="td11"> pi-Zfp382  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-117-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-118-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-118-1" class="td11"> pi-Zfp652  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-118-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-119-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-119-1" class="td11"> pi-Zfp866  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-119-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-120-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-120-1" class="td11"> pi-Zmat3  </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-120-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-121-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-121-1" class="td11"> pi-Zyg11a </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-121-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-122-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-122-1" class="td11"> pi-Zyg11b </td><td style="white-space:nowrap; text-align:center;" id="TBL-11-122-2" class="td11">–</td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-123-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-123-1" class="td11">          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-124-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-124-1" class="td11">          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-125-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-125-1" class="td11">          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-126-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-126-1" class="td11">          </td>
</tr><tr style="vertical-align:baseline;" id="TBL-11-127-"><td style="white-space:nowrap; text-align:center;" id="TBL-11-127-1" class="td11">          </td>
</tr>
</tbody></table></div>
                                                                                

                                                                                
<a id="x1-106003r146"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&nbsp;B</span><br><a id="x1-107000B"></a>Appendix B: Automated Ligamer Assembly</h2>
<a id="x1-107001r182"></a>
<h3 class="sectionHead"><span class="titlemark">B.1 </span> <a id="x1-1080001"></a>Installation</h3>
<!--l. 6--><p class="noindent">Major Steps: </p>
<ul class="itemize1">
<li class="itemize">Create an input csv file with required information
</li>
<li class="itemize">Run this information sequentially through the scripts
</li>
<li class="itemize">Use the results to order oligos from IDT</li></ul>
<!--l. 13--><p class="noindent">Required Tools: </p>
<ul class="itemize1">
<li class="itemize">Perl
</li>
<li class="itemize">BioPerl
</li>
<li class="itemize">Ensembl Perl APIs
</li>
<li class="itemize">String::Random Perl Package</li></ul>
<!--l. 21--><p class="noindent">Items to future improvements </p>
<ul class="itemize1">
<li class="itemize">Use Ensembl Database to initilize queries
</li>
<li class="itemize">Make the use of BioPerl more flexible
</li>
<li class="itemize">Make more web-friedly</li></ul>
<!--l. 28--><p class="noindent">Helpful hints on installing BioPerl and Emsembl Perl APIs:
</p>
<!--l. 31-->
<div class="lstlisting" id="listing-3"><span class="label"><a id="x1-108001r1"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Install</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BioPerl</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">git</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108002r2"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cpan</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">App</span><span class="ectt-1000">::</span><span class="ectt-1000">cpanminus</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">First</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prep</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cpan</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108003r3"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cpanm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">DBI</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Install</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">necessary</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">DBI</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">perl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">module</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108004r4"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">mkdir</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/</span><span class="ectt-1000">src</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cd</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/</span><span class="ectt-1000">src</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108005r5"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">git</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">clone</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">git</span><span class="ectt-1000">://</span><span class="ectt-1000">github</span><span class="ectt-1000">.</span><span class="ectt-1000">com</span><span class="ectt-1000">/</span><span class="ectt-1000">bioperl</span><span class="ectt-1000">/</span><span class="ectt-1000">bioperl</span><span class="ectt-1000">-</span><span class="ectt-1000">live</span><span class="ectt-1000">.</span><span class="ectt-1000">git</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108006r6"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/.</span><span class="ectt-1000">bash_profile</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/.</span><span class="ectt-1000">bash_profile</span><span class="ectt-1000">.</span><span class="ectt-1000">bak</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108007r7"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">echo</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">e</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">=</span><span class="ectt-1000">$HOME</span><span class="ectt-1000">/</span><span class="ectt-1000">src</span><span class="ectt-1000">/</span><span class="ectt-1000">bioperl</span><span class="ectt-1000">-</span><span class="ectt-1000">live</span><span class="ectt-1000">:</span><span class="ectt-1000">$PERL5LIB</span><span class="ectt-1000">’</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&gt;&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/.</span><span class="ectt-1000">bash_profile</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108008r8"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">source</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/.</span><span class="ectt-1000">bash_profile</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108009r9"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Install</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">perl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">apis</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108010r10"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">mkdir</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/</span><span class="ectt-1000">src</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cd</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/</span><span class="ectt-1000">src</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108011r11"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">wget</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ftp</span><span class="ectt-1000">://</span><span class="ectt-1000">ftp</span><span class="ectt-1000">.</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">.</span><span class="ectt-1000">org</span><span class="ectt-1000">/</span><span class="ectt-1000">pub</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">-</span><span class="ectt-1000">api</span><span class="ectt-1000">.</span><span class="ectt-1000">tar</span><span class="ectt-1000">.</span><span class="ectt-1000">gz</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108012r12"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">tar</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">xvfz</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">-</span><span class="ectt-1000">api</span><span class="ectt-1000">.</span><span class="ectt-1000">tar</span><span class="ectt-1000">.</span><span class="ectt-1000">gz</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108013r13"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Add</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">locations</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">perlfile</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">libs</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$PATH</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108014r14"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">echo</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">e</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108015r15"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">=</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">}:</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">HOME</span><span class="ectt-1000">}/</span><span class="ectt-1000">src</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">/</span><span class="ectt-1000">modules</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108016r16"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">=</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">}:</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">HOME</span><span class="ectt-1000">}/</span><span class="ectt-1000">src</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">-</span><span class="ectt-1000">compara</span><span class="ectt-1000">/</span><span class="ectt-1000">modules</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108017r17"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">=</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">}:</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">HOME</span><span class="ectt-1000">}/</span><span class="ectt-1000">src</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">-</span><span class="ectt-1000">variation</span><span class="ectt-1000">/</span><span class="ectt-1000">modules</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108018r18"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">=</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">}:</span><span class="ectt-1000">$</span><span class="ectt-1000">{</span><span class="ectt-1000">HOME</span><span class="ectt-1000">}/</span><span class="ectt-1000">src</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">-</span><span class="ectt-1000">functgenomics</span><span class="ectt-1000">/</span><span class="ectt-1000">modules</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-108019r19"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">export</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PERL5LIB</span><span class="ectt-1000">’</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&gt;&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">~/.</span><span class="ectt-1000">bash_profile</span>

                                                                                

                                                                                
</div>
<a id="x1-108020r185"></a>
<h3 class="sectionHead"><span class="titlemark">B.2    </span> <a id="x1-1090002"></a>Example Input Format</h3>
                                                                                

                                                                                
                                                                                <!--l. 56--><p class="noindent">Here is an example input file to create ligamers investigating the <span class="phvro8t-x-x-120">Gria3 </span>gene in Rats:
</p>
<div class="table">
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 58--><p class="noindent"></p><hr class="float"><div class="float">
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <div class="tabular"> <table id="TBL-12" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-12-1g"><col id="TBL-12-1"><col id="TBL-12-2"><col id="TBL-12-3"><col id="TBL-12-4"><col id="TBL-12-5"><col id="TBL-12-6"><col id="TBL-12-7"><col id="TBL-12-8"><col id="TBL-12-9"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-12-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-1-1" class="td11"><span class="phvr8t-"># Comment lines are ignored                                     </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-12-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-2-1" class="td11"><span class="phvr8t-">#Gene name                                                               </span></td></tr><tr style="vertical-align:baseline;" id="TBL-12-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-3-1" class="td
11"><span class="phvr8t-">&nbsp;GRIA3                                                                       </span></td></tr><tr style="vertical-align:baseline;" id="TBL-12-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-4-1" class="td
11"><span class="phvr8t-"># PCR primers used - Solexa PE adaptor sequences</span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-12-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-5-1" class="td11"><span class="phvr8t-"># Five prime                                                                </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-12-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-6-1" class="td11"><span class="phvr8t-">PCR-Primer-5’-ATCTGAGCGGGCTGGCAAGGC      </span></td></tr><tr style="vertical-align:baseline;" id="TBL-12-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-7-1" class="td
11"><span class="phvr8t-">#Three Prime                                                              </span></td></tr><tr style="vertical-align:baseline;" id="TBL-12-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-8-1" class="td
11"><span class="phvr8t-">PCR-Primer-3’-GCCTCCCTCGCGCCATCAGA         </span></td></tr><tr style="vertical-align:baseline;" id="TBL-12-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-12-9-1" class="td11"></td></tr></tbody></table></div>
                                                                                                                   
                                                                                

                                                                                
</div><hr class="endfloat">
<div class="table">
                                                                                

                                                                                
<!--l. 63--><p class="noindent"></p><hr class="float"><div class="float">
                                                                                

                                                                                
<div class="tabular"><table id="TBL-13" class="tabular" cellspacing="0" cellpadding="0"><colgroup id="TBL-13-1g"><col id="TBL-13-1"><col id="TBL-13-2"><col id="TBL-13-3"><col id="TBL-13-4"><col id="TBL-13-5"><col id="TBL-13-6"><col id="TBL-13-7"><col id="TBL-13-8"><col id="TBL-13-9"></colgroup><tbody><tr style="vertical-align:baseline;" id="TBL-13-1-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-1-1" class="td11"><span class="phvr8t-">ExonId                                   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-2" class="td11"><span class="phvr8t-">LigID</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-3" class="td11"><span class="phvr8t-">Name</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-4" class="td11"><span class="phvr8t-">Strand</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-5" class="td11"><span class="phvr8t-">Code</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-6" class="td11"><span class="phvr8t-">TargetPrime</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-7" class="td11">        <span class="phvr8t-">bedLoc              </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-8" class="td11">   <span class="phvr8t-">SetID      </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-1-9" class="td11">  <span class="phvr8t-">ConstID    </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-2-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-2-1" class="td11"><span class="phvr8t-">&lt;Gria3_201/202-Shared-I14  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-2" class="td11"> <span class="phvr8t-">10  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-5" class="td11"> <span class="phvr8t-">TC  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-6" class="td11">     <span class="phvr8t-">5         </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-7" class="td11"><span class="phvr8t-">X:127903250-127903350</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-8" class="td11"><span class="phvr8t-">NANNNNNN</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-2-9" class="td11"><span class="phvr8t-">201_2_intron</span></td></tr><tr style="vertical-align:baseline;" id="TBL-13-3-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-3-1" class="td
11"><span class="phvr8t-">&lt;Gria33_201/202-Shared-I14</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-2" class="td11">  <span class="phvr8t-">9   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-5" class="td11">  <span class="phvr8t-">T   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-6" class="td11">     <span class="phvr8t-">3         </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-7" class="td11"><span class="phvr8t-">X:127903210-127903249</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-8" class="td11"><span class="phvr8t-">NNNNNNNN</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-3-9" class="td11"><span class="phvr8t-">201_2_intron</span></td></tr><tr style="vertical-align:baseline;" id="TBL-13-4-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-4-1" class="td
11"><span class="phvr8t-">&lt;Gria33_201/202-E15           </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-2" class="td11">  <span class="phvr8t-">8   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-5" class="td11"> <span class="phvr8t-">TC  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-6" class="td11">     <span class="phvr8t-">5         </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-7" class="td11"><span class="phvr8t-">X:127914822-127915069</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-8" class="td11"> <span class="phvr8t-">NNNNNNN </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-4-9" class="td11">  <span class="phvr8t-">201,202    </span></td></tr><tr style="vertical-align:baseline;" id="TBL-13-5-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-5-1" class="td
11"><span class="phvr8t-">&lt;Gria33_202-I14:15               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-2" class="td11">  <span class="phvr8t-">7   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-7" class="td11"><span class="phvr8t-">X:127912345-127914821</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-8" class="td11"> <span class="phvr8t-">TACACAT   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-5-9" class="td11">    <span class="phvr8t-">202       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-6-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-6-1" class="td11"><span class="phvr8t-">&lt;Gria33_202-E14                  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-2" class="td11">  <span class="phvr8t-">6   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-7" class="td11"><span class="phvr8t-">X:127912230-127912344</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-8" class="td11"> <span class="phvr8t-">ACCCCAG  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-6-9" class="td11">    <span class="phvr8t-">202       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-7-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-7-1" class="td11"><span class="phvr8t-">&lt;Gria33_201-I14:15               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-2" class="td11">  <span class="phvr8t-">5   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-7" class="td11"><span class="phvr8t-">X:127897499-127914821</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-8" class="td11"> <span class="phvr8t-">CGCGCAC </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-7-9" class="td11">    <span class="phvr8t-">201       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-8-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-8-1" class="td11"><span class="phvr8t-">&lt;Gria33_201-E14                  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-2" class="td11">  <span class="phvr8t-">4   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-7" class="td11"><span class="phvr8t-">X:127897384-127897498</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-8" class="td11"> <span class="phvr8t-">GTCTCAA  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-8-9" class="td11">    <span class="phvr8t-">201       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-9-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-9-1" class="td11"><span class="phvr8t-">&lt;Gria33_202-I13:14               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-2" class="td11">  <span class="phvr8t-">3   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-7" class="td11"><span class="phvr8t-">X:127896828-127912229</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-8" class="td11"> <span class="phvr8t-">ACCGATT  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-9-9" class="td11">    <span class="phvr8t-">202       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-10-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-10-1" class="td11"><span class="phvr8t-">&lt;Gria33_201-I13:14               </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-2" class="td11">  <span class="phvr8t-">2   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-5" class="td11">  <span class="phvr8t-">I    </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-6" class="td11">    <span class="phvr8t-">N        </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-7" class="td11"><span class="phvr8t-">X:127896828-127897383</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-8" class="td11"> <span class="phvr8t-">CGCTATG  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-10-9" class="td11">    <span class="phvr8t-">201       </span></td>
</tr><tr style="vertical-align:baseline;" id="TBL-13-11-"><td style="white-space:nowrap; text-align:left;" id="TBL-13-11-1" class="td11"><span class="phvr8t-">&lt;Gria33_201/202-E13           </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-2" class="td11">  <span class="phvr8t-">1   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-3" class="td11"> <span class="phvr8t-">rn4  </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-4" class="td11"> <span class="phvr8t-">minus </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-5" class="td11">  <span class="phvr8t-">T   </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-6" class="td11">     <span class="phvr8t-">3         </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-7" class="td11"><span class="phvr8t-">X:127896580-127896827</span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-8" class="td11"> <span class="phvr8t-">NNNNNNN </span></td><td style="white-space:nowrap; text-align:center;" id="TBL-13-11-9" class="td11">  <span class="phvr8t-">201,202    </span></td></tr></tbody></table></div>
                                                                                

                                                                                
</div><hr class="endfloat">
                                                                                

                                                                                
<a id="x1-109001r186"></a>
<h3 class="sectionHead"><span class="titlemark">B.3 </span> <a id="x1-1100003"></a>Ligamer Assembler Source Code</h3>
<!--l. 73--><div class="lstinputlisting">
<a id="x1-110001"></a>
<span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110002r1"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">!</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/</span><span class="ectt-1000">usr</span><span class="ectt-1000">/</span><span class="ectt-1000">bin</span><span class="ectt-1000">/</span><span class="ectt-1000">perl</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110003r2"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110004r3"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">Pre</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">requisites</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110005r4"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">These</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">are</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">on</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">02/19/13</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110006r5"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">lib</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">/</span><span class="ectt-1000">home</span><span class="ectt-1000">/</span><span class="ectt-1000">royc</span><span class="ectt-1000">/</span><span class="ectt-1000">perl5</span><span class="ectt-1000">/</span><span class="ectt-1000">lib</span><span class="ectt-1000">/</span><span class="ectt-1000">perl5</span><span class="ectt-1000">/</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BioPerl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">location</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110007r6"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">lib</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"/</span><span class="ectt-1000">home</span><span class="ectt-1000">/</span><span class="ectt-1000">royc</span><span class="ectt-1000">/</span><span class="ectt-1000">lib</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">.</span><span class="ectt-1000">perl</span><span class="ectt-1000">.</span><span class="ectt-1000">zpi</span><span class="ectt-1000">/</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">/</span><span class="ectt-1000">modules</span><span class="ectt-1000">";</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">packages</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110008r7"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110009r8"></a></span><span class="ectt-1000">=</span><span class="ectt-1000">head1</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Assembler</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110010r9"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110011r10"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">This</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">script</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">will</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">automatically</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">create</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamers</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110012r11"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110013r12"></a></span><span class="ectt-1000">=</span><span class="ectt-1000">head2</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Contact</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110014r13"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110015r14"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Script</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">made</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">by</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Christian</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Roy</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Umass</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Medical</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">School</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110016r15"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">christian</span><span class="ectt-1000">.</span><span class="ectt-1000">roy@umassmed</span><span class="ectt-1000">.</span><span class="ectt-1000">edu</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110017r16"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110018r17"></a></span><span class="ectt-1000">=</span><span class="ectt-1000">cut</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110019r18"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110020r19"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strict</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">To</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">help</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">wtih</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110021r20"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">warnings</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">To</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">help</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">me</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">catch</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">mistakes</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110022r21"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110023r22"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">EnsEMBL</span><span class="ectt-1000">::</span><span class="ectt-1000">Registry</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">To</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">load</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">remote</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">EnsEMBL</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Registry</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110024r23"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">EnsEMBL</span><span class="ectt-1000">::</span><span class="ectt-1000">Slice</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">To</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">retreave</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequences</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">EnsEMBL</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">registry</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110025r24"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">DB</span><span class="ectt-1000">::</span><span class="ectt-1000">Fasta</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BioPerl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">tool</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">retreave</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequnce</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">local</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">FastA</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110026r25"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">SeqFeature</span><span class="ectt-1000">::</span><span class="ectt-1000">Primer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BioPerl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tool</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">normalization</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110027r26"></a></span><span class="ectt-1000">use</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Cwd</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">To</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">retreave</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">current</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">directory</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110028r27"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110029r28"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$dir</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">getcwd</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Assign</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">current</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">directory</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">scalar</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110030r29"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$timestamp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">localtime</span><span class="ectt-1000">()</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">time</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">at</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">script</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">start</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110031r30"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110032r31"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Variables</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110033r32"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110034r33"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$file_input</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">specified</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110035r34"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$output_file</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">too</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110036r35"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">species</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ensembl</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110037r36"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequences</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110038r37"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110039r38"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line_counter</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Keep</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">track</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">stepping</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">through</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110040r39"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@arguments</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Keep</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">track</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">arguments</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110041r40"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$fa_reference</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">using</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">a</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">local</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">FASTA</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Reference</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110042r41"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Obvious</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110043r42"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$coordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Interim</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">splitting</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">UCSC</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110044r43"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obvious</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110045r44"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Obvious</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110046r45"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">target</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">gene</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">name</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110047r46"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110048r47"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Broad</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">define</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">type</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">see</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">man</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110049r48"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Obvious</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110050r49"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">appropriate</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PCR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">terminal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">oligos</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110051r50"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">will</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">between</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">regions</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">comp</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110052r51"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$note_line</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">notes</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">a</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">query</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110053r52"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">three</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PCR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110054r53"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fill</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">five</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PCR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110055r54"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Internal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">varialbe</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">assembling</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamers</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">see</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">man</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110056r55"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$set</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Move</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">set</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">assembly</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110057r56"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110058r57"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110059r58"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">Variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Defaults</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110060r59"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$verbose</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Verbose</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">databases</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110061r60"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">=62;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Default</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">database</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">database</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110062r61"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">58</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Defalt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">normalization</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110063r62"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">0.05</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Default</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">concetration</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">calculation</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">M</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110064r63"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">0.00000025</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Defeult</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">conc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">calc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">M</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110065r64"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$man_print</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printing</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">manual</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110066r65"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$help_print</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">For</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printing</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">help</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">informatio</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HTML</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110067r66"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Internal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variable</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequental</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">numbering</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamers</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110068r67"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$remote</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">set</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">database</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110069r68"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">=20;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Default</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">nt</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110070r69"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$plname</span><span class="ectt-1000">=</span><span class="ectt-1000">$0</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">assign</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$plname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">scalar</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">script</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">help</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printing</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110071r70"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110072r71"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">Print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Usage</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">nothing</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">is</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">entered</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">at</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">commandline</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110073r72"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">==0)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">system</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">pod2text</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$0</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">|</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">less</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">die</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110074r73"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110075r74"></a></span><span class="ectt-1000">=</span><span class="ectt-1000">head2</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Usage</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110076r75"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110077r76"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110078r77"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">hp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HTML</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">scriptname</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110079r78"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">mp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">and</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">view</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Manual</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">scriptname</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110080r79"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">i</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">File</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">File</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Input</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110081r80"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">o</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">File</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">File</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110082r81"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">v</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Verbose</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110083r82"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">d</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data_base</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110084r83"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">t</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">degrees</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">celcius</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110085r84"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">concentration</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">mM</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110086r85"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">lig_conc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">]</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">concentration</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">nM</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110087r86"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">c</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">#</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Minimum</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Control</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamers</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">default</span><span class="ectt-1000">=20)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110088r87"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110089r88"></a></span><span class="ectt-1000">=</span><span class="ectt-1000">cut</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110090r89"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Finish</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">message</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">run</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">no</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">arguments</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110091r90"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110092r91"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">Parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">command</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110093r92"></a></span><span class="ectt-1000">while</span><span class="ectt-1000">(</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">&gt;0)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110094r93"></a></span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110095r94"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@arguments</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Store</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">command</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printing</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">later</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110096r95"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110097r96"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">(</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110098r97"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110099r98"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">hp</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HTML</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Data</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110100r99"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$help_print</span><span class="ectt-1000">=1;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110101r100"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110102r101"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">mp</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">a</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">manual</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110103r102"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$man_print</span><span class="ectt-1000">=1;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110104r103"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110105r104"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">i</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">What</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">is</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110106r105"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$file_input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110107r106"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110108r107"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">f</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">n</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">fasta</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">your</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequences</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">are</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110109r108"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$fa_reference</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110110r109"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110111r110"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">r</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">fetch</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequences</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110112r111"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$remote</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110113r112"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110114r113"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">o</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110115r114"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$output_file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">(</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110116r115"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110117r116"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">v</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">see</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">load</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110118r117"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$verbose</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110119r118"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110120r119"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">d</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">What</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">use</span><span class="ectt-1000">?#</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110121r120"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110122r121"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110123r122"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">What</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">temperature</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">degrees</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">C</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">want</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">norm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110124r123"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110125r124"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110126r125"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">salt</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">concentration</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">calculations</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110127r126"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110128r127"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1000;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">micro</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Molar</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Molar</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110129r128"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110130r129"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">lig_conc</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Concentration</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">calculations</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110131r130"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">=</span><span class="ectt-1000">shift@ARGV</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110132r131"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1000000000;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">nM</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">M</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110133r132"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110134r133"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$next_arg</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">-</span><span class="ectt-1000">c</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">What</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">would</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">you</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">like</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">min</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligs</span><span class="ectt-1000">?</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110135r134"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@ARGV</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110136r135"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">-1</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110137r136"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110138r137"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Finish</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Parsing</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">command</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110139r138"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110140r139"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####################</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HELP</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">SUBROUTINE</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">CALLS</span><span class="ectt-1000">################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110141r140"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">=</span><span class="ectt-1000">$0</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110142r141"></a></span><span class="ectt-1000">podhelp</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$help_print</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$man_print</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$dir</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110143r142"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">####################</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HTML</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">CALLS</span><span class="ectt-1000">#################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110144r143"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110145r144"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###################</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">open</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">registry</span><span class="ectt-1000">#################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110146r145"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110147r146"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$fa_reference</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110148r147"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">DB</span><span class="ectt-1000">::</span><span class="ectt-1000">Fasta</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">new</span><span class="ectt-1000">(</span><span class="ectt-1000">$fa_reference</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110149r148"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110150r149"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110151r150"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$remote</span><span class="ectt-1000">==1)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110152r151"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl_database</span><span class="ectt-1000">(</span><span class="ectt-1000">$verbose</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110153r152"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110154r153"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110155r154"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">open</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110156r155"></a></span><span class="ectt-1000">open</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">OUT</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">’</span><span class="ectt-1000">.</span><span class="ectt-1000">$output_file</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">||</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">die</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">could</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">not</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">be</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">created</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110157r156"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">headers</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110158r157"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110159r158"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Start</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">general</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">assembler</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110160r159"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Source</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Program</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$dir</span><span class="ectt-1000">,</span><span class="ectt-1000">$0</span><span class="ectt-1000">,</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110161r160"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Date</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Run</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t$timestamp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110162r161"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Arguments</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">entered</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">@arguments</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110163r162"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">filename</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t$file_input</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110164r163"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">filename</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t$output_file</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110165r164"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Control</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t$control_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">plus</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110166r165"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Normalization</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">temperature</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">t$temp</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110167r166"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">####</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">now</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">all</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">on</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligmaer</span><span class="ectt-1000">-</span><span class="ectt-1000">specific</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">information</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110168r167"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">Gene</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">1</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110169r168"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Ligamer_Number</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">2</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110170r169"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Species</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110171r170"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Strand</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">4</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110172r171"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Joiner</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Code</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">5</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110173r172"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Target</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">6</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110174r173"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">UCSC</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">coordinates</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">7</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110175r174"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">PCR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Used</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">8</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110176r175"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Used</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">9</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110177r176"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Total</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Query</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">span</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">10</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110178r177"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Five</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Sequence</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">11</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110179r178"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Length</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">12</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110180r179"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Five</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110181r180"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Sequence</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110182r181"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Length</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110183r182"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110184r183"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">"</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Identifier</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">".</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110185r184"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Sequence</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110186r185"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Length</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110187r186"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Notes</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110188r187"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">Set</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110189r188"></a></span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110190r189"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110191r190"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">open</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110192r191"></a></span><span class="ectt-1000">open</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">INPUT</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$file_input</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">||</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">die</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$file_input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">couldn</span><span class="ectt-1000">’</span><span class="ectt-1000">t</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">be</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">opened</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110193r192"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">##############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110194r193"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110195r194"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">################</span><span class="ectt-1000">read</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">and</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">analyze</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">each</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110196r195"></a></span><span class="ectt-1000">while</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">=&lt;</span><span class="ectt-1000">INPUT</span><span class="ectt-1000">&gt;)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">starting</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">brace</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">read</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">through</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">csv</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110197r196"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110198r197"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^</span><span class="ectt-1000">#</span><span class="ectt-1000">/)</span><span class="ectt-1000">{</span><span class="ectt-1000">next</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">skips</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">comments</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110199r198"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^&gt;/)</span><span class="ectt-1000">{</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">next</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">skips</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">and</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">trans</span><span class="ectt-1000">.</span><span class="ectt-1000">these</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">lines</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110200r199"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^~/)</span><span class="ectt-1000">{</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/~//;</span><span class="ectt-1000">chomp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">=</span><span class="ectt-1000">$line</span><span class="ectt-1000">;}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">find</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">gene</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">identifier</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110201r200"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/[\</span><span class="ectt-1000">s</span><span class="ectt-1000">]+//</span><span class="ectt-1000">g</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110202r201"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^\</span><span class="ectt-1000">@</span><span class="ectt-1000">/)</span><span class="ectt-1000">{</span><span class="ectt-1000">chomp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">$note_line</span><span class="ectt-1000">=</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">next</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">store</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">notes</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110203r202"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110204r203"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">chomp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110205r204"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110206r205"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^</span><span class="ectt-1000">PCR</span><span class="ectt-1000">-</span><span class="ectt-1000">Primer</span><span class="ectt-1000">-5</span><span class="ectt-1000">’</span><span class="ectt-1000">-/</span><span class="ectt-1000">g</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Find</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">adaptor</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110207r206"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110208r207"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/</span><span class="ectt-1000">PCR</span><span class="ectt-1000">-</span><span class="ectt-1000">Primer</span><span class="ectt-1000">-5</span><span class="ectt-1000">’</span><span class="ectt-1000">-//;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110209r208"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/[\</span><span class="ectt-1000">s</span><span class="ectt-1000">]+//</span><span class="ectt-1000">g</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110210r209"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;5</span><span class="ectt-1000">_pcr</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">.</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110211r210"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">next</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110212r211"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110213r212"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110214r213"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^</span><span class="ectt-1000">PCR</span><span class="ectt-1000">-</span><span class="ectt-1000">Primer</span><span class="ectt-1000">-3</span><span class="ectt-1000">’</span><span class="ectt-1000">-/)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{##</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">find</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">adaptor</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110215r214"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$line</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110216r215"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/</span><span class="ectt-1000">PCR</span><span class="ectt-1000">-</span><span class="ectt-1000">Primer</span><span class="ectt-1000">-3</span><span class="ectt-1000">’</span><span class="ectt-1000">-//;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110217r216"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/[\</span><span class="ectt-1000">s</span><span class="ectt-1000">]+//</span><span class="ectt-1000">g</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110218r217"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">&gt;3</span><span class="ectt-1000">_pcr</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">.</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110219r218"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">next</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110220r219"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110221r220"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110222r221"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">=~/^&lt;/)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">#</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">query</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">lines</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">start</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">a</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’&lt;’</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110223r222"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">unless</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$note_line</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$note_line</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">;}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110224r223"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110225r224"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110226r225"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110227r226"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110228r227"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line_counter</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110229r228"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110230r229"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110231r230"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110232r231"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110233r232"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110234r233"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110235r234"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110236r235"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110237r236"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110238r237"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$set</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110239r238"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse_the_line</span><span class="ectt-1000">(</span><span class="ectt-1000">$line</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110240r239"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110241r240"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">uc</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110242r241"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110243r242"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">query</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110244r243"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">)</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse_coordinates</span><span class="ectt-1000">(</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110245r244"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110246r245"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110247r246"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$end</span><span class="ectt-1000">-</span><span class="ectt-1000">$start</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110248r247"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110249r248"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###########</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">registry</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110250r249"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$remote</span><span class="ectt-1000">==1)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110251r250"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110252r251"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get_genomic_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110253r252"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110254r253"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110255r254"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110256r255"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110257r256"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110258r257"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110259r258"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110260r259"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110261r260"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">####################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110262r261"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110263r262"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###########</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">local</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Fasta</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110264r263"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$fa_reference</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110265r264"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">$chr</span><span class="ectt-1000">="</span><span class="ectt-1000">chr</span><span class="ectt-1000">".</span><span class="ectt-1000">$chr</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110266r265"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$obj</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get_Seq_by_id</span><span class="ectt-1000">(</span><span class="ectt-1000">$chr</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110267r266"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$obj</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">subseq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$start</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110268r267"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110269r268"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110270r269"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110271r270"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">correct</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">orientation</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110272r271"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110273r272"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">revcom_slice_based_on_strand</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110274r273"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110275r274"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110276r275"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110277r276"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110278r277"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110279r278"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">end</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110280r279"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110281r280"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T5_tm_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110282r281"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110283r282"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110284r283"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110285r284"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110286r285"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110287r286"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110288r287"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110289r288"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">end</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110290r289"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110291r290"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T3_tm_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110292r291"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110293r292"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110294r293"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110295r294"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110296r295"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110297r296"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110298r297"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110299r298"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110300r299"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110301r300"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">start</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">build</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">your</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HASH</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110302r301"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110303r302"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110304r303"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110305r304"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110306r305"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">UCSCcoordinates</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110307r306"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">UCSC_chr</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110308r307"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">UCSC_start</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110309r308"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">UCSC_end</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110310r309"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">gene</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110311r310"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110312r311"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">species</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110313r312"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110314r313"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110315r314"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">five_prime_PCR_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$five_prime_PCR_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110316r315"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">three_prime_PCR_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$three_prime_PCR_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110317r316"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110318r317"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">target_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_seq_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110319r318"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">seed</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110320r319"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110321r320"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110322r321"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110323r322"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110324r323"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110325r324"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110326r325"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">notes</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$note_line</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110327r326"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">set</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$set</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110328r327"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110329r328"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110330r329"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">5</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Terminal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">targeted</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110331r330"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Advance</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">number</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110332r331"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110333r332"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110334r333"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Add</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">hash</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">table</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110335r334"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-5</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110336r335"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110337r336"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_5</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110338r337"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110339r338"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110340r339"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110341r340"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110342r341"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110343r342"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">TC</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">5</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">internal</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110344r343"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110345r344"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110346r345"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-5-</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110347r346"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110348r347"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_5</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110349r348"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110350r349"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final_internal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110351r350"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110352r351"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110353r352"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110354r353"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_results</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110355r354"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110356r355"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110357r356"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Now</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">inside</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110358r357"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110359r358"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_ctrl_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110360r359"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_ctrl_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_ctrl_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110361r360"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">substr</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">$T5_ctrl_length</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110362r361"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">IC</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110363r362"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110364r363"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T5_tm_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110365r364"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110366r365"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110367r366"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110368r367"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110369r368"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110370r369"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110371r370"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110372r371"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110373r372"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110374r373"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110375r374"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_tm</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110376r375"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110377r376"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110378r377"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110379r378"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-5-</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110380r379"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110381r380"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110382r381"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110383r382"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final_internal</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110384r383"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110385r384"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110386r385"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110387r386"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">3</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110388r387"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110389r388"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110390r389"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-3</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110391r390"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110392r391"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110393r392"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110394r393"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110395r394"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110396r395"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110397r396"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110398r397"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">TC</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">3</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110399r398"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">Grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110400r399"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110401r400"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110402r401"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-3-</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110403r402"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110404r403"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110405r404"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final_internal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110406r405"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Grab</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">internal</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">control</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110407r406"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110408r407"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_ctrl_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110409r408"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_ctrl_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_ctrl_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110410r409"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110411r410"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">substr</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,0,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_ctrl_length</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110412r411"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">IC</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110413r412"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110414r413"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T3_tm_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110415r414"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110416r415"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110417r416"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110418r417"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110419r418"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110420r419"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110421r420"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110422r421"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110423r422"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_tm</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110424r423"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110425r424"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">bed_start</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110426r425"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">bed_end</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110427r426"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110428r427"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110429r428"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-3-</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110430r429"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110431r430"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110432r431"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110433r432"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110434r433"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final_internal</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110435r434"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110436r435"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110437r436"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_seq_length</span><span class="ectt-1000">&gt;60)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110438r437"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110439r438"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110440r439"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110441r440"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">C</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110442r441"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">L</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110443r442"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110444r443"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110445r444"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110446r445"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$final</span><span class="ectt-1000">{</span><span class="ectt-1000">pcrsequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110447r446"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110448r447"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110449r448"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110450r449"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110451r450"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">N</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110452r451"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">L</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110453r452"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110454r453"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110455r454"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110456r455"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$final</span><span class="ectt-1000">{</span><span class="ectt-1000">pcrsequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110457r456"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110458r457"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110459r458"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prep_bed</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110460r459"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110461r460"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110462r461"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110463r462"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&amp;&amp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_seq_length</span><span class="ectt-1000">&lt;=60)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110464r463"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110465r464"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110466r465"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">S</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110467r466"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$common</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110468r467"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_short_interal_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">common</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110469r468"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">lig_results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110470r469"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$final</span><span class="ectt-1000">{</span><span class="ectt-1000">pcrsequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110471r470"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">final</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110472r471"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">final</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110473r472"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">bed_final</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">prep_bed</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">bed_output</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110474r473"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110475r474"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110476r475"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">matching</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">brace</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">lines</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110477r476"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110478r477"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">else</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">next</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110479r478"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Matching</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">brace</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">csv</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">test</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110480r479"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">######</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">LIGAMERS</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ASSEMBLY</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PORTION</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110481r480"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110482r481"></a></span><span class="ectt-1000">close</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110483r482"></a></span><span class="ectt-1000">close</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">INPUT</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110484r483"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Program</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Finished</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110485r484"></a></span><span class="ectt-1000">exit</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110486r485"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110487r486"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">######</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">MAJOR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">WORK</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OF</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PROGRAM</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">!!</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110488r487"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">##############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110489r488"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Begin</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">section</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">program</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110490r489"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110491r490"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110492r491"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110493r492"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110494r493"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110495r494"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110496r495"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110497r496"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110498r497"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110499r498"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_tm</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110500r499"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110501r500"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110502r501"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110503r502"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110504r503"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110505r504"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110506r505"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110507r506"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Terminal_3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110508r507"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110509r508"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110510r509"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110511r510"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110512r511"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110513r512"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110514r513"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110515r514"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_tm</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110516r515"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110517r516"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110518r517"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110519r518"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110520r519"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110521r520"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110522r521"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">for</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">short</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slices</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#############################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110523r522"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_short_interal_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110524r523"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110525r524"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110526r525"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110527r526"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110528r527"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence_tm_obj</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110529r528"></a></span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">SeqFeature</span><span class="ectt-1000">::</span><span class="ectt-1000">Primer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">new</span><span class="ectt-1000">(-</span><span class="ectt-1000">seq</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110530r529"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence_tm_obj</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110531r530"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110532r531"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110533r532"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">salt</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110534r533"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">oligo</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110535r534"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110536r535"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110537r536"></a></span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">=</span><span class="ectt-1000">substr</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,0,5)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110538r537"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110539r538"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110540r539"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">working_sequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110541r540"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110542r541"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_tm</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110543r542"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110544r543"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(%</span><span class="ectt-1000">results</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110545r544"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110546r545"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110547r546"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110548r547"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110549r548"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">output</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">=(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110550r549"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110551r550"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">gene</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">0</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110552r551"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110553r552"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">species</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">2</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110554r553"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">3</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110555r554"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">4</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110556r555"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">5</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110557r556"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">UCSCcoordinates</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">6</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110558r557"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">pcrsequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">7</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110559r558"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">8</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110560r559"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">target_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">9</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110561r560"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">10</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110562r561"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">11</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110563r562"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">12</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110564r563"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">13</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110565r564"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">14</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110566r565"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">15</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110567r566"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">16</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110568r567"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">warning</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110569r568"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">17</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110570r569"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">notes</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">18</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110571r570"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">set</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">t</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">19</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110572r571"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110573r572"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110574r573"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">Commented</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">on</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">022013</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110575r574"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">defined</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_ctrl_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110576r575"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5_ctrl_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},"\</span><span class="ectt-1000">t</span><span class="ectt-1000">";</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110577r576"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110578r577"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110579r578"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">defined</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_ctrl_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110580r579"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3_ctrl_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">},"\</span><span class="ectt-1000">t</span><span class="ectt-1000">";</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110581r580"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110582r581"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110583r582"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">OUT</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110584r583"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110585r584"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110586r585"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110587r586"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110588r587"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">csv</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">into</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">########################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110589r588"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse_the_line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110590r589"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110591r590"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110592r591"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110593r592"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110594r593"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110595r594"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110596r595"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110597r596"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110598r597"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110599r598"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110600r599"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110601r600"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110602r601"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$set</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110603r602"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110604r603"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">split</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/\</span><span class="ectt-1000">t</span><span class="ectt-1000">/</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110605r604"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110606r605"></a></span><span class="ectt-1000">$gene</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/^&lt;//;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110607r606"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Gene</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110608r607"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Ligmamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">name</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_name</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110609r608"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">species</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110610r609"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110611r610"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110612r611"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">target_prime</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110613r612"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">UCSC</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110614r613"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">]\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110615r614"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Set</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">[</span><span class="ectt-1000">$set</span><span class="ectt-1000">]\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110616r615"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110617r616"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">=~/</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/)</span><span class="ectt-1000">{</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">//}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">GO</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HERE</span><span class="ectt-1000">!</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110618r617"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110619r618"></a></span><span class="ectt-1000">$gene</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/&lt;//;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110620r619"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110621r620"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110622r621"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110623r622"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$gene</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110624r623"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110625r624"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110626r625"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110627r626"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$target_prime</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110628r627"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$UCSCcoordinates</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110629r628"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110630r629"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$set</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110631r630"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110632r631"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110633r632"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110634r633"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110635r634"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">csv</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">into</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">############</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110636r635"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110637r636"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">coordnates</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">into</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">############</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110638r637"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse_coordinates</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110639r638"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110640r639"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$input</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110641r640"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110642r641"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">$coordinates</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">split</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/\:/,</span><span class="ectt-1000">$input</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110643r642"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110644r643"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">$end</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">split</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">/\-/,</span><span class="ectt-1000">$coordinates</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110645r644"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">$chr</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/</span><span class="ectt-1000">chr</span><span class="ectt-1000">//;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">I</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">have</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">comment</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">out</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">this</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">behave</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">with</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">local</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">fasta</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">files</span><span class="ectt-1000">!</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110646r645"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110647r646"></a></span><span class="ectt-1000">$start</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/\,//</span><span class="ectt-1000">g</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110648r647"></a></span><span class="ectt-1000">$end</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/\,//</span><span class="ectt-1000">g</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110649r648"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110650r649"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110651r650"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110652r651"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110653r652"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">parse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">coordnates</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">into</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">variables</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110654r653"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110655r654"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">make</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">revcom</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">depending</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">on</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">annoation</span><span class="ectt-1000">###################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110656r655"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">revcom_slice_based_on_strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110657r656"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110658r657"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$strand</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110659r658"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110660r659"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">is</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">positive</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">make</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">reverse</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">compliment</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110661r660"></a></span><span class="ectt-1000">$strand</span><span class="ectt-1000">=</span><span class="ectt-1000">lc</span><span class="ectt-1000">(</span><span class="ectt-1000">$strand</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110662r661"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110663r662"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">plus</span><span class="ectt-1000">’</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110664r663"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">reverse</span><span class="ectt-1000">(</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110665r664"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=~</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">tr</span><span class="ectt-1000">/</span><span class="ectt-1000">ACGTacgt</span><span class="ectt-1000">/</span><span class="ectt-1000">TGCAtgca</span><span class="ectt-1000">/;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110666r665"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110667r666"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">is</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">minus</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">do</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">nothing</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110668r667"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">minus</span><span class="ectt-1000">’</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110669r668"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110670r669"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110671r670"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110672r671"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110673r672"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110674r673"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110675r674"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#####</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">SUBROUTINE</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">revcom_slice_based_on_strand</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">####################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110676r675"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110677r676"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110678r677"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtained</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">only</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">5’</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">end</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">##</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110679r678"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T5_tm_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110680r679"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110681r680"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110682r681"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110683r682"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110684r683"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110685r684"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110686r685"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110687r686"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110688r687"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110689r688"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110690r689"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">TC</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">=</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110691r690"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">=19};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110692r691"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">=19};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110693r692"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110694r693"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110695r694"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110696r695"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_out</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110697r696"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110698r697"></a></span><span class="ectt-1000">while</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&lt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110699r698"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110700r699"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">++;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110701r700"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110702r701"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">=</span><span class="ectt-1000">substr</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,0,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110703r702"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_primer</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110704r703"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">SeqFeature</span><span class="ectt-1000">::</span><span class="ectt-1000">Primer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110705r704"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">new</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110706r705"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110707r706"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">seq</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110708r707"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110709r708"></a></span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_primer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110710r709"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110711r710"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110712r711"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">salt</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110713r712"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">oligo</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110714r713"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110715r714"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">=</span><span class="ectt-1000">substr</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,0,5)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110716r715"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence_length</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">last</span><span class="ectt-1000">;}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110717r716"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">&gt;=</span><span class="ectt-1000">$temp</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110718r717"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110719r718"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_out</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110720r719"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110721r720"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">last</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110722r721"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110723r722"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110724r723"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">33)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110725r724"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110726r725"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_out</span><span class="ectt-1000">=</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110727r726"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110728r727"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">STDERR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Warning</span><span class="ectt-1000">:</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110729r728"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Assembly</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">at</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T5</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">side</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cut</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110730r729"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">off</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">due</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">low</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110731r730"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">last</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110732r731"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110733r732"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">elsif</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">&lt;=</span><span class="ectt-1000">$temp</span><span class="ectt-1000">)</span><span class="ectt-1000">{</span><span class="ectt-1000">next</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110734r733"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110735r734"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T5_seq_out</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110736r735"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110737r736"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110738r737"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtained</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">only</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">3’</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">end</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">##</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110739r738"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtain_T3_tm_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110740r739"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110741r740"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110742r741"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110743r742"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110744r743"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110745r744"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110746r745"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">=</span><span class="ectt-1000">shift</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">@_</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110747r746"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110748r747"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110749r748"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110750r749"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">TC</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">=(-</span><span class="ectt-1000">$control_length</span><span class="ectt-1000">)</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110751r750"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">=(-19)</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110752r751"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_location</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">=(-19)</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110753r752"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110754r753"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">=0;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110755r754"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110756r755"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_out</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110757r756"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110758r757"></a></span><span class="ectt-1000">while</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&lt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110759r758"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110760r759"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110761r760"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">--;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110762r761"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110763r762"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">substr</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$working_sequence</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110764r763"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_primer</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110765r764"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">SeqFeature</span><span class="ectt-1000">::</span><span class="ectt-1000">Primer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110766r765"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">new</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110767r766"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110768r767"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">seq</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110769r768"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110770r769"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_primer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110771r770"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110772r771"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110773r772"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">salt</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$salt</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110774r773"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">oligo</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">$lig_conc</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110775r774"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110776r775"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">=</span><span class="ectt-1000">substr</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">,0,5)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110777r776"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(-</span><span class="ectt-1000">$working_sequence_length</span><span class="ectt-1000">)</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">last</span><span class="ectt-1000">;}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110778r777"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">&lt;(-80)</span><span class="ectt-1000">)</span><span class="ectt-1000">{</span><span class="ectt-1000">die</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110779r778"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">&gt;=</span><span class="ectt-1000">$temp</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110780r779"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110781r780"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_out</span><span class="ectt-1000">=</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110782r781"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110783r782"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">last</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110784r783"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110785r784"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110786r785"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(-33)</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110787r786"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110788r787"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_out</span><span class="ectt-1000">=</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110789r788"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110790r789"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">STDERR</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Warning</span><span class="ectt-1000">:</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110791r790"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Assembly</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">at</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">T3</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">side</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cut</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110792r791"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">off</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">due</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">low</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Tm</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110793r792"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">last</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110794r793"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110795r794"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">&lt;</span><span class="ectt-1000">$temp</span><span class="ectt-1000">)</span><span class="ectt-1000">{</span><span class="ectt-1000">next</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110796r795"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110797r796"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$T3_seq_out</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_tm</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq_length</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110798r797"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110799r798"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110800r799"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtained</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">only</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">3’</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">end</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">working</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">####</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110801r800"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110802r801"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">joined</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">pieces</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">of</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#########</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110803r802"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer_piece_joiner</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110804r803"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110805r804"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110806r805"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110807r806"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">lig_joiner_code</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110808r807"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T5_seq</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110809r808"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">barcode</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110810r809"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">T3_seq</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110811r810"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110812r811"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$short_sequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110813r812"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110814r813"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110815r814"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$warning</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110816r815"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">=</span><span class="ectt-1000">"</span><span class="ectt-1000">\/5</span><span class="ectt-1000">Phos</span><span class="ectt-1000">\/</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110817r816"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110818r817"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-5</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110819r818"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110820r819"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">three_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110821r820"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110822r821"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110823r822"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110824r823"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110825r824"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110826r825"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-5-</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110827r826"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110828r827"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">three_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110829r828"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">$short_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110830r829"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110831r830"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110832r831"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110833r832"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110834r833"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-5-</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110835r834"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110836r835"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">three_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110837r836"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110838r837"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110839r838"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110840r839"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110841r840"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110842r841"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-3</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110843r842"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110844r843"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">five_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110845r844"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">,</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110846r845"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110847r846"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110848r847"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110849r848"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-3-</span><span class="ectt-1000">I</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110850r849"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110851r850"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">five_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110852r851"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110853r852"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110854r853"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110855r854"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110856r855"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110857r856"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">T</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">-3-</span><span class="ectt-1000">T</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110858r857"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110859r858"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">=</span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">five_prime_PCR_sequence</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110860r859"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">,</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110861r860"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110862r861"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110863r862"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110864r863"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">S</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110865r864"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110866r865"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">$short_sequence</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110867r866"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110868r867"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110869r868"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110870r869"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110871r870"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">L</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110872r871"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110873r872"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110874r873"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110875r874"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110876r875"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110877r876"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110878r877"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$lig_joiner_code</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">I</span><span class="ectt-1000">-</span><span class="ectt-1000">L</span><span class="ectt-1000">-</span><span class="ectt-1000">C</span><span class="ectt-1000">"</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110879r878"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110880r879"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">join</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">"</span><span class="ectt-1000">"</span><span class="ectt-1000">,</span><span class="ectt-1000">$Phos_mod_code</span><span class="ectt-1000">,</span><span class="ectt-1000">$T5_seq</span><span class="ectt-1000">,</span><span class="ectt-1000">$barcode</span><span class="ectt-1000">,</span><span class="ectt-1000">$T3_seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110881r880"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110882r881"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">-7;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110883r882"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110884r883"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110885r884"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">60)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110886r885"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110887r886"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">STDERR</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110888r887"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Warning</span><span class="ectt-1000">!</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">The</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">from</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">input</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">file</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">data</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">line</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$</span><span class="ectt-1000">.</span><span class="ectt-1000">"</span><span class="ectt-1000">.</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110889r888"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">has</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">a</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">length</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">greater</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">than</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">60!\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110890r889"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">};</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110891r890"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110892r891"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110893r892"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">pcrsequence</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$pcrsequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110894r893"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110895r894"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">ligamer_length</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$ligamer_length</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110896r895"></a></span><span class="ectt-1000">$results</span><span class="ectt-1000">{</span><span class="ectt-1000">warning</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$warning</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110897r896"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110898r897"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">%</span><span class="ectt-1000">results</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110899r898"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110900r899"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110901r900"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110902r901"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110903r902"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110904r903"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">#</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Load</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">the</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">latest</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Registry</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110905r904"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl_database</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110906r905"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110907r906"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$verbose</span><span class="ectt-1000">=</span><span class="ectt-1000">shift@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110908r907"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">=</span><span class="ectt-1000">shift@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110909r908"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110910r909"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$registry</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">Bio</span><span class="ectt-1000">::</span><span class="ectt-1000">EnsEMBL</span><span class="ectt-1000">::</span><span class="ectt-1000">Registry</span><span class="ectt-1000">’</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110911r910"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Beginning</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">login</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">database</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110912r911"></a></span><span class="ectt-1000">$registry</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">load_registry_from_db</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110913r912"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110914r913"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">host</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">ensembldb</span><span class="ectt-1000">.</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">.</span><span class="ectt-1000">org</span><span class="ectt-1000">’</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110915r914"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">user</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">anonymous</span><span class="ectt-1000">’</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110916r915"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">db_version</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db_version</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110917r916"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">verbose</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=&gt;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$verbose</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110918r917"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110919r918"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110920r919"></a></span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">Done</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">loading</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">ensembl</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">database</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110921r920"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$registry</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110922r921"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110923r922"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110924r923"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110925r924"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtained</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">#######</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110926r925"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get_genomic_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110927r926"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110928r927"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110929r928"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110930r929"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_adaptor</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$db</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">get_adaptor</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$species</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">Core</span><span class="ectt-1000">’</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">Slice</span><span class="ectt-1000">’</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110931r930"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110932r931"></a></span><span class="ectt-1000">$chr</span><span class="ectt-1000">=~</span><span class="ectt-1000">s</span><span class="ectt-1000">/^</span><span class="ectt-1000">chr</span><span class="ectt-1000">//;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110933r932"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110934r933"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_adaptor</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110935r934"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">fetch_by_region</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110936r935"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110937r936"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">’</span><span class="ectt-1000">chromosome</span><span class="ectt-1000">’</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110938r937"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$chr</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110939r938"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$start</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110940r939"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$end</span><span class="ectt-1000">,</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110941r940"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110942r941"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110943r942"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$slice</span><span class="ectt-1000">-&gt;</span><span class="ectt-1000">seq</span><span class="ectt-1000">)</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110944r943"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110945r944"></a></span><span class="ectt-1000">return</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$slice_sequence</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110946r945"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110947r946"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">obtained</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">get</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">genomic</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">sequence</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">slice</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">######</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110948r947"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############################################################################</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110949r948"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">BEGIN</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PROVIDE</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HELP</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">DATA</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">######</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110950r949"></a></span><span class="ectt-1000">sub</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">podhelp</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110951r950"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110952r951"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110953r952"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$help_print</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110954r953"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$man_print</span><span class="ectt-1000">=</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">shift@_</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110955r954"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$perlname</span><span class="ectt-1000">=</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110956r955"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$htmlname</span><span class="ectt-1000">=</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110957r956"></a></span><span class="ectt-1000">my</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$manname</span><span class="ectt-1000">=</span><span class="ectt-1000">$scriptname</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110958r957"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110959r958"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$help_print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110960r959"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110961r960"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$htmlname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=~</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">s</span><span class="ectt-1000">/\.</span><span class="ectt-1000">pl</span><span class="ectt-1000">/\.</span><span class="ectt-1000">html</span><span class="ectt-1000">/;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110962r961"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">system</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">pod2html</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$perlname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">title</span><span class="ectt-1000">=</span><span class="ectt-1000">$perlname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">--</span><span class="ectt-1000">outfile</span><span class="ectt-1000">=</span><span class="ectt-1000">$htmlname</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110963r962"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">\</span><span class="ectt-1000">t$htmlname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printed</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">cwd</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110964r963"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">exit</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110965r964"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110966r965"></a></span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110967r966"></a></span><span class="ectt-1000">if</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">(</span><span class="ectt-1000">$man_print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">eq</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">1)</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110968r967"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">{</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110969r968"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$manname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">=~</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">s</span><span class="ectt-1000">/\.</span><span class="ectt-1000">pl</span><span class="ectt-1000">/\.</span><span class="ectt-1000">man</span><span class="ectt-1000">/;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110970r969"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">system</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">pod2man</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$perlname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$manname</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110971r970"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">print</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">\</span><span class="ectt-1000">t$manname</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">printed</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">in</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$dir</span><span class="ectt-1000">.\</span><span class="ectt-1000">n</span><span class="ectt-1000">\</span><span class="ectt-1000">n</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110972r971"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">system</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">"</span><span class="ectt-1000">man</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">-</span><span class="ectt-1000">l</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">$manname</span><span class="ectt-1000">|</span><span class="ectt-1000">less</span><span class="ectt-1000">"</span><span class="ectt-1000">;</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110973r972"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">exit</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110974r973"></a></span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110975r974"></a></span><span class="ectt-1000">}</span><span class="ectt-1000">&nbsp;</span><br><span class="label"><a id="x1-110976r975"></a></span><span class="ectt-1000">#</span><span class="ectt-1000">###############</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">END</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">Subroutine</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">to</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">PROVIDE</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">POD</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">HELP</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">DATA</span><span class="ectt-1000">&nbsp;</span><span class="ectt-1000">######</span>

</div>
                                                                                

                                                                                
<a id="x1-110977r8"></a>
<a id="Q1-1-189"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a id="x1-1110008"></a>Bibliography</h2>
<div class="thebibliography">
<p class="bibitem"><span class="biblabel">
<a id="XAltschul1990"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Altschul, S., Gish, W., and Miller, W. (1990).  Basic local alignment search
tool. <span class="phvro8t-x-x-120">Journal of molecular </span><span class="phvro8t-x-x-120">…</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAmitsur1987d"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Amitsur,  M.,  Levitz,  R.,  and  Kaufmann,  G.  (1987).    Bacteriophage  T4
anticodon nuclease,  polynucleotide  kinase  and  RNA  ligase  reprocess  the
host lysine tRNA. <span class="phvro8t-x-x-120">The EMBO journal</span>, 6(8):2499–503.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAnders2010a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Anders, S.  and  Huber,  W.  (2010).    Differential  expression  analysis  for
sequence count data. <span class="phvro8t-x-x-120">Genome Biology</span>, 11(10):R106.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAnstey2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Anstey, M.&nbsp;L.,  Rogers,  S.&nbsp;M.,  Ott,  S.&nbsp;R.,  Burrows,  M.,  and  Simpson,
S.&nbsp;J. (2009).  Serotonin mediates behavioral gregarization underlying swarm
formation in desert locusts. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 323(5914):627–30.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.,  Gaidatzis,  D.,  Pfeffer,  S.,  Lagos-Quintana,  M.,  Landgraf,  P.,
Iovino, N., Morris, P., Brownstein, M.&nbsp;J., Kuramochi-Miyagawa, S., Nakano,
T., Chien, M., Russo, J.&nbsp;J., Ju, J., Sheridan, R., Sander, C., Zavolan, M., and
Tuschl, T. (2006).  A novel class of small RNAs bind to MILI protein in mouse
testes. <span class="phvro8t-x-x-120">Nature</span>, 442(7099):203–207.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2008a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.&nbsp;A. and Hannon, G.&nbsp;J. (2008).  Small RNA silencing pathways in
                                                                                

                                                                                
germ and stem cells.  <span class="phvro8t-x-x-120">Cold Spring Harbor symposia on quantitative biology</span>,
73:283–90.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2007a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.&nbsp;A., Hannon, G.&nbsp;J., and Brennecke, J. (2007a).  The Piwi-piRNA
Pathway Provides  an  Adaptive  Defense  in  the  Transposon  Arms  Race.
<span class="phvro8t-x-x-120">Science</span>, 318(5851):761–764.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.&nbsp;A., Naumova, N.&nbsp;M., Tulin, A.&nbsp;V., Vagin, V.&nbsp;V., Rozovsky, Y.&nbsp;M.,
and Gvozdev,  V.&nbsp;A.  (2001).    Double-stranded  RNA-mediated  silencing  of
genomic tandem repeats and transposable elements in the D. melanogaster
germline. <span class="phvro8t-x-x-120">Current Biology</span>, 11(13):1017–1027.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.&nbsp;A., Sachidanandam, R., Bourc’his, D., Schaefer, C., Pezic, D.,
Toth, K.&nbsp;F., Bestor, T., and Hannon, G.&nbsp;J. (2008).  A piRNA Pathway Primed
by Individual Transposons Is Linked to De Novo DNA Methylation in Mice.
<span class="phvro8t-x-x-120">Molecular Cell</span>, 31(6):785–799.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAravin2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Aravin, A.&nbsp;A., Sachidanandam, R., Girard, A., Fejes-Toth, K., and Hannon,
G.&nbsp;J. (2007b).  Developmentally regulated piRNA clusters implicate MILI in
transposon control. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 316(5825):744–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XAshe2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ashe, A.,  Sapetschnig,  A.,  Weick,  E.-M.,  Mitchell,  J.,  Bagijn,  M.&nbsp;P.,
Cording, A.&nbsp;C.,  Doebley,  A.-L.,  Goldstein,  L.&nbsp;D.,  Lehrbach,  N.&nbsp;J.,  Le  Pen,
J., Pintacuda, G., Sakaguchi, A., Sarkies, P., Ahmed, S., and Miska, E.&nbsp;a.
(2012). piRNAs  can  trigger  a  multigenerational  epigenetic  memory  in  the
germline of C. elegans. <span class="phvro8t-x-x-120">Cell</span>, 150(1):88–99.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBailey2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bailey, T.&nbsp;L.,  Boden,  M.,  Buske,  F.&nbsp;a.,  Frith,  M.,  Grant,  C.&nbsp;E.,  Clementi,
L., Ren, J.,  Li,  W.&nbsp;W.,  and  Noble,  W.&nbsp;S.  (2009).   MEME  SUITE:  tools  for
motif discovery  and  searching.    <span class="phvro8t-x-x-120">Nucleic  acids  research</span>,  37(Web  Server
issue):W202–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBailey1994"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bailey, T.&nbsp;L. and Elkan, C. (1994).  Fitting a mixture model by expectation
maximization to discover motifs in biopolymers.   <span class="phvro8t-x-x-120">Proceedings International</span>
                                                                                

                                                                                
<span class="phvro8t-x-x-120">Conference on Intelligent Systems for Molecular Biology ISMB International</span>
<span class="phvro8t-x-x-120">Conference on Intelligent Systems for Molecular Biology</span>, 2:28–36.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBarany1991"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Barany, F.  (1991).    Genetic  disease  detection  and  DNA  amplification
using cloned thermostable ligase.  <span class="phvro8t-x-x-120">Proceedings of the National Academy of</span>
<span class="phvro8t-x-x-120">Sciences of the United States of America</span>, 88(1):189–93.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBarash2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Barash, Y., Calarco, J.&nbsp;A., Gao, W., Pan, Q., Wang, X., Shai, O., Blencowe,
B.&nbsp;J., and  Frey,  B.&nbsp;J.  (2010).    Deciphering  the  splicing  code.    <span class="phvro8t-x-x-120">Nature</span>,
465(7294):53–59.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBarbosa-Morais2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Barbosa-Morais,  N.&nbsp;L.,  Irimia,  M.,  Pan,  Q.,  Xiong,  H.&nbsp;Y.,  Gueroussov,
S., Lee, L.&nbsp;J.,  Slobodeniuc,  V.,  Kutter,  C.,  Watt,  S.,  Colak,  R.,  Kim,  T.,
Misquitta-Ali, C.&nbsp;M., Wilson, M.&nbsp;D., Kim, P.&nbsp;M., Odom, D.&nbsp;T., Frey, B.&nbsp;J., and
Blencowe, B.&nbsp;J. (2012). The evolutionary landscape of alternative splicing in
vertebrate species. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 338(6114):1587–93.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBarrangou2007c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Barrangou,  R.,  Fremaux,  C.,  Deveau,  H.,  Richards,  M.,  Boyaval,  P.,
Moineau, S.,  Romero,  D.&nbsp;a.,  and  Horvath,  P.  (2007).    CRISPR  provides
acquired resistance against viruses in prokaryotes. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
315(5819):1709–12.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBarrett2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Barrett,   T.,   Wilhite,   S.&nbsp;E.,   Ledoux,   P.,   Evangelista,   C.,   Kim,   I.&nbsp;F.,
Tomashevsky, M., Marshall, K.&nbsp;a., Phillippy, K.&nbsp;H., Sherman, P.&nbsp;M., Holko,
M., Yefanov, A., Lee, H., Zhang, N., Robertson, C.&nbsp;L., Serova, N., Davis, S.,
and Soboleva, A. (2013).  NCBI GEO: archive for functional genomics data
sets–update. <span class="phvro8t-x-x-120">Nucleic acids research</span>, 41(Database issue):D991–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBaudat2000c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Baudat, F.,  Manova,  K.,  Yuen,  J.&nbsp;P.,  Jasin,  M.,  and  Keeney,  S.  (2000).
Chromosome synapsis defects and sexually dimorphic meiotic progression in
mice lacking Spo11. <span class="phvro8t-x-x-120">Molecular cell</span>, 6(5):989–98.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCordell1980a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bell, G.&nbsp;I.,  Pictet,  R.&nbsp;L.,  Rutter,  W.&nbsp;J.,  Cordell,  B.,  Tischer,  E.,  and
Goodman,  H.&nbsp;M.  (1980).   Sequence  of  the  human  insulin  gene.   <span class="phvro8t-x-x-120">Nature</span>,
                                                                                

                                                                                
284(5751):26–32.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBenson2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Benson, D.&nbsp;a., Karsch-Mizrachi, I., Lipman, D.&nbsp;J., Ostell, J., and Sayers,
E.&nbsp;W. (2011). GenBank. <span class="phvro8t-x-x-120">Nucleic acids research</span>, 39(Database issue):D32–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBentley2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bentley, D.&nbsp;R., Balasubramanian, S., Swerdlow, H.&nbsp;P., Smith, G.&nbsp;P., Milton,
J., Brown,  C.&nbsp;G.,  Hall,  K.&nbsp;P.,  Evers,  D.&nbsp;J.,  Barnes,  C.&nbsp;L.,  Bignell,  H.&nbsp;R.,
Boutell, J.&nbsp;M.,  Bryant,  J.,  Carter,  R.&nbsp;J.,  Keira  Cheetham,  R.,  Cox,  A.&nbsp;J.,
Ellis, D.&nbsp;J., Flatbush, M.&nbsp;R., Gormley, N.&nbsp;a., Humphray, S.&nbsp;J., Irving, L.&nbsp;J.,
Karbelashvili,  M.&nbsp;S.,  Kirk,  S.&nbsp;M.,  Li,  H.,  Liu,  X.,  Maisinger,  K.&nbsp;S.,  Murray,
L.&nbsp;J., Obradovic, B., Ost, T., Parkinson, M.&nbsp;L., Pratt, M.&nbsp;R., Rasolonjatovo,
I. M.&nbsp;J., Reed,  M.&nbsp;T.,  Rigatti,  R.,  Rodighiero,  C.,  Ross,  M.&nbsp;T.,  Sabot,  A.,
Sankar, S.&nbsp;V., Scally, A., Schroth, G.&nbsp;P., Smith, M.&nbsp;E., Smith, V.&nbsp;P., Spiridou,
A., Torrance, P.&nbsp;E., Tzonev, S.&nbsp;S., Vermaas, E.&nbsp;H., Walter, K., Wu, X., Zhang,
L., Alam, M.&nbsp;D., Anastasi, C., Aniebo, I.&nbsp;C., Bailey, D. M.&nbsp;D., Bancarz, I.&nbsp;R.,
Banerjee, S., Barbour, S.&nbsp;G., Baybayan, P.&nbsp;a., Benoit, V.&nbsp;a., Benson, K.&nbsp;F.,
Bevis, C., Black, P.&nbsp;J., Boodhun, A., Brennan, J.&nbsp;S., Bridgham, J.&nbsp;a., Brown,
R.&nbsp;C., Brown, A.&nbsp;a., Buermann, D.&nbsp;H., Bundu, A.&nbsp;a., Burrows, J.&nbsp;C., Carter,
N.&nbsp;P., Castillo,  N.,  Chiara  E  Catenazzi,  M.,  Chang,  S.,  Neil  Cooley,  R.,
Crake, N.&nbsp;R., Dada, O.&nbsp;O., Diakoumakos, K.&nbsp;D., Dominguez-Fernandez, B.,
Earnshaw, D.&nbsp;J., Egbujor, U.&nbsp;C., Elmore, D.&nbsp;W., Etchin, S.&nbsp;S., Ewan, M.&nbsp;R.,
Fedurco, M., Fraser, L.&nbsp;J., Fuentes Fajardo, K.&nbsp;V., Scott Furey, W., George,
D., Gietzen,  K.&nbsp;J.,  Goddard,  C.&nbsp;P.,  Golda,  G.&nbsp;S.,  Granieri,  P.&nbsp;a.,  Green,
D.&nbsp;E., Gustafson,  D.&nbsp;L.,  Hansen,  N.&nbsp;F.,  Harnish,  K.,  Haudenschild,  C.&nbsp;D.,
Heyer, N.&nbsp;I., Hims, M.&nbsp;M., Ho, J.&nbsp;T., Horgan, A.&nbsp;M., Hoschler, K., Hurwitz,
S., Ivanov, D.&nbsp;V., Johnson, M.&nbsp;Q., James, T., Huw Jones, T.&nbsp;a., Kang, G.-D.,
Kerelska, T.&nbsp;H., Kersey, A.&nbsp;D., Khrebtukova, I., Kindwall, A.&nbsp;P., Kingsbury, Z.,
Kokko-Gonzales, P.&nbsp;I., Kumar, A., Laurent, M.&nbsp;a., Lawley, C.&nbsp;T., Lee, S.&nbsp;E.,
Lee, X., Liao, A.&nbsp;K., Loch, J.&nbsp;a., Lok, M., Luo, S., Mammen, R.&nbsp;M., Martin,
J.&nbsp;W., McCauley, P.&nbsp;G., McNitt, P., Mehta, P., Moon, K.&nbsp;W., Mullens, J.&nbsp;W.,
Newington, T., Ning, Z., Ling Ng, B., Novo, S.&nbsp;M., O’Neill, M.&nbsp;J., Osborne,
M.&nbsp;a., Osnowski,  A.,  Ostadan,  O.,  Paraschos,  L.&nbsp;L.,  Pickering,  L.,  Pike,
A.&nbsp;C., Pike, A.&nbsp;C., Chris Pinkard, D., Pliskin, D.&nbsp;P., Podhasky, J., Quijano,
V.&nbsp;J., Raczy,  C.,  Rae,  V.&nbsp;H.,  Rawlings,  S.&nbsp;R.,  Chiva  Rodriguez,  A.,  Roe,
P.&nbsp;M., Rogers, J., Rogert Bacigalupo, M.&nbsp;C., Romanov, N., Romieu, A., Roth,
R.&nbsp;K., Rourke, N.&nbsp;J., Ruediger, S.&nbsp;T., Rusman, E., Sanches-Kuiper, R.&nbsp;M.,
Schenker, M.&nbsp;R., Seoane, J.&nbsp;M., Shaw, R.&nbsp;J., Shiver, M.&nbsp;K., Short, S.&nbsp;W.,
Sizto, N.&nbsp;L.,  Sluis,  J.&nbsp;P.,  Smith,  M.&nbsp;a.,  Ernest  Sohna  Sohna,  J.,  Spence,
E.&nbsp;J., Stevens, K., Sutton, N., Szajkowski, L., Tregidgo, C.&nbsp;L., Turcatti, G.,
                                                                                

                                                                                
Vandevondele,  S.,  Verhovsky,  Y.,  Virk,  S.&nbsp;M.,  Wakelin,  S.,  Walcott,  G.&nbsp;C.,
Wang, J., Worsley, G.&nbsp;J., Yan, J., Yau, L., Zuerlein, M., Rogers, J., Mullikin,
J.&nbsp;C., Hurles,  M.&nbsp;E.,  McCooke,  N.&nbsp;J.,  West,  J.&nbsp;S.,  Oaks,  F.&nbsp;L.,  Lundberg,
P.&nbsp;L., Klenerman, D., Durbin, R., and Smith, A.&nbsp;J. (2008).   Accurate whole
human genome sequencing using reversible terminator chemistry.   <span class="phvro8t-x-x-120">Nature</span>,
456(7218):53–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBenton2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Benton, M.&nbsp;J. and Donoghue, P. C.&nbsp;J. (2007).  Paleontological evidence to
date the tree of life. <span class="phvro8t-x-x-120">Molecular biology and evolution</span>, 24(1):26–53.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBerg2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Berg, M.&nbsp;G.,  Singh,  L.&nbsp;N.,  Younis,  I.,  Liu,  Q.,  Pinto,  A.&nbsp;M.,  Kaida,  D.,
Zhang, Z., Cho, S., Sherrill-Mix, S., Wan, L., and Dreyfuss, G. (2012).   U1
snRNP determines  mRNA  length  and  regulates  isoform  expression.   <span class="phvro8t-x-x-120">Cell</span>,
150(1):53–64.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBerget1977"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Berget, S.&nbsp;M., Moore, C., and Sharp, P.&nbsp;A. (1977).  Spliced segments at
the 5’ terminus  of  adenovirus  2  late  mRNA.   <span class="phvro8t-x-x-120">Reviews  in  medical  virology</span>,
10(6):356–62; discussion 355–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBernstein1983a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bernstein,  S.&nbsp;I.,  Mogami,  K.,  Donady,  J.&nbsp;J.,  and  Emerson,  C.&nbsp;P.  (1983).
Drosophila muscle myosin heavy chain encoded by a single gene in a cluster
of muscle mutations. <span class="phvro8t-x-x-120">Nature</span>, 302(5907):393–397.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBesmer1972b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Besmer,  P.,  Jr.,  R.  C.&nbsp;M.,  Caruthers,  M.&nbsp;H.,  Kumar,  A.,  Minamoto,  K.,
van&nbsp;de Sande,  J.,  Sidarova,  N.,  and  Khorana,  H.  (1972).     Studies  on
polynucleotides. CXVII. Hybridization of polydeoxynucleotides with tyrosine
transfer RNA sequences to the r-strand of phi80psu + 3 DNA.   <span class="phvro8t-x-x-120">Journal of</span>
<span class="phvro8t-x-x-120">Molecular Biology</span>, 72:503–522.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBhattacharjee2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bhattacharjee, Y. (2014). The Vigilante. <span class="phvro8t-x-x-120">Science</span>, 343(6177):1306–1309.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBhattacharya2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bhattacharya, A., Czaplinski, K., Trifillis, P., He, F., Jacobson, A., and Peltz,
S.&nbsp;W. (2000).  Characterization of the biochemical properties of the human
Upf1 gene product that is involved in nonsense-mediated mRNA decay. <span class="phvro8t-x-x-120">RNA</span>
                                                                                

                                                                                
<span class="phvro8t-x-x-120">(New York, N.Y.)</span>, 6:1226–1235.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBieberstein2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bieberstein, N.&nbsp;I., Carrillo Oesterreich, F., Straube, K., and Neugebauer,
K.&nbsp;M. (2012).   First  exon  length  controls  active  chromatin  signatures  and
transcription. <span class="phvro8t-x-x-120">Cell reports</span>, 2(1):62–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBingham2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bingham,   J.&nbsp;L.,   Carrigan,   P.&nbsp;E.,   Miller,   L.&nbsp;J.,   and   Srinivasan,   S.
(2008). Extent  and  diversity  of  human  alternative  splicing  established  by
complementary  database  annotation  and  microarray  analysis.    <span class="phvro8t-x-x-120">Omics:  A</span>
<span class="phvro8t-x-x-120">Journal of Integrative Biology</span>, 12(1):83–92.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBirney2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Birney, E.,  Stamatoyannopoulos,  J.&nbsp;A.,  Dutta,  A.,  Guig|[oacute]|,  R.,
Gingeras, T.&nbsp;R., Margulies, E.&nbsp;H., Weng, Z., Snyder, M., Dermitzakis, E.&nbsp;T.,
Thurman, R.&nbsp;E.,  Kuehn,  M.&nbsp;S.,  Taylor,  C.&nbsp;M.,  Neph,  S.,  Koch,  C.&nbsp;M.,
Asthana, S., Malhotra, A., Adzhubei, I., Greenbaum, J.&nbsp;A., Andrews, R.&nbsp;M.,
Flicek, P., Boyle,  P.&nbsp;J.,  Cao,  H.,  Carter,  N.&nbsp;P.,  Clelland,  G.&nbsp;K.,  Davis,  S.,
Day, N., Dhami,  P.,  Dillon,  S.&nbsp;C.,  Dorschner,  M.&nbsp;O.,  Fiegler,  H.,  Giresi,
P.&nbsp;G., Goldy,  J.,  Hawrylycz,  M.,  Haydock,  A.,  Humbert,  R.,  James,  K.&nbsp;D.,
Johnson, B.&nbsp;E., Johnson, E.&nbsp;M., Frum, T.&nbsp;T., Rosenzweig, E.&nbsp;R., Karnani,
N., Lee, K., Lefebvre, G.&nbsp;C., Navas, P.&nbsp;A., Neri, F., Parker, S. C.&nbsp;J., Sabo,
P.&nbsp;J., Sandstrom,  R.,  Shafer,  A.,  Vetrie,  D.,  Weaver,  M.,  Wilcox,  S.,  Yu1,
M., Collins,  F.&nbsp;S.,  Dekker,  J.,  Lieb,  J.&nbsp;D.,  Tullius,  T.&nbsp;D.,  Crawford,  G.&nbsp;E.,
Sunyaev, S., Noble, W.&nbsp;S., Dunham, I., Denoeud, F., Reymond, A., Kapranov,
P., Rozowsky,  J.,  Zheng,  D.,  Castelo,  R.,  Frankish,  A.,  Harrow,  J.,  Ghosh,
S., Sandelin,  A.,  Hofacker,  I.&nbsp;L.,  Baertsch,  R.,  Keefe,  D.,  Dike,  S.,  Cheng,
J., Hirsch,  H.&nbsp;A.,  Sekinger,  E.&nbsp;A.,  Lagarde,  J.,  Abril,  J.&nbsp;F.,  Shahab,  A.,
Flamm, C.,  Fried,  C.,  Hackerm|[uuml]|ller,  J.,  Hertel,  J.,  Lindemeyer,  M.,
Missal, K., Tanzer, A., Washietl, S., Korbel, J., Emanuelsson, O., Pedersen,
J.&nbsp;S., Holroyd, N., Taylor, R., Swarbreck, D., Matthews, N., Dickson, M.&nbsp;C.,
Thomas, D.&nbsp;J., Weirauch, M.&nbsp;T., Gilbert, J., Drenkow, J., Bell, I., Zhao, X.,
Srinivasan,  K.,  Sung,  W.-K.,  Ooi,  H.&nbsp;S.,  Chiu,  K.&nbsp;P.,  Foissac,  S.,  Alioto,
T., Brent, M.,  Pachter,  L.,  Tress,  M.&nbsp;L.,  Valencia,  A.,  Choo,  S.&nbsp;W.,  Choo,
C.&nbsp;Y., Ucla,  C.,  Manzano,  C.,  Wyss,  C.,  Cheung,  E.,  Clark,  T.&nbsp;G.,  Brown,
J.&nbsp;B., Ganesh,  M.,  Patel,  S.,  Tammana,  H.,  Chrast,  J.,  Henrichsen,  C.&nbsp;N.,
Kai, C., Kawai,  J.,  Nagalakshmi,  U.,  Wu,  J.,  Lian,  Z.,  Lian,  J.,  Newburger,
P., Zhang,  X.,  Bickel,  P.,  Mattick,  J.&nbsp;S.,  Carninci,  P.,  Hayashizaki,  Y.,
Weissman, S., Hubbard, T., Myers, R.&nbsp;M., Rogers, J., Stadler, P.&nbsp;F., Lowe,
T.&nbsp;M., Wei,  C.-L.,  Ruan,  Y.,  Struhl,  K.,  Gerstein,  M.,  Antonarakis,  S.&nbsp;E.,
                                                                                

                                                                                
Fu, Y., Green,  E.&nbsp;D.,  Kara|[ouml]|z,  U.,  Siepel,  A.,  Taylor,  J.,  Liefer,  L.&nbsp;A.,
Wetterstrand,  K.&nbsp;A.,  Good,  P.&nbsp;J.,  Feingold,  E.&nbsp;A.,  Guyer,  M.&nbsp;S.,  Cooper,
G.&nbsp;M., Asimenos, G., Dewey, C.&nbsp;N., Hou, M., Nikolaev, S., Montoya-Burgos,
J.&nbsp;I., L|[ouml]|ytynoja,  A.,  Whelan,  S.,  Pardi,  F.,  Massingham,  T.,  Huang,
H., Zhang,  N.&nbsp;R.,  Holmes,  I.,  Mullikin,  J.&nbsp;C.,  Ureta-Vidal,  A.,  Paten,  B.,
Seringhaus,  M.,  Church,  D.,  Rosenbloom,  K.,  Kent,  W.&nbsp;J.,  Stone,  E.&nbsp;A.,
Batzoglou, S., Goldman, N., Hardison, R.&nbsp;C., Haussler, D., Miller, W., Sidow,
A., Trinklein, N.&nbsp;D., Zhang, Z.&nbsp;D., Barrera, L., Stuart, R., King, D.&nbsp;C., Ameur,
A., Enroth, S., Bieda, M.&nbsp;C., Kim, J., Bhinge, A.&nbsp;A., Jiang, N., Liu, J., Yao,
F., Vega, V.&nbsp;B.,  Lee,  C.&nbsp;W.,  Ng,  P.,  Yang,  A.,  Moqtaderi,  Z.,  Zhu,  Z.,  Xu,
X., Squazzo, S., Oberley, M.&nbsp;J., Inman, D., Singer, M.&nbsp;A., Richmond, T.&nbsp;A.,
Munn, K.&nbsp;J., Rada-Iglesias, A., Wallerman, O., Komorowski, J., Fowler, J.&nbsp;C.,
Couttet, P.,  Bruce,  A.&nbsp;W.,  Dovey,  O.&nbsp;M.,  Ellis,  P.&nbsp;D.,  Langford,  C.&nbsp;F.,  Nix,
D.&nbsp;A., Euskirchen, G., Hartman, S., Urban, A.&nbsp;E., Kraus, P., Calcar, S.&nbsp;V.,
Heintzman,  N.,  Kim,  T.&nbsp;H.,  Wang,  K.,  Qu,  C.,  Hon,  G.,  Luna,  R.,  Glass,
C.&nbsp;K., Rosenfeld, M.&nbsp;G., Aldred, S.&nbsp;F., Cooper, S.&nbsp;J., Halees, A., Lin, J.&nbsp;M.,
Shulha, H.&nbsp;P., Zhang, X., Xu, M., Haidar, J. N.&nbsp;S., Yu, Y., Birney*, E., Iyer,
V.&nbsp;R., Green,  R.&nbsp;D.,  Wadelius,  C.,  Farnham,  P.&nbsp;J.,  Ren,  B.,  Harte,  R.&nbsp;A.,
Hinrichs, A.&nbsp;S.,  Trumbower,  H.,  Clawson,  H.,  Hillman-Jackson,  J.,  Zweig,
A.&nbsp;S., Smith, K., Thakkapallayil, A., Barber, G., Kuhn, R.&nbsp;M., Karolchik, D.,
Armengol,  L.,  Bird,  C.&nbsp;P.,  de&nbsp;Bakker,  P.  I.&nbsp;W.,  Kern,  A.&nbsp;D.,  Lopez-Bigas,
N., Martin,  J.&nbsp;D.,  Stranger,  B.&nbsp;E.,  Woodroffe,  A.,  Davydov,  E.,  Dimas,  A.,
Eyras, E.,  Hallgr|[iacute]|msd|[oacute]|ttir,  I.&nbsp;B.,  Huppert,  J.,  Zody,  M.&nbsp;C.,
Abecasis, G.&nbsp;R.,  Estivill,  X.,  Bouffard,  G.&nbsp;G.,  Guan,  X.,  Hansen,  N.&nbsp;F.,
Idol, J.&nbsp;R., Maduro, V.&nbsp;V., Maskeri, B., McDowell, J.&nbsp;C., Park, M., Thomas,
P.&nbsp;J., Young, A.&nbsp;C., Blakesley, R.&nbsp;W., Muzny, D.&nbsp;M., Sodergren, E., Wheeler,
D.&nbsp;A., Worley,  K.&nbsp;C.,  Jiang,  H.,  Weinstock,  G.&nbsp;M.,  Gibbs,  R.&nbsp;A.,  Graves,
T., Fulton,  R.,  Mardis,  E.&nbsp;R.,  Wilson,  R.&nbsp;K.,  Clamp,  M.,  Cuff,  J.,  Gnerre,
S., Jaffe, D.&nbsp;B.,  Chang,  J.&nbsp;L.,  Lindblad-Toh,  K.,  Lander,  E.&nbsp;S.,  Koriabine,
M., Nefedov, M., Osoegawa, K., Yoshinaga, Y., Zhu, B., and de&nbsp;Jong, P.&nbsp;J.
(2007). Identification and analysis of functional elements in 1% of the human
genome by the ENCODE pilot project. <span class="phvro8t-x-x-120">Nature</span>, 447(7146):799–816.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBlankenberg2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Blankenberg,  D.,  Von  Kuster,  G.,  Coraor,  N.,  Ananda,  G.,  Lazarus,  R.,
Mangan, M.,  Nekrutenko,  A.,  and  Taylor,  J.  (2010).   Galaxy:  a  web-based
genome analysis  tool  for  experimentalists.   <span class="phvro8t-x-x-120">Current  protocols  in  molecular</span>
<span class="phvro8t-x-x-120">biology / edited  by  Frederick  M.  Ausubel  ...  [et  al.]</span>,  Chapter  19:Unit
19.10.1–21.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XBlencowe2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Blencowe,        B.&nbsp;J.,        Ahmad,        S.,        and        Lee,        L.&nbsp;J.
(2009). Current-generation high-throughput sequencing: deepening insights
into mammalian transcriptomes. <span class="phvro8t-x-x-120">Genes &amp; development</span>, 23(12):1379–86.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBlower2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Blower, M.&nbsp;D., Jambhekar, A., Schwarz, D.&nbsp;S., and Toombs, J.&nbsp;a. (2013).
Combining  different  mRNA  capture  methods  to  analyze  the  transcriptome:
analysis of the Xenopus laevis transcriptome. <span class="phvro8t-x-x-120">PloS one</span>, 8(10):e77700.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBolcun-Filas2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bolcun-Filas, E., Bannister, L.&nbsp;A., Barash, A., Schimenti, K.&nbsp;J., Hartford,
S.&nbsp;A., Eppig,  J.&nbsp;J.,  Handel,  M.&nbsp;A.,  Shen,  L.,  and  Schimenti,  J.&nbsp;C.  (2011).
A-MYB (MYBL1) transcription factor is a master regulator of male meiosis.
<span class="phvro8t-x-x-120">Development</span>, 138(15):3319 –3330.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBoley2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Boley, N.,  Stoiber,  M.&nbsp;H.,  Booth,  B.&nbsp;W.,  Wan,  K.&nbsp;H.,  Hoskins,  R.&nbsp;a.,
Bickel, P.&nbsp;J.,  Celniker,  S.&nbsp;E.,  and  Brown,  J.&nbsp;B.  (2014).    Genome-guided
transcript assembly by integrative analysis of RNA sequence data.   <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">biotechnology</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBrennecke2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Brennecke, J., Aravin, A.&nbsp;A., Stark, A., Dus, M., Kellis, M., Sachidanandam,
R., and Hannon, G.&nbsp;J. (2007). Discrete small RNA-generating loci as master
regulators of transposon activity in Drosophila. <span class="phvro8t-x-x-120">Cell</span>, 128(6):1089–1103.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBrennecke2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Brennecke, J. and Malone, C. (2008).  An Epigenetic Role for Maternally
Inherited piRNAs   in   Transposon   Silencing.       <span class="phvro8t-x-x-120">Science   (New   York,   </span><span class="phvro8t-x-x-120">…</span>,
322(November):1387–1392.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBrenner2000a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Brenner, S., Johnson, M., Bridgham, J., Golda, G., Lloyd, D.&nbsp;H., Johnson,
D., Luo, S., McCurdy, S., Foy, M., Ewan, M., Roth, R., George, D., Eletr, S.,
Albrecht, G., Vermaas, E., Williams, S.&nbsp;R., Moon, K., Burcham, T., Pallas,
M., DuBridge,  R.&nbsp;B.,  Kirchner,  J.,  Fearon,  K.,  Mao,  J.,  and  Corcoran,  K.
(2000). Gene expression analysis by massively parallel signature sequencing
(MPSS) on microbead arrays. <span class="phvro8t-x-x-120">Nature biotechnology</span>, 18(6):630–4.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBrown2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Brown, J.&nbsp;B.,  Boley,  N.,  Eisman,  R.,  May,  G.&nbsp;E.,  Stoiber,  M.&nbsp;H.,  Duff,
                                                                                

                                                                                
M.&nbsp;O., Booth, B.&nbsp;W., Wen, J., Park, S., Suzuki, A.&nbsp;M., Wan, K.&nbsp;H., Yu, C.,
Zhang, D., Carlson, J.&nbsp;W., Cherbas, L., Eads, B.&nbsp;D., Miller, D., Mockaitis, K.,
Roberts, J., Davis, C.&nbsp;a., Frise, E., Hammonds, A.&nbsp;S., Olson, S., Shenker,
S., Sturgill, D., Samsonova, A.&nbsp;a., Weiszmann, R., Robinson, G., Hernandez,
J., Andrews,  J.,  Bickel,  P.&nbsp;J.,  Carninci,  P.,  Cherbas,  P.,  Gingeras,  T.&nbsp;R.,
Hoskins, R.&nbsp;a., Kaufman, T.&nbsp;C., Lai, E.&nbsp;C., Oliver, B., Perrimon, N., Graveley,
B.&nbsp;R., and Celniker, S.&nbsp;E. (2014).  Diversity and dynamics of the Drosophila
transcriptome. <span class="phvro8t-x-x-120">Nature</span>, pages 1–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBrown2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Brown,   S.&nbsp;J.,   Stoilov,   P.,   and   Xing,   Y.   (2012).       Chromatin   and
epigenetic regulation of pre-mRNA processing.  <span class="phvro8t-x-x-120">Human molecular genetics</span>,
21(R1):R90–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBullard2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Bullard, D. and Bowater, R. (2006). Direct comparison of nick-joining activity
of the nucleic  acid  ligases  from  bacteriophage  T4.    <span class="phvro8t-x-x-120">Biochemical  Journal</span>,
398(Pt 1):135–144.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBurnette2005a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Burnette, J.&nbsp;M., Miyamoto-Sato, E., Schaub, M.&nbsp;a., Conklin, J., and Lopez,
a.&nbsp;J. (2005).  Subdivision of large introns in Drosophila by recursive splicing
at nonexonic elements. <span class="phvro8t-x-x-120">Genetics</span>, 170(2):661–74.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XBurnside2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Burnside, J., Ouyang, M., Anderson, A., Bernberg, E., Lu, C., Meyers, B.&nbsp;C.,
Green, P.&nbsp;J., Markis, M., Isaacs, G., Huang, E., and Morgan, R.&nbsp;W. (2008).
Deep sequencing of chicken microRNAs. <span class="phvro8t-x-x-120">BMC genomics</span>, 9:185.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCalarco2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Calarco,  J.&nbsp;A.,  Saltzman,  A.&nbsp;L.,  Ip,  J.&nbsp;Y.,  and  Blencowe,  B.&nbsp;J.  (2007a).
Technologies for the global discovery and analysis of alternative splicing.  In
<span class="phvro8t-x-x-120">Advances in Experimental Medicine and Biology</span>, volume 623, pages 64–84.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCalarco2007a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Calarco,  J.&nbsp;A.,  Xing,  Y.,  Cáceres,  M.,  Calarco,  J.&nbsp;P.,  Xiao,  X.,  Pan,  Q.,
Lee, C., Preuss,  T.&nbsp;M.,  and  Blencowe,  B.&nbsp;J.  (2007b).   Global  analysis  of
alternative splicing differences between humans and chimpanzees. <span class="phvro8t-x-x-120">Genes &amp;</span>
<span class="phvro8t-x-x-120">Development</span>, 21(22):2963–75.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XCarmell2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Carmell,  M.&nbsp;A.,  Girard,  A.,  van&nbsp;de  Kant,  H.&nbsp;J.,  Bourc’his,  D.,  Bestor,
T.&nbsp;H., de&nbsp;Rooij,  D.&nbsp;G.,  and  Hannon,  G.&nbsp;J.  (2007).    MIWI2  Is  Essential
for Spermatogenesis  and  Repression  of  Transposons  in  the  Mouse  Male
Germline. <span class="phvro8t-x-x-120">Developmental Cell</span>, 12(4):503–514.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCelotto2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Celotto, a.&nbsp;M.  and  Graveley,  B.&nbsp;R.  (2001).    Alternative  splicing  of  the
Drosophila  Dscam  pre-mRNA  is  both  temporally  and  spatially  regulated.
<span class="phvro8t-x-x-120">Genetics</span>, 159(2):599–608.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCenik2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Cenik, E.&nbsp;S. and Zamore, P.&nbsp;D. (2011). Argonaute proteins. <span class="phvro8t-x-x-120">Current biology</span>
<span class="phvro8t-x-x-120">: CB</span>, 21(12):R446–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChang2014b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chang, H., Lim, J., Ha, M., and Kim, V.&nbsp;N. (2014a). TAIL-seq: Genome-wide
Determination of Poly(A) Tail Length and 3’ End Modifications. <span class="phvro8t-x-x-120">Molecular cell</span>,
53(6):1044–52.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChang2014c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chang, Z., Wang, Z., and Li, G. (2014b).  The Impacts of Read Length and
Transcriptome Complexity for De Novo Assembly: A Simulation Study.  <span class="phvro8t-x-x-120">PloS</span>
<span class="phvro8t-x-x-120">one</span>, 9(4):e94825.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChauhan2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chauhan,  A.&nbsp;K.,  Iaconcig,  A.,  Baralle,  F.&nbsp;E.,  and  Muro,  A.&nbsp;F.  (2004).
Alternative splicing of fibronectin: a mouse model demonstrates the identity of
in vitro and in vivo systems and the processing autonomy of regulated exons
in adult mice. <span class="phvro8t-x-x-120">Gene</span>, 324:55–63.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChauleau2013b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chauleau, M. and Shuman, S. (2013).  Kinetic mechanism of nick sealing
by T4 RNA ligase 2 and effects of 3’-OH base mispairs and damaged base
lesions. <span class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>, 19(12):1840–1847.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChen2009a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chen, C.,  Jin,  J.,  James,  D.&nbsp;A.,  Adams-cioaba,  M.&nbsp;A.,  Gyoon,  J.,  Guo,
Y., Tenaglia, E., Xu, C., Gish, G., Min, J., and Pawson, T. (2009).   Mouse
Piwi interactome  identifies  binding  mechanism  of  Tdrkh  Tudor  domain  to
arginine methylated Miwi. <span class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>,
106(48):20336–20341.
                                                                                

                                                                                
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChen2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chen, C., Nott, T.&nbsp;J., Jin, J., and Pawson, T. (2011).  Deciphering arginine
methylation:  Tudor  tells  the  tale.    <span class="phvro8t-x-x-120">Nature  reviews.  Molecular  cell  biology</span>,
12(10):629–42.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChen2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chen, X., Xu, H., Yuan, P., Fang, F., Huss, M., Vega, V.&nbsp;B., Wong, E., Orlov,
Y.&nbsp;L., Zhang,  W.,  Jiang,  J.,  Loh,  Y.-H.,  Yeo,  H.&nbsp;C.,  Yeo,  Z.&nbsp;X.,  Narang,  V.,
Govindarajan, K.&nbsp;R., Leong, B., Shahab, A., Ruan, Y., Bourque, G., Sung,
W.-K., Clarke, N.&nbsp;D., Wei, C.-L., and Ng, H.-H. (2008). Integration of external
signaling pathways with the core transcriptional network in embryonic stem
cells. <span class="phvro8t-x-x-120">Cell</span>, 133(6):1106–17.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChen2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chen, Z. and Duan, X. (2011).   Ribosomal RNA depletion for massively
parallel bacterial RNA-sequencing applications. <span class="phvro8t-x-x-120">Methods in molecular biology</span>
<span class="phvro8t-x-x-120">(Clifton, N.J.)</span>, 733:93–103.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChodosh2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chodosh, L.&nbsp;a. (2001). UV crosslinking of proteins to nucleic acids. <span class="phvro8t-x-x-120">Current</span>
<span class="phvro8t-x-x-120">protocols in molecular biology</span>, Chapter 12:Unit 12.5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChow1977"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chow, L.&nbsp;T., Gelinas, R.&nbsp;E., Broker, T.&nbsp;R., and Roberts, R.&nbsp;J. (1977).  An
amazing sequence arrangement at the 5’ ends of adenovirus 2 messenger
RNA. <span class="phvro8t-x-x-120">Cell</span>, 12(1):1–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChu2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chu, C., Quinn, J., and Chang, H.&nbsp;Y. (2012).  Chromatin isolation by RNA
purification (ChIRP). <span class="phvro8t-x-x-120">Journal of visualized experiments : JoVE</span>, (61).
</p>
<p class="bibitem"><span class="biblabel">
<a id="XChuma2006a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Chuma, S. and Hosokawa, M. (2006).  Tdrd1/Mtr-1 ,a male tudor-related
gene, is essential  for  male  germ-cell  differentiationand  nuage/germinal
grandule formation in mice. <span class="phvro8t-x-x-120">Proceedings of the </span><span class="phvro8t-x-x-120">…</span>, 103(43):1–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHattori2005a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Consortium, I. H. G.&nbsp;S. (2004). Finishing the euchromatic sequence of the
human genome. <span class="phvro8t-x-x-120">Nature</span>, 50(2):162–8.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XConze2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Conze, T.,  Goransson,  J.,  Razzaghian,  H.&nbsp;R.,  Ericsson,  O.,  Oberg,  D.,
Akusjarvi, G.,  Landegren,  U.,  and  Nilsson,  M.  (2010).    Single  molecule
analysis of combinatorial splicing. <span class="phvro8t-x-x-120">Nucl. Acids Res.</span>, page gkq581.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XConze2009c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Conze, T.,  Shetye,  A.,  Tanaka,  Y.,  Gu,  J.,  Larsson,  C.,  Göransson,  J.,
Tavoosidana,  G.,  Söderberg,  O.,  Nilsson,  M.,  and  Landegren,  U.  (2009).
Analysis of genes, transcripts, and proteins via DNA ligation.  <span class="phvro8t-x-x-120">Annual review</span>
<span class="phvro8t-x-x-120">of analytical chemistry (Palo Alto, Calif.)</span>, 2:215–39.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCora2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Cora, E., Pandey, R.&nbsp;R., Xiol, J., Taylor, J., Sachidanandam, R., Mccarthy,
A.&nbsp;A., and  Pillai,  R.&nbsp;S.  (2014).    The  MID-PIWI  module  of  Piwi  proteins
specifies nucleotide- and strand-biases of piRNAs. <span class="phvro8t-x-x-120">RNA</span>, 2014.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCore2008a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Core, L.&nbsp;J., Waterfall, J.&nbsp;J., and Lis, J.&nbsp;T. (2008). Nascent RNA sequencing
reveals widespread  pausing  and  divergent  initiation  at  human  promoters.
<span class="phvro8t-x-x-120">Science</span>, 322:1845–1848.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCox2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Cox, D.&nbsp;N., Chao, A., and Lin, H. (2000).  piwi encodes a nucleoplasmic
factor whose activity modulates the number and division rate of germline stem
cells. <span class="phvro8t-x-x-120">Development (Cambridge, England)</span>, 127(3):503–14.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XCramer1997"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Cramer,  P.,  Pesce,  C.&nbsp;G.,  Baralle,  F.&nbsp;E.,  and  Kornblihtt,  A.&nbsp;R.  (1997).
Functional association between promoter structure and transcript alternative
splicing. <span class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of  Sciences  of  the  United</span>
<span class="phvro8t-x-x-120">States of America</span>, 94(21):11456–60.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDeFazio2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> De Fazio, S., Bartonicek, N., Di Giacomo, M., Abreu-Goodger, C., Sankar,
A., Funaya,  C.,  Antony,  C.,  Moreira,  P.&nbsp;N.,  Enright,  A.&nbsp;J.,  and  O/’Carroll,
D. (2011).   The endonuclease activity of Mili fuels piRNA amplification that
silences LINE1 elements. <span class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDeng1996"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Deng, Q. Q.-l., Ishii, S., and Sarai, A. (1996). Binding site analysis of c-Myb
: screening  of  potential  binding  sites  by  using  the  mutation  matrix  derived
                                                                                

                                                                                
from systematic  binding  affinity  measurements.   <span class="phvro8t-x-x-120">Nucleic  Acids  Research</span>,
24(4):766–774.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDeng2002c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Deng, W. and Lin, H. (2002).   miwi, a Murine Homolog of piwi, Encodes
a Cytoplasmic Protein Essential for Spermatogenesis.  <span class="phvro8t-x-x-120">Developmental cell</span>,
2(6):819–830.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDerrien2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Derrien, T., Johnson, R., Bussotti, G., Tanzer, A., Djebali, S., Tilgner, H.,
Guernec, G., Martin, D., Merkel, A., Knowles, D.&nbsp;G., Lagarde, J., Veeravalli,
L., Ruan, X.,  Ruan,  Y.,  Lassmann,  T.,  Carninci,  P.,  Brown,  J.&nbsp;B.,  Lipovich,
L., Gonzalez,  J.&nbsp;M.,  Thomas,  M.,  Davis,  C.&nbsp;a.,  Shiekhattar,  R.,  Gingeras,
T.&nbsp;R., Hubbard, T.&nbsp;J., Notredame, C., Harrow, J., and Guigó, R. (2012).  The
GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene
structure, evolution, and expression. <span class="phvro8t-x-x-120">Genome research</span>, 22(9):1775–89.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDiGiacomo2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Di Giacomo,  M.,  Comazzetto,  S.,  Saini,  H.,  De  Fazio,  S.,  Carrieri,  C.,
Morgan, M.,  Vasiliauskaite,  L.,  Benes,  V.,  Enright,  A.&nbsp;J.,  and  O’Carroll,  D.
(2013). Multiple  epigenetic  mechanisms  and  the  piRNA  pathway  enforce
LINE1 silencing during adult spermatogenesis. <span class="phvro8t-x-x-120">Molecular cell</span>, 50(4):601–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDjebali2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Djebali, S., Davis, C.&nbsp;a., Merkel, A., Dobin, A., Lassmann, T., Mortazavi,
A., Tanzer,  A.,  Lagarde,  J.,  Lin,  W.,  Schlesinger,  F.,  Xue,  C.,  Marinov,
G.&nbsp;K., Khatun,  J.,  Williams,  B.&nbsp;a.,  Zaleski,  C.,  Rozowsky,  J.,  Röder,  M.,
Kokocinski,  F.,  Abdelhamid,  R.&nbsp;F.,  Alioto,  T.,  Antoshechkin,  I.,  Baer,  M.&nbsp;T.,
Bar, N.&nbsp;S.,  Batut,  P.,  Bell,  K.,  Bell,  I.,  Chakrabortty,  S.,  Chen,  X.,  Chrast,
J., Curado, J., Derrien, T., Drenkow, J., Dumais, E., Dumais, J., Duttagupta,
R., Falconnet,  E.,  Fastuca,  M.,  Fejes-Toth,  K.,  Ferreira,  P.,  Foissac,  S.,
Fullwood, M.&nbsp;J.,  Gao,  H.,  Gonzalez,  D.,  Gordon,  A.,  Gunawardena,  H.,
Howald, C.,  Jha,  S.,  Johnson,  R.,  Kapranov,  P.,  King,  B.,  Kingswood,  C.,
Luo, O.&nbsp;J., Park, E., Persaud, K., Preall, J.&nbsp;B., Ribeca, P., Risk, B., Robyr,
D., Sammeth, M., Schaffer, L., See, L.-H., Shahab, A., Skancke, J., Suzuki,
A.&nbsp;M., Takahashi, H., Tilgner, H., Trout, D., Walters, N., Wang, H., Wrobel,
J., Yu, Y., Ruan, X., Hayashizaki, Y., Harrow, J., Gerstein, M., Hubbard, T.,
Reymond, A.,  Antonarakis,  S.&nbsp;E.,  Hannon,  G.,  Giddings,  M.&nbsp;C.,  Ruan,  Y.,
Wold, B., Carninci, P., Guigó, R., and Gingeras, T.&nbsp;R. (2012).  Landscape of
transcription in human cells. <span class="phvro8t-x-x-120">Nature</span>, 489(7414):101–8.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XDjebali2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Djebali, S.,  Kapranov,  P.,  Foissac,  S.,  Lagarde,  J.,  Reymond,  A.,  Ucla,
C., Wyss, C.,  Drenkow,  J.,  Dumais,  E.,  Murray,  R.&nbsp;R.,  Lin,  C.,  Szeto,  D.,
Denoeud, F.,  Calvo,  M.,  Frankish,  A.,  Harrow,  J.,  Makrythanasis,  P.,  Vidal,
M., Salehi-Ashtiani, K., Antonarakis, S.&nbsp;E., Gingeras, T.&nbsp;R., and Guigo, R.
(2008). Efficient targeted transcript discovery via array-based normalization
of RACE libraries. <span class="phvro8t-x-x-120">Nat Meth</span>, 5(7):629–635.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDong2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Dong, Y.,  Taylor,  H.&nbsp;E.,  and  Dimopoulos,  G.  (2006).     AgDscam,  a
hypervariable immunoglobulin domain-containing receptor of the Anopheles
gambiae innate immune system. <span class="phvro8t-x-x-120">PLoS Biology</span>, 4:1137–1146.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XDunham2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Dunham, I., Kundaje, A., Aldred, S.&nbsp;F., Collins, P.&nbsp;J., Davis, C.&nbsp;a., Doyle,
F., Epstein, C.&nbsp;B., Frietze, S., Harrow, J., Kaul, R., Khatun, J., Lajoie, B.&nbsp;R.,
Landt, S.&nbsp;G.,  Lee,  B.-K.,  Pauli,  F.,  Rosenbloom,  K.&nbsp;R.,  Sabo,  P.,  Safi,  A.,
Sanyal, A., Shoresh, N., Simon, J.&nbsp;M., Song, L., Trinklein, N.&nbsp;D., Altshuler,
R.&nbsp;C., Birney,  E.,  Brown,  J.&nbsp;B.,  Cheng,  C.,  Djebali,  S.,  Dong,  X.,  Ernst,
J., Furey, T.&nbsp;S.,  Gerstein,  M.,  Giardine,  B.,  Greven,  M.,  Hardison,  R.&nbsp;C.,
Harris, R.&nbsp;S., Herrero, J., Hoffman, M.&nbsp;M., Iyer, S., Kelllis, M., Kheradpour, P.,
Lassman, T., Li, Q., Lin, X., Marinov, G.&nbsp;K., Merkel, A., Mortazavi, A., Parker,
S. C.&nbsp;J., Reddy, T.&nbsp;E., Rozowsky, J., Schlesinger, F., Thurman, R.&nbsp;E., Wang,
J., Ward, L.&nbsp;D., Whitfield, T.&nbsp;W., Wilder, S.&nbsp;P., Wu, W., Xi, H.&nbsp;S., Yip, K.&nbsp;Y.,
Zhuang, J.,  Bernstein,  B.&nbsp;E.,  Green,  E.&nbsp;D.,  Gunter,  C.,  Snyder,  M.,  Pazin,
M.&nbsp;J., Lowdon,  R.&nbsp;F.,  Dillon,  L.  a.&nbsp;L.,  Adams,  L.&nbsp;B.,  Kelly,  C.&nbsp;J.,  Zhang,
J., Wexler, J.&nbsp;R., Good, P.&nbsp;J., Feingold, E.&nbsp;a., Crawford, G.&nbsp;E., Dekker, J.,
Elinitski, L.,  Farnham,  P.&nbsp;J.,  Giddings,  M.&nbsp;C.,  Gingeras,  T.&nbsp;R.,  Guigó,  R.,
Hubbard, T.&nbsp;J., Kellis, M., Kent, W.&nbsp;J., Lieb, J.&nbsp;D., Margulies, E.&nbsp;H., Myers,
R.&nbsp;M., Starnatoyannopoulos,  J.&nbsp;a.,  Tennebaum,  S.&nbsp;a.,  Weng,  Z.,  White,
K.&nbsp;P., Wold, B., Yu, Y., Wrobel, J., Risk, B.&nbsp;a., Gunawardena, H.&nbsp;P., Kuiper,
H.&nbsp;C., Maier, C.&nbsp;W., Xie, L., Chen, X., Mikkelsen, T.&nbsp;S., Gillespie, S., Goren,
A., Ram, O., Zhang, X., Wang, L., Issner, R., Coyne, M.&nbsp;J., Durham, T., Ku,
M., Truong, T., Eaton, M.&nbsp;L., Dobin, A., Lassmann, T., Tanzer, A., Lagarde,
J., Lin, W., Xue, C., Williams, B.&nbsp;a., Zaleski, C., Röder, M., Kokocinski, F.,
Abdelhamid,  R.&nbsp;F.,  Alioto,  T.,  Antoshechkin,  I.,  Baer,  M.&nbsp;T.,  Batut,  P.,  Bell,
I., Bell, K.,  Chakrabortty,  S.,  Chrast,  J.,  Curado,  J.,  Derrien,  T.,  Drenkow,
J., Dumais,  E.,  Dumais,  J.,  Duttagupta,  R.,  Fastuca,  M.,  Fejes-Toth,  K.,
Ferreira, P.,  Foissac,  S.,  Fullwood,  M.&nbsp;J.,  Gao,  H.,  Gonzalez,  D.,  Gordon,
A., Howald, C., Jha, S., Johnson, R., Kapranov, P., King, B., Kingswood, C.,
Li, G., Luo, O.&nbsp;J., Park, E., Preall, J.&nbsp;B., Presaud, K., Ribeca, P., Robyr, D.,
                                                                                

                                                                                
Ruan, X., Sammeth, M., Sandu, K.&nbsp;S., Schaeffer, L., See, L.-H., Shahab, A.,
Skancke, J., Suzuki, A.&nbsp;M., Takahashi, H., Tilgner, H., Trout, D., Walters, N.,
Wang, H., Hayashizaki, Y., Hubbard, T.&nbsp;J., Reymond, A., Antonarakis, S.&nbsp;E.,
Hannon, G.&nbsp;J.,  Ruan,  Y.,  Carninci,  P.,  Sloan,  C.&nbsp;a.,  Learned,  K.,  Malladi,
V.&nbsp;S., Wong,  M.&nbsp;C.,  Barber,  G.&nbsp;P.,  Cline,  M.&nbsp;S.,  Dreszer,  T.&nbsp;R.,  Heitner,
S.&nbsp;G., Karolchik, D., Kirkup, V.&nbsp;M., Meyer, L.&nbsp;R., Long, J.&nbsp;C., Maddren, M.,
Raney, B.&nbsp;J.,  Grasfeder,  L.&nbsp;L.,  Giresi,  P.&nbsp;G.,  Lee,  B.-K.,  Battenhouse,  A.,
Sheffield, N.&nbsp;C.,  Showers,  K.&nbsp;a.,  London,  D.,  Bhinge,  A.&nbsp;a.,  Shestak,  C.,
Schaner, M.&nbsp;R., Kim, S.&nbsp;K., Zhang, Z.&nbsp;Z., Mieczkowski, P.&nbsp;a., Mieczkowska,
J.&nbsp;O., Liu,  Z.,  McDaniell,  R.&nbsp;M.,  Ni,  Y.,  Rashid,  N.&nbsp;U.,  Kim,  M.&nbsp;J.,  Adar,
S., Zhang,  Z.,  Wang,  T.,  Winter,  D.,  Keefe,  D.,  Iyer,  V.&nbsp;R.,  Sandhu,  K.&nbsp;S.,
Zheng, M.,  Wang,  P.,  Gertz,  J.,  Vielmetter,  J.,  Partridge,  E.&nbsp;C.,  Varley,
K.&nbsp;E., Gasper,  C.,  Bansal,  A.,  Pepke,  S.,  Jain,  P.,  Amrhein,  H.,  Bowling,
K.&nbsp;M., Anaya, M., Cross, M.&nbsp;K., Muratet, M.&nbsp;a., Newberry, K.&nbsp;M., McCue,
K., Nesmith, A.&nbsp;S., Fisher-Aylor, K.&nbsp;I., Pusey, B., DeSalvo, G., Parker, S.&nbsp;L.,
Balasubramanian, S., Davis, N.&nbsp;S., Meadows, S.&nbsp;K., Eggleston, T., Newberry,
J.&nbsp;S., Levy,  S.&nbsp;E.,  Absher,  D.&nbsp;M.,  Wong,  W.&nbsp;H.,  Blow,  M.&nbsp;J.,  Visel,  A.,
Pennachio,  L.&nbsp;a.,  Elnitski,  L.,  Petrykowska,  H.&nbsp;M.,  Abyzov,  A.,  Aken,  B.,
Barrell, D.,  Barson,  G.,  Berry,  A.,  Bignell,  A.,  Boychenko,  V.,  Bussotti,  G.,
Davidson, C., Despacio-Reyes, G., Diekhans, M., Ezkurdia, I., Frankish, A.,
Gilbert, J.,  Gonzalez,  J.&nbsp;M.,  Griffiths,  E.,  Harte,  R.,  Hendrix,  D.&nbsp;a.,  Hunt,
T., Jungreis,  I.,  Kay,  M.,  Khurana,  E.,  Leng,  J.,  Lin,  M.&nbsp;F.,  Loveland,  J.,
Lu, Z., Manthravadi, D., Mariotti, M., Mudge, J., Mukherjee, G., Notredame,
C., Pei, B.,  Rodriguez,  J.&nbsp;M.,  Saunders,  G.,  Sboner,  A.,  Searle,  S.,  Sisu,
C., Snow, C.,  Steward,  C.,  Tapanari,  E.,  Tress,  M.&nbsp;L.,  van  Baren,  M.&nbsp;J.,
Washieti, S., Wilming, L., Zadissa, A., Zhengdong, Z., Brent, M., Haussler,
D., Valencia,  A.,  Raymond,  A.,  Addleman,  N.,  Alexander,  R.&nbsp;P.,  Auerbach,
R.&nbsp;K., Balasubramanian, S., Bettinger, K., Bhardwaj, N., Boyle, A.&nbsp;P., Cao,
A.&nbsp;R., Cayting,  P.,  Charos,  A.,  Cheng,  Y.,  Eastman,  C.,  Euskirchen,  G.,
Fleming, J.&nbsp;D.,  Grubert,  F.,  Habegger,  L.,  Hariharan,  M.,  Harmanci,  A.,
Iyenger, S., Jin, V.&nbsp;X., Karczewski, K.&nbsp;J., Kasowski, M., Lacroute, P., Lam, H.,
Larnarre-Vincent, N., Lian, J., Lindahl-Allen, M., Min, R., Miotto, B., Monahan,
H., Moqtaderi,  Z.,  Mu,  X.&nbsp;J.,  O’Geen,  H.,  Ouyang,  Z.,  Patacsil,  D.,  Raha,
D., Ramirez, L., Reed, B., Shi, M., Slifer, T., Witt, H., Wu, L., Xu, X., Yan,
K.-K., Yang, X., Zhang, Z., Struhl, K., Weissman, S.&nbsp;M., Tenebaum, S.&nbsp;a.,
Penalva, L.&nbsp;O., Karmakar, S., Bhanvadia, R.&nbsp;R., Choudhury, A., Domanus,
M., Ma, L., Moran, J., Victorsen, A., Auer, T., Centarin, L., Eichenlaub, M.,
Gruhl, F., Heerman, S., Hoeckendorf, B., Inoue, D., Kellner, T., Kirchmaier, S.,
Mueller, C., Reinhardt, R., Schertel, L., Schneider, S., Sinn, R., Wittbrodt, B.,
Wittbrodt, J., Jain, G., Balasundaram, G., Bates, D.&nbsp;L., Byron, R., Canfield,
T.&nbsp;K., Diegel, M.&nbsp;J., Dunn, D., Ebersol, A.&nbsp;K., Frum, T., Garg, K., Gist, E.,
                                                                                

                                                                                
Hansen, R.&nbsp;S.,  Boatman,  L.,  Haugen,  E.,  Humbert,  R.,  Johnson,  A.&nbsp;K.,
Johnson, E.&nbsp;M., Kutyavin, T.&nbsp;M., Lee, K., Lotakis, D., Maurano, M.&nbsp;T., Neph,
S.&nbsp;J., Neri, F.&nbsp;V., Nguyen, E.&nbsp;D., Qu, H., Reynolds, A.&nbsp;P., Roach, V., Rynes,
E., Sanchez,  M.&nbsp;E.,  Sandstrom,  R.&nbsp;S.,  Shafer,  A.&nbsp;O.,  Stergachis,  A.&nbsp;B.,
Thomas, S.,  Vernot,  B.,  Vierstra,  J.,  Vong,  S.,  Wang,  H.,  Weaver,  M.&nbsp;a.,
Yan, Y., Zhang, M., Akey, J.&nbsp;a., Bender, M., Dorschner, M.&nbsp;O., Groudine, M.,
MacCoss, M.&nbsp;J., Navas, P., Stamatoyannopoulos, G., Stamatoyannopoulos,
J.&nbsp;a., Beal,  K.,  Brazma,  A.,  Flicek,  P.,  Johnson,  N.,  Lukk,  M.,  Luscombe,
N.&nbsp;M., Sobral, D., Vaquerizas, J.&nbsp;M., Batzoglou, S., Sidow, A., Hussami, N.,
Kyriazopoulou-Panagiotopoulou, S., Libbrecht, M.&nbsp;W., Schaub, M.&nbsp;a., Miller,
W., Bickel, P.&nbsp;J., Banfai, B., Boley, N.&nbsp;P., Huang, H., Li, J.&nbsp;J., Noble, W.&nbsp;S.,
Bilmes, J.&nbsp;a.,  Buske,  O.&nbsp;J.,  Sahu,  A.&nbsp;O.,  Kharchenko,  P.&nbsp;V.,  Park,  P.&nbsp;J.,
Baker, D., Taylor, J., and Lochovsky, L. (2012). An integrated encyclopedia of
DNA elements in the human genome. <span class="phvro8t-x-x-120">Nature</span>, 489(7414):57–74.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XEmerick2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Emerick, M., Parmigiani, G., and Agnew, W. (2007).  Multivariate Analysis
and Visualization  of  Splicing  Correlations  in  Single-Gene  Transcriptomes.
<span class="phvro8t-x-x-120">BMC Bioinformatics</span>, 8(1):16.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XEngreitz2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Engreitz,  J.&nbsp;M.,  Pandya-Jones,  A.,  McDonel,  P.,  Shishkin,  A.,  Sirokman,
K., Surka, C.,  Kadri,  S.,  Xing,  J.,  Goren,  A.,  Lander,  E.&nbsp;S.,  Plath,  K.,  and
Guttman, M. (2013).   The Xist lncRNA exploits three-dimensional genome
architecture to spread across the X chromosome.  <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
341(6147):1237973.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XEngstrom2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Engström,  P.&nbsp;G.,  Steijger,  T.,  Sipos,  B.,  Grant,  G.&nbsp;R.,  Kahles,  A.,  Alioto,
T., Behr, J.,  Bertone,  P.,  Bohnert,  R.,  Campagna,  D.,  Davis,  C.&nbsp;A.,  Dobin,
A., Gingeras,  T.&nbsp;R.,  Goldman,  N.,  Guigó,  R.,  Harrow,  J.,  Hubbard,  T.&nbsp;J.,
Jean, G., Kosarev, P., Li, S., Liu, J., Mason, C.&nbsp;E., Molodtsov, V., Ning, Z.,
Ponstingl, H., Prins, J.&nbsp;F., Rätsch, G., Ribeca, P., Seledtsov, I., Solovyev, V.,
Valle, G., Vitulo, N., Wang, K., Wu, T.&nbsp;D., and Zeller, G. (2013).  Systematic
evaluation of spliced alignment programs for RNA-seq data. <span class="phvro8t-x-x-120">Nature methods</span>,
10(12):1185–91.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFagnani2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Fagnani, M., Barash, Y., Ip, J.&nbsp;Y., Misquitta, C., Pan, Q., Saltzman, A.&nbsp;L.,
Shai, O., Lee,  L.,  Rozenhek,  A.,  Mohammad,  N.,  Willaime-Morawek,  S.,
Babak, T.,  Zhang,  W.,  Hughes,  T.&nbsp;R.,  van&nbsp;der  Kooy,  D.,  Frey,  B.&nbsp;J.,  and
                                                                                

                                                                                
Blencowe, B.&nbsp;J. (2007).  Functional coordination of alternative splicing in the
mammalian central nervous system. <span class="phvro8t-x-x-120">Genome Biology</span>, 8(6):R108.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFarazi2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Farazi, T.&nbsp;a., Juranek, S.&nbsp;a., and Tuschl, T. (2008).  The growing catalog
of small RNAs  and  their  association  with  distinct  Argonaute/Piwi  family
members. <span class="phvro8t-x-x-120">Development (Cambridge, England)</span>, 135(7):1201–14.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFareed1971"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Fareed, G.&nbsp;C.,  Wilt,  E.&nbsp;M.,  and  Richardson,  C.&nbsp;C.  (1971).    Enzymatic
Breakage and  Joining  of  Deoxyribonucleic  Acid.     <span class="phvro8t-x-x-120">Journal  of  Biological</span>
<span class="phvro8t-x-x-120">Chemistry</span>, 246:925–932.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFededa2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Fededa,  J.&nbsp;P.,  Petrillo,  E.,  Gelfand,  M.&nbsp;S.,  Neverov,  A.&nbsp;D.,  Kadener,  S.,
Nogués, G.,  Pelisch,  F.,  Baralle,  F.&nbsp;E.,  Muro,  A.&nbsp;F.,  and  Kornblihtt,  A.&nbsp;R.
(2005). A  polar  mechanism  coordinates  different  regions  of  alternative
splicing within a single gene. <span class="phvro8t-x-x-120">Molecular cell</span>, 19(3):393–404.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFire1998"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Fire, A., Xu, S., Montgomery, M.&nbsp;K., Kostas, S.&nbsp;A., Driver, S.&nbsp;E., and Mello,
C.&nbsp;C. (1998).   Potent and specific genetic interference by double-stranded
RNA in Caenorhabditis elegans. <span class="phvro8t-x-x-120">Nature</span>, 391:806–811.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFodor2009a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Fodor, A.&nbsp;a. and Aldrich, R.&nbsp;W. (2009). Convergent evolution of alternative
splices at domain boundaries of the BK channel. <span class="phvro8t-x-x-120">Annual review of physiology</span>,
71:19–36.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XFuruichi1975"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Furuichi, Y. (1975). 5’-terminal m7G(5’)ppp(5’)Gmp In Vivo: Identification in
Reovirus Genome RNA. <span class="phvro8t-x-x-120">Proceedings of the </span><span class="phvro8t-x-x-120">…</span>, 72(2):742–745.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGan2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gan, H., Lin, X., Zhang, Z., Zhang, W., Liao, S., Wang, L., and Han, C.
(2011). piRNA profiling during specific stages of mouse spermatogenesis.
<span class="phvro8t-x-x-120">RNA</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGan2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gan, H., Wen, L., Liao, S., Lin, X., Ma, T., Liu, J., Song, C.-X., Wang, M.,
He, C., Han, C., and Tang, F. (2013).  Dynamics of 5-hydroxymethylcytosine
                                                                                

                                                                                
during mouse spermatogenesis. <span class="phvro8t-x-x-120">Nature communications</span>, 4:1995.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGarber2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Garber,  M.,  Grabherr,  M.&nbsp;G.,  Guttman,  M.,  and  Trapnell,  C.  (2011).
Computational methods for transcriptome annotation and quantification using
RNA-seq. <span class="phvro8t-x-x-120">Nature Methods</span>, 8(6):469–477.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGhildiyal2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ghildiyal, M., Seitz, H., Horwich, M.&nbsp;D., Li, C., Du, T., Lee, S., Xu, J., Kittler,
E. L.&nbsp;W., Zapp, M.&nbsp;L., Weng, Z., and Zamore, P.&nbsp;D. (2008).   Endogenous
siRNAs Derived from Transposons and mRNAs in Drosophila Somatic Cells.
<span class="phvro8t-x-x-120">Science</span>, 320(5879):1077–1081.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGhosh2010a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ghosh, S. and Jacobson, A. (2010). RNA decay modulates gene expression
and controls its fidelity. <span class="phvro8t-x-x-120">WIREs RNA</span>, 1:351–361.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGilbert1978a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gilbert, W. (1978). Why genes in pieces? <span class="phvro8t-x-x-120">Nature</span>, 271(5645):501.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGirard2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Girard, A., Sachidanandam, R., Hannon, G.&nbsp;J., and Carmell, M.&nbsp;A. (2006).
A germline-specific  class  of  small  RNAs  binds  mammalian  Piwi  proteins.
<span class="phvro8t-x-x-120">Nature</span>, 442(7099):199–202.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGlauser2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Glauser,  D.&nbsp;A.,  Johnson,  B.&nbsp;E.,  Aldrich,  R.&nbsp;W.,  and  Goodman,  M.&nbsp;B.
(2011).       Intragenic   alternative   splicing   coordination   is   essential   for
Caenorhabditis  elegans  slo-1  gene  function.   <span class="phvro8t-x-x-120">Proceedings  of  the  National</span>
<span class="phvro8t-x-x-120">Academy of Sciences of the United States of America</span>, 108(51):20790–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGoriaux2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Goriaux,  C.,  Desset,  S.,  Renaud,  Y.,  Vaury,  C.,  and  Brasset,  E.  (2014).
Transcriptional properties and splicing of the flamenco piRNA cluster. <span class="phvro8t-x-x-120">EMBO</span>
<span class="phvro8t-x-x-120">reports</span>, 15(4):411–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGrabherr2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Grabherr, M.&nbsp;G., Haas, B.&nbsp;J., Yassour, M., Levin, J.&nbsp;Z., Thompson, D.&nbsp;A.,
Amit, I., Adiconis, X., Fan, L., Raychowdhury, R., Zeng, Q., Chen, Z., Mauceli,
E., Hacohen, N., Gnirke, A., Rhind, N., di&nbsp;Palma, F., Birren, B.&nbsp;W., Nusbaum,
C., Lindblad-Toh,  K.,  Friedman,  N.,  and  Regev,  A.  (2011).     Full-length
                                                                                

                                                                                
transcriptome  assembly  from  RNA-Seq  data  without  a  reference  genome.
<span class="phvro8t-x-x-120">Nature Biotechnology</span>, 29(7):644–652.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGrant2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Grant, C.&nbsp;E., Bailey, T.&nbsp;L., and Noble, W.&nbsp;S. (2011).  FIMO: scanning for
occurrences of a given motif. <span class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 27(7):1017–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGraur2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Graur, D., Zheng, Y., Price, N., Azevedo, R. B.&nbsp;R., Zufall, R.&nbsp;a., and Elhaik,
E. (2013).   On  the  immortality  of  television  sets:  "function"  in  the  human
genome according to the evolution-free gospel of ENCODE. <span class="phvro8t-x-x-120">Genome biology</span>
<span class="phvro8t-x-x-120">and evolution</span>, 5(3):578–590.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGraveley2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Graveley, B.&nbsp;R. (2000).  Sorting out the complexity of SR protein functions.
<span class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>, 6(9):1197–211.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGraveley2005b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Graveley, B.&nbsp;R. (2005).   Mutually exclusive splicing of the insect Dscam
pre-mRNA directed by competing intronic RNA secondary structures.  <span class="phvro8t-x-x-120">Cell</span>,
123(1):65–73.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGregersen2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gregersen, L.&nbsp;H., Schueler, M., Munschauer, M., Mastrobuoni, G., Chen,
W., Kempa,  S.,  Dieterich,  C.,  and  Landthaler,  M.  (2014).   MOV10  Is  a  5’
to 3’ RNA  Helicase  Contributing  to  UPF1  mRNA  Target  Degradation  by
Translocation along 3’ UTRs. <span class="phvro8t-x-x-120">Molecular cell</span>, pages 1–13.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGrivna2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Grivna, S.&nbsp;T.,  Beyret,  E.,  Wang,  Z.,  and  Lin,  H.  (2006).   A  novel  class
of small RNAs  in  mouse  spermatogenic  cells.     <span class="phvro8t-x-x-120">Genes  &amp;  development</span>,
20(13):1709–14.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGu2010a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gu, A., Ji, G., Shi, X., Long, Y., Xia, Y., Song, L., Wang, S., and Wang,
X. (2010). Genetic Variants in Piwi-Interacting RNA Pathway Genes Confer
Susceptibility  to  Spermatogenic  Failure  in  a  Chinese  Population.   <span class="phvro8t-x-x-120">Human</span>
<span class="phvro8t-x-x-120">Reproduction</span>, 25(12):2955–2961.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGu2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gu, W., Lee, H.-C., Chaves, D., Youngman, E.&nbsp;M., Pazour, G.&nbsp;J., Conte,
                                                                                

                                                                                
D., and Mello, C.&nbsp;C. (2012).  CapSeq and CIP-TAP identify Pol II start sites
and reveal  capped  small  RNAs  as  C.  elegans  piRNA  precursors.    <span class="phvro8t-x-x-120">Cell</span>,
151(7):1488–500.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGuenther2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Guenther,  M.&nbsp;G.,  Levine,  S.&nbsp;S.,  Boyer,  L.&nbsp;A.,  Jaenisch,  R.,  and  Young,
R.&nbsp;A. (2007).    A  chromatin  landmark  and  transcription  initiation  at  most
promoters in human cells. <span class="phvro8t-x-x-120">Cell</span>, 130:77–88.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGunawardane2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gunawardane, L.&nbsp;S., Saito, K., Nishida, K.&nbsp;M., Miyoshi, K., Kawamura, Y.,
Nagami, T., Siomi, H., and Siomi, M.&nbsp;C. (2007). A slicer-mediated mechanism
for repeat-associated siRNA 5’ end formation in Drosophila.  <span class="phvro8t-x-x-120">Science (New</span>
<span class="phvro8t-x-x-120">York, N.Y.)</span>, 315(5818):1587–90.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGupta2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Gupta, S.,  Stamatoyannopoulos,  J.&nbsp;a.,  Bailey,  T.&nbsp;L.,  and  Noble,  W.&nbsp;S.
(2007). Quantifying similarity between motifs. <span class="phvro8t-x-x-120">Genome biology</span>, 8(2):R24.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGuttman2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Guttman,  M.,  Amit,  I.,  Garber,  M.,  French,  C.,  Lin,  M.&nbsp;F.,  Feldser,  D.,
Huarte, M., Zuk, O., Carey, B.&nbsp;W., Cassady, J.&nbsp;P., Cabili, M.&nbsp;N., Jaenisch, R.,
Mikkelsen, T.&nbsp;S., Jacks, T., Hacohen, N., Bernstein, B.&nbsp;E., Kellis, M., Regev,
A., Rinn, J.&nbsp;L., and Lander, E.&nbsp;S. (2009).  Chromatin signature reveals over
a thousand highly conserved large non-coding RNAs in mammals.  <span class="phvro8t-x-x-120">Nature</span>,
458(7235):223–227.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGuttman2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Guttman, M., Garber, M., Levin, J.&nbsp;Z., Donaghey, J., Robinson, J., Adiconis,
X., Fan, L.,  Koziol,  M.&nbsp;J.,  Gnirke,  A.,  Nusbaum,  C.,  Rinn,  J.&nbsp;L.,  Lander,
E.&nbsp;S., and  Regev,  A.  (2010).   Ab  initio  reconstruction  of  cell  type-specific
transcriptomes  in  mouse  reveals  the  conserved  multi-exonic  structure  of
lincRNAs. <span class="phvro8t-x-x-120">Nature Biotechnology</span>, 28(5):503–510.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XGuttman2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Guttman, M., Russell, P., Ingolia, N.&nbsp;T., Weissman, J.&nbsp;S., and Lander, E.&nbsp;S.
(2013). Ribosome Profiling Provides Evidence that Large Noncoding RNAs
Do Not Encode Proteins. <span class="phvro8t-x-x-120">Cell</span>, 154(1):1–12.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHaas2013c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Haas, B.&nbsp;J.,  Papanicolaou,  A.,  Yassour,  M.,  Grabherr,  M.,  Blood,  P.&nbsp;D.,
                                                                                

                                                                                
Bowden, J., Couger, M.&nbsp;B., Eccles, D., Li, B., Lieber, M., Macmanes, M.&nbsp;D.,
Ott, M., Orvis, J., Pochet, N., Strozzi, F., Weeks, N., Westerman, R., William,
T., Dewey, C.&nbsp;N., Henschel, R., LeDuc, R.&nbsp;D., Friedman, N., and Regev, A.
(2013). De  novo  transcript  sequence  reconstruction  from  RNA-seq  using
the Trinity platform for reference generation and analysis.  <span class="phvro8t-x-x-120">Nature Protocols</span>,
8(8):1494–1512.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHaase2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Haase, A.&nbsp;D.,  Fenoglio,  S.,  Muerdter,  F.,  Guzzardo,  P.&nbsp;M.,  Czech,  B.,
Pappin, D.&nbsp;J., Chen, C., Gordon, A., and Hannon, G.&nbsp;J. (2010).  Probing the
initiation and effector phases of the somatic piRNA pathway in Drosophila.
<span class="phvro8t-x-x-120">Genes &amp; development</span>, 24(22):2499–504.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHafner2008a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hafner, M.,  Landgraf,  P.,  Ludwig,  J.,  Rice,  A.,  Ojo,  T.,  Lin,  C.,  Holoch,
D., Lim, C.,  and  Tuschl,  T.  (2008).   Identification  of  microRNAs  and  other
small regulatory RNAs using cDNA library sequencing. <span class="phvro8t-x-x-120">Methods (San Diego,</span>
<span class="phvro8t-x-x-120">Calif.)</span>, 44(1):3–12.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHafner2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hafner, M., Landthaler, M., Burger, L., Khorshid, M., Hausser, J., Berninger,
P., Rothballer,  A.,  Ascano  Jr.,  M.,  Jungkamp,  A.-C.,  Munschauer,  M.,
Ulrich, A., Wardle,  G.&nbsp;S.,  Dewell,  S.,  Zavolan,  M.,  and  Tuschl,  T.  (2010).
Transcriptome-wide  Identification  of  RNA-Binding  Protein  and  MicroRNA
Target Sites by PAR-CLIP. <span class="phvro8t-x-x-120">Cell</span>, 141(1):129–141.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHandler2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Handler,  D.,  Olivieri,  D.,  Novatchkova,  M.,  Gruber,  F.&nbsp;S.,  Meixner,  K.,
Mechtler, K.,  Stark,  A.,  Sachidanandam,  R.,  and  Brennecke,  J.  (2011).
A systematic  analysis  of  Drosophila  TUDOR  domain-containing  proteins
identifies Vreteno and the Tdrd12 family as essential primary piRNA pathway
factors. <span class="phvro8t-x-x-120">The EMBO journal</span>, 30(19):3977–93.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHartig2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hartig, J.&nbsp;V., Tomari, Y., and Förstemann, K. (2007).  piRNAs–the ancient
hunters of genome invaders. <span class="phvro8t-x-x-120">Genes &amp; development</span>, 21(14):1707–13.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHattori2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hattori, D., Chen, Y., Matthews, B.&nbsp;J., Salwinski, L., Sabatti, C., Grueber,
W.&nbsp;B., and  Zipursky,  S.&nbsp;L.  (2009).     Robust  discrimination  between  self
and non-self  neurites  requires  thousands  of  Dscam1  isoforms.    <span class="phvro8t-x-x-120">Nature</span>,
                                                                                

                                                                                
461(7264):644–648.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHattori2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hattori,  D.,  Demir,  E.,  Kim,  H.&nbsp;W.,  Viragh,  E.,  Zipursky,  S.&nbsp;L.,  and
Dickson, B.&nbsp;J. (2007).  Dscam diversity is essential for neuronal wiring and
self-recognition. <span class="phvro8t-x-x-120">Nature</span>, 449(7159):223–227.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHattori2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hattori, D.,  Millard,  S.&nbsp;S.,  Wojtowicz,  W.&nbsp;M.,  and  Zipursky,  S.&nbsp;L.  (2008).
Dscam-Mediated   Cell   Recognition   Regulates   Neural   Circuit   Formation.
<span class="phvro8t-x-x-120">Annual Review of Cell and Developmental Biology</span>, 24(1):597–620.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHelwak2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Helwak, A. and Tollervey, D. (2014).  Mapping the miRNA interactome by
cross-linking ligation and sequencing of hybrids (CLASH).  <span class="phvro8t-x-x-120">Nature protocols</span>,
9(3):711–28.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHemani2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hemani,  Y.  and  Soller,  M.  (2012).    Mechanisms  of  Drosophila  Dscam
mutually exclusive  splicing  regulation.    <span class="phvro8t-x-x-120">Biochemical  Society  transactions</span>,
40(4):804–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHirose2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hirose, T., Mishima, Y., and Tomari, Y. (2014).  Elements and machinery of
non-coding RNAs : toward their taxonomy. <span class="phvro8t-x-x-120">EMBO reports</span>, pages 1–19.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHo1997b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ho, C.&nbsp;K., Etten, J. L.&nbsp;V., and Shuman, S. (1997).  Characterization of an
ATP-dependent DNA ligase encoded by Chlorella virus PBCV-1.  <span class="phvro8t-x-x-120">Journal of</span>
<span class="phvro8t-x-x-120">virology</span>, 71(3).
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHo2002b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ho, C.&nbsp;K. and Shuman, S. (2002). Bacteriophage T4 RNA ligase 2 (gp24.1)
exemplifies  a  family  of  RNA  ligases  found  in  all  phylogenetic  domains.
<span class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>, 99(20):12709–12714.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHo2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ho, C.&nbsp;K., Wang, L.&nbsp;K., Lima, C.&nbsp;D., and Shuman, S. (2004).   Structure
and mechanism  of  RNA  ligase.      <span class="phvro8t-x-x-120">Structure  (London,  England:  1993)</span>,
12(2):327–339.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XHoeffer2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hoeffer, C.&nbsp;A., Sanyal, S., and Ramaswami, M. (2003).   Acute induction
of conserved synaptic signaling pathways in Drosophila melanogaster.  <span class="phvro8t-x-x-120">The</span>
<span class="phvro8t-x-x-120">Journal of neuroscience : the official journal of the Society for Neuroscience</span>,
23:6362–6372.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHorvath2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Horvath,   G.&nbsp;C.,   Kistler,   M.&nbsp;K.,   and   Kistler,   W.&nbsp;S.   (2009).       RFX2
is a candidate   downstream   amplifier   of   A-MYB   regulation   in   mouse
spermatogenesis. <span class="phvro8t-x-x-120">BMC Developmental Biology</span>, 9(1):63.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHorwich2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Horwich, M.&nbsp;D., Li, C., Matranga, C., Vagin, V., Farley, G., Wang, P., and
Zamore, P.&nbsp;D.  (2007).   The  Drosophila  RNA  methyltransferase,  DmHen1,
modifies germline  piRNAs  and  single-stranded  siRNAs  in  RISC.   <span class="phvro8t-x-x-120">Current</span>
<span class="phvro8t-x-x-120">biology : CB</span>, 17(14):1265–72.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHosokawa2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hosokawa, M., Shoji, M., Kitamura, K., Tanaka, T., Noce, T., Chuma, S.,
and Nakatsuji, N. (2007). Tudor-related proteins TDRD1/MTR-1, TDRD6 and
TDRD7/TRAP: domain composition, intracellular localization, and function in
male germ cells in mice. <span class="phvro8t-x-x-120">Developmental biology</span>, 301(1):38–52.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHouse2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> House,  A.&nbsp;E.  and  Lynch,  K.&nbsp;W.  (2008).      Regulation  of  alternative
splicing: more  than  just  the  ABCs.    <span class="phvro8t-x-x-120">The  Journal  of  Biological  Chemistry</span>,
283(3):1217–21.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHouseley2010a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Houseley, J. and Tollervey, D. (2010). Apparent non-canonical trans-splicing
is generated by reverse transcriptase in vitro. <span class="phvro8t-x-x-120">PloS one</span>, 5(8):e12271.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHouwing2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Houwing, S., Kamminga, L.&nbsp;M., Berezikov, E., Cronembold, D., Girard, A.,
van&nbsp;den Elst, H., Filippov, D.&nbsp;V., Blaser, H., Raz, E., Moens, C.&nbsp;B., Plasterk,
R.&nbsp;H., Hannon, G.&nbsp;J., Draper, B.&nbsp;W., and Ketting, R.&nbsp;F. (2007).  A Role for
Piwi and piRNAs  in  Germ  Cell  Maintenance  and  Transposon  Silencing  in
Zebrafish. <span class="phvro8t-x-x-120">Cell</span>, 129(1):69–82.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHuang2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Huang, H., Gao, Q., Peng, X., Choi, S.-Y., Sarma, K., Ren, H., Morris, A.&nbsp;J.,
                                                                                

                                                                                
and Frohman, M.&nbsp;a. (2011). piRNA-associated germline nuage formation and
spermatogenesis  require  MitoPLD  profusogenic  mitochondrial-surface  lipid
signaling. <span class="phvro8t-x-x-120">Developmental cell</span>, 20(3):376–87.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XHughes2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Hughes,  T.&nbsp;A.  (2006).     Regulation  of  gene  expression  by  alternative
untranslated regions. <span class="phvro8t-x-x-120">Trends in Genetics: TIG</span>, 22(3):119–22.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XIngolia2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ingolia, N.&nbsp;T., Ghaemmaghami, S., Newman, J. R.&nbsp;S., and Weissman, J.&nbsp;S.
(2009). Genome-wide analysis in vivo of translation with nucleotide resolution
using ribosome profiling. <span class="phvro8t-x-x-120">Science</span>, 324:218–223.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XIngolia2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ingolia, N.&nbsp;T.,  Lareau,  L.&nbsp;F.,  and  Weissman,  J.&nbsp;S.  (2011).    Ribosome
profiling of mouse embryonic stem cells reveals the complexity and dynamics
of mammalian proteomes. <span class="phvro8t-x-x-120">Cell</span>, 147(4):789–802.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XIpsaro2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ipsaro, J.&nbsp;J., Haase, A.&nbsp;D., Knott, S.&nbsp;R., Joshua-Tor, L., and Hannon, G.&nbsp;J.
(2012). The structural biochemistry of Zucchini implicates it as a nuclease in
piRNA biogenesis. <span class="phvro8t-x-x-120">Nature</span>, 491(7423):279–83.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XJin2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Jin, Y., Suzuki,  H.,  Maegawa,  S.,  Endo,  H.,  Sugano,  S.,  Hashimoto,  K.,
Yasuda, K., and Inoue, K. (2003).  A vertebrate RNA-binding protein Fox-1
regulates tissue-specific splicing via the pentanucleotide GCAUG. <span class="phvro8t-x-x-120">The EMBO</span>
<span class="phvro8t-x-x-120">Journal</span>, 22(4):905–12.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XJohnson2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Johnson, B.&nbsp;E., Glauser, D.&nbsp;A., Dan-Glauser, E.&nbsp;S., Halling, D.&nbsp;B., Aldrich,
R.&nbsp;W., and Goodman, M.&nbsp;B. (2011).  Alternatively Spliced Domains Interact
to Regulate  BK  Potassium  Channel  Gating.   <span class="phvro8t-x-x-120">Proceedings  of  the  National</span>
<span class="phvro8t-x-x-120">Academy of Sciences</span>, 108(51):20784–20789.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XJohnson2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Johnson,  J.&nbsp;M.,  Castle,  J.,  Garrett-Engele,  P.,  Kan,  Z.,  Loerch,  P.&nbsp;M.,
Armour, C.&nbsp;D.,  Santos,  R.,  Schadt,  E.&nbsp;E.,  Stoughton,  R.,  and  Shoemaker,
D.&nbsp;D. (2003).     Genome-Wide  Survey  of  Human  Alternative  Pre-mRNA
Splicing with Exon Junction Microarrays. <span class="phvro8t-x-x-120">Science</span>, 302(5653):2141–2144.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XJones1999"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Jones, R.&nbsp;C. (1999). To store or mature spermatozoa? The primary role of
the epididymis. <span class="phvro8t-x-x-120">International Journal of Andrology</span>, 22(2):57–67.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XJudo1998"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Judo, M.&nbsp;S.,  Wedel,  a.&nbsp;B.,  and  Wilson,  C.  (1998).     Stimulation  and
suppression  of  PCR-mediated  recombination.     <span class="phvro8t-x-x-120">Nucleic  acids  research</span>,
26(7):1819–25.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKawaoka2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kawaoka,  S.,  Hara,  K.,  Shoji,  K.,  Kobayashi,  M.,  Shimada,  T.,  Sugano,
S., Tomari,  Y.,  Suzuki,  Y.,  and  Katsuma,  S.  (2012).    The  comprehensive
epigenome map of piRNA clusters. pages 1–10.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKawaoka2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kawaoka,  S.,  Hayashi,  N.,  Suzuki,  Y.,  Abe,  H.,  Sugano,  S.,  Tomari,  Y.,
Shimada, T., and Katsuma, S. (2009).   The Bombyx ovary-derived cell line
endogenously expresses PIWI/PIWI-interacting RNA complexes.  <span class="phvro8t-x-x-120">RNA (New</span>
<span class="phvro8t-x-x-120">York, N.Y.)</span>, 15(7):1258–64.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKawaoka2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kawaoka, S., Izumi, N., Katsuma, S., and Tomari, Y. (2011). 3’ end formation
of PIWI-interacting RNAs in vitro. <span class="phvro8t-x-x-120">Molecular cell</span>, 43(6):1015–22.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKe2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ke, R., Mignardi, M., Pacureanu, A., Svedlund, J., Botling, J., Wählby, C.,
and Nilsson, M. (2013).   In situ sequencing for RNA analysis in preserved
tissue and cells. <span class="phvro8t-x-x-120">Nature methods</span>, 10(July):857–60.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKennard1927"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kennard, E.&nbsp;H. (1927). Zur Quantenmechanik einfacher Bewegungstypen.
<span class="phvro8t-x-x-120">Zeitschrift for Physik</span>, 44(4-5):326–352.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKent2002"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kent, W.&nbsp;J.  (2002).    BLAT—The  BLAST-Like  Alignment  Tool.    <span class="phvro8t-x-x-120">Genome</span>
<span class="phvro8t-x-x-120">Research</span>, 12(4):656–664.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKhalil2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Khalil, A.&nbsp;M., Guttman, M., Huarte, M., Garber, M., Raj, A., Rivea Morales,
D., Thomas, K., Presser, A., Bernstein, B.&nbsp;E., van Oudenaarden, A., Regev,
A., Lander,  E.&nbsp;S.,  and  Rinn,  J.&nbsp;L.  (2009).    Many  human  large  intergenic
                                                                                

                                                                                
noncoding RNAs associate with chromatin-modifying complexes and affect
gene expression.     <span class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of  Sciences</span>,
106(28):11667 –11672.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKim2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kim, V.&nbsp;N., Han, J., and Siomi, M.&nbsp;C. (2009).  Biogenesis of small RNAs in
animals. <span class="phvro8t-x-x-120">Nature reviews. Molecular cell biology</span>, 10(2):126–39.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKirino2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kirino, Y.  and  Mourelatos,  Z.  (2007).   The  mouse  homolog  of  HEN1  is
a potential  methylase  for  Piwi-interacting  RNAs.    <span class="phvro8t-x-x-120">RNA  (New  York,  N.Y.)</span>,
13(9):1397–401.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKlattenhoff2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Klattenhoff,  C.,  Xi,  H.,  Li,  C.,  Lee,  S.,  Xu,  J.,  Khurana,  J.&nbsp;S.,  Zhang,  F.,
Schultz, N.,  Koppetsch,  B.&nbsp;S.,  Nowosielska,  A.,  Seitz,  H.,  Zamore,  P.&nbsp;D.,
Weng, Z., and Theurkauf, W.&nbsp;E. (2009). The Drosophila HP1 Homolog Rhino
Is Required for Transposon Silencing and piRNA Production by Dual-Strand
Clusters. <span class="phvro8t-x-x-120">Cell</span>, 138(6):1137–1149.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKleppe1970b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span>
Kleppe, K., Van de Sande, J.&nbsp;H., and Khorana, H.&nbsp;G. (1970). Polynucleotide
ligase-catalyzed joining of deoxyribo-oligonucleotides on ribopolynucleotide
templates and of ribo-oligonucleotides on deoxyribopolynucleotide templates.
<span class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of</span>
<span class="phvro8t-x-x-120">America</span>, 67(1):68–73.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKojima2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kojima, K.,  Kuramochi-Miyagawa,  S.,  Chuma,  S.,  Tanaka,  T.,  Nakatsuji,
N., Kimura, T., and Nakano, T. (2009).  Associations between PIWI proteins
and TDRD1/MTR-1 are critical for integrated subcellular localization in murine
male germ cells. <span class="phvro8t-x-x-120">Genes to cells : devoted to molecular &amp; cellular mechanisms</span>,
14(10):1155–65.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKolasinska-Zwierz2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kolasinska-Zwierz, P., Down, T., Latorre, I., Liu, T., Liu, X.&nbsp;S., and Ahringer,
J. (2009). Differential chromatin marking of introns and expressed exons by
H3K36me3. <span class="phvro8t-x-x-120">Nat Genet</span>, 41(3):376–381.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XKonig2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> König, J., Zarnack, K., Luscombe, N.&nbsp;M., and Ule, J. (2011).  Protein-RNA
interactions: new genomic technologies and perspectives.   <span class="phvro8t-x-x-120">Nature reviews.</span>
<span class="phvro8t-x-x-120">Genetics</span>, 13(2):77–83.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKonig2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> König, J., Zarnack, K., Rot, G., Curk, T., Kayikci, M., Zupan, B., Turner, D.&nbsp;J.,
Luscombe, N.&nbsp;M., and Ule, J. (2010).  iCLIP reveals the function of hnRNP
particles in splicing at individual nucleotide resolution.   <span class="phvro8t-x-x-120">Nature structural &amp;</span>
<span class="phvro8t-x-x-120">molecular biology</span>, 17(7):909–15.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKoren2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Koren, S.,  Schatz,  M.&nbsp;C.,  Walenz,  B.&nbsp;P.,  Martin,  J.,  Howard,  J.&nbsp;T.,
Ganapathy, G., Wang, Z., Rasko, D.&nbsp;a., McCombie, W.&nbsp;R., Jarvis, E.&nbsp;D., and
Adam M Phillippy (2012).  Hybrid error correction and de novo assembly of
single-molecule sequencing reads. <span class="phvro8t-x-x-120">Nature biotechnology</span>, 30(7):693–700.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKornblihtt2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kornblihtt,  A.&nbsp;R.,  Schor,  I.&nbsp;E.,  Alló,  M.,  Dujardin,  G.,  Petrillo,  E.,  and
Muñoz, M.&nbsp;J. (2013).  Alternative splicing: a pivotal step between eukaryotic
transcription   and   translation.      <span class="phvro8t-x-x-120">Nature   reviews.   Molecular   cell   biology</span>,
14(3):153–65.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKreahling2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kreahling,  J.&nbsp;M.  and  Graveley,  B.&nbsp;R.  (2005).   The  iStem,  a  long-range
RNA secondary  structure  element  required  for  efficient  exon  inclusion
in the Drosophila  Dscam  pre-mRNA.     <span class="phvro8t-x-x-120">Molecular  and  Cellular  Biology</span>,
25(23):10251–60.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKuhn2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kuhn, H.  and  Frank-Kamenetskii,  M.&nbsp;D.  (2005).    Template-independent
ligation of  single-stranded  DNA  by  T4  DNA  ligase.    <span class="phvro8t-x-x-120">The  FEBS  journal</span>,
272(23):5991–6000.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKumar1998"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kumar, M. and Carmichael, G.&nbsp;G. (1998).   Antisense RNA: function and
fate of duplex RNA in cells of higher eukaryotes. <span class="phvro8t-x-x-120">Microbiology and molecular</span>
<span class="phvro8t-x-x-120">biology reviews : MMBR</span>, 62(4):1415–34.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKuramochi-Miyagawa2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kuramochi-Miyagawa, S., Kimura, T., Ijiri, T.&nbsp;W., Isobe, T., Asada, N., Fujita,
Y., Ikawa, M., Iwai, N., Okabe, M., Deng, W., Lin, H., Matsuda, Y., and Nakano,
                                                                                

                                                                                
T. (2004). Mili, a Mammalian Member of Piwi Family Gene, Is Essential for
Spermatogenesis. <span class="phvro8t-x-x-120">Development</span>, 131(4):839–849.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKuramochi2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kuramochi-Miyagawa, S., Watanabe, T., Gotoh, K., Totoki, Y., Toyoda, A.,
Ikawa, M.,  Asada,  N.,  Kojima,  K.,  Yamaguchi,  Y.,  Ijiri,  T.&nbsp;W.,  Hata,  K.,  Li,
E., Matsuda, Y., Kimura, T., Okabe, M., Sakaki, Y., Sasaki, H., and Nakano,
T. (2008).    DNA  Methylation  of  Retrotransposon  Genes  Is  Regulated  by
Piwi Family  Members  MILI  and  MIWI2  in  Murine  Fetal  Testes.   <span class="phvro8t-x-x-120">Genes  &amp;</span>
<span class="phvro8t-x-x-120">Development</span>, 22(7):908–917.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XKutter2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Kutter, C., Brown, G.&nbsp;D., Gonçalves, A., Wilson, M.&nbsp;D., Watt, S., Brazma,
A., White, R.&nbsp;J.,  and  Odom,  D.&nbsp;T.  (2011).   Pol  III  binding  in  six  mammals
shows conservation among amino acid isotypes despite divergence among
tRNA genes. <span class="phvro8t-x-x-120">Nature genetics</span>, 43(10):948–55.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLadd2002"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ladd, A.&nbsp;N.   and   Cooper,   T.&nbsp;A.   (2002).        Finding   signals   that
regulate alternative  splicing  in  the  post-genomic  era.    <span class="phvro8t-x-x-120">Genome  Biology</span>,
3(11):reviews0008.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLaiho2013a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Laiho, A., Kotaja, N., Gyenesei, A., and Sironen, A. (2013).  Transcriptome
profiling of the murine testis during the first wave of spermatogenesis.  <span class="phvro8t-x-x-120">PloS</span>
<span class="phvro8t-x-x-120">one</span>, 8(4):e61558.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLander2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lander, E.&nbsp;S. (2011). Initial impact of the sequencing of the human genome.
<span class="phvro8t-x-x-120">Nature</span>, 470(7333):187–97.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLangmead2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Langmead, B. and Salzberg, S.&nbsp;L. (2012). Fast gapped-read alignment with
Bowtie 2. <span class="phvro8t-x-x-120">Nature methods</span>, 9(4):357–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLangmead2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.&nbsp;L. (2009).  Ultrafast
and memory-efficient  alignment  of  short  DNA  sequences  to  the  human
genome. <span class="phvro8t-x-x-120">Genome Biology</span>, 10(3):R25.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XLasko2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lasko, P. (2013).  The DEAD-box helicase Vasa: evidence for a multiplicity
of functions  in  RNA  processes  and  developmental  biology.   <span class="phvro8t-x-x-120">Biochimica  et</span>
<span class="phvro8t-x-x-120">biophysica acta</span>, 1829(8):810–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLatham1996"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Latham, K.&nbsp;E., Litvin, J., Orth, J.&nbsp;M., Patel, B., Mettus, R., and Reddy, E.&nbsp;P.
(1996). Temporal patterns of A-myb and B-myb gene expression during testis
development. <span class="phvro8t-x-x-120">Oncogene</span>, 13:1161–1168.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLau2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lau, N.&nbsp;C.,  Seto,  A.&nbsp;G.,  Kim,  J.,  Kuramochi-Miyagawa,  S.,  Nakano,  T.,
Bartel, D.&nbsp;P.,  and  Kingston,  R.&nbsp;E.  (2006).   Characterization  of  the  piRNA
complex from rat testes. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 313(5785):363–367.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLecuyer2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lécuyer, E., Yoshida, H., Parthasarathy, N., Alm, C., Babak, T., Cerovina, T.,
Hughes, T.&nbsp;R., Tomancak, P., and Krause, H.&nbsp;M. (2007).  Global analysis of
mRNA localization reveals a prominent role in organizing cellular architecture
and function. <span class="phvro8t-x-x-120">Cell</span>, 131(1):174–87.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLee2010b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lee, C., Kim, N., Roy, M., and Graveley, B.&nbsp;R. (2010). Massive expansions
of Dscam splicing diversity via staggered homologous recombination during
arthropod evolution. <span class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>, 16(1):91–105.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLee2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lee, H.-C., Gu, W., Shirayama, M., Youngman, E., Conte, D., and Mello,
C.&nbsp;C. (2012).  C. elegans piRNAs mediate the genome-wide surveillance of
germline transcripts. <span class="phvro8t-x-x-120">Cell</span>, 150(1):78–87.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLee2014a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lee, J.&nbsp;H.,  Daugharthy,  E.&nbsp;R.,  Scheiman,  J.,  Kalhor,  R.,  Yang,  J.&nbsp;L.,
Ferrante, T.&nbsp;C., Terry, R., Jeanty, S. S.&nbsp;F., Li, C., Amamoto, R., Peters, D.&nbsp;T.,
Turczyk, B.&nbsp;M., Marblestone, A.&nbsp;H., Inverso, S.&nbsp;A., Bernard, A., Mali, P., Rios,
X., Aach, J., and Church, G.&nbsp;M. (2014a). Highly Multiplexed Subcellular RNA
Sequencing in Situ. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 309:1728–32.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLee2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lee, K., Cui, Y., Lee, L.&nbsp;P., and Irudayaraj, J. (2014b). Quantitative imaging
of single mRNA  splice  variants  in  living  cells.    <span class="phvro8t-x-x-120">Nature  nanotechnology</span>,
(April):1–7.
                                                                                

                                                                                
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLeGault2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> LeGault, L.&nbsp;H. and Dewey, C.&nbsp;N. (2013).  Inference of alternative splicing
from RNA-Seq data with probabilistic splice graphs.  <span class="phvro8t-x-x-120">Bioinformatics (Oxford,</span>
<span class="phvro8t-x-x-120">England)</span>, 29(18):2300–10.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLenasi2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lenasi, T.,  Peterlin,  B.&nbsp;M.,  and  Dovc,  P.  (2006).    Distal  regulation  of
alternative splicing by splicing enhancer in equine <span class="cmmi-12">&#946;</span>-casein intron 1.  <span class="phvro8t-x-x-120">RNA</span>,
12(3):498 –507.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2009d"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth,
G., Abecasis,  G.,  and  Durbin,  R.  (2009a).   The  Sequence  Alignment/Map
format and SAMtools. <span class="phvro8t-x-x-120">Bioinformatics</span>, 25:2078–2079.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2012c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, H., Qiu,  J.,  and  Fu,  X.-D.  (2012a).   RASL-seq  for  massively  parallel
and quantitative analysis of gene expression.  <span class="phvro8t-x-x-120">Current protocols in molecular</span>
<span class="phvro8t-x-x-120">biology / edited  by  Frederick  M.  Ausubel  ...  [et  al.]</span>,  Chapter  4(April):Unit
4.13.1–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, M., Wang, I.&nbsp;X., Li, Y., Bruzel, A., Richards, A.&nbsp;L., Toung, J.&nbsp;M., and
Cheung, V.&nbsp;G. (2011).  Widespread RNA and DNA sequence differences in
the human transcriptome. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 333(6038):53–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2009a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, X.&nbsp;C., Barringer, B.&nbsp;C., and Barbash, D.&nbsp;a. (2009b).   The pachytene
checkpoint and its relationship to evolutionary patterns of polyploidization and
hybrid sterility. <span class="phvro8t-x-x-120">Heredity</span>, 102(1):24–30.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, X.&nbsp;C.  and  Schimenti,  J.&nbsp;C.  (2007).   Mouse  pachytene  checkpoint  2
(trip13) is required  for  completing  meiotic  recombination  but  not  synapsis.
<span class="phvro8t-x-x-120">PLoS genetics</span>, 3(8):e130.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2013h"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, X.&nbsp;Z., Roy, C.&nbsp;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&nbsp;W., Xu,
J., Moore, M.&nbsp;J., Schimenti, J.&nbsp;C., Weng, Z., and Zamore, P.&nbsp;D. (2013a).  An
ancient transcription factor initiates the burst of piRNA production during early
                                                                                

                                                                                
meiosis in mouse testes. <span class="phvro8t-x-x-120">Molecular cell</span>, 50(1):67–81.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2013e"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, X.&nbsp;Z., Roy, C.&nbsp;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&nbsp;W., Xu,
J., Moore, M.&nbsp;J., Schimenti, J.&nbsp;C., Weng, Z., and Zamore, P.&nbsp;D. (2013b).  An
Ancient Transcription Factor Initiates the Burst of piRNA Production during
Early Meiosis in Mouse Testes. <span class="phvro8t-x-x-120">Molecular Cell</span>, pages 1–15.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, X.&nbsp;Z., Roy, C.&nbsp;K., Moore, M.&nbsp;J., and Zamore, P.&nbsp;D. (2013c).  Defining
piRNA primary transcripts. <span class="phvro8t-x-x-120">Cell cycle (Georgetown, Tex.)</span>, 12(11):1657–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLi2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Li, Y., Wang, H.-Y., Wan, F.-C., Liu, F.-J., Liu, J., Zhang, N., Jin, S.-H., and Li,
J.-Y. (2012b).  Deep sequencing analysis of small non-coding RNAs reveals
the diversity  of  microRNAs  and  piRNAs  in  the  human  epididymis.   <span class="phvro8t-x-x-120">Gene</span>,
497(2):330–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLicatalosi2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Licatalosi,  D.&nbsp;D.  and  Darnell,  R.&nbsp;B.  (2006).     Splicing  Regulation  in
Neurologic Disease. <span class="phvro8t-x-x-120">Neuron</span>, 52(1):93–101.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLicatalosi2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Licatalosi, D.&nbsp;D., Mele, A., Fak, J.&nbsp;J., Ule, J., Kayikci, M., Chi, S.&nbsp;W., Clark,
T.&nbsp;A., Schweitzer, A.&nbsp;C., Blume, J.&nbsp;E., Wang, X., Darnell, J.&nbsp;C., and Darnell,
R.&nbsp;B. (2008).  HITS-CLIP yields genome-wide insights into brain alternative
RNA processing. <span class="phvro8t-x-x-120">Nature</span>, 456(7221):464–469.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLim2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lim, A.&nbsp;K. and Kai, T. (2007). Unique germ-line organelle, nuage, functions
to repress selfish genetic elements in Drosophila melanogaster. <span class="phvro8t-x-x-120">Proceedings</span>
<span class="phvro8t-x-x-120">of the National  Academy  of  Sciences  of  the  United  States  of  America</span>,
104(16):6714–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLister2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lister, R., O’Malley, R.&nbsp;C., Tonti-Filippini, J., Gregory, B.&nbsp;D., Berry, C.&nbsp;C.,
Millar, A.&nbsp;H., and Ecker, J.&nbsp;R. (2008). Highly integrated single-base resolution
maps of the epigenome in Arabidopsis. <span class="phvro8t-x-x-120">Cell</span>, 133(3):523–36.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLiu2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Liu, Y., Zhou, J., and White, K.&nbsp;P. (2014).  RNA-seq differential expression
                                                                                

                                                                                
studies: more  sequence  or  more  replication?      <span class="phvro8t-x-x-120">Bioinformatics  (Oxford,</span>
<span class="phvro8t-x-x-120">England)</span>, 30:301–4.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLohman2013c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lohman, G. J.&nbsp;S., Zhang, Y., Zhelkovsky, A.&nbsp;M., Cantor, E.&nbsp;J., Evans, T.&nbsp;C.,
and Jr, T. C.&nbsp;E. (2013).  Efficient DNA ligation in DNA – RNA hybrid helices
by Chlorella virus DNA ligase. <span class="phvro8t-x-x-120">Nucleic acids research</span>, 42(36):1–14.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLong2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Long, J.&nbsp;C. and Caceres, J.&nbsp;F. (2009).  The SR protein family of splicing
factors: master  regulators  of  gene  expression.    <span class="phvro8t-x-x-120">The  Biochemical  journal</span>,
417(1):15–27.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLuco2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Luco, R.&nbsp;F., Allo, M., Schor, I.&nbsp;E., Kornblihtt, A.&nbsp;R., and Misteli, T. (2011).
Epigenetics in alternative pre-mRNA splicing. <span class="phvro8t-x-x-120">Cell</span>, 144(1):16–26.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLuteijn2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Luteijn, M.&nbsp;J.  and  Ketting,  R.&nbsp;F.  (2013).    PIWI-interacting  RNAs:  from
generation  to  transgenerational  epigenetics.     <span class="phvro8t-x-x-120">Nature  reviews.  Genetics</span>,
14(8):523–34.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLynch2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Lynch, K.&nbsp;W. (2004). Consequences of regulated pre-mRNA splicing in the
immune system. <span class="phvro8t-x-x-120">Nat Rev Immunol</span>, 4(12):931–940.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMacilwain2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Macilwain, C. (2010). Scientists vs engineers: this time it’s financial. <span class="phvro8t-x-x-120">Nature</span>,
467(7318):885.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMalone2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Malone, C.&nbsp;D. and Hannon, G.&nbsp;J. (2009).  Small RNAs as guardians of the
genome. <span class="phvro8t-x-x-120">Cell</span>, 136(4):656–668.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMangan2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Mangan, S. and Alon, U. (2003). Structure and function of the feed-forward
loop network  motif.    <span class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of  Sciences</span>,
100(21):11980 –11985.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMarinov2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Marinov, G.&nbsp;K., Williams, B.&nbsp;a., McCue, K., Schroth, G.&nbsp;P., Gertz, J., Myers,
                                                                                

                                                                                
R.&nbsp;M., and Wold, B.&nbsp;J. (2013).  From single-cell to cell-pool transcriptomes:
stochasticity in gene expression and RNA splicing. <span class="phvro8t-x-x-120">Genome research</span>, pages
496–510.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMarioni2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Marioni, J.&nbsp;C.,  Mason,  C.&nbsp;E.,  Mane,  S.&nbsp;M.,  Stephens,  M.,  and  Gilad,  Y.
(2008). RNA-seq: an assessment of technical reproducibility and comparison
with gene expression arrays. <span class="phvro8t-x-x-120">Genome Research</span>, 18(9):1509–17.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMartinez-Contreras2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Martinez-Contreras,  R.,  Cloutier,  P.,  Shkreta,  L.,  Fisette,  J.-F.,  Revil,  T.,
and Chabot, B. (2007).  hnRNP proteins and splicing control.  <span class="phvro8t-x-x-120">Advances in</span>
<span class="phvro8t-x-x-120">experimental medicine and biology</span>, 623:123–147.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMaxam1977a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Maxam, A.&nbsp;M. and Gilbert, W. (1992). A new method for sequencing DNA.
1977. <span class="phvro8t-x-x-120">Biotechnology (Reading, Mass.)</span>, 24(2):99–103.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMeikar2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Meikar, O., Da Ros, M., Korhonen, H., and Kotaja, N. (2011).  Chromatoid
body and small  RNAs  in  male  germ  cells.     <span class="phvro8t-x-x-120">Reproduction  (Cambridge,</span>
<span class="phvro8t-x-x-120">England)</span>, 142(2):195–209.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMeikar2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Meikar, O., Vagin, V.&nbsp;V., Chalmel, F., Sõstar, K., Lardenois, A., Hammell,
M., Jin, Y., Da Ros, M., Wasik, K.&nbsp;a., Toppari, J., Hannon, G.&nbsp;J., and Kotaja,
N. (2014). An atlas of chromatoid body components.  <span class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>,
20(4):483–95.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMeister2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Meister, G. (2013).  Argonaute proteins: functional insights and emerging
roles. <span class="phvro8t-x-x-120">Nature reviews. Genetics</span>, 14(7):447–59.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMercer2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Mercer, T.&nbsp;R., Clark, M.&nbsp;B., Crawford, J., Brunck, M.&nbsp;E., Gerhardt, D.&nbsp;J.,
Taft, R.&nbsp;J., Nielsen, L.&nbsp;K., Dinger, M.&nbsp;E., and Mattick, J.&nbsp;S. (2014). Targeted
sequencing  for  gene  discovery  and  quantification  using  RNA  CaptureSeq.
<span class="phvro8t-x-x-120">Nature Protocols</span>, 9(5):989–1009.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMerkhofer2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Merkhofer,  E.&nbsp;C.,  Hu,  P.,  and  Johnson,  T.&nbsp;L.  (2014).    Introduction  to
                                                                                

                                                                                
cotranscriptional RNA splicing.  <span class="phvro8t-x-x-120">Methods in molecular biology (Clifton, N.J.)</span>,
1126:83–96.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMerkin2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Merkin, J., Russell, C., Chen, P., and Burge, C.&nbsp;B. (2012).   Evolutionary
Dynamics of Gene and Isoform Regulation in Mammalian Tissues.  <span class="phvro8t-x-x-120">Science</span>,
338(6114):1593–1599.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMettus1994"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Mettus, R.&nbsp;V., Litvin, J., Wali, A., Toscani, A., Latham, K., Hatton, K., and
Reddy, E.&nbsp;P.  (1994).   Murine  A-myb:  evidence  for  differential  splicing  and
tissue-specific expression. <span class="phvro8t-x-x-120">Oncogene</span>, 9(10):3077–86.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMeyerhans1990a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Meyerhans,  A.,  Vartanian,  J.-P.,  and  Wain-Hobson,  S.  (1990).     DNA
recombination during PCR. <span class="phvro8t-x-x-120">Nucleic Acids Research</span>, 18(7):1687–1691.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMiller2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Miller, J.&nbsp;a.,  Ding,  S.-L.,  Sunkin,  S.&nbsp;M.,  Smith,  K.&nbsp;a.,  Ng,  L.,  Szafer,  A.,
Ebbert, A.,  Riley,  Z.&nbsp;L.,  Royall,  J.&nbsp;J.,  Aiona,  K.,  Arnold,  J.&nbsp;M.,  Bennet,  C.,
Bertagnolli, D., Brouner, K., Butler, S., Caldejon, S., Carey, A., Cuhaciyan,
C., Dalley, R.&nbsp;a., Dee, N., Dolbeare, T.&nbsp;a., Facer, B. a.&nbsp;C., Feng, D., Fliss,
T.&nbsp;P., Gee, G., Goldy, J., Gourley, L., Gregor, B.&nbsp;W., Gu, G., Howard, R.&nbsp;E.,
Jochim, J.&nbsp;M., Kuan, C.&nbsp;L., Lau, C., Lee, C.-K., Lee, F., Lemon, T.&nbsp;a., Lesnar,
P., McMurray, B., Mastan, N., Mosqueda, N., Naluai-Cecchini, T., Ngo, N.-K.,
Nyhus, J.,  Oldre,  A.,  Olson,  E.,  Parente,  J.,  Parker,  P.&nbsp;D.,  Parry,  S.&nbsp;E.,
Stevens, A.,  Pletikos,  M.,  Reding,  M.,  Roll,  K.,  Sandman,  D.,  Sarreal,  M.,
Shapouri, S., Shapovalova, N.&nbsp;V., Shen, E.&nbsp;H., Sjoquist, N., Slaughterbeck,
C.&nbsp;R., Smith,  M.,  Sodt,  A.&nbsp;J.,  Williams,  D.,  Zöllei,  L.,  Fischl,  B.,  Gerstein,
M.&nbsp;B., Geschwind,  D.&nbsp;H.,  Glass,  I.&nbsp;a.,  Hawrylycz,  M.&nbsp;J.,  Hevner,  R.&nbsp;F.,
Huang, H.,  Jones,  A.&nbsp;R.,  Knowles,  J.&nbsp;a.,  Levitt,  P.,  Phillips,  J.&nbsp;W.,  Šestan,
N., Wohnoutka, P., Dang, C., Bernard, A., Hohmann, J.&nbsp;G., and Lein, E.&nbsp;S.
(2014). Transcriptional landscape of the prenatal human brain. <span class="phvro8t-x-x-120">Nature</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMiura2013b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Miura, S.&nbsp;K., Martins, A., Zhang, K.&nbsp;X., Graveley, B.&nbsp;R., and Zipursky, S.&nbsp;L.
(2013). Probabilistic splicing of Dscam1 establishes identity at the level of
single neurons. <span class="phvro8t-x-x-120">Cell</span>, 155(5):1166–77.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XModrek2002"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Modrek, B.  and  Lee,  C.  (2002).   A  genomic  view  of  alternative  splicing.
                                                                                

                                                                                
<span class="phvro8t-x-x-120">Nature Genetics</span>, 30(1):13–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XModrich1973a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Modrich,  P.,  Anraku,  Y.,  and  Lehman,  I.&nbsp;R.  (1973).    Deoxyribonucleic
Acid Ligase  ISOLATION  AND  PHYSICAL  CHARACTERIZATION  OF  THE
HOMOGENEOUS  ENZYME  FROM  ESCHERICHIA  COLI.      <span class="phvro8t-x-x-120">Journal  of</span>
<span class="phvro8t-x-x-120">Biological Chemistry</span>, 248(21):7495–7501.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XModzelewski2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Modzelewski,  A.&nbsp;J.,  Holmes,  R.&nbsp;J.,  Hilz,  S.,  Grimson,  A.,  and  Cohen,
P.&nbsp;E. (2012).   AGO4  regulates  entry  into  meiosis  and  influences  silencing
of sex chromosomes  in  the  male  mouse  germline.    <span class="phvro8t-x-x-120">Developmental  cell</span>,
23(2):251–64.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMontgomery1998"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Montgomery,  M.&nbsp;K.,  Xu,  S.,  and  Fire,  a.  (1998).     RNA  as  a  target
of double-stranded  RNA-mediated  genetic  interference  in  Caenorhabditis
elegans. <span class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United</span>
<span class="phvro8t-x-x-120">States of America</span>, 95(26):15502–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMortazavi2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Mortazavi,  A.,  Williams,  B.&nbsp;A.,  McCue,  K.,  Schaeffer,  L.,  and  Wold,  B.
(2008). Mapping and quantifying mammalian transcriptomes by RNA-Seq.
<span class="phvro8t-x-x-120">Nat Meth</span>, 5(7):621–628.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XMutz2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Mutz, K.-O.,  Heilkenbrinker,  A.,  Lönne,  M.,  Walter,  J.-G.,  and  Stahl,  F.
(2013). Transcriptome analysis using next-generation sequencing.  <span class="phvro8t-x-x-120">Current</span>
<span class="phvro8t-x-x-120">opinion in biotechnology</span>, 24(1):22–30.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNagalakshmi2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nagalakshmi, U., Wang, Z., Waern, K., Shou, C., Raha, D., Gerstein, M.,
and Snyder, M. (2008).  The transcriptional landscape of the yeast genome
defined by RNA sequencing. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 320(5881):1344–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNakano2003a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nakano, M., Komatsu, J., Matsuura, S.-i., Takashima, K., Katsura, S., and
Mizuno, A. (2003). Single-molecule PCR using water-in-oil emulsion. <span class="phvro8t-x-x-120">Journal</span>
<span class="phvro8t-x-x-120">of Biotechnology</span>, 102(2):117–124.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XNamekawa2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Namekawa,  S.&nbsp;H.  and  Lee,  J.&nbsp;T.  (2009).     XY  and  ZW:  is  meiotic
sex chromosome   inactivation   the   rule   in   evolution?      <span class="phvro8t-x-x-120">PLoS   genetics</span>,
5(5):e1000493.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNandakumar2004a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nandakumar,   J.,   Ho,   C.&nbsp;K.,   Lima,   C.&nbsp;D.,   and   Shuman,   S.   (2004).
RNA substrate   specificity   and   structure-guided   mutational   analysis   of
bacteriophage  T4  RNA  ligase  2.     <span class="phvro8t-x-x-120">The  Journal  of  biological  chemistry</span>,
279(30):31337–47.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNandakumar2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nandakumar,  J.,  Shuman,  S.,  and  Lima,  C.&nbsp;D.  (2006).     RNA  ligase
structures reveal the basis for RNA specificity and conformational changes
that drive ligation forward. <span class="phvro8t-x-x-120">Cell</span>, 127(1):71–84.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNEBEL1961"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nebel, B.&nbsp;R.,  Amarose,  A.&nbsp;P.,  and  Hacket,  E.&nbsp;M.  (1961).    Calendar  of
gametogenic development in the prepuberal male mouse. <span class="phvro8t-x-x-120">Science (New York,</span>
<span class="phvro8t-x-x-120">N.Y.)</span>, 134(3482):832–3.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNellaker2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nellå&nbsp;ker, C., Keane, T.&nbsp;M., Yalcin, B., Wong, K., Agam, A., Belgard, T.&nbsp;G.,
Flint, J., Adams,  D.&nbsp;J.,  Frankel,  W.&nbsp;N.,  and  Ponting,  C.&nbsp;P.  (2012).    The
genomic landscape shaped by selection on transposable elements across 18
mouse strains. <span class="phvro8t-x-x-120">Genome biology</span>, 13(6):R45.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNeves2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Neves, G.,  Zucker,  J.,  Daly,  M.,  and  Chess,  A.  (2004).    Stochastic  yet
biased expression of multiple Dscam splice variants by individual cells. <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">genetics</span>, 36(3):240–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNewburger2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Newburger, D.&nbsp;E. and Bulyk, M.&nbsp;L. (2009). UniPROBE: an online database
of protein binding microarray data on protein-DNA interactions. <span class="phvro8t-x-x-120">Nucleic acids</span>
<span class="phvro8t-x-x-120">research</span>, 37(Database issue):D77–82.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNilsen2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nilsen, T.&nbsp;W.  and  Graveley,  B.&nbsp;R.  (2010).   Expansion  of  the  eukaryotic
proteome by alternative splicing. <span class="phvro8t-x-x-120">Nature</span>, 463(7280):457–463.
</p>
<p class="bibitem"><span class="biblabel">
                                                                                

                                                                                
<a id="XNilsson2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nilsson,  M.,  Antson,  D.-O.,  Barbany,  G.,  and  Landegren,  U.  (2001).
RNA-templated DNA ligation for transcript analysis. <span class="phvro8t-x-x-120">Nucleic Acids Research</span>,
29(2):578–581.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNilsson2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nilsson, M.,  Barbany,  G.,  Antson,  D.&nbsp;O.,  Gertow,  K.,  and  Landegren,  U.
(2000). Enhanced  detection  and  distinction  of  RNA  by  enzymatic  probe
ligation. <span class="phvro8t-x-x-120">Nature biotechnology</span>, 18(7):791–3.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNishimasu2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Nishimasu,  H.,  Ishizu,  H.,  Saito,  K.,  Fukuhara,  S.,  Kamatani,  M.&nbsp;K.,
Bonnefond, L., Matsumoto, N., Nishizawa, T., Nakanaga, K., Aoki, J., Ishitani,
R., Siomi, H., Siomi, M.&nbsp;C., and Nureki, O. (2012).  Structure and function of
Zucchini endoribonuclease in piRNA biogenesis. <span class="phvro8t-x-x-120">Nature</span>, 491(7423):284–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNorth2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> North, A.&nbsp;J. (2006).   Seeing is believing? A beginners’ guide to practical
pitfalls in image acquisition. <span class="phvro8t-x-x-120">The Journal of cell biology</span>, 172(1):9–18.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XNotredame2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Notredame, C., Higgins, D.&nbsp;G., and Heringa, J. (2000).  T-Coffee: A novel
method for  fast  and  accurate  multiple  sequence  alignment.    <span class="phvro8t-x-x-120">Journal  of</span>
<span class="phvro8t-x-x-120">molecular biology</span>, 302(1):205–17.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOakberg1956b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Oakberg, E. and Oakberq, F. (1956).  A Description of Spermiogenesis in
the mouse and its use in analysis of the cycle of the seminiferous epithelium
and germ cell renewal. <span class="phvro8t-x-x-120">The American journal of anatomy</span>, 99(3):391–419.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOh1999"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Oh, I.&nbsp;H. and Reddy, E.&nbsp;P. (1999).   The myb gene family in cell growth,
differentiation and apoptosis. <span class="phvro8t-x-x-120">Oncogene</span>, 18(19):3017–33.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOlivera1967b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Olivera, B.  and  Lehman,  I.  (1967).   Linkage  of  polynucleotides  through
phosphodiester bonds by an enzyme from Escherichia coli.  <span class="phvro8t-x-x-120">Proceedings of</span>
<span class="phvro8t-x-x-120">the National Academy of </span><span class="phvro8t-x-x-120">…</span>, pages 1426–1433.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOlson2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Olson, S., Blanchette, M., Park, J., Savva, Y., Yeo, G.&nbsp;W., Yeakley, J.&nbsp;M.,
Rio, D.&nbsp;C.,  and  Graveley,  B.&nbsp;R.  (2007).    A  regulator  of  Dscam  mutually
                                                                                

                                                                                
exclusive splicing fidelity. <span class="phvro8t-x-x-120">Nat Struct Mol Biol</span>, 14(12):1134–1140.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOsella2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Osella, M.,  Bosia,  C.,  Corá,  D.,  and  Caselle,  M.  (2011).    The  role  of
incoherent microRNA-mediated feedforward loops in noise buffering.  <span class="phvro8t-x-x-120">PLoS</span>
<span class="phvro8t-x-x-120">computational biology</span>, 7(3):e1001101.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOsheim1985"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Osheim, Y.&nbsp;N., Miller, O.&nbsp;L., and Beyer, a.&nbsp;L. (1985). RNP particles at splice
junction sequences on Drosophila chorion transcripts. <span class="phvro8t-x-x-120">Cell</span>, 43(1):143–51.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XOzsolak2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ozsolak,  F.  and  Milos,  P.&nbsp;M.  (2010).      RNA  sequencing:  advances,
challenges and opportunities. <span class="phvro8t-x-x-120">Nature Reviews Genetics</span>, 12(2):87–98.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPan2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Pan, Q., Shai, O., Lee, L.&nbsp;J., Frey, B.&nbsp;J., and Blencowe, B.&nbsp;J. (2008). Deep
surveying of alternative splicing complexity in the human transcriptome by
high-throughput sequencing. <span class="phvro8t-x-x-120">Nature Genetics</span>, 40(12):1413–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPan2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Pan, T. (2013).  N6-methyl-adenosine modification in messenger and long
non-coding RNA. <span class="phvro8t-x-x-120">Trends in biochemical sciences</span>, 38(4):204–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPane2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Pane, A., Wehr, K., and Schüpbach, T. (2007). zucchini and squash encode
two putative  nucleases  required  for  rasiRNA  production  in  the  Drosophila
germline. <span class="phvro8t-x-x-120">Developmental cell</span>, 12(6):851–62.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPark2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Park, H.&nbsp;Y.,  Lim,  H.,  Yoon,  Y.&nbsp;J.,  Follenzi,  A.,  Nwokafor,  C.,  Meng,  X.,
Singer, R.&nbsp;H.,  Yoon,  H.,  and  Lopez-Jones,  M.  (2014).    Visualization  of
dynamics of single endogenous mRNA labeled in live mouse.  <span class="phvro8t-x-x-120">Science (New</span>
<span class="phvro8t-x-x-120">York, N.Y.)</span>, 343(6169):422–4.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPark2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Park, J.&nbsp;W.  and  Graveley,  B.&nbsp;R.  (2007).    Complex  alternative  splicing.
<span class="phvro8t-x-x-120">Advances in Experimental Medicine and Biology</span>, 623:50–63.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XParra2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Parra, M.&nbsp;K., Gallagher, T.&nbsp;L., Amacher, S.&nbsp;L., Mohandas, N., and Conboy,
                                                                                

                                                                                
J.&nbsp;G. (2012).    Deep  intron  elements  mediate  nested  splicing  events  at
consecutive AG dinucleotides to regulate alternative 3’ splice site choice in
vertebrate 4.1 genes. <span class="phvro8t-x-x-120">Molecular and cellular biology</span>, 32(11):2044–53.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XParra2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Parra, M.&nbsp;K.,  Tan,  J.&nbsp;S.,  Mohandas,  N.,  and  Conboy,  J.&nbsp;G.  (2008).
Intrasplicing coordinates alternative first exons with alternative splicing in the
protein 4.1R gene. <span class="phvro8t-x-x-120">EMBO J</span>, 27(1):122–131.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPauli2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Pauli, A.,  Rinn,  J.&nbsp;L.,  and  Schier,  A.&nbsp;F.  (2011).    Non-coding  RNAs  as
regulators of embryogenesis. <span class="phvro8t-x-x-120">Nature reviews. Genetics</span>, 12(2):136–49.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPeirson2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Peirson, S.&nbsp;N. (2003).  Experimental validation of novel and conventional
approaches  to  quantitative  real-time  PCR  data  analysis.    <span class="phvro8t-x-x-120">Nucleic  Acids</span>
<span class="phvro8t-x-x-120">Research</span>, 31(14):73e–73.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPelisson1994"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Pélisson, a., Song, S.&nbsp;U., Prud’homme, N., Smith, P.&nbsp;a., Bucheton, a., and
Corces, V.&nbsp;G.  (1994).   Gypsy  transposition  correlates  with  the  production
of a retroviral  envelope-like  protein  under  the  tissue-specific  control  of  the
Drosophila flamenco gene. <span class="phvro8t-x-x-120">The EMBO journal</span>, 13(18):4401–11.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPeng2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Peng, T.,  Xue,  C.,  Bi,  J.,  Li,  T.,  Wang,  X.,  Zhang,  X.,  and  Li,  Y.  (2008).
Functional importance of different patterns of correlation between adjacent
cassette exons in human and mouse. <span class="phvro8t-x-x-120">BMC Genomics</span>, 9(1):191.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPlocik2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Plocik, A. M.&nbsp;A. and Graveley, B.&nbsp;R. (2013).   New insights from existing
sequence data: generating breakthroughs without a pipette.  <span class="phvro8t-x-x-120">Molecular cell</span>,
49(4):605–17.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPonta2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ponta, H., Sherman, L., and Herrlich, P.&nbsp;a. (2003).  CD44: from adhesion
molecules to signalling regulators.   <span class="phvro8t-x-x-120">Nature reviews. Molecular cell biology</span>,
4(1):33–45.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XPurandare2014b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Purandare, S.&nbsp;R., Tenhumberg, B., and Brisson, J.&nbsp;a. (2014). Comparison
                                                                                

                                                                                
of the wing polyphenic response of pea aphids ( Acyrthosiphon pisum ) to
crowding and predator cues. <span class="phvro8t-x-x-120">Ecological Entomology</span>, 39(2):263–266.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XQuinlan2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Quinlan,  A.&nbsp;R.  and  Hall,  I.&nbsp;M.  (2010).    BEDTools:  a  flexible  suite  of
utilities for comparing genomic features.   <span class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>,
26(6):841–842.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRDevelopmentCoreTeam2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> R Development Core Team, R. (2011). R: A Language and Environment for
Statistical Computing.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRaney2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Raney, B.&nbsp;J.,  Dreszer,  T.&nbsp;R.,  Barber,  G.&nbsp;P.,  Clawson,  H.,  Fujita,  P.&nbsp;a.,
Wang, T., Nguyen, N., Paten, B., Zweig, a.&nbsp;S., Karolchik, D., and Kent, W.&nbsp;J.
(2013). Track Data Hubs enable visualization of user-defined genome-wide
annotations  on  the  UCSC  Genome  Browser.      <span class="phvro8t-x-x-120">Bioinformatics  (Oxford,</span>
<span class="phvro8t-x-x-120">England)</span>, pages 1–3.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRebagliati1985"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Rebagliati, M., Weeks, D., Harvey, R., and Melton, D. (1985).  Identification
and cloning   of   localized   maternal   RNAs   from   xenopus   eggs.      <span class="phvro8t-x-x-120">Cell</span>,
42(3):769–777.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRehrauer2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Rehrauer, H., Opitz, L., Tan, G., Sieverling, L., and Schlapbach, R. (2013).
Blind spots  of  quantitative  RNA-seq:  the  limits  for  assessing  abundance,
differential expression, and isoform switching. <span class="phvro8t-x-x-120">BMC bioinformatics</span>, 14:370.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XReuter2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Reuter, J.&nbsp;S. and Mathews, D.&nbsp;H. (2010).  RNAstructure: software for RNA
secondary structure prediction and analysis. <span class="phvro8t-x-x-120">BMC bioinformatics</span>, 11:129.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XReuter2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Reuter, M., Berninger, P., Chuma, S., Shah, H., Hosokawa, M., Funaya,
C., Antony, C., Sachidanandam, R., and Pillai, R.&nbsp;S. (2011).  Miwi catalysis
is required for piRNA amplification-independent LINE1 transposon silencing.
<span class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XReuter2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Reuter, M., Chuma, S., Tanaka, T., Franz, T., Stark, A., and Pillai, R.&nbsp;S.
                                                                                

                                                                                
(2009). Loss  of  the  Mili-interacting  Tudor  domain-containing  protein-1
activates transposons  and  alters  the  Mili-associated  small  RNA  profile.
<span class="phvro8t-x-x-120">Nature structural &amp; molecular biology</span>, 16(6):639–46.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRichardson1965a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Richardson, C. (1965).   Phosphorylation of Nucleic Acid by and Enzyme
from T4 Bacteriophate-infected Escherichia Coli. <span class="phvro8t-x-x-120">Proceedings of the National</span>
<span class="phvro8t-x-x-120">Academy of </span><span class="phvro8t-x-x-120">…</span>, 1372(1961):158–165.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRiley2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Riley, K.&nbsp;J.  and  Steitz,  J.&nbsp;A.  (2013).   Minireview  The  ‘‘  Observer  Effect
’’ in Genome-wide Surveys of Protein-RNA Interactions Minireview.   pages
2011–2014.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRo2007"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ro, S.,   Park,   C.,   Song,   R.,   Nguyen,   D.,   Jin,   J.,   Sanders,   K.&nbsp;M.,
McCarrey, J.&nbsp;R., and Yan, W. (2007).   Cloning and Expression Profiling of
Testis-Expressed piRNA-Like RNAs. <span class="phvro8t-x-x-120">RNA</span>, 13(10):1693–1702.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRoach1995"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Roach, J.  J.&nbsp;C.,  Boysen,  C.,  Wang,  K.,  Hood,  L.,  and  Wang,  I.  K.  A.&nbsp;I.
(1995). Pairwise End Sequencing: A unified approach to genomic mapping
and sequencing. <span class="phvro8t-x-x-120">Genomics</span>, 353:345–353.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRobine2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span>                     Robine,                     N.,                     Lau,                     N.&nbsp;C.,
Balla, S., Jin, Z., Okamura, K., Kuramochi-Miyagawa, S., Blower, M.&nbsp;D., and
Lai, E.&nbsp;C. (2009). A Broadly Conserved Pathway Generates 3’ UTR-Directed
Primary piRNAs. <span class="phvro8t-x-x-120">Current Biology</span>, 19(24):2066–2076.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRobinson2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Robinson,  J.&nbsp;T.,  Thorvaldsdóttir,  H.,  Winckler,  W.,  Guttman,  M.,  Lander,
E.&nbsp;S., Getz, G., and Mesirov, J.&nbsp;P. (2011). Integrative genomics viewer.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRomanienko2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Romanienko,  P.&nbsp;J.  and  Camerini-Otero,  R.&nbsp;D.  (2000).      The  mouse
Spo11 gene is required for meiotic chromosome synapsis.   <span class="phvro8t-x-x-120">Molecular cell</span>,
6(5):975–87.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRomero2009a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Romero,  P.&nbsp;a.  and  Arnold,  F.&nbsp;H.  (2009).      Exploring  protein  fitness
                                                                                

                                                                                
landscapes  by  directed  evolution.   <span class="phvro8t-x-x-120">Nature  reviews.  Molecular  cell  biology</span>,
10(12):866–76.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRonaghi1998a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ronaghi, M., Uhlén, M., and Nyrén, P. (1998). A sequencing method based
on real-time pyrophosphate. <span class="phvro8t-x-x-120">Science</span>, 281:363, 365.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XRumsfeld2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Rumsfeld, D. (2011). Known and unknown: a memoir.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSaito2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Saito, K.,  Inagaki,  S.,  Mituyama,  T.,  Kawamura,  Y.,  Ono,  Y.,  Sakota,  E.,
Kotani, H.,  Asai,  K.,  Siomi,  H.,  and  Siomi,  M.&nbsp;C.  (2009).    A  regulatory
circuit for piwi  by  the  large  Maf  gene  traffic  jam  in  Drosophila.    <span class="phvro8t-x-x-120">Nature</span>,
461(7268):1296–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSaito2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Saito, K., Ishizu, H., Komai, M., Kotani, H., Kawamura, Y., Nishida, K.&nbsp;M.,
Siomi, H., and  Siomi,  M.&nbsp;C.  (2010).   Roles  for  the  Yb  body  components
Armitage and  Yb  in  primary  piRNA  biogenesis  in  Drosophila.    <span class="phvro8t-x-x-120">Genes  &amp;</span>
<span class="phvro8t-x-x-120">development</span>, 24(22):2493–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSalehi-Ashtiani2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Salehi-Ashtiani, K., Yang, X., Derti, A., Tian, W., Hao, T., Lin, C., Makowski,
K., Shen, L., Murray, R.&nbsp;R., Szeto, D., Tusneem, N., Smith, D.&nbsp;R., Cusick,
M.&nbsp;E., Hill,  D.&nbsp;E.,  Roth,  F.&nbsp;P.,  and  Vidal,  M.  (2008).    Isoform  discovery
by targeted  cloning,  ’deep-well’  pooling  and  parallel  sequencing.    <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">Methods</span>, 5(7):597–600.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSander2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sander, J.&nbsp;D. and Joung, J.&nbsp;K. (2014).  CRISPR-Cas systems for editing,
regulating and targeting genomes. <span class="phvro8t-x-x-120">Nature biotechnology</span>, 32(4):347–355.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSanger1975a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sanger, F.  and  Coulson,  A.&nbsp;R.  (1975).   A  rapid  method  for  determining
sequences in DNA by primed synthesis with DNA polymerase.   <span class="phvro8t-x-x-120">Journal of</span>
<span class="phvro8t-x-x-120">molecular biology</span>, 94(3):441–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSanger1978a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sanger, F., Coulson, A.&nbsp;R., Friedmann, T., Air, G.&nbsp;M., Barrell, B.&nbsp;G., Brown,
N.&nbsp;L., Fiddes,  J.&nbsp;C.,  Hutchison,  C.&nbsp;A.,  Slocombe,  P.&nbsp;M.,  and  Smith,  M.
                                                                                

                                                                                
(1978). The  nucleotide  sequence  of  bacteriophage  phiX174.   <span class="phvro8t-x-x-120">Journal  of</span>
<span class="phvro8t-x-x-120">molecular biology</span>, 125(2):225–46.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSanger1977b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sanger, F.,  Nicklen,  S.,  and  Coulson,  A.&nbsp;R.  (1977).    DNA  sequencing
with chain-terminating inhibitors.   <span class="phvro8t-x-x-120">Proceedings of the National Academy of</span>
<span class="phvro8t-x-x-120">Sciences of the United States of America</span>, 74(12):5463–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchena1995a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schena,   M.,   Shalon,   D.,   Davis,   R.&nbsp;W.,   and   Brown,   P.&nbsp;O.   (1995).
Quantitative monitoring of gene expression patterns with a complementary
DNA microarray. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 270(5235):467–70.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchmucker2000"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schmucker, D., Clemens, J.&nbsp;C., Shu, H., Worby, C.&nbsp;A., Xiao, J., Muda, M.,
Dixon, J.&nbsp;E., and Zipursky, S. (2000). Drosophila Dscam Is an Axon Guidance
Receptor Exhibiting Extraordinary Molecular Diversity. <span class="phvro8t-x-x-120">Cell</span>, 101(6):671–684.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchneider1972"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schneider,  I.  (1972).     Cell  lines  derived  from  late  embryonic  stages
of Drosophila  melanogaster.     <span class="phvro8t-x-x-120">Journal  of  embryology  and  experimental</span>
<span class="phvro8t-x-x-120">morphology</span>, 27(2):353–65.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchoenmakers2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schoenmakers, S., Wassenaar, E., Hoogerbrugge, J.&nbsp;W., Laven, J. S.&nbsp;E.,
Grootegoed,  J.&nbsp;A.,  and  Baarends,  W.&nbsp;M.  (2009).     Female  meiotic  sex
chromosome inactivation in chicken. <span class="phvro8t-x-x-120">PLoS genetics</span>, 5(5):e1000466.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchulz2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schulz, M.&nbsp;H., Zerbino, D.&nbsp;R., Vingron, M., and Birney, E. (2012).  Oases:
robust de novo RNA-seq assembly across the dynamic range of expression
levels. <span class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 28:1086–92.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchwartz2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schwartz, S. and Ast, G. (2010). Chromatin density and splicing destiny: on
the cross-talk between chromatin structure and splicing.  <span class="phvro8t-x-x-120">The EMBO journal</span>,
29(10):1629–36.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSchwarzbauer1983"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Schwarzbauer, J. and Tamkun, J. (1983). Three different fibronectin mRNAs
arise by alternative splicing within the coding region. <span class="phvro8t-x-x-120">Cell</span>, 35(2 Pt 1):421–31.
                                                                                

                                                                                
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSeitz2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Seitz, H.,  Ghildiyal,  M.,  and  Zamore,  P.&nbsp;D.  (2008).    Argonaute  loading
improves the 5’ precision of both MicroRNAs and their miRNA* strands in
flies. <span class="phvro8t-x-x-120">Current biology : CB</span>, 18(2):147–51.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSerquina2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Serquiña,  A.  K.&nbsp;P.,  Das,  S.&nbsp;R.,  Popova,  E.,  Ojelabi,  O.&nbsp;A.,  Roy,  C.&nbsp;K.,
and Göttlinger,  H.&nbsp;G.  (2013).   UPF1  is  crucial  for  the  infectivity  of  human
immunodeficiency   virus   type   1   progeny   virions.      <span class="phvro8t-x-x-120">Journal   of   virology</span>,
87(16):8853–61.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShalek2013b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shalek, A.&nbsp;K., Satija, R., Adiconis, X., Gertner, R.&nbsp;S., Gaublomme, J.&nbsp;T.,
Raychowdhury, R., Schwartz, S., Yosef, N., Malboeuf, C., Lu, D., Trombetta,
J.&nbsp;T., Gennert,  D.,  Gnirke,  A.,  Goren,  A.,  Hacohen,  N.,  Levin,  J.&nbsp;Z.,  Park,
H., and Regev, A. (2013).   Single-cell transcriptomics reveals bimodality in
expression and splicing in immune cells. <span class="phvro8t-x-x-120">Nature</span>, pages 1–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShapiro2013b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shapiro,   E.,   Biezuner,   T.,   and   Linnarsson,   S.   (2013).      Single-cell
sequencing-based  technologies  will  revolutionize  whole-organism  science.
<span class="phvro8t-x-x-120">Nature reviews. Genetics</span>, 14(9):618–630.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSharon2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sharon,   D.,   Tilgner,   H.,   Grubert,   F.,   and   Snyder,   M.   (2013).      A
single-molecule  long-read  survey  of  the  human  transcriptome.     <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">biotechnology</span>, 31(11):1009–1014.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSharp2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sharp, P.&nbsp;A. (2014). Nobel Lectures in Physiology or Medicine 1991-1995.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShen-Orr2002"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shen-Orr, S.&nbsp;S., Milo, R., Mangan, S., and Alon, U. (2002). Network motifs
in the transcriptional regulation network of Escherichia coli.  <span class="phvro8t-x-x-120">Nature genetics</span>,
31(1):64–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShendure2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shendure,  J.  and  Aiden,  E.&nbsp;L.  (2012).    The  expanding  scope  of  DNA
sequencing. <span class="phvro8t-x-x-120">Nature Biotechnology</span>, 30(11):1084–1094.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XShendure2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shendure, J. and Ji, H. (2008).  Next-generation DNA sequencing.  <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">Biotechnology</span>, 26(10):1135–1145.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShendure2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shendure, J., Mitra, R.&nbsp;D., Varma, C., and Church, G.&nbsp;M. (2004). Advanced
sequencing  technologies:  methods  and  goals.   <span class="phvro8t-x-x-120">Nature  Reviews.  Genetics</span>,
5(5):335–44.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShendure2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shendure, J., Porreca, G.&nbsp;J., Reppas, N.&nbsp;B., Lin, X., McCutcheon, J.&nbsp;P.,
Rosenbaum,  A.&nbsp;M.,  Wang,  M.&nbsp;D.,  Zhang,  K.,  Mitra,  R.&nbsp;D.,  and  Church,
G.&nbsp;M. (2005). Accurate Multiplex Polony Sequencing of an Evolved Bacterial
Genome. <span class="phvro8t-x-x-120">Science</span>, 309(5741):1728–1732.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShepard2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shepard, P.&nbsp;J., Choi, E.-A., Lu, J., Flanagan, L.&nbsp;A., Hertel, K.&nbsp;J., and Shi,
Y. (2011). Complex and dynamic landscape of RNA polyadenylation revealed
by PAS-Seq. <span class="phvro8t-x-x-120">RNA</span>, 17(4):761 –772.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShi2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shi, L.   and   Lee,   T.   (2012).       Molecular   diversity   of   Dscam   and
self-recognition.        <span class="phvro8t-x-x-120">Advances   in   experimental   medicine   and   biology</span>,
739:262–75.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShingleton2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shingleton, A., Sisk, G., and Stern, D. (2003).  Diapause in the pea aphid
(Acyrthosiphon pisum) is a slowing but not a cessation of development. <span class="phvro8t-x-x-120">BMC</span>
<span class="phvro8t-x-x-120">developmental biology</span>, 12:1–12.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShiraki2003a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shiraki, T.,  Kondo,  S.,  Katayama,  S.,  Waki,  K.,  Kasukawa,  T.,  Kawaji,
H., Kodzius,  R.,  Watahiki,  A.,  Nakamura,  M.,  Arakawa,  T.,  Fukuda,  S.,
Sasaki, D.,   Podhajska,   A.,   Harbers,   M.,   Kawai,   J.,   Carninci,   P.,   and
Hayashizaki, Y. (2003).   Cap analysis gene expression for high-throughput
analysis of transcriptional starting point and identification of promoter usage.
<span class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of</span>
<span class="phvro8t-x-x-120">America</span>, 100:15776–15781.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShirayama2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shirayama, M., Seth, M., Lee, H.-C., Gu, W., Ishidate, T., Conte, D., and
                                                                                

                                                                                
Mello, C.&nbsp;C. (2012). piRNAs initiate an epigenetic memory of nonself RNA in
the C. elegans germline. <span class="phvro8t-x-x-120">Cell</span>, 150(1):65–77.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShiroguchi2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shiroguchi, K., Jia, T.&nbsp;Z., Sims, P.&nbsp;a., and Xie, X.&nbsp;S. (2012).  Digital RNA
sequencing minimizes sequence-dependent bias and amplification noise with
optimized single-molecule barcodes.  <span class="phvro8t-x-x-120">Proceedings of the National Academy</span>
<span class="phvro8t-x-x-120">of Sciences of the United States of America</span>, 109(4):1347–52.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XShoji2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Shoji, M.,  Tanaka,  T.,  Hosokawa,  M.,  Reuter,  M.,  Stark,  A.,  Kato,  Y.,
Kondoh, G., Okawa, K., Chujo, T., Suzuki, T., Hata, K., Martin, S.&nbsp;L., Noce,
T., Kuramochi-Miyagawa, S., Nakano, T., Sasaki, H., Pillai, R.&nbsp;S., Nakatsuji,
N., and Chuma,  S.  (2009).    The  TDRD9-MIWI2  complex  is  essential  for
piRNA-mediated  retrotransposon  silencing  in  the  mouse  male  germline.
<span class="phvro8t-x-x-120">Developmental cell</span>, 17(6):775–87.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSingh2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Singh, G.,   Ricci,   E.&nbsp;P.,   and   Moore,   M.&nbsp;J.   (2014).       RIPiT-Seq:   a
high-throughput approach for footprinting RNA:protein complexes.  <span class="phvro8t-x-x-120">Methods</span>
<span class="phvro8t-x-x-120">(San Diego, Calif.)</span>, 65(3):320–32.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSiomi2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Siomi, M.&nbsp;C., Sato, K., Pezic, D., and Aravin, A.&nbsp;A. (2011). PIWI-interacting
small RNAs:  the  vanguard  of  genome  defence.    <span class="phvro8t-x-x-120">Nat  Rev  Mol  Cell  Biol</span>,
12(4):246–258.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSmith2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Smith, J.&nbsp;M.,  Bowles,  J.,  Wilson,  M.,  Teasdale,  R.&nbsp;D.,  and  Koopman,  P.
(2004). Expression of the tudor-related gene Tdrd5 during development of
the male germline in mice. <span class="phvro8t-x-x-120">Gene expression patterns : GEP</span>, 4(6):701–5.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSouthern2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Southern, E.&nbsp;M. (2001). DNA microarrays. History and overview. <span class="phvro8t-x-x-120">Methods</span>
<span class="phvro8t-x-x-120">in Molecular Biology (Clifton, N.J.)</span>, 170:1–15.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSriskanda1998c"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sriskanda,   V.   and   Shuman,   S.   (1998).      Specificity   and   fidelity   of
strand joining  by  Chlorella  virus  DNA  ligase.     <span class="phvro8t-x-x-120">Nucleic  acids  research</span>,
26(15):3536–41.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XStaden1979"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Staden, R.  (1979).   A  strategy  of  DNA  sequencing  employing  computer
programs. <span class="phvro8t-x-x-120">Nucleic Acids Research</span>, 6(7):2601–2610.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSteijger2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Steijger, T., Abril, J.&nbsp;F., Engström, P.&nbsp;G., Kokocinski, F., Akerman, M., Alioto,
T., Ambrosini,  G.,  Antonarakis,  S.&nbsp;E.,  Behr,  J.,  Bertone,  P.,  Bohnert,  R.,
Bucher, P., Cloonan, N., Derrien, T., Djebali, S., Du, J., Dudoit, S., Gerstein,
M., Gingeras, T.&nbsp;R., Gonzalez, D., Grimmond, S.&nbsp;M., Guigó, R., Habegger, L.,
Harrow, J., Hubbard, T.&nbsp;J., Iseli, C., Jean, G., Kahles, A., Lagarde, J., Leng,
J., Lefebvre, G., Lewis, S., Mortazavi, A., Niermann, P., Rätsch, G., Reymond,
A., Ribeca,  P.,  Richard,  H.,  Rougemont,  J.,  Rozowsky,  J.,  Sammeth,  M.,
Sboner, A.,  Schulz,  M.&nbsp;H.,  Searle,  S.  M.&nbsp;J.,  Solorzano,  N.&nbsp;D.,  Solovyev,
V., Stanke,  M.,  Stevenson,  B.&nbsp;J.,  Stockinger,  H.,  Valsesia,  A.,  Weese,  D.,
White, S., Wold, B.&nbsp;J., Wu, J., Wu, T.&nbsp;D., Zeller, G., Zerbino, D., and Zhang,
M.&nbsp;Q. (2013). Assessment of transcript reconstruction methods for RNA-seq.
<span class="phvro8t-x-x-120">Nature methods</span>, 10(12):1177–84.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XStemmer1994"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Stemmer,  W.&nbsp;P.  (1994).    Rapid  evolution  of  a  protein  in  vitro  by  DNA
shuffling. <span class="phvro8t-x-x-120">Nature</span>, 370(6488):389–91.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSubtelny2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Subtelny, A.&nbsp;O., Eichhorn, S.&nbsp;W., Chen, G.&nbsp;R., Sive, H., and Bartel, D.&nbsp;P.
(2014). Poly(A)-tail  profiling  reveals  an  embryonic  switch  in  translational
control. <span class="phvro8t-x-x-120">Nature</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSultan2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sultan, M., Schulz, M.&nbsp;H., Richard, H., Magen, A., Klingenhoff, A., Scherf,
M., Seifert,  M.,  Borodina,  T.,  Soldatov,  A.,  Parkhomchuk,  D.,  Schmidt,  D.,
O’Keeffe, S., Haas, S., Vingron, M., Lehrach, H., and Yaspo, M.-L. (2008).  A
Global View of Gene Activity and Alternative Splicing by Deep Sequencing of
the Human Transcriptome. <span class="phvro8t-x-x-120">Science</span>, 321(5891):956–960.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSummers1970b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Summers, W. and Siegel, R. (1970).  Transcription of Late Phage RNA by
T7 RNA Polymerase. <span class="phvro8t-x-x-120">Nature</span>, 228:1160–1162.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XSun2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Sun, W., You, X., Gogol-Döring, A., He, H., Kise, Y., Sohn, M., Chen, T.,
Klebes, A.,  Schmucker,  D.,  and  Chen,  W.  (2013).    Ultra-deep  profiling  of
                                                                                

                                                                                
alternatively  spliced  Drosophila  Dscam  isoforms  by  circularization-assisted
multi-segment sequencing. <span class="phvro8t-x-x-120">The EMBO journal</span>, 32(14):2029–38.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTabor1987a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Tabor, S. (1987).   DNA Ligases.   <span class="phvro8t-x-x-120">Current Protocols in Molecular Biology</span>,
pages 3.14.1–3.14.4.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTani2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Tani, H.,  Mizutani,  R.,  Salam,  K.&nbsp;A.,  Tano,  K.,  Ijiri,  K.,  Wakamatsu,  A.,
Isogai, T., Suzuki, Y., and Akimitsu, N. (2012).  Genome-wide determination
of RNA stability  reveals  hundreds  of  short-lived  noncoding  transcripts  in
mammals. <span class="phvro8t-x-x-120">Genome research</span>, 22(5):947–56.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTazi2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Tazi, J.,  Bakkour,  N.,  and  Stamm,  S.  (2009).    Alternative  splicing  and
disease. <span class="phvro8t-x-x-120">Biochimica et biophysica acta</span>, 1792(1):14–26.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XThierry-Mieg2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Thierry-Mieg, D. and Thierry-Mieg, J. (2006).  AceView: a comprehensive
cDNA-supported gene and transcripts annotation.  <span class="phvro8t-x-x-120">Genome biology</span>, 7 Suppl
1(Suppl 1):S12.1–14.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XThomson2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Thomson,  T.  and  Lin,  H.  (2009).   The  biogenesis  and  function  of  PIWI
proteins and  piRNAs:  progress  and  prospect.    <span class="phvro8t-x-x-120">Annual  review  of  cell  and</span>
<span class="phvro8t-x-x-120">developmental biology</span>, 25:355–76.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XToscani1997"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Toscani,  A.,  Mettus,  R.&nbsp;V.,  Coupland,  R.,  Simpkins,  H.,  Litvin,  J.,  Orth,
J., Hatton,  K.&nbsp;S.,  and  Reddy,  E.&nbsp;P.  (1997).    Arrest  of  spermatogenesis
and defective   breast   development   in   mice   lacking   A-myb.       <span class="phvro8t-x-x-120">Nature</span>,
386(6626):713–717.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTrapnell2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Trapnell, C., Pachter, L., and Salzberg, S.&nbsp;L. (2009).  TopHat: discovering
splice junctions with RNA-Seq. <span class="phvro8t-x-x-120">Bioinformatics</span>, 25(9):1105 –1111.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTrapnell2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Trapnell,  C.,  Roberts,  A.,  Goff,  L.,  Pertea,  G.,  Kim,  D.,  Kelley,  D.&nbsp;R.,
Pimentel, H., Salzberg, S.&nbsp;L., Rinn, J.&nbsp;L., and Pachter, L. (2012). Differential
gene and transcript expression analysis of RNA-seq experiments with TopHat
                                                                                

                                                                                
and Cufflinks. <span class="phvro8t-x-x-120">Nature Protocols</span>, 7(3):562–578.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTrapnell2010"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Trapnell,  C.,  Williams,  B.&nbsp;A.,  Pertea,  G.,  Mortazavi,  A.,  Kwan,  G.,  van
Baren, M.&nbsp;J., Salzberg, S.&nbsp;L., Wold, B.&nbsp;J., and Pachter, L. (2010). Transcript
assembly and quantification by RNA-Seq reveals unannotated transcripts and
isoform switching during cell differentiation. <span class="phvro8t-x-x-120">Nat Biotech</span>, 28(5):511–515.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTrauth1994"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Trauth, K., Mutschler, B., Jenkins, N.&nbsp;A., Gilbert, D.&nbsp;J., Copeland, N.&nbsp;G.,
and Klempnauer, K.-h. (1994). Mouse A-myb encodes a trans-activator and is
expressed in mitotically active cells of the developing central nervous system,
adult testis and B lymphocytes. <span class="phvro8t-x-x-120">EMBO</span>, 13(24):5994–6005.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XTreutlein2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Treutlein,   B.,   Gokce,   O.,   Quake,   S.&nbsp;R.,   and   Südhof,   T.&nbsp;C.   (2014).
Cartography  of  neurexin  alternative  splicing  mapped  by  single-molecule
long-read mRNA  sequencing.    <span class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of</span>
<span class="phvro8t-x-x-120">Sciences of the United States of America</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XUle2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ule, J.,  Jensen,  K.,  Mele,  A.,  and  Darnell,  R.&nbsp;B.  (2005).     CLIP:  A
method for identifying protein-RNA interaction sites in living cells.  <span class="phvro8t-x-x-120">Methods</span>,
37(4):376–386.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XUshkaryov1992"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Ushkaryov, Y.&nbsp;a., Petrenko, a.&nbsp;G., Geppert, M., and Südhof, T.&nbsp;C. (1992).
Neurexins:  synaptic  cell  surface  proteins  related  to  the  alpha-latrotoxin
receptor and laminin. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 257(5066):50–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVagin2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Vagin, V.&nbsp;V.,  Klenov,  M.&nbsp;S.,  Stolyarenko,  A.&nbsp;D.,  and  Kotelnikov,  R.&nbsp;N.
(2004). The  RNA  Interference  Proteins  and  Vasa  Locus  are  Involved  in
the Silencing  of  Retrotransposons  in  the  Female  Germline  of  Drosophila
melanogaster. <span class="phvro8t-x-x-120">RNA Biology</span>, (June):54–58.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVagin2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Vagin, V.&nbsp;V., Sigova, A., Li, C., Seitz, H., Gvozdev, V., and Zamore, P.&nbsp;D.
(2006). A Distinct Small RNA Pathway Silences Selfish Genetic Elements in
the Germline. <span class="phvro8t-x-x-120">Science</span>, 313(5785):320 –324.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XVagin2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Vagin, V.&nbsp;V.,  Wohlschlegel,  J.,  Qu,  J.,  Jonsson,  Z.,  Huang,  X.,  Chuma,
S., Girard, A., Sachidanandam, R., Hannon, G.&nbsp;J., and Aravin, A.&nbsp;a. (2009).
Proteomic  analysis  of  murine  Piwi  proteins  reveals  a  role  for  arginine
methylation in specifying interaction with Tudor family members.   <span class="phvro8t-x-x-120">Genes &amp;</span>
<span class="phvro8t-x-x-120">development</span>, 23(15):1749–62.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVasileva2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Vasileva, A., Tiedau, D., Firooznia, A., Müller-Reichert, T., and Jessberger,
R. (2009).       Tdrd6   is   required   for   spermiogenesis,   chromatoid   body
architecture,  and  regulation  of  miRNA  expression.   <span class="phvro8t-x-x-120">Current  biology  :  CB</span>,
19(8):630–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVelculescu1995a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Velculescu,   V.&nbsp;E.,   Zhang,   L.,   Vogelstein,   B.,   and   Kinzler,   K.&nbsp;W.
(1995). Serial  analysis  of  gene  expression.    <span class="phvro8t-x-x-120">Science  (New  York,  N.Y.)</span>,
270(5235):484–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVenter2008a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Venter, J.  (2007).     <span class="phvro8t-x-x-120">A  life  decoded:  my  genome,  my  life</span>.     Penguin
(Non-Classics).
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVenter2001"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Venter, J.&nbsp;C., Adams, M.&nbsp;D., Myers, E.&nbsp;W., Li, P.&nbsp;W., Mural, R.&nbsp;J., Sutton,
G.&nbsp;G., Smith, H.&nbsp;O., Yandell, M., Evans, C.&nbsp;a., Holt, R.&nbsp;a., Gocayne, J.&nbsp;D.,
Amanatides,  P.,  Ballew,  R.&nbsp;M.,  Huson,  D.&nbsp;H.,  Wortman,  J.&nbsp;R.,  Zhang,  Q.,
Kodira, C.&nbsp;D.,  Zheng,  X.&nbsp;H.,  Chen,  L.,  Skupski,  M.,  Subramanian,  G.,
Thomas, P.&nbsp;D., Zhang, J., Gabor Miklos, G.&nbsp;L., Nelson, C., Broder, S., Clark,
a.&nbsp;G., Nadeau, J., McKusick, V.&nbsp;a., Zinder, N., Levine, a.&nbsp;J., Roberts, R.&nbsp;J.,
Simon, M., Slayman, C., Hunkapiller, M., Bolanos, R., Delcher, A., Dew, I.,
Fasulo, D., Flanigan, M., Florea, L., Halpern, A., Hannenhalli, S., Kravitz, S.,
Levy, S., Mobarry, C., Reinert, K., Remington, K., Abu-Threideh, J., Beasley,
E., Biddick, K., Bonazzi, V., Brandon, R., Cargill, M., Chandramouliswaran, I.,
Charlab, R., Chaturvedi, K., Deng, Z., Di Francesco, V., Dunn, P., Eilbeck, K.,
Evangelista, C., Gabrielian, a.&nbsp;E., Gan, W., Ge, W., Gong, F., Gu, Z., Guan,
P., Heiman, T.&nbsp;J., Higgins, M.&nbsp;E., Ji, R.&nbsp;R., Ke, Z., Ketchum, K.&nbsp;a., Lai, Z.,
Lei, Y., Li, Z., Li, J., Liang, Y., Lin, X., Lu, F., Merkulov, G.&nbsp;V., Milshina, N.,
Moore, H.&nbsp;M., Naik, a.&nbsp;K., Narayan, V.&nbsp;a., Neelam, B., Nusskern, D., Rusch,
D.&nbsp;B., Salzberg, S., Shao, W., Shue, B., Sun, J., Wang, Z., Wang, A., Wang,
X., Wang, J.,  Wei,  M.,  Wides,  R.,  Xiao,  C.,  Yan,  C.,  Yao,  A.,  Ye,  J.,  Zhan,
M., Zhang, W., Zhang, H., Zhao, Q., Zheng, L., Zhong, F., Zhong, W., Zhu,
                                                                                

                                                                                
S., Zhao, S., Gilbert, D., Baumhueter, S., Spier, G., Carter, C., Cravchik, A.,
Woodage, T., Ali, F., An, H., Awe, A., Baldwin, D., Baden, H., Barnstead, M.,
Barrow, I., Beeson, K., Busam, D., Carver, A., Center, A., Cheng, M.&nbsp;L., Curry,
L., Danaher,  S.,  Davenport,  L.,  Desilets,  R.,  Dietz,  S.,  Dodson,  K.,  Doup,
L., Ferriera,  S.,  Garg,  N.,  Gluecksmann,  A.,  Hart,  B.,  Haynes,  J.,  Haynes,
C., Heiner,  C.,  Hladun,  S.,  Hostin,  D.,  Houck,  J.,  Howland,  T.,  Ibegwam,
C., Johnson, J., Kalush, F., Kline, L., Koduru, S., Love, A., Mann, F., May,
D., McCawley, S., McIntosh, T., McMullen, I., Moy, M., Moy, L., Murphy, B.,
Nelson, K.,  Pfannkoch,  C.,  Pratts,  E.,  Puri,  V.,  Qureshi,  H.,  Reardon,  M.,
Rodriguez, R., Rogers, Y.&nbsp;H., Romblad, D., Ruhfel, B., Scott, R., Sitter, C.,
Smallwood,  M.,  Stewart,  E.,  Strong,  R.,  Suh,  E.,  Thomas,  R.,  Tint,  N.&nbsp;N.,
Tse, S., Vech, C., Wang, G., Wetter, J., Williams, S., Williams, M., Windsor,
S., Winn-Deen,  E.,  Wolfe,  K.,  Zaveri,  J.,  Zaveri,  K.,  Abril,  J.&nbsp;F.,  Guigó,  R.,
Campbell, M.&nbsp;J., Sjolander, K.&nbsp;V., Karlak, B., Kejariwal, A., Mi, H., Lazareva,
B., Hatton, T., Narechania, A., Diemer, K., Muruganujan, A., Guo, N., Sato, S.,
Bafna, V., Istrail, S., Lippert, R., Schwartz, R., Walenz, B., Yooseph, S., Allen,
D., Basu, A., Baxendale, J., Blick, L., Caminha, M., Carnes-Stine, J., Caulk, P.,
Chiang, Y.&nbsp;H., Coyne, M., Dahlke, C., Mays, A., Dombroski, M., Donnelly, M.,
Ely, D., Esparham, S., Fosler, C., Gire, H., Glanowski, S., Glasser, K., Glodek,
A., Gorokhov, M., Graham, K., Gropman, B., Harris, M., Heil, J., Henderson,
S., Hoover, J., Jennings, D., Jordan, C., Jordan, J., Kasha, J., Kagan, L., Kraft,
C., Levitsky, A., Lewis, M., Liu, X., Lopez, J., Ma, D., Majoros, W., McDaniel,
J., Murphy,  S.,  Newman,  M.,  Nguyen,  T.,  Nguyen,  N.,  Nodell,  M.,  Pan,  S.,
Peck, J., Peterson, M., Rowe, W., Sanders, R., Scott, J., Simpson, M., Smith,
T., Sprague, A., Stockwell, T., Turner, R., Venter, E., Wang, M., Wen, M., Wu,
D., Wu, M., Xia, A., Zandieh, A., and Zhu, X. (2001).  The sequence of the
human genome. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 291(5507):1304–51.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XViollet2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Viollet, S., Fuchs, R.&nbsp;T., Munafo, D.&nbsp;B., Zhuang, F., and Robb, G.&nbsp;B. (2011).
T4 RNA Ligase  2  truncated  active  site  mutants:  improved  tools  for  RNA
analysis. <span class="phvro8t-x-x-120">BMC Biotechnology</span>, 11(1):72.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XVourekas2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Vourekas, A., Zheng, Q., Alexiou, P., Maragkakis, M., Kirino, Y., Gregory,
B.&nbsp;D., and Mourelatos, Z. (2012). Mili and Miwi target RNA repertoire reveals
piRNA biogenesis and function of Miwi in spermiogenesis.  <span class="phvro8t-x-x-120">Nature Structural</span>
<span class="phvro8t-x-x-120">&amp; Molecular Biology</span>, 19(8):773–781.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWan2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wan, Y.,  Qu,  K.,  Zhang,  Q.&nbsp;C.,  Flynn,  R.&nbsp;a.,  Manor,  O.,  Ouyang,  Z.,
                                                                                

                                                                                
Zhang, J., Spitale, R.&nbsp;C., Snyder, M.&nbsp;P., Segal, E., and Chang, H.&nbsp;Y. (2014).
Landscape  and  variation  of  RNA  secondary  structure  across  the  human
transcriptome. <span class="phvro8t-x-x-120">Nature</span>, 505(7485):706–9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWang2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wang, E.&nbsp;T.,  Sandberg,  R.,  Luo,  S.,  Khrebtukova,  I.,  Zhang,  L.,  Mayr,
C., Kingsmore, S.&nbsp;F., Schroth, G.&nbsp;P., and Burge, C.&nbsp;B. (2008).   Alternative
isoform regulation in human tissue transcriptomes. <span class="phvro8t-x-x-120">Nature</span>, 456(7221):470–6.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWang2013b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wang, L. and Yi, R. (2013). 3’ UTRs take a long shot in the brain. <span class="phvro8t-x-x-120">BioEssays</span>,
pages 1–7.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWang2002b"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wang, L.&nbsp;K.,  Lima,  C.&nbsp;D.,  and  Shuman,  S.  (2002).     Structure  and
mechanism of T4 polynucleotide kinase: an RNA repair enzyme. <span class="phvro8t-x-x-120">The EMBO</span>
<span class="phvro8t-x-x-120">journal</span>, 21(14):3873–80.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWashietl2014"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Washietl, S., Kellis, M., and Garber, M. (2014).  Evolutionary dynamics and
tissue specificity of human long noncoding RNAs in six mammals.  <span class="phvro8t-x-x-120">Genome</span>
<span class="phvro8t-x-x-120">research</span>, pages 616–628.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWatanabe2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Watanabe, T., Chuma, S., Yamamoto, Y., Kuramochi-Miyagawa, S., Totoki,
Y., Toyoda, A., Hoki, Y., Fujiyama, A., Shibata, T., Sado, T., Noce, T., Nakano,
T., Nakatsuji, N., Lin, H., and Sasaki, H. (2011a). MITOPLD Is a Mitochondrial
Protein Essential for Nuage Formation and piRNA Biogenesis in the Mouse
Germline. <span class="phvro8t-x-x-120">Developmental Cell</span>, 20(3):364–375.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWatanabe2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Watanabe,  T.,  Tomizawa,  S.-i.,  Mitsuya,  K.,  Totoki,  Y.,  Yamamoto,  Y.,
Kuramochi-Miyagawa, S., Iida, N., Hoki, Y., Murphy, P.&nbsp;J., Toyoda, A., Gotoh,
K., Hiura, H., Arima, T., Fujiyama, A., Sado, T., Shibata, T., Nakano, T., Lin, H.,
Ichiyanagi, K., Soloway, P.&nbsp;D., and Sasaki, H. (2011b).  Role for piRNAs and
Noncoding RNA in de Novo DNA Methylation of the Imprinted Mouse Rasgrf1
Locus. <span class="phvro8t-x-x-120">Science</span>, 332(6031):848 –852.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWaterhouse2009"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Waterhouse, A.&nbsp;M., Procter, J.&nbsp;B., Martin, D. M.&nbsp;A., Clamp, M., and Barton,
G.&nbsp;J. (2009).   Jalview Version 2–a multiple sequence alignment editor and
                                                                                

                                                                                
analysis workbench. <span class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 25(9):1189–91.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWatson2005"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Watson, F.&nbsp;L., Püttmann-Holgado, R., Thomas, F., Lamar, D.&nbsp;L., Hughes,
M., Kondo, M., Rebel, V.&nbsp;I., and Schmucker, D. (2005). Extensive diversity of
Ig-superfamily proteins in the immune system of insects. <span class="phvro8t-x-x-120">Science (New York,</span>
<span class="phvro8t-x-x-120">N.Y.)</span>, 309(5742):1874–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWatson2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Watson, J.&nbsp;D.,  Gann,  A.,  and  Witkowski,  J.  (2012).   <span class="phvro8t-x-x-120">The  Annotated  and</span>
<span class="phvro8t-x-x-120">Illustrated Double Helix</span>. Simon &amp; Schuster.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWatson1953a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Watson, J. D.&nbsp;J. and Crick, F. (1953).  Molecular structure of nucleic acids.
<span class="phvro8t-x-x-120">Nature</span>, 4356(171):737–738.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWei1975"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wei, C.&nbsp;M., Gershowitz, a., and Moss, B. (1975).  Methylated nucleotides
block 5’ terminus of HeLa cell messenger RNA. <span class="phvro8t-x-x-120">Cell</span>, 4(4):379–86.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWeiss1967a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Weiss, B. and Richardson, C. (1967).  Enzymatic breakage and joining of
deoxyribonucleic acid, I. Repair of single-strand breaks in DNA by an enzyme
system from Escherichia coli infected with T4 bacteriophage. <span class="phvro8t-x-x-120">Proceedings of</span>
<span class="phvro8t-x-x-120">the National Academy of </span><span class="phvro8t-x-x-120">…</span>.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWeston1992"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Weston, K. (1992). Extension of the DNA binding consensus of the chicken
c-Myb and v-Myb proteins. <span class="phvro8t-x-x-120">Nucleic acids research</span>, 20(12).
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWhite2011a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> White, E.&nbsp;S.   and   Muro,   A.&nbsp;F.   (2011).      Fibronectin   splice   variants:
understanding  their  multiple  roles  in  health  and  disease  using  engineered
mouse models. <span class="phvro8t-x-x-120">IUBMB life</span>, 63(7):538–46.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWills2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wills, Q.&nbsp;F., Livak, K.&nbsp;J., Tipping, A.&nbsp;J., Enver, T., Goldson, A.&nbsp;J., Sexton,
D.&nbsp;W., and  Holmes,  C.  (2013).      Single-cell  gene  expression  analysis
reveals genetic  associations  masked  in  whole-tissue  experiments.   <span class="phvro8t-x-x-120">Nature</span>
<span class="phvro8t-x-x-120">biotechnology</span>, 31(8):748–752.
</p>
                                                                                

                                                                                
<p class="bibitem"><span class="biblabel">
<a id="XWojtowicz2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wojtowicz,  W.&nbsp;M.,  Flanagan,  J.&nbsp;J.,  Millard,  S.&nbsp;S.,  Zipursky,  S.&nbsp;L.,  and
Clemens, J.&nbsp;C. (2004).  Alternative Splicing of Drosophila Dscam Generates
Axon Guidance Receptors that Exhibit Isoform-Specific Homophilic Binding.
<span class="phvro8t-x-x-120">Cell</span>, 118(5):619–633.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XWu1999"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Wu, Q. and Maniatis, T. (1999).  A striking organization of a large family of
human neural cadherin-like cell adhesion genes. <span class="phvro8t-x-x-120">Cell</span>, 97(6):779–90.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XXia1998"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Xia, T., SantaLucia,  J.,  Burkard,  M.&nbsp;E.,  Kierzek,  R.,  Schroeder,  S.&nbsp;J.,
Jiao, X., Cox, C., and Turner, D.&nbsp;H. (1998).  Thermodynamic parameters for
an expanded  nearest-neighbor  model  for  formation  of  RNA  duplexes  with
Watson-Crick base pairs. <span class="phvro8t-x-x-120">Biochemistry</span>, 37(42):14719–35.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYabuta2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Yabuta, Y.,  Ohta,  H.,  Abe,  T.,  Kurimoto,  K.,  Chuma,  S.,  and  Saitou,
M. (2011).    TDRD5  is  required  for  retrotransposon  silencing,  chromatoid
body assembly,  and  spermiogenesis  in  mice.   <span class="phvro8t-x-x-120">The  Journal  of  cell  biology</span>,
192(5):781–95.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYamakawa1998a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Yamakawa,  K.,  Huot,  Y.&nbsp;K.,  Haendelt,  M.&nbsp;a.,  Hubert,  R.,  Chen,  X.&nbsp;N.,
Lyons, G.&nbsp;E., and Korenberg, J.&nbsp;R. (1998).  DSCAM: a novel member of the
immunoglobulin superfamily maps in a Down syndrome region and is involved
in the development  of  the  nervous  system.    <span class="phvro8t-x-x-120">Human  molecular  genetics</span>,
7(2):227–37.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYang2011"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Yang, Y., Zhan, L., Zhang, W., Sun, F., Wang, W., Tian, N., Bi, J., Wang, H.,
Shi, D., Jiang, Y., Zhang, Y., and Jin, Y. (2011).  RNA secondary structure in
mutually exclusive splicing. <span class="phvro8t-x-x-120">Nat Struct Mol Biol</span>, 18(2):159–168.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYeakley2002"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Yeakley, J.&nbsp;M., Fan, J.-B., Doucet, D., Luo, L., Wickham, E., Ye, Z., Chee,
M.&nbsp;S., and Fu, X.-D. (2002). Profiling alternative splicing on fiber-optic arrays.
<span class="phvro8t-x-x-120">Nature Biotechnology</span>, 20(4):353–358.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYin2003d"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Yin, S., Ho, C.&nbsp;K., and Shuman, S. (2003).  Structure-function analysis of
                                                                                

                                                                                
T4 RNA ligase 2. <span class="phvro8t-x-x-120">The Journal of biological chemistry</span>, 278(20):17601–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XYou2006"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> You, Y., Moreira, B.&nbsp;G., Behlke, M.&nbsp;a., and Owczarzy, R. (2006). Design of
LNA probes that improve mismatch discrimination.  <span class="phvro8t-x-x-120">Nucleic acids research</span>,
34(8):e60.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZerbino2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zerbino, D.&nbsp;R. and Birney, E. (2008).  Velvet: algorithms for de novo short
read assembly using de Bruijn graphs. <span class="phvro8t-x-x-120">Genome research</span>, 18:821–829.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZhan2004"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zhan, X.-L.,  Clemens,  J.&nbsp;C.,  Neves,  G.,  Hattori,  D.,  Flanagan,  J.&nbsp;J.,
Hummel, T.,  Vasconcelos,  M.&nbsp;L.,  Chess,  A.,  and  Zipursky,  S.&nbsp;L.  (2004).
Analysis of  Dscam  Diversity  in  Regulating  Axon  Guidance  in  Drosophila
Mushroom Bodies. <span class="phvro8t-x-x-120">Neuron</span>, 43(5):673–686.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZhang2012"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zhang, F.,  Wang,  J.,  Xu,  J.,  Zhang,  Z.,  Koppetsch,  B.&nbsp;S.,  Schultz,
N., Vreven,  T.,  Meignin,  C.,  Davis,  I.,  Zamore,  P.&nbsp;D.,  Weng,  Z.,  and
Theurkauf, W.&nbsp;E. (2012a). UAP56 Couples piRNA Clusters to the Perinuclear
Transposon Silencing Machinery. <span class="phvro8t-x-x-120">Cell</span>, 151(4):871–884.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZhang2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zhang, Y., Liu, T., Meyer, C.&nbsp;a., Eeckhoute, J., Johnson, D.&nbsp;S., Bernstein,
B.&nbsp;E., Nusbaum, C., Myers, R.&nbsp;M., Brown, M., Li, W., and Liu, X.&nbsp;S. (2008).
Model-based analysis of ChIP-Seq (MACS). <span class="phvro8t-x-x-120">Genome biology</span>, 9(9):R137.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZhang2012a"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zhang, Z.,  Theurkauf,  W.&nbsp;E.,  Weng,  Z.,  and  Zamore,  P.&nbsp;D.  (2012b).
Strand-specific  libraries  for  high  throughput  RNA  sequencing  (  RNA-Seq  )
prepared without poly ( A ) selection. <span class="phvro8t-x-x-120">Silence</span>, 3(1):9.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZhu2003"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zhu, J., Shendure,  J.,  Mitra,  R.&nbsp;D.,  and  Church,  G.&nbsp;M.  (2003).   Single
molecule profiling of alternative pre-mRNA splicing. <span class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
301(5634):836–8.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZikherman2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zikherman, J. and Weiss, A. (2008).  Alternative Splicing of CD45: The Tip
of the Iceberg. <span class="phvro8t-x-x-120">Immunity</span>, 29(6):839–841.
                                                                                

                                                                                
</p>
<p class="bibitem"><span class="biblabel">
<a id="XLawrenceZipursky2013"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zipursky,  S.&nbsp;L.  and  Grueber,  W.&nbsp;B.  (2013).     The  molecular  basis  of
self-avoidance. <span class="phvro8t-x-x-120">Annual review of neuroscience</span>, 36(1):547–68.
</p>
<p class="bibitem"><span class="biblabel">
<a id="XZweig2008"></a><span class="bibsp">&nbsp;&nbsp;&nbsp;</span></span> Zweig, A.&nbsp;S.,  Karolchik,  D.,  Kuhn,  R.&nbsp;M.,  Haussler,  D.,  and  Kent,  W.&nbsp;J.
(2008). UCSC genome browser tutorial.
</p>
</div>
                                                                                

                                                                                
 
 

                                                                                

                                                                                
                                                                                


</div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></div></body></html>