<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"  
  "http://www.w3.org/TR/html4/loose.dtd">  
<html > 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<!-- html,htex4ht --> 
<meta name="src" content="RoyC_Umass_Thesis.v0.5.tex"> 
<meta name="date" content="2014-04-05 21:38:00"> 
<link rel="stylesheet" type="text/css" href="RoyC_Umass_Thesis.v0.5.css"> 
</head><body 
>
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<!--l. 4--><p class="noindent" >
                                                                                                    
                                                                                                    
<div class="center" 
>
<!--l. 5--><p class="noindent" >
<!--l. 6--><p class="noindent" ><span 
class="phvr8t-x-x-248">EXAMINATION OF DYNAMIC LONG RNAS</span><br /><br />
<span 
class="phvr8t-x-x-144">A Dissertation Presented</span><br /><br />
<span 
class="phvr8t-x-x-144">By</span><br /><br />
<span 
class="phvr8t-x-x-144">Christian K. Roy</span><br /><br />
<span 
class="phvr8t-x-x-144">Submitted to the Faculty of the University of the</span><br />
<span 
class="phvr8t-x-x-144">Massachusetts Graduate School of Biomedical Sciences, Worcester</span><br />
<span 
class="phvr8t-x-x-144">in partial filfillment of the requirements</span><br />
<span 
class="phvr8t-x-x-144">for the degree of</span><br /><br />
<span 
class="phvr8t-x-x-144">DOCTOR OF PHILOSOPHY</span><br /><br />
<span 
class="phvr8t-x-x-144">MAY 21st 2014</span><br /><br />
<span 
class="phvr8t-x-x-144">BIOCHEMISTRY</span><br /><br />
</div>
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<!--l. 4--><p class="noindent" >
                                                                                                    
                                                                                                    
<a 
 id="x1-1doc"></a>
<div class="center" 
>
<!--l. 5--><p class="noindent" >
<!--l. 6--><p class="noindent" >EXAMINATION OF DYNAMIC LONG RNAS<br />
A Dissertation Presented<br />
By<br />
Christian K. Roy<br />
The signatures of the Dissertation Defense Committee signify<br />
completion and approval as to style and content of the Dissertation<br /><br />
_________________________________________________________
                                                         <br />Melissa J. Moore, Co-Thesis Advisor<br /><br />
_________________________________________________________
                                                         <br />Phillip D. Zamore, Co-Thesis Advisor<br /><br />
_________________________________________________________
                                                         <br />Scot Wolfe, Member of Committee<br /><br />
_________________________________________________________
                                                         <br />Job Dekker, Member of Committee<br />
The signature of the Chair of the Committee signifies that the written dissertation meets the
requirements of the Dissertation Committee<br /><br />
_____________
                                                         <br />Zhiping Weng, Chair of Committee<br />
The signature of the Dean of the Graduate School of Biomedical Sciences signifies that the student
has met all graduation requirements of the school.<br /><br />
______
                                                         <br />Anthony Carruthers, Ph.D.,<br />
Dean of the Graduate School of Biomedical Sciences<br />
Biochemistry and Molecular Pharmacology<br />
MAY 21st 2014
                                                                                                    
                                                                                                    
</div>
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<a 
 id="x1-2r1"></a>
<a 
 id="Q1-1-1"></a>
<div class="center" 
>
<!--l. 6--><p class="noindent" >
<!--l. 6--><p class="noindent" ><a 
href="University" Web Site URL Here (include http://) >UNIVERSITY NAME (IN BLOCK CAPITALS)</a>
<!--l. 6--><p class="noindent" ><span 
class="phvro8t-x-x-248">Abstract</span>
<!--l. 6--><p class="noindent" ><a 
href="Faculty" Web Site URL Here (include http://) >Faculty Name</a>
<!--l. 6--><p class="noindent" ><a 
href="http://www.umassmed.edu/bmp/index.aspx(include" http://) >BIOCHEMISTRY AND MOLECULAR PHARMACOLOGY</a>
<!--l. 6--><p class="noindent" >Doctor of Philosophy
<!--l. 6--><p class="noindent" ><span 
class="phvb8t-x-x-120">Examination of dynamic long RNAs</span>
<!--l. 6--><p class="noindent" >by Christian Knauf <span 
class="phvrc8t-x-x-120">R<span 
class="small-caps">O</span><span 
class="small-caps">Y</span></span>
</div>
<!--l. 8--><p class="noindent" >The Thesis Abstract is written here (and usually kept to just this page). The page is kept centered vertically
so can expand into the blank space above the title too&#x2026;
                                                                                                    
                                                                                                    
<a 
 id="x1-3r2"></a>
<a 
 id="Q1-1-2"></a>
<div class="center" 
>
<!--l. 33--><p class="noindent" >
<!--l. 33--><p class="noindent" ><span 
class="phvro8t-x-x-248">Acknowledgements</span>
</div>
<!--l. 33--><p class="noindent" >First I would like to acknowledge and thank Laura Geagan and Rebecca Sendak at Genzyme. They were
my supervisors while I was a research associate there, and assisted and encouraged my transition back to
graduate school. Without the confidence they instilled in my abilities as a young scientist in, I doubt I would
have ever signed up for more school.
<!--l. 33--><p class="noindent" >Next I&#8217;d like to thank Melissa. During my 1st year retreat at Wood&#8217;s Hole I first learned that Melissa is a
fantastic communicator of interesting and important science. When she brought out her rope
representing the unspliced pre-mRNA of dystrophin&#8212;a rope that reached to the back of a rather
large auditorium&#8212;and then dramatically held up a No.2 pencil representing the final mRNA
product, both to scale, I knew the that I wanted to do my graduate research in her lab. I have never
once doubted the decision to join Melissa&#8217;s lab, and have learned so much from the broad and
interconnected approach she takes to important scientific questions. Thank you so much for
teaching me to always consider the big picture, go for the answer, and to just ask when I need
help.
<!--l. 33--><p class="noindent" >Soon after joining Melissa&#8217;s lab, and a project going well, it was proposed to me that I be a joint student
between Melissa and Phil. It was not difficult to not jump at the opportunity to be advised by two Howard
Hughes Investigators, and I also haven&#8217;t regretted the decision. Over the past few years, I have been
continually amazed at the depth of Phil&#8217;s knowledge, in scientific and general topics. He is a careful,
meticulous, quantitative, and calculating mentor. While I feel that I clicked &#8216;on the level&#8217; with
Melissa, interacting with Phil forced me to think and act outside my comfort zone, something I
always tell myself is a critical aspect of change and growth. Thank you Phil for everything I&#8217;ve
learned.
<!--l. 33--><p class="noindent" >My committee has also been very supportive throughput my PhD. I hardly believed the ease with
which I passed my qualifying exam, and took it as a big confidence boost. The following years of
TRAC meetings confirmed that I was not thinking way off-base. The one-on-one meetings just
prior to my QE were especially helpful. Thanks to Scot, Job, and especially Zhiping for all the
guidance.
                                                                                                    
                                                                                                    
<ul class="itemize1">
<li class="itemize">Lab Members; Aaron; Alper; Amrit
</li>
<li class="itemize">Eric and Erin
</li>
<li class="itemize">Collaborators
</li>
<li class="itemize">Dave Weaver
</li>
<li class="itemize">Muro
</li>
<li class="itemize">Graveley
</li>
<li class="itemize">Heinrich
</li>
<li class="itemize">Anna
</li>
<li class="itemize">Ogo
</li>
<li class="itemize">Family
</li>
<li class="itemize">Jul Owen
</li></ul>
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-10002"></a>Contents</h2> <div class="tableofcontents">
<span class="chapterToc" ><a 
href="#Q1-1-1">Abstract</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-2">Acknowledgements</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-4">List of Figures</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-6">List of Tables</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-8">Abreviations</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-9">Symbols</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-10">Definitions</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-11">Preface</a></span>
<br /><span class="chapterToc" >1 <a 
href="#x1-40001" id="QQ2-1-12">Introduction</a></span>
<br />&#x00A0;<span class="sectionToc" >1.1 <a 
href="#x1-50001" id="QQ2-1-13">On the importance of gene expression</a></span>
<br />&#x00A0;<span class="sectionToc" >1.2 <a 
href="#x1-60002" id="QQ2-1-15">Nucleic Acid Sequencing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.1 <a 
href="#x1-70001" id="QQ2-1-16">DNA Sequencing History</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.2 <a 
href="#x1-80002" id="QQ2-1-17">History of High-throughput Sequencing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.3 <a 
href="#x1-90003" id="QQ2-1-19">Deep-sequencing RNA methodologies</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.4 <a 
href="#x1-100004" id="QQ2-1-21">Measuing RNA Expression via Sequencing</a></span>
<br />&#x00A0;<span class="sectionToc" >1.3 <a 
href="#x1-110003" id="QQ2-1-22">Nucleic Acid Splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.1 <a 
href="#x1-120001" id="QQ2-1-23">Alternative Splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.2 <a 
href="#x1-130002" id="QQ2-1-24">Deciphering a splicing code</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.3 <a 
href="#x1-140003" id="QQ2-1-25">The Isoform Problem</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.4 <a 
href="#x1-150004" id="QQ2-1-27">Coordination in splicing</a></span>
                                                                                                    
                                                                                                    
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.5 <a 
href="#x1-160005" id="QQ2-1-29">Many isoforms per gene</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.6 <a 
href="#x1-170006" id="QQ2-1-33"><span 
class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></a></span>
<br />&#x00A0;<span class="sectionToc" >1.4 <a 
href="#x1-180004" id="QQ2-1-36">Nucleic Acid Ligation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.4.1 <a 
href="#x1-190001" id="QQ2-1-37">RNA Sequence investigation by ligation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.4.2 <a 
href="#x1-200002" id="QQ2-1-39">T4 RNA Ligase 2 (Rnl2)</a></span>
<br />&#x00A0;<span class="sectionToc" >1.5 <a 
href="#x1-210005" id="QQ2-1-42">Long Nucleic Acid Polymers</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.5.1 <a 
href="#x1-220001" id="QQ2-1-43">Mouse piRNAs are different</a></span>
<br /><span class="chapterToc" >2 <a 
href="#x1-230002" id="QQ2-1-47">SeqZip - Development and Applications</a></span>
<br />&#x00A0;<span class="sectionToc" >2.1 <a 
href="#x1-240001" id="QQ2-1-48">SeqZip Overview</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.1.1 <a 
href="#x1-250001" id="QQ2-1-49">Subsection 1</a></span>
<br />&#x00A0;<span class="sectionToc" >2.2 <a 
href="#x1-260002" id="QQ2-1-51">Multiplex Gene Study</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.2.1 <a 
href="#x1-270001" id="QQ2-1-53">Subsection 1</a></span>
<br />&#x00A0;<span class="sectionToc" >2.3 <a 
href="#x1-280003" id="QQ2-1-55">Determining RNA integrity using SeqZip</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.1 <a 
href="#x1-290001" id="QQ2-1-56">Demonstration of Concept</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.2 <a 
href="#x1-300002" id="QQ2-1-59">Investigating HIV viral genome integrity using SeqZip</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.3 <a 
href="#x1-310003" id="QQ2-1-60">Design of HIV ligamers</a></span>
<br />&#x00A0;<span class="sectionToc" >2.4 <a 
href="#x1-320004" id="QQ2-1-62">Continuity of piRNA precursor transcripts</a></span>
<br /><span class="chapterToc" >3 <a 
href="#x1-330003" id="QQ2-1-69">SeqZip Publication</a></span>
<br />&#x00A0;<span class="sectionToc" >3.1 <a 
href="#x1-340001" id="QQ2-1-70">Abstract</a></span>
<br />&#x00A0;<span class="sectionToc" >3.2 <a 
href="#x1-350002" id="QQ2-1-71">Introduction</a></span>
<br />&#x00A0;<span class="sectionToc" >3.3 <a 
href="#x1-360003" id="QQ2-1-73">Results</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.1 <a 
href="#x1-370001" id="QQ2-1-74">Subsection 2</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.2 <a 
href="#x1-380002" id="QQ2-1-76">Subsection 2</a></span>
<br />&#x00A0;<span class="sectionToc" >3.4 <a 
href="#x1-390004" id="QQ2-1-78">Discussion</a></span>
<br />&#x00A0;<span class="sectionToc" >3.5 <a 
href="#x1-400005" id="QQ2-1-79">Methods</a></span>
<br />&#x00A0;<span class="sectionToc" >3.6 <a 
href="#x1-410006" id="QQ2-1-80">Supplemental Text</a></span>
<br /><span class="chapterToc" >4 <a 
href="#x1-420004" id="QQ2-1-81">MolCel2013</a></span>
                                                                                                    
                                                                                                    
<br />&#x00A0;<span class="sectionToc" >4.1 <a 
href="#x1-430001" id="QQ2-1-82">INTRODUCTION</a></span>
<br />&#x00A0;<span class="sectionToc" >4.2 <a 
href="#x1-440002" id="QQ2-1-83">RESULTS</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.1 <a 
href="#x1-450001" id="QQ2-1-84">Defining piRNA-Producing Transcripts in the Mouse Testis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.2 <a 
href="#x1-460002" id="QQ2-1-87">piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.3 <a 
href="#x1-470003" id="QQ2-1-88">A Transcript-based Set of piRNA Loci</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.4 <a 
href="#x1-480004" id="QQ2-1-89">Three Classes of piRNAs During Post-Natal Spermatogenesis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.5 <a 
href="#x1-490005" id="QQ2-1-94"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.6 <a 
href="#x1-500006" id="QQ2-1-97"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.7 <a 
href="#x1-510007" id="QQ2-1-100"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.8 <a 
href="#x1-520008" id="QQ2-1-103">A-MYB and the Pachytene piRNA Regulatory Circuitry</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.9 <a 
href="#x1-530009" id="QQ2-1-106">Feed-Forward Regulation of piRNA Production is Evolutionarily
Conserved</a></span>
<br />&#x00A0;<span class="sectionToc" >4.3 <a 
href="#x1-540003" id="QQ2-1-109">DISCUSSION</a></span>
<br />&#x00A0;<span class="sectionToc" >4.4 <a 
href="#x1-550004" id="QQ2-1-110">EXPERIMENTAL PROCEDURES</a></span>
<br /><span class="chapterToc" >5 <a 
href="#x1-560005" id="QQ2-1-111">Discussion</a></span>
<br />&#x00A0;<span class="sectionToc" >5.1 <a 
href="#x1-570001" id="QQ2-1-112">Future of Dynamic long RNAs</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.1 <a 
href="#x1-580001" id="QQ2-1-113">The Known Knowns</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.2 <a 
href="#x1-590002" id="QQ2-1-114">The Known Unknowns</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.3 <a 
href="#x1-600003" id="QQ2-1-115">The Unknown Unknowns</a></span>
<br />&#x00A0;<span class="sectionToc" >5.2 <a 
href="#x1-610002" id="QQ2-1-116">Ligation-based investigation of long RNAs</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.1 <a 
href="#x1-620001" id="QQ2-1-117">SeqZip Other Applications</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.2 <a 
href="#x1-630002" id="QQ2-1-118">SeqZip Technical Improvements</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.3 <a 
href="#x1-640003" id="QQ2-1-119">SeqZip Alternatives</a></span>
<br />&#x00A0;<span class="sectionToc" >5.3 <a 
href="#x1-650003" id="QQ2-1-120">Mammalian piRNA-precursors, a special type of long RNA</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.1 <a 
href="#x1-660001" id="QQ2-1-121">What are they doing?</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.2 <a 
href="#x1-670002" id="QQ2-1-122">How are they generated?</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.3 <a 
href="#x1-680003" id="QQ2-1-123">Why should we care?</a></span>
<br /><span class="appendixToc" >A <a 
href="#x1-69000A" id="QQ2-1-124">Appendix - Misc Information</a></span>
                                                                                                    
                                                                                                    
<br />&#x00A0;<span class="sectionToc" >A.1 <a 
href="#x1-700001" id="QQ2-1-125">Buffers</a></span>
<br />&#x00A0;<span class="sectionToc" >A.2 <a 
href="#x1-710002" id="QQ2-1-127">Equations</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.1 <a 
href="#x1-720001" id="QQ2-1-128">f</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.2 <a 
href="#x1-730002" id="QQ2-1-129">b</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.3 <a 
href="#x1-740003" id="QQ2-1-130">Normalize oxidized small RNA libraries size to time-matched
unoxidized library</a></span>
<br />&#x00A0;<span class="sectionToc" >A.3 <a 
href="#x1-750003" id="QQ2-1-131">PCR Programs</a></span>
<br />&#x00A0;<span class="sectionToc" >A.4 <a 
href="#x1-760004" id="QQ2-1-132">Source Code</a></span>
<br /><span class="appendixToc" >B <a 
href="#x1-77000B" id="QQ2-1-133">Appendix B: piRNA precurors are spliced</a></span>
<br />&#x00A0;<span class="sectionToc" >B.1 <a 
href="#x1-780001" id="QQ2-1-134">Evidence for spliced piRNA precusor transcripts</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-140">Bibliography</a></span>
</div><a 
 id="x1-1001r3"></a><a 
 id="Q1-1-4"></a>
                                                                                                    
                                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-20003"></a>List of Figures</h2> <div class="tableofcontents"><span class="lofToc" >1.1&#x00A0;<a 
href="#x1-5001r1">The Solitary and Gregarious forms of <span 
class="phvro8t-x-x-120">Schistocerca</span>
<span 
class="phvro8t-x-x-120">gregaria</span></a></span><br /><span class="lofToc" >1.2&#x00A0;<a 
href="#x1-8001r2">Cost of sequencing the human genome over time</a></span><br /><span class="lofToc" >1.3&#x00A0;<a 
href="#x1-9001r3">Methods
for High-throughput sequencing of RNA</a></span><br /><span class="lofToc" >1.4&#x00A0;<a 
href="#x1-14001r4">HTS read lengths are not
sufficient to maintain AS connectivity</a></span><br /><span class="lofToc" >1.5&#x00A0;<a 
href="#x1-15001r5">Mouse <span 
class="phvro8t-x-x-120">Fn1 </span>contains multipe
sites of Alternative Splicing</a></span><br /><span class="lofToc" >1.6&#x00A0;<a 
href="#x1-16001r6">Number of hg19 Alternative Event
types per gene</a></span><br /><span class="lofToc" >1.7&#x00A0;<a 
href="#x1-16002r7">Number of transcripts per <span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>
gene</a></span><br /><span class="lofToc" >1.8&#x00A0;<a 
href="#x1-17001r8">The architecture of the <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>gene
<span 
class="phvro8t-x-x-120">Dscam1</span></a></span><br /><span class="lofToc" >1.9&#x00A0;<a 
href="#x1-17002r9">Important <span 
class="phvro8t-x-x-120">Dscam1 </span>expression during <span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>
life cycle</a></span><br /><span class="lofToc" >1.10&#x00A0;<a 
href="#x1-19001r10">Mechanism of Rnl2 ATP-dependent ligation</a></span><br /><span class="lofToc" >1.11&#x00A0;<a 
href="#x1-20001r11">Structure
and active site of pre-adenylated of Rnl2</a></span><br /><span class="lofToc" >1.12&#x00A0;<a 
href="#x1-20002r12">Active site of T4 RNA Ligase
2 with highlighted residues</a></span><br /><span class="lofToc" >1.13&#x00A0;<a 
href="#x1-22001r13">Genetic evidence for long, continuous
fly piRNA precursor transcripts</a></span><br /><span class="lofToc" >1.14&#x00A0;<a 
href="#x1-22002r14">Different Classes of mammalian
piRNAs</a></span><br /><span class="lofToc" >1.15&#x00A0;<a 
href="#x1-22003r15">A model for Mammalian piRNA biogenesis</a></span><br /><span class="lofToc" >2.1&#x00A0;<a 
href="#x1-25001r1">Original
SeqZip Diagram</a></span><br /><span class="lofToc" >2.2&#x00A0;<a 
href="#x1-26001r2">10 Gene Set schematic</a></span><br /><span class="lofToc" >2.3&#x00A0;<a 
href="#x1-29001r3">Ligation product tied
to RNA integrity</a></span><br /><span class="lofToc" >2.4&#x00A0;<a 
href="#x1-29002r4">Trans Transcript investigation</a></span><br /><span class="lofToc" >2.5&#x00A0;<a 
href="#x1-31001r5">SeqZip can
examine HIV transcript integrity</a></span><br /><span class="lofToc" >2.6&#x00A0;<a 
href="#x1-32001r6">piRNA precursor locations</a></span><br /><span class="lofToc" >2.7&#x00A0;<a 
href="#x1-32003r7">pRT
Doesn&#8217;t Work for piRNA precursors</a></span><br /><span class="lofToc" >2.8&#x00A0;<a 
href="#x1-32004r8">Dst1 by SeqZip</a></span><br /><span class="lofToc" >2.9&#x00A0;<a 
href="#x1-32005r9">Testes
Specific RNA precursor expression</a></span><br /><span class="lofToc" >2.10&#x00A0;<a 
href="#x1-32006r10">piRNA precursor analysis
via SeqZip</a></span><br /><span class="lofToc" >3.1&#x00A0;<a 
href="#x1-35001r1">SeqZip Diagram</a></span><br /><span class="lofToc" >3.2&#x00A0;<a 
href="#x1-37001r2">SeqZip Diagram</a></span><br /><span class="lofToc" >3.3&#x00A0;<a 
href="#x1-38001r3">SeqZip
Diagram</a></span><br /><span class="lofToc" >4.1&#x00A0;<a 
href="#x1-45001r1">piRNA Precursors are RNA Pol II Transcripts</a></span><br /><span class="lofToc" >4.2&#x00A0;<a 
href="#x1-45002r2">The Major
piRNA-Producing Genes of the Post-Partum Mouse Testis</a></span><br /><span class="lofToc" >4.3&#x00A0;<a 
href="#x1-48001r3">Three
Classes of piRNA-Generating Loci</a></span><br /><span class="lofToc" >4.4&#x00A0;<a 
href="#x1-48002r4">Pre-pachytene piRNAs Persist
in Pachytene Spermatocytes</a></span><br /><span class="lofToc" >4.5&#x00A0;<a 
href="#x1-48003r5">Examples of Pachytene piRNA
Genes</a></span><br /><span class="lofToc" >4.6&#x00A0;<a 
href="#x1-48004r6">Examples of Pre-Pachytene piRNA Genes</a></span><br /><span class="lofToc" >4.7&#x00A0;<a 
href="#x1-49001r7">A-MYB Binds
the Promoters of Pachytene piRNA Genes</a></span><br /><span class="lofToc" >4.8&#x00A0;<a 
href="#x1-49002r8">ChIP-qPCR Confirms
ChIP-seq Data</a></span><br /><span class="lofToc" >4.9&#x00A0;<a 
href="#x1-50001r9">Pachytene piRNAs and Precursors Decrease in <span 
class="phvro8t-x-x-120">A-Myb</span>
Mutant Testes</a></span><br /><span class="lofToc" >4.10&#x00A0;<a 
href="#x1-50002r10">Change in piRNA Expression in <span 
class="phvro8t-x-x-120">Spo11</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, <span 
class="phvro8t-x-x-120">Tdrd6</span>,
and <span 
class="phvro8t-x-x-120">Tdrd9 </span>Mutants</a></span><br /><span class="lofToc" >4.11&#x00A0;<a 
href="#x1-51001r11">Examples of the Effect of the <span 
class="phvro8t-x-x-120">A-Myb </span>Mutation
on piRNA Expression</a></span><br /><span class="lofToc" >4.12&#x00A0;<a 
href="#x1-51002r12">Pachytene piRNA Precursor Abundance in
<span 
class="phvro8t-x-x-120">A-Myb</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, and <span 
class="phvro8t-x-x-120">Trip13 </span>Mutants</a></span><br /><span class="lofToc" >4.13&#x00A0;<a 
href="#x1-52001r13">A-MYB Regulates Expression
of mRNAs Encoding piRNA Pathway Proteins</a></span><br /><span class="lofToc" >4.14&#x00A0;<a 
href="#x1-52002r14"><span 
class="phvro8t-x-x-120">A-Myb </span>mutants, but
Not <span 
class="phvro8t-x-x-120">Miwi </span>Mutants, Change the Expression of RNA Silencing Pathway
Genes</a></span><br /><span class="lofToc" >4.15&#x00A0;<a 
href="#x1-53001r15">Feed-Forward Regulation of piRNA Biogenesis by A-MYB is
Conserved in Rooster</a></span><br /><span class="lofToc" >4.16&#x00A0;<a 
href="#x1-53002r16">Genomic Locations of piRNA Clusters in the
Rooster (Gallus gallus) Testis.</a></span><br /><span class="lofToc" >B.1&#x00A0;<a 
href="#x1-78001r1">RNA-Seq evidence for piRNA precursor
splicing</a></span><br /><span class="lofToc" >B.2&#x00A0;<a 
href="#x1-78002r2">Lack of piRNAs within precusor introns</a></span><br /><span class="lofToc" >B.3&#x00A0;<a 
href="#x1-78003r3">piRNAs map to
Splice Junctions of precursor transcripts</a></span><br /><span class="lofToc" >B.4&#x00A0;<a 
href="#x1-78004r4"><span 
class="phvro8t-x-x-120">A-Myb </span>Mutants produce no
splice-junction mapping piRNAs</a></span><br /><span class="lofToc" >B.5&#x00A0;<a 
href="#x1-78005r5">General features of piRNA transcripts</a></span><br />
                                                                                                    
                                                                                                    
</div>
                                                                                                    
                                                                                                    
<a 
 id="x1-2001r4"></a>
<a 
 id="Q1-1-6"></a>
                                                                                                    
                                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-30004"></a>List of Tables</h2> <div class="tableofcontents"><span class="lotToc" >1.1&#x00A0;<a 
href="#x1-16003r1">Fly genes with &#x003E;2,000 assembled transcripts</a></span><br /><span class="lotToc" >2.1&#x00A0;<a 
href="#x1-27001r1">Genes
with big spans in between</a></span><br /><span class="lotToc" >2.2&#x00A0;<a 
href="#x1-32002r2">Just 9 piRNA genes create &#x003E;50% of
mammalian piRNAs</a></span><br /><span class="lotToc" >A.1&#x00A0;<a 
href="#x1-70001r1">SeqZip Hybridization and Ligation Buffer</a></span><br />
</div>
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<a 
 id="x1-3001r5"></a>
<a 
 id="Q1-1-8"></a>
<div class="center" 
>
<!--l. 7--><p class="noindent" >
<!--l. 7--><p class="noindent" ><span 
class="phvro8t-x-x-144">List of Abreviations</span>
</div>
<div class="table">
<!--l. 8--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-1" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1"></colgroup><colgroup id="TBL-1-2g"><col 
id="TBL-1-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-1"  
class="td11">AS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-2"  
class="td11">Alternative Splicing                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-1"  
class="td11">DNA        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-2"  
class="td11">Deoxyribonucleic acid                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-1"  
class="td11">ssDNA    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-2"  
class="td11">Single-stranded DNA                                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-1"  
class="td11">RNA        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-2"  
class="td11">Ribonucleic acid                                                              </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-1"  
class="td
11">ssRNA    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-2"  
class="td11">Single-stranded RNA                                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-1"  
class="td11">ATP         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-2"  
class="td11">Adenosine triphosphate                                                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-1"  
class="td11">NAD        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-2"  
class="td11">Nicotinamide adenine dinucleotide                                 </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-1"  
class="td
11">ChIP-Seq</td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-2"  
class="td11">Chromatin Immunoprecipitation followed by sequencing</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-1"  
class="td11">HTS        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-2"  
class="td11">High-throughput sequencing (see also NGS)                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-1"  
class="td11">NGS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-2"  
class="td11">Next-generation sequencing                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-1"  
class="td11">nt </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-2"  
class="td11">A nucleotide of either DNA or RNA                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-1"  
class="td11">bp </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-2"  
class="td11">A base pair of DNA                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-1"  
class="td11">SRE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-2"  
class="td11">Splicing Regulatory Element                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-14-1"  
class="td11">IRE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-14-2"  
class="td11">Intron Recognition Element                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-15-1"  
class="td11">CNS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-15-2"  
class="td11">Central Nervous System                                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-16-1"  
class="td11">TSS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-16-2"  
class="td11">{Transcription or Translation} Start Site                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-17-1"  
class="td11">TTS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-17-2"  
class="td11">{Transcription or Translation} Termination Site                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-18-1"  
class="td11">SAGE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-18-2"  
class="td11">Serial Analysis of Gene Expression                                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-19-1"  
class="td11"> </td></tr></table></div>
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
                                                                                                    
                                                                                                    
<a 
 id="x1-3002r6"></a>
<a 
 id="Q1-1-9"></a>
<div class="center" 
>
<!--l. 6--><p class="noindent" >
<!--l. 6--><p class="noindent" ><span 
class="phvro8t-x-x-144">List of Symbols</span>
</div>
<div class="table">
                                                                                                    
                                                                                                    
<!--l. 8--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1"><col 
id="TBL-2-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td11">5<span 
class="cmsy-8">&#x2032;</span> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-2"  
class="td11">The 5 prime end of a DNA or RNA molecule</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-1"  
class="td11">3<span 
class="cmsy-8">&#x2032;</span> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-2"  
class="td11">The 5 prime end of a DNA or RNA molecule</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-1"  
class="td11"><span 
class="cmmi-12">&#x03BC; </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-2"  
class="td11">Micro. A value of 1x10<sup class="textsuperscript"><span 
class="phvr8t-x-x-109">-6</span></sup> standard units        </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-1"  
class="td11"> </td></tr></table>
</div>
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
                                                                                                    
                                                                                                    
<a 
 id="x1-3003r7"></a>
<a 
 id="Q1-1-10"></a>
<div class="center" 
>
<!--l. 6--><p class="noindent" >
<!--l. 6--><p class="noindent" ><span 
class="phvro8t-x-x-144">Definitions</span>
</div>
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">RNA-Seq</span></dt><dd 
class="description">                                                                                         <br 
class="newline" />A technology   wherein   RNA   is   fragmented,   converted   to   DNA,   and   analyzed   on   a
high-throughput sequencing instrument
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">A &#8216;Read&#8217;</span></dt><dd 
class="description">                                                                                          <br 
class="newline" />The sequence  of  nucleotides  produced  from  each  spot  on  a  high-throughput  sequencing
machine
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Insert</span></dt><dd 
class="description">                                                                                             <br 
class="newline" />The RNA molecule captured between two cloning sequences in a high-throughput sequencing
library preparation workflow
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Read length</span> </dt><dd 
class="description">                                                                                                                                       <br 
class="newline" />The number of nucleotides for each given &#8217;read&#8217;
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Read depth</span></dt><dd 
class="description">                                                                                       <br 
class="newline" />The number of reads obtained from each high-throughput sequencing analysis
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Coverage</span></dt><dd 
class="description">                                                                                         <br 
class="newline" />A measure of the number of times each nt of a genome is sequenced. E.g. 100 million reads
of a 10 million nt genome = 10X coverage, assuming uniform distribution of the &#8217;reads&#8217;
                                                                                                    
                                                                                                    
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Paired-end</span></dt><dd 
class="description">                                                                                       <br 
class="newline" />oach1995a When both sides of a DNA insert or template are sequenced, utilizing the original
length of DNA between the reads to facilitate mapping (<a 
href="#XRoach1995">Roach et&#x00A0;al.</a>&#x00A0;[<a 
href="#XRoach1995">1995</a>]).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Scaffold or contig</span> </dt><dd 
class="description">                                                                                                                             <br 
class="newline" />A draft sequence of nucleotides, meant to represent the actual biological sequence as closely
as possible, examples include unassembled fragments of chromosomes or fragments of mRNA
transcripts.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Argonaute</span></dt><dd 
class="description">                                                                                        <br 
class="newline" />Protein(s) belonging to a group containing a Piwi (P-element induced wimpy testes) domain,
that bind  nucleic  acids  and  participate  in  many  target-guided  processes,  including  RNA
Interference, and RNA-indicuded transcript/gene silencing.</dd></dl>
                                                                                                    
                                                                                                    
<div class="center" 
>
<!--l. 9--><p class="noindent" >
<!--l. 9--><p class="noindent" ><span 
class="phvro8t-x-x-172">I would like to dedicate this Doctoral dissertation to my grandfather,</span>
<span 
class="phvro8t-x-x-172">George Knauf. My grandfather passed away on September 23rd,</span>
<span 
class="phvro8t-x-x-172">2011, just one week shy of his 82nd birthday. I find it difficult to</span>
<span 
class="phvro8t-x-x-172">articulate how much I miss him. He spoke carefully and never</span>
<span 
class="phvro8t-x-x-172">without purpose or conviction. While I hear from others that he was</span>
<span 
class="phvro8t-x-x-172">proud of me, he rarely, if ever, betrayed that type of emotion directly.</span>
<span 
class="phvro8t-x-x-172">It is my goal to build as solid a life as he, founded on hard work,</span>
<span 
class="phvro8t-x-x-172">playing the long game, responsibility, and maintaining friendships.</span>
<span 
class="phvro8t-x-x-172">These are just a few of the personality traits that I observed and try</span>
<span 
class="phvro8t-x-x-172">to emulate. The fact that he passed before he could meet our son</span>
<span 
class="phvro8t-x-x-172">Owen is one of my biggest regrets. Of all the possessions he left</span>
<span 
class="phvro8t-x-x-172">behind, it is the memory of our time together that I will cherish the</span>
<span 
class="phvro8t-x-x-172">most. Rest in peace, Grump. I did it.</span></div>
                                                                                                    
                                                                                                    
<a 
 id="x1-3004r8"></a>
<a 
 id="Q1-1-11"></a>
<div class="center" 
>
<!--l. 7--><p class="noindent" >
<!--l. 7--><p class="noindent" ><span 
class="phvro8t-x-x-248">Preface</span>
</div>
<!--l. 8--><p class="noindent" >The work reported in this dissertation has been published in the following articles. Chapter <a 
href="#x1-420004">4<!--tex4ht:ref: Chapter4 --></a> has been
published previously as:<br 
class="newline" />Li, X. Z. Z., Roy, C. K. K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B. W. W., &#x2026; Zamore, P. D. D. (2013). An
Ancient Transcription Factor Initiates the Burst of piRNA Production during Early Meiosis in Mouse Testes.
Molecular Cell, 50(1), 1&#8211;15. doi:10.1016/j.molcel.2013.02.016
<!--l. 12--><p class="noindent" >Some contents of Chapter <a 
href="#x1-330003">3<!--tex4ht:ref: Chapter3 --></a> are included in a currently submitted manuscript.
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
                                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;1</span><br /><a 
 id="x1-40001"></a>Introduction</h2>
<a 
 id="x1-4001r1"></a>
<h3 class="sectionHead"><span class="titlemark">1.1 </span> <a 
 id="x1-50001"></a>On the importance of gene expression</h3>
<!--l. 11--><p class="noindent" >Exodus tells of the liberation of the Israelites from Egyptian slavery. Humble and reluctant Moses, their
divine-apointed leader, attempts to force the Pharaoh Ramses to release the Israelites through infliction of a
series of 10 plagues. Pharaoh is stalwart and stubborn through plauges that turn water to blood,
flood the streets with frogs, lice, and flies. Even as livestock fell dead from disease, people
and animals both are covered in boils, and the land burns in storms of fire, Pharaoh does not
bend.
<!--l. 13--><p class="noindent" >The 8th plague was a swarm of Locusts, described in Exodus 10: 14&#8211;15:
<div class="quote">
<!--l. 19--><p class="noindent" ><sup class="textsuperscript"><span 
class="phvro8t-x-x-109">14</span></sup><span 
class="phvro8t-x-x-120">And the locusts went up over all the land of Egypt, and rested in all the coasts of</span>
<span 
class="phvro8t-x-x-120">Egypt: very grievous were they; before them there were no such locusts as they, neither</span>
<span 
class="phvro8t-x-x-120">after them shall be such.</span><br 
class="newline" /><sup class="textsuperscript"><span 
class="phvro8t-x-x-109">15</span></sup><span 
class="phvro8t-x-x-120">For they covered the face of the whole earth, so that the land was darkened; and they</span>
<span 
class="phvro8t-x-x-120">did eat every herb of the land, and all the fruit of the trees which the hail had left: and</span>
<span 
class="phvro8t-x-x-120">there remained not any green thing in the trees, or in the herbs of the field, through all</span>
<span 
class="phvro8t-x-x-120">the land of Egypt.</span></div>
<!--l. 23--><p class="noindent" >The desolation left by the locust plague was still not enough to persuade Ramses. Nor was three days of
darkness. Only the death of all first-born Egyptians, included Ramses own son, was enough to persuade
Pharaoh to liberate the Israelites.
<!--l. 25--><p class="noindent" >The power of a locust swarm is not just a fanciful biblical story, and is perhaps the most <span 
class="phvro8t-x-x-120">believable </span>of the 10
plagues. In current times, the United Nations&#8217; (UN) Food and Agriculture division maintains a
<a 
href="http://www.fao.org/ag/locusts/en/info/info/news/index.html" >Locust watch website</a> providing weekly updates on potential locust swarms in northern Africa and
the Middle East. Locusts have long been, and continues to be, a powerful and feared force of
Nature.
<!--l. 27--><p class="noindent" >Unlike fire and brimstone from the heavens, locusts are something that can be observed and studied. What
triggers them to swam and cause massive destruction? We know that the desert locust, <span 
class="phvro8t-x-x-120">Schistocerca</span>
<span 
class="phvro8t-x-x-120">gregaria</span>, is the one of 10 others species that swarm and cause massive crop damage. <span 
class="phvro8t-x-x-120">Schistocerca</span>
<span 
class="phvro8t-x-x-120">gregaria </span>are in the insect Order Orthoptera, along with crickets and katydids. Orthoptern members
make sound known as <span 
class="phvro8t-x-x-120">stridulation </span>by vigorously rubbing their wings, making for a noisy cloud of
devastation.
<!--l. 32--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-5001r1"></a>
                                                                                                    
                                                                                                    

<!--l. 34--><p class="noindent" ><img 
src="Figures/Chapter1/DesertLocust.jpeg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.1: </span><span  
class="content">The Solitary and Gregarious forms of <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span><br 
class="newline" /> The  two  phenotypic  forms  of  Schistocerca  gregaria  appear  very  different.  The  Solitary  form  is
green an generally larger, while its gregarious form is more brightly colored, smaller, and capable of
swarming in vast numbers, destroying crops vegetation. Photo from <a 
href="http://www.wikicommons.com" >Wikicommons</a>. </span></div><!--tex4ht:label?: x1-5001r1 -->
                                                                                                    
                                                                                                    
<!--l. 41--><p class="noindent" ></div><hr class="endfigure">
<!--l. 44--><p class="noindent" >A Desert Locust only weighs 0.05&#8211;0.07 ounces, and is less than 2.5 inches long. They can consume their
own body weight in vegetation per day. One <a 
href="http://animaldiversity.ummz.umich.edu/site/accounts/information/Melanoplus_spretus.html" >swarm</a> of the infamous, and now curiously extinct, Rocky
Mountain locust contained 12.2 trillion insects, weighed 27.5 million tons, covered almost 200 square miles
(2/3 the size of ), and could travel 60 miles in a day. A locust swarm is truely a modern biblical
plague.
<!--l. 46--><p class="noindent" >By definition swarms are temporary; the movement, en masse, from one location to another. Where
do 12.2 trillion locusts go when not swarming? Does anyone care if their crops aren&#8217;t under
assault? It seemed no one cared enough, until about 1921, when an important realization was
made.
<!--l. 49--><p class="noindent" >The power and destruction <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>can inflict makes it difficult to believe that they are nothing
more than common grasshoppers. Nothing more than grasshoppers not just by analogy, but by actual
<span 
class="phvro8t-x-x-120">Taxonomy</span>. Desert locusts are actually the <span 
class="phvro8t-x-x-120">gregarious </span>form of <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>(See Figure &#x00A0;<a 
href="#x1-5001r1">1.1<!--tex4ht:ref: fig:Locust --></a>), while
the more familiar and docile looking grasshopper is the <span 
class="phvro8t-x-x-120">solitary form</span>. What makes it possible for such a
dichotomy to exist within the same organism, indeed the same <span 
class="phvro8t-x-x-120">genome</span>, is just now beginning to be
understood.
<!--l. 51--><p class="noindent" ><span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>are <span 
class="phvro8t-x-x-120">polyphenic</span>, meaning that they have multiple (poly) physical forms (phenotypes).
Polyphenism is a general feature among insects. Phenotypes are often extremely different. For example,
pea aphids (<span 
class="phvro8t-x-x-120">Acyrthosiphon pisum</span>), which usually exist in an asexually reproducing, wingless
female form, respond to reduced food supply overcrowding by producing winged offspring.
Winged organisms travel to new sources of food and revert back to the asexually reproducing
form [<a 
href="#XPurandare2014b">Purandare et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPurandare2014b">2014</a>,&#x00A0;<a 
href="#XShingleton2003">Shingleton et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShingleton2003">2003</a>]. In the case of <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span>,
phenotyically the gregarious form is smaller and more brightly colored compared to its solarity cousins.
This transformation can happen in as little as two hours. What is the underlying cause of this
transformation?
<!--l. 53--><p class="noindent" >In 2009, <a 
href="#XAnstey2009">Anstey et&#x00A0;al.</a>&#x00A0;[<a 
href="#XAnstey2009">2009</a>] reported that in just two hours after forced crowding of <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span>,
elevated levels of the neurotransmitter serotonin could be detected in the gangilia (brain). These levels were
strongly correlated gregarious form indicators. [<a 
href="#XAnstey2009">Anstey et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAnstey2009">2009</a>]
<!--l. 58--><p class="noindent" >In an extremely interesting <a 
href="http://aeon.co/magazine/nature-and-cosmos/why-its-time-to-lay-the-selfish-gene-to-rest/" >article</a>, David Dobbs compares the two forms of <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>to that of
Dr Jekyll and Mr. Hyde, the principle characters of Robert Louis Stevenson novella. For Dr Jekyll in fiction,
and for <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>in reality, the power to morph into multiple forms demonsrates the incredible
power and plastic nature of gene expression.
<a 
 id="x1-5002r13"></a>
<h3 class="sectionHead"><span class="titlemark">1.2 </span> <a 
 id="x1-60002"></a>Nucleic Acid Sequencing</h3>
<a 
 id="x1-6001r1"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.1 </span> <a 
 id="x1-70001"></a>DNA Sequencing History</h4>
<!--l. 67--><p class="noindent" >Soon after it was realized that DNA is the source of genetic information in all living organisms [<a 
href="#XWatson1953a">Watson and
Crick</a>,&#x00A0;<a 
href="#XWatson1953a">1953</a>], and the <span 
class="phvro8t-x-x-120">pretty </span>and <span 
class="phvro8t-x-x-120">elegant </span>arrangement of complementary, antiparrallel DNA strands, was
known [<a 
href="#XWatson2012a">Watson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatson2012a">2012</a>], the ability to determine specific arrangements of nucleotide bases (i.e. to
sequence) in a given length of DNA was seen as a critical missing piece of technology. It took 25 years after
the nature of DNA&#8217;s architecture to be able to determine the specific arrangement of nucleotides in the
polymer&#8212;to sequence it. By 1977, two completely different methods developed by Sanger [<a 
href="#XSanger1975a">Sanger and
                                                                                                    
                                                                                                    
Coulson</a>,&#x00A0;<a 
href="#XSanger1975a">1975</a>,&#x00A0;<a 
href="#XSanger1977b">Sanger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSanger1977b">1977</a>] and Maxam-Gilbert [<a 
href="#XMaxam1977a">Maxam and Gilbert</a>,&#x00A0;<a 
href="#XMaxam1977a">1992</a>] were
reported. These sequencing technologies, from then on referred to eponymously as &#8216;Sanger&#8217;
or &#8216;Maxam-Gilbert&#8217; sequencing, were used to determine the specific order of a small piece
of DNA (200&#8211;300 nt). Sanger sequencing soon dominated most sequencing reactions, likely
due to the conceptually more intuitive nature of the technology, and over the past 35 years,
DNA sequences have been slowly cloned, sequenced, analyzed, and dutifully cataloged into
knowledge.
<!--l. 69--><p class="noindent" >During the late 1970&#8217;s and throughout the 1980&#8217;s, DNA sequences were typically communicated in
important publications [<a 
href="#XCordell1980a">Bell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCordell1980a">1980</a>,&#x00A0;<a 
href="#XSanger1978a">Sanger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSanger1978a">1978</a>]. The birth of the Internet in the 1990&#8217;s made
essential publically-funded repositories for sequence information easily available [<a 
href="#XBenson2011a">Benson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBenson2011a">2011</a>].
However, it was the human genome project [<a 
href="#XLander2011a">Lander</a>,&#x00A0;<a 
href="#XLander2011a">2011</a>,&#x00A0;<a 
href="#XVenter2001">Venter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVenter2001">2001</a>], that provided the
important activation energy that brought DNA sequencing from a hard-to-perform, but necessary, analysis,
to an organized large-scale effort of assembling the complete genetic material complex genomes. An often
criticized, but undeniably disrupting force in the human genome project was the competing efforts of the
privately-owned company Celera [<a 
href="#XVenter2008a">Venter</a>,&#x00A0;<a 
href="#XVenter2008a">2007</a>]. Taking a higher-throughput and centralized
approach to determining the sequence of the human genome, Celera fundamentally changed
the landscape of genome assembly. Instead of assigning specific sections of the genome to
be worked out by individual labs, Celera parallelized the effort, by collecting many of the best
&#8220;high-throughput&#8221; Sanger-sequencing devices from Agilent (ABI 3700 DNA Analyzer). Using "shotgun"
approach [<a 
href="#XStaden1979">Staden</a>,&#x00A0;<a 
href="#XStaden1979">1979</a>], sequenced pairwise [<a 
href="#XRoach1995">Roach et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRoach1995">1995</a>], and combined with sequence
scaffolds made available by the publicly-funded project, Celera was able to assemble high-quality
genomic sequences very quickly. Arguably, this was the first deep sequencing effort, and changed
the landscape of molecular and biochemical research, coincident with the beginning of a new
millennium.
<a 
 id="x1-7001r16"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.2 </span> <a 
 id="x1-80002"></a>History of High-throughput Sequencing</h4>
<!--l. 75--><p class="noindent" >Sequencing DNA by Sanger&#8217;s technology remains a valuable and critical tool in every biological scientist&#8217;s
arsenal. However, the technology has a practical throughput limit. Each DNA molecule to be sequenced
must be isolated and clonally amplified, typically using bacteria. Given that the human genome
[<a 
href="#XHattori2005a">Consortium</a>,&#x00A0;<a 
href="#XHattori2005a">2004</a>] comprises &#x003E; 3 billion nt (on just one strand), and that each Sanger reaction will provide
&#x00A0;800nt of quality sequence, we need at least &#x00A0;4 million individual reactions to determine the sequence of
the human genome, assuming that all of our reads are of sufficient quality, length, and do not overlap by
even 1 nt. Even the best practical improvements to work-flows could not bring the Sanger approach to DNA
sequencing in-line with aspirations of analyzing genomes of many different species or individual
organisms.
<!--l. 77--><p class="noindent" >In the early 2000&#8217;s, efforts to change the approach to DNA sequencing, first using MPSS [<a 
href="#XBrenner2000a">Brenner
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrenner2000a">2000</a>], but perhaps more importantly, by Pyrosequencing [<a 
href="#XRonaghi1998a">Ronaghi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRonaghi1998a">1998</a>] and Polony
sequencing [<a 
href="#XShendure2005">Shendure et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShendure2005">2005</a>]. Both of the latter methods utilize emulsion PCR [<a 
href="#XNakano2003a">Nakano et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNakano2003a">2003</a>]
for clonal amplification prior to sequencing, removing the bottleneck of bacterial cloning. In
contrast to Sanger sequencing, where the signal is from fluorescence of the last incorporated
chain-terminating nucleotide, Pyrosequencing visualizes light given off by luciferase as it reacts with ATP
generated from the pyrophosphate (PPi) by-product of nucleotide addition. Pyrosequencing has
been commercialized by 454 technologies. Polony sequencing involves a more complicated
sequencing-by-ligation method, eventually commercialized by Applied Biosystems and branded as SOLiD
sequencing. While both of these technologies provided valuable, high-throughput sequences, neither
has been as successful as the approach commercialized by Solexa, eventually purchased and
                                                                                                    
                                                                                                    
now known as Illumina. Illumina uses a sequencing-by-synthesis approach using fluorescent
nucleotides after clonal amplification of DNA on a slide surface [<a 
href="#XBentley2008">Bentley et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBentley2008">2008</a>]. Since 2006,
iterations of the Illumina platform (eg. GE, GE-II(x), Hi-Seq, Hi-Seq 2500) have demonstrated
a steady and impressive increases in both read depth and length. On February 15th 2012,
Illumina announced on its <a 
href="http://blog.basespace.illumina.com/" >Basespace blog</a>, that they had sequenced a HapMap sample at 40X
coverage, using the HiSeq 2500 platform and paired-end 100 nt reads in a single run. This
announcement demonstrated that in a single analysis attempt (but certainly not the day claimed by
the title), analysis and assembly of a human genome is no longer the monumental endeavor
it once was, and that completely new experimental possibilities are a reality for life science
research.
<!--l. 81--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-8001r2"></a>
                                                                                                    
                                                                                                    
<!--l. 83--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.50x.png" alt="PIC" class="graphics" width="346.29375pt" height="155.58124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.50x.png" src="Figures/Chapter1/Sequencing_costs_over_time.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.2: </span><span  
class="content">Cost of sequencing the human genome over time<br 
class="newline" />The costs of sequencing the human genome has decreased on a log scale over a roughly 10 year
period thanks to major improvements in high-throughput sequencing. Data from Wetterstrand KA.
DNA Sequencing Costs: Data from the NHGRI Genome Sequencing Program (GSP) Available at:
<a 
href="www.genome.gov/sequencingcosts" class="url" ><span 
class="ectt-1200">www.genome.gov/sequencingcosts</span></a>. Accessed 2013-09-03). </span></div><!--tex4ht:label?: x1-8001r2 -->
                                                                                                    
                                                                                                    
<!--l. 92--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-8002r17"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.3 </span> <a 
 id="x1-90003"></a>Deep-sequencing RNA methodologies</h4>
<!--l. 105--><p class="noindent" >The first widely-accepted method for measuring gene expression via sequencing by proxy of cDNA
molecules was Serial Analysis of Gene Expression (SAGE) [<a 
href="#XVelculescu1995a">Velculescu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelculescu1995a">1995</a>]. While the
importance of microarrays in the measurement of gene expression via cannot be overstated [<a 
href="#XMarioni2008">Marioni
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMarioni2008">2008</a>,&#x00A0;<a 
href="#XShendure2008">Shendure and Ji</a>,&#x00A0;<a 
href="#XShendure2008">2008</a>] the technologies limited ability to investigate novel sequences, and
analogue signal, makes their relevance to this section somewhat off-topic. However, SAGE, (similar to the
previously discussed MPSS technique) produces a digital output of gene expression using a cleaver
procedure of cleaving cDNA molecules using restriction endonucleases that leaves a sticky end.
After cleavage, these molecules are ligated and concatenated together to form longer DNA
fragments. Fragments are cloned into a vector, amplified, and Sanger sequenced. Using known
sequences incorporated during concatenation, the number of sequenced &#8217;fragments&#8217; that align to a
given gene is related to the abundance of the original mRNA molecule. While SAGE was a
cleaver molecular trick allowing researches to dip into the 5-log range of expression typically
seen in mRNA expression, it is still limited by read lengths and practical read depth of Sanger
sequencing.
<!--l. 107--><p class="noindent" >Not long after the Solexa/Illumina platform produced read lengths of sufficient length of depth to consider
measuring gene expression were the first RNA-Seq papers published [<a 
href="#XLister2008">Lister et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLister2008">2008</a>,&#x00A0;<a 
href="#XMortazavi2008">Mortazavi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMortazavi2008">2008</a>,&#x00A0;<a 
href="#XNagalakshmi2008">Nagalakshmi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNagalakshmi2008">2008</a>]. These papers gave a powerful glimpse into the future of molecular
biology. Indeed, in the years since, analysis by RNA-Seq has quickly overtaken other forms of
gene expression analysis, as demonstrated by the number of accessions deposited in GEO per
year [<a 
href="#XBarrett2013">Barrett et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarrett2013">2013</a>]. RNA-Seq allows for digital quantification of RNA expression across
physiologically-relevant ranges [<a 
href="#XBlencowe2009">Blencowe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBlencowe2009">2009</a>]. While simultaneously measuring gene expression,
the data can be used for novel sequence discovery, measuring RNA-editing [<a 
href="#XLi2011">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2011">2011</a>],
transcript assembly [<a 
href="#XTrapnell2010">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>]. By modifying the basic protocol or performing additional
biochemical steps, RNA-Seq can be used to investigate many aspects of RNA biology (see
<a 
href="#x1-9001r3">1.3<!--tex4ht:ref: fig:htsMethods --></a>).
<!--l. 110--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-9001r3"></a>
                                                                                                    
                                                                                                    
<!--l. 112--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.51x.png" alt="PIC" class="graphics" width="420.57124pt" height="311.16249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.51x.png" src="Figures/Chapter1/RNA_Sequencing_methodologies.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.3: </span><span  
class="content">Methods for High-throughput sequencing of RNA<br 
class="newline" />In the short years since the first report of RNA-Seq, many variations have been reported. The figure
above provides an incomplete graphical illustration of some of these variations. A more complete list
of *Seq applications is maintained on this <a 
href="http://liorpachter.wordpress.com/seq/" >blog</a>. </span></div><!--tex4ht:label?: x1-9001r3 -->
                                                                                                    
                                                                                                    
<!--l. 119--><p class="noindent" ></div><hr class="endfigure">
<!--l. 122--><p class="noindent" >RNA processing begins the moment the nascent RNA is exposed from the polymerase exit
channel. Many methodologies have been developed that enrich RNA-Seq libraries for RNA
molecules. For example, measurement of nascently transcribed RNA can be performed via GRO-Seq
[<a 
href="#XCore2008a">Core et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCore2008a">2008</a>]. Measuring the extremely complicated process of RNA turnover (referring to
the rates at which RNAs both are produced and degraded) [<a 
href="#XGhosh2010a">Ghosh and Jacobson</a>,&#x00A0;<a 
href="#XGhosh2010a">2010</a>], can
be done using XXX-Seq after incorporation of XX nucleotides or a biochemical handle such
as biotin. RNA::protein interactions can be measured with or without cross-linking the protein
to the RNA, via CLIP or RIP, respectively. Once an RNA has been fully transcribed, known
processing steps such as Cap formation and poly(A) tail formation can be measured using any of the
Cap-Seq/CAGE methodologies [<a 
href="#XShiraki2003a">Shiraki et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShiraki2003a">2003</a>], or PAS-Seq [<a 
href="#XShepard2011">Shepard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShepard2011">2011</a>]. With
appropriate size-selection steps, small RNAs [<a 
href="#XGhildiyal2008">Ghildiyal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>] can also be captured into a
sequencing library. Finally, traditional RNA-Seq, can effectively capture fragments of all of the above
mentioned libraries, even though it is mainly associated with measurement or analysis of traditional
mRNAs.
<!--l. 131--><p class="noindent" >RNA-Seq and its associated flavors are also traditionally associated with measuring gene expression in
tissue culture cells, or RNA extracted from particular tissues. Recently, efforts to measure the RNA
expression occurring in individual cells has gained attention [<a 
href="#XShapiro2013b">Shapiro et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShapiro2013b">2013</a>]. Perhaps the most
interesting concept when thinking about measurement of gene expression in a single cell is the <span class="underline">biological
uncertainty principle</span>, wherein it is possible to either know, or change &#8212;but not both &#8212;the RNA composition
of a single cell. The name borrows from Heisenberg&#8217;s uncertainty principle [<a 
href="#XKennard1927">Kennard</a>,&#x00A0;<a 
href="#XKennard1927">1927</a>] and is often
confused with the more appropriate &#8216;observed effect&#8217; [<a 
href="#XRiley2013">Riley and Steitz</a>,&#x00A0;<a 
href="#XRiley2013">2013</a>]. Leaving that issue aside,
measuring the unique transcriptome of a given cell amoung cells of a common tissue is an exciting
and informative endeavor [<a 
href="#XShalek2013b">Shalek et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShalek2013b">2013</a>,&#x00A0;<a 
href="#XWills2013">Wills et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWills2013">2013</a>]. Compared to DNA, the
diversity of RNA synthesis within living cells is potentially much more complicated [<a 
href="#XShendure2012">Shendure
and Aiden</a>,&#x00A0;<a 
href="#XShendure2012">2012</a>], and the ability to accurately measure RNA dynamics should allow us to
make much more informative observations concerning biology then is currently possible [<a 
href="#XDjebali2012">Djebali
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2012">2012</a>].
<a 
 id="x1-9002r19"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.4 </span> <a 
 id="x1-100004"></a>Measuing RNA Expression via Sequencing</h4>
<!--l. 142--><p class="noindent" >The encode project revealed that most of the genome is transcribed into RNA. This was done in cancerous
cell lines, and while it revealed the potential for transcription, it did not reveal much biology beyond cells in
culture simply perpetuating their existence.
<a 
 id="x1-10001r15"></a>
<h3 class="sectionHead"><span class="titlemark">1.3 </span> <a 
 id="x1-110003"></a>Nucleic Acid Splicing</h3>
<a 
 id="x1-11001r21"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.1 </span> <a 
 id="x1-120001"></a>Alternative Splicing</h4>
<!--l. 166--><p class="noindent" >+ See the chapter 4 of the Spliceosomal pre-mRNA splicing book. 2014
<!--l. 168--><p class="noindent" >+ Yeast + Caenorhabditis elegans is 25342&#8211;348
<!--l. 174--><p class="noindent" >Soon after the discovery of introns, it was reasoned that genes could be arranged in different
combinations, greatly increasing the coding potential of a genome [<a 
href="#XGilbert1978a">Gilbert</a>,&#x00A0;<a 
href="#XGilbert1978a">1978</a>]. The process of
                                                                                                    
                                                                                                    
rearranging genes, now known as alternative splicing (AS), has proven to be an integral phase of
gene expression in most eukaryotes. In just 15 years, the number of genes estimated to be
alternatively spliced has grown considerably. Phillip Sharp, Co-Nobel-prize winner for the discovery of
splicing, stated that: &#8220;Approximately, one of every twenty genes is expressed by alternative
pathways of RNA splicing in different cell types or growth states&#8221; <a 
href="#XSharp2014">Sharp</a>&#x00A0;[<a 
href="#XSharp2014">2014</a>]. Not long after the
assembly of the first human genome, a number of groups combed through Expressed Sequence
Tag (EST) databases to increase that estimate to 35%-59% [<a 
href="#XModrek2002">Modrek and Lee</a>,&#x00A0;<a 
href="#XModrek2002">2002</a>]. Soon
after, analysis using specially designed microarrays resulted in an increased estimate of 74%
[<a 
href="#XJohnson2003">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2003">2003</a>]. However, in late 2008, three groups utilizing high-throughput sequencing
(HTS) of cDNA (referred to as RNA-Seq) demonstrated that between 86% and 95% of human
multi-exon genes are subject to AS [<a 
href="#XPan2008">Pan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPan2008">2008</a>,&#x00A0;<a 
href="#XSultan2008">Sultan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSultan2008">2008</a>,&#x00A0;<a 
href="#XWang2008">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>]. Not
only did they demonstrate that almost all genes are alternatively spliced, they also showed that
AS often occurs in a tissue- and cell type-specific manner. In combination with regulation of
transcription itself, the study of AS is critical to our understanding of the connections between
the comparably static genomic DNA sequence and the highly flexible and adaptive abilities of
organisms.
<a 
 id="x1-12001r23"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.2 </span> <a 
 id="x1-130002"></a>Deciphering a splicing code</h4>
<!--l. 180--><p class="noindent" >A gene is alternatively spliced when, as a result of transcription and processing, there are at least two
unique transcripts produced from one genomic sequence. Beyond counting observed isoforms, one major
area of effort is to decode sequence regulatory elements (SREs) contained in pre-mRNA that define AS site
selection [<a 
href="#XWang2008">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>]. In contrast to the core splicing signals, we have limited knowledge of the
SREs that serve to increase, or decrease, the strength of a particular splice site, often within a sea of other
potential sites. Through a variety of mechanisms, these elements serve as cis-acting sequences and
binding sites for trans-acting factors. Some of the best-studied SREs include Exon Splicing Enhancers and
Silencers (ESEs and ESSs). Members of the Serine-Arginine (SR) protein family typically bind to
ESEs located in an exon, promoting its definition and thereby increasing the probability that
the exon will be included in the final transcript [<a 
href="#XGraveley2000">Graveley</a>,&#x00A0;<a 
href="#XGraveley2000">2000</a>,&#x00A0;<a 
href="#XLong2009">Long and Caceres</a>,&#x00A0;<a 
href="#XLong2009">2009</a>].
Meanwhile, ESSs serve to squelch inclusion, often through binding trans-acting heterogeneous
ribonucleoprotein particles (hnRNPs) [<a 
href="#XMartinez-Contreras2007">Martinez-Contreras et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMartinez-Contreras2007">2007</a>]. Therefore, binding of these
trans-acting factors to their appropriate SREs can either promote or inhibit interactions between the
splicing machinery and the pre-mRNA. The current working hypothesis is that a finely tuned
combination of these binding events determines the final exon content of each isoform [<a 
href="#XHouse2008">House and
Lynch</a>,&#x00A0;<a 
href="#XHouse2008">2008</a>].
<!--l. 182--><p class="noindent" >Sequence motifs that compose the AS code have been teased out [<a 
href="#XBarash2010">Barash et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarash2010">2010</a>,&#x00A0;<a 
href="#XLadd2002">Ladd and
Cooper</a>,&#x00A0;<a 
href="#XLadd2002">2002</a>]. Additionally, assignment of the binding motifs to tissue-specific trans-acting factors has also
progressed [<a 
href="#XJin2003">Jin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJin2003">2003</a>,&#x00A0;<a 
href="#XLicatalosi2008">Licatalosi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLicatalosi2008">2008</a>,&#x00A0;<a 
href="#XUle2005">Ule et&#x00A0;al.</a>,&#x00A0;<a 
href="#XUle2005">2005</a>]. Many of these binding motifs were
identified using combined computational and biochemical approaches. Computational approaches usually
involve searching for a comparative enrichment of sequences near splice sites. Biochemical
approaches typically include gel shift, SELEX, and cross-linking. Many of these approaches
are performed in vitro and disregard the importance of cellular context on binding affinities.
However, with the increasing accessibility of deep sequencing, many groups are extracting
physiologically relevant, high-resolution data from traditional biochemical techniques [<a 
href="#XIngolia2009">Ingolia
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XIngolia2009">2009</a>,&#x00A0;<a 
href="#XIngolia2011">2011</a>]. Deep-sequencing approaches are also being applied to questions involving
mechanisms of AS. In addition to the RNA-Seq experiments, High-Throughput Sequencing [following]
Cross-Linking Immunoprecipitation (HTS-CLIP) has confirmed SRE motif data predicted from
computational and microarray experiments [<a 
href="#XHafner2010">Hafner et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHafner2010">2010</a>,&#x00A0;<a 
href="#XLicatalosi2008">Licatalosi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLicatalosi2008">2008</a>]. Using this
                                                                                                    
                                                                                                    
approach, researchers can now enrich their samples for sequences that bind trans-acting factors of
interest.
<a 
 id="x1-13001r24"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.3 </span> <a 
 id="x1-140003"></a>The Isoform Problem</h4>
<!--l. 188--><p class="noindent" >As with many areas of basic research, the field of AS relies on large-scale (aka &#8211; global, genome-wide,
high-throughput) techniques. Two of the most widely applied technologies employed for large-scale analysis
of gene expression are microarrays and &#8217;2nd generation&#8217; HTS sequencing. Unfortunately, both of these
techniques have fundamental limitations, with the major issues being probe specificity for the former and
read length for the latter.
<!--l. 191--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-14001r4"></a>
                                                                                                    
                                                                                                    
<!--l. 193--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.52x.png" alt="PIC" class="graphics" width="415.55249pt" height="327.22249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.52x.png" src="Figures/Chapter1/SeqLengths_and_Connectivity.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.4: </span><span  
class="content">HTS read lengths are not sufficient to maintain AS connectivity<br 
class="newline" />A) Long RNAs may have multiple sites of AS, separated by 1000&#8217;s of nt; B) Most mRNAs have
&#x00A0;10 exons of &#x00A0;150 nt each. Some have many more (and longer) exons. Read lengths of current
sequencing technologies do not maintain connectivity between distant sites. </span></div><!--tex4ht:label?: x1-14001r4 -->
                                                                                                    
                                                                                                    
<!--l. 200--><p class="noindent" ></div><hr class="endfigure">
<!--l. 203--><p class="noindent" >Microarrays rely on hybridization of a target sequence to a known probe averaging 25 to 100 nt in length
[<a 
href="#XSouthern2001">Southern</a>,&#x00A0;<a 
href="#XSouthern2001">2001</a>]. Therefore, microarrays indicate only the presence of short sequences in the target
sample and do not provide adequate linkage information of these sequences. A hypothetical scenario can
be used to describe it another way. Say we are investigating a transcript known to display two different
regions of AS (See Figure <a 
href="#x1-14001r4">1.4<!--tex4ht:ref: fig:NoConnectivityInHTSMethods --></a>). Probes targeting these two regions demonstrated an increase in signal for
both AS events. Unfortunately, we could not determine if we observed an increase in unique
transcripts, each containing only one region of AS, or an increase in production of a single transcript
containing both regions [<a 
href="#XCalarco2007">Calarco et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007">2007a</a>]. This binary analysis is the heart of the "connectivity
problem." Microarrays have proven extremely informative and will likely continue to do so in
more targeted applications. However, this issue, combined with concerns of cross-hybridization,
reproducibility, and a comparably small dynamic range, will likely hasten microarray displacement by
RNA-Seq as the preferred method for comprehensive analysis of gene expression [<a 
href="#XShendure2008">Shendure and
Ji</a>,&#x00A0;<a 
href="#XShendure2008">2008</a>].
<!--l. 205--><p class="noindent" >Many researchers are turning toward 2nd generation HTS methodologies for comprehensive transcriptome
analysis. This sequencing approach has significance advantages over microarrays. Specifically, it allows de
novo identification of isoforms, over a larger dynamic range, in a quantitative fashion [<a 
href="#XMortazavi2008">Mortazavi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMortazavi2008">2008</a>]. Additionally, techniques exist to enrich samples for low-abundance isoforms, making the
complete cataloging of AS events a possibility [<a 
href="#XDjebali2008">Djebali et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2008">2008</a>,&#x00A0;<a 
href="#XSalehi-Ashtiani2008">Salehi-Ashtiani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSalehi-Ashtiani2008">2008</a>].
Unfortunately, the current read-length abilities (depicted in Figure 1-1,B) of all sequencing platforms do not
solve the connectivity problem. Excluding single-molecule HTS read lengths of sufficient length
[<a 
href="#XShendure2004">Shendure et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShendure2004">2004</a>], other approaches proposed to solve the connectivity problem include
traditional cloning and sequencing or hybridization of query oligos to single-molecule transcripts
[<a 
href="#XCalarco2007">Calarco et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007">2007a</a>,&#x00A0;<a 
href="#XEmerick2007">Emerick et&#x00A0;al.</a>,&#x00A0;<a 
href="#XEmerick2007">2007</a>,&#x00A0;<a 
href="#XZhu2003">Zhu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhu2003">2003</a>]. While these approaches can
determine exon sequence connectivity, they scale poorly and are not feasible for large-scale
applications. Clearly, AS is an essential regulatory mechanism involved in the control of human gene
expression. Its combinatorial nature could potentially answer many questions, such as a physical
explanation of what separates us from our closest evolutionary ancestor, the chimpanzee [<a 
href="#XCalarco2007a">Calarco
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007a">2007b</a>]. Additionally, the influence of AS on disease and cancer is slowly coming to light
[<a 
href="#XTazi2009">Tazi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTazi2009">2009</a>]. Unfortunately, because of the limitations of methods currently used for the
large-scale analysis of isoform expression we fail to obtain the complete picture of AS. One specific
missing element of that picture is the prevalence of coordination between different regions of AS
separated by large spans of sequence. An efficient, large-scale, single-molecule technique that
maintains isoform sequence connectivity is required to complete the complicated picture of
AS.
<a 
 id="x1-14002r25"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.4 </span> <a 
 id="x1-150004"></a>Coordination in splicing</h4>
<!--l. 218--><p class="noindent" >Identification of proximally acting SREs is progressing at a rapid pace. New and traditional biochemical
methods, coupled with HTS, will undoubtedly fuel this progress. Unfortunately, a critical component of AS
regulation currently neglected by the field is that of SREs acting across a considerable distance (&#x003E;800 nt).
One observation that may lead to the identification of long-range SREs is intramolecular coordination
between distal splicing decisions. Figure <a 
href="#x1-14001r4">1.4<!--tex4ht:ref: fig:NoConnectivityInHTSMethods --></a> a model transcript that may exhibit coordinated
distal regions of AS. In this model, the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; region of AS contains a cassette exon, which may or
may not be included. This region is separated from the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;region of AS by many thousands of
nucleotides. Does the decision to include the cassette exon have an effect on which of the mutually
                                                                                                    
                                                                                                    
exclusive exons is included? This type of AS regulation may represent a general and pervasive
phenomenon.
<!--l. 221--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-15001r5"></a>
                                                                                                    
                                                                                                    
<!--l. 223--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.53x.png" alt="PIC" class="graphics" width="390.45874pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.53x.png" src="Figures/Chapter1/Fibronectin.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.5: </span><span  
class="content">Mouse <span 
class="phvro8t-x-x-120">Fn1 </span>contains multiple sites of Alternative Splicing<br 
class="newline" />A) There are three highly-studied regions of AS in mouse <span 
class="phvro8t-x-x-120">Fn1</span>: The cassette exons EDB and EDA,
and the Variable(V)-region (AKA the IIICS) exon, which displays multiple 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; splice sites. Each of
these sites is separated by multiple constitutive exons.; B) Considering simplistic splicing of these
three exons, there are 12 different isoforms of mouse <span 
class="phvro8t-x-x-120">Fn1</span>. </span></div><!--tex4ht:label?: x1-15001r5 -->
                                                                                                    
                                                                                                    
<!--l. 230--><p class="noindent" ></div><hr class="endfigure">
<!--l. 233--><p class="noindent" >There is precedence in the literature for genes known to display coordinated regions of AS. One of the
clearest examples is mouse Fibronectin <span 
class="phvro8t-x-x-120">Fn1 </span>(Figure <a 
href="#x1-15001r5">1.5<!--tex4ht:ref: fig:mouseFn1 --></a>) [<a 
href="#XSchwarzbauer1983">Schwarzbauer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchwarzbauer1983">1983</a>,&#x00A0;<a 
href="#XWhite2011a">White and
Muro</a>,&#x00A0;<a 
href="#XWhite2011a">2011</a>]. In this gene, inclusion of the alternatively spliced Extra Domain A (EDI or EDA) region
promotes splicing from one of three alternative 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; Splice Site (3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; SS) in the type III homology connecting
segment (IIICS) region, resulting in more frequent production of shorter transcripts [<a 
href="#XFededa2005">Fededa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFededa2005">2005</a>].
This effect occurs over six constitutively expressed exons and 800 nt of sequence (5400 nt if introns are
considered). [<a 
href="#XFededa2005">Fededa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFededa2005">2005</a>] also analyzed EST databases, concluding that approximately 25% of
human genes contain multiple regions of AS. How many of these regions could show a coordinated effect,
similar to that observed in Fibronectin? Providing some insight into this question, Fagnani et
al used microarrays designed to report on inclusion levels of cassette exons in mammalian
central nervous system tissues [<a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFagnani2007">2007</a>]. The results produced a set of 38 pairs of
exons mapping to the same gene that showed a coordinated increase or decrease of inclusion
levels.
<!--l. 235--><p class="noindent" >There have been a few studies that investigate forms of splicing coordination between adjacent exons
present in mRNA. The vertebrate genes 4.1B and 4.1R, members of the protein 4.1 family encoding for
cytoskeletal adaptor proteins, both undergo splicing of upstream 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; first exons to distal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
second exons, skipping a stronger proximal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; second exon [<a 
href="#XParra2012">Parra et&#x00A0;al.</a>,&#x00A0;<a 
href="#XParra2012">2012</a>,&#x00A0;<a 
href="#XParra2008">2008</a>]. This is
accomplished through &#8217;intrasplicing&#8217; involving an intronic sequence element (the &#8217;intraexon&#8217;) only
present when transcription begins at the upstream 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; exon, allowing the exon to ligate do the
weaker distal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; second exon via an intermediate splicing event. Importantly, this type of splicing
would be similar, but different from recursive splicing seen in drosophila [<span 
class="phvb8t-x-x-120">?</span>]. Another example of
the importance of intron sequence elements on AS is observed in the equine <span 
class="cmmi-12">&#x03B2;</span>-casin gene,
where the authors propose a model involving an intronic splicing enhancer bound to the exit
channel of the elongating polymerase, promoting inclusion of downstream cassette exons [<a 
href="#XLenasi2006">Lenasi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLenasi2006">2006</a>]. Taking a more genome-wide approach Peng et al. examined human and mouse
EST data looking for correlations between adjacent AS cassette exons [<a 
href="#XPeng2008">Peng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPeng2008">2008</a>].
The authors note that positively correlated pairs of adjacent cassette exons typically resemble
constitutive exons in splice strength, whereas negatively, or weakly correlated pairs are likely to be
newly emerging exons, whose strength of splicing has not evolved enough to be constitutively
included.
<!--l. 237--><p class="noindent" >The last, most current, and thorough study of intra-gene splicing coordination involves the <span 
class="phvro8t-x-x-120">Caenorhabditis</span>
<span 
class="phvro8t-x-x-120">elegans </span>gene <span 
class="phvro8t-x-x-120">slo1 </span>[<a 
href="#XGlauser2011">Glauser et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGlauser2011">2011</a>,&#x00A0;<a 
href="#XJohnson2011">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2011">2011</a>]. <span 
class="phvro8t-x-x-120">slo1 </span>is the <span 
class="phvro8t-x-x-120">C.elegans </span>orthologue of the
human BK channel gene <span 
class="phvro8t-x-x-120">Kcnma</span>, also known to undergo extensive alternative splicing [<a 
href="#XNilsen2010">Nilsen and
Graveley</a>,&#x00A0;<a 
href="#XNilsen2010">2010</a>] via 13 cassette exons, potentially coding for over 1,000 different isoforms. <span 
class="phvro8t-x-x-120">Kcnma </span>is
highly developmentally, spatially, and tissue regulated. It is involved in a diverse range of cellular
processes, including hearing, circadian rhythms, urinary function, and vasoregulation [<a 
href="#XFodor2009a">Fodor
and Aldrich</a>,&#x00A0;<a 
href="#XFodor2009a">2009</a>]. While the gene is highly conserved, as organism complexity grows, so
does the apparent transcriptional diversity of this gene. In worms, <span 
class="phvro8t-x-x-120">slo1 </span>can produce up to 12
different isoforms. Glauser et al. used QPCR to demonstrate individual, AS region inclusion
frequencies do not correspond to complete isoform frequencies, when measured via a TaqMan
probe approach. They go on to describe a interdependent-splicing model that best fits the data,
and support interdependence via mutations at one sight altering splicing at both upstream and
downstream sites of AS, separated by atleast one other splicing event. After measuring the biophysical
properties of the isoforms [<a 
href="#XJohnson2011">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2011">2011</a>], they conclude that coordinated AS is critical for
proper BK channel function in vivo. It is interesting to note that this study also identified an
intronic sequence element that displayed some type of coordinated, or co-regulated effect on
                                                                                                    
                                                                                                    
AS.
<a 
 id="x1-15002r27"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.5 </span> <a 
 id="x1-160005"></a>Many isoforms per gene</h4>
<!--l. 246--><p class="noindent" >It is easy to think of AS as a binary process. Isoform A or B is produced based upon picking either exon A or
B. What quickly becomes evident, and is far too real for researchers building transcriptome assembly
algorithms, is that the combinatorial nature of AS makes it both a power means of generating isoform
diversity and a difficult problem to study [<a 
href="#XTrapnell2012a">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2012a">2012</a>].
<!--l. 249--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-16001r6"></a>
                                                                                                    
                                                                                                    
<!--l. 251--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.54x.png" alt="PIC" class="graphics" width="393.46999pt" height="197.73874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.54x.png" src="Figures/Chapter1/ASEventTypesPlot.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.6: </span><span  
class="content">Number of hg19 Alternative Event types per gene<br 
class="newline" />Alternative Event types per gene. RefSeq on 2014-03-24 </span></div><!--tex4ht:label?: x1-16001r6 -->
                                                                                                    
                                                                                                    
<!--l. 258--><p class="noindent" ></div><hr class="endfigure">
<!--l. 261--><p class="noindent" >One of the most recent attempts to investigate the breath of combinations produced by AS is the already
mentioned ENCODE project [<a 
href="#XDjebali2012">Djebali et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2012">2012</a>]. ENCODE performed extremely in depth analysis of 15
cell lines, and find that each isoform produces &#x00A0;10 transcripts per gene, with a broad distribution in terms of
isoforms expressed per sample.
<!--l. 263--><p class="noindent" >The ENCODE project clearly demonstrated that most human genes can under AS in many more ways than
previously appreciated. Most genes could be considered as undergoing &#8217;complex&#8217; AS, with numerous forms
of AS ( See figure <a 
href="#x1-16001r6">1.6<!--tex4ht:ref: fig:asEventsBarChart --></a>). Despite the prevalence of complex alternative spliced genes, just a few genes are
used as examples to illustrate numerical possibilities and biological significance. For example the human
immune system relies heavily on AS to be plastic toward antigen recognition and response [<a 
href="#XLynch2004">Lynch</a>,&#x00A0;<a 
href="#XLynch2004">2004</a>].
Modulation of extracellular signaling proteins such as <span 
class="phvro8t-x-x-120">CD44 </span>and cellular adhesion protein <span 
class="phvro8t-x-x-120">CD45 </span>have
been well-studied [<a 
href="#XPonta2003b">Ponta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPonta2003b">2003</a>,&#x00A0;<a 
href="#XZikherman2008">Zikherman and Weiss</a>,&#x00A0;<a 
href="#XZikherman2008">2008</a>]. Alternative splicing in
humans, however, does not seem to produce the number of unique possible combinations as AS
of genes in simpler organisms, such as fruit flies, perhaps due to specialization of genes, or
different genes that work in combination or complexes, as oppose to utilizing unique gene isoforms
[<a 
href="#XPark2007">Park and Graveley</a>,&#x00A0;<a 
href="#XPark2007">2007</a>]. For example, the fruit fly gene muscle myosin heavy chain (<span 
class="phvro8t-x-x-120">Mhc</span>)
can produce up to 480 different isoforms through AS of 17 different cassette exons [<a 
href="#XBernstein1983a">Bernstein
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBernstein1983a">1983</a>].
<!--l. 266--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-16002r7"></a>
                                                                                                    
                                                                                                    
<!--l. 268--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.55x.png" alt="PIC" class="graphics" width="413.545pt" height="291.0875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.55x.png" src="Figures/Chapter1/NumberOFTranscriptsPerFlyGene.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.7: </span><span  
class="content">Number of transcripts per <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>gene<br 
class="newline" />Data from [<a 
href="#XBrown2014">Brown et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>], Supplemental Table 3. Number of transcript per bin, with bin sizes
&#8217;closed&#8217; on the upper part of range. </span></div><!--tex4ht:label?: x1-16002r7 -->
                                                                                                    
                                                                                                    
<!--l. 275--><p class="noindent" ></div><hr class="endfigure">
<div class="table">
                                                                                                    
                                                                                                    
<!--l. 2--><p class="noindent" ><a 
 id="x1-16003r1"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-3" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-3-1g"><col 
id="TBL-3-1"></colgroup><colgroup id="TBL-3-2g"><col 
id="TBL-3-2"></colgroup><colgroup id="TBL-3-3g"><col 
id="TBL-3-3"></colgroup><colgroup id="TBL-3-4g"><col 
id="TBL-3-4"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-1-1"  
class="td11"><span 
class="phvb8t-x-x-120">Gene Name</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-2"  
class="td11"><span 
class="phvb8t-x-x-120"># Introns</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-3"  
class="td11"><span 
class="phvb8t-x-x-120"># Transcripts</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-4"  
class="td11"><span 
class="phvb8t-x-x-120"># Proteins</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-2-1"  
class="td11">Mhc              </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-2"  
class="td11">      60</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-3"  
class="td11">        2040</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-4"  
class="td11">      511</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-3-1"  
class="td
11">slo                </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-2"  
class="td11">      49</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-3"  
class="td11">        2070</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-4"  
class="td11">      279</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-4-1"  
class="td
11">ps                 </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-2"  
class="td11">      30</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-3"  
class="td11">        2099</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-4"  
class="td11">       27</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-5-1"  
class="td11">rg                  </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-2"  
class="td11">      45</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-3"  
class="td11">        2178</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-4"  
class="td11">       23</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-6-1"  
class="td11">shot              </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-2"  
class="td11">      60</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-3"  
class="td11">        2478</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-4"  
class="td11">      886</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-7-1"  
class="td
11">scrib             </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-2"  
class="td11">      53</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-3"  
class="td11">        2555</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-4"  
class="td11">      259</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-8-1"  
class="td
11">heph             </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-2"  
class="td11">      75</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-3"  
class="td11">        2876</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-4"  
class="td11">       52</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-9-1"  
class="td11">CG42748     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-2"  
class="td11">      26</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-3"  
class="td11">        2876</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-4"  
class="td11">       51</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-10-1"  
class="td
11">rdgA             </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-2"  
class="td11">      35</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-3"  
class="td11">        3003</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-4"  
class="td11">       89</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-11-1"  
class="td11">Mbs              </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-2"  
class="td11">      39</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-3"  
class="td11">        3080</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-4"  
class="td11">      119</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-12-1"  
class="td
11">CaMKI          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-2"  
class="td11">      41</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-3"  
class="td11">        3992</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-4"  
class="td11">        7</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-13-1"  
class="td11">par-1            </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-2"  
class="td11">      48</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-3"  
class="td11">        4410</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-4"  
class="td11">      142</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-14-1"  
class="td11">GluClalpha   </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-2"  
class="td11">      27</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-3"  
class="td11">        4945</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-4"  
class="td11">      188</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-15-1"  
class="td11">Sap47          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-2"  
class="td11">      24</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-3"  
class="td11">        5011</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-4"  
class="td11">       49</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-16-1"  
class="td11">Patronin        </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-2"  
class="td11">      50</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-3"  
class="td11">        5615</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-4"  
class="td11">      590</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-17-1"  
class="td11">CG17838     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-2"  
class="td11">      37</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-3"  
class="td11">        8333</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-4"  
class="td11">      147</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-18-1"  
class="td11">unc-13          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-2"  
class="td11">      52</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-3"  
class="td11">        8391</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-4"  
class="td11">      279</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-19-1"  
class="td11">A2bp1          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-2"  
class="td11">      29</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-3"  
class="td11">        9055</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-4"  
class="td11">       58</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-20-1"  
class="td11">Imp               </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-2"  
class="td11">      33</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-3"  
class="td11">        9131</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-4"  
class="td11">       12</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-21-1"  
class="td11">pan               </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-2"  
class="td11">      38</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-3"  
class="td11">        9432</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-4"  
class="td11">       72</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-22-1"  
class="td11">Sh                </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-2"  
class="td11">      40</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-3"  
class="td11">       15995</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-4"  
class="td11">       66</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-23-1"  
class="td11">gish              </td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-2"  
class="td11">      48</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-3"  
class="td11">       18972</td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-4"  
class="td11">      142</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-24-1"  
class="td11">           </td></tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;1.1: </span><span  
class="content">Fly genes with &#x003E;2,000 assembled transcripts according to [<a 
href="#XBrown2014">Brown et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>].</span></div><!--tex4ht:label?: x1-16003r1 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
<a 
 id="x1-16004r29"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.6 </span> <a 
 id="x1-170006"></a><span 
class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></h4>
<!--l. 286--><p class="noindent" >Unquestionably, the gene most frequently used to demonstrate the combinatorial power of AS is
fly <span 
class="phvro8t-x-x-120">Dscam1</span>. The &#8217;architecture&#8217; of <span 
class="phvro8t-x-x-120">Dscam1 </span>is rather unique among other organisms, but as we
saw in Section <a 
href="#x1-160005">1.3.5<!--tex4ht:ref: sec:IsoformsPerGene --></a>, contain some genes that generate tremendous isoform diversity from a
single genetic locus [<a 
href="#XBrown2014">Brown et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>]. The basic structure of <span 
class="phvro8t-x-x-120">Dscam1 </span>is shown in Figure
<a 
href="#x1-17001r8">1.8<!--tex4ht:ref: fig:DscamArch --></a>.
<!--l. 289--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-17001r8"></a>
                                                                                                    
                                                                                                    
<!--l. 291--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.56x.png" alt="PIC" class="graphics" width="391.4625pt" height="73.27374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.56x.png" src="Figures/Chapter1/DscamArch.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.8: </span><span  
class="content">The architecture of the <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>gene <span 
class="phvro8t-x-x-120">Dscam1</span><br 
class="newline" /><span 
class="phvro8t-x-x-120">Dscam1 </span>has  three  <span 
class="phvro8t-x-x-120">clusters  </span>or  &#8217;banks&#8217;  of  alternative  cassette  exons  that  are  splicing  out  in  a
mutually-exclusive manner. The first bank, &#8217;Exon 4&#8217;, contains 12 different variants, of which one one
is ever included into the mRNA. Similarly, banks 6 &amp; 9 each contain 48 and 33 different variants,
respectively. These three banks code for extracellular IgG domains, while the final region of AS, exon
17, encodes two different trans-membrane domains, again of which only one is included in the final
mRNA. </span></div><!--tex4ht:label?: x1-17001r8 -->
                                                                                                    
                                                                                                    
<!--l. 298--><p class="noindent" ></div><hr class="endfigure">
<!--l. 301--><p class="noindent" >Human <span 
class="phvro8t-x-x-120">Dscam</span>, for which <span 
class="phvro8t-x-x-120">Dscam1 </span>was named, was identified while looking for genes on chromosome 21,
specifically band 21q22, where extra copies expressed in Down syndrome patients, a trisomy 21 disorder,
may be causative for disease [<a 
href="#XYamakawa1998b">Yamakawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYamakawa1998b">1998</a>]. <span 
class="phvro8t-x-x-120">Dscam </span>(Down Syndrome Cellular Adhesion
Molecule) was named according to this association, and its membership in the immunoglobulin super family
of proteins with extracellular adhesion functions. Human <span 
class="phvro8t-x-x-120">Dscam </span>does undergo some alternatively splicing
and broadly expressed in the developing nervous system. However, it does not contain the architecture of
cassette exon banks as <span 
class="phvro8t-x-x-120">Dscam1</span>.
<!--l. 303--><p class="noindent" >Complex AS of<span 
class="phvro8t-x-x-120">Dscam1 </span>was first noticed by the Zipursky lab in 2000 [<a 
href="#XSchmucker2000">Schmucker et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchmucker2000">2000</a>]. While
looking for proteins associated with <span 
class="phvro8t-x-x-120">dock </span>and <span 
class="phvro8t-x-x-120">pak</span>, two proteins important for neuronal growth cone
guidance, they biochemically co-purified DSCAM1. Sequencing of <span 
class="phvro8t-x-x-120">Dscam1 </span>clones revealed that virtually all
contained different combinations of exons 4,6, and 9. In fact, these three exons are chosen from
three clusters of mutually-exclusive cassette exons, containing 12, 48, and 33 different options
each(see Figure <a 
href="#x1-17001r8">1.8<!--tex4ht:ref: fig:DscamArch --></a>). The initial report kicked off an exciting period of research into <span 
class="phvro8t-x-x-120">Dscam1</span>
structure and function. The functional significance of <span 
class="phvro8t-x-x-120">Dscam1 </span>AS was a major goal of multiple
labs.
<!--l. 305--><p class="noindent" >Before the highlights of <span 
class="phvro8t-x-x-120">Dscam1 </span>research are reviewed, it is illustrative to discuss some basic
<span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>anatomy. There are 4 main regions where <span 
class="phvro8t-x-x-120">Dscam1 </span>expression has been
highly-studied.
<ul class="itemize1">
<li class="itemize">Hemocyte cells of the immune system
</li>
<li class="itemize">Larva Class IV da Neurons
</li>
<li class="itemize">Pupal Mushroom-body neurons in the developing brain
</li>
<li class="itemize">Tetrad synapses of the eye</li></ul>
<!--l. 314--><p class="noindent" ><span 
class="phvro8t-x-x-120">Dscam1 </span>involvement/expression in three of these four biologically important roles is shown in Figure <a 
href="#x1-17002r9">1.9<!--tex4ht:ref: fig:DscamAnatomy --></a>.
First, <span 
class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells of the immune system is important for recognition of foreign
antigens [<a 
href="#XWatson2005">Watson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatson2005">2005</a>]. During larval development, <span 
class="phvro8t-x-x-120">Dscam1 </span>is expressed in the da neurons of the
larval body wall, these neurons create a uniform sensory feed, allowing the larva to respond to mechanical
stimulus. In the developing brain, <span 
class="phvro8t-x-x-120">Dscam1 </span>is expressed in both axonal projections of neurons
extend from their Kenyon cell bodies and bifurcate into two different mushroom body lobs [<a 
href="#XZhan2004">Zhan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhan2004">2004</a>].
<!--l. 317--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-17002r9"></a>
                                                                                                    
                                                                                                    
<!--l. 319--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.57x.png" alt="PIC" class="graphics" width="397.48499pt" height="516.93124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.57x.png" src="Figures/Chapter1/DscamAnatomy.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.9: </span><span  
class="content">Important <span 
class="phvro8t-x-x-120">Dscam1 </span>expression during <span 
class="phvro8t-x-x-120">Drosophila melanogaster life </span>cycle<br 
class="newline" /><span 
class="phvro8t-x-x-120">Dscam1 </span>has been high-studied in four different regions/cell types. 1) Hemocytes of the immune
system, where DSCAM1 is involved in antigen recognition; 2) In Class IV da neurons, which sense
mechanical stimulation of the larval body wall; 3) In mushroom body neurons of the pupal developing
brain; and 4) (not shown) in Tetrad neurons of the eyes. </span></div><!--tex4ht:label?: x1-17002r9 -->
                                                                                                    
                                                                                                    
<!--l. 326--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-17003r22"></a>
<h3 class="sectionHead"><span class="titlemark">1.4 </span> <a 
 id="x1-180004"></a>Nucleic Acid Ligation</h3>
<a 
 id="x1-18001r33"></a>
<h4 class="subsectionHead"><span class="titlemark">1.4.1 </span> <a 
 id="x1-190001"></a>RNA Sequence investigation by ligation</h4>
<!--l. 348--><p class="noindent" >In the late 1960&#8217;s and early 1970&#8217;s, the Lehman and Richardson labs characterized two workhorse-enzymes
of modern molecular biology. Robert Lehman and colleagues, working at Standford Medical School, first
described the activity of <span 
class="phvro8t-x-x-120">polynucleotide-joining enzyme </span>from <span 
class="phvro8t-x-x-120">Escherichia coli </span>(now known as <span 
class="phvro8t-x-x-120">E. Coli </span>DNA
Ligase) [<a 
href="#XOlivera1967b">Olivera and Lehman</a>,&#x00A0;<a 
href="#XOlivera1967b">1967</a>]. Work on this enzyme paralleled that from the Richardson lab at
Harvard Medical School, where they focused on <span 
class="phvro8t-x-x-120">polynucleotide ligase </span>from <span 
class="phvro8t-x-x-120">Escherichia coli </span>infected with T4
bacteriophage (now known as T4 DNA ligase) [<a 
href="#XWeiss1967a">Weiss and Richardson</a>,&#x00A0;<a 
href="#XWeiss1967a">1967</a>]. It became clear
that while these two enzyme&#8217;s shared a common mechanism&#8212;later elucidated by [<a 
href="#XModrich1973a">Modrich
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModrich1973a">1973</a>]&#8212;they had important differences. First, T4 DNA ligase required ATP as a cofactor,
which <span 
class="phvro8t-x-x-120">E. Coli </span>DNA Ligase did not (though it was later discovered that DNA ligase required
NAD as a cofactor). Second, only T4 DNA ligase could catalyze ligation of blunt-ended DNA
[<a 
href="#XTabor1987a">Tabor</a>,&#x00A0;<a 
href="#XTabor1987a">1987</a>].
<!--l. 351--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-19001r10"></a>
                                                                                                    
                                                                                                    
<!--l. 353--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.58x.png" alt="PIC" class="graphics" width="374.39874pt" height="518.93874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.58x.png" src="Figures/Chapter1/LigationMechanism.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.10: </span><span  
class="content">Mechanism of ATP-dependent ligation<br 
class="newline" />Adapted from [<a 
href="#XNandakumar2006">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>] and specifically for that of T4 RNA ligase 2. </span></div><!--tex4ht:label?: x1-19001r10 -->
                                                                                                    
                                                                                                    
<!--l. 360--><p class="noindent" ></div><hr class="endfigure">
<!--l. 363--><p class="noindent" >The general mechanism of ligation, shown in Figure <a 
href="#x1-19001r10">1.10<!--tex4ht:ref: fig:Ligation Mechanism --></a>, involves three steps: Step 1 (A) involves the
<span 
class="cmmi-12">&#x03F5;</span>-amino group from the active site lysine performs a nucleophilic attack on the <span 
class="cmmi-12">&#x03B1;</span>-phosphate of ATP in
solution. B) The ligase is now charged with AMP and inorganic phosphate (PPi) is freed into
solution. C) Step 2: Nucleophilic attack by the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; DNA phosphate on the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick to the
AMP:ligase phosphate. D) &#8217;Adenylated&#8217; DNA is now competent for DNA ligation. E) Step 3: the
3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; OH on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick performs a nucleophilic attack on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> across the
DNA nick, liberating AMP into solution. F) Sealed nick resulting in: Ligase; AMP; and intact
dsDNA.
<!--l. 365--><p class="noindent" >In addition to elucidated the general mechanism of ligation, it was also discovered that T4 DNA ligase lacks
a preference for terminal polynucleotide structures. The Khorana and Richardson labs both reported the
activity of this enzyme on combinations of RNA and DNA duplexes [<a 
href="#XFareed1971">Fareed et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFareed1971">1971</a>,&#x00A0;<a 
href="#XKleppe1970b">Kleppe
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKleppe1970b">1970</a>]. Both of these papers describe an activity on T4 DNA ligase, <span class="underline">RNA-templated DNA to DNA
ligation</span>, that is of particular relevance to this thesis work. Unlike T4 DNA ligase, <span 
class="phvro8t-x-x-120">E. Coli </span>DNA Ligase, will not
join DNA strands on an RNA template [<a 
href="#XBullard2006">Bullard and Bowater</a>,&#x00A0;<a 
href="#XBullard2006">2006</a>]. Soon after demonstrating these
activities in vitro, the Khorana lab reported detection of organism-generated DNA [<a 
href="#XBesmer1972b">Besmer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBesmer1972b">1972</a>],
setting up an orthogonal field (respective to PCR) of nucleic acid sequence characterization [<a 
href="#XConze2009c">Conze
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XConze2009c">2009</a>].
<!--l. 367--><p class="noindent" >An enzyme that can catalyze an RNA-templated DNA:DNA ligation is a very useful molecular biology tool
for two main reasons. First, using RNA as a ligation guide means no modification is made to
the template molecule. This contrasts cDNA analysis, where the RNA has been enzymatically
converted by reverse transcription, potentially losing valuable RNA-coded information, such as
modified bases. Second, synthesis of the DNA probes used in ligation is inherently easier and
cheaper compared to synthesis of RNA probes. In addition to being cheaper, synthesis of DNA
probes has become high-throughput since the adoption of microarrays as a standard gene
expression measurement tool [<a 
href="#XSchena1995a">Schena et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchena1995a">1995</a>]. A pair of papers from the Landegren lab first
reported the utility of RNA-templated DNA:DNA ligation for analysis of RNA transcripts [<a 
href="#XNilsson2001">Nilsson
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNilsson2001">2001</a>,&#x00A0;<a 
href="#XNilsson2000">2000</a>]. The Fu lab applied this approach in a multiplex experimental design in collaboration
with Illumina [<a 
href="#XLi2012c">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2012c">2012</a>,&#x00A0;<a 
href="#XYeakley2002">Yeakley et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYeakley2002">2002</a>], while Mats Nilsson and Ulf Landegren
developed a single molecule application [<a 
href="#XConze2010">Conze et&#x00A0;al.</a>,&#x00A0;<a 
href="#XConze2010">2010</a>]. It is important to note that <span 
class="phvro8t-x-x-120">all </span>of
these studies used T4 DNA ligase. Clearly, there is interest and utility in analyzing RNA in both
high-throughput and multiplex experimental designs, using cheap DNA probes, and without cDNA
conversion.
<!--l. 369--><p class="noindent" >For more than 40 years after its first description, T4 DNA ligase was the only choice for RNA-templated
DNA:DNA ligation. However, a recent publication from New England Biolabs (NEB) describes this activity by
another well-studied ligase, Chlorella Virus PBCV-1 DNA ligase (herein Chlorella DNA ligase) [<span 
class="phvb8t-x-x-120">?</span>]. Chlorella
DNA ligase is a long-studied enzyme and had been reported to <span 
class="phvro8t-x-x-120">not </span>display RNA-templated DNA:DNA
ligation activity [<a 
href="#XHo1997b">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo1997b">1997</a>,&#x00A0;<a 
href="#XSriskanda1998c">Sriskanda and Shuman</a>,&#x00A0;<a 
href="#XSriskanda1998c">1998</a>]. However, at high enough concentrations
and under special buffer conditions (specifically a critical concentration of ATP), Lohman et al have shown
that Chlorella DNA ligase will join two DNA strands hybridized to an RNA template [<span 
class="phvb8t-x-x-120">?</span>]. They further
demonstrated that it performs no worse in this activity than traditional T4 DNA ligase [<a 
href="#XNilsson2001">Nilsson
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNilsson2001">2001</a>,&#x00A0;<a 
href="#XYeakley2002">Yeakley et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYeakley2002">2002</a>].
<!--l. 371--><p class="noindent" >Building on the list of available enzymes that join hybrid polymer substrates Chapter <a 
href="#x1-230002">2<!--tex4ht:ref: Chapter2 --></a> presents
data supporting RNA-templated DNA:DNA ligation activity for another enzyme, T4 RNA Ligase
2.
<a 
 id="x1-19002r37"></a>
                                                                                                    
                                                                                                    
<h4 class="subsectionHead"><span class="titlemark">1.4.2 </span> <a 
 id="x1-200002"></a>T4 RNA Ligase 2 (Rnl2)</h4>
<!--l. 377--><p class="noindent" >Proteins of the T4 and T7 bacteriophages have been a boon for molecular biology. Without enzymes like
polynucleotide kinase [<a 
href="#XRichardson1965a">Richardson</a>,&#x00A0;<a 
href="#XRichardson1965a">1965</a>], T7 RNA polymerase [<a 
href="#XSummers1970b">Summers and Siegel</a>,&#x00A0;<a 
href="#XSummers1970b">1970</a>], and T4 DNA
ligase [<a 
href="#XWeiss1967a">Weiss and Richardson</a>,&#x00A0;<a 
href="#XWeiss1967a">1967</a>], many essential manipulations of nucleic acids would have been
impossible for decades. Obviously, these enzymes also have essential phage functions. T7 RNA
polymerase is responsible for late stage replication of T7 phage transcripts, while T4 PNK works in concert
with T4 DNA and RNA ligases to repair cleaved nucleic acids resulting from bacterial pathogens defense
systems [<a 
href="#XWang2002b">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2002b">2002</a>]. Specifically, T4 RNA ligase 1 (herein "Rnl1", also known as <span 
class="phvro8t-x-x-120">gene 63</span>
maintains phage replication by repairing tRNAs cleaved by an anticodon nuclease produces from the <span 
class="phvro8t-x-x-120">prr</span>
locus [<a 
href="#XAmitsur1987d">Amitsur et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAmitsur1987d">1987</a>].
<!--l. 379--><p class="noindent" >Given the utility and importance of these enzymes, novel enzyme discovery is a fruitful area of research.
The Shuman lab has a distinguished record of discovering and characterizing numerous such enzymes,
including any involved in nucleic acid synthesis, modification, and repair. Through a blast search looking for
novel ligases with sequences related to <span 
class="phvro8t-x-x-120">Trypanosoma brucei </span>RNA-editing ligases TbMP52 and TbMP48 [<a 
href="#XHo2002b">Ho
and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>], they identified motifs in correct arrangement, spacing, and number indicative of an
RNA ligase. The gene, identified as <span 
class="phvro8t-x-x-120">gp24.1</span>, has quickly become an essential tool in the era of modern
genomics.
<!--l. 381--><p class="noindent" >Initial biochemical purification and characterization of <span 
class="phvro8t-x-x-120">gp24.1 </span>[<a 
href="#XHo2002b">Ho and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>] revealed that it
indeed codes for an RNA ligase, which was renamed T4 RNA ligase 2 (herein "Rnl2"). Rnl2 is a 374
amino acid monomeric protein composed of 2 distinct domains initially purified as a 42-kDA
His-tagged recombinant protein. The N-terminal domain (1&#8211;243) is responsible for steps (1)
and (3) of the general ligation mechanisms (see Figure <a 
href="#x1-19001r10">1.10<!--tex4ht:ref: fig:Ligation Mechanism --></a>), while the C-terminal domain
(244&#8211;329) is responsible for adenylation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; residue at the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of
the nick, as shown in step (2). Additionally, Rnl2 is routinely purified as a pre-adenylated and
immediately poised for its first ligation. In contrast to the N-terminal domain, which is composed of
motifs typical to main ligases, the C-terminal domain is significantly different from all other DNA
ligases and has no structural homologue. While the biological function of Rnl1 is known, the
biological function of Rnl2 remains a mystery, more than 12 years after its discovery [<a 
href="#XChauleau2013b">Chauleau and
Shuman</a>,&#x00A0;<a 
href="#XChauleau2013b">2013</a>]. However, there is some speculation that the flurry of research into bacterial
CRISPR phage defense may reveal a role for Rnl2 [<a 
href="#XBarrangou2007c">Barrangou et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarrangou2007c">2007</a>,&#x00A0;<a 
href="#XChauleau2013b">Chauleau and
Shuman</a>,&#x00A0;<a 
href="#XChauleau2013b">2013</a>].
<!--l. 384--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-20001r11"></a>
                                                                                                    
                                                                                                    
<!--l. 386--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.59x.png" alt="PIC" class="graphics" width="355.3275pt" height="292.09125pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.59x.png" src="Figures/Chapter1/Rnl2_Structure.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.11: </span><span  
class="content">Structure and active site of pre-adenylated of Rnl2<br 
class="newline" />Rnl2 as crystalized and described by [<a 
href="#XNandakumar2006">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>]. Structures from PDB:2HVQ were
generated with PyMol. Top left) Rnl2 is composed of a C-terminal and N-terminal domain. Top Right)
The active site of Rnl2 is highlighted. Bottom left) Active site of Rnl2 as shown from bottom. This
face interacts with substrate. Residue numbering refers to that of the crystal structure. </span></div><!--tex4ht:label?: x1-20001r11 -->
                                                                                                    
                                                                                                    
<!--l. 393--><p class="noindent" ></div><hr class="endfigure">
<!--l. 396--><p class="noindent" >Mutational analysis of Rnl2, and later a crystal structure of the enzyme, have identified key functional
residues [<a 
href="#XHo2004">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo2004">2004</a>,&#x00A0;<a 
href="#XNandakumar2004a">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2004a">2004</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>,&#x00A0;<a 
href="#XYin2003d">Yin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYin2003d">2003</a>]. The lysine residue at
position 35 (K35) receives the AMP in Step 1. The K227 residue in the C-terminal domain is essential for
both forward and reverse adenylation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> at the nick [<a 
href="#XViollet2011">Viollet et&#x00A0;al.</a>,&#x00A0;<a 
href="#XViollet2011">2011</a>]. Mutation of H37 results
in an &#x00A0;102 reduced ligation rate, and therefore indicates the essential nature of this residue. Finally, T39 has
been shown to interact with the 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH on the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick, preferring a C3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; endo sugar pucker
confirmation (see Figure <a 
href="#x1-20002r12">1.12<!--tex4ht:ref: fig:Rnl2 Active Site Residues --></a>). Rnl2 has a minimal footprint of 13nt, centered on the nick, and only requires
magnesium for transfer of AMP to the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; phosphate. Work done in the Shuman lab [<span 
class="phvb8t-x-x-120">?</span>] observed that 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
deoxyribose residues on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick (i.e. DNA) adopt an RNA-like sugar pucker,
leading to the correct orientation of the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH relative to the AMP leaving group and resulting
in ligation. This conformation is of particular importance to this results presented in Chapter
<a 
href="#x1-230002">2<!--tex4ht:ref: Chapter2 --></a>.
<!--l. 399--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-20002r12"></a>
                                                                                                    
                                                                                                    
<!--l. 401--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.510x.png" alt="PIC" class="graphics" width="216.81pt" height="217.81374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.510x.png" src="Figures/Chapter1/Rnl2_Active_Site_Residues.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.12: </span><span  
class="content">Structure and active site of pre-adenylated of Rnl2<br 
class="newline" />Rnl2 complexed with nicked dsDNA as crystalized and described by [<a 
href="#XNandakumar2006">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>].
Structures from PDB:2HVR and generated with PyMol </span></div><!--tex4ht:label?: x1-20002r12 -->
                                                                                                    
                                                                                                    
<!--l. 408--><p class="noindent" ></div><hr class="endfigure">
<!--l. 411--><p class="noindent" >While Rnl2 is extremely efficient at high concentration, displaying little or no reversible chemistry, a modified
version of the enzyme containing only the N-terminal domain and a K227A point mutation (&#8220;Truncated
mutant&#8221;) has no adenyltransferase activity. In this case, adenyltransferase refers to the ligase transferring
AMP from an adenylated substrate to itself; reverse chemistry of step 2 in Figure <a 
href="#x1-19001r10">1.10<!--tex4ht:ref: fig:Ligation Mechanism --></a>). This mutant has
been used in specialized cloning applications [<a 
href="#XGhildiyal2008">Ghildiyal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XHafner2008a">Hafner et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHafner2008a">2008</a>,&#x00A0;<a 
href="#XViollet2011">Viollet
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XViollet2011">2011</a>] that take advantage of this activity. In these reactions, the use of pre-adenylated 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; DNA
adaptors allows for selective ligation among already phosphorylated species by limiting the
enzyme-catalyzed transfer of AMP from the adaptor to other phosphorylated species. Use of this truncated
mutant to create a hybrid RNA/DNA molecule has greatly improved many high-throughput sequencing work
flows.
<!--l. 413--><p class="noindent" >Ligation of hybrid substrates (eg. DNA-templated RNA:DNA vs DNA-templated DNA:DNA) have revealed
general substrate preferences. DNA ligases appear to prefer the residue bearing the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; phosphate on the
3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; side of the nick to be 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0; deoxyribose, and have a relaxed requirement for the sugar on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the
nick. RNA ligases have the reverse preference, demonstrating higher activities when the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; strand, 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH
residue also bears a 2<span 
class="cmsy-8">&#x2032;</span>OH. Rnl2 has an additional preference for an RNA residue at the penultimate 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
side of a nick [<a 
href="#XHo2002b">Ho and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>,&#x00A0;<a 
href="#XHo2004">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo2004">2004</a>,&#x00A0;<a 
href="#XNandakumar2004a">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2004a">2004</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. The
two base requirement for RNA at the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the double stranded nick biases Rnl2 to join
RNA:[RNA/DNA] strands. Independent labs have measured this preference and have reported that the
RNA-templated DNA:DNA joining activity of Rnl2 is below assay limits of detection [<a 
href="#XBullard2006">Bullard
and Bowater</a>,&#x00A0;<a 
href="#XBullard2006">2006</a>]. However, results discussed in this work clearly show that with enough
enzyme, and sensitive downstream measurements, Rnl2 will catalyze RNA-templated DNA:DNA
ligation (see Chapter <a 
href="#x1-230002">2<!--tex4ht:ref: Chapter2 --></a>. Previous reports of Rnl2 lacking this activity are likely due to a single
turnover mechanism in this reaction, owning to the poor dissociation rate of nucleic acid-interacting
enzymes.
<a 
 id="x1-20003r36"></a>
<h3 class="sectionHead"><span class="titlemark">1.5 </span> <a 
 id="x1-210005"></a>Long Nucleic Acid Polymers</h3>
<a 
 id="x1-21001r39"></a>
<h4 class="subsectionHead"><span class="titlemark">1.5.1 </span> <a 
 id="x1-220001"></a>Mouse piRNAs are different</h4>
<!--l. 434--><p class="noindent" >Mammalian spermatogenesis is critical for the future of the species. Recently the importance a specific kind
of small RNA&#8212;piRNAs&#8212;for proper spermatogenesis has become clear [<a 
href="#XSiomi2011">Siomi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSiomi2011">2011</a>]. Even after &#x003E;12
years since their discovery in <span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>[<a 
href="#XAravin2001">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2001">2001</a>], and &#x003E;6 since their identification
in rodents [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>], the essential mechanisms of piRNA biogenesis to proper
mammalian spermatogenesis remains largely unknown. These unknown mechanisms include: biogenesis
from transcript to small RNA, physiological targets, and terminal function of sterility maintenance. What
is known is that without a functioning piRNA pathway males are sterile. Studies in humans
have also linked SNPs in the Argonaute proteins that bind piRNAs to decreased fertility [<a 
href="#XGu2010a">Gu
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGu2010a">2010</a>].
<!--l. 437--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-22001r13"></a>
                                                                                                    
                                                                                                    
<!--l. 439--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.511x.png" alt="PIC" class="graphics" width="375.4025pt" height="234.87749pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.511x.png" src="Figures/Chapter1/FlamencoLocus.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.13:  </span><span  
class="content">A  the  <span 
class="phvro8t-x-x-120">Drosophila  melanogaster   </span>gene  <span 
class="phvro8t-x-x-120">flamenco  </span>is  a  graveyard  for  transposon
sequences [<a 
href="#XPelisson1994">Plisson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPelisson1994">1994</a>]. Evidence for expression of a single-contiguous RNA transcript
from <span 
class="phvro8t-x-x-120">flamenco  </span>(A)  is  provided  by  a  P-element  insertion  into  the  suspected  promoter  region  (B).
[<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>] could not detect specific piRNAs (red X&#8217;s) by northern blot in the P-element
mutant. </span></div><!--tex4ht:label?: x1-22001r13 -->
                                                                                                    
                                                                                                    
<!--l. 445--><p class="noindent" ></div><hr class="endfigure">
<!--l. 449--><p class="noindent" >piRNAs are so-called because they bind PIWI proteins, a sub-group of the Argonaute protein family, whose
other members utilize small RNA as guides to target nucleic acids for many forms of post-transcriptional
regulation [<a 
href="#XSiomi2011">Siomi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSiomi2011">2011</a>]. There are three PIWI proteins in mice, each displaying a distinct expression
profile during development and an association with piRNAs of a specific length. The first PIWI protein
expressed, even before a mouse is born, is MIWI2 [<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>], followed quickly by the more
consistent player, MILI (see Figure <span 
class="phvb8t-x-x-120">??</span> [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">2004</a>]. It is
during the &#8217;fetal&#8217; stage of piRNA biogenesis in mice that MIWI2 and MILI undergo ping-pong
amplification, similar to that observed in flies, in order to silence expression of transposons
during germ line formation [<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">2008</a>]. After
birth, and during the &#8217;neonatal stage&#8217; only MILI is expressed, and piRNAs shift from mostly
transposon-mapping to 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; UTR mapping [<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>]. Once the &#8217;first wave&#8217; of spermatogenesis
[<a 
href="#XLaiho2013a">Laiho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLaiho2013a">2013</a>,&#x00A0;<a 
href="#XOakberg1956b">Oakberg and Oakberq</a>,&#x00A0;<a 
href="#XOakberg1956b">1956</a>] reaches meiosis, the pachytene piRNAs are
expressed [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>,&#x00A0;<a 
href="#XLi2013h">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013h">2013a</a>]. Pachytene piRNAs tend to
map to intergenic &#8217;clusters&#8217; of unique genomic sequence. These clusters, from here after called
&#8217;piRNA-generating genes&#8217; appear to produce a single, continuous, relatively long, and un-spliced Pol II
transcript [<a 
href="#XLi2013h">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013h">2013a</a>]. This is comparable to piRNA clusters in flies, such as flamenco, whose
transcription can be abolished with a P-element insertion into a putative promoter, as measured by
northern blot looking for piRNAs generated 168 kb downstream (see Figure <a 
href="#x1-22001r13">1.13<!--tex4ht:ref: fig:flamenco --></a> [<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>].
<!--l. 452--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-22002r14"></a>
                                                                                                    
                                                                                                    
<!--l. 454--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.512x.png" alt="PIC" class="graphics" width="415.55249pt" height="324.21124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.512x.png" src="Figures/Chapter1/MammalianPiRNAClassesOverTime.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.14: </span><span  
class="content"><span class="texthl"><span id="textcolor1">Write a nice caption here!</span></span> </span></div><!--tex4ht:label?: x1-22002r14 -->
                                                                                                    
                                                                                                    
<!--l. 460--><p class="noindent" ></div><hr class="endfigure">
<!--l. 463--><p class="noindent" >Mammalian piRNAs can be divided into three major classes (#Figure of my own design on
piRNAs) [<a 
href="#XKuramochi-Miyagawa2008">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">2008</a>]. The first, present before birth, are those of the <span 
class="phvro8t-x-x-120">fetal</span>
<span 
class="phvro8t-x-x-120">piRNAs</span>. These piRNAs tend to be short, bind the PIWI protein MILI2 in mice, and have sequences
found in transposable elements (Carmell et al. 2007). The next class of piRNAs, historically but
confusing grouped with the previous class, are called Pre-pachytene piRNAs. Pre-pachytene
piRNAs are expressed just before birth, and continue to be expressed in functioning testes,
often associated with spermotoXX and spermotXX, precursor cells to mature sperm(?), and
tend to map to traditional, and annotated, protein coding genes. Finally, due to their unique
sequence in the genome, the genetic origin of a millions of piRNAs belonging to the third class, the
pachytene piRNAs, was immediately known. Pachytene piRNAs are extremely abundant after, not
coincidently, the pachytene stage of meiosis I when (descriptor) chromosomes pair up, cross over, and
rearrange their genetic material. Extremely abundant means XX fold enrichment of pachytene
piRNAs compared to miRNAs, as compared to developmental time point just two days earlier.
The genomic origins of these piRNAs, while unique in terms of sequence, are often in &#8216;gene
deserts,&#8217; unannotated, and devoid of intronic sequences. This gene architecture makes the
pachytene piRNA loci some of the most interesting RNA-producing regions of the mammalian
genome.
<!--l. 466--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-22003r15"></a>
                                                                                                    
                                                                                                    

<!--l. 468--><p class="noindent" ><img 
src="Figures/Chapter1/mammalian_piRNA_pathway.png" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.15: </span><span  
class="content">Figure taken from [<a 
href="#XLi2013e">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013e">2013b</a>]: A model for piRNA biogenesis. Primary piRNA
transcripts are transcribed by RNA polymerase II and contain 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; caps, exons and introns and poly(A)
tails. The transcription of pachytene piRNA genes is controlled by A-MYB; transcription factor(s)
(TF)  controlling  pre-pachytene  piRNA  genes  remain  to  be  discovered.  Current  models  of  piRNA
biogenesis  propose  that  PLD6  determines  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  piRNA  intermediates  with  lengths  &#x003E;30
nt. These intermediates are proposed to then be loaded into PIWI proteins. After PIWI binding, a
nuclease is thought to trim the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of the piRNA to the length characteristic of the particular
bound PIWI protein. Finally, further trimming is prevented by addition of a 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0;-O-methyl group to the
3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of the mature piRNA by the S-adenosylmethionine-dependent methyltransferase HEN1 </span></div><!--tex4ht:label?: x1-22003r15 -->
                                                                                                    
                                                                                                    
<!--l. 474--><p class="noindent" ></div><hr class="endfigure">
<!--l. 478--><p class="noindent" ><a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>&#x00A0;[<a 
href="#XVourekas2012">2012</a>] demonstrated that many MIWI, as measured by ClIP, actually bind mRNAs directly,
without a small RNA guide.
<!--l. 480--><p class="noindent" >Two studies [<a 
href="#XDeFazio2011">De Fazio et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDeFazio2011">2011</a>,&#x00A0;<a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2011">2011</a>] used point mutations in the catalytic triad of
MIWI2, MILI to remove slicer activity. The MIWI2 and MILI studies found that the mice were sterile, and did
not accumulate transposon-mapping piRNAs. <a 
href="#XDeFazio2011">De Fazio et&#x00A0;al.</a>&#x00A0;[<a 
href="#XDeFazio2011">2011</a>] found that the mice were also sterile,
and demonstrated increased LINE1 transcript accumulation. <a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>&#x00A0;[<a 
href="#XReuter2011">2011</a>] states that much the
biological activity of MIWI depends on its slicer activity.
<!--l. 483--><p class="noindent" >The transposon-mapping nature of the fetal piRNA class made obvious comparisons to the fly piRNA
system nature. In the fly system, primary piRNAs transcribed from discrete loci and fed into an amplification
loop between two PIWI proteins PIWI(3) and AGO3 (&#8216;the ping-pong&#8217; cycle) (REF Brenneki cell paper 2007).
It is believed that PIWI proteins loaded with piRNAs bind and silence transposon messages, using the
cleaved transposon transcripts, in combination with primary piRNA transcripts, as a substrates in the
Ping-Pong cycle
                                                                                                    
                                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;2</span><br /><a 
 id="x1-230002"></a>SeqZip - Development and Applications</h2>
<a 
 id="x1-23001r42"></a>
<h3 class="sectionHead"><span class="titlemark">2.1 </span> <a 
 id="x1-240001"></a>SeqZip Overview</h3>
<a 
 id="x1-24001r43"></a>
<h4 class="subsectionHead"><span class="titlemark">2.1.1 </span> <a 
 id="x1-250001"></a>Subsection 1</h4>
<!--l. 14--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-25001r1"></a>
                                                                                                    
                                                                                                    
<!--l. 16--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.513x.png" alt="PIC" class="graphics" width="396.48125pt" height="197.73874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.513x.png" src="Figures/Chapter2/OriginalSeqZipDiagram.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.1: </span><span  
class="content">Original SeqZip Diagram<br 
class="newline" />This is the original concept diagram of the SeqZip methodology. (A) Specific DNA oligos target an
mRNA and loop out the RNA sequence. Ligases is added to join the DNA oligos together; (B) &amp; (C)
Two different possibilities of ligation products templated from the RNA in (A), where Exon 2 is an
cassette exon. </span></div><!--tex4ht:label?: x1-25001r1 -->
                                                                                                    
                                                                                                    
<!--l. 23--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-25002r48"></a>
<h3 class="sectionHead"><span class="titlemark">2.2 </span> <a 
 id="x1-260002"></a>Multiplex Gene Study</h3>
<!--l. 30--><p class="noindent" >One of the major goals of developing the SeqZip methodology was investigating potential coordination
genome-wide. By genome-wide, what we really mean is to analyze many (or all) of the RNA transcripts in a
tissue for evidence of coordinated splicing decisions. When development of the method reached the point
that it could be applied in a multiplex study, I did not posses the bioinformatic skills necessary to 1) design
ligamers in an automated and high-throughput fashion and 2) identify target transcripts, exons, and
sequences to investigate for potential connectivity. Both of these points are discussed later, see &#x003C;automated
ligamer design | appendix&#x003E; and &#x003C;ideas on transcript identification for SeqZip coordination investigation |
Discussion/Perspective&#x003E;.
<!--l. 32--><p class="noindent" >A hypothesis we wanted to use SeqZip to test was that coordination among distant regions of AS in the
same transcript is a general phenomenon. It is important to note that we are not limiting our scope of
coordination to that between two internal sites of AS (&#8211;e.g., cassette or mutually exclusive exon events).
From microarrays studies, it has been estimated that approximately 30% of all AS events involve alternative
first and last exons (Bingham et al., 2008). It is known that, through alternative use of first and last exons,
cells can fine-tune a transcript&#8217;s untranslated region (UTR) and control many aspects of mRNA
regulation including nuclear export, localization, expression, and stability (Hughes, 2006). In
support of the importance of alternative UTRs in tuning of gene expression, a landmark RNA-Seq
study demonstrated a high occurrence of alternative first and last exon splicing, with alternative
tandem 3&#8217; UTR usage being the most highly tissue-dependent form of AS observed (Wang et al.,
2008). The current model of spatial proximity between 5&#8217; and 3&#8217; UTRs is suggestive of their
possible interdependence. In our large-scale analysis, we included genes potentially displaying
interdependence between first and last exons. Discovery of interdependence would lead to many
questions into how specific combinations of UTRs can influence mRNA processing downstream of
AS.
<!--l. 36--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-26001r2"></a>
                                                                                                    
                                                                                                    
<!--l. 38--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.514x.png" alt="PIC" class="graphics" width="344.28624pt" height="370.38374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.514x.png" src="Figures/Chapter2/10GeneSetSchematic.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.2: </span><span  
class="content">10 Gene set schematic<br 
class="newline" /><span class="texthl"><span id="textcolor2">caption next</span></span> </span></div><!--tex4ht:label?: x1-26001r2 -->
                                                                                                    
                                                                                                    
<!--l. 45--><p class="noindent" ></div><hr class="endfigure">
<!--l. 48--><p class="noindent" >In an effort not to let my lack of bioinformatic ability hold back application of SeqZip to many transcripts at
once, I designed a series of experiments that I termed a &#8217;multiplex&#8217; application of SeqZip. It was based off of
a very important paper from the Blencowe lab, where they used AS-sensitive microarrays to investigate
transcripts in the Mouse CNS (Fagnani et al. 2007). This paper identified genes displaying tissue-specific
splicing patterns, focusing on those with CNS-specific patterns. Once section of their paper focused on
"Coordination between AS events belonging to the same genes," and seemed to be the exact type of
experiment we were interested in applying the SeqZip method too. Five hundred of the 3,044 genes
investigated by their microarrays contained 2&#8211;5 alternative exons. Fagnani et all contained an additional
data file a list of all pair-wise combinations of alternative exons in the same gene (with that
gene having significant expression in &#x003E;20 different tissues), along with the standard and partial
spearman correlations. Using this dataset, I filtered the exon pairs to those with a distance &#x003E;
350 nt in the final pre-mRNA. I also visualized their transcript achetecture, and EST evidence
using NCBIs AceView tool (Thierry-Mieg and Thierry-Mieg 2006). For example, the exons with
strong correlation of expression in the Chl1 gene are in the beginning (second exon) and end
(fourth from last exon, accession BC060216) with plenty of supporting evidence for these exons
being expressed, and skipped. After combing through the Fagnani data for a group of about
10 genes displaying these characteristics, I designed ligamers for the alternative, and flank
constitutive exons. These oligos were then ordered from IDT in a 96-well plate format, pooled
according to gene, and used to develop a multiplex approach to applying SeqZip, as well as
investigate coordination between these exons, in these genes, using mouse total RNA from
brains.
<!--l. 50--><p class="noindent" >After synthesis, ligamers targeting a particular gene will be pooled at the predetermined appropriate
concentration. Poly(A) samples from a treatment condition, cell-, or tissue-type of interest will be isolated.
These samples will be parsed out into individual wells of a 96-well plate, and different ligamer sets will be
added to individual wells followed by analysis using SeqZip. Analysis of the resulting FLLPs can be carried
out manually using semi-quantitative PCR, followed by denaturing PAGE or by one 454 sequencing run.
After analysis, the lengths of observed FLLPs will be related back to those predicted during ligamer
construction. Our primary data will represent the relative abundance of observed gene-specific isoforms for
the cell- or tissue-type examined.
<a 
 id="x1-26002r49"></a>
<h4 class="subsectionHead"><span class="titlemark">2.2.1 </span> <a 
 id="x1-270001"></a>Subsection 1</h4>
<div class="table">
                                                                                                    
                                                                                                    
<!--l. 56--><p class="noindent" ><a 
 id="x1-27001r1"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1"></colgroup><colgroup id="TBL-4-2g"><col 
id="TBL-4-2"></colgroup><colgroup id="TBL-4-3g"><col 
id="TBL-4-3"></colgroup><colgroup id="TBL-4-4g"><col 
id="TBL-4-4"></colgroup><colgroup id="TBL-4-5g"><col 
id="TBL-4-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-1-1"  
class="td11"><span 
class="phvb8t-x-x-120">Gene name</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-2"  
class="td11"><span 
class="phvb8t-x-x-120">nt of mRNA between exons</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-3"  
class="td11"><span 
class="phvb8t-x-x-120">possible isoforms</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-4"  
class="td11"><span 
class="phvb8t-x-x-120">Exon 1</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-5"  
class="td11"><span 
class="phvb8t-x-x-120">Exon 2</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-2-1"  
class="td11">Chl1             </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-2"  
class="td11">                     4665</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-3"  
class="td11">              18</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-4"  
class="td11">     2</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-5"  
class="td11">    24</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-3-1"  
class="td
11">Mdm1          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-2"  
class="td11">                     1846</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-4"  
class="td11">  EDA</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-5"  
class="td11">  IIICS</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-4-1"  
class="td
11">PTPRF-Y     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-2"  
class="td11">                     1633</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-4"  
class="td11">     2</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-5"  
class="td11">    13</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-5-1"  
class="td
11">Cacna1c      </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-2"  
class="td11">                     1403</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-4"  
class="td11">    15</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-5"  
class="td11">  21/22</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-6-1"  
class="td
11">PTPRF-X     </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-2"  
class="td11">                      936</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-4"  
class="td11">   9/10</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-5"  
class="td11">    21</td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-7-1"  
class="td
11">FN1              </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-2"  
class="td11">                      813</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-3"  
class="td11">               8</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-4"  
class="td11">  13/14</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-5"  
class="td11">  21/22</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-8-1"  
class="td11">Apbb1          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-2"  
class="td11">                      802</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-3"  
class="td11">             260</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-4"  
class="td11">   1/2b</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-5"  
class="td11">   2/3e</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-9-1"  
class="td11">Agrn             </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-2"  
class="td11">                      736</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-3"  
class="td11">               8</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-4"  
class="td11"> 33/34c</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-5"  
class="td11"> 33/34a</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-10-1"  
class="td11">Exoc7          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-2"  
class="td11">                      513</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-4"  
class="td11">     7</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-5"  
class="td11">    13</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-11-1"  
class="td11">Prom1          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-2"  
class="td11">                      512</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-3"  
class="td11">               4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-4"  
class="td11">     7</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-5"  
class="td11">     9</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-12-1"  
class="td11">Lphn2          </td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-2"  
class="td11">                      396</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-3"  
class="td11">              32</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-4"  
class="td11">    19</td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-5"  
class="td11"> 24/25a</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-13-1"  
class="td11">           </td></tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;2.1: </span><span  
class="content"><span class="texthl"><span id="textcolor3">caption</span></span></span></div><!--tex4ht:label?: x1-27001r1 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
<a 
 id="x1-27002r51"></a>
<h3 class="sectionHead"><span class="titlemark">2.3 </span> <a 
 id="x1-280003"></a>Determining RNA integrity using SeqZip</h3>
<a 
 id="x1-28001r53"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.1 </span> <a 
 id="x1-290001"></a>Demonstration of Concept</h4>
<!--l. 85--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-29001r3"></a>
                                                                                                    
                                                                                                    
<!--l. 87--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.515x.png" alt="PIC" class="graphics" width="288.07625pt" height="251.94124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.515x.png" src="Figures/Chapter2/DegreadedRNABySeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.3: </span><span  
class="content">Ligation product tied to RNA integrity<br 
class="newline" /><span class="texthl"><span id="textcolor4">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-29001r3 -->
                                                                                                    
                                                                                                    
<!--l. 94--><p class="noindent" ></div><hr class="endfigure">
<!--l. 97--><p class="noindent" >Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Ut purus elit, vestibulum ut, placerat
ac, adipiscing vitae, felis. Curabitur dictum gravida mauris. Nam arcu libero, nonummy eget,
consectetuer id, vulputate a, magna. Donec vehicula augue eu neque. Pellentesque habitant
morbi tristique senectus et netus et malesuada fames ac turpis egestas. Mauris ut leo. Cras
viverra metus rhoncus sem. Nulla et lectus vestibulum urna fringilla ultrices. Phasellus eu tellus
sit amet tortor gravida placerat. Integer sapien est, iaculis in, pretium quis, viverra ac, nunc.
Praesent eget sem vel leo ultrices bibendum. Aenean faucibus. Morbi dolor nulla, malesuada eu,
pulvinar at, mollis ac, nulla. Curabitur auctor semper nulla. Donec varius orci eget risus. Duis nibh
mi, congue eu, accumsan eleifend, sagittis quis, diam. Duis eget orci sit amet orci dignissim
rutrum.
<!--l. 97--><p class="noindent" >Nam dui ligula, fringilla a, euismod sodales, sollicitudin vel, wisi. Morbi auctor lorem non justo. Nam lacus
libero, pretium at, lobortis vitae, ultricies et, tellus. Donec aliquet, tortor sed accumsan bibendum, erat ligula
aliquet magna, vitae ornare odio metus a mi. Morbi ac orci et nisl hendrerit mollis. Suspendisse ut massa.
Cras nec ante. Pellentesque a nulla. Cum sociis natoque penatibus et magnis dis parturient montes,
nascetur ridiculus mus. Aliquam tincidunt urna. Nulla ullamcorper vestibulum turpis. Pellentesque cursus
luctus mauris.
<!--l. 97--><p class="noindent" >Nulla malesuada porttitor diam. Donec felis erat, congue non, volutpat at, tincidunt tristique, libero. Vivamus
viverra fermentum felis. Donec nonummy pellentesque ante. Phasellus adipiscing semper elit. Proin
fermentum massa ac quam. Sed diam turpis, molestie vitae, placerat a, molestie nec, leo. Maecenas
lacinia. Nam ipsum ligula, eleifend at, accumsan nec, suscipit a, ipsum. Morbi blandit ligula feugiat magna.
Nunc eleifend consequat lorem. Sed lacinia nulla vitae enim. Pellentesque tincidunt purus vel magna.
Integer non enim. Praesent euismod nunc eu purus. Donec bibendum quam in tellus. Nullam cursus
pulvinar lectus. Donec et mi. Nam vulputate metus eu enim. Vestibulum pellentesque felis eu
massa.
<!--l. 100--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-29002r4"></a>
                                                                                                    
                                                                                                    
<!--l. 102--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.516x.png" alt="PIC" class="graphics" width="363.3575pt" height="258.9675pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.516x.png" src="Figures/Chapter2/TransRNAWithSeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.4: </span><span  
class="content">Trans Transcript investigation<br 
class="newline" /><span class="texthl"><span id="textcolor5">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-29002r4 -->
                                                                                                    
                                                                                                    
<!--l. 109--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-29003r56"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.2 </span> <a 
 id="x1-300002"></a>Investigating HIV viral genome integrity using SeqZip</h4>
<!--l. 119--><p class="noindent" >MOV10L1 is implicated in HIV genome stability and intactness. We sought to measure the effects of a point
mutant in the ATPase domain of MOV10L1 on the &#8216;intactness&#8217; of the HIV genome. Viral particles contain two
copies of the ssRNA HIV genome. Proteomics studies have measured MOV10L1 and Upf1 in viral
particles, implicated these proteins, which are known helicases, in the maintained and infectivity of
HIV.
<a 
 id="x1-30001r59"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.3 </span> <a 
 id="x1-310003"></a>Design of HIV ligamers</h4>
<!--l. 125--><p class="noindent" >Research into the integrity of the HIV RNA genome using SeqZip began with designing a set of ligamers
against two different clones. The first clone, targeting transcripts from the M19921 plasmid (so called &#8217;M&#8217;
clone), and transcripts from the K03455 clone contain nearly identical sequences with respect to the
genome itself, and differ mostly in plasmid originating sequences. We targeted a difference in sequence for
one site of ligation (Fig3-11A). Three different pools of ligamers were created initially: a Five(5) ligamer pool,
with three ligamers design to test for the presence of sequence in the first 1,140 nt of the HIV genome,
importantly the first site of ligation in the 5 region pool should contain a mismatch in the K clone
sequence; a three(3) pool, testing the last 1,210 nt of the genome, and a Long (L) ligamer pool, also
containing three ligamers, but the middle ligamer of which would span the 5 and 3 regions,
looping out 8,633 nt of sequence in the middle of the HIV genome. In vitro transcripts were
created using both the K and M clone plasmids. These transcripts were added to a background of
total MEF RNA, and the SeqZIp assay was performed. Ligation products were successfully
amplified from all ligamer pools when using the M clone transcript and all three ligamer pools.
Also the abundance of these ligation products, as measured by endpoint PCR, seemed to be
spike-concentration dependent. Notably, Ligation products were not obtained from the K clone
using either the 5 or L ligamer pools, likely due to the mismatch between the transcript and
the ligamers at the site of ligation. Also of note was the appearance of ligation products from
purified endogenous virons of the M clone from all three ligamer pools, and the absence of
products from virions purified from plasmids containing a defective protein, Gag, essential for viral
packaging.
<!--l. 128--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-31001r5"></a>
                                                                                                    
                                                                                                    
<!--l. 130--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.517x.png" alt="PIC" class="graphics" width="392.46625pt" height="346.29375pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.517x.png" src="Figures/Chapter2/HIVviaSeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.5: </span><span  
class="content">SeqZip can examine HIV transcript integrity<br 
class="newline" /><span class="texthl"><span id="textcolor6">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-31001r5 -->
                                                                                                    
                                                                                                    
<!--l. 137--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-31002r55"></a>
<h3 class="sectionHead"><span class="titlemark">2.4 </span> <a 
 id="x1-320004"></a>Continuity of piRNA precursor transcripts</h3>
<!--l. 145--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-32001r6"></a>
                                                                                                    
                                                                                                    
<!--l. 147--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.518x.png" alt="PIC" class="graphics" width="378.41374pt" height="294.09874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.518x.png" src="Figures/Chapter2/PrecursorLocations.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.6: </span><span  
class="content">piRNA precursor locations<br 
class="newline" /><span class="texthl"><span id="textcolor7">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-32001r6 -->
                                                                                                    
                                                                                                    
<!--l. 154--><p class="noindent" ></div><hr class="endfigure">
<div class="table">
                                                                                                    
                                                                                                    
<!--l. 1--><p class="noindent" ><a 
 id="x1-32002r2"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1"><col 
id="TBL-5-2"><col 
id="TBL-5-3"><col 
id="TBL-5-4"><col 
id="TBL-5-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-1"  
class="td11"><span 
class="phvb8t-x-x-120">Cluster Name     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-2"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-6-1-1"  
class="td00"><span 
class="phvb8t-x-x-120">Matched</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-6-2-1"  
class="td00"> <span 
class="phvb8t-x-x-120">Cluster</span></td></tr></table>                                                                                                                                          </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-3"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-7-1g"><col 
id="TBL-7-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-7-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-1-1"  
class="td00"><span 
class="phvb8t-x-x-120">Unique-mapping</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-7-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-2-1"  
class="td00">   <span 
class="phvb8t-x-x-120">piRNAs @      </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-7-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-3-1"  
class="td00">    <span 
class="phvb8t-x-x-120">wt.14dpp       </span></td></tr></table>                                                                                                                             </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-4"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-1-1"  
class="td00"><span 
class="phvb8t-x-x-120">Fraction of</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-2-1"  
class="td00"> <span 
class="phvb8t-x-x-120">pachytene</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-3-1"  
class="td00">  <span 
class="phvb8t-x-x-120">piRNAs   </span></td></tr></table>                                                                                                                                      </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-5"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-9-1g"><col 
id="TBL-9-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-9-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-1-1"  
class="td00"><span 
class="phvb8t-x-x-120">Cumulative</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-9-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-2-1"  
class="td00"> <span 
class="phvb8t-x-x-120">pachytene</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-9-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-3-1"  
class="td00">  <span 
class="phvb8t-x-x-120">piRNAs    </span></td></tr></table>                                                                                                                                      </div></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-1"  
class="td11">17-qA3.3-26735.1</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-2"  
class="td11">17-qA3.3-27363                                                                                                                                    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-3"  
class="td11">                                                                             3,021,022</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-4"  
class="td11">                                                                                 17.2</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-5"  
class="td11">                                                                                 17.2</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-1"  
class="td11">17-qA3.3-27363.1</td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-2"  
class="td11">17-qA3.3-26735                                                                                                                                    </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-3"  
class="td11">                                                                             1,742,695</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-4"  
class="td11">                                                                                  9.9</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-5"  
class="td11">                                                                                 27.2</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-1"  
class="td11">9-qC-31469.1       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-2"  
class="td11">9-qC-10667                                                                                                                                           </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-3"  
class="td11">                                                                             1,006,333</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-4"  
class="td11">                                                                                  5.7</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-5"  
class="td11">                                                                                 32.9</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-1"  
class="td11">9-qC-10667.1       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-2"  
class="td11">9-qC-31469                                                                                                                                           </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-3"  
class="td11">                                                                              272,385</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-4"  
class="td11">                                                                                  1.6</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-5"  
class="td11">                                                                                 34.5</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-1"  
class="td11">7-qD2-24830.1     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-2"  
class="td11">7-qD2-11976                                                                                                                                         </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-3"  
class="td11">                                                                              652,564</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-4"  
class="td11">                                                                                  3.7</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-5"  
class="td11">                                                                                 38.2</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-1"  
class="td11">7-qD2-11976.1     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-2"  
class="td11">7-qD2-24830                                                                                                                                         </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-3"  
class="td11">                                                                              280,312</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-4"  
class="td11">                                                                                  1.6</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-5"  
class="td11">                                                                                 39.8</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-1"  
class="td11">6-qF3-28913.1     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-2"  
class="td11">6-qF3-8009                                                                                                                                           </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-3"  
class="td11">                                                                              564,930</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-4"  
class="td11">                                                                                  3.2</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-5"  
class="td11">                                                                                 43.0</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-1"  
class="td11">6-qF3-8009.1       </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-2"  
class="td11">6-qF3-28913                                                                                                                                         </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-3"  
class="td11">                                                                              180,210</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-4"  
class="td11">                                                                                  1.0</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-5"  
class="td11">                                                                                 44.0</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-1"  
class="td11">2-qE1-35981.1     </td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-2"  
class="td11">NA                                                                                                                                                         </td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-3"  
class="td11">                                                                              1121042</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-4"  
class="td11">                                                                                  6.4</td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-5"  
class="td11">                                                                                 50.4</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-1"  
class="td11">                </td></tr></table>
</div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;2.2: </span><span  
class="content"><span class="texthl"><span id="textcolor8">caption</span></span></span></div><!--tex4ht:label?: x1-32002r2 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
<!--l. 162--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-32003r7"></a>
                                                                                                    
                                                                                                    
<!--l. 164--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.519x.png" alt="PIC" class="graphics" width="341.275pt" height="346.29375pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.519x.png" src="Figures/Chapter2/RTDoesntWork.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.7: </span><span  
class="content">RT Doesn&#8217;t Work for piRNA precursors<br 
class="newline" /><span class="texthl"><span id="textcolor9">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-32003r7 -->
                                                                                                    
                                                                                                    
<!--l. 171--><p class="noindent" ></div><hr class="endfigure">
<!--l. 176--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-32004r8"></a>
                                                                                                    
                                                                                                    
<!--l. 178--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.520x.png" alt="PIC" class="graphics" width="291.0875pt" height="160.59999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.520x.png" src="Figures/Chapter2/dst1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.8: </span><span  
class="content">Dst1 by SeqZip<br 
class="newline" /><span class="texthl"><span id="textcolor10">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-32004r8 -->
                                                                                                    
                                                                                                    
<!--l. 185--><p class="noindent" ></div><hr class="endfigure">
<!--l. 190--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-32005r9"></a>
                                                                                                    
                                                                                                    
<!--l. 192--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.521x.png" alt="PIC" class="graphics" width="336.25624pt" height="208.78pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.521x.png" src="Figures/Chapter2/testesSpecificRnaseqPrecursors.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.9: </span><span  
class="content">Testes Specific RNA precursor expression<br 
class="newline" /><span class="texthl"><span id="textcolor11">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-32005r9 -->
                                                                                                    
                                                                                                    
<!--l. 199--><p class="noindent" ></div><hr class="endfigure">
<!--l. 204--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-32006r10"></a>
                                                                                                    
                                                                                                    
<!--l. 206--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.522x.png" alt="PIC" class="graphics" width="352.31624pt" height="288.07625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.522x.png" src="Figures/Chapter2/piRNAPrecurserAnalyisBySeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.10: </span><span  
class="content">piRNA precursor analysis via SeqZip<br 
class="newline" /><span class="texthl"><span id="textcolor12">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-32006r10 -->
                                                                                                    
                                                                                                    
<!--l. 213--><p class="noindent" ></div><hr class="endfigure">
                                                                                                    
                                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;3</span><br /><a 
 id="x1-330003"></a>SeqZip Publication</h2>
<a 
 id="x1-33001r62"></a>
<h3 class="sectionHead"><span class="titlemark">3.1 </span> <a 
 id="x1-340001"></a>Abstract</h3>
<a 
 id="x1-34001r70"></a>
<h3 class="sectionHead"><span class="titlemark">3.2 </span> <a 
 id="x1-350002"></a>Introduction</h3>
<!--l. 13--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-35001r1"></a>
                                                                                                    
                                                                                                    
<!--l. 15--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.523x.png" alt="PIC" class="graphics" width="409.53pt" height="143.53624pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.523x.png" src="Figures/Chapter3/Roy2014Fig1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.1: </span><span  
class="content">SeqZip Diagram]<br 
class="newline" /><span class="texthl"><span id="textcolor13">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-35001r1 -->
                                                                                                    
                                                                                                    
<!--l. 22--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-35002r71"></a>
<h3 class="sectionHead"><span class="titlemark">3.3 </span> <a 
 id="x1-360003"></a>Results</h3>
<a 
 id="x1-36001r60"></a>
<h4 class="subsectionHead"><span class="titlemark">3.3.1 </span> <a 
 id="x1-370001"></a>Subsection 2</h4>
<!--l. 35--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-37001r2"></a>
                                                                                                    
                                                                                                    
<!--l. 37--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.524x.png" alt="PIC" class="graphics" width="387.4475pt" height="358.33875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.524x.png" src="Figures/Chapter3/Roy2014Fig2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.2: </span><span  
class="content">Rnl2 Panel]<br 
class="newline" /><span class="texthl"><span id="textcolor14">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-37001r2 -->
                                                                                                    
                                                                                                    
<!--l. 44--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-37002r74"></a>
<h4 class="subsectionHead"><span class="titlemark">3.3.2 </span> <a 
 id="x1-380002"></a>Subsection 2</h4>
<!--l. 53--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-38001r3"></a>
                                                                                                    
                                                                                                    
<!--l. 55--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.525x.png" alt="PIC" class="graphics" width="390.45874pt" height="478.78874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.525x.png" src="Figures/Chapter3/Roy2014Fig3.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.3: </span><span  
class="content">Rnl2 Panel]<br 
class="newline" /><span class="texthl"><span id="textcolor15">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-38001r3 -->
                                                                                                    
                                                                                                    
<!--l. 62--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-38002r73"></a>
<h3 class="sectionHead"><span class="titlemark">3.4 </span> <a 
 id="x1-390004"></a>Discussion</h3>
<a 
 id="x1-39001r78"></a>
<h3 class="sectionHead"><span class="titlemark">3.5 </span> <a 
 id="x1-400005"></a>Methods</h3>
<a 
 id="x1-40001r79"></a>
<h3 class="sectionHead"><span class="titlemark">3.6 </span> <a 
 id="x1-410006"></a>Supplemental Text</h3>
                                                                                                    
                                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;4</span><br /><a 
 id="x1-420004"></a>MolCel2013</h2>
<a 
 id="x1-42001r80"></a>
<h3 class="sectionHead"><span class="titlemark">4.1 </span> <a 
 id="x1-430001"></a>INTRODUCTION</h3>
<!--l. 12--><p class="noindent" >P-element induced wimpy testis (PIWI)-interacting RNAs (piRNAs) can be distinguished from other
animal small silencing RNAs by their longer length (typically 23&#8211;35 nt), 2<span 
class="cmsy-8">&#x2032;</span>-O-methyl-modified
3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; termini, and association with PIWI proteins, a distinct subgroup of Argonaute proteins, the
small RNA-guided proteins responsible for RNA interference and related pathways [<a 
href="#XCenik2011">Cenik and
Zamore</a>,&#x00A0;<a 
href="#XCenik2011">2011</a>,&#x00A0;<a 
href="#XFarazi2008">Farazi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFarazi2008">2008</a>,&#x00A0;<a 
href="#XKim2009">Kim et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKim2009">2009</a>,&#x00A0;<a 
href="#XKumar1998">Kumar and Carmichael</a>,&#x00A0;<a 
href="#XKumar1998">1998</a>,&#x00A0;<a 
href="#XThomson2009">Thomson and
Lin</a>,&#x00A0;<a 
href="#XThomson2009">2009</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. piRNA production does not require Dicer, the double-stranded RNA endonuclease that
makes microRNAs (miRNAs) and small interfering RNAs (siRNAs), and piRNAs are thought to
derive from single-stranded rather than double-stranded RNA [<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>,&#x00A0;<a 
href="#XVagin2006">Vagin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2006">2006</a>].
<!--l. 14--><p class="noindent" >In most bilateral animals, germline piRNAs protect the genome from transposon activation, but also have
other functions [<a 
href="#XAravin2008a">Aravin and Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XAravin2007a">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XAravin2001">2001</a>,&#x00A0;<a 
href="#XAshe2012">Ashe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAshe2012">2012</a>,&#x00A0;<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>,&#x00A0;<a 
href="#XHartig2007">Hartig et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHartig2007">2007</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">2008</a>,&#x00A0;<a 
href="#XLee2012">Lee
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLee2012">2012</a>,&#x00A0;<a 
href="#XShirayama2012">Shirayama et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShirayama2012">2012</a>,&#x00A0;<a 
href="#XVagin2004">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2004">2004</a>]. A few days after birth, the majority of piRNAs in
the mouse testis are pre-pachytene piRNAs; 25% of these piRNA species map to more than one location in
the genome. A second class of piRNAs, typically derived from intergenic regions, has been reported to
emerge in the mouse testis 14.5 days postpartum (dpp), when the developing spermatocytes synchronously
enter the pachytene phase of meiotic prophase I. These pachytene piRNAs compose &#x003E;95% of piRNAs in
the adult mouse testis. Loss of genes required to make pachytene piRNAs blocks production of
mature sperm [<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>,&#x00A0;<a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2011">2011</a>,&#x00A0;<a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVourekas2012">2012</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. What
triggers the accumulation of pachytene piRNAs when spermatocytes enter the pachynema is
unknown.
<!--l. 17--><p class="noindent" >In Caenorhabditis elegans, each piRNA is processed from its own short RNA polymerase II
(Pol II) transcript [<a 
href="#XGu2012">Gu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGu2012">2012</a>]. In contrast, insect and mouse piRNAs are thought to be
processed from long RNAs transcribed from large piRNA loci. Supporting this view, a transposon
inserted into the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of the flamenco piRNA cluster in flies reduces the production of flamenco
piRNAs 168 kbp 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; to the insertion, suggesting that it disrupts transcription of the entire locus
[<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>]. High-throughput sequencing and chromatin immunoprecipitation (ChIP) has
been used to define the genomic structure of the piRNA-producing genes of immortalized,
cultured silk moth BmN4 cells [<span 
class="phvb8t-x-x-120">?</span>]. However, for flies and mice, we do not know the structure
of piRNA-producing genes, their transcripts, or the nature of the promoters that control their
expression.
<!--l. 19--><p class="noindent" >Instead, piRNA loci have been defined as clusters: regions of the genome with a high density of mapping
piRNA sequences [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XGrivna2006">Grivna
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrivna2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>,&#x00A0;<a 
href="#XRo2007">Ro et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRo2007">2007</a>]. In reality, piRNA-producing loci correspond to discrete
transcription units that include both intergenic loci believed to encode no protein [<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XBrennecke2008">Brennecke and Malone</a>,&#x00A0;<a 
href="#XBrennecke2008">2008</a>,&#x00A0;<a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVourekas2012">2012</a>] and protein-coding genes that also
produce piRNAs [<a 
href="#XAravin2007">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007">2007b</a>,&#x00A0;<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>,&#x00A0;<a 
href="#XSaito2009">Saito et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSaito2009">2009</a>].
<!--l. 21--><p class="noindent" >We used high-throughput sequencing data to define the genes and transcripts that produce piRNAs in the
juvenile and adult mouse testis. Using these data, we identified the factor that initiates transcription of
pachytene piRNA genes: A-MYB (MYBL1), a spermatocyte protein that serves as a master regulator of
genes encoding proteins required for cell-cycle progression through the pachytene stage of meiosis
[<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XTrauth1994">Trauth et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrauth1994">1994</a>]. A-MYB also initiates transcription of the genes encoding
                                                                                                    
                                                                                                    
many piRNA biogenesis factors. The combined action of A-MYB at the promoters of genes
producing pachytene piRNA precursor transcripts and genes encoding piRNA biogenesis proteins
creates a coherent feedforward loop that triggers a &#x003E;6,000-fold increase in pachytene piRNA
abundance during the ~5 days between the early and late phases of the pachytene stage of
male meiosis. A-MYB also promotes its own transcription through a positive feedback loop.
The A-MYB-regulated feedforward loop is evolutionarily conserved: A-MYB is bound to the
promoters of both piRNA clusters and PIWIL1, TDRD1, and TDRD3 in the rooster (Gallus gallus)
testis.
<a 
 id="x1-43001r82"></a>
<h3 class="sectionHead"><span class="titlemark">4.2 </span> <a 
 id="x1-440002"></a>RESULTS</h3>
<a 
 id="x1-44001r76"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.1 </span> <a 
 id="x1-450001"></a>Defining piRNA-Producing Transcripts in the Mouse Testis</h4>
<!--l. 32--><p class="noindent" >To define the structure of piRNA-producing loci in the testis of wild-type adult mice, we assembled the
transcripts detected by three biological replicates of strand-specific, paired-end, rRNA-depleted, total RNA
sequencing (RNA-seq; Figure <a 
href="#x1-45001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>A). We mapped reads to the mouse genome using TopHat [<a 
href="#XTrapnell2009">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2009">2009</a>] and performed de novo transcriptome assembly using Trinity [<a 
href="#XGrabherr2011">Grabherr et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrabherr2011">2011</a>] to identify
unannotated exon-exon junctions. We used all mapped reads, including reads corresponding to
unannotated exon-exon junctions, to perform reference-based transcript assembly (Cufflinks; [<a 
href="#XTrapnell2010">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>].
<!--l. 36--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-45001r1"></a>
                                                                                                    
                                                                                                    
<!--l. 38--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.526x.png" alt="PIC" class="graphics" width="320.19624pt" height="444.66124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.526x.png" src="Figures/Chapter4/MolCel2013_Fig1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.1: </span><span  
class="content">piRNA Precursors are RNA Pol II Transcripts<br 
class="newline" />(A) Strategy to assemble the mouse testis transcriptome. Rectangles with rounded corners, input
or output data; rectangles, processes. Decisions are shown without boxing.(B) Aggregated data for
piRNA-producing transcripts (5% trimmed mean). Oxidized small RNA (&#x003E;23 nt) sequencing data
were used to detect piRNAs; transcript abundance was measured using total RNA depleted of rRNA
(RNA-seq). RNA Pol III data were from SRA001030. Dotted lines show the transcriptional start site
(Start) and site of polyadenylation (End). See also Figure <a 
href="#x1-45002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>. </span></div><!--tex4ht:label?: x1-45001r1 -->
                                                                                                    
                                                                                                    
<!--l. 45--><p class="noindent" ></div><hr class="endfigure">
<!--l. 48--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-45002r2"></a>
                                                                                                    
                                                                                                    
<!--l. 50--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.527x.png" alt="PIC" class="graphics" width="369.37999pt" height="504.88625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.527x.png" src="Figures/Chapter4/MolCel2013_FigS1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.2: </span><span  
class="content">(A) Positions of the 214 major piRNA-producing genes on the 19 autosomes of mice.
We detected no loci on the X or Y chromosomes. (B) Cumulative distributions for all piRNAs and
for uniquely mapping piRNAs comparing the piRNA loci defined by our methods and by previous
approaches [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. (C) Histogram of distances (in 200 bp bins) from
the annotated 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; or 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of a piRNA gene (this study) or cluster to the nearest peak of reads from
high-throughput sequencing for transcript 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; (CAGE-seq) or 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; (PAS-seq) ends, transcription start
sites (H3K4me3) or A-MYB binding. </span></div><!--tex4ht:label?: x1-45002r2 -->
                                                                                                    
                                                                                                    
<!--l. 56--><p class="noindent" ></div><hr class="endfigure">
<!--l. 59--><p class="noindent" >To identify the transcripts that produce piRNAs, we sequenced piRNAs from six developmental stages of
mouse testes (10.5 dpp, 12.5 dpp, 14.5 dpp, 17.5 dpp, 20.5 dpp, and adult) and mapped them to the
assembled transcripts. The first round of spermatogenesis proceeds synchronously among the tubules
of the testis: mouse testes at 10.5 dpp advance no further than the zygotene stage (staging
according to [<a 
href="#XNEBEL1961">NEBEL et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNEBEL1961">1961</a>]; 12.5 dpp to the early pachytene; 14.5 dpp to the middle
pachytene; 17.5 to the late pachytene; and 20.5 dpp to the round spermatid stage. For each
stage, we prepared two sequencing libraries: one comprising all small RNAs and one in which
oxidation was used to enrich for piRNAs by virtue of their 2<span 
class="cmsy-8">&#x2032;</span>-O-methyl-modified 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; termini [<a 
href="#XGhildiyal2008">Ghildiyal
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>].
<!--l. 61--><p class="noindent" >To qualify as a piRNA-producing transcript, an assembled RNA was required to produce either a sufficiently
high piRNA abundance (&#x003E;100 ppm; parts per million uniquely mapped reads) or density (&#x003E;100 rpkm; reads
per kilobase of transcript per million uniquely mapped reads). These criteria retained both long transcripts
producing an abundance of piRNAs and short transcripts generating many piRNAs per unit of length. To
refine the termini of each piRNA-producing transcript, we supplemented the RNA-seq data with
high-throughput sequencing of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of RNAs bearing an N(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)ppp(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)N cap structure (cap analysis
of gene expression; CAGE) and the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of transcripts preceding the poly(A) tail (polyadenylation site
sequencing; PAS-seq). The assembled piRNA-producing transcripts likely correspond to continuous RNAs
in vivo because the CAGE library used to annotate transcript 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends was constructed after
two rounds of poly(A) selection. Thus, the RNA molecules in the library derive from complete
transcripts extending from the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; cap to the poly(A) tail (Figure <a 
href="#x1-45001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>B). Conventional 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
RACE (rapid amplification of cDNA ends) analysis of piRNA-producing transcripts confirmed
the ends of 16 loci (data not shown). To provide additional confirmation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of each
piRNA-producing transcript, we also determined the locations of histone H3 bearing trimethylated lysine 4
(H3K4me3), a histone modification associated with RNA Pol II transcription start sites <a 
href="#XGuenther2007">Guenther
et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGuenther2007">2007</a>].
<a 
 id="x1-45003r84"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.2 </span> <a 
 id="x1-460002"></a>piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</h4>
<!--l. 68--><p class="noindent" >The presence of 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; caps and poly(A) tails and the binding of histone H3K4me3 to the genomic DNA
immediately upstream of the transcription start site of each piRNA locus suggest that piRNA transcripts are
produced by RNA pol II <a 
href="#x1-45001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>. Moreover, using antibodies to RNA pol II but not RNA pol III, ChIP-seq showed
a peak at the transcription start site as well as polymerase occupancy across the entire piRNA gene (Figure
<a 
href="#x1-45001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>B; [<a 
href="#XKutter2011">Kutter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKutter2011">2011</a>]. We conclude that piRNA transcripts are conventional RNA pol II transcripts
bearing 5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; caps and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; poly(A) tails.
<a 
 id="x1-46001r87"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.3 </span> <a 
 id="x1-470003"></a>A Transcript-based Set of piRNA Loci</h4>
<!--l. 75--><p class="noindent" >Our transcriptome assembly yielded 467 piRNA-producing transcripts that define 214 genomic loci (Figure
<a 
href="#x1-45002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>A and <span class="texthl"><span id="textcolor16">Table S1</span></span>). Among the ~2.2 million distinct piRNA species and ~8.8 million piRNA reads from the
adult mouse testis, the 214 genomic loci account for 95% of all piRNAs.
<!--l. 77--><p class="noindent" >Previous studies defined piRNA clusters based solely on small RNA sequencing data [<a 
href="#XAravin2007a">Aravin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. Our approach differs in that it (1) uses RNA-seq data,
whose greater read length facilitates the identification of introns, allowing us to define the architecture of
piRNA precursor transcripts and (2) uses CAGE, PAS-seq, and H3K4me3 ChIP-seq data to refine the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
                                                                                                    
                                                                                                    
and 3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; ends of the piRNA transcripts. Consequently, the piRNA loci presented here account for more
piRNAs using fewer genomic base pairs than those previously defined (Figures <a 
href="#x1-45002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>B and <a 
href="#x1-45002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>C;
[<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. Our piRNA-producing loci include 41 piRNA loci that
escaped previous detection [<a 
href="#XAravin2007a">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>], 37 of
which contain introns. The 41 loci account for 2% of piRNAs at 10.5 dpp and 0.36% in the adult
testis.
<a 
 id="x1-47001r88"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.4 </span> <a 
 id="x1-480004"></a>Three Classes of piRNAs During Post-Natal Spermatogenesis</h4>
<!--l. 84--><p class="noindent" >Mice produce three PIWI proteins: MIWI2 (PIWIL4), which binds piRNAs in perinatal testis [<a 
href="#XAravin2008a">Aravin and
Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>]; MILI (PIWIL2), which binds piRNAs at least until the round spermatid
stage of spermatogenesis [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">2004</a>]; and MIWI
(PIWILl), which is first produced during the pachytene stage of meiosis [<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>]. From 10.5 to
20.5 dpp, piRNA abundance increases and longer piRNAs appear, reflecting a switch from
MILI-bound piRNAs, which have a 26&#8211;27 nt modal length [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XAravin2008a">Aravin and
Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XMontgomery1998">Montgomery et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMontgomery1998">1998</a>,&#x00A0;<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>], to MIWI-bound piRNAs, which have a 30
nt modal length (Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>A; [<a 
href="#XReuter2009">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2009">2009</a>,&#x00A0;<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>]. This switch occurs at the
pachytene phase of meiosis. MILI-bound pre-pachytene piRNAs predominate before the onset of
pachynema; at the pachytene and round spermatid stages, most piRNAs are MIWI-bound pachytene
piRNAs.
<!--l. 86--><p class="noindent" >We used hierarchical clustering to analyze the change in piRNA abundance from 10.5 to 20.5 dpp for the
214 genes defined by our data (Figures <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>A and <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>A and <span class="texthl"><span id="textcolor17">Table S2</span></span>). Three types of piRNA-producing
genes were identified according to when their piRNAs first accumulate and how their expression changes
during spermatogenesis: 84 pre-pachytene, 100 pachytene, and 30 hybrid loci. At 10.5 dpp, the
earliest time we evaluated, 84 genes dominate piRNA production (median piRNA abundance per
gene = 16 rpkm; Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B). Nearly all (81 out of 84) were congruent with protein-coding
genes. The 84 pre-pachytene piRNA genes account for 13% of piRNAs at 10.5 dpp, but only
0.31% of piRNAs in the adult testis. Of the pre-pachytene piRNAs accounted for by the 84 loci,
15% derive from 31 piRNA-producing genes that, to our knowledge, have not previously been
described.
<!--l. 89--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-48001r3"></a>
                                                                                                    
                                                                                                    
<!--l. 91--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.528x.png" alt="PIC" class="graphics" width="370.38374pt" height="485.81499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.528x.png" src="Figures/Chapter4/MolCel2013_Fig2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.3: </span><span  
class="content">(A)  Normalized  piRNA  density  (rpkm)  for  each  piRNA-producing  gene  is  shown  as
a heatmap across the developmental stages. Hierarchical clustering divided the genes into three
classes. Arrowheads mark seven pachytene piRNA genes that were not classified as pachytene
according to the change in the abundance of their precursor RNAs from 10.5 to 17.5 dpp.(B) Top:
box plots present piRNA density per gene as spermatogenesis progresses (here and elsewhere,
pre-pachytene in yellow and pachytene in purple). Middle: expression of <span 
class="phvro8t-x-x-120">A-Myb</span>, <span 
class="phvro8t-x-x-120">B-Myb</span>, <span 
class="phvro8t-x-x-120">Mili</span>, and
<span 
class="phvro8t-x-x-120">Miwi </span>was measured by RNA-seq. Bottom: box plots present piRNA precursor expression per gene,
measured by RNA-seq, from 10.5 to 20.5 dpp. See also Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a> and <span class="texthl"><span id="textcolor18">Table S2</span></span>. </span></div><!--tex4ht:label?: x1-48001r3 -->
                                                                                                    
                                                                                                    
<!--l. 97--><p class="noindent" ></div><hr class="endfigure">
<!--l. 100--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-48002r4"></a>
                                                                                                    
                                                                                                    
<!--l. 102--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.529x.png" alt="PIC" class="graphics" width="408.52625pt" height="536.00249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.529x.png" src="Figures/Chapter4/MolCel2013_FigS2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.4: </span><span  
class="content">(A) As shown previously by others using lower temporal resolution, the modal length
of piRNAs increases  as  spermatogenesis  proceeds  to  more  advanced  stages.  (B)  Total  piRNA
rpkm abundance and piRNA transcript abundance per locus by class, from purified spermatogonia,
spermatocytes,  round  spermatids,  and  3  dpp  and  8  dpp  testis  [<a 
href="#XGan2011">Gan  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>].   (C)   Correlation   between   piRNA   abundance   per   locus   and   piRNA   precursor
transcription from 10.5 to 20.5 dpp. Throughout the Figures, gold indicates pre-pachytene and purple
indicates pachytene piRNA loci. </span></div><!--tex4ht:label?: x1-48002r4 -->
                                                                                                    
                                                                                                    
<!--l. 108--><p class="noindent" ></div><hr class="endfigure">
<!--l. 112--><p class="noindent" >A parallel analysis of piRNA precursor transcription using RNA-seq (&#x003E;100 nt) corroborated the classification
based on piRNA abundance; of the 100 piRNA genes classified as pachytene based on the developmental
expression profile of their piRNAs, 93 were grouped as pachytene according to the developmental
expression profile of their transcripts. Of these 93, 89 are intergenic. All 84 piRNA genes designated
pre-pachytene using piRNA data were classified as pre-pachytene according to their transcript
abundance.
<!--l. 114--><p class="noindent" >Despite their name, pre-pachytene piRNAs were readily detected in &#x003E;90% and ~95% pure pachytene
spermatocytes, as well as round spermatids (Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B; [<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. Transcript
abundance from the 84 pre-pachytene loci was high at 3 dpp (median abundance = 11 rpkm), higher by 8
dpp (18 rpkm), and lower in purified leptotene/zygotene spermatocytes (3.3 rpkm; <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B). Yet piRNA
precursor transcripts were readily detectable in purified pachytene spermatocytes at a level (4.6 rpkm)
comparable to that in purified leptotene/zygotene spermatocytes (Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B); [<a 
href="#XModzelewski2012">Modzelewski
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. From 10.5 to 20.5 dpp, the steady-state level of pre-pachytene piRNA precursor transcripts
remained constant (Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B).
<!--l. 116--><p class="noindent" >Finally, the abundance of pre-pachytene piRNA precursor transcripts was better correlated with
pre-pachytene piRNA abundance at 17.5 dpp (<span 
class="cmmi-12">&#x03C1; </span>= 0.47), when pachytene spermatocytes compose a larger
fraction of the testis, than at 10.5, 12.5, or 14.5 dpp (0.32 <span 
class="cmsy-10x-x-120">&#x2265; </span><span 
class="cmmi-12">&#x03C1; </span><span 
class="cmsy-10x-x-120">&#x2264; </span>0.40; Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>C). Our data suggest that
the pre-pachytene loci continue to be transcribed and processed into piRNAs long after spermatocytes
enter the pachytene stage of meiosis. Thus, the name pre-pachytene piRNA is a misnomer that should be
retained only for historical reasons.
<!--l. 118--><p class="noindent" >Hierarchical clustering identified 100 pachytene genes whose piRNAs emerge at 12.5 dpp,
2 days earlier than previously reported [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>]. Nearly all the pachytene genes
are intergenic (93 out of 100). piRNA expression from pachytene piRNA genes peaks at 17.5
dpp (Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B). Overall, the median abundance of piRNAs from these 100 loci increased
&#x003E;6,000-fold from 10.5 to 17.5 dpp. Transcripts from pachytene genes were low at 10.5 dpp (median
abundance = 0.15 rpkm) and increased 116-fold from 10.5 to 17.5 dpp. From 10.5 to 20.5 dpp, the
dynamics of pachytene piRNA abundance from each piRNA gene correlated with the increase in
abundance of its precursor transcripts (0.39 <span 
class="cmsy-10x-x-120">&#x2265; </span><span 
class="cmmi-12">&#x03C1; </span><span 
class="cmsy-10x-x-120">&#x2264; </span>0.63; <span 
class="cmmi-12">&#x03C1;value </span><span 
class="cmsy-10x-x-120">&#x2264; </span><span 
class="cmr-12">7</span><span 
class="cmmi-12">.</span><span 
class="cmr-12">3</span><span 
class="cmmi-12">x</span><span 
class="cmr-12">10 </span><span 
class="cmsy-10x-x-120">- </span><span 
class="cmr-12">5</span>; Figure <a 
href="#x1-48002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>C). The 100
pachytene genes account for 92% of piRNAs in the adult testis, making it unlikely that biologically
functional pachytene piRNAs originate from thousands of genomic loci [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>]. Figures
<span 
class="phvb8t-x-x-120">??</span>and <a 
href="#x1-48004r6">4.6<!--tex4ht:ref: fig:MolCelS3 --></a> provide examples of pachytene and pre-pachytene piRNA genes defined by our
data.
                                                                                                    
                                                                                                    
<!--l. 122--><p class="noindent" ><a 
 id="x1-48003r5"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<img 
src="RoyC_Umass_Thesis.v0.530x.png" alt="PIC" class="graphics" width="482.80374pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.530x.png" src="Figures/Chapter4/MolCel2013_Fig3.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.5: </span><span  
class="content">Previous cluster boundaries are from <a 
href="#XLau2006">Lau et&#x00A0;al.</a>&#x00A0;[<a 
href="#XLau2006">2006</a>] in gray and <a 
href="#XGirard2006">Girard et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGirard2006">2006</a>]
in dark gray). </span></div><!--tex4ht:label?: x1-48003r5 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
                                                                                                    
                                                                                                    
<!--l. 133--><p class="noindent" ><a 
 id="x1-48004r6"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<img 
src="RoyC_Umass_Thesis.v0.531x.png" alt="PIC" class="graphics" width="387.4475pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.531x.png" src="Figures/Chapter4/MolCel2013_FigS3.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.6: </span><span  
class="content">Previous cluster boundaries are from <a 
href="#XLau2006">Lau et&#x00A0;al.</a>&#x00A0;[<a 
href="#XLau2006">2006</a>] in gray and <a 
href="#XGirard2006">Girard et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGirard2006">2006</a>]
in dark gray). </span></div><!--tex4ht:label?: x1-48004r6 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
<!--l. 144--><p class="noindent" >Hierarchical clustering detected a third class, hybrid piRNAs, which derives from 30 genes with
characteristics of both pre-pachytene and pachytene piRNA loci. Like pre-pachytene, hybrid piRNAs
were detected at 10.5 dpp (median abundance = 3.7 rpkm) and in purified spermatogonia [<a 
href="#XGan2011">Gan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>]. Like pachytene piRNAs, hybrid piRNA abundance increased during the pachytene stage
of meiosis, but the increase was delayed until late (17.5 dpp) rather than early pachynema
(14.5 dpp). Overall, piRNAs from hybrid genes increased &#x003E;10-fold from 14.5 to 17.5 dpp. The
median abundance of piRNAs from hybrid piRNA genes ranged from 90&#8211;120 rpkm in purified
pachytene spermatocytes, &#x003E;20-fold greater than their median abundance in spermatogonia [<a 
href="#XGan2011">Gan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]. Moreover, hybrid piRNA precursor transcripts were readily
detected in purified pachytene spermatocytes (median abundance = 9.0 rpkm; [<a 
href="#XModzelewski2012">Modzelewski
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]).
<a 
 id="x1-48005r89"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.5 </span> <a 
 id="x1-490005"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</h4>
<!--l. 151--><p class="noindent" >The coordinated increase in pachytene piRNA precursor transcripts suggests their regulation by a common
transcription factor or factors. Among the 100 pachytene piRNA genes, 15 pairs (30 genes) are divergently
transcribed. The 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of the piRNA precursor RNAs from each pair are close in genomic distance
(median = 127 bp), suggesting that a shared promoter lies between the two transcription start
sites.
<!--l. 153--><p class="noindent" >We took advantage of the unique genomic organization of these 15 pairs of divergently transcribed piRNA
genes to search for sequence motifs common to their promoters. The MEME algorithm [<a 
href="#XBailey1994">Bailey and
Elkan</a>,&#x00A0;<a 
href="#XBailey1994">1994</a>] revealed a motif highly enriched in these bidirectional promoters (E = 8.3 x 10<sup><span 
class="cmr-8">12</span></sup>; Figure <a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A).
This motif matches the binding site of the Myb family of transcription factors (Figure <a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A; [<a 
href="#XGupta2007">Gupta
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGupta2007">2007</a>,&#x00A0;<a 
href="#XNewburger2009">Newburger and Bulyk</a>,&#x00A0;<a 
href="#XNewburger2009">2009</a>]. The Myb motif is not restricted to bidirectional promoters;
MEME identified the same motif using the promoters of all pachytene piRNA genes (E = 9.1 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">28</span></sup>; Figure
<a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B).
<!--l. 157--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-49001r7"></a>
                                                                                                    
                                                                                                    
<!--l. 159--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.532x.png" alt="PIC" class="graphics" width="393.46999pt" height="391.4625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.532x.png" src="Figures/Chapter4/MolCel2013_Fig4.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.7: </span><span  
class="content">(A) Top: MEME identified a sequence motif in the bidirectional promoters of the 15
pairs of divergently transcribed pachytene piRNA genes. E value computed by MEME measures
the statistical  significance  of  the  motif.  Middle:  Myb  motif  from  the  mouse  UniPROBE  database.
Bottom: MEME-reported motif for the top 500 (by peak score) A-MYB ChIP-seq peaks from adult
mouse testes.(B) A-MYB ChIP-seq data for the common promoter of the divergently transcribed
pachytene piRNA  genes  <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1  </span>and  <span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>.(C)  The  distance  from  the
annotated transcription start site (TSS) of each piRNA gene to the nearest A-MYB peak. See also
Figure <a 
href="#x1-49002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>. </span></div><!--tex4ht:label?: x1-49001r7 -->
                                                                                                    
                                                                                                    
<!--l. 165--><p class="noindent" ></div><hr class="endfigure">
<!--l. 168--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-49002r8"></a>
                                                                                                    
                                                                                                    
<!--l. 170--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.533x.png" alt="PIC" class="graphics" width="385.43999pt" height="454.69875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.533x.png" src="Figures/Chapter4/MolCel2013_FigS4.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.8: </span><span  
class="content">(A) A-MYB binds to the common promoter of divergently transcribed pachytene piRNA
loci <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and <span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>. The abundance of DNA fragments at the amplified
region relative to a control region (mean <span 
class="cmsy-10x-x-120"> </span>standard deviation; n = 3) was measured by qPCR (top).
The A-MYB ChIP-seq (red) and input (black) data for this pair of genes is presented as in Figure
<a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B. (B) ChIP-seq and qPCR were as in (A), but for the promoter region of <span 
class="phvro8t-x-x-120">Miwi </span>(Piwil1). Also shown
is the RefSeq gene model. Exons, black; introns, gray. </span></div><!--tex4ht:label?: x1-49002r8 -->
                                                                                                    
                                                                                                    
<!--l. 176--><p class="noindent" ></div><hr class="endfigure">
<!--l. 179--><p class="noindent" >The Myb transcription factor family is conserved among eukaryotes. Like other vertebrates, mice produce
three Myb proteins, A-MYB (MYBL1), B-MYB (MYBL2), and C-MYB (MYB), each with a distinct
tissue distribution [<a 
href="#XLatham1996">Latham et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLatham1996">1996</a>,&#x00A0;<a 
href="#XMettus1994">Mettus et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMettus1994">1994</a>,&#x00A0;<a 
href="#XOh1999">Oh and Reddy</a>,&#x00A0;<a 
href="#XOh1999">1999</a>,&#x00A0;<a 
href="#XTrauth1994">Trauth
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrauth1994">1994</a>]. Testes produce both A- and B-MYB proteins. Multiple lines of evidence implicate
A-MYB, rather than B-MYB, as a candidate for regulating pachytene piRNA transcription. First, the
expression of <span 
class="phvro8t-x-x-120">A-Myb </span>during spermatogenesis resembles that of pachytene piRNAs: <span 
class="phvro8t-x-x-120">A-Myb</span>
transcripts appear at ~12.5 dpp and peak at 17.5 dpp (Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B; [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>].
The expression of <span 
class="phvro8t-x-x-120">A-Myb </span>messenger RNA (mRNA) increases ~15-fold from 8 dpp to 19 dpp,
whereas <span 
class="phvro8t-x-x-120">B-Myb </span>mRNA expression remains constant and low during the same time frame and into
adulthood [<a 
href="#XHorvath2009">Horvath et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>]. Our RNA-seq data (Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B) corroborate these findings.
Indeed, in our RNA-seq analysis of adult testes, <span 
class="phvro8t-x-x-120">A-Myb </span>mRNA was 24-fold more abundant than
<span 
class="phvro8t-x-x-120">B-Myb</span>. Second, a testis-specific <span 
class="phvro8t-x-x-120">A-Myb </span>point-mutant allele, <span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>, which is caused by a
cytosine-to-adenine transversion that changes alanine 213 to glutamic acid, leads to meiotic arrest at the
pachytene stage with subtle defects in autosome synapsis; <span 
class="phvro8t-x-x-120">A-Myb </span>null mutant mice have defects in
multiple tissues, including the testis and the mammary gland [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XToscani1997">Toscani
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XToscani1997">1997</a>]. Third, our RNA-seq analysis of <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes shows that there is no significant
change in <span 
class="phvro8t-x-x-120">B-Myb </span>expression in the mutant, compared to the heterozygous controls, at 14.5
or 17.5 dpp. Finally, B-MYB protein is not detectable in pachytene spermatocytes [<a 
href="#XHorvath2009">Horvath
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>].
<!--l. 181--><p class="noindent" >To assess more directly the role of A-MYB in pachytene piRNA precursor transcription, we used anti-A-MYB
antibody to perform ChIP followed by high-throughput sequencing of the A-MYB-bound DNA. The
anti-A-MYB antibody is specific for A-MYB, and the peptide used to raise the antibody is not present in
B-MYB. The model-based analysis of ChIP-seq (MACS) algorithm [<a 
href="#XZhang2008">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>] reported 3,815
genomic regions with significant A-MYB binding (false discovery rate, FDR &#x003C; 10<sup><span 
class="cmr-8">25</span></sup>); we call these
regions A-MYB peaks or peaks. Among the 500 peaks with the lowest FDR values, 394 (80%)
contained at least one significant site (<span 
class="cmmi-12">&#x03C1; &#x003C; </span><span 
class="cmr-12">10</span><sup><span 
class="cmr-8">4</span></sup>) for the MYB binding motif (Figure <a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A). Figure
<a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B shows an example of such an A-MYB peak at the bidirectional promoter of the divergently
transcribed pair of pachytene piRNA genes <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and <span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>. A-MYB
occupancy of this genomic site was confirmed by ChIP and quantitative PCR (ChIP-qPCR) (Figure
<a 
href="#x1-49002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>A).
<!--l. 183--><p class="noindent" >The median distance from the transcription start site to the nearest A-MYB peak was ~43 bp for the 100
pachytene piRNA genes but &#x003E;66,000 bp for the 84 pre-pachytene genes (Figure <a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>C). Our data suggest
that during mouse spermatogenesis A-MYB binds to the promoters of both divergently and unidirectionally
transcribed pachytene piRNA genes.
<!--l. 185--><p class="noindent" >To test the idea that A-MYB promotes transcription of pachytene, but not pre-pachytene, piRNA genes, we
used RNA-seq to measure the abundance of RNA &#x003E; 100 nt long from the testes of <span 
class="phvro8t-x-x-120">A-Myb </span>point-mutant
(<span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span> ) mice and their heterozygous littermates (Figure <a 
href="#x1-50001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). Pachytene piRNA precursor
transcripts&#8212;both divergently and unidirectionally transcribed&#8212;were significantly depleted in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant
testes compared to the heterozygotes: the median decrease was 45-fold at 14.5 dpp (q = 1.1 x
10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">13</span> </sup> ) and 248-fold at 17.5 dpp (q = 3.9 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">23</span></sup>). The abundance of pre-pachytene piRNA
transcripts was not significantly changed (q <span 
class="cmsy-10x-x-120">&#x2265; </span>0.34). The binding of A-MYB to the promoters of
pachytene piRNA genes, together with the depletion of pachytene piRNA transcripts in the <span 
class="phvro8t-x-x-120">A-Myb</span>
mutant, further supports the view that A-MYB directly regulates transcription of pachytene piRNA
genes.
<a 
 id="x1-49003r94"></a>
                                                                                                    
                                                                                                    
<h4 class="subsectionHead"><span class="titlemark">4.2.6 </span> <a 
 id="x1-500006"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</h4>
<!--l. 192--><p class="noindent" >To test the consequences of the loss of piRNA precursor transcripts, we measured piRNA abundance in the
<span 
class="phvro8t-x-x-120">A-Myb </span>mutant. Like pachytene piRNA precursor transcription, pachytene piRNA abundance significantly
decreased in mutant testes. At 14.5 dpp, median piRNA abundance per pachytene gene decreased 87-fold
in <span 
class="phvro8t-x-x-120">A-Myb </span>homozygous mutant testes compared to heterozygotes (<span 
class="cmmi-12">&#x03C1; &#x003C; </span><span 
class="cmr-12">2</span><span 
class="cmmi-12">.</span><span 
class="cmr-12">2</span><span 
class="cmmi-12">X</span><span 
class="cmr-12">10</span><sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">16</span></sup>; Figure <a 
href="#x1-50001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>. By 17.5 dpp,
median pachytene piRNA abundance was &#x003E;9,000 times lower in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant than the heterozygotes
(P &#x003C; 2.2 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">16</span></sup>). In contrast, pre-pachytene piRNA levels were essentially unaltered. Figure 6
presents examples of the effect at 14.5 and 17.5 dpp of the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant on piRNA precursor
transcript and mature piRNA abundance for one pre-pachytene and three pachytene piRNA
genes.
<!--l. 195--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-50001r9"></a>
                                                                                                    
                                                                                                    
<!--l. 197--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.534x.png" alt="PIC" class="graphics" width="399.4925pt" height="415.55249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.534x.png" src="Figures/Chapter4/MolCel2013_Fig5.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.9: </span><span  
class="content">The change in transcript or piRNA abundance per gene in <span 
class="phvro8t-x-x-120">A-Myb </span>(n = 3) and <span 
class="phvro8t-x-x-120">Miwi </span>(n
= 1) mutants compared to heterozygotes in testes isolated at 14.5 and 17.5 dpp. See also Figure
<a 
href="#x1-50002r10">4.10<!--tex4ht:ref: fig:MolCelS5 --></a>. </span></div><!--tex4ht:label?: x1-50001r9 -->
                                                                                                    
                                                                                                    
<!--l. 203--><p class="noindent" ></div><hr class="endfigure">
<!--l. 206--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-50002r10"></a>
                                                                                                    
                                                                                                    
<!--l. 208--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.535x.png" alt="PIC" class="graphics" width="308.15125pt" height="254.9525pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.535x.png" src="Figures/Chapter4/MolCel2013_FigS5.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.10: </span><span  
class="content">Change in piRNA abundance per locus (rpkm) for <span 
class="phvro8t-x-x-120">Spo11 </span>(14.5 dpp), <span 
class="phvro8t-x-x-120">Miwi </span>(<span 
class="phvro8t-x-x-120">Piwil2</span>; 10.5
dpp), <span 
class="phvro8t-x-x-120">Tdrd6 </span>(18 dpp), and <span 
class="phvro8t-x-x-120">Tdrd9 </span>(14 dpp) mutants compared to heterozygous controls. </span></div><!--tex4ht:label?: x1-50002r10 -->
                                                                                                    
                                                                                                    
<!--l. 214--><p class="noindent" ></div><hr class="endfigure">
<!--l. 217--><p class="noindent" >Our data show that A-MYB binds to the promoters of pachytene piRNA genes; <span 
class="phvro8t-x-x-120">A-Myb</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, and pachytene
piRNA precursor transcription begins at 12.5 dpp; and <span 
class="phvro8t-x-x-120">A-Myb </span>mutant spermatocytes reach pachynema
with subtle defects in autosome synapsis [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Could pachytene piRNA
depletion nonetheless be an indirect consequence of the meiotic arrest caused by the <span 
class="phvro8t-x-x-120">A-Myb</span>
mutant? To test this possibility, we sequenced small RNAs from <span 
class="phvro8t-x-x-120">Spo11 </span>mutant testes, which
failed to generate double-stranded DNA breaks at the leptotene stage and display a meiotic
arrest [<a 
href="#XRomanienko2000">Romanienko and Camerini-Otero</a>,&#x00A0;<a 
href="#XRomanienko2000">2000</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. The median abundance of piRNAs from
pre-pachytene genes did not decrease at 14.5 dpp. By 17.5 dpp, piRNA from pachytene genes
decreased just 5.9-fold in the <span 
class="phvro8t-x-x-120">Spo11 </span>mutant testes compared to the heterozygotes (Figure
<a 
href="#x1-50002r10">4.10<!--tex4ht:ref: fig:MolCelS5 --></a>). We note that A-MYB protein abundance is reduced in the <span 
class="phvro8t-x-x-120">Spo11 </span>mutant [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>].
<!--l. 219--><p class="noindent" ><span 
class="phvro8t-x-x-120">Trip13 </span>is required to complete the repair of double-strand DNA breaks on fully synapsed chromosomes.
<span 
class="phvro8t-x-x-120">Trip13 </span>mutants display a meiotic arrest similar to that in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes [<a 
href="#XLi2007">Li and Schimenti</a>,&#x00A0;<a 
href="#XLi2007">2007</a>]:
pachytene arrest with synapsed chromosomes. To further test whether the loss of pachytene piRNA
precursor transcripts in <span 
class="phvro8t-x-x-120">A-Myb </span>mutants reflects a general effect of meiotic arrest, we measured piRNA
precursor transcript abundance in <span 
class="phvro8t-x-x-120">Trip13 </span>mutant testes at 17.5 dpp. Unlike <span 
class="phvro8t-x-x-120">A-Myb</span>, piRNA precursor
transcripts were readily detectable in the <span 
class="phvro8t-x-x-120">Trip13 </span>mutant (Figure <a 
href="#x1-51002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>). We conclude that the loss of
pachytene piRNA precursor transcripts and piRNAs in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes is a direct consequence of the
requirement for A-MYB to transcribe pachytene piRNA genes and not a general feature of meiotic arrest at
the pachytene stage.
<a 
 id="x1-50003r97"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.7 </span> <a 
 id="x1-510007"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</h4>
<!--l. 226--><p class="noindent" >The <span 
class="phvro8t-x-x-120">A-Myb </span>mutant more strongly affected pachytene piRNA accumulation than it did the steady-state
abundance of the corresponding piRNA precursor transcripts (Figure <a 
href="#x1-50001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>; the median decrease in
pachytene piRNA abundance was 2-fold greater at 14.5 dpp and 38-fold greater at 17.5 dpp than the
decrease in the steady-state abundance of pachytene precursor transcripts <span class="texthl"><span id="textcolor19">(Table S1)</span></span>. These data suggest
that A-MYB exerts a layer of control on piRNA accumulation beyond its role in promoting pachytene piRNA
precursor transcription.
<!--l. 228--><p class="noindent" ><span 
class="phvro8t-x-x-120">Miwi </span>has previously been proposed to be a direct target of A-MYB; <span 
class="phvro8t-x-x-120">Miwi </span>mRNA abundance is reduced in
A-MYB mutant testes, and ChIP microarray data place A-MYB on the <span 
class="phvro8t-x-x-120">Miwi </span>promoter [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Our RNA-seq data confirm that accumulation of <span 
class="phvro8t-x-x-120">Miwi </span>mRNA requires A-MYB: <span 
class="phvro8t-x-x-120">Miwi </span>mRNA
decreased more than 50-fold in testes isolated from <span 
class="phvro8t-x-x-120">A-Myb </span>mutant mice at 14.5 dpp compared to their
heterozygous littermates (Figures <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>A and <a 
href="#x1-52002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a> and <span class="texthl"><span id="textcolor20">Table S3</span></span>). Furthermore, our ChIP data confirm that
A-MYB binds the <span 
class="phvro8t-x-x-120">Miwi </span>promoter in vivo (Figures <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B, <a 
href="#x1-49002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>B, and <a 
href="#x1-52002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a>). Like pachytene piRNAs,
<span 
class="phvro8t-x-x-120">Miwi </span>transcripts first appear at 12.5 dpp (Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B), and MIWI protein is first detected in
testes at 14.5 dpp [<span 
class="phvb8t-x-x-120">?</span>]. Loss of MIWI arrests spermatogenesis at the round spermatid stage
[<span 
class="phvb8t-x-x-120">?</span>].
<!--l. 232--><p class="noindent" >A previous study reported that piRNAs fail to accumulate to wild-type levels in <span 
class="phvro8t-x-x-120">Miwi </span>mutant testes [<span 
class="phvb8t-x-x-120">?</span>].
However, our data suggest that the overall change in piRNA abundance caused by loss of MIWI is quite
small: RNA-seq detected no change at 14.5 dpp (change in total piRNA abundance = 1.1; n = 2) and only a
modest decrease at 17.5 dpp (change in total piRNA abundance = 0.58; n = 1). piRNAs from pachytene loci
decreased just 2.7-fold at 14.5 dpp (p = 0.0046) and 3.5-fold at 17.5 dpp (p = 1.8 x 10-6) in <span 
class="phvro8t-x-x-120">Miwi </span>mutant
testes (Figure <a 
href="#x1-50001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). By comparison, pachytene piRNAs declined 87-fold at 14.5 dpp and 9,400-fold at 17.5
                                                                                                    
                                                                                                    
dpp in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant.
<!--l. 234--><p class="noindent" >Does the loss of MIWI affect piRNA precursor transcription? We measured transcript abundance and piRNA
expression in <span 
class="phvro8t-x-x-120">Miwi </span>null mutant testes at 14.5 and 17.5 dpp. In <span 
class="phvro8t-x-x-120">Miwi</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span></sup> testes, pachytene piRNA precursor
transcripts were present at levels indistinguishable from <span 
class="phvro8t-x-x-120">Miwi </span>heterozygotes (median change = 1.0- to
1.4-fold; q = 1; Figure <a 
href="#x1-50001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). Thus, loss of MIWI does not explain loss of pachytene piRNA precursor
transcripts in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes.
<!--l. 237--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-51001r11"></a>
                                                                                                    
                                                                                                    
<!--l. 239--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.536x.png" alt="PIC" class="graphics" width="396.48125pt" height="345.29pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.536x.png" src="Figures/Chapter4/MolCel2013_Fig6.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.11: </span><span  
class="content">Transcript and piRNA abundance in heterozygous (Het) and homozygous <span 
class="phvro8t-x-x-120">A-Myb </span>(Mut)
point-mutant testes is shown for four illustrative examples at 14.5 and 17.5 dpp. Also shown is the
abundance of piRNA sequencing reads that map to the exon-exon junctions. Gene <span 
class="phvro8t-x-x-120">11-qE1-9443</span>
does not have an intron. Exons, blue boxes; splice junctions, gaps; the last exon is compressed and
not to scale. See also Figure <a 
href="#x1-51002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>. </span></div><!--tex4ht:label?: x1-51001r11 -->
                                                                                                    
                                                                                                    
<!--l. 245--><p class="noindent" ></div><hr class="endfigure">
<!--l. 249--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-51002r12"></a>
                                                                                                    
                                                                                                    
<!--l. 251--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.537x.png" alt="PIC" class="graphics" width="342.27875pt" height="357.33499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.537x.png" src="Figures/Chapter4/MolCel2013_FigS6.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.12: </span><span  
class="content">Transcripts were detected in total RNA from adult testes by RT-PCR (using random
primers) for five pachytene piRNA loci as well as <span 
class="phvro8t-x-x-120">Miwi </span>and <span 
class="phvro8t-x-x-120">Actin</span>. Mut, mutant; Het or H, heterozygote;
wt, wild type. </span></div><!--tex4ht:label?: x1-51002r12 -->
                                                                                                    
                                                                                                    
<!--l. 257--><p class="noindent" ></div><hr class="endfigure">
<!--l. 260--><p class="noindent" >In addition to <span 
class="phvro8t-x-x-120">Miwi</span>, ChIP-seq detected A-MYB bound to the promoters of 12 other RNA-silencing-pathway
genes (Figure <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B and <span class="texthl"><span id="textcolor21">Table S3</span></span>). Of these, the mRNA abundance&#8212;measured by three biologically
independent RNA-seq experiments&#8212;of <span 
class="phvro8t-x-x-120">Ago2</span>, <span 
class="phvro8t-x-x-120">Ddx39 </span>(uap56 in flies), <span 
class="phvro8t-x-x-120">Mael</span>, <span 
class="phvro8t-x-x-120">Mili</span>, <span 
class="phvro8t-x-x-120">Mov10l1</span>, <span 
class="phvro8t-x-x-120">Tdrd9</span>, and <span 
class="phvro8t-x-x-120">Vasa</span>
did not change significantly at 14.5 dpp in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes compared to heterozygotes (q &#x003E; 0.05);
except for <span 
class="phvro8t-x-x-120">Ago2</span>, all decreased significantly in the mutant at 17.5 dpp. In contrast, the abundance of the
mRNAs encoding Tudor domain proteins decreased significantly in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes: <span 
class="phvro8t-x-x-120">Tdrd6 </span>(64-fold
decrease; q = 3.1 x 10-5) and <span 
class="phvro8t-x-x-120">Tdrd5 </span>(7.5-fold decrease; q = 1.0 x 10-5). <span 
class="phvro8t-x-x-120">Tdrd5 </span>is expressed in embryonic
testes then decreases around birth [<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. <span 
class="phvro8t-x-x-120">TDRD5 </span>protein reappears at 12 dpp, increasing
throughout the pachynema [<a 
href="#XSmith2004">Smith et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSmith2004">2004</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. Our data indicate that A-MYB
activates <span 
class="phvro8t-x-x-120">Tdrd5 </span>transcription at the onset of the pachytene stage of meiosis. Similarly, <span 
class="phvro8t-x-x-120">Tdrd6</span>
mRNA can be detected at the middle pachytene, but not the zygotene stage, and peaks after late
pachytene; TDRD6 protein can be detected at 17 dpp and continues to increase until 21 dpp
[<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>]. The findings that TDRD5 and TDRD6 colocalize with MIWI in pachytene
spermatocytes [<a 
href="#XHosokawa2007">Hosokawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHosokawa2007">2007</a>,&#x00A0;<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>] and that TDRD6
binds MIWI [<a 
href="#XChen2009a">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2009a">2009</a>,&#x00A0;<a 
href="#XVagin2009">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>,&#x00A0;<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>] suggest a role for
these Tudor domain proteins in pachytene piRNA production or function. As in <span 
class="phvro8t-x-x-120">Miwi</span>-/- testes,
spermatogenesis arrests at the round spermatid stage in <span 
class="phvro8t-x-x-120">Tdrd5</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span></sup> and <span 
class="phvro8t-x-x-120">Tdrd6</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span></sup> mutant testes
[<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. Loss of <span 
class="phvro8t-x-x-120">Tdrd6 </span>expression has little effect on piRNA
levels (Figure <a 
href="#x1-48004r6">4.6<!--tex4ht:ref: fig:MolCelS3 --></a>; [<a 
href="#XVagin2009">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>], perhaps because the functions of Tudor domain proteins
overlap.
<!--l. 262--><p class="noindent" >Other genes encoding piRNA pathway proteins whose promoters are bound by A-MYB and whose
expression decreased significantly in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes include <span 
class="phvro8t-x-x-120">MitoPld </span>(<span 
class="phvro8t-x-x-120">Pld6</span>; 3.9-fold decrease; q =
0.0095) and <span 
class="phvro8t-x-x-120">Tdrd12 </span>(5.3-fold decrease; q = 0.0046). <span 
class="phvro8t-x-x-120">MitoPld </span>encodes an endoribonuclease implicated in an
early step in piRNA biogenesis in mice and flies [<a 
href="#XHaase2010">Haase et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHaase2010">2010</a>,&#x00A0;<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>,&#x00A0;<a 
href="#XHuang2011">Huang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHuang2011">2011</a>,&#x00A0;<a 
href="#XIpsaro2012">Ipsaro et&#x00A0;al.</a>,&#x00A0;<a 
href="#XIpsaro2012">2012</a>,&#x00A0;<a 
href="#XNishimasu2012">Nishimasu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNishimasu2012">2012</a>,&#x00A0;<a 
href="#XPane2007">Pane et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPane2007">2007</a>,&#x00A0;<a 
href="#XWatanabe2011a">Watanabe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatanabe2011a">2011</a>].
The function of Tdrd12 is not known, but its fly homologs (Yb, Brother of Yb, and Sister of Yb) are all
required for piRNA production [<a 
href="#XHandler2011">Handler et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHandler2011">2011</a>]. <span 
class="phvro8t-x-x-120">Tdrd1 </span>decreased 3.4-fold, but with q value = 0.015.
<span 
class="phvro8t-x-x-120">Tdrd1 </span>is first expressed in fetal prospermatogonia, then re-expressed in pachytene spermatocytes [<a 
href="#XChuma2006a">Chuma
and Hosokawa</a>,&#x00A0;<a 
href="#XChuma2006a">2006</a>]. In Tdrd1 mutant testes, spermatogenesis fails, with no spermatocytes progressing
past the round spermatid stage [<a 
href="#XChuma2006a">Chuma and Hosokawa</a>,&#x00A0;<a 
href="#XChuma2006a">2006</a>]. TDRD1 binds MILI and MIWI [<a 
href="#XChen2009a">Chen
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2009a">2009</a>,&#x00A0;<a 
href="#XKojima2009">Kojima et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKojima2009">2009</a>] and colocalizes with TDRD5 and TDRD6 in the chromatoid body
[<a 
href="#XHosokawa2007">Hosokawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHosokawa2007">2007</a>].
<!--l. 264--><p class="noindent" >Together, these data support the idea that at the onset of the pachytene phase of meiosis, A-MYB
coordinately activates transcription of many genes encoding piRNA pathway proteins.
<a 
 id="x1-51003r100"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.8 </span> <a 
 id="x1-520008"></a>A-MYB and the Pachytene piRNA Regulatory Circuitry</h4>
<!--l. 271--><p class="noindent" >A number of genes encoding known and suspected piRNA pathway proteins are bound and regulated by
A-MYB (Figures <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B and <a 
href="#x1-52002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a>C). Our data support a model in which A-MYB drives both the transcription
of pachytene piRNA genes and the mRNAs encoding genes required for piRNA production including <span 
class="phvro8t-x-x-120">Miwi</span>,
<span 
class="phvro8t-x-x-120">MitoPld</span>, and <span 
class="phvro8t-x-x-120">Tdrd9</span>. Regulation by A-MYB of both the sources of pachytene piRNAs and the piRNA
biogenesis machinery creates a coherent feedforward loop (Figure <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>C). Feedforward loops amplify
initiating signals to increase target gene expression. Furthermore, they function as switches that
are sensitive to sustained signals; they reject transient signals [<a 
href="#XOsella2011">Osella et&#x00A0;al.</a>,&#x00A0;<a 
href="#XOsella2011">2011</a>,&#x00A0;<a 
href="#XShen-Orr2002">Shen-Orr
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShen-Orr2002">2002</a>].
                                                                                                    
                                                                                                    
<!--l. 273--><p class="noindent" >A-MYB also bound to the <span 
class="phvro8t-x-x-120">A-Myb </span>promoter (Figure <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B), and <span 
class="phvro8t-x-x-120">A-Myb </span>transcripts decreased 4.2-fold in
testes from an <span 
class="phvro8t-x-x-120">A-Myb </span>point mutant (<span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>; Figure <a 
href="#x1-52001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B). The <span 
class="phvro8t-x-x-120">A-Myb </span>mutant fails to produce the high
level of A-MYB protein observed in wild-type testes at the late pachytene stage of meiosis [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Instead, A-MYB protein never becomes more abundant than the level achieved in wild-type
testes by the beginning of the pachytene stage. While the lower level of A-MYB in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant may
reflect instability of the mutant protein, a simpler explanation is that mutant A-MYB cannot activate <span 
class="phvro8t-x-x-120">A-Myb</span>
transcription.
<!--l. 276--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-52001r13"></a>
                                                                                                    
                                                                                                    
<!--l. 278--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.538x.png" alt="PIC" class="graphics" width="415.55249pt" height="502.87874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.538x.png" src="Figures/Chapter4/MolCel2013_Fig7.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.13: </span><span  
class="content">(A) mRNA abundance in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant versus heterozygous testes. The 407 genes
with a significant (q &#x003C; 0.05) change in steady-state mRNA levels are shown as red circles. The 203
with A-MYB peaks within 500 bp of their transcription start site are filled. (B) A-MYB ChIP-seq signal
at the transcription start sites of <span 
class="phvro8t-x-x-120">A-Myb </span>and genes implicated in RNA silencing pathways. For each,
the figure reports the change in mRNA abundance between 17.5 and 10.5 dpp in wild-type testes
and the mean change between <span 
class="phvro8t-x-x-120">A-Myb </span>mutant and heterozygous testes at 14.5 dpp (mean <span 
class="cmsy-10x-x-120"> </span>SD; n
= 3). (C) A model for the regulation of pachytene piRNA biogenesis by A-MYB. See also Figure <a 
href="#x1-52002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a>
and <span class="texthl"><span id="textcolor22">Table S3</span></span>. </span></div><!--tex4ht:label?: x1-52001r13 -->
                                                                                                    
                                                                                                    
<!--l. 286--><p class="noindent" ></div><hr class="endfigure">
<!--l. 290--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-52002r14"></a>
                                                                                                    
                                                                                                    
<!--l. 292--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.539x.png" alt="PIC" class="graphics" width="373.39499pt" height="557.08124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.539x.png" src="Figures/Chapter4/MolCel2013_FigS7.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.14: </span><span  
class="content">A) mRNA abundance in 17.5 dpp <span 
class="phvro8t-x-x-120">A-Myb </span>versus heterozygous testes. The 2,853 genes
with a significant  (q  &#x003C;  0.051)  change  in  steady-state  mRNA  abundance  are  shown  as  open  red
circles. Among them, 8721,009 genes also had A-MYB peaks within 500 bp of their transcription
start sites. These &#8220;A-MYB targets&#8221; are marked with filled red circles. (B) Same as (A) but in 14.5 dpp
<span 
class="phvro8t-x-x-120">Miwi </span>mutant versus heterozygous testes. The genes encoding proteins implicated in RNA silencing
pathways that were labeled in (A) and that showed no change in expression in <span 
class="phvro8t-x-x-120">Miwi </span>mutant testes
are highlighted as green filled circles. As expected, <span 
class="phvro8t-x-x-120">Miwi</span>, showed a significant decrease in mRNA
abundance in <span 
class="phvro8t-x-x-120">Miwi</span>-/- testes. (C) The change in mRNA abundance (rpkm) in <span 
class="phvro8t-x-x-120">A-Myb </span>and <span 
class="phvro8t-x-x-120">Miwi </span>mutant
testes versus heterozygous controls for the RNA silencing genes highlighted in (A) and (B). </span></div><!--tex4ht:label?: x1-52002r14 -->
                                                                                                    
                                                                                                    
<!--l. 298--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-52003r103"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.9 </span> <a 
 id="x1-530009"></a>Feed-Forward Regulation of piRNA Production is Evolutionarily Conserved</h4>
<!--l. 306--><p class="noindent" >Is A-MYB-mediated, feedforward control a general feature of regulation of piRNA production among
vertebrates? To test whether A-MYB control of piRNA precursor transcription is evolutionarily conserved, we
used high-throughput sequencing to identify piRNAs in adult rooster testes. Birds and mammals diverged
330 million years ago [<a 
href="#XBenton2007">Benton and Donoghue</a>,&#x00A0;<a 
href="#XBenton2007">2007</a>]. After removing the sequences of identifiable miRNAs
[<a 
href="#XBurnside2008">Burnside et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBurnside2008">2008</a>] and annotated noncoding RNAs, total small RNA from the adult rooster
testis showed peaks at both 23 and 25 nt (Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>A). When the RNA was oxidized before
being prepared for sequencing, only a single 25 nt peak remained, consistent with the 25 nt
small RNAs corresponding to piRNAs containing 2<span 
class="cmsy-8">&#x2032;</span>-O-methyl-modified 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; termini. These longer,
oxidation-resistant species typically began with uracil (62% of species and 65% of reads; Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>B), and
we detected a significant Ping-Pong amplification signature (Z score = 31; Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>C). We
conclude that the oxidation-resistant, 24&#8211;30 nt long small RNAs correspond to rooster piRNAs.
Like piRNAs generally, rooster piRNAs are diverse, with 5,742,529 species present among
81,121,893 genome-mapping reads. Like mouse pachytene piRNAs, 70% of piRNAs from adult
rooster testes mapped to unannotated intergenic regions, 19% mapped to transposons, and
14% mapped to protein-coding genes. Of the piRNAs that map to protein-coding genes, &#x003E;95%
derive from introns. Forty-two percent of piRNA species mapped uniquely to the Gallus gallus
genome.
<!--l. 308--><p class="noindent" >Using 24&#8211;30 nt piRNAs from oxidized libraries, we identified 327 rooster piRNA clusters (Figure <a 
href="#x1-53002r16">4.16<!--tex4ht:ref: fig:MolCelS8 --></a>).
These account for 76% of all uniquely mapping piRNAs. Of the 327 clusters, 25 overlapped with
protein-coding genes. To begin to identify the transcription start sites for the rooster piRNA clusters, we
analyzed adult rooster testes by H3K4me3 ChIP-seq. More than 81% (268 out of 327) of the clusters
contained a readily detectable H3K4me3 peak within 1 kbp of the piRNA cluster. In contrast, the
median distance from a cluster to the nearest transcription start site of an annotated gene was 73
kbp, suggesting that the H3K4me3 peaks reflect the start sites for rooster piRNA precursor
transcripts.
<!--l. 310--><p class="noindent" >Next, we asked where in the genome A-MYB bound in adult rooster testes. A-MYB ChIP-seq identified
5,509 significant peaks (FDR &#x003C; 10-25). MEME analysis of the top 500 peaks with the lowest FDR values
identified a motif (E = 2.6 x 10-201; Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>D) similar to that found in the mouse (Figure <a 
href="#x1-49001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A). A-MYB is
the only one of the three chicken MYB genes expressed in adult testis (X.Z.L. and P.D.Z., unpublished data),
supporting the view that these peaks correspond to A-MYB binding. The core sequence motif associated
with A-MYB binding in mouse differs at one position (CAGTT) from that in rooster (C C/G GTT). This
difference between mammalian and chicken MYB proteins has been noted previously [<a 
href="#XDeng1996">Deng
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDeng1996">1996</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>].
<!--l. 313--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-53001r15"></a>
                                                                                                    
                                                                                                    
<!--l. 315--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.540x.png" alt="PIC" class="graphics" width="401.5pt" height="526.96875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.540x.png" src="Figures/Chapter4/MolCel2013_Fig8.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.15:  </span><span  
class="content">(A)  Length  distributions  of  total  rooster  testis  small  RNAs  (black)  and  miRNAs
(gray).(B) Sequence logo showing the nucleotide composition of piRNA reads and species.(C) The
5<span 
class="cmsy-8">&#x2032;</span> &#x00A0;-5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; overlap between piRNAs from opposite strands was analyzed to determine if rooster piRNAs
display Ping-Pong amplification. The number of pairs of piRNA reads at each position is reported. Z
score indicates that a significant 10 nt overlap (Ping-Pong) was detected. Z score &#x003E; 1.96 corresponds
to p value &#x003C; 0.05.(D) MEME-reported motif of the top 500 (by peak score) A-MYB ChIP-seq peaks
from adult rooster testes.(E) A-MYB, H3K4me3, and input ChIP-seq signals at the transcription start
sites of rooster PIWIL1, TDRD1, and TDRD3. See also Figure S8. </span></div><!--tex4ht:label?: x1-53001r15 -->
                                                                                                    
                                                                                                    
<!--l. 321--><p class="noindent" ></div><hr class="endfigure">
<!--l. 325--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-53002r16"></a>
                                                                                                    
                                                                                                    
<!--l. 327--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.541x.png" alt="PIC" class="graphics" width="407.52249pt" height="459.7175pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.541x.png" src="Figures/Chapter4/MolCel2013_FigS8.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.16:   </span><span  
class="content">Black   horizontal   lines   denote   the   locations   on   the   Gallus   gallus   (galGal3)
chromosomes of the piRNA clusters identified by small RNA sequencing. The figure shows 324
clusters; clusters on E64 (cluster 370) and E22C19W28_E50C23 (clusters 109 and 563) are not
shown. </span></div><!--tex4ht:label?: x1-53002r16 -->
                                                                                                    
                                                                                                    
<!--l. 333--><p class="noindent" ></div><hr class="endfigure">
<!--l. 336--><p class="noindent" >To determine whether chicken A-MYB might regulate transcription of some piRNA clusters in the testis, we
compared the A-MYB peak nearest to each piRNA cluster with the nearest H3K4me3 peak. Of the 327
rooster piRNA clusters, at least 104 were occupied by A-MYB at their promoters, as defined by an
overlapping H3K4me3 peak. These 104 clusters account for 31% of uniquely mapping rooster
piRNAs.
<!--l. 338--><p class="noindent" >The chicken genome encodes at least two PIWI proteins: PIWIL1 and PIWIL2. Remarkably, the promoter of
Gallus gallus PIWIL1, the homolog of mouse <span 
class="phvro8t-x-x-120">Miwi</span>, contained a prominent A-MYB peak (Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E).
TDRD1 and TDRD3 also showed A-MYB peaks (Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E). Thus, as in mice, Gallus gallus A-MYB
controls the transcription of both piRNA clusters and genes encoding piRNA pathway proteins. We conclude
that A-MYB-mediated feedforward regulation of piRNA production was likely present in the last common
ancestor of birds and mammals.
<!--l. 340--><p class="noindent" >In mice, we found no piRNA-producing genes on the sex chromosomes (Figure <a 
href="#x1-45002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>A), perhaps because
mouse sex chromosomes are silenced during the pachytene stage [<a 
href="#XLi2009d">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009d">2009a</a>]. Birds use a ZW rather
than an XY mechanism for sex determination, so roosters are homogametic (ZZ), allowing the sex
chromosomes to remain transcriptionally active in males [<a 
href="#XNamekawa2009">Namekawa and Lee</a>,&#x00A0;<a 
href="#XNamekawa2009">2009</a>,&#x00A0;<a 
href="#XSchoenmakers2009">Schoenmakers
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchoenmakers2009">2009</a>]. Indeed, we find that 39 of the 327 rooster piRNA clusters are on the Z chromosome,
accounting for 12% of uniquely mapping piRNAs (Figure <a 
href="#x1-53002r16">4.16<!--tex4ht:ref: fig:MolCelS8 --></a>). Of the 39 Z chromosome clusters, 18 had an
A-MYB peak at their promoter.
<a 
 id="x1-53003r83"></a>
<h3 class="sectionHead"><span class="titlemark">4.3 </span> <a 
 id="x1-540003"></a>DISCUSSION</h3>
<!--l. 347--><p class="noindent" >The data presented here provide strong support for the view that piRNAs in mammals begin as long,
single-stranded precursors generated by testis-specific, RNA Pol II transcription of individual piRNA genes
(see also <a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>&#x00A0;[<a 
href="#XVourekas2012">2012</a>]. Transcription by RNA Pol II affords piRNA genes the same rich set of
transcriptional controls available to regulate mRNA expression. Our data establish that developmentally
regulated transcription of piRNA genes determines when specific classes of piRNAs emerge during
spermatogenesis.
<!--l. 349--><p class="noindent" >During mouse spermatogenesis, transcription of pachytene piRNA genes begins at the onset of the
pachytene stage of meiosis; pachytene piRNAs accumulate subsequently. The presence of the
MYB binding motif near the transcription start sites of pachytene piRNA genes, the physical
binding of A-MYB to those genes, and the loss of pachytene piRNA precursor transcripts and
piRNAs in testes from <span 
class="phvro8t-x-x-120">A-Myb </span>mutant mice all argue that A-MYB regulates pachytene piRNA
production.
<!--l. 351--><p class="noindent" >A-MYB also drives increased expression of piRNA pathway genes. Among these, <span 
class="phvro8t-x-x-120">Miwi </span>expression shows
the greatest dependence on A-MYB, but A-MYB also drives transcription of genes encoding other proteins
in the piRNA pathway, including MitoPld, Mael, and five genes encoding Tudor domain proteins. For
example, A-MYB increases expression of Tdrd6 more than 500-fold. Loss of A-MYB function more strongly
depletes pachytene piRNAs than loss of MIWI, in part because pachytene piRNAs can still be loaded into
MILI in <span 
class="phvro8t-x-x-120">Miwi </span>mutant testes, although MILI-loaded pachytene piRNAs do not suffice to produce functional
sperm. In the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant, expression of mRNAs encoding multiple piRNA pathway proteins decreases.
We speculate that in wild-type male mice, the increased expression of these mRNAs at the
onset of the pachytene stage of meiosis ensures that sufficient piRNA-precursor-processing and
MIWI-loading factors are available to cope with the large increase in pachytene piRNA precursor
                                                                                                    
                                                                                                    
transcription.
<!--l. 353--><p class="noindent" >We propose that induction of A-MYB during the early pachytene stage of spermatogenesis initiates a
feedforward loop that ensures the precisely timed production of these piRNAs. Coherent feedforward loops
show delayed kinetics in order to reject background stimuli [<a 
href="#XMangan2003">Mangan and Alon</a>,&#x00A0;<a 
href="#XMangan2003">2003</a>]. Indeed, we observed
a delay from the early to middle pachytene in the accumulation of pachytene piRNAs, despite the continued
increase in <span 
class="phvro8t-x-x-120">A-Myb </span>expression (Figure <a 
href="#x1-48001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>A). Pachytene piRNA levels increase 75-fold (median for the 100
genes) from 10.5 to 12.5 dpp, coincident with increased expression of <span 
class="phvro8t-x-x-120">A-Myb</span>. However, from 12.5 to 14.5
dpp, pachytene piRNAs increase only 1.2-fold. Pachytene piRNAs subsequently resume their
accumulation, increasing 65-fold from 14.5 to 17.5 dpp. We believe this delay is a consequence of
a feedforward loop that ensures the production of pachytene piRNAs only at the pachytene
stage of spermatogenesis. Regulation by a feedforward loop also predicts a rapid shutdown of
pachytene piRNA pathways at round spermatid stage VIII, when A-MYB protein levels decrease
[<a 
href="#XHorvath2009">Horvath et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>]. Supporting this idea, the abundance of MIWI decreases sharply by the
elongated spermatid stage of spermatogenesis [<span 
class="phvb8t-x-x-120">?</span>]. Testing this proposal is a clear challenge for the
future.
<!--l. 355--><p class="noindent" >In fruit flies and zebrafish [<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>], most piRNAs map to repetitive
regions, whereas in mammals, uniquely mapping intergenic piRNAs predominate in the adult testis. The
discovery that 70% of rooster piRNA reads map to intergenic regions suggests that the expansion of
intergenic piRNAs controlled by A-MYB feedforward regulation arose before the divergence of birds and
mammals. In the future, detailed analysis of piRNA production across avian spermatogenesis should
provide insight into the evolutionary origins and functions of pachytene piRNAs, a class of piRNAs thus far
only detected in mammals.
<!--l. 357--><p class="noindent" >In summary, we have shown that mouse piRNA genes are coregulated transcriptionally, establishing that
A-MYB coordinately regulates the biogenesis of an entire piRNA class, the pachytene piRNAs. The
discovery that a loss-of-function <span 
class="phvro8t-x-x-120">A-Myb </span>mutant, <span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>, disrupts piRNA precursor transcription in
vertebrates provides a tool to understand the transformation of long, single-stranded piRNA precursors into
mature piRNAs and to explore the functions and targets of the pachytene piRNAs.
<a 
 id="x1-54001r109"></a>
<h3 class="sectionHead"><span class="titlemark">4.4 </span> <a 
 id="x1-550004"></a>EXPERIMENTAL PROCEDURES</h3>
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">Mice</span></dt><dd 
class="description">                                                                                              <br 
class="newline" /><span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>,  <span 
class="phvro8t-x-x-120">Spo11</span><sup><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Sky</span></sup>,  and  <span 
class="phvro8t-x-x-120">Piwil1</span><sup><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Hf</span></sup>  mice  were  maintained  and  used  according  to
the guidelines  of  the  Institutional  Animal  Care  and  Use  Committee  of  the  University  of
Massachusetts Medical School and genotyped as described [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Sequencing</span></dt><dd 
class="description">                                                                                      <br 
class="newline" />Small [<a 
href="#XGhildiyal2008">Ghildiyal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XSeitz2008">Seitz et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSeitz2008">2008</a>] and long RNA-seq [<a 
href="#XZhang2012a">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2012a">2012b</a>] and
analysis  [<a 
href="#XLi2009a">Li  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>]  were  as  described.  Reads  that  did  not  map  to  mouse  genome
mm9 were  mapped  to  piRNA  precursor  transcripts  to  obtain  splice  junction  mapping  small
RNAs. Total small RNA libraries from different developmental stages and from mutants were
normalized to the sum of all miRNA hairpin mapping reads. Oxidized samples were calibrated
to the corresponding total small RNA library via the abundance of shared, uniquely mapped
piRNA species.  piRNA  expression  data  were  grouped  with  Cluster  3.0.  Differential  gene
expression  was  analyzed  with  DESeq  R  [<a 
href="#XAnders2010a">Anders  and  Huber</a>,&#x00A0;<a 
href="#XAnders2010a">2010</a>];  ChIP-seq  reads  were
aligned to the genome using Bowtie version 0.12.7 [<a 
href="#XLangmead2009">Langmead et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangmead2009">2009</a>], and peaks were
                                                                                                    
                                                                                                    
identified using MACS [<a 
href="#XZhang2008">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Acknowledgments</span> </dt><dd 
class="description">                                                                                                                            <br 
class="newline" />We thank K. Chase and K. Schimenti for help collecting tissues; C. Tipping for help with mouse
husbandry;  P.  Johnson  and  B.  Keagle  for  providing  rooster  testes;  G.  Farley  for  technical
assistance; H. Lin for reagents; Xi Chen, Xiaotu Ma, Oliver Rando, and Benjamin Carone for
advice on ChIP; and members of our laboratories for critical comments on the manuscript.
X.Z.L. was supported by the Lalor Foundation and the Jane Coffin Childs Memorial Fund for
Medical Research.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Accession Numbers</span> </dt><dd 
class="description">                                                                                                                         <br 
class="newline" />The Gene  Expression  Omnibus  (GEO)  accession  number  for  the  RNA-seq,  ChIP-seq,  and
small RNA data reported in this paper is GSE44690.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Animals</span></dt><dd 
class="description">                                                                                          <br 
class="newline" />Mice were  maintained  and  used  according  to  the  guidelines  of  the  Institutional  Animal
Care and  Use  Committee  of  the  University  of  Massachusetts  Medical  School.  C57BL/6J
(Jackson Labs, Bar Harbor, ME, USA; stock number 664); <span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span> in a mixed 129X1/SvJ x
C57BL/6J background; Spo11tm1Sky in a C57BL/6J background; and Piwil<sup><span 
class="cmr-8">1</span><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Hf</span></sup> in a mixed
129X1/SvJ x C57BL/6J background (&#8220;<span 
class="phvro8t-x-x-120">Miwi</span>&#8221;) mice were genotyped as described [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. Rooster testes from White Leghorn of the Cornell Special C strain, about
15 months old, were used for small RNA analysis; and testes from the Brown Leghorn strain,
about one year old, were used for ChIP analysis.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">RNA Sequencing</span> </dt><dd 
class="description">                                                                                                                               <br 
class="newline" />Small                                                            RNA                                                            libraries
were constructed  and  sequenced  as  described  [<a 
href="#XGhildiyal2008">Ghildiyal  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XSeitz2008">Seitz  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSeitz2008">2008</a>]
except that  18&#8211;35  nt  RNA  was  isolated  and  2S  rRNA  depletion  omitted.  Sequencing  was
performed using either a Genome Analyzer GAII (36 or 76 nt reads) or HiSeq 2000 (50 nt)
instrument (Illumina, San Diego, CA, USA). We analyzed published small RNA libraries from
purified  mouse  spermatogonia  (SRR069809),  spermatocytes  (SRR069810,  GSE39652),  or
spermatids (SRR069811; [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]; from <span 
class="phvro8t-x-x-120">Mili  </span>mutant or
heterozygous  testes  at  10  dpp  (SRX003089  and  SRX003088;  [<a 
href="#XAravin2008a">Aravin  and  Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>];
from Tdrd6 mutant or heterozygous testes at 18 dpp (SRX012165 and SRX012166; [<a 
href="#XVagin2009">Vagin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>];  and  MILI  IP  samples  from  Tdrd9  mutant  or  heterozygous  testes  at  14  dpp
(SRX015795, SRX015796, SRX015797, and SRX015798; [<a 
href="#XShoji2009">Shoji et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShoji2009">2009</a>].
<!--l. 383--><p class="noindent" >Strand-specific  RNA-seq  libraries  [<a 
href="#XZhang2012">Zhang  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2012">2012a</a>]  using  Ribo-Zero  Gold  (Epicentre
Biotechnologies, Madison, WI, USA) were sequenced using the paired-end protocol on a HiSeq
2000.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd 
class="description">                                                                                                                          <br 
class="newline" />Small RNA  sequence  analysis  was  as  described  [<a 
href="#XLi2009a">Li  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>]  using  mouse  genome
release mm9 and chicken genome release galGal3. Non-coding RNA annotations comprised
data from ncRNAscan, the known tRNAs from UCSC, and 18S, 28S and 5.8S rRNAs. miRNA
hairpin and mature miRNA annotation was from miRBase Release 19. Mouse and chicken
transposons were annotated using Repeat Masker from UCSC. Reads that did not map to
                                                                                                    
                                                                                                    
the mouse  genome  (mm9)  were  mapped  to  piRNA  precursor  transcripts  to  obtain  splice
junction-mapping small RNAs. Total small RNA libraries from different developmental stages
and from mutants were normalized to the sum of all miRNA hairpin-mapping reads. Oxidized
samples were calibrated to the corresponding total small RNA library via the abundance of
shared, uniquely mapped piRNA species. <span class="texthl"><span id="textcolor23">Table S1</span></span> reports the statistics for high-throughput
sequencing. For oxidized (i.e., piRNA-enriched) samples, uniquely mapping small RNAs &#x003E;23
nt were mapped to each assembled piRNA precursor transcript and reported as reads per
kilobase pair per million reads mapped to the genome (rpkm) using a pseudo count of 0.001.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd 
class="description">                                                                                                                          <br 
class="newline" />RNA-seq  reads  were  aligned  to  the  genome  (NCBI  37/mm9)  using  TopHat  2.0.4  [<a 
href="#XTrapnell2009">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2009">2009</a>]. Reads were mapped uniquely using the &#8216;-g 1&#8217; switch. We assembled the mouse
testes transcriptome  (see  below).  For  genes  with  multiple  isoforms,  the  transcript  with  the
highest  average  rpkm  value  among  the  three  replicates  of  adult  testes  was  selected  for
further analysis. Fragments with both reads mapped within a transcript, or to piRNA precursor
transcripts, were counted using BEDTools [<a 
href="#XQuinlan2010">Quinlan and Hall</a>,&#x00A0;<a 
href="#XQuinlan2010">2010</a>].  The sum of the reads
aligning  to  the  top  quartile  of  expressed  transcripts  per  library  was  used  to  calibrate  the
samples.  The  number  of  reads  per  transcript  was  normalized  by  length,  divided  by  the
library-specific calibration factor, and reported as rpkm with a pseudo count of 0.001. Table
S1 presents the statistics for the RNA-seq data. Sequences mapping to five genes <span class="texthl"><span id="textcolor24">(Table S1)</span></span>
that overlapped with or were embedded within a piRNA gene were excluded when calculating
piRNA precursor transcript abundance.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">PAS-seq Library Construction and Analysis</span> </dt><dd 
class="description">                                                                                  <br 
class="newline" />PAS-seq libraries <span class="texthl"><span id="textcolor25">(Table S1)</span></span> were prepared essentially as described [<a 
href="#XShepard2011">Shepard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShepard2011">2011</a>]
and sequenced  using  a  HiSeq  2000  (100  nt  read  length).  We  first  removed  adaptors  and
performed  quality  control  using  Flexbar  2.2  (http://sourceforge.net/projects/theflexibleadap)
with the parameters &#8220;-at 3 -ao 10 &#8211;min-readlength 30 &#8211;max-uncalled 70 &#8211;phred-pre-trim 10.&#8221;
For reads beginning with GGG including (NGG, NNG or GNG) and ending with three or more
adenosines, we removed the first three nucleotides and mapped the remaining sequence with
and without  the  tailing  adenosines  to  the  mouse  genome  using  TopHat  2.0.4.  We  retained
only those reads that could be mapped to the genome without the trailing adenosine residues.
Genome-mapping reads containing trailing adenosines were regarded as potentially originating
from internal  priming  and  thus  discarded.  The  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  the  mapped,  retained  read  was
reported as the site of cleavage and polyadenylation.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">CAGE Library Construction and Analysis</span> </dt><dd 
class="description">                                                                                      <br 
class="newline" />CAGE (cap  analysis  of  gene  expression;  Table  S1)  was  as  described  [<a 
href="#XYang2011">Yang  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYang2011">2011</a>]
and sequenced using a HiSeq 2000 (100 nt reads). After removing adaptor sequences and
checking read quality using Flexbar 2.2 with the parameters of &#8220;-at 3 -ao 10 &#8211;min-readlength 20
&#8211;max-uncalled 70 &#8211;phred-pre-trim 10&#8221;, we retained only reads beginning with NG or GG (the
last two nucleotides on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; adaptor). We then removed the first two nucleotides and mapped
the sequences to the mouse genome using TopHat 2.0.4. All unique 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of the mapped
positions  were  considered  as  CAGE-tag  starting  sites  and  grouped  into  tag  clusters  using
a distance-based method in which the maximal distance between two neighboring tags was
required to be &#x003C;20 bp. The peak position of a tag cluster was then reported as the transcription
start site.
                                                                                                    
                                                                                                    
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Transcriptome Assembly and Annotation</span> </dt><dd 
class="description">                                                                                      <br 
class="newline" />De novo transcriptome assembly from three biological replicates of strand-specific RNA-seq
data from  adult  testes  was  performed  using  Trinity  (r2012-06-08)  with  default  parameters
[<a 
href="#XGrabherr2011">Grabherr et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrabherr2011">2011</a>]. The assembled RNA sequences were aligned to the mouse genome
(mm9) with BLAT [<a 
href="#XKent2002">Kent</a>,&#x00A0;<a 
href="#XKent2002">2002</a>], and the alignments with more than 95% of sequence length
mapped and fewer than 1% mismatches retained.
<!--l. 400--><p class="noindent" >We extracted novel junctions from Trinity (i.e., reads with [0-9]+M[0-9]+N[0-9]+M pattern in the
CIGAR string of SAM output), and re-mapped all RNA-seq reads to these junctions, rescuing
1,402,444 reads in three replicates. Rescued reads were combined with TopHat alignments
(supplied with "&#8211;max-multihits 100" to assembly through repetitive regions) and used as input
for reference-based assembly.
<!--l. 402--><p class="noindent" >We used    Cufflinks    v2.0.2    [<a 
href="#XTrapnell2010">Trapnell    et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>]    with    parameters    of    &#8220;-u    -j    0.2
&#8211;min-frags-per-transfrag 40&#8221; to assemble transcripts. To join small transcript fragments caused
by insufficient  read  coverage  or  embedded  repetitive  elements,  two  different  gap-joining
distance  cutoffs  were  used  for  the  assembly  of  genes  (&#8220;&#8211;overlap-radius  100&#8221;)  and  piRNA
loci (&#8220;&#8211;overlap-radius 250&#8221;). We used Cuffcompare v2.0.2 [<a 
href="#XTrapnell2010">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>] to annotate
the 49,840 Cufflinks-assembled transcripts using parameters optimized for genic conditions
(&#8220;&#8211;overlap-radius 100&#8221;).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">piRNA Precursor Transcript Annotation</span> </dt><dd 
class="description">                                                                                         <br 
class="newline" />We combined transcripts from the two Cufflink assemblies with those from the Trinity assembly,
producing 136,069 unique transcripts. Those transcripts with 100 ppm or 100 rpkm unique
mapping piRNAs at any time point (10.5, 12.5, 14.5, 17.5, 20.5 dpp and adult oxidized small
RNA from testis) were selected for manual annotation.
<!--l. 407--><p class="noindent" >To refine the termini of the piRNA-producing transcripts, we supplemented the RNA-seq data
with high-throughput  sequencing  of  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  ends  of  RNAs  bearing  (5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)ppp(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)  cap  structures
(CAGE)  and  of  the  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  ends  of  transcripts  flanking  the  poly(A)  tail  (PAS-seq).  To  provide
independent  confirmation  of  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  ends  of  each  piRNA-producing  transcript,  we  used
chromatin  immunoprecipitation  (ChIP-seq)  of  RNA  polymerase  II  (pol  II)  and  histone  H3
bearing trimethylated lysine-4 (H3K4me3). Refinement of transcriptional starts required both a
CAGE and a H3K4me3 peak to support the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of the transcript. When no H3K4me3 peak
corroborated alternative transcription start sites proposed by the CAGE data, the alternative
transcripts were merged with the fully substantiated transcript.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">piRNA Gene Nomenclature</span> </dt><dd 
class="description">                                                                                                              <br 
class="newline" />When piRNA-producing genes overlap an annotated protein coding gene, we refer to them
using the name of the overlapping gene preceded by &#8216;pi-&#8216;; when they do not, their names refer
to their genomic location followed by a number indicating the piRNA abundance in ppm at 6
weeks post-partum. The last digit of a piRNA gene name specifies the rank order of expression
among isoforms, determined by the highest abundance of transcripts (rpkm) observed for that
gene among the six developmental stages of testis.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Grouping piRNA Precursor Transcripts</span> </dt><dd 
class="description">                                                                                          <br 
class="newline" />For the most abundant transcript in each locus, the abundance (rpkm) of piRNAs at each stage
was expressed as a fraction of the maximum abundance reached during the developmental
                                                                                                    
                                                                                                    
time course. These data were then analyzed by hierarchical clustering according to Euclidean
distance and complete linkage using Cluster 3.0. Clustering results were visualized using Java
Tree View 1.1.3.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Analysis of Differential Gene Expression</span>  </dt><dd 
class="description">                                                                                      <br 
class="newline" />We determined differential gene expression using DESeq R [<a 
href="#XAnders2010a">Anders and Huber</a>,&#x00A0;<a 
href="#XAnders2010a">2010</a>]. For
each annotated mRNA, reads from each library were aligned to the most abundant assembled
transcript. Transcripts with q &#x003C; 0.05 were considered to be differentially expressed. <span class="texthl"><span id="textcolor26">Table S3</span></span>
lists the  genes  that  were  differentially  expressed  in  <span 
class="phvro8t-x-x-120">A-Myb  </span>at  14.5  dpp.  Three  biologically
independent replicates were used for <span 
class="phvro8t-x-x-120">A-Myb</span>homozygotes and heterozygotes at 14.5 and at
17.5 dpp.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Motif Discovery</span> </dt><dd 
class="description">                                                                                                                                 <br 
class="newline" />For divergently transcribed piRNA gene pairs, the promoter region was defined as the region
between  the  transcription  start  sites  defined  by  CAGE  peaks.  Sequence  motifs  in  these
putative  promoter  regions  were  detected  ab  initio  using  MEME  [<a 
href="#XBailey2009">Bailey  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBailey2009">2009</a>,&#x00A0;<a 
href="#XBailey1994">Bailey
and Elkan</a>,&#x00A0;<a 
href="#XBailey1994">1994</a>] in TCM mod (any number of repetitions per sequence) and compared to
existing JASPAR and TRANSFAC libraries via TOMTOM [<a 
href="#XGupta2007">Gupta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGupta2007">2007</a>]. FIMO was used
to detect motif sites within the putative promoters (default p &#x003C; 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">4</span></sup>; [<a 
href="#XGrant2011">Grant et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrant2011">2011</a>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Chromatin Immunoprecipitation (ChIP)</span> </dt><dd 
class="description">                                                                                          <br 
class="newline" />ChIP was  performed  as  described  [<a 
href="#XChen2008">Chen  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2008">2008</a>]  except  that  testes  were  macerated
on ice and then fixed with 1.5% (w/v) formaldehyde for 20 min. Samples were then further
crushed  using  20  strokes  with  a  &#8216;B&#8217;  pestle  in  a  Dounce  homogenizer  (Kimble-Chase,
Vineland,  NJ,  USA).  Chromatin  was  sheared  by  sonication  and  immunoprecipitated  using
anti-A-MYB  (HPA008791;  Sigma,  St.  Louis,  MO,  USA)  or  anti-H3K4me3  (ab8580;  Abcam,
Cambridge,  MA,  USA)  antibody;  immunoglobulin  G  (IgG;  Sigma,  item  2729)  served  as  a
control.  ChIP-quantitative  PCR  (qPCR)  was  performed  using  the  CFX96  Real-Time  PCR
Detection System with SsoFast EvaGreen Supermix (Bio-Rad, Hercules, CA, USA). Data were
analyzed using DART-PCR [<a 
href="#XPeirson2003">Peirson</a>,&#x00A0;<a 
href="#XPeirson2003">2003</a>]. Relative ChIP enrichment values were normalized
to <span 
class="phvro8t-x-x-120">MyoD1</span>,  a  gene  not  expressed  in  testes.  <span class="texthl"><span id="textcolor27">Table  S1</span></span>  lists  ChIP-qPCR  primers.  ChIP-seq
libraries for anti-A-MYB and control input DNA were prepared following the Illumina ChIP-seq
protocol and sequenced on a HiSeq 2000 (50 nt reads).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">ChIP-seq Analysis</span> </dt><dd 
class="description">                                                                                                                             <br 
class="newline" />ChIP-seq  reads  were  aligned  to  the  genome  using  Bowtie  version  0.12.7  [<a 
href="#XLangmead2009">Langmead
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangmead2009">2009</a>]. Reads were mapped uniquely using the &#8216;-M 1 &#8211;best &#8211;strata&#8217; switches and one
mismatch was allowed (-v 1). ChIP peaks were identified using MACS version 1.4.1 [<a 
href="#XZhang2008">Zhang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>]  using  default  arguments,  input  as  control,  and  a  cutoff  p-value  =  10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">25</span></sup>  was
used. BEDTools was used to assign peaks to the nearest 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of genes. <span class="texthl"><span id="textcolor28">Table S1</span></span> reports
sequencing statistics for ChIP-seq.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">RT-PCR</span></dt><dd 
class="description">                                                                                           <br 
class="newline" />Total RNA  was  treated  with  Turbo  DNase  (Ambion,  Austin,  TX,  USA),  and  then  reverse
transcribed  using  SuperScript  III  (Invitrogen,  Eugene,  OR,  USA)  with  random  primers
(Promega, Madison, WI, USA). The resulting cDNA was analyzed by conventional PCR. <span class="texthl"><span id="textcolor29">Table
                                                                                                    
                                                                                                    
S1</span></span> lists the primers used in Figure <a 
href="#x1-51002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Ping-Pong Analysis</span> </dt><dd 
class="description">                                                                                                                          <br 
class="newline" />Ping-Pong  amplification  was  analyzed  by  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;&#8211;5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  overlap  between  piRNA  pairs  from
opposite genomic strands [<a 
href="#XLi2009a">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>]. Overlap scores for each overlapping pair were
the product of the number of reads of each of the piRNAs from opposite strands. The overall
score for each overlap extend (1&#8211;30) was the sum of all such products for all chromosomes.
Heterogeneity at the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of small RNAs was neglected. Z-score for 10 bp overlap was
calculated using the scores of overlaps from 1&#8211;9 and 11&#8211;30 as background.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Rooster piRNA Cluster Detection</span> </dt><dd 
class="description">                                                                                                    <br 
class="newline" />We developed  a  dynamic  programming  algorithm  to  identify  the  genomic  regions  with  the
highest piRNA density. We used oxidized small RNA reads (&#x003E;23 nt) to detect clusters. We used
the conservative assumption that piRNA clusters compose at most 2% of the chicken genome.
We first split the genome into 1 kbp non-overlapping windows and computed piRNA abundance
for each window. The mean of the top 2% of windows was used as the penalty score for the
dynamic programming algorithm. The algorithm computes the cumulative piRNA abundance
score as a function of the window index along each chromosome. The score at a window is
the sum of the score in the previous window and the piRNA abundance in the current window,
minus the penalty score; if the resulting score was negative it was reset to 0. The maximal
score points to the largest piRNA cluster. We extracted the largest piRNA cluster, recomputed
the scores  at  the  corresponding  windows,  and  searched  for  the  next  cluster.  The  process
continued until the scores for all windows were zero. The boundaries of each cluster were
further refined by including those base pairs for which piRNA abundance exceeded the mean
piRNA abundance of the top 2% windows. We considered only those clusters with abundance
&#x003E;10 ppm for uniquely mapping piRNAs. In Figure <a 
href="#x1-53001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E, gene models were corrected using
data from our unpublished adult rooster testis RNA-seq data.
</dd></dl>
                                                                                                    
                                                                                                    
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;5</span><br /><a 
 id="x1-560005"></a>Discussion</h2>
<a 
 id="x1-56001r110"></a>
<h3 class="sectionHead"><span class="titlemark">5.1 </span> <a 
 id="x1-570001"></a>Future of Dynamic long RNAs</h3>
<!--l. 20--><p class="noindent" >Deep sequencing of transcriptomes has revolutionized biology. Previously, transcript discovery was a
cumbersome task. Transcript identification and characterization involved significant labor, cost, and
materials. In the mid-90&#8217;s, microarray technology [<a 
href="#XSchena1995a">Schena et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchena1995a">1995</a>] gave us a tantalizing glimpse into
how genes were expressed, but were limited to probed, and therefore known, sequences. However, the
green and red landscapes of a microarray analysis hinted at incredible complexity&#8212;a complexity that would
have to wait for technology to catch up.
<!--l. 22--><p class="noindent" >Like many transformative technologies, RNA-seq was made possible by incremental improvements to
numerous supportive technologies such as: 1) digital optics, 2) microscopy, 3) slide chemistry and on-slide
PCR and 4) nucleic-acid alignment. A HiSeq 2500 relies on all of these technologies (and others) to
produce the 100M+ sequences that allow Scientists to peer every day into the transcriptional output of a
genome.
<!--l. 24--><p class="noindent" >In the past 5 years, biologists have started to think way beyond mRNAs and small RNAs. The former
captured out interest for 30+ years [<a 
href="#XFuruichi1975">Furuichi</a>,&#x00A0;<a 
href="#XFuruichi1975">1975</a>,&#x00A0;<a 
href="#XWei1975">Wei et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWei1975">1975</a>], and the later has been on a
run-away trail since capturing out attention in 1998 [<a 
href="#XFire1998">Fire et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFire1998">1998</a>]. HTS has added long RNAs (among
others) to these classes of gene RNA products. However, many biologically-trained and minded Scientists
find themselves overwhelmed by the complete different methods and approaches to tackling the &#8217;big data&#8217;
created by modern genome-wide experiments. Experimental training does not currently provide
students with the required skills in statistics, computer programing, and experimental design
that are needed to work with genome-wide data. The richness of this data often leaves many
unasked (and unanswered) test-able hypothesis just sitting in public repositories [<a 
href="#XPlocik2013">Plocik and
Graveley</a>,&#x00A0;<a 
href="#XPlocik2013">2013</a>].
<!--l. 26--><p class="noindent" >Being such a novel area of extremely basic research, and to borrow a few seemingly inane but
rather insightful trio of phrases from the United States Secretary of Defense Donald Rumsfeld
[<a 
href="#XRumsfeld2011">Rumsfeld</a>,&#x00A0;<a 
href="#XRumsfeld2011">2011</a>], their remain at least three important areas of knowledge concerning long RNAs
of the transcriptome: <a 
href="#x1-580001">&#8221;The Known Knowns&#8221;</a>; <a 
href="#x1-590002">&#8221;The Known Unknowns&#8221;</a>; and the <a 
href="#x1-600003">&#8221;The Unknown
Unknowns&#8221;</a>.
<a 
 id="x1-57001r106"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.1 </span> <a 
 id="x1-580001"></a>The Known Knowns</h4>
<!--l. 36--><p class="noindent" >At this point, it is important to remember that in this document <span 
class="phvro8t-x-x-120">long RNAs </span>may also refer to products
containing characteristics of traditional mRNAs, that is a 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;m7G Cap, ligated exons, and a Poly(A) tail.
However, many of these long mRNAs are extemely dynamic. So much so that until HTS and RNA-Seq,
comprehensive investigation of their complexity was not possible.
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">Pervasive transcription</span> </dt><dd 
class="description">Here you can put some information from ENCODE and your thoughts on
it.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Tissue and cell specificity</span> </dt><dd 
class="description">Your feelings on Specificity of long RNA expression
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Functional</span></dt><dd 
class="description">We <span 
class="phvro8t-x-x-120">know </span>that some long RNAs are functional. What are these?
                                                                                                    
                                                                                                    
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Chromatin regulation</span> </dt><dd 
class="description">We also know that some long RNAs regulation Chromatin structure. What
are your feelings as to the importance of this fact?
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">PTGS</span></dt><dd 
class="description">Long RNAs  ability  to  do  post  transcriptional  gene  regulation,  including  piRNAs,  and  Xist,
etc....
</dd></dl>
<a 
 id="x1-58001r113"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.2 </span> <a 
 id="x1-590002"></a>The Known Unknowns</h4>
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">How are they important?</span> </dt><dd 
class="description">Conservation of these things is not obvious - if they are not conserved
- are they important? Maybe talk about how MALAT1 is highly expressed, but seems to be
dispensable.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">What regulates their tissue-specific expression?</span> </dt><dd 
class="description">Do they important some of the special sauce
that makes tissues different from one other, more so then the mRNAs changes which can be
extreme, but not terribly so....
</dd><dt class="description">
</dt><dd 
class="description">
</dd></dl>
<a 
 id="x1-59001r114"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.3 </span> <a 
 id="x1-600003"></a>The Unknown Unknowns</h4>
<!--l. 78--><p class="noindent" >This is the area of knowledge keeps many motivated to perform basic research every day. What secrets
does the transcriptome have in store that we haven&#8217;t even <span 
class="phvro8t-x-x-120">thought </span>about? Only through pushing the
boundaries of the last two sections can we begin to think beyond the edge of map and formulate testable
hypothesis. Here I propose a few outlandish ideas for Unknown Unkowns.
<a 
 id="x1-60001r112"></a>
<h3 class="sectionHead"><span class="titlemark">5.2 </span> <a 
 id="x1-610002"></a>Ligation-based investigation of long RNAs</h3>
<a 
 id="x1-61001r115"></a>
<h4 class="subsectionHead"><span class="titlemark">5.2.1 </span> <a 
 id="x1-620001"></a>SeqZip Other Applications</h4>
<a 
 id="x1-62001r117"></a>
<h4 class="subsectionHead"><span class="titlemark">5.2.2 </span> <a 
 id="x1-630002"></a>SeqZip Technical Improvements</h4>
<ul class="itemize1">
<li class="itemize">Use of   T39A   mutation   to   aliviate   penultimate   2<span 
class="cmsy-8">&#x2032;</span>OH   requirement   of   T4   Rnl2   (See
Nandakumar...Lima, Cell 2006)
</li>
<li class="itemize">Use of thermostable ligase, allowing for multiple rounds of ligation. Need a good reference, DO
NOT USE Ref 27 from Conze et al 2009!
                                                                                                    
                                                                                                    
</li>
<li class="itemize">Elevated ligation temperatures, minimizing blut-ended NTL events
</li>
<li class="itemize">Make a note into the future directions that you would like to explore LNA&#8217;s at the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;OH position
of all ligation results, leading to increased ligation efficiency, however both this and the use of
penultimate 2<span 
class="cmsy-8">&#x2032;</span>OH (Ribosome) suger in your ligamers would lead to added costs, and the latter
maybe better served with a T39A mutation. Giggity
</li>
<li class="itemize">Digital PCR of the PCR products ala [<a 
href="#XShiroguchi2012a">Shiroguchi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShiroguchi2012a">2012</a>].
</li>
<li class="itemize">SeqZip on the SOLiD platform
</li>
<li class="itemize">SeqZip on single-cell RNA samples.</li></ul>
<a 
 id="x1-63001r118"></a>
<h4 class="subsectionHead"><span class="titlemark">5.2.3 </span> <a 
 id="x1-640003"></a>SeqZip Alternatives</h4>
<a 
 id="x1-64001r116"></a>
<h3 class="sectionHead"><span class="titlemark">5.3 </span> <a 
 id="x1-650003"></a>Mammalian piRNA-precursors, a special type of long RNA</h3>
<a 
 id="x1-65001r119"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.1 </span> <a 
 id="x1-660001"></a>What are they doing?</h4>
<a 
 id="x1-66001r121"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.2 </span> <a 
 id="x1-670002"></a>How are they generated?</h4>
<a 
 id="x1-67001r122"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.3 </span> <a 
 id="x1-680003"></a>Why should we care?</h4>
                                                                                                    
                                                                                                    
<a 
 id="x1-68001r111"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;A</span><br /><a 
 id="x1-69000A"></a>Appendix - Misc Information</h2>
<a 
 id="x1-69001r120"></a>
<h3 class="sectionHead"><span class="titlemark">A.1 </span> <a 
 id="x1-700001"></a>Buffers</h3>
<div class="table">
                                                                                                    
                                                                                                    
<!--l. 14--><p class="noindent" ><a 
 id="x1-70001r1"></a><hr class="float"><div class="float" 
>
                                                                                                    
                                                                                                    
<div class="tabular"><table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-10-1g"><col 
id="TBL-10-1"></colgroup><colgroup id="TBL-10-2g"><col 
id="TBL-10-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-10-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-1"  
class="td11">Component</td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-2"  
class="td11">Concentration</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-1"  
class="td11">  Tris-HCl   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-2"  
class="td11">   50 mM      </td></tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-1"  
class="td
11">  MgCl2     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-2"  
class="td11">    2 mM       </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-1"  
class="td11"> DTT      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-2"  
class="td11">    1 mM       </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-1"  
class="td11"> ATP       </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-2"  
class="td11">   400 <span 
class="cmmi-12">&#x03BC;</span>M     </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-1"  
class="td11"> pH       </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-2"  
class="td11"> 7.5 @ 25<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C </td>
</tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;A.1: </span><span  
class="content">SeqZip Hybridization and Ligation Buffer </span></div><!--tex4ht:label?: x1-70001r1 -->
                                                                                                    
                                                                                                    
</div><hr class="endfloat" />
</div>
<a 
 id="x1-70002r125"></a>
<h3 class="sectionHead"><span class="titlemark">A.2 </span> <a 
 id="x1-710002"></a>Equations</h3>
<a 
 id="x1-71001r123"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.1 </span> <a 
 id="x1-720001"></a>f</h4>
<!--l. 40--><p class="noindent" >rom <sup><span 
class="cmr-8">32</span> </sup> P-<span 
class="cmmi-12">&#x03B1;</span>-UTP used during vitro transcription]Determining [RNA] from <sup><span 
class="cmr-8">32</span></sup>P-<span 
class="cmmi-12">&#x03B1;</span>-UTP used during vitro
transcription
<!--l. 43--><p class="noindent" ><div class="eqnarray">
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.542x.png" alt="  (  )  (                     ) (               )  (              )
&#x03BC;M=  pm&#x03BC;oLl = ccppmmabfteferorpepuruifirificactioatnion&#x00D7;&#x00D7;diluditiluontiofnacfatoctror &#x00D7;  molURTPeiacntioornigVinaollumreeaction-&#x00D7;  NumberUT1Psintranscript &#x00D7;10-12
" class="math-display" ></center>
</div>
<a 
 id="x1-72002r128"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.2 </span> <a 
 id="x1-730002"></a>b</h4>
<!--l. 60--><p class="noindent" >ased on A<sub><span 
class="cmr-8">260</span> </sub> ]Determining [RNA] based on A<sub><span 
class="cmr-8">260</span></sub>
<!--l. 63--><p class="noindent" >
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.543x.png" alt="              (             A   &#x00D7; Dilution  Factor            )
[RNA   in M ] =   -------------260------------------------------
                10,313 &#x003C; note 1 &#x003E;  &#x00D7; nucleotides in message " class="math-display" ></center> note
1: This value represents an average RNA extinction (<span 
class="cmmi-12">&#x03F5;</span>) coefficient value <br 
class="newline" />
<a 
 id="x1-73001r129"></a>
                                                                                                    
                                                                                                    
<h4 class="subsectionHead"><span class="titlemark">A.2.3 </span> <a 
 id="x1-740003"></a>Normalize oxidized small RNA libraries size to time-matched unoxidized library</h4>
<!--l. 71--><p class="noindent" >NB: this equation assumes calibration against a specific time-point , in this case data obtained from 6
week-old testes.
<!--l. 74--><p class="noindent" ><div class="eqnarray">
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.544x.png" alt="                  ( (  &#x2211;                    )               )
                           miRNA   reads &#x03C4;
                  |   &#x2211;---------------------   &#x00D7; depth  6wk |
                  ||       miRNA   reads  6wk                 ||
unox &#x03C4; norm1   =   | ----------------------------------------|
                  |(                1,000,000                |)

                                  (   &#x2211;                                   )
                                         oxidized shared  &#x2265;  23 nt reads
 ox &#x03C4; norm1   =   unox &#x03C4; norm1  &#x00D7; ( &#x2211;-------------------------------------)
                                        unoxidized shared   &#x2265; 23 nt reads
" class="math-display" ></center>
</div>
<a 
 id="x1-74002r127"></a>
<h3 class="sectionHead"><span class="titlemark">A.3 </span> <a 
 id="x1-750003"></a>PCR Programs</h3>
<!--l. 87--><p class="noindent" ><span 
class="phvb8t-x-x-120">Ligamer Hybridization </span>ROY-H2 | Ligamer Hybridization<br 
class="newline" />Steps 1&#8211;9 are 10 minute incubations at the following temperatures:<br 
class="newline" />69;66;63;58;54;52;50;48;46<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C<br 
class="newline" />Step 10 is a 45<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C incubation for 1 hour<br 
class="newline" />Steps 11&#8211;14 are 10 minute incubations at the following templates:<br 
class="newline" />43;41;39;37<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C<br 
class="newline" />Final incubation is at 37<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C for <span 
class="cmsy-10x-x-120">&#x221E;</span><br 
class="newline" />
<!--l. 96--><p class="noindent" ><span 
class="phvb8t-x-x-120">SeqZip ligation program </span>ROY-37-4 | T4 Rnl2 RNA-template DNA:DNA ligation<br 
class="newline" />1. 37<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C for 18 hours<br 
class="newline" />2. 10<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C for <span 
class="cmsy-10x-x-120">&#x221E;</span><br 
class="newline" />
<a 
 id="x1-75001r131"></a>
<h3 class="sectionHead"><span class="titlemark">A.4 </span> <a 
 id="x1-760004"></a>Source Code</h3>
<!--l. 106-->
                                                                                                    
                                                                                                    
<div class="lstlisting" id="listing-1"><span class="label"><a 
 id="x1-76001r1"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">!</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">usr</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">bin</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">perl</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-76002r2"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-76003r3"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ONE</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ARGV</span><span 
class="ectt-1095">[1];</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-76004r4"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-76005r5"></a></span><span 
class="ectt-1095">chomp</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-76006r6"></a></span><span 
class="ectt-1095">while</span><span 
class="ectt-1095">(&#x003C;&#x003E;)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$_</span><span 
class="ectt-1095">};</span>

</div>
                                                                                                    
                                                                                                    
<a 
 id="x1-76007r111"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;B</span><br /><a 
 id="x1-77000B"></a>Appendix B: piRNA precurors are spliced</h2>
<a 
 id="x1-77001r132"></a>
<h3 class="sectionHead"><span class="titlemark">B.1 </span> <a 
 id="x1-780001"></a>Evidence for spliced piRNA precusor transcripts</h3>
<!--l. 7--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-78001r1"></a>
                                                                                                    
                                                                                                    
<!--l. 9--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.545x.png" alt="PIC" class="graphics" width="415.55249pt" height="224.84pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.545x.png" src="Figures/AppendixB/evidenceForPrecursorSplicing.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.1: </span><span  
class="content">RNA-Seq evidence for piRNA precursor splicing]<br 
class="newline" /><span class="texthl"><span id="textcolor30">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-78001r1 -->
                                                                                                    
                                                                                                    
<!--l. 16--><p class="noindent" ></div><hr class="endfigure">
<!--l. 20--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-78002r2"></a>
                                                                                                    
                                                                                                    
<!--l. 22--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.546x.png" alt="PIC" class="graphics" width="403.50749pt" height="367.3725pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.546x.png" src="Figures/AppendixB/noPiRNAswithinPrecusorIntrons.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.2: </span><span  
class="content">Lack of piRNAs within precusor introns<br 
class="newline" /><span class="texthl"><span id="textcolor31">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-78002r2 -->
                                                                                                    
                                                                                                    
<!--l. 29--><p class="noindent" ></div><hr class="endfigure">
<!--l. 33--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-78003r3"></a>
                                                                                                    
                                                                                                    
<!--l. 35--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.547x.png" alt="PIC" class="graphics" width="415.55249pt" height="258.9675pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.547x.png" src="Figures/AppendixB/smallRNAsMapToPrecursorSJ.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.3: </span><span  
class="content">piRNAs map to Splice Junctions of precursor transcripts<br 
class="newline" /><span class="texthl"><span id="textcolor32">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-78003r3 -->
                                                                                                    
                                                                                                    
<!--l. 42--><p class="noindent" ></div><hr class="endfigure">
<!--l. 46--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-78004r4"></a>
                                                                                                    
                                                                                                    
<!--l. 48--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.548x.png" alt="PIC" class="graphics" width="415.55249pt" height="188.70499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.548x.png" src="Figures/AppendixB/aggregatePiRNAsatSpliceJunctions.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.4: </span><span  
class="content"><span 
class="phvro8t-x-x-120">A-Myb </span>Mutants produce no splice-junction mapping piRNAs<br 
class="newline" /><span class="texthl"><span id="textcolor33">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-78004r4 -->
                                                                                                    
                                                                                                    
<!--l. 55--><p class="noindent" ></div><hr class="endfigure">
<!--l. 61--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                    
                                                                                                    
<a 
 id="x1-78005r5"></a>
                                                                                                    
                                                                                                    
<!--l. 63--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.549x.png" alt="PIC" class="graphics" width="415.55249pt" height="221.82874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.549x.png" src="Figures/AppendixB/piRNAPrecusorTXFeatures.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.5: </span><span  
class="content">General features of piRNA transcripts<br 
class="newline" /><span class="texthl"><span id="textcolor34">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-78005r5 -->
                                                                                                    
                                                                                                    
<!--l. 70--><p class="noindent" ></div><hr class="endfigure">
                                                                                                    
                                                                                                    
<a 
 id="x1-78006r9"></a>
<a 
 id="Q1-1-140"></a>
                                                                                                    
                                                                                                    
<h2 class="likechapterHead"><a 
 id="x1-790009"></a>Bibliography</h2>
<div class="thebibliography">
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAmitsur1987d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Amitsur,  M.,  Levitz,  R.,  and  Kaufmann,  G.  (1987).    Bacteriophage  T4  anticodon  nuclease,
polynucleotide  kinase  and  RNA  ligase  reprocess  the  host  lysine  tRNA.    <span 
class="phvro8t-x-x-120">The  EMBO  journal</span>,
6(8):2499&#8211;503.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAnders2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Anders, S. and Huber, W. (2010).   Differential expression analysis for sequence count data.
<span 
class="phvro8t-x-x-120">Genome Biology</span>, 11(10):R106.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAnstey2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Anstey, M.&#x00A0;L., Rogers, S.&#x00A0;M., Ott, S.&#x00A0;R., Burrows, M., and Simpson, S.&#x00A0;J. (2009).  Serotonin
mediates behavioral gregarization underlying swarm formation in desert locusts.  <span 
class="phvro8t-x-x-120">Science (New</span>
<span 
class="phvro8t-x-x-120">York, N.Y.)</span>, 323(5914):627&#8211;30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A., Gaidatzis, D., Pfeffer, S., Lagos-Quintana, M., Landgraf, P., Iovino, N., Morris, P.,
Brownstein, M.&#x00A0;J., Kuramochi-Miyagawa, S., Nakano, T., Chien, M., Russo, J.&#x00A0;J., Ju, J., Sheridan,
R., Sander, C., Zavolan, M., and Tuschl, T. (2006). A novel class of small RNAs bind to MILI protein
in mouse testes. <span 
class="phvro8t-x-x-120">Nature</span>, 442(7099):203&#8211;207.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A. and Hannon, G.&#x00A0;J. (2008). Small RNA silencing pathways in germ and stem cells.
<span 
class="phvro8t-x-x-120">Cold Spring Harbor symposia on quantitative biology</span>, 73:283&#8211;90.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2007a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A., Hannon, G.&#x00A0;J., and Brennecke, J. (2007a). The Piwi-piRNA Pathway Provides an
Adaptive Defense in the Transposon Arms Race. <span 
class="phvro8t-x-x-120">Science</span>, 318(5851):761&#8211;764.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A., Naumova, N.&#x00A0;M., Tulin, A.&#x00A0;V., Vagin, V.&#x00A0;V., Rozovsky, Y.&#x00A0;M., and Gvozdev, V.&#x00A0;A.
(2001). Double-stranded RNA-mediated silencing of genomic tandem repeats and transposable
elements in the D. melanogaster germline. <span 
class="phvro8t-x-x-120">Current Biology</span>, 11(13):1017&#8211;1027.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A.,  Sachidanandam,  R.,  Girard,  A.,  Fejes-Toth,  K.,  and  Hannon,  G.&#x00A0;J.  (2007b).
                                                                                                    
                                                                                                    
Developmentally regulated piRNA clusters implicate MILI in transposon control.   <span 
class="phvro8t-x-x-120">Science (New</span>
<span 
class="phvro8t-x-x-120">York, N.Y.)</span>, 316(5825):744&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAshe2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ashe, A., Sapetschnig, A., Weick, E.-M., Mitchell, J., Bagijn, M.&#x00A0;P., Cording, A.&#x00A0;C., Doebley,
A.-L., Goldstein, L.&#x00A0;D., Lehrbach, N.&#x00A0;J., Le Pen, J., Pintacuda, G., Sakaguchi, A., Sarkies, P.,
Ahmed, S., and Miska, E.&#x00A0;a. (2012).  piRNAs can trigger a multigenerational epigenetic memory
in the germline of C. elegans. <span 
class="phvro8t-x-x-120">Cell</span>, 150(1):88&#8211;99.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBailey2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bailey, T.&#x00A0;L., Boden, M., Buske, F.&#x00A0;a., Frith, M., Grant, C.&#x00A0;E., Clementi, L., Ren, J., Li, W.&#x00A0;W.,
and Noble, W.&#x00A0;S. (2009).  MEME SUITE: tools for motif discovery and searching.  <span 
class="phvro8t-x-x-120">Nucleic acids</span>
<span 
class="phvro8t-x-x-120">research</span>, 37(Web Server issue):W202&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBailey1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bailey, T.&#x00A0;L.  and  Elkan,  C.  (1994).   Fitting  a  mixture  model  by  expectation  maximization  to
discover motifs in biopolymers.  <span 
class="phvro8t-x-x-120">Proceedings International Conference on Intelligent Systems for</span>
<span 
class="phvro8t-x-x-120">Molecular Biology ISMB International Conference on Intelligent Systems for Molecular Biology</span>,
2:28&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarash2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barash, Y., Calarco, J.&#x00A0;A., Gao, W., Pan, Q., Wang, X., Shai, O., Blencowe, B.&#x00A0;J., and Frey, B.&#x00A0;J.
(2010). Deciphering the splicing code. <span 
class="phvro8t-x-x-120">Nature</span>, 465(7294):53&#8211;59.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarrangou2007c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barrangou, R., Fremaux, C., Deveau, H., Richards, M., Boyaval, P., Moineau, S., Romero, D.&#x00A0;a.,
and Horvath, P. (2007).   CRISPR provides acquired resistance against viruses in prokaryotes.
<span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 315(5819):1709&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarrett2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barrett, T., Wilhite, S.&#x00A0;E., Ledoux, P., Evangelista, C., Kim, I.&#x00A0;F., Tomashevsky, M., Marshall,
K.&#x00A0;a., Phillippy, K.&#x00A0;H., Sherman, P.&#x00A0;M., Holko, M., Yefanov, A., Lee, H., Zhang, N., Robertson,
C.&#x00A0;L., Serova, N., Davis, S., and Soboleva, A. (2013). NCBI GEO: archive for functional genomics
data sets&#8211;update. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 41(Database issue):D991&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCordell1980a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bell, G.&#x00A0;I., Pictet, R.&#x00A0;L., Rutter, W.&#x00A0;J., Cordell, B., Tischer, E., and Goodman, H.&#x00A0;M. (1980).
Sequence of the human insulin gene. <span 
class="phvro8t-x-x-120">Nature</span>, 284(5751):26&#8211;32.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBenson2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Benson,  D.&#x00A0;a.,  Karsch-Mizrachi,  I.,  Lipman,  D.&#x00A0;J.,  Ostell,  J.,  and  Sayers,  E.&#x00A0;W.  (2011).
GenBank. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 39(Database issue):D32&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBentley2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bentley, D.&#x00A0;R., Balasubramanian, S., Swerdlow, H.&#x00A0;P., Smith, G.&#x00A0;P., Milton, J., Brown, C.&#x00A0;G.,
Hall, K.&#x00A0;P., Evers, D.&#x00A0;J., Barnes, C.&#x00A0;L., Bignell, H.&#x00A0;R., Boutell, J.&#x00A0;M., Bryant, J., Carter, R.&#x00A0;J.,
Keira Cheetham, R., Cox, A.&#x00A0;J., Ellis, D.&#x00A0;J., Flatbush, M.&#x00A0;R., Gormley, N.&#x00A0;a., Humphray, S.&#x00A0;J.,
Irving, L.&#x00A0;J.,  Karbelashvili,  M.&#x00A0;S.,  Kirk,  S.&#x00A0;M.,  Li,  H.,  Liu,  X.,  Maisinger,  K.&#x00A0;S.,  Murray,  L.&#x00A0;J.,
Obradovic, B., Ost, T., Parkinson, M.&#x00A0;L., Pratt, M.&#x00A0;R., Rasolonjatovo, I. M.&#x00A0;J., Reed, M.&#x00A0;T., Rigatti,
                                                                                                    
                                                                                                    
R., Rodighiero, C., Ross, M.&#x00A0;T., Sabot, A., Sankar, S.&#x00A0;V., Scally, A., Schroth, G.&#x00A0;P., Smith, M.&#x00A0;E.,
Smith, V.&#x00A0;P.,  Spiridou,  A.,  Torrance,  P.&#x00A0;E.,  Tzonev,  S.&#x00A0;S.,  Vermaas,  E.&#x00A0;H.,  Walter,  K.,  Wu,  X.,
Zhang, L., Alam, M.&#x00A0;D., Anastasi, C., Aniebo, I.&#x00A0;C., Bailey, D. M.&#x00A0;D., Bancarz, I.&#x00A0;R., Banerjee, S.,
Barbour, S.&#x00A0;G., Baybayan, P.&#x00A0;a., Benoit, V.&#x00A0;a., Benson, K.&#x00A0;F., Bevis, C., Black, P.&#x00A0;J., Boodhun,
A., Brennan, J.&#x00A0;S., Bridgham, J.&#x00A0;a., Brown, R.&#x00A0;C., Brown, A.&#x00A0;a., Buermann, D.&#x00A0;H., Bundu, A.&#x00A0;a.,
Burrows, J.&#x00A0;C., Carter, N.&#x00A0;P., Castillo, N., Chiara E Catenazzi, M., Chang, S., Neil Cooley, R.,
Crake, N.&#x00A0;R.,  Dada,  O.&#x00A0;O.,  Diakoumakos,  K.&#x00A0;D.,  Dominguez-Fernandez,  B.,  Earnshaw,  D.&#x00A0;J.,
Egbujor, U.&#x00A0;C., Elmore, D.&#x00A0;W., Etchin, S.&#x00A0;S., Ewan, M.&#x00A0;R., Fedurco, M., Fraser, L.&#x00A0;J., Fuentes
Fajardo, K.&#x00A0;V., Scott Furey, W., George, D., Gietzen, K.&#x00A0;J., Goddard, C.&#x00A0;P., Golda, G.&#x00A0;S., Granieri,
P.&#x00A0;a., Green, D.&#x00A0;E., Gustafson, D.&#x00A0;L., Hansen, N.&#x00A0;F., Harnish, K., Haudenschild, C.&#x00A0;D., Heyer,
N.&#x00A0;I., Hims, M.&#x00A0;M., Ho, J.&#x00A0;T., Horgan, A.&#x00A0;M., Hoschler, K., Hurwitz, S., Ivanov, D.&#x00A0;V., Johnson,
M.&#x00A0;Q., James, T., Huw Jones, T.&#x00A0;a., Kang, G.-D., Kerelska, T.&#x00A0;H., Kersey, A.&#x00A0;D., Khrebtukova,
I., Kindwall, A.&#x00A0;P., Kingsbury, Z., Kokko-Gonzales, P.&#x00A0;I., Kumar, A., Laurent, M.&#x00A0;a., Lawley, C.&#x00A0;T.,
Lee, S.&#x00A0;E., Lee, X., Liao, A.&#x00A0;K., Loch, J.&#x00A0;a., Lok, M., Luo, S., Mammen, R.&#x00A0;M., Martin, J.&#x00A0;W.,
McCauley,  P.&#x00A0;G.,  McNitt,  P.,  Mehta,  P.,  Moon,  K.&#x00A0;W.,  Mullens,  J.&#x00A0;W.,  Newington,  T.,  Ning,  Z.,
Ling Ng, B., Novo, S.&#x00A0;M., O&#8217;Neill, M.&#x00A0;J., Osborne, M.&#x00A0;a., Osnowski, A., Ostadan, O., Paraschos,
L.&#x00A0;L., Pickering,  L.,  Pike,  A.&#x00A0;C.,  Pike,  A.&#x00A0;C.,  Chris  Pinkard,  D.,  Pliskin,  D.&#x00A0;P.,  Podhasky,  J.,
Quijano, V.&#x00A0;J., Raczy, C., Rae, V.&#x00A0;H., Rawlings, S.&#x00A0;R., Chiva Rodriguez, A., Roe, P.&#x00A0;M., Rogers,
J., Rogert Bacigalupo, M.&#x00A0;C., Romanov, N., Romieu, A., Roth, R.&#x00A0;K., Rourke, N.&#x00A0;J., Ruediger,
S.&#x00A0;T., Rusman, E., Sanches-Kuiper, R.&#x00A0;M., Schenker, M.&#x00A0;R., Seoane, J.&#x00A0;M., Shaw, R.&#x00A0;J., Shiver,
M.&#x00A0;K., Short,  S.&#x00A0;W.,  Sizto,  N.&#x00A0;L.,  Sluis,  J.&#x00A0;P.,  Smith,  M.&#x00A0;a.,  Ernest  Sohna  Sohna,  J.,  Spence,
E.&#x00A0;J., Stevens,  K.,  Sutton,  N.,  Szajkowski,  L.,  Tregidgo,  C.&#x00A0;L.,  Turcatti,  G.,  Vandevondele,  S.,
Verhovsky, Y., Virk, S.&#x00A0;M., Wakelin, S., Walcott, G.&#x00A0;C., Wang, J., Worsley, G.&#x00A0;J., Yan, J., Yau,
L., Zuerlein, M., Rogers, J., Mullikin, J.&#x00A0;C., Hurles, M.&#x00A0;E., McCooke, N.&#x00A0;J., West, J.&#x00A0;S., Oaks,
F.&#x00A0;L., Lundberg, P.&#x00A0;L., Klenerman, D., Durbin, R., and Smith, A.&#x00A0;J. (2008). Accurate whole human
genome sequencing using reversible terminator chemistry. <span 
class="phvro8t-x-x-120">Nature</span>, 456(7218):53&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBenton2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Benton, M.&#x00A0;J. and Donoghue, P. C.&#x00A0;J. (2007).  Paleontological evidence to date the tree of life.
<span 
class="phvro8t-x-x-120">Molecular biology and evolution</span>, 24(1):26&#8211;53.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBernstein1983a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bernstein, S.&#x00A0;I., Mogami, K., Donady, J.&#x00A0;J., and Emerson, C.&#x00A0;P. (1983).   Drosophila muscle
myosin heavy  chain  encoded  by  a  single  gene  in  a  cluster  of  muscle  mutations.    <span 
class="phvro8t-x-x-120">Nature</span>,
302(5907):393&#8211;397.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBesmer1972b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Besmer,  P.,  Jr.,  R.  C.&#x00A0;M.,  Caruthers,  M.&#x00A0;H.,  Kumar,  A.,  Minamoto,  K.,  van&#x00A0;de  Sande,  J.,
Sidarova, N.,  and  Khorana,  H.  (1972).    Studies  on  polynucleotides.  CXVII.  Hybridization  of
polydeoxynucleotides with tyrosine transfer RNA sequences to the r-strand of phi80psu + 3 DNA.
<span 
class="phvro8t-x-x-120">Journal of Molecular Biology</span>, 72:503&#8211;522.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBlencowe2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Blencowe,  B.&#x00A0;J.,  Ahmad,  S.,  and  Lee,  L.&#x00A0;J.  (2009).     Current-generation  high-throughput
sequencing:  deepening  insights  into  mammalian  transcriptomes.      <span 
class="phvro8t-x-x-120">Genes  &amp;  development</span>,
23(12):1379&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBolcun-Filas2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bolcun-Filas, E., Bannister, L.&#x00A0;A., Barash, A., Schimenti, K.&#x00A0;J., Hartford, S.&#x00A0;A., Eppig, J.&#x00A0;J.,
                                                                                                    
                                                                                                    
Handel, M.&#x00A0;A., Shen, L., and Schimenti, J.&#x00A0;C. (2011).  A-MYB (MYBL1) transcription factor is a
master regulator of male meiosis. <span 
class="phvro8t-x-x-120">Development</span>, 138(15):3319 &#8211;3330.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrennecke2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brennecke, J., Aravin, A.&#x00A0;A., Stark, A., Dus, M., Kellis, M., Sachidanandam, R., and Hannon,
G.&#x00A0;J. (2007).  Discrete small RNA-generating loci as master regulators of transposon activity in
Drosophila. <span 
class="phvro8t-x-x-120">Cell</span>, 128(6):1089&#8211;1103.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrennecke2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brennecke, J. and Malone, C. (2008).  An Epigenetic Role for Maternally Inherited piRNAs in
Transposon Silencing. <span 
class="phvro8t-x-x-120">Science (New York, </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, 322(November):1387&#8211;1392.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrenner2000a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brenner, S., Johnson, M., Bridgham, J., Golda, G., Lloyd, D.&#x00A0;H., Johnson, D., Luo, S., McCurdy,
S., Foy, M.,  Ewan,  M.,  Roth,  R.,  George,  D.,  Eletr,  S.,  Albrecht,  G.,  Vermaas,  E.,  Williams,
S.&#x00A0;R., Moon,  K.,  Burcham,  T.,  Pallas,  M.,  DuBridge,  R.&#x00A0;B.,  Kirchner,  J.,  Fearon,  K.,  Mao,  J.,
and Corcoran, K. (2000).  Gene expression analysis by massively parallel signature sequencing
(MPSS) on microbead arrays. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 18(6):630&#8211;4.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrown2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brown, J.&#x00A0;B., Boley, N., Eisman, R., May, G.&#x00A0;E., Stoiber, M.&#x00A0;H., Duff, M.&#x00A0;O., Booth, B.&#x00A0;W., Wen,
J., Park, S., Suzuki, A.&#x00A0;M., Wan, K.&#x00A0;H., Yu, C., Zhang, D., Carlson, J.&#x00A0;W., Cherbas, L., Eads, B.&#x00A0;D.,
Miller, D., Mockaitis, K., Roberts, J., Davis, C.&#x00A0;a., Frise, E., Hammonds, A.&#x00A0;S., Olson, S., Shenker,
S., Sturgill,  D.,  Samsonova,  A.&#x00A0;a.,  Weiszmann,  R.,  Robinson,  G.,  Hernandez,  J.,  Andrews,  J.,
Bickel, P.&#x00A0;J., Carninci, P., Cherbas, P., Gingeras, T.&#x00A0;R., Hoskins, R.&#x00A0;a., Kaufman, T.&#x00A0;C., Lai, E.&#x00A0;C.,
Oliver, B., Perrimon, N., Graveley, B.&#x00A0;R., and Celniker, S.&#x00A0;E. (2014).  Diversity and dynamics of
the Drosophila transcriptome. <span 
class="phvro8t-x-x-120">Nature</span>, pages 1&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBullard2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bullard, D. and Bowater, R. (2006).  Direct comparison of nick-joining activity of the nucleic acid
ligases from bacteriophage T4. <span 
class="phvro8t-x-x-120">Biochemical Journal</span>, 398(Pt 1):135&#8211;144.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBurnside2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Burnside,  J.,  Ouyang,  M.,  Anderson,  A.,  Bernberg,  E.,  Lu,  C.,  Meyers,  B.&#x00A0;C.,  Green,  P.&#x00A0;J.,
Markis, M.,  Isaacs,  G.,  Huang,  E.,  and  Morgan,  R.&#x00A0;W.  (2008).   Deep  sequencing  of  chicken
microRNAs. <span 
class="phvro8t-x-x-120">BMC genomics</span>, 9:185.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCalarco2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Calarco, J.&#x00A0;A., Saltzman, A.&#x00A0;L., Ip, J.&#x00A0;Y., and Blencowe, B.&#x00A0;J. (2007a).   Technologies for the
global discovery and analysis of alternative splicing.  In <span 
class="phvro8t-x-x-120">Advances in Experimental Medicine and</span>
<span 
class="phvro8t-x-x-120">Biology</span>, volume 623, pages 64&#8211;84.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCalarco2007a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Calarco, J.&#x00A0;A., Xing, Y., Cceres, M., Calarco, J.&#x00A0;P., Xiao, X., Pan, Q., Lee, C., Preuss, T.&#x00A0;M.,
and Blencowe, B.&#x00A0;J. (2007b). Global analysis of alternative splicing differences between humans
and chimpanzees. <span 
class="phvro8t-x-x-120">Genes &amp; Development</span>, 21(22):2963&#8211;75.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCarmell2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Carmell, M.&#x00A0;A., Girard, A., van&#x00A0;de Kant, H.&#x00A0;J., Bourc&#8217;his, D., Bestor, T.&#x00A0;H., de&#x00A0;Rooij, D.&#x00A0;G., and
                                                                                                    
                                                                                                    
Hannon, G.&#x00A0;J. (2007).  MIWI2 Is Essential for Spermatogenesis and Repression of Transposons
in the Mouse Male Germline. <span 
class="phvro8t-x-x-120">Developmental Cell</span>, 12(4):503&#8211;514.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCenik2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Cenik, E.&#x00A0;S.  and  Zamore,  P.&#x00A0;D.  (2011).     Argonaute  proteins.     <span 
class="phvro8t-x-x-120">Current  biology  :  CB</span>,
21(12):R446&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChauleau2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chauleau, M. and Shuman, S. (2013).   Kinetic mechanism of nick sealing by T4 RNA ligase
2 and effects  of  3&#8217;-OH  base  mispairs  and  damaged  base  lesions.    <span 
class="phvro8t-x-x-120">RNA  (New  York,  N.Y.)</span>,
19(12):1840&#8211;1847.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, C., Jin, J., James, D.&#x00A0;A., Adams-cioaba, M.&#x00A0;A., Gyoon, J., Guo, Y., Tenaglia, E., Xu, C.,
Gish, G., Min, J., and Pawson, T. (2009).  Mouse Piwi interactome identifies binding mechanism
of Tdrkh Tudor domain to arginine methylated Miwi.   <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of</span>
<span 
class="phvro8t-x-x-120">Sciences</span>, 106(48):20336&#8211;20341.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, X., Xu, H., Yuan, P., Fang, F., Huss, M., Vega, V.&#x00A0;B., Wong, E., Orlov, Y.&#x00A0;L., Zhang, W.,
Jiang, J., Loh, Y.-H., Yeo, H.&#x00A0;C., Yeo, Z.&#x00A0;X., Narang, V., Govindarajan, K.&#x00A0;R., Leong, B., Shahab,
A., Ruan, Y., Bourque, G., Sung, W.-K., Clarke, N.&#x00A0;D., Wei, C.-L., and Ng, H.-H. (2008). Integration
of external signaling pathways with the core transcriptional network in embryonic stem cells. <span 
class="phvro8t-x-x-120">Cell</span>,
133(6):1106&#8211;17.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChuma2006a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chuma, S. and Hosokawa, M. (2006).  Tdrd1/Mtr-1 ,a male tudor-related gene, is essential for
male germ-cell differentiationand nuage/germinal grandule formation in mice.  <span 
class="phvro8t-x-x-120">Proceedings of the</span>
<span 
class="phvro8t-x-x-120">&#x2026;</span>, 103(43):1&#8211;6.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHattori2005a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Consortium, I. H. G.&#x00A0;S. (2004).   Finishing the euchromatic sequence of the human genome.
<span 
class="phvro8t-x-x-120">Nature</span>, 50(2):162&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XConze2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Conze, T., Goransson, J., Razzaghian, H.&#x00A0;R., Ericsson, O., Oberg, D., Akusjarvi, G., Landegren,
U., and Nilsson, M. (2010).  Single molecule analysis of combinatorial splicing.  <span 
class="phvro8t-x-x-120">Nucl. Acids Res.</span>,
page gkq581.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XConze2009c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Conze, T.,  Shetye,  A.,  Tanaka,  Y.,  Gu,  J.,  Larsson,  C.,  Gransson,  J.,  Tavoosidana,  G.,
Sderberg, O., Nilsson, M., and Landegren, U. (2009). Analysis of genes, transcripts, and proteins
via DNA ligation. <span 
class="phvro8t-x-x-120">Annual review of analytical chemistry (Palo Alto, Calif.)</span>, 2:215&#8211;39.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCore2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Core, L.&#x00A0;J., Waterfall, J.&#x00A0;J., and Lis, J.&#x00A0;T. (2008). Nascent RNA sequencing reveals widespread
pausing and divergent initiation at human promoters. <span 
class="phvro8t-x-x-120">Science</span>, 322:1845&#8211;1848.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                                    
                                                                                                    
<a 
 id="XDeFazio2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> De Fazio,  S.,  Bartonicek,  N.,  Di  Giacomo,  M.,  Abreu-Goodger,  C.,  Sankar,  A.,  Funaya,  C.,
Antony, C., Moreira, P.&#x00A0;N., Enright, A.&#x00A0;J., and O/&#8217;Carroll, D. (2011).  The endonuclease activity of
Mili fuels piRNA amplification that silences LINE1 elements. <span 
class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeng1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Deng, Q.  Q.-l.,  Ishii,  S.,  and  Sarai,  A.  (1996).    Binding  site  analysis  of  c-Myb  :  screening
of potential  binding  sites  by  using  the  mutation  matrix  derived  from  systematic  binding  affinity
measurements. <span 
class="phvro8t-x-x-120">Nucleic Acids Research</span>, 24(4):766&#8211;774.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeng2002c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Deng, W. and Lin, H. (2002).  miwi, a Murine Homolog of piwi, Encodes a Cytoplasmic Protein
Essential for Spermatogenesis. <span 
class="phvro8t-x-x-120">Developmental cell</span>, 2(6):819&#8211;830.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDjebali2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Djebali, S., Davis, C.&#x00A0;a., Merkel, A., Dobin, A., Lassmann, T., Mortazavi, A., Tanzer, A., Lagarde,
J., Lin, W.,  Schlesinger,  F.,  Xue,  C.,  Marinov,  G.&#x00A0;K.,  Khatun,  J.,  Williams,  B.&#x00A0;a.,  Zaleski,  C.,
Rozowsky, J., Rder, M., Kokocinski, F., Abdelhamid, R.&#x00A0;F., Alioto, T., Antoshechkin, I., Baer, M.&#x00A0;T.,
Bar, N.&#x00A0;S., Batut, P., Bell, K., Bell, I., Chakrabortty, S., Chen, X., Chrast, J., Curado, J., Derrien,
T., Drenkow, J., Dumais, E., Dumais, J., Duttagupta, R., Falconnet, E., Fastuca, M., Fejes-Toth,
K., Ferreira, P., Foissac, S., Fullwood, M.&#x00A0;J., Gao, H., Gonzalez, D., Gordon, A., Gunawardena,
H., Howald, C., Jha, S., Johnson, R., Kapranov, P., King, B., Kingswood, C., Luo, O.&#x00A0;J., Park, E.,
Persaud, K., Preall, J.&#x00A0;B., Ribeca, P., Risk, B., Robyr, D., Sammeth, M., Schaffer, L., See, L.-H.,
Shahab, A., Skancke, J., Suzuki, A.&#x00A0;M., Takahashi, H., Tilgner, H., Trout, D., Walters, N., Wang,
H., Wrobel, J., Yu, Y., Ruan, X., Hayashizaki, Y., Harrow, J., Gerstein, M., Hubbard, T., Reymond,
A., Antonarakis, S.&#x00A0;E., Hannon, G., Giddings, M.&#x00A0;C., Ruan, Y., Wold, B., Carninci, P., Guig, R.,
and Gingeras, T.&#x00A0;R. (2012). Landscape of transcription in human cells. <span 
class="phvro8t-x-x-120">Nature</span>, 489(7414):101&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDjebali2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Djebali, S., Kapranov, P., Foissac, S., Lagarde, J., Reymond, A., Ucla, C., Wyss, C., Drenkow,
J., Dumais, E., Murray, R.&#x00A0;R., Lin, C., Szeto, D., Denoeud, F., Calvo, M., Frankish, A., Harrow, J.,
Makrythanasis, P., Vidal, M., Salehi-Ashtiani, K., Antonarakis, S.&#x00A0;E., Gingeras, T.&#x00A0;R., and Guigo,
R. (2008). Efficient targeted transcript discovery via array-based normalization of RACE libraries.
<span 
class="phvro8t-x-x-120">Nat Meth</span>, 5(7):629&#8211;635.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XEmerick2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Emerick, M., Parmigiani, G., and Agnew, W. (2007).  Multivariate Analysis and Visualization of
Splicing Correlations in Single-Gene Transcriptomes. <span 
class="phvro8t-x-x-120">BMC Bioinformatics</span>, 8(1):16.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFagnani2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fagnani, M., Barash, Y., Ip, J.&#x00A0;Y., Misquitta, C., Pan, Q., Saltzman, A.&#x00A0;L., Shai, O., Lee, L.,
Rozenhek,  A.,  Mohammad,  N.,  Willaime-Morawek,  S.,  Babak,  T.,  Zhang,  W.,  Hughes,  T.&#x00A0;R.,
van&#x00A0;der Kooy, D., Frey, B.&#x00A0;J., and Blencowe, B.&#x00A0;J. (2007).  Functional coordination of alternative
splicing in the mammalian central nervous system. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 8(6):R108.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFarazi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Farazi, T.&#x00A0;a., Juranek, S.&#x00A0;a., and Tuschl, T. (2008). The growing catalog of small RNAs and their
association with distinct Argonaute/Piwi family members.   <span 
class="phvro8t-x-x-120">Development (Cambridge, England)</span>,
135(7):1201&#8211;14.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFareed1971"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fareed, G.&#x00A0;C., Wilt, E.&#x00A0;M., and Richardson, C.&#x00A0;C. (1971).  Enzymatic Breakage and Joining of
Deoxyribonucleic Acid. <span 
class="phvro8t-x-x-120">Journal of Biological Chemistry</span>, 246:925&#8211;932.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFededa2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fededa, J.&#x00A0;P., Petrillo, E., Gelfand, M.&#x00A0;S., Neverov, A.&#x00A0;D., Kadener, S., Nogus, G., Pelisch, F.,
Baralle, F.&#x00A0;E., Muro, A.&#x00A0;F., and Kornblihtt, A.&#x00A0;R. (2005). A Polar Mechanism Coordinates Different
Regions of Alternative Splicing within a Single Gene. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, 19(3):393&#8211;404.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFire1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fire, A., Xu, S., Montgomery, M.&#x00A0;K., Kostas, S.&#x00A0;A., Driver, S.&#x00A0;E., and Mello, C.&#x00A0;C. (1998). Potent
and specific genetic interference by double-stranded RNA in Caenorhabditis elegans.   <span 
class="phvro8t-x-x-120">Nature</span>,
391:806&#8211;811.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFodor2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fodor, A.&#x00A0;a. and Aldrich, R.&#x00A0;W. (2009).  Convergent evolution of alternative splices at domain
boundaries of the BK channel. <span 
class="phvro8t-x-x-120">Annual review of physiology</span>, 71:19&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFuruichi1975"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Furuichi, Y. (1975).  5&#8217;-terminal m7G(5&#8217;)ppp(5&#8217;)Gmp In Vivo: Identification in Reovirus Genome
RNA. <span 
class="phvro8t-x-x-120">Proceedings of the </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, 72(2):742&#8211;745.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGan2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gan, H., Lin, X., Zhang, Z., Zhang, W., Liao, S., Wang, L., and Han, C. (2011).  piRNA profiling
during specific stages of mouse spermatogenesis. <span 
class="phvro8t-x-x-120">RNA</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGhildiyal2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ghildiyal, M., Seitz, H., Horwich, M.&#x00A0;D., Li, C., Du, T., Lee, S., Xu, J., Kittler, E. L.&#x00A0;W., Zapp,
M.&#x00A0;L., Weng, Z., and Zamore, P.&#x00A0;D. (2008).  Endogenous siRNAs Derived from Transposons and
mRNAs in Drosophila Somatic Cells. <span 
class="phvro8t-x-x-120">Science</span>, 320(5879):1077&#8211;1081.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGhosh2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ghosh, S. and Jacobson, A. (2010).  RNA decay modulates gene expression and controls its
fidelity. <span 
class="phvro8t-x-x-120">WIREs RNA</span>, 1:351&#8211;361.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGilbert1978a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gilbert, W. (1978). Why genes in pieces? <span 
class="phvro8t-x-x-120">Nature</span>, 271(5645):501.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGirard2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Girard, A., Sachidanandam, R., Hannon, G.&#x00A0;J., and Carmell, M.&#x00A0;A. (2006). A germline-specific
class of small RNAs binds mammalian Piwi proteins. <span 
class="phvro8t-x-x-120">Nature</span>, 442(7099):199&#8211;202.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGlauser2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Glauser,  D.&#x00A0;A.,  Johnson,  B.&#x00A0;E.,  Aldrich,  R.&#x00A0;W.,  and  Goodman,  M.&#x00A0;B.  (2011).    Intragenic
alternative  splicing  coordination  is  essential  for  Caenorhabditis  elegans  slo-1  gene  function.
<span 
class="phvro8t-x-x-120">Proceedings   of   the   National   Academy   of   Sciences   of   the   United   States   of   America</span>,
108(51):20790&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                                    
                                                                                                    
<a 
 id="XGrabherr2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grabherr, M.&#x00A0;G., Haas, B.&#x00A0;J., Yassour, M., Levin, J.&#x00A0;Z., Thompson, D.&#x00A0;A., Amit, I., Adiconis, X.,
Fan, L., Raychowdhury, R., Zeng, Q., Chen, Z., Mauceli, E., Hacohen, N., Gnirke, A., Rhind, N.,
di&#x00A0;Palma, F., Birren, B.&#x00A0;W., Nusbaum, C., Lindblad-Toh, K., Friedman, N., and Regev, A. (2011).
Full-length  transcriptome  assembly  from  RNA-Seq  data  without  a  reference  genome.   <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Biotechnology</span>, 29(7):644&#8211;652.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGrant2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grant, C.&#x00A0;E., Bailey, T.&#x00A0;L., and Noble, W.&#x00A0;S. (2011). FIMO: scanning for occurrences of a given
motif. <span 
class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 27(7):1017&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGraveley2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Graveley, B.&#x00A0;R. (2000).  Sorting out the complexity of SR protein functions.  <span 
class="phvro8t-x-x-120">RNA (New York,</span>
<span 
class="phvro8t-x-x-120">N.Y.)</span>, 6(9):1197&#8211;211.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGrivna2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grivna, S.&#x00A0;T., Beyret, E., Wang, Z., and Lin, H. (2006).  A novel class of small RNAs in mouse
spermatogenic cells. <span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 20(13):1709&#8211;14.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGu2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gu, A., Ji, G., Shi, X., Long, Y., Xia, Y., Song, L., Wang, S., and Wang, X. (2010).   Genetic
Variants in Piwi-Interacting RNA Pathway Genes Confer Susceptibility to Spermatogenic Failure
in a Chinese Population. <span 
class="phvro8t-x-x-120">Human Reproduction</span>, 25(12):2955&#8211;2961.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGu2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gu, W., Lee, H.-C., Chaves, D., Youngman, E.&#x00A0;M., Pazour, G.&#x00A0;J., Conte, D., and Mello, C.&#x00A0;C.
(2012). CapSeq  and  CIP-TAP  identify  Pol  II  start  sites  and  reveal  capped  small  RNAs  as  C.
elegans piRNA precursors. <span 
class="phvro8t-x-x-120">Cell</span>, 151(7):1488&#8211;500.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuenther2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Guenther,  M.&#x00A0;G.,  Levine,  S.&#x00A0;S.,  Boyer,  L.&#x00A0;A.,  Jaenisch,  R.,  and  Young,  R.&#x00A0;A.  (2007).    A
chromatin landmark and transcription initiation at most promoters in human cells. <span 
class="phvro8t-x-x-120">Cell</span>, 130:77&#8211;88.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGupta2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gupta, S.,  Stamatoyannopoulos,  J.&#x00A0;a.,  Bailey,  T.&#x00A0;L.,  and  Noble,  W.&#x00A0;S.  (2007).   Quantifying
similarity between motifs. <span 
class="phvro8t-x-x-120">Genome biology</span>, 8(2):R24.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHaase2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Haase, A.&#x00A0;D., Fenoglio, S., Muerdter, F., Guzzardo, P.&#x00A0;M., Czech, B., Pappin, D.&#x00A0;J., Chen, C.,
Gordon, A., and Hannon, G.&#x00A0;J. (2010).  Probing the initiation and effector phases of the somatic
piRNA pathway in Drosophila. <span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 24(22):2499&#8211;504.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHafner2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hafner, M., Landgraf, P., Ludwig, J., Rice, A., Ojo, T., Lin, C., Holoch, D., Lim, C., and Tuschl,
T. (2008).    Identification  of  microRNAs  and  other  small  regulatory  RNAs  using  cDNA  library
sequencing. <span 
class="phvro8t-x-x-120">Methods (San Diego, Calif.)</span>, 44(1):3&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHafner2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hafner, M., Landthaler, M., Burger, L., Khorshid, M., Hausser, J., Berninger, P., Rothballer, A.,
Ascano Jr., M., Jungkamp, A.-C., Munschauer, M., Ulrich, A., Wardle, G.&#x00A0;S., Dewell, S., Zavolan,
                                                                                                    
                                                                                                    
M., and Tuschl, T. (2010). Transcriptome-wide Identification of RNA-Binding Protein and MicroRNA
Target Sites by PAR-CLIP. <span 
class="phvro8t-x-x-120">Cell</span>, 141(1):129&#8211;141.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHandler2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Handler, D., Olivieri, D., Novatchkova, M., Gruber, F.&#x00A0;S., Meixner, K., Mechtler, K., Stark, A.,
Sachidanandam,  R.,  and  Brennecke,  J.  (2011).   A  systematic  analysis  of  Drosophila  TUDOR
domain-containing proteins identifies Vreteno and the Tdrd12 family as essential primary piRNA
pathway factors. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 30(19):3977&#8211;93.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHartig2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hartig, J.&#x00A0;V., Tomari, Y., and Frstemann, K. (2007).  piRNAs&#8211;the ancient hunters of genome
invaders. <span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 21(14):1707&#8211;13.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo1997b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K., Etten, J. L.&#x00A0;V., and Shuman, S. (1997).  Characterization of an ATP-dependent DNA
ligase encoded by Chlorella virus PBCV-1. <span 
class="phvro8t-x-x-120">Journal of virology</span>, 71(3).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo2002b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K.  and  Shuman,  S.  (2002).   Bacteriophage  T4  RNA  ligase  2  (gp24.1)  exemplifies  a
family of RNA ligases found in all phylogenetic domains. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of</span>
<span 
class="phvro8t-x-x-120">Sciences</span>, 99(20):12709&#8211;12714.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K., Wang, L.&#x00A0;K., Lima, C.&#x00A0;D., and Shuman, S. (2004). Structure and mechanism of RNA
ligase. <span 
class="phvro8t-x-x-120">Structure (London, England: 1993)</span>, 12(2):327&#8211;339.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHorvath2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Horvath,  G.&#x00A0;C.,  Kistler,  M.&#x00A0;K.,  and  Kistler,  W.&#x00A0;S.  (2009).   RFX2  is  a  candidate  downstream
amplifier of A-MYB regulation in mouse spermatogenesis. <span 
class="phvro8t-x-x-120">BMC Developmental Biology</span>, 9(1):63.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHosokawa2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hosokawa,  M.,  Shoji,  M.,  Kitamura,  K.,  Tanaka,  T.,  Noce,  T.,  Chuma,  S.,  and  Nakatsuji,  N.
(2007). Tudor-related proteins TDRD1/MTR-1, TDRD6 and TDRD7/TRAP: domain composition,
intracellular  localization,  and  function  in  male  germ  cells  in  mice.    <span 
class="phvro8t-x-x-120">Developmental  biology</span>,
301(1):38&#8211;52.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHouse2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> House, A.&#x00A0;E. and Lynch, K.&#x00A0;W. (2008).  Regulation of alternative splicing: more than just the
ABCs. <span 
class="phvro8t-x-x-120">The Journal of Biological Chemistry</span>, 283(3):1217&#8211;21.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHouwing2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Houwing, S., Kamminga, L.&#x00A0;M., Berezikov, E., Cronembold, D., Girard, A., van&#x00A0;den Elst, H.,
Filippov, D.&#x00A0;V., Blaser, H., Raz, E., Moens, C.&#x00A0;B., Plasterk, R.&#x00A0;H., Hannon, G.&#x00A0;J., Draper, B.&#x00A0;W.,
and Ketting, R.&#x00A0;F. (2007). A Role for Piwi and piRNAs in Germ Cell Maintenance and Transposon
Silencing in Zebrafish. <span 
class="phvro8t-x-x-120">Cell</span>, 129(1):69&#8211;82.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHuang2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Huang, H., Gao, Q., Peng, X., Choi, S.-Y., Sarma, K., Ren, H., Morris, A.&#x00A0;J., and Frohman,
M.&#x00A0;a. (2011). piRNA-associated germline nuage formation and spermatogenesis require MitoPLD
                                                                                                    
                                                                                                    
profusogenic mitochondrial-surface lipid signaling. <span 
class="phvro8t-x-x-120">Developmental cell</span>, 20(3):376&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIngolia2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ingolia,   N.&#x00A0;T.,   Ghaemmaghami,   S.,   Newman,   J.   R.&#x00A0;S.,   and   Weissman,   J.&#x00A0;S.   (2009).
Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.
<span 
class="phvro8t-x-x-120">Science</span>, 324:218&#8211;223.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIngolia2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ingolia, N.&#x00A0;T.,  Lareau,  L.&#x00A0;F.,  and  Weissman,  J.&#x00A0;S.  (2011).    Ribosome  profiling  of  mouse
embryonic  stem  cells  reveals  the  complexity  and  dynamics  of  mammalian  proteomes.    <span 
class="phvro8t-x-x-120">Cell</span>,
147(4):789&#8211;802.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIpsaro2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ipsaro, J.&#x00A0;J.,  Haase,  A.&#x00A0;D.,  Knott,  S.&#x00A0;R.,  Joshua-Tor,  L.,  and  Hannon,  G.&#x00A0;J.  (2012).    The
structural biochemistry  of  Zucchini  implicates  it  as  a  nuclease  in  piRNA  biogenesis.   <span 
class="phvro8t-x-x-120">Nature</span>,
491(7423):279&#8211;83.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJin2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Jin, Y., Suzuki, H., Maegawa, S., Endo, H., Sugano, S., Hashimoto, K., Yasuda, K., and Inoue,
K. (2003).    A  vertebrate  RNA-binding  protein  Fox-1  regulates  tissue-specific  splicing  via  the
pentanucleotide GCAUG. <span 
class="phvro8t-x-x-120">The EMBO Journal</span>, 22(4):905&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJohnson2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Johnson,  B.&#x00A0;E.,  Glauser,  D.&#x00A0;A.,  Dan-Glauser,  E.&#x00A0;S.,  Halling,  D.&#x00A0;B.,  Aldrich,  R.&#x00A0;W.,  and
Goodman,  M.&#x00A0;B.  (2011).    Alternatively  Spliced  Domains  Interact  to  Regulate  BK  Potassium
Channel Gating. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>, 108(51):20784&#8211;20789.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJohnson2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Johnson, J.&#x00A0;M., Castle, J., Garrett-Engele, P., Kan, Z., Loerch, P.&#x00A0;M., Armour, C.&#x00A0;D., Santos,
R., Schadt, E.&#x00A0;E., Stoughton, R., and Shoemaker, D.&#x00A0;D. (2003). Genome-Wide Survey of Human
Alternative Pre-mRNA Splicing with Exon Junction Microarrays. <span 
class="phvro8t-x-x-120">Science</span>, 302(5653):2141&#8211;2144.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKennard1927"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kennard, E.&#x00A0;H. (1927). Zur Quantenmechanik einfacher Bewegungstypen. <span 
class="phvro8t-x-x-120">Zeitschrift for Physik</span>,
44(4-5):326&#8211;352.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKent2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kent, W.&#x00A0;J. (2002). BLAT&#8212;The BLAST-Like Alignment Tool. <span 
class="phvro8t-x-x-120">Genome Research</span>, 12(4):656&#8211;664.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKim2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kim, V.&#x00A0;N., Han, J., and Siomi, M.&#x00A0;C. (2009).  Biogenesis of small RNAs in animals.  <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">reviews. Molecular cell biology</span>, 10(2):126&#8211;39.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKleppe1970b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kleppe, K., Van de Sande, J.&#x00A0;H., and Khorana, H.&#x00A0;G. (1970).  Polynucleotide ligase-catalyzed
joining of deoxyribo-oligonucleotides on ribopolynucleotide templates and of ribo-oligonucleotides
on deoxyribopolynucleotide templates.  <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the</span>
<span 
class="phvro8t-x-x-120">United States of America</span>, 67(1):68&#8211;73.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKojima2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kojima, K.,  Kuramochi-Miyagawa,  S.,  Chuma,  S.,  Tanaka,  T.,  Nakatsuji,  N.,  Kimura,  T.,  and
Nakano, T.  (2009).    Associations  between  PIWI  proteins  and  TDRD1/MTR-1  are  critical  for
integrated subcellular localization in murine male germ cells. <span 
class="phvro8t-x-x-120">Genes to cells : devoted to molecular</span>
<span 
class="phvro8t-x-x-120">&amp; cellular mechanisms</span>, 14(10):1155&#8211;65.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKumar1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kumar, M. and Carmichael, G.&#x00A0;G. (1998).  Antisense RNA: function and fate of duplex RNA in
cells of higher eukaryotes. <span 
class="phvro8t-x-x-120">Microbiology and molecular biology reviews : MMBR</span>, 62(4):1415&#8211;34.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKuramochi-Miyagawa2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kuramochi-Miyagawa, S., Kimura, T., Ijiri, T.&#x00A0;W., Isobe, T., Asada, N., Fujita, Y., Ikawa, M., Iwai,
N., Okabe, M., Deng, W., Lin, H., Matsuda, Y., and Nakano, T. (2004). Mili, a Mammalian Member
of Piwi Family Gene, Is Essential for Spermatogenesis. <span 
class="phvro8t-x-x-120">Development</span>, 131(4):839&#8211;849.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKuramochi-Miyagawa2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kuramochi-Miyagawa, S., Watanabe, T., Gotoh, K., Totoki, Y., Toyoda, A., Ikawa, M., Asada, N.,
Kojima, K., Yamaguchi, Y., Ijiri, T.&#x00A0;W., Hata, K., Li, E., Matsuda, Y., Kimura, T., Okabe, M., Sakaki,
Y., Sasaki, H., and Nakano, T. (2008).  DNA Methylation of Retrotransposon Genes Is Regulated
by Piwi Family  Members  MILI  and  MIWI2  in  Murine  Fetal  Testes.    <span 
class="phvro8t-x-x-120">Genes  &amp;  Development</span>,
22(7):908&#8211;917.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKutter2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kutter, C.,  Brown,  G.&#x00A0;D.,  Gonalves,  A.,  Wilson,  M.&#x00A0;D.,  Watt,  S.,  Brazma,  A.,  White,  R.&#x00A0;J.,
and Odom, D.&#x00A0;T. (2011).  Pol III binding in six mammals shows conservation among amino acid
isotypes despite divergence among tRNA genes. <span 
class="phvro8t-x-x-120">Nature genetics</span>, 43(10):948&#8211;55.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLadd2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ladd, A.&#x00A0;N. and Cooper, T.&#x00A0;A. (2002).  Finding signals that regulate alternative splicing in the
post-genomic era. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 3(11):reviews0008.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLaiho2013a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Laiho, A., Kotaja, N., Gyenesei, A., and Sironen, A. (2013). Transcriptome profiling of the murine
testis during the first wave of spermatogenesis. <span 
class="phvro8t-x-x-120">PloS one</span>, 8(4):e61558.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLander2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lander, E.&#x00A0;S.  (2011).    Initial  impact  of  the  sequencing  of  the  human  genome.    <span 
class="phvro8t-x-x-120">Nature</span>,
470(7333):187&#8211;97.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLangmead2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.&#x00A0;L. (2009). Ultrafast and memory-efficient
alignment of short DNA sequences to the human genome. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 10(3):R25.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLatham1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Latham,  K.&#x00A0;E.,  Litvin,  J.,  Orth,  J.&#x00A0;M.,  Patel,  B.,  Mettus,  R.,  and  Reddy,  E.&#x00A0;P.  (1996).
Temporal patterns of A-myb and B-myb gene expression during testis development.  <span 
class="phvro8t-x-x-120">Oncogene</span>,
13:1161&#8211;1168.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                                    
                                                                                                    
<a 
 id="XLau2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lau, N.&#x00A0;C.,  Seto,  A.&#x00A0;G.,  Kim,  J.,  Kuramochi-Miyagawa,  S.,  Nakano,  T.,  Bartel,  D.&#x00A0;P.,  and
Kingston, R.&#x00A0;E. (2006).  Characterization of the piRNA complex from rat testes.  <span 
class="phvro8t-x-x-120">Science (New</span>
<span 
class="phvro8t-x-x-120">York, N.Y.)</span>, 313(5785):363&#8211;367.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lee, H.-C., Gu, W., Shirayama, M., Youngman, E., Conte, D., and Mello, C.&#x00A0;C. (2012). C. elegans
piRNAs mediate the genome-wide surveillance of germline transcripts. <span 
class="phvro8t-x-x-120">Cell</span>, 150(1):78&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLenasi2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lenasi, T., Peterlin, B.&#x00A0;M., and Dovc, P. (2006). Distal regulation of alternative splicing by splicing
enhancer in equine <span 
class="cmmi-12">&#x03B2;</span>-casein intron 1. <span 
class="phvro8t-x-x-120">RNA</span>, 12(3):498 &#8211;507.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2009d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth, G., Abecasis, G.,
and Durbin, R. (2009a).   The Sequence Alignment/Map format and SAMtools.   <span 
class="phvro8t-x-x-120">Bioinformatics</span>,
25:2078&#8211;2079.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2012c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, H., Qiu, J., and Fu, X.-D. (2012).  RASL-seq for massively parallel and quantitative analysis
of gene expression. <span 
class="phvro8t-x-x-120">Current protocols in molecular biology / edited by Frederick M. Ausubel ... [et</span>
<span 
class="phvro8t-x-x-120">al.]</span>, Chapter 4(April):Unit 4.13.1&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, M., Wang, I.&#x00A0;X., Li, Y., Bruzel, A., Richards, A.&#x00A0;L., Toung, J.&#x00A0;M., and Cheung, V.&#x00A0;G. (2011).
Widespread RNA and DNA sequence differences in the human transcriptome. <span 
class="phvro8t-x-x-120">Science (New York,</span>
<span 
class="phvro8t-x-x-120">N.Y.)</span>, 333(6038):53&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;C.,  Barringer,  B.&#x00A0;C.,  and  Barbash,  D.&#x00A0;a.  (2009b).   The  pachytene  checkpoint  and  its
relationship to evolutionary patterns of polyploidization and hybrid sterility. <span 
class="phvro8t-x-x-120">Heredity</span>, 102(1):24&#8211;30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;C. and Schimenti, J.&#x00A0;C. (2007).  Mouse pachytene checkpoint 2 (trip13) is required for
completing meiotic recombination but not synapsis. <span 
class="phvro8t-x-x-120">PLoS genetics</span>, 3(8):e130.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2013h"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;Z., Roy, C.&#x00A0;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&#x00A0;W., Xu, J., Moore, M.&#x00A0;J.,
Schimenti, J.&#x00A0;C., Weng, Z., and Zamore, P.&#x00A0;D. (2013a).  An Ancient Transcription Factor Initiates
the Burst of piRNA Production during Early Meiosis in Mouse Testes. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, 50(1):1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2013e"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;Z., Roy, C.&#x00A0;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&#x00A0;W., Xu, J., Moore, M.&#x00A0;J.,
Schimenti, J.&#x00A0;C., Weng, Z., and Zamore, P.&#x00A0;D. (2013b).  An Ancient Transcription Factor Initiates
the Burst of piRNA Production during Early Meiosis in Mouse Testes. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, pages 1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLicatalosi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Licatalosi, D.&#x00A0;D., Mele, A., Fak, J.&#x00A0;J., Ule, J., Kayikci, M., Chi, S.&#x00A0;W., Clark, T.&#x00A0;A., Schweitzer,
A.&#x00A0;C., Blume,  J.&#x00A0;E.,  Wang,  X.,  Darnell,  J.&#x00A0;C.,  and  Darnell,  R.&#x00A0;B.  (2008).    HITS-CLIP  yields
genome-wide insights into brain alternative RNA processing. <span 
class="phvro8t-x-x-120">Nature</span>, 456(7221):464&#8211;469.
                                                                                                    
                                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLister2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lister, R.,  O&#8217;Malley,  R.&#x00A0;C.,  Tonti-Filippini,  J.,  Gregory,  B.&#x00A0;D.,  Berry,  C.&#x00A0;C.,  Millar,  A.&#x00A0;H.,
and Ecker,  J.&#x00A0;R.  (2008).   Highly  integrated  single-base  resolution  maps  of  the  epigenome  in
Arabidopsis. <span 
class="phvro8t-x-x-120">Cell</span>, 133(3):523&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLong2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Long, J.&#x00A0;C.  and  Caceres,  J.&#x00A0;F.  (2009).    The  SR  protein  family  of  splicing  factors:  master
regulators of gene expression. <span 
class="phvro8t-x-x-120">The Biochemical journal</span>, 417(1):15&#8211;27.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLynch2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lynch, K.&#x00A0;W. (2004).  Consequences of regulated pre-mRNA splicing in the immune system.
<span 
class="phvro8t-x-x-120">Nat Rev Immunol</span>, 4(12):931&#8211;940.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMangan2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mangan, S. and Alon, U. (2003).  Structure and function of the feed-forward loop network motif.
<span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>, 100(21):11980 &#8211;11985.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMarioni2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Marioni, J.&#x00A0;C., Mason, C.&#x00A0;E., Mane, S.&#x00A0;M., Stephens, M., and Gilad, Y. (2008).  RNA-seq: an
assessment of technical reproducibility and comparison with gene expression arrays.  <span 
class="phvro8t-x-x-120">Genome</span>
<span 
class="phvro8t-x-x-120">Research</span>, 18(9):1509&#8211;17.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMartinez-Contreras2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Martinez-Contreras,  R.,  Cloutier,  P.,  Shkreta,  L.,  Fisette,  J.-F.,  Revil,  T.,  and  Chabot,  B.
(2007). hnRNP proteins and splicing control.  <span 
class="phvro8t-x-x-120">Advances in experimental medicine and biology</span>,
623:123&#8211;147.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMaxam1977a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Maxam, A.&#x00A0;M. and Gilbert, W. (1992). A new method for sequencing DNA. 1977. <span 
class="phvro8t-x-x-120">Biotechnology</span>
<span 
class="phvro8t-x-x-120">(Reading, Mass.)</span>, 24(2):99&#8211;103.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMettus1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mettus, R.&#x00A0;V.,  Litvin,  J.,  Wali,  A.,  Toscani,  A.,  Latham,  K.,  Hatton,  K.,  and  Reddy,  E.&#x00A0;P.
(1994). Murine A-myb: evidence for differential splicing and tissue-specific expression. <span 
class="phvro8t-x-x-120">Oncogene</span>,
9(10):3077&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModrek2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modrek,  B.  and  Lee,  C.  (2002).   A  genomic  view  of  alternative  splicing.   <span 
class="phvro8t-x-x-120">Nature  Genetics</span>,
30(1):13&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModrich1973a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modrich,   P.,   Anraku,   Y.,   and   Lehman,   I.&#x00A0;R.   (1973).       Deoxyribonucleic   Acid   Ligase
ISOLATION AND PHYSICAL CHARACTERIZATION OF THE HOMOGENEOUS ENZYME FROM
ESCHERICHIA COLI. <span 
class="phvro8t-x-x-120">Journal of Biological Chemistry</span>, 248(21):7495&#8211;7501.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModzelewski2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modzelewski,  A.&#x00A0;J.,  Holmes,  R.&#x00A0;J.,  Hilz,  S.,  Grimson,  A.,  and  Cohen,  P.&#x00A0;E.  (2012).   AGO4
regulates entry  into  meiosis  and  influences  silencing  of  sex  chromosomes  in  the  male  mouse
                                                                                                    
                                                                                                    
germline. <span 
class="phvro8t-x-x-120">Developmental cell</span>, 23(2):251&#8211;64.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMontgomery1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Montgomery,  M.&#x00A0;K.,  Xu,  S.,  and  Fire,  a.  (1998).     RNA  as  a  target  of  double-stranded
RNA-mediated  genetic  interference  in  Caenorhabditis  elegans.    <span 
class="phvro8t-x-x-120">Proceedings  of  the  National</span>
<span 
class="phvro8t-x-x-120">Academy of Sciences of the United States of America</span>, 95(26):15502&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMortazavi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mortazavi, A., Williams, B.&#x00A0;A., McCue, K., Schaeffer, L., and Wold, B. (2008).   Mapping and
quantifying mammalian transcriptomes by RNA-Seq. <span 
class="phvro8t-x-x-120">Nat Meth</span>, 5(7):621&#8211;628.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNagalakshmi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nagalakshmi, U., Wang, Z., Waern, K., Shou, C., Raha, D., Gerstein, M., and Snyder, M. (2008).
The transcriptional landscape of the yeast genome defined by RNA sequencing.  <span 
class="phvro8t-x-x-120">Science (New</span>
<span 
class="phvro8t-x-x-120">York, N.Y.)</span>, 320(5881):1344&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNakano2003a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nakano, M., Komatsu, J., Matsuura, S.-i., Takashima, K., Katsura, S., and Mizuno, A. (2003).
Single-molecule PCR using water-in-oil emulsion. <span 
class="phvro8t-x-x-120">Journal of Biotechnology</span>, 102(2):117&#8211;124.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNamekawa2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Namekawa, S.&#x00A0;H. and Lee, J.&#x00A0;T. (2009).  XY and ZW: is meiotic sex chromosome inactivation
the rule in evolution? <span 
class="phvro8t-x-x-120">PLoS genetics</span>, 5(5):e1000493.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNandakumar2004a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nandakumar, J., Ho, C.&#x00A0;K., Lima, C.&#x00A0;D., and Shuman, S. (2004). RNA substrate specificity and
structure-guided mutational analysis of bacteriophage T4 RNA ligase 2. <span 
class="phvro8t-x-x-120">The Journal of biological</span>
<span 
class="phvro8t-x-x-120">chemistry</span>, 279(30):31337&#8211;47.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNandakumar2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nandakumar, J., Shuman, S., and Lima, C.&#x00A0;D. (2006).  RNA ligase structures reveal the basis
for RNA specificity and conformational changes that drive ligation forward. <span 
class="phvro8t-x-x-120">Cell</span>, 127(1):71&#8211;84.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNEBEL1961"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> NEBEL,  B.&#x00A0;R.,  AMAROSE,  A.&#x00A0;P.,  and  HACKET,  E.&#x00A0;M.  (1961).    Calendar  of  gametogenic
development in the prepuberal male mouse. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 134(3482):832&#8211;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNewburger2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Newburger, D.&#x00A0;E. and Bulyk, M.&#x00A0;L. (2009).  UniPROBE: an online database of protein binding
microarray data on protein-DNA interactions. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 37(Database issue):D77&#8211;82.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsen2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsen, T.&#x00A0;W. and Graveley, B.&#x00A0;R. (2010).  Expansion of the eukaryotic proteome by alternative
splicing. <span 
class="phvro8t-x-x-120">Nature</span>, 463(7280):457&#8211;463.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsson2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsson, M., Antson, D.-O., Barbany, G., and Landegren, U. (2001). RNA-templated DNA ligation
for transcript analysis. <span 
class="phvro8t-x-x-120">Nucleic Acids Research</span>, 29(2):578&#8211;581.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsson2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsson, M.,  Barbany,  G.,  Antson,  D.&#x00A0;O.,  Gertow,  K.,  and  Landegren,  U.  (2000).   Enhanced
detection and distinction of RNA by enzymatic probe ligation. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 18(7):791&#8211;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNishimasu2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nishimasu, H., Ishizu, H., Saito, K., Fukuhara, S., Kamatani, M.&#x00A0;K., Bonnefond, L., Matsumoto,
N., Nishizawa, T., Nakanaga, K., Aoki, J., Ishitani, R., Siomi, H., Siomi, M.&#x00A0;C., and Nureki, O.
(2012). Structure  and  function  of  Zucchini  endoribonuclease  in  piRNA  biogenesis.    <span 
class="phvro8t-x-x-120">Nature</span>,
491(7423):284&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOakberg1956b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Oakberg, E. and Oakberq, F. (1956).   A Description of Spermiogenesis in the mouse and its
use in analysis of the cycle of the seminiferous epithelium and germ cell renewal.  <span 
class="phvro8t-x-x-120">The American</span>
<span 
class="phvro8t-x-x-120">journal of anatomy</span>, 99(3):391&#8211;419.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOh1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Oh, I.&#x00A0;H.  and  Reddy,  E.&#x00A0;P.  (1999).   The  myb  gene  family  in  cell  growth,  differentiation  and
apoptosis. <span 
class="phvro8t-x-x-120">Oncogene</span>, 18(19):3017&#8211;33.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOlivera1967b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Olivera, B. and Lehman, I. (1967). Linkage of polynucleotides through phosphodiester bonds by
an enzyme from Escherichia coli. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, pages 1426&#8211;1433.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOsella2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Osella,   M.,   Bosia,   C.,   Cor,   D.,   and   Caselle,   M.   (2011).       The   role   of   incoherent
microRNA-mediated   feedforward   loops   in   noise   buffering.      <span 
class="phvro8t-x-x-120">PLoS   computational   biology</span>,
7(3):e1001101.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPan2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Pan, Q.,  Shai,  O.,  Lee,  L.&#x00A0;J.,  Frey,  B.&#x00A0;J.,  and  Blencowe,  B.&#x00A0;J.  (2008).   Deep  surveying  of
alternative splicing complexity in the human transcriptome by high-throughput sequencing. <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Genetics</span>, 40(12):1413&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPane2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Pane, A.,  Wehr,  K.,  and  Schpbach,  T.  (2007).   zucchini  and  squash  encode  two  putative
nucleases required  for  rasiRNA  production  in  the  Drosophila  germline.    <span 
class="phvro8t-x-x-120">Developmental  cell</span>,
12(6):851&#8211;62.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPark2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Park, J.&#x00A0;W. and Graveley, B.&#x00A0;R. (2007). Complex alternative splicing. <span 
class="phvro8t-x-x-120">Advances in Experimental</span>
<span 
class="phvro8t-x-x-120">Medicine and Biology</span>, 623:50&#8211;63.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XParra2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Parra, M.&#x00A0;K.,  Gallagher,  T.&#x00A0;L.,  Amacher,  S.&#x00A0;L.,  Mohandas,  N.,  and  Conboy,  J.&#x00A0;G.  (2012).
Deep intron elements mediate nested splicing events at consecutive AG dinucleotides to regulate
alternative  3&#8217;  splice  site  choice  in  vertebrate  4.1  genes.     <span 
class="phvro8t-x-x-120">Molecular  and  cellular  biology</span>,
32(11):2044&#8211;53.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                                    
                                                                                                    
<a 
 id="XParra2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Parra, M.&#x00A0;K., Tan, J.&#x00A0;S., Mohandas, N., and Conboy, J.&#x00A0;G. (2008).   Intrasplicing coordinates
alternative first exons with alternative splicing in the protein 4.1R gene. <span 
class="phvro8t-x-x-120">EMBO J</span>, 27(1):122&#8211;131.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPeirson2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Peirson,  S.&#x00A0;N.  (2003).    Experimental  validation  of  novel  and  conventional  approaches  to
quantitative real-time PCR data analysis. <span 
class="phvro8t-x-x-120">Nucleic Acids Research</span>, 31(14):73e&#8211;73.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPelisson1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Plisson, a., Song, S.&#x00A0;U., Prud&#8217;homme, N., Smith, P.&#x00A0;a., Bucheton, a., and Corces, V.&#x00A0;G. (1994).
Gypsy transposition correlates with the production of a retroviral envelope-like protein under the
tissue-specific control of the Drosophila flamenco gene. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 13(18):4401&#8211;11.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPeng2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Peng, T., Xue, C., Bi, J., Li, T., Wang, X., Zhang, X., and Li, Y. (2008).  Functional importance
of different patterns of correlation between adjacent cassette exons in human and mouse.  <span 
class="phvro8t-x-x-120">BMC</span>
<span 
class="phvro8t-x-x-120">Genomics</span>, 9(1):191.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPlocik2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Plocik, A.&#x00A0;M.  and  Graveley,  B.&#x00A0;R.  (2013).     New  Insights  from  Existing  Sequence  Data:
Generating Breakthroughs without a Pipette. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, 49(4):605&#8211;617.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPonta2003b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ponta, H., Sherman, L., and Herrlich, P.&#x00A0;a. (2003). CD44: from adhesion molecules to signalling
regulators. <span 
class="phvro8t-x-x-120">Nature reviews. Molecular cell biology</span>, 4(1):33&#8211;45.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPurandare2014b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Purandare,  S.&#x00A0;R.,  Tenhumberg,  B.,  and  Brisson,  J.&#x00A0;a.  (2014).    Comparison  of  the  wing
polyphenic  response  of  pea  aphids  (  Acyrthosiphon  pisum  )  to  crowding  and  predator  cues.
<span 
class="phvro8t-x-x-120">Ecological Entomology</span>, 39(2):263&#8211;266.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XQuinlan2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Quinlan, A.&#x00A0;R. and Hall, I.&#x00A0;M. (2010). BEDTools: a flexible suite of utilities for comparing genomic
features. <span 
class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 26(6):841&#8211;842.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XReuter2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Reuter,   M.,   Berninger,   P.,   Chuma,   S.,   Shah,   H.,   Hosokawa,   M.,   Funaya,   C.,   Antony,
C., Sachidanandam,  R.,  and  Pillai,  R.&#x00A0;S.  (2011).      Miwi  catalysis  is  required  for  piRNA
amplification-independent LINE1 transposon silencing. <span 
class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XReuter2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Reuter, M.,  Chuma,  S.,  Tanaka,  T.,  Franz,  T.,  Stark,  A.,  and  Pillai,  R.&#x00A0;S.  (2009).    Loss
of the Mili-interacting  Tudor  domain-containing  protein-1  activates  transposons  and  alters  the
Mili-associated small RNA profile. <span 
class="phvro8t-x-x-120">Nature structural &amp; molecular biology</span>, 16(6):639&#8211;46.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRichardson1965a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Richardson,   C.   (1965).       Phosphorylation   of   Nucleic   Acid   by   and   Enzyme   from   T4
Bacteriophate-infected   Escherichia   Coli.       <span 
class="phvro8t-x-x-120">Proceedings   of   the   National   Academy   of   </span><span 
class="phvro8t-x-x-120">&#x2026;</span>,
1372(1961):158&#8211;165.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRiley2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Riley, K.&#x00A0;J. and Steitz, J.&#x00A0;A. (2013). Minireview The &#8216;&#8216; Observer Effect &#8217;&#8217; in Genome-wide Surveys
of Protein-RNA Interactions Minireview. pages 2011&#8211;2014.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRo2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ro, S., Park,  C.,  Song,  R.,  Nguyen,  D.,  Jin,  J.,  Sanders,  K.&#x00A0;M.,  McCarrey,  J.&#x00A0;R.,  and  Yan,
W. (2007).    Cloning  and  Expression  Profiling  of  Testis-Expressed  piRNA-Like  RNAs.    <span 
class="phvro8t-x-x-120">RNA</span>,
13(10):1693&#8211;1702.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRoach1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Roach, J. J.&#x00A0;C., Boysen, C., Wang, K., Hood, L., and Wang, I. K. A.&#x00A0;I. (1995).  Pairwise End
Sequencing: A unified approach to genomic mapping and sequencing. <span 
class="phvro8t-x-x-120">Genomics</span>, 353:345&#8211;353.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRobine2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Robine, N., Lau, N.&#x00A0;C., Balla, S., Jin, Z., Okamura, K., Kuramochi-Miyagawa, S., Blower, M.&#x00A0;D.,
and Lai, E.&#x00A0;C. (2009). A Broadly Conserved Pathway Generates 3&#8217; UTR-Directed Primary piRNAs.
<span 
class="phvro8t-x-x-120">Current Biology</span>, 19(24):2066&#8211;2076.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRomanienko2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Romanienko, P.&#x00A0;J. and Camerini-Otero, R.&#x00A0;D. (2000).  The mouse Spo11 gene is required for
meiotic chromosome synapsis. <span 
class="phvro8t-x-x-120">Molecular cell</span>, 6(5):975&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRonaghi1998a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ronaghi,  M.,  Uhln,  M.,  and  Nyrn,  P.  (1998).    A  sequencing  method  based  on  real-time
pyrophosphate. <span 
class="phvro8t-x-x-120">Science</span>, 281:363, 365.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRumsfeld2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Rumsfeld, D. (2011). Known and unknown: a memoir.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSaito2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Saito, K., Inagaki, S., Mituyama, T., Kawamura, Y., Ono, Y., Sakota, E., Kotani, H., Asai, K.,
Siomi, H., and Siomi, M.&#x00A0;C. (2009).  A regulatory circuit for piwi by the large Maf gene traffic jam
in Drosophila. <span 
class="phvro8t-x-x-120">Nature</span>, 461(7268):1296&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSalehi-Ashtiani2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Salehi-Ashtiani, K., Yang, X., Derti, A., Tian, W., Hao, T., Lin, C., Makowski, K., Shen, L., Murray,
R.&#x00A0;R., Szeto, D., Tusneem, N., Smith, D.&#x00A0;R., Cusick, M.&#x00A0;E., Hill, D.&#x00A0;E., Roth, F.&#x00A0;P., and Vidal, M.
(2008). Isoform discovery by targeted cloning, &#8217;deep-well&#8217; pooling and parallel sequencing. <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Methods</span>, 5(7):597&#8211;600.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1975a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F. and Coulson, A.&#x00A0;R. (1975).  A rapid method for determining sequences in DNA by
primed synthesis with DNA polymerase. <span 
class="phvro8t-x-x-120">Journal of molecular biology</span>, 94(3):441&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1978a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F.,  Coulson,  A.&#x00A0;R.,  Friedmann,  T.,  Air,  G.&#x00A0;M.,  Barrell,  B.&#x00A0;G.,  Brown,  N.&#x00A0;L.,  Fiddes,
J.&#x00A0;C., Hutchison, C.&#x00A0;A., Slocombe, P.&#x00A0;M., and Smith, M. (1978).   The nucleotide sequence of
bacteriophage phiX174. <span 
class="phvro8t-x-x-120">Journal of molecular biology</span>, 125(2):225&#8211;46.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1977b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F., Nicklen, S., and Coulson, A.&#x00A0;R. (1977).   DNA sequencing with chain-terminating
inhibitors. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of America</span>,
74(12):5463&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchena1995a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schena,  M.,  Shalon,  D.,  Davis,  R.&#x00A0;W.,  and  Brown,  P.&#x00A0;O.  (1995).   Quantitative  monitoring  of
gene expression  patterns  with  a  complementary  DNA  microarray.   <span 
class="phvro8t-x-x-120">Science  (New  York,  N.Y.)</span>,
270(5235):467&#8211;70.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchmucker2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schmucker, D., Clemens, J.&#x00A0;C., Shu, H., Worby, C.&#x00A0;A., Xiao, J., Muda, M., Dixon, J.&#x00A0;E., and
Zipursky, S. (2000).  Drosophila Dscam Is an Axon Guidance Receptor Exhibiting Extraordinary
Molecular Diversity. <span 
class="phvro8t-x-x-120">Cell</span>, 101(6):671&#8211;684.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchoenmakers2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schoenmakers, S., Wassenaar, E., Hoogerbrugge, J.&#x00A0;W., Laven, J. S.&#x00A0;E., Grootegoed, J.&#x00A0;A.,
and Baarends, W.&#x00A0;M. (2009).   Female meiotic sex chromosome inactivation in chicken.   <span 
class="phvro8t-x-x-120">PLoS</span>
<span 
class="phvro8t-x-x-120">genetics</span>, 5(5):e1000466.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchwarzbauer1983"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schwarzbauer, J.&#x00A0;E., Tamkun, J.&#x00A0;W., Lemischka, I.&#x00A0;R., and Hynes, R.&#x00A0;O. (1983). Three different
fibronectin mRNAs arise by alternative splicing within the coding region. <span 
class="phvro8t-x-x-120">Cell</span>, 35(2 Pt 1):421&#8211;31.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSeitz2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Seitz, H., Ghildiyal, M., and Zamore, P.&#x00A0;D. (2008).  Argonaute loading improves the 5&#8217; precision
of both MicroRNAs and their miRNA* strands in flies. <span 
class="phvro8t-x-x-120">Current biology : CB</span>, 18(2):147&#8211;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShalek2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shalek, A.&#x00A0;K., Satija, R., Adiconis, X., Gertner, R.&#x00A0;S., Gaublomme, J.&#x00A0;T., Raychowdhury, R.,
Schwartz, S., Yosef, N., Malboeuf, C., Lu, D., Trombetta, J.&#x00A0;T., Gennert, D., Gnirke, A., Goren,
A., Hacohen, N., Levin, J.&#x00A0;Z., Park, H., and Regev, A. (2013).  Single-cell transcriptomics reveals
bimodality in expression and splicing in immune cells. <span 
class="phvro8t-x-x-120">Nature</span>, pages 1&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShapiro2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shapiro, E., Biezuner, T., and Linnarsson, S. (2013). Single-cell sequencing-based technologies
will revolutionize whole-organism science. <span 
class="phvro8t-x-x-120">Nature reviews. Genetics</span>, 14(9):618&#8211;630.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSharp2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sharp, P.&#x00A0;A. (2014). Nobel Lectures in Physiology or Medicine 1991-1995.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShen-Orr2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shen-Orr, S.&#x00A0;S., Milo, R., Mangan, S., and Alon, U. (2002). Network motifs in the transcriptional
regulation network of Escherichia coli. <span 
class="phvro8t-x-x-120">Nature genetics</span>, 31(1):64&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure,  J.  and  Aiden,  E.&#x00A0;L.  (2012).   The  expanding  scope  of  DNA  sequencing.   <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Biotechnology</span>, 30(11):1084&#8211;1094.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure,  J.  and  Ji,  H.  (2008).   Next-generation  DNA  sequencing.   <span 
class="phvro8t-x-x-120">Nature  Biotechnology</span>,
26(10):1135&#8211;1145.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure,  J.,  Mitra,  R.&#x00A0;D.,  Varma,  C.,  and  Church,  G.&#x00A0;M.  (2004).    Advanced  sequencing
technologies: methods and goals. <span 
class="phvro8t-x-x-120">Nature Reviews. Genetics</span>, 5(5):335&#8211;44.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure, J., Porreca, G.&#x00A0;J., Reppas, N.&#x00A0;B., Lin, X., McCutcheon, J.&#x00A0;P., Rosenbaum, A.&#x00A0;M.,
Wang, M.&#x00A0;D.,  Zhang,  K.,  Mitra,  R.&#x00A0;D.,  and  Church,  G.&#x00A0;M.  (2005).   Accurate  Multiplex  Polony
Sequencing of an Evolved Bacterial Genome. <span 
class="phvro8t-x-x-120">Science</span>, 309(5741):1728&#8211;1732.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShepard2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shepard, P.&#x00A0;J., Choi, E.-A., Lu, J., Flanagan, L.&#x00A0;A., Hertel, K.&#x00A0;J., and Shi, Y. (2011).  Complex
and dynamic landscape of RNA polyadenylation revealed by PAS-Seq. <span 
class="phvro8t-x-x-120">RNA</span>, 17(4):761 &#8211;772.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShingleton2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shingleton, A., Sisk, G., and Stern, D. (2003). Diapause in the pea aphid (Acyrthosiphon pisum)
is a slowing but not a cessation of development. <span 
class="phvro8t-x-x-120">BMC developmental biology</span>, 12:1&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShiraki2003a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shiraki, T., Kondo, S., Katayama, S., Waki, K., Kasukawa, T., Kawaji, H., Kodzius, R., Watahiki,
A., Nakamura, M., Arakawa, T., Fukuda, S., Sasaki, D., Podhajska, A., Harbers, M., Kawai, J.,
Carninci, P.,  and  Hayashizaki,  Y.  (2003).    Cap  analysis  gene  expression  for  high-throughput
analysis of transcriptional starting point and identification of promoter usage.  <span 
class="phvro8t-x-x-120">Proceedings of the</span>
<span 
class="phvro8t-x-x-120">National Academy of Sciences of the United States of America</span>, 100:15776&#8211;15781.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShirayama2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shirayama, M., Seth, M., Lee, H.-C., Gu, W., Ishidate, T., Conte, D., and Mello, C.&#x00A0;C. (2012).
piRNAs initiate  an  epigenetic  memory  of  nonself  RNA  in  the  C.  elegans  germline.     <span 
class="phvro8t-x-x-120">Cell</span>,
150(1):65&#8211;77.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShiroguchi2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shiroguchi,  K.,  Jia,  T.&#x00A0;Z.,  Sims,  P.&#x00A0;a.,  and  Xie,  X.&#x00A0;S.  (2012).     Digital  RNA  sequencing
minimizes sequence-dependent  bias  and  amplification  noise  with  optimized  single-molecule
barcodes.  <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of America</span>,
109(4):1347&#8211;52.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShoji2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shoji, M.,  Tanaka,  T.,  Hosokawa,  M.,  Reuter,  M.,  Stark,  A.,  Kato,  Y.,  Kondoh,  G.,  Okawa,
K., Chujo, T.,  Suzuki,  T.,  Hata,  K.,  Martin,  S.&#x00A0;L.,  Noce,  T.,  Kuramochi-Miyagawa,  S.,  Nakano,
T., Sasaki,  H.,  Pillai,  R.&#x00A0;S.,  Nakatsuji,  N.,  and  Chuma,  S.  (2009).      The  TDRD9-MIWI2
complex is essential for piRNA-mediated retrotransposon silencing in the mouse male germline.
<span 
class="phvro8t-x-x-120">Developmental cell</span>, 17(6):775&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSiomi2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Siomi, M.&#x00A0;C., Sato, K., Pezic, D., and Aravin, A.&#x00A0;A. (2011).  PIWI-interacting small RNAs: the
vanguard of genome defence. <span 
class="phvro8t-x-x-120">Nat Rev Mol Cell Biol</span>, 12(4):246&#8211;258.
                                                                                                    
                                                                                                    
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSmith2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Smith, J.&#x00A0;M., Bowles, J., Wilson, M., Teasdale, R.&#x00A0;D., and Koopman, P. (2004).  Expression of
the tudor-related gene Tdrd5 during development of the male germline in mice. <span 
class="phvro8t-x-x-120">Gene expression</span>
<span 
class="phvro8t-x-x-120">patterns : GEP</span>, 4(6):701&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSouthern2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Southern, E.&#x00A0;M. (2001). DNA microarrays. History and overview. <span 
class="phvro8t-x-x-120">Methods in Molecular Biology</span>
<span 
class="phvro8t-x-x-120">(Clifton, N.J.)</span>, 170:1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSriskanda1998c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sriskanda, V. and Shuman, S. (1998). Specificity and fidelity of strand joining by Chlorella virus
DNA ligase. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 26(15):3536&#8211;41.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XStaden1979"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Staden, R. (1979). A strategy of DNA sequencing employing computer programs. <span 
class="phvro8t-x-x-120">Nucleic Acids</span>
<span 
class="phvro8t-x-x-120">Research</span>, 6(7):2601&#8211;2610.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSultan2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sultan, M.,  Schulz,  M.&#x00A0;H.,  Richard,  H.,  Magen,  A.,  Klingenhoff,  A.,  Scherf,  M.,  Seifert,  M.,
Borodina, T., Soldatov, A., Parkhomchuk, D., Schmidt, D., O&#8217;Keeffe, S., Haas, S., Vingron, M.,
Lehrach, H., and Yaspo, M.-L. (2008).  A Global View of Gene Activity and Alternative Splicing by
Deep Sequencing of the Human Transcriptome. <span 
class="phvro8t-x-x-120">Science</span>, 321(5891):956&#8211;960.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSummers1970b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Summers, W. and Siegel, R. (1970).  Transcription of Late Phage RNA by T7 RNA Polymerase.
<span 
class="phvro8t-x-x-120">Nature</span>, 228:1160&#8211;1162.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTabor1987a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Tabor, S. (1987). DNA Ligases. <span 
class="phvro8t-x-x-120">Current Protocols in Molecular Biology</span>, pages 3.14.1&#8211;3.14.4.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTazi2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Tazi, J., Bakkour, N., and Stamm, S. (2009).  Alternative splicing and disease.  <span 
class="phvro8t-x-x-120">Biochimica et</span>
<span 
class="phvro8t-x-x-120">biophysica acta</span>, 1792(1):14&#8211;26.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XThomson2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Thomson, T. and Lin, H. (2009).   The biogenesis and function of PIWI proteins and piRNAs:
progress and prospect. <span 
class="phvro8t-x-x-120">Annual review of cell and developmental biology</span>, 25:355&#8211;76.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XToscani1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Toscani, A., Mettus, R.&#x00A0;V., Coupland, R., Simpkins, H., Litvin, J., Orth, J., Hatton, K.&#x00A0;S., and
Reddy, E.&#x00A0;P. (1997). Arrest of spermatogenesis and defective breast development in mice lacking
A-myb. <span 
class="phvro8t-x-x-120">Nature</span>, 386(6626):713&#8211;717.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrapnell2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell, C., Pachter, L., and Salzberg, S.&#x00A0;L. (2009).  TopHat: discovering splice junctions with
RNA-Seq. <span 
class="phvro8t-x-x-120">Bioinformatics</span>, 25(9):1105 &#8211;1111.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                                    
                                                                                                    
<a 
 id="XTrapnell2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell, C., Roberts, A., Goff, L., Pertea, G., Kim, D., Kelley, D.&#x00A0;R., Pimentel, H., Salzberg,
S.&#x00A0;L., Rinn, J.&#x00A0;L., and Pachter, L. (2012).  Differential gene and transcript expression analysis of
RNA-seq experiments with TopHat and Cufflinks. <span 
class="phvro8t-x-x-120">Nature Protocols</span>, 7(3):562&#8211;578.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrapnell2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell, C., Williams, B.&#x00A0;A., Pertea, G., Mortazavi, A., Kwan, G., van Baren, M.&#x00A0;J., Salzberg,
S.&#x00A0;L., Wold, B.&#x00A0;J., and Pachter, L. (2010).  Transcript assembly and quantification by RNA-Seq
reveals unannotated  transcripts  and  isoform  switching  during  cell  differentiation.   <span 
class="phvro8t-x-x-120">Nat  Biotech</span>,
28(5):511&#8211;515.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrauth1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trauth, K., Mutschler, B., Jenkins, N.&#x00A0;A., Gilbert, D.&#x00A0;J., Copeland, N.&#x00A0;G., and Klempnauer, K.-h.
(1994). Mouse A-myb encodes a trans-activator and is expressed in mitotically active cells of the
developing central nervous system, adult testis and B lymphocytes. <span 
class="phvro8t-x-x-120">EMBO</span>, 13(24):5994&#8211;6005.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XUle2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ule, J., Jensen,  K.,  Mele,  A.,  and  Darnell,  R.&#x00A0;B.  (2005).    CLIP:  A  method  for  identifying
protein-RNA interaction sites in living cells. <span 
class="phvro8t-x-x-120">Methods</span>, 37(4):376&#8211;386.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V.,  Klenov,  M.&#x00A0;S.,  Stolyarenko,  A.&#x00A0;D.,  and  Kotelnikov,  R.&#x00A0;N.  (2004).    The  RNA
Interference Proteins and Vasa Locus are Involved in the Silencing of Retrotransposons in the
Female Germline of Drosophila melanogaster. <span 
class="phvro8t-x-x-120">RNA Biology</span>, (June):54&#8211;58.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V., Sigova, A., Li, C., Seitz, H., Gvozdev, V., and Zamore, P.&#x00A0;D. (2006). A Distinct Small
RNA Pathway Silences Selfish Genetic Elements in the Germline. <span 
class="phvro8t-x-x-120">Science</span>, 313(5785):320 &#8211;324.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V.,  Wohlschlegel,  J.,  Qu,  J.,  Jonsson,  Z.,  Huang,  X.,  Chuma,  S.,  Girard,  A.,
Sachidanandam, R., Hannon, G.&#x00A0;J., and Aravin, A.&#x00A0;a. (2009).  Proteomic analysis of murine Piwi
proteins reveals a role for arginine methylation in specifying interaction with Tudor family members.
<span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 23(15):1749&#8211;62.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVasileva2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vasileva, A., Tiedau, D., Firooznia, A., Mller-Reichert, T., and Jessberger, R. (2009).  Tdrd6 is
required for spermiogenesis, chromatoid body architecture, and regulation of miRNA expression.
<span 
class="phvro8t-x-x-120">Current biology : CB</span>, 19(8):630&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVelculescu1995a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Velculescu, V.&#x00A0;E., Zhang, L., Vogelstein, B., and Kinzler, K.&#x00A0;W. (1995).  Serial analysis of gene
expression. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 270(5235):484&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVenter2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Venter, J. (2007). <span 
class="phvro8t-x-x-120">A life decoded: my genome, my life</span>. Penguin (Non-Classics).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVenter2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Venter, J.&#x00A0;C.,  Adams,  M.&#x00A0;D.,  Myers,  E.&#x00A0;W.,  Li,  P.&#x00A0;W.,  Mural,  R.&#x00A0;J.,  Sutton,  G.&#x00A0;G.,  Smith,
H.&#x00A0;O., Yandell,  M.,  Evans,  C.&#x00A0;a.,  Holt,  R.&#x00A0;a.,  Gocayne,  J.&#x00A0;D.,  Amanatides,  P.,  Ballew,  R.&#x00A0;M.,
                                                                                                    
                                                                                                    
Huson, D.&#x00A0;H., Wortman, J.&#x00A0;R., Zhang, Q., Kodira, C.&#x00A0;D., Zheng, X.&#x00A0;H., Chen, L., Skupski, M.,
Subramanian, G., Thomas, P.&#x00A0;D., Zhang, J., Gabor Miklos, G.&#x00A0;L., Nelson, C., Broder, S., Clark,
a.&#x00A0;G., Nadeau, J., McKusick, V.&#x00A0;a., Zinder, N., Levine, a.&#x00A0;J., Roberts, R.&#x00A0;J., Simon, M., Slayman,
C., Hunkapiller, M., Bolanos, R., Delcher, A., Dew, I., Fasulo, D., Flanigan, M., Florea, L., Halpern,
A., Hannenhalli, S., Kravitz, S., Levy, S., Mobarry, C., Reinert, K., Remington, K., Abu-Threideh,
J., Beasley, E., Biddick, K., Bonazzi, V., Brandon, R., Cargill, M., Chandramouliswaran, I., Charlab,
R., Chaturvedi, K., Deng, Z., Di Francesco, V., Dunn, P., Eilbeck, K., Evangelista, C., Gabrielian,
a.&#x00A0;E., Gan, W., Ge, W., Gong, F., Gu, Z., Guan, P., Heiman, T.&#x00A0;J., Higgins, M.&#x00A0;E., Ji, R.&#x00A0;R., Ke, Z.,
Ketchum, K.&#x00A0;a., Lai, Z., Lei, Y., Li, Z., Li, J., Liang, Y., Lin, X., Lu, F., Merkulov, G.&#x00A0;V., Milshina, N.,
Moore, H.&#x00A0;M., Naik, a.&#x00A0;K., Narayan, V.&#x00A0;a., Neelam, B., Nusskern, D., Rusch, D.&#x00A0;B., Salzberg, S.,
Shao, W., Shue, B., Sun, J., Wang, Z., Wang, A., Wang, X., Wang, J., Wei, M., Wides, R., Xiao, C.,
Yan, C., Yao, A., Ye, J., Zhan, M., Zhang, W., Zhang, H., Zhao, Q., Zheng, L., Zhong, F., Zhong,
W., Zhu, S., Zhao, S., Gilbert, D., Baumhueter, S., Spier, G., Carter, C., Cravchik, A., Woodage,
T., Ali, F., An, H., Awe, A., Baldwin, D., Baden, H., Barnstead, M., Barrow, I., Beeson, K., Busam,
D., Carver, A., Center, A., Cheng, M.&#x00A0;L., Curry, L., Danaher, S., Davenport, L., Desilets, R., Dietz,
S., Dodson, K., Doup, L., Ferriera, S., Garg, N., Gluecksmann, A., Hart, B., Haynes, J., Haynes,
C., Heiner, C., Hladun, S., Hostin, D., Houck, J., Howland, T., Ibegwam, C., Johnson, J., Kalush,
F., Kline, L., Koduru, S., Love, A., Mann, F., May, D., McCawley, S., McIntosh, T., McMullen, I.,
Moy, M., Moy, L., Murphy, B., Nelson, K., Pfannkoch, C., Pratts, E., Puri, V., Qureshi, H., Reardon,
M., Rodriguez, R., Rogers, Y.&#x00A0;H., Romblad, D., Ruhfel, B., Scott, R., Sitter, C., Smallwood, M.,
Stewart, E., Strong, R., Suh, E., Thomas, R., Tint, N.&#x00A0;N., Tse, S., Vech, C., Wang, G., Wetter, J.,
Williams, S., Williams, M., Windsor, S., Winn-Deen, E., Wolfe, K., Zaveri, J., Zaveri, K., Abril, J.&#x00A0;F.,
Guig, R., Campbell, M.&#x00A0;J., Sjolander, K.&#x00A0;V., Karlak, B., Kejariwal, A., Mi, H., Lazareva, B., Hatton,
T., Narechania, A., Diemer, K., Muruganujan, A., Guo, N., Sato, S., Bafna, V., Istrail, S., Lippert,
R., Schwartz, R., Walenz, B., Yooseph, S., Allen, D., Basu, A., Baxendale, J., Blick, L., Caminha,
M., Carnes-Stine, J., Caulk, P., Chiang, Y.&#x00A0;H., Coyne, M., Dahlke, C., Mays, A., Dombroski, M.,
Donnelly, M., Ely, D., Esparham, S., Fosler, C., Gire, H., Glanowski, S., Glasser, K., Glodek, A.,
Gorokhov, M., Graham, K., Gropman, B., Harris, M., Heil, J., Henderson, S., Hoover, J., Jennings,
D., Jordan, C., Jordan, J., Kasha, J., Kagan, L., Kraft, C., Levitsky, A., Lewis, M., Liu, X., Lopez,
J., Ma, D., Majoros, W., McDaniel, J., Murphy, S., Newman, M., Nguyen, T., Nguyen, N., Nodell,
M., Pan, S., Peck, J., Peterson, M., Rowe, W., Sanders, R., Scott, J., Simpson, M., Smith, T.,
Sprague, A., Stockwell, T., Turner, R., Venter, E., Wang, M., Wen, M., Wu, D., Wu, M., Xia, A.,
Zandieh, A., and Zhu, X. (2001). The sequence of the human genome. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
291(5507):1304&#8211;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XViollet2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Viollet, S., Fuchs, R.&#x00A0;T., Munafo, D.&#x00A0;B., Zhuang, F., and Robb, G.&#x00A0;B. (2011).  T4 RNA Ligase 2
truncated active site mutants: improved tools for RNA analysis. <span 
class="phvro8t-x-x-120">BMC Biotechnology</span>, 11(1):72.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVourekas2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vourekas, A., Zheng, Q., Alexiou, P., Maragkakis, M., Kirino, Y., Gregory, B.&#x00A0;D., and Mourelatos,
Z. (2012). Mili and Miwi target RNA repertoire reveals piRNA biogenesis and function of Miwi in
spermiogenesis. <span 
class="phvro8t-x-x-120">Nature Structural &amp; Molecular Biology</span>, 19(8):773&#8211;781.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wang, E.&#x00A0;T.,  Sandberg,  R.,  Luo,  S.,  Khrebtukova,  I.,  Zhang,  L.,  Mayr,  C.,  Kingsmore,  S.&#x00A0;F.,
Schroth, G.&#x00A0;P.,  and  Burge,  C.&#x00A0;B.  (2008).     Alternative  isoform  regulation  in  human  tissue
transcriptomes. <span 
class="phvro8t-x-x-120">Nature</span>, 456(7221):470&#8211;6.
</p>
                                                                                                    
                                                                                                    
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2002b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wang, L.&#x00A0;K.,  Lima,  C.&#x00A0;D.,  and  Shuman,  S.  (2002).     Structure  and  mechanism  of  T4
polynucleotide kinase: an RNA repair enzyme. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 21(14):3873&#8211;80.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatanabe2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watanabe, T., Chuma, S., Yamamoto, Y., Kuramochi-Miyagawa, S., Totoki, Y., Toyoda, A., Hoki,
Y., Fujiyama, A., Shibata, T., Sado, T., Noce, T., Nakano, T., Nakatsuji, N., Lin, H., and Sasaki, H.
(2011). MITOPLD Is a Mitochondrial Protein Essential for Nuage Formation and piRNA Biogenesis
in the Mouse Germline. <span 
class="phvro8t-x-x-120">Developmental Cell</span>, 20(3):364&#8211;375.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson, F.&#x00A0;L., Pttmann-Holgado, R., Thomas, F., Lamar, D.&#x00A0;L., Hughes, M., Kondo, M., Rebel,
V.&#x00A0;I., and Schmucker,  D.  (2005).   Extensive  diversity  of  Ig-superfamily  proteins  in  the  immune
system of insects. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 309(5742):1874&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson, J.&#x00A0;D., Gann, A., and Witkowski, J. (2012). <span 
class="phvro8t-x-x-120">The Annotated and Illustrated Double Helix</span>.
Simon &amp; Schuster.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson1953a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson,  J.  D.&#x00A0;J.  and  Crick,  F.  (1953).     Molecular  structure  of  nucleic  acids.     <span 
class="phvro8t-x-x-120">Nature</span>,
4356(171):737&#8211;738.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWei1975"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wei, C.&#x00A0;M., Gershowitz, a., and Moss, B. (1975).  Methylated nucleotides block 5&#8217; terminus of
HeLa cell messenger RNA. <span 
class="phvro8t-x-x-120">Cell</span>, 4(4):379&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWeiss1967a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Weiss, B. and Richardson, C. (1967). Enzymatic breakage and joining of deoxyribonucleic acid,
I. Repair of single-strand breaks in DNA by an enzyme system from Escherichia coli infected with
T4 bacteriophage. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of </span><span 
class="phvro8t-x-x-120">&#x2026;</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWhite2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> White, E.&#x00A0;S. and Muro, A.&#x00A0;F. (2011).  Fibronectin splice variants: understanding their multiple
roles in health and disease using engineered mouse models. <span 
class="phvro8t-x-x-120">IUBMB life</span>, 63(7):538&#8211;46.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWills2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wills, Q.&#x00A0;F., Livak, K.&#x00A0;J., Tipping, A.&#x00A0;J., Enver, T., Goldson, A.&#x00A0;J., Sexton, D.&#x00A0;W., and Holmes, C.
(2013). Single-cell gene expression analysis reveals genetic associations masked in whole-tissue
experiments. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 31(8):748&#8211;752.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYabuta2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yabuta, Y.,  Ohta,  H.,  Abe,  T.,  Kurimoto,  K.,  Chuma,  S.,  and  Saitou,  M.  (2011).   TDRD5  is
required for retrotransposon silencing, chromatoid body assembly, and spermiogenesis in mice.
<span 
class="phvro8t-x-x-120">The Journal of cell biology</span>, 192(5):781&#8211;95.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYamakawa1998b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yamakawa,  K.,  Huot,  Y.&#x00A0;K.,  Haendelt,  M.&#x00A0;a.,  Hubert,  R.,  Chen,  X.&#x00A0;N.,  Lyons,  G.&#x00A0;E.,  and
Korenberg, J.&#x00A0;R. (1998).  DSCAM: a novel member of the immunoglobulin superfamily maps in
                                                                                                    
                                                                                                    
a Down syndrome  region  and  is  involved  in  the  development  of  the  nervous  system.   <span 
class="phvro8t-x-x-120">Human</span>
<span 
class="phvro8t-x-x-120">molecular genetics</span>, 7(2):227&#8211;37.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYang2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yang, Y., Zhan, L., Zhang, W., Sun, F., Wang, W., Tian, N., Bi, J., Wang, H., Shi, D., Jiang, Y.,
Zhang, Y., and Jin, Y. (2011).  RNA secondary structure in mutually exclusive splicing.  <span 
class="phvro8t-x-x-120">Nat Struct</span>
<span 
class="phvro8t-x-x-120">Mol Biol</span>, 18(2):159&#8211;168.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYeakley2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yeakley, J.&#x00A0;M., Fan, J.-B., Doucet, D., Luo, L., Wickham, E., Ye, Z., Chee, M.&#x00A0;S., and Fu, X.-D.
(2002). Profiling alternative splicing on fiber-optic arrays. <span 
class="phvro8t-x-x-120">Nature Biotechnology</span>, 20(4):353&#8211;358.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYin2003d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yin, S., Ho, C.&#x00A0;K., and Shuman, S. (2003). Structure-function analysis of T4 RNA ligase 2. <span 
class="phvro8t-x-x-120">The</span>
<span 
class="phvro8t-x-x-120">Journal of biological chemistry</span>, 278(20):17601&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhan2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhan, X.-L., Clemens, J.&#x00A0;C., Neves, G., Hattori, D., Flanagan, J.&#x00A0;J., Hummel, T., Vasconcelos,
M.&#x00A0;L., Chess, A., and Zipursky, S.&#x00A0;L. (2004).   Analysis of Dscam Diversity in Regulating Axon
Guidance in Drosophila Mushroom Bodies. <span 
class="phvro8t-x-x-120">Neuron</span>, 43(5):673&#8211;686.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, F., Wang, J., Xu, J., Zhang, Z., Koppetsch, B.&#x00A0;S., Schultz, N., Vreven, T., Meignin, C.,
Davis, I., Zamore, P.&#x00A0;D., Weng, Z., and Theurkauf, W.&#x00A0;E. (2012a). UAP56 Couples piRNA Clusters
to the Perinuclear Transposon Silencing Machinery. <span 
class="phvro8t-x-x-120">Cell</span>, 151(4):871&#8211;884.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, Y.,  Liu,  T.,  Meyer,  C.&#x00A0;a.,  Eeckhoute,  J.,  Johnson,  D.&#x00A0;S.,  Bernstein,  B.&#x00A0;E.,  Nusbaum,
C., Myers, R.&#x00A0;M., Brown, M., Li, W., and Liu, X.&#x00A0;S. (2008).  Model-based analysis of ChIP-Seq
(MACS). <span 
class="phvro8t-x-x-120">Genome biology</span>, 9(9):R137.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, Z., Theurkauf, W.&#x00A0;E., Weng, Z., and Zamore, P.&#x00A0;D. (2012b).   Strand-specific libraries
for high throughput RNA sequencing ( RNA-Seq ) prepared without poly ( A ) selection.  <span 
class="phvro8t-x-x-120">Silence</span>,
3(1):9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhu2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhu, J., Shendure,  J.,  Mitra,  R.&#x00A0;D.,  and  Church,  G.&#x00A0;M.  (2003).   Single  molecule  profiling  of
alternative pre-mRNA splicing. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 301(5634):836&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZikherman2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zikherman,  J.  and  Weiss,  A.  (2008).   Alternative  Splicing  of  CD45:  The  Tip  of  the  Iceberg.
<span 
class="phvro8t-x-x-120">Immunity</span>, 29(6):839&#8211;841.
</p>
</div>
 
</body></html> 

                                                                                                    


