<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"  
  "http://www.w3.org/TR/html4/loose.dtd">  
<html > 
<head><title></title> 
<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"> 
<meta name="generator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<meta name="originator" content="TeX4ht (http://www.tug.org/tex4ht/)"> 
<!-- html,htex4ht --> 
<meta name="src" content="RoyC_Umass_Thesis.v0.5.tex"> 
<meta name="date" content="2014-04-10 22:30:00"> 
<link rel="stylesheet" type="text/css" href="RoyC_Umass_Thesis.v0.5.css"> 
</head><body 
>
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 4--><p class="noindent" >
                                                                                

                                                                                
<div class="center" 
>
<!--l. 5--><p class="noindent" >
<!--l. 6--><p class="noindent" ><span 
class="phvr8t-x-x-248">"Putting the Pieces Together: Exons</span>
<span 
class="phvr8t-x-x-248">and piRNAs"</span><br /><br />
<span 
class="phvr8t-x-x-144">A Dissertation Presented</span><br /><br />
<span 
class="phvr8t-x-x-144">By</span><br /><br />
<span 
class="phvr8t-x-x-144">Christian K. Roy</span><br /><br />
<span 
class="phvr8t-x-x-144">Submitted to the Faculty of the University of the</span><br />
<span 
class="phvr8t-x-x-144">Massachusetts Graduate School of Biomedical Sciences,</span>
<span 
class="phvr8t-x-x-144">Worcester</span><br />
<span 
class="phvr8t-x-x-144">in partial filfillment of the requirements</span><br />
<span 
class="phvr8t-x-x-144">for the degree of</span><br /><br />
<span 
class="phvr8t-x-x-144">DOCTOR OF PHILOSOPHY</span><br /><br />
<span 
class="phvr8t-x-x-144">MAY 21st 2014</span><br />
                                                                                

                                                                                
<br /><span 
class="phvr8t-x-x-144">BIOCHEMISTRY</span><br /><br />
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 4--><p class="noindent" >
                                                                                

                                                                                
<a 
 id="x1-1doc"></a>
<div class="center" 
>
<!--l. 5--><p class="noindent" >
<!--l. 6--><p class="noindent" >"Putting the Pieces Together: Exons and piRNAs"<br />
A Dissertation Presented<br />
By<br />
Christian K. Roy<br />
The signatures of the Dissertation Defense Committee signify<br />
completion and approval as to style and content of the Dissertation<br /><br />
_________________________________________________________
                                                         <br />Melissa J. Moore, Co-Thesis Advisor<br /><br />
_________________________________________________________
                                                         <br />Phillip D. Zamore, Co-Thesis Advisor<br /><br />
_________________________________________________________
                                                         <br />Scot Wolfe, Member of Committee<br /><br />
_________________________________________________________
                                                         <br />Job Dekker, Member of Committee<br />
The signature of the Chair of the Committee signifies that the written
dissertation meets the requirements of the Dissertation Committee<br /><br />
_________________________________________________________
                                                         <br />Zhiping Weng, Chair of Committee<br />
The signature of the Dean of the Graduate School of Biomedical Sciences
signifies that the student has met all graduation requirements of the school.<br />
                                                                                

                                                                                
<br />_________________________________________________________
                                                         <br />Anthony Carruthers, Ph.D.,<br />
Dean of the Graduate School of Biomedical Sciences<br />
Biochemistry and Molecular Pharmacology<br />
MAY 21st 2014
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<a 
 id="x1-2r1"></a>
<a 
 id="Q1-1-1"></a>
<div class="center" 
>
<!--l. 6--><p class="noindent" >
<!--l. 6--><p class="noindent" ><a 
href="http://www.umassmed.edu)" >UNIVERSITY OF MASSACHUSETTS MEDICAL SCHOOL</a>
<!--l. 6--><p class="noindent" ><span 
class="phvro8t-x-x-248">Abstract</span>
<!--l. 6--><p class="noindent" ><a 
href="http://www.umassmed.edu/bmp/index.aspx(include" http://) >BIOCHEMISTRY AND MOLECULAR PHARMACOLOGY</a>
<!--l. 6--><p class="noindent" >Doctor of Philosophy
<!--l. 6--><p class="noindent" ><span 
class="phvb8t-x-x-120">"Putting the Pieces Together: Exons and piRNAs"</span>
<!--l. 6--><p class="noindent" >by Christian K. <span 
class="phvrc8t-x-x-120">R<span 
class="small-caps">O</span><span 
class="small-caps">Y</span></span>
</div>
                                                                                

                                                                                
<!--l. 8--><p class="noindent" >Analysis of gene expression has undergone a technical revolution. What was
impossible 6 years ago is now routine. The application of high-throughput DNA
sequencing machines capable of generating hundreds of millions of reads allows,
indeed forces, a major revision in how we think about the genome&#8217;s functional
output&#8211;the transcriptome.
<!--l. 10--><p class="noindent" >This thesis examines the history of DNA sequencing, the application of sequencing to
the measurement of gene expression, and some of the more interesting
features of transcripts, including Alternative Splicing and mammalian piRNA
biogenesis. Examination of these topics is framed around development of a novel
RNA-templated DNA:DNA ligation assay (SeqZip). SeqZip allows for a more efficient
analysis of the abundant and function long RNA molecules present in the
transcriptome.
<!--l. 12--><p class="noindent" >The thesis closes with a discussion on future applications of the SeqZip methodology,
when major advancements in high-throughput sequencing are a certainty. Finally,
challenges facing biomedical researches in the new climate of extremely large and
rich datasets are discussed.
                                                                                

                                                                                
<a 
 id="x1-3r2"></a>
<a 
 id="Q1-1-2"></a>
<div class="center" 
>
<!--l. 7--><p class="noindent" >
<!--l. 7--><p class="noindent" ><span 
class="phvro8t-x-x-248">Acknowledgements</span>
</div>
<!--l. 10--><p class="noindent" >First I would like to acknowledge and thank Laura Geagan and Rebecca Sendak at
Genzyme. They were my supervisors while I was a research associate there, and
assisted and encouraged my transition back to graduate school. Without the
confidence they instilled in my abilities as a young scientist in, I doubt I would have
ever signed up for more school.
<!--l. 12--><p class="noindent" >Next I&#8217;d like to thank Melissa. During my 1st year retreat at Wood&#8217;s Hole
I first learned that Melissa is a fantastic communicator of interesting and
important science. When she brought out her rope representing the unspliced
pre-mRNA of dystrophin&#8212;a rope that reached to the back of a rather large
auditorium&#8212;and then dramatically held up a No.2 pencil representing the final mRNA
product, both to scale, I knew the that I wanted to do my graduate research in
her lab. I have never once doubted the decision to join Melissa&#8217;s lab, and
have learned so much from the broad and interconnected approach she
takes to important scientific questions. Thank you so much for teaching me to
always consider the big picture, go for the answer, and to just ask when I need
help.
<!--l. 14--><p class="noindent" >Soon after joining Melissa&#8217;s lab, and a project going well, it was proposed to me that I
be a joint student between Melissa and Phil. It was not difficult to not jump at the
opportunity to be advised by two Howard Hughes Investigators, and I also
haven&#8217;t regretted the decision. Over the past few years, I have been continually
amazed at the depth of Phil&#8217;s knowledge, in scientific and general topics. He
is a careful, meticulous, quantitative, and calculating mentor. While I feel
that I clicked &#8216;on the level&#8217; with Melissa, interacting with Phil forced me to
think and act outside my comfort zone, something I always tell myself is
a critical aspect of change and growth. Thank you Phil for everything I&#8217;ve
learned.
                                                                                

                                                                                
<!--l. 16--><p class="noindent" >My committee has also been very supportive throughput my PhD. I hardly
believed the ease with which I passed my qualifying exam, and took it as a big
confidence boost. The following years of TRAC meetings confirmed that I was not
thinking way off-base. The one-on-one meetings just prior to my QE were
especially helpful. Thanks to Scot, Job, and especially Zhiping for all the
guidance.
<ul class="itemize1">
<li class="itemize">Lab Members; Aaron; Alper; Amrit
</li>
<li class="itemize">Eric and Erin
</li>
<li class="itemize">Collaborators
</li>
<li class="itemize">Dave Weaver
</li>
<li class="itemize">Muro
</li>
<li class="itemize">Graveley
</li>
<li class="itemize">Heinrich
</li>
<li class="itemize">Anna
</li>
<li class="itemize">Ogo
</li>
<li class="itemize">Family
</li>
<li class="itemize">Jul Owen
</li></ul>
                                                                                

                                                                                
                                                                                

                                                                                
<h2 class="likechapterHead"><a 
 id="x1-10002"></a>Contents</h2> <div class="tableofcontents">
<span class="chapterToc" ><a 
href="#Q1-1-1">Abstract</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-2">Acknowledgements</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-4">List of Figures</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-6">List of Tables</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-8">Abreviations</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-9">Symbols</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-10">Definitions</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-11">Dedication</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-12">Preface</a></span>
<br /><span class="chapterToc" >1 <a 
href="#x1-40001" id="QQ2-1-13">Introduction</a></span>
<br />&#x00A0;<span class="sectionToc" >1.1 <a 
href="#x1-50001" id="QQ2-1-14">Fixed Genomes and Flexible Genes</a></span>
<br />&#x00A0;<span class="sectionToc" >1.2 <a 
href="#x1-60002" id="QQ2-1-16">Nucleic Acid Sequencing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.1 <a 
href="#x1-70001" id="QQ2-1-17">DNA Sequencing History</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.2 <a 
href="#x1-80002" id="QQ2-1-18">History of High-throughput Sequencing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.3 <a 
href="#x1-90003" id="QQ2-1-20">Deep-sequencing of RNA</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.4 <a 
href="#x1-100004" id="QQ2-1-22">Measuring RNA Expression via Sequencing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.2.5 <a 
href="#x1-110005" id="QQ2-1-23">Assembling Transcripts from Short Reads</a></span>
<br />&#x00A0;<span class="sectionToc" >1.3 <a 
href="#x1-120003" id="QQ2-1-24">Nucleic Acid Splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.1 <a 
href="#x1-130001" id="QQ2-1-25">Alternative Splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.2 <a 
href="#x1-140002" id="QQ2-1-27">Deciphering a splicing code</a></span>
                                                                                

                                                                                
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.3 <a 
href="#x1-150003" id="QQ2-1-28">The Isoform Problem</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.4 <a 
href="#x1-160004" id="QQ2-1-30">Coordination in splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.5 <a 
href="#x1-170005" id="QQ2-1-32">Many isoforms per gene</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.3.6 <a 
href="#x1-180006" id="QQ2-1-36"><span 
class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></a></span>
<br />&#x00A0;<span class="sectionToc" >1.4 <a 
href="#x1-190004" id="QQ2-1-39">Nucleic Acid Ligation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.4.1 <a 
href="#x1-200001" id="QQ2-1-40">RNA Sequence investigation by ligation</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.4.2 <a 
href="#x1-210002" id="QQ2-1-42">T4 RNA Ligase 2 (Rnl2)</a></span>
<br />&#x00A0;<span class="sectionToc" >1.5 <a 
href="#x1-220005" id="QQ2-1-45">Nucleic Acid Polymers</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.5.1 <a 
href="#x1-230001" id="QQ2-1-46">piRNAs original from very long PolII Transcripts</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.1.1 <a 
href="#x1-240001" id="QQ2-1-47">Brief history of piRNAs</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.1.2 <a 
href="#x1-250002" id="QQ2-1-50">Fly and Mouse piRNAs have important differences</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.1.3 <a 
href="#x1-260003" id="QQ2-1-52">Known functions of mammalian piRNAs</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >1.5.1.4 <a 
href="#x1-270004" id="QQ2-1-53">Integration of multiple HTS datatypes of piRNA analysis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >1.5.2 <a 
href="#x1-280002" id="QQ2-1-54">Assembling full length transcripts from Short read data</a></span>
<br /><span class="chapterToc" >2 <a 
href="#x1-290002" id="QQ2-1-55">SeqZip - Development and Applications</a></span>
<br />&#x00A0;<span class="sectionToc" >2.1 <a 
href="#x1-300001" id="QQ2-1-56">SeqZip Overview</a></span>
<br />&#x00A0;<span class="sectionToc" >2.2 <a 
href="#x1-310002" id="QQ2-1-58">Multiplex Gene Study</a></span>
<br />&#x00A0;<span class="sectionToc" >2.3 <a 
href="#x1-320003" id="QQ2-1-61">Determining RNA integrity using SeqZip</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.1 <a 
href="#x1-330001" id="QQ2-1-62">Demonstration of Concept</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.2 <a 
href="#x1-340002" id="QQ2-1-65">Investigating HIV viral genome integrity using SeqZip</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >2.3.3 <a 
href="#x1-350003" id="QQ2-1-66">Design of HIV ligamers</a></span>
<br />&#x00A0;<span class="sectionToc" >2.4 <a 
href="#x1-360004" id="QQ2-1-68">Continuity of piRNA precursor transcripts</a></span>
<br /><span class="chapterToc" >3 <a 
href="#x1-370003" id="QQ2-1-76">SeqZip Publication</a></span>
<br />&#x00A0;<span class="sectionToc" >3.1 <a 
href="#x1-380001" id="QQ2-1-77">Abstract</a></span>
<br />&#x00A0;<span class="sectionToc" >3.2 <a 
href="#x1-390002" id="QQ2-1-78">Introduction</a></span>
<br />&#x00A0;<span class="sectionToc" >3.3 <a 
href="#x1-400003" id="QQ2-1-80">Results</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.1 <a 
href="#x1-410001" id="QQ2-1-81">Subsection 2</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >3.3.2 <a 
href="#x1-420002" id="QQ2-1-83">Subsection 2</a></span>
                                                                                

                                                                                
<br />&#x00A0;<span class="sectionToc" >3.4 <a 
href="#x1-430004" id="QQ2-1-85">Discussion</a></span>
<br />&#x00A0;<span class="sectionToc" >3.5 <a 
href="#x1-440005" id="QQ2-1-86">Methods</a></span>
<br />&#x00A0;<span class="sectionToc" >3.6 <a 
href="#x1-450006" id="QQ2-1-87">Supplemental Text</a></span>
<br /><span class="chapterToc" >4 <a 
href="#x1-460004" id="QQ2-1-88">An Ancient Transcription Factor Initiates the Burst of piRNA
Production during Early Meiosis in Mouse Testes</a></span>
<br />&#x00A0;<span class="sectionToc" >4.1 <a 
href="#x1-470001" id="QQ2-1-89">INTRODUCTION</a></span>
<br />&#x00A0;<span class="sectionToc" >4.2 <a 
href="#x1-480002" id="QQ2-1-90">RESULTS</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.1 <a 
href="#x1-490001" id="QQ2-1-91">Defining piRNA-Producing Transcripts in the Mouse Testis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.2 <a 
href="#x1-500002" id="QQ2-1-94">piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.3 <a 
href="#x1-510003" id="QQ2-1-95">A Transcript-based Set of piRNA Loci</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.4 <a 
href="#x1-520004" id="QQ2-1-96">Three Classes of piRNAs During Post-Natal Spermatogenesis</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.5 <a 
href="#x1-530005" id="QQ2-1-101"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.6 <a 
href="#x1-540006" id="QQ2-1-104"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.7 <a 
href="#x1-550007" id="QQ2-1-107"><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.8 <a 
href="#x1-560008" id="QQ2-1-110">A-MYB and the Pachytene piRNA Regulatory Circuitry</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >4.2.9 <a 
href="#x1-570009" id="QQ2-1-113">Feed-Forward Regulation of piRNA Production is Evolutionarily
Conserved</a></span>
<br />&#x00A0;<span class="sectionToc" >4.3 <a 
href="#x1-580003" id="QQ2-1-116">DISCUSSION</a></span>
<br />&#x00A0;<span class="sectionToc" >4.4 <a 
href="#x1-590004" id="QQ2-1-117">EXPERIMENTAL PROCEDURES</a></span>
<br /><span class="chapterToc" >5 <a 
href="#x1-600005" id="QQ2-1-118">Discussion</a></span>
<br />&#x00A0;<span class="sectionToc" >5.1 <a 
href="#x1-610001" id="QQ2-1-119">The Future of Dynamic long RNAs</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.1 <a 
href="#x1-620001" id="QQ2-1-120">The Known Knowns</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.1 <a 
href="#x1-630001" id="QQ2-1-121">Alternative Splicing turns gene expression into a game of cards</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.2 <a 
href="#x1-640002" id="QQ2-1-122">Pervasive transcription</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.3 <a 
href="#x1-650003" id="QQ2-1-123">Tissue and cell specificity</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.4 <a 
href="#x1-660004" id="QQ2-1-124">Function of long RNAs</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.5 <a 
href="#x1-670005" id="QQ2-1-125">Chromatin regulation</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.1.6 <a 
href="#x1-680006" id="QQ2-1-126">PTGS</a></span>
                                                                                

                                                                                
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.2 <a 
href="#x1-690002" id="QQ2-1-127">The Known Unknowns</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.2.1 <a 
href="#x1-700001" id="QQ2-1-128">How are they important?</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.2.2 <a 
href="#x1-710002" id="QQ2-1-129">What regulates their tissue-specific expression?</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.1.2.3 <a 
href="#x1-720003" id="QQ2-1-130">Technological Improvements</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.1.3 <a 
href="#x1-730003" id="QQ2-1-131">The Unknown Unknowns</a></span>
<br />&#x00A0;<span class="sectionToc" >5.2 <a 
href="#x1-740002" id="QQ2-1-132">Future SeqZip development and use</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.1 <a 
href="#x1-750001" id="QQ2-1-133">Assay Modifications</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.1 <a 
href="#x1-760001" id="QQ2-1-134">Rnl2 T29A mutation</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.2 <a 
href="#x1-770002" id="QQ2-1-135">Thermostable Ligases</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.3 <a 
href="#x1-780003" id="QQ2-1-136">LNA-containing Ligamers</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.4 <a 
href="#x1-790004" id="QQ2-1-137">Ribose-containing Ligamers</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.5 <a 
href="#x1-800005" id="QQ2-1-138">Repurposing the SOLiD Platform</a></span>
<br />&#x00A0;&#x00A0;&#x00A0;<span class="subsubsectionToc" >5.2.1.6 <a 
href="#x1-810006" id="QQ2-1-139">Others</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.2 <a 
href="#x1-820002" id="QQ2-1-140">An ideal SeqZip experiment to query coordinated splicing</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.2.3 <a 
href="#x1-830003" id="QQ2-1-141">SeqZip and single-molecule FISH</a></span>
<br />&#x00A0;<span class="sectionToc" >5.3 <a 
href="#x1-840003" id="QQ2-1-142">In the haystack: piRNA precursor transcripts surrounded by RNA</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.1 <a 
href="#x1-850001" id="QQ2-1-143">In vivo chemical labeling of precursor piRNA transcripts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.2 <a 
href="#x1-860002" id="QQ2-1-144">In vivo imaging of precursor piRNA transcripts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.3 <a 
href="#x1-870003" id="QQ2-1-145">What are they doing?</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.4 <a 
href="#x1-880004" id="QQ2-1-146">How are they generated?</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.3.5 <a 
href="#x1-890005" id="QQ2-1-147">Why should we care?</a></span>
<br />&#x00A0;<span class="sectionToc" >5.4 <a 
href="#x1-900004" id="QQ2-1-148">Lingering Questions for <span 
class="phvro8t-x-x-120">Dscam1</span></a></span>
<br />&#x00A0;<span class="sectionToc" >5.5 <a 
href="#x1-910005" id="QQ2-1-149">Final thoughts</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.5.1 <a 
href="#x1-920001" id="QQ2-1-150">Why did I want to study AS?</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.5.2 <a 
href="#x1-930002" id="QQ2-1-151">Biologists need Computation Biological Skills</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.5.3 <a 
href="#x1-940003" id="QQ2-1-152">Science versus Engineering: Two thoughts in one school</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >5.5.4 <a 
href="#x1-950004" id="QQ2-1-153">Dealing with the data deluge</a></span>
<br /><span class="appendixToc" >A <a 
href="#x1-96000A" id="QQ2-1-154">Appendix - Misc Information</a></span>
                                                                                

                                                                                
<br />&#x00A0;<span class="sectionToc" >A.1 <a 
href="#x1-970001" id="QQ2-1-155">Buffers</a></span>
<br />&#x00A0;<span class="sectionToc" >A.2 <a 
href="#x1-980002" id="QQ2-1-157">Equations</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.1 <a 
href="#x1-990001" id="QQ2-1-158">f</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.2 <a 
href="#x1-1000002" id="QQ2-1-159">b</a></span>
<br />&#x00A0;&#x00A0;<span class="subsectionToc" >A.2.3 <a 
href="#x1-1010003" id="QQ2-1-160">Normalize oxidized small RNA libraries size to time-matched
unoxidized library</a></span>
<br />&#x00A0;<span class="sectionToc" >A.3 <a 
href="#x1-1020003" id="QQ2-1-161">PCR Programs</a></span>
<br /><span class="appendixToc" >B <a 
href="#x1-103000B" id="QQ2-1-162">Appendix B: piRNA precurors are spliced</a></span>
<br />&#x00A0;<span class="sectionToc" >B.1 <a 
href="#x1-1040001" id="QQ2-1-163">Evidence for spliced piRNA precusor transcripts</a></span>
<br /><span class="appendixToc" >C <a 
href="#x1-105000C" id="QQ2-1-169">Automated Ligamer Assembly</a></span>
<br />&#x00A0;<span class="sectionToc" >C.1 <a 
href="#x1-1060001" id="QQ2-1-170">Installation</a></span>
<br />&#x00A0;<span class="sectionToc" >C.2 <a 
href="#x1-1070002" id="QQ2-1-171">Example Input Format</a></span>
<br />&#x00A0;<span class="sectionToc" >C.3 <a 
href="#x1-1080003" id="QQ2-1-172">Ligamer Assmebly Source Code</a></span>
<br /><span class="chapterToc" ><a 
href="#Q1-1-173">Bibliography</a></span>
</div><a 
 id="x1-1001r3"></a><a 
 id="Q1-1-4"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a 
 id="x1-20003"></a>List of Figures</h2> <div class="tableofcontents"><span class="lofToc" >1.1&#x00A0;<a 
href="#x1-5001r1">The Solitary and Gregarious forms of <span 
class="phvro8t-x-x-120">Schistocerca</span>
<span 
class="phvro8t-x-x-120">gregaria</span></a></span><br /><span class="lofToc" >1.2&#x00A0;<a 
href="#x1-8001r2">Cost of sequencing the human genome over time</a></span><br /><span class="lofToc" >1.3&#x00A0;<a 
href="#x1-9001r3">Methods
for High-throughput sequencing of RNA</a></span><br /><span class="lofToc" >1.4&#x00A0;<a 
href="#x1-13001r4">Estimates of number of
human genes, and percentage alternatively spliced over time</a></span><br /><span class="lofToc" >1.5&#x00A0;<a 
href="#x1-15001r5">HTS
read lengths are not sufficient to maintain AS connectivity</a></span><br /><span class="lofToc" >1.6&#x00A0;<a 
href="#x1-16001r6">Mouse
<span 
class="phvro8t-x-x-120">Fn1 </span>contains multipe sites of Alternative Splicing</a></span><br /><span class="lofToc" >1.7&#x00A0;<a 
href="#x1-17001r7">Number of hg19
Alternative Event types per gene</a></span><br /><span class="lofToc" >1.8&#x00A0;<a 
href="#x1-17002r8">Number of transcripts per <span 
class="phvro8t-x-x-120">Drosophila</span>
<span 
class="phvro8t-x-x-120">melanogaster </span>gene</a></span><br /><span class="lofToc" >1.9&#x00A0;<a 
href="#x1-18001r9">The architecture of the <span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>
gene <span 
class="phvro8t-x-x-120">Dscam1</span></a></span><br /><span class="lofToc" >1.10&#x00A0;<a 
href="#x1-18002r10">Important <span 
class="phvro8t-x-x-120">Dscam1 </span>expression during <span 
class="phvro8t-x-x-120">Drosophila</span>
<span 
class="phvro8t-x-x-120">melanogaster </span>life cycle</a></span><br /><span class="lofToc" >1.11&#x00A0;<a 
href="#x1-20001r11">Mechanism of Rnl2 ATP-dependent
ligation</a></span><br /><span class="lofToc" >1.12&#x00A0;<a 
href="#x1-21001r12">Structure and active site of pre-adenylated of Rnl2</a></span><br /><span class="lofToc" >1.13&#x00A0;<a 
href="#x1-21002r13">Active
site of T4 RNA Ligase 2 with highlighted residues</a></span><br /><span class="lofToc" >1.14&#x00A0;<a 
href="#x1-24001r14">Genetic evidence
for long, continuous fly piRNA precursor transcripts</a></span><br /><span class="lofToc" >1.15&#x00A0;<a 
href="#x1-24002r15">Different
Classes of mammalian piRNAs</a></span><br /><span class="lofToc" >1.16&#x00A0;<a 
href="#x1-25001r16">A model for Mammalian
piRNA biogenesis</a></span><br /><span class="lofToc" >2.1&#x00A0;<a 
href="#x1-30001r1">Original SeqZip Diagram</a></span><br /><span class="lofToc" >2.2&#x00A0;<a 
href="#x1-31002r2">10 Gene Set
schematic</a></span><br /><span class="lofToc" >2.3&#x00A0;<a 
href="#x1-33001r3">Ligation product tied to RNA integrity</a></span><br /><span class="lofToc" >2.4&#x00A0;<a 
href="#x1-33002r4">Trans Transcript
investigation</a></span><br /><span class="lofToc" >2.5&#x00A0;<a 
href="#x1-35001r5">SeqZip can examine HIV transcript integrity</a></span><br /><span class="lofToc" >2.6&#x00A0;<a 
href="#x1-36001r6">piRNA
precursor locations</a></span><br /><span class="lofToc" >2.7&#x00A0;<a 
href="#x1-36003r7">pRT Doesn&#8217;t Work for piRNA precursors</a></span><br /><span class="lofToc" >2.8&#x00A0;<a 
href="#x1-36004r8">Dst1
by SeqZip</a></span><br /><span class="lofToc" >2.9&#x00A0;<a 
href="#x1-36005r9">Three sites of AS in <span 
class="phvro8t-x-x-120">Fn1 </span>by SeqZip</a></span><br /><span class="lofToc" >2.10&#x00A0;<a 
href="#x1-36006r10">Testes
Specific RNA precursor expression</a></span><br /><span class="lofToc" >2.11&#x00A0;<a 
href="#x1-36007r11">piRNA precursor analysis
via SeqZip</a></span><br /><span class="lofToc" >3.1&#x00A0;<a 
href="#x1-39001r1">SeqZip Diagram</a></span><br /><span class="lofToc" >3.2&#x00A0;<a 
href="#x1-41001r2">SeqZip Diagram</a></span><br /><span class="lofToc" >3.3&#x00A0;<a 
href="#x1-42001r3">SeqZip
Diagram</a></span><br /><span class="lofToc" >4.1&#x00A0;<a 
href="#x1-49001r1">piRNA Precursors are RNA Pol II Transcripts</a></span><br /><span class="lofToc" >4.2&#x00A0;<a 
href="#x1-49002r2">The Major
piRNA-Producing Genes of the Post-Partum Mouse Testis</a></span><br /><span class="lofToc" >4.3&#x00A0;<a 
href="#x1-52001r3">Three
Classes of piRNA-Generating Loci</a></span><br /><span class="lofToc" >4.4&#x00A0;<a 
href="#x1-52002r4">Pre-pachytene piRNAs Persist
in Pachytene Spermatocytes</a></span><br /><span class="lofToc" >4.5&#x00A0;<a 
href="#x1-52003r5">Examples of Pachytene piRNA
Genes</a></span><br /><span class="lofToc" >4.6&#x00A0;<a 
href="#x1-52004r6">Examples of Pre-Pachytene piRNA Genes</a></span><br /><span class="lofToc" >4.7&#x00A0;<a 
href="#x1-53001r7">A-MYB Binds
the Promoters of Pachytene piRNA Genes</a></span><br /><span class="lofToc" >4.8&#x00A0;<a 
href="#x1-53002r8">ChIP-qPCR Confirms
ChIP-seq Data</a></span><br /><span class="lofToc" >4.9&#x00A0;<a 
href="#x1-54001r9">Pachytene piRNAs and Precursors Decrease in <span 
class="phvro8t-x-x-120">A-Myb</span>
Mutant Testes</a></span><br /><span class="lofToc" >4.10&#x00A0;<a 
href="#x1-54002r10">Change in piRNA Expression in <span 
class="phvro8t-x-x-120">Spo11</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, <span 
class="phvro8t-x-x-120">Tdrd6</span>,
and <span 
class="phvro8t-x-x-120">Tdrd9 </span>Mutants</a></span><br /><span class="lofToc" >4.11&#x00A0;<a 
href="#x1-55001r11">Examples of the Effect of the <span 
class="phvro8t-x-x-120">A-Myb </span>Mutation
on piRNA Expression</a></span><br /><span class="lofToc" >4.12&#x00A0;<a 
href="#x1-55002r12">Pachytene piRNA Precursor Abundance in
<span 
class="phvro8t-x-x-120">A-Myb</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, and <span 
class="phvro8t-x-x-120">Trip13 </span>Mutants</a></span><br /><span class="lofToc" >4.13&#x00A0;<a 
href="#x1-56001r13">A-MYB Regulates Expression
                                                                                

                                                                                
of mRNAs Encoding piRNA Pathway Proteins</a></span><br /><span class="lofToc" >4.14&#x00A0;<a 
href="#x1-56002r14"><span 
class="phvro8t-x-x-120">A-Myb </span>mutants, but
Not <span 
class="phvro8t-x-x-120">Miwi </span>Mutants, Change the Expression of RNA Silencing Pathway
Genes</a></span><br /><span class="lofToc" >4.15&#x00A0;<a 
href="#x1-57001r15">Feed-Forward Regulation of piRNA Biogenesis by A-MYB is
Conserved in Rooster</a></span><br /><span class="lofToc" >4.16&#x00A0;<a 
href="#x1-57002r16">Genomic Locations of piRNA Clusters in the
Rooster (Gallus gallus) Testis.</a></span><br /><span class="lofToc" >B.1&#x00A0;<a 
href="#x1-104001r1">RNA-Seq evidence for piRNA precursor
splicing</a></span><br /><span class="lofToc" >B.2&#x00A0;<a 
href="#x1-104002r2">Lack of piRNAs within precusor introns</a></span><br /><span class="lofToc" >B.3&#x00A0;<a 
href="#x1-104003r3">piRNAs map to
Splice Junctions of precursor transcripts</a></span><br /><span class="lofToc" >B.4&#x00A0;<a 
href="#x1-104004r4"><span 
class="phvro8t-x-x-120">A-Myb </span>Mutants produce no
splice-junction mapping piRNAs</a></span><br /><span class="lofToc" >B.5&#x00A0;<a 
href="#x1-104005r5">General features of piRNA transcripts</a></span><br />
</div>
                                                                                

                                                                                
<a 
 id="x1-2001r4"></a>
<a 
 id="Q1-1-6"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a 
 id="x1-30004"></a>List of Tables</h2> <div class="tableofcontents"><span class="lotToc" >1.1&#x00A0;<a 
href="#x1-17003r1">Fly genes with &#x003E;2,000 assembled transcripts</a></span><br /><span class="lotToc" >2.1&#x00A0;<a 
href="#x1-31001r1">Genes
with big spans in between</a></span><br /><span class="lotToc" >2.2&#x00A0;<a 
href="#x1-36002r2">Just 9 piRNA genes create &#x003E;50% of
mammalian piRNAs</a></span><br /><span class="lotToc" >A.1&#x00A0;<a 
href="#x1-97001r1">SeqZip Hybridization and Ligation Buffer</a></span><br />
</div>
                                                                                

                                                                                
                                                                                

                                                                                
<a 
 id="x1-3001r5"></a>
<a 
 id="Q1-1-8"></a>
<div class="center" 
>
<!--l. 4--><p class="noindent" >
<!--l. 4--><p class="noindent" ><span 
class="phvro8t-x-x-144">List of Abreviations</span>
</div>
<div class="table">
<!--l. 5--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-1" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-1-1g"><col 
id="TBL-1-1"></colgroup><colgroup id="TBL-1-2g"><col 
id="TBL-1-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-1-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-1"  
class="td11">AS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-1-2"  
class="td11">Alternative Splicing                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-1"  
class="td11">DNA        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-2-2"  
class="td11">Deoxyribonucleic acid                                                     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-1"  
class="td11">ssDNA    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-3-2"  
class="td11">Single-stranded DNA                                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-1"  
class="td11">RNA        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-4-2"  
class="td11">Ribonucleic acid                                                              </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-1"  
class="td
11">ssRNA    </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-5-2"  
class="td11">Single-stranded RNA                                                      </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-1"  
class="td11">ATP         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-6-2"  
class="td11">Adenosine triphosphate                                                  </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-1"  
class="td11">NAD        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-7-2"  
class="td11">Nicotinamide adenine dinucleotide                                 </td></tr><tr  
 style="vertical-align:baseline;" id="TBL-1-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-1"  
class="td
11">ChIP-Seq</td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-8-2"  
class="td11">Chromatin Immunoprecipitation followed by sequencing</td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-1"  
class="td11">EST         </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-9-2"  
class="td11">Expressed Sequence Tag                                               </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-1"  
class="td11">HTS        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-10-2"  
class="td11">High-throughput sequencing (see also NGS)                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-1"  
class="td11">NGS        </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-11-2"  
class="td11">Next-generation sequencing                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-1"  
class="td11">CAP </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-12-2"  
class="td11">5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; 7meG structure attached to mRNAs                          </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-1"  
class="td11">nt </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-13-2"  
class="td11">A nucleotide of either DNA or RNA                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-14-1"  
class="td11">bp </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-14-2"  
class="td11">A base pair of DNA                                                         </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-15-1"  
class="td11">SRE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-15-2"  
class="td11">Splicing Regulatory Element                                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-16-1"  
class="td11">IRE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-16-2"  
class="td11">Intron Recognition Element                                             </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-17-1"  
class="td11">CNS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-17-2"  
class="td11">Central Nervous System                                                 </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-18-1"  
class="td11">TSS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-18-2"  
class="td11">{Transcription or Translation} Start Site                           </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-19-1"  
class="td11">TTS </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-19-2"  
class="td11">{Transcription or Translation} Termination Site                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-20-1"  
class="td11">SAGE </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-20-2"  
class="td11">Serial Analysis of Gene Expression                                </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-21-1"  
class="td11">hnRNP </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-21-2"  
class="td11">heterogeneous nuclear ribonucleoprotein                       </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-22-1"  
class="td11">FISH </td><td  style="white-space:nowrap; text-align:left;" id="TBL-1-22-2"  
class="td11">Fluorescence in situ hybridization                                   </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-1-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-1-23-1"  
class="td11"> </td></tr></table></div>
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
                                                                                

                                                                                
<a 
 id="x1-3002r6"></a>
<a 
 id="Q1-1-9"></a>
<div class="center" 
>
<!--l. 2--><p class="noindent" >
<!--l. 2--><p class="noindent" ><span 
class="phvro8t-x-x-144">List of Symbols</span>
</div>
<div class="table">
<!--l. 3--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-2" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-2-1g"><col 
id="TBL-2-1"><col 
id="TBL-2-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-2-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-1"  
class="td11">5<span 
class="cmsy-8">&#x2032;</span> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-1-2"  
class="td11">The 5 prime end of a DNA or RNA molecule     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-1"  
class="td11">3<span 
class="cmsy-8">&#x2032;</span> </td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-2-2"  
class="td11">The 5 prime end of a DNA or RNA molecule     </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-1"  
class="td11"><span 
class="cmmi-12">&#x03BC; </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-3-2"  
class="td11">Micro. A value of 1x10<sup class="textsuperscript"><span 
class="phvr8t-x-x-109">-6</span></sup> standard units              </td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-2-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-1"  
class="td11"><span 
class="cmmi-12">&#x03C1; </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-2-4-2"  
class="td11">Pearson product-moment correlation coefficient</td></tr></table></div>
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
                                                                                

                                                                                
<a 
 id="x1-3003r7"></a>
<a 
 id="Q1-1-10"></a>
<div class="center" 
>
<!--l. 2--><p class="noindent" >
<!--l. 2--><p class="noindent" ><span 
class="phvro8t-x-x-144">Definitions</span>
</div>
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">RNA-Seq</span></dt><dd 
class="description">                                                                    <br 
class="newline" />A technology  wherein  RNA  is  fragmented,  converted  to  DNA,  and
analyzed on a high-throughput sequencing instrument
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">A &#8216;Read&#8217;</span></dt><dd 
class="description">                                                                     <br 
class="newline" />The sequence   of   nucleotides   produced   from   each   spot   on   a
high-throughput sequencing machine
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Insert</span></dt><dd 
class="description">                                                                        <br 
class="newline" />The RNA  molecule  captured  between  two  cloning  sequences  in  a
high-throughput sequencing library preparation workflow
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Read length</span> </dt><dd 
class="description">                                                                                                       <br 
class="newline" />The number of nucleotides for each given &#8220;read&#8221;
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Read depth</span></dt><dd 
class="description">                                                                  <br 
class="newline" />The number  of  reads  obtained  from  each  high-throughput  sequencing
                                                                                

                                                                                
analysis
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Coverage</span></dt><dd 
class="description">                                                                    <br 
class="newline" />A measure of the number of times each nt of a genome is sequenced. E.g.
100 million reads of a 10 million nt genome = 10X coverage, assuming
uniform distribution of the &#8220;reads&#8221;
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Paired-end</span></dt><dd 
class="description">                                                                   <br 
class="newline" />When both sides of a DNA insert or template are sequenced, utilizing the
original  length  of  DNA  between  the  reads  to  facilitate  mapping  (<a 
href="#XRoach1995">Roach
et&#x00A0;al.</a>&#x00A0;[<a 
href="#XRoach1995">1995</a>]).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Scaffold or contig</span> </dt><dd 
class="description">                                                                                             <br 
class="newline" />A draft sequence of nucleotides, meant to represent the actual biological
sequence   as   closely   as   possible,   examples   include   unassembled
fragments of chromosomes or fragments of mRNA transcripts.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Argonaute</span></dt><dd 
class="description">                                                                   <br 
class="newline" />Protein(s)  belonging  to  a  group  containing  a  Piwi  (P-element  induced
wimpy testes)  domain,  that  bind  nucleic  acids  and  participate  in  many
target-guided processes, including RNA Interference, and RNA-indicuded
transcript/gene silencing.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Ligamer</span></dt><dd 
class="description">                                                                      <br 
class="newline" />A DNA oligonucleotide containing two distinct regions of complementarity
to a 5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; section of RNA. Each region is normalized for <sup><span 
class="cmmi-8">Tm</span></sup> such that
                                                                                

                                                                                
the length of each section is ~15&#8211;30 nt. These two regions are connected
by a short sequence of the designer&#8217;s choice, usually &#x003E;5 nt in length. Each
ligamers overall length is ~45&#8211;60 nt. See figures <a 
href="#x1-30001r1">2.1<!--tex4ht:ref: fig:Original SeqZip Diagram --></a> and <a 
href="#x1-39001r1">3.1<!--tex4ht:ref: fig:Roy2014 SeqZip Diagram --></a>.
</dd></dl>
                                                                                

                                                                                
<a 
 id="x1-3004r8"></a>
<a 
 id="Q1-1-11"></a>
<div class="center" 
>
<!--l. 7--><p class="noindent" >
<!--l. 7--><p class="noindent" ><span 
class="phvro8t-x-x-172">I would like to dedicate this Doctoral dissertation to my</span>
<span 
class="phvro8t-x-x-172">grandfather, George Knauf. My grandfather passed</span>
<span 
class="phvro8t-x-x-172">away on September 23rd, 2011, just one week shy of</span>
<span 
class="phvro8t-x-x-172">his 82nd birthday. I find it difficult to articulate how</span>
<span 
class="phvro8t-x-x-172">much I miss him. He spoke carefully and never without</span>
<span 
class="phvro8t-x-x-172">purpose or conviction. While I hear from others that he</span>
<span 
class="phvro8t-x-x-172">was proud of me, he rarely, if ever, betrayed that type</span>
<span 
class="phvro8t-x-x-172">of emotion directly. It is my goal to build as solid a life</span>
<span 
class="phvro8t-x-x-172">as he, founded on hard work, playing the long game,</span>
<span 
class="phvro8t-x-x-172">responsibility, and maintaining friendships. These are</span>
<span 
class="phvro8t-x-x-172">just a few of the personality traits that I observed and</span>
<span 
class="phvro8t-x-x-172">try to emulate. The fact that he passed before he could</span>
<span 
class="phvro8t-x-x-172">meet our son Owen is one of my biggest regrets. Of all</span>
<span 
class="phvro8t-x-x-172">the possessions he left behind, it is the memory of our</span>
<span 
class="phvro8t-x-x-172">time together that I will cherish the most. Rest in</span>
<span 
class="phvro8t-x-x-172">peace, Grump. I did it.</span></div>
                                                                                

                                                                                
<a 
 id="x1-3005r9"></a>
<a 
 id="Q1-1-12"></a>
<div class="center" 
>
<!--l. 3--><p class="noindent" >
<!--l. 3--><p class="noindent" ><span 
class="phvro8t-x-x-248">Preface</span>
</div>
<!--l. 5--><p class="noindent" >The work reported in this dissertation has been published in the following articles.
Chapter <a 
href="#x1-460004">4<!--tex4ht:ref: Chapter4 --></a> has been published previously as:<br 
class="newline" />Li, X. Z. Z., Roy, C. K. K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B. W. W., &#x2026;
Zamore, P. D. D. (2013). An Ancient Transcription Factor Initiates the Burst of piRNA
Production during Early Meiosis in Mouse Testes. Molecular Cell, 50(1), 1&#8211;15.
doi:10.1016/j.molcel.2013.02.016
<!--l. 9--><p class="noindent" >Some contents of Chapter <a 
href="#x1-370003">3<!--tex4ht:ref: Chapter3 --></a> are included in a currently submitted manuscript.
                                                                                

                                                                                
                                                                                

                                                                                
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;1</span><br /><a 
 id="x1-40001"></a>Introduction</h2>
<a 
 id="x1-4001r1"></a>
<h3 class="sectionHead"><span class="titlemark">1.1 </span> <a 
 id="x1-50001"></a>Fixed Genomes and Flexible Genes</h3>
<!--l. 6--><p class="noindent" >Exodus tells of the liberation of the Israelites from Egyptian slavery. Humble and
reluctant Moses, their divine-appointed leader, attempts to force the Pharaoh
Ramses to release the Israelites through inflicting 10 plagues. Pharaoh is
stalwart and stubborn as water turns to blood, streets are flooded with frogs,
lice, and flies. As livestock falls dead from disease, people and animals both
are covered in boils, and the land burns in storms of fire, Pharaoh does not
bend.
<!--l. 8--><p class="noindent" >The 8th plague was a swarm of Locusts, described in Exodus 10: 14&#8211;15:
<div class="quote">
<!--l. 13--><p class="noindent" ><sup class="textsuperscript"><span 
class="phvro8t-x-x-109">14</span></sup><span 
class="phvro8t-x-x-120">And the locusts went up over all the land of Egypt, and rested in</span>
<span 
class="phvro8t-x-x-120">all the coasts of Egypt: very grievous were they; before them there</span>
<span 
class="phvro8t-x-x-120">were no such locusts as they, neither after them shall be such.</span><br 
class="newline" /><sup class="textsuperscript"><span 
class="phvro8t-x-x-109">15</span></sup><span 
class="phvro8t-x-x-120">For they covered the face of the whole earth, so that the land was</span>
<span 
class="phvro8t-x-x-120">darkened; and they did eat every herb of the land, and all the fruit of</span>
<span 
class="phvro8t-x-x-120">the trees which the hail had left: and there remained not any green</span>
<span 
class="phvro8t-x-x-120">thing in the trees, or in the herbs of the field, through all the land of</span>
<span 
class="phvro8t-x-x-120">Egypt.</span></div>
<!--l. 17--><p class="noindent" >The desolation of a locust plague was still not enough to persuade Ramses. Nor three
days of darkness. Only the death of all first-born Egyptians, included Ramses own
son, was enough to persuade Pharaoh to liberate the Israelites.
<!--l. 19--><p class="noindent" >Locust swarms are not biblical fantasy. It is perhaps the most <span 
class="phvro8t-x-x-120">believable </span>of the 10
plagues. Today the United Nations&#8217; (UN) Food and Agriculture division maintains a
<a 
href="http://www.fao.org/ag/locusts/en/info/info/news/index.html" >Locust watch website</a> that provides weekly updates on potential swarms in northern
Africa and the Middle East. Locusts have long been, and continue to be, a powerful
and feared force of Nature.
<!--l. 21--><p class="noindent" >Unlike fire and brimstone, locusts are something that can be observed and
studied. What triggers a swam? We know that the desert locust, <span 
class="phvro8t-x-x-120">Schistocerca</span>
<span 
class="phvro8t-x-x-120">gregaria</span>, is the one of 10 locust species that swarm and cause massive
crop damage. <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>are in the insect Order Orthoptera,
along with crickets and katydids. Orthoptern members make sound known as
<span 
class="phvro8t-x-x-120">stridulation </span>by vigorously rubbing their wings, making for a noisy cloud of
devastation.
                                                                                

                                                                                
<!--l. 23--><p class="noindent" >A Desert Locust only weighs 0.05&#8211;0.07 ounces, and is less than 2.5 inches long.
They consume their own body weight in vegetation per day. One <a 
href="http://animaldiversity.ummz.umich.edu/site/accounts/information/Melanoplus_spretus.html" >swarm</a> of the
infamous, and now curiously extinct, Rocky Mountain locust contained 12.2 trillion
insects. Its estimated total weight was 27.5 million tons. The swarm covered almost
200 square miles (2/3 the size of Manhattan), and could travel 60 miles in a day. A
locust swarm is truly a modern biblical plague.
<!--l. 25--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-5001r1"></a>
                                                                                

                                                                                

<!--l. 27--><p class="noindent" ><img 
src="Figures/Chapter1/DesertLocust.jpeg" alt="PIC"  
>
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;1.1: </span><span  
class="content">The Solitary and Gregarious forms of <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span><br 
class="newline" /> The two phenotypic forms of <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>appear very different. The
Solitary form is green and generally larger, while its gregarious form is more
brightly colored, smaller, and capable of swarming in vast numbers, destroying
crops and vegetation. Photo from <a 
href="http://www.wikicommons.com" >Wikicommons</a>. </span></div><!--tex4ht:label?: x1-5001r1 -->
                                                                                

                                                                                
<!--l. 34--><p class="noindent" ></div><hr class="endfigure">
<!--l. 36--><p class="noindent" >By definition swarms are temporary; the movement, en masse, from one location to
another. Where do 12.2 trillion locusts go when not swarming? Does anyone care if
their crops aren&#8217;t under assault? It seemed no one cared enough, until about 1921,
when an important realization was made.
<!--l. 38--><p class="noindent" >The power and destruction <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>can inflict makes it difficult to
believe that they are nothing more than common grasshoppers. Nothing more than
grasshoppers not just by analogy, but by actual <span 
class="phvro8t-x-x-120">Taxonomy</span>. &#8220;Desert locusts&#8221; are
actually the <span 
class="phvro8t-x-x-120">gregarious </span>form of <span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>(See Figure <a 
href="#x1-5001r1">1.1<!--tex4ht:ref: fig:Locust --></a>), while
the more familiar and docile looking &#8220;grasshopper&#8221; is the <span 
class="phvro8t-x-x-120">solitary form</span>. How
does such a dichotomy to exist within the same organism, indeed the same
<span 
class="phvro8t-x-x-120">genome</span>?
<!--l. 40--><p class="noindent" ><span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>are <span 
class="phvro8t-x-x-120">polyphenic</span>, meaning that they have multiple (poly)
physical forms (phenotypes). Polyphenism is a general feature among insects.
Phenotypes are often extremely different. For example, pea aphids (<span 
class="phvro8t-x-x-120">Acyrthosiphon</span>
<span 
class="phvro8t-x-x-120">pisum</span>), which usually exist in an asexually reproducing, wingless female
form, respond to reduced food supply overcrowding by producing winged
offspring. Winged organisms travel to new sources of food and revert back to the
asexually reproducing form [<a 
href="#XPurandare2014b">Purandare et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPurandare2014b">2014</a>,&#x00A0;<a 
href="#XShingleton2003">Shingleton et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShingleton2003">2003</a>].
In the case of <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span>, the gregarious form is smaller and
more brightly colored compared to its solitary cousins. This transformation
can happen in as little as two hours. What is the underlying cause of this
transformation?
<!--l. 42--><p class="noindent" >In 2009, <a 
href="#XAnstey2009">Anstey et&#x00A0;al.</a>&#x00A0;[<a 
href="#XAnstey2009">2009</a>] reported that in just two hours of forced crowding,
<span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>displayed elevated levels of the neurotransmitter serotonin in
the ganglia (brain). These levels were strongly correlated gregarious form indicators.
Serotonin is an important molecule in regulating neuronal junctions and wiring in the
brain [<a 
href="#XHoeffer2003">Hoeffer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHoeffer2003">2003</a>]. Through the integration of environmental and social
cues, the brain of the insect was being re-wired, resulting in tremendous changes in
behavior and phenotype. These changes prepare the organism to deal with a different
world in order for the species to survive, but to the detriment of surrounding
agriculture.
<!--l. 44--><p class="noindent" >In an extremely interesting <a 
href="http://aeon.co/magazine/nature-and-cosmos/why-its-time-to-lay-the-selfish-gene-to-rest/" >article</a>, David Dobbs compares the two forms of
<span 
class="phvro8t-x-x-120">Schistocerca gregaria </span>to that of Dr Jekyll and Mr. Hyde, the principle characters of
Robert Louis Stevenson novella. For Dr Jekyll in fiction, and for <span 
class="phvro8t-x-x-120">Schistocerca gregaria</span>
in reality, the power to morph into multiple forms demonstrates the incredible power of
a fixed genome yet plastic gene expression. It is often said that something is &#8220;in the
genes.&#8221; This thesis will illustrate that another oft-head idiom is more important: &#8220;it&#8217;s
how you use them.&#8221;
                                                                                

                                                                                
<!--l. 46--><p class="noindent" >Before we could understand how genes were used, we needed to identify them, and
understand what they were.
<a 
 id="x1-5002r14"></a>
<h3 class="sectionHead"><span class="titlemark">1.2 </span> <a 
 id="x1-60002"></a>Nucleic Acid Sequencing</h3>
<a 
 id="x1-6001r1"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.1 </span> <a 
 id="x1-70001"></a>DNA Sequencing History</h4>
<!--l. 51--><p class="noindent" >Soon after it was realized that DNA is the source of genetic information in all living
organisms [<a 
href="#XWatson1953a">Watson and Crick</a>,&#x00A0;<a 
href="#XWatson1953a">1953</a>], and the <span 
class="phvro8t-x-x-120">pretty </span>and <span 
class="phvro8t-x-x-120">elegant </span>arrangement of
complementary, antiparrallel, DNA strands was known [<a 
href="#XWatson2012a">Watson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatson2012a">2012</a>], the
ability to determine specific arrangements of nucleotide bases (i.e. to sequence) in a
given length of DNA was seen as a critical missing piece of technology. It
took 25 years after the nature of DNA&#8217;s architecture to be able to determine
the specific arrangement of nucleotides in the polymer&#8212;to sequence it. By
1977, two completely different methods developed by Sanger [<a 
href="#XSanger1975a">Sanger and
Coulson</a>,&#x00A0;<a 
href="#XSanger1975a">1975</a>,&#x00A0;<a 
href="#XSanger1977b">Sanger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSanger1977b">1977</a>] and Maxam-Gilbert [<a 
href="#XMaxam1977a">Maxam and Gilbert</a>,&#x00A0;<a 
href="#XMaxam1977a">1992</a>]
were reported. These sequencing technologies, from then on referred to
eponymously as &#8220;Sanger&#8221; or &#8220;Maxam-Gilbert&#8221; sequencing, were used to determine
the specific order of a small piece of DNA (200&#8211;300 nt). Sanger sequencing soon
dominated most sequencing reactions, likely due to the conceptually more
intuitive nature of the technology, and over the past 35 years, DNA sequences
have been slowly cloned, sequenced, analyzed, and dutifully cataloged into
knowledge.
<!--l. 53--><p class="noindent" >During the late 1970&#8217;s and throughout the 1980&#8217;s, DNA sequences were typically
communicated in important publications [<a 
href="#XCordell1980a">Bell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCordell1980a">1980</a>,&#x00A0;<a 
href="#XSanger1978a">Sanger et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSanger1978a">1978</a>]. The
birth of the Internet in the 1990&#8217;s made essential publically-funded repositories for
sequence information easily available [<a 
href="#XBenson2011a">Benson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBenson2011a">2011</a>]. However, it was the
human genome project [<a 
href="#XLander2011a">Lander</a>,&#x00A0;<a 
href="#XLander2011a">2011</a>,&#x00A0;<a 
href="#XVenter2001">Venter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVenter2001">2001</a>], that provided the
important activation energy that brought DNA sequencing from a hard-to-perform, but
necessary, analysis, to an organized large-scale effort of assembling the complete
genetic material complex genomes. An often criticized, but undeniably disrupting
force in the human genome project was the competing efforts of the privately-owned
company Celera [<a 
href="#XVenter2008a">Venter</a>,&#x00A0;<a 
href="#XVenter2008a">2007</a>]. Taking a higher-throughput and centralized
approach to determining the sequence of the human genome, Celera fundamentally
changed the landscape of genome assembly. Instead of assigning specific sections
of the genome to be worked out by individual labs, Celera parallelized the
effort, by collecting many of the best &#8220;high-throughput&#8221; Sanger-sequencing
devices from Agilent (ABI 3700 DNA Analyzer). Using a &#8220;shotgun&#8221; approach
                                                                                

                                                                                
[<a 
href="#XStaden1979">Staden</a>,&#x00A0;<a 
href="#XStaden1979">1979</a>], sequenced pairwise [<a 
href="#XRoach1995">Roach et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRoach1995">1995</a>], and combined with
sequence scaffolds made available by the publicly-funded project, Celera was
able to assemble high-quality genomic sequences very quickly. Arguably,
this was the first deep sequencing effort, and changed the landscape of
molecular and biochemical research, coincident with the beginning of a new
millennium.
<a 
 id="x1-7001r17"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.2 </span> <a 
 id="x1-80002"></a>History of High-throughput Sequencing</h4>
<!--l. 57--><p class="noindent" >Sequencing DNA by Sanger&#8217;s technology remains a valuable and critical tool in
every biological scientist&#8217;s arsenal. However, the technology has a practical
throughput limit. Each DNA molecule to be sequenced must be isolated and
clonally amplified, typically using bacteria. Given that the human genome
[<a 
href="#XHattori2005a">Consortium</a>,&#x00A0;<a 
href="#XHattori2005a">2004</a>] comprises &#x003E;3 billion bp, and each Sanger reaction provides &#x00A0;800
nt of quality sequence, at least &#x00A0;4 million individual reactions are needed
to determine the sequence of the human genome. This also assumes all
of reads are of sufficient quality, length, and do not overlap by even 1 nt.
No overlap is out of the question, as overlaps are essential for assembling
individual sequences via their overlaps, allowing assembly through repeated
sequences.
<!--l. 59--><p class="noindent" >Even the best practical improvements to work-flows could not bring Sanger DNA
sequencing in-line with aspirations of analyzing genomes of multiple species
and/or organisms. The early 2000&#8217;s saw multiple efforts to improve the scale
of DNA sequencing, first using MPSS [<a 
href="#XBrenner2000a">Brenner et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrenner2000a">2000</a>], but perhaps
more importantly, by Pyro- [<a 
href="#XRonaghi1998a">Ronaghi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRonaghi1998a">1998</a>] and Polony sequencing
[<a 
href="#XShendure2005">Shendure et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShendure2005">2005</a>]. Both pyro- and polonysequencing utilize emulsion PCR
[<a 
href="#XNakano2003a">Nakano et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNakano2003a">2003</a>] for clonal amplification prior to sequencing, removing the
bottleneck of bacterial cloning. In contrast to Sanger sequencing, where
the signal is from fluorescence of the last incorporated chain-terminating
nucleotide, pyrosequencing visualizes light given off by luciferase reacting with
pyrophosphate (PPi), a by-product of nucleotide incorporation. This approach was
later commercialized by 454 technologies. Polony sequencing involves a more
complicated sequencing-by-ligation method, eventually commercialized by
Applied Biosystems and branded as SOLiD sequencing. While both of these
technologies provided valuable, high-throughput sequences, neither has been
as successful as the approach commercialized by Solexa, now known as
Illumina.
<!--l. 61--><p class="noindent" >Illumina-produced sequencers use a sequencing-by-synthesis approach. After clonal
amplification of DNA on a slide surface [<a 
href="#XBentley2008">Bentley et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBentley2008">2008</a>], fluorescent nucleotides
are visualized as they are incorporated into the growing DNA strand. Since 2006,
                                                                                

                                                                                
iterations of the Illumina platform (eg. GE, GE-II(x), Hi-Seq, Hi-Seq 2500, Hi X) have
demonstrated a steady and impressive increases in both read depth and length.
On February 15th 2012, Illumina announced on its <a 
href="http://blog.basespace.illumina.com/" >Basespace blog</a>, that
they had sequenced a HapMap sample at 40X coverage, using the HiSeq
2500 platform and paired-end 100 nt reads in a single run. On Janurary
14th, 2014, Illumina <a 
href="http://bit.ly/PZpegZ" >announced</a> its HiSeq X system, the first platform to truly
attain produce the $1,000 genome. These machines can demonstrate that
sequencing genomes is no longer the monumental endeavor it once was, and
that completely new experimental possibilities are a reality for life science
research.
<!--l. 63--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-8001r2"></a>
                                                                                

                                                                                
<!--l. 65--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.50x.png" alt="PIC" class="graphics" width="346.29375pt" height="155.58124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.50x.png" src="Figures/Chapter1/Sequencing_costs_over_time.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.2: </span><span  
class="content">Cost of sequencing the human genome over time<br 
class="newline" />The costs of  sequencing  the  human  genome  has  decreased  on  a  log
scale over a  roughly  10  year  period  thanks  to  major  improvements  in
high-throughput  sequencing.  Data  from  Wetterstrand  KA.  DNA  Sequencing
Costs: Data from the NHGRI Genome Sequencing Program (GSP) Available at:
<a 
href="www.genome.gov/sequencingcosts" class="url" ><span 
class="ectt-1200">www.genome.gov/sequencingcosts</span></a>. Accessed 2013-09-03). </span></div><!--tex4ht:label?: x1-8001r2 -->
                                                                                

                                                                                
<!--l. 72--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-8002r18"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.3 </span> <a 
 id="x1-90003"></a>Deep-sequencing of RNA</h4>
<!--l. 76--><p class="noindent" >The first widely-accepted large scale method used to measured gene expression was
Serial Analysis of Gene Expression (SAGE) [<a 
href="#XVelculescu1995a">Velculescu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVelculescu1995a">1995</a>]. While the
importance of microarrays in the measurement of gene expression via cannot be
overstated [<a 
href="#XMarioni2008">Marioni et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMarioni2008">2008</a>,&#x00A0;<a 
href="#XShendure2008">Shendure and Ji</a>,&#x00A0;<a 
href="#XShendure2008">2008</a>], the limited ability of
microarrays to investigate novel sequences combined with their analogue signal,
makes their relevance to this section off-topic. Yet, SAGE, (like the before mentioned
MPSS technique) produces a digital output of gene expression using a cleaver
procedure of restriction endonucleolytic cleavage of cDNA molecules. Cleaved
product sticky ends are ligated using sticky ends left from digestion and concatenated
together to form longer DNA fragments. Fragments are cloned into a vector, amplified,
and Sanger sequenced. Using sequences incorporated during concatenation, the
number of sequenced &#8220;fragments&#8221; that align to a given gene is related to the
abundance of the original RNA molecule. While SAGE was a clever molecular trick
allowing researches to dip into the 5-<span 
class="cmmi-12">log </span>range of expression typically seen in
mRNA expression, it is still limited by Sanger sequencing read length and
depth.
<!--l. 78--><p class="noindent" >The Solexa/Illumina platform relies on clonal amplification of a single template directly
on a slide surface. Imaging these spots with sensitive digital cameras after
sequential addition of fluorescent nucleotides (<span 
class="phvro8t-x-x-120">sequencing by synthesis</span>) turned
out to be the right mix for a &#8220;second generation&#8221; HTS platform. Not long
after the Solexa/Illumina platform achieved read lengths of sufficient length
of depth necessary to measure gene expression were the first RNA-Seq
papers published [<a 
href="#XLister2008">Lister et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLister2008">2008</a>,&#x00A0;<a 
href="#XMortazavi2008">Mortazavi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMortazavi2008">2008</a>,&#x00A0;<a 
href="#XNagalakshmi2008">Nagalakshmi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNagalakshmi2008">2008</a>]. These papers gave a powerful glimpse into the future of molecular
biology. Indeed, in the years since, analysis by RNA-Seq has quickly overtaken
other forms of gene expression analysis, as demonstrated by the number of
accessions deposited in GEO per year [<a 
href="#XBarrett2013">Barrett et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarrett2013">2013</a>]. RNA-Seq allows for
digital quantification of RNA expression across physiologically-relevant ranges
[<a 
href="#XBlencowe2009">Blencowe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBlencowe2009">2009</a>]. While simultaneously measuring gene expression, the
data can be used for novel sequence discovery, measuring RNA-editing [<a 
href="#XLi2011">Li
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2011">2011</a>], and transcript assembly [<a 
href="#XTrapnell2010">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>]. By modifying the
basic protocol or performing additional biochemical steps, RNA-Seq can
be used to investigate many aspects of RNA biology (see <a 
href="#x1-9001r3">1.3<!--tex4ht:ref: fig:htsMethods --></a> and [<a 
href="#XMutz2013">Mutz
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMutz2013">2013</a>]).
<!--l. 80--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-9001r3"></a>
                                                                                

                                                                                
<!--l. 82--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.51x.png" alt="PIC" class="graphics" width="420.57124pt" height="311.16249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.51x.png" src="Figures/Chapter1/RNA_Sequencing_methodologies.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.3: </span><span  
class="content">Methods for High-throughput sequencing of RNA<br 
class="newline" />In the short years since the first report of RNA-Seq, many variations have been
reported. The figure above provides an incomplete graphical illustration of some
of these variations. A more complete list of *Seq applications is maintained on
this <a 
href="http://liorpachter.wordpress.com/seq/" >blog</a>. </span></div><!--tex4ht:label?: x1-9001r3 -->
                                                                                

                                                                                
<!--l. 89--><p class="noindent" ></div><hr class="endfigure">
<!--l. 91--><p class="noindent" >RNA processing begins the moment the nascent RNA is exposed from the
polymerase exit channel. Many methodologies have been developed that enrich
RNA-Seq libraries for particular types of RNA. For example, measurement of
nascent transcripts can be performed via GRO-Seq [<a 
href="#XCore2008a">Core et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCore2008a">2008</a>],
and the extremely complicated process of RNA turnover (referring to the
rates at which RNAs both are produced and degraded) has been examined
[<a 
href="#XGhosh2010a">Ghosh and Jacobson</a>,&#x00A0;<a 
href="#XGhosh2010a">2010</a>,&#x00A0;<a 
href="#XTani2012">Tani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTani2012">2012</a>]. RNA::Protein interactions
can be measured with or without cross-linking the protein to the RNA, via
CLIP or RIP, respectively. Once an RNA has been fully transcribed, known
processing steps such as 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; 7meG (CAP) formation and poly(A) tail formation
can be measured using any of the Cap-Seq/CAGE methodologies [<a 
href="#XShiraki2003a">Shiraki
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShiraki2003a">2003</a>], or PAS/TAIL/PAL [<a 
href="#XChang2014b">Chang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChang2014b">2014</a>,&#x00A0;<a 
href="#XShepard2011">Shepard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShepard2011">2011</a>,&#x00A0;<a 
href="#XSubtelny2014">Subtelny
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSubtelny2014">2014</a>]. With appropriate size-selection steps, small RNAs [<a 
href="#XGhildiyal2008">Ghildiyal
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>] can also be captured into a sequencing library. Finally, traditional
RNA-Seq, can effectively capture fragments of all of the above mentioned
libraries, even though it is mainly associated with measurement of traditional
mRNAs.
<!--l. 93--><p class="noindent" >RNA-Seq and its associated flavors are also traditionally associated quantifying RNA
obtained from <span 
class="phvro8t-x-x-120">many </span>tissue culture cells bulk pieces of a particular tissues. Recently,
efforts to measure the RNA expression occurring in individual cells has gained
attention [<a 
href="#XShapiro2013b">Shapiro et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShapiro2013b">2013</a>]. Perhaps the most interesting concept when thinking
about measurement of gene expression in a single cell is the &#8220;biological uncertainty
principle&#8221;, wherein it is possible to either know, or change &#8212;but not both&#8212;&#x00A0;the RNA
composition of a single cell. The name borrows from Heisenberg&#8217;s uncertainty
principle [<a 
href="#XKennard1927">Kennard</a>,&#x00A0;<a 
href="#XKennard1927">1927</a>] and is often confused with the more appropriate &#8220;Observer
effect&#8221; [<a 
href="#XRiley2013">Riley and Steitz</a>,&#x00A0;<a 
href="#XRiley2013">2013</a>]. Leaving that issue aside, measuring the unique
transcriptome of a given cell among cells of a common tissue is surly an exciting
and informative endeavor [<a 
href="#XMarinov2013">Marinov et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMarinov2013">2013</a>,&#x00A0;<a 
href="#XShalek2013b">Shalek et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShalek2013b">2013</a>,&#x00A0;<a 
href="#XWills2013">Wills
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWills2013">2013</a>]. Compared to DNA, the diversity of RNA synthesis within living cells is
potentially much more complicated [<a 
href="#XShendure2012">Shendure and Aiden</a>,&#x00A0;<a 
href="#XShendure2012">2012</a>], and the ability to
accurately measure RNA dynamics should allow us to make much more
informative observations concerning biology then is currently possible [<a 
href="#XDjebali2012">Djebali
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2012">2012</a>].
<a 
 id="x1-9002r20"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.4 </span> <a 
 id="x1-100004"></a>Measuring RNA Expression via Sequencing</h4>
<!--l. 99--><p class="noindent" >The encode project revealed that most of the genome is transcribed into RNA. This
was done in cancerous cell lines, and while it revealed the potential for transcription, it
                                                                                

                                                                                
did not reveal much biology beyond cells in culture simply perpetuating their
existence.
<a 
 id="x1-10001r22"></a>
<h4 class="subsectionHead"><span class="titlemark">1.2.5 </span> <a 
 id="x1-110005"></a>Assembling Transcripts from Short Reads</h4>
<a 
 id="x1-11001r16"></a>
<h3 class="sectionHead"><span class="titlemark">1.3 </span> <a 
 id="x1-120003"></a>Nucleic Acid Splicing</h3>
<a 
 id="x1-12001r23"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.1 </span> <a 
 id="x1-130001"></a>Alternative Splicing</h4>
<!--l. 110--><p class="noindent" >[<span 
class="phvb8t-x-x-120">?</span>] [<a 
href="#XBarbosa-Morais2012">Barbosa-Morais et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarbosa-Morais2012">2012</a>,&#x00A0;<a 
href="#XMerkin2012">Merkin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMerkin2012">2012</a>]
<!--l. 114--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-13001r4"></a>
                                                                                

                                                                                
<!--l. 116--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.52x.png" alt="PIC" class="graphics" width="332.24124pt" height="160.59999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.52x.png" src="Figures/Chapter1/numberHumanGenesAndNumberSpliced.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.4: </span><span  
class="content">Dark Grey - Estimates of number of human genes; Light Grey -
Estimates of what percent of genes undergo some form of alternative splicing. </span></div><!--tex4ht:label?: x1-13001r4 -->
                                                                                

                                                                                
<!--l. 122--><p class="noindent" ></div><hr class="endfigure">
<!--l. 124--><p class="noindent" >Soon after the discovery of introns, it was reasoned that genes could be arranged in
different combinations, greatly increasing the coding potential of a genome
[<a 
href="#XGilbert1978a">Gilbert</a>,&#x00A0;<a 
href="#XGilbert1978a">1978</a>]. The process of rearranging genes, now known as alternative splicing
(AS), has proven to be an integral phase of gene expression in most eukaryotes. In
just 15 years, the number of genes estimated to be alternatively spliced has grown
considerably. Phillip Sharp, Co-Nobel-prize winner for the discovery of splicing, stated
that: &#8220;Approximately, one of every twenty genes is expressed by alternative
pathways of RNA splicing in different cell types or growth states&#8221; <a 
href="#XSharp2014">Sharp</a>&#x00A0;[<a 
href="#XSharp2014">2014</a>].
Not long after the assembly of the first human genome, a number of groups
combed through Expressed Sequence Tag (EST) databases to increase that
estimate to 35%-59% [<a 
href="#XModrek2002">Modrek and Lee</a>,&#x00A0;<a 
href="#XModrek2002">2002</a>]. Soon after, analysis using
specially designed microarrays resulted in an increased estimate of 74%
[<a 
href="#XJohnson2003">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2003">2003</a>]. However, in late 2008, three groups used RNA-Seq to
demonstrate that between 86% and 95% of human multi-exon genes are
subject to AS [<a 
href="#XPan2008">Pan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPan2008">2008</a>,&#x00A0;<a 
href="#XSultan2008">Sultan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSultan2008">2008</a>,&#x00A0;<a 
href="#XWang2008">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>]. Not
only did they demonstrate that almost all genes are alternatively spliced,
they also showed that AS often occurs in a tissue- and cell type-specific
manner. In combination with regulation of transcription itself, the study of AS is
critical to our understanding of the connections between the comparably
static genomic DNA sequence and the highly flexible and adaptive abilities of
organisms.
<a 
 id="x1-13002r25"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.2 </span> <a 
 id="x1-140002"></a>Deciphering a splicing code</h4>
<!--l. 128--><p class="noindent" >A gene is alternatively spliced when, as a result of transcription and processing, there
are at least two unique transcripts produced from one genomic sequence. Beyond
counting observed isoforms, one major area of effort is to decode sequence
regulatory elements (SREs) contained in pre-mRNA that define AS site selection
[<a 
href="#XWang2008">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>]. In contrast to the core splicing signals, we have limited
knowledge of the SREs that serve to increase, or decrease, the strength of a
particular splice site, often within a sea of other potential sites. Through a variety of
mechanisms, these elements serve as cis-acting sequences and binding sites for
trans-acting factors. Some of the best-studied SREs include Exon Splicing Enhancers
and Silencers (ESEs and ESSs). Members of the Serine-Arginine (SR) protein family
typically bind to ESEs located in an exon, promoting its definition and thereby
increasing the probability that the exon will be included in the final transcript
[<a 
href="#XGraveley2000">Graveley</a>,&#x00A0;<a 
href="#XGraveley2000">2000</a>,&#x00A0;<a 
href="#XLong2009">Long and Caceres</a>,&#x00A0;<a 
href="#XLong2009">2009</a>]. Meanwhile, ESSs serve to squelch
inclusion, often through binding trans-acting heterogeneous ribonucleoprotein
                                                                                

                                                                                
particles (hnRNPs) [<a 
href="#XMartinez-Contreras2007">Martinez-Contreras et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMartinez-Contreras2007">2007</a>]. Therefore, binding
of these trans-acting factors to their appropriate SREs can either promote
or inhibit interactions between the splicing machinery and the pre-mRNA.
The current working hypothesis is that a finely tuned combination of these
binding events determines the final exon content of each isoform [<a 
href="#XHouse2008">House and
Lynch</a>,&#x00A0;<a 
href="#XHouse2008">2008</a>].
<!--l. 130--><p class="noindent" >Sequence motifs that compose the AS code have been teased out [<a 
href="#XBarash2010">Barash
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarash2010">2010</a>,&#x00A0;<a 
href="#XLadd2002">Ladd and Cooper</a>,&#x00A0;<a 
href="#XLadd2002">2002</a>]. Additionally, assignment of the binding motifs
to tissue-specific trans-acting factors has also progressed [<a 
href="#XJin2003">Jin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJin2003">2003</a>,&#x00A0;<a 
href="#XLicatalosi2008">Licatalosi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLicatalosi2008">2008</a>,&#x00A0;<a 
href="#XUle2005">Ule et&#x00A0;al.</a>,&#x00A0;<a 
href="#XUle2005">2005</a>]. Many of these binding motifs were identified using
combined computational and biochemical approaches. Computational approaches
usually involve searching for a comparative enrichment of sequences near splice
sites. Biochemical approaches typically include gel shift, SELEX, and cross-linking.
Many of these approaches are performed in vitro and disregard the importance of
cellular context on binding affinities. However, with the increasing accessibility of deep
sequencing, many groups are extracting physiologically relevant, high-resolution
data from traditional biochemical techniques [<a 
href="#XIngolia2009">Ingolia et&#x00A0;al.</a>,&#x00A0;<a 
href="#XIngolia2009">2009</a>,&#x00A0;<a 
href="#XIngolia2011">2011</a>].
Deep-sequencing approaches are also being applied to questions involving
mechanisms of AS. In addition to the RNA-Seq experiments, High-Throughput
Sequencing [following] Cross-Linking Immunoprecipitation (HTS-CLIP) has confirmed
SRE motif data predicted from computational and microarray experiments [<a 
href="#XHafner2010">Hafner
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHafner2010">2010</a>,&#x00A0;<a 
href="#XLicatalosi2008">Licatalosi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLicatalosi2008">2008</a>]. Using this approach, researchers can
now enrich their samples for sequences that bind trans-acting factors of
interest.
<a 
 id="x1-14001r27"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.3 </span> <a 
 id="x1-150003"></a>The Isoform Problem</h4>
<!--l. 134--><p class="noindent" >As with many areas of basic research, the field of AS relies on large-scale (aka:
<span 
class="phvro8t-x-x-120">global, genome-wide, high-throughput</span>) techniques. Two of the most widely applied
technologies employed for large-scale analysis of gene expression are microarrays
and &#8220;2nd generation&#8221; HTS. Unfortunately, both of these techniques have fundamental
limitations, with the major issues being probe specificity for the former and read
length for the latter.
<!--l. 136--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-15001r5"></a>
                                                                                

                                                                                
<!--l. 138--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.53x.png" alt="PIC" class="graphics" width="415.55249pt" height="327.22249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.53x.png" src="Figures/Chapter1/SeqLengths_and_Connectivity.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.5: </span><span  
class="content">HTS read lengths are not sufficient to maintain AS connectivity<br 
class="newline" />A) Long RNAs may have multiple sites of AS, separated by 1000&#8217;s of nt; B)
Most mRNAs have &#x00A0;10 exons of &#x00A0;150 nt each. Some have many more (and
longer) exons. Read lengths of current sequencing technologies do not maintain
connectivity between distant sites. </span></div><!--tex4ht:label?: x1-15001r5 -->
                                                                                

                                                                                
<!--l. 145--><p class="noindent" ></div><hr class="endfigure">
<!--l. 147--><p class="noindent" >Microarrays rely on hybridization of a target sequence to a known probe
averaging 25 to 100 nt in length [<a 
href="#XSouthern2001">Southern</a>,&#x00A0;<a 
href="#XSouthern2001">2001</a>]. Therefore, microarrays
indicate only the presence of short sequences in the target sample and do not
provide adequate linkage information of these sequences. A hypothetical
scenario can be used to describe it another way. Say we are investigating
a transcript known to display two different regions of AS (See Figure <a 
href="#x1-15001r5">1.5<!--tex4ht:ref: fig:NoConnectivityInHTSMethods --></a>).
Probes targeting these two regions demonstrated an increase in signal for
both AS events. Unfortunately, we could not determine if we observed an
increase in unique transcripts, each containing only one region of AS, or an
increase in production of a single transcript containing both regions [<a 
href="#XCalarco2007">Calarco
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007">2007a</a>]. This binary analysis is the heart of the &#8220;connectivity problem.&#8221;
Microarrays have proven extremely informative and will likely continue to
do so in more targeted applications. However, this issue, combined with
concerns of cross-hybridization, reproducibility, and a comparably small dynamic
range, has hastened the displacement of microarray by RNA-Seq as the
preferred method for comprehensive analysis of gene expression [<a 
href="#XShendure2008">Shendure and
Ji</a>,&#x00A0;<a 
href="#XShendure2008">2008</a>].
<!--l. 149--><p class="noindent" >Many researchers are turning toward 2nd generation HTS methodologies for
comprehensive transcriptome analysis. HTS allows for <span 
class="phvro8t-x-x-120">de novo </span>identification of
isoforms, over a larger dynamic range, in a quantitative fashion [<a 
href="#XMortazavi2008">Mortazavi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMortazavi2008">2008</a>]. Additionally, techniques exist to enrich samples for low-abundance
isoforms, making the complete cataloging of AS events a possibility [<a 
href="#XDjebali2008">Djebali
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2008">2008</a>,&#x00A0;<a 
href="#XSalehi-Ashtiani2008">Salehi-Ashtiani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSalehi-Ashtiani2008">2008</a>]. Unfortunately, the current read-length
abilities (depicted in Figure <a 
href="#x1-15001r5">1.5<!--tex4ht:ref: fig:NoConnectivityInHTSMethods --></a>) of all sequencing platforms do not solve the
connectivity problem. Excluding single-molecule HTS read lengths of sufficient
length [<a 
href="#XShendure2004">Shendure et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShendure2004">2004</a>], other approaches proposed to solve the
connectivity problem include traditional cloning and sequencing or hybridization of
query oligos to single-molecule transcripts [<a 
href="#XCalarco2007">Calarco et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007">2007a</a>,&#x00A0;<a 
href="#XEmerick2007">Emerick
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XEmerick2007">2007</a>,&#x00A0;<a 
href="#XZhu2003">Zhu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhu2003">2003</a>]. While these approaches can determine exon
sequence connectivity, they scale poorly and are not feasible for large-scale
applications.
<!--l. 151--><p class="noindent" >Clearly, AS is an essential regulatory mechanism involved in the control of human
gene expression. Its combinatorial nature could potentially answer many questions,
such as a physical explanation of what separates us from our closest evolutionary
ancestor, the chimpanzee [<a 
href="#XCalarco2007a">Calarco et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCalarco2007a">2007b</a>]. Additionally, the influence
of AS on disease and cancer is slowly coming to light [<a 
href="#XTazi2009">Tazi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTazi2009">2009</a>].
Unfortunately, because of the limitations of methods currently used for the
large-scale analysis of isoform expression we fail to obtain the complete
picture of AS. One specific missing element of that picture is the prevalence of
                                                                                

                                                                                
coordination between different regions of AS separated by large spans of sequence.
An efficient, large-scale, single-molecule technique that maintains isoform
sequence connectivity is required to complete the complicated picture of
AS.
<!--l. 153--><p class="noindent" >Identification of proximally acting SREs is progressing at a rapid pace. New and
traditional biochemical methods, coupled with HTS, will undoubtedly fuel
this progress. Unfortunately, a critical component of AS regulation currently
neglected by the field is that of SREs acting across a considerable distance
(&#x003E;800 nt). One observation that may lead to the identification of long-range
SREs is intramolecular coordination between distal splicing decisions. In
Figure <a 
href="#x1-15001r5">1.5<!--tex4ht:ref: fig:NoConnectivityInHTSMethods --></a> a model transcript that may exhibit coordinated distal regions
of AS. In this model, the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; region of AS contains a cassette exon, which
may or may not be included. This region is separated from the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;region
of AS by many thousands of nucleotides. Does the decision to include the
cassette exon have an effect on which of the mutually exclusive exons is
included? This type of AS regulation may represent a general and pervasive
phenomenon.
<a 
 id="x1-15002r28"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.4 </span> <a 
 id="x1-160004"></a>Coordination in splicing</h4>
<!--l. 157--><p class="noindent" >The provocative &#8220;Miller Spread&#8221; showing spliceosomes associated with RNA
transcripts likely ignited the first thought that transcription and splicing are intracately
linked [<a 
href="#XOsheim1985">Osheim et&#x00A0;al.</a>,&#x00A0;<a 
href="#XOsheim1985">1985</a>]. Twelve years later, the first observation that polymerase
speed can affect downstream splicing decisions was reported [<a 
href="#XCramer1997">Cramer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCramer1997">1997</a>],
spawning a new field of research into linked transcription and splicing. But how linked
are these two processes?
<!--l. 160--><p class="noindent" >How would linkage between transcription and splicing manifest? One of the clearest
examples is mouse Fibronectin <span 
class="phvro8t-x-x-120">Fn1 </span>(Figure <a 
href="#x1-16001r6">1.6<!--tex4ht:ref: fig:mouseFn1 --></a>) [<a 
href="#XSchwarzbauer1983">Schwarzbauer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchwarzbauer1983">1983</a>,&#x00A0;<a 
href="#XWhite2011a">White
and Muro</a>,&#x00A0;<a 
href="#XWhite2011a">2011</a>]. In this gene, inclusion of the alternatively spliced Extra Domain A
(EDI or EDA) region promotes splicing from one of three alternative 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; Splice Site
(3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; SS) in the type III homology connecting segment (IIICS) region, resulting in more
frequent production of shorter transcripts [<a 
href="#XFededa2005">Fededa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFededa2005">2005</a>]. This effect occurs
over six constitutively expressed exons and 800 nt of sequence (5400 nt if
introns are considered). [<a 
href="#XFededa2005">Fededa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFededa2005">2005</a>] also analyzed EST databases,
concluding that approximately 25% of human genes contain multiple regions of
AS. How many of these regions could show a coordinated effect, similar
to that observed in Fibronectin? Providing some insight into this question,
[<a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFagnani2007">2007</a>] used microarrays designed to report on inclusion levels
of cassette exons in mammalian central nervous system tissues [<a 
href="#XFagnani2007">Fagnani
                                                                                

                                                                                
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFagnani2007">2007</a>]. The results produced a set of 38 pairs of exons mapping to the
same gene that showed a coordinated increase or decrease of inclusion
levels.
<!--l. 162--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-16001r6"></a>
                                                                                

                                                                                
<!--l. 164--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.54x.png" alt="PIC" class="graphics" width="390.45874pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.54x.png" src="Figures/Chapter1/Fibronectin.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.6: </span><span  
class="content">Mouse <span 
class="phvro8t-x-x-120">Fn1 </span>contains multiple sites of Alternative Splicing<br 
class="newline" />A) There are three highly-studied regions of AS in mouse <span 
class="phvro8t-x-x-120">Fn1</span>: The cassette
exons EDB and EDA, and the Variable(V)-region (AKA the IIICS) exon, which
displays multiple 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; splice sites. Each of these sites is separated by multiple
constitutive exons.; B) Considering simplistic splicing of these three exons, there
are 12 different isoforms of mouse <span 
class="phvro8t-x-x-120">Fn1</span>. </span></div><!--tex4ht:label?: x1-16001r6 -->
                                                                                

                                                                                
<!--l. 171--><p class="noindent" ></div><hr class="endfigure">
<!--l. 173--><p class="noindent" >There have been a few studies that investigate forms of splicing coordination between
adjacent exons present in mRNA. The vertebrate genes 4.1B and 4.1R,
members of the protein 4.1 family encoding for cytoskeletal adaptor proteins,
both undergo splicing of upstream 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; first exons to distal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; second exons,
skipping a stronger proximal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; second exon [<a 
href="#XParra2012">Parra et&#x00A0;al.</a>,&#x00A0;<a 
href="#XParra2012">2012</a>,&#x00A0;<a 
href="#XParra2008">2008</a>]. This is
accomplished through &#8220;intrasplicing&#8221; involving an intronic sequence element
(the &#8220;intraexon&#8221;) only present when transcription begins at the upstream 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
exon, allowing the exon to ligate do the weaker distal 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; second exon via
an intermediate splicing event. Importantly, this type of splicing would be
similar, but different from recursive splicing seen in drosophila [<a 
href="#XBurnette2005a">Burnette
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBurnette2005a">2005</a>]. Another example of the importance of intron sequence elements on
AS is observed in the equine <span 
class="cmmi-12">&#x03B2;</span>-casin gene, where the authors propose a
model involving an intronic splicing enhancer bound to the exit channel of the
elongating polymerase, promoting inclusion of downstream cassette exons [<a 
href="#XLenasi2006">Lenasi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLenasi2006">2006</a>]. Taking a more genome-wide approach Peng et al. examined human
and mouse EST data looking for correlations between adjacent AS cassette
exons [<a 
href="#XPeng2008">Peng et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPeng2008">2008</a>]. The authors note that positively correlated pairs of
adjacent cassette exons typically resemble constitutive exons in splice strength,
whereas negatively, or weakly correlated pairs are likely to be newly emerging
exons, whose strength of splicing has not evolved enough to be constitutively
included.
<!--l. 177--><p class="noindent" >The last, most current, and thorough study of intra-gene splicing coordination involves
the <span 
class="phvro8t-x-x-120">Caenorhabditis elegans </span>gene <span 
class="phvro8t-x-x-120">slo1 </span>[<a 
href="#XGlauser2011">Glauser et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGlauser2011">2011</a>,&#x00A0;<a 
href="#XJohnson2011">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2011">2011</a>].
<span 
class="phvro8t-x-x-120">slo1 </span>is the <span 
class="phvro8t-x-x-120">Caenorhabditis elegans </span>orthologue of the human BK channel gene
<span 
class="phvro8t-x-x-120">Kcnma</span>, also known to undergo extensive alternative splicing [<a 
href="#XNilsen2010">Nilsen and
Graveley</a>,&#x00A0;<a 
href="#XNilsen2010">2010</a>] via 13 cassette exons, potentially coding for over 1,000
different isoforms. <span 
class="phvro8t-x-x-120">Kcnma </span>is highly developmentally, spatially, and tissue
regulated. It is involved in a diverse range of cellular processes, including
hearing, circadian rhythms, urinary function, and vasoregulation [<a 
href="#XFodor2009a">Fodor and
Aldrich</a>,&#x00A0;<a 
href="#XFodor2009a">2009</a>]. While the gene is highly conserved, as organism complexity grows, so
does the apparent transcriptional diversity of this gene. In worms, <span 
class="phvro8t-x-x-120">slo1 </span>can
produce up to 12 different isoforms. Glauser et al. used QPCR to demonstrate
individual, AS region inclusion frequencies do not correspond to complete isoform
frequencies, when measured via a TaqMan probe approach. They go on to
describe a interdependent-splicing model that best fits the data, and support
interdependence via mutations at one sight altering splicing at both upstream and
downstream sites of AS, separated by atleast one other splicing event. After
measuring the biophysical properties of the isoforms [<a 
href="#XJohnson2011">Johnson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XJohnson2011">2011</a>], they
conclude that coordinated AS is critical for proper BK channel function in vivo.
                                                                                

                                                                                
It is interesting to note that this study also identified an intronic sequence
element that displayed some type of coordinated, or co-regulated effect on
AS.
<a 
 id="x1-16002r30"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.5 </span> <a 
 id="x1-170005"></a>Many isoforms per gene</h4>
<!--l. 181--><p class="noindent" >It is easy to think of AS as a binary process. Isoform A or B is produced based upon
picking either exon A or B. What quickly becomes evident, and is far too real for
researchers building transcriptome assembly algorithms, is that the combinatorial
nature of AS makes it both a power means of generating isoform diversity and a
difficult problem to study [<a 
href="#XTrapnell2012a">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2012a">2012</a>].
<!--l. 183--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-17001r7"></a>
                                                                                

                                                                                
<!--l. 185--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.55x.png" alt="PIC" class="graphics" width="393.46999pt" height="172.64499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.55x.png" src="Figures/Chapter1/ASEventTypesPlot.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.7: </span><span  
class="content">Number of hg19 Alternative Event types per gene<br 
class="newline" />Alternative Event types per gene. RefSeq on 2014-03-24 </span></div><!--tex4ht:label?: x1-17001r7 -->
                                                                                

                                                                                
<!--l. 192--><p class="noindent" ></div><hr class="endfigure">
<!--l. 195--><p class="noindent" >One of the most recent attempts to investigate the breath of combinations produced
by AS is the already mentioned ENCODE project [<a 
href="#XDjebali2012">Djebali et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDjebali2012">2012</a>]. ENCODE
performed extremely in depth analysis of 15 cell lines, and find that each isoform
produces &#x00A0;10 transcripts per gene, with a broad distribution in terms of isoforms
expressed per sample.
<!--l. 197--><p class="noindent" >The ENCODE project clearly demonstrated that most human genes can under AS in
many more ways than previously appreciated. Most genes could be considered as
undergoing &#8220;complex&#8221; AS, with numerous forms of AS ( See figure <a 
href="#x1-17001r7">1.7<!--tex4ht:ref: fig:asEventsBarChart --></a>). Despite the
prevalence of complex alternative spliced genes, just a few genes are used as
examples to illustrate numerical possibilities and biological significance. For
example the human immune system relies heavily on AS to be plastic toward
antigen recognition and response [<a 
href="#XLynch2004">Lynch</a>,&#x00A0;<a 
href="#XLynch2004">2004</a>]. Modulation of extracellular
signaling proteins such as <span 
class="phvro8t-x-x-120">CD44 </span>and cellular adhesion protein <span 
class="phvro8t-x-x-120">CD45 </span>have been
well-studied [<a 
href="#XZikherman2008">Zikherman and Weiss</a>,&#x00A0;<a 
href="#XZikherman2008">2008</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. Alternative splicing in humans,
however, does not seem to produce the number of unique possible combinations
as AS of genes in simpler organisms, such as fruit flies, perhaps due to
specialization of genes, or different genes that work in combination or complexes, as
oppose to utilizing unique gene isoforms [<a 
href="#XPark2007">Park and Graveley</a>,&#x00A0;<a 
href="#XPark2007">2007</a>]. For
example, the fruit fly gene muscle myosin heavy chain (<span 
class="phvro8t-x-x-120">Mhc</span>) can produce up to
480 different isoforms through AS of 17 different cassette exons [<a 
href="#XBernstein1983a">Bernstein
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBernstein1983a">1983</a>].
<!--l. 199--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-17002r8"></a>
                                                                                

                                                                                
<!--l. 201--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.56x.png" alt="PIC" class="graphics" width="413.545pt" height="291.0875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.56x.png" src="Figures/Chapter1/NumberOFTranscriptsPerFlyGene.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.8: </span><span  
class="content">Number of transcripts per <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>gene<br 
class="newline" />Data from [<a 
href="#XBrown2014">Brown et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>], Supplemental Table 3. Number of transcript
per bin, with bin sizes &#8220;closed&#8221; on the upper part of range. </span></div><!--tex4ht:label?: x1-17002r8 -->
                                                                                

                                                                                
<!--l. 208--><p class="noindent" ></div><hr class="endfigure">
<div class="table">
                                                                                

                                                                                
<!--l. 2--><p class="noindent" ><a 
 id="x1-17003r1"></a><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-3" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-3-1g"><col 
id="TBL-3-1"></colgroup><colgroup id="TBL-3-2g"><col 
id="TBL-3-2"></colgroup><colgroup id="TBL-3-3g"><col 
id="TBL-3-3"></colgroup><colgroup id="TBL-3-4g"><col 
id="TBL-3-4"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-1-1"  
class="td11"><span 
class="phvb8t-x-x-109">Gene Name</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-2"  
class="td11"><span 
class="phvb8t-x-x-109"># Introns</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-3"  
class="td11"><span 
class="phvb8t-x-x-109"># Transcripts</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-1-4"  
class="td11"><span 
class="phvb8t-x-x-109"># Proteins</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-2-1"  
class="td11"><span 
class="phvr8t-x-x-109">Mhc              </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-2"  
class="td11">      <span 
class="phvr8t-x-x-109">60</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2040</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-2-4"  
class="td11">      <span 
class="phvr8t-x-x-109">511</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-3-1"  
class="td
11"><span 
class="phvr8t-x-x-109">slo                </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-2"  
class="td11">      <span 
class="phvr8t-x-x-109">49</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2070</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-3-4"  
class="td11">      <span 
class="phvr8t-x-x-109">279</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-4-1"  
class="td
11"><span 
class="phvr8t-x-x-109">ps                 </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-2"  
class="td11">      <span 
class="phvr8t-x-x-109">30</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2099</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-4-4"  
class="td11">       <span 
class="phvr8t-x-x-109">27</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-5-1"  
class="td11"><span 
class="phvr8t-x-x-109">rg                  </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-2"  
class="td11">      <span 
class="phvr8t-x-x-109">45</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2178</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-5-4"  
class="td11">       <span 
class="phvr8t-x-x-109">23</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-6-1"  
class="td11"><span 
class="phvr8t-x-x-109">shot              </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-2"  
class="td11">      <span 
class="phvr8t-x-x-109">60</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2478</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-6-4"  
class="td11">      <span 
class="phvr8t-x-x-109">886</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-7-1"  
class="td
11"><span 
class="phvr8t-x-x-109">scrib             </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-2"  
class="td11">      <span 
class="phvr8t-x-x-109">53</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2555</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-7-4"  
class="td11">      <span 
class="phvr8t-x-x-109">259</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-8-1"  
class="td
11"><span 
class="phvr8t-x-x-109">heph             </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-2"  
class="td11">      <span 
class="phvr8t-x-x-109">75</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2876</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-8-4"  
class="td11">       <span 
class="phvr8t-x-x-109">52</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-9-1"  
class="td11"><span 
class="phvr8t-x-x-109">CG42748     </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-2"  
class="td11">      <span 
class="phvr8t-x-x-109">26</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-3"  
class="td11">        <span 
class="phvr8t-x-x-109">2876</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-9-4"  
class="td11">       <span 
class="phvr8t-x-x-109">51</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-10-1"  
class="td
11"><span 
class="phvr8t-x-x-109">rdgA             </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-2"  
class="td11">      <span 
class="phvr8t-x-x-109">35</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-3"  
class="td11">        <span 
class="phvr8t-x-x-109">3003</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-10-4"  
class="td11">       <span 
class="phvr8t-x-x-109">89</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-11-1"  
class="td11"><span 
class="phvr8t-x-x-109">Mbs              </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-2"  
class="td11">      <span 
class="phvr8t-x-x-109">39</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-3"  
class="td11">        <span 
class="phvr8t-x-x-109">3080</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-11-4"  
class="td11">      <span 
class="phvr8t-x-x-109">119</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-12-1"  
class="td
11"><span 
class="phvr8t-x-x-109">CaMKI          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-2"  
class="td11">      <span 
class="phvr8t-x-x-109">41</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-3"  
class="td11">        <span 
class="phvr8t-x-x-109">3992</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-12-4"  
class="td11">        <span 
class="phvr8t-x-x-109">7</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-13-1"  
class="td11"><span 
class="phvr8t-x-x-109">par-1            </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-2"  
class="td11">      <span 
class="phvr8t-x-x-109">48</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-3"  
class="td11">        <span 
class="phvr8t-x-x-109">4410</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-13-4"  
class="td11">      <span 
class="phvr8t-x-x-109">142</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-14-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-14-1"  
class="td11"><span 
class="phvr8t-x-x-109">GluClalpha   </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-2"  
class="td11">      <span 
class="phvr8t-x-x-109">27</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-3"  
class="td11">        <span 
class="phvr8t-x-x-109">4945</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-14-4"  
class="td11">      <span 
class="phvr8t-x-x-109">188</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-15-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-15-1"  
class="td11"><span 
class="phvr8t-x-x-109">Sap47          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-2"  
class="td11">      <span 
class="phvr8t-x-x-109">24</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-3"  
class="td11">        <span 
class="phvr8t-x-x-109">5011</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-15-4"  
class="td11">       <span 
class="phvr8t-x-x-109">49</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-16-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-16-1"  
class="td11"><span 
class="phvr8t-x-x-109">Patronin        </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-2"  
class="td11">      <span 
class="phvr8t-x-x-109">50</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-3"  
class="td11">        <span 
class="phvr8t-x-x-109">5615</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-16-4"  
class="td11">      <span 
class="phvr8t-x-x-109">590</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-17-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-17-1"  
class="td11"><span 
class="phvr8t-x-x-109">CG17838     </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-2"  
class="td11">      <span 
class="phvr8t-x-x-109">37</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-3"  
class="td11">        <span 
class="phvr8t-x-x-109">8333</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-17-4"  
class="td11">      <span 
class="phvr8t-x-x-109">147</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-18-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-18-1"  
class="td11"><span 
class="phvr8t-x-x-109">unc-13          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-2"  
class="td11">      <span 
class="phvr8t-x-x-109">52</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-3"  
class="td11">        <span 
class="phvr8t-x-x-109">8391</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-18-4"  
class="td11">      <span 
class="phvr8t-x-x-109">279</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-19-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-19-1"  
class="td11"><span 
class="phvr8t-x-x-109">A2bp1          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-2"  
class="td11">      <span 
class="phvr8t-x-x-109">29</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-3"  
class="td11">        <span 
class="phvr8t-x-x-109">9055</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-19-4"  
class="td11">       <span 
class="phvr8t-x-x-109">58</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-20-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-20-1"  
class="td11"><span 
class="phvr8t-x-x-109">Imp               </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-2"  
class="td11">      <span 
class="phvr8t-x-x-109">33</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-3"  
class="td11">        <span 
class="phvr8t-x-x-109">9131</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-20-4"  
class="td11">       <span 
class="phvr8t-x-x-109">12</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-21-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-21-1"  
class="td11"><span 
class="phvr8t-x-x-109">pan               </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-2"  
class="td11">      <span 
class="phvr8t-x-x-109">38</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-3"  
class="td11">        <span 
class="phvr8t-x-x-109">9432</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-21-4"  
class="td11">       <span 
class="phvr8t-x-x-109">72</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-22-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-22-1"  
class="td11"><span 
class="phvr8t-x-x-109">Sh                </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-2"  
class="td11">      <span 
class="phvr8t-x-x-109">40</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-3"  
class="td11">       <span 
class="phvr8t-x-x-109">15995</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-22-4"  
class="td11">       <span 
class="phvr8t-x-x-109">66</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-23-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-23-1"  
class="td11"><span 
class="phvr8t-x-x-109">gish              </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-2"  
class="td11">      <span 
class="phvr8t-x-x-109">48</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-3"  
class="td11">       <span 
class="phvr8t-x-x-109">18972</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-3-23-4"  
class="td11">      <span 
class="phvr8t-x-x-109">142</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-3-24-"><td  style="white-space:nowrap; text-align:left;" id="TBL-3-24-1"  
class="td11">           </td></tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;1.1: </span><span  
class="content">Fly genes with &#x003E;2,000 assembled transcripts according to [<a 
href="#XBrown2014">Brown
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>].</span></div><!--tex4ht:label?: x1-17003r1 -->
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
<a 
 id="x1-17004r32"></a>
<h4 class="subsectionHead"><span class="titlemark">1.3.6 </span> <a 
 id="x1-180006"></a><span 
class="phvro8t-x-x-120">Drosophila melanogaster Dscam1</span></h4>
<!--l. 214--><p class="noindent" >Unquestionably, the gene most frequently used to demonstrate the combinatorial
power of AS is fly <span 
class="phvro8t-x-x-120">Dscam1</span>. The &#8220;architecture&#8221; of <span 
class="phvro8t-x-x-120">Dscam1 </span>is rather unique among
other organisms, but as we saw in Section <a 
href="#x1-170005">1.3.5<!--tex4ht:ref: sec:IsoformsPerGene --></a>, contain some genes that generate
tremendous isoform diversity from a single genetic locus [<a 
href="#XBrown2014">Brown et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrown2014">2014</a>]. The
basic structure of <span 
class="phvro8t-x-x-120">Dscam1 </span>is shown in Figure <a 
href="#x1-18001r9">1.9<!--tex4ht:ref: fig:DscamArch --></a>.
<!--l. 216--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-18001r9"></a>
                                                                                

                                                                                
<!--l. 218--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.57x.png" alt="PIC" class="graphics" width="391.4625pt" height="73.27374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.57x.png" src="Figures/Chapter1/DscamArch.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.9: </span><span  
class="content">The architecture of the <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>gene <span 
class="phvro8t-x-x-120">Dscam1</span><br 
class="newline" /><span 
class="phvro8t-x-x-120">Dscam1 </span>has three <span 
class="phvro8t-x-x-120">clusters </span>or &#8220;banks&#8221; of alternative cassette exons that are
splicing out in a mutually-exclusive manner. The first bank, &#8220;Exon 4&#8221;, contains
12 different variants, of which one one is ever included into the mRNA. Similarly,
banks 6 &amp; 9 each contain 48 and 33 different variants, respectively. These three
banks code for extracellular IgG domains, while the final region of AS, exon
17, encodes two different trans-membrane domains, again of which only one is
included in the final mRNA. </span></div><!--tex4ht:label?: x1-18001r9 -->
                                                                                

                                                                                
<!--l. 225--><p class="noindent" ></div><hr class="endfigure">
<!--l. 227--><p class="noindent" >Human <span 
class="phvro8t-x-x-120">Dscam</span>, for which <span 
class="phvro8t-x-x-120">Dscam1 </span>was named, was identified while looking for genes
on chromosome 21, specifically band 21q22, where extra copies expressed in Down
syndrome patients, a trisomy 21 disorder, may be causative for disease [<span 
class="phvb8t-x-x-120">?</span>]. <span 
class="phvro8t-x-x-120">Dscam</span>
(Down Syndrome Cellular Adhesion Molecule) was named according to this
association, and its membership in the immunoglobulin super family of proteins
with extracellular adhesion functions. Human <span 
class="phvro8t-x-x-120">Dscam </span>does undergo some
alternatively splicing and broadly expressed in the developing nervous system.
Yet, it does not contain the same architecture of cassette exon banks as
<span 
class="phvro8t-x-x-120">Dscam1</span>.
<!--l. 229--><p class="noindent" >Complex AS of<span 
class="phvro8t-x-x-120">Dscam1 </span>was first noticed by the Zipursky lab in 2000 [<a 
href="#XSchmucker2000">Schmucker
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchmucker2000">2000</a>]. While looking for proteins associated with <span 
class="phvro8t-x-x-120">dock </span>and <span 
class="phvro8t-x-x-120">pak</span>, two
proteins important for neuronal growth cone guidance, they biochemically
co-purified DSCAM1. Sequencing of <span 
class="phvro8t-x-x-120">Dscam1 </span>clones revealed that virtually all
contained different combinations of exons 4,6, and 9. In fact, these three
exons are chosen from three clusters of mutually-exclusive cassette exons,
containing 12, 48, and 33 different options each(see Figure <a 
href="#x1-18001r9">1.9<!--tex4ht:ref: fig:DscamArch --></a>). The initial
report kicked off an exciting period of research into <span 
class="phvro8t-x-x-120">Dscam1 </span>structure and
function. The functional significance of <span 
class="phvro8t-x-x-120">Dscam1 </span>AS was a major goal of multiple
labs.
<!--l. 231--><p class="noindent" >Before the highlights of <span 
class="phvro8t-x-x-120">Dscam1 </span>research are reviewed, it is illustrative to discuss
some basic <span 
class="phvro8t-x-x-120">Drosophila melanogaster </span>anatomy. There are 4 main regions where
<span 
class="phvro8t-x-x-120">Dscam1 </span>expression has been highly-studied. These four biologically important roles
are shown in Figure <a 
href="#x1-18002r10">1.10<!--tex4ht:ref: fig:DscamAnatomy --></a>.
<ul class="itemize1">
<li class="itemize">Hemocyte cells of the immune system
</li>
<li class="itemize">Larva Class IV da Neurons
</li>
<li class="itemize">Pupal Mushroom-body neurons in the developing brain
</li>
<li class="itemize">Tetrad synapses of the eye</li></ul>
<!--l. 240--><p class="noindent" >First, <span 
class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells of the immune system is important for
recognition of foreign antigens [<a 
href="#XWatson2005">Watson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatson2005">2005</a>].
<!--l. 242--><p class="noindent" >During larval development, <span 
class="phvro8t-x-x-120">Dscam1 </span>is expressed in the da neurons of the larval body
wall. Da neurons create a uniform sensory feed, allowing the larva to respond to
mechanical stimulus. Morphologicially da neurons resemble an oak tree growing in a
sunny field, allowing the larvae to sense as much as possible. In order to maximize
                                                                                

                                                                                
coverage of the field, every cell:cell interaction (i.e. every synapse) must be a
productive one. Molecularly, this is accomplished via an extracellular handshake
between two copies of DSCAM1. If this handskake feels too familiar, a stable, lasting,
and *productive* synapse is actively discouraged until a new and different handshake
is felt.
<!--l. 244--><p class="noindent" >The use of DSCAM1 to discern self from non-self determination is not unique
to da nuerons, but is also essential on equally critical, and arguably more
complex, nervous systems including the eye and brain. In the developing brain,
<span 
class="phvro8t-x-x-120">Dscam1 </span>is expressed in both axonal projections of neurons extend from their
Kenyon cell bodies and bifurcate into two different mushroom body lobs [<a 
href="#XZhan2004">Zhan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhan2004">2004</a>].
<!--l. 246--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-18002r10"></a>
                                                                                

                                                                                
<!--l. 248--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.58x.png" alt="PIC" class="graphics" width="397.48499pt" height="516.93124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.58x.png" src="Figures/Chapter1/DscamAnatomy.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.10: </span><span  
class="content">Important <span 
class="phvro8t-x-x-120">Dscam1 </span>expression during <span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>
cycle<br 
class="newline" /><span 
class="phvro8t-x-x-120">Dscam1 </span>has   been   high-studied   in   four   different   regions/cell   types.   1)
Hemocytes of  the  immune  system,  where  DSCAM1  is  involved  in  antigen
                                                                                

                                                                                
recognition; 2) In Class IV da neurons, which sense mechanical stimulation of
the larval body  wall;  3)  In  mushroom  body  neurons  of  the  pupal  developing
brain; and 4) (not shown) in Tetrad neurons of the eyes. </span></div><!--tex4ht:label?: x1-18002r10 -->
                                                                                

                                                                                
<!--l. 255--><p class="noindent" ></div><hr class="endfigure">
<!--l. 257--><p class="noindent" ><a 
href="#XCelotto2001">Celotto and Graveley</a>&#x00A0;[<a 
href="#XCelotto2001">2001</a>] investigated developmental regulation of <span 
class="phvro8t-x-x-120">Dscam1</span>. They
focused on the 12 variants of cluster 4, and observed regulation of exon 4.2, with
embryonic transcripts showing little inclusion of this exon, while adult transcripts
showed frequent inclusion. Exon 4.8 demonstrated the opposite behavior. Similar
regulation of exons with cluster 4 was also observed in a closely related species,
<span 
class="phvro8t-x-x-120">Drosophila yakuba</span>.
<!--l. 259--><p class="noindent" >Four years after <span 
class="phvro8t-x-x-120">Dscam1 </span>complexity was first reported, [<a 
href="#XNeves2004">Neves et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNeves2004">2004</a>] used a
specially designed microarray to robustly characterize its molecular diversity. They
observed, at some rate, inclusion of virtually all alternative exons with each of the
three clusters examined. Additionally, they examined <span 
class="phvro8t-x-x-120">Dscam1 </span>transcripts obtained
from colonies grown from single cells and reported that multiple <span 
class="phvro8t-x-x-120">Dscam1 </span>transcripts
were expressed per cell, estimating the number to be between 7&#8211;50 different
combinations, depending on the cell type. As discussed above, the use of microarrays
to perform this analysis precluded observing any potential coordination between
variant exons.
<!--l. 261--><p class="noindent" >Quickly after <a 
href="#XNeves2004">Neves et&#x00A0;al.</a>&#x00A0;[<a 
href="#XNeves2004">2004</a>] published their results, the Zipursky lab also
published a microarray study of <span 
class="phvro8t-x-x-120">Dscam1 </span>isoforms [<a 
href="#XZhan2004">Zhan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhan2004">2004</a>]. They focused
their analysis to neurons of the developing Mushroom body (see Figure <a 
href="#x1-18002r10">1.10<!--tex4ht:ref: fig:DscamAnatomy --></a>). Not
only did they also show that most <span 
class="phvro8t-x-x-120">Dscam1 </span>combinations are likely produced at some
level, but the diversity of isoforms is required for bifurcation of neurons into
different lobes of the developing mushroom body. These results highlighted a
critical function for DSCAM&#8211;mediating extracellular interactions via homophillic
binding.
<!--l. 263--><p class="noindent" >Having some functional purpose for <span 
class="phvro8t-x-x-120">Dscam1 </span>diversity, the next area of research
focused on mechanisms of mutually exon selection. <a 
href="#XGraveley2005b">Graveley</a>&#x00A0;[<a 
href="#XGraveley2005b">2005</a>] observed a
single &#8220;docking site&#8221; within the intronic sequence just 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; to exon 6.1. This docking
site was conserved among 15 insect species examined, from closely-related
<span 
class="phvro8t-x-x-120">Drosophila simulans </span>to a distantly-related <span 
class="phvro8t-x-x-120">Tribolium castaneum </span>(Red flour beetle).
Astonishingly, the docking site was complementary to &#8220;selector sites&#8221; within
intronic regions just 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; of each of the 48 variant exons. A model is proposed
where docking::selector site interactions is required to choosing which of
the variant exons is included, while a splicing regulator protein, likely and
hnRNP due to the repressive nature of the interaction [<a 
href="#XGraveley2000">Graveley</a>,&#x00A0;<a 
href="#XGraveley2000">2000</a>].
Additional mechanisms have been reported for other clusters, including the <span 
class="phvro8t-x-x-120">iStem</span>
[<a 
href="#XKreahling2005">Kreahling and Graveley</a>,&#x00A0;<a 
href="#XKreahling2005">2005</a>] in cluster 4, and the hnRNP protein hrp36 [<a 
href="#XOlson2007">Olson
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XOlson2007">2007</a>].
<!--l. 265--><p class="noindent" >[<a 
href="#XNeves2004">Neves et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNeves2004">2004</a>] examined <span 
class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells, and their
results clearly show reduced variability in cluster 9 inclusion. Virtually all of the signal
                                                                                

                                                                                
obtained from hemocyte cells for cluster 9 was seen in variants 9.[6,9,13,30,and 31].
[<a 
href="#XWatson2005">Watson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatson2005">2005</a>] also examined <span 
class="phvro8t-x-x-120">Dscam1 </span>expression in hemocyte cells,
comparing it to that of neuronal cells. They propose that secreted forms of
<span 
class="phvro8t-x-x-120">Dscam1 </span>are essential for a robust innate immune system in insects. These
studies highlight how nature has applied one gene that produced extreme
molecular diversity to multiple problems involving determing self from non-self
[<a 
href="#XHattori2008">Hattori et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHattori2008">2008</a>,&#x00A0;<a 
href="#XShi2012a">Shi and Lee</a>,&#x00A0;<a 
href="#XShi2012a">2012</a>]. <span 
class="phvro8t-x-x-120">Dscam1 </span>use in these two very
different biologically roles has been summarized previously [<a 
href="#XHemani2012">Hemani and
Soller</a>,&#x00A0;<a 
href="#XHemani2012">2012</a>].
<!--l. 267--><p class="noindent" >Beginning in 2005 Hattori series &#8211; Minimal require divsersity [<a 
href="#XHattori2005a">Consortium</a>,&#x00A0;<a 
href="#XHattori2005a">2004</a>,&#x00A0;<a 
href="#XHattori2009">Hattori
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHattori2009">2009</a>,&#x00A0;<a 
href="#XHattori2008">2008</a>]
<a 
 id="x1-18003r24"></a>
<h3 class="sectionHead"><span class="titlemark">1.4 </span> <a 
 id="x1-190004"></a>Nucleic Acid Ligation</h3>
<a 
 id="x1-19001r36"></a>
<h4 class="subsectionHead"><span class="titlemark">1.4.1 </span> <a 
 id="x1-200001"></a>RNA Sequence investigation by ligation</h4>
<!--l. 277--><p class="noindent" >In the late 1960&#8217;s and early 1970&#8217;s, the Lehman and Richardson labs characterized
two workhorse-enzymes of modern molecular biology. Robert Lehman and
colleagues, working at Standford Medical School, first described the activity of
<span 
class="phvro8t-x-x-120">polynucleotide-joining enzyme </span>from <span 
class="phvro8t-x-x-120">Escherichia coli </span>(now known as <span 
class="phvro8t-x-x-120">E. Coli </span>DNA
Ligase) [<a 
href="#XOlivera1967b">Olivera and Lehman</a>,&#x00A0;<a 
href="#XOlivera1967b">1967</a>]. Work on this enzyme paralleled that
from the Richardson lab at Harvard Medical School, where they focused on
<span 
class="phvro8t-x-x-120">polynucleotide ligase </span>from <span 
class="phvro8t-x-x-120">Escherichia coli </span>infected with T4 bacteriophage
(now known as T4 DNA ligase) [<a 
href="#XWeiss1967a">Weiss and Richardson</a>,&#x00A0;<a 
href="#XWeiss1967a">1967</a>]. It became
clear that while these two enzyme&#8217;s shared a common mechanism&#8212;later
elucidated by [<a 
href="#XModrich1973a">Modrich et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModrich1973a">1973</a>]&#8212;they had important differences. First, T4
DNA ligase required ATP as a cofactor, which <span 
class="phvro8t-x-x-120">E. Coli </span>DNA Ligase did not
(though it was later discovered that DNA ligase required NAD as a cofactor).
Second, only T4 DNA ligase could catalyze ligation of blunt-ended DNA
[<a 
href="#XTabor1987a">Tabor</a>,&#x00A0;<a 
href="#XTabor1987a">1987</a>].
<!--l. 279--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-20001r11"></a>
                                                                                

                                                                                
<!--l. 281--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.59x.png" alt="PIC" class="graphics" width="374.39874pt" height="518.93874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.59x.png" src="Figures/Chapter1/LigationMechanism.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.11: </span><span  
class="content">Mechanism of ATP-dependent ligation<br 
class="newline" />Adapted from [<a 
href="#XNandakumar2006">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>] and specifically for that of T4 RNA
ligase 2. </span></div><!--tex4ht:label?: x1-20001r11 -->
                                                                                

                                                                                
<!--l. 288--><p class="noindent" ></div><hr class="endfigure">
<!--l. 290--><p class="noindent" >The general mechanism of ligation, shown in Figure <a 
href="#x1-20001r11">1.11<!--tex4ht:ref: fig:Ligation Mechanism --></a>, involves three steps: Step 1
(A) involves the <span 
class="cmmi-12">&#x03F5;</span>-amino group from the active site lysine performs a nucleophilic
attack on the <span 
class="cmmi-12">&#x03B1;</span>-phosphate of ATP in solution. B) The ligase is now charged with AMP
and inorganic phosphate (PPi) is freed into solution. C) Step 2: Nucleophilic attack by
the 5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; DNA phosphate on the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick to the AMP:ligase phosphate. D)
Adenylated DNA is now competent for DNA ligation. E) Step 3: the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
side of the nick performs a nucleophilic attack on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> across the DNA nick,
liberating AMP into solution. F) Sealed nick resulting in: Ligase; AMP; and intact
dsDNA.
<!--l. 292--><p class="noindent" >In addition to elucidated the general mechanism of ligation, it was also discovered
that T4 DNA ligase lacks a preference for terminal polynucleotide structures. The
Khorana and Richardson labs both reported the activity of this enzyme on
combinations of RNA and DNA duplexes [<a 
href="#XFareed1971">Fareed et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFareed1971">1971</a>,&#x00A0;<a 
href="#XKleppe1970b">Kleppe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKleppe1970b">1970</a>].
Both of these papers describe an activity on T4 DNA ligase, <span class="underline">RNA-templated DNA to
DNA ligation</span>, that is of particular relevance to this thesis work. Unlike T4 DNA ligase,
<span 
class="phvro8t-x-x-120">E. Coli </span>DNA Ligase, will not join DNA strands on an RNA template [<a 
href="#XBullard2006">Bullard and
Bowater</a>,&#x00A0;<a 
href="#XBullard2006">2006</a>]. Soon after demonstrating these activities in vitro, the Khorana lab
reported detection of organism-generated DNA [<a 
href="#XBesmer1972b">Besmer et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBesmer1972b">1972</a>], setting up an
orthogonal field (respective to PCR) of nucleic acid sequence characterization [<a 
href="#XConze2009c">Conze
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XConze2009c">2009</a>].
<!--l. 294--><p class="noindent" >An enzyme that can catalyze an RNA-templated DNA:DNA ligation is a very useful
molecular biology tool for two main reasons. First, using RNA as a ligation guide
means no modification is made to the template molecule. This contrasts cDNA
analysis, where the RNA has been enzymatically converted by reverse transcription,
potentially losing valuable RNA-coded information, such as modified bases. Second,
synthesis of the DNA probes used in ligation is inherently easier and cheaper
compared to synthesis of RNA probes. In addition to being cheaper, synthesis of DNA
probes has become high-throughput since the adoption of microarrays as a standard
gene expression measurement tool [<a 
href="#XSchena1995a">Schena et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchena1995a">1995</a>]. A pair of papers
from the Landegren lab first reported the utility of RNA-templated DNA:DNA
ligation for analysis of RNA transcripts [<a 
href="#XNilsson2001">Nilsson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNilsson2001">2001</a>,&#x00A0;<a 
href="#XNilsson2000">2000</a>]. The Fu lab
applied this approach in a multiplex experimental design in collaboration with
Illumina [<a 
href="#XLi2012c">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2012c">2012</a>,&#x00A0;<a 
href="#XYeakley2002">Yeakley et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYeakley2002">2002</a>], while Mats Nilsson and Ulf
Landegren developed a single molecule application [<a 
href="#XConze2010">Conze et&#x00A0;al.</a>,&#x00A0;<a 
href="#XConze2010">2010</a>]. It
is important to note that <span 
class="phvro8t-x-x-120">all </span>of these studies used T4 DNA ligase. Clearly,
there is interest and utility in analyzing RNA in both high-throughput and
multiplex experimental designs, using cheap DNA probes, and without cDNA
conversion.
                                                                                

                                                                                
<!--l. 296--><p class="noindent" >For more than 40 years after its first description, T4 DNA ligase was the only choice
for RNA-templated DNA:DNA ligation. However, a recent publication from New
England Biolabs (NEB) describes this activity by another well-studied ligase, Chlorella
Virus PBCV-1 DNA ligase (herein Chlorella DNA ligase) [<a 
href="#XLohman2013c">Lohman et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLohman2013c">2013</a>].
Chlorella DNA ligase is a long-studied enzyme and had been reported to <span 
class="phvro8t-x-x-120">not </span>display
RNA-templated DNA:DNA ligation activity [<a 
href="#XHo1997b">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo1997b">1997</a>,&#x00A0;<a 
href="#XSriskanda1998c">Sriskanda and
Shuman</a>,&#x00A0;<a 
href="#XSriskanda1998c">1998</a>]. However, at high enough concentrations and under special buffer
conditions (specifically a critical concentration of ATP), Lohman et al have shown that
Chlorella DNA ligase will join two DNA strands hybridized to an RNA template
[<a 
href="#XLohman2013c">Lohman et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLohman2013c">2013</a>]. They further demonstrated that it performs no worse in
this activity than traditional T4 DNA ligase [<a 
href="#XNilsson2001">Nilsson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNilsson2001">2001</a>,&#x00A0;<a 
href="#XYeakley2002">Yeakley
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYeakley2002">2002</a>].
<!--l. 298--><p class="noindent" >Building on the list of available enzymes that join hybrid polymer substrates Chapter <a 
href="#x1-290002">2<!--tex4ht:ref: Chapter2 --></a>
presents data supporting RNA-templated DNA:DNA ligation activity for another
enzyme, T4 RNA Ligase 2.
<a 
 id="x1-20002r40"></a>
<h4 class="subsectionHead"><span class="titlemark">1.4.2 </span> <a 
 id="x1-210002"></a>T4 RNA Ligase 2 (Rnl2)</h4>
<!--l. 302--><p class="noindent" >Proteins of the T4 and T7 bacteriophages have been a boon for molecular biology.
Without enzymes like polynucleotide kinase [<a 
href="#XRichardson1965a">Richardson</a>,&#x00A0;<a 
href="#XRichardson1965a">1965</a>], T7 RNA
polymerase [<a 
href="#XSummers1970b">Summers and Siegel</a>,&#x00A0;<a 
href="#XSummers1970b">1970</a>], and T4 DNA ligase [<a 
href="#XWeiss1967a">Weiss and
Richardson</a>,&#x00A0;<a 
href="#XWeiss1967a">1967</a>], many essential manipulations of nucleic acids would
have been impossible for decades. Obviously, these enzymes also have
essential phage functions. T7 RNA polymerase is responsible for late stage
replication of T7 phage transcripts, while T4 PNK works in concert with T4
DNA and RNA ligases to repair cleaved nucleic acids resulting from bacterial
pathogens defense systems [<a 
href="#XWang2002b">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2002b">2002</a>]. Specifically, T4 RNA ligase 1
(herein &#8220;Rnl1&#8221;, also known as <span 
class="phvro8t-x-x-120">gene 63 </span>maintains phage replication by repairing
tRNAs cleaved by an anticodon nuclease produces from the <span 
class="phvro8t-x-x-120">prr </span>locus [<a 
href="#XAmitsur1987d">Amitsur
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAmitsur1987d">1987</a>].
<!--l. 304--><p class="noindent" >Given the utility and importance of these enzymes, novel enzyme discovery is a
fruitful area of research. The Shuman lab has a distinguished record of discovering
and characterizing numerous such enzymes, including any involved in nucleic acid
synthesis, modification, and repair. Through a blast search looking for novel
ligases with sequences related to <span 
class="phvro8t-x-x-120">Trypanosoma brucei </span>RNA-editing ligases
TbMP52 and TbMP48 [<a 
href="#XHo2002b">Ho and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>], they identified motifs in correct
arrangement, spacing, and number indicative of an RNA ligase. The gene,
identified as <span 
class="phvro8t-x-x-120">gp24.1</span>, has quickly become an essential tool in the era of modern
genomics.
                                                                                

                                                                                
<!--l. 306--><p class="noindent" >Initial biochemical purification and characterization of <span 
class="phvro8t-x-x-120">gp24.1 </span>[<a 
href="#XHo2002b">Ho and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>]
revealed that it indeed codes for an RNA ligase, which was renamed T4 RNA ligase 2
(herein &#8220;Rnl2&#8221;). Rnl2 is a 374 amino acid monomeric protein composed of 2
distinct domains initially purified as a 42-kDA His-tagged recombinant protein.
The N-terminal domain (1&#8211;243) is responsible for steps (1) and (3) of the
general ligation mechanisms (see Figure <a 
href="#x1-20001r11">1.11<!--tex4ht:ref: fig:Ligation Mechanism --></a>), while the C-terminal domain
(244&#8211;329) is responsible for adenylation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; residue at
the 3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; side of the nick, as shown in step (2). Additionally, Rnl2 is routinely
purified as a pre-adenylated and immediately poised for its first ligation. In
contrast to the N-terminal domain, which is composed of motifs typical to main
ligases, the C-terminal domain is significantly different from all other DNA
ligases and has no structural homologue. While the biological function of Rnl1
is known, the biological function of Rnl2 remains a mystery, more than 12
years after its discovery [<a 
href="#XChauleau2013b">Chauleau and Shuman</a>,&#x00A0;<a 
href="#XChauleau2013b">2013</a>]. However, there is
some speculation that the flurry of research into bacterial CRISPR phage
defense may reveal a role for Rnl2 [<a 
href="#XBarrangou2007c">Barrangou et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBarrangou2007c">2007</a>,&#x00A0;<a 
href="#XChauleau2013b">Chauleau and
Shuman</a>,&#x00A0;<a 
href="#XChauleau2013b">2013</a>].
<!--l. 308--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-21001r12"></a>
                                                                                

                                                                                
<!--l. 310--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.510x.png" alt="PIC" class="graphics" width="355.3275pt" height="292.09125pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.510x.png" src="Figures/Chapter1/Rnl2_Structure.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.12: </span><span  
class="content">Structure and active site of pre-adenylated of Rnl2<br 
class="newline" />Rnl2 as crystalized and described by [<a 
href="#XNandakumar2006">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>]. Structures
from PDB:2HVQ  were  generated  with  PyMol.  Top  left)  Rnl2  is  composed
of a C-terminal  and  N-terminal  domain.  Top  Right)  The  active  site  of  Rnl2
is highlighted.  Bottom  left)  Active  site  of  Rnl2  as  shown  from  bottom.  This
face interacts with substrate. Residue numbering refers to that of the crystal
structure. </span></div><!--tex4ht:label?: x1-21001r12 -->
                                                                                

                                                                                
<!--l. 317--><p class="noindent" ></div><hr class="endfigure">
<!--l. 319--><p class="noindent" >Mutational analysis of Rnl2, and later a crystal structure of the enzyme, have identified
key functional residues [<a 
href="#XHo2004">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo2004">2004</a>,&#x00A0;<a 
href="#XNandakumar2004a">Nandakumar et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2004a">2004</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>,&#x00A0;<a 
href="#XYin2003d">Yin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYin2003d">2003</a>]. The lysine residue at position 35 (K35) receives the AMP in Step 1. The
K227 residue in the C-terminal domain is essential for both forward and reverse
adenylation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; PO<sub>
<span 
class="cmr-8">4</span></sub> at the nick [<a 
href="#XViollet2011">Viollet et&#x00A0;al.</a>,&#x00A0;<a 
href="#XViollet2011">2011</a>]. Mutation of H37 results in
an &#x00A0;102 reduced ligation rate, and therefore indicates the essential nature of this
residue. Finally, T39 has been shown to interact with the 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH on the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the
nick, preferring a C3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; endo sugar pucker confirmation (see Figure <a 
href="#x1-21002r13">1.13<!--tex4ht:ref: fig:Rnl2 Active Site Residues --></a>). Rnl2 has a
minimal footprint of 13nt, centered on the nick, and only requires magnesium for
transfer of AMP to the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; phosphate. Work done in the Shuman lab [<a 
href="#XNandakumar2006">Nandakumar
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>] observed that 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0; deoxyribose residues on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick (i.e.
DNA) adopt an RNA-like sugar pucker, leading to the correct orientation of
the 3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; OH relative to the AMP leaving group and resulting in ligation. This
conformation is of particular importance to this results presented in Chapter
<a 
href="#x1-290002">2<!--tex4ht:ref: Chapter2 --></a>.
<!--l. 321--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-21002r13"></a>
                                                                                

                                                                                
<!--l. 323--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.511x.png" alt="PIC" class="graphics" width="216.81pt" height="217.81374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.511x.png" src="Figures/Chapter1/Rnl2_Active_Site_Residues.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.13: </span><span  
class="content">Structure and active site of pre-adenylated of Rnl2<br 
class="newline" />Rnl2 complexed   with   nicked   dsDNA   as   crystalized   and   described   by
[<a 
href="#XNandakumar2006">Nandakumar  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>].  Structures  from  PDB:2HVR  and  generated  with
PyMol </span></div><!--tex4ht:label?: x1-21002r13 -->
                                                                                

                                                                                
<!--l. 330--><p class="noindent" ></div><hr class="endfigure">
<!--l. 332--><p class="noindent" >While Rnl2 is extremely efficient at high concentration, displaying little or no
reversible chemistry, a modified version of the enzyme containing only the
N-terminal domain and a K227A point mutation (&#8220;Truncated mutant&#8221;) has no
adenyltransferase activity. In this case, adenyltransferase refers to the ligase
transferring AMP from an adenylated substrate to itself; reverse chemistry of step 2 in
Figure <a 
href="#x1-20001r11">1.11<!--tex4ht:ref: fig:Ligation Mechanism --></a>). This mutant has been used in specialized cloning applications
[<a 
href="#XGhildiyal2008">Ghildiyal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XHafner2008a">Hafner et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHafner2008a">2008</a>,&#x00A0;<a 
href="#XViollet2011">Viollet et&#x00A0;al.</a>,&#x00A0;<a 
href="#XViollet2011">2011</a>] that take
advantage of this activity. In these reactions, the use of pre-adenylated 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; DNA
adaptors allows for selective ligation among already phosphorylated species by
limiting the enzyme-catalyzed transfer of AMP from the adaptor to other
phosphorylated species. Use of this truncated mutant to create a hybrid
RNA/DNA molecule has greatly improved many high-throughput sequencing work
flows.
<!--l. 334--><p class="noindent" >Ligation of hybrid substrates (eg. DNA-templated RNA:DNA vs DNA-templated
DNA:DNA) have revealed general substrate preferences. DNA ligases appear to
prefer the residue bearing the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; phosphate on the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the nick to be
2<span 
class="cmsy-8">&#x2032;</span> &#x00A0; deoxyribose, and have a relaxed requirement for the sugar on the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
side of the nick. RNA ligases have the reverse preference, demonstrating
higher activities when the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; strand, 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH residue also bears a 2<span 
class="cmsy-8">&#x2032;</span>OH.
Rnl2 has an additional preference for an RNA residue at the penultimate
3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; side of a nick [<a 
href="#XHo2002b">Ho and Shuman</a>,&#x00A0;<a 
href="#XHo2002b">2002</a>,&#x00A0;<a 
href="#XHo2004">Ho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHo2004">2004</a>,&#x00A0;<a 
href="#XNandakumar2004a">Nandakumar
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNandakumar2004a">2004</a>,&#x00A0;<a 
href="#XNandakumar2006">2006</a>]. The two base requirement for RNA at the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; side of the double
stranded nick biases Rnl2 to join RNA:[RNA/DNA] strands. Independent labs
have measured this preference and have reported that the RNA-templated
DNA:DNA joining activity of Rnl2 is below assay limits of detection [<a 
href="#XBullard2006">Bullard and
Bowater</a>,&#x00A0;<a 
href="#XBullard2006">2006</a>]. However, results discussed in this work clearly show that
with enough enzyme, and sensitive downstream measurements, Rnl2 will
catalyze RNA-templated DNA:DNA ligation (see Chapter <a 
href="#x1-290002">2<!--tex4ht:ref: Chapter2 --></a>. Previous reports of
Rnl2 lacking this activity are likely due to a single turnover mechanism in
this reaction, owning to the poor dissociation rate of nucleic acid-interacting
enzymes.
<a 
 id="x1-21003r39"></a>
<h3 class="sectionHead"><span class="titlemark">1.5 </span> <a 
 id="x1-220005"></a>Nucleic Acid Polymers</h3>
<a 
 id="x1-22001r42"></a>
<h4 class="subsectionHead"><span class="titlemark">1.5.1 </span> <a 
 id="x1-230001"></a>piRNAs original from very long PolII Transcripts</h4>
<a 
 id="x1-23001r1"></a>
                                                                                

                                                                                
<h5 class="subsubsectionHead"><span class="titlemark">1.5.1.1 </span> <a 
 id="x1-240001"></a>Brief history of piRNAs</h5>
<!--l. 352--><p class="noindent" >Mammalian spermatogenesis is critical for the future of the species. Recently the
importance a specific kind of small RNA&#8212;piRNAs&#8212;for proper spermatogenesis has
become clear [<a 
href="#XSiomi2011">Siomi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSiomi2011">2011</a>]. Even after &#x003E;12 years since their discovery in
<span 
class="phvro8t-x-x-120">Drosophila melanogaster</span>[<a 
href="#XAravin2001">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2001">2001</a>], and &#x003E;6 since their identification in
rodents [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>], the essential mechanisms of piRNA
biogenesis to proper mammalian spermatogenesis remains largely unknown. These
unknown mechanisms include: biogenesis from transcript to small RNA, physiological
targets, and terminal function of sterility maintenance. What is known is that without a
functioning piRNA pathway males are sterile. Studies in humans have also linked
SNPs in the Argonaute proteins that bind piRNAs to decreased fertility [<a 
href="#XGu2010a">Gu
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGu2010a">2010</a>].
<!--l. 354--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-24001r14"></a>
                                                                                

                                                                                
<!--l. 356--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.512x.png" alt="PIC" class="graphics" width="375.4025pt" height="234.87749pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.512x.png" src="Figures/Chapter1/FlamencoLocus.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.14:  </span><span  
class="content">A  the  <span 
class="phvro8t-x-x-120">Drosophila  melanogaster  </span>gene  <span 
class="phvro8t-x-x-120">flamenco  </span>is  a  graveyard
for transposon sequences [<a 
href="#XPelisson1994">Plisson et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPelisson1994">1994</a>]. Evidence for expression of a
single-contiguous RNA transcript from <span 
class="phvro8t-x-x-120">flamenco </span>(A) is provided by a P-element
insertion into the suspected promoter region (B). [<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>] could
not detect specific piRNAs (red X&#8217;s) by northern blot in the P-element mutant. </span></div><!--tex4ht:label?: x1-24001r14 -->
                                                                                

                                                                                
<!--l. 362--><p class="noindent" ></div><hr class="endfigure">
<!--l. 364--><p class="noindent" >piRNAs are so-called because they bind PIWI proteins, a sub-group of the Argonaute
protein family, whose other members utilize small RNA as guides to target nucleic
acids for many forms of post-transcriptional regulation [<a 
href="#XSiomi2011">Siomi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSiomi2011">2011</a>]. There are
three PIWI proteins in mice, each displaying a distinct expression profile
during development and an association with piRNAs of a specific length.
The first PIWI protein expressed, even before a mouse is born, is MIWI2
[<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>], followed quickly by the more consistent player, MILI (see
Figure <a 
href="#x1-24002r15">1.15<!--tex4ht:ref: fig:Mammalian piRNA classes --></a> [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">2004</a>]. It is
during the &#8220;fetal&#8221; stage of piRNA biogenesis in mice that MIWI2 and MILI
undergo ping-pong amplification, similar to that observed in flies, in order to
silence expression of transposons during germ line formation [<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">2008</a>]. After birth, and during
the &#8220;neonatal stage&#8221; only MILI is expressed, and piRNAs shift from mostly
transposon-mapping to 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; UTR mapping [<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>]. Once the &#8220;first wave&#8221;
of spermatogenesis [<a 
href="#XLaiho2013a">Laiho et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLaiho2013a">2013</a>,&#x00A0;<a 
href="#XOakberg1956b">Oakberg and Oakberq</a>,&#x00A0;<a 
href="#XOakberg1956b">1956</a>] reaches
meiosis, the pachytene piRNAs are expressed [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>,&#x00A0;<a 
href="#XLi2013h">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013h">2013a</a>]. Pachytene piRNAs tend to map to intergenic
&#8220;clusters&#8221; of unique genomic sequence. These clusters, from here after called
&#8220;piRNA-generating genes&#8221; appear to produce a single, continuous, relatively long,
and un-spliced Pol II transcript [<a 
href="#XLi2013h">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013h">2013a</a>]. This is comparable to piRNA
clusters in flies, such as flamenco, whose transcription can be abolished with a
P-element insertion into a putative promoter, as measured by northern blot
looking for piRNAs generated 168 kb downstream (see Figure <a 
href="#x1-24001r14">1.14<!--tex4ht:ref: fig:flamenco --></a> [<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>].
<!--l. 366--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-24002r15"></a>
                                                                                

                                                                                
<!--l. 368--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.513x.png" alt="PIC" class="graphics" width="415.55249pt" height="324.21124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.513x.png" src="Figures/Chapter1/MammalianPiRNAClassesOverTime.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;1.15: </span><span  
class="content"><span class="texthl"><span id="textcolor1">Write a nice caption here!</span></span> </span></div><!--tex4ht:label?: x1-24002r15 -->
                                                                                

                                                                                
<!--l. 373--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-24003r47"></a>
<h5 class="subsubsectionHead"><span class="titlemark">1.5.1.2 </span> <a 
 id="x1-250002"></a>Fly and Mouse piRNAs have important differences</h5>
<!--l. 377--><p class="noindent" >piRNAs are small RNAs with lengths 24&#8211;31 nt long, making them longer than
other small RNAs (eg. miRNAs or siRNAs). They are believed to derive from
single-stranded RNA precursors because also unlike other small RNAs, their
biogenesis does not require double-stranded RNA-specific ribonuclease Dicer
[<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>,&#x00A0;<a 
href="#XVagin2006">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2006">2006</a>]. Yet, similar to other small RNAs, they
bind a sub group of the Argonaute family of proteins, PIWI proteins, from which their
name is derived (<span 
class="phvro8t-x-x-120">PIWI Interacting RNAs</span>.
<!--l. 379--><p class="noindent" >Mammalian piRNAs can be divided into three major classes (see Figure <a 
href="#x1-24002r15">1.15<!--tex4ht:ref: fig:Mammalian piRNA classes --></a>.
<span 
class="phvro8t-x-x-120">Fetal piRNAs </span>are present before birth. These piRNAs tend to be short, bind
the PIWI protein MILI2 in mice, and have sequences found in transposable
elements [<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>]. The next class of piRNAs, historically but
confusing grouped with the previous class, are called <span 
class="phvro8t-x-x-120">Pre-pachytene piRNAs</span>.
Pre-pachytene piRNAs are expressed just before birth, and continue to be expressed
in functioning testes. These piRNAs tend to map to traditional, annotated,
protein-coding genes. Finally, due to their unique sequence in the genome, the
genetic origin of millions of piRNAs belonging to the third class, the <span 
class="phvro8t-x-x-120">pachytene</span>
<span 
class="phvro8t-x-x-120">piRNAs</span>, was immediately known. Pachytene piRNAs are extremely abundant
after the pachytene stage of meiosis I when chromosomes pair up, cross
over, and exchange genetic material. The genomic origin of these piRNAs,
while unique in terms of sequence, are often in &#8220;gene deserts&#8221;&#8211;unannotated
and devoid of introns. This gene architecture makes the pachytene piRNA
loci some of the most interesting RNA-producing regions of the mammalian
genome.
<!--l. 381--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-25001r16"></a>
                                                                                

                                                                                

<!--l. 383--><p class="noindent" ><img 
src="Figures/Chapter1/mammalian_piRNA_pathway.png" alt="PIC"  
>
<br />  <div class="caption" 
><span class="id">Figure&#x00A0;1.16:   </span><span  
class="content">Figure   taken   from   [<a 
href="#XLi2013e">Li   et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013e">2013b</a>]:   A   model   for   piRNA
biogenesis. Primary piRNA transcripts are transcribed by RNA polymerase II
and  contain  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  caps,  exons  and  introns  and  poly(A)  tails.  The  transcription
of  pachytene  piRNA  genes  is  controlled  by  A-MYB;  transcription  factor(s)
(TF) controlling pre-pachytene piRNA genes remain to be discovered. Current
models  of  piRNA  biogenesis  propose  that  PLD6  determines  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of
piRNA intermediates with lengths &#x003E;30 nt. These intermediates are proposed
to  then  be  loaded  into  PIWI  proteins.  After  PIWI  binding,  a  nuclease  is
thought  to  trim  the  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  the  piRNA  to  the  length  characteristic  of
the  particular  bound  PIWI  protein.  Finally,  further  trimming  is  prevented  by
addition  of  a  2<span 
class="cmsy-8">&#x2032;</span>&#x00A0;-O-methyl  group  to  the  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  the  mature  piRNA  by
the S-adenosylmethionine-dependent methyltransferase HEN1. Figure adapted
from [<a 
href="#XLi2013">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013">2013c</a>]. </span></div><!--tex4ht:label?: x1-25001r16 -->
                                                                                

                                                                                
<!--l. 389--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-25002r50"></a>
<h5 class="subsubsectionHead"><span class="titlemark">1.5.1.3 </span> <a 
 id="x1-260003"></a>Known functions of mammalian piRNAs</h5>
<!--l. 393--><p class="noindent" >Two studies [<a 
href="#XDeFazio2011">De Fazio et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDeFazio2011">2011</a>,&#x00A0;<a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2011">2011</a>] used point mutations in the
catalytic triad of MIWI2, MILI to remove slicer activity. The MIWI2 and MILI studies
found that the mice were sterile, and did not accumulate transposon-mapping
piRNAs. <a 
href="#XDeFazio2011">De Fazio et&#x00A0;al.</a>&#x00A0;[<a 
href="#XDeFazio2011">2011</a>] found that the mice were also sterile, and
demonstrated increased LINE1 transcript accumulation. <a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>&#x00A0;[<a 
href="#XReuter2011">2011</a>] states
that much the biological activity of MIWI depends on its slicer activity.
<!--l. 395--><p class="noindent" >Mouse piRNAs have also been implicated in gene imprinting [<a 
href="#XWatanabe2011">Watanabe
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatanabe2011">2011b</a>].
<!--l. 397--><p class="noindent" >The transposon-mapping nature of the fetal piRNA class made obvious comparisons
to the fly piRNA system nature. In the fly system, primary piRNAs transcribed from
discrete loci and fed into an amplification loop between two PIWI proteins PIWI(3) and
AGO3 (&#8216;the ping-pong&#8217; cycle) (REF Brenneki cell paper 2007). It is believed that PIWI
proteins loaded with piRNAs bind and silence transposon messages, using the
cleaved transposon transcripts, in combination with primary piRNA transcripts, as a
substrates in the Ping-Pong cycle
<a 
 id="x1-26001r52"></a>
<h5 class="subsubsectionHead"><span class="titlemark">1.5.1.4 </span> <a 
 id="x1-270004"></a>Integration of multiple HTS datatypes of piRNA analysis</h5>
<!--l. 403--><p class="noindent" >+ Critical importance of integrating many different HTS datasets into mammalian
piRNA study
<!--l. 405--><p class="noindent" >+ RNA-Seq does not give precision necessary for annotation of 5&#8217; and 3&#8217;
ends.
<!--l. 407--><p class="noindent" >piRNAs lend themselves to study using HTS
<!--l. 409--><p class="noindent" >Require more datasets to work back to molecular precursors
<!--l. 411--><p class="noindent" >Cap-seq is too noisey to not have orthogonal dataset for comparison
<!--l. 413--><p class="noindent" >Precision of 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end is critical for proper measurement of proximity to suspected
transcription factor binding motifs and experimentally-determined ChIP signal
<!--l. 415--><p class="noindent" >PAS-Seq challenging due to internal priming sites, difficulty of HTS platforms to read
through homopolymers, and dynamic nature of 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end processing.
<a 
 id="x1-27001r46"></a>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">1.5.2 </span> <a 
 id="x1-280002"></a>Assembling full length transcripts from Short read data</h4>
<!--l. 419--><p class="noindent" >+ Current status and future of transcriptome assembly
<!--l. 421--><p class="noindent" >+ Reference most recent transcript assembly review.
<!--l. 423--><p class="noindent" >+ Most successful methods are guided by known annotations
<!--l. 425--><p class="noindent" >+ Unguided typically provide many discontinuous &#8216;contigs&#8217;
<!--l. 427--><p class="noindent" >+ Maybe assisted by deeper RNA-Seq
<!--l. 429--><p class="noindent" >+ Paired-end is critically important
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;2</span><br /><a 
 id="x1-290002"></a>SeqZip - Development and Applications</h2>
<a 
 id="x1-29001r45"></a>
<h3 class="sectionHead"><span class="titlemark">2.1 </span> <a 
 id="x1-300001"></a>SeqZip Overview</h3>
<!--l. 6--><p class="noindent" >Development of the SeqZip methodology began with an attempt to circumvent an
obvious shortcoming in second generation HTS&#8212;the short nature of the reads. Until
second generation HTS (i.e. reads &#x003C;100nt on either the Illumina or SOLiD platforms),
most sequencing was done using cloned fragments, stored in a bacteria,
and analyzed using dideoxy Sanger Sequencing ( see <a 
href="#x1-70001">1.2.1<!--tex4ht:ref: sec:DNA Sequencing History --></a>. Indeed, this is
how most <span 
class="phvb8t-x-x-120">??</span> <a 
href="\protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 70 undefined on input line 6.
}\endgroup \let \endcsname \relax \protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 70 undefined on input line 6.
}\endgroup \let ::)Q\endcsname \relax #x1-3001r\protect \protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 70 undefined on input line 6.
}\endgroup \let ">EST</a>s where analyzed. An extremely powerful feature of these
ESTs is that as they represented the sequence of a single clone, from a
single original molecule of RNA, the connectivity between sequences that
were far apart (&#x003E;1,000 nt) in the original sequence was maintained. It is
this very feature, the continuity of sequence, that allowed whole genome
shotgun sequenced to be used, and ESTs to be assembled, into complete
genomes, despite sometimes lengthly, and highly repetitive, sretches of DNA (see
<a 
href="#x1-70001">1.2.1<!--tex4ht:ref: sec:DNA Sequencing History --></a>). Once research transitioned over to heavy use of the second generation
HTS, all of that connectivity was lost, and all the inherent information with
it.
<!--l. 8--><p class="noindent" >Second generation HTS can be supplemented with other technologies. This has been
demonstrated perhaps most successfully with long-read assisted genome assembly
[<a 
href="#XKoren2012a">Koren et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKoren2012a">2012</a>]. Why not supplemental the disconnected nature of short reads
with another technilogy? To that end, Phillip Zamore proposed an RNA-templated
DNA to DNA ligation approach as drawn in Figure <a 
href="#x1-30001r1">2.1<!--tex4ht:ref: fig:Original SeqZip Diagram --></a> (see also US Patent application
<a 
href="http://1.usa.gov/PTG9BB" >12/906,678</a>). Using this approach, 2 or more distant sequences of RNA are
investigated using short DNA oligonucleotides that hybridize to target sequences,
forcing the intervening sequences to loop out. Incorporation of the hybridized DNA via
ligation with those of DNAs adjacently hybridized generates a positive read out of
sequence presence.
<!--l. 10--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-30001r1"></a>
                                                                                

                                                                                
<!--l. 12--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.514x.png" alt="PIC" class="graphics" width="396.48125pt" height="197.73874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.514x.png" src="Figures/Chapter2/OriginalSeqZipDiagram.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.1: </span><span  
class="content">Original SeqZip Diagram<br 
class="newline" />This is the original concept diagram of the SeqZip methodology. (A) Specific
DNA oligos target an mRNA and loop out the RNA sequence. Ligases is added
to join the DNA oligos together; (B) &amp; (C) Two different possibilities of ligation
products templated from the RNA in (A), where Exon 2 is an cassette exon. </span></div><!--tex4ht:label?: x1-30001r1 -->
                                                                                

                                                                                
<!--l. 19--><p class="noindent" ></div><hr class="endfigure">
<!--l. 21--><p class="noindent" >Along with Patent <a 
href="http://1.usa.gov/PTG9BB" >12/906,678</a>, Chapter <a 
href="#x1-370003">3<!--tex4ht:ref: Chapter3 --></a> presents much of the early and important
developmental work demonstrating reduction to practice of this method (termed
&#8220;SeqZip"), and its application to investgating connectivity of sequence content in
biologically-interesting genes <span 
class="phvro8t-x-x-120">Fn1 </span>and <span 
class="phvro8t-x-x-120">Dscam1</span>.
<!--l. 23--><p class="noindent" >Presented in this Chapter are experiences demonstrating SeqZip application to the
following questions and issues:
<ul class="itemize1">
<li class="itemize">Section  <a 
href="#x1-310002">2.2<!--tex4ht:ref: sec: Multiplex Gene Study --></a>:  Investgation  of  10  genes,  simultaneously  ("Multiplex")  for
connected AS decisions
</li>
<li class="itemize">Section <a 
href="#x1-320003">2.3<!--tex4ht:ref: sec: SeqZip Integrity --></a>: Using SeqZip to investigate the interigy of RNA molecules
</li>
<li class="itemize">Section   <a 
href="#x1-360004">2.4<!--tex4ht:ref: sec: precursor TX --></a>:   Using   SeqZip   to   demonstrate   the   presence   of   long,
continuous piRNA precusors</li></ul>
<!--l. 31--><p class="noindent" >The three sections contrast with Chapter <a 
href="#x1-370003">3<!--tex4ht:ref: Chapter3 --></a> in some important ways. First, Section <a 
href="#x1-310002">2.2<!--tex4ht:ref: sec: Multiplex Gene Study --></a>
demonstrates that the SeqZip method can not only be used to investigate one,
extremely complex Alternatively splice gene, such as <span 
class="phvro8t-x-x-120">Dscam1 </span>in a c comprehensive
manner, but can also be applied to looking at multiple genes at once. Seciton <a 
href="#x1-320003">2.3<!--tex4ht:ref: sec: SeqZip Integrity --></a>
expoits an important subtle feature of the method&#8212;that the RNA must be
intact in order to produce a ligation product. This can be used to report on a
fraction of RNA that is intact, and deduce meaningful information such as the
amount of RNA virus that is intact (see <a 
href="#x1-340002">2.3.2<!--tex4ht:ref: subsec: HIV --></a>), or the existance of as-yet
unobserved mega transcripts, like mammalian piRNA precursors (see <a 
href="#x1-360004">2.4<!--tex4ht:ref: sec: precursor TX --></a> and [<a 
href="#XLi2013h">Li
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2013h">2013a</a>,<a 
href="#XLi2013">c</a>]).
<a 
 id="x1-30002r56"></a>
<h3 class="sectionHead"><span class="titlemark">2.2 </span> <a 
 id="x1-310002"></a>Multiplex Gene Study</h3>
<!--l. 35--><p class="noindent" >Is the coordination discussed in section <a 
href="#x1-160004">1.3.4<!--tex4ht:ref: sec: Coordination in splicing --></a> a general phenomenon? One of
the major goals of developing the SeqZip methodology was investigating
potential coordination genome-wide. By genome-wide, what we really mean is
to analyze many (or all) of the RNA transcripts in a tissue for evidence of
coordinated splicing decisions. When development of the method reached
the point that it could be applied in a multiplex study, I did not posses the
bioinformatic skills necessary to 1) design ligamers in an automated and
high-throughput fashion and 2) identify target transcripts, exons, and sequences to
                                                                                

                                                                                
investigate for potential connectivity. Both of these points are discussed later(see
<a 
href="#x1-105000C">C<!--tex4ht:ref: AppendixC --></a><span 
class="phvb8t-x-x-120">??</span> .
<!--l. 37--><p class="noindent" >In order to make some progress on applying the technique to multiple genes at
once, I used data presented by <a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>&#x00A0;[<a 
href="#XFagnani2007">2007</a>]. This paper identified
genes displaying tissue-specific splicing patterns, focusing on those with
CNS-specific patterns. Once section focused on &#8220;Coordination between AS events
belonging to the same genes,&#8221; and seemed to be the exact type of data we were
interested in applying the SeqZip method too. Five hundred of the 3,044
genes investigated by their microarrays contained 2&#8211;5 alternative exons.
<a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>&#x00A0;[<a 
href="#XFagnani2007">2007</a>] contained an additional data file listing all pair-wise
combinations of alternative exons in the same gene (with that gene having significant
expression in &#x003E;20 different tissues), along with the standard and partial spearman
correlations.
<!--l. 39--><p class="noindent" >It is important to note that the genes above also contain alternative first exons, a
prominant type of AS (see figure <a 
href="#x1-17001r7">1.7<!--tex4ht:ref: fig:asEventsBarChart --></a>). Indeed, from microarrays studies, it has been
estimated that approximately 16%&#8211;23% of all AS events involve alternative first and
last exons [<a 
href="#XBingham2008">Bingham et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBingham2008">2008</a>]. It is known that, through alternative use of first and
last exons, cells can fine-tune a transcript&#8217;s untranslated region (UTR) and control
many aspects of mRNA regulation including nuclear export, localization, expression,
and stability [<a 
href="#XHughes2006">Hughes</a>,&#x00A0;<a 
href="#XHughes2006">2006</a>]. In support of the importance of alternative UTRs in
tuning of gene expression, a landmark RNA-Seq study demonstrated a high
occurrence of alternative first and last exon splicing, with alternative tandem 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; UTR
usage being the most highly tissue-dependent form of AS observed [<a 
href="#XWang2008">Wang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>]. The current model of spatial proximity between 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; UTRs is
suggestive of their possible interdependence. In our multiplez analysis, we
included genes potentially displaying interdependence between first and last
exons. Discovery of interdependence would lead to many questions into how
specific combinations of UTRs can influence mRNA processing downstream of
AS.
<!--l. 41--><p class="noindent" >Using the <a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>&#x00A0;[<a 
href="#XFagnani2007">2007</a>] data, I filtered exon pairs to those with a distance
&#x003E;350 nt in the final pre-mRNA. I also visualized their transcript achetecture, and EST
evidence using NCBIs AceView tool [<a 
href="#XThierry-Mieg2006">Thierry-Mieg and Thierry-Mieg</a>,&#x00A0;<a 
href="#XThierry-Mieg2006">2006</a>]. For
example, the exons with strong correlation of expression in <span 
class="phvro8t-x-x-120">Chl1 </span>are in the beginning
(second exon) and end (fourth from last exon, accession BC060216) with plenty of
supporting evidence for these exons being expressed and skipped. After combing
through [<a 
href="#XFagnani2007">Fagnani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFagnani2007">2007</a>] data for a group of 11 genes. The genes examined are
presented in Table <a 
href="#x1-31001r1">2.1<!--tex4ht:ref: table: BigSpanGenes --></a>.
<div class="table">
                                                                                

                                                                                
<!--l. 1--><p class="noindent" ><a 
 id="x1-31001r1"></a><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-4" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-4-1g"><col 
id="TBL-4-1"></colgroup><colgroup id="TBL-4-2g"><col 
id="TBL-4-2"></colgroup><colgroup id="TBL-4-3g"><col 
id="TBL-4-3"></colgroup><colgroup id="TBL-4-4g"><col 
id="TBL-4-4"></colgroup><colgroup id="TBL-4-5g"><col 
id="TBL-4-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-1-1"  
class="td11"><span 
class="phvb8t-x-x-109">Gene name</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-2"  
class="td11"><span 
class="phvb8t-x-x-109">nt mRNA between</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-3"  
class="td11"><span 
class="phvb8t-x-x-109">possible isoforms</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-4"  
class="td11"><span 
class="phvb8t-x-x-109">Exon 1</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-1-5"  
class="td11"><span 
class="phvb8t-x-x-109">Exon 2</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-2-1"  
class="td11"><span 
class="phvr8t-x-x-109">Chl1             </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-2"  
class="td11">            <span 
class="phvr8t-x-x-109">4665</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-3"  
class="td11">              <span 
class="phvr8t-x-x-109">18</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-4"  
class="td11">     <span 
class="phvr8t-x-x-109">2</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-2-5"  
class="td11">    <span 
class="phvr8t-x-x-109">24</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-3-1"  
class="td
11"><span 
class="phvr8t-x-x-109">Mdm1          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-2"  
class="td11">            <span 
class="phvr8t-x-x-109">1846</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-4"  
class="td11">  <span 
class="phvr8t-x-x-109">EDA</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-3-5"  
class="td11">  <span 
class="phvr8t-x-x-109">IIICS</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-4-1"  
class="td
11"><span 
class="phvr8t-x-x-109">PTPRF-Y     </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-2"  
class="td11">            <span 
class="phvr8t-x-x-109">1633</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-4"  
class="td11">     <span 
class="phvr8t-x-x-109">2</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-4-5"  
class="td11">    <span 
class="phvr8t-x-x-109">13</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-5-1"  
class="td
11"><span 
class="phvr8t-x-x-109">Cacna1c      </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-2"  
class="td11">            <span 
class="phvr8t-x-x-109">1403</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-4"  
class="td11">    <span 
class="phvr8t-x-x-109">15</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-5-5"  
class="td11">  <span 
class="phvr8t-x-x-109">21/22</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-6-1"  
class="td
11"><span 
class="phvr8t-x-x-109">PTPRF-X     </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-2"  
class="td11">             <span 
class="phvr8t-x-x-109">936</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-4"  
class="td11">   <span 
class="phvr8t-x-x-109">9/10</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-6-5"  
class="td11">    <span 
class="phvr8t-x-x-109">21</span></td></tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-7-1"  
class="td
11"><span 
class="phvr8t-x-x-109">FN1              </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-2"  
class="td11">             <span 
class="phvr8t-x-x-109">813</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-3"  
class="td11">               <span 
class="phvr8t-x-x-109">8</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-4"  
class="td11">  <span 
class="phvr8t-x-x-109">13/14</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-7-5"  
class="td11">  <span 
class="phvr8t-x-x-109">21/22</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-8-1"  
class="td11"><span 
class="phvr8t-x-x-109">Apbb1          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-2"  
class="td11">             <span 
class="phvr8t-x-x-109">802</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-3"  
class="td11">             <span 
class="phvr8t-x-x-109">260</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-4"  
class="td11">   <span 
class="phvr8t-x-x-109">1/2b</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-8-5"  
class="td11">   <span 
class="phvr8t-x-x-109">2/3e</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-9-1"  
class="td11"><span 
class="phvr8t-x-x-109">Agrn             </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-2"  
class="td11">             <span 
class="phvr8t-x-x-109">736</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-3"  
class="td11">               <span 
class="phvr8t-x-x-109">8</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-4"  
class="td11"> <span 
class="phvr8t-x-x-109">33/34c</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-9-5"  
class="td11"> <span 
class="phvr8t-x-x-109">33/34a</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-10-1"  
class="td11"><span 
class="phvr8t-x-x-109">Exoc7          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-2"  
class="td11">             <span 
class="phvr8t-x-x-109">513</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-4"  
class="td11">     <span 
class="phvr8t-x-x-109">7</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-10-5"  
class="td11">    <span 
class="phvr8t-x-x-109">13</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-11-1"  
class="td11"><span 
class="phvr8t-x-x-109">Prom1          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-2"  
class="td11">             <span 
class="phvr8t-x-x-109">512</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-3"  
class="td11">               <span 
class="phvr8t-x-x-109">4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-4"  
class="td11">     <span 
class="phvr8t-x-x-109">7</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-11-5"  
class="td11">     <span 
class="phvr8t-x-x-109">9</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-12-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-12-1"  
class="td11"><span 
class="phvr8t-x-x-109">Lphn2          </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-2"  
class="td11">             <span 
class="phvr8t-x-x-109">396</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-3"  
class="td11">              <span 
class="phvr8t-x-x-109">32</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-4"  
class="td11">    <span 
class="phvr8t-x-x-109">19</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-4-12-5"  
class="td11"> <span 
class="phvr8t-x-x-109">24/25a</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-4-13-"><td  style="white-space:nowrap; text-align:left;" id="TBL-4-13-1"  
class="td11">           </td></tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;2.1: </span><span  
class="content"><span class="texthl"><span id="textcolor2">caption</span></span></span></div><!--tex4ht:label?: x1-31001r1 -->
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
<!--l. 45--><p class="noindent" >I hand-designed ligamers to observe potentially coordinated splicing decisions. These
oligos were then ordered from IDT in a 96-well plate format, pooled according to
gene, and used to develop a multiplex approach to applying SeqZip, as well as
investigate coordination between these exons, in these genes, using mouse total
RNA from brains.
<!--l. 47--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-31002r2"></a>
                                                                                

                                                                                
<!--l. 49--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.515x.png" alt="PIC" class="graphics" width="344.28624pt" height="370.38374pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.515x.png" src="Figures/Chapter2/10GeneSetSchematic.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.2: </span><span  
class="content">10 Gene set schematic<br 
class="newline" /><span class="texthl"><span id="textcolor3">caption next</span></span> </span></div><!--tex4ht:label?: x1-31002r2 -->
                                                                                

                                                                                
<!--l. 56--><p class="noindent" ></div><hr class="endfigure">
<!--l. 58--><p class="noindent" >After attempts at performing the SeqZip assay on all 10 genes in one ligation failed, I
reverted back to per-gene ligation reactions in order to trouble shoot and optimize the
assay. Once I had obtained ligaiton products from per-gene ligation reactions for both
the individual and combination ligamers pools, I pooled all the ligation products and
amplified them via PCR. Amplified products were sent for PE100 sequencing on the
Illumina GEIIx platform.
<!--l. 60--><p class="noindent" >After considerable delay and optimization from the Umass Sequencing Core (likely
due to little sequence diversity in the library), the analyzed data demonstrated little
alternative splicing in the genes examined. Put another way- most of the transcripts
observed via SeqZip were uniform in exon inclusion, and showed little variability for
cassette exon inclusion. These results were disappointing, and forced us to rethink a
better way to apply the SeqZip methodology to multiple genes, or complex alternative
splicing (i.e. <span 
class="phvro8t-x-x-120">Dscam1</span>; see <a 
href="#x1-370003">3<!--tex4ht:ref: Chapter3 --></a>. For a discussion of an &#8220;ideal&#8221; multiplex study, see
section <a 
href="#x1-820002">5.2.2<!--tex4ht:ref: subsec: Ideal multiplex study --></a>.
<a 
 id="x1-31003r58"></a>
<h3 class="sectionHead"><span class="titlemark">2.3 </span> <a 
 id="x1-320003"></a>Determining RNA integrity using SeqZip</h3>
<a 
 id="x1-32001r54"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.1 </span> <a 
 id="x1-330001"></a>Demonstration of Concept</h4>
<!--l. 66--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-33001r3"></a>
                                                                                

                                                                                
<!--l. 68--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.516x.png" alt="PIC" class="graphics" width="288.07625pt" height="251.94124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.516x.png" src="Figures/Chapter2/DegreadedRNABySeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.3: </span><span  
class="content">Ligation product tied to RNA integrity<br 
class="newline" /><span class="texthl"><span id="textcolor4">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-33001r3 -->
                                                                                

                                                                                
<!--l. 75--><p class="noindent" ></div><hr class="endfigure">
<!--l. 77--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-33002r4"></a>
                                                                                

                                                                                
<!--l. 79--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.517x.png" alt="PIC" class="graphics" width="363.3575pt" height="258.9675pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.517x.png" src="Figures/Chapter2/TransRNAWithSeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.4: </span><span  
class="content">Trans Transcript investigation<br 
class="newline" /><span class="texthl"><span id="textcolor5">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-33002r4 -->
                                                                                

                                                                                
<!--l. 86--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-33003r62"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.2 </span> <a 
 id="x1-340002"></a>Investigating HIV viral genome integrity using SeqZip</h4>
<!--l. 93--><p class="noindent" >MOV10 is implicated in HIV genome stability and intactness. We sought to measure
the effects of a point mutant in the ATPase domain of MOV10L1 on the &#8216;intactness&#8217; of
the HIV genome. Viral particles contain two copies of the ssRNA HIV genome.
Proteomics studies have measured MOV10L1 and Upf1 in viral particles, implicated
these proteins, which are known helicases, in the maintained and infectivity of
HIV.
<!--l. 95--><p class="noindent" >1
<a 
 id="x1-34001r65"></a>
<h4 class="subsectionHead"><span class="titlemark">2.3.3 </span> <a 
 id="x1-350003"></a>Design of HIV ligamers</h4>
<!--l. 97--><p class="noindent" >Research into the integrity of the HIV RNA genome using SeqZip began with
designing a set of ligamers against two different clones. The first clone, targeting
transcripts from the M19921 plasmid (so called &#8220;M&#8221; clone), and transcripts from the
K03455 clone contain nearly identical sequences with respect to the genome itself,
and differ mostly in plasmid originating sequences. We targeted a difference in
sequence for one site of ligation (Fig3-11A). Three different pools of ligamers were
created initially: a Five(5) ligamer pool, with three ligamers design to test for the
presence of sequence in the first 1,140 nt of the HIV genome, importantly the first site
of ligation in the 5 region pool should contain a mismatch in the K clone
sequence; a three(3) pool, testing the last 1,210 nt of the genome, and a Long (L)
ligamer pool, also containing three ligamers, but the middle ligamer of which
would span the 5 and 3 regions, looping out 8,633 nt of sequence in the
middle of the HIV genome. In vitro transcripts were created using both the K
and M clone plasmids. These transcripts were added to a background of
total MEF RNA, and the SeqZIp assay was performed. Ligation products
were successfully amplified from all ligamer pools when using the M clone
transcript and all three ligamer pools. Also the abundance of these ligation
products, as measured by endpoint PCR, seemed to be spike-concentration
dependent. Notably, Ligation products were not obtained from the K clone
using either the 5 or L ligamer pools, likely due to the mismatch between
the transcript and the ligamers at the site of ligation. Also of note was the
appearance of ligation products from purified endogenous virons of the M
clone from all three ligamer pools, and the absence of products from virions
purified from plasmids containing a defective protein, Gag, essential for viral
packaging.
                                                                                

                                                                                
<!--l. 99--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-35001r5"></a>
                                                                                

                                                                                
<!--l. 101--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.518x.png" alt="PIC" class="graphics" width="392.46625pt" height="346.29375pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.518x.png" src="Figures/Chapter2/HIVviaSeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.5: </span><span  
class="content">SeqZip can examine HIV transcript integrity<br 
class="newline" /><span class="texthl"><span id="textcolor6">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-35001r5 -->
                                                                                

                                                                                
<!--l. 108--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-35002r61"></a>
<h3 class="sectionHead"><span class="titlemark">2.4 </span> <a 
 id="x1-360004"></a>Continuity of piRNA precursor transcripts</h3>
<!--l. 112--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36001r6"></a>
                                                                                

                                                                                
<!--l. 114--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.519x.png" alt="PIC" class="graphics" width="378.41374pt" height="294.09874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.519x.png" src="Figures/Chapter2/PrecursorLocations.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.6: </span><span  
class="content">piRNA precursor locations<br 
class="newline" /><span class="texthl"><span id="textcolor7">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36001r6 -->
                                                                                

                                                                                
<!--l. 121--><p class="noindent" ></div><hr class="endfigure">
<div class="table">
                                                                                

                                                                                
<!--l. 1--><p class="noindent" ><a 
 id="x1-36002r2"></a><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-5" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-5-1g"><col 
id="TBL-5-1"><col 
id="TBL-5-2"><col 
id="TBL-5-3"><col 
id="TBL-5-4"><col 
id="TBL-5-5"></colgroup><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-1"  
class="td11"><span 
class="phvb8t-x-x-109">Cluster Name     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-1-2"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-6" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-6-1g"><col 
id="TBL-6-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-6-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-6-1-1"  
class="td00"><span 
class="phvb8t-x-x-109">Matched</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-6-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-6-2-1"  
class="td00"> <span 
class="phvb8t-x-x-109">Cluster</span></td></tr></table>                                                                                                                      </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-3"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-7" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-7-1g"><col 
id="TBL-7-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-7-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-1-1"  
class="td00"><span 
class="phvb8t-x-x-109">Unique-mapping</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-7-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-2-1"  
class="td00">   <span 
class="phvb8t-x-x-109">piRNAs @      </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-7-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-7-3-1"  
class="td00">    <span 
class="phvb8t-x-x-109">wt.14dpp       </span></td></tr></table>                                                                                                        </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-4"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-8" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-8-1g"><col 
id="TBL-8-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-8-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-1-1"  
class="td00"><span 
class="phvb8t-x-x-109">Fraction of</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-8-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-2-1"  
class="td00"> <span 
class="phvb8t-x-x-109">pachytene</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-8-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-8-3-1"  
class="td00">  <span 
class="phvb8t-x-x-109">piRNAs   </span></td></tr></table>                                                                                                                  </div></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-1-5"  
class="td11"><!--tex4ht:inline--><div class="tabular"> <table id="TBL-9" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-9-1g"><col 
id="TBL-9-1"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-9-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-1-1"  
class="td00"><span 
class="phvb8t-x-x-109">Cumulative</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-9-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-2-1"  
class="td00"> <span 
class="phvb8t-x-x-109">pachytene</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-9-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-9-3-1"  
class="td00">  <span 
class="phvb8t-x-x-109">piRNAs    </span></td></tr></table>                                                                                                                 </div></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-1"  
class="td11"><span 
class="phvr8t-x-x-109">17-qA3.3-26735.1</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-2-2"  
class="td11"><span 
class="phvr8t-x-x-109">17-qA3.3-27363                                                                                                               </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-3"  
class="td11">                                                                 <span 
class="phvr8t-x-x-109">3,021,022</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-4"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">17.2</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-2-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">17.2</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-1"  
class="td11"><span 
class="phvr8t-x-x-109">17-qA3.3-27363.1</span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-3-2"  
class="td11"><span 
class="phvr8t-x-x-109">17-qA3.3-26735                                                                                                               </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-3"  
class="td11">                                                                 <span 
class="phvr8t-x-x-109">1,742,695</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">9.9</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-3-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">27.2</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-1"  
class="td11"><span 
class="phvr8t-x-x-109">9-qC-31469.1       </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-4-2"  
class="td11"><span 
class="phvr8t-x-x-109">9-qC-10667                                                                                                                      </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-3"  
class="td11">                                                                 <span 
class="phvr8t-x-x-109">1,006,333</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">5.7</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-4-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">32.9</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-1"  
class="td11"><span 
class="phvr8t-x-x-109">9-qC-10667.1       </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-5-2"  
class="td11"><span 
class="phvr8t-x-x-109">9-qC-31469                                                                                                                      </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-3"  
class="td11">                                                                   <span 
class="phvr8t-x-x-109">272,385</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">1.6</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-5-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">34.5</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-1"  
class="td11"><span 
class="phvr8t-x-x-109">7-qD2-24830.1     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-6-2"  
class="td11"><span 
class="phvr8t-x-x-109">7-qD2-11976                                                                                                                    </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-3"  
class="td11">                                                                   <span 
class="phvr8t-x-x-109">652,564</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">3.7</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-6-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">38.2</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-1"  
class="td11"><span 
class="phvr8t-x-x-109">7-qD2-11976.1     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-7-2"  
class="td11"><span 
class="phvr8t-x-x-109">7-qD2-24830                                                                                                                    </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-3"  
class="td11">                                                                   <span 
class="phvr8t-x-x-109">280,312</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">1.6</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-7-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">39.8</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-1"  
class="td11"><span 
class="phvr8t-x-x-109">6-qF3-28913.1     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-8-2"  
class="td11"><span 
class="phvr8t-x-x-109">6-qF3-8009                                                                                                                       </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-3"  
class="td11">                                                                   <span 
class="phvr8t-x-x-109">564,930</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">3.2</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-8-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">43.0</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-1"  
class="td11"><span 
class="phvr8t-x-x-109">6-qF3-8009.1       </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-9-2"  
class="td11"><span 
class="phvr8t-x-x-109">6-qF3-28913                                                                                                                     </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-3"  
class="td11">                                                                   <span 
class="phvr8t-x-x-109">180,210</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">1.0</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-9-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">44.0</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-5-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-1"  
class="td11"><span 
class="phvr8t-x-x-109">2-qE1-35981.1     </span></td><td  style="white-space:nowrap; text-align:left;" id="TBL-5-10-2"  
class="td11"><span 
class="phvr8t-x-x-109">NA                                                                                                                                    </span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-3"  
class="td11">                                                                  <span 
class="phvr8t-x-x-109">1121042</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-4"  
class="td11">                                                                       <span 
class="phvr8t-x-x-109">6.4</span></td><td  style="white-space:nowrap; text-align:right;" id="TBL-5-10-5"  
class="td11">                                                                      <span 
class="phvr8t-x-x-109">50.4</span></td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td><td><hr></td><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-5-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-5-11-1"  
class="td11">               </td></tr></table>
</div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;2.2: </span><span  
class="content"><span class="texthl"><span id="textcolor8">caption</span></span></span></div><!--tex4ht:label?: x1-36002r2 -->
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
<!--l. 125--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36003r7"></a>
                                                                                

                                                                                
<!--l. 127--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.520x.png" alt="PIC" class="graphics" width="341.275pt" height="346.29375pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.520x.png" src="Figures/Chapter2/RTDoesntWork.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.7: </span><span  
class="content">RT Doesn&#8217;t Work for piRNA precursors<br 
class="newline" /><span class="texthl"><span id="textcolor9">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36003r7 -->
                                                                                

                                                                                
<!--l. 134--><p class="noindent" ></div><hr class="endfigure">
<!--l. 136--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36004r8"></a>
                                                                                

                                                                                
<!--l. 138--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.521x.png" alt="PIC" class="graphics" width="291.0875pt" height="160.59999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.521x.png" src="Figures/Chapter2/dst1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.8: </span><span  
class="content">Dst1 by SeqZip<br 
class="newline" /><span class="texthl"><span id="textcolor10">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36004r8 -->
                                                                                

                                                                                
<!--l. 145--><p class="noindent" ></div><hr class="endfigure">
<!--l. 147--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36005r9"></a>
                                                                                

                                                                                
<!--l. 149--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.522x.png" alt="PIC" class="graphics" width="372.39124pt" height="371.3875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.522x.png" src="Figures/Chapter2/fn1ThreeSite.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.9: </span><span  
class="content">Capturing AS at three sites and &#x003E;2kb of mRNA<br 
class="newline" /><span class="texthl"><span id="textcolor11">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36005r9 -->
                                                                                

                                                                                
<!--l. 156--><p class="noindent" ></div><hr class="endfigure">
<!--l. 158--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36006r10"></a>
                                                                                

                                                                                
<!--l. 160--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.523x.png" alt="PIC" class="graphics" width="336.25624pt" height="208.78pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.523x.png" src="Figures/Chapter2/testesSpecificRnaseqPrecursors.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.10: </span><span  
class="content">Testes Specific RNA precursor expression<br 
class="newline" /><span class="texthl"><span id="textcolor12">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36006r10 -->
                                                                                

                                                                                
<!--l. 167--><p class="noindent" ></div><hr class="endfigure">
<!--l. 169--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-36007r11"></a>
                                                                                

                                                                                
<!--l. 171--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.524x.png" alt="PIC" class="graphics" width="352.31624pt" height="288.07625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.524x.png" src="Figures/Chapter2/piRNAPrecurserAnalyisBySeqZip.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;2.11: </span><span  
class="content">piRNA precursor analysis via SeqZip<br 
class="newline" /><span class="texthl"><span id="textcolor13">figure Caption</span></span> </span></div><!--tex4ht:label?: x1-36007r11 -->
                                                                                

                                                                                
<!--l. 178--><p class="noindent" ></div><hr class="endfigure">
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;3</span><br /><a 
 id="x1-370003"></a>SeqZip Publication</h2>
<a 
 id="x1-37001r68"></a>
<h3 class="sectionHead"><span class="titlemark">3.1 </span> <a 
 id="x1-380001"></a>Abstract</h3>
<a 
 id="x1-38001r77"></a>
<h3 class="sectionHead"><span class="titlemark">3.2 </span> <a 
 id="x1-390002"></a>Introduction</h3>
<!--l. 13--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-39001r1"></a>
                                                                                

                                                                                
<!--l. 15--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.525x.png" alt="PIC" class="graphics" width="409.53pt" height="143.53624pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.525x.png" src="Figures/Chapter3/Roy2014Fig1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.1: </span><span  
class="content">SeqZip Diagram]<br 
class="newline" /><span class="texthl"><span id="textcolor14">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-39001r1 -->
                                                                                

                                                                                
<!--l. 22--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-39002r78"></a>
<h3 class="sectionHead"><span class="titlemark">3.3 </span> <a 
 id="x1-400003"></a>Results</h3>
<a 
 id="x1-40001r66"></a>
<h4 class="subsectionHead"><span class="titlemark">3.3.1 </span> <a 
 id="x1-410001"></a>Subsection 2</h4>
<!--l. 35--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-41001r2"></a>
                                                                                

                                                                                
<!--l. 37--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.526x.png" alt="PIC" class="graphics" width="387.4475pt" height="358.33875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.526x.png" src="Figures/Chapter3/Roy2014Fig2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.2: </span><span  
class="content">Rnl2 Panel]<br 
class="newline" /><span class="texthl"><span id="textcolor15">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-41001r2 -->
                                                                                

                                                                                
<!--l. 44--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-41002r81"></a>
<h4 class="subsectionHead"><span class="titlemark">3.3.2 </span> <a 
 id="x1-420002"></a>Subsection 2</h4>
<!--l. 53--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-42001r3"></a>
                                                                                

                                                                                
<!--l. 55--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.527x.png" alt="PIC" class="graphics" width="390.45874pt" height="478.78874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.527x.png" src="Figures/Chapter3/Roy2014Fig3.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;3.3: </span><span  
class="content">Rnl2 Panel]<br 
class="newline" /><span class="texthl"><span id="textcolor16">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-42001r3 -->
                                                                                

                                                                                
<!--l. 62--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-42002r80"></a>
<h3 class="sectionHead"><span class="titlemark">3.4 </span> <a 
 id="x1-430004"></a>Discussion</h3>
<a 
 id="x1-43001r85"></a>
<h3 class="sectionHead"><span class="titlemark">3.5 </span> <a 
 id="x1-440005"></a>Methods</h3>
<a 
 id="x1-44001r86"></a>
<h3 class="sectionHead"><span class="titlemark">3.6 </span> <a 
 id="x1-450006"></a>Supplemental Text</h3>
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;4</span><br /><a 
 id="x1-460004"></a>An Ancient Transcription Factor Initiates the Burst of piRNA Production
during Early Meiosis in Mouse Testes</h2>
<a 
 id="x1-46001r87"></a>
<h3 class="sectionHead"><span class="titlemark">4.1 </span> <a 
 id="x1-470001"></a>INTRODUCTION</h3>
<!--l. 9--><p class="noindent" >P-element induced wimpy testis (PIWI)-interacting RNAs (piRNAs) can be
distinguished from other animal small silencing RNAs by their longer length (typically
23&#8211;35 nt), 2<span 
class="cmsy-8">&#x2032;</span> -O-methyl-modified 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; termini, and association with PIWI proteins, a
distinct subgroup of Argonaute proteins, the small RNA-guided proteins responsible
for RNA interference and related pathways [<a 
href="#XCenik2011">Cenik and Zamore</a>,&#x00A0;<a 
href="#XCenik2011">2011</a>,&#x00A0;<a 
href="#XFarazi2008">Farazi
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFarazi2008">2008</a>,&#x00A0;<a 
href="#XKim2009">Kim et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKim2009">2009</a>,&#x00A0;<a 
href="#XKumar1998">Kumar and Carmichael</a>,&#x00A0;<a 
href="#XKumar1998">1998</a>,&#x00A0;<a 
href="#XThomson2009">Thomson and
Lin</a>,&#x00A0;<a 
href="#XThomson2009">2009</a>,&#x00A0;<span 
class="phvb8t-x-x-120">?</span>]. piRNA production does not require Dicer, the double-stranded RNA
endonuclease that makes microRNAs (miRNAs) and small interfering RNAs
(siRNAs), and piRNAs are thought to derive from single-stranded rather than
double-stranded RNA [<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>,&#x00A0;<a 
href="#XVagin2006">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2006">2006</a>].
<!--l. 11--><p class="noindent" >In most bilateral animals, germline piRNAs protect the genome from transposon
activation, but also have other functions [<a 
href="#XAravin2008a">Aravin and Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XAravin2007a">Aravin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XAravin2001">2001</a>,&#x00A0;<a 
href="#XAshe2012">Ashe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAshe2012">2012</a>,&#x00A0;<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XCarmell2007">Carmell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>,&#x00A0;<a 
href="#XHartig2007">Hartig et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHartig2007">2007</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2008">2008</a>,&#x00A0;<a 
href="#XLee2012">Lee
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLee2012">2012</a>,&#x00A0;<a 
href="#XShirayama2012">Shirayama et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShirayama2012">2012</a>,&#x00A0;<a 
href="#XVagin2004">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2004">2004</a>]. A few days after
birth, the majority of piRNAs in the mouse testis are pre-pachytene piRNAs;
25% of these piRNA species map to more than one location in the genome.
A second class of piRNAs, typically derived from intergenic regions, has
been reported to emerge in the mouse testis 14.5 days postpartum (dpp),
when the developing spermatocytes synchronously enter the pachytene
phase of meiotic prophase I. These pachytene piRNAs compose &#x003E;95% of
piRNAs in the adult mouse testis. Loss of genes required to make pachytene
piRNAs blocks production of mature sperm [<a 
href="#XAravin2001">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2001">2001</a>,&#x00A0;<a 
href="#XDeng2002c">Deng and
Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>,&#x00A0;<a 
href="#XReuter2011">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2011">2011</a>,&#x00A0;<a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVourekas2012">2012</a>]. What triggers the
accumulation of pachytene piRNAs when spermatocytes enter the pachynema is
unknown.
<!--l. 13--><p class="noindent" >In Caenorhabditis elegans, each piRNA is processed from its own short RNA
polymerase II (Pol II) transcript [<a 
href="#XGu2012">Gu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGu2012">2012</a>]. In contrast, insect and mouse
piRNAs are thought to be processed from long RNAs transcribed from large
piRNA loci. Supporting this view, a transposon inserted into the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of the
flamenco piRNA cluster in flies reduces the production of flamenco piRNAs 168
kbp 3<span 
class="cmsy-8">&#x2032;</span> &#x00A0; to the insertion, suggesting that it disrupts transcription of the entire
locus [<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>]. High-throughput sequencing and chromatin
immunoprecipitation (ChIP) has been used to define the genomic structure of the
piRNA-producing genes of immortalized, cultured silk moth BmN4 cells [<a 
href="#XKawaoka2012">Kawaoka
                                                                                

                                                                                
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKawaoka2012">2012</a>]. However, for flies and mice, we do not know the structure of
piRNA-producing genes, their transcripts, or the nature of the promoters that control
their expression.
<!--l. 15--><p class="noindent" >Instead, piRNA loci have been defined as clusters: regions of the genome with a
high density of mapping piRNA sequences [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XGrivna2006">Grivna et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrivna2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>,&#x00A0;<a 
href="#XRo2007">Ro
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRo2007">2007</a>]. In reality, piRNA-producing loci correspond to discrete transcription
units that include both intergenic loci believed to encode no protein [<a 
href="#XBrennecke2007">Brennecke
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XBrennecke2008">Brennecke and Malone</a>,&#x00A0;<a 
href="#XBrennecke2008">2008</a>,&#x00A0;<a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVourekas2012">2012</a>] and
protein-coding genes that also produce piRNAs [<a 
href="#XAravin2007">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007">2007b</a>,&#x00A0;<a 
href="#XRobine2009">Robine
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>,&#x00A0;<a 
href="#XSaito2009">Saito et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSaito2009">2009</a>].
<!--l. 17--><p class="noindent" >We used high-throughput sequencing data to define the genes and transcripts that
produce piRNAs in the juvenile and adult mouse testis. Using these data, we
identified the factor that initiates transcription of pachytene piRNA genes: A-MYB
(MYBL1), a spermatocyte protein that serves as a master regulator of genes
encoding proteins required for cell-cycle progression through the pachytene stage of
meiosis [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XTrauth1994">Trauth et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrauth1994">1994</a>]. A-MYB also initiates
transcription of the genes encoding many piRNA biogenesis factors. The
combined action of A-MYB at the promoters of genes producing pachytene
piRNA precursor transcripts and genes encoding piRNA biogenesis proteins
creates a coherent feedforward loop that triggers a &#x003E;6,000-fold increase in
pachytene piRNA abundance during the ~5 days between the early and late
phases of the pachytene stage of male meiosis. A-MYB also promotes its own
transcription through a positive feedback loop. The A-MYB-regulated feedforward
loop is evolutionarily conserved: A-MYB is bound to the promoters of both
piRNA clusters and PIWIL1, TDRD1, and TDRD3 in the rooster (Gallus gallus)
testis.
<a 
 id="x1-47001r89"></a>
<h3 class="sectionHead"><span class="titlemark">4.2 </span> <a 
 id="x1-480002"></a>RESULTS</h3>
<a 
 id="x1-48001r83"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.1 </span> <a 
 id="x1-490001"></a>Defining piRNA-Producing Transcripts in the Mouse Testis</h4>
<!--l. 23--><p class="noindent" >To define the structure of piRNA-producing loci in the testis of wild-type adult
mice, we assembled the transcripts detected by three biological replicates of
strand-specific, paired-end, rRNA-depleted, total RNA sequencing (RNA-seq;
Figure <a 
href="#x1-49001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>A). We mapped reads to the mouse genome using TopHat [<a 
href="#XTrapnell2009">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2009">2009</a>] and performed de novo transcriptome assembly using Trinity
[<a 
href="#XGrabherr2011">Grabherr et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrabherr2011">2011</a>] to identify unannotated exon-exon junctions. We used all
mapped reads, including reads corresponding to unannotated exon-exon
                                                                                

                                                                                
junctions, to perform reference-based transcript assembly (Cufflinks; [<a 
href="#XTrapnell2010">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>].
<!--l. 25--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-49001r1"></a>
                                                                                

                                                                                
<!--l. 27--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.528x.png" alt="PIC" class="graphics" width="320.19624pt" height="444.66124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.528x.png" src="Figures/Chapter4/MolCel2013_Fig1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.1: </span><span  
class="content">piRNA Precursors are RNA Pol II Transcripts<br 
class="newline" />(A) Strategy  to  assemble  the  mouse  testis  transcriptome.  Rectangles  with
rounded corners,  input  or  output  data;  rectangles,  processes.  Decisions  are
shown without boxing.(B) Aggregated data for piRNA-producing transcripts (5%
trimmed mean). Oxidized small RNA (&#x003E;23 nt) sequencing data were used to
detect piRNAs; transcript abundance was measured using total RNA depleted
of rRNA (RNA-seq). RNA Pol III data were from SRA001030. Dotted lines show
the transcriptional start site (Start) and site of polyadenylation (End). See also
Figure <a 
href="#x1-49002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>. </span></div><!--tex4ht:label?: x1-49001r1 -->
                                                                                

                                                                                
                                                                                

                                                                                
<!--l. 34--><p class="noindent" ></div><hr class="endfigure">
<!--l. 35--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-49002r2"></a>
                                                                                

                                                                                
<!--l. 37--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.529x.png" alt="PIC" class="graphics" width="369.37999pt" height="504.88625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.529x.png" src="Figures/Chapter4/MolCel2013_FigS1.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.2: </span><span  
class="content">(A)  Positions  of  the  214  major  piRNA-producing  genes  on  the
19 autosomes  of  mice.  We  detected  no  loci  on  the  X  or  Y  chromosomes.
(B) Cumulative distributions for all piRNAs and for uniquely mapping piRNAs
comparing the piRNA loci defined by our methods and by previous approaches
[<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. (C) Histogram of distances (in 200 bp
bins) from the annotated 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; or 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of a piRNA gene (this study) or cluster
                                                                                

                                                                                
to the nearest  peak  of  reads  from  high-throughput  sequencing  for  transcript
5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; (CAGE-seq) or 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; (PAS-seq) ends, transcription start sites (H3K4me3) or
A-MYB binding. </span></div><!--tex4ht:label?: x1-49002r2 -->
                                                                                

                                                                                
<!--l. 43--><p class="noindent" ></div><hr class="endfigure">
<!--l. 45--><p class="noindent" >To identify the transcripts that produce piRNAs, we sequenced piRNAs from six
developmental stages of mouse testes (10.5 dpp, 12.5 dpp, 14.5 dpp, 17.5 dpp, 20.5
dpp, and adult) and mapped them to the assembled transcripts. The first round of
spermatogenesis proceeds synchronously among the tubules of the testis:
mouse testes at 10.5 dpp advance no further than the zygotene stage (staging
according to [<a 
href="#XNEBEL1961">NEBEL et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNEBEL1961">1961</a>]; 12.5 dpp to the early pachytene; 14.5
dpp to the middle pachytene; 17.5 to the late pachytene; and 20.5 dpp to
the round spermatid stage. For each stage, we prepared two sequencing
libraries: one comprising all small RNAs and one in which oxidation was used to
enrich for piRNAs by virtue of their 2<span 
class="cmsy-8">&#x2032;</span>-O-methyl-modified 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; termini [<a 
href="#XGhildiyal2008">Ghildiyal
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>].
<!--l. 47--><p class="noindent" >To qualify as a piRNA-producing transcript, an assembled RNA was required to
produce either a sufficiently high piRNA abundance (&#x003E;100 ppm; parts per million
uniquely mapped reads) or density (&#x003E;100 rpkm; reads per kilobase of transcript per
million uniquely mapped reads). These criteria retained both long transcripts
producing an abundance of piRNAs and short transcripts generating many piRNAs
per unit of length. To refine the termini of each piRNA-producing transcript, we
supplemented the RNA-seq data with high-throughput sequencing of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of
RNAs bearing an N(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)ppp(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)N cap structure (cap analysis of gene expression;
CAGE) and the 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of transcripts preceding the poly(A) tail (polyadenylation site
sequencing; PAS-seq). The assembled piRNA-producing transcripts likely correspond
to continuous RNAs in vivo because the CAGE library used to annotate transcript 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
ends was constructed after two rounds of poly(A) selection. Thus, the RNA
molecules in the library derive from complete transcripts extending from the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
cap to the poly(A) tail (Figure <a 
href="#x1-49001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>B). Conventional 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; RACE (rapid
amplification of cDNA ends) analysis of piRNA-producing transcripts confirmed
the ends of 16 loci (data not shown). To provide additional confirmation of
the 5<span 
class="cmsy-8">&#x2032;</span> &#x00A0; end of each piRNA-producing transcript, we also determined the
locations of histone H3 bearing trimethylated lysine 4 (H3K4me3), a histone
modification associated with RNA Pol II transcription start sites <a 
href="#XGuenther2007">Guenther
et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGuenther2007">2007</a>].
<a 
 id="x1-49003r91"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.2 </span> <a 
 id="x1-500002"></a>piRNA Precursor RNAs are Canonical RNA Pol II Transcripts</h4>
<!--l. 51--><p class="noindent" >The presence of 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; caps and poly(A) tails and the binding of histone H3K4me3 to the
genomic DNA immediately upstream of the transcription start site of each piRNA
locus suggest that piRNA transcripts are produced by RNA pol II <a 
href="#x1-49001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>. Moreover, using
antibodies to RNA pol II but not RNA pol III, ChIP-seq showed a peak at the
                                                                                

                                                                                
transcription start site as well as polymerase occupancy across the entire
piRNA gene (Figure <a 
href="#x1-49001r1">4.1<!--tex4ht:ref: fig:MolCelF1 --></a>B; [<a 
href="#XKutter2011">Kutter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKutter2011">2011</a>]. We conclude that piRNA
transcripts are conventional RNA pol II transcripts bearing 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; caps and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; poly(A)
tails.
<a 
 id="x1-50001r94"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.3 </span> <a 
 id="x1-510003"></a>A Transcript-based Set of piRNA Loci</h4>
<!--l. 55--><p class="noindent" >Our transcriptome assembly yielded 467 piRNA-producing transcripts that define 214
genomic loci (Figure <a 
href="#x1-49002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>A and <span class="texthl"><span id="textcolor17">Table S1</span></span>). Among the ~2.2 million distinct piRNA
species and ~8.8 million piRNA reads from the adult mouse testis, the 214 genomic
loci account for 95% of all piRNAs.
<!--l. 57--><p class="noindent" >Previous studies defined piRNA clusters based solely on small RNA sequencing data
[<a 
href="#XAravin2007a">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. Our approach differs in
that it (1) uses RNA-seq data, whose greater read length facilitates the identification
of introns, allowing us to define the architecture of piRNA precursor transcripts and
(2) uses CAGE, PAS-seq, and H3K4me3 ChIP-seq data to refine the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; and 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;
ends of the piRNA transcripts. Consequently, the piRNA loci presented here
account for more piRNAs using fewer genomic base pairs than those previously
defined (Figures <a 
href="#x1-49002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>B and <a 
href="#x1-49002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>C; [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>]. Our
piRNA-producing loci include 41 piRNA loci that escaped previous detection [<a 
href="#XAravin2007a">Aravin
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>,&#x00A0;<a 
href="#XLau2006">Lau et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLau2006">2006</a>], 37 of which contain introns.
The 41 loci account for 2% of piRNAs at 10.5 dpp and 0.36% in the adult
testis.
<a 
 id="x1-51001r95"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.4 </span> <a 
 id="x1-520004"></a>Three Classes of piRNAs During Post-Natal Spermatogenesis</h4>
<!--l. 61--><p class="noindent" >Mice produce three PIWI proteins: MIWI2 (PIWIL4), which binds piRNAs
in perinatal testis [<a 
href="#XAravin2008a">Aravin and Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XCarmell2007">Carmell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XCarmell2007">2007</a>]; MILI
(PIWIL2), which binds piRNAs at least until the round spermatid stage of
spermatogenesis [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XAravin2007a">2007a</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">Kuramochi-Miyagawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKuramochi-Miyagawa2004">2004</a>];
and MIWI (PIWILl), which is first produced during the pachytene stage of
meiosis [<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>]. From 10.5 to 20.5 dpp, piRNA abundance
increases and longer piRNAs appear, reflecting a switch from MILI-bound
piRNAs, which have a 26&#8211;27 nt modal length [<a 
href="#XAravin2006">Aravin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAravin2006">2006</a>,&#x00A0;<a 
href="#XAravin2008a">Aravin and
Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>,&#x00A0;<a 
href="#XMontgomery1998">Montgomery et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMontgomery1998">1998</a>,&#x00A0;<a 
href="#XRobine2009">Robine et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>], to MIWI-bound
piRNAs, which have a 30 nt modal length (Figure <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>A; [<a 
href="#XReuter2009">Reuter et&#x00A0;al.</a>,&#x00A0;<a 
href="#XReuter2009">2009</a>,&#x00A0;<a 
href="#XRobine2009">Robine
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XRobine2009">2009</a>]. This switch occurs at the pachytene phase of meiosis. MILI-bound
pre-pachytene piRNAs predominate before the onset of pachynema; at the
pachytene and round spermatid stages, most piRNAs are MIWI-bound pachytene
piRNAs.
                                                                                

                                                                                
<!--l. 63--><p class="noindent" >We used hierarchical clustering to analyze the change in piRNA abundance from 10.5
to 20.5 dpp for the 214 genes defined by our data (Figures <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>A and <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>A and <span class="texthl"><span id="textcolor18">Table
S2</span></span>). Three types of piRNA-producing genes were identified according to when their
piRNAs first accumulate and how their expression changes during spermatogenesis:
84 pre-pachytene, 100 pachytene, and 30 hybrid loci. At 10.5 dpp, the earliest time
we evaluated, 84 genes dominate piRNA production (median piRNA abundance per
gene = 16 rpkm; Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B). Nearly all (81 out of 84) were congruent with
protein-coding genes. The 84 pre-pachytene piRNA genes account for 13% of
piRNAs at 10.5 dpp, but only 0.31% of piRNAs in the adult testis. Of the
pre-pachytene piRNAs accounted for by the 84 loci, 15% derive from 31
piRNA-producing genes that, to our knowledge, have not previously been
described.
<!--l. 65--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-52001r3"></a>
                                                                                

                                                                                
<!--l. 67--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.530x.png" alt="PIC" class="graphics" width="370.38374pt" height="485.81499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.530x.png" src="Figures/Chapter4/MolCel2013_Fig2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.3: </span><span  
class="content">(A) Normalized piRNA density (rpkm) for each piRNA-producing
gene is shown  as  a  heatmap  across  the  developmental  stages.  Hierarchical
clustering divided  the  genes  into  three  classes.  Arrowheads  mark  seven
pachytene piRNA genes that were not classified as pachytene according to the
change in the abundance of their precursor RNAs from 10.5 to 17.5 dpp.(B)
Top: box plots present piRNA density per gene as spermatogenesis progresses
(here and elsewhere, pre-pachytene in yellow and pachytene in purple). Middle:
expression of  <span 
class="phvro8t-x-x-120">A-Myb</span>,  <span 
class="phvro8t-x-x-120">B-Myb</span>,  <span 
class="phvro8t-x-x-120">Mili</span>,  and  <span 
class="phvro8t-x-x-120">Miwi  </span>was  measured  by  RNA-seq.
                                                                                

                                                                                
Bottom: box plots present piRNA precursor expression per gene, measured by
RNA-seq, from 10.5 to 20.5 dpp. See also Figure <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a> and <span class="texthl"><span id="textcolor19">Table S2</span></span>. </span></div><!--tex4ht:label?: x1-52001r3 -->
                                                                                

                                                                                
<!--l. 73--><p class="noindent" ></div><hr class="endfigure">
<!--l. 74--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-52002r4"></a>
                                                                                

                                                                                
<!--l. 76--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.531x.png" alt="PIC" class="graphics" width="408.52625pt" height="536.00249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.531x.png" src="Figures/Chapter4/MolCel2013_FigS2.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.4: </span><span  
class="content">(A) As shown previously by others using lower temporal resolution,
the modal length of piRNAs increases as spermatogenesis proceeds to more
advanced stages.  (B)  Total  piRNA  rpkm  abundance  and  piRNA  transcript
abundance per  locus  by  class,  from  purified  spermatogonia,  spermatocytes,
                                                                                

                                                                                
round spermatids, and 3 dpp and 8 dpp testis [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]. (C) Correlation between piRNA abundance per locus and piRNA
precursor transcription  from  10.5  to  20.5  dpp.  Throughout  the  Figures,  gold
indicates pre-pachytene and purple indicates pachytene piRNA loci. </span></div><!--tex4ht:label?: x1-52002r4 -->
                                                                                

                                                                                
<!--l. 82--><p class="noindent" ></div><hr class="endfigure">
<!--l. 84--><p class="noindent" >A parallel analysis of piRNA precursor transcription using RNA-seq (&#x003E;100 nt)
corroborated the classification based on piRNA abundance; of the 100 piRNA genes
classified as pachytene based on the developmental expression profile of their
piRNAs, 93 were grouped as pachytene according to the developmental expression
profile of their transcripts. Of these 93, 89 are intergenic. All 84 piRNA genes
designated pre-pachytene using piRNA data were classified as pre-pachytene
according to their transcript abundance.
<!--l. 86--><p class="noindent" >Despite their name, pre-pachytene piRNAs were readily detected in &#x003E;90% and
~95% pure pachytene spermatocytes, as well as round spermatids (Figure
<a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B; [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]. Transcript abundance
from the 84 pre-pachytene loci was high at 3 dpp (median abundance = 11
rpkm), higher by 8 dpp (18 rpkm), and lower in purified leptotene/zygotene
spermatocytes (3.3 rpkm; <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B). Yet piRNA precursor transcripts were readily
detectable in purified pachytene spermatocytes at a level (4.6 rpkm) comparable
to that in purified leptotene/zygotene spermatocytes (Figure <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B); [<a 
href="#XGan2011">Gan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]. From 10.5 to 20.5 dpp, the steady-state
level of pre-pachytene piRNA precursor transcripts remained constant (Figure
<a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>B).
<!--l. 88--><p class="noindent" >Finally, the abundance of pre-pachytene piRNA precursor transcripts was better
correlated with pre-pachytene piRNA abundance at 17.5 dpp (<span 
class="cmmi-12">&#x03C1; </span>= 0.47), when
pachytene spermatocytes compose a larger fraction of the testis, than at 10.5,
12.5, or 14.5 dpp (0.32 <span 
class="cmsy-10x-x-120">&#x2265; </span><span 
class="cmmi-12">&#x03C1; </span><span 
class="cmsy-10x-x-120">&#x2264; </span>0.40; Figure <a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>C). Our data suggest that the
pre-pachytene loci continue to be transcribed and processed into piRNAs long
after spermatocytes enter the pachytene stage of meiosis. Thus, the name
pre-pachytene piRNA is a misnomer that should be retained only for historical
reasons.
<!--l. 90--><p class="noindent" >Hierarchical clustering identified 100 pachytene genes whose piRNAs emerge at 12.5
dpp, 2 days earlier than previously reported [<a 
href="#XGirard2006">Girard et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGirard2006">2006</a>]. Nearly all the
pachytene genes are intergenic (93 out of 100). piRNA expression from pachytene
piRNA genes peaks at 17.5 dpp (Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B). Overall, the median abundance of
piRNAs from these 100 loci increased &#x003E;6,000-fold from 10.5 to 17.5 dpp. Transcripts
from pachytene genes were low at 10.5 dpp (median abundance = 0.15 rpkm) and
increased 116-fold from 10.5 to 17.5 dpp. From 10.5 to 20.5 dpp, the dynamics of
pachytene piRNA abundance from each piRNA gene correlated with the increase in
abundance of its precursor transcripts (0.39 <span 
class="cmsy-10x-x-120">&#x2265; </span><span 
class="cmmi-12">&#x03C1; </span><span 
class="cmsy-10x-x-120">&#x2264; </span>0.63; <span 
class="cmmi-12">&#x03C1;value </span><span 
class="cmsy-10x-x-120">&#x2264; </span><span 
class="cmr-12">7</span><span 
class="cmmi-12">.</span><span 
class="cmr-12">3</span><span 
class="cmmi-12">x</span><span 
class="cmr-12">10 </span><span 
class="cmsy-10x-x-120">- </span><span 
class="cmr-12">5</span>; Figure
<a 
href="#x1-52002r4">4.4<!--tex4ht:ref: fig:MolCelS2 --></a>C). The 100 pachytene genes account for 92% of piRNAs in the adult testis,
making it unlikely that biologically functional pachytene piRNAs originate from
thousands of genomic loci [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>]. Figures <span 
class="phvb8t-x-x-120">??</span> and <a 
href="#x1-52004r6">4.6<!--tex4ht:ref: fig:MolCelS3 --></a> provide
                                                                                

                                                                                
examples of pachytene and pre-pachytene piRNA genes defined by our
data.
                                                                                

                                                                                
                                                                                <!--l. 94--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <a 
 id="x1-52003r5"></a>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 96--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.532x.png" alt="PIC" class="graphics" width="482.80374pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.532x.png" src="Figures/Chapter4/MolCel2013_Fig3.eps"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;4.5: </span><span  
class="content">Previous cluster boundaries are from <a 
href="#XLau2006">Lau et&#x00A0;al.</a>&#x00A0;[<a 
href="#XLau2006">2006</a>] in gray and <a 
href="#XGirard2006">Girard et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGirard2006">2006</a>] in dark gray). </span></div><!--tex4ht:label?: x1-52003r5 -->
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 102--><p class="noindent" ></div><hr class="endfigure">
<!--l. 103--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <a 
 id="x1-52004r6"></a>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 105--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.533x.png" alt="PIC" class="graphics" width="387.4475pt" height="289.07999pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.533x.png" src="Figures/Chapter4/MolCel2013_FigS3.eps"  
-->
<br /> <div class="caption" 
><span class="id">Figure&#x00A0;4.6: </span><span  
class="content">Previous cluster boundaries are from <a 
href="#XLau2006">Lau et&#x00A0;al.</a>&#x00A0;[<a 
href="#XLau2006">2006</a>] in gray and <a 
href="#XGirard2006">Girard et&#x00A0;al.</a>&#x00A0;[<a 
href="#XGirard2006">2006</a>] in dark gray). </span></div><!--tex4ht:label?: x1-52004r6 -->
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 111--><p class="noindent" ></div><hr class="endfigure">
                                                                                                                                                                                      
                                                                                

                                                                                
<!--l. 114--><p class="noindent" >Hierarchical clustering detected a third class, hybrid piRNAs, which derives from 30
genes with characteristics of both pre-pachytene and pachytene piRNA loci. Like
pre-pachytene, hybrid piRNAs were detected at 10.5 dpp (median abundance = 3.7
rpkm) and in purified spermatogonia [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>]. Like pachytene piRNAs,
hybrid piRNA abundance increased during the pachytene stage of meiosis, but the
increase was delayed until late (17.5 dpp) rather than early pachynema (14.5 dpp).
Overall, piRNAs from hybrid genes increased &#x003E;10-fold from 14.5 to 17.5 dpp. The
median abundance of piRNAs from hybrid piRNA genes ranged from 90&#8211;120 rpkm
in purified pachytene spermatocytes, &#x003E;20-fold greater than their median
abundance in spermatogonia [<a 
href="#XGan2011">Gan et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>].
Moreover, hybrid piRNA precursor transcripts were readily detected in purified
pachytene spermatocytes (median abundance = 9.0 rpkm; [<a 
href="#XModzelewski2012">Modzelewski
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>]).
<a 
 id="x1-52005r96"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.5 </span> <a 
 id="x1-530005"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Precursor Transcription</h4>
<!--l. 118--><p class="noindent" >The coordinated increase in pachytene piRNA precursor transcripts suggests their
regulation by a common transcription factor or factors. Among the 100 pachytene
piRNA genes, 15 pairs (30 genes) are divergently transcribed. The 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of the
piRNA precursor RNAs from each pair are close in genomic distance (median = 127
bp), suggesting that a shared promoter lies between the two transcription start
sites.
<!--l. 120--><p class="noindent" >We took advantage of the unique genomic organization of these 15 pairs of
divergently transcribed piRNA genes to search for sequence motifs common to their
promoters. The MEME algorithm [<a 
href="#XBailey1994">Bailey and Elkan</a>,&#x00A0;<a 
href="#XBailey1994">1994</a>] revealed a motif highly
enriched in these bidirectional promoters (E = 8.3 x 10<sup><span 
class="cmr-8">12</span></sup>; Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A). This motif
matches the binding site of the Myb family of transcription factors (Figure
<a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A; [<a 
href="#XGupta2007">Gupta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGupta2007">2007</a>,&#x00A0;<a 
href="#XNewburger2009">Newburger and Bulyk</a>,&#x00A0;<a 
href="#XNewburger2009">2009</a>]. The Myb motif is
not restricted to bidirectional promoters; MEME identified the same motif
using the promoters of all pachytene piRNA genes (E = 9.1 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">28</span></sup>; Figure
<a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B).
<!--l. 122--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-53001r7"></a>
                                                                                

                                                                                
<!--l. 124--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.534x.png" alt="PIC" class="graphics" width="393.46999pt" height="391.4625pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.534x.png" src="Figures/Chapter4/MolCel2013_Fig4.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.7: </span><span  
class="content">(A)  Top:  MEME  identified  a  sequence  motif  in  the  bidirectional
promoters of the 15 pairs of divergently transcribed pachytene piRNA genes.
E value computed  by  MEME  measures  the  statistical  significance  of  the
motif. Middle:  Myb  motif  from  the  mouse  UniPROBE  database.  Bottom:
MEME-reported motif for the top 500 (by peak score) A-MYB ChIP-seq peaks
from adult mouse testes.(B) A-MYB ChIP-seq data for the common promoter
of the divergently transcribed pachytene piRNA genes <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and
<span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>.(C) The distance from the annotated transcription start site
(TSS) of each piRNA gene to the nearest A-MYB peak. See also Figure <a 
href="#x1-53002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>. </span></div><!--tex4ht:label?: x1-53001r7 -->
                                                                                

                                                                                
<!--l. 130--><p class="noindent" ></div><hr class="endfigure">
<!--l. 131--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-53002r8"></a>
                                                                                

                                                                                
<!--l. 133--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.535x.png" alt="PIC" class="graphics" width="385.43999pt" height="454.69875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.535x.png" src="Figures/Chapter4/MolCel2013_FigS4.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.8:   </span><span  
class="content">(A)   A-MYB   binds   to   the   common   promoter   of   divergently
transcribed pachytene  piRNA  loci  <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1  </span>and  <span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>.
The abundance of DNA fragments at the amplified region relative to a control
region (mean <span 
class="cmsy-10x-x-120"> </span>standard deviation; n = 3) was measured by qPCR (top). The
A-MYB ChIP-seq (red) and input (black) data for this pair of genes is presented
as in Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B. (B) ChIP-seq and qPCR were as in (A), but for the promoter
region of <span 
class="phvro8t-x-x-120">Miwi  </span>(Piwil1). Also shown is the RefSeq gene model. Exons, black;
introns, gray. </span></div><!--tex4ht:label?: x1-53002r8 -->
                                                                                

                                                                                
<!--l. 139--><p class="noindent" ></div><hr class="endfigure">
<!--l. 141--><p class="noindent" >The Myb transcription factor family is conserved among eukaryotes. Like other
vertebrates, mice produce three Myb proteins, A-MYB (MYBL1), B-MYB
(MYBL2), and C-MYB (MYB), each with a distinct tissue distribution [<a 
href="#XLatham1996">Latham
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLatham1996">1996</a>,&#x00A0;<a 
href="#XMettus1994">Mettus et&#x00A0;al.</a>,&#x00A0;<a 
href="#XMettus1994">1994</a>,&#x00A0;<a 
href="#XOh1999">Oh and Reddy</a>,&#x00A0;<a 
href="#XOh1999">1999</a>,&#x00A0;<a 
href="#XTrauth1994">Trauth et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrauth1994">1994</a>].
Testes produce both A- and B-MYB proteins. Multiple lines of evidence implicate
A-MYB, rather than B-MYB, as a candidate for regulating pachytene piRNA
transcription. First, the expression of <span 
class="phvro8t-x-x-120">A-Myb </span>during spermatogenesis resembles that
of pachytene piRNAs: <span 
class="phvro8t-x-x-120">A-Myb </span>transcripts appear at ~12.5 dpp and peak at 17.5 dpp
(Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B; [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. The expression of <span 
class="phvro8t-x-x-120">A-Myb </span>messenger RNA
(mRNA) increases ~15-fold from 8 dpp to 19 dpp, whereas <span 
class="phvro8t-x-x-120">B-Myb </span>mRNA expression
remains constant and low during the same time frame and into adulthood [<a 
href="#XHorvath2009">Horvath
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>]. Our RNA-seq data (Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B) corroborate these findings. Indeed, in
our RNA-seq analysis of adult testes, <span 
class="phvro8t-x-x-120">A-Myb </span>mRNA was 24-fold more abundant
than <span 
class="phvro8t-x-x-120">B-Myb</span>. Second, a testis-specific <span 
class="phvro8t-x-x-120">A-Myb </span>point-mutant allele, <span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>,
which is caused by a cytosine-to-adenine transversion that changes alanine
213 to glutamic acid, leads to meiotic arrest at the pachytene stage with
subtle defects in autosome synapsis; <span 
class="phvro8t-x-x-120">A-Myb </span>null mutant mice have defects in
multiple tissues, including the testis and the mammary gland [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XToscani1997">Toscani et&#x00A0;al.</a>,&#x00A0;<a 
href="#XToscani1997">1997</a>]. Third, our RNA-seq analysis of <span 
class="phvro8t-x-x-120">A-Myb</span>
mutant testes shows that there is no significant change in <span 
class="phvro8t-x-x-120">B-Myb </span>expression in
the mutant, compared to the heterozygous controls, at 14.5 or 17.5 dpp.
Finally, B-MYB protein is not detectable in pachytene spermatocytes [<a 
href="#XHorvath2009">Horvath
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>].
<!--l. 143--><p class="noindent" >To assess more directly the role of A-MYB in pachytene piRNA precursor
transcription, we used anti-A-MYB antibody to perform ChIP followed by
high-throughput sequencing of the A-MYB-bound DNA. The anti-A-MYB
antibody is specific for A-MYB, and the peptide used to raise the antibody is not
present in B-MYB. The model-based analysis of ChIP-seq (MACS) algorithm
[<a 
href="#XZhang2008">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>] reported 3,815 genomic regions with significant A-MYB
binding (false discovery rate, FDR &#x003C; 10<sup><span 
class="cmr-8">25</span></sup>); we call these regions A-MYB peaks
or peaks. Among the 500 peaks with the lowest FDR values, 394 (80%)
contained at least one significant site (<span 
class="cmmi-12">&#x03C1; &#x003C; </span><span 
class="cmr-12">10</span><sup><span 
class="cmr-8">4</span></sup>) for the MYB binding motif
(Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A). Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>B shows an example of such an A-MYB peak at the
bidirectional promoter of the divergently transcribed pair of pachytene piRNA
genes <span 
class="phvro8t-x-x-120">17-qA3.3-27363.1 </span>and <span 
class="phvro8t-x-x-120">17-qA3.3-26735.1</span>. A-MYB occupancy of this
genomic site was confirmed by ChIP and quantitative PCR (ChIP-qPCR) (Figure
<a 
href="#x1-53002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>A).
<!--l. 145--><p class="noindent" >The median distance from the transcription start site to the nearest A-MYB peak was
~43 bp for the 100 pachytene piRNA genes but &#x003E;66,000 bp for the 84 pre-pachytene
                                                                                

                                                                                
genes (Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>C). Our data suggest that during mouse spermatogenesis A-MYB
binds to the promoters of both divergently and unidirectionally transcribed pachytene
piRNA genes.
<!--l. 147--><p class="noindent" >To test the idea that A-MYB promotes transcription of pachytene, but not
pre-pachytene, piRNA genes, we used RNA-seq to measure the abundance of RNA &#x003E;
100 nt long from the testes of <span 
class="phvro8t-x-x-120">A-Myb </span>point-mutant (<span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>) mice and their
heterozygous littermates (Figure <a 
href="#x1-54001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). Pachytene piRNA precursor transcripts&#8212;both
divergently and unidirectionally transcribed&#8212;were significantly depleted in <span 
class="phvro8t-x-x-120">A-Myb</span>
mutant testes compared to the heterozygotes: the median decrease was 45-fold at
14.5 dpp (q = 1.1 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">13</span></sup>) and 248-fold at 17.5 dpp (q = 3.9 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">23</span></sup>). The abundance
of pre-pachytene piRNA transcripts was not significantly changed (q <span 
class="cmsy-10x-x-120">&#x2265; </span>0.34). The
binding of A-MYB to the promoters of pachytene piRNA genes, together with the
depletion of pachytene piRNA transcripts in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant, further supports
the view that A-MYB directly regulates transcription of pachytene piRNA
genes.
<a 
 id="x1-53003r101"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.6 </span> <a 
 id="x1-540006"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Pachytene piRNA Production</h4>
<!--l. 151--><p class="noindent" >To test the consequences of the loss of piRNA precursor transcripts, we measured
piRNA abundance in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant. Like pachytene piRNA precursor
transcription, pachytene piRNA abundance significantly decreased in mutant testes.
At 14.5 dpp, median piRNA abundance per pachytene gene decreased 87-fold in
<span 
class="phvro8t-x-x-120">A-Myb </span>homozygous mutant testes compared to heterozygotes (<span 
class="cmmi-12">&#x03C1; &#x003C; </span><span 
class="cmr-12">2</span><span 
class="cmmi-12">.</span><span 
class="cmr-12">2</span><span 
class="cmmi-12">X</span><span 
class="cmr-12">10</span><sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">16</span></sup>; Figure
<a 
href="#x1-54001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>. By 17.5 dpp, median pachytene piRNA abundance was &#x003E;9,000 times lower in the
<span 
class="phvro8t-x-x-120">A-Myb </span>mutant than the heterozygotes (P &#x003C; 2.2 x 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">16</span></sup>). In contrast, pre-pachytene
piRNA levels were essentially unaltered. Figure 6 presents examples of the effect at
14.5 and 17.5 dpp of the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant on piRNA precursor transcript and
mature piRNA abundance for one pre-pachytene and three pachytene piRNA
genes.
<!--l. 153--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-54001r9"></a>
                                                                                

                                                                                
<!--l. 155--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.536x.png" alt="PIC" class="graphics" width="399.4925pt" height="415.55249pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.536x.png" src="Figures/Chapter4/MolCel2013_Fig5.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.9: </span><span  
class="content">The change in transcript or piRNA abundance per gene in <span 
class="phvro8t-x-x-120">A-Myb</span>
(n = 3) and <span 
class="phvro8t-x-x-120">Miwi </span>(n = 1) mutants compared to heterozygotes in testes isolated
at 14.5 and 17.5 dpp. See also Figure <a 
href="#x1-54002r10">4.10<!--tex4ht:ref: fig:MolCelS5 --></a>. </span></div><!--tex4ht:label?: x1-54001r9 -->
                                                                                

                                                                                
<!--l. 161--><p class="noindent" ></div><hr class="endfigure">
<!--l. 162--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-54002r10"></a>
                                                                                

                                                                                
<!--l. 164--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.537x.png" alt="PIC" class="graphics" width="308.15125pt" height="254.9525pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.537x.png" src="Figures/Chapter4/MolCel2013_FigS5.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.10: </span><span  
class="content">Change in piRNA abundance per locus (rpkm) for <span 
class="phvro8t-x-x-120">Spo11 </span>(14.5
dpp), <span 
class="phvro8t-x-x-120">Miwi </span>(<span 
class="phvro8t-x-x-120">Piwil2</span>;  10.5  dpp),  <span 
class="phvro8t-x-x-120">Tdrd6  </span>(18  dpp),  and  <span 
class="phvro8t-x-x-120">Tdrd9  </span>(14  dpp)  mutants
compared to heterozygous controls. </span></div><!--tex4ht:label?: x1-54002r10 -->
                                                                                

                                                                                
<!--l. 170--><p class="noindent" ></div><hr class="endfigure">
<!--l. 172--><p class="noindent" >Our data show that A-MYB binds to the promoters of pachytene piRNA genes;
<span 
class="phvro8t-x-x-120">A-Myb</span>, <span 
class="phvro8t-x-x-120">Miwi</span>, and pachytene piRNA precursor transcription begins at 12.5 dpp; and
<span 
class="phvro8t-x-x-120">A-Myb </span>mutant spermatocytes reach pachynema with subtle defects in autosome
synapsis [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Could pachytene piRNA depletion nonetheless
be an indirect consequence of the meiotic arrest caused by the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant? To
test this possibility, we sequenced small RNAs from <span 
class="phvro8t-x-x-120">Spo11 </span>mutant testes, which failed
to generate double-stranded DNA breaks at the leptotene stage and display a meiotic
arrest [<a 
href="#XBaudat2000c">Baudat et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBaudat2000c">2000</a>,&#x00A0;<a 
href="#XRomanienko2000">Romanienko and Camerini-Otero</a>,&#x00A0;<a 
href="#XRomanienko2000">2000</a>]. The median
abundance of piRNAs from pre-pachytene genes did not decrease at 14.5 dpp. By
17.5 dpp, piRNA from pachytene genes decreased just 5.9-fold in the <span 
class="phvro8t-x-x-120">Spo11</span>
mutant testes compared to the heterozygotes (Figure <a 
href="#x1-54002r10">4.10<!--tex4ht:ref: fig:MolCelS5 --></a>). We note that
A-MYB protein abundance is reduced in the <span 
class="phvro8t-x-x-120">Spo11 </span>mutant [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>].
<!--l. 174--><p class="noindent" ><span 
class="phvro8t-x-x-120">Trip13 </span>is required to complete the repair of double-strand DNA breaks on fully
synapsed chromosomes. <span 
class="phvro8t-x-x-120">Trip13 </span>mutants display a meiotic arrest similar to
that in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes [<a 
href="#XLi2007">Li and Schimenti</a>,&#x00A0;<a 
href="#XLi2007">2007</a>]: pachytene arrest with
synapsed chromosomes. To further test whether the loss of pachytene piRNA
precursor transcripts in <span 
class="phvro8t-x-x-120">A-Myb </span>mutants reflects a general effect of meiotic
arrest, we measured piRNA precursor transcript abundance in <span 
class="phvro8t-x-x-120">Trip13 </span>mutant
testes at 17.5 dpp. Unlike <span 
class="phvro8t-x-x-120">A-Myb</span>, piRNA precursor transcripts were readily
detectable in the <span 
class="phvro8t-x-x-120">Trip13 </span>mutant (Figure <a 
href="#x1-55002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>). We conclude that the loss of
pachytene piRNA precursor transcripts and piRNAs in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes is a
direct consequence of the requirement for A-MYB to transcribe pachytene
piRNA genes and not a general feature of meiotic arrest at the pachytene
stage.
<a 
 id="x1-54003r104"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.7 </span> <a 
 id="x1-550007"></a><span 
class="phvro8t-x-x-120">A-Myb </span>Regulates Expression of piRNA Biogenesis Factors</h4>
<!--l. 178--><p class="noindent" >The <span 
class="phvro8t-x-x-120">A-Myb </span>mutant more strongly affected pachytene piRNA accumulation than it did
the steady-state abundance of the corresponding piRNA precursor transcripts (Figure
<a 
href="#x1-54001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>; the median decrease in pachytene piRNA abundance was 2-fold greater at 14.5
dpp and 38-fold greater at 17.5 dpp than the decrease in the steady-state abundance
of pachytene precursor transcripts <span class="texthl"><span id="textcolor20">(Table S1)</span></span>. These data suggest that A-MYB exerts
a layer of control on piRNA accumulation beyond its role in promoting pachytene
piRNA precursor transcription.
<!--l. 180--><p class="noindent" ><span 
class="phvro8t-x-x-120">Miwi </span>has previously been proposed to be a direct target of A-MYB; <span 
class="phvro8t-x-x-120">Miwi </span>mRNA
abundance is reduced in A-MYB mutant testes, and ChIP microarray data
                                                                                

                                                                                
place A-MYB on the <span 
class="phvro8t-x-x-120">Miwi </span>promoter [<a 
href="#XBolcun-Filas2011">Bolcun-Filas et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Our RNA-seq
data confirm that accumulation of <span 
class="phvro8t-x-x-120">Miwi </span>mRNA requires A-MYB: <span 
class="phvro8t-x-x-120">Miwi </span>mRNA
decreased more than 50-fold in testes isolated from <span 
class="phvro8t-x-x-120">A-Myb </span>mutant mice at
14.5 dpp compared to their heterozygous littermates (Figures <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>A and
<a 
href="#x1-56002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a> and <span class="texthl"><span id="textcolor21">Table S3</span></span>). Furthermore, our ChIP data confirm that A-MYB binds
the <span 
class="phvro8t-x-x-120">Miwi </span>promoter in vivo (Figures <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B, <a 
href="#x1-53002r8">4.8<!--tex4ht:ref: fig:MolCelS4 --></a>B, and <a 
href="#x1-56002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a>). Like pachytene
piRNAs, <span 
class="phvro8t-x-x-120">Miwi </span>transcripts first appear at 12.5 dpp (Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>B), and MIWI
protein is first detected in testes at 14.5 dpp [<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>]. Loss
of MIWI arrests spermatogenesis at the round spermatid stage [<a 
href="#XDeng2002c">Deng and
Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>].
<!--l. 182--><p class="noindent" >A previous study reported that piRNAs fail to accumulate to wild-type levels in <span 
class="phvro8t-x-x-120">Miwi</span>
mutant testes [<a 
href="#XGrivna2006">Grivna et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrivna2006">2006</a>]. However, our data suggest that the overall
change in piRNA abundance caused by loss of MIWI is quite small: RNA-seq
detected no change at 14.5 dpp (change in total piRNA abundance = 1.1; n = 2) and
only a modest decrease at 17.5 dpp (change in total piRNA abundance = 0.58; n = 1).
piRNAs from pachytene loci decreased just 2.7-fold at 14.5 dpp (p = 0.0046) and
3.5-fold at 17.5 dpp (p = 1.8 x 10-6) in <span 
class="phvro8t-x-x-120">Miwi </span>mutant testes (Figure <a 
href="#x1-54001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). By
comparison, pachytene piRNAs declined 87-fold at 14.5 dpp and 9,400-fold at 17.5
dpp in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant.
<!--l. 184--><p class="noindent" >Does the loss of MIWI affect piRNA precursor transcription? We measured transcript
abundance and piRNA expression in <span 
class="phvro8t-x-x-120">Miwi </span>null mutant testes at 14.5 and 17.5 dpp. In
<span 
class="phvro8t-x-x-120">Miwi</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span> </sup> testes, pachytene piRNA precursor transcripts were present at levels
indistinguishable from <span 
class="phvro8t-x-x-120">Miwi </span>heterozygotes (median change = 1.0- to 1.4-fold; q = 1;
Figure <a 
href="#x1-54001r9">4.9<!--tex4ht:ref: fig:MolCelF5 --></a>). Thus, loss of MIWI does not explain loss of pachytene piRNA precursor
transcripts in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes.
<!--l. 186--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-55001r11"></a>
                                                                                

                                                                                
<!--l. 188--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.538x.png" alt="PIC" class="graphics" width="396.48125pt" height="345.29pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.538x.png" src="Figures/Chapter4/MolCel2013_Fig6.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.11:  </span><span  
class="content">Transcript  and  piRNA  abundance  in  heterozygous  (Het)  and
homozygous  <span 
class="phvro8t-x-x-120">A-Myb  </span>(Mut)  point-mutant  testes  is  shown  for  four  illustrative
examples at  14.5  and  17.5  dpp.  Also  shown  is  the  abundance  of  piRNA
sequencing reads  that  map  to  the  exon-exon  junctions.  Gene  <span 
class="phvro8t-x-x-120">11-qE1-9443</span>
does not have an intron. Exons, blue boxes; splice junctions, gaps; the last exon
is compressed and not to scale. See also Figure <a 
href="#x1-55002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>. </span></div><!--tex4ht:label?: x1-55001r11 -->
                                                                                

                                                                                
<!--l. 194--><p class="noindent" ></div><hr class="endfigure">
<!--l. 195--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-55002r12"></a>
                                                                                

                                                                                
<!--l. 197--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.539x.png" alt="PIC" class="graphics" width="342.27875pt" height="357.33499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.539x.png" src="Figures/Chapter4/MolCel2013_FigS6.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.12:  </span><span  
class="content">Transcripts  were  detected  in  total  RNA  from  adult  testes  by
RT-PCR (using random primers) for five pachytene piRNA loci as well as <span 
class="phvro8t-x-x-120">Miwi</span>
and <span 
class="phvro8t-x-x-120">Actin</span>. Mut, mutant; Het or H, heterozygote; wt, wild type. </span></div><!--tex4ht:label?: x1-55002r12 -->
                                                                                

                                                                                
<!--l. 203--><p class="noindent" ></div><hr class="endfigure">
<!--l. 205--><p class="noindent" >In addition to <span 
class="phvro8t-x-x-120">Miwi</span>, ChIP-seq detected A-MYB bound to the promoters of 12 other
RNA-silencing-pathway genes (Figure <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B and <span class="texthl"><span id="textcolor22">Table S3</span></span>). Of these, the mRNA
abundance&#8212;measured by three biologically independent RNA-seq experiments&#8212;of
<span 
class="phvro8t-x-x-120">Ago2</span>, <span 
class="phvro8t-x-x-120">Ddx39 </span>(uap56 in flies), <span 
class="phvro8t-x-x-120">Mael</span>, <span 
class="phvro8t-x-x-120">Mili</span>, <span 
class="phvro8t-x-x-120">Mov10l1</span>, <span 
class="phvro8t-x-x-120">Tdrd9</span>, and <span 
class="phvro8t-x-x-120">Vasa </span>did not change
significantly at 14.5 dpp in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes compared to heterozygotes (q &#x003E;
0.05); except for <span 
class="phvro8t-x-x-120">Ago2</span>, all decreased significantly in the mutant at 17.5 dpp. In
contrast, the abundance of the mRNAs encoding Tudor domain proteins decreased
significantly in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes: <span 
class="phvro8t-x-x-120">Tdrd6 </span>(64-fold decrease; q = 3.1 x 10-5) and
<span 
class="phvro8t-x-x-120">Tdrd5 </span>(7.5-fold decrease; q = 1.0 x 10-5). <span 
class="phvro8t-x-x-120">Tdrd5 </span>is expressed in embryonic testes
then decreases around birth [<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. <span 
class="phvro8t-x-x-120">TDRD5 </span>protein reappears at
12 dpp, increasing throughout the pachynema [<a 
href="#XSmith2004">Smith et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSmith2004">2004</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. Our data indicate that A-MYB activates <span 
class="phvro8t-x-x-120">Tdrd5 </span>transcription at the
onset of the pachytene stage of meiosis. Similarly, <span 
class="phvro8t-x-x-120">Tdrd6 </span>mRNA can be
detected at the middle pachytene, but not the zygotene stage, and peaks after
late pachytene; TDRD6 protein can be detected at 17 dpp and continues
to increase until 21 dpp [<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>]. The findings that TDRD5
and TDRD6 colocalize with MIWI in pachytene spermatocytes [<a 
href="#XHosokawa2007">Hosokawa
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHosokawa2007">2007</a>,&#x00A0;<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>] and that TDRD6 binds
MIWI [<a 
href="#XChen2009a">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2009a">2009</a>,&#x00A0;<a 
href="#XVagin2009">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>,&#x00A0;<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>] suggest
a role for these Tudor domain proteins in pachytene piRNA production or
function. As in <span 
class="phvro8t-x-x-120">Miwi</span>-/- testes, spermatogenesis arrests at the round spermatid
stage in <span 
class="phvro8t-x-x-120">Tdrd5</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span></sup> and <span 
class="phvro8t-x-x-120">Tdrd6</span><sup><span 
class="cmsy-8">-</span><span 
class="cmmi-8">&#x2215;</span><span 
class="cmsy-8">-</span></sup> mutant testes [<a 
href="#XVasileva2009">Vasileva et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVasileva2009">2009</a>,&#x00A0;<a 
href="#XYabuta2011">Yabuta
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYabuta2011">2011</a>]. Loss of <span 
class="phvro8t-x-x-120">Tdrd6 </span>expression has little effect on piRNA levels (Figure <a 
href="#x1-52004r6">4.6<!--tex4ht:ref: fig:MolCelS3 --></a>;
[<a 
href="#XVagin2009">Vagin et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>], perhaps because the functions of Tudor domain proteins
overlap.
<!--l. 207--><p class="noindent" >Other genes encoding piRNA pathway proteins whose promoters are bound by
A-MYB and whose expression decreased significantly in <span 
class="phvro8t-x-x-120">A-Myb </span>mutant testes include
<span 
class="phvro8t-x-x-120">MitoPld </span>(<span 
class="phvro8t-x-x-120">Pld6</span>; 3.9-fold decrease; q = 0.0095) and <span 
class="phvro8t-x-x-120">Tdrd12 </span>(5.3-fold decrease; q
= 0.0046). <span 
class="phvro8t-x-x-120">MitoPld </span>encodes an endoribonuclease implicated in an early
step in piRNA biogenesis in mice and flies [<a 
href="#XHaase2010">Haase et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHaase2010">2010</a>,&#x00A0;<a 
href="#XHouwing2007">Houwing
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>,&#x00A0;<a 
href="#XHuang2011">Huang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHuang2011">2011</a>,&#x00A0;<a 
href="#XIpsaro2012">Ipsaro et&#x00A0;al.</a>,&#x00A0;<a 
href="#XIpsaro2012">2012</a>,&#x00A0;<a 
href="#XNishimasu2012">Nishimasu et&#x00A0;al.</a>,&#x00A0;<a 
href="#XNishimasu2012">2012</a>,&#x00A0;<a 
href="#XPane2007">Pane
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPane2007">2007</a>,&#x00A0;<a 
href="#XWatanabe2011a">Watanabe et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWatanabe2011a">2011a</a>]. The function of Tdrd12 is not known, but its
fly homologs (Yb, Brother of Yb, and Sister of Yb) are all required for piRNA
production [<a 
href="#XHandler2011">Handler et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHandler2011">2011</a>]. <span 
class="phvro8t-x-x-120">Tdrd1 </span>decreased 3.4-fold, but with q value = 0.015.
<span 
class="phvro8t-x-x-120">Tdrd1 </span>is first expressed in fetal prospermatogonia, then re-expressed in
pachytene spermatocytes [<a 
href="#XChuma2006a">Chuma and Hosokawa</a>,&#x00A0;<a 
href="#XChuma2006a">2006</a>]. In Tdrd1 mutant testes,
spermatogenesis fails, with no spermatocytes progressing past the round spermatid
stage [<a 
href="#XChuma2006a">Chuma and Hosokawa</a>,&#x00A0;<a 
href="#XChuma2006a">2006</a>]. TDRD1 binds MILI and MIWI [<a 
href="#XChen2009a">Chen
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2009a">2009</a>,&#x00A0;<a 
href="#XKojima2009">Kojima et&#x00A0;al.</a>,&#x00A0;<a 
href="#XKojima2009">2009</a>] and colocalizes with TDRD5 and TDRD6 in the
                                                                                

                                                                                
chromatoid body [<a 
href="#XHosokawa2007">Hosokawa et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHosokawa2007">2007</a>].
<!--l. 209--><p class="noindent" >Together, these data support the idea that at the onset of the pachytene phase of
meiosis, A-MYB coordinately activates transcription of many genes encoding piRNA
pathway proteins.
<a 
 id="x1-55003r107"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.8 </span> <a 
 id="x1-560008"></a>A-MYB and the Pachytene piRNA Regulatory Circuitry</h4>
<!--l. 213--><p class="noindent" >A number of genes encoding known and suspected piRNA pathway proteins are
bound and regulated by A-MYB (Figures <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B and <a 
href="#x1-56002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a>C). Our data support a
model in which A-MYB drives both the transcription of pachytene piRNA genes and
the mRNAs encoding genes required for piRNA production including <span 
class="phvro8t-x-x-120">Miwi</span>, <span 
class="phvro8t-x-x-120">MitoPld</span>,
and <span 
class="phvro8t-x-x-120">Tdrd9</span>. Regulation by A-MYB of both the sources of pachytene piRNAs
and the piRNA biogenesis machinery creates a coherent feedforward loop
(Figure <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>C). Feedforward loops amplify initiating signals to increase target
gene expression. Furthermore, they function as switches that are sensitive to
sustained signals; they reject transient signals [<a 
href="#XOsella2011">Osella et&#x00A0;al.</a>,&#x00A0;<a 
href="#XOsella2011">2011</a>,&#x00A0;<a 
href="#XShen-Orr2002">Shen-Orr
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShen-Orr2002">2002</a>].
<!--l. 215--><p class="noindent" >A-MYB also bound to the <span 
class="phvro8t-x-x-120">A-Myb </span>promoter (Figure <a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B), and <span 
class="phvro8t-x-x-120">A-Myb </span>transcripts
decreased 4.2-fold in testes from an <span 
class="phvro8t-x-x-120">A-Myb </span>point mutant (<span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>; Figure
<a 
href="#x1-56001r13">4.13<!--tex4ht:ref: fig:MolCelF7 --></a>B). The <span 
class="phvro8t-x-x-120">A-Myb </span>mutant fails to produce the high level of A-MYB protein
observed in wild-type testes at the late pachytene stage of meiosis [<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>]. Instead, A-MYB protein never becomes more abundant than the level
achieved in wild-type testes by the beginning of the pachytene stage. While the
lower level of A-MYB in the <span 
class="phvro8t-x-x-120">A-Myb </span>mutant may reflect instability of the mutant
protein, a simpler explanation is that mutant A-MYB cannot activate <span 
class="phvro8t-x-x-120">A-Myb</span>
transcription.
<!--l. 217--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-56001r13"></a>
                                                                                

                                                                                
<!--l. 219--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.540x.png" alt="PIC" class="graphics" width="415.55249pt" height="502.87874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.540x.png" src="Figures/Chapter4/MolCel2013_Fig7.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.13:  </span><span  
class="content">(A)  mRNA  abundance  in  <span 
class="phvro8t-x-x-120">A-Myb  </span>mutant  versus  heterozygous
testes. The 407 genes with a significant (q &#x003C; 0.05) change in steady-state mRNA
levels are shown as red circles. The 203 with A-MYB peaks within 500 bp of their
transcription start site are filled. (B) A-MYB ChIP-seq signal at the transcription
start sites of  <span 
class="phvro8t-x-x-120">A-Myb  </span>and  genes  implicated  in  RNA  silencing  pathways.  For
each, the figure  reports  the  change  in  mRNA  abundance  between  17.5  and
                                                                                

                                                                                
10.5 dpp in wild-type testes and the mean change between <span 
class="phvro8t-x-x-120">A-Myb </span>mutant and
heterozygous  testes  at  14.5  dpp  (mean  <span 
class="cmsy-10x-x-120"> </span>SD;  n  =  3).  (C)  A  model  for  the
regulation of pachytene piRNA biogenesis by A-MYB. See also Figure <a 
href="#x1-56002r14">4.14<!--tex4ht:ref: fig:MolCelS7 --></a> and
<span class="texthl"><span id="textcolor23">Table S3</span></span>. </span></div><!--tex4ht:label?: x1-56001r13 -->
                                                                                

                                                                                
<!--l. 227--><p class="noindent" ></div><hr class="endfigure">
<!--l. 228--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-56002r14"></a>
                                                                                

                                                                                
<!--l. 230--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.541x.png" alt="PIC" class="graphics" width="373.39499pt" height="557.08124pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.541x.png" src="Figures/Chapter4/MolCel2013_FigS7.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.14: </span><span  
class="content">A) mRNA abundance in 17.5 dpp <span 
class="phvro8t-x-x-120">A-Myb  </span>versus heterozygous
testes. The 2,853 genes with a significant (q &#x003C; 0.051) change in steady-state
mRNA abundance  are  shown  as  open  red  circles.  Among  them,  8721,009
                                                                                

                                                                                
genes also had A-MYB peaks within 500 bp of their transcription start sites.
These &#8220;A-MYB targets&#8221; are marked with filled red circles. (B) Same as (A) but in
14.5 dpp <span 
class="phvro8t-x-x-120">Miwi </span>mutant versus heterozygous testes. The genes encoding proteins
implicated in RNA silencing pathways that were labeled in (A) and that showed
no change in expression in <span 
class="phvro8t-x-x-120">Miwi  </span>mutant testes are highlighted as green filled
circles. As expected, <span 
class="phvro8t-x-x-120">Miwi</span>, showed a significant decrease in mRNA abundance
in <span 
class="phvro8t-x-x-120">Miwi</span>-/- testes. (C) The change in mRNA abundance (rpkm) in <span 
class="phvro8t-x-x-120">A-Myb  </span>and
<span 
class="phvro8t-x-x-120">Miwi </span>mutant testes versus heterozygous controls for the RNA silencing genes
highlighted in (A) and (B). </span></div><!--tex4ht:label?: x1-56002r14 -->
                                                                                

                                                                                
<!--l. 236--><p class="noindent" ></div><hr class="endfigure">
<a 
 id="x1-56003r110"></a>
<h4 class="subsectionHead"><span class="titlemark">4.2.9 </span> <a 
 id="x1-570009"></a>Feed-Forward Regulation of piRNA Production is Evolutionarily Conserved</h4>
<!--l. 240--><p class="noindent" >Is A-MYB-mediated, feedforward control a general feature of regulation of piRNA
production among vertebrates? To test whether A-MYB control of piRNA precursor
transcription is evolutionarily conserved, we used high-throughput sequencing to
identify piRNAs in adult rooster testes. Birds and mammals diverged 330 million years
ago [<a 
href="#XBenton2007">Benton and Donoghue</a>,&#x00A0;<a 
href="#XBenton2007">2007</a>]. After removing the sequences of identifiable
miRNAs [<a 
href="#XBurnside2008">Burnside et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBurnside2008">2008</a>] and annotated noncoding RNAs, total small
RNA from the adult rooster testis showed peaks at both 23 and 25 nt (Figure
<a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>A). When the RNA was oxidized before being prepared for sequencing,
only a single 25 nt peak remained, consistent with the 25 nt small RNAs
corresponding to piRNAs containing 2<span 
class="cmsy-8">&#x2032;</span>-O-methyl-modified 3<span 
class="cmsy-8">&#x2032;</span>&#x00A0; termini. These
longer, oxidation-resistant species typically began with uracil (62% of species
and 65% of reads; Figure <a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>B), and we detected a significant Ping-Pong
amplification signature (Z score = 31; Figure <a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>C). We conclude that the
oxidation-resistant, 24&#8211;30 nt long small RNAs correspond to rooster piRNAs. Like
piRNAs generally, rooster piRNAs are diverse, with 5,742,529 species present among
81,121,893 genome-mapping reads. Like mouse pachytene piRNAs, 70% of
piRNAs from adult rooster testes mapped to unannotated intergenic regions,
19% mapped to transposons, and 14% mapped to protein-coding genes. Of
the piRNAs that map to protein-coding genes, &#x003E;95% derive from introns.
Forty-two percent of piRNA species mapped uniquely to the Gallus gallus
genome.
<!--l. 242--><p class="noindent" >Using 24&#8211;30 nt piRNAs from oxidized libraries, we identified 327 rooster piRNA
clusters (Figure <a 
href="#x1-57002r16">4.16<!--tex4ht:ref: fig:MolCelS8 --></a>). These account for 76% of all uniquely mapping piRNAs. Of the
327 clusters, 25 overlapped with protein-coding genes. To begin to identify the
transcription start sites for the rooster piRNA clusters, we analyzed adult
rooster testes by H3K4me3 ChIP-seq. More than 81% (268 out of 327) of
the clusters contained a readily detectable H3K4me3 peak within 1 kbp of
the piRNA cluster. In contrast, the median distance from a cluster to the
nearest transcription start site of an annotated gene was 73 kbp, suggesting
that the H3K4me3 peaks reflect the start sites for rooster piRNA precursor
transcripts.
<!--l. 244--><p class="noindent" >Next, we asked where in the genome A-MYB bound in adult rooster testes. A-MYB
ChIP-seq identified 5,509 significant peaks (FDR &#x003C; 10-25). MEME analysis of the top
500 peaks with the lowest FDR values identified a motif (E = 2.6 x 10-201; Figure
<a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>D) similar to that found in the mouse (Figure <a 
href="#x1-53001r7">4.7<!--tex4ht:ref: fig:MolCelF4 --></a>A). A-MYB is the only one of the
                                                                                

                                                                                
three chicken MYB genes expressed in adult testis (X.Z.L. and P.D.Z., unpublished
data), supporting the view that these peaks correspond to A-MYB binding. The
core sequence motif associated with A-MYB binding in mouse differs at one
position (CAGTT) from that in rooster (C C/G GTT). This difference between
mammalian and chicken MYB proteins has been noted previously [<a 
href="#XDeng1996">Deng
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XDeng1996">1996</a>,&#x00A0;<a 
href="#XWeston1992">Weston</a>,&#x00A0;<a 
href="#XWeston1992">1992</a>].
<!--l. 247--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-57001r15"></a>
                                                                                

                                                                                
<!--l. 249--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.542x.png" alt="PIC" class="graphics" width="401.5pt" height="526.96875pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.542x.png" src="Figures/Chapter4/MolCel2013_Fig8.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.15: </span><span  
class="content">(A) Length distributions of total rooster testis small RNAs (black)
and miRNAs  (gray).(B)  Sequence  logo  showing  the  nucleotide  composition
of piRNA reads  and  species.(C)  The  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;-5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  overlap  between  piRNAs  from
opposite strands   was   analyzed   to   determine   if   rooster   piRNAs   display
                                                                                

                                                                                
Ping-Pong amplification. The number of pairs of piRNA reads at each position
is reported. Z score indicates that a significant 10 nt overlap (Ping-Pong) was
detected. Z score &#x003E; 1.96 corresponds to p value &#x003C; 0.05.(D) MEME-reported
motif of the top 500 (by peak score) A-MYB ChIP-seq peaks from adult rooster
testes.(E) A-MYB, H3K4me3, and input ChIP-seq signals at the transcription
start sites of rooster PIWIL1, TDRD1, and TDRD3. See also Figure S8. </span></div><!--tex4ht:label?: x1-57001r15 -->
                                                                                

                                                                                
<!--l. 255--><p class="noindent" ></div><hr class="endfigure">
<!--l. 256--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-57002r16"></a>
                                                                                

                                                                                
<!--l. 258--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.543x.png" alt="PIC" class="graphics" width="407.52249pt" height="459.7175pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.543x.png" src="Figures/Chapter4/MolCel2013_FigS8.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;4.16:  </span><span  
class="content">Black  horizontal  lines  denote  the  locations  on  the  Gallus
gallus (galGal3) chromosomes of the piRNA clusters identified by small RNA
sequencing. The figure shows 324 clusters; clusters on E64 (cluster 370) and
E22C19W28_E50C23 (clusters 109 and 563) are not shown. </span></div><!--tex4ht:label?: x1-57002r16 -->
                                                                                

                                                                                
<!--l. 264--><p class="noindent" ></div><hr class="endfigure">
<!--l. 266--><p class="noindent" >To determine whether chicken A-MYB might regulate transcription of some piRNA
clusters in the testis, we compared the A-MYB peak nearest to each piRNA cluster
with the nearest H3K4me3 peak. Of the 327 rooster piRNA clusters, at least 104
were occupied by A-MYB at their promoters, as defined by an overlapping
H3K4me3 peak. These 104 clusters account for 31% of uniquely mapping rooster
piRNAs.
<!--l. 268--><p class="noindent" >The chicken genome encodes at least two PIWI proteins: PIWIL1 and PIWIL2.
Remarkably, the promoter of Gallus gallus PIWIL1, the homolog of mouse <span 
class="phvro8t-x-x-120">Miwi</span>,
contained a prominent A-MYB peak (Figure <a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E). TDRD1 and TDRD3 also
showed A-MYB peaks (Figure <a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E). Thus, as in mice, Gallus gallus A-MYB
controls the transcription of both piRNA clusters and genes encoding piRNA
pathway proteins. We conclude that A-MYB-mediated feedforward regulation of
piRNA production was likely present in the last common ancestor of birds and
mammals.
<!--l. 270--><p class="noindent" >In mice, we found no piRNA-producing genes on the sex chromosomes (Figure <a 
href="#x1-49002r2">4.2<!--tex4ht:ref: fig:MolCelS1 --></a>A),
perhaps because mouse sex chromosomes are silenced during the pachytene stage
[<a 
href="#XLi2009d">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009d">2009a</a>]. Birds use a ZW rather than an XY mechanism for sex
determination, so roosters are homogametic (ZZ), allowing the sex chromosomes to
remain transcriptionally active in males [<a 
href="#XNamekawa2009">Namekawa and Lee</a>,&#x00A0;<a 
href="#XNamekawa2009">2009</a>,&#x00A0;<a 
href="#XSchoenmakers2009">Schoenmakers
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchoenmakers2009">2009</a>]. Indeed, we find that 39 of the 327 rooster piRNA clusters are on the
Z chromosome, accounting for 12% of uniquely mapping piRNAs (Figure
<a 
href="#x1-57002r16">4.16<!--tex4ht:ref: fig:MolCelS8 --></a>). Of the 39 Z chromosome clusters, 18 had an A-MYB peak at their
promoter.
<a 
 id="x1-57003r90"></a>
<h3 class="sectionHead"><span class="titlemark">4.3 </span> <a 
 id="x1-580003"></a>DISCUSSION</h3>
<!--l. 274--><p class="noindent" >The data presented here provide strong support for the view that piRNAs in mammals
begin as long, single-stranded precursors generated by testis-specific, RNA Pol II
transcription of individual piRNA genes (see also <a 
href="#XVourekas2012">Vourekas et&#x00A0;al.</a>&#x00A0;[<a 
href="#XVourekas2012">2012</a>].
Transcription by RNA Pol II affords piRNA genes the same rich set of transcriptional
controls available to regulate mRNA expression. Our data establish that
developmentally regulated transcription of piRNA genes determines when specific
classes of piRNAs emerge during spermatogenesis.
<!--l. 276--><p class="noindent" >During mouse spermatogenesis, transcription of pachytene piRNA genes begins at
the onset of the pachytene stage of meiosis; pachytene piRNAs accumulate
subsequently. The presence of the MYB binding motif near the transcription start sites
                                                                                

                                                                                
of pachytene piRNA genes, the physical binding of A-MYB to those genes, and
the loss of pachytene piRNA precursor transcripts and piRNAs in testes
from <span 
class="phvro8t-x-x-120">A-Myb </span>mutant mice all argue that A-MYB regulates pachytene piRNA
production.
<!--l. 278--><p class="noindent" >A-MYB also drives increased expression of piRNA pathway genes. Among these,
<span 
class="phvro8t-x-x-120">Miwi </span>expression shows the greatest dependence on A-MYB, but A-MYB also drives
transcription of genes encoding other proteins in the piRNA pathway, including
MitoPld, Mael, and five genes encoding Tudor domain proteins. For example, A-MYB
increases expression of Tdrd6 more than 500-fold. Loss of A-MYB function more
strongly depletes pachytene piRNAs than loss of MIWI, in part because
pachytene piRNAs can still be loaded into MILI in <span 
class="phvro8t-x-x-120">Miwi </span>mutant testes, although
MILI-loaded pachytene piRNAs do not suffice to produce functional sperm. In the
<span 
class="phvro8t-x-x-120">A-Myb </span>mutant, expression of mRNAs encoding multiple piRNA pathway
proteins decreases. We speculate that in wild-type male mice, the increased
expression of these mRNAs at the onset of the pachytene stage of meiosis
ensures that sufficient piRNA-precursor-processing and MIWI-loading factors
are available to cope with the large increase in pachytene piRNA precursor
transcription.
<!--l. 280--><p class="noindent" >We propose that induction of A-MYB during the early pachytene stage of
spermatogenesis initiates a feedforward loop that ensures the precisely timed
production of these piRNAs. Coherent feedforward loops show delayed kinetics in
order to reject background stimuli [<a 
href="#XMangan2003">Mangan and Alon</a>,&#x00A0;<a 
href="#XMangan2003">2003</a>]. Indeed, we observed a
delay from the early to middle pachytene in the accumulation of pachytene piRNAs,
despite the continued increase in <span 
class="phvro8t-x-x-120">A-Myb </span>expression (Figure <a 
href="#x1-52001r3">4.3<!--tex4ht:ref: fig:MolCelF2 --></a>A). Pachytene
piRNA levels increase 75-fold (median for the 100 genes) from 10.5 to 12.5
dpp, coincident with increased expression of <span 
class="phvro8t-x-x-120">A-Myb</span>. However, from 12.5
to 14.5 dpp, pachytene piRNAs increase only 1.2-fold. Pachytene piRNAs
subsequently resume their accumulation, increasing 65-fold from 14.5 to
17.5 dpp. We believe this delay is a consequence of a feedforward loop that
ensures the production of pachytene piRNAs only at the pachytene stage of
spermatogenesis. Regulation by a feedforward loop also predicts a rapid shutdown of
pachytene piRNA pathways at round spermatid stage VIII, when A-MYB
protein levels decrease [<a 
href="#XHorvath2009">Horvath et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHorvath2009">2009</a>]. Supporting this idea, the
abundance of MIWI decreases sharply by the elongated spermatid stage of
spermatogenesis [<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>]. Testing this proposal is a clear challenge for
the future.
<!--l. 282--><p class="noindent" >In fruit flies and zebrafish [<a 
href="#XBrennecke2007">Brennecke et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBrennecke2007">2007</a>,&#x00A0;<a 
href="#XHouwing2007">Houwing et&#x00A0;al.</a>,&#x00A0;<a 
href="#XHouwing2007">2007</a>], most
piRNAs map to repetitive regions, whereas in mammals, uniquely mapping intergenic
piRNAs predominate in the adult testis. The discovery that 70% of rooster piRNA
reads map to intergenic regions suggests that the expansion of intergenic piRNAs
                                                                                

                                                                                
controlled by A-MYB feedforward regulation arose before the divergence of birds and
mammals. In the future, detailed analysis of piRNA production across avian
spermatogenesis should provide insight into the evolutionary origins and
functions of pachytene piRNAs, a class of piRNAs thus far only detected in
mammals.
<!--l. 284--><p class="noindent" >In summary, we have shown that mouse piRNA genes are coregulated transcriptionally,
establishing that A-MYB coordinately regulates the biogenesis of an entire piRNA
class, the pachytene piRNAs. The discovery that a loss-of-function <span 
class="phvro8t-x-x-120">A-Myb </span>mutant,
<span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span> , disrupts piRNA precursor transcription in vertebrates provides a tool to
understand the transformation of long, single-stranded piRNA precursors into
mature piRNAs and to explore the functions and targets of the pachytene
piRNAs.
<a 
 id="x1-58001r116"></a>
<h3 class="sectionHead"><span class="titlemark">4.4 </span> <a 
 id="x1-590004"></a>EXPERIMENTAL PROCEDURES</h3>
<dl class="description"><dt class="description">
<span 
class="phvb8t-x-x-120">Mice</span></dt><dd 
class="description">                                                                         <br 
class="newline" /><span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>,  <span 
class="phvro8t-x-x-120">Spo11</span><sup><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Sky</span></sup>,  and  <span 
class="phvro8t-x-x-120">Piwil1</span><sup><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Hf</span></sup>  mice  were  maintained  and
used according   to   the   guidelines   of   the   Institutional   Animal   Care
and Use   Committee   of   the   University   of   Massachusetts   Medical
School and genotyped as described [<a 
href="#XBaudat2000c">Baudat et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBaudat2000c">2000</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Sequencing</span></dt><dd 
class="description">                                                                 <br 
class="newline" />Small [<a 
href="#XGhildiyal2008">Ghildiyal  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XSeitz2008">Seitz  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSeitz2008">2008</a>]  and  long  RNA-seq
[<a 
href="#XZhang2012a">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2012a">2012b</a>] and analysis [<a 
href="#XLi2009a">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>] were as described.
Reads that did not map to mouse genome mm9 were mapped to piRNA
precursor  transcripts  to  obtain  splice  junction  mapping  small  RNAs.
Total small RNA libraries from different developmental stages and from
mutants were normalized to the sum of all miRNA hairpin mapping reads.
Oxidized samples were calibrated to the corresponding total small RNA
library via  the  abundance  of  shared,  uniquely  mapped  piRNA  species.
piRNA expression data were grouped with Cluster 3.0. Differential gene
expression  was  analyzed  with  DESeq  R  [<a 
href="#XAnders2010a">Anders  and  Huber</a>,&#x00A0;<a 
href="#XAnders2010a">2010</a>];
ChIP-seq reads were aligned to the genome using Bowtie version 0.12.7
[<a 
href="#XLangmead2009">Langmead et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangmead2009">2009</a>], and peaks were identified using MACS [<a 
href="#XZhang2008">Zhang
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Acknowledgments</span> </dt><dd 
class="description">                                                                                            <br 
class="newline" />We thank K. Chase and K. Schimenti for help collecting tissues; C. Tipping
for help with mouse husbandry; P. Johnson and B. Keagle for providing
rooster testes; G. Farley for technical assistance; H. Lin for reagents; Xi
Chen, Xiaotu Ma, Oliver Rando, and Benjamin Carone for advice on ChIP;
and members of our laboratories for critical comments on the manuscript.
X.Z.L. was supported by the Lalor Foundation and the Jane Coffin Childs
Memorial Fund for Medical Research.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Accession Numbers</span> </dt><dd 
class="description">                                                                                         <br 
class="newline" />The Gene   Expression   Omnibus   (GEO)   accession   number   for   the
RNA-seq,  ChIP-seq,  and  small  RNA  data  reported  in  this  paper  is
GSE44690.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Animals</span></dt><dd 
class="description">                                                                      <br 
class="newline" />Mice were  maintained  and  used  according  to  the  guidelines  of  the
Institutional   Animal   Care   and   Use   Committee   of   the   University   of
Massachusetts  Medical  School.  C57BL/6J  (Jackson  Labs,  Bar  Harbor,
ME, USA;  stock  number  664);  <span 
class="phvro8t-x-x-120">Mybl1</span><span 
class="cmmi-8">repro</span><span 
class="cmr-8">9</span>   in  a  mixed  129X1/SvJ  x
C57BL/6J  background;  Spo11tm1Sky  in  a  C57BL/6J  background;  and
Piwil<sup><span 
class="cmr-8">1</span><span 
class="cmmi-8">tm</span><span 
class="cmr-8">1</span><span 
class="cmmi-8">Hf</span></sup>  in  a  mixed  129X1/SvJ  x  C57BL/6J  background  (&#8220;<span 
class="phvro8t-x-x-120">Miwi</span>&#8221;)
mice were  genotyped  as  described  [<a 
href="#XBaudat2000c">Baudat  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBaudat2000c">2000</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">Bolcun-Filas
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBolcun-Filas2011">2011</a>,&#x00A0;<a 
href="#XDeng2002c">Deng and Lin</a>,&#x00A0;<a 
href="#XDeng2002c">2002</a>]. Rooster testes from White Leghorn of
the Cornell Special C strain, about 15 months old, were used for small
RNA analysis; and testes from the Brown Leghorn strain, about one year
old, were used for ChIP analysis.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">RNA Sequencing</span> </dt><dd 
class="description">                                                                                              <br 
class="newline" />Small RNA  libraries  were  constructed  and  sequenced  as  described
[<a 
href="#XGhildiyal2008">Ghildiyal et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGhildiyal2008">2008</a>,&#x00A0;<a 
href="#XSeitz2008">Seitz et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSeitz2008">2008</a>] except that 18&#8211;35 nt RNA was
isolated  and  2S  rRNA  depletion  omitted.  Sequencing  was  performed
using either  a  Genome  Analyzer  GAII  (36  or  76  nt  reads)  or  HiSeq
2000 (50  nt)  instrument  (Illumina,  San  Diego,  CA,  USA).  We  analyzed
published   small   RNA   libraries   from   purified   mouse   spermatogonia
(SRR069809), spermatocytes (SRR069810, GSE39652), or spermatids
(SRR069811;  [<a 
href="#XGan2011">Gan  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGan2011">2011</a>,&#x00A0;<a 
href="#XModzelewski2012">Modzelewski  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XModzelewski2012">2012</a>];  from  <span 
class="phvro8t-x-x-120">Mili</span>
mutant or heterozygous testes at 10 dpp (SRX003089 and SRX003088;
[<a 
href="#XAravin2008a">Aravin and Hannon</a>,&#x00A0;<a 
href="#XAravin2008a">2008</a>]; from Tdrd6 mutant or heterozygous testes
at 18 dpp  (SRX012165  and  SRX012166;  [<a 
href="#XVagin2009">Vagin  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XVagin2009">2009</a>];  and
MILI IP  samples  from  Tdrd9  mutant  or  heterozygous  testes  at  14
dpp (SRX015795,  SRX015796,  SRX015797,  and  SRX015798;  [<a 
href="#XShoji2009">Shoji
                                                                                

                                                                                
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShoji2009">2009</a>].
<!--l. 307--><p class="noindent" >Strand-specific RNA-seq libraries [<a 
href="#XZhang2012">Zhang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2012">2012a</a>] using Ribo-Zero
Gold (Epicentre  Biotechnologies,  Madison,  WI,  USA)  were  sequenced
using the paired-end protocol on a HiSeq 2000.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd 
class="description">                                                                                         <br 
class="newline" />Small RNA sequence analysis was as described [<a 
href="#XLi2009a">Li et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>] using
mouse  genome  release  mm9  and  chicken  genome  release  galGal3.
Non-coding  RNA  annotations  comprised  data  from  ncRNAscan,  the
known tRNAs from UCSC, and 18S, 28S and 5.8S rRNAs. miRNA hairpin
and mature miRNA annotation was from miRBase Release 19. Mouse and
chicken transposons were annotated using Repeat Masker from UCSC.
Reads that  did  not  map  to  the  mouse  genome  (mm9)  were  mapped
to piRNA  precursor  transcripts  to  obtain  splice  junction-mapping  small
RNAs. Total small RNA libraries from different developmental stages and
from mutants were normalized to the sum of all miRNA hairpin-mapping
reads. Oxidized samples were calibrated to the corresponding total small
RNA library  via  the  abundance  of  shared,  uniquely  mapped  piRNA
species. <span class="texthl"><span id="textcolor24">Table S1</span></span> reports the statistics for high-throughput sequencing.
For oxidized  (i.e.,  piRNA-enriched)  samples,  uniquely  mapping  small
RNAs &#x003E;23 nt were mapped to each assembled piRNA precursor transcript
and reported as reads per kilobase pair per million reads mapped to the
genome (rpkm) using a pseudo count of 0.001.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Small RNA Analysis</span> </dt><dd 
class="description">                                                                                         <br 
class="newline" />RNA-seq reads were aligned to the genome (NCBI 37/mm9) using TopHat
2.0.4 [<a 
href="#XTrapnell2009">Trapnell et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2009">2009</a>]. Reads were mapped uniquely using the &#8216;-g
1&#8217; switch.  We  assembled  the  mouse  testes  transcriptome  (see  below).
For genes with multiple isoforms, the transcript with the highest average
rpkm value among the three replicates of adult testes was selected for
further analysis. Fragments with both reads mapped within a transcript,
or to piRNA precursor transcripts, were counted using BEDTools [<a 
href="#XQuinlan2010">Quinlan
and Hall</a>,&#x00A0;<a 
href="#XQuinlan2010">2010</a>].  The  sum  of  the  reads  aligning  to  the  top  quartile  of
expressed transcripts per library was used to calibrate the samples. The
number of reads per transcript was normalized by length, divided by the
library-specific  calibration  factor,  and  reported  as  rpkm  with  a  pseudo
count of  0.001.  Table  S1  presents  the  statistics  for  the  RNA-seq  data.
Sequences mapping to five genes <span class="texthl"><span id="textcolor25">(Table S1)</span></span> that overlapped with or were
embedded within a piRNA gene were excluded when calculating piRNA
precursor transcript abundance.
                                                                                

                                                                                
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">PAS-seq Library Construction and Analysis</span> </dt><dd 
class="description">                                                 <br 
class="newline" />PAS-seq  libraries  <span class="texthl"><span id="textcolor26">(Table  S1)</span></span>  were  prepared  essentially  as  described
[<a 
href="#XShepard2011">Shepard  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShepard2011">2011</a>]  and  sequenced  using  a  HiSeq  2000  (100  nt
read length).  We  first  removed  adaptors  and  performed  quality  control
using Flexbar  2.2  (http://sourceforge.net/projects/theflexibleadap)  with
the parameters  &#8220;-at  3  -ao  10  &#8211;min-readlength  30  &#8211;max-uncalled  70
&#8211;phred-pre-trim 10.&#8221; For reads beginning with GGG including (NGG, NNG
or GNG) and ending with three or more adenosines, we removed the first
three nucleotides and mapped the remaining sequence with and without
the tailing  adenosines  to  the  mouse  genome  using  TopHat  2.0.4.  We
retained only those reads that could be mapped to the genome without the
trailing adenosine  residues.  Genome-mapping  reads  containing  trailing
adenosines were regarded as potentially originating from internal priming
and thus  discarded.  The  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  the  mapped,  retained  read  was
reported as the site of cleavage and polyadenylation.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">CAGE Library Construction and Analysis</span> </dt><dd 
class="description">                                                     <br 
class="newline" />CAGE (cap  analysis  of  gene  expression;  Table  S1)  was  as  described
[<a 
href="#XYang2011">Yang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XYang2011">2011</a>] and sequenced using a HiSeq 2000 (100 nt reads).
After removing  adaptor  sequences  and  checking  read  quality  using
Flexbar  2.2  with  the  parameters  of  &#8220;-at  3  -ao  10  &#8211;min-readlength  20
&#8211;max-uncalled 70 &#8211;phred-pre-trim 10&#8221;, we retained only reads beginning
with NG  or  GG  (the  last  two  nucleotides  on  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  adaptor).  We  then
removed  the  first  two  nucleotides  and  mapped  the  sequences  to  the
mouse genome using TopHat 2.0.4. All unique 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of the mapped
positions were considered as CAGE-tag starting sites and grouped into
tag clusters using a distance-based method in which the maximal distance
between  two  neighboring  tags  was  required  to  be  &#x003C;20  bp.  The  peak
position of a tag cluster was then reported as the transcription start site.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Transcriptome Assembly and Annotation</span> </dt><dd 
class="description">                                                     <br 
class="newline" />De novo  transcriptome  assembly  from  three  biological  replicates  of
strand-specific  RNA-seq  data  from  adult  testes  was  performed  using
Trinity (r2012-06-08) with default parameters [<a 
href="#XGrabherr2011">Grabherr et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrabherr2011">2011</a>]. The
assembled RNA sequences were aligned to the mouse genome (mm9)
with BLAT  [<a 
href="#XKent2002">Kent</a>,&#x00A0;<a 
href="#XKent2002">2002</a>],  and  the  alignments  with  more  than  95%  of
sequence length mapped and fewer than 1% mismatches retained.
<!--l. 324--><p class="noindent" >We                                            extracted                                            novel
junctions from Trinity (i.e., reads with [0-9]+M[0-9]+N[0-9]+M pattern in the
                                                                                

                                                                                
CIGAR string of SAM output), and re-mapped all RNA-seq reads to these
junctions,  rescuing  1,402,444  reads  in  three  replicates.  Rescued  reads
were combined  with  TopHat  alignments  (supplied  with  &#8220;&#8211;max-multihits
100&#8221; to  assembly  through  repetitive  regions)  and  used  as  input  for
reference-based assembly.
<!--l. 326--><p class="noindent" >We used  Cufflinks  v2.0.2  [<a 
href="#XTrapnell2010">Trapnell  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>]  with  parameters  of
&#8220;-u -j 0.2  &#8211;min-frags-per-transfrag  40&#8221;  to  assemble  transcripts.  To  join
small transcript  fragments  caused  by  insufficient  read  coverage  or
embedded repetitive elements, two different gap-joining distance cutoffs
were used  for  the  assembly  of  genes  (&#8220;&#8211;overlap-radius  100&#8221;)  and
piRNA loci (&#8220;&#8211;overlap-radius 250&#8221;). We used Cuffcompare v2.0.2 [<a 
href="#XTrapnell2010">Trapnell
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XTrapnell2010">2010</a>] to annotate the 49,840 Cufflinks-assembled transcripts using
parameters optimized for genic conditions (&#8220;&#8211;overlap-radius 100&#8221;).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">piRNA Precursor Transcript Annotation</span> </dt><dd 
class="description">                                                        <br 
class="newline" />We combined  transcripts  from  the  two  Cufflink  assemblies  with  those
from the Trinity assembly, producing 136,069 unique transcripts. Those
transcripts with 100 ppm or 100 rpkm unique mapping piRNAs at any time
point (10.5, 12.5, 14.5, 17.5, 20.5 dpp and adult oxidized small RNA from
testis) were selected for manual annotation.
<!--l. 331--><p class="noindent" >To refine the termini of the piRNA-producing transcripts, we supplemented
the RNA-seq data with high-throughput sequencing of 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of RNAs
bearing  (5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)ppp(5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;)  cap  structures  (CAGE)  and  of  the  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  ends  of
transcripts  flanking  the  poly(A)  tail  (PAS-seq).  To  provide  independent
confirmation of the 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; ends of each piRNA-producing transcript, we used
chromatin immunoprecipitation (ChIP-seq) of RNA polymerase II (pol II)
and histone  H3  bearing  trimethylated  lysine-4  (H3K4me3).  Refinement
of transcriptional  starts  required  both  a  CAGE  and  a  H3K4me3  peak
to support  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  end  of  the  transcript.  When  no  H3K4me3  peak
corroborated alternative transcription start sites proposed by the CAGE
data, the alternative transcripts were merged with the fully substantiated
transcript.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">piRNA Gene Nomenclature</span> </dt><dd 
class="description">                                                                             <br 
class="newline" />When piRNA-producing genes overlap an annotated protein coding gene,
we refer to them using the name of the overlapping gene preceded by &#8216;pi-&#8216;;
when they do not, their names refer to their genomic location followed by a
number indicating the piRNA abundance in ppm at 6 weeks post-partum.
The last digit of a piRNA gene name specifies the rank order of expression
                                                                                

                                                                                
among  isoforms,  determined  by  the  highest  abundance  of  transcripts
(rpkm) observed  for  that  gene  among  the  six  developmental  stages  of
testis.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Grouping piRNA Precursor Transcripts</span> </dt><dd 
class="description">                                                         <br 
class="newline" />For the  most  abundant  transcript  in  each  locus,  the  abundance  (rpkm)
of piRNAs at each stage was expressed as a fraction of the maximum
abundance reached during the developmental time course. These data
were then  analyzed  by  hierarchical  clustering  according  to  Euclidean
distance and complete linkage using Cluster 3.0. Clustering results were
visualized using Java Tree View 1.1.3.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Analysis of Differential Gene Expression</span>  </dt><dd 
class="description">                                                     <br 
class="newline" />We determined differential gene expression using DESeq R [<a 
href="#XAnders2010a">Anders and
Huber</a>,&#x00A0;<a 
href="#XAnders2010a">2010</a>]. For each annotated mRNA, reads from each library were
aligned to the most abundant assembled transcript. Transcripts with q &#x003C;
0.05 were  considered  to  be  differentially  expressed.  <span class="texthl"><span id="textcolor27">Table  S3</span></span>  lists  the
genes that  were  differentially  expressed  in  <span 
class="phvro8t-x-x-120">A-Myb  </span>at  14.5  dpp.  Three
biologically independent replicates were used for <span 
class="phvro8t-x-x-120">A-Myb</span>homozygotes and
heterozygotes at 14.5 and at 17.5 dpp.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Motif Discovery</span> </dt><dd 
class="description">                                                                                                 <br 
class="newline" />For divergently transcribed piRNA gene pairs, the promoter region was
defined  as  the  region  between  the  transcription  start  sites  defined
by CAGE  peaks.  Sequence  motifs  in  these  putative  promoter  regions
were detected  ab  initio  using  MEME  [<a 
href="#XBailey2009">Bailey  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XBailey2009">2009</a>,&#x00A0;<a 
href="#XBailey1994">Bailey  and
Elkan</a>,&#x00A0;<a 
href="#XBailey1994">1994</a>] in TCM mod (any number of repetitions per sequence) and
compared  to  existing  JASPAR  and  TRANSFAC  libraries  via  TOMTOM
[<a 
href="#XGupta2007">Gupta et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGupta2007">2007</a>].  FIMO  was  used  to  detect  motif  sites  within  the
putative promoters (default p &#x003C; 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">4</span></sup>; [<a 
href="#XGrant2011">Grant et&#x00A0;al.</a>,&#x00A0;<a 
href="#XGrant2011">2011</a>].
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Chromatin Immunoprecipitation (ChIP)</span> </dt><dd 
class="description">                                                         <br 
class="newline" />ChIP was performed as described [<a 
href="#XChen2008">Chen et&#x00A0;al.</a>,&#x00A0;<a 
href="#XChen2008">2008</a>] except that testes
were macerated on ice and then fixed with 1.5% (w/v) formaldehyde for 20
min. Samples were then further crushed using 20 strokes with a &#8216;B&#8217; pestle
in a Dounce homogenizer (Kimble-Chase, Vineland, NJ, USA). Chromatin
was sheared  by  sonication  and  immunoprecipitated  using  anti-A-MYB
(HPA008791;  Sigma,  St.  Louis,  MO,  USA)  or  anti-H3K4me3  (ab8580;
Abcam, Cambridge, MA, USA) antibody; immunoglobulin G (IgG; Sigma,
                                                                                

                                                                                
item 2729)  served  as  a  control.  ChIP-quantitative  PCR  (qPCR)  was
performed  using  the  CFX96  Real-Time  PCR  Detection  System  with
SsoFast EvaGreen Supermix (Bio-Rad, Hercules, CA, USA). Data were
analyzed  using  DART-PCR  [<a 
href="#XPeirson2003">Peirson</a>,&#x00A0;<a 
href="#XPeirson2003">2003</a>].  Relative  ChIP  enrichment
values were normalized to <span 
class="phvro8t-x-x-120">MyoD1</span>, a gene not expressed in testes. <span class="texthl"><span id="textcolor28">Table
S1</span></span> lists ChIP-qPCR primers. ChIP-seq libraries for anti-A-MYB and control
input DNA  were  prepared  following  the  Illumina  ChIP-seq  protocol  and
sequenced on a HiSeq 2000 (50 nt reads).
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">ChIP-seq Analysis</span> </dt><dd 
class="description">                                                                                            <br 
class="newline" />ChIP-seq reads were aligned to the genome using Bowtie version 0.12.7
[<a 
href="#XLangmead2009">Langmead  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangmead2009">2009</a>].  Reads  were  mapped  uniquely  using  the  &#8216;-M
1 &#8211;best  &#8211;strata&#8217;  switches  and  one  mismatch  was  allowed  (-v  1).  ChIP
peaks were  identified  using  MACS  version  1.4.1  [<a 
href="#XZhang2008">Zhang  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XZhang2008">2008</a>]
using default arguments, input as control, and a cutoff p-value = 10<sup><span 
class="cmsy-8">-</span><span 
class="cmr-8">25</span></sup>
was used. BEDTools was used to assign peaks to the nearest 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0; end of
genes. <span class="texthl"><span id="textcolor29">Table S1</span></span> reports sequencing statistics for ChIP-seq.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">RT-PCR</span></dt><dd 
class="description">                                                                      <br 
class="newline" />Total RNA was treated with Turbo DNase (Ambion, Austin, TX, USA), and
then reverse transcribed using SuperScript III (Invitrogen, Eugene, OR,
USA) with random primers (Promega, Madison, WI, USA). The resulting
cDNA was analyzed by conventional PCR. <span class="texthl"><span id="textcolor30">Table S1</span></span> lists the primers used
in Figure <a 
href="#x1-55002r12">4.12<!--tex4ht:ref: fig:MolCelS6 --></a>.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Ping-Pong Analysis</span> </dt><dd 
class="description">                                                                                          <br 
class="newline" />Ping-Pong  amplification  was  analyzed  by  the  5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;&#8211;5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  overlap  between
piRNA pairs  from  opposite  genomic  strands  [<a 
href="#XLi2009a">Li  et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLi2009a">2009b</a>].  Overlap
scores for  each  overlapping  pair  were  the  product  of  the  number  of
reads of  each  of  the  piRNAs  from  opposite  strands.  The  overall  score
for each  overlap  extend  (1&#8211;30)  was  the  sum  of  all  such  products  for
all chromosomes.  Heterogeneity  at  the  3<span 
class="cmsy-8">&#x2032;</span>&#x00A0;  ends  of  small  RNAs  was
neglected. Z-score for 10 bp overlap was calculated using the scores of
overlaps from 1&#8211;9 and 11&#8211;30 as background.
</dd><dt class="description">
<span 
class="phvb8t-x-x-120">Rooster piRNA Cluster Detection</span> </dt><dd 
class="description">                                                                   <br 
class="newline" />We developed a dynamic programming algorithm to identify the genomic
regions  with  the  highest  piRNA  density.  We  used  oxidized  small  RNA
reads (&#x003E;23 nt) to detect clusters. We used the conservative assumption
                                                                                

                                                                                
that piRNA  clusters  compose  at  most  2%  of  the  chicken  genome.  We
first split the genome into 1 kbp non-overlapping windows and computed
piRNA abundance for each window. The mean of the top 2% of windows
was used as the penalty score for the dynamic programming algorithm.
The algorithm  computes  the  cumulative  piRNA  abundance  score  as  a
function  of  the  window  index  along  each  chromosome.  The  score  at  a
window is the sum of the score in the previous window and the piRNA
abundance in the current window, minus the penalty score; if the resulting
score was  negative  it  was  reset  to  0.  The  maximal  score  points  to  the
largest piRNA cluster. We extracted the largest piRNA cluster, recomputed
the scores  at  the  corresponding  windows,  and  searched  for  the  next
cluster.  The  process  continued  until  the  scores  for  all  windows  were
zero. The  boundaries  of  each  cluster  were  further  refined  by  including
those base pairs for which piRNA abundance exceeded the mean piRNA
abundance  of  the  top  2%  windows.  We  considered  only  those  clusters
with abundance &#x003E;10 ppm for uniquely mapping piRNAs. In Figure <a 
href="#x1-57001r15">4.15<!--tex4ht:ref: fig:MolCelF8 --></a>E,
gene models were corrected using data from our unpublished adult rooster
testis RNA-seq data.
</dd></dl>
                                                                                

                                                                                
<h2 class="chapterHead"><span class="titlemark">Chapter&#x00A0;5</span><br /><a 
 id="x1-600005"></a>Discussion</h2>
<a 
 id="x1-60001r117"></a>
<h3 class="sectionHead"><span class="titlemark">5.1 </span> <a 
 id="x1-610001"></a>The Future of Dynamic long RNAs</h3>
<!--l. 7--><p class="noindent" >Deep sequencing of transcriptomes has revolutionized biology. Previously, transcript
discovery was a cumbersome task. Transcript identification and characterization
involved significant labor, cost, and materials. In the mid-90&#8217;s, microarray technology
[<a 
href="#XSchena1995a">Schena et&#x00A0;al.</a>,&#x00A0;<a 
href="#XSchena1995a">1995</a>] gave us a tantalizing glimpse into how genes were
expressed, but were limited to probed, and therefore known, sequences. Yet,
the green and red landscapes of a microarray analysis hinted at incredible
complexity &#8212;a complexity that would have to wait for technology to catch
up.
<!--l. 9--><p class="noindent" >Like many transformative technologies, RNA-seq was made possible by incremental
improvements to numerous supportive technologies such as: 1) digital optics, 2)
microscopy, 3) slide chemistry and on-slide PCR and 4) nucleic-acid alignment. A
HiSeq 2500 relies on all of these technologies (and others) to produce the 100M+
sequences that allow Scientists to peer every day into the transcriptional output of a
genome.
<!--l. 11--><p class="noindent" >In the past 5 years, biologists have started to think way beyond mRNAs and small
RNAs. The former captured out interest for 30+ years [<a 
href="#XFuruichi1975">Furuichi</a>,&#x00A0;<a 
href="#XFuruichi1975">1975</a>,&#x00A0;<a 
href="#XWei1975">Wei
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWei1975">1975</a>], and the later has been on a run-away trail since capturing out attention
in 1998 [<a 
href="#XFire1998">Fire et&#x00A0;al.</a>,&#x00A0;<a 
href="#XFire1998">1998</a>]. HTS has added long RNAs (among others) to these
classes of gene RNA products. However, many biologically-trained and minded
Scientists find themselves overwhelmed by the complete different methods and
approaches to tackling the &#8220;big data&#8221; created by modern genome-wide experiments.
Experimental training does not currently provide students with the required skills in
statistics, computer programing, and experimental design that are needed to work
with genome-wide data. The richness of this data often leaves many unasked (and
unanswered) test-able hypothesis just sitting in public repositories [<a 
href="#XPlocik2013">Plocik and
Graveley</a>,&#x00A0;<a 
href="#XPlocik2013">2013</a>].
<!--l. 13--><p class="noindent" >Being such a novel area of extremely basic research, and to borrow a few seemingly
inane but rather insightful trio of phrases from the United States Secretary of Defense
Donald Rumsfeld [<a 
href="#XRumsfeld2011">Rumsfeld</a>,&#x00A0;<a 
href="#XRumsfeld2011">2011</a>], their remain at least three important areas of
knowledge concerning long RNAs of the transcriptome: <a 
href="#x1-620001">&#8221;The Known Knowns&#8221;</a>; <a 
href="#x1-690002">&#8221;The
Known Unknowns&#8221;</a>; and the <a 
href="#x1-730003">&#8221;The Unknown Unknowns&#8221;</a>.
<a 
 id="x1-61001r113"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.1 </span> <a 
 id="x1-620001"></a>The Known Knowns</h4>
<a 
 id="x1-62001r53"></a>
                                                                                

                                                                                
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.1 </span> <a 
 id="x1-630001"></a>Alternative Splicing turns gene expression into a game of cards</h5>
<!--l. 24--><p class="noindent" >A deck of cards has only 52 cards. Yet these cards can be delt into XXX
permutations of 7 cards. It is these numbers that makes possible to play Poker for
hours on end, and the likelihood of obtaining that &#8220;Royal&#8221; flush so incredibly
low. Long ago, genes discovered how they could arrange themselves into
unique and rare combinations, especially in eukaryotic organisms. Indeed
the process of splicing is closely correlated with organism complexity. The
process of AS is even more closely tied to organism complexity (see figure
<a 
href="#x1-13001r4">1.4<!--tex4ht:ref: fig:numGenesAndNumSpliced --></a>).
<!--l. 28--><p class="noindent" >At this point, it is important to remember that in this document <span 
class="phvro8t-x-x-120">long RNAs </span>may also
refer to products containing characteristics of traditional mRNAs, that is a 5<span 
class="cmsy-8">&#x2032;</span>&#x00A0;m7G
Cap, ligated exons, and a Poly(A) tail. However, many of these long mRNAs are
extremely dynamic. So much so that until HTS and RNA-Seq, comprehensive
investigation of their complexity was not possible.
<a 
 id="x1-63001r121"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.2 </span> <a 
 id="x1-640002"></a>Pervasive transcription</h5>
<!--l. 31--><p class="noindent" >Here you can put some information from ENCODE and your thoughts on
it.
<a 
 id="x1-64001r122"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.3 </span> <a 
 id="x1-650003"></a>Tissue and cell specificity</h5>
<!--l. 34--><p class="noindent" >Your feelings on Specificity of long RNA expression
<a 
 id="x1-65001r123"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.4 </span> <a 
 id="x1-660004"></a>Function of long RNAs</h5>
<!--l. 37--><p class="noindent" >We <span 
class="phvro8t-x-x-120">know </span>that some long RNAs are functional. What are these?
<a 
 id="x1-66001r124"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.5 </span> <a 
 id="x1-670005"></a>Chromatin regulation</h5>
<!--l. 40--><p class="noindent" >We also know that some long RNAs regulation Chromatin structure. What are your
feelings as to the importance of this fact?
<a 
 id="x1-67001r125"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.1.6 </span> <a 
 id="x1-680006"></a>PTGS</h5>
<!--l. 43--><p class="noindent" >Long RNAs ability to do post transcriptional gene regulation, including piRNAs, and
Xist, etc....
                                                                                

                                                                                
<a 
 id="x1-68001r120"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.2 </span> <a 
 id="x1-690002"></a>The Known Unknowns</h4>
<a 
 id="x1-69001r126"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.2.1 </span> <a 
 id="x1-700001"></a>How are they important?</h5>
<!--l. 48--><p class="noindent" >Conservation of these things is not obvious - if they are not conserved - are they
important? Maybe talk about how MALAT1 is highly expressed, but seems to be
dispensable.
<a 
 id="x1-70001r128"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.2.2 </span> <a 
 id="x1-710002"></a>What regulates their tissue-specific expression?</h5>
<!--l. 51--><p class="noindent" >Do they important some of the special sauce that makes tissues different from one
other, more so then the mRNAs changes which can be extreme, but not terribly
so....
<!--l. 53--><p class="noindent" >What determines AS splicing decisions? It is not connectivity in splicing, and it
seems to be SR and hnRNP proteins. Splicing is tissue specific. So is it the
tissue-specific expression of SR and hnRNP proteins the is the main determinant of
AS outcomes? How does chromatin state and organization play into AS
decisions?
<a 
 id="x1-71001r129"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.1.2.3 </span> <a 
 id="x1-720003"></a>Technological Improvements</h5>
<!--l. 56--><p class="noindent" >How does one perform deep and broad analysis of mRNAs using second-generation
HTS give the tremendous log-range over which they are expressed?
<a 
 id="x1-72001r127"></a>
<h4 class="subsectionHead"><span class="titlemark">5.1.3 </span> <a 
 id="x1-730003"></a>The Unknown Unknowns</h4>
<!--l. 60--><p class="noindent" >This is the area of knowledge keeps many motivated to perform basic research every
day. What secrets does the transcriptome have in store that we haven&#8217;t even <span 
class="phvro8t-x-x-120">thought</span>
about? Only through pushing the boundaries of the last two sections can we begin to
think beyond the edge of map and formulate testable hypothesis. Here I propose a
few outlandish ideas for Unknown Unknowns.
<a 
 id="x1-73001r119"></a>
<h3 class="sectionHead"><span class="titlemark">5.2 </span> <a 
 id="x1-740002"></a>Future SeqZip development and use</h3>
<a 
 id="x1-74001r131"></a>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">5.2.1 </span> <a 
 id="x1-750001"></a>Assay Modifications</h4>
<a 
 id="x1-75001r130"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.1 </span> <a 
 id="x1-760001"></a>Rnl2 T29A mutation</h5>
<!--l. 68--><p class="noindent" >Use of T39A mutation to alleviate penultimate 2<span 
class="cmsy-8">&#x2032;</span>&#x00A0; OH requirement of T4 Rnl2 (See
Nandakumar...Lima, Cell 2006)
<a 
 id="x1-76001r134"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.2 </span> <a 
 id="x1-770002"></a>Thermostable Ligases</h5>
<!--l. 72--><p class="noindent" >Use of thermostable ligase, allowing for multiple rounds of ligation. Need a good
reference, DO NOT USE Ref 27 from Conze et al 2009! Also Elevated ligation
temperatures, minimizing blut-ended NTL events
<a 
 id="x1-77001r135"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.3 </span> <a 
 id="x1-780003"></a>LNA-containing Ligamers</h5>
<!--l. 76--><p class="noindent" >Make a note into the future directions that you would like to explore LNA&#8217;s at the
3 extprime&#x00A0;OH position of all ligation results, leading to increased ligation
efficiency
<a 
 id="x1-78001r136"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.4 </span> <a 
 id="x1-790004"></a>Ribose-containing Ligamers</h5>
<!--l. 80--><p class="noindent" >however both this and the use of penultimate 2 extprime OH (Ribosome) suger in
your ligamers would lead to added costs, and the latter maybe better served with a
T39A mutation. Giggity
<a 
 id="x1-79001r137"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.5 </span> <a 
 id="x1-800005"></a>Repurposing the SOLiD Platform</h5>
<a 
 id="x1-80001r138"></a>
<h5 class="subsubsectionHead"><span class="titlemark">5.2.1.6 </span> <a 
 id="x1-810006"></a>Others</h5>
<ul class="itemize1">
<li class="itemize">Digital PCR of the PCR products ala [<a 
href="#XShiroguchi2012a">Shiroguchi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShiroguchi2012a">2012</a>].
</li>
<li class="itemize">SeqZip on the SOLiD platform
</li>
<li class="itemize">SeqZip on single-cell RNA samples.</li></ul>
<a 
 id="x1-81001r133"></a>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">5.2.2 </span> <a 
 id="x1-820002"></a>An ideal SeqZip experiment to query coordinated splicing</h4>
<!--l. 94--><p class="noindent" >If I could go back 4 years and still possess the knowledge and abilities that I do now, I
would have approached a genome-wide (or multiplex) study of coordination in splicing
using SeqZip much differently. Firstly, I would have focused on alternative first
exon (or promoter, or TSS) use being coordinated with downstream cassette
exon inclusion. I would have mined newly-generated RNA-Seq data [<a 
href="#XPan2008">Pan
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPan2008">2008</a>,&#x00A0;<a 
href="#XWang2008">Wang et&#x00A0;al.</a>,&#x00A0;<a 
href="#XWang2008">2008</a>] for alternative first exons and cassette exons of
sufficient expression levels. I would have used my automated ligamer designing
software (see Appendix <a 
href="#x1-105000C">C<!--tex4ht:ref: AppendixC --></a>), to create a database of the required ligamers to maintain
connectivity and observe connectivity between alternative first exons and a
single downstream cassette exon. This would require at least 3 ligamers per
event, with very little duplicated use of ligamers. To overcome what would
be a significant burden and cost of synthesizing such a large number of
ligamers, I would have pursued having them printed on a custom microarray,
and cleaved into a bulk solution, similar to products offered by (<span 
class="cmmi-12">NUCBIO</span><span 
class="cmr-12">?</span>).
These ligamers would be barcoded and priming sequences included such that
short (50nt) paired-end reads could reliably identify the templating first and
cassette exons. Using this pool of ligamers, I would have performed the SeqZip
assay, also including a barcoding scheme to quantify the number of ligation
events per alt first exon::cassette exon pair, and amplified the library via
digital PCR allowing me to check for PCR jackpots [<a 
href="#XShiroguchi2012a">Shiroguchi et&#x00A0;al.</a>,&#x00A0;<a 
href="#XShiroguchi2012a">2012</a>].
The library would be have been analyzed using a paired-end 50 nt read
apporach on a GEIIx Illumina machine. The data would be aligned against a
reference of all alt first exon::cassette exon pairs, and any potential coordination
determined.
<!--l. 96--><p class="noindent" >If I could have done the experiment designed above, I feel the full potential and utility
of the SeqZip method could have been realized and generated new and valuable
knowledge for the field of gene expression.
<a 
 id="x1-82001r140"></a>
<h4 class="subsectionHead"><span class="titlemark">5.2.3 </span> <a 
 id="x1-830003"></a>SeqZip and single-molecule FISH</h4>
<!--l. 101--><p class="noindent" >The cellular location of precursor piRNA transcript processing is not known. The most
accepted hypothesis is that precursor transcripts are processed into mature piRNAs
with machinery tethered to chromatoid bodies (<span 
class="cmmi-12">REF</span>) or another structure similar to
Drosophila Nuage (<span 
class="cmmi-12">REF</span>) near the mitochondrial cement (<span 
class="cmmi-12">REF</span>). Knowledge of
extitwhere mature piRNAs are generated would provide clues into larger mechanist
details of their biogenesis.
<!--l. 103--><p class="noindent" >I can think of two broadly different ways in which we could pinpoint the physical
location of mature piRNA generation. The first is to chemically label, in some manner,
                                                                                

                                                                                
primary piRNA transcripts. The label would need to 1) not interfere with processing
and 2) be durable to later methods used to analyze the presence or absence of the
modification. A second way to identify the location of mature piRNA processing
would be to actually observe, through in vivo <span 
class="phvb8t-x-x-120">??</span><a 
href="\protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 202 undefined on input line 103.
}\endgroup \let \endcsname \relax \protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 202 undefined on input line 103.
}\endgroup \let ::)Q\endcsname \relax #x1-3001r\protect \protect \begingroup \def \MessageBreak {
(caption)                }\let \protect \immediate\write \@unused {
Package caption Warning: \label without proper \caption on input line 6.
See the caption package documentation for explanation.
}\endgroup \edef hd:abrevs{hd:abrevs}\protect \gdef \protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup {\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: There were undefined references.
}\endgroup }\protect \unhbox \voidb@x \hbox {\a:mbox {\protect \protect \edef T1{T1}\let \enc@update \relax \protect \edef phv{phv}\protect \edef m{m}\protect \edef n{n}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update \ignorespaces \relax \protect \relax \protect \edef m{bx}\protect \xdef \T1/phv/bx/n/12 {\T1/phv/m/n/12 }\T1/phv/bx/n/12 \size@update \enc@update ??}\b:mbox }\protect \begingroup \def \MessageBreak {
               }\let \protect \immediate\write \@unused {
LaTeX Warning: Reference `hd:abrevs' on page 202 undefined on input line 103.
}\endgroup \let ">FISH</a>, the various products
and by-products of biogenesis. In this particular approach, SeqZip maybe
useful.
<!--l. 106--><p class="noindent" >Another idea here would be to use the MCP x MBS mouse as published by the Singer
lab [<a 
href="#XPark2014">Park et&#x00A0;al.</a>,&#x00A0;<a 
href="#XPark2014">2014</a>]. In this paper they insert multiple MS2 binding loops into the 3
extprime&#x00A0;-UTR of the <span 
class="cmmi-12">beta</span>-actin gene, and cross it with another mouse that has MCP
(MS2 Bacteriophage capsid protein) fused to GFP. Using this system they can
visualize extitendogenous mRNAs in cultured MEF cells, and brain sections. How
could we apply this to imaging of piRNA precursor transcripts? If we inserted an MS2
loop into the tail end of transcripts, what would we see? It is certainly worth
it!
<a 
 id="x1-83001r132"></a>
<h3 class="sectionHead"><span class="titlemark">5.3 </span> <a 
 id="x1-840003"></a>In the haystack: piRNA precursor transcripts surrounded by RNA</h3>
<a 
 id="x1-84001r141"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.1 </span> <a 
 id="x1-850001"></a>In vivo chemical labeling of precursor piRNA transcripts</h4>
<!--l. 114--><p class="noindent" >SeqZip accurately reports on the presence of multiple, distant sequences contained
in the same RNA molecule (see Chapters <a 
href="#x1-290002">2<!--tex4ht:ref: Chapter2 --></a> and <a 
href="#x1-370003">3<!--tex4ht:ref: Chapter3 --></a>). Yet I was not able to observe
ligation products templated off some of the most highly-expressed precursor
transcripts (see section <a 
href="#x1-360004">2.4<!--tex4ht:ref: sec: precursor TX --></a>. We hypothesize that the most likely explanation for being
unable to observe ligation products for these transcripts, but successfully observing
those for a very long, but lowly expressed gene extitDst, suggests that precursor
transcripts are rapidly processed into mature piRNAs. What if this is not the case if we
could hybridize ligamers to precursor transcripts in vivo? Put another way extemdash
What if the steady-state amount of precursor transcript is very low due to rapid
processing, but when visualized in real time, the transcripts are of sufficient
abundance for FISH? Ligamers could be engineered to contain different combinations
of fluorescent dyes, and the proximity and intensity of the signal could be used to
infer precursor transcripts. Importantly, this approach could distinguish long,
continuous precursor transcripts from processing intermediates and mature
piRNAs.
<a 
 id="x1-85001r143"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.2 </span> <a 
 id="x1-860002"></a>In vivo imaging of precursor piRNA transcripts</h4>
<!--l. 118--><p class="noindent" >The most important question for mammalian extitpachytene piRNAs is extitWhat are
they doing?. We know that they are essential for the health of the species, as
                                                                                

                                                                                
discussed in section <a 
href="#x1-470001">4.1<!--tex4ht:ref: c4-intro --></a>, and piRNA-pathway mutants are sterile. What could these
small RNAs, with complementarity to nothing but themselves, be doing? Building on
the last section discussing chemical modification precursor transcripts, if the
modification was again durable enough for downstream analysis, perhaps the
modification would remain in the mature piRNAs that are incorporated into mature
sperm. These modifications could be tracked as the sperm move through the
<span class="texthl"><span id="textcolor31">semineferous tubues</span></span> and into the <span 
class="cmmi-12">SPERMANATOMY </span>. One could even track the
piRNAs as they fuse with the oocyte to create an embryo. If this modification was
labile, piRNA interacting proteins or nucleic acids could be captured or marked as
well, providing additional clues as to the function of piRNAs in sperm and early
embryogenesis.
<a 
 id="x1-86001r144"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.3 </span> <a 
 id="x1-870003"></a>What are they doing?</h4>
<a 
 id="x1-87001r145"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.4 </span> <a 
 id="x1-880004"></a>How are they generated?</h4>
<!--l. 134--><p class="noindent" >What determines that a seemingly mRNA-like piRNA precursor is processed
into piRNAs, and not translated like crazy by Ribosomes? Everything meets
the ribosome (<span 
class="cmmi-12">REF</span>), so why is it that precursors are given a different lot in
life?
<a 
 id="x1-88001r146"></a>
<h4 class="subsectionHead"><span class="titlemark">5.3.5 </span> <a 
 id="x1-890005"></a>Why should we care?</h4>
<a 
 id="x1-89001r142"></a>
<h3 class="sectionHead"><span class="titlemark">5.4 </span> <a 
 id="x1-900004"></a>Lingering Questions for <span 
class="phvro8t-x-x-120">Dscam1</span></h3>
<!--l. 141--><p class="noindent" >What controls the stochastic and probabilistic splicing of <span 
class="phvro8t-x-x-120">Dscam1</span>? Why is it different
between hemocytes and neurons? Why would hemocytes need less apparent
diversity, given the range of antigen they could potentially encounter?
<a 
 id="x1-90001r148"></a>
<h3 class="sectionHead"><span class="titlemark">5.5 </span> <a 
 id="x1-910005"></a>Final thoughts</h3>
<a 
 id="x1-91001r147"></a>
<h4 class="subsectionHead"><span class="titlemark">5.5.1 </span> <a 
 id="x1-920001"></a>Why did I want to study AS?</h4>
<a 
 id="x1-92001r150"></a>
                                                                                

                                                                                
<h4 class="subsectionHead"><span class="titlemark">5.5.2 </span> <a 
 id="x1-930002"></a>Biologists need Computation Biological Skills</h4>
<!--l. 153--><p class="noindent" >Just 10 years ago, Graduate students and PhDs in the fields of Molecular Biology or
Biochemistry need not venture far from data analysis within Excel or perhaps a
statistical program with an advanced graphical interface (examples include Prism or
Graphpad). Software knowledge that stops at these tools and the rest of the Microsoft
Office suite of tools is no longer enough to generate big strides in Biomedical
research.
<!--l. 155--><p class="noindent" >Working with tens of even hundreds of lines of data within a spreadsheet is
manageable and computers from 20 years ago had more then enough computing
power to process the data. Yet, this type of data is longer then endpoint of most
cutting edge projects. Many students and post docs often find that they are unable to
analyze the data generated from months or years of tireless bench work. Faced with
learning what is effectively a collection of new languages and awash in a sea of
acronyms (LINUX, BASH, GNU, PERL, R) they reach out for help from a
&#8220;Bioinformatics person.&#8221; Perhaps the relationship and interaction with this
personal is productive, leading to a collaboration and exciting new knowledge.
Sometimes it isn&#8217;t, and the bench scientist shifts into one of three modes: 1)
Wait; 2) Find another bioinformatic-minded collaborator; or 3) collect more
data.
<!--l. 157--><p class="noindent" >In my experience, the most often chosen mode is &#8220;wait.&#8221; This is also the most
damaging, as it delays the progress of one&#8217;s work, and the advancement of science in
general. Personally, I did not want to fall into this mode, and once the multiplex study
described in section <a 
href="#x1-310002">2.2<!--tex4ht:ref: sec: Multiplex Gene Study --></a> reached a point where I had millions of sequencing reads,
but I could not find anyone to help me analyze the data, that I decided to educate
myself on the basic principles of Linux, the command line, and analysis of HTS
data.
<!--l. 159--><p class="noindent" >A Biologically-train individual who posses the knowledge of analysis of HTS datasets
is an extremely powerful and empowering situation. This was recently commucated in
<a 
href="#XPlocik2013">Plocik and Graveley</a>&#x00A0;[<a 
href="#XPlocik2013">2013</a>]:
<div class="quote">
<!--l. 163--><p class="noindent" ><span 
class="phvro8t-x-x-120">Such exercises  will  empower  students  to  explore  and  assess  the</span>
<span 
class="phvro8t-x-x-120">quantitative data published in the manuscripts that they read, which</span>
<span 
class="phvro8t-x-x-120">can no longer be assessed at a glance like the qualitative gel-based</span>
<span 
class="phvro8t-x-x-120">results on which molecular biology was founded. Ultimately, it will</span>
<span 
class="phvro8t-x-x-120">be equally important to know how to write code as it is to pipette. -</span>
<span 
class="phvro8t-x-x-120">[</span><a 
href="#XPlocik2013"><span 
class="phvro8t-x-x-120">Plocik and Graveley</span></a><span 
class="phvro8t-x-x-120">,</span><span 
class="phvro8t-x-x-120">&#x00A0;</span><a 
href="#XPlocik2013"><span 
class="phvro8t-x-x-120">2013</span></a><span 
class="phvro8t-x-x-120">]</span>
                                                                                

                                                                                
</div>
<!--l. 167--><p class="noindent" >The fact is that no one will care about a project as much as the Graduate student or
PostDoc who is the main project driver. Learning and training of computational skills
bent on analyzing large datasets should be central to the education in Biomedical
sciences in the future.
<a 
 id="x1-93001r151"></a>
<h4 class="subsectionHead"><span class="titlemark">5.5.3 </span> <a 
 id="x1-940003"></a>Science versus Engineering: Two thoughts in one school</h4>
<div class="quote">
<!--l. 174--><p class="noindent" ><span 
class="phvro8t-x-x-120">&#8220;There is  a  general  attitude  among  the  scientific  community  that</span>
<span 
class="phvro8t-x-x-120">science is superior to engineering.&#8221; - [</span><a 
href="#XMacilwain2010"><span 
class="phvro8t-x-x-120">Macilwain</span></a><span 
class="phvro8t-x-x-120">,</span><span 
class="phvro8t-x-x-120">&#x00A0;</span><a 
href="#XMacilwain2010"><span 
class="phvro8t-x-x-120">2010</span></a><span 
class="phvro8t-x-x-120">]</span>
<!--l. 176--><p class="noindent" ><span 
class="phvro8t-x-x-120">&#8220;Science  is  about  what  is;  engineering  is  about  what  can  be.</span>
<span 
class="phvro8t-x-x-120">Engineers  are  dedicated  to  solving  problems  and  creating  new,</span>
<span 
class="phvro8t-x-x-120">useful, and efficient things.&#8220; - Niel Armstrong</span></div>
<!--l. 179--><p class="noindent" >A common schism between technically-oriented individuals is whether or not they
identify themselves as an engineer or a scientist. The first quote, from an article
published in Nature, communicates a clear bias in academic circles of the importance
of the extitwhy over the extithow. In essence, how one priorities these questions may
categories individuals as a scientists (why is important) or an engineer (how is more
important). The second quote, from the first man to walk on the Moon, Neil
Armstrong, highlights what motivates a self described &#8221;engineer&#8221; and &#8221;geek.&#8221; How
does a single graduate system, training PhDs for careers in life science,
educate individuals who fall into these two fundamentally different belief
systems?
<!--l. 181--><p class="noindent" >In short extemdash not well. When searching for a lab to call home, I told
professors that I wanted to work on a technology development project. A
typical response was, &#8220;That&#8217;s not what we do here.&#8221; As someone who is
first interested in the &#8220;how&#8221; over the &#8220;why,&#8221; this began a brief period when I
thought I had made the wrong choice in leaving industry to go back to graduate
school. What was the basis for this aversion to technology development? The
same article in Nature states that this feeling toward engineering may be
attributed:
<div class="quote">
                                                                                

                                                                                
<!--l. 186--><p class="noindent" ><span 
class="phvro8t-x-x-120">...partly to a &#8220;linear&#8221; model of innovation, which holds that scientific</span>
<span 
class="phvro8t-x-x-120">discovery  leads  to  technology,  which  in  turn  leads  to  human</span>
<span 
class="phvro8t-x-x-120">betterment.  This  model  is  as  firmly  entrenched  in  policy-makers&#8217;</span>
<span 
class="phvro8t-x-x-120">minds as  it  is  intellectually  discredited.  As  any  engineer  will  tell</span>
<span 
class="phvro8t-x-x-120">you, innovations, such as aviation and the steam engine, commonly</span>
<span 
class="phvro8t-x-x-120">precede scientific understanding of how things work.</span></div>
<!--l. 189--><p class="noindent" >If policy-makers value basic discovery over technological application, perhaps this
explains why many of my professors tried to steer me away from a technology
development project.
<!--l. 191--><p class="noindent" >In spite of policy-makers and my professors holding the viewpoint that discovery
precedes technology, some of the most notable breakthrough scientific discoveries,
including many made by Nobel Laureates, demonstrate a clear integration of both the
scientific method and technological application. For example, the 2007 award in
Physiology and Medicine was given for &#8220;discoveries of principles for introducing
specific gene modifications in mice by the use of embryonic stem cells." By combining
these principle discoveries, an indispensable technique in modern genetics was
created &#8211; gene targeting. The feeling of which is more important, the principle
discoveries or the application thereof, is likely what separates a scientist from an
engineer.
<!--l. 193--><p class="noindent" >The importance of technology to the advancement of science in general is not limited
to anecdotes resulting in a Nobel prize. A quick scan of the most <a 
href="http://www.pnas.org/reports/most-cited" >highly-cited</a> papers
in the journal PNAS reveals that the top 13, indeed extitall 13, are about a
novel methodology or technique. Sequencing of DNA, microarray analysis,
tetracycline-ineducable promoters, recombinant adenovirus, and site-specific
mutagenesis are just a handful of the tools on this list. This effect can be seen in
<a 
href="http://simplystatistics.org/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/" >computation biology</a>, with transformative techniques, such as BLAT [<a 
href="#XAltschul1990">Altschul
et&#x00A0;al.</a>,&#x00A0;<a 
href="#XAltschul1990">1990</a>] and Bowtie [<a 
href="#XLangmead2009">Langmead et&#x00A0;al.</a>,&#x00A0;<a 
href="#XLangmead2009">2009</a>] attaining citations well beyond a
typical paper in their journal of publication and far more than most primary
research-centered articles.
<!--l. 195--><p class="noindent" >How does someone who is motivated by the engineering of science best contribute in
an academic setting? Luckily, I did not have to question my decision to return to
school for long. I found a pair of labs where I could learn how to practice traditional
hypothesis-driven research while also developing new tools necessary to do so. My
project is a perfect fit for me and has been terrific fun to work on. In the past five
years, the technique that I developed, SeqZip, allows for the more efficient study of
mRNA isoforms produced from alternative splicing of pre-mRNA. This is a very
engineering-type accomplishment. However, the technique uses short DNA
oligonucleotides and a novel activity (that I discovered) of an known RNA ligase to
shorten and simplify isoform sequence information. Use of Rnl2 to perform
                                                                                

                                                                                
RNA-templated DNA:DNA ligation is completely new knowledge, and falls squarely
within a scientific purview. The technique simplifies many of the experimental issues
that researchers struggle with when studying the often complex products of
alternative splicing.
<a 
 id="x1-94001r152"></a>
<h4 class="subsectionHead"><span class="titlemark">5.5.4 </span> <a 
 id="x1-950004"></a>Dealing with the data deluge</h4>
<!--l. 200--><p class="noindent" >How do we work with all this data? Mention LabKey, GenomeBridge, Define the
scope of the problem, etc...
<!--l. 202--><p class="noindent" >I would love to have a database of complete transcripts of a cell - think about how you
were panning the data for the MolCel paper, and you could see ChIP marks, Pol II
occupancy, RNA-Seq, and piRNA data. It would be great to be able to do that more
often, and we greater precision
                                                                                

                                                                                
<a 
 id="x1-95001r118"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;A</span><br /><a 
 id="x1-96000A"></a>Appendix - Misc Information</h2>
<a 
 id="x1-96001r149"></a>
<h3 class="sectionHead"><span class="titlemark">A.1 </span> <a 
 id="x1-970001"></a>Buffers</h3>
<div class="table">
                                                                                

                                                                                
<!--l. 14--><p class="noindent" ><a 
 id="x1-97001r1"></a><hr class="float"><div class="float" 
>
                                                                                

                                                                                
<div class="tabular"><table id="TBL-10" class="tabular" 
cellspacing="0" cellpadding="0" rules="groups" 
><colgroup id="TBL-10-1g"><col 
id="TBL-10-1"></colgroup><colgroup id="TBL-10-2g"><col 
id="TBL-10-2"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-10-1-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-1"  
class="td11">Component</td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-1-2"  
class="td11">Concentration</td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-2-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-1"  
class="td11">  Tris-HCl   </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-2-2"  
class="td11">   50 mM      </td></tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-3-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-1"  
class="td
11">  MgCl2     </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-3-2"  
class="td11">    2 mM       </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-4-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-1"  
class="td11"> DTT      </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-4-2"  
class="td11">    1 mM       </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-5-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-1"  
class="td11"> ATP       </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-5-2"  
class="td11">   400 <span 
class="cmmi-12">&#x03BC;</span>M     </td>
</tr><tr 
class="hline"><td><hr></td><td><hr></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-10-6-"><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-1"  
class="td11"> pH       </td><td  style="white-space:nowrap; text-align:center;" id="TBL-10-6-2"  
class="td11"> 7.5 @ 25<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C </td>
</tr></table></div>
<br /><div class="caption" 
><span class="id">Table&#x00A0;A.1: </span><span  
class="content">SeqZip Hybridization and Ligation Buffer </span></div><!--tex4ht:label?: x1-97001r1 -->
                                                                                

                                                                                
</div><hr class="endfloat" />
</div>
<a 
 id="x1-97002r155"></a>
<h3 class="sectionHead"><span class="titlemark">A.2 </span> <a 
 id="x1-980002"></a>Equations</h3>
<a 
 id="x1-98001r153"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.1 </span> <a 
 id="x1-990001"></a>f</h4>
<!--l. 40--><p class="noindent" >rom <sup><span 
class="cmr-8">32</span> </sup> P-<span 
class="cmmi-12">&#x03B1;</span>-UTP used during vitro transcription]Determining [RNA] from <sup><span 
class="cmr-8">32</span></sup>P-<span 
class="cmmi-12">&#x03B1;</span>-UTP
used during vitro transcription
<!--l. 43--><p class="noindent" ><div class="eqnarray">
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.544x.png" alt="()(                  )  (               ) (              )
&#x03BC;M=pmol&#x03BC;L=cpcpmmabefteforrepupriurficifiatcaiotionn&#x00D7;&#x00D7;ddiluiltioutnionfafctacortor &#x00D7; molUTRPeaincotioringiVonalulrmeaection &#x00D7;  NumberUTP1sintranscript &#x00D7;10-12
" class="math-display" ></center>
</div>
<a 
 id="x1-99002r158"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.2 </span> <a 
 id="x1-1000002"></a>b</h4>
<!--l. 60--><p class="noindent" >ased on A<sub><span 
class="cmr-8">260</span> </sub> ]Determining [RNA] based on A<sub><span 
class="cmr-8">260</span></sub>
<!--l. 63--><p class="noindent" >
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.545x.png" alt="         (                                               )
           ------------A260-&#x00D7;-Dilution-Factor-------------
[RNAin M ] =  10, 313 &#x003C; note 1 &#x003E; &#x00D7;  nucleotides in message " class="math-display" ></center>
note 1: This value represents an average RNA extinction (<span 
class="cmmi-12">&#x03F5;</span>) coefficient value
<br 
class="newline" />
<a 
 id="x1-100001r159"></a>
<h4 class="subsectionHead"><span class="titlemark">A.2.3 </span> <a 
 id="x1-1010003"></a>Normalize oxidized small RNA libraries size to time-matched unoxidized
library</h4>
<!--l. 71--><p class="noindent" >NB: this equation assumes calibration against a specific time-point , in this case data
obtained from 6 week-old testes.
<!--l. 74--><p class="noindent" ><div class="eqnarray">
<center class="math-display" >
<img 
src="RoyC_Umass_Thesis.v0.546x.png" alt="       (  (  &#x2211;                    )               )
            &#x2211;----miRNA---reads--&#x03C4;--
       ||       miRNA   reads  6wk   &#x00D7;  depth 6wk  ||
unox&#x03C4;norm   =  ||  ----------------------------------------||
 1     |                 1,000,000                |
       (                                          )

                        (  &#x2211;                                   )
                               oxidized shared  &#x2265;  23 nt reads
ox&#x03C4;norm1  =  unox  &#x03C4; norm1 &#x00D7;  ( &#x2211;------------------------------------)
                              unoxidized shared  &#x2265;  23 nt reads
" class="math-display" ></center>
</div>
<a 
 id="x1-101002r157"></a>
<h3 class="sectionHead"><span class="titlemark">A.3 </span> <a 
 id="x1-1020003"></a>PCR Programs</h3>
<!--l. 87--><p class="noindent" ><span 
class="phvb8t-x-x-120">Ligamer Hybridization </span>ROY-H2 | Ligamer Hybridization<br 
class="newline" />Steps 1&#8211;9 are 10 minute incubations at the following temperatures:<br 
class="newline" />69;66;63;58;54;52;50;48;46<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C<br 
class="newline" />Step 10 is a 45<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C incubation for 1 hour<br 
class="newline" />Steps 11&#8211;14 are 10 minute incubations at the following templates:<br 
class="newline" />43;41;39;37<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C<br 
class="newline" />Final incubation is at 37<span 
class="cmsy-8">&#x2218;</span>&#x00A0;C for <span 
class="cmsy-10x-x-120">&#x221E;</span><br 
class="newline" />
<!--l. 96--><p class="noindent" ><span 
class="phvb8t-x-x-120">SeqZip ligation program </span>ROY-37-4 | T4 Rnl2 RNA-template DNA:DNA
ligation<br 
class="newline" />1. 37<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C for 18 hours<br 
class="newline" />2. 10<span 
class="cmsy-8">&#x2218;</span> &#x00A0;C for <span 
class="cmsy-10x-x-120">&#x221E;</span><br 
class="newline" />
                                                                                

                                                                                
<a 
 id="x1-102001r118"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;B</span><br /><a 
 id="x1-103000B"></a>Appendix B: piRNA precurors are spliced</h2>
<a 
 id="x1-103001r161"></a>
<h3 class="sectionHead"><span class="titlemark">B.1 </span> <a 
 id="x1-1040001"></a>Evidence for spliced piRNA precusor transcripts</h3>
<!--l. 7--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-104001r1"></a>
                                                                                

                                                                                
<!--l. 9--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.547x.png" alt="PIC" class="graphics" width="415.55249pt" height="224.84pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.547x.png" src="Figures/AppendixB/evidenceForPrecursorSplicing.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.1: </span><span  
class="content">RNA-Seq evidence for piRNA precursor splicing]<br 
class="newline" /><span class="texthl"><span id="textcolor32">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-104001r1 -->
                                                                                

                                                                                
<!--l. 16--><p class="noindent" ></div><hr class="endfigure">
<!--l. 20--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-104002r2"></a>
                                                                                

                                                                                
<!--l. 22--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.548x.png" alt="PIC" class="graphics" width="403.50749pt" height="367.3725pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.548x.png" src="Figures/AppendixB/noPiRNAswithinPrecusorIntrons.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.2: </span><span  
class="content">Lack of piRNAs within precusor introns<br 
class="newline" /><span class="texthl"><span id="textcolor33">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-104002r2 -->
                                                                                

                                                                                
<!--l. 29--><p class="noindent" ></div><hr class="endfigure">
<!--l. 33--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-104003r3"></a>
                                                                                

                                                                                
<!--l. 35--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.549x.png" alt="PIC" class="graphics" width="415.55249pt" height="258.9675pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.549x.png" src="Figures/AppendixB/smallRNAsMapToPrecursorSJ.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.3: </span><span  
class="content">piRNAs map to Splice Junctions of precursor transcripts<br 
class="newline" /><span class="texthl"><span id="textcolor34">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-104003r3 -->
                                                                                

                                                                                
<!--l. 42--><p class="noindent" ></div><hr class="endfigure">
<!--l. 46--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-104004r4"></a>
                                                                                

                                                                                
<!--l. 48--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.550x.png" alt="PIC" class="graphics" width="415.55249pt" height="188.70499pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.550x.png" src="Figures/AppendixB/aggregatePiRNAsatSpliceJunctions.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.4: </span><span  
class="content"><span 
class="phvro8t-x-x-120">A-Myb </span>Mutants produce no splice-junction mapping piRNAs<br 
class="newline" /><span class="texthl"><span id="textcolor35">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-104004r4 -->
                                                                                

                                                                                
<!--l. 55--><p class="noindent" ></div><hr class="endfigure">
<!--l. 61--><p class="noindent" ><hr class="figure"><div class="figure" 
>
                                                                                

                                                                                
<a 
 id="x1-104005r5"></a>
                                                                                

                                                                                
<!--l. 63--><p class="noindent" ><img 
src="RoyC_Umass_Thesis.v0.551x.png" alt="PIC" class="graphics" width="415.55249pt" height="221.82874pt" ><!--tex4ht:graphics  
name="RoyC_Umass_Thesis.v0.551x.png" src="Figures/AppendixB/piRNAPrecusorTXFeatures.eps"  
-->
<br /><div class="caption" 
><span class="id">Figure&#x00A0;B.5: </span><span  
class="content">General features of piRNA transcripts<br 
class="newline" /><span class="texthl"><span id="textcolor36">Insert Figure Text</span></span> </span></div><!--tex4ht:label?: x1-104005r5 -->
                                                                                

                                                                                
<!--l. 70--><p class="noindent" ></div><hr class="endfigure">
                                                                                

                                                                                
<a 
 id="x1-104006r118"></a>
<h2 class="appendixHead"><span class="titlemark">Appendix&#x00A0;C</span><br /><a 
 id="x1-105000C"></a>Automated Ligamer Assembly</h2>
<a 
 id="x1-105001r163"></a>
<h3 class="sectionHead"><span class="titlemark">C.1 </span> <a 
 id="x1-1060001"></a>Installation</h3>
<!--l. 6--><p class="noindent" >Major Steps:
<ul class="itemize1">
<li class="itemize">Create an input csv file with required information
</li>
<li class="itemize">Run this information sequentially through the scripts
</li>
<li class="itemize">Use the results to order oligos from IDT</li></ul>
<!--l. 13--><p class="noindent" >Required Tools:
<ul class="itemize1">
<li class="itemize">Perl
</li>
<li class="itemize">BioPerl
</li>
<li class="itemize">Ensembl Perl APIs
</li>
<li class="itemize">String::Random Perl Package</li></ul>
<!--l. 21--><p class="noindent" >Items to future improvements
<ul class="itemize1">
<li class="itemize">Use Ensembl Database to initilize queries
</li>
<li class="itemize">Make the use of BioPerl more flexible
</li>
<li class="itemize">Make more web-friedly</li></ul>
<!--l. 28--><p class="noindent" >Helpful hints on installing BioPerl and Emsembl Perl APIs:
<!--l. 31-->
<div class="lstlisting" id="listing-1"><span class="label"><a 
 id="x1-106001r1"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Install</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BioPerl</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">git</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106002r2"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cpan</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">App</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">cpanminus</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">First</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prep</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cpan</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106003r3"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cpanm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">DBI</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Install</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">necessary</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">DBI</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">perl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">module</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106004r4"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">mkdir</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cd</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106005r5"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">git</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">clone</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">git</span><span 
class="ectt-1095">://</span><span 
class="ectt-1095">github</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">com</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">bioperl</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">bioperl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">live</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">git</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106006r6"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/.</span><span 
class="ectt-1095">bash_profile</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/.</span><span 
class="ectt-1095">bash_profile</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">bak</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106007r7"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">echo</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">e</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$HOME</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">bioperl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">live</span><span 
class="ectt-1095">:</span><span 
class="ectt-1095">$PERL5LIB</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x003E;&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/.</span><span 
class="ectt-1095">bash_profile</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106008r8"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">source</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/.</span><span 
class="ectt-1095">bash_profile</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106009r9"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Install</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">perl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">apis</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106010r10"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">mkdir</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cd</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106011r11"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">wget</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ftp</span><span 
class="ectt-1095">://</span><span 
class="ectt-1095">ftp</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">org</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">pub</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">api</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">tar</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">gz</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106012r12"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">tar</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">xvfz</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">api</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">tar</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">gz</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106013r13"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Add</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">locations</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">perlfile</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">libs</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$PATH</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106014r14"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">echo</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">e</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106015r15"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">}:</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">HOME</span><span 
class="ectt-1095">}/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">modules</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106016r16"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">}:</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">HOME</span><span 
class="ectt-1095">}/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">compara</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">modules</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106017r17"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">}:</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">HOME</span><span 
class="ectt-1095">}/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">variation</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">modules</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106018r18"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">}:</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">HOME</span><span 
class="ectt-1095">}/</span><span 
class="ectt-1095">src</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">functgenomics</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">modules</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-106019r19"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">export</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PERL5LIB</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x003E;&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">~/.</span><span 
class="ectt-1095">bash_profile</span>

                                                                                

                                                                                
</div>
<a 
 id="x1-106020r170"></a>
<h3 class="sectionHead"><span class="titlemark">C.2    </span> <a 
 id="x1-1070002"></a>Example Input Format</h3>
                                                                                

                                                                                
                                                                                <!--l. 60--><p class="noindent" >Here is an example input file to create ligamers investigating the <span 
class="phvro8t-x-x-120">Gria3 </span>gene in Rats: <div class="table">
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <!--l. 1--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                                                                                                                      
                                                                                

                                                                                
                                                                                <div class="tabular"> <table id="TBL-11" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-11-1g"><col 
id="TBL-11-1"><col 
id="TBL-11-2"><col 
id="TBL-11-3"><col 
id="TBL-11-4"><col 
id="TBL-11-5"><col 
id="TBL-11-6"><col 
id="TBL-11-7"><col 
id="TBL-11-8"><col 
id="TBL-11-9"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-11-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-1-1"  
class="td11"><span 
class="phvr8t-"># Comment lines are ignored                                     </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-2-1"  
class="td11"><span 
class="phvr8t-">#Gene name                                                               </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-3-1"  
class="td
11"><span 
class="phvr8t-">&#x00A0;GRIA3                                                                       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-4-1"  
class="td
11"><span 
class="phvr8t-"># PCR primers used - Solexa PE adaptor sequences</span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-5-1"  
class="td11"><span 
class="phvr8t-"># Five prime                                                                </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-11-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-6-1"  
class="td11"><span 
class="phvr8t-">PCR-Primer-5&#8217;-ATCTGAGCGGGCTGGCAAGGC      </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-7-1"  
class="td
11"><span 
class="phvr8t-">#Three Prime                                                              </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-8-1"  
class="td
11"><span 
class="phvr8t-">PCR-Primer-3&#8217;-GCCTCCCTCGCGCCATCAGA         </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-11-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-11-9-1"  
class="td11"></td></tr></table></div>
                                                                                                                   
                                                                                

                                                                                
                                                                                </div><hr class="endfloat" />
</div>
<div class="table">
                                                                                                                   
                                                                                

                                                                                
                                                                                <!--l. 14--><p class="noindent" ><hr class="float"><div class="float" 
>
                                                                                                                   
                                                                                

                                                                                
                                                                                <div class="tabular"> <table id="TBL-12" class="tabular" 
cellspacing="0" cellpadding="0"  
><colgroup id="TBL-12-1g"><col 
id="TBL-12-1"><col 
id="TBL-12-2"><col 
id="TBL-12-3"><col 
id="TBL-12-4"><col 
id="TBL-12-5"><col 
id="TBL-12-6"><col 
id="TBL-12-7"><col 
id="TBL-12-8"><col 
id="TBL-12-9"></colgroup><tr  
 style="vertical-align:baseline;" id="TBL-12-1-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-1-1"  
class="td11"><span 
class="phvr8t-">ExonId                                   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-2"  
class="td11"><span 
class="phvr8t-">LigID</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-3"  
class="td11"><span 
class="phvr8t-">Name</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-4"  
class="td11"><span 
class="phvr8t-">Strand</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-5"  
class="td11"><span 
class="phvr8t-">Code</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-6"  
class="td11"><span 
class="phvr8t-">TargetPrime</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-7"  
class="td11">        <span 
class="phvr8t-">bedLoc              </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-8"  
class="td11">   <span 
class="phvr8t-">SetID      </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-1-9"  
class="td11">  <span 
class="phvr8t-">ConstID    </span></td>
</tr><tr  
 style="vertical-align:baseline;" id="TBL-12-2-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-2-1"  
class="td11"><span 
class="phvr8t-">&#x003C;Gria3_201/202-Shared-I14  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-2"  
class="td11"> <span 
class="phvr8t-">10  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-5"  
class="td11"> <span 
class="phvr8t-">TC  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-6"  
class="td11">     <span 
class="phvr8t-">5         </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-7"  
class="td11"><span 
class="phvr8t-">X:127903250-127903350</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-8"  
class="td11"><span 
class="phvr8t-">NANNNNNN</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-2-9"  
class="td11"><span 
class="phvr8t-">201_2_intron</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-3-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-3-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201/202-Shared-I14</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-2"  
class="td11">  <span 
class="phvr8t-">9   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-5"  
class="td11">  <span 
class="phvr8t-">T   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-6"  
class="td11">     <span 
class="phvr8t-">3         </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-7"  
class="td11"><span 
class="phvr8t-">X:127903210-127903249</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-8"  
class="td11"><span 
class="phvr8t-">NNNNNNNN</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-3-9"  
class="td11"><span 
class="phvr8t-">201_2_intron</span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-4-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-4-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201/202-E15           </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-2"  
class="td11">  <span 
class="phvr8t-">8   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-5"  
class="td11"> <span 
class="phvr8t-">TC  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-6"  
class="td11">     <span 
class="phvr8t-">5         </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-7"  
class="td11"><span 
class="phvr8t-">X:127914822-127915069</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-8"  
class="td11"> <span 
class="phvr8t-">NNNNNNN </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-4-9"  
class="td11">  <span 
class="phvr8t-">201,202    </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-5-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-5-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_202-I14:15               </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-2"  
class="td11">  <span 
class="phvr8t-">7   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-7"  
class="td11"><span 
class="phvr8t-">X:127912345-127914821</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-8"  
class="td11"> <span 
class="phvr8t-">TACACAT   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-5-9"  
class="td11">    <span 
class="phvr8t-">202       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-6-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-6-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_202-E14                  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-2"  
class="td11">  <span 
class="phvr8t-">6   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-7"  
class="td11"><span 
class="phvr8t-">X:127912230-127912344</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-8"  
class="td11"> <span 
class="phvr8t-">ACCCCAG  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-6-9"  
class="td11">    <span 
class="phvr8t-">202       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-7-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-7-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201-I14:15               </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-2"  
class="td11">  <span 
class="phvr8t-">5   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-7"  
class="td11"><span 
class="phvr8t-">X:127897499-127914821</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-8"  
class="td11"> <span 
class="phvr8t-">CGCGCAC </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-7-9"  
class="td11">    <span 
class="phvr8t-">201       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-8-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-8-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201-E14                  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-2"  
class="td11">  <span 
class="phvr8t-">4   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-7"  
class="td11"><span 
class="phvr8t-">X:127897384-127897498</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-8"  
class="td11"> <span 
class="phvr8t-">GTCTCAA  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-8-9"  
class="td11">    <span 
class="phvr8t-">201       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-9-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-9-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_202-I13:14               </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-2"  
class="td11">  <span 
class="phvr8t-">3   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-7"  
class="td11"><span 
class="phvr8t-">X:127896828-127912229</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-8"  
class="td11"> <span 
class="phvr8t-">ACCGATT  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-9-9"  
class="td11">    <span 
class="phvr8t-">202       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-10-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-10-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201-I13:14               </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-2"  
class="td11">  <span 
class="phvr8t-">2   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-5"  
class="td11">  <span 
class="phvr8t-">I    </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-6"  
class="td11">    <span 
class="phvr8t-">N        </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-7"  
class="td11"><span 
class="phvr8t-">X:127896828-127897383</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-8"  
class="td11"> <span 
class="phvr8t-">CGCTATG  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-10-9"  
class="td11">    <span 
class="phvr8t-">201       </span></td></tr><tr  
 style="vertical-align:baseline;" id="TBL-12-11-"><td  style="white-space:nowrap; text-align:left;" id="TBL-12-11-1"  
class="td
11"><span 
class="phvr8t-">&#x003C;Gria33_201/202-E13           </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-2"  
class="td11">  <span 
class="phvr8t-">1   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-3"  
class="td11"> <span 
class="phvr8t-">rn4  </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-4"  
class="td11"> <span 
class="phvr8t-">minus </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-5"  
class="td11">  <span 
class="phvr8t-">T   </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-6"  
class="td11">     <span 
class="phvr8t-">3         </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-7"  
class="td11"><span 
class="phvr8t-">X:127896580-127896827</span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-8"  
class="td11"> <span 
class="phvr8t-">NNNNNNN </span></td><td  style="white-space:nowrap; text-align:center;" id="TBL-12-11-9"  
class="td11">  <span 
class="phvr8t-">201,202    </span></td></tr></table></div>
                                                                                                                                                                                                                          
                                                                                

                                                                                
                                                                                </div><hr class="endfloat" />
</div>
                                                                                                                                                                                                                          
                                                                                

                                                                                
<a 
 id="x1-107001r171"></a>
<h3 class="sectionHead"><span class="titlemark">C.3 </span> <a 
 id="x1-1080003"></a>Ligamer Assmebly Source Code</h3>
<!--l. 71-->
<div class="lstlisting" id="listing-2"><span class="label"><a 
 id="x1-108001r1"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">!</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">usr</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">bin</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">perl</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108002r2"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108003r3"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Pre</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">requisites</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108004r4"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">These</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">are</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">on</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">02/19/13</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108005r5"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">lib</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">home</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">royc</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">perl5</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">lib</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">perl5</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BioPerl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">location</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108006r6"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">lib</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"/</span><span 
class="ectt-1095">home</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">royc</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">lib</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">perl</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">zpi</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">modules</span><span 
class="ectt-1095">";</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">packages</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108007r7"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108008r8"></a></span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">head1</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Assembler</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108009r9"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108010r10"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">This</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">script</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">will</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">automatically</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">create</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamers</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108011r11"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108012r12"></a></span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">head2</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Contact</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108013r13"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108014r14"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Script</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">made</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">by</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Christian</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Roy</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Umass</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Medical</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">School</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108015r15"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">christian</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">roy@umassmed</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">edu</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108016r16"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108017r17"></a></span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">cut</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108018r18"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108019r19"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strict</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">To</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">help</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">wtih</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108020r20"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">warnings</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">To</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">help</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">me</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">catch</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">mistakes</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108021r21"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108022r22"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">EnsEMBL</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Registry</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">To</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">load</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">remote</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">EnsEMBL</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Registry</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108023r23"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">EnsEMBL</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Slice</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">To</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">retreave</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequences</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">EnsEMBL</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">registry</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108024r24"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">DB</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Fasta</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BioPerl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">tool</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">retreave</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequnce</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">local</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">FastA</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108025r25"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">SeqFeature</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BioPerl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tool</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">normalization</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108026r26"></a></span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Cwd</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">To</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">retreave</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">current</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">directory</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108027r27"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108028r28"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$dir</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">getcwd</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Assign</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">current</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">directory</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">scalar</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108029r29"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$timestamp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">localtime</span><span 
class="ectt-1095">()</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">time</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">at</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">script</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">start</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108030r30"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108031r31"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Variables</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108032r32"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108033r33"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$file_input</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">specified</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108034r34"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$output_file</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">too</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108035r35"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">species</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ensembl</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108036r36"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequences</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108037r37"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108038r38"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line_counter</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Keep</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">track</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">stepping</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">through</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108039r39"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@arguments</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Keep</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">track</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">arguments</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108040r40"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$fa_reference</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">using</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">a</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">local</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">FASTA</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Reference</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108041r41"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Obvious</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108042r42"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$coordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Interim</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">splitting</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">UCSC</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108043r43"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obvious</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108044r44"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Obvious</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108045r45"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">target</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">gene</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">name</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108046r46"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108047r47"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Broad</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">define</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">type</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">see</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">man</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108048r48"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Obvious</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108049r49"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">appropriate</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">terminal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">oligos</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108050r50"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">will</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">between</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">regions</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">comp</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108051r51"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$note_line</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">notes</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">a</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">query</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108052r52"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">three</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108053r53"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fill</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">five</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108054r54"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Internal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">varialbe</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">assembling</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamers</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">see</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">man</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108055r55"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Move</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">set</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">assembly</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108056r56"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108057r57"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108058r58"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Defaults</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108059r59"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$verbose</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Verbose</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">databases</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108060r60"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">=62;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Default</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">database</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">database</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108061r61"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">58</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Defalt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">normalization</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108062r62"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">0.05</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Default</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">concetration</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">calculation</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">M</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108063r63"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">0.00000025</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Defeult</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">conc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">calc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">M</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108064r64"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$man_print</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printing</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">manual</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108065r65"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$help_print</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">For</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printing</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">help</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">informatio</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HTML</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108066r66"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Internal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variable</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequental</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">numbering</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamers</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108067r67"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$remote</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">set</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">database</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108068r68"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">=20;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Default</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">nt</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108069r69"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$plname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$0</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">assign</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$plname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">scalar</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">script</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">help</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printing</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108070r70"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108071r71"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Usage</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">nothing</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">is</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">entered</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">at</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">commandline</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108072r72"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">==0)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">system</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">pod2text</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$0</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">|</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">less</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">die</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108073r73"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108074r74"></a></span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">head2</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Usage</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108075r75"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108076r76"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108077r77"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">hp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HTML</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">scriptname</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108078r78"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">mp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">and</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">view</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Manual</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">scriptname</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108079r79"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">i</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">File</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">File</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Input</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108080r80"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">o</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">File</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">File</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108081r81"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">v</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Verbose</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108082r82"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">d</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data_base</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108083r83"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">degrees</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">celcius</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108084r84"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">concentration</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">mM</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108085r85"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">]</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">concentration</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">nM</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108086r86"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">c</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Minimum</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Control</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamers</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">default</span><span 
class="ectt-1095">=20)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108087r87"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108088r88"></a></span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">cut</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108089r89"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Finish</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">message</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">run</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">no</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">arguments</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108090r90"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108091r91"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">command</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108092r92"></a></span><span 
class="ectt-1095">while</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">&#x003E;0)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108093r93"></a></span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108094r94"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@arguments</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Store</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">command</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printing</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">later</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108095r95"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108096r96"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108097r97"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108098r98"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">hp</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HTML</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Data</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108099r99"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$help_print</span><span 
class="ectt-1095">=1;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108100r100"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108101r101"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">mp</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">a</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">manual</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108102r102"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$man_print</span><span 
class="ectt-1095">=1;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108103r103"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108104r104"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">i</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">What</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">is</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108105r105"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$file_input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108106r106"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108107r107"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">f</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">fasta</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">your</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequences</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">are</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108108r108"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$fa_reference</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108109r109"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108110r110"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">r</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">fetch</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequences</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108111r111"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$remote</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108112r112"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108113r113"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">o</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108114r114"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$output_file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108115r115"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108116r116"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">v</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">see</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">load</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108117r117"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$verbose</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108118r118"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108119r119"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">d</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">What</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">use</span><span 
class="ectt-1095">?#</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108120r120"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108121r121"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108122r122"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">What</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">temperature</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">degrees</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">want</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">norm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108123r123"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108124r124"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108125r125"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">concentration</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">calculations</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108126r126"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108127r127"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1000;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">micro</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Molar</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Molar</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108128r128"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108129r129"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">lig_conc</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Concentration</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">calculations</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108130r130"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift@ARGV</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108131r131"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1000000000;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">nM</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">M</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108132r132"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108133r133"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$next_arg</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">c</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">What</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">would</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">you</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">like</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">min</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligs</span><span 
class="ectt-1095">?</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108134r134"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@ARGV</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108135r135"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">-1</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108136r136"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108137r137"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Finish</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Parsing</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">command</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108138r138"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108139r139"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####################</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HELP</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">SUBROUTINE</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">CALLS</span><span 
class="ectt-1095">################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108140r140"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$0</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108141r141"></a></span><span 
class="ectt-1095">podhelp</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$help_print</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$man_print</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$dir</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108142r142"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">####################</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HTML</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">CALLS</span><span 
class="ectt-1095">#################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108143r143"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108144r144"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###################</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">open</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">registry</span><span 
class="ectt-1095">#################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108145r145"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108146r146"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$fa_reference</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108147r147"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">DB</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Fasta</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">new</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$fa_reference</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108148r148"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108149r149"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108150r150"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$remote</span><span 
class="ectt-1095">==1)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108151r151"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl_database</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$verbose</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108152r152"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108153r153"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108154r154"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">open</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108155r155"></a></span><span 
class="ectt-1095">open</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">$output_file</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">||</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">die</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">could</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">not</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">be</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">created</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108156r156"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">headers</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108157r157"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108158r158"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Start</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">general</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">assembler</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108159r159"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Source</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Program</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$dir</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$0</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108160r160"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Date</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Run</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$timestamp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108161r161"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Arguments</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">entered</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">@arguments</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108162r162"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">filename</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$file_input</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108163r163"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">filename</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$output_file</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108164r164"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Control</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$control_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">plus</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108165r165"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Normalization</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">temperature</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$temp</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108166r166"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">####</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">now</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">all</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">on</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligmaer</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">specific</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">information</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108167r167"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">Gene</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">1</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108168r168"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligamer_Number</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">2</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108169r169"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Species</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108170r170"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Strand</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">4</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108171r171"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Joiner</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Code</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108172r172"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Target</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">6</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108173r173"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">UCSC</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">coordinates</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">7</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108174r174"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Used</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">8</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108175r175"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Used</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">9</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108176r176"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Total</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Query</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">span</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">10</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108177r177"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Five</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Sequence</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">11</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108178r178"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Length</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">12</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108179r179"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Five</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108180r180"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Sequence</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108181r181"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Length</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108182r182"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108183r183"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Identifier</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">".</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108184r184"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Sequence</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108185r185"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Length</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108186r186"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Notes</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108187r187"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Set</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108188r188"></a></span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108189r189"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108190r190"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">open</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108191r191"></a></span><span 
class="ectt-1095">open</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">INPUT</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$file_input</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">||</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">die</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$file_input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">couldn</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">be</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">opened</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108192r192"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">##############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108193r193"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108194r194"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">################</span><span 
class="ectt-1095">read</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">and</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">analyze</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">each</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108195r195"></a></span><span 
class="ectt-1095">while</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=&#x003C;</span><span 
class="ectt-1095">INPUT</span><span 
class="ectt-1095">&#x003E;)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">starting</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">brace</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">read</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">through</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">csv</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108196r196"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108197r197"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">/)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">skips</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">comments</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108198r198"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^&#x003E;/)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">skips</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">and</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">trans</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">these</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">lines</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108199r199"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^~/)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/~//;</span><span 
class="ectt-1095">chomp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">find</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">gene</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">identifier</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108200r200"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/[\</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">]+//</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108201r201"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^\</span><span 
class="ectt-1095">@</span><span 
class="ectt-1095">/)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">chomp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">$note_line</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">store</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">notes</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108202r202"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108203r203"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">chomp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108204r204"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108205r205"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">-5</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">-/</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Find</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">adaptor</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108206r206"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108207r207"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">-5</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">-//;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108208r208"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/[\</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">]+//</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108209r209"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;5</span><span 
class="ectt-1095">_pcr</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108210r210"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108211r211"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108212r212"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108213r213"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">-3</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">-/)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{##</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">find</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">adaptor</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108214r214"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108215r215"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">PCR</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">-3</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">-//;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108216r216"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/[\</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">]+//</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108217r217"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x003E;3</span><span 
class="ectt-1095">_pcr</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108218r218"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108219r219"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108220r220"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108221r221"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">=~/^&#x003C;/)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">query</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">lines</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">start</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">a</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;&#x003C;&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108222r222"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">unless</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$note_line</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$note_line</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108223r223"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108224r224"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108225r225"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108226r226"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108227r227"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line_counter</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108228r228"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108229r229"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108230r230"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108231r231"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108232r232"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108233r233"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108234r234"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108235r235"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108236r236"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108237r237"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108238r238"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse_the_line</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108239r239"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108240r240"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">uc</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108241r241"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108242r242"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">query</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108243r243"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse_coordinates</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108244r244"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108245r245"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108246r246"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108247r247"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108248r248"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###########</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">registry</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108249r249"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$remote</span><span 
class="ectt-1095">==1)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108250r250"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108251r251"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get_genomic_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108252r252"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108253r253"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108254r254"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108255r255"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108256r256"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108257r257"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108258r258"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108259r259"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108260r260"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">####################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108261r261"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108262r262"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###########</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">local</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Fasta</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108263r263"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$fa_reference</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108264r264"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">="</span><span 
class="ectt-1095">chr</span><span 
class="ectt-1095">".</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108265r265"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$obj</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get_Seq_by_id</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108266r266"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$obj</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">subseq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108267r267"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108268r268"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108269r269"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108270r270"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">correct</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">orientation</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108271r271"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108272r272"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">revcom_slice_based_on_strand</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108273r273"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108274r274"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108275r275"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108276r276"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108277r277"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108278r278"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">end</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108279r279"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108280r280"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T5_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108281r281"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108282r282"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108283r283"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108284r284"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108285r285"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108286r286"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108287r287"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108288r288"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">end</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108289r289"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108290r290"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T3_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108291r291"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108292r292"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108293r293"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108294r294"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108295r295"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108296r296"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108297r297"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108298r298"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108299r299"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108300r300"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">start</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">build</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">your</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HASH</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108301r301"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108302r302"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108303r303"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108304r304"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108305r305"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">UCSCcoordinates</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108306r306"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">UCSC_chr</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108307r307"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">UCSC_start</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108308r308"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">UCSC_end</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108309r309"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">gene</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108310r310"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108311r311"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">species</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108312r312"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108313r313"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108314r314"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">five_prime_PCR_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$five_prime_PCR_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108315r315"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">three_prime_PCR_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$three_prime_PCR_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108316r316"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108317r317"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">target_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_seq_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108318r318"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">seed</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108319r319"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108320r320"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108321r321"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108322r322"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108323r323"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108324r324"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108325r325"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">notes</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$note_line</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108326r326"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">set</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108327r327"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108328r328"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108329r329"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Terminal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">targeted</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108330r330"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Advance</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">number</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108331r331"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108332r332"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108333r333"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Add</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">hash</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">table</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108334r334"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-5</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108335r335"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108336r336"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_5</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108337r337"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108338r338"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108339r339"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108340r340"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108341r341"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108342r342"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">TC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">internal</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108343r343"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108344r344"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108345r345"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-5-</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108346r346"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108347r347"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_5</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108348r348"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108349r349"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final_internal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108350r350"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108351r351"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108352r352"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108353r353"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108354r354"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108355r355"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108356r356"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Now</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">inside</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108357r357"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108358r358"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_ctrl_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108359r359"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_ctrl_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_ctrl_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108360r360"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T5_ctrl_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108361r361"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">IC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108362r362"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108363r363"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T5_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108364r364"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108365r365"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108366r366"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108367r367"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108368r368"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108369r369"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108370r370"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108371r371"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108372r372"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108373r373"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108374r374"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_tm</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108375r375"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108376r376"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108377r377"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108378r378"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-5-</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108379r379"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108380r380"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108381r381"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108382r382"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final_internal</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108383r383"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108384r384"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108385r385"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108386r386"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108387r387"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108388r388"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108389r389"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-3</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108390r390"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108391r391"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108392r392"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108393r393"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108394r394"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108395r395"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108396r396"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108397r397"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">TC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108398r398"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108399r399"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108400r400"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108401r401"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-3-</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108402r402"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108403r403"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108404r404"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final_internal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108405r405"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Grab</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">internal</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">control</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108406r406"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108407r407"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_ctrl_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108408r408"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_ctrl_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_ctrl_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108409r409"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108410r410"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,0,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_ctrl_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108411r411"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">IC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108412r412"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108413r413"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T3_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108414r414"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108415r415"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108416r416"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108417r417"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108418r418"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108419r419"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108420r420"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108421r421"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108422r422"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_tm</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108423r423"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108424r424"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">bed_start</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108425r425"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">bed_end</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108426r426"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108427r427"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108428r428"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-3-</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108429r429"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108430r430"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108431r431"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108432r432"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108433r433"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final_internal</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108434r434"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108435r435"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108436r436"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_seq_length</span><span 
class="ectt-1095">&#x003E;60)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108437r437"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108438r438"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108439r439"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108440r440"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108441r441"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">L</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108442r442"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108443r443"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108444r444"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108445r445"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$final</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">pcrsequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108446r446"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108447r447"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108448r448"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108449r449"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108450r450"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">N</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108451r451"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">L</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108452r452"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108453r453"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108454r454"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108455r455"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$final</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">pcrsequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108456r456"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108457r457"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108458r458"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prep_bed</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108459r459"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108460r460"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108461r461"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108462r462"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&amp;&amp;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_seq_length</span><span 
class="ectt-1095">&#x003C;=60)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108463r463"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108464r464"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108465r465"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">S</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108466r466"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$common</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108467r467"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_short_interal_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">common</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108468r468"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">lig_results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108469r469"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$final</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">pcrsequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108470r470"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108471r471"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">final</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108472r472"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">bed_final</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">prep_bed</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">bed_output</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108473r473"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108474r474"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108475r475"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">matching</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">brace</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">lines</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108476r476"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108477r477"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">else</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108478r478"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Matching</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">brace</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">csv</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">test</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108479r479"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">######</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">LIGAMERS</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ASSEMBLY</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PORTION</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108480r480"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108481r481"></a></span><span 
class="ectt-1095">close</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108482r482"></a></span><span 
class="ectt-1095">close</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">INPUT</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108483r483"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Program</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Finished</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108484r484"></a></span><span 
class="ectt-1095">exit</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108485r485"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108486r486"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">######</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">MAJOR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">WORK</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OF</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PROGRAM</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">!!</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108487r487"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">##############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108488r488"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Begin</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">section</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">program</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108489r489"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108490r490"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108491r491"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108492r492"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108493r493"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108494r494"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108495r495"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108496r496"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108497r497"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108498r498"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_tm</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108499r499"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108500r500"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108501r501"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108502r502"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108503r503"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108504r504"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108505r505"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108506r506"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Terminal_3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108507r507"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108508r508"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108509r509"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108510r510"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108511r511"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108512r512"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108513r513"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108514r514"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_tm</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108515r515"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108516r516"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108517r517"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108518r518"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108519r519"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108520r520"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108521r521"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">for</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">short</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slices</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#############################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108522r522"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_short_interal_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108523r523"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108524r524"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108525r525"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108526r526"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108527r527"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence_tm_obj</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108528r528"></a></span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">SeqFeature</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">new</span><span 
class="ectt-1095">(-</span><span 
class="ectt-1095">seq</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108529r529"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence_tm_obj</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108530r530"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108531r531"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108532r532"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108533r533"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">oligo</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108534r534"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108535r535"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108536r536"></a></span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,0,5)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108537r537"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108538r538"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108539r539"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">working_sequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108540r540"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108541r541"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_tm</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108542r542"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108543r543"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108544r544"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108545r545"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108546r546"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108547r547"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108548r548"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">output</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">=(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108549r549"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108550r550"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">gene</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">0</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108551r551"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108552r552"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">species</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">2</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108553r553"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">3</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108554r554"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">4</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108555r555"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">5</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108556r556"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">UCSCcoordinates</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">6</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108557r557"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">pcrsequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">7</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108558r558"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">8</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108559r559"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">target_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">9</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108560r560"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">10</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108561r561"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">11</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108562r562"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">12</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108563r563"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">13</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108564r564"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">14</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108565r565"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">15</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108566r566"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">16</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108567r567"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">warning</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108568r568"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">17</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108569r569"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">notes</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">18</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108570r570"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">set</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">19</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108571r571"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108572r572"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108573r573"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">Commented</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">on</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">022013</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108574r574"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">defined</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_ctrl_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108575r575"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5_ctrl_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},"\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">";</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108576r576"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108577r577"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108578r578"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">defined</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_ctrl_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108579r579"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3_ctrl_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">},"\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">";</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108580r580"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108581r581"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108582r582"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">OUT</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108583r583"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108584r584"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108585r585"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108586r586"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108587r587"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">csv</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">into</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">########################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108588r588"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse_the_line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108589r589"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108590r590"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108591r591"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108592r592"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108593r593"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108594r594"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108595r595"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108596r596"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108597r597"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108598r598"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108599r599"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108600r600"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108601r601"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108602r602"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108603r603"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">split</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/\</span><span 
class="ectt-1095">t</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108604r604"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108605r605"></a></span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/^&#x003C;//;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108606r606"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Gene</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108607r607"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Ligmamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">name</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_name</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108608r608"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">species</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108609r609"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108610r610"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108611r611"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">target_prime</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108612r612"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">UCSC</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108613r613"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">]\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108614r614"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Set</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">[</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">]\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108615r615"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108616r616"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">=~/</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">//}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">GO</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HERE</span><span 
class="ectt-1095">!</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108617r617"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108618r618"></a></span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/&#x003C;//;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108619r619"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108620r620"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108621r621"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108622r622"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$gene</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108623r623"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108624r624"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108625r625"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108626r626"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$target_prime</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108627r627"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$UCSCcoordinates</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108628r628"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108629r629"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$set</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108630r630"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108631r631"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108632r632"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108633r633"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108634r634"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">csv</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">into</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">############</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108635r635"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108636r636"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">coordnates</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">into</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">############</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108637r637"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse_coordinates</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108638r638"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108639r639"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$input</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108640r640"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108641r641"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$coordinates</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">split</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/\:/,</span><span 
class="ectt-1095">$input</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108642r642"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108643r643"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">split</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">/\-/,</span><span 
class="ectt-1095">$coordinates</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108644r644"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">chr</span><span 
class="ectt-1095">//;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">have</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">comment</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">out</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">this</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">behave</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">with</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">local</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">fasta</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">files</span><span 
class="ectt-1095">!</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108645r645"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108646r646"></a></span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/\,//</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108647r647"></a></span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/\,//</span><span 
class="ectt-1095">g</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108648r648"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108649r649"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108650r650"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108651r651"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108652r652"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">parse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">coordnates</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">into</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">variables</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108653r653"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108654r654"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">make</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">revcom</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">depending</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">on</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">annoation</span><span 
class="ectt-1095">###################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108655r655"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">revcom_slice_based_on_strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108656r656"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108657r657"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108658r658"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108659r659"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">is</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">positive</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">make</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">reverse</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">compliment</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108660r660"></a></span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">lc</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108661r661"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108662r662"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">plus</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108663r663"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">reverse</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108664r664"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">tr</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">ACGTacgt</span><span 
class="ectt-1095">/</span><span 
class="ectt-1095">TGCAtgca</span><span 
class="ectt-1095">/;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108665r665"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108666r666"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">is</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">minus</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">do</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">nothing</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108667r667"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">minus</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108668r668"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108669r669"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108670r670"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108671r671"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108672r672"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108673r673"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108674r674"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#####</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">SUBROUTINE</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">revcom_slice_based_on_strand</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">####################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108675r675"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108676r676"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108677r677"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtained</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">only</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">5&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">end</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">##</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108678r678"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T5_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108679r679"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108680r680"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108681r681"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108682r682"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108683r683"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108684r684"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108685r685"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108686r686"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108687r687"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108688r688"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108689r689"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">TC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108690r690"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">=19};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108691r691"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">=19};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108692r692"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108693r693"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108694r694"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108695r695"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_out</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108696r696"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108697r697"></a></span><span 
class="ectt-1095">while</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x003C;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108698r698"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108699r699"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">++;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108700r700"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108701r701"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,0,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108702r702"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_primer</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108703r703"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">SeqFeature</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108704r704"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">new</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108705r705"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108706r706"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">seq</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108707r707"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108708r708"></a></span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_primer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108709r709"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108710r710"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108711r711"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108712r712"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">oligo</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108713r713"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108714r714"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,0,5)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108715r715"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">;}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108716r716"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">&#x003E;=</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108717r717"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108718r718"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_out</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108719r719"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108720r720"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108721r721"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108722r722"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108723r723"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">33)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108724r724"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108725r725"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_out</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108726r726"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108727r727"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">STDERR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Warning</span><span 
class="ectt-1095">:</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108728r728"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Assembly</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">at</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T5</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">side</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cut</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108729r729"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">off</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">due</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">low</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108730r730"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108731r731"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108732r732"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">elsif</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">&#x003C;=</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108733r733"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108734r734"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T5_seq_out</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108735r735"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108736r736"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108737r737"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtained</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">only</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">3&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">end</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">##</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108738r738"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtain_T3_tm_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108739r739"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108740r740"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108741r741"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108742r742"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108743r743"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108744r744"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108745r745"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108746r746"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108747r747"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108748r748"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108749r749"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">TC</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">=(-</span><span 
class="ectt-1095">$control_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108750r750"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">=(-19)</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108751r751"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_location</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">=(-19)</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108752r752"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108753r753"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">=0;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108754r754"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108755r755"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_out</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108756r756"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108757r757"></a></span><span 
class="ectt-1095">while</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x003C;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108758r758"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108759r759"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108760r760"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">--;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108761r761"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108762r762"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$working_sequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108763r763"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_primer</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108764r764"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">SeqFeature</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Primer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108765r765"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">new</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108766r766"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108767r767"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">seq</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108768r768"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108769r769"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_primer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108770r770"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108771r771"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108772r772"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">salt</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$salt</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108773r773"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">oligo</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">$lig_conc</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108774r774"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108775r775"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">substr</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">,0,5)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108776r776"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(-</span><span 
class="ectt-1095">$working_sequence_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">;}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108777r777"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">&#x003C;(-80)</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">die</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108778r778"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">&#x003E;=</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108779r779"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108780r780"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_out</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108781r781"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108782r782"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108783r783"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108784r784"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108785r785"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(-33)</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108786r786"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108787r787"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_out</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108788r788"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108789r789"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">STDERR</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Warning</span><span 
class="ectt-1095">:</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108790r790"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Assembly</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">at</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">T3</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">side</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cut</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108791r791"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">off</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">due</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">low</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Tm</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108792r792"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">last</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108793r793"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108794r794"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">&#x003C;</span><span 
class="ectt-1095">$temp</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">next</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108795r795"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108796r796"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$T3_seq_out</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_tm</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq_length</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108797r797"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108798r798"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108799r799"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtained</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">only</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">3&#8217;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">end</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">working</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">####</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108800r800"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108801r801"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">joined</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">pieces</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">of</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#########</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108802r802"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer_piece_joiner</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108803r803"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108804r804"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108805r805"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108806r806"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">lig_joiner_code</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108807r807"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T5_seq</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108808r808"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">barcode</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108809r809"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">T3_seq</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108810r810"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108811r811"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$short_sequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108812r812"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108813r813"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108814r814"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$warning</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108815r815"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\/5</span><span 
class="ectt-1095">Phos</span><span 
class="ectt-1095">\/</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108816r816"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108817r817"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-5</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108818r818"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108819r819"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">three_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108820r820"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108821r821"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108822r822"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108823r823"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108824r824"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108825r825"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-5-</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108826r826"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108827r827"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">three_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108828r828"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$short_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108829r829"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108830r830"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108831r831"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108832r832"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108833r833"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-5-</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108834r834"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108835r835"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">three_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108836r836"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108837r837"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108838r838"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108839r839"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108840r840"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108841r841"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-3</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108842r842"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108843r843"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">five_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108844r844"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108845r845"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108846r846"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108847r847"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108848r848"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-3-</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108849r849"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108850r850"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">five_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108851r851"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108852r852"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108853r853"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108854r854"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108855r855"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108856r856"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">-3-</span><span 
class="ectt-1095">T</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108857r857"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108858r858"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">five_prime_PCR_sequence</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108859r859"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108860r860"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108861r861"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108862r862"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108863r863"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">S</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108864r864"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108865r865"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$short_sequence</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108866r866"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108867r867"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108868r868"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108869r869"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108870r870"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">L</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108871r871"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108872r872"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108873r873"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108874r874"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108875r875"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108876r876"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108877r877"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$lig_joiner_code</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">I</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">L</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">C</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108878r878"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108879r879"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">join</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$Phos_mod_code</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T5_seq</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$barcode</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">$T3_seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108880r880"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108881r881"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">-7;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108882r882"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108883r883"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108884r884"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">60)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108885r885"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108886r886"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">STDERR</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108887r887"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Warning</span><span 
class="ectt-1095">!</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">The</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">from</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">input</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">file</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">data</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">line</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108888r888"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">has</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">a</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">length</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">greater</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">than</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">60!\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108889r889"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">};</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108890r890"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108891r891"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108892r892"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">pcrsequence</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$pcrsequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108893r893"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108894r894"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">ligamer_length</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$ligamer_length</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108895r895"></a></span><span 
class="ectt-1095">$results</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">warning</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$warning</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108896r896"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108897r897"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">%</span><span 
class="ectt-1095">results</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108898r898"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108899r899"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108900r900"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108901r901"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108902r902"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108903r903"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Load</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">the</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">latest</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Registry</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108904r904"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl_database</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108905r905"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108906r906"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$verbose</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108907r907"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">shift@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108908r908"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108909r909"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$registry</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">Bio</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">EnsEMBL</span><span 
class="ectt-1095">::</span><span 
class="ectt-1095">Registry</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108910r910"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Beginning</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">login</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">database</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108911r911"></a></span><span 
class="ectt-1095">$registry</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">load_registry_from_db</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108912r912"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108913r913"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">host</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">ensembldb</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">.</span><span 
class="ectt-1095">org</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108914r914"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">user</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">anonymous</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108915r915"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">db_version</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db_version</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108916r916"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">verbose</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$verbose</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108917r917"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108918r918"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108919r919"></a></span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">Done</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">loading</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">ensembl</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">database</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108920r920"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$registry</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108921r921"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108922r922"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108923r923"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108924r924"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtained</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">#######</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108925r925"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get_genomic_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108926r926"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108927r927"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108928r928"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108929r929"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_adaptor</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$db</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">get_adaptor</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$species</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">Core</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">Slice</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108930r930"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108931r931"></a></span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/^</span><span 
class="ectt-1095">chr</span><span 
class="ectt-1095">//;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108932r932"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108933r933"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_adaptor</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108934r934"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">fetch_by_region</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108935r935"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108936r936"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">chromosome</span><span 
class="ectt-1095">&#8217;</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108937r937"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$chr</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108938r938"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$start</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108939r939"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$end</span><span 
class="ectt-1095">,</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108940r940"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108941r941"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108942r942"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$slice</span><span 
class="ectt-1095">-&#x003E;</span><span 
class="ectt-1095">seq</span><span 
class="ectt-1095">)</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108943r943"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108944r944"></a></span><span 
class="ectt-1095">return</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$slice_sequence</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108945r945"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108946r946"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">obtained</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">get</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">genomic</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">sequence</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">slice</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">######</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108947r947"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############################################################################</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108948r948"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">BEGIN</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PROVIDE</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HELP</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">DATA</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">######</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108949r949"></a></span><span 
class="ectt-1095">sub</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">podhelp</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108950r950"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108951r951"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108952r952"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$help_print</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108953r953"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$man_print</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">shift@_</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108954r954"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$perlname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108955r955"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$htmlname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108956r956"></a></span><span 
class="ectt-1095">my</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$manname</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$scriptname</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108957r957"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108958r958"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$help_print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108959r959"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108960r960"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$htmlname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/\.</span><span 
class="ectt-1095">pl</span><span 
class="ectt-1095">/\.</span><span 
class="ectt-1095">html</span><span 
class="ectt-1095">/;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108961r961"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">system</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">pod2html</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$perlname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">--</span><span 
class="ectt-1095">title</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$perlname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">--</span><span 
class="ectt-1095">outfile</span><span 
class="ectt-1095">=</span><span 
class="ectt-1095">$htmlname</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108962r962"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$htmlname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printed</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">cwd</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108963r963"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">exit</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108964r964"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108965r965"></a></span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108966r966"></a></span><span 
class="ectt-1095">if</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">(</span><span 
class="ectt-1095">$man_print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">eq</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">1)</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108967r967"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">{</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108968r968"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$manname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">=~</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">s</span><span 
class="ectt-1095">/\.</span><span 
class="ectt-1095">pl</span><span 
class="ectt-1095">/\.</span><span 
class="ectt-1095">man</span><span 
class="ectt-1095">/;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108969r969"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">system</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">pod2man</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$perlname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$manname</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108970r970"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">print</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">t$manname</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">printed</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">in</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$dir</span><span 
class="ectt-1095">.\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">\</span><span 
class="ectt-1095">n</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108971r971"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">system</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">man</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">-</span><span 
class="ectt-1095">l</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">$manname</span><span 
class="ectt-1095">|</span><span 
class="ectt-1095">less</span><span 
class="ectt-1095">"</span><span 
class="ectt-1095">;</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108972r972"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">exit</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108973r973"></a></span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108974r974"></a></span><span 
class="ectt-1095">}</span><span 
class="ectt-1095">&#x00A0;</span><br /><span class="label"><a 
 id="x1-108975r975"></a></span><span 
class="ectt-1095">#</span><span 
class="ectt-1095">###############</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">END</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">Subroutine</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">to</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">PROVIDE</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">POD</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">HELP</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">DATA</span><span 
class="ectt-1095">&#x00A0;</span><span 
class="ectt-1095">######</span>

</div>
                                                                                

                                                                                
<a 
 id="x1-108978r10"></a>
<a 
 id="Q1-1-173"></a>
                                                                                

                                                                                
<h2 class="likechapterHead"><a 
 id="x1-10900010"></a>Bibliography</h2>
<div class="thebibliography">
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAltschul1990"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Altschul, S., Gish, W., and Miller, W. (1990).  Basic local alignment search
tool. <span 
class="phvro8t-x-x-120">Journal of molecular </span><span 
class="phvro8t-x-x-120">&#x2026;</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAmitsur1987d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Amitsur,  M.,  Levitz,  R.,  and  Kaufmann,  G.  (1987).    Bacteriophage  T4
anticodon nuclease,  polynucleotide  kinase  and  RNA  ligase  reprocess  the
host lysine tRNA. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 6(8):2499&#8211;503.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAnders2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Anders, S.  and  Huber,  W.  (2010).    Differential  expression  analysis  for
sequence count data. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 11(10):R106.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAnstey2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Anstey, M.&#x00A0;L.,  Rogers,  S.&#x00A0;M.,  Ott,  S.&#x00A0;R.,  Burrows,  M.,  and  Simpson,
S.&#x00A0;J. (2009).  Serotonin mediates behavioral gregarization underlying swarm
formation in desert locusts. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 323(5914):627&#8211;30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.,  Gaidatzis,  D.,  Pfeffer,  S.,  Lagos-Quintana,  M.,  Landgraf,  P.,
Iovino, N., Morris, P., Brownstein, M.&#x00A0;J., Kuramochi-Miyagawa, S., Nakano,
T., Chien, M., Russo, J.&#x00A0;J., Ju, J., Sheridan, R., Sander, C., Zavolan, M., and
Tuschl, T. (2006).  A novel class of small RNAs bind to MILI protein in mouse
testes. <span 
class="phvro8t-x-x-120">Nature</span>, 442(7099):203&#8211;207.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A. and Hannon, G.&#x00A0;J. (2008).  Small RNA silencing pathways in
                                                                                

                                                                                
germ and stem cells.  <span 
class="phvro8t-x-x-120">Cold Spring Harbor symposia on quantitative biology</span>,
73:283&#8211;90.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2007a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A., Hannon, G.&#x00A0;J., and Brennecke, J. (2007a).  The Piwi-piRNA
Pathway Provides  an  Adaptive  Defense  in  the  Transposon  Arms  Race.
<span 
class="phvro8t-x-x-120">Science</span>, 318(5851):761&#8211;764.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A., Naumova, N.&#x00A0;M., Tulin, A.&#x00A0;V., Vagin, V.&#x00A0;V., Rozovsky, Y.&#x00A0;M.,
and Gvozdev,  V.&#x00A0;A.  (2001).    Double-stranded  RNA-mediated  silencing  of
genomic tandem repeats and transposable elements in the D. melanogaster
germline. <span 
class="phvro8t-x-x-120">Current Biology</span>, 11(13):1017&#8211;1027.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAravin2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Aravin, A.&#x00A0;A., Sachidanandam, R., Girard, A., Fejes-Toth, K., and Hannon,
G.&#x00A0;J. (2007b).  Developmentally regulated piRNA clusters implicate MILI in
transposon control. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 316(5825):744&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XAshe2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ashe, A.,  Sapetschnig,  A.,  Weick,  E.-M.,  Mitchell,  J.,  Bagijn,  M.&#x00A0;P.,
Cording, A.&#x00A0;C.,  Doebley,  A.-L.,  Goldstein,  L.&#x00A0;D.,  Lehrbach,  N.&#x00A0;J.,  Le  Pen,
J., Pintacuda, G., Sakaguchi, A., Sarkies, P., Ahmed, S., and Miska, E.&#x00A0;a.
(2012). piRNAs  can  trigger  a  multigenerational  epigenetic  memory  in  the
germline of C. elegans. <span 
class="phvro8t-x-x-120">Cell</span>, 150(1):88&#8211;99.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBailey2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bailey, T.&#x00A0;L.,  Boden,  M.,  Buske,  F.&#x00A0;a.,  Frith,  M.,  Grant,  C.&#x00A0;E.,  Clementi,
L., Ren, J.,  Li,  W.&#x00A0;W.,  and  Noble,  W.&#x00A0;S.  (2009).   MEME  SUITE:  tools  for
motif discovery  and  searching.    <span 
class="phvro8t-x-x-120">Nucleic  acids  research</span>,  37(Web  Server
issue):W202&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBailey1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bailey, T.&#x00A0;L. and Elkan, C. (1994).  Fitting a mixture model by expectation
maximization to discover motifs in biopolymers.   <span 
class="phvro8t-x-x-120">Proceedings International</span>
<span 
class="phvro8t-x-x-120">Conference on Intelligent Systems for Molecular Biology ISMB International</span>
<span 
class="phvro8t-x-x-120">Conference on Intelligent Systems for Molecular Biology</span>, 2:28&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarash2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barash, Y., Calarco, J.&#x00A0;A., Gao, W., Pan, Q., Wang, X., Shai, O., Blencowe,
B.&#x00A0;J., and  Frey,  B.&#x00A0;J.  (2010).    Deciphering  the  splicing  code.    <span 
class="phvro8t-x-x-120">Nature</span>,
                                                                                

                                                                                
465(7294):53&#8211;59.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarbosa-Morais2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barbosa-Morais,  N.&#x00A0;L.,  Irimia,  M.,  Pan,  Q.,  Xiong,  H.&#x00A0;Y.,  Gueroussov,
S., Lee, L.&#x00A0;J.,  Slobodeniuc,  V.,  Kutter,  C.,  Watt,  S.,  Colak,  R.,  Kim,  T.,
Misquitta-Ali, C.&#x00A0;M., Wilson, M.&#x00A0;D., Kim, P.&#x00A0;M., Odom, D.&#x00A0;T., Frey, B.&#x00A0;J., and
Blencowe, B.&#x00A0;J. (2012). The evolutionary landscape of alternative splicing in
vertebrate species. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 338(6114):1587&#8211;93.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarrangou2007c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barrangou,  R.,  Fremaux,  C.,  Deveau,  H.,  Richards,  M.,  Boyaval,  P.,
Moineau, S.,  Romero,  D.&#x00A0;a.,  and  Horvath,  P.  (2007).    CRISPR  provides
acquired resistance against viruses in prokaryotes. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
315(5819):1709&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBarrett2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Barrett,   T.,   Wilhite,   S.&#x00A0;E.,   Ledoux,   P.,   Evangelista,   C.,   Kim,   I.&#x00A0;F.,
Tomashevsky, M., Marshall, K.&#x00A0;a., Phillippy, K.&#x00A0;H., Sherman, P.&#x00A0;M., Holko,
M., Yefanov, A., Lee, H., Zhang, N., Robertson, C.&#x00A0;L., Serova, N., Davis, S.,
and Soboleva, A. (2013).  NCBI GEO: archive for functional genomics data
sets&#8211;update. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 41(Database issue):D991&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBaudat2000c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Baudat, F.,  Manova,  K.,  Yuen,  J.&#x00A0;P.,  Jasin,  M.,  and  Keeney,  S.  (2000).
Chromosome synapsis defects and sexually dimorphic meiotic progression in
mice lacking Spo11. <span 
class="phvro8t-x-x-120">Molecular cell</span>, 6(5):989&#8211;98.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCordell1980a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bell, G.&#x00A0;I.,  Pictet,  R.&#x00A0;L.,  Rutter,  W.&#x00A0;J.,  Cordell,  B.,  Tischer,  E.,  and
Goodman,  H.&#x00A0;M.  (1980).   Sequence  of  the  human  insulin  gene.   <span 
class="phvro8t-x-x-120">Nature</span>,
284(5751):26&#8211;32.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBenson2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Benson, D.&#x00A0;a., Karsch-Mizrachi, I., Lipman, D.&#x00A0;J., Ostell, J., and Sayers,
E.&#x00A0;W. (2011). GenBank. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 39(Database issue):D32&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBentley2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bentley, D.&#x00A0;R., Balasubramanian, S., Swerdlow, H.&#x00A0;P., Smith, G.&#x00A0;P., Milton,
J., Brown,  C.&#x00A0;G.,  Hall,  K.&#x00A0;P.,  Evers,  D.&#x00A0;J.,  Barnes,  C.&#x00A0;L.,  Bignell,  H.&#x00A0;R.,
Boutell, J.&#x00A0;M.,  Bryant,  J.,  Carter,  R.&#x00A0;J.,  Keira  Cheetham,  R.,  Cox,  A.&#x00A0;J.,
Ellis, D.&#x00A0;J., Flatbush, M.&#x00A0;R., Gormley, N.&#x00A0;a., Humphray, S.&#x00A0;J., Irving, L.&#x00A0;J.,
                                                                                

                                                                                
Karbelashvili,  M.&#x00A0;S.,  Kirk,  S.&#x00A0;M.,  Li,  H.,  Liu,  X.,  Maisinger,  K.&#x00A0;S.,  Murray,
L.&#x00A0;J., Obradovic, B., Ost, T., Parkinson, M.&#x00A0;L., Pratt, M.&#x00A0;R., Rasolonjatovo,
I. M.&#x00A0;J., Reed,  M.&#x00A0;T.,  Rigatti,  R.,  Rodighiero,  C.,  Ross,  M.&#x00A0;T.,  Sabot,  A.,
Sankar, S.&#x00A0;V., Scally, A., Schroth, G.&#x00A0;P., Smith, M.&#x00A0;E., Smith, V.&#x00A0;P., Spiridou,
A., Torrance, P.&#x00A0;E., Tzonev, S.&#x00A0;S., Vermaas, E.&#x00A0;H., Walter, K., Wu, X., Zhang,
L., Alam, M.&#x00A0;D., Anastasi, C., Aniebo, I.&#x00A0;C., Bailey, D. M.&#x00A0;D., Bancarz, I.&#x00A0;R.,
Banerjee, S., Barbour, S.&#x00A0;G., Baybayan, P.&#x00A0;a., Benoit, V.&#x00A0;a., Benson, K.&#x00A0;F.,
Bevis, C., Black, P.&#x00A0;J., Boodhun, A., Brennan, J.&#x00A0;S., Bridgham, J.&#x00A0;a., Brown,
R.&#x00A0;C., Brown, A.&#x00A0;a., Buermann, D.&#x00A0;H., Bundu, A.&#x00A0;a., Burrows, J.&#x00A0;C., Carter,
N.&#x00A0;P., Castillo,  N.,  Chiara  E  Catenazzi,  M.,  Chang,  S.,  Neil  Cooley,  R.,
Crake, N.&#x00A0;R., Dada, O.&#x00A0;O., Diakoumakos, K.&#x00A0;D., Dominguez-Fernandez, B.,
Earnshaw, D.&#x00A0;J., Egbujor, U.&#x00A0;C., Elmore, D.&#x00A0;W., Etchin, S.&#x00A0;S., Ewan, M.&#x00A0;R.,
Fedurco, M., Fraser, L.&#x00A0;J., Fuentes Fajardo, K.&#x00A0;V., Scott Furey, W., George,
D., Gietzen,  K.&#x00A0;J.,  Goddard,  C.&#x00A0;P.,  Golda,  G.&#x00A0;S.,  Granieri,  P.&#x00A0;a.,  Green,
D.&#x00A0;E., Gustafson,  D.&#x00A0;L.,  Hansen,  N.&#x00A0;F.,  Harnish,  K.,  Haudenschild,  C.&#x00A0;D.,
Heyer, N.&#x00A0;I., Hims, M.&#x00A0;M., Ho, J.&#x00A0;T., Horgan, A.&#x00A0;M., Hoschler, K., Hurwitz,
S., Ivanov, D.&#x00A0;V., Johnson, M.&#x00A0;Q., James, T., Huw Jones, T.&#x00A0;a., Kang, G.-D.,
Kerelska, T.&#x00A0;H., Kersey, A.&#x00A0;D., Khrebtukova, I., Kindwall, A.&#x00A0;P., Kingsbury, Z.,
Kokko-Gonzales, P.&#x00A0;I., Kumar, A., Laurent, M.&#x00A0;a., Lawley, C.&#x00A0;T., Lee, S.&#x00A0;E.,
Lee, X., Liao, A.&#x00A0;K., Loch, J.&#x00A0;a., Lok, M., Luo, S., Mammen, R.&#x00A0;M., Martin,
J.&#x00A0;W., McCauley, P.&#x00A0;G., McNitt, P., Mehta, P., Moon, K.&#x00A0;W., Mullens, J.&#x00A0;W.,
Newington, T., Ning, Z., Ling Ng, B., Novo, S.&#x00A0;M., O&#8217;Neill, M.&#x00A0;J., Osborne,
M.&#x00A0;a., Osnowski,  A.,  Ostadan,  O.,  Paraschos,  L.&#x00A0;L.,  Pickering,  L.,  Pike,
A.&#x00A0;C., Pike, A.&#x00A0;C., Chris Pinkard, D., Pliskin, D.&#x00A0;P., Podhasky, J., Quijano,
V.&#x00A0;J., Raczy,  C.,  Rae,  V.&#x00A0;H.,  Rawlings,  S.&#x00A0;R.,  Chiva  Rodriguez,  A.,  Roe,
P.&#x00A0;M., Rogers, J., Rogert Bacigalupo, M.&#x00A0;C., Romanov, N., Romieu, A., Roth,
R.&#x00A0;K., Rourke, N.&#x00A0;J., Ruediger, S.&#x00A0;T., Rusman, E., Sanches-Kuiper, R.&#x00A0;M.,
Schenker, M.&#x00A0;R., Seoane, J.&#x00A0;M., Shaw, R.&#x00A0;J., Shiver, M.&#x00A0;K., Short, S.&#x00A0;W.,
Sizto, N.&#x00A0;L.,  Sluis,  J.&#x00A0;P.,  Smith,  M.&#x00A0;a.,  Ernest  Sohna  Sohna,  J.,  Spence,
E.&#x00A0;J., Stevens, K., Sutton, N., Szajkowski, L., Tregidgo, C.&#x00A0;L., Turcatti, G.,
Vandevondele,  S.,  Verhovsky,  Y.,  Virk,  S.&#x00A0;M.,  Wakelin,  S.,  Walcott,  G.&#x00A0;C.,
Wang, J., Worsley, G.&#x00A0;J., Yan, J., Yau, L., Zuerlein, M., Rogers, J., Mullikin,
J.&#x00A0;C., Hurles,  M.&#x00A0;E.,  McCooke,  N.&#x00A0;J.,  West,  J.&#x00A0;S.,  Oaks,  F.&#x00A0;L.,  Lundberg,
P.&#x00A0;L., Klenerman, D., Durbin, R., and Smith, A.&#x00A0;J. (2008).   Accurate whole
human genome sequencing using reversible terminator chemistry.   <span 
class="phvro8t-x-x-120">Nature</span>,
456(7218):53&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBenton2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Benton, M.&#x00A0;J. and Donoghue, P. C.&#x00A0;J. (2007).  Paleontological evidence to
date the tree of life. <span 
class="phvro8t-x-x-120">Molecular biology and evolution</span>, 24(1):26&#8211;53.
</p>
                                                                                

                                                                                
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBernstein1983a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bernstein,  S.&#x00A0;I.,  Mogami,  K.,  Donady,  J.&#x00A0;J.,  and  Emerson,  C.&#x00A0;P.  (1983).
Drosophila muscle myosin heavy chain encoded by a single gene in a cluster
of muscle mutations. <span 
class="phvro8t-x-x-120">Nature</span>, 302(5907):393&#8211;397.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBesmer1972b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Besmer,  P.,  Jr.,  R.  C.&#x00A0;M.,  Caruthers,  M.&#x00A0;H.,  Kumar,  A.,  Minamoto,  K.,
van&#x00A0;de Sande,  J.,  Sidarova,  N.,  and  Khorana,  H.  (1972).     Studies  on
polynucleotides. CXVII. Hybridization of polydeoxynucleotides with tyrosine
transfer RNA sequences to the r-strand of phi80psu + 3 DNA.   <span 
class="phvro8t-x-x-120">Journal of</span>
<span 
class="phvro8t-x-x-120">Molecular Biology</span>, 72:503&#8211;522.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBingham2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bingham,   J.&#x00A0;L.,   Carrigan,   P.&#x00A0;E.,   Miller,   L.&#x00A0;J.,   and   Srinivasan,   S.
(2008). Extent  and  diversity  of  human  alternative  splicing  established  by
complementary  database  annotation  and  microarray  analysis.    <span 
class="phvro8t-x-x-120">Omics:  A</span>
<span 
class="phvro8t-x-x-120">Journal of Integrative Biology</span>, 12(1):83&#8211;92.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBlencowe2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Blencowe,        B.&#x00A0;J.,        Ahmad,        S.,        and        Lee,        L.&#x00A0;J.
(2009). Current-generation high-throughput sequencing: deepening insights
into mammalian transcriptomes. <span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 23(12):1379&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBolcun-Filas2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bolcun-Filas, E., Bannister, L.&#x00A0;A., Barash, A., Schimenti, K.&#x00A0;J., Hartford,
S.&#x00A0;A., Eppig,  J.&#x00A0;J.,  Handel,  M.&#x00A0;A.,  Shen,  L.,  and  Schimenti,  J.&#x00A0;C.  (2011).
A-MYB (MYBL1) transcription factor is a master regulator of male meiosis.
<span 
class="phvro8t-x-x-120">Development</span>, 138(15):3319 &#8211;3330.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrennecke2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brennecke, J., Aravin, A.&#x00A0;A., Stark, A., Dus, M., Kellis, M., Sachidanandam,
R., and Hannon, G.&#x00A0;J. (2007). Discrete small RNA-generating loci as master
regulators of transposon activity in Drosophila. <span 
class="phvro8t-x-x-120">Cell</span>, 128(6):1089&#8211;1103.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrennecke2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brennecke, J. and Malone, C. (2008).  An Epigenetic Role for Maternally
Inherited piRNAs   in   Transposon   Silencing.       <span 
class="phvro8t-x-x-120">Science   (New   York,   </span><span 
class="phvro8t-x-x-120">&#x2026;</span>,
322(November):1387&#8211;1392.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrenner2000a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brenner, S., Johnson, M., Bridgham, J., Golda, G., Lloyd, D.&#x00A0;H., Johnson,
                                                                                

                                                                                
D., Luo, S., McCurdy, S., Foy, M., Ewan, M., Roth, R., George, D., Eletr, S.,
Albrecht, G., Vermaas, E., Williams, S.&#x00A0;R., Moon, K., Burcham, T., Pallas,
M., DuBridge,  R.&#x00A0;B.,  Kirchner,  J.,  Fearon,  K.,  Mao,  J.,  and  Corcoran,  K.
(2000). Gene expression analysis by massively parallel signature sequencing
(MPSS) on microbead arrays. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 18(6):630&#8211;4.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBrown2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Brown, J.&#x00A0;B.,  Boley,  N.,  Eisman,  R.,  May,  G.&#x00A0;E.,  Stoiber,  M.&#x00A0;H.,  Duff,
M.&#x00A0;O., Booth, B.&#x00A0;W., Wen, J., Park, S., Suzuki, A.&#x00A0;M., Wan, K.&#x00A0;H., Yu, C.,
Zhang, D., Carlson, J.&#x00A0;W., Cherbas, L., Eads, B.&#x00A0;D., Miller, D., Mockaitis, K.,
Roberts, J., Davis, C.&#x00A0;a., Frise, E., Hammonds, A.&#x00A0;S., Olson, S., Shenker,
S., Sturgill, D., Samsonova, A.&#x00A0;a., Weiszmann, R., Robinson, G., Hernandez,
J., Andrews,  J.,  Bickel,  P.&#x00A0;J.,  Carninci,  P.,  Cherbas,  P.,  Gingeras,  T.&#x00A0;R.,
Hoskins, R.&#x00A0;a., Kaufman, T.&#x00A0;C., Lai, E.&#x00A0;C., Oliver, B., Perrimon, N., Graveley,
B.&#x00A0;R., and Celniker, S.&#x00A0;E. (2014).  Diversity and dynamics of the Drosophila
transcriptome. <span 
class="phvro8t-x-x-120">Nature</span>, pages 1&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBullard2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Bullard, D. and Bowater, R. (2006). Direct comparison of nick-joining activity
of the nucleic  acid  ligases  from  bacteriophage  T4.    <span 
class="phvro8t-x-x-120">Biochemical  Journal</span>,
398(Pt 1):135&#8211;144.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBurnette2005a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Burnette, J.&#x00A0;M., Miyamoto-Sato, E., Schaub, M.&#x00A0;a., Conklin, J., and Lopez,
a.&#x00A0;J. (2005).  Subdivision of large introns in Drosophila by recursive splicing
at nonexonic elements. <span 
class="phvro8t-x-x-120">Genetics</span>, 170(2):661&#8211;74.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XBurnside2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Burnside, J., Ouyang, M., Anderson, A., Bernberg, E., Lu, C., Meyers, B.&#x00A0;C.,
Green, P.&#x00A0;J., Markis, M., Isaacs, G., Huang, E., and Morgan, R.&#x00A0;W. (2008).
Deep sequencing of chicken microRNAs. <span 
class="phvro8t-x-x-120">BMC genomics</span>, 9:185.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCalarco2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Calarco,  J.&#x00A0;A.,  Saltzman,  A.&#x00A0;L.,  Ip,  J.&#x00A0;Y.,  and  Blencowe,  B.&#x00A0;J.  (2007a).
Technologies for the global discovery and analysis of alternative splicing.  In
<span 
class="phvro8t-x-x-120">Advances in Experimental Medicine and Biology</span>, volume 623, pages 64&#8211;84.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCalarco2007a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Calarco,  J.&#x00A0;A.,  Xing,  Y.,  Cceres,  M.,  Calarco,  J.&#x00A0;P.,  Xiao,  X.,  Pan,  Q.,
Lee, C., Preuss,  T.&#x00A0;M.,  and  Blencowe,  B.&#x00A0;J.  (2007b).   Global  analysis  of
alternative splicing differences between humans and chimpanzees. <span 
class="phvro8t-x-x-120">Genes &amp;</span>
                                                                                

                                                                                
<span 
class="phvro8t-x-x-120">Development</span>, 21(22):2963&#8211;75.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCarmell2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Carmell,  M.&#x00A0;A.,  Girard,  A.,  van&#x00A0;de  Kant,  H.&#x00A0;J.,  Bourc&#8217;his,  D.,  Bestor,
T.&#x00A0;H., de&#x00A0;Rooij,  D.&#x00A0;G.,  and  Hannon,  G.&#x00A0;J.  (2007).    MIWI2  Is  Essential
for Spermatogenesis  and  Repression  of  Transposons  in  the  Mouse  Male
Germline. <span 
class="phvro8t-x-x-120">Developmental Cell</span>, 12(4):503&#8211;514.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCelotto2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Celotto, a.&#x00A0;M.  and  Graveley,  B.&#x00A0;R.  (2001).    Alternative  splicing  of  the
Drosophila  Dscam  pre-mRNA  is  both  temporally  and  spatially  regulated.
<span 
class="phvro8t-x-x-120">Genetics</span>, 159(2):599&#8211;608.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCenik2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Cenik, E.&#x00A0;S. and Zamore, P.&#x00A0;D. (2011). Argonaute proteins. <span 
class="phvro8t-x-x-120">Current biology</span>
<span 
class="phvro8t-x-x-120">: CB</span>, 21(12):R446&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChang2014b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chang, H., Lim, J., Ha, M., and Kim, V. (2014).  TAIL-seq: Genome-wide
Determination of Poly(A) Tail Length and 3 End Modifications. <span 
class="phvro8t-x-x-120">Molecular Cell</span>,
pages 1&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChauleau2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chauleau, M. and Shuman, S. (2013).  Kinetic mechanism of nick sealing
by T4 RNA ligase 2 and effects of 3&#8217;-OH base mispairs and damaged base
lesions. <span 
class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>, 19(12):1840&#8211;1847.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, C.,  Jin,  J.,  James,  D.&#x00A0;A.,  Adams-cioaba,  M.&#x00A0;A.,  Gyoon,  J.,  Guo,
Y., Tenaglia, E., Xu, C., Gish, G., Min, J., and Pawson, T. (2009).   Mouse
Piwi interactome  identifies  binding  mechanism  of  Tdrkh  Tudor  domain  to
arginine methylated Miwi. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>,
106(48):20336&#8211;20341.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChen2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chen, X., Xu, H., Yuan, P., Fang, F., Huss, M., Vega, V.&#x00A0;B., Wong, E., Orlov,
Y.&#x00A0;L., Zhang,  W.,  Jiang,  J.,  Loh,  Y.-H.,  Yeo,  H.&#x00A0;C.,  Yeo,  Z.&#x00A0;X.,  Narang,  V.,
Govindarajan, K.&#x00A0;R., Leong, B., Shahab, A., Ruan, Y., Bourque, G., Sung,
W.-K., Clarke, N.&#x00A0;D., Wei, C.-L., and Ng, H.-H. (2008). Integration of external
signaling pathways with the core transcriptional network in embryonic stem
cells. <span 
class="phvro8t-x-x-120">Cell</span>, 133(6):1106&#8211;17.
                                                                                

                                                                                
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XChuma2006a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Chuma, S. and Hosokawa, M. (2006).  Tdrd1/Mtr-1 ,a male tudor-related
gene, is essential  for  male  germ-cell  differentiationand  nuage/germinal
grandule formation in mice. <span 
class="phvro8t-x-x-120">Proceedings of the </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, 103(43):1&#8211;6.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHattori2005a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Consortium, I. H. G.&#x00A0;S. (2004). Finishing the euchromatic sequence of the
human genome. <span 
class="phvro8t-x-x-120">Nature</span>, 50(2):162&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XConze2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Conze, T.,  Goransson,  J.,  Razzaghian,  H.&#x00A0;R.,  Ericsson,  O.,  Oberg,  D.,
Akusjarvi, G.,  Landegren,  U.,  and  Nilsson,  M.  (2010).    Single  molecule
analysis of combinatorial splicing. <span 
class="phvro8t-x-x-120">Nucl. Acids Res.</span>, page gkq581.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XConze2009c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Conze, T.,  Shetye,  A.,  Tanaka,  Y.,  Gu,  J.,  Larsson,  C.,  Gransson,  J.,
Tavoosidana,  G.,  Sderberg,  O.,  Nilsson,  M.,  and  Landegren,  U.  (2009).
Analysis of genes, transcripts, and proteins via DNA ligation.  <span 
class="phvro8t-x-x-120">Annual review</span>
<span 
class="phvro8t-x-x-120">of analytical chemistry (Palo Alto, Calif.)</span>, 2:215&#8211;39.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCore2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Core, L.&#x00A0;J., Waterfall, J.&#x00A0;J., and Lis, J.&#x00A0;T. (2008). Nascent RNA sequencing
reveals widespread  pausing  and  divergent  initiation  at  human  promoters.
<span 
class="phvro8t-x-x-120">Science</span>, 322:1845&#8211;1848.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XCramer1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Cramer,   P.,   Pesce,   C.&#x00A0;G.,   Baralle,   F.&#x00A0;E.,   and   Kornblihtt,   A.&#x00A0;R.
(1997). Functional  association  between  promoter  structure  and  transcript
alternativesplicing.     <span 
class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of  Sciences</span>,
94(21):11456 &#8211;11460.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeFazio2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> De Fazio, S., Bartonicek, N., Di Giacomo, M., Abreu-Goodger, C., Sankar,
A., Funaya,  C.,  Antony,  C.,  Moreira,  P.&#x00A0;N.,  Enright,  A.&#x00A0;J.,  and  O/&#8217;Carroll,
D. (2011).   The endonuclease activity of Mili fuels piRNA amplification that
silences LINE1 elements. <span 
class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeng1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Deng, Q. Q.-l., Ishii, S., and Sarai, A. (1996). Binding site analysis of c-Myb
: screening  of  potential  binding  sites  by  using  the  mutation  matrix  derived
                                                                                

                                                                                
from systematic  binding  affinity  measurements.   <span 
class="phvro8t-x-x-120">Nucleic  Acids  Research</span>,
24(4):766&#8211;774.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDeng2002c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Deng, W. and Lin, H. (2002).   miwi, a Murine Homolog of piwi, Encodes
a Cytoplasmic Protein Essential for Spermatogenesis.  <span 
class="phvro8t-x-x-120">Developmental cell</span>,
2(6):819&#8211;830.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDjebali2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Djebali, S., Davis, C.&#x00A0;a., Merkel, A., Dobin, A., Lassmann, T., Mortazavi,
A., Tanzer,  A.,  Lagarde,  J.,  Lin,  W.,  Schlesinger,  F.,  Xue,  C.,  Marinov,
G.&#x00A0;K., Khatun,  J.,  Williams,  B.&#x00A0;a.,  Zaleski,  C.,  Rozowsky,  J.,  Rder,  M.,
Kokocinski,  F.,  Abdelhamid,  R.&#x00A0;F.,  Alioto,  T.,  Antoshechkin,  I.,  Baer,  M.&#x00A0;T.,
Bar, N.&#x00A0;S.,  Batut,  P.,  Bell,  K.,  Bell,  I.,  Chakrabortty,  S.,  Chen,  X.,  Chrast,
J., Curado, J., Derrien, T., Drenkow, J., Dumais, E., Dumais, J., Duttagupta,
R., Falconnet,  E.,  Fastuca,  M.,  Fejes-Toth,  K.,  Ferreira,  P.,  Foissac,  S.,
Fullwood, M.&#x00A0;J.,  Gao,  H.,  Gonzalez,  D.,  Gordon,  A.,  Gunawardena,  H.,
Howald, C.,  Jha,  S.,  Johnson,  R.,  Kapranov,  P.,  King,  B.,  Kingswood,  C.,
Luo, O.&#x00A0;J., Park, E., Persaud, K., Preall, J.&#x00A0;B., Ribeca, P., Risk, B., Robyr,
D., Sammeth, M., Schaffer, L., See, L.-H., Shahab, A., Skancke, J., Suzuki,
A.&#x00A0;M., Takahashi, H., Tilgner, H., Trout, D., Walters, N., Wang, H., Wrobel,
J., Yu, Y., Ruan, X., Hayashizaki, Y., Harrow, J., Gerstein, M., Hubbard, T.,
Reymond, A.,  Antonarakis,  S.&#x00A0;E.,  Hannon,  G.,  Giddings,  M.&#x00A0;C.,  Ruan,  Y.,
Wold, B., Carninci, P., Guig, R., and Gingeras, T.&#x00A0;R. (2012).  Landscape of
transcription in human cells. <span 
class="phvro8t-x-x-120">Nature</span>, 489(7414):101&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XDjebali2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Djebali, S.,  Kapranov,  P.,  Foissac,  S.,  Lagarde,  J.,  Reymond,  A.,  Ucla,
C., Wyss, C.,  Drenkow,  J.,  Dumais,  E.,  Murray,  R.&#x00A0;R.,  Lin,  C.,  Szeto,  D.,
Denoeud, F.,  Calvo,  M.,  Frankish,  A.,  Harrow,  J.,  Makrythanasis,  P.,  Vidal,
M., Salehi-Ashtiani, K., Antonarakis, S.&#x00A0;E., Gingeras, T.&#x00A0;R., and Guigo, R.
(2008). Efficient targeted transcript discovery via array-based normalization
of RACE libraries. <span 
class="phvro8t-x-x-120">Nat Meth</span>, 5(7):629&#8211;635.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XEmerick2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Emerick, M., Parmigiani, G., and Agnew, W. (2007).  Multivariate Analysis
and Visualization  of  Splicing  Correlations  in  Single-Gene  Transcriptomes.
<span 
class="phvro8t-x-x-120">BMC Bioinformatics</span>, 8(1):16.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFagnani2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fagnani, M., Barash, Y., Ip, J.&#x00A0;Y., Misquitta, C., Pan, Q., Saltzman, A.&#x00A0;L.,
                                                                                

                                                                                
Shai, O., Lee,  L.,  Rozenhek,  A.,  Mohammad,  N.,  Willaime-Morawek,  S.,
Babak, T.,  Zhang,  W.,  Hughes,  T.&#x00A0;R.,  van&#x00A0;der  Kooy,  D.,  Frey,  B.&#x00A0;J.,  and
Blencowe, B.&#x00A0;J. (2007).  Functional coordination of alternative splicing in the
mammalian central nervous system. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 8(6):R108.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFarazi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Farazi, T.&#x00A0;a., Juranek, S.&#x00A0;a., and Tuschl, T. (2008).  The growing catalog
of small RNAs  and  their  association  with  distinct  Argonaute/Piwi  family
members. <span 
class="phvro8t-x-x-120">Development (Cambridge, England)</span>, 135(7):1201&#8211;14.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFareed1971"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fareed, G.&#x00A0;C.,  Wilt,  E.&#x00A0;M.,  and  Richardson,  C.&#x00A0;C.  (1971).    Enzymatic
Breakage and  Joining  of  Deoxyribonucleic  Acid.     <span 
class="phvro8t-x-x-120">Journal  of  Biological</span>
<span 
class="phvro8t-x-x-120">Chemistry</span>, 246:925&#8211;932.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFededa2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fededa,  J.&#x00A0;P.,  Petrillo,  E.,  Gelfand,  M.&#x00A0;S.,  Neverov,  A.&#x00A0;D.,  Kadener,  S.,
Nogus, G.,  Pelisch,  F.,  Baralle,  F.&#x00A0;E.,  Muro,  A.&#x00A0;F.,  and  Kornblihtt,  A.&#x00A0;R.
(2005). A  Polar  Mechanism  Coordinates  Different  Regions  of  Alternative
Splicing within a Single Gene. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, 19(3):393&#8211;404.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFire1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fire, A., Xu, S., Montgomery, M.&#x00A0;K., Kostas, S.&#x00A0;A., Driver, S.&#x00A0;E., and Mello,
C.&#x00A0;C. (1998).   Potent and specific genetic interference by double-stranded
RNA in Caenorhabditis elegans. <span 
class="phvro8t-x-x-120">Nature</span>, 391:806&#8211;811.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFodor2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Fodor, A.&#x00A0;a. and Aldrich, R.&#x00A0;W. (2009). Convergent evolution of alternative
splices at domain boundaries of the BK channel. <span 
class="phvro8t-x-x-120">Annual review of physiology</span>,
71:19&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XFuruichi1975"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Furuichi, Y. (1975). 5&#8217;-terminal m7G(5&#8217;)ppp(5&#8217;)Gmp In Vivo: Identification in
Reovirus Genome RNA. <span 
class="phvro8t-x-x-120">Proceedings of the </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, 72(2):742&#8211;745.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGan2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gan, H., Lin, X., Zhang, Z., Zhang, W., Liao, S., Wang, L., and Han, C.
(2011). piRNA profiling during specific stages of mouse spermatogenesis.
<span 
class="phvro8t-x-x-120">RNA</span>.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XGhildiyal2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ghildiyal, M., Seitz, H., Horwich, M.&#x00A0;D., Li, C., Du, T., Lee, S., Xu, J., Kittler,
E. L.&#x00A0;W., Zapp, M.&#x00A0;L., Weng, Z., and Zamore, P.&#x00A0;D. (2008).   Endogenous
siRNAs Derived from Transposons and mRNAs in Drosophila Somatic Cells.
<span 
class="phvro8t-x-x-120">Science</span>, 320(5879):1077&#8211;1081.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGhosh2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ghosh, S. and Jacobson, A. (2010). RNA decay modulates gene expression
and controls its fidelity. <span 
class="phvro8t-x-x-120">WIREs RNA</span>, 1:351&#8211;361.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGilbert1978a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gilbert, W. (1978). Why genes in pieces? <span 
class="phvro8t-x-x-120">Nature</span>, 271(5645):501.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGirard2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Girard, A., Sachidanandam, R., Hannon, G.&#x00A0;J., and Carmell, M.&#x00A0;A. (2006).
A germline-specific  class  of  small  RNAs  binds  mammalian  Piwi  proteins.
<span 
class="phvro8t-x-x-120">Nature</span>, 442(7099):199&#8211;202.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGlauser2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Glauser,  D.&#x00A0;A.,  Johnson,  B.&#x00A0;E.,  Aldrich,  R.&#x00A0;W.,  and  Goodman,  M.&#x00A0;B.
(2011).       Intragenic   alternative   splicing   coordination   is   essential   for
Caenorhabditis  elegans  slo-1  gene  function.   <span 
class="phvro8t-x-x-120">Proceedings  of  the  National</span>
<span 
class="phvro8t-x-x-120">Academy of Sciences of the United States of America</span>, 108(51):20790&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGrabherr2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grabherr, M.&#x00A0;G., Haas, B.&#x00A0;J., Yassour, M., Levin, J.&#x00A0;Z., Thompson, D.&#x00A0;A.,
Amit, I., Adiconis, X., Fan, L., Raychowdhury, R., Zeng, Q., Chen, Z., Mauceli,
E., Hacohen, N., Gnirke, A., Rhind, N., di&#x00A0;Palma, F., Birren, B.&#x00A0;W., Nusbaum,
C., Lindblad-Toh,  K.,  Friedman,  N.,  and  Regev,  A.  (2011).     Full-length
transcriptome  assembly  from  RNA-Seq  data  without  a  reference  genome.
<span 
class="phvro8t-x-x-120">Nature Biotechnology</span>, 29(7):644&#8211;652.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGrant2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grant, C.&#x00A0;E., Bailey, T.&#x00A0;L., and Noble, W.&#x00A0;S. (2011).  FIMO: scanning for
occurrences of a given motif. <span 
class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>, 27(7):1017&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGraveley2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Graveley, B.&#x00A0;R. (2000).  Sorting out the complexity of SR protein functions.
<span 
class="phvro8t-x-x-120">RNA (New York, N.Y.)</span>, 6(9):1197&#8211;211.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGraveley2005b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Graveley, B.&#x00A0;R. (2005).   Mutually exclusive splicing of the insect Dscam
                                                                                

                                                                                
pre-mRNA directed by competing intronic RNA secondary structures.  <span 
class="phvro8t-x-x-120">Cell</span>,
123(1):65&#8211;73.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGrivna2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Grivna, S.&#x00A0;T.,  Beyret,  E.,  Wang,  Z.,  and  Lin,  H.  (2006).   A  novel  class
of small RNAs  in  mouse  spermatogenic  cells.     <span 
class="phvro8t-x-x-120">Genes  &amp;  development</span>,
20(13):1709&#8211;14.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGu2010a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gu, A., Ji, G., Shi, X., Long, Y., Xia, Y., Song, L., Wang, S., and Wang,
X. (2010). Genetic Variants in Piwi-Interacting RNA Pathway Genes Confer
Susceptibility  to  Spermatogenic  Failure  in  a  Chinese  Population.   <span 
class="phvro8t-x-x-120">Human</span>
<span 
class="phvro8t-x-x-120">Reproduction</span>, 25(12):2955&#8211;2961.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGu2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gu, W., Lee, H.-C., Chaves, D., Youngman, E.&#x00A0;M., Pazour, G.&#x00A0;J., Conte,
D., and Mello, C.&#x00A0;C. (2012).  CapSeq and CIP-TAP identify Pol II start sites
and reveal  capped  small  RNAs  as  C.  elegans  piRNA  precursors.    <span 
class="phvro8t-x-x-120">Cell</span>,
151(7):1488&#8211;500.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGuenther2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Guenther,  M.&#x00A0;G.,  Levine,  S.&#x00A0;S.,  Boyer,  L.&#x00A0;A.,  Jaenisch,  R.,  and  Young,
R.&#x00A0;A. (2007).    A  chromatin  landmark  and  transcription  initiation  at  most
promoters in human cells. <span 
class="phvro8t-x-x-120">Cell</span>, 130:77&#8211;88.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XGupta2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Gupta, S.,  Stamatoyannopoulos,  J.&#x00A0;a.,  Bailey,  T.&#x00A0;L.,  and  Noble,  W.&#x00A0;S.
(2007). Quantifying similarity between motifs. <span 
class="phvro8t-x-x-120">Genome biology</span>, 8(2):R24.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHaase2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Haase, A.&#x00A0;D.,  Fenoglio,  S.,  Muerdter,  F.,  Guzzardo,  P.&#x00A0;M.,  Czech,  B.,
Pappin, D.&#x00A0;J., Chen, C., Gordon, A., and Hannon, G.&#x00A0;J. (2010).  Probing the
initiation and effector phases of the somatic piRNA pathway in Drosophila.
<span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 24(22):2499&#8211;504.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHafner2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hafner, M.,  Landgraf,  P.,  Ludwig,  J.,  Rice,  A.,  Ojo,  T.,  Lin,  C.,  Holoch,
D., Lim, C.,  and  Tuschl,  T.  (2008).   Identification  of  microRNAs  and  other
small regulatory RNAs using cDNA library sequencing. <span 
class="phvro8t-x-x-120">Methods (San Diego,</span>
<span 
class="phvro8t-x-x-120">Calif.)</span>, 44(1):3&#8211;12.
</p>
                                                                                

                                                                                
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHafner2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hafner, M., Landthaler, M., Burger, L., Khorshid, M., Hausser, J., Berninger,
P., Rothballer,  A.,  Ascano  Jr.,  M.,  Jungkamp,  A.-C.,  Munschauer,  M.,
Ulrich, A., Wardle,  G.&#x00A0;S.,  Dewell,  S.,  Zavolan,  M.,  and  Tuschl,  T.  (2010).
Transcriptome-wide  Identification  of  RNA-Binding  Protein  and  MicroRNA
Target Sites by PAR-CLIP. <span 
class="phvro8t-x-x-120">Cell</span>, 141(1):129&#8211;141.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHandler2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Handler,  D.,  Olivieri,  D.,  Novatchkova,  M.,  Gruber,  F.&#x00A0;S.,  Meixner,  K.,
Mechtler, K.,  Stark,  A.,  Sachidanandam,  R.,  and  Brennecke,  J.  (2011).
A systematic  analysis  of  Drosophila  TUDOR  domain-containing  proteins
identifies Vreteno and the Tdrd12 family as essential primary piRNA pathway
factors. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 30(19):3977&#8211;93.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHartig2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hartig, J.&#x00A0;V., Tomari, Y., and Frstemann, K. (2007).  piRNAs&#8211;the ancient
hunters of genome invaders. <span 
class="phvro8t-x-x-120">Genes &amp; development</span>, 21(14):1707&#8211;13.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHattori2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hattori, D., Chen, Y., Matthews, B.&#x00A0;J., Salwinski, L., Sabatti, C., Grueber,
W.&#x00A0;B., and  Zipursky,  S.&#x00A0;L.  (2009).     Robust  discrimination  between  self
and non-self  neurites  requires  thousands  of  Dscam1  isoforms.    <span 
class="phvro8t-x-x-120">Nature</span>,
461(7264):644&#8211;648.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHattori2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hattori, D.,  Millard,  S.&#x00A0;S.,  Wojtowicz,  W.&#x00A0;M.,  and  Zipursky,  S.&#x00A0;L.  (2008).
Dscam-Mediated   Cell   Recognition   Regulates   Neural   Circuit   Formation.
<span 
class="phvro8t-x-x-120">Annual Review of Cell and Developmental Biology</span>, 24(1):597&#8211;620.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHemani2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hemani,  Y.  and  Soller,  M.  (2012).    Mechanisms  of  Drosophila  Dscam
mutually exclusive  splicing  regulation.    <span 
class="phvro8t-x-x-120">Biochemical  Society  transactions</span>,
40(4):804&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo1997b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K., Etten, J. L.&#x00A0;V., and Shuman, S. (1997).  Characterization of an
ATP-dependent DNA ligase encoded by Chlorella virus PBCV-1.  <span 
class="phvro8t-x-x-120">Journal of</span>
<span 
class="phvro8t-x-x-120">virology</span>, 71(3).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo2002b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K. and Shuman, S. (2002). Bacteriophage T4 RNA ligase 2 (gp24.1)
                                                                                

                                                                                
exemplifies  a  family  of  RNA  ligases  found  in  all  phylogenetic  domains.
<span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences</span>, 99(20):12709&#8211;12714.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHo2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ho, C.&#x00A0;K., Wang, L.&#x00A0;K., Lima, C.&#x00A0;D., and Shuman, S. (2004).   Structure
and mechanism  of  RNA  ligase.      <span 
class="phvro8t-x-x-120">Structure  (London,  England:  1993)</span>,
12(2):327&#8211;339.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHoeffer2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hoeffer, C.&#x00A0;A., Sanyal, S., and Ramaswami, M. (2003).   Acute induction
of conserved synaptic signaling pathways in Drosophila melanogaster.  <span 
class="phvro8t-x-x-120">The</span>
<span 
class="phvro8t-x-x-120">Journal of neuroscience : the official journal of the Society for Neuroscience</span>,
23:6362&#8211;6372.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHorvath2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Horvath,   G.&#x00A0;C.,   Kistler,   M.&#x00A0;K.,   and   Kistler,   W.&#x00A0;S.   (2009).       RFX2
is a candidate   downstream   amplifier   of   A-MYB   regulation   in   mouse
spermatogenesis. <span 
class="phvro8t-x-x-120">BMC Developmental Biology</span>, 9(1):63.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHosokawa2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hosokawa, M., Shoji, M., Kitamura, K., Tanaka, T., Noce, T., Chuma, S.,
and Nakatsuji, N. (2007). Tudor-related proteins TDRD1/MTR-1, TDRD6 and
TDRD7/TRAP: domain composition, intracellular localization, and function in
male germ cells in mice. <span 
class="phvro8t-x-x-120">Developmental biology</span>, 301(1):38&#8211;52.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHouse2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> House,  A.&#x00A0;E.  and  Lynch,  K.&#x00A0;W.  (2008).      Regulation  of  alternative
splicing: more  than  just  the  ABCs.    <span 
class="phvro8t-x-x-120">The  Journal  of  Biological  Chemistry</span>,
283(3):1217&#8211;21.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHouwing2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Houwing, S., Kamminga, L.&#x00A0;M., Berezikov, E., Cronembold, D., Girard, A.,
van&#x00A0;den Elst, H., Filippov, D.&#x00A0;V., Blaser, H., Raz, E., Moens, C.&#x00A0;B., Plasterk,
R.&#x00A0;H., Hannon, G.&#x00A0;J., Draper, B.&#x00A0;W., and Ketting, R.&#x00A0;F. (2007).  A Role for
Piwi and piRNAs  in  Germ  Cell  Maintenance  and  Transposon  Silencing  in
Zebrafish. <span 
class="phvro8t-x-x-120">Cell</span>, 129(1):69&#8211;82.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHuang2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Huang, H., Gao, Q., Peng, X., Choi, S.-Y., Sarma, K., Ren, H., Morris, A.&#x00A0;J.,
and Frohman, M.&#x00A0;a. (2011). piRNA-associated germline nuage formation and
spermatogenesis  require  MitoPLD  profusogenic  mitochondrial-surface  lipid
                                                                                

                                                                                
signaling. <span 
class="phvro8t-x-x-120">Developmental cell</span>, 20(3):376&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XHughes2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Hughes,  T.&#x00A0;A.  (2006).     Regulation  of  gene  expression  by  alternative
untranslated regions. <span 
class="phvro8t-x-x-120">Trends in Genetics: TIG</span>, 22(3):119&#8211;22.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIngolia2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ingolia, N.&#x00A0;T., Ghaemmaghami, S., Newman, J. R.&#x00A0;S., and Weissman, J.&#x00A0;S.
(2009). Genome-wide analysis in vivo of translation with nucleotide resolution
using ribosome profiling. <span 
class="phvro8t-x-x-120">Science</span>, 324:218&#8211;223.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIngolia2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ingolia, N.&#x00A0;T.,  Lareau,  L.&#x00A0;F.,  and  Weissman,  J.&#x00A0;S.  (2011).    Ribosome
profiling of mouse embryonic stem cells reveals the complexity and dynamics
of mammalian proteomes. <span 
class="phvro8t-x-x-120">Cell</span>, 147(4):789&#8211;802.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XIpsaro2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ipsaro, J.&#x00A0;J., Haase, A.&#x00A0;D., Knott, S.&#x00A0;R., Joshua-Tor, L., and Hannon, G.&#x00A0;J.
(2012). The structural biochemistry of Zucchini implicates it as a nuclease in
piRNA biogenesis. <span 
class="phvro8t-x-x-120">Nature</span>, 491(7423):279&#8211;83.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJin2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Jin, Y., Suzuki,  H.,  Maegawa,  S.,  Endo,  H.,  Sugano,  S.,  Hashimoto,  K.,
Yasuda, K., and Inoue, K. (2003).  A vertebrate RNA-binding protein Fox-1
regulates tissue-specific splicing via the pentanucleotide GCAUG. <span 
class="phvro8t-x-x-120">The EMBO</span>
<span 
class="phvro8t-x-x-120">Journal</span>, 22(4):905&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJohnson2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Johnson, B.&#x00A0;E., Glauser, D.&#x00A0;A., Dan-Glauser, E.&#x00A0;S., Halling, D.&#x00A0;B., Aldrich,
R.&#x00A0;W., and Goodman, M.&#x00A0;B. (2011).  Alternatively Spliced Domains Interact
to Regulate  BK  Potassium  Channel  Gating.   <span 
class="phvro8t-x-x-120">Proceedings  of  the  National</span>
<span 
class="phvro8t-x-x-120">Academy of Sciences</span>, 108(51):20784&#8211;20789.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XJohnson2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Johnson,  J.&#x00A0;M.,  Castle,  J.,  Garrett-Engele,  P.,  Kan,  Z.,  Loerch,  P.&#x00A0;M.,
Armour, C.&#x00A0;D.,  Santos,  R.,  Schadt,  E.&#x00A0;E.,  Stoughton,  R.,  and  Shoemaker,
D.&#x00A0;D. (2003).     Genome-Wide  Survey  of  Human  Alternative  Pre-mRNA
Splicing with Exon Junction Microarrays. <span 
class="phvro8t-x-x-120">Science</span>, 302(5653):2141&#8211;2144.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKawaoka2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kawaoka,  S.,  Hara,  K.,  Shoji,  K.,  Kobayashi,  M.,  Shimada,  T.,  Sugano,
                                                                                

                                                                                
S., Tomari,  Y.,  Suzuki,  Y.,  and  Katsuma,  S.  (2012).    The  comprehensive
epigenome map of piRNA clusters. pages 1&#8211;10.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKennard1927"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kennard, E.&#x00A0;H. (1927). Zur Quantenmechanik einfacher Bewegungstypen.
<span 
class="phvro8t-x-x-120">Zeitschrift for Physik</span>, 44(4-5):326&#8211;352.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKent2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kent, W.&#x00A0;J.  (2002).    BLAT&#8212;The  BLAST-Like  Alignment  Tool.    <span 
class="phvro8t-x-x-120">Genome</span>
<span 
class="phvro8t-x-x-120">Research</span>, 12(4):656&#8211;664.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKim2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kim, V.&#x00A0;N., Han, J., and Siomi, M.&#x00A0;C. (2009).  Biogenesis of small RNAs in
animals. <span 
class="phvro8t-x-x-120">Nature reviews. Molecular cell biology</span>, 10(2):126&#8211;39.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKleppe1970b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>
Kleppe, K., Van de Sande, J.&#x00A0;H., and Khorana, H.&#x00A0;G. (1970). Polynucleotide
ligase-catalyzed joining of deoxyribo-oligonucleotides on ribopolynucleotide
templates and of ribo-oligonucleotides on deoxyribopolynucleotide templates.
<span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of</span>
<span 
class="phvro8t-x-x-120">America</span>, 67(1):68&#8211;73.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKojima2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kojima, K.,  Kuramochi-Miyagawa,  S.,  Chuma,  S.,  Tanaka,  T.,  Nakatsuji,
N., Kimura, T., and Nakano, T. (2009).  Associations between PIWI proteins
and TDRD1/MTR-1 are critical for integrated subcellular localization in murine
male germ cells. <span 
class="phvro8t-x-x-120">Genes to cells : devoted to molecular &amp; cellular mechanisms</span>,
14(10):1155&#8211;65.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKoren2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Koren, S.,  Schatz,  M.&#x00A0;C.,  Walenz,  B.&#x00A0;P.,  Martin,  J.,  Howard,  J.&#x00A0;T.,
Ganapathy, G., Wang, Z., Rasko, D.&#x00A0;a., McCombie, W.&#x00A0;R., Jarvis, E.&#x00A0;D., and
Adam M Phillippy (2012).  Hybrid error correction and de novo assembly of
single-molecule sequencing reads. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 30(7):693&#8211;700.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKreahling2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kreahling,  J.&#x00A0;M.  and  Graveley,  B.&#x00A0;R.  (2005).   The  iStem,  a  long-range
RNA secondary  structure  element  required  for  efficient  exon  inclusion
in the Drosophila  Dscam  pre-mRNA.     <span 
class="phvro8t-x-x-120">Molecular  and  Cellular  Biology</span>,
25(23):10251&#8211;60.
                                                                                

                                                                                
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKumar1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kumar, M. and Carmichael, G.&#x00A0;G. (1998).   Antisense RNA: function and
fate of duplex RNA in cells of higher eukaryotes. <span 
class="phvro8t-x-x-120">Microbiology and molecular</span>
<span 
class="phvro8t-x-x-120">biology reviews : MMBR</span>, 62(4):1415&#8211;34.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKuramochi-Miyagawa2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kuramochi-Miyagawa, S., Kimura, T., Ijiri, T.&#x00A0;W., Isobe, T., Asada, N., Fujita,
Y., Ikawa, M., Iwai, N., Okabe, M., Deng, W., Lin, H., Matsuda, Y., and Nakano,
T. (2004). Mili, a Mammalian Member of Piwi Family Gene, Is Essential for
Spermatogenesis. <span 
class="phvro8t-x-x-120">Development</span>, 131(4):839&#8211;849.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKuramochi-Miyagawa2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kuramochi-Miyagawa, S., Watanabe, T., Gotoh, K., Totoki, Y., Toyoda, A.,
Ikawa, M.,  Asada,  N.,  Kojima,  K.,  Yamaguchi,  Y.,  Ijiri,  T.&#x00A0;W.,  Hata,  K.,  Li,
E., Matsuda, Y., Kimura, T., Okabe, M., Sakaki, Y., Sasaki, H., and Nakano,
T. (2008).    DNA  Methylation  of  Retrotransposon  Genes  Is  Regulated  by
Piwi Family  Members  MILI  and  MIWI2  in  Murine  Fetal  Testes.   <span 
class="phvro8t-x-x-120">Genes  &amp;</span>
<span 
class="phvro8t-x-x-120">Development</span>, 22(7):908&#8211;917.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XKutter2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Kutter, C., Brown, G.&#x00A0;D., Gonalves, A., Wilson, M.&#x00A0;D., Watt, S., Brazma,
A., White, R.&#x00A0;J.,  and  Odom,  D.&#x00A0;T.  (2011).   Pol  III  binding  in  six  mammals
shows conservation among amino acid isotypes despite divergence among
tRNA genes. <span 
class="phvro8t-x-x-120">Nature genetics</span>, 43(10):948&#8211;55.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLadd2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ladd, A.&#x00A0;N.   and   Cooper,   T.&#x00A0;A.   (2002).        Finding   signals   that
regulate alternative  splicing  in  the  post-genomic  era.    <span 
class="phvro8t-x-x-120">Genome  Biology</span>,
3(11):reviews0008.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLaiho2013a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Laiho, A., Kotaja, N., Gyenesei, A., and Sironen, A. (2013).  Transcriptome
profiling of the murine testis during the first wave of spermatogenesis.  <span 
class="phvro8t-x-x-120">PloS</span>
<span 
class="phvro8t-x-x-120">one</span>, 8(4):e61558.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLander2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lander, E.&#x00A0;S. (2011). Initial impact of the sequencing of the human genome.
<span 
class="phvro8t-x-x-120">Nature</span>, 470(7333):187&#8211;97.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XLangmead2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Langmead, B., Trapnell, C., Pop, M., and Salzberg, S.&#x00A0;L. (2009).  Ultrafast
and memory-efficient  alignment  of  short  DNA  sequences  to  the  human
genome. <span 
class="phvro8t-x-x-120">Genome Biology</span>, 10(3):R25.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLatham1996"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Latham, K.&#x00A0;E., Litvin, J., Orth, J.&#x00A0;M., Patel, B., Mettus, R., and Reddy, E.&#x00A0;P.
(1996). Temporal patterns of A-myb and B-myb gene expression during testis
development. <span 
class="phvro8t-x-x-120">Oncogene</span>, 13:1161&#8211;1168.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLau2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lau, N.&#x00A0;C.,  Seto,  A.&#x00A0;G.,  Kim,  J.,  Kuramochi-Miyagawa,  S.,  Nakano,  T.,
Bartel, D.&#x00A0;P.,  and  Kingston,  R.&#x00A0;E.  (2006).   Characterization  of  the  piRNA
complex from rat testes. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 313(5785):363&#8211;367.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLee2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lee, H.-C., Gu, W., Shirayama, M., Youngman, E., Conte, D., and Mello,
C.&#x00A0;C. (2012).  C. elegans piRNAs mediate the genome-wide surveillance of
germline transcripts. <span 
class="phvro8t-x-x-120">Cell</span>, 150(1):78&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLenasi2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lenasi, T.,  Peterlin,  B.&#x00A0;M.,  and  Dovc,  P.  (2006).    Distal  regulation  of
alternative splicing by splicing enhancer in equine <span 
class="cmmi-12">&#x03B2;</span>-casein intron 1.  <span 
class="phvro8t-x-x-120">RNA</span>,
12(3):498 &#8211;507.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2009d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, H., Handsaker, B., Wysoker, A., Fennell, T., Ruan, J., Homer, N., Marth,
G., Abecasis,  G.,  and  Durbin,  R.  (2009a).   The  Sequence  Alignment/Map
format and SAMtools. <span 
class="phvro8t-x-x-120">Bioinformatics</span>, 25:2078&#8211;2079.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2012c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, H., Qiu,  J.,  and  Fu,  X.-D.  (2012).    RASL-seq  for  massively  parallel
and quantitative analysis of gene expression.  <span 
class="phvro8t-x-x-120">Current protocols in molecular</span>
<span 
class="phvro8t-x-x-120">biology / edited  by  Frederick  M.  Ausubel  ...  [et  al.]</span>,  Chapter  4(April):Unit
4.13.1&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, M., Wang, I.&#x00A0;X., Li, Y., Bruzel, A., Richards, A.&#x00A0;L., Toung, J.&#x00A0;M., and
Cheung, V.&#x00A0;G. (2011).  Widespread RNA and DNA sequence differences in
the human transcriptome. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 333(6038):53&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XLi2009a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;C., Barringer, B.&#x00A0;C., and Barbash, D.&#x00A0;a. (2009b).   The pachytene
checkpoint and its relationship to evolutionary patterns of polyploidization and
hybrid sterility. <span 
class="phvro8t-x-x-120">Heredity</span>, 102(1):24&#8211;30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;C.  and  Schimenti,  J.&#x00A0;C.  (2007).   Mouse  pachytene  checkpoint  2
(trip13) is required  for  completing  meiotic  recombination  but  not  synapsis.
<span 
class="phvro8t-x-x-120">PLoS genetics</span>, 3(8):e130.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2013h"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;Z., Roy, C.&#x00A0;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&#x00A0;W., Xu,
J., Moore, M.&#x00A0;J., Schimenti, J.&#x00A0;C., Weng, Z., and Zamore, P.&#x00A0;D. (2013a).  An
Ancient Transcription Factor Initiates the Burst of piRNA Production during
Early Meiosis in Mouse Testes. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, 50(1):1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2013e"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;Z., Roy, C.&#x00A0;K., Dong, X., Bolcun-Filas, E., Wang, J., Han, B.&#x00A0;W., Xu,
J., Moore, M.&#x00A0;J., Schimenti, J.&#x00A0;C., Weng, Z., and Zamore, P.&#x00A0;D. (2013b).  An
Ancient Transcription Factor Initiates the Burst of piRNA Production during
Early Meiosis in Mouse Testes. <span 
class="phvro8t-x-x-120">Molecular Cell</span>, pages 1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLi2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Li, X.&#x00A0;Z., Roy, C.&#x00A0;K., Moore, M.&#x00A0;J., and Zamore, P.&#x00A0;D. (2013c).  Defining
piRNA primary transcripts. <span 
class="phvro8t-x-x-120">Cell cycle (Georgetown, Tex.)</span>, 12(11):1657&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLicatalosi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Licatalosi, D.&#x00A0;D., Mele, A., Fak, J.&#x00A0;J., Ule, J., Kayikci, M., Chi, S.&#x00A0;W., Clark,
T.&#x00A0;A., Schweitzer, A.&#x00A0;C., Blume, J.&#x00A0;E., Wang, X., Darnell, J.&#x00A0;C., and Darnell,
R.&#x00A0;B. (2008).  HITS-CLIP yields genome-wide insights into brain alternative
RNA processing. <span 
class="phvro8t-x-x-120">Nature</span>, 456(7221):464&#8211;469.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLister2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lister, R., O&#8217;Malley, R.&#x00A0;C., Tonti-Filippini, J., Gregory, B.&#x00A0;D., Berry, C.&#x00A0;C.,
Millar, A.&#x00A0;H., and Ecker, J.&#x00A0;R. (2008). Highly integrated single-base resolution
maps of the epigenome in Arabidopsis. <span 
class="phvro8t-x-x-120">Cell</span>, 133(3):523&#8211;36.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLohman2013c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lohman, G. J.&#x00A0;S., Zhang, Y., Zhelkovsky, A.&#x00A0;M., Cantor, E.&#x00A0;J., Evans, T.&#x00A0;C.,
and Jr, T. C.&#x00A0;E. (2013).  Efficient DNA ligation in DNA &#8211; RNA hybrid helices
by Chlorella virus DNA ligase. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 42(36):1&#8211;14.
</p>
                                                                                

                                                                                
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLong2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Long, J.&#x00A0;C. and Caceres, J.&#x00A0;F. (2009).  The SR protein family of splicing
factors: master  regulators  of  gene  expression.    <span 
class="phvro8t-x-x-120">The  Biochemical  journal</span>,
417(1):15&#8211;27.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XLynch2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Lynch, K.&#x00A0;W. (2004). Consequences of regulated pre-mRNA splicing in the
immune system. <span 
class="phvro8t-x-x-120">Nat Rev Immunol</span>, 4(12):931&#8211;940.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMacilwain2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Macilwain, C. (2010). Scientists vs engineers: this time it&#8217;s financial. <span 
class="phvro8t-x-x-120">Nature</span>,
467(7318):885.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMangan2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mangan, S. and Alon, U. (2003). Structure and function of the feed-forward
loop network  motif.    <span 
class="phvro8t-x-x-120">Proceedings  of  the  National  Academy  of  Sciences</span>,
100(21):11980 &#8211;11985.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMarinov2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Marinov, G.&#x00A0;K., Williams, B.&#x00A0;a., McCue, K., Schroth, G.&#x00A0;P., Gertz, J., Myers,
R.&#x00A0;M., and Wold, B.&#x00A0;J. (2013).  From single-cell to cell-pool transcriptomes:
stochasticity in gene expression and RNA splicing. <span 
class="phvro8t-x-x-120">Genome research</span>, pages
496&#8211;510.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMarioni2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Marioni, J.&#x00A0;C.,  Mason,  C.&#x00A0;E.,  Mane,  S.&#x00A0;M.,  Stephens,  M.,  and  Gilad,  Y.
(2008). RNA-seq: an assessment of technical reproducibility and comparison
with gene expression arrays. <span 
class="phvro8t-x-x-120">Genome Research</span>, 18(9):1509&#8211;17.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMartinez-Contreras2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Martinez-Contreras,  R.,  Cloutier,  P.,  Shkreta,  L.,  Fisette,  J.-F.,  Revil,  T.,
and Chabot, B. (2007).  hnRNP proteins and splicing control.  <span 
class="phvro8t-x-x-120">Advances in</span>
<span 
class="phvro8t-x-x-120">experimental medicine and biology</span>, 623:123&#8211;147.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMaxam1977a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Maxam, A.&#x00A0;M. and Gilbert, W. (1992). A new method for sequencing DNA.
1977. <span 
class="phvro8t-x-x-120">Biotechnology (Reading, Mass.)</span>, 24(2):99&#8211;103.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMerkin2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Merkin, J., Russell, C., Chen, P., and Burge, C.&#x00A0;B. (2012).   Evolutionary
Dynamics of Gene and Isoform Regulation in Mammalian Tissues.  <span 
class="phvro8t-x-x-120">Science</span>,
                                                                                

                                                                                
338(6114):1593&#8211;1599.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMettus1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mettus, R.&#x00A0;V., Litvin, J., Wali, A., Toscani, A., Latham, K., Hatton, K., and
Reddy, E.&#x00A0;P.  (1994).   Murine  A-myb:  evidence  for  differential  splicing  and
tissue-specific expression. <span 
class="phvro8t-x-x-120">Oncogene</span>, 9(10):3077&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModrek2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modrek, B.  and  Lee,  C.  (2002).   A  genomic  view  of  alternative  splicing.
<span 
class="phvro8t-x-x-120">Nature Genetics</span>, 30(1):13&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModrich1973a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modrich,  P.,  Anraku,  Y.,  and  Lehman,  I.&#x00A0;R.  (1973).    Deoxyribonucleic
Acid Ligase  ISOLATION  AND  PHYSICAL  CHARACTERIZATION  OF  THE
HOMOGENEOUS  ENZYME  FROM  ESCHERICHIA  COLI.      <span 
class="phvro8t-x-x-120">Journal  of</span>
<span 
class="phvro8t-x-x-120">Biological Chemistry</span>, 248(21):7495&#8211;7501.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XModzelewski2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Modzelewski,  A.&#x00A0;J.,  Holmes,  R.&#x00A0;J.,  Hilz,  S.,  Grimson,  A.,  and  Cohen,
P.&#x00A0;E. (2012).   AGO4  regulates  entry  into  meiosis  and  influences  silencing
of sex chromosomes  in  the  male  mouse  germline.    <span 
class="phvro8t-x-x-120">Developmental  cell</span>,
23(2):251&#8211;64.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMontgomery1998"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Montgomery,  M.&#x00A0;K.,  Xu,  S.,  and  Fire,  a.  (1998).     RNA  as  a  target
of double-stranded  RNA-mediated  genetic  interference  in  Caenorhabditis
elegans. <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United</span>
<span 
class="phvro8t-x-x-120">States of America</span>, 95(26):15502&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMortazavi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mortazavi,  A.,  Williams,  B.&#x00A0;A.,  McCue,  K.,  Schaeffer,  L.,  and  Wold,  B.
(2008). Mapping and quantifying mammalian transcriptomes by RNA-Seq.
<span 
class="phvro8t-x-x-120">Nat Meth</span>, 5(7):621&#8211;628.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XMutz2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Mutz, K.-O.,  Heilkenbrinker,  A.,  Lnne,  M.,  Walter,  J.-G.,  and  Stahl,  F.
(2013). Transcriptome analysis using next-generation sequencing.  <span 
class="phvro8t-x-x-120">Current</span>
<span 
class="phvro8t-x-x-120">opinion in biotechnology</span>, 24(1):22&#8211;30.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNagalakshmi2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nagalakshmi, U., Wang, Z., Waern, K., Shou, C., Raha, D., Gerstein, M.,
                                                                                

                                                                                
and Snyder, M. (2008).  The transcriptional landscape of the yeast genome
defined by RNA sequencing. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 320(5881):1344&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNakano2003a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nakano, M., Komatsu, J., Matsuura, S.-i., Takashima, K., Katsura, S., and
Mizuno, A. (2003). Single-molecule PCR using water-in-oil emulsion. <span 
class="phvro8t-x-x-120">Journal</span>
<span 
class="phvro8t-x-x-120">of Biotechnology</span>, 102(2):117&#8211;124.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNamekawa2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Namekawa,  S.&#x00A0;H.  and  Lee,  J.&#x00A0;T.  (2009).     XY  and  ZW:  is  meiotic
sex chromosome   inactivation   the   rule   in   evolution?      <span 
class="phvro8t-x-x-120">PLoS   genetics</span>,
5(5):e1000493.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNandakumar2004a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nandakumar,   J.,   Ho,   C.&#x00A0;K.,   Lima,   C.&#x00A0;D.,   and   Shuman,   S.   (2004).
RNA substrate   specificity   and   structure-guided   mutational   analysis   of
bacteriophage  T4  RNA  ligase  2.     <span 
class="phvro8t-x-x-120">The  Journal  of  biological  chemistry</span>,
279(30):31337&#8211;47.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNandakumar2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nandakumar,  J.,  Shuman,  S.,  and  Lima,  C.&#x00A0;D.  (2006).     RNA  ligase
structures reveal the basis for RNA specificity and conformational changes
that drive ligation forward. <span 
class="phvro8t-x-x-120">Cell</span>, 127(1):71&#8211;84.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNEBEL1961"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> NEBEL, B.&#x00A0;R., AMAROSE, A.&#x00A0;P., and HACKET, E.&#x00A0;M. (1961).  Calendar
of gametogenic development in the prepuberal male mouse.  <span 
class="phvro8t-x-x-120">Science (New</span>
<span 
class="phvro8t-x-x-120">York, N.Y.)</span>, 134(3482):832&#8211;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNeves2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Neves, G.,  Zucker,  J.,  Daly,  M.,  and  Chess,  A.  (2004).    Stochastic  yet
biased expression of multiple Dscam splice variants by individual cells. <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">genetics</span>, 36(3):240&#8211;6.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNewburger2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Newburger, D.&#x00A0;E. and Bulyk, M.&#x00A0;L. (2009). UniPROBE: an online database
of protein binding microarray data on protein-DNA interactions. <span 
class="phvro8t-x-x-120">Nucleic acids</span>
<span 
class="phvro8t-x-x-120">research</span>, 37(Database issue):D77&#8211;82.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsen2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsen, T.&#x00A0;W.  and  Graveley,  B.&#x00A0;R.  (2010).   Expansion  of  the  eukaryotic
                                                                                

                                                                                
proteome by alternative splicing. <span 
class="phvro8t-x-x-120">Nature</span>, 463(7280):457&#8211;463.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsson2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsson,  M.,  Antson,  D.-O.,  Barbany,  G.,  and  Landegren,  U.  (2001).
RNA-templated DNA ligation for transcript analysis. <span 
class="phvro8t-x-x-120">Nucleic Acids Research</span>,
29(2):578&#8211;581.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNilsson2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nilsson, M.,  Barbany,  G.,  Antson,  D.&#x00A0;O.,  Gertow,  K.,  and  Landegren,  U.
(2000). Enhanced  detection  and  distinction  of  RNA  by  enzymatic  probe
ligation. <span 
class="phvro8t-x-x-120">Nature biotechnology</span>, 18(7):791&#8211;3.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XNishimasu2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Nishimasu,  H.,  Ishizu,  H.,  Saito,  K.,  Fukuhara,  S.,  Kamatani,  M.&#x00A0;K.,
Bonnefond, L., Matsumoto, N., Nishizawa, T., Nakanaga, K., Aoki, J., Ishitani,
R., Siomi, H., Siomi, M.&#x00A0;C., and Nureki, O. (2012).  Structure and function of
Zucchini endoribonuclease in piRNA biogenesis. <span 
class="phvro8t-x-x-120">Nature</span>, 491(7423):284&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOakberg1956b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Oakberg, E. and Oakberq, F. (1956).  A Description of Spermiogenesis in
the mouse and its use in analysis of the cycle of the seminiferous epithelium
and germ cell renewal. <span 
class="phvro8t-x-x-120">The American journal of anatomy</span>, 99(3):391&#8211;419.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOh1999"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Oh, I.&#x00A0;H. and Reddy, E.&#x00A0;P. (1999).   The myb gene family in cell growth,
differentiation and apoptosis. <span 
class="phvro8t-x-x-120">Oncogene</span>, 18(19):3017&#8211;33.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOlivera1967b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Olivera, B.  and  Lehman,  I.  (1967).   Linkage  of  polynucleotides  through
phosphodiester bonds by an enzyme from Escherichia coli.  <span 
class="phvro8t-x-x-120">Proceedings of</span>
<span 
class="phvro8t-x-x-120">the National Academy of </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, pages 1426&#8211;1433.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOlson2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Olson, S., Blanchette, M., Park, J., Savva, Y., Yeo, G.&#x00A0;W., Yeakley, J.&#x00A0;M.,
Rio, D.&#x00A0;C.,  and  Graveley,  B.&#x00A0;R.  (2007).    A  regulator  of  Dscam  mutually
exclusive splicing fidelity. <span 
class="phvro8t-x-x-120">Nat Struct Mol Biol</span>, 14(12):1134&#8211;1140.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOsella2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Osella, M.,  Bosia,  C.,  Cor,  D.,  and  Caselle,  M.  (2011).    The  role  of
incoherent microRNA-mediated feedforward loops in noise buffering.  <span 
class="phvro8t-x-x-120">PLoS</span>
<span 
class="phvro8t-x-x-120">computational biology</span>, 7(3):e1001101.
                                                                                

                                                                                
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XOsheim1985"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Osheim, Y.&#x00A0;N., Miller, O.&#x00A0;L., and Beyer, a.&#x00A0;L. (1985). RNP particles at splice
junction sequences on Drosophila chorion transcripts. <span 
class="phvro8t-x-x-120">Cell</span>, 43(1):143&#8211;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPan2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Pan, Q., Shai, O., Lee, L.&#x00A0;J., Frey, B.&#x00A0;J., and Blencowe, B.&#x00A0;J. (2008). Deep
surveying of alternative splicing complexity in the human transcriptome by
high-throughput sequencing. <span 
class="phvro8t-x-x-120">Nature Genetics</span>, 40(12):1413&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPane2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Pane, A., Wehr, K., and Schpbach, T. (2007). zucchini and squash encode
two putative  nucleases  required  for  rasiRNA  production  in  the  Drosophila
germline. <span 
class="phvro8t-x-x-120">Developmental cell</span>, 12(6):851&#8211;62.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPark2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Park, H.&#x00A0;Y.,  Lim,  H.,  Yoon,  Y.&#x00A0;J.,  Follenzi,  A.,  Nwokafor,  C.,  Meng,  X.,
Singer, R.&#x00A0;H.,  and  Yoon,  H.  (2014).    Visualization  of  dynamics  of  single
endogenous  mRNA  labeled  in  live  mouse.     <span 
class="phvro8t-x-x-120">Science  (New  York,  N.Y.)</span>,
343(6169):422&#8211;4.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPark2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Park, J.&#x00A0;W.  and  Graveley,  B.&#x00A0;R.  (2007).    Complex  alternative  splicing.
<span 
class="phvro8t-x-x-120">Advances in Experimental Medicine and Biology</span>, 623:50&#8211;63.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XParra2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Parra, M.&#x00A0;K., Gallagher, T.&#x00A0;L., Amacher, S.&#x00A0;L., Mohandas, N., and Conboy,
J.&#x00A0;G. (2012).    Deep  intron  elements  mediate  nested  splicing  events  at
consecutive AG dinucleotides to regulate alternative 3&#8217; splice site choice in
vertebrate 4.1 genes. <span 
class="phvro8t-x-x-120">Molecular and cellular biology</span>, 32(11):2044&#8211;53.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XParra2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Parra, M.&#x00A0;K.,  Tan,  J.&#x00A0;S.,  Mohandas,  N.,  and  Conboy,  J.&#x00A0;G.  (2008).
Intrasplicing coordinates alternative first exons with alternative splicing in the
protein 4.1R gene. <span 
class="phvro8t-x-x-120">EMBO J</span>, 27(1):122&#8211;131.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPeirson2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Peirson, S.&#x00A0;N. (2003).  Experimental validation of novel and conventional
approaches  to  quantitative  real-time  PCR  data  analysis.    <span 
class="phvro8t-x-x-120">Nucleic  Acids</span>
<span 
class="phvro8t-x-x-120">Research</span>, 31(14):73e&#8211;73.
</p>
                                                                                

                                                                                
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPelisson1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Plisson, a., Song, S.&#x00A0;U., Prud&#8217;homme, N., Smith, P.&#x00A0;a., Bucheton, a., and
Corces, V.&#x00A0;G.  (1994).   Gypsy  transposition  correlates  with  the  production
of a retroviral  envelope-like  protein  under  the  tissue-specific  control  of  the
Drosophila flamenco gene. <span 
class="phvro8t-x-x-120">The EMBO journal</span>, 13(18):4401&#8211;11.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPeng2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Peng, T.,  Xue,  C.,  Bi,  J.,  Li,  T.,  Wang,  X.,  Zhang,  X.,  and  Li,  Y.  (2008).
Functional importance of different patterns of correlation between adjacent
cassette exons in human and mouse. <span 
class="phvro8t-x-x-120">BMC Genomics</span>, 9(1):191.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPlocik2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Plocik, A. M.&#x00A0;A. and Graveley, B.&#x00A0;R. (2013).   New insights from existing
sequence data: generating breakthroughs without a pipette.  <span 
class="phvro8t-x-x-120">Molecular cell</span>,
49(4):605&#8211;17.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XPurandare2014b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Purandare, S.&#x00A0;R., Tenhumberg, B., and Brisson, J.&#x00A0;a. (2014). Comparison
of the wing polyphenic response of pea aphids ( Acyrthosiphon pisum ) to
crowding and predator cues. <span 
class="phvro8t-x-x-120">Ecological Entomology</span>, 39(2):263&#8211;266.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XQuinlan2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Quinlan,  A.&#x00A0;R.  and  Hall,  I.&#x00A0;M.  (2010).    BEDTools:  a  flexible  suite  of
utilities for comparing genomic features.   <span 
class="phvro8t-x-x-120">Bioinformatics (Oxford, England)</span>,
26(6):841&#8211;842.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XReuter2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Reuter, M., Berninger, P., Chuma, S., Shah, H., Hosokawa, M., Funaya,
C., Antony, C., Sachidanandam, R., and Pillai, R.&#x00A0;S. (2011).  Miwi catalysis
is required for piRNA amplification-independent LINE1 transposon silencing.
<span 
class="phvro8t-x-x-120">Nature</span>, advance on.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XReuter2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Reuter, M., Chuma, S., Tanaka, T., Franz, T., Stark, A., and Pillai, R.&#x00A0;S.
(2009). Loss  of  the  Mili-interacting  Tudor  domain-containing  protein-1
activates transposons  and  alters  the  Mili-associated  small  RNA  profile.
<span 
class="phvro8t-x-x-120">Nature structural &amp; molecular biology</span>, 16(6):639&#8211;46.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRichardson1965a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Richardson, C. (1965).   Phosphorylation of Nucleic Acid by and Enzyme
from T4 Bacteriophate-infected Escherichia Coli. <span 
class="phvro8t-x-x-120">Proceedings of the National</span>
                                                                                

                                                                                
<span 
class="phvro8t-x-x-120">Academy of </span><span 
class="phvro8t-x-x-120">&#x2026;</span>, 1372(1961):158&#8211;165.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRiley2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Riley, K.&#x00A0;J.  and  Steitz,  J.&#x00A0;A.  (2013).   Minireview  The  &#8216;&#8216;  Observer  Effect
&#8217;&#8217; in Genome-wide Surveys of Protein-RNA Interactions Minireview.   pages
2011&#8211;2014.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRo2007"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ro, S.,   Park,   C.,   Song,   R.,   Nguyen,   D.,   Jin,   J.,   Sanders,   K.&#x00A0;M.,
McCarrey, J.&#x00A0;R., and Yan, W. (2007).   Cloning and Expression Profiling of
Testis-Expressed piRNA-Like RNAs. <span 
class="phvro8t-x-x-120">RNA</span>, 13(10):1693&#8211;1702.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRoach1995"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Roach, J.  J.&#x00A0;C.,  Boysen,  C.,  Wang,  K.,  Hood,  L.,  and  Wang,  I.  K.  A.&#x00A0;I.
(1995). Pairwise End Sequencing: A unified approach to genomic mapping
and sequencing. <span 
class="phvro8t-x-x-120">Genomics</span>, 353:345&#8211;353.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRobine2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span>                     Robine,                     N.,                     Lau,                     N.&#x00A0;C.,
Balla, S., Jin, Z., Okamura, K., Kuramochi-Miyagawa, S., Blower, M.&#x00A0;D., and
Lai, E.&#x00A0;C. (2009). A Broadly Conserved Pathway Generates 3&#8217; UTR-Directed
Primary piRNAs. <span 
class="phvro8t-x-x-120">Current Biology</span>, 19(24):2066&#8211;2076.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRomanienko2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Romanienko,  P.&#x00A0;J.  and  Camerini-Otero,  R.&#x00A0;D.  (2000).      The  mouse
Spo11 gene is required for meiotic chromosome synapsis.   <span 
class="phvro8t-x-x-120">Molecular cell</span>,
6(5):975&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRonaghi1998a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ronaghi, M., Uhln, M., and Nyrn, P. (1998). A sequencing method based
on real-time pyrophosphate. <span 
class="phvro8t-x-x-120">Science</span>, 281:363, 365.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XRumsfeld2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Rumsfeld, D. (2011). Known and unknown: a memoir.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSaito2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Saito, K.,  Inagaki,  S.,  Mituyama,  T.,  Kawamura,  Y.,  Ono,  Y.,  Sakota,  E.,
Kotani, H.,  Asai,  K.,  Siomi,  H.,  and  Siomi,  M.&#x00A0;C.  (2009).    A  regulatory
circuit for piwi  by  the  large  Maf  gene  traffic  jam  in  Drosophila.    <span 
class="phvro8t-x-x-120">Nature</span>,
461(7268):1296&#8211;9.
</p>
                                                                                

                                                                                
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSalehi-Ashtiani2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Salehi-Ashtiani, K., Yang, X., Derti, A., Tian, W., Hao, T., Lin, C., Makowski,
K., Shen, L., Murray, R.&#x00A0;R., Szeto, D., Tusneem, N., Smith, D.&#x00A0;R., Cusick,
M.&#x00A0;E., Hill,  D.&#x00A0;E.,  Roth,  F.&#x00A0;P.,  and  Vidal,  M.  (2008).    Isoform  discovery
by targeted  cloning,  &#8217;deep-well&#8217;  pooling  and  parallel  sequencing.    <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Methods</span>, 5(7):597&#8211;600.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1975a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F.  and  Coulson,  A.&#x00A0;R.  (1975).   A  rapid  method  for  determining
sequences in DNA by primed synthesis with DNA polymerase.   <span 
class="phvro8t-x-x-120">Journal of</span>
<span 
class="phvro8t-x-x-120">molecular biology</span>, 94(3):441&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1978a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F., Coulson, A.&#x00A0;R., Friedmann, T., Air, G.&#x00A0;M., Barrell, B.&#x00A0;G., Brown,
N.&#x00A0;L., Fiddes,  J.&#x00A0;C.,  Hutchison,  C.&#x00A0;A.,  Slocombe,  P.&#x00A0;M.,  and  Smith,  M.
(1978). The  nucleotide  sequence  of  bacteriophage  phiX174.   <span 
class="phvro8t-x-x-120">Journal  of</span>
<span 
class="phvro8t-x-x-120">molecular biology</span>, 125(2):225&#8211;46.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSanger1977b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sanger, F.,  Nicklen,  S.,  and  Coulson,  A.&#x00A0;R.  (1977).    DNA  sequencing
with chain-terminating inhibitors.   <span 
class="phvro8t-x-x-120">Proceedings of the National Academy of</span>
<span 
class="phvro8t-x-x-120">Sciences of the United States of America</span>, 74(12):5463&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchena1995a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schena,   M.,   Shalon,   D.,   Davis,   R.&#x00A0;W.,   and   Brown,   P.&#x00A0;O.   (1995).
Quantitative monitoring of gene expression patterns with a complementary
DNA microarray. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 270(5235):467&#8211;70.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchmucker2000"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schmucker, D., Clemens, J.&#x00A0;C., Shu, H., Worby, C.&#x00A0;A., Xiao, J., Muda, M.,
Dixon, J.&#x00A0;E., and Zipursky, S. (2000). Drosophila Dscam Is an Axon Guidance
Receptor Exhibiting Extraordinary Molecular Diversity. <span 
class="phvro8t-x-x-120">Cell</span>, 101(6):671&#8211;684.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchoenmakers2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schoenmakers, S., Wassenaar, E., Hoogerbrugge, J.&#x00A0;W., Laven, J. S.&#x00A0;E.,
Grootegoed,  J.&#x00A0;A.,  and  Baarends,  W.&#x00A0;M.  (2009).     Female  meiotic  sex
chromosome inactivation in chicken. <span 
class="phvro8t-x-x-120">PLoS genetics</span>, 5(5):e1000466.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSchwarzbauer1983"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Schwarzbauer, J.&#x00A0;E., Tamkun, J.&#x00A0;W., Lemischka, I.&#x00A0;R., and Hynes, R.&#x00A0;O.
(1983). Three different fibronectin mRNAs arise by alternative splicing within
                                                                                

                                                                                
the coding region. <span 
class="phvro8t-x-x-120">Cell</span>, 35(2 Pt 1):421&#8211;31.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSeitz2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Seitz, H.,  Ghildiyal,  M.,  and  Zamore,  P.&#x00A0;D.  (2008).    Argonaute  loading
improves the 5&#8217; precision of both MicroRNAs and their miRNA* strands in
flies. <span 
class="phvro8t-x-x-120">Current biology : CB</span>, 18(2):147&#8211;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShalek2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shalek, A.&#x00A0;K., Satija, R., Adiconis, X., Gertner, R.&#x00A0;S., Gaublomme, J.&#x00A0;T.,
Raychowdhury, R., Schwartz, S., Yosef, N., Malboeuf, C., Lu, D., Trombetta,
J.&#x00A0;T., Gennert,  D.,  Gnirke,  A.,  Goren,  A.,  Hacohen,  N.,  Levin,  J.&#x00A0;Z.,  Park,
H., and Regev, A. (2013).   Single-cell transcriptomics reveals bimodality in
expression and splicing in immune cells. <span 
class="phvro8t-x-x-120">Nature</span>, pages 1&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShapiro2013b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shapiro,   E.,   Biezuner,   T.,   and   Linnarsson,   S.   (2013).      Single-cell
sequencing-based  technologies  will  revolutionize  whole-organism  science.
<span 
class="phvro8t-x-x-120">Nature reviews. Genetics</span>, 14(9):618&#8211;630.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSharp2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sharp, P.&#x00A0;A. (2014). Nobel Lectures in Physiology or Medicine 1991-1995.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShen-Orr2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shen-Orr, S.&#x00A0;S., Milo, R., Mangan, S., and Alon, U. (2002). Network motifs
in the transcriptional regulation network of Escherichia coli.  <span 
class="phvro8t-x-x-120">Nature genetics</span>,
31(1):64&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure,  J.  and  Aiden,  E.&#x00A0;L.  (2012).    The  expanding  scope  of  DNA
sequencing. <span 
class="phvro8t-x-x-120">Nature Biotechnology</span>, 30(11):1084&#8211;1094.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure, J. and Ji, H. (2008).  Next-generation DNA sequencing.  <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">Biotechnology</span>, 26(10):1135&#8211;1145.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShendure2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure, J., Mitra, R.&#x00A0;D., Varma, C., and Church, G.&#x00A0;M. (2004). Advanced
sequencing  technologies:  methods  and  goals.   <span 
class="phvro8t-x-x-120">Nature  Reviews.  Genetics</span>,
5(5):335&#8211;44.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XShendure2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shendure, J., Porreca, G.&#x00A0;J., Reppas, N.&#x00A0;B., Lin, X., McCutcheon, J.&#x00A0;P.,
Rosenbaum,  A.&#x00A0;M.,  Wang,  M.&#x00A0;D.,  Zhang,  K.,  Mitra,  R.&#x00A0;D.,  and  Church,
G.&#x00A0;M. (2005). Accurate Multiplex Polony Sequencing of an Evolved Bacterial
Genome. <span 
class="phvro8t-x-x-120">Science</span>, 309(5741):1728&#8211;1732.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShepard2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shepard, P.&#x00A0;J., Choi, E.-A., Lu, J., Flanagan, L.&#x00A0;A., Hertel, K.&#x00A0;J., and Shi,
Y. (2011). Complex and dynamic landscape of RNA polyadenylation revealed
by PAS-Seq. <span 
class="phvro8t-x-x-120">RNA</span>, 17(4):761 &#8211;772.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShi2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shi, L.   and   Lee,   T.   (2012).       Molecular   diversity   of   Dscam   and
self-recognition.        <span 
class="phvro8t-x-x-120">Advances   in   experimental   medicine   and   biology</span>,
739:262&#8211;75.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShingleton2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shingleton, A., Sisk, G., and Stern, D. (2003).  Diapause in the pea aphid
(Acyrthosiphon pisum) is a slowing but not a cessation of development. <span 
class="phvro8t-x-x-120">BMC</span>
<span 
class="phvro8t-x-x-120">developmental biology</span>, 12:1&#8211;12.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShiraki2003a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shiraki, T.,  Kondo,  S.,  Katayama,  S.,  Waki,  K.,  Kasukawa,  T.,  Kawaji,
H., Kodzius,  R.,  Watahiki,  A.,  Nakamura,  M.,  Arakawa,  T.,  Fukuda,  S.,
Sasaki, D.,   Podhajska,   A.,   Harbers,   M.,   Kawai,   J.,   Carninci,   P.,   and
Hayashizaki, Y. (2003).   Cap analysis gene expression for high-throughput
analysis of transcriptional starting point and identification of promoter usage.
<span 
class="phvro8t-x-x-120">Proceedings of the National Academy of Sciences of the United States of</span>
<span 
class="phvro8t-x-x-120">America</span>, 100:15776&#8211;15781.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShirayama2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shirayama, M., Seth, M., Lee, H.-C., Gu, W., Ishidate, T., Conte, D., and
Mello, C.&#x00A0;C. (2012). piRNAs initiate an epigenetic memory of nonself RNA in
the C. elegans germline. <span 
class="phvro8t-x-x-120">Cell</span>, 150(1):65&#8211;77.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XShiroguchi2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shiroguchi, K., Jia, T.&#x00A0;Z., Sims, P.&#x00A0;a., and Xie, X.&#x00A0;S. (2012).  Digital RNA
sequencing minimizes sequence-dependent bias and amplification noise with
optimized single-molecule barcodes.  <span 
class="phvro8t-x-x-120">Proceedings of the National Academy</span>
<span 
class="phvro8t-x-x-120">of Sciences of the United States of America</span>, 109(4):1347&#8211;52.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XShoji2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Shoji, M.,  Tanaka,  T.,  Hosokawa,  M.,  Reuter,  M.,  Stark,  A.,  Kato,  Y.,
Kondoh, G., Okawa, K., Chujo, T., Suzuki, T., Hata, K., Martin, S.&#x00A0;L., Noce,
T., Kuramochi-Miyagawa, S., Nakano, T., Sasaki, H., Pillai, R.&#x00A0;S., Nakatsuji,
N., and Chuma,  S.  (2009).    The  TDRD9-MIWI2  complex  is  essential  for
piRNA-mediated  retrotransposon  silencing  in  the  mouse  male  germline.
<span 
class="phvro8t-x-x-120">Developmental cell</span>, 17(6):775&#8211;87.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSiomi2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Siomi, M.&#x00A0;C., Sato, K., Pezic, D., and Aravin, A.&#x00A0;A. (2011). PIWI-interacting
small RNAs:  the  vanguard  of  genome  defence.    <span 
class="phvro8t-x-x-120">Nat  Rev  Mol  Cell  Biol</span>,
12(4):246&#8211;258.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSmith2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Smith, J.&#x00A0;M.,  Bowles,  J.,  Wilson,  M.,  Teasdale,  R.&#x00A0;D.,  and  Koopman,  P.
(2004). Expression of the tudor-related gene Tdrd5 during development of
the male germline in mice. <span 
class="phvro8t-x-x-120">Gene expression patterns : GEP</span>, 4(6):701&#8211;5.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSouthern2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Southern, E.&#x00A0;M. (2001). DNA microarrays. History and overview. <span 
class="phvro8t-x-x-120">Methods</span>
<span 
class="phvro8t-x-x-120">in Molecular Biology (Clifton, N.J.)</span>, 170:1&#8211;15.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSriskanda1998c"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sriskanda,   V.   and   Shuman,   S.   (1998).      Specificity   and   fidelity   of
strand joining  by  Chlorella  virus  DNA  ligase.     <span 
class="phvro8t-x-x-120">Nucleic  acids  research</span>,
26(15):3536&#8211;41.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XStaden1979"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Staden, R.  (1979).   A  strategy  of  DNA  sequencing  employing  computer
programs. <span 
class="phvro8t-x-x-120">Nucleic Acids Research</span>, 6(7):2601&#8211;2610.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSubtelny2014"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Subtelny, A.&#x00A0;O., Eichhorn, S.&#x00A0;W., Chen, G.&#x00A0;R., Sive, H., and Bartel, D.&#x00A0;P.
(2014). Poly(A)-tail  profiling  reveals  an  embryonic  switch  in  translational
control. <span 
class="phvro8t-x-x-120">Nature</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSultan2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Sultan, M., Schulz, M.&#x00A0;H., Richard, H., Magen, A., Klingenhoff, A., Scherf,
M., Seifert,  M.,  Borodina,  T.,  Soldatov,  A.,  Parkhomchuk,  D.,  Schmidt,  D.,
O&#8217;Keeffe, S., Haas, S., Vingron, M., Lehrach, H., and Yaspo, M.-L. (2008).  A
Global View of Gene Activity and Alternative Splicing by Deep Sequencing of
the Human Transcriptome. <span 
class="phvro8t-x-x-120">Science</span>, 321(5891):956&#8211;960.
                                                                                

                                                                                
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XSummers1970b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Summers, W. and Siegel, R. (1970).  Transcription of Late Phage RNA by
T7 RNA Polymerase. <span 
class="phvro8t-x-x-120">Nature</span>, 228:1160&#8211;1162.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTabor1987a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Tabor, S. (1987).   DNA Ligases.   <span 
class="phvro8t-x-x-120">Current Protocols in Molecular Biology</span>,
pages 3.14.1&#8211;3.14.4.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTani2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Tani, H.,  Mizutani,  R.,  Salam,  K.&#x00A0;A.,  Tano,  K.,  Ijiri,  K.,  Wakamatsu,  A.,
Isogai, T., Suzuki, Y., and Akimitsu, N. (2012).  Genome-wide determination
of RNA stability  reveals  hundreds  of  short-lived  noncoding  transcripts  in
mammals. <span 
class="phvro8t-x-x-120">Genome research</span>, 22(5):947&#8211;56.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTazi2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Tazi, J.,  Bakkour,  N.,  and  Stamm,  S.  (2009).    Alternative  splicing  and
disease. <span 
class="phvro8t-x-x-120">Biochimica et biophysica acta</span>, 1792(1):14&#8211;26.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XThierry-Mieg2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Thierry-Mieg, D. and Thierry-Mieg, J. (2006).  AceView: a comprehensive
cDNA-supported gene and transcripts annotation.  <span 
class="phvro8t-x-x-120">Genome biology</span>, 7 Suppl
1(Suppl 1):S12.1&#8211;14.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XThomson2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Thomson,  T.  and  Lin,  H.  (2009).   The  biogenesis  and  function  of  PIWI
proteins and  piRNAs:  progress  and  prospect.    <span 
class="phvro8t-x-x-120">Annual  review  of  cell  and</span>
<span 
class="phvro8t-x-x-120">developmental biology</span>, 25:355&#8211;76.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XToscani1997"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Toscani,  A.,  Mettus,  R.&#x00A0;V.,  Coupland,  R.,  Simpkins,  H.,  Litvin,  J.,  Orth,
J., Hatton,  K.&#x00A0;S.,  and  Reddy,  E.&#x00A0;P.  (1997).    Arrest  of  spermatogenesis
and defective   breast   development   in   mice   lacking   A-myb.       <span 
class="phvro8t-x-x-120">Nature</span>,
386(6626):713&#8211;717.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrapnell2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell, C., Pachter, L., and Salzberg, S.&#x00A0;L. (2009).  TopHat: discovering
splice junctions with RNA-Seq. <span 
class="phvro8t-x-x-120">Bioinformatics</span>, 25(9):1105 &#8211;1111.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrapnell2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell,  C.,  Roberts,  A.,  Goff,  L.,  Pertea,  G.,  Kim,  D.,  Kelley,  D.&#x00A0;R.,
                                                                                

                                                                                
Pimentel, H., Salzberg, S.&#x00A0;L., Rinn, J.&#x00A0;L., and Pachter, L. (2012). Differential
gene and transcript expression analysis of RNA-seq experiments with TopHat
and Cufflinks. <span 
class="phvro8t-x-x-120">Nature Protocols</span>, 7(3):562&#8211;578.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrapnell2010"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trapnell,  C.,  Williams,  B.&#x00A0;A.,  Pertea,  G.,  Mortazavi,  A.,  Kwan,  G.,  van
Baren, M.&#x00A0;J., Salzberg, S.&#x00A0;L., Wold, B.&#x00A0;J., and Pachter, L. (2010). Transcript
assembly and quantification by RNA-Seq reveals unannotated transcripts and
isoform switching during cell differentiation. <span 
class="phvro8t-x-x-120">Nat Biotech</span>, 28(5):511&#8211;515.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XTrauth1994"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Trauth, K., Mutschler, B., Jenkins, N.&#x00A0;A., Gilbert, D.&#x00A0;J., Copeland, N.&#x00A0;G.,
and Klempnauer, K.-h. (1994). Mouse A-myb encodes a trans-activator and is
expressed in mitotically active cells of the developing central nervous system,
adult testis and B lymphocytes. <span 
class="phvro8t-x-x-120">EMBO</span>, 13(24):5994&#8211;6005.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XUle2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Ule, J.,  Jensen,  K.,  Mele,  A.,  and  Darnell,  R.&#x00A0;B.  (2005).     CLIP:  A
method for identifying protein-RNA interaction sites in living cells.  <span 
class="phvro8t-x-x-120">Methods</span>,
37(4):376&#8211;386.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V.,  Klenov,  M.&#x00A0;S.,  Stolyarenko,  A.&#x00A0;D.,  and  Kotelnikov,  R.&#x00A0;N.
(2004). The  RNA  Interference  Proteins  and  Vasa  Locus  are  Involved  in
the Silencing  of  Retrotransposons  in  the  Female  Germline  of  Drosophila
melanogaster. <span 
class="phvro8t-x-x-120">RNA Biology</span>, (June):54&#8211;58.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2006"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V., Sigova, A., Li, C., Seitz, H., Gvozdev, V., and Zamore, P.&#x00A0;D.
(2006). A Distinct Small RNA Pathway Silences Selfish Genetic Elements in
the Germline. <span 
class="phvro8t-x-x-120">Science</span>, 313(5785):320 &#8211;324.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVagin2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vagin, V.&#x00A0;V.,  Wohlschlegel,  J.,  Qu,  J.,  Jonsson,  Z.,  Huang,  X.,  Chuma,
S., Girard, A., Sachidanandam, R., Hannon, G.&#x00A0;J., and Aravin, A.&#x00A0;a. (2009).
Proteomic  analysis  of  murine  Piwi  proteins  reveals  a  role  for  arginine
methylation in specifying interaction with Tudor family members.   <span 
class="phvro8t-x-x-120">Genes &amp;</span>
<span 
class="phvro8t-x-x-120">development</span>, 23(15):1749&#8211;62.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVasileva2009"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vasileva, A., Tiedau, D., Firooznia, A., Mller-Reichert, T., and Jessberger,
                                                                                

                                                                                
R. (2009).       Tdrd6   is   required   for   spermiogenesis,   chromatoid   body
architecture,  and  regulation  of  miRNA  expression.   <span 
class="phvro8t-x-x-120">Current  biology  :  CB</span>,
19(8):630&#8211;9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVelculescu1995a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Velculescu,   V.&#x00A0;E.,   Zhang,   L.,   Vogelstein,   B.,   and   Kinzler,   K.&#x00A0;W.
(1995). Serial  analysis  of  gene  expression.    <span 
class="phvro8t-x-x-120">Science  (New  York,  N.Y.)</span>,
270(5235):484&#8211;7.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVenter2008a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Venter, J.  (2007).     <span 
class="phvro8t-x-x-120">A  life  decoded:  my  genome,  my  life</span>.     Penguin
(Non-Classics).
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVenter2001"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Venter, J.&#x00A0;C., Adams, M.&#x00A0;D., Myers, E.&#x00A0;W., Li, P.&#x00A0;W., Mural, R.&#x00A0;J., Sutton,
G.&#x00A0;G., Smith, H.&#x00A0;O., Yandell, M., Evans, C.&#x00A0;a., Holt, R.&#x00A0;a., Gocayne, J.&#x00A0;D.,
Amanatides,  P.,  Ballew,  R.&#x00A0;M.,  Huson,  D.&#x00A0;H.,  Wortman,  J.&#x00A0;R.,  Zhang,  Q.,
Kodira, C.&#x00A0;D.,  Zheng,  X.&#x00A0;H.,  Chen,  L.,  Skupski,  M.,  Subramanian,  G.,
Thomas, P.&#x00A0;D., Zhang, J., Gabor Miklos, G.&#x00A0;L., Nelson, C., Broder, S., Clark,
a.&#x00A0;G., Nadeau, J., McKusick, V.&#x00A0;a., Zinder, N., Levine, a.&#x00A0;J., Roberts, R.&#x00A0;J.,
Simon, M., Slayman, C., Hunkapiller, M., Bolanos, R., Delcher, A., Dew, I.,
Fasulo, D., Flanigan, M., Florea, L., Halpern, A., Hannenhalli, S., Kravitz, S.,
Levy, S., Mobarry, C., Reinert, K., Remington, K., Abu-Threideh, J., Beasley,
E., Biddick, K., Bonazzi, V., Brandon, R., Cargill, M., Chandramouliswaran, I.,
Charlab, R., Chaturvedi, K., Deng, Z., Di Francesco, V., Dunn, P., Eilbeck, K.,
Evangelista, C., Gabrielian, a.&#x00A0;E., Gan, W., Ge, W., Gong, F., Gu, Z., Guan,
P., Heiman, T.&#x00A0;J., Higgins, M.&#x00A0;E., Ji, R.&#x00A0;R., Ke, Z., Ketchum, K.&#x00A0;a., Lai, Z.,
Lei, Y., Li, Z., Li, J., Liang, Y., Lin, X., Lu, F., Merkulov, G.&#x00A0;V., Milshina, N.,
Moore, H.&#x00A0;M., Naik, a.&#x00A0;K., Narayan, V.&#x00A0;a., Neelam, B., Nusskern, D., Rusch,
D.&#x00A0;B., Salzberg, S., Shao, W., Shue, B., Sun, J., Wang, Z., Wang, A., Wang,
X., Wang, J.,  Wei,  M.,  Wides,  R.,  Xiao,  C.,  Yan,  C.,  Yao,  A.,  Ye,  J.,  Zhan,
M., Zhang, W., Zhang, H., Zhao, Q., Zheng, L., Zhong, F., Zhong, W., Zhu,
S., Zhao, S., Gilbert, D., Baumhueter, S., Spier, G., Carter, C., Cravchik, A.,
Woodage, T., Ali, F., An, H., Awe, A., Baldwin, D., Baden, H., Barnstead, M.,
Barrow, I., Beeson, K., Busam, D., Carver, A., Center, A., Cheng, M.&#x00A0;L., Curry,
L., Danaher,  S.,  Davenport,  L.,  Desilets,  R.,  Dietz,  S.,  Dodson,  K.,  Doup,
L., Ferriera,  S.,  Garg,  N.,  Gluecksmann,  A.,  Hart,  B.,  Haynes,  J.,  Haynes,
C., Heiner,  C.,  Hladun,  S.,  Hostin,  D.,  Houck,  J.,  Howland,  T.,  Ibegwam,
C., Johnson, J., Kalush, F., Kline, L., Koduru, S., Love, A., Mann, F., May,
D., McCawley, S., McIntosh, T., McMullen, I., Moy, M., Moy, L., Murphy, B.,
Nelson, K.,  Pfannkoch,  C.,  Pratts,  E.,  Puri,  V.,  Qureshi,  H.,  Reardon,  M.,
                                                                                

                                                                                
Rodriguez, R., Rogers, Y.&#x00A0;H., Romblad, D., Ruhfel, B., Scott, R., Sitter, C.,
Smallwood,  M.,  Stewart,  E.,  Strong,  R.,  Suh,  E.,  Thomas,  R.,  Tint,  N.&#x00A0;N.,
Tse, S., Vech, C., Wang, G., Wetter, J., Williams, S., Williams, M., Windsor,
S., Winn-Deen,  E.,  Wolfe,  K.,  Zaveri,  J.,  Zaveri,  K.,  Abril,  J.&#x00A0;F.,  Guig,  R.,
Campbell, M.&#x00A0;J., Sjolander, K.&#x00A0;V., Karlak, B., Kejariwal, A., Mi, H., Lazareva,
B., Hatton, T., Narechania, A., Diemer, K., Muruganujan, A., Guo, N., Sato, S.,
Bafna, V., Istrail, S., Lippert, R., Schwartz, R., Walenz, B., Yooseph, S., Allen,
D., Basu, A., Baxendale, J., Blick, L., Caminha, M., Carnes-Stine, J., Caulk, P.,
Chiang, Y.&#x00A0;H., Coyne, M., Dahlke, C., Mays, A., Dombroski, M., Donnelly, M.,
Ely, D., Esparham, S., Fosler, C., Gire, H., Glanowski, S., Glasser, K., Glodek,
A., Gorokhov, M., Graham, K., Gropman, B., Harris, M., Heil, J., Henderson,
S., Hoover, J., Jennings, D., Jordan, C., Jordan, J., Kasha, J., Kagan, L., Kraft,
C., Levitsky, A., Lewis, M., Liu, X., Lopez, J., Ma, D., Majoros, W., McDaniel,
J., Murphy,  S.,  Newman,  M.,  Nguyen,  T.,  Nguyen,  N.,  Nodell,  M.,  Pan,  S.,
Peck, J., Peterson, M., Rowe, W., Sanders, R., Scott, J., Simpson, M., Smith,
T., Sprague, A., Stockwell, T., Turner, R., Venter, E., Wang, M., Wen, M., Wu,
D., Wu, M., Xia, A., Zandieh, A., and Zhu, X. (2001).  The sequence of the
human genome. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>, 291(5507):1304&#8211;51.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XViollet2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Viollet, S., Fuchs, R.&#x00A0;T., Munafo, D.&#x00A0;B., Zhuang, F., and Robb, G.&#x00A0;B. (2011).
T4 RNA Ligase  2  truncated  active  site  mutants:  improved  tools  for  RNA
analysis. <span 
class="phvro8t-x-x-120">BMC Biotechnology</span>, 11(1):72.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XVourekas2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Vourekas, A., Zheng, Q., Alexiou, P., Maragkakis, M., Kirino, Y., Gregory,
B.&#x00A0;D., and Mourelatos, Z. (2012). Mili and Miwi target RNA repertoire reveals
piRNA biogenesis and function of Miwi in spermiogenesis.  <span 
class="phvro8t-x-x-120">Nature Structural</span>
<span 
class="phvro8t-x-x-120">&amp; Molecular Biology</span>, 19(8):773&#8211;781.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wang, E.&#x00A0;T.,  Sandberg,  R.,  Luo,  S.,  Khrebtukova,  I.,  Zhang,  L.,  Mayr,
C., Kingsmore, S.&#x00A0;F., Schroth, G.&#x00A0;P., and Burge, C.&#x00A0;B. (2008).   Alternative
isoform regulation in human tissue transcriptomes. <span 
class="phvro8t-x-x-120">Nature</span>, 456(7221):470&#8211;6.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWang2002b"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wang, L.&#x00A0;K.,  Lima,  C.&#x00A0;D.,  and  Shuman,  S.  (2002).     Structure  and
mechanism of T4 polynucleotide kinase: an RNA repair enzyme. <span 
class="phvro8t-x-x-120">The EMBO</span>
<span 
class="phvro8t-x-x-120">journal</span>, 21(14):3873&#8211;80.
</p>
<p class="bibitem" ><span class="biblabel">
                                                                                

                                                                                
<a 
 id="XWatanabe2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watanabe, T., Chuma, S., Yamamoto, Y., Kuramochi-Miyagawa, S., Totoki,
Y., Toyoda, A., Hoki, Y., Fujiyama, A., Shibata, T., Sado, T., Noce, T., Nakano,
T., Nakatsuji, N., Lin, H., and Sasaki, H. (2011a). MITOPLD Is a Mitochondrial
Protein Essential for Nuage Formation and piRNA Biogenesis in the Mouse
Germline. <span 
class="phvro8t-x-x-120">Developmental Cell</span>, 20(3):364&#8211;375.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatanabe2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watanabe,  T.,  Tomizawa,  S.-i.,  Mitsuya,  K.,  Totoki,  Y.,  Yamamoto,  Y.,
Kuramochi-Miyagawa, S., Iida, N., Hoki, Y., Murphy, P.&#x00A0;J., Toyoda, A., Gotoh,
K., Hiura, H., Arima, T., Fujiyama, A., Sado, T., Shibata, T., Nakano, T., Lin, H.,
Ichiyanagi, K., Soloway, P.&#x00A0;D., and Sasaki, H. (2011b).  Role for piRNAs and
Noncoding RNA in de Novo DNA Methylation of the Imprinted Mouse Rasgrf1
Locus. <span 
class="phvro8t-x-x-120">Science</span>, 332(6031):848 &#8211;852.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson2005"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson, F.&#x00A0;L., Pttmann-Holgado, R., Thomas, F., Lamar, D.&#x00A0;L., Hughes,
M., Kondo, M., Rebel, V.&#x00A0;I., and Schmucker, D. (2005). Extensive diversity of
Ig-superfamily proteins in the immune system of insects. <span 
class="phvro8t-x-x-120">Science (New York,</span>
<span 
class="phvro8t-x-x-120">N.Y.)</span>, 309(5742):1874&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson, J.&#x00A0;D.,  Gann,  A.,  and  Witkowski,  J.  (2012).   <span 
class="phvro8t-x-x-120">The  Annotated  and</span>
<span 
class="phvro8t-x-x-120">Illustrated Double Helix</span>. Simon &amp; Schuster.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWatson1953a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Watson, J. D.&#x00A0;J. and Crick, F. (1953).  Molecular structure of nucleic acids.
<span 
class="phvro8t-x-x-120">Nature</span>, 4356(171):737&#8211;738.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWei1975"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wei, C.&#x00A0;M., Gershowitz, a., and Moss, B. (1975).  Methylated nucleotides
block 5&#8217; terminus of HeLa cell messenger RNA. <span 
class="phvro8t-x-x-120">Cell</span>, 4(4):379&#8211;86.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWeiss1967a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Weiss, B. and Richardson, C. (1967).  Enzymatic breakage and joining of
deoxyribonucleic acid, I. Repair of single-strand breaks in DNA by an enzyme
system from Escherichia coli infected with T4 bacteriophage. <span 
class="phvro8t-x-x-120">Proceedings of</span>
<span 
class="phvro8t-x-x-120">the National Academy of </span><span 
class="phvro8t-x-x-120">&#x2026;</span>.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWeston1992"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Weston, K. (1992). Extension of the DNA binding consensus of the chicken
c-Myb and v-Myb proteins. <span 
class="phvro8t-x-x-120">Nucleic acids research</span>, 20(12).
                                                                                

                                                                                
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWhite2011a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> White, E.&#x00A0;S.   and   Muro,   A.&#x00A0;F.   (2011).      Fibronectin   splice   variants:
understanding  their  multiple  roles  in  health  and  disease  using  engineered
mouse models. <span 
class="phvro8t-x-x-120">IUBMB life</span>, 63(7):538&#8211;46.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XWills2013"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Wills, Q.&#x00A0;F., Livak, K.&#x00A0;J., Tipping, A.&#x00A0;J., Enver, T., Goldson, A.&#x00A0;J., Sexton,
D.&#x00A0;W., and  Holmes,  C.  (2013).      Single-cell  gene  expression  analysis
reveals genetic  associations  masked  in  whole-tissue  experiments.   <span 
class="phvro8t-x-x-120">Nature</span>
<span 
class="phvro8t-x-x-120">biotechnology</span>, 31(8):748&#8211;752.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYabuta2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yabuta, Y.,  Ohta,  H.,  Abe,  T.,  Kurimoto,  K.,  Chuma,  S.,  and  Saitou,
M. (2011).    TDRD5  is  required  for  retrotransposon  silencing,  chromatoid
body assembly,  and  spermiogenesis  in  mice.   <span 
class="phvro8t-x-x-120">The  Journal  of  cell  biology</span>,
192(5):781&#8211;95.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYang2011"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yang, Y., Zhan, L., Zhang, W., Sun, F., Wang, W., Tian, N., Bi, J., Wang, H.,
Shi, D., Jiang, Y., Zhang, Y., and Jin, Y. (2011).  RNA secondary structure in
mutually exclusive splicing. <span 
class="phvro8t-x-x-120">Nat Struct Mol Biol</span>, 18(2):159&#8211;168.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYeakley2002"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yeakley, J.&#x00A0;M., Fan, J.-B., Doucet, D., Luo, L., Wickham, E., Ye, Z., Chee,
M.&#x00A0;S., and Fu, X.-D. (2002). Profiling alternative splicing on fiber-optic arrays.
<span 
class="phvro8t-x-x-120">Nature Biotechnology</span>, 20(4):353&#8211;358.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XYin2003d"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Yin, S., Ho, C.&#x00A0;K., and Shuman, S. (2003).  Structure-function analysis of
T4 RNA ligase 2. <span 
class="phvro8t-x-x-120">The Journal of biological chemistry</span>, 278(20):17601&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhan2004"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhan, X.-L.,  Clemens,  J.&#x00A0;C.,  Neves,  G.,  Hattori,  D.,  Flanagan,  J.&#x00A0;J.,
Hummel, T.,  Vasconcelos,  M.&#x00A0;L.,  Chess,  A.,  and  Zipursky,  S.&#x00A0;L.  (2004).
Analysis of  Dscam  Diversity  in  Regulating  Axon  Guidance  in  Drosophila
Mushroom Bodies. <span 
class="phvro8t-x-x-120">Neuron</span>, 43(5):673&#8211;686.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2012"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, F.,  Wang,  J.,  Xu,  J.,  Zhang,  Z.,  Koppetsch,  B.&#x00A0;S.,  Schultz,
N., Vreven,  T.,  Meignin,  C.,  Davis,  I.,  Zamore,  P.&#x00A0;D.,  Weng,  Z.,  and
                                                                                

                                                                                
Theurkauf, W.&#x00A0;E. (2012a). UAP56 Couples piRNA Clusters to the Perinuclear
Transposon Silencing Machinery. <span 
class="phvro8t-x-x-120">Cell</span>, 151(4):871&#8211;884.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, Y., Liu, T., Meyer, C.&#x00A0;a., Eeckhoute, J., Johnson, D.&#x00A0;S., Bernstein,
B.&#x00A0;E., Nusbaum, C., Myers, R.&#x00A0;M., Brown, M., Li, W., and Liu, X.&#x00A0;S. (2008).
Model-based analysis of ChIP-Seq (MACS). <span 
class="phvro8t-x-x-120">Genome biology</span>, 9(9):R137.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhang2012a"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhang, Z.,  Theurkauf,  W.&#x00A0;E.,  Weng,  Z.,  and  Zamore,  P.&#x00A0;D.  (2012b).
Strand-specific  libraries  for  high  throughput  RNA  sequencing  (  RNA-Seq  )
prepared without poly ( A ) selection. <span 
class="phvro8t-x-x-120">Silence</span>, 3(1):9.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZhu2003"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zhu, J., Shendure,  J.,  Mitra,  R.&#x00A0;D.,  and  Church,  G.&#x00A0;M.  (2003).   Single
molecule profiling of alternative pre-mRNA splicing. <span 
class="phvro8t-x-x-120">Science (New York, N.Y.)</span>,
301(5634):836&#8211;8.
</p>
<p class="bibitem" ><span class="biblabel">
<a 
 id="XZikherman2008"></a><span class="bibsp">&#x00A0;&#x00A0;&#x00A0;</span></span> Zikherman, J. and Weiss, A. (2008).  Alternative Splicing of CD45: The Tip
of the Iceberg. <span 
class="phvro8t-x-x-120">Immunity</span>, 29(6):839&#8211;841.
</p>
</div>
 
</body></html> 

                                                                                


