% !TEX root = /Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.tex
\chapter{Introduction}  \label{Intro}
\lhead{Chapter 1. \emph{Introduction}}
%----- -----------------------------------------------------------------------------------

\section{Fixed Genomes and Flexible Genes}
  \label{Intro:sec:Fixed Genomes and Flexible Genes}

  Exodus tells of the liberation of the Israelites from Egyptian slavery. Humble and reluctant Moses, their divine-appointed messiah, attempts to force the Pharaoh Ramses to release the Israelites through inflicting 10 plagues. Pharaoh is stalwart and stubborn as water turns to blood and the streets are flooded with frogs, lice, and flies. As livestock falls dead from disease, people and animals both are covered in boils, and the land burns in storms of fire, Pharaoh does not bend.

  The 8th plague was a swarm of Locusts, described in Exodus 10: 14-15:

  \begin{quote}
    \itshape
    \singlespacing
    \textsuperscript{14} And the locusts went up over all the land of Egypt, and rested in all the coasts of Egypt: very grievous were they; before them there were no such locusts as they, neither after them shall be such.\\
      \\
    \textsuperscript{15} For they covered the face of the whole earth, so that the land was darkened; and they did eat every herb of the land, and all the fruit of the trees which the hail had left: and there remained not any green thing in the trees, or in the herbs of the field, through all the land of Egypt.
    \end{quote}

  The desolation of a locust plague was still not enough to persuade Ramses. Nor was three days of darkness. Only after the death of all first-born Egyptians, including Ramses own son, was Pharaoh persuaded to liberate the Israelites.

  Locust swarms are not biblical fantasy. Today the United Nations' Food and Agriculture division maintains a \href{http://www.fao.org/ag/locusts/en/info/info/news/index.html}{Locust watch website} that provides weekly updates on potential swarms in northern Africa and the Middle East. Locusts have long been, and continue to be, a powerful and feared force of Nature.

  Unlike fire and brimstone, locusts are something that can be observed and studied. What triggers a swarm? We know that the desert locust, \locusts{}, is the one of 10 species that swarm and cause massive crop damage. \locusts{} are in the insect Order Orthoptera, along with crickets and katydids. Orthoptern members make sounds known as \textit{stridulation} by vigorously rubbing their wings, making for a noisy cloud of devastation. They only weigh 0.05--0.07 ounces and are less than 2.5 inches long but can consume their own body weight in vegetation per day. One \href{http://animaldiversity.ummz.umich.edu/site/accounts/information/Melanoplus_spretus.html}{swarm} of the infamous, and now curiously extinct, Rocky Mountain locust contained 12.2 \textit{trillion} insects. Its estimated total weight was 27.5 \textit{million} tons. The swarm covered almost 200 square miles (2/3 the size of Manhattan), and could travel 60 miles in a day. A locust swarm is truly a modern biblical plague.

  \begin{figure} % Desert Locus Figure
    \centering
    \includegraphics[width=10cm,keepaspectratio]{Figures/Intro/DesertLocust.jpeg}
    \caption[The Solitary and Gregarious forms of \locusts{}]
    {
      The Solitary and Gregarious forms of \locusts{}\\[0.25cm]
      The two phenotypic forms of \locusts{} appear very different. The solitary form is green and generally larger, while its gregarious form is more brightly colored, smaller, and swarms in vast numbers. Photo from \href{http://www.wikicommons.com}{Wikicommons}.
      }
    \label{Intro:fig:Locust}
    \end{figure}

  By definition swarms are temporary; the movement, en masse, from one location to another. But where do 12.2 trillion locusts go when not swarming? Does anyone care if their crops aren't under assault? It seemed no one cared until 1921 when an important realization was made.

  The power and destruction \locusts{} can inflict makes it difficult to believe that they are nothing more than common grasshoppers. Nothing more than grasshoppers not just by analogy, but by actual \textit{Taxonomy}. ``Desert'' locusts are actually the \textit{gregarious} form  of \locusts{} (Figure \ref{Intro:fig:Locust}), while the more familiar and docile looking ``grasshopper'' is the \textit{solitary form}. How does such a dichotomy exist within the same organism---indeed the same \textit{genome}?

  \locusts{} are \textit{polyphenic}, meaning that they have multiple (poly) physical forms (phenotypes). Polyphenism is a general feature among insects. These phenotypes are often extremely different. For example, pea aphids (\textit{Acyrthosiphon pisum}), which usually exist in an asexually reproducing, wingless female form, respond to reduced food supply and overcrowding by producing winged sexually-reproducing offspring. Winged organisms travel to new sources of food and revert back to the asexually reproducing form \citep{Shingleton2003,Purandare2014b}. In the case of \locusts{}, the gregarious form is smaller and more brightly colored compared to its solitary cousins. This transformation can happen in as little as two hours. What is the underlying cause of this transformation?

  In 2009, \citet{Anstey2009} reported that after two hours of forced crowding \locusts{} displayed elevated levels of the neurotransmitter serotonin in the ganglia (brain). Serotonin levels were strongly correlated with other gregarious form indicators. Serotonin regulates neuronal junctions and wiring in the brain \citep{Hoeffer2003}. Through the integration of environmental and social cues, the grasshopper brain can be re-wired, resulting in tremendous changes in behavior and phenotype. These changes prepare the organism to deal with a different world. It allows the organism to survive. Survival that is to the detriment of surrounding agriculture.

  In an extremely interesting \href{http://aeon.co/magazine/nature-and-cosmos/why-its-time-to-lay-the-selfish-gene-to-rest/}{article}, David Dobbs compares the two forms of \locusts{} to that of Dr. Jekyll and Mr. Hyde, the principle characters in the Robert Louis Stevenson novella. For Dr. Jekyll in fiction, and for \locusts{} in reality, the power to morph into multiple forms demonstrates the incredible power of a fixed genome yet plastic gene expression.

  It is often said that something is ``in the genes.'' Another oft-heard idiom that is perhaps more appropriate is: ``it's how you use them.'' This thesis will illustrate that, with ever increasing resolution in the measurement of functional gene products (i.e. the ``transcriptome''), we are beginning to realize the tremendous diversity and complexity of gene expression.

\section{Nucleic Acid Sequencing}
  \label{Intro:sec:Nucleic Acid Sequencing}

  \subsection{DNA Sequencing}
    \label{Intro:subsec:DNA Sequencing History}

    That DNA is the source of genetic information in all living organisms was first realized in 1953 \citep{Watson1953a}. The \textit{``pretty''} and \textit{``elegant''} arrangement of complementary, antiparallel, DNA strands captivated everyone, including one of DNAs co-discoverers \citep{Watson2012a}. Yet it took 25 years after the structure was known to be able to determine specific arrangements of nucleotide bases in a given length of DNA (i.e. to ``sequence''). By 1977, two completely different methods developed by Sanger \citep{Sanger1975a,Sanger1977b} and Maxam-Gilbert \citep{Maxam1977a} were reported. These sequencing technologies, from then on referred to eponymously as ``Sanger'' or ``Maxam-Gilbert'' sequencing, were used to determine the specific order of a small piece of DNA (200--300 nt). Over the next 35 years, DNA sequences were slowly cloned, sequenced, analyzed, and dutifully cataloged into knowledge.

    During the late 1970's and throughout the 1980's, DNA sequences were typically communicated in important publications \citep{Cordell1980a,Sanger1978a}. The birth of the Internet in the 1990's allowed publicly-funded repositories to store sequence information \citep{Benson2011a}. Yet it took the human genome project to transform tedious and balkanized DNA sequencing efforts into an organized process capable of assembling complex genomes \citep{Lander2011a,Venter2001}. An often criticized, but undeniably disrupting force in the human genome project was the competing efforts by the privately-owned company Celera \citep{Venter2008a}. Instead of assigning specific sections of the genome to be worked out by individual labs, Celera centralized the efforts by collecting many of the best ``high-throughput'' Sanger-sequencing devices from Agilent (ABI 3700 DNA Analyzer). Celera used a ``shotgun'' sequencing approach \citep{Staden1979}, combined with sequence scaffolds from the publicly-funded project, to quickly assemble a high-quality genome. Arguably, this was the first deep sequencing effort. Coincident with the beginning of a new millennium. It changed the landscape of molecular and biochemical research.

  \subsection{High-throughput Sequencing}
    \label{Intro:subsec: History of HTS}

    Sanger's DNA sequencing technology remains a valuable tool for every biological scientist. However, Sanger sequencing has a practical throughput limit. Each DNA molecule to be sequenced must be isolated, cloned, and amplified---using bacteria. Given that the human genome \citep{Hattori2005a} comprises >3 billion bp, and each Sanger reaction provides \textasciitilde800 nt of quality sequence, at least \textasciitilde4 million individual reactions are needed to determine the sequence of the human genome. This number assumes all ``reads'' are of sufficient quality, length, and do not overlap by even 1 nt.

    Even the best practical improvements to Sanger work-flows could not bring the technology in-line with aspirations of analyzing many species and/or organisms. The early 2000's saw multiple efforts to improve the scale of DNA sequencing, first using Massively Parallel Signature sequencing (MPSS) \citep{Brenner2000a}, but perhaps more importantly, by Pyro- \citep{Ronaghi1998a} and Polony-sequencing \citep{Shendure2005}. Both pyro- and polony sequencing use emulsion PCR \citep{Nakano2003a} for clonal amplification prior to sequencing, removing the bottleneck of bacterial cloning. In contrast to Sanger sequencing, where fluorescence signal from the last incorporated chain-terminating nucleotide is observed, pyrosequencing visualizes light given off by luciferase reacting with pyrophosphate (PPi), a by-product of nucleotide incorporation. This approach was later commercialized by 454 technologies. Polony sequencing involves a sequencing-by-ligation method, eventually commercialized by Applied BioSystems and branded as SOLiD sequencing.
    % ER Comment: Suggests figure showing how these two technologies work
    While both of these technologies provided valuable high-throughput sequences, neither has been as successful as the approach commercialized by Solexa, now known as Illumina.

    Illumina sequencers use a sequencing-by-synthesis approach. After clonal amplification of DNA on a slide surface \citep{Bentley2008}, fluorescent nucleotides are visualized as they are incorporated into the growing DNA strand. Iterations of the Illumina platform (e.g. GE, GE-II(x), HiSeq, HiSeq 2500, Hi X) have demonstrated steady and impressive increases in both read depth and length. On February 15th 2012, Illumina announced on its \href{http://blog.basespace.illumina.com/}{Basespace blog}, that they had sequenced a haplotype map (HapMap) sample at 40X coverage, using the HiSeq 2500 platform and paired-end 100 nt reads in a single run. On January 14th, 2014, Illumina \href{http://bit.ly/PZpegZ}{announced} its HiSeq X system, the first platform to truly attain the benchmark \$1,000 genome \citep{Service2006,Hayden2014}. These machines demonstrate that sequencing genomes is no longer the monumental endeavor it once was and that completely new experimental possibilities are a reality for life science researchers (Figure \ref{Intro:fig:SeqCosts}).

    \begin{figure} % Sequence Costs over time
      \centering
      \includegraphics{Figures/Intro/Sequencing_costs_over_time.eps}
      \caption[Cost of sequencing the human genome over time]
      {
        Cost of sequencing the human genome over time\\[0.25cm]
        The costs of sequencing the human genome has decreased on a log scale over a 10 year period due to major improvements in high-throughput sequencing. Data from Wetterstrand KA. DNA Sequencing Costs: Data from the NHGRI Genome Sequencing Program (GSP) Available at: \url{www.genome.gov/sequencingcosts}. Accessed 2014-05-04).
        }
      \label{Intro:fig:SeqCosts}
      \end{figure}

  \subsection{RNA Sequencing}
    \label{Intro:subsec:Types of HTS}

    The first widely-accepted large scale method used to measure gene expression was Serial Analysis of Gene Expression (SAGE) \citep{Velculescu1995a}. SAGE, like the before mentioned MPSS technique, produces a digital output of gene expression using a clever procedure of restriction endonucleolytic cDNA cleavage. Cleaved-product sticky ends are concatenated together to form long DNA fragments.
    % ER Comment: Suggests SAGE figure showing how it works
    Fragments are cloned into a vector, amplified, and Sanger sequenced. Using known sequences incorporated during concatenation, the number of sequenced fragments that align to a given gene is related to the abundance of the original RNA molecule. A clever molecular trick, SAGE allowed researchers to dip into the 5-log range of mRNA expression. However, the technique is still limited by Sanger sequencing read lengths and depth.

    After SAGE but prior to second generation high-throughput sequencing (HTS) technology, microarrays were the goto approach for gene expression analysis. The importance of microarrays in the measurement of gene expression cannot be overstated \citep{Shendure2008,Marioni2008}. However, limitations of novel sequence discovery combined with analogue signal, make the relevance of microarray technology limited in application to complete isoform discovery and annotation (see Section\ref{Intro:sec:Isoform Problem} and Figure \ref{Intro:fig:NoConnectivityInHTSMethods}). Yet Microarray technology is still very relevant. For example, use of microarrays in the recent definition of the developing human brain transcriptome---where sample was precious and quantification of known genes was of more importance than novel isoform discovery--was a prudent analysis platform choice \cite{Miller2014}.

    Different from SAGE and microarrays, the Solexa/Illumina platform relies on clonal amplification of a single template directly on a slide surface and is therefore not restricted by bacterial cloning. This new form of ``massively parrallel'' amplification uses imaging of clonal cDNA spots with sensitive digital cameras during sequential addition of fluorescent nucleotides (\textit{sequencing by synthesis}). Machines from Illumina turned out to posses the right mix for a ``second generation'' HTS platform. Soon after the Solexa/Illumina platform achieved read lengths of sufficient length and depth to measure gene expression, the first RNA-Seq papers were published \citep{Mortazavi2008, Nagalakshmi2008,Lister2008}, providing a glimpse into the future of molecular biology. Indeed, in the years since, analysis by RNA-Seq has quickly overtaken other forms of gene expression analysis, as demonstrated by the number of accessions created in the publically-funded repository of sequencing data, the Gene Expression OmniBus (GEO) \citep{Barrett2013}. RNA-Seq allows for digital quantification of RNA expression across more physiologically-relevant ranges \citep{Blencowe2009}, novel sequence discovery, measuring RNA-editing \citep{Li2011}, and fuels the novel area of transcript assembly \citep{Trapnell2010}. Through modification of the basic protocol or performing additional biochemical steps, RNA-Seq can be used to investigate many aspects of RNA biology (Figure \ref{Intro:fig:htsMethods} and \citep{Mutz2013}).

    \begin{figure} % RNA Sequencing Methodologies
      \centering
      \includegraphics{Figures/Intro/RNA_Sequencing_methodologies.eps}
      \caption[Methods for High-throughput sequencing of RNA]
      {
      Methods for High-throughput sequencing of RNA\\[0.25cm]
      In the short years since the first report of RNA-Seq, many variations have been reported. The figure above provides an incomplete graphical illustration of these variations. A more complete list of ``*Seq'' applications is maintained on this \href{http://liorpachter.wordpress.com/seq/}{blog} \url{http://liorpachter.wordpress.com/seq}.
      }
      \label{Intro:fig:htsMethods}
      \end{figure}

    Numerous methodologies enrich RNA-Seq libraries for particular types of RNA. For example, measurement of nascent transcripts can be performed via Global Nuclear Run-on sequencing (GRO-Seq) \citep{Core2008a}, and the extremely complicated process of RNA turnover (referring to the rates at which RNAs both are produced and degraded) has been examined \citep{Ghosh2010a, Tani2012}. RNA::Protein interactions can be measured with or without cross-linking the protein to the RNA, via CLIP or RIP, respectively (see section \ref{Disc:subsec:Labeling of precursors}) \citep{Ule2005,Licatalosi2006,Singh2013}. Once an RNA has been fully transcribed, known processing steps such as 5\textprime~7meG CAP formation and poly(A)+ tail formation can be measured using any of the Cap-Seq/CAGE \citep{Shiraki2003a} or PAS/TAIL/PAL methodologies \citep{Shepard2011, Chang2014b, Subtelny2014}. Importantly, nascent RNA can also be captured via Cap-Seq \citep{Kruesi2013}. With appropriate size-selection steps, small RNAs \citep{Ghildiyal2008} can also be captured. Finally, traditional RNA-Seq can capture many of the same RNA fragments as the above mentioned methods, even though it is mainly associated with measurement of traditional mRNA.

    RNA-Seq (and all its flavors) are traditionally associated with quantification of RNA obtained from \textit{many} tissue culture cells or bulk pieces of tissue. Recently, efforts to measure RNA expression in a single cell has gained attention \citep{Shapiro2013b}. Perhaps the most interesting concept concerning single-cell gene expression is the ``biological uncertainty principle'', wherein it is possible to either know, or change --- but not both---~the RNA composition of a single cell. The name borrows from Heisenberg's uncertainty principle \citep{Kennard1927} and is often confused with the more appropriate ``Observer effect'' \citep{Riley2013}. Leaving that issue aside, measuring the unique transcriptome of a cell is surely an exciting and informative endeavor \citep{Marinov2013, Shalek2013b,Wills2013}. Compared to DNA, the diversity of RNA synthesis within living cells is more complicated \citep{Shendure2012} and the ability to accurately measure RNA dynamics among cells should allow for more informative observations concerning biology than is currently possible using bulk measurements.
    %MO Comment : "This last PP feels a little mis-placed or irrelevant - I think you want to talk about the future of RNA Seq and single-cell is definitely futuristic. I'd kill the uncertainty principle and would rather hear what I can learn from single-cell that I can't from bulk, and what are some of the barriers to further adoption of this method - what are the current limits of single-cell RNASeq which need to be overcome?"

\section{Nucleic Acid Splicing}
  \label{Intro:sec:Nucleic Acid Splicing}

  1977 brought the discovery of ``split genes'' \citep{Berget1977,Chow1977}. Almost immediately it was reasoned that RNA transcribed from split genes could be arranged in different combinations, greatly increasing the coding potential of a genome \citep{Gilbert1978a}. Differential arrangement of gene products via transcription and splicing, where at least two unique transcripts produced, known as \textit{alternative splicing}, has proven to be an integral part of eukaryotic gene expression.

  \subsection{Alternative Splicing}
    \label{Intro:subsec:Alternative Splicing}

    The number of genes estimated to be alternatively spliced has grown considerably. In 1993, Phillip Sharp, Co-Nobel-prize winner for the discovery of splicing, stated that: ``Approximately, one of every twenty genes is expressed by alternative pathways of RNA splicing in different cell types or growth states'' \cite{Sharp2014}. Not long after the assembly of the first human genome, a number of groups combed through Expressed Sequence Tag (EST) databases to increase that estimate to 35\%-59\% \citep{Modrek2002}. Soon after, analysis using specially designed ``splicing sensitive'' microarrays resulted in an increased estimate of 74\% \citep{Johnson2003}. In late 2008, three groups used RNA-Seq to demonstrate that between 86\% and 95\% of human multi-exon genes are subject to alternative splicing \citep{Pan2008, Wang2008, Sultan2008}. Not only did they demonstrate that almost all genes are alternatively spliced, they also showed that alternative splicing often occurs in a tissue- and cell type-specific manner. In combination with transcription regulation, the study of alternative splicing is critical to advance our understanding how comparably static genomic DNA sequence produce the highly flexible and adaptive transcriptomes of organisms.

    A pair of papers recently published in Science best illustrate the amazing complexity alternative splicing can generate between, and perhaps more importantly \textit{within}, organisms \citep{Barbosa-Morais2012,Merkin2012}. RNA-Seq performed on a diverse array of organisms and tissues has revealed that splicing patterns are shared more closely between organs of different species than between different organs within a species. Alternative splicing is essential for physiologically-specialized organs to use a common genome.

    \begin{figure} % # of Genes and % Alternatively Spliced
      \centering
      \includegraphics{Figures/Intro/numberHumanGenesAndNumberSpliced}
      \caption[Estimates of number of human genes, and percentage alternatively spliced over time]
      {
      Dark Grey - Estimates of number of human genes; Light Grey - Estimates of what percent of genes undergo some form of alternative splicing.
      }
      \label{Intro:fig:numGenesAndNumSpliced}
      \end{figure}

    Alternative splicing is an essential regulatory mechanism involved in the control of gene expression. Its combinatorial nature could potentially answer many basic questions concerning gene expression, such as a physical explanation of what separates us from our closest evolutionary ancestor, the chimpanzee \citep{Calarco2007a}. Additionally, the influence of alternative splicing on disease and cancer is slowly coming to light \citep{Tazi2009}. Unfortunately, because of the limitations of methods currently used for the large-scale analysis of isoform expression, we fail to obtain the complete picture of alternative splicing. One specific missing element of that picture is the prevalence of coordination between different regions of alternative splicing separated by large spans of sequence. An efficient, large-scale, single-molecule technique that maintains isoform sequence connectivity is required to complete the complicated picture of alternative splicing.

  \subsection{The Connectivity Problem}
    \label{Intro:sec:Isoform Problem}

    Alternative splicing research now relies on large-scale (aka: \textit{global}, \textit{genome-wide}, \textit{high-throughput}) techniques. Two of the most widely-applied technologies employed for large-scale analysis of gene expression are microarrays and ``2nd generation'' HTS. Unfortunately, both of these techniques have fundamental limitations, with the major issues being probe specificity for the former and read length for the latter.

    \begin{figure} % Seqlengths and Connectivity
      \centering
      \includegraphics{Figures/Intro/SeqLengths_and_Connectivity.eps}
      \caption[HTS read lengths are not sufficient to maintain alternative splicing connectivity]
      {
        HTS read lengths are not sufficient to maintain alternative splicing connectivity\\[0.25cm]
        A) Long RNAs may have multiple sites of alternative splicing, separated by 1000's of nt; B) Most mRNAs have \textasciitilde10 exons of \textasciitilde150 nt each. Some have many more (and longer) exons. Read lengths of current sequencing technologies do not maintain connectivity between distant sites.
        }
      \label{Intro:fig:NoConnectivityInHTSMethods}
      \end{figure}

    Microarrays rely on hybridization of a target sequence to a known probe averaging 25--100 nt in length \citep{Southern2001}. Therefore, microarrays only report the presence of short sequences in the target sample and do not provide linkage information between the sequences. A hypothetical scenario can be used to describe this issue. The goal of this example is to investigate a transcript known to display two different regions of alternative splicing (Figure \ref{Intro:fig:NoConnectivityInHTSMethods}). Probes targeting these two regions demonstrate an increase in signal for both alternative splicing events. While probes designed to hybridize accross splice junctions could be used to report on splicing (i.e. ``splicing sensitive arrays''), combinations of independent splicing decisions potentially contained in the same transcript would not be known. Put another way - it is not known if we observe an increase in unique transcripts, each containing only one region of  alternative splicing, or an increase in production of a single transcript containing both regions \citep{Calarco2007}. This binary analysis is the heart of the ``connectivity problem.'' Microarrays have proven extremely informative and will continue to do so for targeted applications. However, this issue, combined with concerns of cross-hybridization, reproducibility, and a comparably small dynamic range, has hastened the displacement of microarray by RNA-Seq as the preferred method for comprehensive analysis of gene expression \citep{Shendure2008}.

    RNA-Seq is now the \textit{de facto} method for comprehensive transcriptome analysis. Additionally, RNA-Seq allows for \textit{de novo} identification of isoforms, and is quantitative over a larger dynamic range \citep{Mortazavi2008}. Techniques exist to enrich samples for low-abundance isoforms, making the complete cataloging of alternative splicing events a possibility \citep{Djebali2008, Salehi-Ashtiani2008}. Unfortunately, current read-lengths (Figure \ref{Intro:fig:NoConnectivityInHTSMethods}) of all 2nd generation sequencing platforms do not solve the connectivity problem. Excluding single-molecule read lengths of sufficient length (i.e. ``third generation platforms'') \citep{Shendure2004}, other approaches proposed to solve the connectivity problem include traditional cloning and sequencing or hybridization of query oligos to single-molecule transcripts \citep{Zhu2003, Calarco2007, Emerick2007}. While these approaches can determine exon sequence connectivity, they scale poorly and are not feasible for large-scale applications.
    % MO Comment: "Aren't there some the statistical approaches that can be used to infer connectivity from RNA Seq data? Can you talk about those and their benefits and limitations?"

  \subsection{A Splicing Code}
    \label{Intro:subsec:Splicing Code}

    Beyond RNA-Seq isoform quantification and annotation, a major area of effort in alternative splicing research is decoding sequence regulatory elements (SREs) contained in pre-mRNA that define alternative splicing site selection. In contrast to core splicing signals, there exists limited knowledge of the SREs that serve to increase and decrease the strength of a particular splice site. SREs serve as \textit{cis}-acting sequences and binding sites for \textit{trans}-acting factors. Some of the best-studied SREs include Exon Splicing Enhancers and Silencers (ESEs and ESSs). Members of the Serine-Arginine (SR) protein family typically bind to ESEs located in an exon, promoting exon definition and thereby increasing the probability that the exon will be included in the final transcript \citep{Graveley2000,Long2009,Nilsen2010}. In contrast, ESS recognition reduces inclusion through binding trans-acting heterogeneous ribonucleoprotein particles (hnRNPs) \citep{Martinez-Contreras2007}. Therefore, trans-acting factor SRE binding can either promote or inhibit splicing machinery::pre-mRNA interactions. The current working hypothesis is that a finely tuned combination of these binding events, constituting a ``a splicing code'', determines the final exonic content of each isoform \citep{House2008}.

    Sequence motifs that compose the alternative splicing code have been teased out \citep{Ladd2002, Barash2010}. Assignment of binding motifs to tissue-specific trans-acting factors has also progressed \citep{Jin2003,Ule2005,Licatalosi2008}. Many of these binding motifs were identified using combined computational and biochemical approaches. Computational approaches involve searching for a comparative enrichment of sequences near splice sites. Biochemical approaches include gel shift assays, Systematic Evolution of Ligands by Exponential Enrichment (SELEX), and cross-linking. Many of these approaches are performed \textit{in vitro} and disregard the importance of cellular context on binding affinities. However, with increasing accessibility of HTS, many groups are extracting physiologically relevant, high-resolution data from traditional biochemical techniques \citep{Ingolia2009, Ingolia2011}. Deep-sequencing approaches are also being applied to questions involving mechanisms of alternative splicing. In addition to the RNA-Seq experiments, High-Throughput Sequencing [following] Cross-Linking Immunoprecipitation (HITS-CLIP) has confirmed SRE motif data predicted from computational and microarray experiments \citep{Licatalosi2008,Hafner2010}. Using HITS-CLIP, researchers can now enrich their samples for sequences that bind trans-acting factors of interest.

    Identification of proximally-acting SREs is progressing at a rapid pace. New and traditional biochemical methods, coupled with HTS, will undoubtedly fuel this progress. Unfortunately, a critical component of alternative splicing regulation currently neglected by the field is that of SREs acting across a considerable distance (>800 nt). One observation that may lead to the identification of long-range SREs is intramolecular coordination between distal splicing decisions. Figure \ref{Intro:fig:NoConnectivityInHTSMethods} shows a model transcript that may exhibit coordinated distal regions of alternative splicing. In this model, the 5\textprime~region of alternative splicing contains a cassette exon, which may or may not be included. This region is separated from the 3\textprime~region of alternative splicing by many thousands of nucleotides. Does the decision to include the cassette exon have an effect on which of the mutually exclusive exons is included? This type of alternative splicing regulation may represent a general and pervasive phenomenon.

  \subsection{Coordinated Splicing}
    \label{Intro:subsec:Coordination in splicing}

    The ``Miller Spread'' showing spliceosomes associated with nascent RNA transcripts suggested transcription and splicing are intricately linked \citep{Osheim1985}. Twelve years later, the observation that polymerase speed can affect downstream splicing decisions was reported \citep{Cramer1997}, spawning new research into co-transcriptional splicing.

    One way that linked splicing decisions (coordinated splicing) could manifest is dependence of a splicing decision in the 3\textprime~portion of a transcript on a splicing event in the 5\textprime~portion, especially if seperated by other non-dependant splicing events. One of the clearest examples of such regulation is mouse \textit{Fibronectin} (\fn{}) (Figure \ref{Intro:fig:mouseFn1}) \citep{Schwarzbauer1983, White2011a}. In this gene, inclusion of the alternatively spliced Extra Domain A (aka ``EDI'' or ``EDA'') region promotes splicing from one of three alternative 3\textprime~Splice Sites (3\textprime~SS) in the type III homology connecting segment (IIICS) region, resulting in more frequent production of shorter transcripts \citep{Fededa2005}. This effect occurs over six constitutively expressed exons and 800 nt of sequence (5,400 nt if introns are considered). \citet{Fededa2005} also analyzed EST databases, concluding that approximately 25\% of human genes contain multiple regions of alternative splicing. How many of these regions could show a coordinated effect similar to that observed in \fn{}? Providing some insight into this question, \citep{Fagnani2007} used microarrays designed to report on inclusion levels of cassette exons in mammalian central nervous system tissues \citep{Fagnani2007}. The results produced a set of 38 pairs of exons mapping to the same gene that showed a coordinated increase or decrease of inclusion levels.

    \begin{figure} % Fibronectin Architecture Figure
      \centering
      \includegraphics{Figures/Intro/Fibronectin.eps}
      \caption[Mouse \fn{} contains multiple sites of Alternative Splicing]
      {
        Mouse \fn{} contains multiple sites of Alternative Splicing\\[0.25cm]
        A) There are three highly-studied regions of alternative splicing in mouse \fn{}: Cassette exons EDB and EDA and the Variable(V)-region exon, which displays multiple 3\textprime~splice sites.  Each of these sites is separated by multiple constitutive exons. B) Considering simplistic splicing of these three exons, there are 12 different isoforms of mouse \fn{}.
        }
      \label{Intro:fig:mouseFn1}
      \end{figure}

    Some studies have investigated coordinated splicing between adjacent exons present in mRNA. The vertebrate genes \textit{4.1B} and \textit{4.1R} are members of the protein 4.1 family which encode cytoskeletal adaptor proteins. Both genes undergo splicing of a 5\textprime~first exon to distal second exons, skipping a stronger proximal 3\textprime~second exon \citep{Parra2008, Parra2012}. This is accomplished through ``intrasplicing'' involving an intronic sequence element (``intraexon'') only present when transcription begins at the upstream 5\textprime~exon. This allows the exon to ligate to the weaker distal 3\textprime~second exon via an intermediate splicing event.

    \textit{Cis}-acting sequences contained in intronic regions of a gene, a so-called Intronic Recognition Elements (IRE), has also been reported for the equine \textit{$\beta$-casin} gene. In this gene, an IRE bound to the exit channel of the elongating polymerase \citep{Lenasi2006}. IRE binding of the nascent RNA promotes inclusion of downstream cassette exons.

    Taking a more genome-wide approach \citet{Peng2008} examined human and mouse EST data looking for correlations between inclusion and exclusion of adjacent alternative splicing cassette exons. The authors note that positively correlated pairs of adjacent cassette exons typically resemble constitutive exons in similarity to the consensus splice sequences. Negatively and weakly correlated pairs are likely to be newly evolving exons whose sites have not evolved enough to be constitutively included.

    The most current and thorough study of intra-gene splicing coordination involves the \worms{} gene \slo{} \citep{Glauser2011, Johnson2011}. \slo{} is the \worms{} orthologue of the human BK channel gene \kcnma{}, which undergoes extensive alternative splicing \citep{Nilsen2010} via 13 cassette exons, potentially coding for over 1,000 different isoforms. \kcnma{} is developmentally, spatially, and tissue regulated and is involved in a diverse range of cellular processes, including hearing, circadian rhythms, urinary function, and vasoregulation \citep{Fodor2009a}.

    In worms, \slo{} can produce up to 12 different isoforms. \citet{Glauser2011} used TaqMan qPCR to demonstrate that individual alternative region inclusion frequencies do not correspond to complete isoform frequencies, suggesting an interdependent-splicing model. Interdependence was supported when mutations at one site altered both upstream and downstream sites of alternative splicing, separated by at least one other splicing event. After measuring the biophysical properties of the resulting protein isoforms \citep{Johnson2011} \citet{Glauser2011} conclude that coordinated alternative splicing is critical for proper BK channel function \textit{in vivo}. This study also identified an IRE that displayed some type of coordinated (or co-regulated effect) on alternative splicing.

    Indeed the Miller Spread was an early glimpse into another aspect of Nature's complexity. Described here are only a few examples of coordinated splicing. Genes like \fn{} and \slo{} have been carefully studied for decades. Yet each was done on a small, targeted scale. Increasing resolution of genome-wide data, better transcriptome assembly, and more rigorous analysis may reveal more examples of coordinated splicing decisions.

  \subsection{One Gene. Many Isoforms}
    \label{Intro:subsec:IsoformsPerGene}

    Researchers often uncouple evolutionarily intertwined processes such as transcription and splicing. A similar reductionist approach is to think of alternative splicing as a binary process: isoform A or B is produced by picking either exon A or B. What quickly becomes evident (to the detriment of researchers building transcriptome assembly algorithms) is that the combinatorial nature of alternative splicing makes it both a powerful means of generating isoform diversity \textit{and} a difficult problem to study \citep{Trapnell2012a}.

    \begin{figure} % alternative splicing Events per type barchart
      \centering
      \includegraphics{Figures/Intro/ASEventTypesPlot.eps}
      \caption[Number of hg19 Alternative Event types]
      {
        Number of hg19 Alternative event types per gene\\[0.25cm]
        Alternative Event types per gene. ``cassetteExon'''s are complete exons that are either included or not. ``altPromoter'' indicates a different transcriptional start site, and thus typically a different first exon identity. ``alt[Five|Three]Prime'' refers to different use of 5\textprime~ and 3\textprime~ splice site use for a given exon. ``retainedIntron'' refers to including an intronic region of a gene in the final mRNA. ``atacIntron'' refers to an intron whose remove of which is performed via the minor spliceosome. ``strangeSplice'' according to UCSC is ``An intron with ends that are not GT/AG, GC/AG, or AT/AC. These are usually artifacts of some sort due to sequencing error or polymorphism.'' For complete list of definitions refer here: \url{http://genome.ucsc.edu/cgi-bin/hgTables} and refer to hg19:UCSC Genes:Alt Events schema. Accessed from RefSeq on 2014-03-24.
        }
      \label{Intro:fig:asEventsBarChart}
      \end{figure}

    A current attempt to investigate the breadth of combinations produced by alternative splicing is the ENCODE project \citep{Birney2007,Dunham2012}. The transcriptional annotation arm of the ENCODE project \citep{Djebali2012,Derrien2012} used data from 15 human cancerous cell lines and found genes produce \textasciitilde10 isoforms.

    The ENCODE project builds on prior evidence for the combinational quality of isoform expression \citep{Wang2008,Pan2008}. Most genes undergo multiple forms of alternative splicing (Figure \ref{Intro:fig:asEventsBarChart}). Despite the prevalence of complex alternative spliced genes, just a few genes are routinely used as examples to illustrate numerical possibilities and biological significance. For example, the human immune system relies heavily on alternative splicing for plastic antigen recognition and response \citep{Lynch2004}. Modulation of extracellular signaling proteins such as \textit{CD44} and cellular adhesion protein \textit{CD45} have been well-studied \citep{Zikherman2008,Ponta2003}.

    Alternative splicing in humans, however, does not seem to produce the extreme number of unique isoforms as alternative splicing of genes in some simpler animals, such as \flies{} (Figure \ref{Intro:fig:txPerFlyGene} and Table \ref{Intro:fig:txPerFlyGene}). Perhaps this reduced alternative splicing \textit{per gene} is due to gene specialization, with transcripts from different genes working in combination, as oppose to unique transcripts from \textit{same} gene \citep{Park2007}. For example, \flies{} have a single muscle myosin heavy chain gene (\textit{Mhc}) capable of producing up to 480 different isoforms through alternative splicing of 17 different cassette exons \citep{Bernstein1983a}. In contrast, mammalian genomes encode whole families of \textit{Mhc} genes that have duplicated, diversified, and specialized in function \citep{Weiss1996}. The use of gene families reduces the necessity for alternative splicing to generate molecular diversity. Section \ref{Intro:subsec:Dscam} discusses another example of \flies{} generating isoform diversity from a single gene, while the comparable human gene does not---the extracellular binding protein DSCAM.

    \begin{figure} % Number of transcripts per gene
      \centering
      \includegraphics{Figures/Intro/NumberOFTranscriptsPerFlyGene.eps}
      \caption[Number of transcripts per \flies{} gene]
      {
        Number of transcripts per \flies{} gene\\[0.25cm]
        Data from \citep{Brown2014}, Supplemental Table 3. Number of transcript per bin, with bin sizes ``closed'' on the upper part of range.
        }
      \label{Intro:fig:txPerFlyGene}
      \end{figure}

    \begin{table} % Fly genes with 1000's of Tx
      \caption{Fly genes with >2,000 assembled transcripts according to \citep{Brown2014}.}
      \label{Intro:tab:FlyGenesWithManyTx}
      \input{Tables/FlyGenesWith1000sofTx}
      \end{table}

  \subsection{\flies{} \dscam{}}
    \label{Intro:subsec:Dscam}

    The gene most frequently used to demonstrate the combinatorial power of alternative splicing is \flies{} \dscam{}. The ``architecture'' of \dscam{} is rather unique, but as we see in Figure \ref{Intro:fig:txPerFlyGene} and Table \ref{Intro:tab:FlyGenesWithManyTx}, \flies{} contain numerous genes that generate tremendous isoform diversity \citep{Brown2014}. The basic structure of \dscam{} is shown in Figure \ref{Intro:fig:DscamArch}.

    Human \textit{Dscam} (Down Syndrome Cellular Adhesion Molecule) was identified while looking for genes on chromosome 21, specifically band 21q22, where extra copies are expressed in Down syndrome patients \citep{Yamakawa1998a}. \textit{Dscam} is a member of the immunoglobulin super family of proteins with extracellular adhesion functions. Human \textit{Dscam} undergoes some alternative splicing and is broadly expressed in the developing nervous system. Yet, it does not contain the same impressive number of cassette exons as \flies{} \dscam{}.

    \begin{figure} % Dscam Architecture
      \centering
      \includegraphics{Figures/Intro/DscamArch.eps}
      \caption[The architecture of the \flies{} gene \dscam{}]
      {
        The architecture of the \flies{} gene \dscam{}\\[0.25cm]
        \dscam{} has three \textit{clusters} or ``banks'' of alternative cassette exons that are included in a mutually-exclusive manner. The first bank, ``Exon 4'', contains 12 different variants, of which only one is ever included into the mRNA. Similarly, banks 6 \& 9 each contain 48 and 33 different variants, respectively. These three banks code for extracellular IgG domains, while the final region of alternative splicing, exon 17, encodes two different trans-membrane domains, again only one of which is included in the final mRNA.
          }
        \label{Intro:fig:DscamArch}
        \end{figure}

    Complex alternative splicing of \dscam{} was first noticed by the Zipursky lab in 2000 \citep{Schmucker2000}. While looking for proteins associated with \textit{dock} and \textit{pak}, two proteins important for neuronal growth cone guidance, they biochemically co-purified DSCAM1. Sequencing of \dscam{} clones revealed that all clones contained different combinations of exons 4,6, and 9. These three exons are chosen from three clusters of mutually-exclusive cassette exons, containing 12, 48, and 33 exons (Figure \ref{Intro:fig:DscamArch}). The initial report kicked off an exciting period of research into \dscam{} structure and function.

    Before the highlights of \dscam{} research are reviewed, it is illustrative to discuss some basic \flies{} anatomy. There are four anatomic regions where \dscam{} expression has been highly-studied:

    \begin{itemize} \itemsep0.5pt \parskip0pt \parsep0pt % Dscam expression regions
      \singlespacing
      \item Hemocyte cells of the immune system
      \item Larva Class IV da Neurons
      \item Pupal Mushroom-body neurons in the developing brain
      \item Tetrad synapses of the eye
      \end{itemize}

    During larval development, \dscam{} is expressed in the da neurons of the larval body wall (Figure \ref{Intro:fig:DscamAnatomy}). The da neurons create a uniform sensory field that allow larva to respond to mechanical stimulus. Morphologically, da neurons resemble oak trees with broadly dispersed branches. In order to maximize coverage of the field, every \{cell::cell\} interaction (i.e. every synapse) must be a productive one. Molecularly, this is accomplished via an extracellular handshake between copies of DSCAM1. If this handshake feels too familiar, a stable, lasting, and \textit{productive} synapse is not encouraged \citep{Wojtowicz2004}.

    The use of DSCAM1 to discern self from non-self is not unique to da neurons. It is also essential in the developing pupal brain. Here, \dscam{} is expressed in both axonal projections of neurons as they extend from Kenyon cell bodies and bifurcate into the two different mushroom body lobes \citep{Zhan2004}.

    Finally, the involvement of \dscam{} in the innate immune system of insects has been demonstrated \citep{Watson2005,Dong2006}. DSCAM1 recognizes antigen via similar self vs non-self interactions.

    \begin{figure} % Dscam Anatomy
      \centering
      \includegraphics{Figures/Intro/DscamAnatomy.eps}
      \caption[Important sites of \dscam{} expression in \flies{}]
      {
        Important sites of \dscam{} expression in \flies{} \\[0.25cm]
        \dscam{} has been high-studied in four different regions/cell types. (1) Hemocytes of the immune system, where DSCAM1 is involved in antigen recognition; (2) In Class IV da neurons, which sense mechanical stimulation of the larval body wall; (3) In mushroom body neurons of the pupal developing brain; and (4) (not shown) in Tetrad neurons of the eyes.
        }
      \label{Intro:fig:DscamAnatomy}
      \end{figure}

    How diverse are \dscam{} isoforms? Are isoforms different between cells? How is diversity generated? Are isoforms different among tissues or in individual cells? These are the questions that research into \dscam{} has sought to answer over the last 14 years.

    Soon after the initial \dscam{} report \citet{Celotto2001} investigated \dscam{} developmental regulation. They focused on the 12 variants of cluster 4 and observed regulation of exon 4.2. Embryonic transcripts show little inclusion of this exon while adult transcripts show frequent inclusion. Exon 4.8 displayed the opposite behavior. Similar regulation of cluster 4 exons was also observed in a closely related species, \textit{Drosophila yakuba}.

    In 2004, \citet{Neves2004} used a specially designed microarray to robustly characterize \dscam{} molecular diversity. They observed inclusion of virtually all alternative exons from clusters 4, 6, and 9. Additionally, they examined \dscam{} transcripts obtained from single-cell originating colonies and reported that multiple \dscam{} transcripts were expressed per cell. They estimated that each cell, depending on type, contained between 7--50 different combinations. As discussed above, the use of microarrays to perform this analysis precluded observing any potential coordination between variant exons.

    Quickly after \citet{Neves2004} published their results, the Zipursky lab also published a microarray study of \dscam{} isoforms \citep{Zhan2004}. They focused their analysis on neurons of the developing mushroom body (Figure \ref{Intro:fig:DscamAnatomy}). Not only did they also show that most \dscam{} combinations are likely produced at some level, but that diversity of isoforms is required for bifurcation of neurons into different lobes of the developing mushroom body. These results highlighted a critical function for self vs non-self determination via DSCAM1-mediating extracellular interaction.

    How is mutually-exclusive exon usage among 48 different options possible? \citet{Graveley2005b} observed a single ``Docking site'' within the intronic sequence just 5\textprime~to exon 6.1. This docking site was conserved among 15 insect species examined, from closely-related \textit{Drosophila simulans} to a distantly-related \textit{Tribolium castaneum} (Red flour beetle). Astonishingly, the docking site was complementary to ``selector sites'' within intronic regions just 5\textprime~of each of the 48 variant exons. A model was proposed where \{docking::selector\} interaction is required to choose which variant exons is included, while a splicing regulator protein, likely an hnRNP due to the repressive nature of the interaction, binds to unused selector sites contained in the pre-mRNA \citep{Graveley2000}. Additional mechanisms have been reported for other clusters, including the \textit{iStem} \citep{Kreahling2005} in cluster 4, and the hnRNP protein hrp36 \citep{Olson2007}.

    \citep{Neves2004} examined \dscam{} expression in hemocyte cells, and their results clearly show reduced variability in cluster 9 inclusion. Virtually all of the signal obtained from hemocyte cells for cluster 9 was seen in variants 9.[6,9,13,30,and 31]. \citep{Watson2005} also examined \dscam{} expression in hemocyte cells, comparing it to that of neuronal cells. They propose that secreted forms of \dscam{} are essential for a robust innate immune system in insects, a finding that has also been observed in mosquitoes \citep{Dong2006}. Involvement of \dscam{} in the insect innate immune system highlights how nature has applied one gene that produces extreme molecular diversity to multiple problems involving determining self from non-self \citep{Hemani2012,Shi2012a, Hattori2008}.

    In 2007 the Zipersky lab published \citep{Hattori2007} the first in a series of genetic reports describing the function and diversity of \dscam{}. Using homologous recombination, \citet{Hattori2007} showed that \dscam{} diversity is required for proper neural wiring but that individual neuronal-isoform identity is not important. Two years later, \citet{Hattori2009} observed that flies capable of expressing at least 4,752 different \dscam{} isoforms were indistinguishable from wild-type controls. This series was recently advanced with another tour-de-force of genetic manipulation. \citet{Miura2013b} used a collection of \dscam{} mutants allowing for visualization via GFP of specific cluster 4.X variant expression in real time. They concluded that a single neuron expresses multiple \dscam{} isoforms over time, and \dscam{} is expressed via ``stochastic and probabilistic'' mechanisms.

    Research into \flies{} \dscam{} has provided major advancements to our understanding of multiple aspects of transcription, including: 1) exon definition; 2) alternative splicing of cassette exons; 3) neuronal and cellular recognition; and finally 4) allows comparisons between how points 1--3 are accomplished among model organisms. See sections \ref{SeqZipPaper:sec:Results} for more information concerning \dscam{}.

\section{Nucleic Acid Ligation}
  \label{Intro:sec:Nucleic Acid Ligation}

  Section \ref{Intro:sec:Nucleic Acid Sequencing} discusses implications of cheap DNA and RNA sequencing to biomedical research. This section discusses how the ability to \textit{join} pieces of nucleic acid has also advanced our understanding of biology. A particular focus is placed on an enzyme with relevance to Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod}---T4 RNA Ligase 2.

  \subsection{RNA-templated DNA-DNA ligation}
    \label{Intro:subsec:Ligation}

    In the late 1960's and early 1970's, the Lehman and Richardson labs characterized two workhorse-enzymes of molecular biology. Robert Lehman and colleagues, working at Stanford Medical School, first described the activity of \textit{polynucleotide-joining enzyme} from \textit{Escherichia coli} (now known as \textit{E. coli} DNA Ligase) \citep{Olivera1967b}. Work on this enzyme paralleled that from the Richardson lab at Harvard Medical School, where they focused on \textit{polynucleotide ligase} from \textit{Escherichia coli} infected with T4 bacteriophage (now known as T4 DNA ligase) \citep{Weiss1967a}. It became clear that while these two enzyme's shared a common mechanism---later elucidated by \citet{Modrich1973a}---they had important differences. First, T4 DNA ligase required ATP as a cofactor, which \textit{E. coli} DNA Ligase did not (it was later discovered that DNA ligase required NAD as a cofactor). Second, only T4 DNA ligase could catalyze ligation of blunt-ended DNA \citep{Tabor1987a}.

    \begin{figure}% Ligation Mechanism
      \centering
      \includegraphics{Figures/Intro/LigationMechanism.eps}
      \caption[Mechanism of Rnl2 ATP-dependent ligation]
      {
        Mechanism of ATP-dependent ligation\\[0.25cm]
        Adapted from \citep{Nandakumar2006} and specifically for that of T4 RNA ligase 2.
        }
      \label{Intro:fig:Ligation Mechanism}
      \end{figure}

    The general mechanism of ligation (Figure \ref{Intro:fig:Ligation Mechanism}) involves three steps: Step 1 (A) the $\epsilon$-amino group from the active site lysine performs a nucleophilic attack on the $\alpha$-phosphate of ATP in solution. B) The ligase is now charged with AMP and inorganic phosphate (PPi) is freed into solution. C) Step 2: Nucleophilic attack by the 5\textprime~DNA phosphate on the 3\textprime~side of the nick to the AMP:ligase phosphate. D) Adenylated DNA is now competent for DNA ligation. E) Step 3: the 3\textprime~OH on the 5\textprime~side of the nick performs a nucleophilic attack on the 5\textprime~PO$_{4}$ across the nick, liberating AMP into solution. F) Sealed nick resulting in: Ligase, AMP, and dsDNA.

    In addition to elucidating the general mechanism of ligation, it was also discovered that T4 DNA ligase lacks a preference for terminal polynucleotide structures. The Khorana and Richardson labs both reported the activity of this enzyme on combinations of RNA and DNA duplexes \citep{Fareed1971, Kleppe1970b}. Both described an activity of T4 DNA ligase, RNA-templated DNA-DNA ligation, that is of particular relevance to this thesis work. Unlike T4 DNA ligase, \textit{E. coli} DNA Ligase, will not join DNA strands on an RNA template \citep{Bullard2006}. Soon after demonstrating these activities \textit{in vitro}, the Khorana lab reported detection of DNA generated \textit{in vivo} (i.e. by and organism) \citep{Besmer1972b}, setting up an orthogonal field (respective to PCR) of nucleic acid sequence characterization \citep{Conze2009c}.

    An enzyme that can catalyze an RNA-templated DNA-DNA ligation is a very useful molecular biology tool for two main reasons. First, using RNA as a ligation guide means no modification is made to the template. This contrasts cDNA analysis, where the RNA has been enzymatically converted by reverse transcription, potentially losing valuable RNA-coded information, such as modified bases. Second, synthesis of the DNA probes used in ligation is inherently easier and cheaper compared to synthesis of RNA probes (see section \ref{Disc:subsec:LNA-Containing ligamers and T39A Rnk2}). In addition to being cheaper, synthesis of DNA probes has become high-throughput since the adoption of microarrays as a standard gene expression measurement tool \citep{Schena1995a}.

    A pair of papers from the Landegren lab first reported the utility of RNA-templated DNA-DNA ligation for analysis of RNA transcripts \citep{Nilsson2000,Nilsson2001}. The Fu lab applied this approach in a multiplex experimental design in collaboration with Illumina \citep{Li2012c,Yeakley2002}, while the Nilsson and Landegren labs developed a single molecule application \citep{Conze2010}. It is important to note that \textit{all} of these studies used T4 DNA ligase. Clearly, there is interest and utility in analyzing RNA in both high-throughput and multiplex experimental designs, using cheap DNA probes, and without cDNA conversion.
    % ER Comment:"It is still not clear how the used RNA-templated DNA-DNA ligation to analyse RNA without cDNA conversion. Maybe you could say a little bit more so that the reader does not have to read the papers to understand how they did it). "

    For more than 40 years after its first description, T4 DNA ligase was the only choice for RNA-templated DNA-DNA ligation. However, a recent publication from New England Biolabs (NEB) describes this activity by another well-studied ligase, Chlorella Virus PBCV-1 DNA ligase (herein Chlorella DNA ligase) \citep{Lohman2013c}. Chlorella DNA ligase is a long-studied enzyme and had been reported to \textit{not} display RNA-templated DNA:DNA ligation activity \citep{Ho1997b,Sriskanda1998c}. However, at high enough concentrations and under special buffer conditions (specifically a critical concentration of ATP), \citet{Lohman2013c} have shown that Chlorella DNA ligase will join two DNA strands hybridized to an RNA template. They further demonstrated that it performs no worse in this activity than traditional T4 DNA ligase \citep{Nilsson2001,Yeakley2002}.

    Building on the list of available enzymes that join hybrid polymer substrates Chapter \ref{SeqZipPaper} presents data supporting RNA-templated DNA-DNA ligation activity for another enzyme, T4 RNA Ligase 2.

  \subsection{T4 RNA Ligase 2}
    \label{Intro:subsec:Rnl2}

    Proteins of the T4 and T7 bacteriophages have been a boon for molecular biology. Without enzymes like polynucleotide kinase \citep{Richardson1965a}, T7 RNA polymerase \citep{Summers1970b}, and T4 DNA ligase \citep{Weiss1967a}, many essential manipulations of nucleic acids would have been impossible for decades. Obviously, these enzymes also have essential phage functions. T7 RNA polymerase is responsible for late stage replication of T7 phage transcripts, while T4 PNK works in concert with T4 DNA and RNA ligases to repair cleaved nucleic acids resulting from bacterial pathogen defense systems \citep{Wang2002b}. Specifically, T4 RNA ligase 1 (herein ``Rnl1'', also known as \textit{gene 63}) maintains phage replication by repairing tRNAs cleaved by an anticodon nuclease produced from the \textit{prr} locus \citep{Amitsur1987d}.

    Given the utility and importance of these enzymes, novel enzyme discovery is a fruitful area of research. The Shuman lab has a distinguished record of discovering and characterizing numerous such enzymes, including many involved in nucleic acid synthesis, modification, and repair. Through a BLAST search looking for novel ligases with sequences related to \textit{Trypanosoma brucei} RNA-editing ligases TbMP52 and TbMP48 \citep{Ho2002b}, they identified a gene in the T4 genome (\textit{gp24.1}) with motifs in correct arrangement, spacing, and number indicative of an RNA ligase.

    Initial biochemical purification and characterization of \textit{gp24.1} \citep{Ho2002b} revealed that it indeed codes for an RNA ligase, which was renamed T4 RNA ligase 2 (herein ``Rnl2''). Rnl2 is a 374 amino acid monomeric protein composed of 2 distinct domains initially purified as a 42-kDA His-tagged recombinant protein. The N-terminal domain (1--243) is responsible for steps (1) and (3) of the general ligation mechanisms (Figure \ref{Intro:fig:Ligation Mechanism}), while the C-terminal domain (244--329) is responsible for adenylation of the 5\textprime~PO$_{4}$ on the 5\textprime~residue at the 3\textprime~side of the nick, as shown in step (2). Rnl2 is routinely purified pre-adenylated and immediately poised for its first ligation.

    In contrast to the N-terminal domain, which is composed of motifs typical to main ligases, the C-terminal domain is not contained in other DNA ligases. While the biological function of Rnl1 is known, the biological function of Rnl2 remains a mystery more than 12 years after its discovery \citep{Chauleau2013b}. However, there is some speculation that the flurry of research into bacterial CRISPR phage defense may reveal a role for Rnl2 \citep{Barrangou2007c,Chauleau2013b}.

    \begin{figure} % Rnl2 structure
      \centering
      \includegraphics{Figures/Intro/Rnl2_Structure.eps}
      \caption[Structure and active site of pre-adenylated Rnl2]
      {
        Structure and active site of pre-adenylated of Rnl2\\[0.25cm]
        Rnl2 as crystalized and described by \citep{Nandakumar2006}. Structures from \{PDB:2HVQ\} were generated with PyMol. Top left) Rnl2 is composed of a C-terminal and N-terminal domain. Top Right) The active site of Rnl2 is highlighted. Bottom left) Active site of Rnl2 as shown from bottom. This face interacts with substrate.
        }
      \label{Intro:fig:Rnl2 General Structure}
      \end{figure}

    Mutational analysis crystal structure analysis of Rnl2 have identified key functional residues \citep{Ho2004, Nandakumar2006,Nandakumar2004a,Yin2003d}. The lysine residue at position 35 (K35) receives the AMP in Step 1. The K227 residue in the C-terminal domain is essential for both forward and reverse adenylation of the 5\textprime~PO$_4$ at the nick \citep{Viollet2011}. Mutation of H37 results in an \textasciitilde102 reduced ligation rate, indicating the important nature of this residue. Finally, T39 has been shown to interact with the 2\textprime~OH on the 3\textprime~side of the nick, preferring a C3\textprime~endo sugar pucker conformation (Figure \ref{Intro:fig:Rnl2 Active Site Residues}).

    Rnl2 has a minimal footprint of 13 nt, centered on the nick, and only requires magnesium for transfer of AMP to the 5\textprime~phosphate. Work done in the Shuman lab \citep{Nandakumar2006} observed that 2\textprime~deoxyribose residues on the 5\textprime~side of the nick (i.e. DNA) adopt an RNA-like sugar pucker, leading to the correct orientation of the 3\textprime~OH relative to the AMP leaving group and resulting in ligation. This conformation is of particular importance to results presented in Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod}.

    \begin{figure} % Rnl2 Active site residues
      \centering
      \includegraphics{Figures/Intro/Rnl2_Active_Site_Residues.eps}
      \caption[Active site of T4 RNA Ligase 2 with highlighted residues]
      {
        Active site of T4 RNA Ligase 2 with highlighted residues\\[0.25cm]
        Rnl2 complexed with nicked dsDNA as crystallized and described by \citep{Nandakumar2006}. Structures from \{PDB:2HVR\} and images generated with PyMol.
        }
      \label{Intro:fig:Rnl2 Active Site Residues}
      \end{figure}

    A modified version of Rnl2 containing only the N-terminal domain and a K227A point mutation (``Truncated mutant'') has no adenyltransferase activity \citep{Viollet2011}. In this case, adenyltransferase refers to the ligase transferring AMP from an adenylated substrate to itself; reverse chemistry of step 2 in Figure \ref{Intro:fig:Ligation Mechanism}). This mutant has been used in specialized cloning applications \citep{Ghildiyal2008, Hafner2008a, Viollet2011} that take advantage of this activity. In these reactions, the use of pre-adenylated 3\textprime~DNA adapters allows for selective ligation among already phosphorylated species by limiting the enzyme-catalyzed transfer of AMP from the adapter to other phosphorylated species. Use of this truncated mutant to create a hybrid RNA/DNA molecule has greatly improved high-throughput sequencing work-flows.

    Ligation of hybrid substrates (e.g.. DNA-templated RNA-DNA vs. DNA-templated DNA-DNA) have revealed general ligase substrate preferences. DNA ligases appear to prefer the residue bearing the 5\textprime~phosphate on the 3\textprime~side of the nick to be 2\textprime~deoxyribose, and have a relaxed requirement for the sugar on the 5\textprime~side of the nick. RNA ligases have the reverse preference, demonstrating higher activities when the 5\textprime~strand, 3\textprime~OH residue also bears a 2\textprime~OH. Rnl2 has an additional preference for an RNA residue at the penultimate 3\textprime~side of a residue \citep{Ho2002b,Ho2004, Nandakumar2004a, Nandakumar2006}. The two base requirement for RNA at the 5\textprime~side of the double stranded nick biases Rnl2 to join RNA:[RNA/DNA] strands.

    Independent labs have measured that RNA-templated DNA-DNA joining activity of Rnl2 is below assay limits of detection \citep{Bullard2006}. However, results discussed Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod} show that with enough enzyme and sensitive downstream measurements, Rnl2 will catalyze RNA-templated DNA-DNA ligation. Previous reports of Rnl2 lacking this activity are likely due to a single turnover mechanism in this reaction imposed by a non-typical sugar pucker of the ligated DNA trapping the enzyme on the duplex.

  \subsection{Ligases as molecular tools}

    Section \ref{Intro:subsec:Ligation} describes the identification and development of ligases as tools in molecular biology. Ligation of templated duplexes has multiple uses in cloning and sequence characterization. The following section (\ref{Intro:sec:Nucleic Acid Polymers}) discusses long nucleic acid polymers, specifically mammalian piRNA precursor transcripts. Little biology is known concerning these long transcripts. Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod} discuss the application of Rnl2 to characterize long nucleic acid polymers.

\section{Nucleic Acid Polymers}
  \label{Intro:sec:Nucleic Acid Polymers}

  \citet{Fire1998} brought small RNAs to the forefront of research. Recently lncRNA research has been in similarly exciting period \citep{Khalil2009,Guttman2009}. Whether ``small'' or ``long'' all classes of RNA are polymers of ribonucleotides. This section will focus on an interesting class of nucleic acid polymer---mammalian piRNA precursor transcripts. These transcripts, which share similarities to traditional mRNAs, are processed into piRNAs. The section ends with a history of transcript assembly using HTS data.

  \subsection{It Started Small: Mammalian piRNAs}

    piRNAs are small RNAs that are 23--35 nt long. They are slightly longer than other small RNAs (e.g. miRNAs or siRNAs, which are 21 to 25 nt long). Contrary to other small RNAs, piRNA biogenesis does not require the double-stranded RNA-specific ribonuclease Dicer \citep{Vagin2006, Houwing2007} and it is believed they originate from single-stranded RNA precursor transcripts. Yet, similar to other small RNAs, they do bind a subgroup of the Argonaute family of proteins, PIWI proteins, from which their name is derived (\textit{PIWI Interacting RNAs}).

    \citet{Aravin2001} first identified piRNAs in \flies{} originating from the \textit{Su(Ste)} locus as heterogeneous 25--27 RNAs essential for silencing of \textit{Stellate} and, more importantly, male fertility. In the few years since the initial report, piRNAs have been cataloged, characterized, manipulated and mutated, especially in \flies{} \citep{Siomi2011,Luteijn2013,Hirose2014}. The most famous function for piRNAs in \flies{} is suppression of transposon transcripts during gametogenesis \citep{Malone2009}. The Ping-Pong model elegantly explains how this might be accomplished: cyclic cleavage of transposon transcripts and piRNA precursor transcripts \citep{Brennecke2007,Gunawardane2007}. Yet, it appears that piRNAs have diversified beyond transposon silencing.

    Four reports in 2006 defined the beginning of mammalian piRNA research \citep{Aravin2006,Grivna2006,Girard2006,Lau2006}. Each observed small 23--35 nt RNA species that bound PIWI proteins. They also noticed that when aligned to the genome, most mapped to ``clusters'' of discrete genomic loci, similar to flies.

    Overtime, it became clear that mammalian piRNAs can divided into three major classes (Figure \ref{Intro:fig:Mammalian piRNA classes}). There are also three PIWI proteins in mice, each displaying a distinct expression profile during development and an association with piRNAs of a specific length.

    \begin{landscape}
      \begin{figure} % Mammalian piRNA classes
        \centering
        \includegraphics{Figures/Intro/MammalianPiRNAClassesOverTime.eps}
        \caption[Different Classes of mammalian piRNAs]
        {
          Overall view of the three classes of mammalian piRNAs. Figure design assisted by \href{mailto:xin.li@umassmed.edu}{Xin Zhiguo Li}.
          } \label{Intro:fig:Mammalian piRNA classes}
        \end{figure}
      \end{landscape}

    \textit{Fetal piRNAs} (or ``prenatal'') are present before birth. These piRNAs tend to be short, bind the PIWI protein MIWI2 (PIWIL4) in mice, and have sequences found in transposable elements \citep{Carmell2007}. Quickly following MIWI2 in expression is the PIWI protein MILI (PIWIL2). It is during the ``fetal'' stage of piRNA biogenesis in mice that, in order to silence expression of transposons during germ line formation, MIWI2 and MILI undergo ping-pong amplification, similar to that observed in flies \citep{Kuramochi-Miyagawa2004, Aravin2006, Aravin2008a,Aravin2008}. Importantly, this activity has not been observed in adult testes.

    During the first three weeks of a male mouse's life the process of spermatogensis is in its ``first wave'' and sperm cells are synchronized \citep{Oakberg1956b, Laiho2013a}. After the first wave and for the rest of the adult lifespan, sperm in the testes are not synchronized. Instead there is a continuum of sperm production. Therefore, it is during the first wave that specific stages of can be easily isolated and studied. The next two classes of piRNAs are named according to their expression respective to an important milestone in gametogensis---the pachytene stage of meiosis I when chromosomes pair up, cross over, and exchange genetic material.

    \textit{Pre-pachytene piRNAs}, historically but confusingly grouped with fetal piRNAs, are expressed just before birth and continue to be expressed throughout the mouse lifespan. These piRNAs tend to map to traditional, annotated, protein-coding genes. During the ``neonatal stage'' pre-pachytene piRNAs are bound by the only PIWI protein expressed at that time, MILI. Also, piRNA expressed during the pre-pachytene stage shift from mostly transposon-mapping to protein coding gene 3\textprime~UTR mapping \citep{Robine2009}.

    The last class, \textit{pachytene piRNAs}, are extremely abundant compared to pre-pachytene piRNAs in adult testes \citep{Girard2006, Lau2006, Li2013h}. They bind another Piwi protein MIWI (PIWIL1). The genomic origin of pachytene piRNAs, often unique in terms of genomic sequence, often fall within ``gene deserts.'' Pachytene piRNA clusters are actually genes (aka: ``piRNA-producing loci'') encoding very long single-stranded transcripts devoid of introns (see section \ref{SeqZipMethod}) \citep{Li2013h}. This gene architecture makes the pachytene piRNA loci some of the most interesting RNA-producing regions of the mammalian genome.

    Except for the uniquely-mapping quality of pachytene piRNA loci, their transcripts are comparable to piRNA clusters in flies, such as \textit{flamenco}. \textit{Flamenco} transcripts can be abolished by inserting a P-element into a putative promoter, as measured by northern blot looking for piRNAs generated 168 kb downstream (Figure \ref{Intro:fig:flamenco} \citep{Brennecke2007,Goriaux2014}. Similarly, transcription of pachytene piRNA loci requires \amyb{}, and piRNAs hundreds of thousands of nt downstream from annotated 5\textprime~ ends are not seen in \amyb{} mutant mice (see Chapter \ref{MolCel}).

    \begin{figure} % Flemenco Locus
      \centering
      \includegraphics{Figures/Intro/FlamencoLocus.eps}
      \caption[Genetic evidence for long, continuous fly piRNA precursor transcripts]
      {
        A the \flies{} gene \textit{flamenco} is a graveyard for transposon sequences \citep{Pelisson1994}. Evidence for expression of a single-contiguous RNA transcript from \textit{flamenco} (A) is provided by a P-element insertion into the suspected promoter region (B). \citep{Brennecke2007} could not detect specific piRNAs (red X's) by northern blot in the P-element mutant.
        }
      \label{Intro:fig:flamenco}
      \end{figure}

  \subsection{From Long to Small: Precursor processing to mature piRNAs}
    \label{Intro:subsec:Processing of piRNAs in mice}

    The process by which a long, single-stranded piRNA precursor transcript become small mature piRNA is full of black boxes and question marks \citep{Li2013e}. Indeed, we are very unsure of many steps between transcription and terminal function of \{piRNA::PIWI\} complexes (PIWI-piRISC).

    For example, how do piRNA precursor transcripts exit the nucleus? This is not known in mice, but there are clues from \flies{}, where some piRNA clusters are bidirectionally transcribed and bound by the HP1 homologue Rhino \citep{Klattenhoff2009}. Rhino co-localizes with the DEAD box protein UAP56 near the perinuclear compartment known as nuage \citep{Zhang2012}. It is believed that Rhino and UAP56 assist in a hand off of large precursor transcripts across the nuclear envelope where they are bound by the nuage protein VASA \citep{Zhang2012}.

    Once precursor transcripts exit the nucleus they may enter chromatoid bodies (comparable to nuage in flies) \citep{Lim2007,Meikar2011,Zhang2012,Meikar2014} where they are proposed to be ``fragmented'' into shorter \textit{piRNA intermediates} \citep{Saito2010,Li2013}. However, the location of fragmentation is currently unknown in mice. In mice, the protein MitoPLD (aka: PLD6, or \textit{Zucchini} in \flies{}) is the proposed enzyme that catalyzes fragmentation, but this has only been studied in 10.5 dpp mice and therefore only for pre-pachytene piRNAs \citep{Watanabe2011a}.

    Slicing activity for \textit{Zucchini} has been observed \textit{in vitro} and is supported structurally \citep{Nishimasu2012,Ipsaro2012}. Its activity has yet to be shown \textit{in vivo} \citep{Luteijn2013}. Fragmentation may, or may not, impart the 5\textprime~U preference seen in mature piRNAs \citep{Gunawardane2007,Brennecke2007} and indeed Zucchini does not show a 5\textprime~U bias \textit{in vitro} \citep{Nishimasu2012,Ipsaro2012}. However, this preference may result from downstream sequence preference of PIWI-protein binding \citep{Cora2014}.

    Once fragmented into shorter RNAs, piRNA intermediates seem to be ``loaded,'' into a specific time- and expression-appropriate PIWI proteins (Figure \ref{Intro:fig:Mammalian piRNA classes}). Following ``loading,'' piRNA intermediates are trimmed down to the length characteristic of bound Piwi by the appropriately named, but \textit{hypothetical}, enzyme ``Trimmer'' \citep{Li2013}. Both ``Loading'' and ``Trimmer'' activity have not been shown in mammalian systems but are inferred from Silk worm (\textit{Bombyx mori}) cellular extracts of ovary-derived BmN4 cells \citep{Kawaoka2009}. Once trimmed, piRNAs are methylated on the 2\textprime~OH position by the enzyme HEN1 \citep{Horwich2007,Kirino2007,Ohara2007,Kawaoka2011}, but again this activity is not well-studied in mice. At this point, a mature piRNA, complexed with a PIWI protein (PIWI-piRISC), is poised to perform cellular function(s).

    \begin{figure}\small % Mammalian piRNA Pathway
     \centering
     \includegraphics{Figures/Intro/mammalian_piRNA_pathway.png}
     \caption[A model for Mammalian piRNA biogenesis]
     {
       Figure taken from \citep{Li2013e}: A model for piRNA biogenesis. Primary piRNA transcripts are transcribed by RNA polymerase II and contain 5\textprime~caps, exons, introns, and poly(A) tails. The transcription of pachytene piRNA genes is controlled by A-MYB; transcription factor(s) (TF) controlling pre-pachytene piRNA genes remain to be discovered. Current models of piRNA biogenesis propose that PLD6 determines the 5\textprime~end of piRNA intermediates with lengths >30 nt. These intermediates are proposed to then be loaded into PIWI proteins. After PIWI binding, a nuclease is thought to trim the 3\textprime~end of the piRNA to the length characteristic of the particular bound PIWI protein. Finally, further trimming is prevented by addition of a 2\textprime~O-methyl group to the 3\textprime~end of the mature piRNA by the S--adenosylmethionine-dependent methyltransferase HEN1. Figure adapted from \citep{Li2013}.
       }
     \label{Intro:fig:Mammalian piRNA BioGensis}
     \end{figure}

    What are the cellular activities of PIWI-piRISC? MILI and MIWI2 have been shown to direct epigenetic LINE1 and IAP transposon silencing in the embryonic male germline \citep{Aravin2007,Carmell2007,Kuramochi2008}. Two studies \citep{DeFazio2011,Reuter2011} used point mutations in the catalytic triad of MIWI, MIWI2, and MILI to remove slicer activity. \citet{DeFazio2011} found that MIWI2-deficient mice are fertile, silence transposons, and display all signs of secondary piRNA biogenesis and concluded that MILI (which is sterile) was required for transposon silencing. This finding was later elaborated upon by \citet{DiGiacomo2013} to work in concert with other forms of epigenetic silencing to repress LINE1 expression. \citet{Reuter2011} focused on MIWI and found that it required for silencing of LINE1 transcripts long after they were epigenetically silenced (i.e. in the adolescent mouse).

    The above studies point to a familiar scenario of piRNA-mediated target cleavage and/or transcriptional silencing by PIWI-piRISC \citep{Meister2013}. Yet confusingly, HITS-CLIP of MIWI revealed that MIWI binds spermiogenic mRNAs without a piRNA guide \citep{Vourekas2012} and \citet{Reuter2011} demonstrated that slicing of target by MIWI RISC requires near perfect binding.

    How does does one reconcile these findings with the extremely uniquely-mapping quality of virtually all pachytene piRNAs? Put another way, if MIWI requires near perfect pairing between guide and target, and pachytene piRNAs perfectly pair with nothing else in the genome but antisense transcripts from their own loci, what is the mechanism of target recognition?

    Taken together, frustratingly little is known or internally consistent concerning biogenesis or function of mouse piRNAs. Indeed, even the catalytic nature of PIWI proteins is a debated topic \citep{Luteijn2013,Meister2013}. A recent report that the DNA modification 5hmC is high in piRNA intergenic gene bodies \citep{Gan2013}, combined with known functions of fetal PIWI-piRISC alludes to a function for self-mapping pachytene piRNAs.

    Perhaps the site of PIWI-piRISC function is not cytoplasmic. Fly PIWI is localized in the nucleus, and MILI and MIWI2 have been shown in induce DNA methylation \citep{Cox2000,Aravin2008}. This is a potentially misleading course of logic. Localization does not confirm interaction \citep{North2006} or function and inferring such from localization can be as dangerous as assuming cars function in parking lots. Finally, a extremely tantalizing additional potential function for mammalian piRNAs is that of genomic imprinting \citep{Watanabe2011}. This function is in good agreement with germ line-specific and developmentally timed nature of Piwi protein expression.

    In summary, there are many holes and black boxes in the story of mammalian piRNAs. Continued study is easily justified by the sterile phenotypes of all pathway mutants. Time will tell if mammalian piRNAs are involved in a satisfying process of biology or are crude side quest of Nature.

  \subsection{From Short to Long: Transcript Assembly}
    \label{Intro:subsec:Tx Assembly}

    Initial genome-wide HTS of piRNAs revealed a tremendous amount of biology \citep{Gunawardane2007,Brennecke2007}, but could provide little information as to the original transcriptional unit. The ability to reconstruct piRNA precursors had to wait for technological improvements in HTS read length and alignment algorithms.

    Working backwards from small RNA-Seq data to original transcription units was impossible. Mammalian piRNAs are too short (\textasciitilde30 nt) to allow for quality assembly using even the most current algorithms. They simply do not provide the sequence overlap necessary to build scaffolds. Also, repeat elements are extremely abundant in mice \citep{Nellaker2012}, and combined with short reads further reduce the ability to assemble full-length sequences. Therefore, it was necessary to sequence RNAs prior to mature piRNA formation.

    Even with longer read lengths and the best assembly algorithms, the 5\textprime~ and 3\textprime~ ends of long and diverse transcripts like piRNA precursors often requires a combination of multiple HTS datatypes \citep{Blower2013,Li2013e}. Tailored versions of RNA-Seq, such as CAP-Seq (see section \ref{Intro:subsec: History of HTS}), are not sufficient for accurate 5\textprime~end determination, and require orthogonal datasets to verify TSSs. Taking a page from lncRNA transcript discovery, complementary data sets such as ChIP-Seq of H3K4 methylated histones, a marker for transcriptional initiation can supplement RNA expression data \citep{Khalil2009}. More information about how multiple HTS datasets can be---and were used---to define the transcriptional unit of piRNA precursors transcripts is provided in Chapter \ref{MolCel}.

    General assembly of full length transcripts (not just piRNA precursor transcripts) is difficult for at least 3 reasons: (1) The transcriptome is expressed across 5 orders of magnitude and a typical RNA-Seq library contains many reads from a few highly-expressed genes and many fewer reads from lowly-expressed genes \citep{Blencowe2009}; (2) RNA-Seq libraries are often not created from a completely pure source of mRNA and can contain reads from other RNA classes (e.g. tRNAs) or intronic reads from pre-mRNAs; and (3) Reads are often much shorter than a typical mRNA, making it difficult to assign which read goes to which isoform of a given gene (see the ``connectivity problem'' discussed in section \ref{Intro:sec:Isoform Problem}. With these challenges in mind, what is the current state of transcript reconstruction (herein \textit{transcript assembly})?

    Computational transcriptome assembly of short reads is currently performed in one of two modes: genome-guided and genome-independent \citep{Garber2011a}. The difference between these two approaches is use of a high-quality genome during the assembly process. Popular assembly programs such as Cufflinks \citep{Trapnell2010} and Scripture \citep{Guttman2010} use genome-aligned short reads as the bases for calling transcripts. Genome-independent methods include Trinity, Oasis, and Velvet \citep{Haas2013c,Schulz2012,Zerbino2008}.

    As mentioned previously, constraints imposed by the dynamic range of RNA expression is the major complicating factor with current transcript assembly programs. These programs frequently generate short transcript fragments (``contigs'') due to poor coverage of long and lowly-expressed transcripts \citep{Rehrauer2013,Steijger2013}. Merging contigs into  continuous transcripts is a major goal. Improvements will surely come from greater sequencing depth, longer reads, and mRNA enrichment schemes, albeit with diminishing returns \citep{Chang2014c}. See section \ref{Disc:subsec:need for Tx assembly} for more thoughts concerning transcript assembly. Longer-term barriers include repetitive sequences, transcript secondary structure \citep{Wan2014}, and mRNA processing including hydrolysis and RT processivity \citep{Sharon2013}. Finally, multiple forms of mRNA enrichment and purification, specifically combining poly(A)+ tail and 5\textprime~CAP selection, can be used to increase the accuracy of mRNA transcript assembly \citep{Blower2013}.

\section{Nucleic Acid 'Omics}
  \label{Intro:sec:Unique Time in Omics}

  Biomedical science has just taken a very sharp step (Figure \ref{Intro:fig:SeqCosts}) into an era of cheap genomics. Most questions, including those of gene expression, molecular interactions, and evolution no longer need be investigated on a small scale. Indeed formulating questions and hypothesis on a ``big scale'' should be considered from the very onset of a project. Combined broad and focused approaches will allow for maximal gains in knowledge. Yet much of the work required to reap maximal benefits from genome-wide approaches falls squarely on our own education and experience. See section \ref{Disc:sec:Final Thoughts} for concluding thoughts.

\cleardoublepage % Finish the chapter
