% !TEX root = /Users/royc/Google_Drive/Thesis/RoyC_Umass_Thesis.tex
\chapter{Discussion} 
\label{Disc}
\lhead{Chapter 5. \emph{Discussion}} 
%----------------------------------------------------------------------------------------
\section{Concerning the transcriptome}
  \label{Disc:sec:Future of Dynamic long RNAs}

  Deep sequencing of transcriptomes has revolutionized biology. Previously, transcript identification and characterization involved significant labor, cost, and materials. In the mid-90's, microarray technology \citep{Schena1995a} gave us a tantalizing glimpse into how genes were expressed, but were limited to probed, and therefore known, sequences. Yet, these green and red landscapes hinted at incredible complexity---a complexity that would have to wait for technology to catch up.

  RNA-seq was made possible by incremental improvements in numerous supportive technologies. These included: 1) digital optics; 2) microscopy; 3) slide chemistry ; 4) colony PCR; and 4) nucleic-acid alignment. A HiSeq 2500 relies on all of these technologies (and others) to produce the 100M+ sequences that allow scientists to peer into the transcriptional output of a genome.

  The past five years have brought further improvements, allowing biologists to think beyond mRNAs and small RNAs. The former captured our interest for 30+ years \citep{Furuichi1975,Wei1975}, while the later has been on a run-away train since 1998 \citep{Fire1998}. Long RNAs (including lncRNAs) are now routinely analyzed by HTS. However, many biologically-trained and minded Scientists find themselves overwhelmed by the completely different methods and approaches used to tackle these biological ``big data.'' Current experimental training does not provide most with the required skills in statistics, programing, and experimental design needed to work with genome-wide data (see section \ref{Disc:subsec:Biologists need Comp Skills}). The richness of these data often results in unasked, and unanswered, testable hypothesis just sitting in public data repositories \citep{Plocik2013}.

  This discussion will focus on \textit{long RNAs} containing traditional mRNA features---a 5\textprime~m7G Cap, ligated exons, and a poly(A)+ tail. Many of these long mRNAs are extremely dynamic. So much so that until HTS and RNA-Seq, comprehensive investigation of their complexity was not possible.

  \subsection{Pervasive transcription}
    \label{Disc:subsec:Pervasive Tx}

    There are $2,598,960$ different Poker hands possible from a 52-card deck. There are $1,098,240$ different single-pair combinations, with a probability of obtaining one being almost 50\%. Compare this to a ``Royal Flush'', for which there are only 4 options, and a probability of $649,739:1$ or $1.54 * 10^{-6}$!. It is these numbers that makes possible to play Poker for hours on end. Biology uses similar combinatorics to arrange exons into unique and rare combinations, especially in complex eukaryotic organisms. Indeed the process of splicing is closely correlated with organism complexity. The process of AS is even more closely tied to organism complexity (see Figure \ref{Intro:fig:numGenesAndNumSpliced}). Accurate determination and assembly of each card (exon) that comprises a hand (transcript) is a major ``known unknown'' \citep{Rumsfeld2011} of research into long RNAs.

    The ENCODE papers from late 2012 suggested that 95\% of the genome is functional \citep{Dunham2012}, a finding that continues to be heavily debated \citep{Graur2013,Bhattacharjee2014}. \citet{Djebali2012} and focused on issues of transcription in these cell lines, as discussed in section \ref{Intro:subsec:IsoformsPerGene}, and concluded that 75\% of the genome is transcribed into RNA. Additionally, GENCODEv7 includes 9,640 manually curated lncRNA loci, some of the most novel and functionally interesting class of long RNAs \citep{Derrien2012,Pauli2011}. While ENCODE was performed using multiple human cancerous cell lines, these results support the ability of HTS to reveal transcriptional diversity.

  \subsection{A need for transcript assembly}
    \label{Disc:subsec:need for Tx assembly}

    The current state of the transcriptome assembly, a field in its relative infancy, is described in section \ref{Intro:subsec:Tx Assembly}. Current transcript assembly algorithms only provide predictions and probabilities for the existence of real molecules. Until RNA can be directly sequenced, in their entirety, and from single cells or molecular compartments, researchers will always be making compromises in annotation and quantification \citep{Ozsolak2010}. Once technology advances to the point where a transcriptome can be as accurately and quickly determined as a genome, extremely exciting research into the more subtle and nuanced complexity (e.g. what makes one twin \textit{molecularly} different from another?) of biology can be unlocked.

    What is required to improve our ability to quickly and accurately assemble transcriptomes? Simulations indicate that improvements will not come from longer read lengths \citep{Chang2014c}. These simulations also demonstrate that the accuracy of current \textit{de novo} (see section \ref{Intro:subsec:Tx Assembly}) assemblers decreases sharply with increased increase AS within the transcriptome.
    \todo[inline]{More thoughts on what is needed for better transcript assembly} % **************************

  \subsection{Tissue and cell specificity}
    \label{Disc:subsec:Tissue-specific Tx expression}

    As discussed in section \ref{Intro:subsec:Alternative Splicing}, mechanisms of AS are frequently tissue-, time-. and cell-specific. Landmark studies examining AS in different organ systems, from evolutionarily-distant organims, found that AS is more comparable between organs of different animals than between different organs from the \textit{same} animal \citep{Barbosa-Morais2012,Merkin2012}. \citet{Brown2014} analyzed the \flies{} transcriptome, and observed that AS could be better desribed as ``tissue-specific splicing''. 

    The concept of ``tissue-specific splicing'' brings up a subtle, but important consideration concerning AS research. The concept of AS conjures an image of dynamic post-transcription RNA processing, allowing cells to quickly respond changes in environment, programming, or stimuli. Yet, as demonstrated by the studies just mentioned, most AS is not that dynamic. In fact, the main reason much of these events are even considered \textit{alternative} is because we are comparing the transcriptomes of different tissues. Is that a fair comparison?

    By the current definition, these events are indeed \textit{alternative}, but does that label carry significance? The label of alternative communicate that RNA products of a gene can be processed into multiple ways, but if they are typically \textit{not} alternatively splicing in the local environment, does this capability matter when HTS allows analysis of very specific biological samples and time points? These questions underscore the importance of advances in transcriptme assembly keeping step with advancements in HTS technology and ever-increasing sample resolution.

  \subsection{Lingering Questions for \dscam{}}
    \label{Disc:sec:Dscam}

    What controls the stochastic and probabilistic splicing of \dscam{}? Why is it different between hemocytes and neurons? Why would hemocytes need less apparent diversity, given the range of antigen they could potentially encounter?

    Can methods such as RNACapture be used to perform more careful and targeted sequencing of \dscam{} transcripts \citep{Mercer2014}?

\section{In the haystack: piRNA precursors}
  \label{Disc:sec:piRNA precursors}

  Chapter \ref{MolCel} describes efforts to annotate 467 transcripts from 214 loci that account for 95\% of the total piRNAs in 14.5 dpp mice. Also described is how these transcripts are typical Pol II transcripts, possessing all the archetypical molecular signatures including 5\textprime~MeGppp caps, introns, and poly(A) tails. Yet RNA from these molecules appears to be rapidly consumed and processed into millions of unique 24\textendash32 nt species. How does the cell partition long RNAs for translation by the ribosome (as the case for most mRNAs) or cleavage and maturation into piRNAs? The sections below propose experiments aimed toward answering this question.

  \subsection{Precursor Identity}
    \label{Disc:subsec:How are precursors generated}

    % From the resources paper: Indeed, two isoforms of pi-Wdfy3 are transcribed from different promoters in the testis (Figure S2D). The short isoform is transcribed from a low CpG promoter bound by A-MYB, while the long isoform is transcribed from a high CpG with no detectable A-MYB binding. piRNAs derive from the genomic region corresponding to the short transcript. In an A-Myb mutant, piRNA from the pi-Wdfy3 loci are reduced 165-fold (31.6 rpkm in heterozygotes versus 0.191 in mutants). The abundance of long isoform transcripts did not decrease (2.08 rpkm in heterozygotes versus 3.86 rpkm in mutant), while the small isoforms became undetectable (10.0 rpkm versus 0 rpkm), consistent with the idea that the short, but not the long, Wdfy3 transcript is a piRNA precursor RNA. Little is known about the regulation of genic piRNAs, because only seven genic piRNA loci are A-MYB-dependent (Li, et al., accompanying manuscript). We do not yet know if such differential regulation of transcript isoforms occurs more generally for genic piRNA loci.

    %How does cell partition prepachytene transcripts to mRNA or piRNA use?
    What determines that a seemingly mRNA-like piRNA precursor is processed into piRNAs, and not translated like crazy by Ribosomes?  Everything meets the ribosome ($REF$), so why is it that precursors are given a different lot in life?

    \begin{figure} % pi-Wdfy3 expresses both mRNA and piRNA precursor transcript
      \centering 
      \includegraphics{Figures/Discussion/pi-wdfy3.eps}
      \caption[\wdfy{} locus expresses both mRNA and piRNA precusor form in testes]
      {\wdfy{} locus expresses both mRNA and piRNA precusor form in testes\\[0.25cm]
        The mouse genomic locus \wdfy{} expresses both a traditional mRNA form, originating from an upstream TSS, and a piRNA precursor transcript from a downstream TSS. The piRNA precursor form appears to originate from an A-MYB-bound promoter, and is expressed in \amyb{} heterzygous mice. Also smalll RNAs (piRNAs) mapping to this locus are only observed in \amyb{} heterzygous mice, and not in \amyb{} mutant mice.
        }
      \label{Disc:fig:wdfy3}
      \end{figure}

    %Interaction w/ ribosomes?

  \subsection{Precursor Labeling}
    \label{Disc:subsec:Labeling of precursors}

    % From Resources Paper: Despite sharing many features with protein-coding genes, intergenic piRNA loci stand out as a unique class of transcriptional units. Almost half contain no introns

    % Need section here on the chemical modification - what were you thinking? Azide? Par-Clip?  Maybe a CLASH-like assay?  What about what Mihir is developing?
    
    SeqZip accurately reports on the presence of multiple, distant sequences contained in the same RNA molecule (see Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod}). Yet I was not able to observe ligation products templated off some of the most highly-expressed precursor transcripts (see section \ref{Disc:sec:piRNA precursors}. We hypothesize that the most likely explanation for being unable to observe ligation products for these transcripts, but successfully observing those for a very long, but lowly expressed gene \textit{Dst}, suggests that precursor transcripts are rapidly processed into mature piRNAs. What if this is not the case if we could hybridize ligamers to precursor transcripts in vivo? Put another way \textemdash What if the steady-state amount of precursor transcript is very low due to rapid processing, but when visualized in real time, the transcripts are of sufficient abundance for FISH? Ligamers could be engineered to contain different combinations of fluorescent dyes, and the proximity and intensity of the signal could be used to infer precursor transcripts. Importantly, this approach could distinguish long, continuous precursor transcripts from processing intermediates and mature piRNAs.

  \subsection{Precursor Imaging}
    \label{Disc:subsec:Imaging of precursors}

    %Another idea would be to image the precursors in real time somehow. How about tagging them and following them in the sperm? That could be cool. You could even follow them into the developing embryo. 
    The most important question for mammalian \textit{pachytene} piRNAs is \textit{What are they doing?}. We know that they are essential for the health of the species, as discussed in section \ref{MolCel:sec:Introduction}, and piRNA-pathway mutants are sterile. What could these small RNAs, with complementarity to nothing but themselves, be doing? Building on the last section discussing chemical modification precursor transcripts, if the modification was again durable enough for downstream analysis, perhaps the modification would remain in the mature piRNAs that are incorporated into mature sperm. These modifications could be tracked as the sperm move through the seminiferous tubules and into the epididymis where sperm mature \citep{Jones1999}. One could even track the piRNAs as they fuse with the oocyte to create an embryo. If this modification was labile, piRNA interacting proteins or nucleic acids could be captured or marked as well, providing additional clues as to the function of piRNAs in sperm and early embryogenesis.

    The cellular location of precursor piRNA transcript processing is not known. The most accepted hypothesis is that precursor transcripts are processed into mature piRNAs with machinery tethered to chromatoid bodies \citep{Meikar2011,Meikar2014} or another structure similar to Drosophila Nuage near the mitochondrial cement. Knowledge of \textit{where} mature piRNAs are generated would provide clues into larger mechanist details of their biogenesis. 

    I can think of two broadly different ways in which we could pinpoint the physical location of mature piRNA generation. The first is to chemically label, in some manner, primary piRNA transcripts. The label would need to 1) not interfere with processing and 2) be durable to later methods used to analyze the presence or absence of the modification. A second way to identify the location of mature piRNA processing would be to actually observe, through in vivo \hyperref[hd:abrevs]{FISH}, the various products and by-products of biogenesis. In this particular approach, SeqZip maybe useful.

    % MS2 lopp introduction into piRNA precursor transcripts.
    Another idea here would be to use the MCP x MBS mouse as published by the Singer lab \citep{Park2014}. In this paper they insert multiple MS2 binding loops into the 3\textprime~-UTR of \textit{$beta$-actin}, and cross it with another mouse that has MCP (MS2 Bacteriophage capsid protein) fused to GFP. Using this system they can visualize \textit{endogenous} mRNAs in cultured MEF cells, and brain sections. How could we apply this to imaging of piRNA precursor transcripts? If we inserted an MS2 loop into the tail end of transcripts, what would we see? It is certainly worth it!

  \subsection{Precursor Sequencing}
    \label{Disc:subsec:Sequencing of Precursors}

    \todo[inline]{Write FISSEQ application to piRNA precursor Tx} % **************************

    % When researchers measure gene expression, they typically read the aggregate RNA abundance over a population of cells. But just as not all stadium-goers cheer when the home team scores during a baseball game, single cells don't necessarily behave the same as the population as a whole. As a result, researchers increasingly are turning to single-cell methods.

    % Single-cell transcriptome sequencing (RNA-seq) is one option, but it loses subcellular localization information by effectively treating the cell as a homogenous sphere. Another choice, RNA fluorescent in situ hybridization (FISH), maintains subcellular detail but supports exploration of only four or five transcripts at once.

    %Now George Church, Professor of Genetics at Harvard Medical School, and colleagues have developed a technique they call FISSEQ (fluorescent in situ RNA sequencing), which blends the power of RNA FISH and RNA-seq to report the subcellular localization of several thousand genes simultaneously (1).

    %The new RNA-focused method relies on rolling circle amplification to create a three-dimensional grid of “DNA nanoballs,” each representing a single RNA molecule locked in its original subcellular location. SOLiD ligation-based sequencing then reads 27 to 30 bases from each nanoball, and the fluorescent output is read in three dimensions via confocal microscopy.

    %The method identified several thousand transcripts per experiment (4171 in one experiment, 8102 in a second), showing relatively good concordance with both RNA-seq and DNA microarray datasets for moderately expressed genes. Low and high abundance genes correlated less well, but the former are less reliably quantitated in all methods, Church noted, whereas the latter are mostly protein- and RNA-processing transcripts, which typically are bound up in nucleoprotein complexes. “Probably the only ribosomal RNAs we detect haven't been packaged into ribosomes,” he speculated. Importantly, FISSEQ not only picked up expected transcripts, but also matched their expected subcellular localizations.

    %Mats Nilsson, Professor of Biochemistry and Biophysics at Stockholm University, who in 2013 described a similar in situ transcriptomic approach (2), noted that Church's method is unbiased, whereas Nilsson's used molecular inversion (padlock) probes to amplify specific transcripts.

    %“I really think this is an important step towards more contextual sequencing, bringing next-generation sequencing into the spatial domain,” he said.

    %Church's group is applying FISSEQ to studies of mouse and human brains, where the method can be applied to classify cells and, if paired with barcoding, even map cellular connectivity. “We see this as one of the key enabling technologies for the BRAIN initiative,” said Church.
    %References
    %[1] J.H. Lee et al., “Highly multiplexed subcellular RNA sequencing in situ,” Science, 343:1360-3, Feb. 27, 2014.
    %[2] R. Ke et al., “In situ sequencing for RNA analysis in preserved tissue and cells,” Nat Meth, 10:857-60, 2013.

\section{A future for RNA-templated DNA-to-DNA ligation}
  \label{Disc:sec:SeqZip Improvements}

  The SeqZip methodology as developed and described in Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod} works adequately and robustly for characterization of relatively simple (\cd{}) to extremely complex (\dscam{}) genes. However, there is still substantial room for method optimization. Two obvious areas of improvement include the reduction of ligation time and NTL product formation. The improvements and modifications discussed below should assist in achieving these two (among other) goals.

  \subsection{An optimized SeqZip examination of Coordinated splicing}
    \label{Disc:subsec: Ideal SeqZip exp. to look for Coordination}

    If I could go back in time 4 years and still possess the knowledge and abilities that I do now, I would have approached a genome-wide study of coordination in splicing using SeqZip much differently. First, I would have focused on alternative first exon (or promoter, or TSS) and potential coordination with downstream cassette exons. I would have mined newly-generated RNA-Seq data \citep{Wang2008, Pan2008} for alternative first exons and cassette exons of sufficient expression. Then, I would have used my automated ligamer design software (see Appendix \ref{Appendix:AutoMatedLigamerAssembly}), to create a database of the required ligamers. As this would require at least 3 ligamers per event, with very little duplicated use of ligamers, the number of ligamers would make standard synthesis, even using 384-well plates, impossible. Therefore, I would have pursued printing the ligamers on a custom microarray, and cleaved them into solution, similar to products offered by \href{http://www.nimblegen.com/}{Nimblogen}. These ligamers would be barcoded and priming sequences included such that short (50nt) paired-end reads could reliably identify the templated first and cassette exons. Using this pool of ligamers, I would have performed the SeqZip assay, including a barcoding scheme to quantify the number of ligation events per {alt first exon::cassette exon} pair. Also, the libraries would have been amplified via digital PCR allowing me to check for PCR jackpots \citep{Shiroguchi2012a}. Finally, the data would be aligned against a reference of all \{alt first exon::cassette exon\} pairs, and any potential coordination determined.

    If I could have done the experiment designed above, I feel the full potential and utility of the SeqZip method could have been realized and generated new and valuable knowledge for the field of gene expression.

  \subsection{LNA-containing ligamers and T39A Rnl2}
    \label{Disc:subsec:LNA-Containing ligamers and T39A Rnk2}

    The use of an RNA-base on the 5\textprime~side of the nick encourages a C3\textprime~\textit{endo} sugar pucker for the base, placing the 3\textprime~OH in an apical orientation relative to the the AMP leaving group (see Figure \ref{Disc:fig:Rnl2 and suger pucker}) \citep{Nandakumar2006}.

    \begin{figure} % Rnl2 Suger Pucker
      \centering 
      \includegraphics{Figures/Discussion/Rnl2_and_suger_pucker.eps}
      \caption[Suger pucker in Rnl2 structures]
      {Suger pucker in Rnl2 structures \\[0.25cm]
        Using two different nucleic acid substrate combinations crystallized with Rnl2, \citet{Nandakumar2006} demonstrates the effect of 3\textprime~and 2\textprime~ identify of the base at the 5\textprime~side of the nick: Left) The crystal structure (PDB: 2HVS), containing a 2\textprime~ position deoxy residue, displays a DNA-like C2\textprime~\textit{endo} sugar pucker. In contrast to Right) where the crystal structure (2HVR) contains a 2\textprime~ hydroxyl and displays an RNA-like C3\textprime~\textit{endo} sugar pucker.
        }
        \label{Disc:fig:Rnl2 and suger pucker}
        \end{figure}

    With lowered costs of oligo synthesis, incorporation of 2\textprime~OMe at the penultimate and ultimate bases of the 5\textprime~nick ligamers should greatly increase ligaiton efficiency, as these are the primary substrate-specificity determinants of Rnl2 \citep{Nandakumar2004a, Nandakumar2006}.

    Interestingly, \citet{Nandakumar2004a} demonstrated that the ribosome at the penultimate position, evidenced by a 50-fold reduction in turnover number for 2\textprime~H substitutions at this position, compared to full activity for 2\textprime~OMe. \citet{Nandakumar2006} (see Figure \ref{Disc:fig:Rnl2 and suger pucker} shows that Threonine 39 (T39) hydrogen bonds with both the 2\textprime~OMe and 3\textprime~O of the penultimate suger. A T39A mutation did not phenocopy the 2\textprime~H substitution on the penultimate suger, indicating that suger pucker is the structural constraint on ligation efficiency. These results indicate that a T39A mutant of Rnl2 may demonstrate increased RNA-templated DNA:DNA ligation efficiency, as it would have one less molecular requirement for RNA.

    Future versions of the SeqZip assay could use a combination of LNA modified bases \citep{You2006} at either the penultimate or terminal residues on the 5\textprime~side of the nick in order to increase 1) specificity, and 2) enzyme efficiency. The combination of these modifications in the ligamers could be combined with a T39A mutant form of Rnl2 for a potentially greatly enhanced ligation rate.

  \subsection{Thermostable Ligases}
    \label{Disc:subsec:Thermostable Ligases}

    The use of LNA-containing brings up issues involving off-target hybridization. Future iterations of the SeqZip methodology could use directed protein evolution of Rnl2 \citep{Stemmer1994, Romero2009a} to develop a thermostable variant of Rnl2, similar to variations of DNA ligase that have been used for years \citep{Barany1991}. Use of LNAs and elevated ligation temperatures could alviate off-target hybridization events reducing both nonproductive hybridization and non-templated ligation events. This would also allow for use of reduced overall ligamer concentrations, in line with the optimal SeqZip experiment described in section \ref{Disc:subsec: Ideal SeqZip exp. to look for Coordination} and synthesis on microarrays.

  \subsection{Other SeqZip Applications}
    \label{Disc:subsec:Future Uses of SeqZip}

    SeqZip can be used to many different forms of RNA sequence characterization. An incomplete illustration of these applications is shown in Figure \ref{Disc:fig: Panel of SeqZip Applications}. The three novel applications are discussed below:

    \begin{figure} % SeqZip uses Panel
      \centering 
      \includegraphics{Figures/Discussion/SeqZipUses.eps}
      \caption[Proposed uses of the SeqZip methodology]
      {Proposed uses of the SeqZip methodology \\[0.25cm]
        Shown here are general applications of the SeqZip method to profiling RNA sequences. Top row examples are substantiated by experiements described on Chapters \ref{SeqZipPaper} and \ref{SeqZipMethod}, the middle and lower rows are hypothetical but logical extensions of the method.
        }
      \label{Disc:fig: Panel of SeqZip Applications}
      \end{figure}

    \subsubsection{Multi-site SNP detection}
      \label{Disc:subsubsec: Multi-site SNP Detection}

      \todo[inline]{Mention Which side to put the Mismatch} % **************************
      % Cite the newest Shuman paper, and the Shuman crystal papers

    \subsubsection{SeqZip and single-molecule multi-site FISH}
      \label{Disc:subsubsec:SeqZip and Single-Molecule FISH}

      \begin{figure} % Multi-site Fish using SeqZip
        \centering 
        \includegraphics{Figures/Discussion/MultiSiteFish.eps}
        \caption[Multi-Site smFISH using flourophore-containg ligamers]
        {Multi-Site smFISH using flourophore-containg ligamers \\[0.25cm]
          \hl{Insert Caption Here}
          }
        \label{Disc:fig:MultiSite FISH using SeqZip}
        \end{figure}

        % What about applying the Church lab's new technique of in situ sequencing? Would this help us differentiate transcripts from mature piRNAs? 

    \subsubsection{Introduction of destruction sequences}
      \label{Disc:subsubsec:Intro of Desctruction Sequences}

      \todo[inline]{Discuss Type III restriction enzymes, etc} % **************************

    \subsubsection{Re-purposing the SOLiD Platform}
      \label{Disc:subsubsec:SOLiD Platform for SeqZip}

      The ability to incorporate custom sequences into ligations, which subsequently are incorporated into ligation products provides tremendous opportunities for downstream analysis (see figure \ref{Disc:fig: Panel of SeqZip Applications}).

      \todo[inline]{SOLiD Application of SeqZip} % **************************

\section{Final thoughts}
  \label{Disc:sec:Final Thoughts}

  This thesis has introduced the complexity, purpose, potential, and challenges of transcriptome study. There is no comparison between these issues with that of the DNA genome. The next period of biomedical knowledge will be heralded by advances in transcriptome analysis. This section discusses how scientists need to grow with technology.

  \subsection{Science vs. Engineering}
    \label{Disc:subsec:Science and Engineering}

    \begin{quote}
      \itshape 
      \singlespacing
        “There is a general attitude among the scientific community that science is superior to engineering.” - \citep{Macilwain2010}

      “Science is about what is; engineering is about what can be. Engineers are dedicated to solving problems and creating new, useful, and efficient things.“ - Niel Armstrong
      \end{quote}

    A common schism between technically-oriented individuals is whether or not they identify themselves as an engineer or a scientist. The first quote, from an article published in Nature, communicates a clear bias in academic circles of the importance of the \textit{why} over the \textit{how}. In essence, how one prioritizes these questions categorizes individuals as scientists (why is important) or engineers (how is more important). The second quote, from the first man to walk on the Moon, Neil Armstrong, highlights what motivates a self described ``engineer'' and ``geek.'' How does a graduate system--- training PhDs for careers in life science--educate individuals who fall into these two fundamentally different belief systems?

    Not well. When searching for a lab I told professors that I wanted to work on a technology development project. A typical response was, ``That's not what we do here.'' As someone who is more interested in the ``how'' over the ``why,'' this began a brief period when I thought I had made the wrong choice leaving industry and going back to graduate school. What was the basis for this aversion to technology development? The same article in Nature states that this feeling toward engineering may be attributed...

    \begin{quote} 
      \itshape 
      \singlespacing
      ...partly to a ``linear'' model of innovation, which holds that scientific discovery leads to technology, which in turn leads to human betterment. This model is as firmly entrenched in policy-makers' minds as it is intellectually discredited. As any engineer will tell you, innovations, such as aviation and the steam engine, commonly precede scientific understanding of how things work.
      \end{quote} 

    If policy-makers value basic discovery over technological application, perhaps this explains why many of my professors tried to steer me away from a technology development project.

    In spite of policy-makers and my professors holding the viewpoint that discovery precedes technology, some of the most notable breakthrough scientific discoveries, including many made by Nobel Laureates, demonstrate a clear integration of both the scientific method and technological application. For example, the 2007 award in Physiology and Medicine was given for ``discoveries of principles for introducing specific gene modifications in mice by the use of embryonic stem cells."  By combining these principle discoveries, an indispensable technique in modern genetics was created---gene targeting. The feeling of which is more important, the principle discoveries or the application thereof, is likely what separates a scientist from an engineer.

    The importance of technology to the advancement of science in general is not limited to anecdotes resulting in a Nobel prize. A quick scan of the most \href{http://www.pnas.org/reports/most-cited}{highly-cited} papers in the journal PNAS reveals that the top 13, indeed \textit{all} 13, are about a novel methodology or technique. Sequencing of DNA, microarray analysis, tetracycline-inducible promoters, recombinant adenovirus, and site-specific mutagenesis are just a handful of the tools on the list. This effect can also be seen in \href{http://simplystatistics.org/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/}{computation biology}, with transformative algorithms, such as BLAT \citep{Altschul1990} and Bowtie \citep{Langmead2009} attain citations well beyond a typical paper in their journal of publication and far more than most primary research articles.    

    The growth of big datasets is forcing all in biomedical research to think like an Engineer. At least two major concerns require immediate attention: How to store the data and how to analyze it?

  \subsection{The Data Deluge}
    \label{Disc:subsec:Dealing with Data Deluge}

    \begin{quote}
      \itshape
      \singlespacing
      The HiSeq X Ten is sold as a set of 10 or more ultra-high throughput sequencing systems, each generating up to 1.8 terabases (Tb) of sequencing data in less than three days or up to 600 gigabases (Gb) per day, per system, providing the throughput to sequence tens of thousands of high-quality, high-coverage genomes per year. ---\href{http://bit.ly/PZpegZ}{Illumina Press Release}
      \end{quote}

    In a world where the HiSeq X, described above, is a reality, biomedical researchers need to change how they approach every aspect of data analysis. This includes storage, processing, and visualization. 

    How do we work with all this data? Mention LabKey, GenomeBridge, Define the scope of the problem, etc...

    I would love to have a database of complete transcripts of a cell - think about how you were panning the data for the MolCel paper, and you could see ChIP marks, Pol II occupancy, RNA-Seq, and piRNA data. It would be great to be able to do that more often, and we greater precision

  \subsection{Biologists need Computation Biological Skills}
    \label{Disc:subsec:Biologists need Comp Skills}

    Just 10 years ago,  Graduate students and PhDs in the fields of Molecular Biology or Biochemistry need not venture far from data analysis within Excel or perhaps a statistical program with an advanced graphical interface (examples include Prism or Graphpad). Software knowledge that stops at these tools and the rest of the Microsoft Office suite of tools is no longer enough to generate big strides in Biomedical research.

    Working with tens of even hundreds of lines of data within a spreadsheet is manageable and computers from 20 years ago had more then enough computing power to process the data. Yet, this type of data is longer then endpoint of most cutting edge projects. Many students and post docs often find that they are unable to analyze the data generated from months or years of tireless bench work. Faced with learning what is effectively a collection of new languages and awash in a sea of acronyms (LINUX, BASH, GNU, PERL, R) they reach out for help from a ``Bioinformatics person.'' Perhaps the relationship and interaction with this personal is productive, leading to a collaboration and exciting new knowledge. Sometimes it isn't, and the bench scientist shifts into one of three modes: 1) Wait; 2) Find another bioinformatic-minded collaborator; or 3) collect more data.

    In my experience, the most often chosen mode is ``wait.'' This is also the most damaging, as it delays the progress of one's work, and the advancement of science in general. Personally, I did not want to fall into this mode, and once the multiplex study described in section \ref{SeqZipMethod:sec:Multiplex Gene Study} reached a point where I had millions of sequencing reads, but I could not find anyone to help me analyze the data, that I decided to educate myself on the basic principles of Linux, the command line, and analysis of HTS data.

    A Biologically-train individual who posses the knowledge of analysis of HTS datasets is an extremely powerful and empowering situation. This was recently commucated in \citet{Plocik2013}:

    \begin{quote} % Polick2013 Quote about insight without a Pipette
      \itshape 
      Such exercises will empower students to explore and assess the quantitative data published in the manuscripts that they read, which can no longer be assessed at a glance like the qualitative gel-based results on which molecular biology was founded. Ultimately, it will be equally important to know how to write code as it is to pipette. - \citep{Plocik2013}
      \singlespacing
      \end{quote}

    The fact is that no one will care about a project as much as the Graduate student or PostDoc who is the main project driver. Learning and training of computational skills bent on analyzing large datasets should be central to the education in Biomedical sciences in the future.
