\chapter{Discussion}   \label{Chapter 5} 
\lhead{Chapter 5. \emph{Discussion}} 
%----------------------------------------------------------------------------------------
\section{Orphan Discussion Writing Fragments}

\section{What are the questions generated by my work?}

    \begin{itemize}
      \item What determines AS splicing decisions? It is not connectivity in splicing, and it seems to be SR and hnRNP proteins. Splicing is tissue specific. So is it the tissue-specific expression of SR and hnRNP proteins the is the main determinant of AS outcomes? How does chromatin state and organization play into AS decisions? 
      \item What determines that a seemingly mRNA-like piRNA precursor is processed into piRNAs, and not translated like crazy by Ribosomes?  Everything meets the ribosome ($REF$), so why is it that precursors are given a different lot in life?
      \item How does one perform deep and broad analysis of mRNAs using second-generation HTS give the tremendous log-range over which they are expressed?
      \item What controls the stochastic and probabilistic splicing of \dscam{}? Why is it different between hemocytes and neurons? Why would hemocytes need less apparent diversity, given the range of antigen they could potentially encounter?
      \item The field needs better tools to study RNA expression in a high-throughput manner. This is discussed in Brent's paper. Many molecular biologists do not have the necessary skills to perform the required analysis. The barrier of performing these type of analysis needs to be lowered by 1) Better software tools 2) More appropriate and targeted education 3) A change in how molecular biologists think 4) Redefining how we think of a gene 5) others?
      \item What experiment would I really like to be able to do??
      \item I would love to have a database of complete transcripts of a cell - think about how you were panning the data for the MolCel paper, and you could see ChIP marks, Pol II occupancy, RNA-Seq, and piRNA data. It would be great to be able to do that more often, and we greater precision
      \item How to you organise and visualize this type of data?
      \end{itemize}

\section{Alternative Splicing turns gene expression into a game of cards}
    %Start with Deck of Cards. Wikicommons on Deck of cards? Combinatoral nature of splicing turns RNA-Seq into a BIG Data problm. 20K genes, but >100K possible transcripts.  This would let me lead into the Graveley paper and then lead into RNA-Seq

    A deck of cards has only 52 cards. Yet these cards can be delt into XXX permutations of 7 cards. It is these numbers that makes possible to play Poker for hours on end, and the likelyhood of obtaining that ``Royal'' flush so incredably low. Long ago, genes discovered how they could arrange themselves into unique and rare combinations, especially in eukaryotic organisms. Indeed the process of splicing is closely correlated with organism complexity. The process of AS is even more closely tied to organism complexity (see figure \ref{fig:numGenesAndNumSpliced}). 

   % Implications for discrination past one's DNA as it is the actual PRODUCT of the DNA and the actual biology (or at least closer to the functional biology) that is going on inside of every person

\subsection{The Future of Dynamic long RNAs}

    Deep sequencing of transcriptomes has revolutionized biology. Previously, transcript discovery was a cumbersome task. Transcript identification and characterization involved significant labor, cost, and materials. In the mid-90's, microarray technology \citep{Schena1995a} gave us a tantalizing glimpse into how genes were expressed, but were limited to probed, and therefore known, sequences. Yet, the green and red landscapes of a microarray analysis hinted at incredible complexity \textemdash a complexity that would have to wait for technology to catch up.

    Like many transformative technologies, RNA-seq was made possible by incremental improvements to numerous supportive technologies such as: 1) digital optics, 2) microscopy, 3) slide chemistry and on-slide PCR and 4) nucleic-acid alignment. A HiSeq 2500 relies on all of these technologies (and others) to produce the 100M+ sequences that allow Scientists to peer every day into the transcriptional output of a genome.

    In the past 5 years, biologists have started to think way beyond mRNAs and small RNAs. The former captured out interest for 30+ years \citep{Furuichi1975,Wei1975}, and the later has been on a run-away trail since capturing out attention in 1998 \citep{Fire1998}. HTS has added long RNAs (among others) to these classes of gene RNA products. However, many biologically-trained and minded Scientists find themselves overwhelmed by the complete different methods and approaches to tackling the ``big data'' created by modern genome-wide experiments. Experimental training does not currently provide students with the required skills in statistics, computer programing, and experimental design that are needed to work with genome-wide data. The richness of this data often leaves many unasked (and unanswered) test-able hypothesis just sitting in public repositories \citep{Plocik2013}.

    Being such a novel area of extremely basic research, and to borrow a few seemingly inane but rather insightful trio of phrases from the United States Secretary of Defense Donald Rumsfeld \citep{Rumsfeld2011}, their remain at least three important areas of knowledge concerning long RNAs of the transcriptome: 
    \hyperref[subsec: The Known Knowns]{''The Known Knowns''}; 
    \hyperref[subsec: The Known Unknowns]{''The Known Unknowns''}; 
    and the \hyperref[subsec: The Unknown Unknowns]{''The Unknown Unknowns''}.

  \subsection{The Known Knowns}\label{subsec: The Known Knowns}
    %-----------------------------------

    At this point, it is important to remember that in this document   \textit{long RNAs} may also refer to products containing characteristics of traditional mRNAs, that is a 5  \textprime~m7G Cap, ligated exons, and a Poly(A) tail. However, many of these long mRNAs are extremely dynamic. So much so that until HTS and RNA-Seq, comprehensive investigation of their complexity was not possible.


    \begin{description}
      \item[Pervasive transcription]
      Here you can put some information from ENCODE and your thoughts on it.

      \item[Tissue and cell specificity]
      Your feelings on Specificity of long RNA expression

      \item[Functional]
      We   extit{know} that some long RNAs are functional. What are these?

      \item[Chromatin regulation]
      We also know that some long RNAs regulation Chromatin structure. What are your feelings as to the importance of this fact?

      \item[PTGS]
      Long RNAs ability to do post transcriptional gene regulation, including piRNAs, and Xist, etc....

      \end{description}

  \subsection{The Known Unknowns}\label{subsec: The Known Unknowns}

    \begin{description}
      \item[How are they important?] 
      Conservation of these things is not obvious - if they are not conserved - are they important? Maybe talk about how MALAT1 is highly expressed, but seems to be dispensable.

      \item[What regulates their tissue-specific expression?]
      Do they important some of the special sauce that makes tissues different from one other, more so then the mRNAs changes which can be extreme, but not terribly so....

      \item[]  
      \end{description}

  \subsection{The Unknown Unknowns}\label{subsec: The Unknown Unknowns}

    This is the area of knowledge keeps many motivated to perform basic research every day. What secrets does the transcriptome have in store that we haven't even \textit{thought} about? Only through pushing the boundaries of the last two sections can we begin to think beyond the edge of map and formulate testable hypothesis. Here I propose a few outlandish ideas for Unknown Unknowns. 

\section{SeqZip Improvements}

  \subsection{Assay Modifications}

    \subsubsection{Rnl2 T29A mutation}

    Use of T39A mutation to alleviate penultimate 2\textprime~ OH requirement of T4 Rnl2 (See Nandakumar...Lima, Cell 2006)

    \subsubsection{Thermostable Ligases}

    Use of thermostable ligase, allowing for multiple rounds of ligation. Need a good reference, DO NOT USE Ref 27 from Conze et al 2009! Also Elevated ligation temperatures, minimizing blut-ended NTL events

    \subsubsection{LNA-containing Ligamers}

    Make a note into the future directions that you would like to explore LNA’s at the 3  extprime~OH position of all ligation results, leading to increased ligation efficiency

    \subsubsection{Ribose-containing Ligamers}

    however both this and the use of penultimate 2  extprime OH (Ribosome) suger in your ligamers would lead to added costs, and the latter maybe better served with a T39A mutation. Giggity

    \subsubsection{Repurposing the SOLiD Platform}

    \subsubsection{Others}

    \begin{itemize}
      \item Digital PCR of the PCR products ala \citep{Shiroguchi2012a}. 
      \item SeqZip on the SOLiD platform
      \item SeqZip on single-cell RNA samples.  
      \end{itemize}

  \subsection{An ideal SeqZip experiment to query coordinated splicing}
    %Need to discuss coordination in splicing. A systematic examination of alternative first exon use tied to downstream splicing events would be very useful. How best to do this with current or near, technologies? Enrichment for Cap would be a good start. But how do get valuable downstream? Do you combine the analysis with a Seqzip approach? Or some other sequencing platform? We are only interested in the cassette exons and alternative processing events... How to enrich for them? Again that would be a Seqzip like approach. 
    If I could go back 4 years and still possess the knowledge and abilities that I do now, I would have approached a genome-wide (or multiplex) study of coordination in splicing using SeqZip much differently. Firstly, I would have focused on alternative first exon (or promoter, or TSS) use being coordinated with downstream cassette exon inclusion. I would have mined newly-generated RNA-Seq data \citep{Wang2008, Pan2008} for alternative first exons and cassette exons of sufficient expression levels. I would have used my automated ligamer designing software (see Appendix \ref{AppendixC}), to create a database of the required ligamers to maintain connectivity and observe connectivity between alternative first exons and a single downstream cassette exon. This would require at least 3 ligamers per event, with very little duplicated use of ligamers. To overcome what would be a significant burden and cost of synthesizing such a large number of ligamers, I would have pursued having them printed on a custom microarray, and cleaved into a bulk solution, similar to products offered by ($NUC BIO?$). These ligamers would be barcoded and priming sequences included such that short (50nt) paired-end reads could reliably identify the templating first and cassette exons. Using this pool of ligamers, I would have performed the SeqZip assay, also including a barcoding scheme to quantify the number of ligation events per alt first exon::cassette exon pair, and amplified the library via digital PCR allowing me to check for PCR jackpots \citep{Shiroguchi2012a}. The library would be have been analyzed using a paired-end 50 nt read apporach on a GEIIx Illumina machine. The data would be aligned against a reference of all alt first exon::cassette exon pairs, and any potential coordination determined.

    If I could have done the experiment designed above, I feel the full potential and utility of the SeqZip method could have been realized and generated new and valuable knowledge for the field of gene expression.

  \subsection{SeqZip and single-molecule FISH}
    %Can you use Seqzip as a platform for imaging? I should ask Akiko for her references concerning in vivo fish. 
    The cellular location of precursor piRNA transcript processing is not known. The most accepted hypothesis is that precursor transcripts are processed into mature piRNAs with machinery tethered to chromatoid bodies ($REF$) or another structure similar to Drosophila Nuage ($REF$) near the mitochondrial cement ($REF$). Knowledge of   extit{where} mature piRNAs are generated would provide clues into larger mechanist details of their biogenesis. 

    I can think of two broadly different ways in which we could pinpoint the physical location of mature piRNA generation. The first is to chemically label, in some manner, primary piRNA transcripts. The label would need to 1) not interfere with processing and 2) be durable to later methods used to analyze the presence or absence of the modification. A second way to identify the location of mature piRNA processing would be to actually observe, through in vivo \hyperref[hd:abrevs]{FISH}, the various products and by-products of biogenesis. In this particular approach, SeqZip maybe useful.

    % MS2 lopp introduction into piRNA precursor transcripts.
    Another idea here would be to use the MCP x MBS mouse as published by the Singer lab \citep{Park2014}. In this paper they insert multiple MS2 binding loops into the 3  extprime~-UTR of the $beta$-actin gene, and cross it with another mouse that has MCP (MS2 Bacteriophage capsid protein) fused to GFP. Using this system they can visualize   extit{endogenous} mRNAs in cultured MEF cells, and brain sections. How could we apply this to imaging of piRNA precursor transcripts? If we inserted an MS2 loop into the tail end of transcripts, what would we see? It is certainly worth it!
    % What about applying the Church lab's new technique of in situ sequencing? Would this help us differentiate transcripts from mature piRNAs?  

\section{Mammalian piRNA-precursors, a special type of long RNA}

  %Unanswered questions/issues of the resources paper?

  % What about the work that MJM asked  you to do considering the mRNAs that change over spermatogensis?

  \subsection{What are they doing?}
    %-----------------------------------

    %Interaction w/ ribosomes?

  \subsection{How are they generated?}
    %-----------------------------------

    %Interaction w/ ribosomes?
    %How does cell partition prepachytene transcripts to mRNA or piRNA use?

  \subsection{Why should we care?}
    %-----------------------------------

\section{Needle in a haystack: piRNA precursor transcripts surrounded by RNA}

  \subsection{In vivo chemical labeling of precursor piRNA transcripts}
    % Need section here on the chemical modification - what were you thinking? Azide? Par-Clip?  Maybe a CLASH-like assay?  What about what Mihir is developing?
    
    SeqZip accurately reports on the presence of multiple, distant sequences contained in the same RNA molecule (see Chapters \ref{Chapter2} and \ref{Chapter3}). Yet I was not able to observe ligation products templated off some of the most highly-expressed precursor transcripts (see section \ref{sec: precursor TX}. We hypothesize that the most likely explanation for being unable to observe ligation products for these transcripts, but successfully observing those for a very long, but lowly expressed gene   extit{Dst}, suggests that precursor transcripts are rapidly processed into mature piRNAs. What if this is not the case if we could hybridize ligamers to precursor transcripts in vivo? Put another way  extemdash What if the steady-state amount of precursor transcript is very low due to rapid processing, but when visualized in real time, the transcripts are of sufficient abundance for FISH? Ligamers could be engineered to contain different combinations of fluorescent dyes, and the proximity and intensity of the signal could be used to infer precursor transcripts. Importantly, this approach could distinguish long, continuous precursor transcripts from processing intermediates and mature piRNAs.

  \subsection{In vivo imaging of precursor piRNA transcripts}
    %Another idea would be to image the precursors in real time somehow. How about tagging them and following them in the sperm? That could be cool. You could even follow them into the developing embryo. 
    The most important question for mammalian   extit{pachytene} piRNAs is   extit{What are they doing?}. We know that they are essential for the health of the species, as discussed in section \ref{c4-intro}, and piRNA-pathway mutants are sterile. What could these small RNAs, with complementarity to nothing but themselves, be doing? Building on the last section discussing chemical modification precursor transcripts, if the modification was again durable enough for downstream analysis, perhaps the modification would remain in the mature piRNAs that are incorporated into mature sperm. These modifications could be tracked as the sperm move through the \hl{semineferous tubues} and into the $SPERM ANATOMY$. One could even track the piRNAs as they fuse with the oocyte to create an embryo. If this modification was labile, piRNA interacting proteins or nucleic acids could be captured or marked as well, providing additional clues as to the function of piRNAs in sperm and early embryogenesis.

\section{Final thoughts}

  %\subsection{Why did I want to study AS?}

    % Highschool Biology class. That is where I first learned about how 'genes' - this things which I always heard everthing was ``in'' - actually worked.  There was a picture of a squiglly line called mRNA. This line was broken into boxes marked exons.  mRNA went on to code for protein, and along with a good part water, is what I was made of.  Made sense.  A sublte note on the page drew my eye. It pointed to one of the exon boxes and said ``alternative splicing.'' What's this?  The arrangement of the exons is not set in stone? It was around this time that the human genome was being finialized, we had soemthing like 3.9 billion basepairs, and this little box only contained ~150 bases.  The implication struck me immediately.  If these boxes could be alternatively arranged, then the possibilities of unique mRNA molecules is staggering!  Far more then the 20K protein coding genes we are now known to possess.  Something that also made sense - this MUST be the reason why humans are so much more advanced then everything else - our mRNAs are alternatively spliced ( I didn't realize that even X a very simple organims also alternatively splices its genes.  

    %That was ~10 years  before I found myself development a methodology that would greatly assist in the routine measurement of mRNAs that are alternatively spliced.  Strange to think that I was amazed with the biological process I would go on to study.  I was similaly amazed to learn about sanger sequencing - it seemed so elegant to me.  Another topic that truely interests me is HTS. 

  \subsection{All Molecular Biologists now need Computation Biological Skills}

    Just 10 years ago,  Graduate students and PhDs in the fields of Molecular Biology or Biochemistry need not venture far from data analysis within Excel or perhaps a statistical program with an advanced graphical interface (examples include Prism or Graphpad). Software knowledge that stops at these tools and the rest of the Microsoft Office suite of tools is no longer enough to generate big strides in Biomedical research.

   Working with tens of even hundreds of lines of data within a spreadsheet is manageable and computers from 20 years ago had more then enough computing power to process the data. Yet, this type of data is longer then endpoint of most cutting edge projects. Many students and post docs often find that they are unable to analyze the data generated from months or years of tireless bench work. Faced with learning what is effectively a collection of new languages and awash in a sea of acronyms (LINUX, BASH, GNU, PERL, R) they reach out for help from a ``Bioinformatics person.'' Perhaps the relationship and interaction with this personal is productive, leading to a collaboration and exciting new knowledge. Sometimes it isn't, and the bench scientist shifts into one of three modes: 1) Wait; 2) Find another bioinformatic-minded collaborator; or 3) collect more data.

    In my experience, the most often chosen mode is ``wait.'' This is also the most damaging, as it delays the progress of one's work, and the advancement of science in general. Personally, I did not want to fall into this mode, and once the multiplex study described in section \ref{sec: Multiplex Gene Study} reached a point where I had millions of sequencing reads, but I could not find anyone to help me analyze the data, that I decided to educate myself on the basic principles of Linux, the command line, and analysis of HTS data.

    A Biologically-train individual who posses the knowledge of analysis of HTS datasets is an extremely powerful and empowering situation. This was recently commucated in \citet{Plocik2013}:

    \begin{quote} % Exodus Quote
      \itshape 
      Such exercises will empower students to explore and assess the quantitative data published in the manuscripts that they read, which can no longer be assessed at a glance like the qualitative gel-based results on which molecular biology was founded. Ultimately, it will be equally important to know how to write code as it is to pipette. - \citep{Plocik2013}
      \singlespacing
      \end{quote}

    The fact is that no one will care about a project as much as the Graduate student or PostDoc that is the main driver. Learning and training of computational skills bent on analyzing large datasets should be central to the education in Biomedical sciences in the future.

  \subsection{Science versus Engineering: Two thoughts in one school}

    \begin{quote}
      \itshape 
      \singlespacing
        “There is a general attitude among the scientific community that science is superior to engineering.” - \citep{Macilwain2010}

      “Science is about what is; engineering is about what can be. Engineers are dedicated to solving problems and creating new, useful, and efficient things.“ - Niel Armstrong
      \end{quote}

    A common schism between technically-oriented individuals is whether or not they identify themselves as an engineer or a scientist. The first quote, from an article published in Nature, communicates a clear bias in academic circles of the importance of the   extit{why} over the   extit{how}. In essence, how one priorities these questions may categories individuals as a scientists (why is important) or an engineer (how is more important). The second quote, from the first man to walk on the Moon, Neil Armstrong, highlights what motivates a self described ”engineer” and ”geek.” How does a single graduate system, training PhDs for careers in life science, educate individuals who fall into these two fundamentally different belief systems?

    In short  extemdash not well. When searching for a lab to call home, I told professors that I wanted to work on a technology development project. A typical response was, “That's not what we do here.” As someone who is first interested in the ``how'' over the ``why,'' this began a brief period when I thought I had made the wrong choice in leaving industry to go back to graduate school. What was the basis for this aversion to technology development? The same article in Nature states that this feeling toward engineering may be attributed:

    \begin{quote} 
      \itshape 
      \singlespacing
      ...partly to a ``linear'' model of innovation, which holds that scientific discovery leads to technology, which in turn leads to human betterment.  This model is as firmly entrenched in policy-makers' minds as it is intellectually discredited.  As any engineer will tell you, innovations, such as aviation and the steam engine, commonly precede scientific understanding of how things work.
      \end{quote} 

    If policy-makers value basic discovery over technological application, perhaps this explains why many of my professors tried to steer me away from a technology development project.

    In spite of policy-makers and my professors holding the viewpoint that discovery precedes technology, some of the most notable breakthrough scientific discoveries, including many made by Nobel Laureates, demonstrate a clear integration of both the scientific method and technological application.  For example, the 2007 award in Physiology and Medicine was given for “discoveries of principles for introducing specific gene modifications in mice by the use of embryonic stem cells."  By combining these principle discoveries, an indispensable technique in modern genetics was created – gene targeting.  The feeling of which is more important, the principle discoveries or the application thereof, is likely what separates a scientist from an engineer. 

    The importance of technology to the advancement of science in general is not limited to anecdotes resulting in a Nobel prize. A quick scan of the most \href{http://www.pnas.org/reports/most-cited}{highly-cited} papers in the journal PNAS reveals that the top 13, indeed   extit{all} 13, are about a novel methodology or technique. Sequencing of DNA, microarray analysis, tetracycline-ineducable promoters, recombinant adenovirus, and site-specific mutagenesis are just a handful of the tools on this list. This effect can be seen in \href{http://simplystatistics.org/2014/04/07/writing-good-software-can-have-more-impact-than-publishing-in-high-impact-journals-for-genomic-statisticians/}{computation biology}, with transformative techniques, such as BLAT \citep{Altschul1990} and Bowtie \citep{Langmead2009} attaining citations well beyond a typical paper in their journal of publication and far more than most primary research-centered articles.

    How does someone who is motivated by the engineering of science best contribute in an academic setting? Luckily, I did not have to question my decision to return to school for long. I found a pair of labs where I could learn how to practice traditional hypothesis-driven research while also developing new tools necessary to do so. My project is a perfect fit for me and has been terrific fun to work on. In the past five years, the technique that I developed, SeqZip, allows for the more efficient study of  mRNA isoforms produced from alternative splicing of pre-mRNA. This is a very engineering-type accomplishment. However, the technique uses short DNA oligonucleotides and a novel activity (that I discovered) of an known RNA ligase to shorten and simplify isoform sequence information. Use of Rnl2 to perform RNA-templated DNA:DNA ligation is completely new knowledge, and falls squarely within a scientific purview. The technique simplifies many of the experimental issues that researchers struggle with when studying the often complex products of alternative splicing.
    % Need transition or concluding paragraph
