% My Thesis Introduction
% 2014-03-25
% Christian Roy
\chapter{Introduction} % Main chapter title
\label{Chapter1} % To refernce this chapter elsewhere, use \ref{ChapterX}
\lhead{Chapter 1. \emph{Introduction}} % Change X to a consecutive number
% this is for the header on each page - perhaps a shortened title
%----------------------------------------------------------------------------------------

%----------------------------------------------------------------------------------------
\section{On the importance of gene expression} %	SECTION 1
%----------------------------------------------------------------------------------------

%% I don't think I need to start with this!
%Man has always been fearful of Nature's power. Before man's transition into civilization, virtually everything was feared. Lightning, hurricanes, and blizzards\textemdash all seem divine in their origin and power. After the start of civilization, man's dependence on agriculture to supply crowded populations with food encouraged any number of prayers, customs, sacrifices, to higher powers for a good harvest. Today, while we harvest both food and energy, we are still at the mercy of natural forces.

The Old Testament chapter Exodus tells of the liberation of the Israelite people from Egyptian slavery. Their humble and reluctant leader Moses, acting under the direction of God, forces the Pharaoh Ramses to release the people of Israel through a series of 10 plagues. Pharaoh is stalwart and stubborn as he watches water turn to blood. As frogs, lice, and flies flood the city streets, he refuses to free the Israelites. When Egyptian livestock fell dead from disease,   people and animals both were covered in boils, and land burned in storms of fire, Pharaoh did not bend. The 8th plague was a swarm of Locusts, described in Exodus 10: 14–15:

\begin{quote}
	\itshape % This will be italicized quote
	\singlespacing
	\textsuperscript{14} And the locusts went up over all the land of Egypt, and rested in all the coasts of Egypt: very grievous were they; before them there were no such locusts as they, neither after them shall be such.\\
	\textsuperscript{15} For they covered the face of the whole earth, so that the land was darkened; and they did eat every herb of the land, and all the fruit of the trees which the hail had left: and there remained not any green thing in the trees, or in the herbs of the field, through all the land of Egypt.
\end{quote}

The desolation left by the locust plague was not enough to persuade Ramses. Nor was three days of darkness. Only the death of all first-born Egyptians, included Ramses own son, was enough to persuade Pharaoh to let the Israelites leave Egypt.

The power of a locust swarm is not just a fanciful biblical story, and is perhaps the most *believable* of the 10 plagues. In current times, the United Nations' (UN) Food and Agriculture division maintains a \href{http://www.fao.org/ag/locusts/en/info/info/news/index.html}{Locust watch website} providing weekly updates on potential locust swarms in northern Africa and middle east. The locust has long been, and continues to be, a powerful and feared force of Nature.

Unlike fire and brimstone from the heavens, locusts are something we can hold and study. Surely science can help us understand what triggers them to swam and cause massive destruction. We know that the desert locust, Schistocerca gregaria, is the main species of about 10 that swarms in vast numbers and causes extensive crop damage. They are members of the insect Order Orthoptera, whose other famous members include crickets and katydids. Orthoptern members make sound known as stridulation by vigorously rubbing their wings. They also undergo incomplete metamorphosis (formally Hemimetabolism), and do not have a pupal stage during development. 

\InsertFigure
{DesertLocust.jpeg}
	{The Solitary and Gregarious forms of \textit{Schistocerca gregaria}}
	{
		The two phenotypic forms of Schistocerca gregaria appear very different.  The Solitary form is green an generally larger, while its "gregarious" form is more brightly colored, smaller, and capable of swarming in vast numbers, destroying crops vegetation. Photo from \href{http://www.wikicommons.com}{Wikicommons}.
	}
	{fig:Locust}

\begin{itemize}
% < Insert other scary facts about Locusts>>.
  \item While only 2–2.5 inches and weighing 0.05–0.07 oz, can consume its own weight in food per day
  \item Can fly 60 miles in 5–8 hours
  \item Thought to be separate species from solitary form until 1921 
\end{itemize}

The power and destruction this animal can inflict makes it difficult to believe that it is nothing more than a grasshopper. It is nothing more than a grasshopper not just by analogy, but by actual Taxonomy. The infamous desert locust is actually the \textit{gregarious} form  of \textit{Schistocerca gregaria} (See Figure ~\ref{fig:Locust}), while the more familiar and docile looking Grasshopper is the \textit{solitary form}. Scientists are just now beginning to understand how it is possible for such a dichotomy to exist within the same organism, or more specifically, within the same \textit{genome}.

\textit{Schistocerca gregaria} is a polyphenic organism. Grasshoppers become locusts by going through a phase transition. Polyphenism is a general feature of insects, often stark in transformation. For example, pea aphids (\textit{Acyrthosiphon pisum}), which usually exist in an asexually reproducing, wingless female form, responds to overcrowding (often as a result of dwindling food supply) by producing winged offspring that travel to new sources of food \citep{Shingleton2003,Purandare2014b}.

%<Facts about polyphenism in plants> 
%<< Compare and contrast locusts and grasshoppers>>.

%----------------------------------------------------------------------------------------
\section{DNA Sequencing History} %	SECTION 2
%----------------------------------------------------------------------------------------

Soon after it was realized that DNA is the source of genetic information in all living organisms \citep{Watson1953a}, and the "pretty" and "elegant" arrangement of complementary, antiparrallel DNA strands was known \citep{Watson2012a}, the ability to determine the specific arrangement, or "sequence", of nucleotide bases in a given length of DNA was seen as a critical missing piece of technology. It took 25 years after the nature of DNA's architecture to be able to determine the specific arrangement of nucleotides in the polymer\textemdash to sequence it. By 1977, two completely different methods developed by Sanger \citep{Sanger1975a,Sanger1977b} and Maxam-Gilbert \citep{Maxam1977a} were reported. These sequencing technologies, from then on referred to eponymously as ‘Sanger’ or ‘Maxam-Gilbert’ sequencing, were used to determine the specific order of a small piece of DNA (200–300 nt). Sanger sequencing soon dominated most sequencing reactions, likely due to the conceptually more intuitive nature of the technology, and over the past 35 years, DNA sequences have been slowly cloned, sequenced, analyzed, and dutifully cataloged into knowledge.

During the late 1970’s and throughout the 1980’s, DNA sequences were typically communicated in important publications \citep{Cordell1980a,Sanger1978a}. The birth of the Internet in the 1990’s made essential publically-funded repositories for sequence information easily available \citep{Benson2011a}. However, it was the human genome project \citep{Lander2011a,Venter2001}, that provided the important activation energy that brought DNA sequencing from a hard-to-perform, but necessary, analysis, to an organized large-scale effort of assembling the complete genetic material complex genomes. An often criticized, but undeniably disrupting force in the human genome project was the competing efforts of the privately-owned company Celera \citep{Venter2008a}. Taking a higher-throughput and centralized approach to determining the sequence of the human genome, Celera fundamentally changed the landscape of genome assembly. Instead of assigning specific sections of the genome to be worked out by individual labs, Celera parallelized the effort, by collecting many of the best “high-throughput” Sanger-sequencing devices from Agilent (ABI 3700 DNA Analyzer). Using "shotgun" approach \citep{Staden1979}, sequenced pairwise \citep{Roach1995}, and combined with sequence scaffolds made available by the publicly-funded project, Celera was able to assemble high-quality genomic sequences very quickly. Arguably, this was the first deep sequencing effort, and changed the landscape of molecular and biochemical research, coincident with the beginning of a new millennium.

%----------------------------------------------------------------------------------------
\section{History of High-throughput Sequencing}
%----------------------------------------------------------------------------------------

Sequencing DNA by Sanger’s technology remains a valuable and critical tool in every biological scientist’s arsenal. However, the technology has a practical throughput limit. Each DNA molecule to be sequenced must be isolated and clonally amplified, typically using bacteria. Given that the human genome \citep{Hattori2005a} comprises > 3 billion nt (on just one strand), and that each Sanger reaction will provide ~800nt of quality sequence, we need at least ~4 million individual reactions to determine the sequence of the human genome, assuming that all of our reads are of sufficient quality, length, and do not overlap by even 1 nt. Even the best practical improvements to work-flows could not bring the Sanger approach to DNA sequencing in-line with aspirations of analyzing genomes of many different species or individual organisms.

In the early 2000’s, efforts to change the approach to DNA sequencing, first using MPSS \citep{Brenner2000a}, but perhaps more importantly, by Pyrosequencing \citep{Ronaghi1998a} and Polony sequencing \citep{Shendure2005}. Both of the latter methods utilize emulsion PCR \citep{Nakano2003a} for clonal amplification prior to sequencing, removing the bottleneck of bacterial cloning. In contrast to Sanger sequencing, where the signal is from fluorescence of the last incorporated chain-terminating nucleotide, Pyrosequencing visualizes light given off by luciferase as it reacts with ATP generated from the pyrophosphate (PPi) by-product of nucleotide addition. Pyrosequencing has been commercialized by 454 technologies. Polony sequencing involves a more complicated sequencing-by-ligation method, eventually commercialized by Applied Biosystems and branded as SOLiD sequencing. While both of these technologies provided valuable, high-throughput sequences, neither has been as successful as the approach commercialized by Solexa, eventually purchased and now known as Illumina.
Illumina uses a sequencing-by-synthesis approach using fluorescent nucleotides after clonal amplification of DNA on a slide surface \citep{Bentley2008}. Since 2006, iterations of the Illumina platform (eg. GE, GE-II(x), Hi-Seq, Hi-Seq 2500) have demonstrated a steady and impressive increases in both read depth and length. On February 15th 2012, Illumina announced on its \href{http://blog.basespace.illumina.com/}{Basespace blog}, that they had sequenced a HapMap sample at 40X coverage, using the HiSeq 2500 platform and paired-end 100 nt reads in a single run. This announcement demonstrated that in a single analysis attempt (but certainly not the day claimed by the title), analysis and assembly of a human genome is no longer the monumental endeavor it once was, and that completely new experimental possibilities are a reality for life science research.

\InsertFigure
	{Sequencing_costs_over_time.pdf}
	{Cost of sequencing the human genome over time}
	{
		The costs of sequencing the human genome has decreased on a log scale over a roughly 10 year period thanks 
		to major improvements in high-throughput sequencing. Data from Wetterstrand KA. DNA Sequencing Costs: 
		Data from the NHGRI Genome Sequencing Program (GSP) Available at: \url{www.genome.gov/sequencingcosts}.
		Accessed 2013-09-03).
	}
	{fig:SeqCosts}

%----------------------------------------------------------------------------------------
\section{Deep-sequencing RNA methodologies}
%----------------------------------------------------------------------------------------

%Just found the mother load of literature reviews http://blog.sbgenomics.com/history-of-rna-seq/ 

%Also this appears to be a recent (but not great) review about transcriptome analysis using RNA-Seq (Mutz et al. 2013)

%<History> - start with a simple history of splicing, mention 

The first widely-accepted method for measuring gene expression via sequencing by proxy of cDNA molecules was Serial Analysis of Gene Expression (SAGE) \citep{Velculescu1995a}. While the importance of microarrays in the measurement of gene expression via cannot be overstated \citep{Shendure2008,Marioni2008} the technologies limited ability to investigate novel sequences, and analogue signal, makes their relevance to this section somewhat off-topic. However, \textbf{SAGE}, (similar to the previously discussed MPSS technique) produces a digital output of gene expression using a cleaver procedure of cleaving cDNA molecules using restriction endonucleases that leaves a ‘sticky end’. After cleavage, these molecules are ligated and concatenated together to form longer DNA fragments. Fragments are cloned into a vector, amplified, and Sanger sequenced. Using known sequences incorporated during concatenation, the number of sequenced 'fragments' that align to a given gene is related to the abundance of the original mRNA molecule. While SAGE was a cleaver molecular trick allowing researches to dip into the 5-log range of expression typically seen in mRNA expression, it is still limited by read lengths and practical read depth of Sanger sequencing. 
Not long after the Solexa/Illumina platform produced read lengths of sufficient length of depth to consider measuring gene expression were the first RNA-Seq papers published \citep{Mortazavi2008, Nagalakshmi2008,Lister2008}. These papers gave a powerful glimpse into the future of molecular biology. Indeed, in the years since, analysis by RNA-Seq has quickly overtaken other forms of gene expression analysis, as demonstrated by the number of accessions deposited in GEO per year \citep{Barrett2013}. RNA-Seq allows for digital quantification of RNA expression across physiologically-relevant ranges \citep{Blencowe2009}. While simultaneously measuring gene expression, the data can be used for novel sequence discovery, measuring RNA-editing \citep{Li2011}, transcript assembly \citep{Trapnell2010}. By modifying the basic protocol or performing additional biochemical steps, RNA-Seq can be used to investigate many aspects of RNA biology (see \ref{fig:htsMethods}). 


\InsertFigure
	{RNA_Sequencing_methodologies.pdf}
	{Methods for High-throughput sequencing of RNA}
	{In the short years since the first report of RNA-Seq, many variations have been reported. The figure 
	above provides an incomplete graphical illustration of some of these variations. A more complete list 
	of *Seq applications is maintained on this \href{http://liorpachter.wordpress.com/seq/}{blog}.}
	{fig:htsMethods}

RNA processing begins the moment the nascent RNA is exposed from the polymerase exit channel. Many methodologies have been developed that enrich RNA-Seq libraries for RNA molecules. For example, measurement of nascently transcribed RNA can be performed via GRO-Seq \citep{Core2008a}. Measuring the extremely complicated process of RNA turnover (referring to the rates at which RNAs both are produced and degraded) \citep{Ghosh2010a}, can be done using XXX-Seq after incorporation of XX nucleotides or a biochemical handle such as biotin. RNA:protein interactions can be measured with or without crosslinking the protein to the RNA, via CLIP or RIP, respectively. Once an RNA has been fully transcribed, known processing steps such as Cap formation and poly(A) tail formation can be measured using any of the Cap-Seq/CAGE methodologies (Shiraki et al. 2003), or PAS-Seq (Shepard et al. 2011). With appropriate size-selection steps, small RNAs (Ghildiyal et al. 2008) can also be captured into a sequencing library. Finally, traditional RNA-Seq, can effectively capture fragments of all of the above mentioned libraries, even though it is mainly associated with measurement or analysis of traditional mRNAs.

%<Need to add PAL-Seq to the figure below> See Paper Phil sent out from the Burge lab.
%<Also we have TAIL-Seq from the Kim Lab> 
%Chang, Hyeshik, Jaechul Lim, Minju Ha, and V. Narry Kim. 2014. “TAIL-Seq: Genome-Wide Determination of Poly(A) Tail Length and 3′ End Modifications.” Molecular Cell (February): 1–9. doi:10.1016/j.molcel.2014.02.007. http://linkinghub.elsevier.com/retrieve/pii/S109727651400121X.
%Subtelny, Alexander O., Stephen W. Eichhorn, Grace R. Chen, Hazel Sive, and David P. Bartel. 2014. “Poly(A)-Tail Profiling Reveals an Embryonic Switch in Translational Control.” Nature (January 29). doi:10.1038/nature13007. http://www.nature.com/doifinder/10.1038/nature13007.

%----------------------------------------------------------------------------------------
\bibliography{Bibliography_2} 