%----------------------------------------------------------------------------------------
%	Definitions
%----------------------------------------------------------------------------------------
\clearpage % Start a new page
\lhead{\emph{Definitions}}
\label{hd:Definitions} \listDefinitions

\begin{description}
  \item[RNA-Seq] \hfill \\
  A technology wherein RNA is fragmented, converted to DNA, and analyzed on a high-throughput sequencing instrument
  \item[A ‘Read’] \hfill \\
  The sequence of nucleotides produced from each spot on a high-throughput sequencing machine
  \item[Insert] \hfill \\
  The RNA molecule captured between two cloning sequences in a high-throughput sequencing library preparation workflow
    \item[Read length] \hfill \\
  The number of nucleotides for each given 'read'
    \item[Read depth] \hfill \\
  The number of reads obtained from each high-throughput sequencing analysis
    \item[Coverage] \hfill \\
  A measure of the number of times each nt of a genome is sequenced. E.g. 100 million reads of a 10 million nt genome = 10X coverage, assuming uniform distribution of the 'reads'
    \item[Paired-end] \hfill \\oach1995a
  When both sides of a DNA insert or template are sequenced, utilizing the original length of DNA between the reads to facilitate mapping (\cite{Roach1995}).
    \item[Scaffold or contig] \hfill \\
  A draft sequence of nucleotides, meant to represent the actual biological sequence as closely as possible, examples include unassembled fragments of chromosomes or fragments of mRNA transcripts.
    \item[Argonaute] \hfill \\
  Protein(s) belonging to a group containing a Piwi (P-element induced wimpy testes) domain, that bind nucleic acids and participate in many target-guided processes, including RNA Interference, and RNA-indicuded transcript/gene silencing.   
\end{description}
\clearpage % Start a new page